# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

BioFire Diagnostics, LLC.

# C Proprietary and Established Names

BIOFIRE SPOTFIRE Respiratory/Sore Throat (R/ST) Panel

# D Regulatory Information

<table><tr><td colspan="1" rowspan="1">ProductCode(s)</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">RegulationSection</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">QOF</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OEM</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OTG</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OZE</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OZX</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panel</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">multiplex nucleic acidassay</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">OZY</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OZZ</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">NSU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570 -Instrumentation forclinical multiplex testsystems</td><td colspan="1" rowspan="1">CH - Clinical Chemistry</td></tr><tr><td colspan="1" rowspan="1">PGX</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.2680 -Streptococcus spp. nucleicacid-based assay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr></table>

Note: The primary regulation for device classification is 21 CFR 866.3981 (Product Code QOF).   
Other regulations and associated product codes are cited for tracking purposes.

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain a substantial equivalence determination for the BIOFIRE SPOTFIRE Respiratory/Sore Throat (R/ST) Panel performed on the BIOFIRE SPOTFIRE System.

# B Measurand:

The BIOFIRE SPOTFIRE Respiratory/Sore Throat (R/ST) Panel detects and identifies nucleic acids from the following pathogens: Adenovirus, Seasonal Coronavirus (229E, HKU1, OC43 and NL63, not differentiated), Severe Acute Respiratory Syndrome (SARS)-Coronavirus-2, Human Metapneumovirus, Human Rhinovirus/Enterovirus (not differentiated), Influenza A virus with subtyping of A/H1-2009 and A/H3 (reported separately), Influenza B virus, Parainfluenza Virus 1-4 (not differentiated), Respiratory Syncytial Virus, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, Mycoplasma pneumoniae, Streptococcus dysgalactiae (Group C/G Streptococcus) and Streptococcus pyogenes (Group A Streptococcus).

# C Type of Test:

Multiplex nucleic acid assay for use with the BIOFIRE SPOTFIRE System for the qualitative detection and identification of viral and/or bacterial pathogens in nasopharyngeal swabs from patients with signs and symptoms of respiratory tract infection (BIOFIRE SPOTFIRE R Panel) or in throat swabs from patients with signs and symptoms of pharyngitis (BIOFIRE SPOTFIRE ST Panel).

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The BIOFIRE SPOTFIRE Respiratory/Sore Throat (R/ST) Panel is a multiplexed polymerase chain reaction (PCR) test intended for use with the BIOFIRE SPOTFIRE System for the simultaneous, qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swab (NPS) specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19; (Respiratory menu) or in throat swab (TS) specimens from individuals with signs and symptoms of pharyngitis (Sore Throat menu).

The following organism types and subtypes are identified and differentiated using the SPOTFIRE R/ST Panel:

<table><tr><td rowspan=1 colspan=1>Respiratory Menu</td><td rowspan=1 colspan=1>Sore Throat Menu</td></tr><tr><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2</td><td rowspan=1 colspan=1>Coronavirus (seasonal)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>Human metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>Human rhinovirus/enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human rhinovirus/enterovirus</td><td rowspan=1 colspan=1>Influenza A virus</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus</td><td rowspan=1 colspan=1>Influenza A virus/H1-2009</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus/H1-2009</td><td rowspan=1 colspan=1>Influenza A virus/H3</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus/H3</td><td rowspan=1 colspan=1>Influenza B virus</td></tr><tr><td rowspan=1 colspan=1>Influenza B virus</td><td rowspan=1 colspan=1>Parainfluenza virus</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus</td><td rowspan=1 colspan=1>Respiratory syncytial virus</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus</td><td></td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Streptococcus dysgalactiae (Group C/G Strep)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Streptococcus pyogenes (Group A Strep)</td></tr></table>

Nucleic acids from the viral and bacterial organisms identified by this test are generally detectable in NPS/TS specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of respiratory infection and/or pharyngitis are indicative of the presence of the identified microorganism and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness and/or pharyngitis may be due to infection with pathogens that are not detected by this test, or a respiratory tract infection that may not be detected by an NPS or TS specimen. Positive results do not rule out coinfection with other

organisms. The agent(s) detected by the SPOTFIRE R/ST Panel may not be the definite cause of disease.

Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection and/or pharyngitis.

C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

D Special Instrument Requirements:

BIOFIRE SPOTFIRE System

# IV Device/System Characteristics:

# A Device Description:

The BIOFIRE SPOTFIRE Respiratory/Sore Throat (R/ST) Panel is a multiplexed nucleic acidamplification-based test that is intended to detect and identify multiple respiratory viral and bacterial pathogens in nasopharyngeal swabs (NPS) and throat swabs (TS) from individuals with signs and symptoms of respiratory tract infection or pharyngitis, respectively. The BIOFIRE SPOTFIRE R/ST Panel is performed using the BIOFIRE SPOTFIRE System, which is an automated polymerase chain reaction (PCR)-based in vitro diagnostic instrument system for use with reagent pouches for specific indications.

The BIOFIRE SPOTFIRE System automates nucleic acid extraction and nested multiplex PCR in closed system. The resulting PCR products are evaluated using assay-specific DNA melting analysis. The BIOFIRE SPOTFIRE Systems Software executes the BIOFIRE SPOTFIRE R/ST Panel test according to parameters specified in the BIOFIRE SPOTFIRE R/ST Panel Software. The same protocol is used for sample preparation, nucleic acid extraction, reverse transcription (RT)-PCR amplification and analysis for both NPS and TS specimens but with different result reporting according to the specimen type selected. The sample is added to the reagent pouch and the pouch is loaded into the BIOFIRE SPOTFIRE System, after which results are reported in approximately 15 minutes without further user intervention.

The BIOFIRE SPOTFIRE System is comprised of between one and four modules that are connected to a single SPOTFIRE Control Station equipped with the SPOTFIRE System Software. The first module is placed on top of the Control Station and up to three additional modules may be stacked on top as required. Each module can be accessed at random to perform a test, independent of the other modules attached to the same Control Station.

The BIOFIRE SPOTFIRE R/ST Panel Reagent Kit includes the following components:

BIOFIRE SPOTFIRE R/ST Panel Pouches (30 ea.) o Each packed under vacuum in a metal canister and outer bag   
BIOFIRE Sample Preparation Reagent Kit (SPRK) (32 ea.) o Fixed volume transfer pipette for addition of the test sample to the Sample Injection Vial

o BIOFIRE Sample Buffer ampoule for addition to the Sample Injection Vial o BIOFIRE Sample Injection Vial for mixing of the test sample and Sample Buffer o BIOFIRE Hydration Injection Vial containing Hydration Solution for pouch rehydration.

# B Principle of Operation:

The BIOFIRE SPOTFIRE Panel Pouch is a closed-system disposable that contains all the reagents needed for sample preparation, reverse transcription, PCR amplification and fluorescent detection, including melt curve analysis of amplified products. The rigid plastic “fitment” of the pouch contains reagents in freeze-dried form while the flexible plastic portion of the pouch is divided into discrete “blisters” in which different chemical processes are carried out. After sample collection, the user adds the sample to the Panel Pouch, selects the applicable specimen type from the SPOTFIRE System touchscreen and then places the pouch into the instrument to start the run. All other operations are automated.

The SPOTFIRE Systems Software is designed to run on the SPOTFIRE Control Station that consists of an integrated computer and touchscreen. The SPOTFIRE Systems Software interfaces with the user via the touchscreen to operate the instrument, collect, save, analyze and retrieve data, report test results and view stored information. The BIOFIRE SPOTFIRE R/ST Panel Software works with the Systems Software and specifies the assay-specific parameters to perform the test, interpret the data and display the reported results.

To perform a test with the BIOFIRE SPOTFIRE R/ST Panel, the patient sample is first mixed with a denaturing Sample Buffer to inactivate nucleases. The user then loads the diluted sample into the BIOFIRE SPOTFIRE R/ST Panel Pouch via the injection port. The vacuum in the pouch draws the liquid into the sample well of the pouch fitment and in the process rehydrates the cellbased Process Control. Upon inserting into the instrument, the sample mixture is moved to the lysis blister where mechanical agitation with zirconia-silica beads is used to complete cell lysis. The lysate is then mixed with glass-coated magnetic beads which adsorb the nucleic acids liberated in the lysis step. The extracted nucleic acids are washed to remove potential inhibitors and then eluted into buffer that is used to rehydrate the contents of the first stage PCR blister, which includes all the components needed for reverse transcription (RT) and first stage PCR. RT-PCR is initiated upon achievement of the requisite temperature within the amplification chamber. After completion of first-stage amplification, the products of the reaction are diluted and combined with fresh reagents that are distributed over a second stage PCR array. The second stage primers are internal (“nested”) with respect to the first stage primers and each analyte or control target is represented by three discrete amplification reactions. The second stage PCR array includes an assay control, the PCR2 Control, failure of which indicates a failure in the second stage PCR and invalidates the run.

Included in the PCR master mix for the second stage PCR is a fluorescent dye (LC Green Plus) that is specific for double-stranded DNA. The dye intercalates with the double stranded-PCR products and fluoresces at a specific wavelength that is detected by the BIOFIRE SPOTFIRE System. At the end of the second stage PCR, the temperature of the array is gradually raised at a controlled ramp rate and the fluorescence is monitored. Fluorescence decreases at the melting temperature of the amplicons, and the temperature at which this occurs is characteristic of each amplification product. The Panel Software automatically evaluates the calculated melting temperature (Tm) in comparison to the predefined range for each specific analyte. The results from each of the three amplification reactions for each analyte are combined by the software to determine the final test result. For an assay to be called positive, at least two of the three wells must have a PCR product with a Tm that is within the specified range for that analyte. Assays that do not meet these criteria are reported as negative.

Positive and negative results for the assay controls (Process Control and PCR2 Control) are compared to their expected values and assigned a “pass” or “fail” result. Panel-specific rules define how control failures affect interpretation of test results. For the BIOFIRE SPOTFIRE R/ST Panel, failure of either the RNA Process Control or PCR2 Control invalidates the results for all analytes. The final sample result is determined by the interpretation rules defined in the Panel Software.

For most bacteria and viruses detected by the BIOFIRE SPOTFIRE R/ST Panel, a “positive” result is reported if two or more of the wells corresponding to a single assay contain detectable PCR product of the appropriate Tm. However, to ensure inclusivity, certain analytes are targeted by multiple assays, the results of which are interpreted in combination, as described below in Tables 1 and 2.

Table 1. Interpretation of results for analytes targeted by multiple assays   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Number ofIndependent Assays</td><td rowspan=1 colspan=1>Positive ResultInterpretation</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>≥ 1 assay positive</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>≥ 1 assay positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>≥ 2 assays positive</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>≥ 1 assay positive</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>≥ 1 assay positive</td></tr></table>

1 At least one of the two pan-influenza A assays and either the H1-2009- or the H3-specific assay must produce positive results to report a positive subtype result (Table 2).

Table 2. Possible result permutations for detection of influenza A virus in patient samples

<table><tr><td rowspan=1 colspan=4>Assay</td><td rowspan=2 colspan=1>Result Reported</td><td rowspan=2 colspan=1>ActionRequired</td></tr><tr><td rowspan=1 colspan=1>FluA-pan1</td><td rowspan=1 colspan=1>FluA-pan2</td><td rowspan=1 colspan=1>FluA-H1-2009</td><td rowspan=1 colspan=1>FluA-H3</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=3 colspan=1>POSITIVEInfluenza A Virus Subtype H1-2009</td><td rowspan=3 colspan=1>Report</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=3 colspan=1>POSITIVEInfluenza A Virus Subtype H3</td><td rowspan=3 colspan=1>Report</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=3 colspan=1>POSITIVEInfluenza A Virus Multiple Subtypes</td><td rowspan=3 colspan=1>Report</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>POSITIVEInfluenza A Virus No Subtype Identified</td><td rowspan=6 colspan=1>Retest 1</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=5 colspan=1>UNCERTAINInfluenza A Virus</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NEGATIVE</td><td rowspan=1 colspan=1>Report</td></tr></table>

1 If the retest result is the same, the user is instructed to contact public health authorities for confirmatory testing.

# C Instrument Description Information:

1. Instrument Name: BIOFIRE SPOTFIRE System

2. Specimen Identification: Specimen identification can be entered manually or by scanning a barcode.

3. Specimen Sampling and Handling:

Use of the BIOFIRE SPOTFIRE R/ST Panel requires collection of either a nasopharyngeal swab (NPS) or throat swab (TS) according to standard procedures. Nasopharyngeal swabs should be placed in $1 { \cdot } 3 ~ \mathrm { m L }$ of viral transport medium, while throat swabs should be placed in $1 ~ \mathrm { m L }$ liquid Amies Medium. The minimum sample volume required to perform a test is 300 $\mu \mathrm { L }$ . Specimens should be tested as soon as possible following collection but may be stored for up to 4 hours at room temperature $1 5 { - } 2 5 ^ { \circ } \mathrm { C } )$ or for up to 3 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or up to 30 days at $\leq - 1 5 ~ ^ { \circ } \mathrm { C }$ .

The SPOTFIRE System Software includes step-by-step on-screen instructions that guide the user through the process of starting a run on the instrument. After cleaning the work area and Pouch Loading Station, the user removes a BIOFIRE SPOTFIRE R /ST Panel pouch from its vacuum packaging and places it into the Pouch Loading Station. They then hydrate the pouch using the Hydration Injection Vial by injecting the contents through the Hydration Solution Injection Port and transfer a fixed volume of sample to the Sample Injection Vial, together with the entire contents of the Sample Buffer ampoule. After mixing the Sample Injection Vial by inversion, the user injects the mixture into the pouch via the Sample Injection Port, enters the sample identity into the instrument (manually or using the barcode scanner), selects the applicable specimen type (NPS or TS) and initiates the run by inserting the pouch into the instrument. The test will then proceed to completion, including interpretation of results, without further user intervention.

4. Calibration:

The BIOFIRE SPOTFIRE System is factory calibrated. User calibration is not required.

5. Quality Control:

Internal Controls Two process controls are included in each BIOFIRE SPOTFIRE R/ST Panel pouch:

1) RNA Process Control

The RNA Process Control comprises freeze-dried Schizosaccharomyces pombe which is rehydrated when the sample is loaded. The control material is carried through all stages of the test process, including nucleic acid extraction and purification, reverse transcription, first and second stage PCR amplification and DNA melt analysis. A positive result indicates that all process steps carried out in the BIOFIRE SPOTFIRE R/ST Panel pouch were completed successfully.

# 2) PCR2 Control

The PCR2 control comprises a DNA target sequence that is dried into specific wells used for the second stage amplification and melting curve analysis. A positive result indicates that these processes were successful.

A positive result is required for both the RNA Process Control and the PCR2 Control for the results for an individual pouch to be considered valid. If either control fails, the sample should be retested using a new pouch.

External Controls

BioFire Diagnostics recommends routine testing of External Positive and Negative Controls that are commercially available from bioMérieux or Maine Molecular Quality Controls, Inc.:

• SPOTFIRE RSP Positive Control (M42638) • SPOTFIRE RSP Negative Control (M42738)

These controls are designed to monitor the performance of each of the individual assays within the BIOFIRE SPOTFIRE R/ST Panel and should be tested as Quality Controls in accordance with their FDA-cleared Instructions For Use (K233611) and the labeling for the BIOFIRE SPOTFIRE R/ST Panel.

In general, External Controls should be tested when training a new operator, or whenever a new shipment of BIOFIRE SPOTFIRE Panels or a new SPOTFIRE Control Station is received. Additional Negative Controls should also be run at least monthly in each testing location to monitor for environmental contamination.

A total of 351 External Positive and Negative Controls were tested during the Prospective Clinical Study described in Section VII (C). Of these, 11 were associated with instrument errors or internal control failures and 24 were tested using an incorrect or expired reagent pouch. As a result, there were 316 External Controls with valid results (141 Positive and 175 Negative), including 4 $( 1 . 3 \% )$ that were incorrectly designated by the operator as patient samples, rather than control materials. Of the 141 valid External Positive Controls, $9 8 . 6 \%$ (139/141) returned the expected results for all analytes as did all 175 $( 1 0 0 \% )$ of the valid Negative External Controls.

V Substantial Equivalence Information:

A Predicate Device Name(s): BIOFIRE SPOTFIRE Respiratory (R) Panel   
B Predicate 510(k) Number(s): K213954

# C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; Predicate Device(s):</td><td colspan="1" rowspan="1">K232954</td><td colspan="1" rowspan="1">K213954</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">BIOFIRE SPOTFIRERespiratory/Sore Throat (R/ST)Panel</td><td colspan="1" rowspan="1">BIOFIRE SPOTFIRE Respiratory(R) Panel</td></tr><tr><td colspan="1" rowspan="1">General Device CharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indications For Use</td><td colspan="1" rowspan="1">The BIOFIRE SPOTFIRERespiratory/Sore Throat (R/ST)Panel is a multiplexed polymerasechain reaction (PCR) test intendedfor use with the BIOFIRESPOTFIRE System for thesimultaneous, qualitative detectionand identification of multiplerespiratory viral and bacterialnucleic acids in nasopharyngeal</td><td colspan="1" rowspan="1">The BIOFIRE SPOTFIRERespiratory (R) Panel (SPOTFIRER Panel) is a multiplexedpolymerase chain reaction (PCR)test intended for use with theBIOFIRE SPOTFIRE System forthe simultaneous, qualitativedetection and identification ofmultiple respiratory viral andbacterial nucleic acids in</td></tr><tr><td colspan="3"></td></tr><tr><td>Device &amp; Predicate Device(s):</td><td>K232954</td><td>K213954</td></tr><tr><td></td><td>swab (NPS) specimens obtained from individuals with signs and</td><td>nasopharyngeal swab (NPS)</td></tr><tr><td></td><td></td><td>specimens obtained from</td></tr><tr><td></td><td>symptoms of respiratory tract</td><td>individuals with signs and</td></tr><tr><td></td><td>infection, including COVID-19;</td><td>symptoms of respiratory tract</td></tr><tr><td></td><td>(Respiratory menu) or in throat</td><td>infection, including COVID-19.</td></tr><tr><td></td><td>swab (TS) specimens from</td><td></td></tr><tr><td></td><td>individuals with signs and</td><td>The following organism types and</td></tr><tr><td></td><td>symptoms of pharyngitis (Sore</td><td>subtypes are identified and</td></tr><tr><td>Throat menu).</td><td></td><td>differentiated using the SPOTFIRE</td></tr><tr><td></td><td>The following organism types and</td><td>R Panel:</td></tr><tr><td></td><td>subtypes are identified and</td><td>Viruses:</td></tr><tr><td></td><td>differentiated using the SPOTFIRE</td><td>Adenovirus</td></tr><tr><td>R/ST Panel:</td><td></td><td>Coronavirus (seasonal)</td></tr><tr><td></td><td></td><td>Coronavirus SARS-CoV-2</td></tr><tr><td></td><td>Respiratory Menu</td><td>Human metapneumovirus</td></tr><tr><td></td><td>Viruses:</td><td>Human rhinovirus/enterovirus</td></tr><tr><td></td><td>Adenovirus</td><td>Influenza A virus</td></tr><tr><td></td><td>Coronavirus SARS-CoV-2</td><td>Influenza A virus A/H1-2009</td></tr><tr><td></td><td>Coronavirus (seasonal)</td><td>Influenza A virus A/H3</td></tr><tr><td></td><td>Human metapneumovirus</td><td>Influenza B virus</td></tr><tr><td></td><td>Human rhinovirus/enterovirus</td><td>Parainfluenza virus</td></tr><tr><td></td><td>Influenza A virus</td><td>Respiratory syncytial virus</td></tr><tr><td></td><td>Influenza A virus/H1-2009</td><td>Bacteria:</td></tr><tr><td></td><td>Influenza A virus/H3</td><td>Bordetella parapertussis</td></tr><tr><td></td><td>Influenza B virus</td><td>Bordetella pertussis</td></tr><tr><td></td><td>Parainfluenza virus</td><td>Chlamydia pneumoniae</td></tr><tr><td></td><td>Respiratory syncytial virus</td><td>Mycoplasma pneumoniae</td></tr><tr><td></td><td>Bacteria:</td><td>Nucleic acids from the viral and</td></tr><tr><td></td><td>Bordetella parapertussis</td><td>bacterial organisms identified by</td></tr><tr><td></td><td>Bordetella pertussis</td><td>this test are generally detectable in</td></tr><tr><td></td><td>Chlamydia pneumoniae</td><td>NPS specimens during the acute</td></tr><tr><td></td><td>Mycoplasma pneumoniae</td><td>phase of infection. The detection</td></tr><tr><td></td><td>Sore Throat Menu</td><td>and identification of specific viral</td></tr><tr><td></td><td>Viruses:</td><td>and bacterial nucleic acids from</td></tr><tr><td></td><td>Adenovirus</td><td>individuals exhibiting signs and/or symptoms of respiratory infection</td></tr><tr><td></td><td>Coronavirus (seasonal)</td><td>are indicative of the presence of the</td></tr><tr><td></td><td>Human metapneumovirus</td><td>identified microorganism and aids</td></tr><tr><td></td><td>Human rhinovirus/enterovirus</td><td>in diagnosis if used in conjunction</td></tr><tr><td></td><td>Influenza A virus</td><td>with other clinical and</td></tr><tr><td></td><td>Influenza A virus/H1-2009</td><td>epidemiological information, and</td></tr><tr><td></td><td>Influenza A virus/H3</td><td>laboratory findings. The results of</td></tr><tr><td></td><td>Influenza B virus</td><td>this test should not be used as the</td></tr><tr><td></td><td>Parainfluenza virus</td><td>sole basis for diagnosis, treatment,</td></tr><tr><td></td><td>Respiratory syncytial virus</td><td>or other patient management</td></tr><tr><td></td><td>Bacteria:</td><td>decisions.</td></tr><tr><td></td><td>Chlamydia pneumoniae</td><td>Negative results in the setting of a</td></tr><tr><td></td><td>Mycoplasma pneumoniae</td><td>respiratory illness may be due to</td></tr><tr><td></td><td>Streptococcus dysgalactiae</td><td>infection with pathogens that are</td></tr><tr><td></td><td>(Group C/G Strep)</td><td>not detected by this test, or lower</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>Streptococcus pyogenes</td><td>respiratory tract infection that may</td></tr><tr><td></td><td>(Group A Strep)</td><td>not be detected by an NPS</td></tr><tr><td></td><td>Nucleic acids from the viral and</td><td>specimen. Positive results do not</td></tr><tr><td></td><td>bacterial organisms identified by</td><td>rule out coinfection with other</td></tr><tr><td></td><td>this test are generally detectable in</td><td>organisms. The agent(s) detected</td></tr><tr><td colspan="1" rowspan="1">Device &amp; Predicate Device(s):</td><td colspan="1" rowspan="1">K232954</td><td colspan="1" rowspan="1">K213954</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NPS/TS specimens during the acutephase of infection. The detectionand identification of specific viraland bacterial nucleic acids fromindividuals exhibiting signs andsymptoms of respiratory infectionand/or pharyngitis are indicative ofthe presence of the identifiedmicroorganism and aids indiagnosis if used in conjunctionwith other clinical andepidemiological information, andlaboratory findings. The results ofthis test should not be used as thesole basis for diagnosis, treatment,or other patient managementdecisions.Negative results in the setting of arespiratory illness and/orpharyngitis may be due to infectionwith pathogens that are notdetected by this test, or arespiratory tract infection that maynot be detected by an NPS or TSspecimen. Positive results do notrule out coinfection with otherorganisms. The agent(s) detectedby the SPOTFIRE R/ST Panel maynot be the definite cause of disease.Additional laboratory testing (e.g.,bacterial and viral culture,immunofluorescence, andradiography) may be necessarywhen evaluating a patient withpossible respiratory tract infectionand/or pharyngitis.</td><td colspan="1" rowspan="1">by the SPOTFIRE R Panel may notbe the definite cause of disease.Additional laboratory testing (e.g.,bacterial and viral culture,immunofluorescence, andradiography) may be necessarywhen evaluating a patient withpossible respiratory tract infection.</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Highly multiplexed PCR with DNAmelting analysis</td></tr><tr><td colspan="1" rowspan="1">Reagents</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Sealed reagent pouch containing allreagents necessary for sampleprocessing (nucleic acidextraction), reverse transcription(RT)-PCR amplification and meltcurve analysis.</td></tr><tr><td colspan="1" rowspan="1">Internal Controls</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">In-built controls for sampleprocessing/nucleic acid extraction,amplification and detection.</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">BIOFIRE SPOTFIRE System</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">~15 minutes</td></tr><tr><td colspan="1" rowspan="1">General Device CharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Disease Syndrome</td><td colspan="1" rowspan="1">Same plus signs and symptoms of</td><td colspan="1" rowspan="1">Signs and symptoms of respiratory</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">pharyngitis</td><td colspan="1" rowspan="1">tract infection</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Same plus throat swabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">Same for Respiratory MenuSore Throat Menu does not reportresults forOCoronavirus SARS-CoV-2Bordetella parapertussisOBordetella pertussisbut reports additional results for:OStreptococcus dysgalactiae(Group C/G Strep)OStreptococcus pyogenes(Group A Strep)</td><td colspan="1" rowspan="1">Viruses:AdenovirusCoronavirus (seasonal)Coronavirus SARS-CoV-2Human metapneumovirusHuman rhinovirus/enterovirusInfluenza A virusInfluenza A virus A/H1-2009Influenza A virus A/H3Influenza B virusParainfluenza virusRespiratory syncytial virusBacteria:Bordetella parapertussisBordetella pertussisChlamydia pneumoniaeMycoplasma pneumoniae</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# Guidance Documents

• Guidance for Clinical Laboratories, Commercial Manufactures, and FDA Staff – Policy for Coronavirus Disease-2019 Tests During the Public Health Emergency (2020) Guidance for Industry - Part 11, Electronic Records; Electronic Signatures - Scope and Application (August 2003) Guidance for Industry - Computerized Systems Used in Clinical Investigations (May 2007) Guidance for Industry - Oversight of Clinical Investigations - A Risk-Based Approach to Monitoring (August 2013) Guidance for Industry - Electronic Source Data in Clinical Investigations (September 2013) Guidance for IRBs, Clinical Investigators, and Sponsors – Informed Consent Information Sheet (July 2014) Guidance for Industry and FDA Staff - Acceptance of Clinical Data to Support Medical Device Applications and Submissions - Frequently Asked Questions (February 2018) Guidance for Industry and FDA Staff - Recommendations for Clinical Laboratory Improvement Amendments of 1988 (CLIA) Waiver Applications for Manufacturers of In Vitro Diagnostic Devices (February 2020) Guidance for Industry and FDA Staff - Recommendations for Dual 510(k) and CLIA Waiver by Application Studies (February 2020) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 2009) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Nucleic Acid Assays (October 2009) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays (October 2009)

Guidance for Industry and FDA Staff - Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, (March 2007)   
Guidance for Industry and FDA Staff, Evaluation and Reporting of Age-, Race-, and Ethnicity-Specific Data in Medical Device Clinical Studies (September 2017)   
Guidance for Sponsors, Investigators, and IRBs - Impact of Certain Provisions of the Revised Common Rule on FDA-Regulated Clinical Investigations (October 2018)   
Guidance for Sponsors, Institutional Review Boards, and FDA Staff - Guidance on   
Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human   
Specimens that are Not Individually Identifiable (April 2006)   
Guidance for Industry and Food and Drug Administration Staff - Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic   
Devices (August 2014)   
Guidance for Industry and FDA Staff - Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests (March 2007)   
CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved   
Guideline - Second Edition. CLSI document EP12-A2. Wayne, PA: Clinical and   
Laboratory Standards Institute; 2008.   
CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25-A. Wayne, PA: Clinical and Laboratory Standards Institute;   
2009.   
CLSI. Interference Testing in Clinical Chemistry. 3rd ed. CLSI guideline EP07. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.   
ICH E6(R1): Guideline for Good Clinical Practice (June 1996)   
ICH Harmonised Guideline: Integrated Addendum to ICH E6(R1): Guideline for Good Clinical Practice E6(R2) (November 2016)   
GHTF: Clinical Evidence for IVD Medical Devices - Clinical Performance Studies for In Vitro Diagnostic Medical Devices (November 2012)   
WMA Declaration of Helsinki, Ethical Principles for Medical Research Involving Human Subjects   
Guidance for Industry and FDA Staff - Guidance for the Content of Premarket   
Submissions for Software Contained in Medical Devices (May 2005)   
Guidance for Industry and FDA Staff - Off-The-Shelf Software Use in Medical   
Devices (September 2019)   
Final Guidance for Industry and FDA Staff - General Principles of Software Validation (January 2002)   
Guidance forIndustry and FDA Staff - Content of Premarket Submissions for   
Management of Cybersecurity in Medical Devices (October 2014)

# Standards

ISO 14971:2019 Medical devices - Application of risk management to medical devices   
IEC 62366-1:2015 Medical devices - Part 1: Application of usability engineering to medical   
devices   
ISO 62304 Edition 1.1b:2015 / EN 62304:2006 + A1:2015 - Medical device software - software   
lifecycle processes   
ISO 15223-1:2016: Medical Devices - Symbols to be used with medical device labels,   
labeling and information to be supplied - Part 1: General requirements   
ISO 13485:2016/EN ISO 13485:2016; Medical devices - Quality Management System -   
Requirements for regulatory purposes   
ISO 20916:2019; In vitro diagnostic medical devices. Clinical performance studies using

specimens from human subjects. Good study practice EN 13612:2002; Performance evaluation of in vitro diagnostic medical devices EN ISO 18113-1:2011; In vitro diagnostic medical devices - Information supplied by the manufacturer (labeling). Terms, definition and general requirements • EN ISO 18113-2:2011; In vitro diagnostic medical devices - Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use EN ISO 23640:2015; In vitro diagnostic medical devices - Evaluation of stability of in vitro diagnostic reagents EN 13641:2002; Elimination or reduction of risk of infection related to in vitro diagnostics reagents

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

The reproducibility and precision of the BIOFIRE SPOTFIRE R/ST Panel on different BIOFIRE SPOTFIRE Systems was evaluated with multiple operators who tested positive and negative samples over multiple days at three study sites, in addition to BioFire Diagnostics. All analytes detected by the panel were evaluated using contrived positive specimens at concentrations equivalent to $\leq 3 \mathrm { X }$ their respective analytical limit of detection (LoD) (Table 3). Negative samples were prepared using artificial matrix only.

Table 3. Contrived samples used to evaluate the precision/reproducibility of the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Analyte/Strain</td><td colspan="1" rowspan="1">Source Identity</td><td colspan="1" rowspan="1">Concentration(per mL)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="6">2</td><td colspan="1" rowspan="1">Bordetella pertussis A639</td><td colspan="1" rowspan="1">ZeptoMetrix0801459</td><td colspan="1" rowspan="1">990 CFU</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae M129</td><td colspan="1" rowspan="1">ZeptoMetrix0801579</td><td colspan="1" rowspan="1">10 CCU</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Species B Type 3</td><td colspan="1" rowspan="1">ZeptoMetrix0810062CF</td><td colspan="1" rowspan="1">2.4 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">ZeptoMetrix0810228CF</td><td colspan="1" rowspan="1">0.0025 TCID50</td></tr><tr><td colspan="1" rowspan="1">Influenza BB/Florida/02/06</td><td colspan="1" rowspan="1">ZeptoMetrix0810037CF</td><td colspan="1" rowspan="1">0.099 TCID50</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4a</td><td colspan="1" rowspan="1">ZeptoMetrix0810060CF</td><td colspan="1" rowspan="1">200 TCID50</td></tr><tr><td colspan="1" rowspan="4">3</td><td colspan="1" rowspan="1">Bordetella parapertussis E595</td><td colspan="1" rowspan="1">ZeptoMetrix0801462</td><td colspan="1" rowspan="1">120 CFU</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae AR-39</td><td colspan="1" rowspan="1">ATCC53592</td><td colspan="1" rowspan="1">20 IFU</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus 3 Type B1</td><td colspan="1" rowspan="1">ZeptoMetrix0810156CF</td><td colspan="1" rowspan="1">0.75 TCID50</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">ZeptoMetrix0810014CF</td><td colspan="1" rowspan="1">4.6 TCID50</td></tr><tr><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="1">Influenza A H1N1pdm Michigan/45/15</td><td colspan="1" rowspan="1">ZeptoMetrix0810538CF</td><td colspan="1" rowspan="1">2.5 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus SARS-CoV-22019-nCoV/USA-WA1/2020</td><td colspan="1" rowspan="1">ATCC VR-1986HK</td><td colspan="1" rowspan="1">250 copies</td></tr><tr><td colspan="1" rowspan="5">4</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae subsp. equisimilis X068</td><td colspan="1" rowspan="1">ZeptoMetrix0810516</td><td colspan="1" rowspan="1">330 CFU</td></tr><tr><td colspan="1" rowspan="1">Human enterovirus D68US/MO/14-18947</td><td colspan="1" rowspan="1">ATCCVR-1823</td><td colspan="1" rowspan="1">11 TCID50</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">ZeptoMetrix0810015CF</td><td colspan="1" rowspan="1">42 TCID50</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus A 2006</td><td colspan="1" rowspan="1">ZeptoMetrix0810040ACF</td><td colspan="1" rowspan="1">0.062 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">ATCCVR-740</td><td colspan="1" rowspan="1">2.0 TCID50</td></tr><tr><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">ZeptoMetrix0810016CF</td><td colspan="1" rowspan="1">8.8 TCID50</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2Hong Kong/4801/14</td><td colspan="1" rowspan="1">ZeptoMetrix0810526CF</td><td colspan="1" rowspan="1">2.6 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">ZeptoMetrix0810024CF</td><td colspan="1" rowspan="1">1.6 x 10-2 TCID50</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenesTyping strain T1</td><td colspan="1" rowspan="1">ATCC 12344</td><td colspan="1" rowspan="1">1,400 cells</td></tr></table>

CCU: Color Changing Units; CFU: Colony Forming Units; IFU: Inclusion Forming Units; $\mathrm { T C I D } _ { 5 0 }$ : Tissue Culture Infectious Dose- $50 \%$ b

At BioFire Diagnostics, testing was performed on three BIOFIRE SPOTFIRE Systems, using three lots of reagents over five consecutive days. On each day of testing, two operators each tested three replicates of each sample (one per reagent lot) on each BIOFIRE SPOTFIRE System for a total of 90 replicates per sample overall (2 operators x 3 samples x 3 instruments x 5 days $= 9 0$ replicates).

At the external study sites, testing was performed with a single BIOFIRE SPOTFIRE System at each site and occurred over five non-consecutive days using a single lot of reagents that was common across each of the sites. On each day, two operators each tested two replicates of each sample for a total of 60 replicates per sample overall (2 operators $_ { \textrm { X 2 } }$ samples x 3 sites x 5 days $= 6 0$ replicates).

For each analyte there were 150 positive and 600 negative samples. Samples that produced an invalid result on initial testing due to process control failure $( \mathtt { n } = 1 2 )$ ) or instrument error (n $= 2$ ) were retested. Precision and reproducibility for each analyte were calculated by assessing the agreement between the reported test results and the expected results for each sample.

A summary of the results of the study, stratified by analyte and BIOFIRE SPOTFIRE System/site is provided in Table 4. Positive agreement for all analytes was $> 9 5 \%$ and there was negligible difference between systems, sites, reagent lots or operators. Negative agreement was $100 \%$ in all cases.

Overall, the reproducibility and precision of the BIOFIRE SPOTFIRE R/ST Panel for detection of the target analytes was determined to be acceptable.

Table 4. Summary of results from the Precision and Reproducibility Study with the BIOFIRE SPOTFIRE R/ST Panel, stratified by BIOFIRE SPOTFIRE System and study site   

<table><tr><td colspan="2" rowspan="3">Assay/Analyte</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="7" rowspan="1">Agreement (%) Stratified by Site and BIOFIRE SPOTFIRE System</td></tr><tr><td colspan="3" rowspan="1">BioFire Diagnostics</td><td colspan="3" rowspan="1">External Site</td><td colspan="1" rowspan="2">Total</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="2" rowspan="4">AdenovirusAdenovirusSpecies B Type 3</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="2" rowspan="1">80/80   80/80(100)    (100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="1" rowspan="8">Coronavirus(seasonal)</td><td colspan="1" rowspan="2">Coronavirus229E</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">29/30(96.7)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">149/150(99.3)</td></tr><tr><td colspan="3" rowspan="1">89/90 (98.9)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">CoronavirusOC43</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">CoronavirusNL63</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">27/30(90.0)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">18/20(90.0)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">18/20(90.0)</td><td colspan="1" rowspan="2">143/150(955.3)</td></tr><tr><td colspan="3" rowspan="1">87/90 (96.7)</td><td colspan="3" rowspan="1">56/60 (93.3)</td></tr><tr><td colspan="1" rowspan="2">None</td><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">60/60(100)</td><td colspan="1" rowspan="1">60/60(100)</td><td colspan="1" rowspan="1">60/60(100)</td><td colspan="1" rowspan="1">40/40(100)</td><td colspan="1" rowspan="1">40/40(100)</td><td colspan="1" rowspan="1">40/40(100)</td><td colspan="1" rowspan="2">300/300(100)</td></tr><tr><td colspan="3" rowspan="1">180/180 (100)</td><td colspan="3" rowspan="1">120/120 (100)</td></tr><tr><td colspan="2" rowspan="4">Coronavirus SARS-CoV-2SARS-CoV-2 2019-nCoV/USA-WA1/2020</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="2" rowspan="4">Human MetapneumovirushMPV 3 Type B1</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">19/20(95.0)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">19/20(95.0)</td><td colspan="1" rowspan="2">148/150(998.7)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">58/60 (96.7)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="2" rowspan="4">Human Rhinovirus/EnterovirusEnterovirus D68US/MO/14-18947</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="1" rowspan="6">Influenza AVirus</td><td colspan="1" rowspan="2">Influenza AH1N1pdm</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Influenza AH3N2</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">19/20 3(95.0)</td><td colspan="1" rowspan="1">19/20(95.0)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">148/15098.7)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">58/60 (96.7)</td></tr><tr><td colspan="1" rowspan="2">None</td><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">90/90(100)</td><td colspan="1" rowspan="1">90/90(100)</td><td colspan="1" rowspan="1">90/90(100)</td><td colspan="1" rowspan="1">60/60(100)</td><td colspan="1" rowspan="1">60/60(100)</td><td colspan="1" rowspan="1">60/60(100)</td><td colspan="1" rowspan="2">450/450(100)</td></tr><tr><td colspan="3" rowspan="1">270/270 (100)</td><td colspan="3" rowspan="1">180/180 (100)</td></tr><tr><td colspan="2" rowspan="4">Influenza A VirusSubtype H1-2009Influenza A H1N1pdmMichigan/45/15</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(10)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="2" rowspan="4">Influenza A VirusSubtype H32Influenza A H3N2Hong Kong/4801/14</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100))</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">19/20 3(95.0)</td><td colspan="1" rowspan="1">19/20(95.0)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">148/150(98.7)</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">58/60 (96.7)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td><td colspan="1" rowspan="1">(100)</td></tr><tr><td colspan="2" rowspan="4">Influenza B VirusInfluenza BB/Florida/02/06</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">29/30(96.7)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100))</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(10)</td><td colspan="1" rowspan="1">149/150</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89/90 (98.9)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">60/60 (100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(99.3)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/6000)</td></tr><tr><td colspan="2" rowspan="1">360/360 (100)</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="1" rowspan="10">ParainfluenzaVirus</td><td colspan="1" rowspan="2">ParainfluenzaVVirus 11</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(10)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150()</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90 (100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">ParainfluenzaVirus 2</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">29/30(96.7)</td><td colspan="1" rowspan="1">28/30(93.3)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">147/15098.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">87/90 (96.7)</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">ParainfluenzaVVirus 3</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">28/30(93.3)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">19/20(95.5)</td><td colspan="1" rowspan="1">18/20(90.0)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">145/150</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">88/90 (97.8)</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">57/60 (95.0)</td><td colspan="1" rowspan="1">(96.7)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 44</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">29/30(96.7)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">149/150(99.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89/90 (98.9)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">None</td><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">150/150</td></tr><tr><td colspan="3" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td><td colspan="1" rowspan="1">(100)</td></tr><tr><td colspan="2" rowspan="4">RespiratorySyncytial VirusRSV A 2006</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100))</td><td colspan="1" rowspan="1">30/30(100))</td><td colspan="1" rowspan="1">30/30(100))</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">19/20(95.0)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">149/150(9.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90 (100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">59/60 (98.3)</td><td colspan="1" rowspan="1">(9.3)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="2" rowspan="4">Bordetella parapertussisB. parapertussis E595</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">30/30(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">150/150(100)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">90/90 (100)</td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr><tr><td colspan="2" rowspan="4">Bordetella pertussisB. pertussis A639</td><td colspan="1" rowspan="2">Positive</td><td colspan="1" rowspan="1">29/30(96.7)</td><td colspan="1" rowspan="1">29/30(96.7)</td><td colspan="1" rowspan="1">30/30(10)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="1">20/20(100)</td><td colspan="1" rowspan="2">148/15098.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">88/90 (97.8)</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">60/60 (100)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">120/120(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="1">80/80(100)</td><td colspan="1" rowspan="2">600/600(100)</td></tr><tr><td colspan="3" rowspan="1">360/360 (100)</td><td colspan="3" rowspan="1">240/240 (100)</td></tr></table>

<table><tr><td rowspan="2">Assay/Analyte</td><td rowspan="2">Expected Result</td><td colspan="7">Agreement (%) Stratified by Site and BIOFIRE SPOTFIRE System</td></tr><tr><td>BioFire Diagnostics</td><td></td><td></td><td></td><td>External Site</td><td>3</td><td>Total</td></tr><tr><td rowspan="4">Chlamydia pneumoniae Chlamydia pneumoniae AR-39</td><td>Positive</td><td>A 30/30</td><td>B 30/30</td><td>C 30/30</td><td>1 20/20</td><td>2 20/20</td><td>20/20</td><td rowspan="2">150/150 (100)</td></tr><tr><td rowspan="2"></td><td>(100)</td><td>(100)</td><td>(100)</td><td>(100)</td><td>(100) 60/60 (100)</td><td>(100)</td></tr><tr><td colspan="3">90/90 (100) 120/120 120/120</td><td colspan="3">80/80 80/80</td><td rowspan="2">600/600 (100)</td></tr><tr><td rowspan="2">Negative</td><td>(100)</td><td>(100) 360/360 (100)</td><td>120/120 (100)</td><td>(100)</td><td>(100) 240/240 (100)</td><td>80/80 (100)</td></tr><tr><td rowspan="2">29/30</td><td></td><td>30/30</td><td>30/30</td><td>20/20</td><td>20/20</td><td>20/20</td><td rowspan="2">149/150 (99.3)</td></tr><tr><td>Positive</td><td>(96.7)</td><td>(100)</td><td>(100)</td><td>(100)</td><td>(100) 60/60 (100)</td><td>(100)</td></tr><tr><td rowspan="2">M. pneumoniae M129</td><td rowspan="2">Negative</td><td>120/120</td><td>89/90 (98.9) 120/120</td><td>120/120</td><td>80/80</td><td>80/80</td><td>80/80</td><td rowspan="2">600/600 (100)</td></tr><tr><td>(100)</td><td>(100) 360/360 (100)</td><td>(100)</td><td>(100)</td><td>(100) 240/240 (100)</td><td>(100)</td></tr><tr><td rowspan="3">Streptococcus dysgalactiae S. dysgalactiae subsp. equisimilis Z068</td><td rowspan="2">Positive</td><td>30/30</td><td>29/30</td><td>27/30</td><td>20/20 (100)</td><td>20/20</td><td>20/20</td><td rowspan="2">146/150 (97.3)</td></tr><tr><td>(100)</td><td>(96.7) 86/90 (95.6)</td><td>(90.0)</td><td></td><td>(100) 60/60 (100)</td><td>(100)</td></tr><tr><td rowspan="2">Negative</td><td>120/120 (100)</td><td>120/120 (100)</td><td>120/120 (100)</td><td>80/80 (100)</td><td>80/80 (100)</td><td>80/80 (100)</td><td rowspan="2">600/600 (100)</td></tr><tr><td></td><td>360/360 (100)</td><td></td><td></td><td>240/240 (100)</td><td></td></tr><tr><td rowspan="3">Streptococcus pyogenes S. pyogenes Typing strain T1</td><td rowspan="2">Positive</td><td>30/30</td><td>30/30 (100)</td><td>30/30 (100)</td><td>20/20 (100)</td><td>19/20 (95.0)</td><td>20/20 (100)</td><td rowspan="2">149/150 (99.3)</td></tr><tr><td>(100)</td><td>90/90 (100)</td><td></td><td></td><td>59/60 (98.3)</td><td></td></tr><tr><td rowspan="2">Negative</td><td>120/120 (100)</td><td>120/120 (100)</td><td>120/120 (100)</td><td>80/80 (100)</td><td>80/80 (100)</td><td>80/80 (100)</td><td rowspan="2">600/600 (100)</td></tr><tr><td></td><td>360/360 (100)</td><td></td><td></td><td>240/240 (100)</td><td></td></tr><tr><td colspan="2" rowspan="2">Positive Agreement</td><td>681/690</td><td>687/690</td><td>684/690</td><td>454/460</td><td>454/460</td><td>457/460</td><td rowspan="2">3417/ 3450 (99.0)</td></tr><tr><td>(98.7)</td><td>(99.6)</td><td>(99.1)</td><td>(98.7)</td><td>(98.7)</td><td>(99.3)</td></tr><tr><td colspan="2" rowspan="2">By System/Site</td><td></td><td>2052/2070 (99.1)</td><td></td><td></td><td>1365/1380 (98.9)</td><td></td></tr><tr><td colspan="3"></td><td colspan="3"></td></tr></table>

1 1 positive sample was initially reported as Positive: Influenza A Virus (No Subtype Identified) and was therefore retested. The retest result was as expected. The retest results from this sample for each analyte are included in the summary table. 2 2 positive samples were initially reported as Positive: Influenza A Virus (No Subtype Identified) and 1 positive sample was initially reported as Uncertain: Influenza A Virus. All 3 samples were retested and the retest results were as expected. The retest results from these samples for each analyte are included in the summary table. $^ 3 1$ sample was reported as Uncertain Influenza A Virus but was not retested.

2. Linearity: Not applicable.

# 3. Analytical Specificity/Interference:

Specificity (Exclusivity)

The analytical specificity (exclusivity) of the BIOFIRE SPOTFIRE R/ST Panel was evaluated through a combination of in silico analysis and laboratory testing of on- and offpanel species, phylogenetically related viruses and bacteria and other microorganisms that may be found in the respiratory tract.

In silico Analysis

In silico analysis of assay specificity/exclusivity was performed by conducting searches with whole amplicon and paired inner primer sequences using the NCBI Basic Local Alignment Search Tool (BLAST) “blastn” tool. The location and orientation of regions of homology with the primer sequences were evaluated to determine the potential for amplification. A summary of the results from the analysis is provided in Table 5. The BIOFIRE SPOTFIRE R/ST panel assays were shown to be specific for their respective analytes with the following exceptions, which are noted in the device labeling:

• Cross-reaction of the SARS-CoV-2 assays with closely-related bat and pangolin coronavirus sequences   
• Cross-reaction of the Human Rhinovirus/Enterovirus assay with $B$ . pertussis, $B _ { \cdot }$ . parapertussis and B. bronchiseptica   
• Cross-reaction of the $B$ . parapertussis assay with strains of $B$ . bronchiseptica that carry IS1001

Table 5. Summary of in silico BLAST analysis to evaluate exclusivity   

<table><tr><td rowspan=1 colspan=2>Viruses</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Predicted Cross-reaction</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>None 1</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2</td><td rowspan=1 colspan=1>Bat coronavirusPangolin coronavirusBat SARS-like coronavirus</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Bovine picornavirusCanine picornavirusB. pertussis, B. parapertussis, B. bronchiseptica</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>None 2</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>Swine Influenza A/H1 sequencesSome sequences of avian Influenza A/H1N2</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=2>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Predicted Cross-reaction</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Strains of B. bronchiseptica carrying IS1001</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Chlamydia gallinacea</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae subsp. equisimilis</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>None</td></tr></table>

1 Some zoonotic coronaviruses may cross-react but these are not expected to be found in human clinical specimens. 2 A sequence from Protopolystoma xenopodis (a parasitic worm found in frogs) exhibited significant homology to the pan-influenza A PCR primers but is not expected to be found in human clinical specimens.

# Laboratory Testing of Analytical Specificity

The analytical specificity (exclusivity) of the BIOFIRE SPOTFIRE R/ST Panel was also evaluated by testing high concentrations of on- and off-panel viruses, bacteria and other microorganisms that are phylogenetically related or may be found in nasopharyngeal and/or throat swab specimens. Analysis of on-panel analytes was performed to evaluate the potential for intra-panel cross-reactivity (i.e., detection of an analyte by an assay other than the targetspecific assay for that analyte). Analysis of off-panel analytes was performed to evaluate the potential for cross-reaction with other respiratory flora that are not intended to be detected by the panel. Each on-panel species was tested in triplicate and the results are shown in Tables 6 and 7. The expected results were obtained with each on-panel virus and no cross-reaction was observed (Table 6). Based on in silico analysis, it was predicted that $B$ . parapertussis and $B$ . pertussis would produce positive results with the Human Rhinovirus/Enterovirus assay. $B .$ . pertussis produced false-positive results at $1 . 3 \mathrm { ~ x ~ } 1 0 ^ { 1 0 }$ CFU/mL but was reported negative when tested at $1 . 3 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { \ C F U / m L }$ (Table 7). False-positive results for Human Rhinovirus/Enterovirus were not observed with $B$ . parapertussis at the concentration tested $( 4 . 6 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { C F U / m L } )$ ). However, the potential for cross-reaction of the human Rhinovirus/Enterovirus assay with both $B _ { \cdot }$ . parapertussis and $B$ . pertussis based on the in silico prediction is noted in the device labeling.

Table 6. Results from cross-reactivity testing with “on-panel” viruses   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Adenovirus A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810073CF</td><td rowspan=1 colspan=1>1.4 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810062CF</td><td rowspan=1 colspan=1>1.2 x 107 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>ZeptoMetrix 0810110CF</td><td rowspan=1 colspan=1>2.2 x 106 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus D</td><td rowspan=1 colspan=1>ZeptoMetrix 0810119CF</td><td rowspan=1 colspan=1>1.7 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E</td><td rowspan=1 colspan=1>ZeptoMetrix 081070CF</td><td rowspan=1 colspan=1>1.5 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F</td><td rowspan=1 colspan=1>ZeptoMetrix 0810085CF</td><td rowspan=1 colspan=1>1.1 x 106 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>8.9 x 106 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical specimen-1 1</td><td rowspan=1 colspan=1>4.5 x 107 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Clinical specimen-2 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None 3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>ZeptoMetrix 0810228CF</td><td rowspan=1 colspan=1>5.0 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>ZeptoMetrix 0810024CF</td><td rowspan=1 colspan=1>3.6 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2</td><td rowspan=1 colspan=1>ATCC VR-1986HK</td><td rowspan=1 colspan=1>7.6 x 107 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>1.6 x 107 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus A1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810161CF</td><td rowspan=1 colspan=1>2.5 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus A2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810164CF</td><td rowspan=1 colspan=1>3.6 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810156CF</td><td rowspan=1 colspan=1>1.6 x 104 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810162CF</td><td rowspan=1 colspan=1>1.3 x 106 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus A1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810012CFN</td><td rowspan=1 colspan=1>1.3 x 106 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1pdm09</td><td rowspan=1 colspan=1>ZeptoMetrix 08100538CF</td><td rowspan=1 colspan=1>1.4 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>ZeptoMetrix 010526CF</td><td rowspan=1 colspan=1>7.2 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Influenza B (Victoria)</td><td rowspan=1 colspan=1>BEI NR-44023</td><td rowspan=1 colspan=1>2.8 x 108 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810037CF</td><td rowspan=1 colspan=1>2.5 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Yamagata)</td><td rowspan=1 colspan=1>ZeptoMetrix 0810256CF</td><td rowspan=1 colspan=1>2.1 x 104 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810014CF</td><td rowspan=1 colspan=1>4.2 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810015CF</td><td rowspan=1 colspan=1>1.2 x 107 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810016CF</td><td rowspan=1 colspan=1>3.4 x 107 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4a</td><td rowspan=1 colspan=1>ZeptoMetrix 0810060CF</td><td rowspan=1 colspan=1>3.4 x 107 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810040ACF</td><td rowspan=1 colspan=1>4.2 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810479CF</td><td rowspan=1 colspan=1>4.2 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr></table>

$\mathrm { C E I D } _ { 5 0 }$ : Chicken Egg Infectious Dose- $50 \%$ ; $\mathrm { T C I D } _ { 5 0 }$ : Tissue Culture Infectious Dose- $50 \%$ 1 Specimen previously reported Positive for Parainfluenza Virus 4 in addition to Coronavirus HKU1. 2 Specimen previously reported Positive for Adenovirus in addition to Coronavirus HKU1. 3 1/3 replicates reported Positive for Adenovirus (not considered to indicate cross-reactivity based on previous test results for the source specimen).

Table 7. Results from cross-reactivity testing with “on-panel” bacteria   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>ZeptoMetrix 0801462</td><td rowspan=1 colspan=1>4.6 x 109 CFU</td><td rowspan=1 colspan=1>None 1</td></tr><tr><td rowspan=2 colspan=1>Bordetella pertussis</td><td rowspan=2 colspan=1>ZeptoMetrix 0801459</td><td rowspan=1 colspan=1>1.3 x 1010 CFU</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus 2</td></tr><tr><td rowspan=1 colspan=1>1.3 x 109 CFU</td><td rowspan=1 colspan=1>None 2</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=1>2.9 x 107 IFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>ZeptoMetrix 0801579</td><td rowspan=1 colspan=1>2.5 x 107 CCU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiaesubsp. equisimilis</td><td rowspan=1 colspan=1>ZeptoMetrix 0801516</td><td rowspan=1 colspan=1>7.8 x 108CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12344</td><td rowspan=1 colspan=1>3.4 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr></table>

CCU: Color Changing Units; CFU: Colony Forming Units; IFU: Inclusion Forming Units 1 In silico analysis predicted detection by the Human Rhinovirus/Enterovirus assay but none was observed. 2 Cross-reaction predicted by in silico analysis and confirmed by laboratory testing a $\mathrm { t } \geq 1 . 3 \mathrm { ~ x ~ } 1 0 ^ { 1 0 } \mathrm { C F U / m L }$ .

The potential for cross-reaction with off-panel analytes was evaluated by testing 87 strains of bacteria, 2 species of yeast and 7 viruses that are either phylogenetically related to the onpanel analytes or which are commonly found in the respiratory tract (Table 8). Included in the study were isolates of S. dysgalactiae subsp. equisimilis of animal origin and S. dysgalactiae subsp. dysgalactiae, both of which lack the region of the S. dysgalactiae genome targeted by the BIOFIRE SPOTFIRE R/ST Panel. No cross-reaction was observed other than the expected detection of one strain of $B _ { \cdot }$ . bronchiseptica by the assay for $B$ . parapertussis.

Table 8. Results from cross-reactivity testing with “off-panel” bacteria, viruses and fungi   

<table><tr><td rowspan=1 colspan=4>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Arcanobacterium bernardiae</td><td rowspan=1 colspan=1>ATCC BAA-441</td><td rowspan=1 colspan=1>1.6 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Arcanobacterium haemolyticum</td><td rowspan=1 colspan=1>ATCC 9345</td><td rowspan=1 colspan=1>1.5 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Arcanobacterium pyogenes</td><td rowspan=1 colspan=1>ATCC 49698</td><td rowspan=1 colspan=1>6.7 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>ATCC 7064</td><td rowspan=1 colspan=1>8.3 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=6 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>ATCC 10580</td><td rowspan=1 colspan=1>8.3 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>ATCC 4617</td><td rowspan=1 colspan=1>7.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>ATCC 19395</td><td rowspan=1 colspan=1>7.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>NRRL B-59914</td><td rowspan=1 colspan=1>7.1 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>NRRL B-59909</td><td rowspan=1 colspan=1>28 cells</td><td rowspan=1 colspan=1>B. parapertussis</td></tr><tr><td rowspan=1 colspan=1>2.8 cells</td><td rowspan=1 colspan=1>B. parapertussis 1</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>ATCC 700052</td><td rowspan=1 colspan=1>8.3 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>ATCC 51671</td><td rowspan=1 colspan=1>7.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Campylobacter rectus</td><td rowspan=1 colspan=1>ATCC 33238</td><td rowspan=1 colspan=1>7.6 x 107 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>ZeptoMetrix 0801775</td><td rowspan=1 colspan=1>1.3 x 108 IFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>ATCC 27010</td><td rowspan=1 colspan=1>8.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumpseudodiphtheriticum</td><td rowspan=1 colspan=1>ATCC 10700</td><td rowspan=1 colspan=1>8.7 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>ATCC 49605</td><td rowspan=1 colspan=1>8.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ZeptoMetrix 0801637</td><td rowspan=1 colspan=1>8.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC BAA-2196</td><td rowspan=1 colspan=1>7.2 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorumsubsp. funduliforme</td><td rowspan=1 colspan=1>ATCC 51357</td><td rowspan=1 colspan=1>4.4 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>ATCC 25586</td><td rowspan=1 colspan=1>4.9 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>ATCC 27725</td><td rowspan=1 colspan=1>1.6 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>ATCC 10379</td><td rowspan=1 colspan=1>4.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>ATCC 27824</td><td rowspan=1 colspan=1>1.0 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Granulicatella adiacens</td><td rowspan=1 colspan=1>ATCC 49175</td><td rowspan=1 colspan=1>1.3 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=2>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 10211</td><td rowspan=1 colspan=2>8.3 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Haemophilus parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 49700</td><td rowspan=1 colspan=2>8.7 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC AR-BANK#0115</td><td rowspan=1 colspan=2>7.3 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus rhamnosus</td><td rowspan=1 colspan=1>ATCC 7469</td><td rowspan=1 colspan=2>7.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus lactis</td><td rowspan=1 colspan=1>ATCC 29146</td><td rowspan=1 colspan=2>6.2 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>ATCC 33215</td><td rowspan=1 colspan=2>7.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Leptotrichia buccalis</td><td rowspan=1 colspan=1>ATCC 14201</td><td rowspan=1 colspan=2>4.4 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>ATCC 43627</td><td rowspan=1 colspan=2>7.2 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>ZeptoMetrix 0801660</td><td rowspan=1 colspan=2>6.1 x 106 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma buccale</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10136</td><td rowspan=1 colspan=2>1.4 x 107 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma faucium</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10174</td><td rowspan=1 colspan=2>1.4 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma fermentans</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10117</td><td rowspan=1 colspan=2>2.8 x 107 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10195</td><td rowspan=1 colspan=2>1.8 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10111</td><td rowspan=1 colspan=2>1.2 x 107 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma lipophilum</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10173</td><td rowspan=1 colspan=2>1.5 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma orale</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10112</td><td rowspan=1 colspan=2>2.2 x 107 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma salivarium</td><td rowspan=1 colspan=1>Mycoplasma Experience NC10113</td><td rowspan=1 colspan=2>4.4 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>ATCC 25295</td><td rowspan=1 colspan=2>8.5 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>ZeptoMetrix 0801482</td><td rowspan=1 colspan=2>4.9 x 107 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>ATCC 23971</td><td rowspan=1 colspan=2>2.7 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13113</td><td rowspan=1 colspan=2>7.4 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>ATCC 9913</td><td rowspan=1 colspan=2>7.2 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria subflava</td><td rowspan=1 colspan=1>ATCC 49275</td><td rowspan=1 colspan=2>8.0 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parvimonas micra</td><td rowspan=1 colspan=1>ATCC 33270</td><td rowspan=1 colspan=2>6.0 x 107 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis carinii</td><td rowspan=1 colspan=1>ATCC PRA-159</td><td rowspan=1 colspan=2>1.0 x 107 nuclei</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Porphyromonas endodontalis</td><td rowspan=1 colspan=1>ATCC 35406</td><td rowspan=1 colspan=2>1.6 x 107 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Porphyromonas gingivalis</td><td rowspan=1 colspan=1>ATCC BAA-308</td><td rowspan=1 colspan=2>5.0 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Prevotella histicola</td><td rowspan=1 colspan=1>BEI HM-471</td><td rowspan=1 colspan=2>9.0 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Prevotella melaninogenica</td><td rowspan=1 colspan=1>ATCC 25845</td><td rowspan=1 colspan=2>6.9 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Prevotella oralis</td><td rowspan=1 colspan=1>ATCC 33322</td><td rowspan=1 colspan=2>6.2 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC AR-BANK#0092</td><td rowspan=1 colspan=2>8.3 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Rhodococcus equi</td><td rowspan=1 colspan=1>ATCC 33706</td><td rowspan=1 colspan=2>7.3 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 27137</td><td rowspan=1 colspan=2>8.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-1700</td><td rowspan=1 colspan=2>7.4 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 12228</td><td rowspan=1 colspan=2>8.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus haemolyticus</td><td rowspan=1 colspan=1>ATCC 29968</td><td rowspan=1 colspan=2>8.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus intermedius</td><td rowspan=1 colspan=1>ATCC 29663</td><td rowspan=1 colspan=2>8.2 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td><td rowspan=1 colspan=1>ATCC 15305</td><td rowspan=1 colspan=2>8.1 x 10 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 13813</td><td rowspan=1 colspan=2>6.0 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 700231</td><td rowspan=1 colspan=2>7.1 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Streptococcus constellatussubsp. pharyngitis</td><td rowspan=1 colspan=1>NCTC 13122</td><td rowspan=1 colspan=2>5.6 x 108 cells</td><td rowspan=2 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>Streptococcus dysgalactiaesubsp. dysgalactiae 2</td><td rowspan=1 colspan=1>ATCC 43078</td><td rowspan=1 colspan=2>6.7 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>NCTC 4669</td><td rowspan=1 colspan=2>7.4 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>NCTC 4335</td><td rowspan=1 colspan=2>8.4 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>NCTC 4670</td><td rowspan=1 colspan=2>6.6 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>CCUG 27665</td><td rowspan=1 colspan=1>7.4 x 10</td><td rowspan=1 colspan=1>cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>CCUG 28112</td><td rowspan=1 colspan=1>6.7 x 10</td><td rowspan=1 colspan=1>cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>CCUG 28114</td><td rowspan=1 colspan=2>7.5 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiaesubsp. equisimilis (pig)</td><td rowspan=1 colspan=1>CCUG 28117</td><td rowspan=1 colspan=2>7.1 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Streptococcus dysgalactiaesubsp. equisimilis (horse)</td><td rowspan=1 colspan=1>CCUG 27664</td><td rowspan=1 colspan=2>7.5 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>ATCC 10009</td><td rowspan=1 colspan=2>6.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gallolyticus</td><td rowspan=1 colspan=1>ATCC 43143</td><td rowspan=1 colspan=2>2.8 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gordonii</td><td rowspan=1 colspan=1>ATCC 10558</td><td rowspan=1 colspan=2>4.5 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus intermedius</td><td rowspan=1 colspan=1>ATCC 27335</td><td rowspan=1 colspan=2>2.9 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 15914</td><td rowspan=1 colspan=2>3.2 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mutans</td><td rowspan=1 colspan=1>ATCC 25175</td><td rowspan=1 colspan=2>2.3 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus oralis</td><td rowspan=1 colspan=1>ATCC 10557</td><td rowspan=1 colspan=2>1.1 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>ATCC 15912</td><td rowspan=1 colspan=1>7.8 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 49619</td><td rowspan=1 colspan=1>2.5 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>ATCC 13419</td><td rowspan=1 colspan=1>6.6 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus sanguinis</td><td rowspan=1 colspan=1>ATCC 10556</td><td rowspan=1 colspan=1>1.1 x 109 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Tannerella forsythia</td><td rowspan=1 colspan=1>ATCC BAA-2717</td><td rowspan=1 colspan=1>2.6 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Treponema denticola</td><td rowspan=1 colspan=1>ATCC 33520</td><td rowspan=1 colspan=1>2.2 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Ureaplasma urealyticum</td><td rowspan=1 colspan=1>ATCC 27618</td><td rowspan=1 colspan=1>5.7 x 107 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Veillonella parvula</td><td rowspan=1 colspan=1>ATCC 10790</td><td rowspan=1 colspan=1>4.7 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Yeast/Fungi</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC MYA-2876</td><td rowspan=1 colspan=1>2.8 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1>ATCC 18824</td><td rowspan=1 colspan=1>1.9 x 108 cells</td><td rowspan=1 colspan=1>None</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Viruses</td></tr><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>ObservedCross-reaction</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>ZeptoMetrix 0810003CF</td><td rowspan=1 colspan=1>1.9 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>ZeptoMetrix 0810008CF</td><td rowspan=1 colspan=1>5.9 x 106 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Simplex Virus 1</td><td rowspan=1 colspan=1>ATCC VR-260</td><td rowspan=1 colspan=1>8.9 x 106 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>ZeptoMetrix 0810025CF</td><td rowspan=1 colspan=1>2.5 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Middle East Respiratory SyndromeCoronavirus</td><td rowspan=1 colspan=1>ZeptoMetrix 0810575CFHI</td><td rowspan=1 colspan=1>1.2 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>ZeptoMetrix 0810079CF</td><td rowspan=1 colspan=1>2.0 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Severe Acute Respiratory SyndromeCoronavirus</td><td rowspan=1 colspan=1>BEI NR-52346</td><td rowspan=1 colspan=1>5.3 x 105 genomes</td><td rowspan=1 colspan=1>None</td></tr></table>

CFU: Colony Forming Units; IFU: Inclusion Forming Units; $\mathrm { T C I D } _ { 5 0 }$ : Tissue Culture Infectious Dose- $5 0 \%$ 1 B. bronchiseptica NRRL B-59909 carries IS1001 which is the target for the BIOFIRE SPOTFIRE R/ST Panel $B .$ . parapertussis assay. Positive results were obtained with all target levels $\ge 2 8$ cells $/ \mathrm { m L }$ and with 1/3 replicates at 2.8 cells/mL. The BIOFIRE SPOTFIRE R/ST Panel assay for $S$ . dysgalactiae is intended to detect strains of $S _ { \cdot }$ . dysgalactiae subsp. equisimilis of human origin. Detection of S. dysgalactiae subsp. dysgalactiae or strains of $S _ { \ast }$ . dysgalactiae subsp. equisimilis of non-human origin is not expected.

Based on a combination of the in silico and laboratory testing data, the potential for crossreaction of the BIOFIRE SPOTFIRE R/ST Panel with certain on- and off-panel analytes is noted in the device labeling, as summarized in Table 9.

Table 9. Summary of known or predicted cross-reactivity   

<table><tr><td colspan="1" rowspan="1">SPOTFIRE R/ST Panel Assay</td><td colspan="1" rowspan="1">Cross-reactive Species</td><td colspan="1" rowspan="1">Description</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Bat coronavirusPangolin coronavirusBat SARS-like coronavirus</td><td colspan="1" rowspan="1">Predicted cross-reaction withclosely related sarbecovirusesisolated from bats and pangolin</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Bordetella bronchisepticaBordetella parapertussisBordetella pertussisBovine and canine picornaviruses</td><td colspan="1" rowspan="1">Homologous sequences in the listedorganisms may be detected andreported as HumanRhinovirus/Enterovirus</td></tr><tr><td colspan="1" rowspan="1">Influenza Virus A/H1-2009</td><td colspan="1" rowspan="1">Influenza A H1N1 (swine)</td><td colspan="1" rowspan="1">The FluA-H1-2009 assay maydetect influenza A H1 genesequences of swine origin that willbe reported as Influenza A Virus(subtype H1-2009) or as InfluenzaA (No Subtype Identified),</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">depending on the strain andconcentration presentCertain strains of avian influenzaA/H1N2 may also be detected andreported as influenza A/H1-2009</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Bordetella bronchiseptica (isolatescarrying IS1001)</td><td colspan="1" rowspan="1">Some strains of B. bronchisepticacarry IS1001, the target for the B.parapertussis assay, and may bedetected and reported as B.parapertussis</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Chlamydia gallinacea</td><td colspan="1" rowspan="1">Homologous sequences in C.gallinacea may be detected andreported as C. pneumoniae</td></tr></table>

# Interference Testing

A study was conducted to evaluate the potential for assay interference due to the presence of endogenous and exogenous substances that may be present in upper respiratory specimens. In addition, testing was also performed with representative on- and off-panel microorganisms at very high concentrations to evaluate the potential for competitive interference between analytes on the BIOFIRE SPORFIRE R/ST Panel. Testing of each potentially interfering substance/microorganism was performed using contrived positive and negative specimens prepared in artificial throat swab matrix. The positive specimens contained representative onpanel analytes at levels close to their respective LoD (Table 10). In a separate study (refer to Sections VII (A)(6) and (B)(2), below) the analytical sensitivity of the BIOFIRE SPOTFIRE R/ST Panel was shown to be similar, irrespective of the specimen matrix (nasopharyngeal or throat swab) and therefore it was considered acceptable to use just one of the two matrices for the evaluation of assay interference. The concentration of each substance tested was chosen to represent a high level that may be found in natural clinical specimens. The results of the study are summarized in Tables 11-16.

Table 10. Representative on-panel analytes used to evaluate assay interference   
Table 11. Results from testing the effect of endogenous substances on the performance of the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=2 colspan=1>Strain/Source</td><td rowspan=2 colspan=1>Characteristics</td><td rowspan=2 colspan=1>Concentration(per mL) 1</td><td rowspan=1 colspan=2>Panel</td></tr><tr><td rowspan=1 colspan=1>Respiratory</td><td rowspan=1 colspan=1>Sore Throat</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenovirus B Type 3ZeptoMetrix 0810062CF</td><td rowspan=1 colspan=1>Non-envelopedDNA virus</td><td rowspan=1 colspan=1>2.4 TCID50</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>Coronavirus NL63ZeptoMetrix 0810228CF</td><td rowspan=1 colspan=1>Enveloped RNAvirus</td><td rowspan=1 colspan=1>0.099 TCID50</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Human Rhinovirus A1ZeptoMetrix 0810012CFN</td><td rowspan=1 colspan=1>Non-envelopedRNA virus</td><td rowspan=1 colspan=1>0.63 TCID50</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>Influenza B virusB/Florida/02/06 (Victoria)ZeptoMetrix 0810255CF</td><td rowspan=1 colspan=1>Enveloped RNAvirus</td><td rowspan=1 colspan=1>60 IFU</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Chlamydia pneumoniaeAR-39 ATCC 53592</td><td rowspan=1 colspan=1>Gram negativebacterium</td><td rowspan=1 colspan=1>0.0075 TCID50</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Streptococcus pyogenesSF-120 T-type 1ATCC 12344</td><td rowspan=1 colspan=1>Gram positivebacterium</td><td rowspan=1 colspan=1>1,400 cells</td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>Yes</td></tr></table>

IFU: Inclusion Forming Units; $\mathrm { T C I D } _ { 5 0 }$ : Tissue Culture Infectious Dose- $50 \%$ 1 3X LoD

<table><tr><td rowspan=3 colspan=1>Substance</td><td rowspan=3 colspan=1>Conc&quot;</td><td rowspan=3 colspan=1>SampleType</td><td rowspan=1 colspan=17>Positive Results Reported/Replicates Tested</td></tr><tr><td rowspan=1 colspan=11>Viruses</td><td rowspan=1 colspan=6>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Arius</td><td rowspan=1 colspan=1>(oos) sn.)</td><td rowspan=1 colspan=1>Z-C-S S.J</td><td rowspan=1 colspan=1>Hmonun uuns</td><td rowspan=1 colspan=1>H  un</td><td rowspan=1 colspan=1>iun V uns</td><td rowspan=1 colspan=1>60/L V Azu</td><td rowspan=1 colspan=1>IH V un3</td><td rowspan=1 colspan=1>In  zus</td><td rowspan=1 colspan=1>Pa.r uus</td><td rowspan=1 colspan=1>Sh T </td><td rowspan=1 colspan=1>PsS</td><td rowspan=1 colspan=1>Paess</td><td rowspan=1 colspan=1>euoumude</td><td rowspan=1 colspan=1>MmuounudW</td><td rowspan=1 colspan=1>s</td><td rowspan=1 colspan=1>sSod S</td></tr><tr><td rowspan=2 colspan=1>Whole human blood(sodium citrate)</td><td rowspan=2 colspan=1>10%v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Humansputum/mucus</td><td rowspan=2 colspan=1>1%v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Humangenomic DNA</td><td rowspan=2 colspan=1>20ng/μL</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>4/51</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>5/5</td></tr></table>

Neg: Expected result - Negative for all SPOTFIRE R/ST Panel analytes. Pos: Expected result - Positive for representative SPOTFIRE R/ST Panel analytes highlighted in red. Contrived Positive Samples contained the representative on-panel analytes at 3X LoD. Contrived Negative Samples contained only the potentially interfering substance. 1 Initially, 1/3 samples was reported Negative for Human Rhinovirus/Enterovirus; therefore, 2 additional replicates were tested, both of which produced the expected results for all analytes.

None of the endogenous substances tested caused reproducible interference with the BIOFIRE SPOTFIRE R/ST Panel.

Table 12. Results from testing the effect of exogenous substances on the performance of the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td colspan="1" rowspan="3">Substance</td><td colspan="1" rowspan="3">Conc"</td><td colspan="1" rowspan="3">SampleType</td><td colspan="17" rowspan="1">Positive Results Reported/Replicates Tested</td></tr><tr><td colspan="11" rowspan="1">Viruses</td><td colspan="6" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Ahus</td><td colspan="1" rowspan="1">(oo) .)</td><td colspan="1" rowspan="1">Zc-SS S.Jr0</td><td colspan="1" rowspan="1">Humonun uunss</td><td colspan="1" rowspan="1">h / .uuwn</td><td colspan="1" rowspan="1">Su V aunus</td><td colspan="1" rowspan="1">G00/L V Az</td><td colspan="1" rowspan="1">Ur  zun</td><td colspan="1" rowspan="1">Tun  zus</td><td colspan="1" rowspan="1">Par zgus</td><td colspan="1" rowspan="1">S T </td><td colspan="1" rowspan="1">Ppg</td><td colspan="1" rowspan="1">Paes</td><td colspan="1" rowspan="1">monud</td><td colspan="1" rowspan="1">aoounnd W</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">seeSodS</td></tr><tr><td colspan="1" rowspan="2">Promethazinehydrochloride</td><td colspan="1" rowspan="2">3.3xmg/mL</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acetaminophen(paracetamol)</td><td colspan="1" rowspan="1">1.0x</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Arls</td><td colspan="1" rowspan="1">(os) r)</td><td colspan="1" rowspan="1">Cc-SS </td><td colspan="1" rowspan="1">Hmon uun</td><td colspan="1" rowspan="1">H  us</td><td colspan="1" rowspan="1">iu v s</td><td colspan="1" rowspan="1">600  Azu</td><td colspan="1" rowspan="1">I  u</td><td colspan="1" rowspan="1">iu  zus</td><td colspan="1" rowspan="1">Pai augus</td><td colspan="1" rowspan="1">Hh  </td><td colspan="1" rowspan="1">sess</td><td colspan="1" rowspan="1">Paess</td><td colspan="1" rowspan="1">Cmuounud</td><td colspan="1" rowspan="1">aoounud W</td><td colspan="1" rowspan="1">Sops</td><td colspan="1" rowspan="1">seeSosd S</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mg/mL</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Acetylsalicylic acid(aspirin)</td><td colspan="1" rowspan="2">30xmg/mL</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Ibuprofen</td><td colspan="1" rowspan="2">$20xmg/mL</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">4/51</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="2">Albuterol sulfate</td><td colspan="1" rowspan="2">50x mg/mL</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Triple antibioticointment(neomycin/polymyxin/BBaitracin)</td><td colspan="1" rowspan="2">2% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Mucinex Severe NasalCongestion ReliefClear &amp; Cool Spray</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg 2</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Saline nasal spray(0.65% disodiumphosphate phenylcarbinol,monosodium phosphate,benzalkonium chloride)</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Vicks VapoRub CoughSuppresant Topical (4.8% camphor, 1.2%eucalyptus oil, 2.%menthol</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Vaselinepetroleum jelly(100%whitepetrolatum)</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Orajel(0.13% benzalkoniumchloride, 20% benzocaine,0.5% menthol, 0.15% zincchloride)</td><td colspan="1" rowspan="2">2% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Chloraseptic SoreThroat Spray(1.4% phenol)</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Vicks Formula 44 DM (0.67 mg/mLdextromethorphanhydrobromide, 13 mg/mLguaifenisin</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos 2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Ar.i</td><td colspan="1" rowspan="1">( r)</td><td colspan="1" rowspan="1">S3-SSouas</td><td colspan="1" rowspan="1">wounudntun</td><td colspan="1" rowspan="1">Shi / uemn</td><td colspan="1" rowspan="1">i V Azus</td><td colspan="1" rowspan="1">600/L  Au</td><td colspan="1" rowspan="1">TH  z</td><td colspan="1" rowspan="1">i  zus</td><td colspan="1" rowspan="1">Pa augus</td><td colspan="1" rowspan="1">S tisH1</td><td colspan="1" rowspan="1">PesB</td><td colspan="1" rowspan="1">psB</td><td colspan="1" rowspan="1">amuounndC</td><td colspan="1" rowspan="1">aoounnd W</td><td colspan="1" rowspan="1">si</td><td colspan="1" rowspan="1">seeond 'S</td></tr><tr><td colspan="1" rowspan="2">Phenylephrinehydrochloride</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Nasal spray(50 μg fluticasonepropionate)</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">4/51</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="2">Sucrets Sore Throat(2.0 mg dycloninehydrochloride/lozenge)</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Benadryl AllergyLiqui-gels(25 mg/capsulediphenhydraminehydrochloide)</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos 2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Zicam Cold Remedy(galphimia glauca,luffa oprculata,sabadilla)</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Cold-eeze(2.3% zincgluconate</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">HALLS lozenge(5 mgmenthol/lozenge)</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Listerine Cool Mint(0.042% menthol, 0.064%thymol, 0.06% methylsalicylate, 0.092%euccalyptol)</td><td colspan="1" rowspan="2">6.5%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Copenhagen Snuff(tobacco)</td><td colspan="1" rowspan="2">1% w/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">4/51</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="2">Juice Head(30% polypropyleneglycol, 70% vegetableglycerin)</td><td colspan="1" rowspan="2">1% v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

Neg: Expected result - Negative for all SPOTFIRE R/ST Panel analytes. Pos: Expected result - Positive for representative SPOTFIRE R/ST Panel analytes highlighted in red. Contrived Positive Samples contained the representative on-panel analytes at 3X LoD. Contrived Negative Samples contained only the potentially interfering substance. 1 1/3 replicates initially reported Negative; 2 additional replicates were both reported Positive. 2 1/3 replicates initially reported Invalid; retesting produced the expected results for all analytes.

None of the exogenous substances tested caused reproducible interference with the BIOFIRE SPOTFIRE R/ST Panel.

Table 13. Results from testing the effect of substances associated with specimen collection on the performance of the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td colspan="1" rowspan="3">Substance</td><td colspan="1" rowspan="3">Conc"</td><td colspan="1" rowspan="3">SampleSType</td><td colspan="17" rowspan="1">Positive Results Reported/Replicates Tested</td></tr><tr><td colspan="11" rowspan="1">Viruses</td><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Aros</td><td colspan="1" rowspan="1">(oo </td><td colspan="1" rowspan="1">Z-a .Jr0</td><td colspan="1" rowspan="1">Hhlont uuns</td><td colspan="1" rowspan="1">Hor n</td><td colspan="1" rowspan="1">Ihu V Azuns</td><td colspan="1" rowspan="1">6001/I V u</td><td colspan="1" rowspan="1">I  zun</td><td colspan="1" rowspan="1">Th  zus</td><td colspan="1" rowspan="1">ar bzuus</td><td colspan="1" rowspan="1">h  </td><td colspan="1" rowspan="1">PdsB</td><td colspan="1" rowspan="1">sess</td><td colspan="1" rowspan="1">aounudC</td><td colspan="1" rowspan="1">aoounnd W</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">sesSoud S</td></tr><tr><td colspan="1" rowspan="2">Rayon swabCopan)Diaagostics)</td><td colspan="1" rowspan="2">1 swab,15 min 1</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Nylonflocked swab(CopanDiagnostics</td><td colspan="1" rowspan="2">1 swab,15 min 1</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Polyester swabCopanDiagnostics)</td><td colspan="1" rowspan="2">1 swab,15 min 1</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Calciumalginate swab(Puritan)</td><td colspan="1" rowspan="2">1 swab,15 min1</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Cary-BlairMedium</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Dulbecco'sModifiedEagles Medium(DMEM)</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">v/v</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">MicroTest M4Viral TransportMedium (VTM)</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">ViralPreservativeMedium (VPM)</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">PhosphateBuffered Saline</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">PrimeStoreMolecularTransportMedium (MTM)</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos 2</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="3">Substance</td><td colspan="1" rowspan="3">Conc"</td><td colspan="1" rowspan="3">SampleType</td><td colspan="17" rowspan="1">Positive Results Reported/Replicates Tested</td></tr><tr><td colspan="11" rowspan="1">Viruses</td><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Whus</td><td colspan="1" rowspan="1">(oo .r)</td><td colspan="1" rowspan="1">G-c-S Sru</td><td colspan="1" rowspan="1">Hhmomudn uu</td><td colspan="1" rowspan="1">Hh / . un</td><td colspan="1" rowspan="1">Inn V zus</td><td colspan="1" rowspan="1">6007/IL V Azn</td><td colspan="1" rowspan="1">TH  un</td><td colspan="1" rowspan="1">In  zuns</td><td colspan="1" rowspan="1">Pa zuus</td><td colspan="1" rowspan="1">h T </td><td colspan="1" rowspan="1">Pess</td><td colspan="1" rowspan="1">PsS</td><td colspan="1" rowspan="1">aououmud</td><td colspan="1" rowspan="1">aooumud W</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">seeSo S</td></tr><tr><td colspan="1" rowspan="2">Stuart TransportMedium</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">eNATMolecularTransportMedium</td><td colspan="1" rowspan="2">90%v/v</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

Neg: Expected result - Negative for all SPOTFIRE R/ST Panel analytes. Pos: Expected result - Positive for representative SPOTFIRE R/ST Panel analytes highlighted in red. Contrived Positive Samples contained the representative on-panel analytes at 3X LoD. Contrived Negative Samples contained only the potentially interfering substance. 1 Swab immersed in artificial throat swab matrix for $1 5 \mathrm { { m i n } }$ prior to testing. 2 1/3 replicates initially reported Invalid; testing of 2 additional replicates produced the expected results for all analytes.

None of the exogenous substances associated with collection of upper respiratory swab specimens that were tested caused interference with the BIOFIRE SPOTFIRE R/ST Panel.

Table 14. Results from testing the effect of disinfectants and decontamination products on the performance of the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td rowspan=3 colspan=1>Substance</td><td rowspan=3 colspan=1>Conc&quot;</td><td rowspan=3 colspan=1>SampleType</td><td rowspan=1 colspan=17>Positive Results Reported/Replicates Tested</td></tr><tr><td rowspan=1 colspan=11>Viruses</td><td rowspan=1 colspan=6>Bacteria</td></tr><tr><td rowspan=1 colspan=1>aru</td><td rowspan=1 colspan=1>(o .)</td><td rowspan=1 colspan=1>Zc- Sr0</td><td rowspan=1 colspan=1>Hhmou uun</td><td rowspan=1 colspan=1>Ho r n</td><td rowspan=1 colspan=1>Ih V uzunus</td><td rowspan=1 colspan=1>600/ V un</td><td rowspan=1 colspan=1>TH  un</td><td rowspan=1 colspan=1>Tnn  uns</td><td rowspan=1 colspan=1>Par uus</td><td rowspan=1 colspan=1>Hh T </td><td rowspan=1 colspan=1>sesS</td><td rowspan=1 colspan=1>PaesSs</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>aoounn </td><td rowspan=1 colspan=1>S s</td><td rowspan=1 colspan=1>Seod S</td></tr><tr><td rowspan=4 colspan=1>Bleach(sodiumhypochlorite)15 min incubation</td><td rowspan=2 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>2% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Bleach(sodiumhypochlorite)4 hour incubation</td><td rowspan=2 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Bleach(sodiumhypochlorite18.5 hour incubation</td><td rowspan=2 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Ethanol</td><td rowspan=2 colspan=1>7% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Ammoniumchloridedisinfectingwipes</td><td rowspan=2 colspan=1>0.25-0. q sample</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>DNAZap</td><td rowspan=2 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>RNaseZap</td><td rowspan=2 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos 1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Neg: Expected result - Negative for all SPOTFIRE R/ST Panel analytes. Pos: Expected result - Positive for representative SPOTFIRE R/ST Panel analytes highlighted in red. Contrived Positive Samples contained the representative on-panel analytes at 3X LoD. Contrived Negative Samples contained only the potentially interfering substance. $^ \mathrm { ~ 1 ~ } 1 / 3$ replicates initially reported Invalid due to an instrument error; retesting and produced the expected results for all analytes.

None of the disinfectants or decontamination products that were tested caused interference with the BIOFIRE SPOTFIRE R/ST Panel. However, although bleach was not shown to have an adverse effect on test performance under the conditions of this study, because of its known

potential to cause nucleic acid degradation, a warning to avoid contact of test samples with bleach is included in the device labeling.

Table 15. Summary of results from testing for competitive interference between on-panel analytes within the BIOFIRE SPOTFIRE R/ST Panel

<table><tr><td rowspan=3 colspan=1>Potentially Interfering pecies </td><td rowspan=3 colspan=1>Conc&quot;(copiees/mL)</td><td rowspan=3 colspan=1>SampleSType</td><td rowspan=1 colspan=17>Positive Results Reported/Replicates Tested</td></tr><tr><td rowspan=1 colspan=11>Viruses</td><td rowspan=1 colspan=6>Bacteria</td></tr><tr><td rowspan=1 colspan=1>whbus</td><td rowspan=1 colspan=1>(oo) </td><td rowspan=1 colspan=1>Z-c- .Jr0</td><td rowspan=1 colspan=1>Hhloun uun</td><td rowspan=1 colspan=1>Hm Ttn</td><td rowspan=1 colspan=1>Ihu V AzunS</td><td rowspan=1 colspan=1>6007/L V Azun</td><td rowspan=1 colspan=1>T  zn</td><td rowspan=1 colspan=1>Tn  zuus</td><td rowspan=1 colspan=1>Pr zus</td><td rowspan=1 colspan=1>S T </td><td rowspan=1 colspan=1>ps</td><td rowspan=1 colspan=1>ess</td><td rowspan=1 colspan=1>pounn</td><td rowspan=1 colspan=1>aoounud W</td><td rowspan=1 colspan=1>S s</td><td rowspan=1 colspan=1>sausSod S</td></tr><tr><td rowspan=2 colspan=1>AdenovirusA31</td><td rowspan=2 colspan=1>1.6 x 107</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Coronavirus229E</td><td rowspan=2 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>EnterovirusD68</td><td rowspan=2 colspan=1>7.8 x 107</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>ParainfluenzaVirus 3</td><td rowspan=2 colspan=1>8.0 x 106</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>RespiratorySyncytial VirusA</td><td rowspan=2 colspan=1>1.5 x 107</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Bordetellapertussis</td><td rowspan=2 colspan=1>1.6 x 109</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Streptococcuspyogenes</td><td rowspan=2 colspan=1>2.2 x 108</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Neg: Expected result - Negative for all SPOTFIRE R/ST Panel analytes except the interfering species. Pos: Expected result - Positive for representative SPOTFIRE R/ST Panel analytes highlighted in red. Contrived Positive Samples contained the representative on-panel analytes at 3X LoD. Contrived Negative Samples contained only the potentially interfering substance. Shaded cells indicate positive results due to the presence of a “potentially interfering species”. All results were as expected based on the design of the SPOTFIRE R/ST Panel. 1 On-panel analytes detected by the BIOFIRE SPOTFIRE R/ST Panel; positive results were obtained for all on-panel analytes, as expected.

No interference was observed with high concentrations of on-panel microbial species that may be present in nasopharyngeal or throat swab specimens.

Table 16. Summary of results from testing for competitive interference by off-panel analytes with the BIOFIRE SPOTFIRE R/ST Panel

<table><tr><td rowspan=3 colspan=1>Potentially InterferingSpecies</td><td rowspan=3 colspan=1>Conc&quot;ie </td><td rowspan=3 colspan=1>SampleType</td><td rowspan=1 colspan=17>Positive Results Reported/Replicates Tested</td></tr><tr><td rowspan=1 colspan=11>Viruses</td><td rowspan=1 colspan=6>Bacteria</td></tr><tr><td rowspan=1 colspan=1>wr</td><td rowspan=1 colspan=1>( )</td><td rowspan=1 colspan=1>Z-C-S S.J</td><td rowspan=1 colspan=1>Hhlou uun</td><td rowspan=1 colspan=1>H  Tn</td><td rowspan=1 colspan=1>Ihn V Azus</td><td rowspan=1 colspan=1>600/ V Azu0</td><td rowspan=1 colspan=1>T  zn</td><td rowspan=1 colspan=1>Il  us</td><td rowspan=1 colspan=1>ar bzunus</td><td rowspan=1 colspan=1>S T </td><td rowspan=1 colspan=1>Spps</td><td rowspan=1 colspan=1>Baes</td><td rowspan=1 colspan=1>muounune</td><td rowspan=1 colspan=1>aoouneud &#x27;W</td><td rowspan=1 colspan=1>S s</td><td rowspan=1 colspan=1>seeSod&#x27;</td></tr><tr><td rowspan=2 colspan=1>Cytomegalovirus</td><td rowspan=2 colspan=1>4.2 x104TCID50</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Herpes SimplexPVVirus 1</td><td rowspan=2 colspan=1>90xTCID50</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Haemophilusinfluenzae</td><td rowspan=2 colspan=1>8.3 x1108CFU</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Staphylococcusaureus</td><td rowspan=2 colspan=1>7.4x108CFU</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Streptococcuspneumoniae</td><td rowspan=2 colspan=1>2.0XCFU</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Candidaalbicans</td><td rowspan=2 colspan=1>2.8 x10CFU</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Neg: Expected result - Negative for all SPOTFIRE R/ST Panel analytes. Pos: Expected result - Positive for representative SPOTFIRE R/ST Panel analytes highlighted in red. Contrived Positive Samples contained the representative on-panel analytes at 3X LoD. Contrived Negative Samples contained only the potentially interfering substance.

No interference was observed with off-panel microbial species that may be present in nasopharyngeal or throat swab specimens.

4. Assay Reportable Range: Not applicable.

Specimen Stability

# Nasopharyngeal Swabs

Analytical studies to demonstrate the stability of the applicable BIOFIRE SPOTFIRE R/ST Panel analytes in nasopharyngeal swab matrix in viral transport medium are documented under K213954. The data support the claimed storage conditions for nasopharyngeal swabs for use with the BIOFIRE SPOTFIRE R/ST Panel of up to 4 hours at $1 5 . 2 5 ^ { \circ } \mathrm { C }$ , 3 days at 2-8 $^ \circ \mathrm { C }$ or 30 days at $\le - 1 5 ^ { \circ } \mathrm { C }$ .

# Throat Swabs

An additional study was conducted to support use of liquid Amies medium for transport and storage of throat swabs for testing with the BIOFIRE SPOTFIRE R/ST Panel. Natural throat swab matrix in Amies medium was obtained from asymptomatic volunteers and was spiked with representative target analytes from the Sore Throat Panel at a concentration of 3X LoD and tested either immediately (baseline) or after storage under different conditions. Five different mixtures of target analytes were included in the study. Each was tested 10 times at each time point after storage under the applicable condition. The results are summarized in Table 17. All the analytes were shown to be stable under the conditions tested. The results of this study support the stability of the BIOFIRE SPOTFIRE R/ST Panel analytes in throat swab specimens in liquid Amies medium for up to 4 hours at $2 5 ~ ^ { \circ } \mathrm { C }$ , 3 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or 30 days at $\leq - 1 5 ~ ^ { \circ } \mathrm { C }$ prior to testing.

Table 17. Stability of representative sore throat analytes in liquid Amies medium   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>SourceIdentity</td><td rowspan=1 colspan=5>Percent Positive (n = 10)</td></tr><tr><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=2>8°C</td><td rowspan=1 colspan=1>≤-15°C</td></tr><tr><td rowspan=1 colspan=1>To</td><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=1>2days</td><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>30 days</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>ZeptoMetrix0810062CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL631</td><td rowspan=1 colspan=1>ZeptoMetrix0810228CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>ZeptoMetrix0810162CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>ZeptoMetrix0810538CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>ZeptoMetrix0810526CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 11</td><td rowspan=1 colspan=1>ZeptoMetrix0810014CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>ZeptoMetrix0810040ACF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>ATCC53592</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>ZeptoMetrix0801579</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>ZeptoMetrix0801516</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC12344</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

Coronavirus NL63 and Parainfluenza Virus 1 were included as the representative analytes for the Coronavirus (seasonal) and Parainfluenza Virus assays, respectively.

# Reagent Stability

Reagent shelf-life was established in a Real-time Reagent Stability Study using 3 lots of reagent kits that were stored at $2 3 \pm 2 ^ { \circ } \mathrm { C }$ , corresponding to the upper end of the recommended storage temperature range $( 1 5 . 2 5 ^ { \circ } \mathrm { C } )$ . At each test point, pouch performance was evaluated using a synthetic RNA mixture that contained the targets for each of the analytes on the BIOFIRE SPOTFIRE R/ST Panel, as well as with negative samples. Additional testing was also performed at selected time points with a representative panel of quantified viral and bacterial species to evaluate the effectiveness of nucleic acid extraction and recovery. Testing of reagents exposed to simulated shipping conditions and to verify performance under the specified in-use workflow parameters was also conducted. To-date, the results of these studies support a shelf-life for the BIOFIRE SPOTFIRE R/ST Panel kit from the date of manufacture of 9 months when stored under the recommended conditions.

# 6. Detection Limit:

# Analytical Sensitivity

The Limit of Detection (LoD) for each analyte on the BIOFIRE SPOTFIRE R/ST Panel was determined with samples prepared in Viral Transport Medium and/or liquid Amies Medium (as applicable) containing artificial clinical matrix. A study showing that the formulation of the artificial matrix neither enhanced nor impaired assay performance is described in Section VII (B)(2). The identity and concentration of each viral or bacterial stock used in the study was either certified by the commercial supplier or confirmed by quantitative real-time or

digital PCR and standard methods. An initial titration of each viral or bacterial stock was performed to estimate the LoD, which was then confirmed by testing 20 replicates at the estimated LoD concentration and at one 10-fold lower dilution. The LoD was considered confirmed if $\geq 1 9 / 2 0$ replicates at the estimated LoD concentration and $\leq 1 8 / 2 0$ replicates at a 10-fold lower concentration were reported as Positive. For most analytes, the confirmed LoD was the same with samples prepared in both types of transport medium (Table 18).

Table 18. Summary of confirmed LoD values for the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td colspan="7" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Assay</td><td colspan="1" rowspan="2">Strain/Type</td><td colspan="1" rowspan="2">Source</td><td colspan="4" rowspan="1">Limit of Detection (per mL)</td></tr><tr><td colspan="2" rowspan="1">Viral Transport Medium</td><td colspan="2" rowspan="1">Liquid Amies Medium</td></tr><tr><td colspan="1" rowspan="6">Adenovirus</td><td colspan="1" rowspan="1">Species ASerotype 31</td><td colspan="1" rowspan="1">ZeptoMetrix0810073CF</td><td colspan="1" rowspan="1">0.0041TCID50</td><td colspan="1" rowspan="1">100copies</td><td colspan="1" rowspan="1">0.0082TCID50</td><td colspan="1" rowspan="1">200copies</td></tr><tr><td colspan="1" rowspan="1">Species BSerotype 3</td><td colspan="1" rowspan="1">ZeptoMetrix0810062CF</td><td colspan="1" rowspan="1">0.8TCID50</td><td colspan="1" rowspan="1">840copies</td><td colspan="1" rowspan="1">0.8TCID50</td><td colspan="1" rowspan="1">840copies</td></tr><tr><td colspan="1" rowspan="1">Species CSerotype 2</td><td colspan="1" rowspan="1">WHO I.S.NIBSC 16-324</td><td colspan="1" rowspan="1">820IU</td><td colspan="1" rowspan="1">820copies</td><td colspan="1" rowspan="1">820IU</td><td colspan="1" rowspan="1">820copies</td></tr><tr><td colspan="1" rowspan="1">Species DSerotye 37</td><td colspan="1" rowspan="1">ZeptoMetrix0810119CF</td><td colspan="1" rowspan="1">0.011TCID50</td><td colspan="1" rowspan="1">450copies</td><td colspan="1" rowspan="1">0.011TCID50</td><td colspan="1" rowspan="1">450copies</td></tr><tr><td colspan="1" rowspan="1">Species ESerotype 4</td><td colspan="1" rowspan="1">ZeptoMetrix0810070CF</td><td colspan="1" rowspan="1">0.018TCID50</td><td colspan="1" rowspan="1">10,000copies</td><td colspan="1" rowspan="1">0.018TCID50</td><td colspan="1" rowspan="1">10,000copies</td></tr><tr><td colspan="1" rowspan="1">Species FSerotype 41</td><td colspan="1" rowspan="1">ZeptoMetrix0810085CF</td><td colspan="1" rowspan="1">0.014TCID50</td><td colspan="1" rowspan="1">100copies</td><td colspan="1" rowspan="1">0.014TCID50</td><td colspan="1" rowspan="1">100copies</td></tr><tr><td colspan="1" rowspan="4">Coronavirus(seasonal)</td><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">ATCC VR-740</td><td colspan="1" rowspan="1">0.65TCID50</td><td colspan="1" rowspan="1">11copies</td><td colspan="1" rowspan="1">0.65TCID50</td><td colspan="1" rowspan="1">11copies</td></tr><tr><td colspan="1" rowspan="1">CoronavirusHKU1</td><td colspan="1" rowspan="1">Clinical Specimen</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">18,000copies</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">18,000copies</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">ZeptoMetrix0810024CF</td><td colspan="1" rowspan="1">0.016TCID50</td><td colspan="1" rowspan="1">63copies</td><td colspan="1" rowspan="1">0.016TCID50</td><td colspan="1" rowspan="1">63copies</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">ZeptoMetrix0810228CF</td><td colspan="1" rowspan="1">0.0025TCID50</td><td colspan="1" rowspan="1">4.7copies</td><td colspan="1" rowspan="1">0.0025TCID50</td><td colspan="1" rowspan="1">4.7copies</td></tr><tr><td colspan="1" rowspan="1">CoronavirusSARS-CoV-2</td><td colspan="1" rowspan="1">USA-WA1/2020(heat inactivated)</td><td colspan="1" rowspan="1">ATCC VR-1986HK</td><td colspan="1" rowspan="1">0.11TCID50</td><td colspan="1" rowspan="1">250copies</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td></tr><tr><td colspan="1" rowspan="4">HumanMetapneumovirus</td><td colspan="1" rowspan="1">A1-16 Iowa10/2003</td><td colspan="1" rowspan="1">ZeptoMetrix0810161CF</td><td colspan="1" rowspan="1">3.2TCID50</td><td colspan="1" rowspan="1">240copies</td><td colspan="1" rowspan="1">3.2TCID50</td><td colspan="1" rowspan="1">240copies</td></tr><tr><td colspan="1" rowspan="1">B1-3 Peru2-2002</td><td colspan="1" rowspan="1">ZeptoMetrix0810156CF</td><td colspan="1" rowspan="1">0.25TCID50</td><td colspan="1" rowspan="1">540copies</td><td colspan="1" rowspan="1">0.25TCID50</td><td colspan="1" rowspan="1">96copies</td></tr><tr><td colspan="1" rowspan="1">A2-27 IowaA/2004</td><td colspan="1" rowspan="1">ZeptoMetrix0810164CF</td><td colspan="1" rowspan="1">0.58TCID50</td><td colspan="1" rowspan="1">1,800copies</td><td colspan="1" rowspan="1">0.58TCID50</td><td colspan="1" rowspan="1">1,800copies</td></tr><tr><td colspan="1" rowspan="1">B2-18 IA18-2003</td><td colspan="1" rowspan="1">ZeptoMetrix0810162CF</td><td colspan="1" rowspan="1">2.0TCID50</td><td colspan="1" rowspan="1">770copies</td><td colspan="1" rowspan="1">2.0TCID50</td><td colspan="1" rowspan="1">770copies</td></tr><tr><td colspan="1" rowspan="2">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Human Rhinovirus1A</td><td colspan="1" rowspan="1">ZeptoMetrix0810012CFN</td><td colspan="1" rowspan="1">0.21TCID50</td><td colspan="1" rowspan="1">1.1copies</td><td colspan="1" rowspan="1">0.21TCID50</td><td colspan="1" rowspan="1">1.1copies</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68US/MO/14-18947</td><td colspan="1" rowspan="1">ATCC VR-1823</td><td colspan="1" rowspan="1">11TCID50</td><td colspan="1" rowspan="1">54copies</td><td colspan="1" rowspan="1">11TCID50</td><td colspan="1" rowspan="1">54copies</td></tr><tr><td colspan="1" rowspan="1">Influenza A VirusSubtype H1-2009</td><td colspan="1" rowspan="1">H1N1 pdm A/Michigan/45/15</td><td colspan="1" rowspan="1">ZeptoMetrix0810538CF</td><td colspan="1" rowspan="1">0.82TCID50</td><td colspan="1" rowspan="1">9,200 1copies</td><td colspan="1" rowspan="1">0.82TCID50</td><td colspan="1" rowspan="1">9,200copies</td></tr><tr><td colspan="1" rowspan="1">Influenza A VirusSubtype H3</td><td colspan="1" rowspan="1">H3N2 A/HongKong/ 4801/14</td><td colspan="1" rowspan="1">ZeptoMetrix0810526CF</td><td colspan="1" rowspan="1">0.86TCID50</td><td colspan="1" rowspan="1">340copies</td><td colspan="1" rowspan="1">0.86TCID50</td><td colspan="1" rowspan="1">340copies</td></tr><tr><td colspan="1" rowspan="3">Influenza B Virus</td><td colspan="1" rowspan="1">B/Florida/02/06(Victoria)</td><td colspan="1" rowspan="1">ZeptoMetrix0810037CF</td><td colspan="1" rowspan="1">0.033TCID50</td><td colspan="1" rowspan="1">160copies</td><td colspan="1" rowspan="1">0.033TCID50</td><td colspan="1" rowspan="1">160copies</td></tr><tr><td colspan="1" rowspan="1">B/Nevada/03/2011(Victoria)</td><td colspan="1" rowspan="1">BEI NR-44023</td><td colspan="1" rowspan="1">1.6CEID50</td><td colspan="1" rowspan="1">4.3copies</td><td colspan="1" rowspan="1">1.6CEID50</td><td colspan="1" rowspan="1">4.3copies</td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/06(Yamagata)</td><td colspan="1" rowspan="1">ZeptoMetrix0810255CF</td><td colspan="1" rowspan="1">0.4TCID50</td><td colspan="1" rowspan="1">32copies</td><td colspan="1" rowspan="1">0.4TCID50</td><td colspan="1" rowspan="1">32copies</td></tr><tr><td colspan="1" rowspan="4">ParainfluenzaVirus</td><td colspan="1" rowspan="1">Serotype 1</td><td colspan="1" rowspan="1">ZeptoMetrix0810014CF</td><td colspan="1" rowspan="1">4.6TCID50</td><td colspan="1" rowspan="1">1,400copies</td><td colspan="1" rowspan="1">2.3TCID50</td><td colspan="1" rowspan="1">710copies</td></tr><tr><td colspan="1" rowspan="1">Serotype 2</td><td colspan="1" rowspan="1">ZeptoMetrix0810015CF</td><td colspan="1" rowspan="1">14TCID50</td><td colspan="1" rowspan="1">160copies</td><td colspan="1" rowspan="1">14TCID50</td><td colspan="1" rowspan="1">160copies</td></tr><tr><td colspan="1" rowspan="1">Serotype 3</td><td colspan="1" rowspan="1">ZeptoMetrix0810016CF</td><td colspan="1" rowspan="1">26TCID50</td><td colspan="1" rowspan="1">61copies</td><td colspan="1" rowspan="1">8.8TCID50</td><td colspan="1" rowspan="1">21copies</td></tr><tr><td colspan="1" rowspan="1">Serotype 4</td><td colspan="1" rowspan="1">ZeptoMetrix0810060CF</td><td colspan="1" rowspan="1">200TCID50</td><td colspan="1" rowspan="1">1,100copies</td><td colspan="1" rowspan="1">100TCID50</td><td colspan="1" rowspan="1">570copies</td></tr><tr><td colspan="1" rowspan="2">RespiratorySyncytial Virus</td><td colspan="1" rowspan="1">Type A 2006</td><td colspan="1" rowspan="1">ZeptoMetrix0810040ACF</td><td colspan="1" rowspan="1">0.062TCID50</td><td colspan="1" rowspan="1">22copies</td><td colspan="1" rowspan="1">0.062TCID50</td><td colspan="1" rowspan="1">22copies</td></tr><tr><td colspan="1" rowspan="1">Type B 3/2015Isolate #1</td><td colspan="1" rowspan="1">ZeptoMetrix0810479CF</td><td colspan="1" rowspan="1">0.028TCID50</td><td colspan="1" rowspan="1">24copies</td><td colspan="1" rowspan="1">0.028TCID50</td><td colspan="1" rowspan="1">24copies</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=2 colspan=1>Strain/Type</td><td rowspan=2 colspan=1>Source</td><td rowspan=1 colspan=4>Limit of Detection (per mL)</td></tr><tr><td rowspan=1 colspan=2>Viral Transport Medium</td><td rowspan=1 colspan=2>Liquid Amies Medium</td></tr><tr><td rowspan=1 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>E595</td><td rowspan=1 colspan=1>ZeptoMetrix8801462</td><td rowspan=1 colspan=1>40CFU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>Notapplicable</td></tr><tr><td rowspan=1 colspan=1>Bordetellapertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>ZeptoMetrix0801459</td><td rowspan=1 colspan=1>330CFU</td><td rowspan=1 colspan=1>380copies</td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>Notapplicable</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=1>20IFU</td><td rowspan=1 colspan=1>140 copies</td><td rowspan=1 colspan=1>20IFU</td><td rowspan=1 colspan=1>140copies</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix08801579</td><td rowspan=1 colspan=1>10CCU</td><td rowspan=1 colspan=1>2,100copies</td><td rowspan=1 colspan=1>10CCU</td><td rowspan=1 colspan=1>2,100copies</td></tr><tr><td rowspan=1 colspan=1>Streptococcusdysgalactiae</td><td rowspan=1 colspan=1>ssp. equisimilisZ068</td><td rowspan=1 colspan=1>ZeptoMetrix0801516</td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>330 CFU</td><td rowspan=1 colspan=1>3,400copies</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspyogenes</td><td rowspan=1 colspan=1>SF-130 T1</td><td rowspan=1 colspan=1>ATCC 12344</td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>Notapplicable</td><td rowspan=1 colspan=1>450cells</td><td rowspan=1 colspan=1>2,900copies</td></tr></table>

CCU: Color Changing Units; $\mathrm { C E I D 5 0 }$ : Chicken Egg Infectious Dose- $50 \%$ ; CFU: Colony Forming Units; IU: International Units; IFU: Inclusion Forming Units; $\mathrm { T C I D } 5 0$ : Tissue Culture Infectious Dose- $50 \%$ The original LoD copy number for Influenza A/H1-2009 in VTM cited under K213954 was incorrect due to assignment of an inaccurate concentration to the virus stock. The correct value is shown here. The LoD value expressed in terms of $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ is unchanged.

For efficiency, the LoD study was conducted by co-spiking multiple analytes into each sample. Therefore, additional testing was performed using a representative subset of target analytes to determine whether testing of multiple species in the same reaction had any effect test performance. Samples containing a single analyte were tested in replicates of 6 in parallel with samples containing the same analyte in addition to other viruses/bacteria on the BIOFIRE SPOTFIRE R/ST Panel. Overall, the results of the study showed that there was negligible difference in the analytical sensitivity of the BIOFIRE SPOTFIRE R/ST Panel with individual (single) or multiple target analytes in a sample (Tables 19 and 20).

Table 19. Comparison of results from co-spiked and single spiked nasopharyngeal swab samples   

<table><tr><td rowspan=2 colspan=1>Test Analyte(Source)</td><td rowspan=2 colspan=1>Co-spiked Species</td><td rowspan=2 colspan=1>Test AnalyteConc&quot; per mL</td><td rowspan=1 colspan=2>Positive</td></tr><tr><td rowspan=1 colspan=1>Co-spiked</td><td rowspan=1 colspan=1>Single Spike</td></tr><tr><td rowspan=4 colspan=1>Coronavirus 229E(ATCC VR-740)</td><td rowspan=4 colspan=1>Adenovirus DHuman MetapneumovirusEnterovirusParainfluenza Virus 4</td><td rowspan=1 colspan=1>13TCID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>1.3TCID50</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>0.13TCID50</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>1/6</td></tr><tr><td rowspan=1 colspan=1>0.013TCID50</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=4 colspan=1>Influenza BVictoriaB/Nevada/03/2011(BEI NR-44023)</td><td rowspan=4 colspan=1>Adenovirus ACoronavirus NL63Influenza A H3N2Respiratory Syncytial Virus BBordetella pertussis</td><td rowspan=1 colspan=1>160CEID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>16CEID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>1.6CEID50</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>0.16CEID50</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>4/6</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza Virus 1(ZeptoMetrix08014CF)</td><td rowspan=4 colspan=1>Adenovirus CCoronavirus OC43Human MetapneumovirusMycoplasma pneumoniae</td><td rowspan=1 colspan=1>23TCID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>2.3TCID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>0.23TCID50</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>0/6</td></tr><tr><td rowspan=1 colspan=1>0.023TCID50</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>0/6</td></tr></table>

CEID50: Chicken Egg Infectious Dose- $50 \%$ ; $\mathrm { T C I D } { 5 0 }$ : Tissue Culture Infectious Dose- $50 \%$ Analyte levels at or above the confirmed LoD are shown in red. Results obtained at the lowest level at which all replicates yielded positive results are highlighted in yellow.

Table 20. Comparison of results from co-spiked and single spiked throat swab samples   

<table><tr><td rowspan=2 colspan=1>Analyte(Source)</td><td rowspan=2 colspan=1>Co-spiked Species</td><td rowspan=2 colspan=1>Analyte Level 1</td><td rowspan=1 colspan=2>Positive</td></tr><tr><td rowspan=1 colspan=1>Co-spiked</td><td rowspan=1 colspan=1>Single Spike</td></tr><tr><td rowspan=3 colspan=1>Coronavirus 229E(ATCC VR-740)</td><td rowspan=3 colspan=1>Adenovirus DHuman MetapneumovirusEnterovirusParainfluenza Virus 4</td><td rowspan=1 colspan=1>13TCID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>1.3TCID50</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>0.13TCID50</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>1/6</td></tr><tr><td rowspan=4 colspan=1>Influenza BVictoriaB/Nevada/03/2011(BEI NR-44023)</td><td rowspan=3 colspan=1>Adenovirus ACoronavirus NL63Influenza A H3N2Respiratory Syncytial Virus BBordetella pertussis</td><td rowspan=1 colspan=1>160CEID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>16CEID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>1.6CEID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.16CEID50</td><td rowspan=1 colspan=1>3/6</td><td rowspan=1 colspan=1>2/6</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 1(ZeptoMetrix0810014CF)</td><td rowspan=3 colspan=1>Adenovirus CCoronavirus OC43Human MetapneumovirusMycoplasma pneumoniae</td><td rowspan=1 colspan=1>23TCID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>2.3TCID50</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>0.23TCID50</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>0/6</td></tr></table>

Analyte levels at or above the confirmed LoD are shown in red. Results obtained at the lowest level at which all replicates yielded positive results are highlighted in yellow.

# Inclusivity (Analytical Reactivity)

The inclusivity of the BIOFIRE SPOTFIRE R/ST Panel was evaluated using a combination of in silico analysis of publicly available sequence information and laboratory testing of contrived specimens containing viral and bacterial isolates that were selected to represent phylogenetic, geographic and temporal diversity.

# In silico Analysis of Inclusivity

BLAST searches were conducted to obtain all available gene target sequences for each assay within the BIOFIRE SPOTFIRE R/ST Panel. In silico analysis of the inclusivity of the panel was performed using concatenated alignments of the nested primer sets for each target region. Mismatches to the primers were evaluated for their potential to affect detection of the target analyte based on their number and location within each primer sequence. In silico analysis of the BIOFIRE SPOTFIRE R/ST Panel primers showed that the target sequences are generally well conserved and that the assays are predicted to be robust to known mutations in these regions, with the ability to detect all or nearly all isolates of the targeted analytes.

In silico analysis of the ability to detect SARS-CoV-2 was performed using all available sequences in the GISAID (Global Initiative on Sharing Avian Influenza Data) database as of December 31, 2023. Included in the analysis were 13,682,523 sequences of which $\geq 9 9 . 9 9 \%$ were predicted to be detected by one or both SARS-CoV-2 assays within the BIOFIRE SPOTFIRE R/ST Panel, including sequences of all lineages, Variants of Concern and Variants Under Investigation identified to-date. Impaired sensitivity for detection by both SARS-CoV-2 assays was predicted for $< 0 . 0 0 6 \%$ of sequences evaluated (824/13,682,523).

# Laboratory Testing of Inclusivity

Contrived samples containing individual viruses and bacteria were prepared at approximately 3X LoD in liquid Amies medium containing artificial throat swab matrix and tested in triplicate (unless noted in the tables below). The analytical sensitivity of the BIOFIRE SPOTFIRE R/ST Panel was shown to be similar with artificial throat swab and nasopharyngeal matrices (refer to Table 18, above) and therefore use of artificial throat swab matrix as the representative matrix for analysis of inclusivity was considered acceptable.

All the viral and bacterial isolates tested were successfully detected apart from one strain of S. dysgalactiae subsp. equisimilis of animal origin and one strain of S. pyogenes that are both known to lack the respective regions of the bacterial genome targeted by the BIOFIRE SPOTFIRE R/ST Panel (Tables 21-36). The inability of the BIOFIRE SPOTFIRE R/ST Panel to detect strains of S. dysgalactiae subsp. equisimilis of animal origin and potential for false-negative results due to deletion of the target region in some strains of S. pyogenes are noted as Limitations in the device labeling.

Table 21. Strains of Adenovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Type [Strain]</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810073CF 2</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-863</td><td rowspan=1 colspan=1>12 [Huie/Massachusetts]</td><td rowspan=1 colspan=1>600</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-19</td><td rowspan=1 colspan=1>18 [Washington DC/1954]</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=11 colspan=1>B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810062CF 2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>840</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-7</td><td rowspan=1 colspan=1>7 [Gomen/California/1954]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810021CF</td><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>UIRF</td><td rowspan=1 colspan=1>7d/d2 [Iowa/2001]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-12</td><td rowspan=1 colspan=1>11 [Slobitski/Massachusetts]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-15</td><td rowspan=1 colspan=1>14 [De Wit/Netherlands/1955]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-17</td><td rowspan=1 colspan=1>16 [CH. 79/Saudi Arabia/1955]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1833</td><td rowspan=1 colspan=1>21 [128/Saudi Arabia/1956]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-716</td><td rowspan=1 colspan=1>34 [Compton/1972]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-718</td><td rowspan=1 colspan=1>35 [Holden]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1602</td><td rowspan=1 colspan=1>50 [WaNAmsterdam/1988]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=4 colspan=1>C</td><td rowspan=1 colspan=1>WHO NIBSC 16/324 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>800</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810050CF</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810020CF</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-6</td><td rowspan=1 colspan=1>6 [Tonsil 99]</td><td rowspan=1 colspan=1>2,500</td></tr><tr><td rowspan=3 colspan=1>D</td><td rowspan=1 colspan=1>ZeptoMetrix 0810119CF 2</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>450</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810069CF</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1,400</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810115CF</td><td rowspan=1 colspan=1>20 [KB]</td><td rowspan=1 colspan=1>1,400</td></tr><tr><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>ZeptoMetrix 0810070CF 2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1572</td><td rowspan=1 colspan=1>4 [RI-67/Missouri/1952-1953]</td><td rowspan=1 colspan=1>30,000</td></tr><tr><td rowspan=1 colspan=1>UIRF</td><td rowspan=1 colspan=1>4a [S. Carolina/2004]</td><td rowspan=1 colspan=1>30,000</td></tr><tr><td rowspan=4 colspan=1>F</td><td rowspan=1 colspan=1>ZeptoMetrix 0810085CF 2</td><td rowspan=1 colspan=1>41 [Tak]</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NCPV 0101141v</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1,400 3</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810084CF</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-930</td><td rowspan=1 colspan=1>41 [Tak (73-3544)]</td><td rowspan=1 colspan=1>48 4</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 Not tested at lower levels. 4 4/5 replicates reported Positive; a higher level was not tested.

Table 22. Strains of seasonal Coronavirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity [Strain]</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>ZeptoMetrix 0810228CF 2</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>Amsterdam/2003 BEI NR-470</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=2 colspan=1>229E</td><td rowspan=1 colspan=1>ATCC VR-740 2</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810229CF</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=4 colspan=1>HKU1</td><td rowspan=1 colspan=1>BIOFIRE 03-0447 [Columbus OH, 2016] 2</td><td rowspan=1 colspan=1>18,000 3</td></tr><tr><td rowspan=1 colspan=1>BIOFIRE 1-063 [South Carolina, 2010]</td><td rowspan=1 colspan=1>54,000</td></tr><tr><td rowspan=1 colspan=1>BIOFIRE 04-0394 [Lyon, France, 2016]</td><td rowspan=1 colspan=1>54,000</td></tr><tr><td rowspan=1 colspan=1>BIOFIRE 04-0194 [Lyon, France, 2016]</td><td rowspan=1 colspan=1>54,000</td></tr><tr><td rowspan=2 colspan=1>0C43</td><td rowspan=1 colspan=1>ZeptoMetrix 0810024CF 2</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-759</td><td rowspan=1 colspan=1>190</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 19/20 replicates reported Positive.

Table 23. Strains of SARS-CoV-2 evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1986HK 2</td><td rowspan=1 colspan=1>USA-WA1/2020 (heat inactivated)</td><td rowspan=1 colspan=1>250 3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1991D</td><td rowspan=1 colspan=1>Hong Kong/VM20001061/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1992D</td><td rowspan=1 colspan=1>2019-nCoV/Italy-INMI1</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1994D</td><td rowspan=1 colspan=1>Germany/BavPat1/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-3326D</td><td rowspan=1 colspan=1>USA/CA CDC 5574/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-52499</td><td rowspan=1 colspan=1>England/02/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-52501</td><td rowspan=1 colspan=1>Singapore/2/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-52503</td><td rowspan=1 colspan=1>USA-IL1/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-52505</td><td rowspan=1 colspan=1>USA-AZ1/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-52507</td><td rowspan=1 colspan=1>USA-CA3/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-52510</td><td rowspan=1 colspan=1>Chile/Santiago_op4d1/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>BEI NR-53518</td><td rowspan=1 colspan=1>New York-PV08410/2020</td><td rowspan=1 colspan=1>750</td></tr><tr><td rowspan=1 colspan=1>LGC SeraCare AccuPlex 0505-0298</td><td rowspan=1 colspan=1>Omicron B.1.1.529 Variant</td><td rowspan=1 colspan=1>750</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 19/20 replicates reported Positive.

Table 24. Strains of Human Rhinovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Type [Strain]</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=8 colspan=1>Rhinovirus A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810012CFN 2</td><td rowspan=1 colspan=1>A1 [1A]</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1187</td><td rowspan=1 colspan=1>A77 [130-63]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>A85 [50-525-CV54]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1365</td><td rowspan=1 colspan=1>A34 [137-3]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1600</td><td rowspan=1 colspan=1>A57 [Ch47]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1601</td><td rowspan=1 colspan=1>A7 [68-CV11]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-283</td><td rowspan=1 colspan=1>A16 [11757]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-482</td><td rowspan=1 colspan=1>A2 [HGP]</td><td rowspan=1 colspan=1>11 3</td></tr><tr><td rowspan=6 colspan=1>Rhinovirus B</td><td rowspan=1 colspan=1>ATCC VR-1663</td><td rowspan=1 colspan=1>B17 [33342]</td><td rowspan=1 colspan=1>3.3 4</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-284</td><td rowspan=1 colspan=1>B14 [1059]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1950</td><td rowspan=1 colspan=1>B42[[56822]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-483</td><td rowspan=1 colspan=1>B3 [FEB]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1137</td><td rowspan=1 colspan=1>B27 [5870]</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1193</td><td rowspan=1 colspan=1>B83 [Baylor 7]</td><td rowspan=1 colspan=1>3.3</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 1/3 replicates reported Positive at 3.3 copies/mL. 4 5/5 replicates reported Positive.

Table 25. Strains of Enterovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=3>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=1 colspan=3>Enterovirus D</td><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>ATCC VR-1823 2</td><td rowspan=1 colspan=1>US/MO/14-18947</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=2 colspan=3>Enterovirus A</td><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>ATCC VR-1432</td><td rowspan=1 colspan=1>71H</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus 10</td><td rowspan=1 colspan=1>ATCC VR-168</td><td rowspan=1 colspan=1>NY/1950</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=6 colspan=3>Enterovirus B</td><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>ZeptoMetrix 0810017CF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>ZeptoMetrix 0810023CF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810074CF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>ZeptoMetrix 0810075CF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>ZeptoMetrix 0810076CF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>ZeptoMetrix 0810077CF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=2 colspan=3>Enterovirus C</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>ATCC VR-583</td><td rowspan=1 colspan=1>DN-19 TX/1963</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>ATCC VR-850</td><td rowspan=1 colspan=1>Kuykendall/CA/1952</td><td rowspan=1 colspan=1>160</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 4/5 replicates reported Positive; a higher level was not tested.

Table 26. Strains of Human Metapneumovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Clade</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ZeptoMetrix 0810161CF 2</td><td rowspan=1 colspan=1>Iowa 10/2003</td><td rowspan=1 colspan=1>240</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>ZeptoMetrix 0810160CF</td><td rowspan=1 colspan=1>Iowa 3/2002</td><td rowspan=1 colspan=1>720</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ZeptoMetrix 0810164CF 2</td><td rowspan=1 colspan=1>Iowa 27/2004</td><td rowspan=1 colspan=1>1,800</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>ZeptoMetrix0810163CF</td><td rowspan=1 colspan=1>Iowa 14/2003</td><td rowspan=1 colspan=1>5,400</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810156CF 2</td><td rowspan=1 colspan=1>Peru 2-2002</td><td rowspan=1 colspan=1>540</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ZeptoMetrix 0810158CF</td><td rowspan=1 colspan=1>Peru 3/2003</td><td rowspan=1 colspan=1>1,600</td></tr><tr><td rowspan=4 colspan=1>B2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ZeptoMetrix 0810162CF 2</td><td rowspan=1 colspan=1>IA18-2003</td><td rowspan=1 colspan=1>770</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ZeptoMetrix 0810157CF</td><td rowspan=1 colspan=1>Peru 1/2002</td><td rowspan=1 colspan=1>2,300</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ZeptoMetrix 0810159CF</td><td rowspan=1 colspan=1>Peru 6/2003</td><td rowspan=1 colspan=1>2,300</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>BEI NR-22232</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>2,300</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD.

Table 27. Strains of Influenza A virus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=8 colspan=1>H1N1pdm09</td><td rowspan=1 colspan=1>ZeptoMetrix 0810538CF 2</td><td rowspan=1 colspan=1>Michigan/45/15</td><td rowspan=1 colspan=1>340</td></tr><tr><td rowspan=1 colspan=1>BEI NR-19823</td><td rowspan=1 colspan=1>Netherlands/2629/2009</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>BEI NR-42938</td><td rowspan=1 colspan=1>Georgia/F32551/2012</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>BEI NR-44345</td><td rowspan=1 colspan=1>Hong Kong/H090-761-V1(0)/2009</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810109CFJ</td><td rowspan=1 colspan=1>Canada/6294/2009</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810165CF</td><td rowspan=1 colspan=1>California/07/2009</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810166CF</td><td rowspan=1 colspan=1>Mexico/4108/2009</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810249CF</td><td rowspan=1 colspan=1>Swine/NY/03/2009</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=6 colspan=1>H1N1 3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810036CF</td><td rowspan=1 colspan=1>New Caledonia/20/1999</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810036CFN</td><td rowspan=1 colspan=1>Solomon Islands/3/2006</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810244CF</td><td rowspan=1 colspan=1>Brisbane/59/2007</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-333</td><td rowspan=1 colspan=1>Swine/Iowa/15/1930</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-897</td><td rowspan=1 colspan=1>Swine/A/New Jersey/8/76</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-99</td><td rowspan=1 colspan=1>Swine/1976/1931</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=9 colspan=1>H3N2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810526CF 2</td><td rowspan=1 colspan=1>Hong Kong/4801/14</td><td rowspan=1 colspan=1>340</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-544</td><td rowspan=1 colspan=1>Hong Kong/8/1968</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-547</td><td rowspan=1 colspan=1>Aichi/2/1968</td><td rowspan=1 colspan=1>3,400 4</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-776</td><td rowspan=1 colspan=1>Alice</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-810</td><td rowspan=1 colspan=1>Port Chalmers/1/1973</td><td rowspan=1 colspan=1>1,000 5</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-822</td><td rowspan=1 colspan=1>Victoria/3/1975</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810138CF</td><td rowspan=1 colspan=1>Brisbane/10/2007</td><td rowspan=1 colspan=1>1,000</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810238CF</td><td rowspan=1 colspan=1>Texas/50/2012</td><td rowspan=1 colspan=1>1,100 5</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810252CF</td><td rowspan=1 colspan=1>Wisconsin/67/2005</td><td rowspan=1 colspan=1>1,200</td></tr><tr><td rowspan=1 colspan=1>H1N2 6</td><td rowspan=1 colspan=1>BEI NR-3478</td><td rowspan=1 colspan=1>Recombinant: Kilbourne F63A/NWS/1934 x A/RockefellerInstitute/5/1957 (NA)</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>H10N76</td><td rowspan=1 colspan=1>BEI NR-2765</td><td rowspan=1 colspan=1>Chicken/Germany/N/49</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>H2N2 3</td><td rowspan=1 colspan=1>BEI NR-2775</td><td rowspan=1 colspan=1>Japan/305/1957</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>H5N3 3</td><td rowspan=1 colspan=1>BEI NR-9682</td><td rowspan=1 colspan=1>Duck/Singapore/645/97</td><td rowspan=1 colspan=1>2.2</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 Reported as Influenza A Virus Positive (No Subtype Identified). 4 3/5 replicates reported Positive for Influenza A Virus A/H3 at 1,000 copies/mL. 5 4/5 replicates reported Positive for Influenza A Virus A/H3; a higher level was not tested. 6 Reported as Uncertain: Influenza A Virus.

Table 28. Strains of Influenza B virus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Lineage</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=4 colspan=1>Yamagata</td><td rowspan=1 colspan=1>ZeptoMetrix 0810255CF 2</td><td rowspan=1 colspan=1>Florida/04/06</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810239CF</td><td rowspan=1 colspan=1>2/Massachusetts/2012</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810241CF</td><td rowspan=1 colspan=1>1/Wisconsin/2010</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810256CF</td><td rowspan=1 colspan=1>07/Florida/2004</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=5 colspan=1>Victoria</td><td rowspan=1 colspan=1>ZeptoMetrix 0810037CF 2</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>BEI NR-44023 2</td><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-823</td><td rowspan=1 colspan=1>5/Hong Kong/1972</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>CDC 2005743348</td><td rowspan=1 colspan=1>1/0hio/2005</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810258CF</td><td rowspan=1 colspan=1>2506/Malaysia/2004</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=6 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC VR-101</td><td rowspan=1 colspan=1>Lee/1940</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-102</td><td rowspan=1 colspan=1>Allen/1945</td><td rowspan=1 colspan=1>13 3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-103</td><td rowspan=1 colspan=1>GL/1739/1954</td><td rowspan=1 colspan=1>13 4</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-295</td><td rowspan=1 colspan=1>2/Taiwan/1962</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-296</td><td rowspan=1 colspan=1>1/Maryland/1959</td><td rowspan=1 colspan=1>13 3</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-786</td><td rowspan=1 colspan=1>Brigit/Russia/1969</td><td rowspan=1 colspan=1>13 3</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 4/5 replicates reported Positive; a higher level was not tested. 4 5/5 replicates reported Positive.

Table 29. Strains of Parainfluenza Virus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810014CF 2</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>710</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>C-35/1957</td><td rowspan=1 colspan=1>2,100</td></tr><tr><td rowspan=1 colspan=1>BEI NR-48680</td><td rowspan=1 colspan=1>FRA/29221106/2009</td><td rowspan=1 colspan=1>2,100</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810015CF 2</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>Greer/1955</td><td rowspan=1 colspan=1>480 3</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810016CF 2</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>C-243/1957</td><td rowspan=1 colspan=1>210 4</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3233</td><td rowspan=1 colspan=1>NIH 47885 Wash/47885/57</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 4A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810060CF 2</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>570</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1378</td><td rowspan=1 colspan=1>M-25/1958</td><td rowspan=1 colspan=1>1,700</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 4B</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>CH-19503/1962</td><td rowspan=1 colspan=1>1,700</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810060BCF</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>1,700</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 4/5 replicates reported Positive; a higher level was not tested. 4 2/5 replicates reported Positive at 63 copies/mL.

Table 30. Strains of Respiratory Syncytial Virus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=5 colspan=1>A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810040ACF 2</td><td rowspan=1 colspan=1>2006</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-26</td><td rowspan=1 colspan=1>Long/Maryland/1956</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>A2/Melbourne/1961</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810474CF</td><td rowspan=1 colspan=1>2/2015/ Isolate #2</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810452CF</td><td rowspan=1 colspan=1>12/2014 Isolate #2</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810479CF 2</td><td rowspan=1 colspan=1>3/2015 Isolate #1</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810040CF</td><td rowspan=1 colspan=1>Ch-93 (18)-18</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>WV/14617/1985</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>9320/Massachusetts/1977</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>18537/WashingtonDC/1962</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810451CF</td><td rowspan=1 colspan=1>11/2014 Isolate #2</td><td rowspan=1 colspan=1>72</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD.

Table 31. Strains of $B$ . parapertussis evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(per/mL) 1</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801462 2</td><td rowspan=1 colspan=1>E595</td><td rowspan=1 colspan=1>41 CFU</td></tr><tr><td rowspan=1 colspan=1>ATCC 9305</td><td rowspan=1 colspan=1>517</td><td rowspan=1 colspan=1>230 cells</td></tr><tr><td rowspan=1 colspan=1>ATCC 53892</td><td rowspan=1 colspan=1>PT28G</td><td rowspan=1 colspan=1>15 cells</td></tr><tr><td rowspan=1 colspan=1>ATCC 53893</td><td rowspan=1 colspan=1>PT 26/28G</td><td rowspan=1 colspan=1>17 cells</td></tr><tr><td rowspan=1 colspan=1>ATCC 15237</td><td rowspan=1 colspan=1>NCTC 10853</td><td rowspan=1 colspan=1>83 CFU</td></tr><tr><td rowspan=1 colspan=1>ATCC 15311</td><td rowspan=1 colspan=1>NCTC 5952</td><td rowspan=1 colspan=1>13 cells</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-587</td><td rowspan=1 colspan=1>12822/Germany/1993</td><td rowspan=1 colspan=1>120 CFU</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801461</td><td rowspan=1 colspan=1>A747</td><td rowspan=1 colspan=1>4.4 CFU</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801643</td><td rowspan=1 colspan=1>C510</td><td rowspan=1 colspan=1>200 CFU</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801644</td><td rowspan=1 colspan=1>E838</td><td rowspan=1 colspan=1>340 CFU 3</td></tr></table>

CFU: Colony Forming Units 1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 4/5 replicates reported Positive; a higher level was not tested.

Table 32. Strains of $B$ . pertussis evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801459 2</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>380</td></tr><tr><td rowspan=1 colspan=1>ATCC 10380</td><td rowspan=1 colspan=1>10-536</td><td rowspan=1 colspan=1>1,100 3</td></tr><tr><td rowspan=1 colspan=1>ATCC 51445</td><td rowspan=1 colspan=1>CNCTC Hp 12/63 [623]</td><td rowspan=1 colspan=1>1,100</td></tr><tr><td rowspan=1 colspan=1>ATCC 8467</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>1,100</td></tr><tr><td rowspan=1 colspan=1>ATCC 9340</td><td rowspan=1 colspan=1>5 [17921]</td><td rowspan=1 colspan=1>1,100</td></tr><tr><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>18323 [NCTC 10739]</td><td rowspan=1 colspan=1>1,100</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1335</td><td rowspan=1 colspan=1>MN2531</td><td rowspan=1 colspan=1>1,100</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-589</td><td rowspan=1 colspan=1>Tohama</td><td rowspan=1 colspan=1>1,100</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801460</td><td rowspan=1 colspan=1>E431</td><td rowspan=1 colspan=1>1,100</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 5/5 replicates reported Positive.

Table 33. Strains of $C .$ . pneumoniae evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=1 colspan=1>ATCC 53592 2</td><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>140</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1310</td><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>420</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1356</td><td rowspan=1 colspan=1>TWAR strain 2023</td><td rowspan=1 colspan=1>420</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1360</td><td rowspan=1 colspan=1>CM-1</td><td rowspan=1 colspan=1>420</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1435</td><td rowspan=1 colspan=1>J-21</td><td rowspan=1 colspan=1>420</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1452</td><td rowspan=1 colspan=1>A03</td><td rowspan=1 colspan=1>420</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>TWAR strain, TW-183</td><td rowspan=1 colspan=1>420</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD.

Table 34. Strains of $M .$ . pneumoniae evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810159 2</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>2,100</td></tr><tr><td rowspan=1 colspan=1>ATCC 29085</td><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=1 colspan=1>ATCC 29342</td><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 15492</td><td rowspan=1 colspan=1>Mac</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=1 colspan=1>ATCC 15531-TTR</td><td rowspan=1 colspan=1>FH strain of Eaton Agent [NCTC 10119)</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=4 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC 15293</td><td rowspan=1 colspan=1>M52</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=1 colspan=1>ATCC 15377</td><td rowspan=1 colspan=1>Bru</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=1 colspan=1>ATCC 39505</td><td rowspan=1 colspan=1>Mutant 22</td><td rowspan=1 colspan=1>6,300</td></tr><tr><td rowspan=1 colspan=1>ATCC 49894</td><td rowspan=1 colspan=1>UTMP-10P</td><td rowspan=1 colspan=1>6,300</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD.

Table 35. Strains of S. dysgalactiae subsp. equisimilis evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL) 1</td></tr><tr><td rowspan=8 colspan=1>Human</td><td rowspan=1 colspan=1>ZeptoMetrix 0801516 2</td><td rowspan=1 colspan=1>Z068</td><td rowspan=1 colspan=1>3,400</td></tr><tr><td rowspan=1 colspan=1>ATCC 12388</td><td rowspan=1 colspan=1>C74</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>NCTC 8543</td><td rowspan=1 colspan=1>655 Chestle</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>CCUG 45841</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>CCUG 45898</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>ATCC 12394</td><td rowspan=1 colspan=1>D166B</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>ATCC 9542</td><td rowspan=1 colspan=1>1180</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-337</td><td rowspan=1 colspan=1>AC-2074</td><td rowspan=1 colspan=1>10,000</td></tr><tr><td rowspan=1 colspan=1>Animal (horse)</td><td rowspan=1 colspan=1>ATCC 10009</td><td rowspan=1 colspan=1>13-166</td><td rowspan=1 colspan=1>Not applicable 3</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 Reported as Negative at up to 100,000 copies $/ \mathrm { m L }$ due to a deletion of the target region. This isolate is from an animal source (horse), all other isolates of $S .$ . dysgalactiae subsp. equisimilis tested were of human origin. BIOFIRE has determined that animal isolates of $S _ { \cdot }$ . dysgalactiae subsp. equisimilis generally lack the region of the genome targeted by the BIOFIRE SPOTFIRE R/ST Panel. Additional animal isolates of S. dysgalactiae subsp. equisimilis were tested as part of the Crossreactivity Study as described in Section VII (A)(3) and none were detected. The labeling for the BIOFIRE SPOTFIRE R/ST Panel indicates that the assay detects isolates of $S _ { \cdot }$ . dysgalactiae subsp. equisimilis of human but not animal origin.

Table 36. Strains of S. pyogenes evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>M type</td><td rowspan=1 colspan=1>Source Identity</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(per/mL)1</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 12344 2</td><td rowspan=1 colspan=1>SF-130 T-type 1</td><td rowspan=1 colspan=1>2,900 copies450 cells</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-947</td><td rowspan=1 colspan=1>MGAS 5005 M type-1</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC 700294</td><td rowspan=1 colspan=1>SF370 M1 GAS M-type 1 Type-1</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 12384</td><td rowspan=1 colspan=1>C203 T-type 3</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-595</td><td rowspan=1 colspan=1>MGAS 315, M type 3</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC 51500</td><td rowspan=1 colspan=1>M-3 [DLS 88002, Weller]</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>NCTC 8193</td><td rowspan=1 colspan=1>T5 B</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC 12348</td><td rowspan=1 colspan=1>S43 T-type 6</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC BAA-1065</td><td rowspan=1 colspan=1>MGAS 2096 [A374]</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC 12356</td><td rowspan=1 colspan=1>Typing Strain J17E [A. Coburn R9]</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC BAA-572</td><td rowspan=1 colspan=1>MGAS 8232</td><td rowspan=1 colspan=1>1,400 cells 3</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>ATCC 8133</td><td rowspan=1 colspan=1>Typing Strain T23 [F. Griffith strain Barts 102]</td><td rowspan=1 colspan=1>1,400 cells 3</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>ATCC 12360</td><td rowspan=1 colspan=1>Typing Strain J17F [A. Coburn R17]</td><td rowspan=1 colspan=1>8,700 copies 4</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>NCTC 8310</td><td rowspan=1 colspan=1>Coggins</td><td rowspan=1 colspan=1>1,400 cells 3</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>NCTC 8195</td><td rowspan=1 colspan=1>Quinn (D24/46)</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>NCTC 8229</td><td rowspan=1 colspan=1>C 95/12B</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>ATCC 12372</td><td rowspan=1 colspan=1>Typing Strain C143 [C143]</td><td rowspan=1 colspan=1>1,400 cells 3</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>ATCC 12377</td><td rowspan=1 colspan=1>C105 [20RS14]</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>NCTC 10880</td><td rowspan=1 colspan=1>R66/3489 (378)</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=6 colspan=1>Not applicable</td><td rowspan=1 colspan=1>ATCC 19615</td><td rowspan=1 colspan=1>Bruno</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC 49399</td><td rowspan=1 colspan=1>AC A62</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC 700466</td><td rowspan=1 colspan=1>CDC-SS-872 [R67/3884]</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC 25663</td><td rowspan=1 colspan=1>[P20080]</td><td rowspan=1 colspan=1>8,700 copies</td></tr><tr><td rowspan=1 colspan=1>ATCC 21060</td><td rowspan=1 colspan=1>Su</td><td rowspan=1 colspan=1>Not applicable 5</td></tr><tr><td rowspan=1 colspan=1>ATCC 49117</td><td rowspan=1 colspan=1>397</td><td rowspan=1 colspan=1>8,700 copies</td></tr></table>

1 Confirmed LoD concentration or lowest concentration at which all replicates reported Positive if $< 2 0$ replicates tested. 2 Isolate used to determine the LoD. 3 Molecular quantification could not be performed. The level tested corresponds to ${ \sim } 3 \mathrm { X }$ LoD for ATCC 12344 as determined in cells/mL. 4 4/5 replicates reported Positive. 5 Reported as Negative at up to 100,000 copies $\mathrm { \Delta } / \mathrm { m L }$ due to deletion of the target region. This is noted in the device labeling.

# 7. Assay Cut-Off:

DNA melt curve analysis is used to determine the presence or absence of the specific amplification product(s) corresponding to each analyte within the BIOFIRE SPOTFIRE R/ST Panel. The Melt Detector software analyzes each well of the second stage PCR array independently. A series of mathematical tests is applied to each fluorescence curve to discriminate target-specific amplicon from system noise and non-specific amplification products, accounting for both system variation and known sequence variation within the amplified regions targeted by the assay. The Melt Detector algorithm was developed by tuning against a large data set comprised of both typical and atypical melt curves that were assigned as positive or negative by manual review performed by expert users. The tuning process optimized the sensitivity and specificity of the algorithm in comparison to manual result interpretation and included optimization of the order of procedural steps and the respective threshold for each parameter.

For each assay within the BIOFIRE SPOTFIRE R/ST Panel, the expected melting temperature (mean Tm) was determined empirically using the most common sequence variants. The acceptable range around the mean was then determined through a combination of empirical testing of known sequence variants and mathematical modeling. Reference data obtained from analysis of templates with known sequences were used to estimate additional sources of system variability, including the BIOFIRE SPOTFIRE instrument and reagents. The melt range parameters for each analyte were validated in the subsequent Analytical and Clinical Studies.

8. Accuracy (Instrument):

# Refer to Section VII (C) Clinical Studies.

# 9. Carry-Over:

The potential for sample-to-sample cross-contamination during operation of the BIOFIRE SPOTFIRE System was evaluated by testing an alternating series of positive and negative contrived samples using the same Pouch Loading Station. The positive samples comprised high levels of representative on-panel analytes spiked separately into artificial throat swab or artificial nasopharyngeal swab matrix ( $\mathrm { \tilde { n } } = 5$ per representative analyte). Negative samples comprised artificial matrix alone $( \mathtt { n } = 2 1$ ). The expected positive results were obtained from pouches containing the target analytes at high target levels. No false-positive results were obtained from these pouches for any other analytes and all true negative samples that contained none of the target analytes produced the expected negative results. These results demonstrated that there is an acceptable, low likelihood of cross-contamination between samples when the SPOTFIRE R/ST Panel is performed according to the instructions for use.

Table 37. Results from evaluation of the potential for cross-contamination on the BIOFIRE SPOTFIRE System   

<table><tr><td rowspan=2 colspan=1>Analyte/Strain</td><td rowspan=1 colspan=2>High Positive Target Level</td><td rowspan=1 colspan=2>SPOTFIRE R/ST Panel Result/Number Tested</td></tr><tr><td rowspan=1 colspan=1>Concentration(per mL)</td><td rowspan=1 colspan=1>Multiple of LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative 4</td></tr><tr><td rowspan=1 colspan=1>EnterovirusATCC VR-1823 1</td><td rowspan=1 colspan=1>1.6 x 107 TCID50</td><td rowspan=1 colspan=1>1.5 x 106</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>36/36</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2Zeptometrix0810526CF 1</td><td rowspan=1 colspan=1>7.2 x 104 TCID50</td><td rowspan=1 colspan=1>8.4 x 104</td><td rowspan=1 colspan=1>5/5 3</td><td rowspan=1 colspan=1>36/36</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2ATCC VR-1986HK 2</td><td rowspan=1 colspan=1>8.4 x102 TCID50</td><td rowspan=1 colspan=1>7.6 x 103</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>36/36</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC 53592 1</td><td rowspan=1 colspan=1>2.9 x 106 IFU</td><td rowspan=1 colspan=1>1.5 x 105</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>36/36</td></tr></table>

IFU: Inclusion Forming Units; TCID50: Tissue Culture Infectious Dose- $50 \%$ 1 Prepared in artificial throat swab matrix 2 Prepared in artificial nasopharyngeal swab matrix 3 1 positive/negative pair of samples was retested due to an Invalid result with the positive sample on initial testing. 4 Denominator includes 21 samples that were negative for all analytes and tested in alternating fashion with positive samples (including 1 negative sample that was retested), plus 15 samples that were spiked with a single analyte $\mathrm { i } \mathtt { n } = 5$ ea.) but which were negative for the target analyte listed.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison:

# Artificial vs Natural Matrix

The BIOFIRE SPOTFIRE R/ST Panel is intended to detect multiple viral and bacterial analytes in nasopharyngeal and throat swab specimens. The high frequency of positive samples for at least one analyte makes it impractical to use natural clinical matrices for analytical validation studies for which large volumes of negative matrix are required. To address this, artificial nasopharyngeal and throat swab matrices were used for most of the analytical studies and therefore additional testing was conducted to demonstrate that the analytical sensitivity of the BIOFIRE SPOTFIRE R/ST Panel was similar with samples prepared in artificial and natural specimen matrices, as described below.

Artificial nasopharyngeal swab and artificial throat swab matrices were comprised of glycerol, porcine mucin, human whole blood, phosphate buffered saline and saline diluted in transport medium (Viral Transport Medium [VTM] for nasopharyngeal swab or liquid Amies medium for throat swab matrices, respectively). Testing was performed using a representative subset of analytes targeted by the BIOFIRE SPOTFIRE R/ST Panel, including DNA and RNA viruses and one Gram positive bacterial species. Ten-fold serial dilutions of 6 representative species were prepared in the artificial matrices and, in parallel, in the respective natural clinical matrices obtained from asymptomatic donors. Prior to conducting the study, the natural clinical specimens were tested using the BIOFIRE SPOTFIRE R/ST Panel to verify that none of the on-panel analytes were present. A summary of the results is presented in Table 38.

Positive results were obtained for all replicates with all 6 analytes when tested at target levels close to $( \leq \sim 3 \mathrm { X } )$ the claimed LoD in the applicable artificial and natural matrices. The same was true to the comparison of natural and artificial nasopharyngeal swab matrices, except for Adenovirus B for which only $4 / 6$ replicates $( 6 7 \% )$ in natural clinical matrix were reported positive at the claimed LoD concentration. However, amplification curve characteristics were similar between positive replicates in both the artificial and natural nasopharyngeal swab matrices at the LoD target level and all replicates in both matrices were positive at a 10-fold higher concentration, indicating that the analytical sensitivity in both matrices was similar.

Overall, the results of the matrix equivalency study showed that the use of artificial nasopharyngeal or throat swab matrix does not enhance or impair device performance and is acceptable for analytical studies in lieu of natural clinical matrix.

Table 38. Comparison of analytical sensitivity in artificial and natural matrices   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>Strain</td><td rowspan=3 colspan=1>Source</td><td rowspan=3 colspan=1>Units</td><td rowspan=3 colspan=1>Per mL</td><td rowspan=3 colspan=1>X LoD</td><td rowspan=1 colspan=4>Positive/Tested (%)</td></tr><tr><td rowspan=1 colspan=2>NasopharyngealSwab</td><td rowspan=1 colspan=2>Throat Swab</td></tr><tr><td rowspan=1 colspan=1>Artificial</td><td rowspan=1 colspan=1>Natural</td><td rowspan=1 colspan=1>Artificial</td><td rowspan=1 colspan=1>Natural</td></tr><tr><td rowspan=4 colspan=1>Adenovirus B</td><td rowspan=4 colspan=1>Type 3</td><td rowspan=4 colspan=1>ZeptoMetrix0810062CF</td><td rowspan=4 colspan=1>TCID50</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>4/6(67)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.082</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>1/6(17)</td><td rowspan=1 colspan=1>1/6(17)</td><td rowspan=1 colspan=1>2/6(33)</td><td rowspan=1 colspan=1>2/6(33)</td></tr><tr><td rowspan=1 colspan=1>0.0082</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>1/6(17)</td></tr><tr><td rowspan=4 colspan=1>Influenza A</td><td rowspan=4 colspan=1>SubtypeH1-2009Michigan/45/15</td><td rowspan=4 colspan=1>ZeptoMetrix0810538CF</td><td rowspan=4 colspan=1>TCID50</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.082</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>1/6(17)</td><td rowspan=1 colspan=1>2/6(33)</td><td rowspan=1 colspan=1>3/6(50)</td><td rowspan=1 colspan=1>1/6(17)</td></tr><tr><td rowspan=1 colspan=1>0.0082</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>0/6(0)</td></tr><tr><td rowspan=4 colspan=1>Influenza B</td><td rowspan=4 colspan=1>Florida/02/06</td><td rowspan=4 colspan=1>ZeptoMetrix08037CF</td><td rowspan=4 colspan=1>TCID50</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5/5(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.033</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>0.0033</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>1/6(17)</td><td rowspan=1 colspan=1>2/6(17)</td><td rowspan=1 colspan=1>3/6(50)</td></tr><tr><td rowspan=4 colspan=1>SARS-CoV-2</td><td rowspan=4 colspan=1>USA-WA1/2020 (heatinactivated)</td><td rowspan=4 colspan=1>ATCCVR-1986HK</td><td rowspan=4 colspan=1>copies</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5/5(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>2/6(33)</td><td rowspan=1 colspan=1>2/6(33)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>0/6(0)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=4 colspan=1>Streptococcuspyogenes</td><td rowspan=4 colspan=1>SF-130 T1</td><td rowspan=4 colspan=1>ATCC 12344</td><td rowspan=4 colspan=1>cells</td><td rowspan=1 colspan=1>15,000</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>1,500</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6/6(100)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5/6(83)</td><td rowspan=1 colspan=1>6/6(100)</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/6(50)</td><td rowspan=1 colspan=1>1/6(17)</td></tr></table>

X LoD: Multiple of confirmed Limit of Detection; N/A: Not Applicable (results for SARS-CoV-2 are not reported from throat swabs and those for S. pyogenes are not reported from nasopharyngeal swabs). Results from target levels below the claimed LoD of each analyte are shaded.

# Fresh vs Frozen Samples

Because evaluation of the clinical performance of the BIOFIRE SPOTFIRE R/ST Panel required testing of archived frozen samples, a study was conducted to determine whether freezing and thawing of throat swab specimens would affect detection of representative target analytes. A similar study was previously performed with nasopharyngeal swab matrix under K160068 for the BioFire FilmArray Respiratory Panel and showed no adverse effects from freezing and thawing of specimens.

Contrived throat swab specimens were prepared by co-spiking representative target analytes into natural throat swab matrix in Amies medium that was known to be negative for the species under investigation. Half the contrived specimens were prepared at 2X LoD, and the remainder were spiked at levels ranging from 10 to 1,000X LoD (except for Influenza A H3 which was spiked at levels ranging from 0.2 to 79X LoD). Each contrived specimen was split into two aliquots, one of which was tested fresh and the second of which was tested after a single freeze-thaw cycle that included $\geq 2 4$ hours at $\mathbf { - 7 0 } \ ^ { \circ } \mathrm { C }$ . The results from testing paired fresh and frozen throat swab samples are summarized in Table 39. Positive and negative agreement ranged from 96.0 to $100 \%$ and 98.2 to $100 \%$ , respectively, and thus support the stability of specimens that are subject to a freeze-thaw cycle.

Table 39. Summary of results from testing paired fresh and frozen contrived throat swab specimens   

<table><tr><td rowspan=2 colspan=1>Analyte[Strains/Isolates Tested]</td><td rowspan=1 colspan=4>% Agreement (Frozen vs Fresh)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>95% ConfidenceInterval </td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>95% ConfidenceInterval </td></tr><tr><td rowspan=1 colspan=1>Adenovirus [5]</td><td rowspan=1 colspan=1>100(49/49)</td><td rowspan=1 colspan=1>92.7-100</td><td rowspan=1 colspan=1>99.1(112/113) 2</td><td rowspan=1 colspan=1>99.1-95.2</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal) [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus [0] 3</td><td rowspan=1 colspan=1>0(0/1)</td><td rowspan=1 colspan=1>0.0-79.3</td><td rowspan=1 colspan=1>100(161/161)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009 [3]</td><td rowspan=1 colspan=1>100(31/31)</td><td rowspan=1 colspan=1>89.0-100</td><td rowspan=1 colspan=1>100(130/130)</td><td rowspan=1 colspan=1>97.1-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2 [2]</td><td rowspan=1 colspan=1>100(20/20) 4</td><td rowspan=1 colspan=1>83.9-100</td><td rowspan=1 colspan=1>100(141/141)</td><td rowspan=1 colspan=1>97.3-100</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus [5] 5</td><td rowspan=1 colspan=1>100(48/48)</td><td rowspan=1 colspan=1>92.6-100</td><td rowspan=1 colspan=1>98.2(112/114) 6</td><td rowspan=1 colspan=1>93.8-99.5</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus [5]7</td><td rowspan=1 colspan=1>96.0(48/50) 8</td><td rowspan=1 colspan=1>86.5-100</td><td rowspan=1 colspan=1>100(112/112)</td><td rowspan=1 colspan=1>96.7-100</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae [5]</td><td rowspan=1 colspan=1>100(50/50)</td><td rowspan=1 colspan=1>92.9-100</td><td rowspan=1 colspan=1>100(112/112)</td><td rowspan=1 colspan=1>96.7-100</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae [0]</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>100(162/162)</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes [5]</td><td rowspan=1 colspan=1>100(51/51)</td><td rowspan=1 colspan=1>93.0-100</td><td rowspan=1 colspan=1>100(111/111)</td><td rowspan=1 colspan=1>96.7-100</td></tr></table>

1 2-sided $9 5 \%$ score confidence interval. 2 1 sample at 2X LoD was reported negative when tested fresh. 3 1 un-spiked sample produced a positive result for Human Rhinovirus/Enterovirus when tested fresh but was negative when tested after freeze-thaw. 4 Although most samples were prepared at target levels below the claimed LoD, in all cases, positive results were obtained with both the fresh and frozen aliquots. 5 All positive samples were prepared with strains of Parainfluenza Virus type 2.

In addition to testing contrived specimens, the effect of a freeze-thaw cycle was also evaluated using 16 clinical throat swab specimens that were tested fresh during the prospective Clinical Study described in Section VII (C)(1) and subsequently retested from a frozen aliquot. All 16 specimens produced concordant results for all analytes on the BIOFIRE SPOTFIRE R/ST Panel (Table 40).

Table 40. Summary of results from testing fresh vs frozen natural clinical throat swab specimens   

<table><tr><td rowspan=2 colspan=1>Analyte 1</td><td rowspan=1 colspan=4>% Agreement (Frozen vs Fresh)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>95% ConfidenceInterval2</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>100(3/3)</td><td rowspan=1 colspan=1>43.9-100</td><td rowspan=1 colspan=1>100(13/13)</td><td rowspan=1 colspan=1>77.2-100</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal) 3</td><td rowspan=1 colspan=1>100(1/1)</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>100(15/15)</td><td rowspan=1 colspan=1>79.6-100</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>100(5/5)</td><td rowspan=1 colspan=1>56.6-100</td><td rowspan=1 colspan=1>100(11/11)</td><td rowspan=1 colspan=1>72.3-100</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>100(6/6)</td><td rowspan=1 colspan=1>61.0-100</td><td rowspan=1 colspan=1>100(10/10)</td><td rowspan=1 colspan=1>72.3-100</td></tr></table>

1 There was $1 0 0 \%$ $( 8 0 . 6 \mathrm { - } 1 0 0 \% )$ ) Negative Agreement (16/16) for all other analytes not listed below. 2 2-sided $9 5 \%$ score confidence interval. 3 Positive for seasonal Coronavirus OC43 by the comparator method.

In combination, the above results indicate that the integrity of throat swab specimens for use with the BIOFIRE SPOTFIRE R/ST Panel is unlikely to be affected by a single freeze-thaw cycle.

Based on the above information, use of frozen nasopharyngeal and throat swab specimens to evaluate the performance of the BIOFIRE SPOTFIRE R/ST Panel was considered acceptable.

# Transport Media Equivalency

The compatibility of the BIOFIRE SPOTFIRE R/ST Panel with specimens collected in alternative transport media $0 . 9 \%$ normal saline, Hank’s Balanced Salt Solution (Gibco), BD Universal Viral Transport (UVT) medium (BD Medical) and Remel MicroTest M4RT (Thermo Fisher) was demonstrated under K213954. Because the reagent formulations and workflow parameters for the BIOFIRE SPOTFIRE R and R/ST Panels are the same, these data also support the compatibility of these alternative transport media with the SPOTFIRE R/ST Panel.

# C Clinical Studies:

# 1. Clinical Sensitivity:

The clinical performance of the BIOFIRE SPOTFIRE R/ST Panel was evaluated using a combination of prospectively collected, archived and contrived specimens as described below.

Prospectively Collected Specimens

A Clinical Study with prospectively collected specimens was conducted at six sites (5 U.S. and 1 ex-U.S.), including adult and pediatric emergency departments/urgent care centers, that were considered representative of the intended use settings for the BIOFIRE SPOTFIRE System and BIOFIRE SPOTFIRE R/ST Panel. The specimens included in the study were either collected under Informed Consent or with parental permission and individual assent (if appropriate) for minors $< 1 8$ years of age or were leftover (residual) samples from standard of care testing. The inclusion and exclusion criteria for the study are summarized in Table 41. Specimen enrollment took place over two periods: both nasopharyngeal and throat swab specimens were enrolled between December 2020 and September 2021 and enrollment of throat swab specimens was continued from September 2022 to May 2023.

Table 41. Inclusion and exclusion criteria for the Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=3>Specimens Collected Under Informed Consent 1, 2</td></tr><tr><td rowspan=6 colspan=1>Inclusion Criteria</td><td rowspan=3 colspan=1>Respiratory Illness</td><td rowspan=1 colspan=1>Subject presented with signs/symptoms of respiratoryinfection included but not limited to fever, cough, sore throat,runny nose, myalgia, headache, chills or fatigue</td></tr><tr><td rowspan=1 colspan=1>If ≥ 18 years of age subject provided Informed Consent 1If &lt; 18 years of age, parental permission and assent obtained(as appropriate) 1</td></tr><tr><td rowspan=1 colspan=1>Subject willing to provide 1 nasopharyngeal swab specimen</td></tr><tr><td rowspan=3 colspan=1>Pharyngitis</td><td rowspan=1 colspan=1>Subject presented with chief complaint of pharyngitis (i.e.,sore throat) or, if a child, fever, headache, chills or nausea</td></tr><tr><td rowspan=1 colspan=1>If ≥ 18 years of age subject provided Informed Consent 1If &lt; 18 years of age, parental permission and assent obtained(as appropriate) 1</td></tr><tr><td rowspan=1 colspan=1>Subject willing to provide 1 throat swab specimen</td></tr><tr><td rowspan=3 colspan=1>Exclusion Criteria</td><td rowspan=1 colspan=2>Subject unable or unwilling to provide Informed Consent or assent (as appropriate)</td></tr><tr><td rowspan=1 colspan=1>Subject unable or unwil</td><td rowspan=1 colspan=1>Subject unable or unwilling to provide the required specimens</td></tr><tr><td rowspan=1 colspan=2>Subject&#x27;s healthcare provider determined that specimen collection represented anunacceptable health risk</td></tr><tr><td rowspan=1 colspan=3>Residual Specimens 3</td></tr><tr><td rowspan=6 colspan=1>Inclusion Criteria</td><td rowspan=3 colspan=1>Respiratory Illness</td><td rowspan=1 colspan=1>Residual nasopharyngeal swab specimen in transport mediumor saline 4 leftover from standard of care testing under aphysician order for analysis for respiratory pathogens</td></tr><tr><td rowspan=1 colspan=1>Specimen held at room temperature for ≤ 4 hours or at 4 °Cfor ≤ 72 hours</td></tr><tr><td rowspan=1 colspan=1>≥ 1.5 mL specimen volume available</td></tr><tr><td rowspan=3 colspan=1>Pharyngitis</td><td rowspan=1 colspan=1>Residual throat swab specimen in transport medium or salineleftover from standard of care testing under a physician orderfor analysis for pharyngitis pathogens</td></tr><tr><td rowspan=1 colspan=1>Specimen held at room temperature ≤ 4 hours or at 4 °C for ≤48 hours</td></tr><tr><td rowspan=1 colspan=1>≥ 0.7 mL specimen volume available</td></tr><tr><td rowspan=3 colspan=1>Exclusion Criteria</td><td rowspan=1 colspan=2>Specimen could not be tested within the specified storage parameters</td></tr><tr><td rowspan=1 colspan=2>Insufficient specimen volume for testing</td></tr><tr><td rowspan=1 colspan=2>Type of transport medium not known</td></tr></table>

1 The Institutional Review Board at one site waived the need for a signed Informed Consent form. 2 Includes specimens from minors collected with parental consent and child assent (as applicable). 3 Residual specimens were exempt from Informed Consent requirements in accordance with FDA’s guidance document “Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable” (April 2006). 4 None of the specimens enrolled in the study were collected in saline. According to the device labeling, saline is not a recommended transport medium for use with the BIOFIRE SPOTFIRE R/ST Panel.

Aliquots of each specimen were prepared for testing with the BIOFIRE SPOTFIRE R/ST Panel and the applicable comparator method(s). Bacterial culture for $\beta$ -hemolytic streptococci was performed from a fresh aliquot of each throat swab specimen. All other aliquots of nasopharyngeal and throat swab specimens for comparator testing were stored at - $7 0 ~ ^ { \circ } \mathrm { C }$ and shipped to BioFire Diagnostics for testing. At five of the study sites, specimen enrollment began prior to the availability of BIOFIRE SPOTFIRE Systems. During this

period, the specimen aliquots for testing with the BIOFIRE SPOTFIRE R/ST Panel were frozen at $\mathbf { - 7 0 ^ { \circ } C }$ (refer to Matrix Comparison in Section VII (B)(2)).

A description of the comparator methods used to establish the performance of the BIOFIRE SPOTFIRE R/ST Panel is provided in Table 42.

Table 42. Comparator methods used to characterize prospectively collected and archived clinical specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>BIOFIRE SPOTFIRE R/ST Panel</td><td rowspan=2 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Respiratory</td><td rowspan=1 colspan=1>Sore Throat</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=12 colspan=1>FDA-clearedMulti-Analyte Panel-13</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza A Subtype H1-2009</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza A Subtype H3</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2 1</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=3 colspan=1>FDA-clearedMulti-Analyte Panel-2</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus 2</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae(Group C/G Streptococcus)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=2 colspan=1>Method 1:Culture for β-hemolyticStreptococcus, followed byspecies identification byPCR of 16S rRNA withbidirectional sequencingperformed on the cultureisolate 4Method 2:2 PCR assays withbidirectional sequencing</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes(Group A Streptococcus)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr></table>

1 Not detected by Multi-Analyte Panel-1 comparator. 2 Due to reduced sensitivity of the Multi-Analyte Panel-1 for Influenza B Virus, Multi-Analyte Panel-2 was used as the comparator for this analyte. 3 The comparator method reported separate results for individual seasonal Coronaviruses 229E, HKU1, NL63 and OC43 and Parainfluenza Virus serotypes 1 – 4, that were collapsed to produce a single comparator result for seasonal Coronaviruses and Parainfluenza Virus, respectively. 4 Culture was not performed for archived specimens.

Initial enrollment in the Prospective Clinical Study included 1215 nasopharyngeal and 1165 throat swabs. Of these, 95 nasopharyngeal swabs and 288 throat swabs were excluded from the analysis of performance for the reasons listed in Table 43. In addition, some specimens were excluded from the analysis of performance for one or more specific analytes due to failure to obtain a valid comparator result. All nasopharyngeal swabs included in the analysis of performance were collected in viral transport medium (VTM), whereas all throat swabs were collected in liquid Amies medium.

Table 43. Summary of data exclusions from the Prospective Clinical Study 1   

<table><tr><td rowspan=2 colspan=1>Reason for Exclusion</td><td rowspan=1 colspan=2>Number of Swab Specimens</td></tr><tr><td rowspan=1 colspan=1>Nasopharyngeal(n = 1215)</td><td rowspan=1 colspan=1>Throat(n = 1165)</td></tr><tr><td rowspan=1 colspan=1>Specimen did not meet inclusion criteria</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Missing informed consent or subject withdrew</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Subject unwilling or unable to provide specimen</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Incorrect residual specimen type</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Residual specimen exceeded storage parameters prior to enrollment</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Incorrect specimen collection</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Valid SPOTFIRE R/ST Panel result not obtained</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>238</td></tr><tr><td rowspan=1 colspan=1>Testing not be performed within the specified time</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>Use of an expired pouch 2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>115</td></tr><tr><td rowspan=1 colspan=1>Invalid result (retest not possible due to specimen volume)</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Specimen handling error</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Specimen not aliquoted within the specified time</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Specimen not received for comparator testing</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Total Excluded</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>288</td></tr><tr><td rowspan=1 colspan=1>Total Included</td><td rowspan=1 colspan=1>1120</td><td rowspan=1 colspan=1>877</td></tr></table>

1 Some additional specimens were excluded from analysis for specific analyte(s) due to failure to comply with parameters for the applicable comparator method. 2 Pouches labeled with incorrect expiration date prevented appropriate activation of the SPOTFIRE System lock-out feature.

A summary of the demographic information of the subjects included in the Prospective Clinical Study is provided in Table 44.

Table 44. Demographics of subjects included in the Prospective Clinical Study   

<table><tr><td rowspan=2 colspan=2>Category</td><td rowspan=1 colspan=2>Nasopharyngeal Swabs</td><td rowspan=1 colspan=2>Throat Swabs</td></tr><tr><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>587</td><td rowspan=1 colspan=1>52.4</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>41.2</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>533</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>516</td><td rowspan=1 colspan=1>58.8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1120</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>877</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=6 colspan=1>Age (years)</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>457</td><td rowspan=1 colspan=1>40.8</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>15.7</td></tr><tr><td rowspan=1 colspan=1>6-18</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>456</td><td rowspan=1 colspan=1>52.0</td></tr><tr><td rowspan=1 colspan=1>19-40</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>21.2</td></tr><tr><td rowspan=1 colspan=1>41 - 60</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>8.2</td></tr><tr><td rowspan=1 colspan=1>≥61</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1120</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>877</td><td rowspan=1 colspan=1>100</td></tr></table>

# Archived Specimens

Several analytes included on the BIOFIRE SPOTFIRE R/ST Panel were not encountered during the Prospective Clinical Study in sufficient numbers to demonstrate system performance. Therefore, the Prospective Clinical Study was supported by additional testing that was performed at four of the U.S. study sites on frozen, archived specimens obtained from various clinical laboratories from around the world. These specimens were selected for inclusion in the study based on their microbial content as originally determined by the source laboratory and were stored frozen prior to testing. The microbial content of the archived specimens was confirmed using the same molecular comparator methods as for the

prospectively collected specimens (Table 42). A mimimum sample volume of $4 5 0 \mu \mathrm { L }$ to 650 $\mu \mathrm { L }$ was required for confirmatory testing depending on the specific comparator method used.

A total of 562 archived nasopharyngeal swabs and 136 throat swabs was obtained. Of these, 20 nasopharyngeal swabs and 8 throat swabs were excluded from the analysis of performance due to invalid BIOFIRE SPOTFIRE R/ST Panel results or to invalid or missing comparator test results (Table 45).

Table 45. Summary of data exclusions from the testing archived clinical specimens   

<table><tr><td rowspan=1 colspan=1>Reason for Exclusion</td><td rowspan=1 colspan=1>NasopharyngealSwab</td><td rowspan=1 colspan=1>Throat Swab</td></tr><tr><td rowspan=1 colspan=1>Invalid BIOFIRE SPOTFIRE R/ST Panel Result</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Invalid or missing comparator test result</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>8</td></tr></table>

A summary of the demographic information of the subjects included in the Clinical Study with preselected, archived specimens is provided in Table 46.

Table 46. Demographics of subjects included in the Clinical Study with preselected, archived specimens   

<table><tr><td rowspan=2 colspan=2>Category</td><td rowspan=1 colspan=2>Nasopharyngeal Swabs</td><td rowspan=1 colspan=2>Throat Swabs</td></tr><tr><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=4 colspan=1>Sex</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>254</td><td rowspan=1 colspan=1>46.9</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>46.1</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>53.9</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>542</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=7 colspan=1>Age (years)</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>43.2</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>17.2</td></tr><tr><td rowspan=1 colspan=1>6-18</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>66.4</td></tr><tr><td rowspan=1 colspan=1>19-40</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>12.5</td></tr><tr><td rowspan=1 colspan=1>41  60</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3.9</td></tr><tr><td rowspan=1 colspan=1>≥61</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>542</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>100</td></tr></table>

# Contrived Specimens

Additional performance data for various low prevalence throat swab analytes were also obtained from testing contrived specimens that were prepared at BioFire Diagnostics, randomized and distributed to three of the sites that also participated in the Prospective Clinical Study. The contrived specimens comprised individual throat swab specimens in Amies medium that were spiked with various isolates/strains of each of the targeted bacteria/viruses. Each analyte was spiked into at least 50 unique clinical specimens, with at least half at a concentration equivalent to $\leq 2 \mathrm { X }$ LoD. Additional samples that were expected to produce negative results for all BIOFIRE SPOTFIRE R/ST Panel analytes were included as controls. Testing of the contrived specimens was integrated alongside testing of the prospectively collected and archived specimens.

In total, 464 contrived specimens were prepared for the study, of which 33 were excluded from the analysis of performance as shown in Table 47.

Table 47. Summary of data exclusions from the testing contrived throat swab specimens   

<table><tr><td rowspan=1 colspan=1>Rationale for Exclusion</td><td rowspan=1 colspan=1>Number</td></tr><tr><td rowspan=1 colspan=1>Invalid BIOFIRE SPOTFIRE R/ST Panel Result</td><td rowspan=1 colspan=1>211</td></tr><tr><td rowspan=1 colspan=1>Delay in processing (&gt; 8 hours post thaw)</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Mislabeled specimen</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Missing specimen</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>33</td></tr></table>

1 19/21 $( 9 0 . 5 \% )$ due to internal control failure and 2 $( 9 . 5 \% )$ due to instrument error.

# Clinical Performance

# Nasopharyngeal Swabs

A summary of the results from testing prospectively collected and archived nasopharyngeal swabs is shown in Table 48. Positive Percent Agreement (PPA) ranged from $9 6 . 3 \substack { - 1 0 0 \% }$ for prospectively collected specimens and from $9 6 . 0 \substack { - 1 0 0 \% }$ for archived specimens, depending on the analyte, whereas Negative Percent Agreement (NPA) ranged from $9 0 . 6 \substack { - 1 0 0 \% }$ for prospectively collected specimens and from $9 6 . 7 – 1 0 0 \%$ for archived specimens. The only analyte for which NPA was $< 9 5 \%$ was Human Rhinovirus/Enterovirus with prospectively collected specimens. However, additional testing provided evidence for the presence of Human Rhinovirus/Enterovirus in 54/72 prospectively collected specimens $( 7 5 . 0 \% )$ with discordant positive BIOFIRE SPOTFIRE R/ST Panel results and therefore the low NPA with this analyte in the prospective study was considered acceptable.

Overall, the BIOFIRE SPOTFIRE R/ST Panel exhibited acceptable PPA and NPA in comparison to other FDA-cleared methods for the detection of the targeted analytes in nasopharyngeal swab specimens.

Table 48. Clinical performance of the BIOFIRE SPOTFIRE R/ST Panel with prospectively collected and archived nasopharyngeal swab specimens   

<table><tr><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Assay</td><td colspan="1" rowspan="2">Study</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TN/(TN+FP)</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">97.0(84.7-99.5)</td><td colspan="1" rowspan="1">32/33</td><td colspan="1" rowspan="1">97.8(96.7-98.5)</td><td colspan="1" rowspan="1">1058/1082</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(89.0-100)</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">96.9(94.9-98.2)</td><td colspan="1" rowspan="1">439/453</td></tr><tr><td colspan="1" rowspan="2">Coronavirus (seasonal)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">99.0(94.7-99.8)</td><td colspan="1" rowspan="1">101/102</td><td colspan="1" rowspan="1">98.7(97.8-99.2)</td><td colspan="1" rowspan="1">1000/1013</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">99.0(94.3-99.8)</td><td colspan="1" rowspan="1">95/96</td><td colspan="1" rowspan="1">98.2(96.3-99.1)</td><td colspan="1" rowspan="1">381/388</td></tr><tr><td colspan="1" rowspan="2">Coronavirus SARS-CoV-2</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">97.3(90.6-99.2)</td><td colspan="1" rowspan="1">71/73</td><td colspan="1" rowspan="1">99.4(98.7-99.7)</td><td colspan="1" rowspan="1">1031/1037</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1114/1114</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">97.0(84.7-99.5)</td><td colspan="1" rowspan="1">32/33</td><td colspan="1" rowspan="1">100(99.2-100)</td><td colspan="1" rowspan="1">451/451</td></tr><tr><td colspan="1" rowspan="2">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">99.1(97.5-99.7)</td><td colspan="1" rowspan="1">345/348</td><td colspan="1" rowspan="1">90.61(88.3-92.5)</td><td colspan="1" rowspan="1">695/767</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">96.7(83.3-99.4)</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7(94.6-98.0)</td><td colspan="1" rowspan="1">439/454</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1115/1115</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">98.3(91.0-99.7)</td><td colspan="1" rowspan="1">58/59</td><td colspan="1" rowspan="1">100(99.1-100)</td><td colspan="1" rowspan="1">423/423</td></tr><tr><td colspan="1" rowspan="2">Influenza A H1-2009</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1115/1115</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">96.9(84.3-99.4)</td><td colspan="1" rowspan="1">31/32</td><td colspan="1" rowspan="1">100(99.2-100)</td><td colspan="1" rowspan="1">450/450</td></tr><tr><td colspan="1" rowspan="2">Influenza A H3</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1115/1115</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(87.5-100)</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100(99.2-100)</td><td colspan="1" rowspan="1">455/455</td></tr><tr><td colspan="1" rowspan="2">Influenza B Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1110/1110</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(88.7-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100(87.9-100)</td><td colspan="1" rowspan="1">28/28</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">98.0(92.9-99.4)</td><td colspan="1" rowspan="1">96/98</td><td colspan="1" rowspan="1">98.9(98.1-99.4)</td><td colspan="1" rowspan="1">1006/1017</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">98.3(94.0-99.5)</td><td colspan="1" rowspan="1">116/118</td><td colspan="1" rowspan="1">98.1(96.1-99.1)</td><td colspan="1" rowspan="1">359/366</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">96.3(81.7-99.3)</td><td colspan="1" rowspan="1">26/27</td><td colspan="1" rowspan="1">99.8(99.3-99.9)</td><td colspan="1" rowspan="1">1086/1088</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(90.6-100)</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">98.4(96.8-99.2)</td><td colspan="1" rowspan="1">440/447</td></tr></table>

<table><tr><td colspan="6" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Study</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TN/(TN+FP)</td></tr><tr><td colspan="1" rowspan="2">Bordetella parapertussis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1110/1110</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">96.0(80.5-99.3)</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">100(89.6-100)</td><td colspan="1" rowspan="1">33/33</td></tr><tr><td colspan="1" rowspan="2">Bordetella pertussis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1115/1115</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">96.4(82.3-99.4)</td><td colspan="1" rowspan="1">27/28</td><td colspan="1" rowspan="1">99.1(97.8-99.7)</td><td colspan="1" rowspan="1">452/456</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1115/1115</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(88.7-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">99.6(98.4-99.9)</td><td colspan="1" rowspan="1">452/454</td></tr><tr><td colspan="1" rowspan="2">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.7-100)</td><td colspan="1" rowspan="1">1115/1115</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(89.6-100)</td><td colspan="1" rowspan="1">33/33</td><td colspan="1" rowspan="1">98.9(97.4-99.5)</td><td colspan="1" rowspan="1">446/451</td></tr></table>

$9 5 \%$ CI: 2-sided $9 5 \%$ score confidence interval; TP: True Positive; FP: False-Positive; False-Negative; True Negative (all as determined with respect to the comparator) 1 54/72 specimens with “false-positive” results for Human Rhinovirus/Enterovirus $( 7 5 . 0 \% )$ were reported positive for these analytes upon retesting with the same FDA-cleared comparator (1), by testing with either an alternative FDA-cleared assay (44) or with PCR followed by bidirectional sequencing (8), or both (1).

# Throat Swabs

The performance of the BIOFIRE SPOTFIRE R/ST Panel with prospectively collected, archived and contrived throat swab specimens is summarized in Table 49. Additional details regarding the analysis of the contrived specimens, stratified by target level, are shown in Table 50.

For all analytes on the BIOFIRE SPOTFIRE R/ST Panel for which positive specimens were available, except Human Metapneumovirus and Respiratory Syncytial Virus, PPA (or sensitivity, as applicable) with prospectively collected throat swab specimens ranged from $9 2 . 3 \substack { - 1 0 0 \% }$ . For Human Metapneumovirus and Respiratory Syncytial Virus, PPA in the Prospective Clinical Study was 88.2 and $8 7 . 5 \%$ , respectively, although in both cases the total number of positive specimens tested was small. Additional positive results were obtained upon retesting of those specimens with initial false-negative BIOFIRE SPOTFIRE R/ST Panel results (Table 49), suggesting the presence of low levels of the target analyte. In addition, for both analytes, each of the archived positive specimens that was tested produced the expected positive result, as did 91.8 to $9 8 . 0 \%$ of contrived positive specimens that were prepared with different strains of each virus. In combination, the results for detection of Human Metapneumovirus and Respiratory Syncytial Virus in throat swabs were considered acceptable in the context of the proposed Intended Use/Indications For Use for the BIOFIRE SPOTFIRE R/ST Panel and associated limitations in the device labeling.

Based on the observed clinical sensitivity of the BIOFIRE SPOTFIRE R/ST Panel for the detection of S. pyogenes (Group A Streptococcus) in throat swab specimens, culture confirmation of negative BIOFIRE SPOTFIRE R/ST Panel results for this analyte is not required. However, the following Limitation is included in the device labeling:

“Additional follow-up testing by culture is required if the SPOTFIRE R/ST Panel Streptococcus pyogenes assay result is negative and clinical symptoms persist, or in the event of an outbreak of acute rheumatic fever (ARF).”

Table 49. Performance of the BIOFIRE SPOTFIRE R/ST Panel with prospectively collected, archived and contrived throat swab specimens   

<table><tr><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Study</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TN/(TN+FP)</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">92.3(83.2-96.7)</td><td colspan="1" rowspan="1">60/65</td><td colspan="1" rowspan="1">98.5(97.4-99.2)</td><td colspan="1" rowspan="1">798/810</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">81.8 1(52.3-94.9)</td><td colspan="1" rowspan="1">9/11</td><td colspan="1" rowspan="1">91.7(80.4-96.7)</td><td colspan="1" rowspan="1">44/48</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">100(92.9-100)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">99.7(98.5-100)</td><td colspan="1" rowspan="1">380/381</td></tr><tr><td colspan="1" rowspan="3">Coronavirus (seasonal)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">97.6(87.7-99.6)</td><td colspan="1" rowspan="1">41/42</td><td colspan="1" rowspan="1">99.2(98.3-99.6)</td><td colspan="1" rowspan="1">826/833</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(75.8-100)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100(92.4-100)</td><td colspan="1" rowspan="1">47/47</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">99.2(95.8-99.9)</td><td colspan="1" rowspan="1">129/130</td><td colspan="1" rowspan="1">100(98.7-100)</td><td colspan="1" rowspan="1">301/301</td></tr><tr><td colspan="1" rowspan="3">Human Metapneumovirus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">88.2 2(65.7-96.7)</td><td colspan="1" rowspan="1">15/17</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">858/858</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(93.5-100)</td><td colspan="1" rowspan="1">55/55</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">91.8 3(80.8-96.8)</td><td colspan="1" rowspan="1">45/49</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">382/382</td></tr><tr><td colspan="1" rowspan="3">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">94.8(91.0-97.1)</td><td colspan="1" rowspan="1">202/213</td><td colspan="1" rowspan="1">93.5 4(91.4-95.1)</td><td colspan="1" rowspan="1">619/662</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">96.5(88.1-99.0)</td><td colspan="1" rowspan="1">55/57</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.0-100)</td><td colspan="1" rowspan="1">431/431</td></tr><tr><td colspan="1" rowspan="3">Influenza A Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">100(90.1-100)</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">840/840</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(92.0-100)</td><td colspan="1" rowspan="1">44/44</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">100(96.0-100)</td><td colspan="1" rowspan="1">93/93</td><td colspan="1" rowspan="1">100(98.9-100)</td><td colspan="1" rowspan="1">332/332</td></tr><tr><td colspan="1" rowspan="3">Influenza A H1-2009</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">100(75.8-100)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">863/863</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(64.6-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100(92.6-100)</td><td colspan="1" rowspan="1">48/48</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">100(92.3-100)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100(99.0-100)</td><td colspan="1" rowspan="1">379/379</td></tr><tr><td colspan="1" rowspan="3">Influenza A H3</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">100(85.7-100)</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">852/852</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(93.0-100)</td><td colspan="1" rowspan="1">51/51</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">100(92.4-100)</td><td colspan="1" rowspan="1">47/47</td><td colspan="1" rowspan="1">100(99.0-100)</td><td colspan="1" rowspan="1">378/378</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">872/872</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(83.9-100)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">95.9(86.3-98.9)</td><td colspan="1" rowspan="1">47/49</td><td colspan="1" rowspan="1">100(98.9-100)</td><td colspan="1" rowspan="1">333/333</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">96.2(87.0-98.9)</td><td colspan="1" rowspan="1">50/52</td><td colspan="1" rowspan="1">99.8(99.1-99.9)</td><td colspan="1" rowspan="1">821/823</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(93.8-100)</td><td colspan="1" rowspan="1">58/58</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">95.3(90.2-97.8)</td><td colspan="1" rowspan="1">122/128</td><td colspan="1" rowspan="1">100(98.7-100)</td><td colspan="1" rowspan="1">303/303</td></tr><tr><td colspan="1" rowspan="3">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">87.5 5(69.0-95.7)</td><td colspan="1" rowspan="1">21/24</td><td colspan="1" rowspan="1">99.8(99.1-99.9)</td><td colspan="1" rowspan="1">849/851</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">98.2(90.7-100)</td><td colspan="1" rowspan="1">56/57</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">98.0(89.5-99.6)</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">100(99.0-100)</td><td colspan="1" rowspan="1">381/381</td></tr></table>

<table><tr><td colspan="6" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Study</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TN/(TN+FP)</td></tr><tr><td colspan="1" rowspan="3">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">875/875</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(93.7-100)</td><td colspan="1" rowspan="1">57/57</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">100(92.9-100)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100(99.0-100)</td><td colspan="1" rowspan="1">381/381</td></tr><tr><td colspan="1" rowspan="3">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">875/875</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(93.5-100)</td><td colspan="1" rowspan="1">55/55</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">96.0(86.5-98.9)</td><td colspan="1" rowspan="1">48/50</td><td colspan="1" rowspan="1">100(99.0-100)</td><td colspan="1" rowspan="1">381/381</td></tr><tr><td colspan="1" rowspan="4">Streptococcus dysgalactiae(Group C, G Streptococcus)</td><td colspan="1" rowspan="1">Prospective(vs PCR seq.)</td><td colspan="1" rowspan="1">100(88.7-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">99.5(98.8-99.8)</td><td colspan="1" rowspan="1">843/847</td></tr><tr><td colspan="1" rowspan="1">Prospective(vs culture) 6</td><td colspan="1" rowspan="1">100(87.1-100)</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">99.3(98.5-99.7)</td><td colspan="1" rowspan="1">837/843</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.1-100)</td><td colspan="1" rowspan="1">431/431</td></tr><tr><td colspan="1" rowspan="2">Streptococcus pyogenes(Group A Streptococcus)</td><td colspan="1" rowspan="1">Prospective(vs PCR seq.)</td><td colspan="1" rowspan="1">96.3(92.9-98.1)</td><td colspan="1" rowspan="1">209/217</td><td colspan="1" rowspan="1">99.1(98.0-99.6)</td><td colspan="1" rowspan="1">654/660</td></tr><tr><td colspan="1" rowspan="1">Prospective(vs culture) 6</td><td colspan="1" rowspan="1">98.3(95.1-99.4)</td><td colspan="1" rowspan="1">174/177</td><td colspan="1" rowspan="1">94.5(92.6-96.0)</td><td colspan="1" rowspan="1">654/692</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Archived(vs PCR seq.)</td><td colspan="1" rowspan="1">97.4(86.8-99.5)</td><td colspan="1" rowspan="1">38/39</td><td colspan="1" rowspan="1">100(72.3-100)</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">Notapplicable</td><td colspan="1" rowspan="1">100(99.1-100)</td><td colspan="1" rowspan="1">431/431</td></tr></table>

$9 5 \%$ CI: 2-sided $9 5 \%$ score confidence interval; TP: True Positive; FP: False-Positive; False-Negative; True Negative (all as determined with respect to the comparator) 1 Positive percent agreement (PPA) for Adenovirus with archived specimens was $< 9 0 \%$ although the number of specimens tested was small. PPA was $> 9 0 \%$ with prospectively collected specimens. Overall, PPA for detection of Adenovirus was considered acceptable. 2 1/2 specimens with false-negative results for Human Metapneumovirus produced a positive result on retest. In combination with the results from testing archived contrived specimens, PPA for Human Metapneumovirus was considered acceptable. 3 Refer to Table 50 for additional analysis. 4 Using an alternative molecular method, Human Rhinovirus/Enterovirus was detected in 14/43 specimens with false-positive BIOFIRE SPOTFIRE R/ST Panel results. 5 3/3 specimens with false-negative results for Respiratory Syncytial Virus produced positive results on retest. 6 Performance compared to culture is expressed in terms of sensitivity and specificity.

Additional analysis of the results from testing contrived throat swab specimens to evaluate the performance of the BIOFIRE SPOTFIRE R/ST Panel with low prevalence analytes is shown in Table 50, stratified by target level. For all analytes included in the Contrived Specimen Study, PPA was $\ge 9 5 \%$ except for Human Metapneumovirus for which unexpected negative results were obtained at the lowest target level tested (2X the claimed LoD). However, the expected positive results were obtained with the same isolates at all other levels tested, suggesting that the false-negative results were associated with reduced precision of the system at target levels close to the LoD.

Table 50. Results from testing contrived throat swab specimens stratified by target level   

<table><tr><td rowspan=1 colspan=8>Viruses</td></tr><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=2># Strains</td><td rowspan=2 colspan=1>Multipleof LoD</td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>Positive/Tested</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Negative/Tested</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=6 colspan=1>Adenovirus</td><td rowspan=6 colspan=2>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=6 colspan=1>380/381</td><td rowspan=6 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=5 colspan=1>Coronavirus (seasonal)</td><td rowspan=3 colspan=2>131</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>73/74</td><td rowspan=1 colspan=1>98.6</td><td rowspan=5 colspan=1>301/301</td><td rowspan=5 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td></tr></table>

<table><tr><td colspan="7" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Species</td><td colspan="1" rowspan="2"># Strains</td><td colspan="1" rowspan="2">Multipleof LoD</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Positive/Tested</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">Negative/Tested</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">129/130</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Human Metapneumovirus</td><td colspan="1" rowspan="6">10</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">21/25 2</td><td colspan="1" rowspan="1">84.0</td><td colspan="1" rowspan="6">382/382</td><td colspan="1" rowspan="6">100</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">45/49</td><td colspan="1" rowspan="1">91.8</td></tr><tr><td colspan="1" rowspan="1">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">431/431</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="7">Influenza A VirusSubtype H1-2009 3</td><td colspan="1" rowspan="7">8</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="7">379/379</td><td colspan="1" rowspan="7">100</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">941</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="6">Influenza A VirusSubtype 3 3</td><td colspan="1" rowspan="6">9</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="6">378/378</td><td colspan="1" rowspan="6">100</td></tr><tr><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">47/47</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="6">Influenza B Virus 4</td><td colspan="1" rowspan="6">10</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">23/25</td><td colspan="1" rowspan="1">92.0</td><td colspan="1" rowspan="6">333/333</td><td colspan="1" rowspan="6">100</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">47/49</td><td colspan="1" rowspan="1">95.9</td></tr><tr><td colspan="1" rowspan="5">Parainfluenza Virus</td><td colspan="1" rowspan="5">175</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">58/64 6</td><td colspan="1" rowspan="1">90.6</td><td colspan="1" rowspan="5">303/303</td><td colspan="1" rowspan="5">100</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">122/128</td><td colspan="1" rowspan="1">95.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Respiratory Syncytial Virus</td><td colspan="1" rowspan="6">8</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="6">381/381</td><td colspan="1" rowspan="6">100</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1># Strains</td><td rowspan=2 colspan=1>Multipleof LoD</td><td rowspan=1 colspan=2>Positive Agreement</td><td rowspan=1 colspan=2>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>Positive/Tested</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Negative/Tested</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=6 colspan=1>Chlamydia pneumoniae</td><td rowspan=6 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=6 colspan=1>381/381</td><td rowspan=6 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=6 colspan=1>Mycoplasma pneumoniae</td><td rowspan=6 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>23/25</td><td rowspan=1 colspan=1>92.0</td><td rowspan=6 colspan=1>381/381</td><td rowspan=6 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=1>96.0</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>431/431</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>431/431</td><td rowspan=1 colspan=1>100</td></tr></table>

Coronavirus 229E (3), HKU1/OC43 (9), NL63 (1); 20 samples were co-spiked with both 229E and NL63 and are stratified in the table according to the higher of the respective target levels used. 2 The 4 false-negative results were associated with 3 different isolates of Human Metapneumovirus which yielded 0/2, 1/2 and 2/3 positive results at 2X LoD, respectively; all 3 isolates were successfully detected at 5X LoD (1/1 ea.). 3 Samples containing isolates that were reported as “Influenza A No Subtype Identified” $[ \mathfrak { n } = 6$ in total; 4 containing influenza A H1-2009 and 2 containing influenza A H3) were excluded from the analysis of performance. 4 49 specimens were excluded from the analysis of performance for Influenza B Virus because they were prepared using the incorrect target level. 5 Parainfluenza Virus 1 (4), Parainfluenza Virus 2 (8), Parainfluenza Virus 4 (5); 19 samples were co-spiked with 2 Parainfluenza Viruses (1 and 4 or 2 and 4) and are stratified in the table according to the higher of the respective target levels used. 6 The 6 false-negative results were associated with Parainfluenza Viruses 1 and 2 $\mathbf { \tilde { n } } = 3$ ea.).

# BIOFIRE SPOTFIRE R/ST Panel Completion Rate

The frequency of BIOFIRE SPOTFIRE System errors and the rate of BIOFIRE SPOTFIRE R/ST Panel internal control failure in each of the Clinical Studies conducted to evaluate panel performance (Prospective, Archived and Contrived) are summarized in Table 51. Based on initial test results for each specimen, the frequency of instrument errors was $< 1 \%$ in each study, while the rate of invalid BIOFIRE SPOTFIRE R/ST Panel results due to internal control failure ranged from 1.0 to $4 . 1 \%$ .

Table 51. Summary of BIOFIRE SPOTFIRE R/ST Panel completion rate on initial testing   

<table><tr><td rowspan=2 colspan=1>Clinical Study</td><td rowspan=1 colspan=5>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Initial Tests</td><td rowspan=1 colspan=1>InstrumentErrors</td><td rowspan=1 colspan=1>RunsCompleted</td><td rowspan=1 colspan=1>ControlFailures</td><td rowspan=1 colspan=1>Valid Results</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>2,120</td><td rowspan=1 colspan=1>14 (0.7)</td><td rowspan=1 colspan=1>2,106 (99.3)</td><td rowspan=1 colspan=1>62 (2.9)</td><td rowspan=1 colspan=1>2,044 (96.4)</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>698</td><td rowspan=1 colspan=1>6 (0.9)</td><td rowspan=1 colspan=1>692 (99.1)</td><td rowspan=1 colspan=1>7 (1.0)</td><td rowspan=1 colspan=1>685 (98.1)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>463</td><td rowspan=1 colspan=1>2 (0.4)</td><td rowspan=1 colspan=1>461 (99.6)</td><td rowspan=1 colspan=1>19 (4.1)</td><td rowspan=1 colspan=1>442 (95.5)</td></tr></table>

# Coinfections

Nasopharyngeal Swab Specimens   
Infections due to a single analyte were reported by the BIOFIRE SPOTFIRE R/ST Panel from $5 1 . 0 \%$ of nasopharyngeal swab specimens (571/1120) in the Prospective Clinical Study, while coinfections with two or more analytes were reported from $9 . 5 \%$ of specimens (106/1120) (Table 52). Among the coinfected specimens, 90 $( 8 4 . 9 \% )$ were positive for two analytes and 16 $( 1 5 . 1 \% )$ were positive for three or more analytes using the BIOFIRE SPOTFIRE R/ST Panel.

Table 52. Coinfections reported by the BIOFIRE SPOTFIRE R/ST Panel in the Prospective Clinical Study with nasopharyngeal swab specimens   

<table><tr><td rowspan=2 colspan=1>Analyte 1</td><td rowspan=2 colspan=1>Analyte 2</td><td rowspan=2 colspan=1>Analyte 3</td><td rowspan=2 colspan=1>Analyte 4</td><td rowspan=1 colspan=2>Number of Specimens</td><td rowspan=2 colspan=1>BIOFIRE SPOTFIRE R/STPanel False-Positive Results 1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>False-Positive</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus, PIV</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Adenovirus (2), hR/EV (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV Seasonal</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>CoV Seasonal (1), hR/EV (2), PIV (2)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>hR/EV (2), PIV (1), RSV (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Adenovirus (11), hR/EV (5)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>canmaGYellowb</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (2)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (2)</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>hR/EV (3), CoV Seasonal (2)</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV Seasonal</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>SARS-CoV-2 (1), hR/EV (5)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>hR/EV (6), PIV (3)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>hR/EV (4), RSV (1)</td></tr><tr><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=4>Total</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>62/229 2</td></tr><tr><td rowspan=1 colspan=4>Total Quadruple Infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2/4 2</td></tr><tr><td rowspan=1 colspan=4>Total Triple Infections</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>13/45 2</td></tr><tr><td rowspan=1 colspan=4>Total Double Infections</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>47/180 2</td></tr></table>

hMPV: Human Metapneumovirus; hR/EV: Human Rhinovirus/Enterovirus; PIV: Parainfluenza Virus; CoV Seasonal: Coronavirus (seasonal); RSV: Respiratory Syncytial Virus 1 Based on comparator test results. 2 Number of BIOFIRE SPOTFIRE R/ST Panel false-positive results/total.

Four nasopharyngeal specimens in the Prospective Clinical Study were positive for two analytes by the comparator methods (i.e., were coinfected) but were reported as negative for one of the analytes using the BIOFIRE SPOTFIRE R/ST Panel (Table 53). A further 5 nasopharyngeal swab specimens were reported positive for 3 on-panel analytes by the comparator methods and, in each case, the results obtained with the BIOFIRE SPOTFIRE R/ST Panel were in agreement.

Table 53. Coinfections identified in nasopharyngeal swab specimens by the comparator methods not reported by the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td rowspan=1 colspan=2>Comparator Positive Results</td><td rowspan=2 colspan=1>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1</td></tr></table>

The analyte in each coinfected specimen that was not detected by the BIOFIRE SPOTFIRE R/ST Panel is highlighted in yellow.

# Throat Swab Specimens

Infections due to a single analyte were reported by the BIOFIRE SPOTFIRE R/ST Panel from $4 8 . 6 \%$ of throat swab specimens (426/877) in the Prospective Clinical Study, while coinfections with two or more analytes were reported from $1 6 . 9 \%$ of specimens (148/877) (Table 54). Among the coinfected specimens, 132 $( 8 9 . 2 \% )$ were positive for two analytes while 16 $( 1 0 . 8 \% )$ were positive for three analytes using the BIOFIRE SPOTFIRE R/ST Panel.

Table 54. Coinfections reported by the BIOFIRE SPOTFIRE R/ST Panel in the Prospective Clinical Study with throat swab specimens   

<table><tr><td rowspan=2 colspan=1>Analyte 1</td><td rowspan=2 colspan=1>Analyte 2</td><td rowspan=2 colspan=1>Analyte 3</td><td rowspan=1 colspan=2>Number ofSpecimens</td><td rowspan=2 colspan=1>BIOFIRE SPOTFIRE R/ST PanelFalse-Positive Results 1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>False-Positive</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>S. dysgalactiae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (1), hR/EV (1), S. pyogenes (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>S. pyogenes (1)</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hR/EV (1)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hR/EV (1), S. pyogenes (1)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>cYanmaGllow</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>cYanmaGllow</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Adenovirus (2), hR/EV (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza AH1-2009</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>cYanmaGllow</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (2)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Adenovirus (1), S. pyogenes (3)</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CoV Seasonal (1), hR/EV (1)</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CoV Seasonal (2)</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CoV Seasonal (1), S. pyogenes (1)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hR/EV (1)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>cYanmaGellowb</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hR/EV (1)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV (1)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>cYanmaGellowb</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV (1)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. dysgalactiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>S. dysgalactiae (1)</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>hR/EV (12), S. pyogenes (4)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>S. pyogenes (1)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>S. dysgalactiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>S. pyogenes (2)</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>S. dysgalactiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>S. dysgalactiae</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>S. dysgalactiae (2), S. pyogenes (2)</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>51/312 2</td></tr><tr><td rowspan=1 colspan=3>Total Triple Infections</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>8/48 2</td></tr><tr><td rowspan=1 colspan=3>Total Double Infections</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43/264 2</td></tr></table>

hMPV: Human Metapneumovirus; hR/EV: Human Rhinovirus/Enterovirus; PIV: Parainfluenza Virus; CoV Seasonal: Coronavirus (seasonal); RSV: Respiratory Syncytial Virus 1 Based on comparator test results. 2 Number of BIOFIRE SPOTFIRE R/ST Panel false-positive results/total.

Thirteen throat swab specimens were positive for multiple analytes by the comparator methods but were reported as negative for one or more of the analytes using the BIOFIRE SPOTFIRE R/ST Panel (Table 55).

Table 55. Coinfections identified in throat swab specimens by the comparator methods not reported by the BIOFIRE SPOTFIRE R/ST Panel   

<table><tr><td rowspan=1 colspan=4>Comparator Positive Results</td><td rowspan=2 colspan=1>Number ofSpecimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte 4</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>PIV</td><td rowspan=1 colspan=1>hR/EV</td><td rowspan=1 colspan=1>cYanmaGYellow</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>CoV Seasonal</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td></tr></table>

CoV Seasonal: Coronavirus (seasonal); hMPV: Human Metapneumovirus; hR/EV; Human Rhinovirus/Enterovirus; PIV: Parainfluenza Virus The analyte(s) in each coinfected specimen that were not detected by the BIOFIRE SPOTFIRE R/ST Panel are highlighted in yellow.

2. Clinical Specificity: Please refer to Section VII (C)(1).

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The rates of positive results for the BIOFIRE SPOTFIRE R/ST Panel analytes in the Prospective Clinical Study described in Section VII (C)(1) are stratified by study site and patient age in Tables 56 and 57 for nasopharyngeal swab specimens and Tables 58 and 59 for throat swab specimens.

Table 56. Expected values and prevalence of analytes in prospectively collected nasopharyngeal swab specimens, as determined by the BIOFIRE SPOTFIRE R/ST Panel and comparator methods, respectively, stratified by site   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">TestMethod</td><td colspan="6" rowspan="1">Positive Results Stratified by Study Site (%)</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">Site 5</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">56/1115(5.0)</td><td colspan="1" rowspan="1">3/361(0.8)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">34/508(6.7)</td><td colspan="1" rowspan="1">19/147(12.9)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">331115(3.0)</td><td colspan="1" rowspan="1">1/361(0.3)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">25/508(4.9)</td><td colspan="1" rowspan="1">7/147(4.8)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus (seasonal)</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">114/1115(10.2)</td><td colspan="1" rowspan="1">24/361(6.6)</td><td colspan="1" rowspan="1">4/36(111.1)</td><td colspan="1" rowspan="1">73/508(14.4)</td><td colspan="1" rowspan="1">13/147(8.8)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">102/1115(9.1)</td><td colspan="1" rowspan="1">19/361(5.3)</td><td colspan="1" rowspan="1">3/36(8.3)</td><td colspan="1" rowspan="1">67/508(13.2)</td><td colspan="1" rowspan="1">13/147(8.8)</td><td colspan="1" rowspan="1">0/63(0</td></tr><tr><td colspan="1" rowspan="2">Coronavirus SARS-CoV-2</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">77/1110(6.9)</td><td colspan="1" rowspan="1">27/361(7.5)</td><td colspan="1" rowspan="1">7/36(19.4)</td><td colspan="1" rowspan="1">8/502(1.6)</td><td colspan="1" rowspan="1">8/147(5.4)</td><td colspan="1" rowspan="1">27/64(42.2)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">73/110(6.6)</td><td colspan="1" rowspan="1">27/361(7.5)</td><td colspan="1" rowspan="1">6/36(16.7)</td><td colspan="1" rowspan="1">8/502(1.6)</td><td colspan="1" rowspan="1">7/147(4.8)</td><td colspan="1" rowspan="1">25/64(39.1)</td></tr><tr><td colspan="1" rowspan="2">Human Metapneumovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">1/1115(0.1)</td><td colspan="1" rowspan="1">1/361(0.3)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/63()</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1/115(0.1)</td><td colspan="1" rowspan="1">1/361(0.3)</td><td colspan="1" rowspan="1">0/36()</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/63()</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">417/1115(37.4)</td><td colspan="1" rowspan="1">59/361(16.3)</td><td colspan="1" rowspan="1">7/36(19.4)</td><td colspan="1" rowspan="1">265(52.2)</td><td colspan="1" rowspan="1">85/147(57.8)</td><td colspan="1" rowspan="1">1/63(1.6)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">348/1115(31.2)</td><td colspan="1" rowspan="1">49/361(13.6)</td><td colspan="1" rowspan="1">5/36(13.9)</td><td colspan="1" rowspan="1">219(43.1)</td><td colspan="1" rowspan="1">74/147(50.3)</td><td colspan="1" rowspan="1">1/63(1.6)</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36()</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus A/H1-2009</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115()</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(0)</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/63(0</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus A/H3</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115()</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115()</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="2">Influenza B Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/361()</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/502()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/64(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1110()</td><td colspan="1" rowspan="1">0/361()</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/502(</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/64()</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">107/1115(9.6)</td><td colspan="1" rowspan="1">20/361(5.5)</td><td colspan="1" rowspan="1">3/36(8.3)</td><td colspan="1" rowspan="1">72/508(14.2)</td><td colspan="1" rowspan="1">11/147(7.5)</td><td colspan="1" rowspan="1">1/63(1.6)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">98/1115(8.8)</td><td colspan="1" rowspan="1">16/361(4.4)</td><td colspan="1" rowspan="1">3/36(8.3)</td><td colspan="1" rowspan="1">68/508(13.4)</td><td colspan="1" rowspan="1">11/147(7.5)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">28/1115(2.5)</td><td colspan="1" rowspan="1">7/361(1.9)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">16/508(3.1)</td><td colspan="1" rowspan="1">5/147(3.4)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">27/11115(2.4)</td><td colspan="1" rowspan="1">7/361(1.9)</td><td colspan="1" rowspan="1">0/36()</td><td colspan="1" rowspan="1">15/508(3.0)</td><td colspan="1" rowspan="1">5/147(3.4)</td><td colspan="1" rowspan="1">0/63()</td></tr><tr><td colspan="1" rowspan="2">Bordetella parapertussis</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/502(0)</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/64()</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/502()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/64()</td></tr><tr><td colspan="1" rowspan="2">Bordetella pertussis</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361()</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115()</td><td colspan="1" rowspan="1">0/361()</td><td colspan="1" rowspan="1">0/36()</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147()</td><td colspan="1" rowspan="1">0/63()</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36()</td><td colspan="1" rowspan="1">0/508()</td><td colspan="1" rowspan="1">0/147)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/361(0)</td><td colspan="1" rowspan="1">0/36(0)</td><td colspan="1" rowspan="1">0/508(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/63(0)</td></tr></table>

Only specimens with both valid BIOFIRE SPOTFIRE R/ST Panel and comparator results for the applicable analyte are included in calculation of expected values.

Table 57. Expected values and prevalence of analytes in prospectively collected nasopharyngeal swab specimens, as determined by the BIOFIRE SPOTFIRE R/ST Panel and comparator methods, respectively, stratified by patient age   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">TestMethod</td><td colspan="6" rowspan="1">Positive Results Stratified by Age [years] (%)</td></tr><tr><td colspan="1" rowspan="1">All Ages</td><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">6-18</td><td colspan="1" rowspan="1">19-40</td><td colspan="1" rowspan="1">41-60</td><td colspan="1" rowspan="1">&gt; 60</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">56/1115(5.0)</td><td colspan="1" rowspan="1">53/455(11.6)</td><td colspan="1" rowspan="1">3/257(1.2)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">33/1115(3.0)</td><td colspan="1" rowspan="1">31/455(6.8)</td><td colspan="1" rowspan="1">2/257(0.8)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus (seasonal)</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">114/1115(10.2)</td><td colspan="1" rowspan="1">71/455(15.6)</td><td colspan="1" rowspan="1">22/257(8.6)</td><td colspan="1" rowspan="1">10/159(6.3)</td><td colspan="1" rowspan="1">9/146(6.2)</td><td colspan="1" rowspan="1">2/98(2.0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">102/1115(9.1)</td><td colspan="1" rowspan="1">64/455(14.1)</td><td colspan="1" rowspan="1">21/257(8.2)</td><td colspan="1" rowspan="1">6/159(3.8)</td><td colspan="1" rowspan="1">9/146(6.2)</td><td colspan="1" rowspan="1">2/98(2.0)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus SARS-CoV-2</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">77/1110(6.9)</td><td colspan="1" rowspan="1">9/450(2.0)</td><td colspan="1" rowspan="1">7/257(2.7)</td><td colspan="1" rowspan="1">27/158(17.1)</td><td colspan="1" rowspan="1">21/147(14.3)</td><td colspan="1" rowspan="1">13/98(13.3)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">73/1110(6.6)</td><td colspan="1" rowspan="1">9/450(2.0)</td><td colspan="1" rowspan="1">5/257(1.9)</td><td colspan="1" rowspan="1">25/158(15.8)</td><td colspan="1" rowspan="1">20/147(13.6)</td><td colspan="1" rowspan="1">14/98(14.3)</td></tr><tr><td colspan="1" rowspan="2">Human Metapneumovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">1/1115(0.1)</td><td colspan="1" rowspan="1">1/455(0.2)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1/1115(0.1)</td><td colspan="1" rowspan="1">1/455(0.2)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">417/1115(37.4)</td><td colspan="1" rowspan="1">236/455(51.9)</td><td colspan="1" rowspan="1">139/257(54.1)</td><td colspan="1" rowspan="1">23/159(14.5)</td><td colspan="1" rowspan="1">14/146(9.6)</td><td colspan="1" rowspan="1">5/98(5.1)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">348/1115(31.2)</td><td colspan="1" rowspan="1">200/455(44.0)</td><td colspan="1" rowspan="1">116/257(45.1)</td><td colspan="1" rowspan="1">17/159(10.7)</td><td colspan="1" rowspan="1">11/146(7.5)</td><td colspan="1" rowspan="1">4/98(4.1)</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257()</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus A/H1-2009</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus A/H3</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Influenza B Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/450(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/158(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/450(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/158(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza Virus</td><td colspan="1" rowspan="2">SPOTFIRE</td><td colspan="1" rowspan="2">107/1115(9.6)</td><td colspan="1" rowspan="2">89/455(19.6)</td><td colspan="1" rowspan="2">4/257(1.6)</td><td colspan="1" rowspan="2">4/159(2.5)</td><td colspan="1" rowspan="2">8/146(5.5)</td><td colspan="1" rowspan="1">2/98</td></tr><tr><td colspan="1" rowspan="1">(2.0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">98/1115(8.8)</td><td colspan="1" rowspan="1">83/455(18.2)</td><td colspan="1" rowspan="1">4/257(1.6)</td><td colspan="1" rowspan="1">3/159(1.9)</td><td colspan="1" rowspan="1">6/146(4.1)</td><td colspan="1" rowspan="1">2/98(2.0)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">28/1115(2.5)</td><td colspan="1" rowspan="1">21/455(4.6)</td><td colspan="1" rowspan="1">6/257(2.3)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">1/146(0.7)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">27/1115(2.4)</td><td colspan="1" rowspan="1">21/455(4.6)</td><td colspan="1" rowspan="1">5/257(1.9)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">1/146(0.7)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Bordetella parapertussis</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/450(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/158(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1110(0)</td><td colspan="1" rowspan="1">0/450(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/158(0)</td><td colspan="1" rowspan="1">0/147(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Bordetella pertussis</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/1115(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/257(0)</td><td colspan="1" rowspan="1">0/159(0)</td><td colspan="1" rowspan="1">0/146(0)</td><td colspan="1" rowspan="1">0/98(0)</td></tr></table>

Only specimens with both valid BIOFIRE SPOTFIRE R/ST Panel and comparator results for the applicable analyte are included in calculation of expected values.

Table 58. Expected values and prevalence of analytes in prospectively collected throat swab specimens, as determined by the BIOFIRE SPOTFIRE R/ST Panel and comparator methods, respectively, stratified by site   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>TestMethod</td><td rowspan=1 colspan=7>Positive Results Stratified by Study Site (%)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Site 6</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=2 colspan=1>SPOTFIRE</td><td rowspan=2 colspan=1>72/875(8.2)</td><td rowspan=2 colspan=1>0/152(0)</td><td rowspan=2 colspan=1>4/97(4.1)</td><td rowspan=1 colspan=1>30/324</td><td rowspan=1 colspan=1>26/253</td><td rowspan=1 colspan=1>1/22</td><td rowspan=1 colspan=1>11/27</td></tr><tr><td rowspan=1 colspan=1>(9.3)</td><td rowspan=1 colspan=1>(10.3)</td><td rowspan=1 colspan=1>(4.5)</td><td rowspan=1 colspan=1>(40.7)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>65/875(7.4)</td><td rowspan=1 colspan=1>0/152(0)</td><td rowspan=1 colspan=1>3/97(3.1)</td><td rowspan=1 colspan=1>26/324(8.0)</td><td rowspan=1 colspan=1>25/253(9.9)</td><td rowspan=1 colspan=1>1/22(4.5)</td><td rowspan=1 colspan=1>10/27(37.0)</td></tr><tr><td rowspan=2 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>48/875(5.5)</td><td rowspan=1 colspan=1>4/152(2.6)</td><td rowspan=1 colspan=1>3/97(3.1)</td><td rowspan=1 colspan=1>13/324(4.0)</td><td rowspan=1 colspan=1>23/253(9.1)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>5/27(18.5)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>42/875(4.8)</td><td rowspan=1 colspan=1>2/152(2.6)</td><td rowspan=1 colspan=1>3/97(3.1)</td><td rowspan=1 colspan=1>11/324(3.4)</td><td rowspan=1 colspan=1>22/253(8.7)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>4/27(14.8)</td></tr><tr><td rowspan=2 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>15/875(1.7)</td><td rowspan=1 colspan=1>3/152(2.0)</td><td rowspan=1 colspan=1>1/97(1.0)</td><td rowspan=1 colspan=1>5/324(1.5)</td><td rowspan=1 colspan=1>6/253(2.4)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>17/875(1.9)</td><td rowspan=1 colspan=1>3/152(2.0)</td><td rowspan=1 colspan=1>2/97(2.1)</td><td rowspan=1 colspan=1>5/324(1.5)</td><td rowspan=1 colspan=1>7/253(2.8)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>245/875(28.0)</td><td rowspan=1 colspan=1>17/152(11.2)</td><td rowspan=1 colspan=1>15/97(15.5)</td><td rowspan=1 colspan=1>125/324(38.6)</td><td rowspan=1 colspan=1>77/253(30.4)</td><td rowspan=1 colspan=1>1/22(4.5)</td><td rowspan=1 colspan=1>10/27(37.0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>213/875(24.3)</td><td rowspan=1 colspan=1>13/152(8.6)</td><td rowspan=1 colspan=1>13/97(13.4)</td><td rowspan=1 colspan=1>109/324(33.6)</td><td rowspan=1 colspan=1>68/253(26.9)</td><td rowspan=1 colspan=1>2/22(9.1)</td><td rowspan=1 colspan=1>8/27(29.6)</td></tr><tr><td rowspan=2 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>35/875(4.0)</td><td rowspan=1 colspan=1>8/152(11.8)</td><td rowspan=1 colspan=1>6/97(6.2)</td><td rowspan=1 colspan=1>8/324(2.5)</td><td rowspan=1 colspan=1>1/253(1.2)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>35/875(4.0)</td><td rowspan=1 colspan=1>18/152(11.8)</td><td rowspan=1 colspan=1>6/97(6.2)</td><td rowspan=1 colspan=1>8/324(2.5)</td><td rowspan=1 colspan=1>1/253(1.2)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>Influenza A VirusA/H1-2009</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>12/875(1.4)</td><td rowspan=1 colspan=1>9/152(5.9)</td><td rowspan=1 colspan=1>1/97(1.0)</td><td rowspan=1 colspan=1>0/324(0)</td><td rowspan=1 colspan=1>2/253(0.8)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>12/875(1.4)</td><td rowspan=1 colspan=1>9/152(5.9)</td><td rowspan=1 colspan=1>1/97(1.0)</td><td rowspan=1 colspan=1>0/324(0)</td><td rowspan=1 colspan=1>2/253(0.8)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=3 colspan=1>Influenza A Virus A/H3</td><td rowspan=2 colspan=1>SPOTFIRE</td><td rowspan=2 colspan=1>23/875(2.6)</td><td rowspan=2 colspan=1>9/152(5.9)</td><td rowspan=1 colspan=1>5/97</td><td rowspan=1 colspan=1>8/324</td><td rowspan=1 colspan=1>1/253</td><td rowspan=1 colspan=1>0/22</td><td rowspan=1 colspan=1>0/27</td></tr><tr><td rowspan=1 colspan=1>(5.2)</td><td rowspan=1 colspan=1>(2.5)</td><td rowspan=1 colspan=1>(0.4)</td><td rowspan=1 colspan=1>(0)</td><td rowspan=1 colspan=1>(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>23/875(2.6)</td><td rowspan=1 colspan=1>9/152(5.9)</td><td rowspan=1 colspan=1>5/97(5.2)</td><td rowspan=1 colspan=1>8/324(2.5)</td><td rowspan=1 colspan=1>1/253(0.4)</td><td rowspan=1 colspan=1>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>Influenza B Virus</td><td rowspan=2 colspan=1>SPOTFIRE</td><td rowspan=2 colspan=1>4/875(0.5)</td><td rowspan=2 colspan=1>1/152(0.7)</td><td rowspan=2 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>3/324</td><td rowspan=1 colspan=1>0/253</td><td rowspan=2 colspan=1>0/22(0)</td><td rowspan=2 colspan=1>0/28(0)</td></tr><tr><td rowspan=1 colspan=1>(0.9)</td><td rowspan=1 colspan=1>(0)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>TestMethod</td><td rowspan=1 colspan=9>Positive Results Stratified by Study Site (%)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=3>Site 5</td><td rowspan=1 colspan=1>Site 6</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>4/875(0.5)</td><td rowspan=1 colspan=1>1/152(0.7)</td><td rowspan=1 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>3/324(0.9)</td><td rowspan=1 colspan=1>0/253(0)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/28(0)</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>52/875(5.9)</td><td rowspan=1 colspan=1>2/152(1.3)</td><td rowspan=1 colspan=1>5/97(5.2)</td><td rowspan=1 colspan=1>33/324(10.2)</td><td rowspan=1 colspan=1>12/253(4.7)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>52/875(5.9)</td><td rowspan=1 colspan=1>2/152(1.3)</td><td rowspan=1 colspan=1>5/97(5.2)</td><td rowspan=1 colspan=1>33/324(10.2)</td><td rowspan=1 colspan=1>12/253(4.7)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>23/875(2.6)</td><td rowspan=1 colspan=1>4/152(2.6)</td><td rowspan=1 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>11/324(3.4)</td><td rowspan=1 colspan=1>8/253(3.2)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>24/875(2.7)</td><td rowspan=1 colspan=1>3/152(2.0)</td><td rowspan=1 colspan=1>1/97(1.0)</td><td rowspan=1 colspan=1>12/324(3.7)</td><td rowspan=1 colspan=1>8/253(3.2)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>0/875(0)</td><td rowspan=1 colspan=1>0/152(0)</td><td rowspan=1 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>0/324(0)</td><td rowspan=1 colspan=1>0/253(0)</td><td rowspan=1 colspan=3>0/22()</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>0/875(0)</td><td rowspan=1 colspan=1>0/152(0)</td><td rowspan=1 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>0/324(0)</td><td rowspan=1 colspan=1>0/253(0)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>0/875(0)</td><td rowspan=1 colspan=1>0/152(0)</td><td rowspan=1 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>0/324(0)</td><td rowspan=1 colspan=1>0/253(0)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>0/875(0)</td><td rowspan=1 colspan=1>0/152(0)</td><td rowspan=1 colspan=1>0/97(0)</td><td rowspan=1 colspan=1>0/324(0)</td><td rowspan=1 colspan=1>0/253(0)</td><td rowspan=1 colspan=3>0/22(0)</td><td rowspan=1 colspan=1>0/27(0)</td></tr><tr><td rowspan=2 colspan=1>Streptococcusdysgalactiae</td><td rowspan=1 colspan=1>SPOTFIRE</td><td rowspan=1 colspan=1>32/869(3.7)</td><td rowspan=1 colspan=1>4/152(2.6)</td><td rowspan=1 colspan=1>6/97(6.2)</td><td rowspan=1 colspan=1>5/320(1.6)</td><td rowspan=1 colspan=1>16/251(6.4)</td><td rowspan=1 colspan=3>1/21(4.8)</td><td rowspan=1 colspan=1>0/28(0)</td></tr><tr><td rowspan=1 colspan=1>Comparator</td><td rowspan=1 colspan=1>26/869(3.0)</td><td rowspan=1 colspan=1>3/152(2.0)</td><td rowspan=1 colspan=1>5/97(5.2)</td><td rowspan=1 colspan=1>4/320(1.3)</td><td rowspan=1 colspan=1>13/251(5.2)</td><td rowspan=1 colspan=3>1/21(4.8)</td><td rowspan=1 colspan=1>0/28(0)</td></tr><tr><td rowspan=4 colspan=1>Streptococcus pyogenes</td><td rowspan=2 colspan=1>SPOTFIRE</td><td rowspan=2 colspan=1>212/869(24.4)</td><td rowspan=2 colspan=1>15/152(9.9)</td><td rowspan=2 colspan=1>27/97(27.8)</td><td rowspan=2 colspan=1>92/320(28.8)</td><td rowspan=1 colspan=1>68/251</td><td rowspan=1 colspan=3>6/21</td><td rowspan=2 colspan=1>4/28(14.3)</td></tr><tr><td rowspan=1 colspan=1>(27.1)</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>(28.6)</td></tr><tr><td rowspan=2 colspan=1>Comparator</td><td rowspan=2 colspan=1>177/869(20.4)</td><td rowspan=2 colspan=1>10/152(6.6)</td><td rowspan=2 colspan=1>21/97(21.6)</td><td rowspan=1 colspan=1>81/320</td><td rowspan=1 colspan=1>61/251</td><td rowspan=2 colspan=3>1/21(4.8)</td><td rowspan=1 colspan=1>−</td><td rowspan=2 colspan=1>3/28(10.7)</td></tr><tr><td rowspan=1 colspan=1>(25.3)</td><td rowspan=1 colspan=1>(24.3)</td></tr></table>

Only specimens with both valid BIOFIRE SPOTFIRE R/ST Panel and comparator results for the applicable analyte are included in calculation of expected values.

Table 59. Expected values and prevalence of analytes in prospectively collected throat swab specimens, as determined by the BIOFIRE SPOTFIRE R/ST Panel and comparator methods, respectively, stratified by patient age   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">TestMethod</td><td colspan="6" rowspan="1">Positive Results Stratified by Age [years] (%)</td></tr><tr><td colspan="1" rowspan="1">All Ages</td><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">6-18</td><td colspan="1" rowspan="1">19-40</td><td colspan="1" rowspan="1">41-60</td><td colspan="1" rowspan="1">&gt; 60</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="2">SPOTFIRE</td><td colspan="1" rowspan="2">72/875(8.2)</td><td colspan="1" rowspan="2">25/137(18.2)</td><td colspan="1" rowspan="2">41/455(9.0)</td><td colspan="1" rowspan="1">5/186</td><td colspan="1" rowspan="2">1/72(1.4)</td><td colspan="1" rowspan="2">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">(2.7)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">65/875(7.4)</td><td colspan="1" rowspan="1">23/137(16.8)</td><td colspan="1" rowspan="1">37/455(8.1)</td><td colspan="1" rowspan="1">5/186(2.7)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus (seasonal)</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">48/875(5.5)</td><td colspan="1" rowspan="1">3/137(2.2)</td><td colspan="1" rowspan="1">35/455(7.7)</td><td colspan="1" rowspan="1">7/186(3.8)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">42/875(4.8)</td><td colspan="1" rowspan="1">2/137(1.5)</td><td colspan="1" rowspan="1">32/455(7.0)</td><td colspan="1" rowspan="1">6/186(3.2)</td><td colspan="1" rowspan="1">2/72(2.8)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Human Metapneumovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">15/875(1.7)</td><td colspan="1" rowspan="1">4/137(2.9)</td><td colspan="1" rowspan="1">7/455(1.5)</td><td colspan="1" rowspan="1">2/186(1.1)</td><td colspan="1" rowspan="1">2/72(2.8)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">17/875(1.9)</td><td colspan="1" rowspan="1">5/137(3.6)</td><td colspan="1" rowspan="1">7/455(1.5)</td><td colspan="1" rowspan="1">2/186(1.1)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">245/875(28.0)</td><td colspan="1" rowspan="1">58/137(42.3)</td><td colspan="1" rowspan="1">152/455(33.4)</td><td colspan="1" rowspan="1">23/186(12.4)</td><td colspan="1" rowspan="1">9/72(12.5)</td><td colspan="1" rowspan="1">3/25(12.0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">213/875(24.3)</td><td colspan="1" rowspan="1">53/137(38.7)</td><td colspan="1" rowspan="1">131/455(28.8)</td><td colspan="1" rowspan="1">20/186(10.8)</td><td colspan="1" rowspan="1">6/72(8.3)</td><td colspan="1" rowspan="1">3/25(12.0)</td></tr><tr><td colspan="1" rowspan="3">Influenza A Virus</td><td colspan="1" rowspan="2">SPOTFIRE</td><td colspan="1" rowspan="2">35/875(4.0)</td><td colspan="1" rowspan="2">2/137(1.5)</td><td colspan="1" rowspan="2">10/455(2.2)</td><td colspan="1" rowspan="1">15/186</td><td colspan="1" rowspan="1">5/72</td><td colspan="1" rowspan="1">3/25</td></tr><tr><td colspan="1" rowspan="1">(8.1)</td><td colspan="1" rowspan="1">(6.9)</td><td colspan="1" rowspan="1">(12.0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">35/875(4.0)</td><td colspan="1" rowspan="1">2/137(1.5)</td><td colspan="1" rowspan="1">10/455(2.2)</td><td colspan="1" rowspan="1">15/186(8.1)</td><td colspan="1" rowspan="1">5/72(6.9)</td><td colspan="1" rowspan="1">3/25(12.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A Virus A/H1-2009</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">12/875(1.4)</td><td colspan="1" rowspan="1">0/137(0)</td><td colspan="1" rowspan="1">2/455(0.4)</td><td colspan="1" rowspan="1">5/186(2.7)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">2/25(8.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">12/875(1.4)</td><td colspan="1" rowspan="1">0/137(0)</td><td colspan="1" rowspan="1">2/455(0.4)</td><td colspan="1" rowspan="1">5/186(2.7)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">2/25(8.0)</td></tr><tr><td colspan="1" rowspan="2">Influenza A Virus A/H3</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">23/875(2.6)</td><td colspan="1" rowspan="1">2/137(1.5)</td><td colspan="1" rowspan="1">8/455(1.8)</td><td colspan="1" rowspan="1">10/186(5.4)</td><td colspan="1" rowspan="1">2/72(2.8)</td><td colspan="1" rowspan="1">1/25(4.0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">23/875(2.6)</td><td colspan="1" rowspan="1">2/137(1.5)</td><td colspan="1" rowspan="1">8/455(1.8)</td><td colspan="1" rowspan="1">10/186(5.4)</td><td colspan="1" rowspan="1">2/72(2.8)</td><td colspan="1" rowspan="1">1/25(4.0)</td></tr><tr><td colspan="1" rowspan="2">Influenza B Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">4/875(0.5)</td><td colspan="1" rowspan="1">1/137(0.7)</td><td colspan="1" rowspan="1">2/455(0.4)</td><td colspan="1" rowspan="1">1/186(0.5)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">4/875(0.5)</td><td colspan="1" rowspan="1">1/137(0.7)</td><td colspan="1" rowspan="1">2/455(0.4)</td><td colspan="1" rowspan="1">1/186(0.5)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">52/875(5.9)</td><td colspan="1" rowspan="1">25/137(18.2)</td><td colspan="1" rowspan="1">18/455(4.0)</td><td colspan="1" rowspan="1">6/186(3.2)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">52/875(5.9)</td><td colspan="1" rowspan="1">26/137(19.0)</td><td colspan="1" rowspan="1">17/455(3.7)</td><td colspan="1" rowspan="1">6/186(3.2)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">23/875(2.6)</td><td colspan="1" rowspan="1">6/137(4.4)</td><td colspan="1" rowspan="1">13/455(2.9)</td><td colspan="1" rowspan="1">0/186(0)</td><td colspan="1" rowspan="1">3/72(4.2)</td><td colspan="1" rowspan="1">1/25(4.0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">24/875(2.7)</td><td colspan="1" rowspan="1">6/137(4.4)</td><td colspan="1" rowspan="1">14/455(3.1)</td><td colspan="1" rowspan="1">0/186(0)</td><td colspan="1" rowspan="1">2/72(2.8)</td><td colspan="1" rowspan="1">2/25(8.0)</td></tr><tr><td colspan="1" rowspan="2">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/875(0)</td><td colspan="1" rowspan="1">0/137(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/186(0)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/875(0)</td><td colspan="1" rowspan="1">0/137(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/186(0)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">0/875(0)</td><td colspan="1" rowspan="1">0/137(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/186(0)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0/875(0)</td><td colspan="1" rowspan="1">0/137(0)</td><td colspan="1" rowspan="1">0/455(0)</td><td colspan="1" rowspan="1">0/186(0)</td><td colspan="1" rowspan="1">0/72(0)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">32/869(3.7)</td><td colspan="1" rowspan="1">4/138(2.9)</td><td colspan="1" rowspan="1">15/451(3.3)</td><td colspan="1" rowspan="1">11/185(5.9)</td><td colspan="1" rowspan="1">2/72(2.8)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">26/869(3.0)</td><td colspan="1" rowspan="1">2/138(1.5)</td><td colspan="1" rowspan="1">14/451(3.1)</td><td colspan="1" rowspan="1">9/185(4.9)</td><td colspan="1" rowspan="1">1/72(1.4)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="2">Streptococcus pyogenes</td><td colspan="1" rowspan="1">SPOTFIRE</td><td colspan="1" rowspan="1">212/869(24.4)</td><td colspan="1" rowspan="1">35/136(25.7)</td><td colspan="1" rowspan="1">130/451(28.8)</td><td colspan="1" rowspan="1">39/185(21.1)</td><td colspan="1" rowspan="1">8/72(11.1)</td><td colspan="1" rowspan="1">0/25(0)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">177/869(20.4)</td><td colspan="1" rowspan="1">31/136(22.8)</td><td colspan="1" rowspan="1">114/451(25.3)</td><td colspan="1" rowspan="1">28/185(15.1)</td><td colspan="1" rowspan="1">4/72(5.6)</td><td colspan="1" rowspan="1">0/25(0)</td></tr></table>

Only specimens with both valid BIOFIRE SPOTFIRE R/ST Panel and comparator results for the applicable analyte are included in calculation of expected values.

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence determination.

The submitted information in this CLIA waiver application supports a CLIA waiver approval decision.