# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

Vela Operations USA

# C Proprietary and Established Names

Great Basin Toxigenic C. difficile Direct Test (CDF2)

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>OZN</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3130 -Clostridium DifficileToxin Gene AmplificationAssay</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: To obtain substantial equivalence for the Great Basin Toxigenic C. difficile Direct Test (CDF2).   
B Measurand: tcdB gene of toxigenic Clostridioides difficile   
C Type of Test: The Toxigenic C. difficile Direct Test (CDF2) is a polymerase chain reaction (PCR) which amplifies specific sequences that are hybridized onto probes immobilized on a silicon chip surface.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Great Basin Toxigenic C. difficile Direct Test, performed on the Great Basin PA500 Analyzer, is a qualitative in vitro diagnostic test for the detection of toxigenic Clostridioides difficile in unformed (liquid or soft) stool samples collected from patients suspected of having $C .$ . difficile infection (CDI). The automated assay utilizes polymerase chain reaction (PCR) to detect a conserved region of the toxin gene (tcdB) associated with toxin-producing C. difficile.

The Toxigenic C. difficile Direct Test is intended for use as an aid in the diagnosis of CDI in conjunction with clinical and epidemiological risk factors

C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

D Special Instrument Requirements:

Great Basin PA500 Analyzer

# IV Device/System Characteristics:

# A Device Description:

The Great Basin Toxigenic C. difficile Direct Test (CDF2) performed on the PA500 Analyzer utilizes automated, hot-start PCR amplification technology to amplify specific nucleic acid sequences that are then detected using hybridization probes immobilized on a modified silicon chip surface, in a single-use, self-contained test cartridge.

A swab volume of the specimen (raw stool) is first processed using the Sample Preparation Device (SPD). During processing through the SPD, the specimen is infused with a synthetic bacterial sample processing control (SPC). An aliquot of the eluate obtained from the SPD, containing diluted specimen mixed with SPC, is loaded into the sample port of the CDF2 Test Cartridge.

Genomic DNA is extracted from microbial cells and diluted to reduce potential inhibitors of PCR. During the PCR process, biotin-labeled primers direct the amplification of specific nucleic acid sequences within a conserved region of the C. difficile Toxin B (tcdB) gene.

Following PCR, biotin-labeled, amplified target DNA sequences are hybridized to sequence specific probes immobilized on the silicon chip surface and incubated with anti-biotin antibody conjugated to the horseradish peroxidase enzyme (HRP). The unbound conjugate is washed away, then tetramethylbenzidine (TMB) is added to produce a colored precipitate at the location of the probe/target sequence complex.

The resulting signal is detected by the automated PA500 Optical Reader within the PA500 Analyzer System. The SPC undergoes the same extraction, amplification, and detection steps as the sample in order to monitor for inhibitory substances, as well as process inefficiency due to instrument or reagent failure. No operator intervention is required once the sample is loaded into the sample port, and the Toxigenic $C .$ . difficile Direct Test cartridge is loaded into the PA500 Analyzer.

# B Principle of Operation:

The Great Basin System is a fully automated in vitro diagnostic system that includes: the PA500 Analyzer device, single-use SPDs, single-use Toxigenic C. difficile Direct Test Cartridges, and the PA500 Data Analysis Software Program.

The PA500 Analyzer is designed to perform automated sample preparation, target amplification via PCR, chip-based (optical) target detection, and integrated data analysis in less than two hours.

# V Substantial Equivalence Information:

A Predicate Device Name(s): GenePOC CDiff   
B Predicate 510(k) Number(s): K172569   
C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">Toxigenic C. difficileDirect Test (CDF2)K232092</td><td colspan="1" rowspan="1">GenePOC CDiffK172569</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Great Basin ToxigenicC. difficile Direct Test</td><td colspan="1" rowspan="1">GenePOC CDiff Assay</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The Great BasinToxigenic C. difficileDirect Test, performedon the Great BasinPA500 Analyzer, is aqualitative in vitrodiagnostic test for thedetection of toxigenicClostridioides difficilein unformed (liquid orsoft) stool samplescollected from patientssuspected of having C.difficile infection (CDI).The automated assayutilizes polymerasechain reaction (PCR) to</td><td colspan="1" rowspan="1">The GenePOC CDiffassay performed on therevogene instrument isa qualitative in vitrodiagnostic test thatutilizes automatedsample processing andreal-time polymerasechain reaction (PCR) todetect the toxin B(tcdB) gene of toxigenicClostridium difficile (C.difficile) in unformed(liquid or soft) stoolspecimens obtainedfrom patients suspectedof having C. difficile</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">detect a conservedregion of the toxin gene(tcdB) associated withtoxin-producing C.difficile.The Toxigenic C.difficile Direct Test isintended for use as anaid in the diagnosis ofCDI in conjunction withclinical andepidemiological riskfactors.</td><td colspan="1" rowspan="1">infection (CDI). TheGenePOC CDiff assayis intended to aid in thediagnosis of CDI.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Unformed (liquid orsoft) stool specimens</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">toxin B gene (tcdB)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Lysis and DNAExtraction</td><td colspan="1" rowspan="1">Automated (incartridge)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Amplification Technology</td><td colspan="1" rowspan="1">Multiplex PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Detection Probes</td><td colspan="1" rowspan="1">Hybridization Probes</td><td colspan="1" rowspan="1">TaqMan Probes</td></tr><tr><td colspan="1" rowspan="1">Detection Technology</td><td colspan="1" rowspan="1">Colorimetric targetspecific hybridization toprobe on a chip surface,optical reader,automated softwarewith built-in resultinterpretation.</td><td colspan="1" rowspan="1">Real-time PCR(fluorescent signals)and embedded detectionalgorithm</td></tr><tr><td colspan="1" rowspan="1">Sample Volume</td><td colspan="1" rowspan="1">1 swab</td><td colspan="1" rowspan="1">5 ul</td></tr><tr><td colspan="1" rowspan="1">Sample per Run</td><td colspan="1" rowspan="1">One</td><td colspan="1" rowspan="1">Up to 8</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">2 hours</td><td colspan="1" rowspan="1">70 min</td></tr></table>

# VI Standards/Guidance Documents Referenced:

1. Guidance for Clinical Investigators, Industry and FDA Staff: Financial Disclosure by Clinical Investigators. February 2013   
2. Guidance on Informed Consent for In Vitro Diagnostics Device Studies Using Leftover Human Specimens that are not Individually Identifiable. April 2006   
3. Class II Special Controls Guideline Document: Toxin Gene Amplification Assays for the Detection of Clostridium difficile – Guideline for Industry and Food and Drug Administration Staff. August 2015   
4. CLSI MM13-A: Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods: Approved Guideline.   
5. CLSI MM3-A2: Molecular Diagnostic Methods for Infectious Diseases: Approved Guideline

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

A between-site reproducibility study was performed at three sites (two external and one internal) by two operators per site, over five non-consecutive days using two cartridge lots and 24 analyzers. A total of 90 negative sample replicates and 360 positive sample replicates were tested. Two toxigenic C. difficile strains were tested: ATCC 43255 (toxinotype 0, ribotype 087) and ATCC BAA-1805 (toxinotype IIIb, NAP1/ribotype 027).

A five-member panel, tested in triplicate, was contrived using a pool of C. difficile-negative clinical stool matrix at the following target concentrations:

Low positive at 1.5X LoD • Moderate positive at 3X LoD True negative (TN) samples without toxigenic C. difficile

The low positive (1.5X LoD) concentrations tested were $3 . 9 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { { C F U / m L } }$ and $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ CFU/mL for ATCC 43255 and ATCC BAA-1805, respectively. The moderate positive (3X LoD) concentrations tested were $7 . 8 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { C F U / m l }$ and $2 . 2 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ for ATCC 43255 and ATCC BAA-1805, respectively.

Results are shown by site and overall, in Table 1. For the between-site reproducibility, the overall percent agreement was $9 7 . 8 \%$ for the low positive (1.5X) samples and $100 \%$ for moderate positive (3X) samples for ATCC strain 43255. The overall percent agreement was $100 \%$ for the low positive (1.5X) and moderate positive (3X) samples of strain ATCC BAA1805. The overall percent agreement was $9 8 . 9 \%$ for the negative samples. The two false negative results from Site 02 were attributed to assay variability and sampling error. The false positive result from Site 02 was attributed to unintentional sample contamination by the operator. There was no variability in results by operator or by lot.

Table 1. Reproducibility Results by Site and Overall   

<table><tr><td colspan="1" rowspan="2">Panel</td><td colspan="1" rowspan="2">Strain</td><td colspan="4" rowspan="1">Assay Results/Expected</td><td colspan="1" rowspan="2">Agreement Total[95% CI]</td></tr><tr><td colspan="1" rowspan="1">Site 01</td><td colspan="1" rowspan="1">Site 02</td><td colspan="1" rowspan="1">Site 03</td><td colspan="1" rowspan="1">All sites</td></tr><tr><td colspan="1" rowspan="2">LowPositive(1.5X LoD)</td><td colspan="1" rowspan="1">ATCC43255</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">28/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">88/90</td><td colspan="1" rowspan="1">97.8%[92.2%-99.7%]</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1805</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100%[96%-100%]</td></tr><tr><td colspan="1" rowspan="2">ModeratePositive(3X LoD)</td><td colspan="1" rowspan="1">ATCC43255</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100%%[96%-100%]</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-1805</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100%[96%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">98.9%[94%-100%]</td></tr></table>

2. Linearity:

Not applicable

3. Analytical Specificity/Interference:

a. Analytical inclusivity:

The analytical inclusivity of the Toxigenic $C .$ difficile Direct Test (CDF2) assay was evaluated by testing 22 strains of toxigenic C. difficile representing six different toxinotypes (Summarized in Table 2). Quantitated cultures of each C. difficile strain were diluted to $9 . 7 \mathrm { { x } \ 1 0 ^ { 4 } \ C F U / m l }$ in clinical negative stool matrix, and three replicates per strain were tested using six cartridge lots. All strains were detected, which is acceptable.

Table 2. C. difficile Strain Evaluated for Inclusivity   

<table><tr><td rowspan=1 colspan=1>Toxigenic C. difficile Strain</td><td rowspan=1 colspan=5>Toxinotype, Toxin</td></tr><tr><td rowspan=1 colspan=1>ATCC9689</td><td rowspan=17 colspan=5>Toxinotype 0, A+, B+</td></tr><tr><td rowspan=1 colspan=1>ATCC700792</td></tr><tr><td rowspan=1 colspan=1>ATCC17858</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1382</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1808</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1871</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1872</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1874</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2156</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>ATCC51695</td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>ATCC43600</td></tr><tr><td rowspan=1 colspan=1>ATCC43599</td></tr><tr><td rowspan=1 colspan=1>ATCC43596</td></tr><tr><td rowspan=1 colspan=1>ATCC43594</td></tr><tr><td rowspan=1 colspan=1>ATCC17857</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1804</td></tr><tr><td rowspan=1 colspan=1>ATCC43598</td><td rowspan=1 colspan=5>Toxinotype VIII, A-, B+</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1812</td><td rowspan=1 colspan=5>Toxinotype XII, A+, B+</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1803</td><td rowspan=1 colspan=5>Toxinotype IIIc, NAP1, A+, B+</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1814</td><td rowspan=1 colspan=5>Toxinotype XXII, A+, B+</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1875</td><td rowspan=1 colspan=5>Toxinotype V, A+, B+</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2155</td><td rowspan=1 colspan=5>Toxinotype XXII, A+, B+</td></tr></table>

# b. Analytical specificity (exclusivity):

The potential cross-reactivity of the assay was assessed by testing concentrations of 82 non-targeted organisms (69 bacteria, one yeast, three parasites, nine viruses) and human DNA preparations that may be found in unformed stool specimens (see Table 3, Table 4, and Table 5). The test panel included commensal and pathogenic microorganisms from the intestinal tract, species phylogenetically related to $C$ .

difficile, non-pathogenic strains of $C .$ . difficile, and human DNA. Bacteria and yeast were tested at a load of $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ , except for a single strain of Clostridium butyricun ATCC 19398 which was tested at $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ CFU/mL. Nucleic acids from viruses and human DNA were tested at a load of ${ \geq } 1 0 ^ { 6 }$ copies/mL or $\geq 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ The test panel was contrived by spiking quantitated cell cultures or nucleic acid solutions into $C .$ . difficile-negative clinical stool matrix. Each organism was tested in triplicate.

Under the conditions of the study, Salmonella enterica subsp. Enterica serovar Newport gave a false positive result by the assay for one out of three replicates. However, no false positives were observed when the culture stock was regrown, and an additional three replicates were tested. All results were acceptable.

Table 3. Cross Reactivity Panel - Non-toxigenic C. difficile, other Clostridium sp.   

<table><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>ID</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (non-toxigenic)</td><td rowspan=1 colspan=1>ATCC43593</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile (non-toxigenic)</td><td rowspan=1 colspan=1>ATCC43601</td></tr><tr><td rowspan=1 colspan=1>Clostridium bifermentans</td><td rowspan=1 colspan=1>ATCC638</td></tr><tr><td rowspan=1 colspan=1>Clostridiumbutyricum</td><td rowspan=1 colspan=1>ATCC19398</td></tr><tr><td rowspan=1 colspan=1>Clostridium novyi</td><td rowspan=1 colspan=1>ATCC19402</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>ATCC13124</td></tr><tr><td rowspan=1 colspan=1>Clostridium scindens</td><td rowspan=1 colspan=1>ATCC35704</td></tr><tr><td rowspan=1 colspan=1>Clostridium septicum</td><td rowspan=1 colspan=1>ATCC12464</td></tr><tr><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>ATCC15579</td></tr><tr><td rowspan=1 colspan=1>Flavonifractor plautii(formerly Clostridium orbiscindens)</td><td rowspan=1 colspan=1>ATCC 49531</td></tr><tr><td rowspan=1 colspan=1>Hathewaya histolytics (formerlyClostridium histolyticum)</td><td rowspan=1 colspan=1>ATCC 19401</td></tr><tr><td rowspan=1 colspan=1>Paeniclostridium (Clostridium)sordellii</td><td rowspan=1 colspan=1>ATCC 9714</td></tr></table>

Table 4. Cross Reactivity Panel - Other Bacteria   

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC 19606</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">ATCC 51903</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">ATCC 35654</td><td colspan="1" rowspan="1">Porphyromonasasaccharolvtica</td><td colspan="1" rowspan="1">ATCC 27908</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">ATCC 15554</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC700524</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 14579</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 23745</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6896</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">ATCC 49943</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">ATCC 49941</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">ATCC 9250</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">ATCC 25936</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 10145</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">ATCC 13525</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 15038</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovarCholeraesuis</td><td colspan="1" rowspan="1">ATCC 13312</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovarNewinoton</td><td colspan="1" rowspan="1">ATCC 29628</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 29212</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovar Newport</td><td colspan="1" rowspan="1">ATCC 6962</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanB)</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovar Typhi</td><td colspan="1" rowspan="1">ATCC 6539</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC19434</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovarTyphimurium</td><td colspan="1" rowspan="1">ATCC 13311</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 4157</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 35551</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7</td><td colspan="1" rowspan="1">ATCC 43895</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O121:H19</td><td colspan="1" rowspan="1">ATCC BAA-2221</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 29928</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O104:H4</td><td colspan="1" rowspan="1">ATCC BAA-2326</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ATCC 9361</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">ATCC 25929</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermanii</td><td colspan="1" rowspan="1">ATCC 33650</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 25931</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">ATCC 43504</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 43300</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 12598</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 19115</td><td colspan="1" rowspan="1">Streptococcus dys alactiae</td><td colspan="1" rowspan="1">ATCC 43078</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 17802</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">ATCC 49397</td></tr></table>

Table 5. Cross Reactivity Panel - Viruses and Other Organisms   

<table><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">ID</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 40</td><td colspan="1" rowspan="1">ATCC VR-931D</td><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">PromegaG147A</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 41</td><td colspan="1" rowspan="1">ATCC VR-930D</td><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">ATCC VR-1775DQ</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">ATCC PRA-67D</td><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">ATCC30459DO</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">ATCC VR-2018DQ</td><td colspan="1" rowspan="1">Echovirus 4</td><td colspan="1" rowspan="1">ATCC VR-1734D</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI</td><td colspan="1" rowspan="1">ATCC VR-3234SD</td><td colspan="1" rowspan="1">Human Herpesvirus5 (Cytomegalovirus)(DNA)</td><td colspan="1" rowspan="1">ATCC VR-538D</td></tr><tr><td colspan="1" rowspan="1">Norovirus GII</td><td colspan="1" rowspan="1">ATCC VR-3235SD</td><td colspan="1" rowspan="1">Giardia intestinalis</td><td colspan="1" rowspan="1">ATCC 50803D</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B2</td><td colspan="1" rowspan="1">ATCC VR-29</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

# c. Microbial Interference

A microbial interference study was conducted to assess the potential inhibitory effect of 82 non-targeted microorganisms (See Table 6) that may be found in stool specimens. Pools of three to five organisms were prepared in $C .$ . difficile negative clinical stool matrix and tested in three replicates each in the presence of toxigenic $C .$ difficile strain ATCC 43255. The bacteria/fungi, DNA/RNA, and viruses were spiked at ${ \geq } 1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ , ${ \geq } 1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ copies $/ \mathrm { m L }$ , and $1 \mathrm { x } 1 0 ^ { 6 }$ T $\mathrm { \ ' C I D 5 0 / m L }$ , respectively. The toxigenic $C .$ . difficile was spiked at 2X LoD of this strain $( 5 . 2 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { { C F U / m L } ) }$ ).

No interference was observed for the detection of $C .$ . difficile strain ATCC 43255.

Table 6. Microbial Interference Panel   

<table><tr><td colspan="1" rowspan="1">Pool</td><td colspan="1" rowspan="1">Potentially Interfering Microorganisms</td></tr><tr><td colspan="1" rowspan="5">1</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td></tr><tr><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td></tr><tr><td colspan="1" rowspan="1">Paeniclostridium (Clostridium) sordellii</td></tr><tr><td colspan="1" rowspan="5">3</td><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157:H7</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="5">4</td><td colspan="1" rowspan="1">Escherichia fergusonii</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli hermanii</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anerobius</td></tr><tr><td colspan="1" rowspan="3">5</td><td colspan="1" rowspan="1">Salmonella enterica serovarCholeraesuis</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Newington</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Newport</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Salmonella enterica serovar Typhi</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarTyphimurium</td></tr><tr><td colspan="1" rowspan="5">6</td><td colspan="1" rowspan="1">Serratia liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="5">7</td><td colspan="1" rowspan="1">Candida albicans</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Yersinia entercolitica</td></tr><tr><td colspan="1" rowspan="5">8</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. Arizonae</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="5">9</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td></tr><tr><td colspan="1" rowspan="5">10</td><td colspan="1" rowspan="1">Abiotrophia defective</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td></tr><tr><td colspan="1" rowspan="1">Clostridium butyricum</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td></tr><tr><td colspan="1" rowspan="5">11</td><td colspan="1" rowspan="1">Hathewaya histolytica (Clostridiumhistolyticum)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus lactis</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Vibrio paraharmolyticus</td></tr><tr><td colspan="1" rowspan="5">12</td><td colspan="1" rowspan="1">Clostridium sporogenes</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td></tr><tr><td colspan="1" rowspan="5">13</td><td colspan="1" rowspan="1">Flavonifractor plautii(formerly Clostridium orbiscindens)</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Newport</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Adenovirus type 40</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">Enterovirus 71</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI</td></tr><tr><td colspan="1" rowspan="1">Norovirius GII</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td></tr><tr><td colspan="1" rowspan="5">15</td><td colspan="1" rowspan="1">Clostridium scindens</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (van B)</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas asaccharolytica</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td></tr><tr><td colspan="1" rowspan="5">16</td><td colspan="1" rowspan="1">Adenovirus type 41</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B2</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus 5 (Cytomegalovirus)</td></tr><tr><td colspan="1" rowspan="1">Echovirus 4</td></tr><tr><td colspan="1" rowspan="1">Giardia intestinalis</td></tr><tr><td colspan="1" rowspan="3">17</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td></tr><tr><td colspan="1" rowspan="1">Human gDNA</td></tr></table>

# d. Interfering Substances

An interfering substances study was conducted to assess the potentially inhibitory effects of 30 substances (Table 7) that may be present in stool specimens. The substances were tested in triplicate in the presence of two strains of toxigenic C. difficile (ATCC 43255 and ATCC BAA-1805) in $C .$ . difficile negative clinical stool matrix. Substances were spiked at higher concentrations than expected in stool specimens, and C. difficile strains were spiked at 2X LoD (ATCC 43255 at $5 . 2 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { C F U / m L }$ , and ATCC BAA-1805 at $1 . 4 \mathrm { ~ x ~ i 0 ^ { 5 } \mathrm { C F U / m L } }$ ).

No interference was observed for the 30 substances at the tested concentrations.

Table 7. Substance Interference Panel   

<table><tr><td colspan="1" rowspan="1">Substance (commercial name)</td><td colspan="1" rowspan="1">Active Ingredient(s)</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Vaginal antifungal / anti-itch(Nystatin)</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">10% w/v</td></tr><tr><td colspan="1" rowspan="1">Creams / ointments(Cortizone-10)</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">30% w/v</td></tr><tr><td colspan="1" rowspan="1">Anti-hemorrhoidal creams /ointments (Preparation H)</td><td colspan="1" rowspan="1">Phenylephrine HCI</td><td colspan="1" rowspan="1">20% w/v</td></tr><tr><td colspan="1" rowspan="1">Antacids (Tums)</td><td colspan="1" rowspan="1">Calcium Carbonate</td><td colspan="1" rowspan="1">38% w/v</td></tr><tr><td colspan="1" rowspan="1">Mineral Oil(Fisher Bioreagents Cat# B2629-1)</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Enemas(Mesalazine Sigma Aldrich Cat#Y0000297 )</td><td colspan="1" rowspan="1">Mesalazine</td><td colspan="1" rowspan="1">2.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Condom with spermicidal lubricant(Trojan)</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">40% w/v</td></tr><tr><td colspan="1" rowspan="1">Anti-diarrheal medication(Pepto Bismol)</td><td colspan="1" rowspan="1">Bismuth Subsalicylate</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Anti-diarrheal medication(Imodium)</td><td colspan="1" rowspan="1">LoperamideHydrochloride</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Laxatives(Senna Glycoside Tablet)</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">17% w/v</td></tr><tr><td colspan="1" rowspan="1">Antibiotics(Vancomycin 100mg/mL in DMSO-Sigma Cat# SBR00001)</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">5% w/v</td></tr><tr><td colspan="1" rowspan="1">Antibiotics(Metronidazole  MP BiomedicalsCat# 155710)</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">0.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Non-steroidal anti-inflammatory(Aleve)</td><td colspan="1" rowspan="1">Naproxen Sodium</td><td colspan="1" rowspan="1">10% w/v</td></tr><tr><td colspan="1" rowspan="1">Moist towelettes (Wet Ones)</td><td colspan="1" rowspan="1">Benzalkonium Chloride</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Suppository(Fleet)</td><td colspan="1" rowspan="1">Liquid Glycerin</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Personal lubricant(KY)</td><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Personal lubricant(Astroglide)</td><td colspan="1" rowspan="1">Silicone</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Aloe leaf juice flushable wipes(Generic)</td><td colspan="1" rowspan="1">Butoxy PEG-4 PG-Amodimethicone</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Anti-fungal cream(Miconazole-3)</td><td colspan="1" rowspan="1">Miconazole Nitrate</td><td colspan="1" rowspan="1">30% w/v</td></tr><tr><td colspan="1" rowspan="1">Petroleum Jelly(Vaseline)</td><td colspan="1" rowspan="1">Petroleum</td><td colspan="1" rowspan="1">30% w/v</td></tr><tr><td colspan="1" rowspan="1">Biotin Supplement(Nature Made, 2500 mcg/capsule)</td><td colspan="1" rowspan="1">Biotin</td><td colspan="1" rowspan="1">51% w/va</td></tr><tr><td colspan="1" rowspan="1">Contrast agent</td><td colspan="1" rowspan="1">Barium Sulfate</td><td colspan="1" rowspan="1">25% w/v</td></tr><tr><td colspan="1" rowspan="1">Stearic Acid(ScienceLab Cat# SLS3742</td><td colspan="1" rowspan="1">Stearic Acid</td><td colspan="1" rowspan="1">1.1% w/v</td></tr><tr><td colspan="1" rowspan="1">Palmitic Acid(Sigma Cat# P0500)</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">1.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Whole Blood(Fisher Scientific Cat# NC0088224)</td><td colspan="1" rowspan="1">Blood components</td><td colspan="1" rowspan="1">40% v/vb</td></tr><tr><td colspan="1" rowspan="1">Biotin(Sigma Cat# B4501)</td><td colspan="1" rowspan="1">Biotin</td><td colspan="1" rowspan="1">0.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Nonoxynol-9(Sigma Aldrich Cat# PHR2731)</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">48% v/v</td></tr><tr><td colspan="1" rowspan="1">Mucin from Porcine Stomach(Sigma Cat# M1778)</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">2.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Topical antibiotic(Neosporin)</td><td colspan="1" rowspan="1">Bacitracin, Neomycin,Polymyxin B</td><td colspan="1" rowspan="1">50% w/v</td></tr><tr><td colspan="1" rowspan="1">Nystatin(Sigma Cat# N6261)</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">10% w/v</td></tr><tr><td colspan="1" rowspan="1">Erythromycin(Alfa Aesar Cat# J62279)</td><td colspan="1" rowspan="1">Erythromycin</td><td colspan="1" rowspan="1">10% w/v</td></tr></table>

a Input listed is the total amount of the gel-capsule biotin supplement tested and equates to approximately $1 . 3 ~ \mathrm { m g }$ of biotin per swab.   
b Raw stool was formulated to $40 \%$ whole blood by volume $\mathbf { \Pi } ( \mathbf { v } / \mathbf { v } )$ . One (1) swab volume of the mixture was evaluated in the CDF2 test.

4. Assay Reportable Range: Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

e. Quality Control:

The Great Basin Toxigenic C. difficile Direct Test (CDF2) assay uses three internal controls: a sample processing control (SPC), a hybridization control (HC), and a detection control (DC). The SPC is included in the Sample Preparation Device supplied with each kit. The SPC is extracted, amplified, and detected along with each specimen tested, and it verifies the efficacy of the sample preparation, PCR amplification, and detection processes. The HC is a control oligonucleotide attached to the silicon chip surface. A biotinylated oligonucleotide with complementary sequence to the HC is added to the hybridization buffer and the oligonucleotide pair provides verification that probe hybridization is functioning properly. The DC is a biotinylated oligonucleotide probe attached to the silicon chip surface that provides a control for the performance of the conjugate and detection reagents for proper signal development.

The use of two external controls is recommended: a positive external control (PEC), and negative external control (NEC). The PEC monitors for substantial reagent failure, and the NEC monitors for environmental contamination or carryover. Each laboratory must establish the number, type and frequency of testing external control materials per applicable regulations or accrediting agencies. External Control materials are recommended but not provided. External Controls are treated as if they are specimens.

Optional PEC materials include:

ZeptoMetrix Clostridium difficile Positive Control IVD CE (Catalog No. NATCDI-6MC-IVD).   
Remnants of a known C. difficile positive clinical stool sample or   
Culture of a known C. difficile strain (e.g., Microbiologics KWIK-STIK C. difficile ATCC 9689, catalog no. 0329)

Optional NEC materials include:

ZeptoMetrix Clostridium difficile Negative Control IVD CE (Catalog No. NATCSO-6MC-IVD) or Remnants of a known C. difficile negative clinical stool sample.

One PEC and one NEC were processed each day of analytical and clinical testing. Of the total 361 external controls that were run across all sites in the clinical study, 350 $( 9 7 . 0 \% )$

were valid, eight $( 2 . 2 \% )$ were canceled by the end-user (test incomplete), and three $( 0 . 8 \% )$ were invalid due to a failed internal control. All failed results repeated successfully.

# f. Sample Stability:

Specimen stability was evaluated analytically by testing pre-characterized positive and negative clinical samples under different storage time and temperature conditions. The results supported the stool specimen storage instructions described in the package insert.

Stool specimens can be stored at refrigerated $( 2 ^ { \circ } { - } 8 ^ { \circ } \mathrm { C } )$ for up to 72 hours (three days), or at room temperature $( 1 8 ^ { \circ } - 2 2 ^ { \circ } \mathrm { C } )$ ) for up to 24 hours.

# g. Freeze/thaw Stability:

Studies were conducted to examine the stability of stool specimens when subjected to freeze/thaw cycles. Contrived positive stool samples contained 5X LoD and 2X LoD of $C$ . difficile (ATCC 43255). The study results supported a recommendation that stool samples stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ can undergo no more than one freeze/thaw cycles.

# 6. Detection Limit:

The analytical sensitivity (Limit of Detection or LoD) of the Toxigenic C. difficile Direct Test (CDF2) assay was determined using negative clinical stool specimens spiked with various concentrations of toxigenic C. difficile bacterial suspensions. The testing to establish and confirm the LoD included four lots, multiple instruments, and operators. Two strains of toxigenic $C .$ . difficile (ATCC 43255, Toxinotype 0, tcdA $^ +$ , tcdB+, and ATCC BAA-1805, NAP 1/027, Toxinotype IIIb, $t c d A +$ , tcdB+) were tested in a minimum of 20 replicates per concentration in the confirmation study. The LoD was defined as the lowest concentration at which $9 5 \%$ or more of all replicates tested positive. The LoD was determined at $2 . 6 \mathrm { \ : X \ : 1 0 ^ { 4 } \ : C F U / m L }$ for ATCC 43255 and $7 . 1 \mathrm { \dot { X } } 1 0 ^ { 4 } \mathrm { C F U } / \mathrm { m l }$ for ATCC BAA-1805.

7. Assay Cut-Off: Not applicable

# B Comparison Studies:

1. Method Comparison with Predicate Device:

The performance of the Toxigenic C. difficile Direct Test (CDF2) was assessed with remnants of unpreserved stool specimens prospectively collected from symptomatic individuals suspected of having an C. difficile infection. Specimens were collected from three independent clinical sites. Stool specimens were tested with a high-performing FDA-cleared molecular $C .$ . difficile assay, which served as the basis for performance evaluation of this assay. Specimens with discrepant results were tested with an alternative FDA-cleared molecular C. difficile assay, performance of which is footnoted accordingly, but the original performance was not changed. Prospective testing consisted of 829 stool specimens that met the inclusion criteria.

The ages of patients ranged from less than 2 years to over 89 years with $5 3 . 3 \%$ of the specimens coming from patients that were 19-64 years of age. Of the 829 patients tested, $52 \%$ were female, $4 7 . 4 \%$ were male, and $0 . 6 \%$ were undeclared gender.

Of the 829 eligible specimens, ten specimens generated false positive with the Toxigenic $C .$ difficile Direct Test (CDF2) and six specimens generated false negative results.

Table 8 and 9 show a summary of the clinical performance of the Toxigenic C. difficile Direct Test (CDF2) for all sites individually and combined. The PPA and NPA values for the Toxigenic C. difficile Direct Test (CDF2) were $9 5 . 8 \%$ and $9 8 . 5 \%$ , respectively. There was no observable difference in performance of the Toxigenic $C .$ difficile Direct Test (CDF2) with respect to study site, kit lot, or patient gender or age. These results are acceptable.

Table 8. Clinical Study Results   

<table><tr><td rowspan=2 colspan=2>All Sites</td><td rowspan=1 colspan=3>Molecular Comparator</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>ToxigenicC. difficileDirect Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>10*</td><td rowspan=1 colspan=1>148</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6*</td><td rowspan=1 colspan=1>675</td><td rowspan=1 colspan=1>681</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>685</td><td rowspan=1 colspan=1>829</td></tr></table>

\* All 16 discrepant results were tested with the alternative FDA-cleared molecular C. difficile assay. None of the 10 false positive samples were positive and five of the six false negatives were negative with the alternative assay.

Table 9. Summary of Clinical Performance   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td><td rowspan=1 colspan=1>PPV(95% CI)</td><td rowspan=1 colspan=1>NPV(95% CI)</td></tr><tr><td rowspan=1 colspan=1>ToxigenicC. difficileDirect Test</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>95.8%(91.2 - 98.5)138/144</td><td rowspan=1 colspan=1>98.5%(97.3  99.3)675/685</td><td rowspan=1 colspan=1>93.2%(87.9 - 96.7)</td><td rowspan=1 colspan=1>99.1%(98.1  99.7)</td></tr></table>

2. Matrix Comparison: Not applicable

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

# 3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

Not applicable

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

In the clinical studies, a total of 829 specimens were enrolled at three geographically diverse clinical sites, from subjects ranging in age from less than two years old to more than 89 years old. Of these specimens, the overall percentage of positive results observed with the Toxigenic C. difficile Direct Test (CDF2) with these specimens was $1 7 . 4 \%$ [144/829; $9 5 \% \mathrm { C I }$ : 14.9 –20.1 $\% ]$ .

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.