# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K232672   
B Applicant Genetic Signatures Limited   
C Proprietary and Established Names EasyScreen Gastrointestinal Parasite Detection Kit   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PCH</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3990 - Gastrointestinal MicroorganismMultiplex Nucleic Acid-Based Assay</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>OOI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 - Instrumentation for clinicalmultiplex test systems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

II Submission/Device Overview:

III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Genetic Signatures EasyScreen Gastrointestinal Parasite Detection Kit is a rapid in vitro nucleic acid amplification assay for the qualitative detection of pathogenic gastrointestinal parasite nucleic acid from the stool of patients with signs and/or symptoms of gastroenteritis. The test, based on real-time PCR, detects the nucleic acid of the following organisms:

Cryptosporidium spp. • Giardia intestinalis Dientamoeba fragilis Entamoeba histolytica • Blastocystis hominis • Enterocytozoon bieneusi

The kit is compatible with stool specimens that are unpreserved or frozen or in transport media including Cary Blair or C&S media from symptomatic patients with suspected gastroenteritis. It is required that the stool is first processed using the EasyScreen Sample Processing Kit. Nucleic acid extraction and real-time PCR set up are performed on the automated Genetic Signatures GS1 platform.

The EasyScreen Gastrointestinal Parasite Detection Kit includes all reagents required to detect the specific protozoan gene sequences using real-time PCR amplification of the extracted nucleic acids and fluorogenic target-specific hybridization probes for the detection of the amplified nucleic acid. The EasyScreen Gastrointestinal Parasite Detection kit also incorporates an Extraction Control (EC) and an Internal Positive Control (IPC) to ensure the reliability of the extracted nucleic acid and to detect the presence of any inhibitors, respectively.

This device is an in vitro diagnostic (IVD) intended to be used by trained personnel in clinical, pathology or hospital laboratories as an aid in the diagnosis of gastrointestinal illness. This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of infections by Dientamoeba fragilis, Blastocystis hominis, Enterocytozoon bieneusi, Cyclospora cayetanensis, Entamoeba histolytica, Encephalitozoon intestinalis, Cryptosporidium spp. (including C. hominis and $C .$ . parvum), and Giardia intestinalis. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decision. Positive results do not rule out coinfection with other organisms that are not detected by this test, and may not indicate the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis and/or colitis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

Following limitations apply: (not an exhaustive list)

i. Whole blood at greater than $0 . 6 3 \%$ and Mucin at greater than $0 . 7 5 ~ \mathrm { m g / m L }$ demonstrated interference with the EasyScreen Gastrointestinal Parasite Detection Kit. Caution must be exercised in interpretation of the results of this assay when testing stool samples with blood and/or mucin potentially present at or above these concentrations.   
ii. This test has been validated for use with only QuantStudio Dx and Applied Biosystems 7500 Fast Dx real-time PCR systems. Other real-time PCR analyzers from different manufacturers may perform differently and should not be used with this this test.   
iii. The performance of this test has been validated with stool specimen that has been collected from patients with signs and symptoms of gastroenteritis and is either unpreserved fresh or diluted in Cary Blair or C&S transport medium per manufacturer’s instructions. It has not been validated for other specimen types, rectal swabs or samples stored in other transport media or fixatives.   
iv. This kit is only compatible with nucleic acid from stool samples prepared using an EasyScreen Sample Processing Kit (SP008B).   
v. Samples subjected to more than one thaw from frozen storage have not been validated for use with the EasyScreen Gastrointestinal Parasite Detection Kit (EP005) and should not be used for the test.

vi. Performance of this assay has been validated only for the target organisms indicated in the Intended Use statement. It has not been validated for any other non-target organisms. The assay is qualitative and does not provide a quantitative value for target organisms present in the stool specimens.

vii. During competitive interference testing, the detection of low positive $D$ . fragilis and $B$ . hominis was attenuated by presence of high amounts of co-targets $C .$ . parvum and $E$ . histolytica, respectively. In regions of high prevalence or endemicity of Cryptosporidium spp and Entamoeba histolytica, negative EasyScreen Gastrointestinal Parasite Detection Kit assay results should be interpreted cautiously. Perform additional tests if gastrointestinal infections/co-infections by $D .$ . fragilis and $B$ . hominis, respectively, are clinically suspected.

# D Special Instrument Requirements:

Genetic Signatures GS1 • QuantStudio Dx Real-Time PCR Instrument with QuantStudio Dx software Applied Biosystems 7500 Fast Dx Real-Time PCR instrument with SDS Software

# IV Device/System Characteristics:

# A Device Description:

EasyScreen Gastrointestinal Parasite Detection Kit consists of four (4) components:

1. The EasyScreen Sample Processing Kit (SP008B), which is a set of reagents for specimen processing, extraction, and purification of nucleic acids. (Required but not provided.)

2. Genetic Signatures GS1, which is an automated platform for processing of human stool samples for the extraction of purified nucleic acids (using SP008B reagents) and setting up of real time PCR runs (using EP005 or EP005-HT reagents). (Required but not provided.)

3. The EasyScreen Gastrointestinal Parasite Detection Kit (EP005, for 100 reactions; EP005-HT, for 500 reactions), which is a set of PCR components, reaction mastermixes, and controls designed for multiplexed detection and identification of eight (8) potentially pathogenic gastrointestinal parasites via real-time PCR.

Amplified targets are detected by fluorophore-labeled probes. The real-time PCR is an 8-plex assay, detecting the following protozoan targets in each fluorescent channel:

<table><tr><td>Detection Channel</td><td>Available Fluorophores</td><td>Reaction Mix (Panel) A</td><td>Reaction Mix (Panel) B</td><td>Reaction Mix (Panel) C</td></tr><tr><td>Channel 1</td><td>FAM, etc.</td><td>Dientamoeba fragilis</td><td>Blastocystis hominis</td><td>Enterocytozoon bieneusi</td></tr><tr><td>Channel 2</td><td>VIC, HEX, TET, etc.</td><td>Extraction Control (EC)</td><td>Internal Process Control (IPC)</td><td>Extraction Control (EC)</td></tr><tr><td>Channel 3</td><td>ROX, Texas Red,</td><td>Cyclospora</td><td>Entamoeba</td><td>Encephalitozoon</td></tr><tr><td>Channel</td><td>etc.</td><td>cayetanensis</td><td>histolytica</td><td>intestinalis</td></tr><tr><td>4</td><td>Cy5, LightCycler Red640, etc.</td><td>Cryptosporidium spp.</td><td>Giardia intestinalis</td><td>Not used</td></tr></table>

4. Real-Time PCR Instruments:

a) QuantStudio Dx Real-Time PCR Instrument with QuantStudio Dx software (K123955). b) Applied Biosystems 7500 Fast Dx Real-Time PCR instrument with SDS Software (K141220).

Additional components indicated as “Required but Not Supplied” are specified in the package insert and include items and laboratory equipment for sample processing and PCR set up.

# B Principle of Operation:

1. EasyScreen Gastrointestinal Parasite Detection Kit is a multiplexed real time PCR assay, designed to simultaneously detect and identify up to eight (8) potentially pathogenic gastrointestinal parasites from human stool samples.

2. The device workflow is compatible with only nucleic acids prepared using the EasyScreen Sample Processing kit (SP008B) in the GS1 automated nucleic acid extraction platform.

3. Kit SP008B contains Reagent 1 and 2 which are combined to make a “Conversion Reagent.” Individual stool samples are mixed and heated with this conversion reagent which lyses microbial cells and converts all Cytosine bases to Uracil (detected as Thymine) to create “3base nucleic acids.”

4. Kit SP008B also contains Nucleic Acid extraction reagents, which are used on the stool lysate for further processing to elute the nucleic acids on the GS1 automated platform. The purified eluate proceeds to the PCR set-up step in the GS1 platform.

5. EasyScreen Gastrointestinal Parasite Detection Kit (EP005 or EP005-HT) contains all the reagents to be used for real-time PCR amplification and detection of the target gastrointestinal parasites. These reagents are appropriately dispensed to uniquely designated sample wells in the GS1 platform.

6. In the GS1 platform, aliquots of the purified nucleic acids eluted from the processed stool sample are then added to EP005 PCR reagents in designated sample wells, which selectively amplify the genetic targets of the target parasites. Each stool eluate is tested across three (3) Reaction Mixes or Panels (i.e., Panel A, B, and C), each of which include an internal control. Panel A and C detect an endogenous gene contained in bacteria present in the stool sample via the Extraction Controls (EC), which act as sampling and extraction controls. Panel B features an incorporated Internal Positive Control (IPC) sequence which acts to detect the presence of any inhibitors after extraction from the primary sample. The IPC also confirms the integrity of the PCR reagents.

7. The amplification reactions in presence of the EP005 PCR Reagents are carried out by the Life Technologies QuantStudio Dx or Applied Biosystems 7500 Fast Dx thermocycler systems. The real-time PCR software displays all collected data and provides the raw threshold cycle (Ct) values for automated analysis using software tools developed by Genetic Signatures.

# C Instrument Description Information:

1. Instrument Name: Genetic Signatures GS1

2. Specimen Identification:

Each specimen barcode is scanned into the system by the user when loading the sample tube into the pedestal. Specimen barcodes are tracked by software tools developed by Genetic Signatures through nucleic acid extraction, PCR setup, run, and result analysis, as well as through result reporting into Laboratory Information Systems at the Customer / End User establishment.

3. Specimen Sampling and Handling:

Specimens are aliquoted into a reagent tube and heated for 3base conversion. After the heating step, the inoculated sample tubes are scanned and loaded onto the GS1 instrument. During extraction, the instrument pipettes samples into the processing plate and, at the end of the process, into the final elution plate. These samples are then pipetted by GS1 from the extraction elution plate into the PCR plate. Sample placements in the elution and PCR plates are tracked via the software.

4. Calibration:

At each semi-annual preventative maintenance, the instrument is calibrated using a manufacturer calibration block and software. If an instrument is found to be out of calibration, alignment is checked and adjusted as needed, then calibration is repeated.

5. Quality Control:

Upon installation, qualification procedures for installation (IQ), operation (OQ), and performance (PQ) are completed by Genetic Signatures to verify performance of the instrument and assay. After each preventative maintenance, OQ is performed. A daily maintenance and weekly maintenance are required before operating the instrument.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BD MAX Enteric Parasite Panel, BD MAX System   
B Predicate 510(k) Number(s): K143648

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">Device: K232672</td><td colspan="1" rowspan="1">Predicate: K143648</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">EasyScreen Gastrointestinal ParasiteDetection Kit</td><td colspan="1" rowspan="1">BD MAX Enteric ParasitePanel</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications ForUse</td><td colspan="1" rowspan="1">The Genetic Signatures EasyScreenGastrointestinal Parasite Detection Kitis a rapid in vitro nucleic acidamplification assay for the qualitativedetection of pathogenic gastrointestinalparasite nucleic acid from the stool ofpatients with signs and/or symptoms ofgastroenteritis. The test, based on real-time PCR, detects the nucleic acid ofthe following organisms:Cryptosporidium spp.Giardia intestinalisDientamoeba fragilisEntamoeba histolytica</td><td colspan="1" rowspan="1">The BD MAX EntericParasite Panel performed onthe BD MAX System is anautomated in vitro diagnostictest for the direct qualitativedetection of enteric parasiticpathogens. The BD MAXEnteric Parasite Panel detectsnucleic acids from:Giardia lambliaCryptosporidium (C.hominis and C. parvumonly)Entamoeba histolytica</td></tr><tr><td>Device &amp; Predicate Device(s):</td><td>Device: K232672</td><td>Predicate: K143648</td></tr><tr><td></td><td>Blastocystis hominis Enterocytozoon bieneusi Encephalitozoon intestinalis Cyclospora cayetanensis The kit is compatible with stool specimens that are unpreserved or frozen or in transport media including Cary Blair or C&amp;S media from symptomatic patients with suspected gastroenteritis. It is required that the stool is first processed using the EasyScreen Sample Processing Kit. Nucleic acid extraction and real-time PCR set up are performed on the automated Genetic Signatures GS1 platform. The EasyScreen Gastrointestinal Parasite Detection Kit includes all reagents required to detect the specific protozoan gene sequences using real- time PCR amplification of the extracted nucleic acids and fluorogenic target- specific hybridization probes for the detection of the amplified nucleic acid. The EasyScreen Gastrointestinal Parasite Detection kit also incorporates an Extraction Control (EC) and an Internal Positive Control (IPC) to ensure the reliability of the extracted nucleic acid and to detect the presence of any inhibitors, respectively. This device is an in vitro diagnostic (IVD) intended to be used by trained personnel in clinical, pathology or hospital laboratories as an aid in the diagnosis of gastrointestinal illness. This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of infections by Dientamoeba fragilis, Blastocystis hominis, Enterocytozoon bieneusi, Cyclospora cayetanensis, Entamoeba histolytica, Encephalitozoon intestinalis, Cryptosporidium spp. (including C.</td><td>Testing is performed on unpreserved or 10% formalin-fixed stool specimens from symptomatic patients with suspected gastroenteritis, enteritis or colitis. The assay is intended to aid in the diagnosis of gastrointestinal infection when used in conjunction with clinical evaluation and other laboratory findings. The test is performed directly on the specimen, utilizing real- time polymerase chain reaction (PCR) for the amplification of specific targets. The test utilizes fluorogenic gene-specific hybridization probes for detection of the amplified DNA. This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of Giardia lamblia, Cryptosporidium hominis and C. parvum, as well as Entamoeba histolytica infections. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decision. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis and/or colitis may be due to infection by pathogens that are not detected by this test or</td></tr><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">Device: K232672</td><td colspan="1" rowspan="1">Predicate: K143648</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">intestinalis. Results of this test shouldnot be used as the sole basis fordiagnosis, treatment, or other patientmanagement decision. Positive resultsdo not rule out co-infection with otherorganisms that are not detected by thistest, and may not indicate the sole ordefinitive cause of patient illness.Negative results in the setting ofclinical illness compatible withgastroenteritis and/or colitis may be dueto infection by pathogens that are notdetected by this test or non-infectiouscauses such as ulcerative colitis,irritable bowel syndrome, or Crohn'sdisease.</td><td colspan="1" rowspan="1">bowel syndrome, or Crohn'sdisease.</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Multiplex nucleic acid amplificationand detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen types</td><td colspan="1" rowspan="1">Unpreserved (fresh) stoolStool specimen in preservative(see below for differences in thepreservatives used)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Organisms detected</td><td colspan="1" rowspan="1">Dientamoeba fragilisBlastocystis hominisEnterocytozoon bieneusiEncephalitozoon intestinalisCyclospora cayetanensis•Giardia lambliaCryptosporidium sppEntamoeba histolytica</td><td colspan="1" rowspan="1">Giardia lambliaCryptosporidium hominisC. parvumEntamoeba histolytica</td></tr><tr><td colspan="1" rowspan="1">Specimen preservativetypes and collectionconditions</td><td colspan="1" rowspan="1">Stool specimens in Cary Blair and C&amp;Stransport media</td><td colspan="1" rowspan="1">10% formalin-fixed stoolspecimens</td></tr><tr><td colspan="1" rowspan="1">Sample processing andPCR analysis platform</td><td colspan="1" rowspan="1">Genetic Signatures GS-1Thermo Scientific QuantStudio DxApplied Biosystems 7500 Dx</td><td colspan="1" rowspan="1">BD Max System</td></tr></table>

# VI Standards/Guidance Documents Referenced:

<table><tr><td>Standard</td><td>Document title</td><td>FDA-Recognition</td><td>Use</td></tr><tr><td>Class II Special Controls Guideline (2015)</td><td>Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens</td><td>FDA guidelines</td><td>Clinical and analytical studies</td></tr><tr><td>CLSI EP37, 1st Ed. (2018) CLSI EP39, 1st Ed</td><td>Supplemental Tables for Interference Testing in Clinical Chemistry</td><td>Complete</td><td>Interference study</td></tr><tr><td>(2021)</td><td>A Hierarchical Approach to Selecting Surrogate Samples for the Evaluation of In Vitro Medical Laboratory Tests</td><td>Complete</td><td>Carry over / cross contamination study</td></tr></table>

# A Analytical Performance:

# 1. Precision/Reproducibility:

Site-to-site reproducibility of the EasyScreen Gastrointestinal Parasite Detection Kit was evaluated in two (2) separate studies—the first study with four (4) targets, namely, $D .$ . fragilis, C. parvum, $E .$ . bieneusi, and $E$ . intestinalis; and a second study with remaining four (4) targets, namely, $C .$ . cayetanensis, $B .$ . hominis, $E .$ . histolytica, and $G .$ . intestinalis—as described below.

# A. First Study (D. fragilis, $C .$ parvum, $E .$ bieneusi, and E. intestinalis)

For the indicated targets, site-to-site reproducibility of the EasyScreen Gastrointestinal Parasite Detection Kit was evaluated at three (3) clinical sites, two (2) in the US and one (1) outside the US (OUS), with two (2) operators at each site for ten (10) days.

For each of the indicated target analyte, a sample panel was contrived by spiking an unpreserved negative stool matrix with varying analyte concentrations relative to the target’s pre-determined LoD, e.g., a Low Positive (LP) at 2X LoD and a Moderate Positive (MP) at 4X LoD. Each panel also featured a True Negative (TN) sample, a negative clinical stool sample with no (or an undetectable amount of the) target analyte.

Each positive panel member was tested in triplicate for each of five (5) extraction runs resulting in a total of 90 (i.e., $3 \mathrm { x } 2 \mathrm { x } 5 \mathrm { x } 3 $ ) data points per panel member. The negative samples were tested at the three (3) sites by two (2) operators at each site for a total of ten (10) extraction runs in triplicate resulting in a total of 180 (i.e., $3 \mathrm { ~ x ~ } 2 \mathrm { ~ x ~ } 1 0 \mathrm { ~ x ~ } 3$ ) data points.

# B. Second Study ( $\mathbf { \nabla } C .$ cayetanensis, $B .$ . hominis, $E .$ . histolytica, and G. intestinalis)

For the indicated targets, site-to-site reproducibility of the EasyScreen Gastrointestinal Parasite Detection Kit was evaluated at one (1) external clinical site in the US and one (1) in-house site outside the US with three (3) operators at each site for five (5) to seven (7) days.

Panel members $B .$ . hominis, $E$ . histolytica, and $G .$ . intestinalis were tested in triplicate for each of five (5) or six (6) extraction runs resulting in a total of 99 (i.e., $3 \mathrm { ~ x ~ } 5 \mathrm { ~ x ~ } 3 + 3 \mathrm { ~ x ~ } 6 \mathrm { ~ x ~ } 3 )$ data points per panel member, whereas $C .$ . cayetanensis was tested in-house in triplicate for each run resulting in a total of 54 (i.e., $3 \mathrm { ~ x ~ } 6 \mathrm { ~ x ~ } 3$ ) data points at 2X LoD (LP) and a total of 51 (i.e., $3 \mathrm { ~ x ~ } 5 \mathrm { ~ x ~ } 3 + 2 \mathrm { ~ x ~ } 1$ $\mathbf { X } { 3 }$ ) data points at 4X LoD (MP) concentration. The negative samples (TN) were tested in triplicate at the two (2) sites by three (3) operators at each site, resulting in a total of 165 (i.e., $3 \mathrm { ~ x ~ } 1 2 \mathrm { ~ x ~ } 3 \mathrm { ~ + ~ }$ $2 \times 2 \times 3 + 3 \times 5 \times 3$ ) data points.

Due to operational difficulties with $C .$ . cayetanensis organisms (i.e., unreliable sourcing, isolates not culturable, transit degradation, time constraints, etc.), testing at adequate numbers could not be completed at the US site and additional testing to fulfil the replicate number requirements was carried out at the OUS site with two (2) operators, testing the target analyte in twelve (12) replicates per run and two (2) runs per operator, resulting in an additional total of 48 (i.e., $2 \mathrm { ~ x ~ } 2 \mathrm { ~ x ~ } 1 2$ ) data points.

# C. Results of site-to-site reproducibility studies.

Analyte-specific site-to-site qualitative reproducibility testing results from both studies are presented in Tables 1 and 2 below as percent agreement (Correct/Total) of tested valid samples with expected results (i.e., proportion of correctly estimated sample over total samples), along with the $9 5 \%$ confidence intervals. Samples testing invalid were removed from analysis.

Table 1: qualitative reproducibility (First study)   

<table><tr><td rowspan="2">Target and Sample</td><td rowspan="2"></td><td colspan="2">Site 1</td><td colspan="2">Site 2</td><td colspan="2">Site 3</td><td colspan="2">Total</td><td colspan="2">95% CI</td></tr><tr><td>%</td><td>Correct IN</td><td>%</td><td>Correct /N</td><td>%</td><td>Correct IN</td><td>%</td><td>Correct IN</td><td>LL</td><td>UL</td></tr><tr><td rowspan="3">D. fragilis</td><td>TN</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(177/177)</td><td>97.94</td><td>100</td></tr><tr><td>LP</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(90/90)</td><td>95.98</td><td>100</td></tr><tr><td>MP</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(90/90)</td><td>95.98</td><td>100</td></tr><tr><td rowspan="3">E. bieneusi</td><td>TN</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(177/177)</td><td>97.94</td><td>100</td></tr><tr><td>LP</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(29/29)</td><td>100</td><td>(89/89)</td><td>95.94</td><td>100</td></tr><tr><td>MP</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(90/90)</td><td>95.98</td><td>100</td></tr><tr><td rowspan="3">C. parvum</td><td>TN</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(177/177)</td><td>97.94</td><td>100</td></tr><tr><td>LP</td><td>100</td><td>(30/30)</td><td>100</td><td>(28/28)</td><td>100</td><td>(30/30)</td><td>100</td><td>(88/88)</td><td>95.90</td><td>100</td></tr><tr><td>MP</td><td>96.4</td><td>(27/28)</td><td>100</td><td>(29/29)</td><td>100</td><td>(30/30)</td><td>98.9</td><td>(87/88)</td><td>93.83</td><td>99.97</td></tr><tr><td>E.</td><td>TN</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(59/59)</td><td>100</td><td>(177/177)</td><td>97.94</td><td>100</td></tr><tr><td rowspan="2">intestinalis</td><td>LP</td><td>100</td><td>(30/30)</td><td>100</td><td>(30/30)</td><td>100</td><td>(28/28)</td><td>100</td><td>(88/88)</td><td>95.90</td><td>100</td></tr><tr><td>MP</td><td>100</td><td>(30/30)</td><td>100</td><td>(29/29)</td><td>100</td><td>(30/30)</td><td>100</td><td>(89/89)</td><td>95.94</td><td>100</td></tr></table>

Table 2: qualitative reproducibility (Second study)   

<table><tr><td colspan="2">Target and Sample</td><td colspan="2">Site 1</td><td colspan="2">Site 2</td><td colspan="2">Total</td><td colspan="2">95% CI</td></tr><tr><td colspan="2"></td><td>%</td><td>Correct IN</td><td>%</td><td>Correct IN</td><td>%</td><td>Correct IN</td><td>LL</td><td>UL</td></tr><tr><td rowspan="4">B. hominis</td><td>TN</td><td>100</td><td>(120/120)</td><td>100</td><td>(45/45)</td><td>100</td><td>(165/165)</td><td>97.79</td><td>100</td></tr><tr><td>LP</td><td>100</td><td>(54/54)</td><td>100</td><td>(45/45)</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr><tr><td>MP</td><td>100</td><td>(54/54)</td><td>100</td><td>(45/45)</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr><tr><td>TN</td><td>100</td><td>(120/120)</td><td></td><td>N/A</td><td>100</td><td>(120/120)</td><td>96.97</td><td>100</td></tr><tr><td rowspan="2">C. cayetanensis $</td><td>LP</td><td>97.1</td><td>(99/102)</td><td></td><td>N/A</td><td>97.1</td><td>(99/102)</td><td>91.64</td><td>99.39</td></tr><tr><td>MP</td><td>100</td><td>(99/99)</td><td></td><td>N/A</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr><tr><td rowspan="3">G. intestinalis</td><td>TN</td><td>100</td><td>(120/120)</td><td>100</td><td>(45/45)</td><td>100</td><td>(165/165)</td><td>97.79</td><td>100</td></tr><tr><td>LP</td><td>100</td><td>(54/54)</td><td>100</td><td>(45/45)</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr><tr><td>MP</td><td>100</td><td>(54/54)</td><td>100</td><td>(45/45)</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr><tr><td rowspan="3">E. histolytica</td><td>TN</td><td>100</td><td>(120/120)</td><td>100</td><td>(45/45)</td><td>100</td><td>(165/165)</td><td>97.79</td><td>100</td></tr><tr><td>LP</td><td>100</td><td>(54/54)</td><td>100</td><td>(45/45)</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr><tr><td>MP</td><td>100</td><td>(54/54)</td><td>100</td><td>(45/45)</td><td>100</td><td>(99/99)</td><td>96.34</td><td>100</td></tr></table>

$\ S C .$ . cayetanensis results are excluded from the site-to-site reproducibility considerations.

For all targets evaluated at ${ > } 1$ sites (which excludes $C .$ . cayetanensis), the overall site-to-site qualitative reproducibility percent agreement was $100 \%$ for all targets at the Low Positive (2X LoD) analyte level, as well as for all targets except $C .$ . parvum at the Medium Positive (4X LoD) analyte level. Overall detection of $C .$ . parvum at 4X LoD across all testing sites was at $9 8 . 9 \%$ $9 5 \%$ CI: 93.8–99.9). $C .$ . cayetanensis was not assessed for site-to-site reproducibility. Within-site reproducibility for $C .$ . cayetanensis (tested twice at a single site) was $9 7 . 1 \%$ ( $9 5 \%$ CI: 91.64–99.39) for LP samples and $100 \%$ $9 5 \%$ CI: 96.3–100) for MP samples. All True Negative (TN) samples $( 1 0 0 \% )$ were correctly identified in these tests.

Therefore, the analysis of site-to-site qualitative reproducibility of the EasyScreen Gastrointestinal Parasite Detection Kit showed acceptably consistent performance of the EP005 workflow across all test sites.

# 2. Analytical Reactivity/Inclusivity:

Analytical reactivity (inclusivity) of the EasyScreen Gastrointestinal Parasite Detection Kit was investigated by testing eighty-two (82) isolates representing the eight (8) target parasites (with 4– 16 isolates per target, as available) (as shown in Table 3). Isolates were selected to represent various temporal, geographic, and phylogenetic diversity for each analyte. Isolates tested represent clinically relevant subspecies or serotypes and are biased toward more common species and known human pathogens. For clinically relevant organisms, genotypes were established with sequencebased analysis using genotyping assays and review of published literature.

For inclusivity testing, organisms were initially diluted to 1X–3X LoD for the corresponding target in unpreserved negative stool or transport media with preservative ( $C .$ . cayetanensis only) and tested with the EasyScreen Gastrointestinal Parasite Detection Kit, which detected all isolates at all tested concentrations.

Table 3. Isolates for inclusivity testing with EasyScreen Gastrointestinal Parasite Detection Kit.   

<table><tr><td colspan="1" rowspan="1">Target organism Strain/ Designation</td><td colspan="1" rowspan="1">Target organism Strain/ Designation</td><td colspan="1" rowspan="1">Product ID / Clinical ID</td><td colspan="1" rowspan="1">Geographic Origin</td></tr><tr><td colspan="1" rowspan="7">Dientamoebafragilis</td><td colspan="1" rowspan="1">Genotype 1</td><td colspan="1" rowspan="1">Culture "B"</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype 2</td><td colspan="1" rowspan="1">Clinical #598</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype 1</td><td colspan="1" rowspan="1">Clinical #465</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype 1</td><td colspan="1" rowspan="1">Clinical #351</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">inconclusive</td><td colspan="1" rowspan="1">Clinical #409</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype 1</td><td colspan="1" rowspan="1">Clinical #862</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype 1</td><td colspan="1" rowspan="1">Clinical #054</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="10">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #17</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #18</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #1</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #4</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #5</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #7</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #12</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #14</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #15</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical #20</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="6">Cryptosporidiumhominis</td><td colspan="1" rowspan="1">IbA10G2</td><td colspan="1" rowspan="1">Clinical#249</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">IbA10G2</td><td colspan="1" rowspan="1">Clinical#246</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">IbA10G2</td><td colspan="1" rowspan="1">Clinical#257</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">IbA10G2</td><td colspan="1" rowspan="1">Clinical#444</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Id</td><td colspan="1" rowspan="1">Clinical#205</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">IfA14G1</td><td colspan="1" rowspan="1">Clinical#058</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="5">Cryptosporidiumparvum</td><td colspan="1" rowspan="1">IaA18G3R1</td><td colspan="1" rowspan="1">Clinical#223</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">IIaA18G3R1</td><td colspan="1" rowspan="1">Clinical#243</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Ha</td><td colspan="1" rowspan="1">Clinical#204</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Ha</td><td colspan="1" rowspan="1">Clinical#244</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">HaA</td><td colspan="1" rowspan="1">Clinical#234</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">HaA</td><td colspan="1" rowspan="1">Clinical#240</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">HaA</td><td colspan="1" rowspan="1">Clinical#248</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">IaA17G2R1</td><td colspan="1" rowspan="1">P102C</td><td colspan="1" rowspan="1">Iowa, US</td></tr><tr><td colspan="1" rowspan="1">IaA20G3R1</td><td colspan="1" rowspan="1">Clinical#017</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">IaA20G3R1</td><td colspan="1" rowspan="1">Clinical#011</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="16">Blastocystishominis</td><td colspan="1" rowspan="1">DL (ST- 3)</td><td colspan="1" rowspan="1">ATCC 50626 a</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">NTY (ST- 1)</td><td colspan="1" rowspan="1">ATCC 50610 a</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">NandII (ST- 1)</td><td colspan="1" rowspan="1">ATCC 50177</td><td colspan="1" rowspan="1">Maryland, US</td></tr><tr><td colspan="1" rowspan="1">BT1 (ST- 4)</td><td colspan="1" rowspan="1">ATCC 50608</td><td colspan="1" rowspan="1">US</td></tr><tr><td colspan="1" rowspan="1">ST-3</td><td colspan="1" rowspan="1">Clinical#851</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST-3</td><td colspan="1" rowspan="1">Clinical#910</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST-3</td><td colspan="1" rowspan="1">Clinical#690</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST-3</td><td colspan="1" rowspan="1">Clinical#835</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST-4</td><td colspan="1" rowspan="1">Clinical#198</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST-8#</td><td colspan="1" rowspan="1">Clinical#155</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST- 2#</td><td colspan="1" rowspan="1">Clinical#123</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">ST- 1</td><td colspan="1" rowspan="1">Clinical#009</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">ST- 1</td><td colspan="1" rowspan="1">Clinical#014</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">ST- 1</td><td colspan="1" rowspan="1">Clinical#022</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">ST-2</td><td colspan="1" rowspan="1">Clinical#023</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">ST-3</td><td colspan="1" rowspan="1">Clinical#053</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="12">Entamoebahistolytica</td><td colspan="1" rowspan="1">HU-21:AMC</td><td colspan="1" rowspan="1">ATCC 30457</td><td colspan="1" rowspan="1">Arkansas, US</td></tr><tr><td colspan="1" rowspan="1">200:NIH</td><td colspan="1" rowspan="1">ATCC 30458</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">H-458: CDC</td><td colspan="1" rowspan="1">ATCC 30889</td><td colspan="1" rowspan="1">Asia</td></tr><tr><td colspan="1" rowspan="1">HK-9 clone 6</td><td colspan="1" rowspan="1">ATCC 50544</td><td colspan="1" rowspan="1">Korea</td></tr><tr><td colspan="1" rowspan="1">HB-301: NIH CL-1-3</td><td colspan="1" rowspan="1">ATCC 50547</td><td colspan="1" rowspan="1">Burma</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#1</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#3</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#4</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#6</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#7</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#047</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">Clinical#050</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="10">Giardia lamblia(a.k.a. G.intestinalis)</td><td colspan="1" rowspan="1">CM</td><td colspan="1" rowspan="1">ATCC PRA-242</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">G1M</td><td colspan="1" rowspan="1">ATCC PRA-251</td><td colspan="1" rowspan="1">unknown</td></tr><tr><td colspan="1" rowspan="1">Mario</td><td colspan="1" rowspan="1">ATCC PRA-244</td><td colspan="1" rowspan="1">US</td></tr><tr><td colspan="1" rowspan="1">DAN</td><td colspan="1" rowspan="1">ATCC PRA-247</td><td colspan="1" rowspan="1">US</td></tr><tr><td colspan="1" rowspan="1">BE-1</td><td colspan="1" rowspan="1">ATCC PRA-249</td><td colspan="1" rowspan="1">Canada</td></tr><tr><td colspan="1" rowspan="1">WB clone C6 (AI)</td><td colspan="1" rowspan="1">ATCC 50803</td><td colspan="1" rowspan="1">Alaska, US</td></tr><tr><td colspan="1" rowspan="1">Portland-1 (AI)</td><td colspan="1" rowspan="1">ATCC 30888</td><td colspan="1" rowspan="1">Oregon, US</td></tr><tr><td colspan="1" rowspan="1">PR-15</td><td colspan="1" rowspan="1">ATCC PRA-42</td><td colspan="1" rowspan="1">Brazil</td></tr><tr><td colspan="1" rowspan="1">JH (AII)</td><td colspan="1" rowspan="1">ATCC 50584</td><td colspan="1" rowspan="1">Alaska, US</td></tr><tr><td colspan="1" rowspan="1">GS clone H7 (B)</td><td colspan="1" rowspan="1">ATCC 50581</td><td colspan="1" rowspan="1">Virginia, US</td></tr><tr><td colspan="1" rowspan="4">Enterocytozoonbieneusi</td><td colspan="1" rowspan="1">Genotype A</td><td colspan="1" rowspan="1">Clinical #7</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype D</td><td colspan="1" rowspan="1">Clinical #8</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype A</td><td colspan="1" rowspan="1">Clinical #5</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype K</td><td colspan="1" rowspan="1">Clinical #587</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Target organism</td><td colspan="1" rowspan="1">Strain/ Designation</td><td colspan="1" rowspan="1">Product ID / Clinical ID</td><td colspan="1" rowspan="1">Geographic Origin</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Genotype D</td><td colspan="1" rowspan="1">Clinical #10</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype D</td><td colspan="1" rowspan="1">Clinical #11</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="1">Genotype K</td><td colspan="1" rowspan="1">Clinical #12</td><td colspan="1" rowspan="1">Sydney, AUS</td></tr><tr><td colspan="1" rowspan="4">Encephalitozoonintestinalis</td><td colspan="1" rowspan="1">Alveolar isolate</td><td colspan="1" rowspan="1">ATCC 50506</td><td colspan="1" rowspan="1">NY, US</td></tr><tr><td colspan="1" rowspan="1">CDC: V297</td><td colspan="1" rowspan="1">ATCC 50651</td><td colspan="1" rowspan="1">CA, US</td></tr><tr><td colspan="1" rowspan="1">CDC: V307</td><td colspan="1" rowspan="1">ATCC 50603</td><td colspan="1" rowspan="1">Georgia, US</td></tr><tr><td colspan="1" rowspan="1">Nasal isolate</td><td colspan="1" rowspan="1">ATCC 50507</td><td colspan="1" rowspan="1">NY, US</td></tr></table>

# 3. Analytical Specificity/Cross reactivity:

# A. Whole organism/genome wet testing:

Analytical Specificity or cross-reactivity of the EasyScreen Gastrointestinal Parasite Detection Kit was established by testing a total of 94 different organisms (including viruses, bacteria, yeasts, and protozoa), as well as some commonly used microbiological media, in the EasyScreen Gastrointestinal Parasite Detection assay. In the whole organism/genome wet testing study, isolates from cultures, where available, or purified nucleic acids were diluted in negative stool matrix to a range of $1 0 ^ { 6 } – 1 0 ^ { 9 }$ CFU or copies $/ \mathrm { m L }$ , with the exception of Entamoeba dispar which could not be procured at a sufficiently high concentration, or in a culturable form, to achieve the desired concentration and was tested at $1 . 5 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ organisms/mL. The potential cross reactants were tested in triplicate, with acceptance criteria for no cross-reactivity set at no positive replicate for a given target in Panels A–C of the EasyScreen Gastrointestinal Parasite Detection assay.

The study results showed no cross-reactivity the viral, fungal, bacterial, and protozoal microorganisms tested at the concentrations indicated in Table 4. None of the microbiological media used for organism propagation were cross-reactive either. However, the whole organism/genome wet testing study demonstrated three (3) cross-reacting protozoa—all of which were predicted from in silico alignments with their congeneric protozoa in the Panels A and C of the EasyScreen Gastrointestinal Parasite Detection assay:

i. Cryptosporidium muris (Strain Waterborne P104, tested at $6 . 2 5 \times 1 0 ^ { 5 }$ organisms/mL), positive signal in Panel A.   
ii. Encephalitozoon cuniculi (ATCC 50789, tested at $1 0 ^ { 6 }$ organisms $/ \mathrm { m L }$ ), positive signal in Panel C, and   
iii. Encephalitozoon hellem (ATCC 50604, tested at $5 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ organisms $ { ' } \mathrm { m L }$ ), positive signal in Panel C.

Table 4. Organisms with NO cross-reactivity observed in the EasyScreen Gastrointestinal Parasite Detection Kit Analytical Specificity Study.   

<table><tr><td>Organism; strain</td><td>ID details</td></tr><tr><td>Viruses (tested at 106-10° copies/mL)</td><td></td></tr><tr><td>Coxsackie virus B5</td><td>Vircell MBC062-R</td></tr><tr><td>Cytomegalovirus</td><td>Vircell MBC016</td></tr><tr><td>Sapovirus</td><td>ATCC VR3237SD</td></tr><tr><td>Adenovirus Type 41</td><td>ATCC VR-930DQ</td></tr><tr><td>Astrovirus</td><td>ATCC VR-3238SD</td></tr><tr><td>Bocavirus</td><td>ATCC VR-3251SD</td></tr><tr><td>Enterovirus 71</td><td>ATCC VR-1775DQ</td></tr><tr><td>Norovirus G1</td><td>ATCC VR-3234SD</td></tr><tr><td>Norovirus G2</td><td>ATCC VR-3235SD</td></tr><tr><td>Rotavirus A</td><td>ATCC VR-2018DQ</td></tr><tr><td>Adenovirus Type 5; Adenoid 75</td><td>ATCC VR-5D</td></tr><tr><td>Yeasts (tested at 0.5-1 x 107 CFU/mL)</td><td></td></tr><tr><td>Saccharomyces cerevisiae</td><td>ATCC MYA-796</td></tr><tr><td>Candida albicans</td><td>ATCC MYA-2876D-5</td></tr><tr><td>Bacterial strains (tested at 106-10° CFU/mL) Abiotrophia defectiva; Strain SC10</td><td></td></tr><tr><td>Acinetobacter baumannii</td><td>ATCC 49176</td></tr><tr><td>Actinomyces naeslundii</td><td>ATCC 19606D-5</td></tr><tr><td>Aeromonas hydrophila; CDC 359-60</td><td>ATCC 12104D5</td></tr><tr><td>Akkermansia muciniphila; Strain Muc</td><td>ATCC 7965D</td></tr><tr><td>Alcaligenes faecalis; subsp. faecalis</td><td>ATCC BAA-835D-5 ATCC 8750D-5</td></tr><tr><td>Anaerococcus tetradius</td><td>ATCC 35098</td></tr><tr><td>Arcobacter butzleri</td><td>ATCC 49616</td></tr><tr><td>Atopobium vaginae</td><td>ATCC BAA55</td></tr><tr><td>Bacillus cereus; Strain 971</td><td>ATCC 14579D</td></tr><tr><td>Bacteroides fragilis</td><td>ATCC 25285D-5</td></tr><tr><td>Bifidobacterium adolescentis</td><td>ATCC 15703D-5</td></tr><tr><td>Bifidobacterium bifidum</td><td>ATCC 29521</td></tr><tr><td>Campylobacter hominis</td><td>ATCC BAA-381D-5</td></tr><tr><td>Campylobacter jejuni</td><td>ATCC 33560D-5</td></tr><tr><td>Campylobacter lari</td><td>ATCC BAA-1060D-5</td></tr><tr><td>Capnocytophaga gingivalis</td><td>ATCC 33624D-5</td></tr><tr><td>Cedecea davisae</td><td>ATCC 33431</td></tr><tr><td>Chlamydia trachomatis; Serovar D</td><td>ATCC VR-885D</td></tr><tr><td>Chryseobacterium gleum</td><td>ATCC 35910</td></tr><tr><td>Citrobacter freundii</td><td>ATCC 8090D</td></tr><tr><td>Clostridioides difficile</td><td>ATCC BAA-1870DQ</td></tr><tr><td>Clostridium perfringens</td><td>ATCC 13124DQ</td></tr><tr><td>Corynebacterium glutamicum; Strain 534</td><td>ATCC 13032D-5</td></tr><tr><td>Cronobacter sakazakii</td><td>ATCC BAA-894D-5</td></tr><tr><td>Desulfovibrio piger; Strain VPI C3-23</td><td>ATCC 29098</td></tr><tr><td>Edwardsiella tarda; Strain CDC 1483-59</td><td>ATCC 15947</td></tr><tr><td>Eggerthella lenta; 1899B</td><td></td></tr><tr><td>Enterococcus faecalis</td><td>ATCC 25559D-5 ATCC 700802DQ</td></tr><tr><td>Enterococcus faecium</td><td>ATCC BAA-472D-5</td></tr><tr><td>Escherichia coli; CFT073</td><td>ATCC 700298D-5</td></tr><tr><td>Escherichia coli; Strain CDC EDL 1284</td><td>ATCC 43893</td></tr><tr><td>Eubacterium rectale</td><td>ATCC 33656</td></tr><tr><td>Faecalibacterium prausnitzii; Strain: VPI C13-51</td><td>ATCC 27768</td></tr><tr><td>Fusobacterium varium</td><td>ATCC 27725</td></tr><tr><td>Gardnerella vaginalis</td><td>ATCC 49145D-5</td></tr><tr><td>Gemella morbillorum</td><td>ATCC 27824</td></tr><tr><td>Hafnia alvei; HER 1272</td><td>ATCC 51873D-5</td></tr><tr><td>Helicobacter pylori; J99</td><td>ATCC 700824D-5</td></tr><tr><td>Klebsiella oxytoca</td><td>ATCC 700324D</td></tr><tr><td>Lactobacillus acidophilus</td><td>ATCC 4357D-5</td></tr><tr><td>Lactococcus lactis; subsp. lactis</td><td>ATCC 19435D-5</td></tr><tr><td>Leminorella grimontii</td><td>ATCC 33999</td></tr><tr><td>Listeria monocytogenes</td><td>ATCC 19115D-5</td></tr><tr><td>Mycobacterium abscessus</td><td>ATCC 19977D-5</td></tr><tr><td>Mycobacterium avium; Strain K-10</td><td>ATCC BAA-968D-5</td></tr><tr><td>Mycobacterium tuberculosis</td><td>ATCC 25177D-5</td></tr><tr><td>Mycoplasma hominis Strain PG21</td><td>ATCC 23114D</td></tr><tr><td>Mycoplasma salivarium</td><td>ATCC 23064D</td></tr><tr><td>Neisseria flava</td><td>ATCC 14221D</td></tr><tr><td>Peptoniphilus asaccharolyticus</td><td>ATCC 14963</td></tr><tr><td>Peptostreptococcus anaerobius</td><td>ATCC 49031D-5</td></tr><tr><td>Plesiomonas shigelloides</td><td>ATCC 51903D</td></tr><tr><td>Porphyromonas asaccharolytica</td><td>ATCC 25260</td></tr><tr><td>Porphyromonas levii</td><td>ATCC 29147</td></tr><tr><td>Prevotella melaninogenica; Strain VPI 2381</td><td>ATCC 25845D-5</td></tr><tr><td>Proteus mirabilis</td><td>ATCC 12453DQ</td></tr><tr><td>Proteus vulgaris</td><td>ATCC 29905DQ</td></tr><tr><td>Providencia stuartii</td><td>ATCC 33672D</td></tr><tr><td>Pseudomonas aeruginosa</td><td>ATCC 47085DQ</td></tr><tr><td>Ruminococcus bromii; Strain VPI 6883</td><td>ATCC 27255</td></tr><tr><td>Salmonella enterica; serovar Typhimurium</td><td>ATCC 700720DQ</td></tr><tr><td>Serratia marcescens; CDC 3100-71</td><td>ATCC 27137D-5</td></tr><tr><td>Shigella flexneri type 2 Strain 24570</td><td>ATCC 29903D-5</td></tr><tr><td>Shigella sonnei</td><td>ATCC 29930</td></tr><tr><td>Staphylococcus aureus; subsp. aureus</td><td>ATCC 25923D-5</td></tr><tr><td>Staphylococcus haemolyticus</td><td>ATCC 29970D-5</td></tr><tr><td>Stenotrophomonas maltophila</td><td>ATCC 13637D-5</td></tr><tr><td>Streptococcus mitis NS 51</td><td>ATCC 49456D-5</td></tr><tr><td>Streptococcus pyogenes</td><td>ATCC 12344D-5</td></tr><tr><td>Streptococcus sanguinis</td><td>ATCC 10556D-5</td></tr><tr><td>Streptococcus thermophilus</td><td>ATCC BAA-250D-5</td></tr><tr><td>Trichomonas vaginalis</td><td>ATCC PRA-98D</td></tr><tr><td>Veillonella parvula</td><td>ATCC 10790D-5</td></tr><tr><td>Vibrio cholerae</td><td>Vircell MBC118-R</td></tr><tr><td>Vibrio parahaemolyticus</td><td>ATCC 17802D-5</td></tr><tr><td>Yersinia pseudotuberculosis</td><td>ATCC 6902D-5</td></tr><tr><td colspan="2">Protozoa (tested at 1.5 x 10³ organisms/mL)</td></tr><tr><td colspan="2">Entamoeba dispar SAW 760* ATCC PRA-260</td></tr></table>

\* Entamoeba dispar was not available for shipment for testing at a higher concentration or culturable form. $E .$ . dispar was thus subsequently investigated as a synthetic RNA target at a higher concentration.

# B. In silico analysis:

Sequence alignments were performed to identify organisms of high similarity using the BLAST tool to interrogate sequences in GenBank. Sequences with identity of $90 \mathrm { - } 1 0 0 \%$ to the target sequence were analyzed in silico for cross reactivity potential where the organisms were not available for wet testing, with a focus on sequence identity under the EP005 primer and probe regions. When assessing the likelihood of cross reactivity, the number of mismatches and location of the mismatches were considered. The location of a mismatch was defined as “significant” if located within the first 5 nucleotides at the $_ 3 \cdot$ end of the primer. (“Significant location” is not applicable to location of mismatches within probes.)

Organisms were assigned into three (3) categories indicating their potential to cross-react based on the GenBank sequence match to the primers and probes (in the context of the “3base” sequences), namely,

(i) High level of sequence match with up to 5 mismatches with non-significant locations (cross reactive potential: High).   
(ii) Moderate level of sequence match with 4-6 mismatches that have non-significant locations (cross reactive potential: Moderate).   
(iii) Low level of sequence match with 7 or more mismatches including in significant locations (cross reactive potential: Low).

For potentially cross-reactive organisms, literature searches were also conducted to identify whether any such organisms were known to infect humans. The results of in silico analyses are summarized in Table 5 (potential cross reactants with no clinical relevance) and 6 (potential cross reactants with clinical relevance as assessed from published literature) below.

Table 5. EasyScreen Gastrointestinal Parasite Detection Kit target in silico analysis: potential cross reactants of no clinical relevance (known animal pathogens).   

<table><tr><td colspan="1" rowspan="1">Organism /representativeGenBank accession</td><td colspan="1" rowspan="1">Cognate EP005target</td><td colspan="1" rowspan="1">Forwardprimermatch *</td><td colspan="1" rowspan="1">Reverseprimermatch *</td><td colspan="1" rowspan="1">Probe match *</td></tr><tr><td colspan="1" rowspan="1">B. cycluri AY590116</td><td colspan="1" rowspan="4">Blastocystishominis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">B. lapemi AY266471</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.3% (1 mismatch, 2deletions)</td></tr><tr><td colspan="1" rowspan="1">B. pythoni AY266472</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.3% (2 mismatches,1 deletion)</td></tr><tr><td colspan="1" rowspan="1">B. ratti AY590114</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">C. cercopitheciAF111185</td><td colspan="1" rowspan="3">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">95.7% (1mismatch)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">C. colobi AF111186</td><td colspan="1" rowspan="1">95.7% (1mismatch)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">C. papionis AF111187</td><td colspan="1" rowspan="1">91.3% (2mis-matches)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">91.3% (2 mismatches)</td></tr><tr><td colspan="1" rowspan="1">C. bovis EF514234</td><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">96.7% (1 insertion)</td></tr><tr><td colspan="1" rowspan="1">C. wrairi U11440 #</td><td colspan="1" rowspan="1">spp.</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Ecytonucleosporahepatopenaei OR168078</td><td colspan="1" rowspan="3">Enterocytozoonbieneusi</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">96% (1 mismatch)</td></tr><tr><td colspan="1" rowspan="1">Enterospora nucleophilaKF135641</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">96% (1 mismatch)</td></tr><tr><td colspan="1" rowspan="1">Obruspora papernaeHG005137</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">96% (1 mismatch)</td></tr><tr><td colspan="1" rowspan="1">E. nuttalli LC042219 #</td><td colspan="1" rowspan="1">Entamoebahistolytica</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">G. microti AF006676</td><td colspan="1" rowspan="1">Giardiaintestinalis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Eimeria hermaniKJ000078</td><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">78.3% (3 mismatches,2 deletions</td></tr><tr><td colspan="1" rowspan="1">C. baileyi KT151546 #</td><td colspan="1" rowspan="1">Cryptosporidiumspp.</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90% (2 mismatches, 1insertion)</td></tr><tr><td colspan="1" rowspan="1">Histomonas meleagridisAJ920323</td><td colspan="1" rowspan="3">Dientamoebafragilis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.5 % (5 mismatches)</td></tr><tr><td colspan="1" rowspan="1">Pseudotrichomonaskeilini HM581663</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">85.2% (4 mismatches)</td></tr><tr><td colspan="1" rowspan="1">Trichomitus sp. 1 (exGeochelone sulcata)JX515400</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.9% (2 deletions, 1mismatch)</td></tr><tr><td colspan="1" rowspan="1">Nucleospora salmonisHQ418210</td><td colspan="1" rowspan="1">Enterocytozoonbieneusi</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88% (3 mismatches)</td></tr><tr><td colspan="1" rowspan="1">E. lacerate AF067144</td><td colspan="1" rowspan="1">Encephalitozoon</td><td colspan="1" rowspan="1">88% (3mis-matches)</td><td colspan="1" rowspan="1">92% (2mis-matches)</td><td colspan="1" rowspan="1">95.7% (1 mismatch)</td></tr><tr><td colspan="1" rowspan="1">E. pogonae KR998311</td><td colspan="1" rowspan="1">intestinalis</td><td colspan="1" rowspan="1">88% (3mis-matches)</td><td colspan="1" rowspan="1">96% (1mismatch)</td><td colspan="1" rowspan="1">95.7% (1 mismatch)</td></tr><tr><td colspan="1" rowspan="1">G. ardeae Z17210</td><td colspan="1" rowspan="1">Giardiaintestinalis</td><td colspan="1" rowspan="1">95.7% (1mismatch)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.8% (4 mismatches)</td></tr><tr><td colspan="1" rowspan="1">Trochochilodon flavusJN867018</td><td colspan="1" rowspan="1">Blastocystishominis</td><td colspan="1" rowspan="1">96% (1mismatch)</td><td colspan="1" rowspan="1">96% (1mismatch)</td><td colspan="1" rowspan="1">80.6% (5 mismatches,1 deletion)</td></tr><tr><td colspan="1" rowspan="1">Colpodella tetrahymenaeAF330214</td><td colspan="1" rowspan="2">Cryptosporidium</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">96% (1mismatch)</td><td colspan="1" rowspan="1">70% (4 mismatches, 4insertions, 1 deletion)</td></tr><tr><td colspan="1" rowspan="1">C. fragile EU162754</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">76.7% (5 mismatches,1 insertion, 1 deletion)</td></tr><tr><td colspan="1" rowspan="1">C. serpentis AF093499</td><td colspan="1" rowspan="2">spp.</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.3% (2 mismatches,3 insertions)</td></tr><tr><td colspan="1" rowspan="1">C. struthionis AJ697751</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">66.7% (7 mismatches,3 insertions)</td></tr><tr><td colspan="1" rowspan="1">Giardia muris X65063</td><td colspan="1" rowspan="1">Giardia.intestinalis</td><td colspan="1" rowspan="1">91.3% (2mis-matches</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">45.5% (7 mismatches,5 insertions)</td></tr></table>

\* For the purpose of calculating $\%$ match, any nucleotide inserted or deleted underneath a primer or probe (relative to the GenBank entry for the species in question) was considered one (1) mismatch. Thus, a primer or probe with four (4) mismatches, four (4) insertions and one (1) deletion would be considered to have nine (9) mismatches. # $< 3$ cases of human infection reported worldwide.

# Table 6. EasyScreen Gastrointestinal Parasite Detection Kit target in silico analysis: potential cross reactants of clinical relevance (known human pathogens).

<table><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Organism /representativeGenBankaccession</td><td rowspan=1 colspan=1>Cognate EP005target</td><td rowspan=1 colspan=1>Forwardprimermatch *</td><td rowspan=1 colspan=1>Reverseprimermatch *</td><td rowspan=1 colspan=1>Probe match *</td><td rowspan=1 colspan=1>Predictedcrossreactivepotential</td></tr><tr><td rowspan=1 colspan=1>C. canisAB210854</td><td rowspan=2 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.3% (2 mis-matches</td><td rowspan=4 colspan=1>High</td></tr><tr><td rowspan=1 colspan=1>C. felisAF112575</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.3% (2 mis-matches)</td></tr><tr><td rowspan=1 colspan=1>C. meleagridisEF179381</td><td rowspan=2 colspan=1>spp.</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>C. tyzzeriOQ826430</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>C. muris L19069</td><td rowspan=1 colspan=1>Cryptosporidiumspp.</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80% (2 mis-matches, 4 inser-tions</td><td rowspan=3 colspan=1>Moderate</td></tr><tr><td rowspan=1 colspan=1>ChilomastixmesniliKC960586</td><td rowspan=1 colspan=1>Giardiaintestinalis</td><td rowspan=1 colspan=1>95.7% (1mismatch)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>81.8% (4 mis-matches)</td></tr><tr><td rowspan=1 colspan=1>E. disparKP722600</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>87.5% (2mismatches,1 insertion)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Isospora belliTDQ060661</td><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>82.6% (4mismatches)</td><td rowspan=1 colspan=1>72% (7mis-matches)</td><td rowspan=1 colspan=1>56.5% (3 mis-matches, 3 inser-tions, 4 deletions)</td><td rowspan=5 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>C. andersoniAB513869</td><td rowspan=1 colspan=1>Cryptosporidiumspp.</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80% (3 mis-matches, 3 inser-tions)</td></tr><tr><td rowspan=1 colspan=1>E. bangladeshiKR025412</td><td rowspan=3 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>62.5% (8mismatches,1 insertion)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>E. ecuadoriensisDQ286373</td><td rowspan=1 colspan=1>62.5% (9mismatches)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>E. moshkovskiiMN536500 $</td><td rowspan=1 colspan=1>70.8% (6mismatches,1 insertion)</td><td rowspan=1 colspan=1>81.5% (5mis-matches)</td><td rowspan=1 colspan=1>100%</td></tr></table>

\* For the purpose of calculating $\%$ match, any nucleotide inserted or deleted underneath a primer or probe (relative to the GenBank entry for the species in question) was considered one (1) mismatch. Thus, a primer or probe with four (4) mismatches, four (4) insertions and one (1) deletion would be considered to have nine (9) mismatches. $\ S E .$ . dispar and $E$ . moshkovskii infect humans but are non-pathogenic.

In silico analysis identified several targets for further investigation as potential cross- reacting organisms. However, only cases with three (3) or more reported human infections world-wide were further analyzed and investigated using synthetic RNA targets as described below.

# C. Confirmatory Wet testing of synthetic RNA targets from clinically relevant protozoa.

Table 6 lists seven (7) potentially cross-reacting organisms (in silico predicted cross-reactivity with the EasyScreen Gastrointestinal Parasite Detection Kit: High or Moderate) that are also clinically relevant (as assessed from literature evidence for human infection). However, these organisms were generally not available in either a culturable form or as clinical samples. For their further testing, synthetic double-stranded DNA targets—incorporating the T7 promoter at the $5 '$ end of the sequence—were commercially designed (‘gBlock’ from IDT) from the Accession numbers in Table 6. Alternatively, where available, whole genome sequences (approximately 800–1000bp in length) were commercially obtained. These sequences were used to make synthetic in vitro transcribed (IVT) RNA. After quantification, the IVT RNA was extracted using the SP008B kit at between $1 0 ^ { 8 }$ and $1 0 ^ { 9 }$ copies/mL (see below) and tested with the EasyScreen Gastrointestinal Parasite Detection Kit. Results are presented in Table 7. With the exception of the Chilomastix mesnili. and Entamoeba dispar, all in silico targets of clinical relevance that were predicted to cross-react in the cognate EasyScreen assay tested positive in that assay.

Table 7. Wet testing of synthetic IVT RNA targets $\boldsymbol { \left( P = \right. }$ positive; $N = n e g a t i v e ) .$ :   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=2>EP005 target towhich similarityexists</td><td rowspan=2 colspan=1>gBlock Lotnumber</td><td rowspan=2 colspan=1>Cop-ies/ mL</td><td rowspan=1 colspan=3>Panel</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>C</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium meleagridis</td><td rowspan=1 colspan=2>Cryptosporidium spp.</td><td rowspan=1 colspan=1>109271385</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium tyzzeri</td><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>109271386</td><td rowspan=1 colspan=2>108</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium canis</td><td rowspan=1 colspan=1>109271381</td><td rowspan=1 colspan=1>8 x 108</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium felis</td><td rowspan=1 colspan=1>109271383</td><td rowspan=1 colspan=1>8 x 108</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Entamoeba dispar ^</td><td rowspan=1 colspan=2>Entamoeba histolytica</td><td rowspan=1 colspan=1>103550577</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Chilomastix mesnili</td><td rowspan=1 colspan=2>Giardia intestinalis</td><td rowspan=1 colspan=1>109609223</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td></tr></table>

^ Wet-testing of $E$ . dispar whole organism was negative at low copy number (Table 4), but it was selected for wet-testing with synthetic targets at a high copy number ( $1 0 ^ { 9 }$ copies/mL).

Therefore, wet testing of whole organisms or whole genomes or of synthetic RNA informed by in silico analysis identified the following seven (7) human pathogenic protozoa that are congeneric to two (2) target protozoal parasites in the EasyScreen Gastrointestinal Parasite Detection Kit and cross-react with the EP005 assays: Cryptosporidium meleagridis, C. tyzzeri, C. canis, C. felis, C. muris, Encephalitozoon cuniculi, and E. hellem.

# 4. Analytical Specificity/Interference:

Twenty-three (23) biological and chemical substances that may be present in clinical stool specimens were evaluated for potential interference with the EasyScreen Gastrointestinal Parasite Detection Kit with three (3) target analytes—one (1) representative chosen per panel, namely, Dientamoeba fragilis (Panel A), Giardia intestinalis / lamblia (Panel B), and Enterocytozoon bieneusi (Panel C)—tested at 2X LoD. Interferent concentrations chosen for evaluation were determined from the recommendations of FDA-recognized Consensus Standard CLSI EP37 along with a review of analytical studies from prior FDA-cleared GI panel devices. In ten (10) replicates tested, Interference (I) was defined as ${ < } 1 0 0 \%$ target positivity $( { \leq } 9 / 1 0 )$ achieved OR a change of ${ > } 1 5 \%$ in the average Ct values in test (i.e., with interferent) samples relative to baseline (i.e., no interferent). Of all substances evaluated (as shown in Table 8), two (2) of the tested substances (i.e., Whole Blood and Mucin) exhibited potential interference at different concentrations tested with the EasyScreen Gastrointestinal Parasite Detection Kit assay.

Table 8. Endogenous and exogenous substances tested with the EasyScreen Gastrointestinal Parasite Detection Kit as potential interferents. $\left( I = \right.$ Interference noted.)

<table><tr><td rowspan=1 colspan=1>Potential Interferent / Active agent (use)</td><td rowspan=1 colspan=1>Interferentconc.</td><td rowspan=1 colspan=1>D.fragilis(panel A)</td><td rowspan=1 colspan=1>G.intestinalis(panel B)</td><td rowspan=1 colspan=1>E.bieneusi(panel C)</td></tr><tr><td rowspan=1 colspan=1>Barium sulfate</td><td rowspan=1 colspan=1>10 % w/v</td><td rowspan=21 colspan=3>No interference observed in anyPanel</td></tr><tr><td rowspan=1 colspan=1>Calcium carbonate</td><td rowspan=1 colspan=1>2.5 % w/v</td></tr><tr><td rowspan=1 colspan=1>Canesten (Clotrimazole 200 mg; 1% v/v)(antifungal)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Diaper rash cream (Zinc oxide)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Doxycycline (antibiotic)</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Dulcolax (Bisacodyl 5mg) (laxative)</td><td rowspan=1 colspan=1>20 % w/v</td></tr><tr><td rowspan=1 colspan=1>Fatty acids (Stearic acid, Palmitic acid)</td><td rowspan=1 colspan=1>5 % w/v</td></tr><tr><td rowspan=1 colspan=1>Fecal fat (Triglycerides, Cholesterol)</td><td rowspan=1 colspan=1>5 % w/v</td></tr><tr><td rowspan=1 colspan=1>Gaviscon 10 mL (Sodium Alginate 500mg,Sodium Bicarbonate 213mg, CalciumCarbonate 325mg) (antacid)</td><td rowspan=1 colspan=1>10 % w/v</td></tr><tr><td rowspan=1 colspan=1>Hydrozole (Hydrocortisone 1% v/v,Clotrimazole 1% v/v) (antifungal)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Imodium (Loperamide hydrochloride) (Anti-diarrheal)</td><td rowspan=1 colspan=1>10 % w/v</td></tr><tr><td rowspan=1 colspan=1>KY gel (Chlorhexidine gluconate; Methylbenzoate) (lubricant)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Metronidazole (antibiotic)</td><td rowspan=1 colspan=1>10 % w/v</td></tr><tr><td rowspan=1 colspan=1>Mineral oil</td><td rowspan=1 colspan=1>50 % w/v</td></tr><tr><td rowspan=1 colspan=1>Naproxen sodium 275 mg (pain reliever)</td><td rowspan=1 colspan=1>10 % w/v</td></tr><tr><td rowspan=1 colspan=1>Nystatin suspension (antifungal)</td><td rowspan=1 colspan=1>25 % w/v</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol Max Strength (Bismuthsubsalicylate) (Anti-diarrheal)</td><td rowspan=1 colspan=1>10 % w/v</td></tr><tr><td rowspan=1 colspan=1>Rectinol (Zinc oxide 200 mg, Cinchocainehydrochloride 5mg) (hemorrhoid cream)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Vagisil (Benzocaine 50mg/g, Resorcinol20mg/g) (feminine itching cream medication)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Vaseline (white petroleum jelly)</td><td rowspan=1 colspan=1>30 % w/v</td></tr><tr><td rowspan=1 colspan=1>Wet Ones (Benzalkonium Chloride, Ethanol)(Antibacterial Hand Wipes)</td><td rowspan=1 colspan=1>30 % v/v</td></tr><tr><td rowspan=3 colspan=1>Purified Mucin protein</td><td rowspan=1 colspan=1>3 mg/mL</td><td rowspan=1 colspan=1>No inter-ference</td><td rowspan=1 colspan=1>No inter-ference</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>1.5 mg/mL</td><td rowspan=1 colspan=1>Nottested</td><td rowspan=1 colspan=1>Not tested</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>0.75mg/mL</td><td rowspan=1 colspan=1>Nottested</td><td rowspan=1 colspan=1>Not tested</td><td rowspan=1 colspan=1>No inter-ference</td></tr><tr><td rowspan=3 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>5 % v/v2.50%</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>No inter-ferenceNot tested</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>1.25%</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>Not tested</td><td rowspan=1 colspan=1>No inter-ference</td></tr><tr><td rowspan=1 colspan=1>0.63%</td><td rowspan=1 colspan=1>No inter-ference</td><td rowspan=1 colspan=1>Not tested</td><td rowspan=1 colspan=1>Nottested</td></tr></table>

Whole blood and Mucin demonstrated potential interference with the EasyScreen Gastrointestinal Parasite Detection Kit at concentrations greater than $0 . 6 3 \%$ and $0 . 7 5 \mathrm { m g / m L }$ , respectively. This is indicated as a limitation in labeling.

# 5. Analytical Specificity/Mixed infections:

# A. Microbial Interference:

The microbial interference study was designed to evaluate the ability of the EasyScreen Gastrointestinal Parasite Detection Kit to detect low positive target analytes in presence of high concentrations of extraneous non-protozoal micro-organisms that may be present in high concentrations in clinical stool specimens. All target analytes in Panel $\mathrm { A } { - } \mathrm { C }$ of the EasyScreen Gastrointestinal Parasite Detection Kit were tested at 2X LoD in a negative stool matrix in presence or absence of the following eight (8) selected bacterial or fungal isolates at $1 0 ^ { 6 }$ CFU/mL: Pseudomonas aeruginosa (ATCC 47085DQ), Enterococcus faecalis (ATCC 700802DQ), Candida albicans (ATCC MYA-2876D-5), Bacteroides fragilis (ATCC 25285D-5), Clostridioides perfringens (ATCC 13124DQ), Klebsiella pneumoniae (ATCC 13883DQ), nonpathogenic Escherichia coli (ATCC 25922DQ), and Saccharomyces cerevisiae (ATCC MYA796).

In the ten (10) target replicates tested, relative to baseline (i.e., with no interferent), microbial interference (I) was defined as ${ < } 1 0 0 \%$ target positivity $( { \leq } 9 / 1 0 )$ achieved OR a change of ${ > } 1 5 \%$ in the average Ct values in test (i.e., with interferent) samples; moderate microbial interference (MI) was defined as $100 \%$ (10/10) target positivity but showing $1 1 - 1 5 \%$ change in test $\mathrm { C t }$ values; whereas no reportable interference (NI) was defined at $100 \%$ (10/10) target positivity with test Ct changes at or below $10 \%$ .

In the presence of potential microbial interferents in the EasyScreen Gastrointestinal Parasite Detection assay, all targets showed ${ < } 1 0 \%$ change in average Ct values relative to baseline, with $100 \%$ (10/10) target positivity amongst replicates. Therefore, no microbial interference was observed when testing target organisms with the EasyScreen Gastrointestinal Parasite Detection Kit.

# B. Competitive Interference:

The competitive interference study was designed to evaluate the ability of the EasyScreen Gastrointestinal Parasite Detection Kit to detect low positive target analytes in presence of high concentrations of potential competitor co-target protozoan analyte(s) that may be present in high concentrations in clinical stool specimens. All target analytes in Panel A–C of the EasyScreen Gastrointestinal Parasite Detection Kit were tested at 2X LoD in a negative stool matrix in presence or absence of other targets as shown in Table 8. Targets used as potential competitors were tested at $1 0 ^ { 5 } ~ \mathrm { { o r g / m L } }$ (whole organisms) except for $C .$ . cayetanensis, for which a synthetic in vitro transcribed RNA target was used at $1 0 ^ { 8 }$ copies/mL to serve as a high concentration competitor.

As before, in the ten (10) target replicates tested, relative to baseline (i.e., with no competitor), competitive interference (I) was defined as ${ < } 1 0 0 \%$ target positivity $( { \leq } 9 / 1 0 )$ achieved OR a change of ${ > } 1 5 \%$ in the average Ct values in test (i.e., with competitor) samples; moderate competitive interference (MI) was defined as $100 \%$ (10/10) target positivity but showing $1 1 - 1 5 \%$ change in test $\mathrm { C t }$ values; whereas no reportable interference $( \mathbf { N I } )$ was defined at $100 \%$ (10/10) target positivity with test Ct changes at or below $10 \%$ .

In the course of these studies, moderate competitive interference was observed when testing low positive (2X LoD) $D .$ . fragilis and $G .$ . intestinalis targets with $C .$ . cayetanensis and $E$ . histolytica competitors, respectively, with average $\mathrm { C t }$ change of $12 \%$ and $1 1 \%$ (relative to baseline). Further, competitive interference (at $20 \%$ positivity) was seen with low positive $D$ . fragilis and $B .$ . hominis targets with $C .$ . parvum and $E$ . histolytica competitors, respectively. These were resolved upon reducing the competitor concentrations to $5 \mathrm { ~ x ~ } 1 0 ^ { \mathrm { { - } } } \mathrm { { \ o r g / m L } }$ and $1 0 ^ { 4 } \mathrm { o r g / m L }$ , respectively. However, a limitation is included in labeling to recommend testing with other methods upon a negative assay if $D$ . fragilis or $B$ . hominis is clinically suspected. All other low positive targets were successfully detected by the EasyScreen Gastrointestinal Parasite Detection Kit when combined with other competing targets at a high concentration, as shown in Table 9 below.

Table 9: Target analytes and potential microbial competitors tested in the EasyScreen Gastrointestinal Parasite Detection Kit assay.   

<table><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Target analyteat 2X LoD</td><td rowspan=1 colspan=1>Competitoranalyte</td><td rowspan=1 colspan=1>Competitor concentra-tion (org/mL, *except C.cayetanensis, copies/mL)</td><td rowspan=1 colspan=1>CompetitiveInterference observed</td></tr><tr><td rowspan=7 colspan=1>A</td><td rowspan=1 colspan=1>D. fragilis</td><td rowspan=1 colspan=1>C. cayetanensis</td><td rowspan=1 colspan=1>108 *</td><td rowspan=1 colspan=1>Moderate</td></tr><tr><td rowspan=1 colspan=1>D. fragilis</td><td rowspan=1 colspan=1>C. parvum</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>Interference</td></tr><tr><td rowspan=1 colspan=1>D. fragilis</td><td rowspan=1 colspan=1>C. parvum</td><td rowspan=1 colspan=1>5 x 104 s</td><td rowspan=5 colspan=1>No reportableinterference</td></tr><tr><td rowspan=1 colspan=1>C. cayetanensis</td><td rowspan=1 colspan=1>D. fragilis</td><td rowspan=1 colspan=1>105}$</td></tr><tr><td rowspan=1 colspan=1>C. cayetanensis</td><td rowspan=1 colspan=1>C. parvum</td><td rowspan=1 colspan=1>105}$</td></tr><tr><td rowspan=1 colspan=1>C. parvum</td><td rowspan=1 colspan=1>D. fragilis</td><td rowspan=1 colspan=1>105}$</td></tr><tr><td rowspan=1 colspan=1>C. parvum</td><td rowspan=1 colspan=1>C. cayetanensis</td><td rowspan=1 colspan=1>108 *</td></tr><tr><td rowspan=7 colspan=1>B</td><td rowspan=1 colspan=1>B. hominis</td><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>1055}$</td><td rowspan=1 colspan=1>Interference</td></tr><tr><td rowspan=1 colspan=1>B. hominis</td><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>104 $</td><td rowspan=5 colspan=1>No reportableinterference</td></tr><tr><td rowspan=1 colspan=1>B. hominis</td><td rowspan=1 colspan=1>G. intestinalis</td><td rowspan=1 colspan=1>10f5}$</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>B. hominis</td><td rowspan=1 colspan=1>10}$</td></tr><tr><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>G. intestinalis</td><td rowspan=1 colspan=1>105}$</td></tr><tr><td rowspan=1 colspan=1>G. intestinalis</td><td rowspan=1 colspan=1>B. hominis</td><td rowspan=1 colspan=1>10\rac5}$</td></tr><tr><td rowspan=1 colspan=1>G. intestinalis</td><td rowspan=1 colspan=1>E. histolytica</td><td rowspan=1 colspan=1>10c}$</td><td rowspan=1 colspan=1>Moderate</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>E. bieneusi</td><td rowspan=1 colspan=1>E. intestinalis</td><td rowspan=1 colspan=1>10ac5}$</td><td rowspan=2 colspan=1>No reportableinterference</td></tr><tr><td rowspan=1 colspan=1>E. intestinalis</td><td rowspan=1 colspan=1>E. bieneusi</td><td rowspan=1 colspan=1>105}$</td></tr></table>

$\ S$ Tests repeated at a lower concentration of competing targets, which resolved the interference.

# 6. Specimen Stability:

To provide evidence in support of the stability of specimens stored for testing with the EasyScreen Gastrointestinal Parasite Detection Kit, a range of storage conditions were evaluated using the methodology described below.

# A. Specimen stability at $2 { - } 8 ^ { o } C .$

From the EP005 Panels A–C, individual target analytes (i.e., whole organisms for targets except C. cayetanensis, which employed a synthetic RNA target) were spiked at 3X LoD into unpreserved negative stool matrix or Cary-Blair medium. Failure of testing in negative stool matrix at 3X LoD for $C .$ . cayetanensis and $B _ { \cdot }$ . hominis necessitated re-testing at 10X LoD for these two (2) targets in the negative stool matrix.

Baseline (time zero) test results were established by testing the contrived samples with EasyScreen Gastrointestinal Parasite Detection Kit on the day of preparation. Aliquots prepared for each analyte in each matrix were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for at least (3) weeks with weekly testing and acceptance criteria set at $100 \%$ positivity (10/10 replicates). Based on the test observations, all target analytes are stable for three (3) weeks in unpreserved negative stool matrix. For analytes in Cary Blair matrix, except $C .$ . cayetanensis and E. histolytica, all targets are stable for three (3) weeks, whereas $C .$ . cayetanensis and E. histolytica are stable for two (2) weeks in the Cary Blair matrix.

# B. Fresh vs. Frozen specimen stability:

A fresh versus frozen study was conducted to support the use of frozen samples in the analytical studies and to provide a scientific rationale for the acceptable use of frozen prospective (Category II) and retrospective (Category III) samples in the clinical studies with the EasyScreen Gastrointestinal Parasite Detection Kit. Individual sample dilutions were prepared for each analyte in the Panels $\mathrm { \bf A } { - } \mathrm { \bf C }$ in unpreserved negative stool matrix, including Negative (no target, 10 replicates), Low Positive (2X LoD, 20 replicates) and Moderately Positive (4X LoD, 10 replicates) samples. Baseline (“fresh”) test results were established by testing the contrived samples on the day of sample preparation. Aliquots of each analyte at each concentration were stored frozen at $- 2 0 \pm 5 ^ { \circ } \mathrm { C }$ and thawed for examination at four (4) weeks (28–30 days). Acceptance criteria for replicate results of each analyte were set at: 4X LoD, $100 \%$ (10/10 replicates) positive; 2X LoD, $2 9 5 \%$ $( \geq 1 9 / 2 0 )$ positive; and Negative, $0 \% ( 0 / 1 0 )$ positive.

All eight (8) target parasites showed $100 \%$ detection at 4X LoD and $2 9 5 \%$ detection at 2X LoD at the 4-week timepoint after being frozen at ${ } - 2 0 \pm 5 { } ^ { \circ } { } C$ , except for $B _ { \cdot }$ . hominis, which was tested at earlier at 3 weeks due to time constraints. Average Ct values at the 4-week timepoint (3-week timepoint for $B$ . hominis) were within $\pm 1 0 \%$ of the baseline average Ct values for all targets and concentrations assessed. The results supported a frozen storage stability claim of three (3) weeks for all targets other than $B .$ . hominis (frozen storage stability of two (2) weeks for $B .$ . hominis was acceptable) when tested with the EasyScreen Gastrointestinal Parasite Detection Kit following manufacturer’s instructions.

# 7. Detection Limit:

The analytical sensitivity (Limit of Detection / LoD) of the EasyScreen Gastrointestinal Parasite Detection Kit was established for all targets in a matrix of either unpreserved confirmed negative stool or Cary Blair media. All samples were evaluated following the device’s Instructions for Use. LoD, evaluated from two (2) isolates per target, is expressed in organisms $( \mathrm { { o r g / m L } ) }$ or genome copy number (copies/mL) per mL of sample, where one organism is defined as one (1) haploid genome, determined by digital PCR or quantitative PCR for either the 18S rRNA gene, or a singlecopy gene target, and where necessary, divided by the published 18S rRNA gene copy number for that organism.

Following a preliminary range-finding study, LoDs were established with a minimum of twenty (20) extraction replicates using two (2) different isolates (or two (2) strains/genotype, where available) for each target. For LoD confirmation, the acceptance criteria were set at $2 9 5 \%$ detection of the specified target AND ${ < } 9 5 \%$ detection at 0.5X LoD. When the acceptance criteria were not met in the first instance, targets were tested at 2-times lower or higher concentration as required until the criteria were met. These studies employed both AB 7500DX and QSDX analyzers. The results of the LoD studies showed comparable performance with minimal variability observed between LoD values obtained across different isolates, PCR analyzers and EP005 reagent batches with all targets showing an LoD within a $\pm 2$ -fold dilution across all variables. The final LoD for each organism was defined for each sample matrix as the lowest concentration tested meeting the LoD criteria when results were combined for all instruments and kit lots tested (see Table 10).

Table 10: Analytical Sensitivity (LoD) results for EasyScreen Gastrointestinal Parasite Detection Kit target parasite in different matrices:   

<table><tr><td rowspan=2 colspan=1>ReagentPanel</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>LoD (analyzers: QSDX and 7500DX)</td></tr><tr><td rowspan=1 colspan=1>unpreserved stool matrix(org/mL)</td><td rowspan=1 colspan=1>Cary Blair matrix(org/mL)</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Dientamoeba fragilis</td><td rowspan=1 colspan=1>62.5</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>1.56</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>2060</td><td rowspan=1 colspan=1>7723</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>6.25</td><td rowspan=1 colspan=1>12.5</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>112.5</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>1425</td><td rowspan=1 colspan=1>981</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>Enterocytozoon bieneusi</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Encephalitozoon intestinalis</td><td rowspan=1 colspan=1>5000</td><td rowspan=1 colspan=1>5000</td></tr></table>

# 8. Assay Cut-Off:

For all target parasites, cut-off values for threshold Cycle (Ct) were determined in analytical studies on both QSDX and AB7500 DX thermocyclers and verified using the multi-site clinical study data. Target-specific cut-offs (defining a negative) were determined during analytical studies using Ct values recorded for LoD or near LoD samples. For each targeted analyte, sample Ct value metrics from the clinical studies were directly compared to the Reference method (i.e., alternative NAAT followed by bidirectional sequencing, as indicated in FDA’s Class II Special Controls Guideline: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens) and used a mathematical validation approach that avoided false negatives and shifted false positive calls to negative calls while retaining the near LoD sample Ct values to positive calls. The accepted and locked in cut-off Ct values for each target (see Table 11 below) were applied to the results of the candidate assay. Apart from target-specific Ct cut-off values, cut-offs for extraction control (EC) and internal positive control (IPC) were established as $\mathrm { C t } \le 3 7$ and $\leq 4 0$ respectively through prior use with other stool-based EasyScreen assays from Genetic Signatures.

Table 11: Cut-offs for EP005 Panel   

<table><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Target                                                  Cutoff Ct</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>D. fragilis                                              &lt;41</td></tr><tr><td rowspan=1 colspan=1>C. cayetanensis                                       &lt;45</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp. (including C. parvum)   &lt;42</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>B. hominis                                              &lt;43</td></tr><tr><td rowspan=1 colspan=1>E. histolytica                                          &lt;45</td></tr><tr><td rowspan=1 colspan=1>G. lamblia/G. intestinalis                          &lt;40</td></tr><tr><td rowspan=1 colspan=2>C            E. bieneusi                                             &lt;44E. intestinalis                                          &lt;41</td></tr></table>

# 9. Carry-Over / Cross Contamination:

A carry-over study was conducted to investigate the potential for cross contamination or carry over between wells when using the workflow of the EasyScreen Gastrointestinal Parasite Detection Kit. Pooled high positive samples were prepared containing one representative member of each panel (Cyclospora cayetanensis, Giardia lamblia and Enterocytozoon bieneusi) consisting of at least 1 $\textbf { x } 1 0 ^ { 6 }$ copies/mL of each target in vitro transcript (IVT) in negative stool matrix, following the recommendations provided in FDA-recognized voluntary consensus standard CLSI EP39. IVTs were used in this study as most of the analytical targets available were not able to be extracted at the high copy numbers required. A negative sample was prepared containing negative stool matrix containing no target analyte and tested in one stage.

Sixteen (16) replicates of pooled high positive sample and fourteen (14) replicates of negative sample were extracted in each run in an alternating negative and positive sample pattern. In this sample pattern, there were 6 and 8 negative wells surrounded by, respectively, 4 and 3 positive wells around them. A total of five (5) extractions and PCR setups were performed with each run containing one negative processing control and thirty (30) samples, which were then seeded into 96 wells of a PCR plate by the GS1 platform. Since each panel gets one set of reagents, one representative organism per panel is acceptable. Target detection in PCR was assessed for all samples, with pre-defined acceptance criteria met at $100 \%$ (240/240) detection for pooled high positive samples and $0 \%$ detection for negative samples (0/210) and the negative processing control (0/15), demonstrating that in the EasyScreen Gastrointestinal Parasite Detection Kit workflow, there was no reportable carry over/cross contamination between the wells during sample preparation or PCR set up.

# B Comparison Studies:

# 1. Method Comparison with Predicate Device: N/A

2. Matrix Equivalency between Cary-Blair and C&S:

A matrix equivalency study was conducted in support of the use of C&S media (e.g., Meridian Parapak #900612) as an alternative to Cary Blair media (e.g., Thermo Scientific Remel #R21610) when storing and/or transporting human clinical stool specimens for later processing with the EasyScreen Gastrointestinal Parasite Detection Kit. For each analyte in the Panels A–C, individual sample dilutions were prepared in C&S matrix to include Negative (no analyte), Low Positive (1X–2X LoD), and High Positive (5X LoD) samples. Performance characteristics of the samples contrived in C&S media were ascertained at the selected analyte levels using five (5) extraction replicates of High Positives, twenty-five (25) replicates of Low Positives, and ten (10) replicates of Negative samples. Acceptance criteria for replicate results of each analyte were set at: 5X LoD, $100 \%$ (5/5 replicates) positive; 1–2X LoD, $2 9 5 \%$ $( { \geq } 2 4 / 2 5 )$ positive; and Negative, $0 \% ( 0 / 1 0 )$ positive. For each target, the initial LoD values previously obtained with Cary-Blair matrix during Analytical Sensitivity studies served as the baseline for this study. As shown in Table 12, for all targets and analyte concentrations tested, the targets diluted in C&S matrix met the acceptance criteria based on LoD in Cary Blair matrix, demonstrating equivalence for use in the EasyScreen Gastrointestinal Parasite Detection Kit.

Table 12. C&S Matrix equivalency for targets with various analyte concentrations tested in EasyScreen Gastrointestinal Parasite Detection Kit assay.   

<table><tr><td colspan="1" rowspan="1">Target analyte</td><td colspan="1" rowspan="1">Tested analyte level in C&amp;Smatrix relative to LoD inCary-Blair matrix</td><td colspan="1" rowspan="1">Detection (%)</td><td colspan="1" rowspan="1">Avg Ct</td></tr><tr><td colspan="1" rowspan="2">D. fragilis</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">24/25 (96%)</td><td colspan="1" rowspan="1">33.17</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">30.9</td></tr><tr><td colspan="1" rowspan="2">C. cayetanensis</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">33.33</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">33.45</td></tr><tr><td colspan="1" rowspan="1">C. parvum</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">29.69</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">26.94</td></tr><tr><td colspan="1" rowspan="2">B. hominis</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">31.60</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">29.36</td></tr><tr><td colspan="1" rowspan="2">E. histolytica</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">34.80</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">32.73</td></tr><tr><td colspan="1" rowspan="2">G. intestinalis</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">34.55</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">31.34</td></tr><tr><td colspan="1" rowspan="2">E. bieneusi</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">34.87</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">32.69</td></tr><tr><td colspan="1" rowspan="2">E. intestinalis</td><td colspan="1" rowspan="1">1X LoD</td><td colspan="1" rowspan="1">25/25 (100%)</td><td colspan="1" rowspan="1">33.66</td></tr><tr><td colspan="1" rowspan="1">5X LoD</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1">29.97</td></tr></table>

# C Clinical Studies:

Clinical Study details:

A multicenter clinical study was conducted to assess the performance of the EasyScreen Gastrointestinal Parasite Detection kit for the identification of Cryptosporidium spp., Cyclospora cayetanensis, Giardia intestinalis, Dientamoeba fragilis, Entamoeba histolytica, Blastocystis hominis, Enterocytozoon bieneusi, and Encephalitozoon intestinalis using stool specimens from symptomatic patients with suspected gastroenteritis. The study evaluated results obtained with the EasyScreen Gastrointestinal Parasite Detection kit, in comparison to those obtained with a reference method. Following FDA’s Class II Special Controls Guidelines “Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens” the reference method for the clinical studies was chosen to be two (2) well-characterized and validated Nucleic Acid Amplification Tests (NAAT) followed by bi-directional sequencing (referred to as “alternative NAAT”).

For the clinical study, sites were selected based on several criteria, including investigator and study staff availability, number of specimens of interest, target prevalence and familiarity with PCR methodology. The three (3) participating US sites for prospective sample collection and testing were geographically diverse by location, and an additional US site performed retrospective sample procurement along with testing.

The clinical study was designed in an All-Comers mode to prospectively collect stool samples from symptomatic subjects to be tested fresh (Category I samples) or after frozen storage (Category II samples). Stool specimens were collected from patients of any age (ranging ${ < } 5$ to ${ \ge } 6 0$ years), who presented with signs and/or symptoms of gastroenteritis and were referred for testing. However, given the low prevalence of some of the target parasites in specific geographic areas as well as problems in enrollment during COVID-19 pandemic-adjacent times, the enrollment of retrospective, positive-identified (Category III) samples was considered acceptable along with the inclusion of randomly distributed negative samples, all to be tested in a masked/blinded manner. Finally, for two (2) low-prevalence target parasites, namely, $E$ . bieneusi and $E$ . intestinalis, for which retrospective samples were not available despite best efforts, contrived (Category IV) samples in a negative stool matrix were prepared and enrolled at an outside the US (OUS) internal site for testing at the US sites.

All specimens were processed with the EasyScreen Sample Processing Kit using the automated Genetic Signatures GS1 platform and tested with the EasyScreen Gastrointestinal Parasite Detection kit (hereafter collectively referred to as “EP005”) device following manufacturer’s instructions.

True analyte status (“positive” or “negative”) of each sample was established for each target parasite by comparison with the reference method of alternative NAATs conducted at the OUS site. For each target analyte, the alternative NAATs consisted of two (2) separate single-plex, PCR amplification tests which targeted regions not covered by or adjacent to the EP005 assay reagents. Amplicons from all PCR-positive reactions sequenced with bi-directional Sanger sequencing at the internal site. Samples producing sequence data that met the acceptability criteria listed in Section VII(D)(1) of the Class II Special Controls guidelines were reported as positive by alternative NAAT.

Results from the EP005 device were also compared against an FDA-cleared assay following its device labeling. In this comparison, only those three (3) protozoan targets were evaluated that are detected by the comparator—namely, Giardia lamblia (a.k.a. G. intestinalis), Cryptosporidium species hominis and parvum, as well as Entamoeba histolytica. Category III samples enrolled in the EP005 clinical study were not tested with the comparator that is not validated for use with that specimen type.

A total of 2,806 specimens (Categories I–IV) were collected with 880 samples excluded due to invalid and/or missing data, storage and volume limitations, improperly contrived samples with incorrect targets, which left 1,926 analyzable specimens for performance evaluation (see Table 13).

Table 13: Sample Details $\mathbf { \Phi } ( N = I , 9 2 6 )$   

<table><tr><td rowspan=2 colspan=1>Prospectiveandretrospective</td><td rowspan=1 colspan=2>US1N = 204</td><td rowspan=1 colspan=2>US2N=483</td><td rowspan=1 colspan=2>US3N=966</td><td rowspan=1 colspan=2>US4N=252</td><td rowspan=1 colspan=2>OUS1N=21</td><td rowspan=1 colspan=2>Total N=1926</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=7>Stool type</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>canmaGlow</td></tr><tr><td rowspan=1 colspan=1>TransportMedia (incl.Cary Blair /C&amp;S)</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>76.96</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>45.34</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>25.05</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>870</td><td rowspan=1 colspan=1>45.17</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>23.04</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>33.95</td><td rowspan=1 colspan=1>674</td><td rowspan=1 colspan=1>69.77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>885</td><td rowspan=1 colspan=1>45.95</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.70</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>5.18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>8.88</td></tr><tr><td rowspan=1 colspan=13>SEX</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>65.2</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>57.97</td><td rowspan=1 colspan=1>579</td><td rowspan=1 colspan=1>59.94</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>40.08</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,093</td><td rowspan=1 colspan=1>56.75</td></tr><tr><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>202</td><td rowspan=1 colspan=1>41.82</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>39.96</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>59.92</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>42.06</td></tr><tr><td rowspan=1 colspan=1>unknown</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>1.19</td></tr><tr><td rowspan=1 colspan=12>AGE (years)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.92</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5.59</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2.80</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>11.51</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.52</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>4.83</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>13.73</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>14.29</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>9.21</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>28.57</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>11.63</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>42.65</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>46.38</td><td rowspan=1 colspan=1>406</td><td rowspan=1 colspan=1>42.03</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>53.97</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>861</td><td rowspan=1 colspan=1>44.7</td></tr><tr><td rowspan=1 colspan=1>60≥</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>39.71</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>33.75</td><td rowspan=1 colspan=1>444</td><td rowspan=1 colspan=1>45.96</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>21.03</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.52</td><td rowspan=1 colspan=1>743</td><td rowspan=1 colspan=1>38.58</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14.29</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.26</td></tr></table>

Of these 1,926 samples with demographic data, 200 samples requiring a repeat test were invalid on retest, making for a $1 0 . 4 \%$ (200/1926) invalid rate. Therefore, 1,726 prospective $( \mathrm { n } = 1 4 6 1 $ ) and retrospective $( \mathbf { n } ~ = ~ 2 6 5$ ) clinical samples were available for analysis with both EP005 and corresponding alternative NAAT reference method. Addition of Category IV samples $\mathrm { \Delta } \mathrm { T } \mathrm { n } = 1 6 5 \mathrm { \Delta }$ ) with $E$ . bieneusi and $E .$ . intestinalis brought the total analyzable study sample count up to 1,891.

Based on evaluation of clinical study data provided from 1,891 stool specimens, the performance of the EasyScreen Gastrointestinal Pathogen Detection Kit is acceptable with PPA for individual target parasites ranging between $91 - 9 9 \%$ with lower limit of $9 5 \%$ CI at $2 8 0 \%$ and NPA $2 9 9 \%$ when compared to the reference method.

# A. Performance Estimates of EP005 relative to the Reference Method:

This section summarizes the target analyte-specific comparison of EasyScreen Gastrointestinal Parasite Detection kit (EP005) results to the reference method for the 1,461 prospective clinical samples, 265 retrospective clinical samples, and the 165 contrived samples with valid results on both tests. For results from clinical studies presented in Tables 14–22, the following abbreviations are used: all relative to the reference method, True Positive, TP; True Negative, TN; False Positive, FP; False negative, FN; Positive Percent Agreement (sensitivity), PPA; Negative Percent Agreement (specificity), NPA; lower (LL), upper limit (UL) of the $9 5 \%$ Confidence Interval (CI).

Table 14. Performance metrics of D. fragilis by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA(%)</td><td rowspan=1 colspan=2>95%CI</td><td rowspan=2 colspan=1>NPA(%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1441</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2 a</td><td rowspan=1 colspan=1>86.67</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>99.65</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2 b</td><td rowspan=1 colspan=1>94.87</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99.56</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>100</td></tr></table>

a) Two (2) FN prospective samples were not available for duplicate, investigational EP005 retests. b) Two (2) FN retrospective samples were again found negative in duplicate EP005 retests and were originally reported negative for $D$ . fragilis by a US site’s laboratory-developed test.

Table 15. Performance metrics of $C .$ cayetanensis by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1454</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.59</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>97.73</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.74</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>99</td></tr></table>

a) One (1) FN retrospective sample was found positive for C. cayetanensis in duplicate, investigational EP005 retests.

Table 16. Performance metrics of Cryptosporidium spp. by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA(%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA(%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1453</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.66</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4 a</td><td rowspan=1 colspan=1>90.7</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>99</td></tr></table>

a) Three of the four (3/4) FN samples were found positive for Cryptosporidium in duplicate, investigational EP005 retests. The fourth (1/4) sample was found negative for Cryptosporidium in duplicate.

Table 17. Performance metrics of B. hominis by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>1401</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2 a</td><td rowspan=1 colspan=1>95.92</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99.22</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.94</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td></tr></table>

a) Two (2) FN prospective samples were not available for duplicate, investigational EP005 retests. b) One (1) FN retrospective sample was found positive for $B .$ . hominis in duplicate, investigational EP005 retests.

Table 18. Performance metrics of E. histolytica by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1457</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>99.73</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>96.88</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99.14</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>100</td></tr></table>

a) One (1) FN sample was co-infected with $B$ . hominis and was found positive for both targets in duplicate, investigational EP005 retests.

Table 19. Performance metrics of G. intestinalis by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA(%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1452</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.52</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>97.06</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>94.81</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>97</td></tr></table>

a) One (1) FN sample was co-infected with $B$ . hominis and was found positive for both targets in duplicate, investigational EP005 retests.

Table 20. Performance metrics of E. bieneusi by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA (%)</td><td rowspan=1 colspan=2>95%CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1448</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.52</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.24</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>98.68</td><td rowspan=1 colspan=1>92.92</td><td rowspan=1 colspan=1>99.77</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.86</td><td rowspan=1 colspan=1>100</td></tr></table>

a) One (1) FN sample was a contrived sample spiked with $E$ . bieneusi at near-LoD.

Table 21. Performance metrics of E. intestinalis by Analysis groups   

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>TN</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>PPA(%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA(%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1461</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1454</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>99.52</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6 a</td><td rowspan=1 colspan=1>92.41</td><td rowspan=1 colspan=1>84.4</td><td rowspan=1 colspan=1>96.47</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.72</td><td rowspan=1 colspan=1>100</td></tr></table>

a) All six (6) FN samples were contrived samples spiked with $E$ . intestinalis at near-LoD.

# B. Performance Estimates of EP005 relative to the Comparator:

This section summarizes the target analyte-specific comparison of EasyScreen Gastrointestinal Parasite Detection kit EP005 results to results obtained from the predicate for 760 samples with valid results on both tests (as shown in Table 22). This performance metrics estimation excluded all samples $\mathbf { \tilde { n } } = 1 2 6 )$ that tested invalid (on EP005) or UNR (i.e., unresolved, on the comparator).

Table 22: 2x2 Performance Comparison Tables for EP005 vs. comparator.   

<table><tr><td rowspan=2 colspan=2>Target: G. intestinalis</td><td rowspan=1 colspan=4>Comparator Test</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>EP005</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>756</td><td rowspan=1 colspan=1>757</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>758</td><td rowspan=1 colspan=1>760</td></tr><tr><td rowspan=2 colspan=2>Target: Cryptosporidium spp.</td><td rowspan=1 colspan=4>Comparator Test</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>EP005</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2>757</td><td rowspan=1 colspan=1>759</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>757</td><td rowspan=1 colspan=1>760</td></tr><tr><td rowspan=2 colspan=2>Target: Entamoeba histolytica</td><td rowspan=1 colspan=4>Comparator Test</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>EP005</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>757</td><td rowspan=1 colspan=1>757</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>760</td><td rowspan=1 colspan=1>760</td></tr></table>

Table 23: Performance estimates of EP005 relative to the comparator.   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>PPA (%)</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=2 colspan=1>NPA (%)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td><td rowspan=1 colspan=1>LL</td><td rowspan=1 colspan=1>UL</td></tr><tr><td rowspan=1 colspan=1>G. intestinalis</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium spp.</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>79.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E. histolytica a</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>100</td></tr></table>

a) EasyScreen Gastrointestinal Parasite Detection kit identified no true-positive and three false-positive Entamoeba histolytica samples.

# C. Co-infections detected by EP005 in clinical study samples with validation by the reference method.

This section summarizes the number of multi-parasite $( \mathtt { n } = 6 8 )$ ) Category I–IV samples detected by the EasyScreen Gastrointestinal Parasite Detection Kit (EP005) as presented in Table 24. Column N represents unique number of samples with targets validated by the reference method. Samples for any analytes with discrepant results between EP005 and reference method were not considered for this summary.

Table 24: Tabulation of Co-Infections as detected by both EP005 and the reference method.   

<table><tr><td colspan="1" rowspan="1">Co-Infections</td><td colspan="1" rowspan="1">Na</td></tr><tr><td colspan="1" rowspan="1">B. hominis, C. cayetanensis</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">B. hominis, Cryptosporidium spp.</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">B. hominis, Cryptosporidium spp., G. intestinalis</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">B. hominis, D. fragilis</td><td colspan="1" rowspan="1">23 b</td></tr><tr><td colspan="1" rowspan="1">B. hominis, D. fragilis, C. cayetanensis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. hominis, D. fragilis, G. intestinalis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. hominis, E. bieneusi</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">B. hominis, E. histolytica</td><td colspan="1" rowspan="1">9c</td></tr><tr><td colspan="1" rowspan="1">B. hominis, G. intestinalis</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">B. hominis, E. bieneusi, G. intestinalis</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">C. cayetanensis, D. fragilis</td><td colspan="1" rowspan="1">[1</td></tr><tr><td colspan="1" rowspan="1">C. cayetanensis, G. intestinalis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium spp., G. intestinalis</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Co-Infections</td><td colspan="1" rowspan="1">N a</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium spp., E. bieneusi</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium spp., D. fragilis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">D. fragilis, G. intestinalis</td><td colspan="1" rowspan="1">2d</td></tr><tr><td colspan="1" rowspan="1">E. bieneusi, G. intestinalis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Totals</td><td colspan="1" rowspan="1">73</td></tr></table>

a) Not counted in this enumeration are five (5) retrospective samples which were discrepant for a second target that was detected upon EP005 retests: #201-1161 $( D .$ . fragilis TP $^ +$ Cryptosporidium spp. $\mathrm { F N } {  } \mathrm { T P }$ ), #201-1187 ( $G .$ . intestinalis $\mathrm { T P } + B$ . hominis $\mathrm { F N } {  } \mathrm { T P }$ ), $\# 2 0 1 .$ - 1198 (B. hominis $\mathrm { T P } + G$ . intestinalis $\mathrm { F N { \to } T P }$ ), #201-1211 ( $B .$ . hominis $\mathrm { T P } + E$ . histolytica $\mathrm { F N } {  } \mathrm { T P }$ ), and #201-1243 ( $G .$ . intestinalis TP $^ +$ Cryptosporidium spp. $\mathrm { F N } {  } \mathrm { T P }$ ).   
b) Not counted in this category was one (1) discrepant prospective sample (#103-0050) that was additionally positive for $D$ . fragilis by the reference method but FN by EP005 and enough sample was not available for EP005 retest.   
c) In this category, one (1) retrospective sample (#201-1066) was additionally positive for $D$ . fragilis by the reference method but persistently FN by EP005. This was not enumerated under D. fragilis.   
d) Not counted in this category was one (1) discrepant retrospective sample (#201-1223) that was additionally positive for $D$ . fragilis by the reference method but persistently FN by EP005.   
D Clinical Cut-Off: N/A   
E Expected Values/Reference Range: N/A   
F Other Supportive Instrument Performance Characteristics Data: N/A

# VIII Proposed Labeling:

Proposed labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.