# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K233352   
B Applicant Hologic, Inc.   
C Proprietary and Established Names Aptima HCV Quant Dx Assay   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>MZP</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3170 -Nucleic Acid-BasedHepatitis C VirusRibonucleic Acid Tests</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

Market clearance of Aptima HCV Quant Dx assay after modifications to the approved device (P160023). Modifications include:

Update the Panther System Software to version 7.2.9 to incorporate: An update to the Assay Definition Module (ADM) to version 5.3.5.1 to implement new background minimum limits (Detect, flag and invalidate results impacted by a flickering/faulty LED lamp); Additional reagent on-board/off-board cycling from 5 to 8 cycles. This will facilitate operators loading reagents onto the Panther System 8 times instead of 5 times.

These changes do not introduce any changes to the original design, method of manufacture, assay procedure, principle of operation, mechanism of action, conditions of use or hardware of the Panther instrument, or to the results interpretation for the cleared assay.

# B Measurand:

HCV RNA

# C Type of Test:

Nucleic acid amplification test that uses real-time TMA technology to detect and quantitate HCV RNA

# III Intended Use/Indications for Use:

# A Intended Use(s):

The Aptima HCV Quant Dx assay is a real-time transcription-mediated amplification (TMA) test used for both detection and quantitation of hepatitis C virus (HCV) RNA in fresh and frozen human serum and plasma from HCV-infected individuals.

Plasma may be prepared in ethylenediaminetetraacetic acid (EDTA), anticoagulant citrate dextrose (ACD) solution, and plasma preparation tubes (PPT). Serum may be prepared in serum tubes and serum separator tubes (SST). Specimens are tested using the Panther system for automated specimen processing, amplification, detection, and quantitation. Specimens containing HCV genotypes 1 to 6 are validated for detection and quantitation in the assay.

The Aptima HCV Quant Dx assay is indicated for use as an aid in the diagnosis of active HCV infection in the following populations: individuals with antibody evidence of HCV infection with evidence of liver disease, individuals suspected to be actively infected with HCV antibody evidence, and individuals at risk for HCV infection with antibodies to HCV. Detection of HCV RNA indicates that the virus is replicating and, therefore, is evidence of active infection. Detection of HCV RNA does not discriminate between acute and chronic states of infection.

The Aptima HCV Quant Dx assay is also indicated for use as an aid in the management of HCV infected patients undergoing HCV antiviral drug therapy. The assay can be used to measure HCV RNA levels periodically prior to, during, and after treatment to determine sustained virological response (SVR) or nonsustained virological response (NSVR). Assay performance characteristics have been established for individuals infected with HCV and treated with certain direct-acting antiviral agents (DAA) regimens. No information is available on the assay’s predictive value when other therapies are used. The results from the Aptima HCV Quant Dx assay must be interpreted within the context of all relevant clinical and laboratory findings.

The Aptima HCV Quant Dx assay is not approved for use as a screening test for the presence of HCV RNA in blood or blood products.

# B Indication(s) for Use:

See Intended Use above C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

# D Special Instrument Requirements:

For use with Panther System

# IV Device/System Characteristics:

# A Device Description:

The Aptima HCV Quant Dx assay has been designed for and validated on the Panther system. The Panther system is an integrated hardware and software system that together with the Aptima HCV Quant Dx assay fully automates all the steps necessary to perform the assay from sample preparation through amplification of nucleic acid, detection, data reduction and amplicon inactivation.

The Aptima HCV Quant Dx assay is a nucleic acid amplification test that uses real-time TMA technology to detect and quantitate HCV RNA for aiding diagnosis or to establish baseline viral load, as well as to measure on-treatment and post-treatment responses. The assay targets a conserved region of the HCV genome, detecting and quantitating genotypes 1, 2, 3, 4, 5, and 6. The assay is standardized against the 2nd WHO International Standard for Hepatitis C Virus (NIBSC Code 96/798).12.

# B Principle of Operation:

The Aptima HCV Quant Dx Assay involves three main steps, which all take place in a single tube on the Panther system: target capture, target amplification by transcription-mediated amplification (TMA), and detection of the amplification products (amplicon) by the fluorescent labeled probes (torches).

During target capture, viral RNA is isolated from specimens. The specimen is treated with a detergent to solubilize the viral envelope, denature proteins, and release viral genomic RNA. Capture oligonucleotides hybridize to highly conserved regions of the HCV genome, if present, in the test specimen. The hybridized target is then captured onto magnetic microparticles that are separated from the specimen in a magnetic field. Wash steps remove extraneous components from the reaction tube.

Target amplification occurs via TMA, which is a transcription-based nucleic acid amplification method that utilizes two enzymes, MMLV (Moloney murine leukemia virus) reverse transcriptase and T7 RNA polymerase. The reverse transcriptase is used to generate a DNA copy (containing a promoter sequence for T7 RNA polymerase) of the target sequence. T7 RNA polymerase produces multiple copies of RNA amplicons from the DNA copy template. The Aptima HCV Quant Dx Assay utilizes the TMA method to amplify the $5 '$ UTR of the HCV genome. Amplification of this region is achieved using specific primers which are designed to amplify HCV genotypes 1, 2, 3, 4, 5, and 6.

Detection is achieved using single-stranded nucleic acid torches that are present during the amplification of the target and that hybridize specifically to the amplicon in real-time. Each torch has a fluorophore and a quencher. When the torch is not hybridized to the amplicon, the quencher is in close proximity of the fluorophore and suppresses the fluorescence. When the torch binds to the amplicon, the quencher is moved away from the fluorophore and it will emit a signal at a specific wavelength when excited by a light source. As more torches hybridize to amplicons, a higher fluorescent signal is generated.

The time taken for the fluorescent signal to reach a specified threshold is inversely proportional to the starting HCV concentration. Each reaction has an internal calibrator/internal control (IC) that controls for variations in specimen processing, amplification, and detection. The level of HCV RNA in a sample is determined by the Panther system software using the HCV and IC signals for each reaction and comparing them to calibration information.

# Assay Components

The reagents required to perform the Aptima HCV Quant Dx assay are available in 100-test kits. The kits are packaged in 3 boxes containing 11 reagents which are required for sample processing.

Box 1: Aptima HCV Quant Dx Assay kit which contains the following reagents:

Amplification Reagent Enzyme Reagent Promoter Reagent Amplification Reconstitution Reagent Enzyme Reconstitution Reagent Promoter Reconstitution Reagent Target Capture Reagent

Box 2: Aptima HCV Quant Dx Calibrator kit which contains the following reagent:  Positive Calibrator

Box 3: Aptima HCV Quant Dx Controls kit which contains the following reagents:  Negative Control Low Positive Control High Positive Control

In addition, two ancillary kits are required to run the assay:

Panther Run Kit for Real Time Assays (for real time assays only)   
Panther System Run Kit (when running non-real time-TMA assays in parallel with real time  
TMA assays)

# Interpretation of Results

The Panther system automatically determines the concentration of HCV RNA for specimens and controls by comparing the results to a calibration curve. HCV RNA concentrations are reported in $\mathrm { I U / m L }$ and $\mathrm { l o g 1 0 \ I U / m L }$ . The interpretation of results is provided in the table below. If a 1:3 or 1:100 dilution is used for diluted specimens, the Panther system automatically calculates the HCV concentration for the neat specimen by multiplying the diluted concentration by the dilution factor and diluted specimens are flagged as diluted. Note: For diluted specimens, results listed as “Not Detected” or $^ { 6 6 } < 1 0$ Detected” may be generated by diluting a specimen with a concentration above, but close to the LoD (limit of detection) or LLoQ (lower limit of quantitation. It is recommended to collect and test another neat specimen if a quantitative result is not obtained with the diluted specimen.

The Panther system does not directly provide a qualitative result (i.e., “Detected” or “Not Detected”) for diagnostic use. The operator must interpret the reported HCV RNA concentration into a qualitative result as provided in Table 1.

Table 1: Aptima HCV Quant Dx assay Results Interpretation   

<table><tr><td rowspan=1 colspan=2>Reported Aptima HCVQuant Dx Result</td><td rowspan=2 colspan=1>HCV RNA ConcentrationInterpretation</td><td rowspan=2 colspan=1>Clinical Interpretation</td></tr><tr><td rowspan=1 colspan=1>IU/mL</td><td rowspan=1 colspan=1>Log10Valueb</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>HCV RNA not detecteda.Report results as &quot;HCV not detected&quot;</td><td rowspan=1 colspan=1>No Current HCV Infection.Follow-up testing is recommended as pernational HCV guidelines for viral loadassessment, and no further testing isrecommended for diagnosis of HCVd.</td></tr><tr><td rowspan=1 colspan=1>&lt;10 Detected</td><td rowspan=1 colspan=1>&lt;1.00</td><td rowspan=1 colspan=1>HCV RNA detected but not quantified.HCV RNA concentration is below theLower Limit of Quantification (LLoQ)of the assay.Report results as &quot;HCV detected, lessthan 10 IU/mL&quot;</td><td rowspan=1 colspan=1>Follow-up testing is recommended as pernational HCV guidelines for viral loadassessment, and results must be interpretedwithin context of all relevant clinical andlaboratory findingsd for diagnosis of HCV.</td></tr><tr><td rowspan=1 colspan=1>10 to 25</td><td rowspan=1 colspan=1>1.00 to 1.40</td><td rowspan=1 colspan=1>HCV RNA detected and quantified.HCV RNA concentration is withinlinear range of the assay ≥ 10 IU/mLand &lt; 25 IU/mL.</td><td rowspan=1 colspan=1>Provide guidance for treatment and care basedon current national HCV treatment guidelinesfordiagnosis of HCV and viral load assessment.</td></tr><tr><td rowspan=1 colspan=1>25 to100,000,000</td><td rowspan=1 colspan=1>1.40 to 8.00</td><td rowspan=1 colspan=1>HCV RNA detected and quantified.HCV RNA concentration is within thelinear range of 25 to100,000,0001U/mL</td><td rowspan=1 colspan=1>Current HCV Infection.Provide guidance for treatment and care basedon current national HCV treatment guidelinesfor diagnosis and viral load assessment of HCV.</td></tr><tr><td rowspan=1 colspan=1>&gt;100,000,000</td><td rowspan=1 colspan=1>&gt;8.00</td><td rowspan=1 colspan=1>HCV RNA is detected above theUpper Limit of Quantification(ULoQ).</td><td rowspan=1 colspan=1>Current HCV Infection.For HCV Diagnosis and Viral Load AssessmenteProvide guidance for treatment and care basedon current national HCV treatment guidelines.</td></tr><tr><td rowspan=1 colspan=1>Invalidc</td><td rowspan=1 colspan=1>Invalidc</td><td rowspan=1 colspan=1>Error indicated in generation of theresult. Specimens should be retested.</td><td rowspan=1 colspan=1>N/A</td></tr></table>

a A diagnostic interpretation should not be made from a "Not Detected" result for serum or plasma specimens that have been diluted. Obtain a new, undiluted specimen and retest. b Value is truncated two decimal places c Invalid results are displayed in blue-colored font. d As per CDC HCV treatment guidelines, repeat HCV RNA testing is recommended after three months for patients exposed to HCV infection within 6 months or patients with clinical evidence of HCV infection. e Serum and Plasma specimens intended for viral load assessment with value above the ULoQ may be diluted and retested to determine a quantitative result within the linear range.

# C Instrument Description Information:

1. Instrument Name: Panther System

2. Specimen Identification:

A barcode reader reads the barcodes assigned to the specimen.

3. Specimen Sampling and Handling:

Fresh and frozen human serum and plasma from HCV-infected individuals. Plasma may be prepared in ethylenediaminetetraacetic acid (EDTA), anticoagulant citrate dextrose (ACD) solution, and plasma preparation tubes (PPT). Serum may be prepared in serum tubes and serum separator tubes (SST).

# 4. Calibration:

To generate valid results, an assay calibration must be completed. A single positive calibrator is run in triplicate each time a reagent kit is loaded on the Panther system. Once established, the calibration is valid for up to 24 hours. Software on the Panther system alerts the operator when a calibration is required. The operator scans a calibration coefficient found on the Master Lot Barcode Sheet provided with each reagent kit.

During processing, criteria for acceptance of the calibrator are automatically verified by the software on the Panther system. If less than two of the calibrator replicates are valid, the software automatically invalidates the run. Samples in an invalidated run must be retested using a freshly prepared calibrator and freshly prepared controls.

The Panther system automatically determines the concentration of HCV RNA for specimens and controls by comparing the results to a calibration curve. HCV RNA concentrations are reported in $\mathrm { I U / m L }$ and $\mathrm { l o g l 0 I U / m L }$ .

5. Quality Control:

# Negative and Positive Controls:

To generate valid results, a set of assay controls must be tested. One replicate of the negative control, the low positive control, and the high positive control must be tested each time a reagent kit is loaded on the Panther system. Once established, the controls are valid for up to 24 hours. Software on the Panther system alerts the operator when controls are required. During processing, criteria for acceptance of controls are automatically verified by software on the Panther system. To generate valid results, the negative control must give a result of “Not Detected” and the positive controls must give results within predefined parameters (LPC Nominal Target: $2 . 3 6 \mathrm { L o g 1 0 \mathrm { I U } / m L }$ , HPC Nominal Target: $5 . 3 6 \mathrm { L o g 1 0 \mathrm { I U / m L } }$ ). If any one of the controls has an invalid result, the software automatically invalidates the run. Samples in an invalidated run must be retested using a freshly prepared calibrator and freshly prepared controls.

# Internal Calibrator/Internal Control:

Each sample contains an internal calibrator/internal control (IC). During processing, IC acceptance criteria are automatically verified by the Panther system software. If an IC result is invalid, the sample result is invalidated. Every sample with an invalid IC result must be retested to obtain a valid result.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Aptima HCV Quant Dx assay   
B Predicate 510(k) Number(s): P160023

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp;PredicateDevice(s):</td><td colspan="1" rowspan="1">K233352</td><td colspan="1" rowspan="1">P160023</td></tr><tr><td colspan="1" rowspan="1">Device TradeName</td><td colspan="1" rowspan="1">Aptima HCV Quant Dx assay</td><td colspan="1" rowspan="1">Aptima HCV Quant Dx assay</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">The Aptima HCV Quant Dx assay isa real-time transcription-mediatedamplification (TMA) test used forboth detection and quantitation ofhepatitis C virus (HCV) RNA infresh and frozen human serum andplasma from HCV-infectedindividuals.Plasma may be prepared inethylenediaminetetraacetic acid(EDTA), anticoagulant citratedextrose (ACD) solution, and plasmapreparation tubes (PPT). Serum maybe prepared in serum tubes andserum separator tubes (SST).Specimens are tested using thePanther system for automatedspecimen processing, amplification,detection, and quantitation.Specimens containing HCVgenotypes 1 to 6 are validated fordetection and quantitation in theassay.The Aptima HCV Quant Dx assay isindicated for use as an aid in thediagnosis of active HCV infection inthe following populations:individuals with antibody evidence ofHCV infection with evidence of liver</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">disease, individuals suspected to beactively infected with HCV antibodyevidence, and individuals at risk forHCV infection with antibodies toHCV. Detection of HCV RNAindicates that the virus is replicatingand, therefore, is evidence of activeinfection. Detection of HCV RNAdoes not discriminate between acuteand chronic states of infection.The Aptima HCV Quant Dx assay isalso indicated for use as an aid in themanagement of HCV infectedpatients undergoing HCV antiviraldrug therapy. The assay can be usedto measure HCV RNA levelsperiodically prior to, during, andafter treatment to determine sustainedvirological response (SVR) ornonsustained virological response(NSVR). Assay performancecharacteristics have been establishedfor individuals infected with HCVand treated with certain direct-actingantiviral agents (DAA) regimens. Noinformation is available on theassay's predictive value when othertherapies are used. The results fromthe Aptima HCV Quant Dx assaymust be interpreted within thecontext of all relevant clinical andlaboratory findings.The Aptima HCV Quant Dx assay isnot approved for use as a screeningtest for the presence of HCV RNA inblood or blood products.</td></tr><tr><td colspan="1" rowspan="1">TechnologyPrinciple ofOperation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Target Capture (TC), Transcription-Mediated Amplification (TMA)</td></tr><tr><td colspan="1" rowspan="1">Platform</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Automated Panther System</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Hepatitis C RNA</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative &amp; quantitative</td></tr><tr><td colspan="1" rowspan="1">Function</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Detection of RNA from Hepatitis C</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Reagents can beloaded onto thePanther system</td><td colspan="1" rowspan="1">8 times</td><td colspan="1" rowspan="1">5 times</td></tr><tr><td colspan="1" rowspan="1">Results impactedby faulty orflickering LED inPanther instrument</td><td colspan="1" rowspan="1">Marked as invalid results</td><td colspan="1" rowspan="1">Read as normal, potentiallyproducing incorrect results</td></tr></table>

# VI Standards/Guidance Documents Referenced:

No standards, special controls or guidance documents were referenced.

# VII Performance Characteristics (if/when applicable):

NOTE: Performance Characteristics of the Aptima HCV Quant Assay other than the analytical studies described below have been described and established in P160023.

# A Analytical Performance:

1. Precision/Reproducibility:

Refer to the original PMA submission P160023

2. Linearity: Refer to the original PMA submission P160023

3. Analytical Specificity/Interference:

Refer to the original PMA submission P160023

4. Assay Reportable Range:

Not applicable.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Estimated Background Process Control Limit Evaluation

The purpose of this study was to establish background process control limits necessary to detect faults such as flickering LED lamps or high background caused by contamination. LIS and fluorescent curve data were obtained from a customer field database. Curve files were processed to generate results processing output; estimated background data from the results processing output was matched with each test order in the LIS files. HCV Estimated Background distributions were analyzed by each channel. The estimated background limits did not increase the number of invalidated results.

# Background Limits System Validation

The purpose of this study was to demonstrate that the Panther System can perform the assay per predefined acceptance criteria and that the assay performance with the revised software version is equivalent to the performance demonstrated with previous software versions. Performance was evaluated using HCV negative plasma and clinical virus spiked into plasma at 3x LLOQ $( { \sim } 3 0 \mathrm { { I U / m L } ) }$ . The results show that all 10 replicates for the negative sample were negative and all 10 replicates for the low positive sample were positive.

# On-Board Stability (OBS) and reagent Cycling:

The purpose of this study was to evaluate the Aptima HCV Quant Assay for an increase in the number of reagent on-board/ off-board cycling to allow operators to load reagents onto the Panther System 8 times instead of 5 times. Performance was evaluated using negative clinical matrix and virus spiked into clinical plasma at the following concentrations: $3 0 \ \mathrm { I U / m L }$ (3x LLoQ), 2E3 IU/mL, and 1E7 IU/mL. The results support on-board/off-board cycling claim of 8 times.

# Software Verification and Validation

Software validation testing of the functionality and feature set incorporated in the Panther and Panther Fusion System Software and associated assays was performed to ensure the software conforms to user needs and intended uses, and that the requirements implemented through software are consistently fulfilled. All verification testing passed meeting all the requirements.

6. Detection Limit: Not applicable

7. Assay Cut-Off: Refer to the original PMA submission P160023

8. Accuracy (Instrument): Not applicable

9. Carry-Over:

Refer to the original PMA submission P160023

# B Comparison Studies:

1. Method Comparison with Predicate Device: Refer to the original PMA submission P160023.

# 2. Matrix Comparison:

Refer to the original PMA submission P160023.

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable

# D Clinical Cut-Off:

# E Expected Values/Reference Range:

# VIII Proposed Labeling:

The labeling is sufficient and satisfies the requirements of 21 CFR Part 809.10.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.