Luminex Corporation   
Tara Viviani   
Sr. Director Molecular Regulatory Affairs   
4088 Commercial Avenue   
Northbrook, Illinois 60062

Re: K233410 Trade/Device Name: LIAISON PLEX Respiratory Flex Assay Regulation Number: 21 CFR 866.3981 Regulation Name: Device To Detect And Identify Nucleic Acid Targets In Respiratory Specimens From Microbial Agents That Cause The SARS-Cov-2 Respiratory Infection And Other Microbial Agents When In A Multi-Target Test Regulatory Class: Class II Product Code: QOF, OEM, OOU, OTG, OZE, OZX, OZY, OZZ, OCC, NSU Dated: October 6, 2023 Received: October 6, 2023

Dear Tara Viviani:

We have reviewed your section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Additional information about changes that may require a new premarket notification are provided in the FDA guidance documents entitled "Deciding When to Submit a 510(k) for a Change to an Existing Device"

(https://www.fda.gov/media/99812/download) and "Deciding When to Submit a 510(k) for a Software Change to an Existing Device" (https://www.fda.gov/media/99785/download).

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

# Joseph Briggs -S

Joseph Briggs, Ph.D.   
Deputy Branch Chief   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K233410

Device Name LIAISON PLEX Respiratory Flex Assay

Indications for Use (Describe) The LIAISON PLEX Respiratory Flex (RSP Flex) Assay is a multiplexed qualitative test for the simultaneous in vitro detection and identification of multiple bacterial and viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of respiratory tract infection, including SARS-CoV-2. The test is performed on the automated LIAISON PLEX System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific nucleic acid gene sequences of the following organism types and subtypes:

Viruses:   
Adenovirus   
Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated)   
Human Enterovirus/Rhinovirus (not differentiated)   
Human Metapneumovirus,   
Influenza A Influenza A (subtype H1) Influenza A (subtype H3)   
Influenza B   
Parainfluenza 1   
Parainfluenza 2   
Parainfluenza 3   
Parainfluenza 4   
Respiratory Syncytial Virus   
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2)

Bacteria: Bordetella holmesii Bordetella parapertussis Bordetella pertussis Chlamydia pneumoniae Mycoplasma pneumoniae

Nucleic acids from the bacterial and viral organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. Detecting and identifying specific bacterial and viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory infection, if used in conjunction with other clinical, epidemiological, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or patient management decisions.

Negative results in the presence of a respiratory illness may be due to infection with pathogens that are not detected by this test or due to lower respiratory tract infection that is not detected by an NPS specimen. Conversely, positive results do not rule out infection or co-infection with organisms not detected by the LIAISON PLEX Respiratory Flex (RSP Flex) Assay. The agent(s) detected may not be the definite cause of disease.

The use of additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography), may be necessary when evaluating a patient with possible respiratory tract infection.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# O NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# Diasorin

# 510(k) Summary

This Summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

Preparation Date: 21‐February‐2024

A. 510(k) Number: K233410   
B. Purpose for Submission: Traditional 510(k), New Device   
C. Measurand:

Adenovirus, Bordetella holmesii, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated), Human Enterovirus/Rhinovirus (not differentiated), Human Metapneumovirus, Influenza A, Influenza A (subtype H1), Influenza A (subtype H3), Influenza B, Mycoplasma pneumoniae, Parainfluenza 1, Parainfluenza 2, Parainfluenza 3, Parainfluenza 4, Respiratory Syncytial Virus (RSV), and Severe Acute Respiratory Syndrome Coronavirus (SARS‐CoV‐2) nucleic acid target sequences

# D. Type of Test:

Qualitative Multiplexed Nucleic Acid Test that Utilizes Reverse Transcription, Real Time Polymerase Chain Reaction (PCR), and Array Hybridization.

# E. Applicant:

Tara Viviani,   
Luminex Corporation   
4088 Commercial Avenue   
Northbrook, IL 60062   
(847) 400‐9000

# F. Proprietary and Established Names:

LIAISON PLEX® Respiratory Flex Assay

# Diasorin

# G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>PrimaryProductCode</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - Device To Detect AndIdentify Nucleic Acid Targets In RespiratorySpecimens From Microbial Agents That CauseThe SARS-CoV-2 Respiratory Infection AndOther Microbial Agents When In A Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# H. Intended Use:

Intended use(s):

The LIAISON PLEX Respiratory Flex (RSP Flex) Assay is a multiplexed qualitative test for the simultaneous in vitro detection and identification of multiple bacterial and viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of respiratory tract infection, including SARS‐CoV‐2. The test is performed on the automated LIAISON PLEX System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific nucleic acid gene sequences of the following organism types and subtypes:

Viruses:   
Adenovirus   
Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated)   
Human Enterovirus/Rhinovirus (not differentiated)   
Human Metapneumovirus,   
Influenza A Influenza A (subtype H1) Influenza A (subtype H3)   
Influenza B   
Parainfluenza 1   
Parainfluenza 2   
Parainfluenza 3   
Parainfluenza 4   
Respiratory Syncytial Virus   
Severe Acute Respiratory Syndrome Coronavirus (SARS‐CoV‐2)

# Bacteria:

Bordetella holmesii Bordetella parapertussis Bordetella pertussis Chlamydia pneumoniae Mycoplasma pneumoniae

# Diasorin

Nucleic acids from the bacterial and viral organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. Detecting and identifying specific bacterial and viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory infection, if used in conjunction with other clinical, epidemiological, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or patient management decisions.

Negative results in the presence of a respiratory illness may be due to infection with pathogens that are not detected by this test or due to lower respiratory tract infection that is not detected by an NPS specimen. Conversely, positive results do not rule out infection or co‐ infection with organisms not detected by the LIAISON PLEX Respiratory Flex (RSP Flex) Assay. The agent(s) detected may not be the definite cause of disease.

The use of additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography), may be necessary when evaluating a patient with possible respiratory tract infection.

Same as intended use.

Special conditions for use statement(s):

For prescription use only.

For in vitro diagnostic use only

Special instrument requirements:

For use with LIAISON PLEX Systems only

# I. Device Description:

The LIAISON $\mathsf { P l E X } ^ { \mathbb { \exp } }$ Respiratory Flex Assay is a multiplexed nucleic acid test system composed of the LIAISON PLEX Instrument, the LIAISON $\mathsf { P l E X } ^ { \otimes }$ System Software (preinstalled on the LIAISON PLEX Instrument), the LIAISON PLEX® Respiratory Flex Assay cartridge, and the LIAISON PLEX® Respiratory Flex Assay File. The LIAISON PLEX® Respiratory Flex Assay cartridge contains the reagents to perform nucleic acid extraction and purification, reverse transcription, PCR, and array hybridization. Specifically, the LIAISON $\mathsf { P l E X } ^ { \mathbb { \exp } }$ Respiratory Flex Assay detects bacteria and viruses from nasopharyngeal swab (NPS) specimens collected from individuals with signs and symptoms of respiratory infection.

The LIAISON PLEX System consists of a touchscreen user interface that includes the software for running and analyzing assay results, one to six processing/imaging LIAISON PLEX modules, and a handheld barcode reader. Each LIAISON PLEX module processes one sample at a time under the control of the LIAISON PLEX System software.

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission

LIAISON $\mathsf { P L E X } ^ { \otimes }$ automates the sample processing through analysis within a single cartridge. Processing steps include 1.) Sample Preparation: Nucleic acid extraction from organisms by chemical and mechanical means and isolation of nucleic acid on magnetic beads 2.) Target Amplification: Multiplex PCR and RT‐PCR based amplification of extracted nucleic acid to generate target specific amplicons 3.) Hybridization: Amplicons hybridize with their target specific DNA probe arranged in a microarray format and that are attached to mediator and gold nanoparticles 4.) Analysis: Gold nanoparticles specifically bound to target amplicons are silver enhanced and the light scatter from microarray spot is measured and analyzed to confirm presence (Detected) or absence (not Detected) of a target.

The LIAISON PLEX Respiratory Flex Assay has the option of creating and processing results for custom panels using $F I e x ^ { \circledast }$ Software. Flex Software allows users to randomly select and group targets in tiers for result processing. Up to 7 targets may be selected for the initial test tier. After the first tier, each additional tier requires a specific number of credits. Flex™ credits allow the end‐user to create custom panels and pay for a smaller subset of results tailored to the individual patient's clinical presentation. Alternatively, a laboratory may choose the fixed price option where all target results are processed at the same time.

# J. Substantial Equivalence Information:

Predicate device name(s):

BioFire Respiratory Panel 2.1 (RP2.1)

Predicate 510(k) number(s):

DEN200031

Comparison with predicate:

The following table compares Luminex’s LIAISON PLEX® Respiratory Flex Assay to the BioFire Respiratory Panel 2.1 (RP2.1) (DEN200031).

Comparison to Predicate

<table><tr><td rowspan=1 colspan=1>Comparison toPredicate Device</td><td rowspan=1 colspan=1>Predicate Device:BioFire Respiratory Panel 2.1 (RP2.1)(DEN200031)</td><td rowspan=1 colspan=1>Candidate Device:LIAISON PLEX® Respiratory FlexAssay</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>QOF</td></tr><tr><td rowspan=1 colspan=1>Regulation Number</td><td rowspan=1 colspan=1>21 CFR 866.3981</td><td rowspan=1 colspan=1>21 CFR 866.3981</td></tr></table>

# Diasorin

<table><tr><td>Comparison to Predicate Device (DEN200031)</td><td>Predicate Device: BioFire Respiratory Panel 2.1 (RP2.1)</td><td>Candidate Device: LIAISON PLEX® Respiratory Flex Assay</td></tr><tr><td>Organisms Detected</td><td>Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2), Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A, including subtypes H1, H1-2009, and H3, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Respiratory Syncytial Virus, Bordetella parapertussis (IS1001), Bordetella pertussis (ptxP), Chlamydia pneumoniae, and Mycoplasma</td><td>Adenovirus, Bordetella holmesii, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Enterovirus/Rhinovirus, Human Metapneumovirus, Influenza A, Influenza A (subtype H1), Influenza A (subtype H3), Influenza B, Mycoplasma pneumoniae, Parainfluenza 1, Parainfluenza 2, Parainfluenza 3, Parainfluenza 4, Respiratory Syncytial Virus, and SARS-CoV-2</td></tr><tr><td>Measurand</td><td>pneumoniae Nucleic acid from Organisms detected</td><td>Nucleic acid from Organisms detected</td></tr><tr><td>Intended Use</td><td>The BioFire Respiratory Panel 2.1 (RP2.1) is a PCR-based multiplexed nucleic acid test intended for use with the BioFire FilmArray 2.0 or BioFire FilmArray Torch systems for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections, including COVID-19. The following organism types and subtypes are identified using the BioFire RP2.1: Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2), Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A, including subtypes H1, H1- 2009, and H3, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2,</td><td>The LIAISON PLEX Respiratory Flex (RSP Flex) Assay is a multiplexed qualitative test for the simultaneous in vitro detection and identification of multiple bacterial and viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of respiratory tract infection, including SARS-CoV-2. The test is performed on the automated LIAISON PLEX System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific nucleic acid gene sequences of the following organism types and subtypes: Viruses: Adenovirus Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated) Human Enterovirus/Rhinovirus (not differentiated Human Metapneumovirus, Influenza A Influenza A (subtype H1) Influenza A (subtype H3)</td></tr></table>

# Diasorin

<table><tr><td rowspan=1 colspan=1>Comparison toPredicate Device</td><td rowspan=1 colspan=1>Predicate Device:BioFire Respiratory Panel 2.1 (RP2.1)(DEN200031)</td><td rowspan=1 colspan=1>Candidate Device:LIAISON PLEX® Respiratory FlexAssay</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Bordetella pertussis (ptxP),Chlamydia pneumoniae, andMycoplasma pneumoniaeNucleic acids from the respiratory viraland bacterial organisms identified by thistest are generally detectable in NPSspecimens during the acute phase ofinfection. The detection and identificationof specific viral and bacterial nucleic acidsfrom individuals exhibiting signs and/orsymptoms of respiratory infection isindicative of the presence of the identifiedmicroorganism and aids in the diagnosisof respiratory infection if used inconjunction with other clinical andepidemiological information. The results ofthis test should not be used as the solebasis for diagnosis, treatment, or otherpatient management decisions.Negative results in the setting of arespiratory illness may be due toinfection with pathogens that are notdetected by this test, or lower respiratorytract infection that may not be detectedby an NPS specimen. Positive results donot rule out coinfection with otherorganisms. The agent(s) detected by theBioFire RP2.1 may not be the definitecause of disease. Additional laboratorytesting (e.g. bacterial and viral culture,immunofluorescence, and radiography)may be necessary when evaluating apatient with possible respiratory tractinfection.</td><td rowspan=1 colspan=1>Bacteria:Bordetella holmesiiBordetella parapertussisBordetella pertussisChlamydia pneumoniaeMycoplasma pneumoniaeNucleic acids from the bacterial andviral organisms identified by this testare generally detectable in NPSspecimens during the acute phase ofinfection. Detecting and identifyingspecific bacterial and viral nucleicacids from individuals exhibiting signsand symptoms of respiratory infectionaids in the diagnosis of respiratoryinfection, if used in conjunction withother clinical, epidemiological, andlaboratory findings. The results of thistest should not be used as the solebasis for diagnosis, treatment, orpatient management decisions.Negative results in the presence of arespiratory illness may be due toinfection with pathogens that are notdetected by this test or due to lowerrespiratory tract infection that is notdetected by an NPS specimen.Conversely, positive results do not ruleout infection or co-infection withorganisms not detected by theLIAISON PLEX Respiratory Flex (RSPFlex) Assay. The agent(s) detectedmay not be the definite cause ofdisease.The use of additional laboratory testing(e.g., bacterial and viral culture,immunofluorescence, and radiography),may be necessary when evaluating apatient with possible respiratory tractinfection.</td></tr><tr><td rowspan=1 colspan=1>Automated System(Sample to Answer)</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>BioFire® FilmArray® 2.0 or BioFire®FilmArray® Torch Systems</td><td rowspan=1 colspan=1>LIAISON PLEX®</td></tr><tr><td rowspan=1 colspan=1>Sample Types</td><td rowspan=1 colspan=1>Nasopharyngeal Swab (NPS) Specimens</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technological Principles</td><td rowspan=1 colspan=1>Highly multiplexed nested nucleic acidamplification with melt analysis.</td><td rowspan=1 colspan=1>Highly multiplexed nucleic acid PCRand RT-PCR test with microarraydetection</td></tr><tr><td rowspan=1 colspan=1>Internal Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagentpouch to control for sample processing</td><td rowspan=1 colspan=1>Multiple internal controls contained inthe cartridge monitor sampleprocessing and RT and PCR</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Comparison toPredicate Device</td><td rowspan=1 colspan=1>Predicate Device:BioFire Respiratory Panel 2.1 (RP2.1)(DEN200031)</td><td rowspan=1 colspan=1>Candidate Device:LIAISON PLEX® Respiratory FlexAssay</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>and both stages of PCR and melt analysis.</td><td rowspan=1 colspan=1>functions.</td></tr><tr><td rowspan=1 colspan=1>Bordetella SpeciesDetected</td><td rowspan=1 colspan=1>Bordetella parapertussisBordetella pertussis</td><td rowspan=1 colspan=1>•  Bordetella parapertussisBordetella pertussis•   Bordetella holmesii</td></tr><tr><td rowspan=1 colspan=1>Human CoronavirusResult Reporting</td><td rowspan=1 colspan=1>Each target human coronavirus species(i.e., HKU1, OC43, 229E, NL63) isreported independently.</td><td rowspan=1 colspan=1>The human coronavirus target species(i.e., HKU1, OC43, 229E, NL63) arenot differentiated.</td></tr><tr><td rowspan=1 colspan=1>Influenza A Subtyping</td><td rowspan=1 colspan=1>Influenza A subtypes H1, H1-2009, and H3detected/reported.</td><td rowspan=1 colspan=1>Influenza A subtypes H1 and H3detected/reported.</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>~45 minutes</td><td rowspan=1 colspan=1>~2 hours</td></tr></table>

# K. Standards/Guidance Documents Referenced:

# Standards

 CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline ‐ Second Edition. CLSI document EP12‐A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.   
 CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25‐A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.   
 CLSI. Interference Testing in Clinical Chemistry. 3rd ed. CLSI guideline EP07. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.   
 ISO 14971:2019 Medical devices ‐ Application of risk management to medical devices   
 IEC 62366‐1:2015 Medical devices ‐ Part 1: Application of usability engineering to medical devices   
 ISO 62304:2006 Medical device software ‐ Software life‐cycle processes ISO 15223‐1:2016: Medical Devices ‐ Symbols to be used with medical device labels, labeling and information to be supplied ‐ Part 1: General requirements IEC 61010‐1 Ed. 3.0 2010: Safety requirements for electrical equipment for measurement, control, and laboratory use ‐ Part 1: General requirements EN 61010‐2‐101:2002/IEC 61010‐2‐101:2015: Safety requirements for electrical equipment for measurement, control and laboratory use ‐ Part 2‐101: Particular requirements for in vitro diagnostic (IVD) medical equipment. IEC 60601‐1‐2:2014 (Edition 4.0): Medical electrical equipment ‐ Part 1‐2: General requirements for basic safety and essential performance ‐ Collateral Standard: Electromagnetic disturbances ‐ Requirements and tests ISO 13485:2016/EN ISO 13485:2016; Medical devices ‐ Quality Management System ‐ Requirements for regulatory purposes ISO 20916:2019; In vitro diagnostic medical devices. Clinical performance studies using specimens from human subjects. Good study practice EN ISO 18113‐1:2011; In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling). Terms, definition and general requirements EN ISO 18113‐2:2011; In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission

 EN ISO 18113‐3:2011; In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling) – Part 3: In vitro diagnostic instruments for professional use EN ISO 23640:2015; In vitro diagnostic medical devices ‐ Evaluation of stability of in vitro IEC 61326‐1:2012; Electrical equipment for measurement control and laboratory use ‐ EMC requirements ‐ Part 1: General requirements   
 EN 61326‐2‐6:2006/IEC 61326‐2‐6:2012; Electrical equipment for measurement control and laboratory use ‐ EMC requirements ‐ Part 2‐6: Particular requirements ‐ In vitro diagnostic (IVD) medical equipment

# Special Controls

Class II Special Controls as per 21 CFR 866.3981

# Guidance Documents

Electronic Submission Template for Medical Device 510(k) Submissions ‐ Guidance for Industry and Food and Drug Administration Staff (October 2, 2023).   
Respiratory Viral Panel Multiplex Nucleic Acid Assay ‐ Class II Special Controls Guidance for Industry and FDA Staff (October 9, 2009).   
Content of Premarket Submissions for Device Software Functions ‐ Guidance for Industry and Food and Drug Administration Staff (June 14, 2023).   
Cybersecurity in Medical Devices: Quality System Considerations and Content of Premarket Submissions ‐ Guidance for Industry and Food and Drug Administration Staff (September 23, 2023).   
Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests ‐ Guidance for Industry and FDA Staff (March 13, 2007).

# L. Test Principle:

The LIAISON $\mathsf { P l E X } ^ { \otimes }$ Respiratory Flex Assay is performed on nasopharyngeal swab (NPS) specimens collected in Copan Universal Transport Medium™ or BD™ Universal Viral Transport Media. The system consists of an instrument and a single‐use, disposable test cartridge and a transfer pipette. The user loads a portion of the sample into the sample port of the LIAISON PLEX Respiratory Flex Assay Cartridge. Next, the user sets up the sample order on the LIAISON PLEX System by first entering the sample information or scanning the barcode ID located on the sample tube, then scanning the barcode ID located on the test cartridge. Last, the user inserts the test cartridge into the processing module to initiate the test. The LIAISON PLEX System identifies the assay being run and automatically initiates the proper testing protocol to process the sample, analyze the data, and generate test results.

The LIAISON PLEX System automates the LIAISON PLEX Respiratory Flex Assay sample analysis through the following steps: a) Sample Preparation: Nucleic acid extraction via mechanical and chemical cell lysis and magnetic bead‐ based nucleic acid isolation of prepared specimens obtained from patients; b) Target Amplification: Multiplex PCR‐ and RT‐PCR‐based amplification of the extracted nucleic acids to generate target‐specific amplicons; c) Hybridization: Amplicons hybridize to target‐specific capture DNA on a microarray format, and target‐specific mediator and gold nanoparticle probes hybridize to captured amplicons; d) Signal Analysis: Gold nanoparticle probes bound specifically to target‐containing spots in the microarray are silver‐enhanced, and light scatter from the spots is measured and further

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission

analyzed to determine the presence (Detected) or absence (Not Detected) of a target.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

Within Laboratory Precision

Within laboratory precision of the LIAISON PLEX Respiratory Flex Assay was evaluated by testing three lots of LIAISON PLEX Respiratory Flex Assay cartridges at a single site over five non‐consecutive days. Three target concentrations were prepared and tested. Targets consisted of a negative sample and a positive sample comprised of five targets (B. pertussis, Adenovirus, influenza B, human metapneumovirus, and SARS‐CoV‐2). All positive samples were diluted in a simulated NPS matrix to a low positive concentration (1.5X LoD) and a moderate positive concentration (5X LoD). Targets were randomized and blinded to the operators in an order that each operator tested each target (negative, 1.5X LoD, and 5X LoD) in triplicate on each of the testing days. Qualitative results of the within laboratory precision study are summarized in Table 1.

Table 1 ‐ Within Laboratory Precision   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Panel Concentration</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>% Agreement withExpected Results(95% CI)</td></tr><tr><td rowspan=3 colspan=1>Bordetellapertussis</td><td rowspan=1 colspan=1>Low Positive (1.5X LoD)</td><td rowspan=1 colspan=1>93.3% (42/45)</td><td rowspan=1 colspan=1>93.3% (82.1-97.7%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (5X LoD)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Low Positive (1.5X LoD)</td><td rowspan=1 colspan=1>97.8% (44/45)</td><td rowspan=1 colspan=1>97.8%(88.4-99.6%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (5X LoD)</td><td rowspan=1 colspan=1>97.8% (44/45)</td><td rowspan=1 colspan=1>97.8%(88.4-99.6%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Low Positive (1.5X LoD)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (5X LoD)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>hMPV</td><td rowspan=1 colspan=1>Low Positive (1.5X LoD)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (5X LoD)</td><td rowspan=1 colspan=1>97.8% (44/45)</td><td rowspan=1 colspan=1>97.8%(88.4-99.6%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Low Positive (1.5X LoD)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (5X LoD)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr></table>

Note: Results are shown only for the intended targets. Panel members co‐spiked with 5 different targets are tested in triplicate across 3 lots and tested over 5 non‐consecutive days (45 total replicates).

# Reproducibility

Reproducibility of the LIAISON PLEX Respiratory Flex Assay was evaluated by testing one lot of LIAISON PLEX Respiratory Flex Assay cartridges with two operators at each of three sites over five non‐consecutive days. Three target concentrations were prepared and tested across all

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission sites and operators to evaluate site‐to‐site reproducibility. Targets consisted of a negative sample and a positive sample comprised of five assay targets (B. pertussis, Adenovirus, influenza B, human metapneumovirus, and SARS‐CoV‐2). All positive samples were diluted in a simulated NPS matrix to a low positive concentration (1.5X LoD) and a moderate positive concentration (5X LoD). Targets were randomized and blinded to the operators in an order that each operator tested each target (negative, 1.5X LoD, and 5X LoD) in triplicate on each of the testing days. Qualitative results of the reproducibility study are summarized in Table 2.

Table 2. Reproducibility Results   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>TargetConcentration</td><td rowspan=1 colspan=4>% Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites(95% Confidence)</td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Low Positive(1.5X LoD)</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>97.8% (88/90)(92.3% - 99.4%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive(5X LoD)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100% (90/90)(95.9% - 100%)</td></tr><tr><td rowspan=2 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Low Positive(1.5X LoD)</td><td rowspan=1 colspan=1>93.3%(28/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>96.7% (87/90)(90.7% - 98.9%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive(5X LoD)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100% (90/90)(95.9% - 100%)</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Low Positive(1.5X LoD)</td><td rowspan=1 colspan=1>93.3%(28/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>96.7% (87/90)(90.7% - 98.9%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive(5X LoD)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100% (90/90)(95.9% - 100%)</td></tr><tr><td rowspan=2 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Low Positive(1.5X LoD)</td><td rowspan=1 colspan=1>90.0%(27/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>93.3%(28/30)</td><td rowspan=1 colspan=1>94.4% (85/90)(87.6% - 97.6%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive(5X LoD)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>93.3%(28/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>97.8% (88/90)(92.3% - 99.4%)</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Low Positive(1.5X LoD)</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>98.9% (89/90)(93.9% - 99.8%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive(5X LoD)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100% (90/90)(95.9% - 100%)</td></tr><tr><td rowspan=1 colspan=1>Negative NPS</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100% (90/90)(95.9% - 100%)</td></tr></table>

b. Linearity/assay reportable range:

Not applicable. The LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Respiratory Flex Assay is a qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls:

Each LIAISON PLEX Respiratory Flex Assay cartridge includes internal controls (extraction control, amplification control, and hybridization control) to ensure performance of sample preparation, amplification, and detection. Extraction control is automatically added to the sample prior to initiation of sample preparation and assesses extraction, nucleic acid recovery, amplification of RNA targets, and detection. Additionally, an amplification control present in the lyophilized PCR

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission master mix, serves as an independent amplification and detection control for DNA targets. Finally, a post‐amplification hybridization control serves as an indicator of successful hybridization. Internal control results are reported as Pass, Fail, or N/A on the printed reports (see Table 3 for detailed explanations of each control result). Internal controls must either (1) generate a signal above threshold in each internal reaction for the system to report a valid test result, or (2) the amplification or extraction control result can be below the signal threshold if a DNA or RNA target pathogen is detected, respectively.

If the Test Result is $^ { \prime \prime } \mathsf { N o } \mathsf { C a l l ^ { \prime \prime } }$ for reasons other than failure of internal controls, the Internal Control Result is reported as $" \mathsf { N } / \mathsf { A } ^ { \prime \prime }$ and the user should repeat the test with a new cartridge. For additional assistance regarding assay failures unrelated to internal controls, please refer to Chapter 8 (Troubleshooting Unexpected Results/Failures) of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ System User Manual.

Table 3. Interpretation of Controls on the LIAISON PLEX Respiratory Flex Assay Report   

<table><tr><td>Internal Control Result</td><td>Explanation</td><td>Suggested Action</td></tr><tr><td>Pass</td><td>The hybridization control was detected, indicating successful hybridization. The amplification control was detected, indicating successful amplification. The extraction control was detected, indicating successful</td><td>Review and report results</td></tr><tr><td>N/A</td><td>extraction. The hybridization control was detected, indicating successful hybridization. A DNA pathogen target was detected, indicating successful amplification. If a DNA pathogen target is detected, the amplification control result is ignored. The extraction control was detected, indicating successful extraction.</td><td>Review and report results</td></tr><tr><td>N/A</td><td>The hybridization control was detected, indicating successful hybridization. The amplification control was detected, indicating successful amplification. A RNA pathogen target was detected, indicating successful extraction. If a RNA pathogen target is detected, the extraction control result is ignored.</td><td>Review and report results</td></tr><tr><td>Fail</td><td>The hybridization control was not detected indicating hybridization was not successful. The amplification control, or a DNA pathogen was detected, indicating successful amplification. The extraction control, or a RNA pathogen was detected, indicating successful extraction.</td><td>Repeat test with a new cartridge</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td>Internal Control Result</td><td>Explanation</td><td>Suggested Action</td></tr><tr><td>Fail</td><td>The hybridization control was detected indicating successful hybridization. The amplification control, or a DNA pathogen was not detected, indicating amplification was not successful. The extraction control, or a RNA pathogen was detected, indicating successful extraction.</td><td>Repeat test with a new cartridge</td></tr><tr><td>Fail</td><td>The hybridization control was detected indicating successful hybridization. The amplification control, or a DNA pathogen was detected, indicating successful amplification. The extraction control, or a RNA pathogen, was not detected, indicating extraction was not successful.</td><td>Repeat test with a new cartridge</td></tr></table>

# External Controls

Positive and negative external controls should be tested with each new lot or shipment of reagents, or monthly, (whichever occurs first), or in accordance with updated local, regional, state, and/or federal guidelines. Positive and negative external controls are not provided with the LIAISON PLEX Respiratory Flex Assay. Verified negative nasopharyngeal swab (NPS) specimens can be used as the negative control. Previously characterized positive samples or verified negative NPS specimens spiked with well characterized organisms may be used as the external positive control. External controls should be used in accordance with laboratory protocols and in accordance with local, state, and federal accrediting organizations, as applicable.

# Stability:

# Specimen Stability

Contrived specimen stability at room temperature $( 1 5 ^ { \circ } \mathsf { C } - 3 0 ^ { \circ } \mathsf { C } )$ , refrigerated $( 2 ^ { \circ } \mathsf { C } - 8 ^ { \circ } \mathsf { C } )$ , and frozen $( \le - 7 0 ^ { \circ } \mathsf { C } )$ storage was evaluated for use with the LIAISON PLEX® Respiratory Flex Assay. A representative panel of 5 Respiratory Flex target organisms (i.e., Bordetella pertussis, adenovirus, influenza B, hMPV, and SARS‐CoV‐2) was co‐spiked into negative clinical NPS matrix at three concentrations ‐ a low positive (2x LoD) sample, a moderate positive (5x LoD) sample, as well as the negative clinical matrix independently. Testing occurred at baseline and various time points up to 36 days for the frozen storage, up to 80 hours for refrigerated storage, and up to 9 hours for room temperature storage.

The results of this study demonstrated that specimens stored frozen $\left( \leq - 7 0 ^ { \circ } \mathsf { C } \right)$ are stable for up to 30 days, specimens stored refrigerated $( 2 ^ { \circ } \mathsf { C } - 8 ^ { \circ } \mathsf { C } )$ are stable for up to 72 hours, and specimens stored at room temperature $\big ( 1 5 ^ { \circ } \mathsf { C } - 3 0 ^ { \circ } \mathsf { C } \big )$ are stable for up to 8 hours.

Fresh vs. Frozen Specimen Stability

Performance of the LIAISON $\mathsf { P l E X } ^ { \otimes }$ Respiratory Flex Assay was assessed using contrived specimens tested fresh (i.e. unfrozen) and specimens tested frozen (stored at $\leq - 7 0 ^ { \circ } \mathsf { C } )$ .

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission

The effect of repeated freeze/thaw cycles was also assessed between freshly prepared contrived specimens and those that had undergone 1, 2, and 3 freeze/thaw cycles. Four contrived sample panels were prepared by co‐spiking 5‐6 targets into clinical negative NPS matrix at three concentrations ‐ a low positive (2x LoD) sample, a moderate positive (5x LoD) sample, as well as the negative clinical matrix independently (see Table 4).

Table 4. Positive Sample Panels Evaluated in the Freeze/Thaw Study   
Table 5. LIAISON PLEX Respiratory Flex Assay Target Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=5 colspan=1>A</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=5 colspan=1>C</td><td rowspan=1 colspan=1>Bordetella parapertussis</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV A</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Parainfluenza 2</td></tr><tr><td rowspan=1 colspan=1>hMPV (A2)</td><td rowspan=1 colspan=1>Influenza A (subtype H3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>hMPV (B2)</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=5 colspan=1>D</td><td rowspan=1 colspan=1>RSV B</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Parainfluenza 1</td></tr><tr><td rowspan=1 colspan=1>Influenza A (subtypeH1)</td><td rowspan=1 colspan=1>Human coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Human coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus A</td><td rowspan=1 colspan=2></td></tr></table>

Positive panels spiked at 2x were tested in replicates of 40 at T0 (fresh) and 20 replicates following 1, 2, and 3 freeze/thaw (F/T) cycles after storage at $\boldsymbol { \mathbf { \ell } } - 7 0 ^ { \circ } \mathsf { C }$ . Positive panels spiked at 5x LoD and the negative sample were tested in replicates of 10 at T0 (fresh) and following 1, 2, and 3 F/T cycles after storage at $\scriptscriptstyle - 7 0 ^ { \circ } \mathsf C$ .

The results of the study support that NPS specimens in UVT/UTM can undergo up to two freeze/thaw cycles prior to testing with the LIAISON PLEX Respiratory Flex Assay.

# d. Detection Limit:

A limit of detection study (LoD) was performed to evaluate the analytical sensitivity of the LIAISON PLEX RSP Flex Assay. Thirty‐nine (39) strains and isolates that represent the 19 reportable targets of the LIAISON PLEX RSP Flex Assay were tested individually by serially diluting each target in NPS matrix. Testing was broken into two parts; LoD Determination and LoD Confirmation. The determined LoD concentrations were evaluated using a 3‐fold dilution series and testing of at least six replicates per dilution. The determined LoD for each target was defined as the lowest concentration at which $100 \%$ of six replicates were positive for the intended reportable target. The confirmed LoD was evaluated using a dilution series around the determined LoD and testing of at least 20 replicates was performed at the confirmed LoD and the dilution below the confirmed LoD. The confirmed LoD for each organism was defined as the lowest concentration at which $\geq 9 5 \%$ of the 20 replicates were positive for the intended reportable target. The confirmed LoD for each target tested is listed in Table 5. The LoD for co‐ analyte spiked samples was also evaluated and shown to be equivalent to single spiked samples.

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Target Organism</td><td rowspan=1 colspan=2>Strain / Isolate</td><td rowspan=1 colspan=2>Concentration at LoD1</td></tr><tr><td rowspan=1 colspan=5>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=2>F061</td><td rowspan=1 colspan=1>7.29E+03 copies/mL</td><td rowspan=1 colspan=1>8.68E+01 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=2>18323 NCTC 10739</td><td rowspan=1 colspan=1>3.80E+03 copies/mL</td><td rowspan=1 colspan=1>1.98E+03 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=2>C510</td><td rowspan=1 colspan=1>7.90E+02 copies/mL</td><td rowspan=1 colspan=1>2.06E+01 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=2>CM-1</td><td rowspan=1 colspan=1>5.68E+02 copies/mL</td><td rowspan=1 colspan=1>1.04E+02 IFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=2>M129</td><td rowspan=1 colspan=1>1.30E+03 copies/mL</td><td rowspan=1 colspan=1>4.24E+01 CCU/mL</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Viruses</td></tr><tr><td rowspan=6 colspan=1>Adenovirus(A, B, C, D, E, F)</td><td rowspan=1 colspan=2>Type 31 (A)</td><td rowspan=1 colspan=1>1.76E+03 copies/mL</td><td rowspan=1 colspan=1>1.09E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>Type 3 (B)</td><td rowspan=1 colspan=1>6.86E+02 copies/mL</td><td rowspan=1 colspan=1>1.69E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>Type 1 (C)</td><td rowspan=1 colspan=1>1.12E+03 copies/mL</td><td rowspan=1 colspan=1>8.97E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>Type 26 (D)</td><td rowspan=1 colspan=1>7.48E+02 copies/mL</td><td rowspan=1 colspan=1>1.10E-02 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=2>Type 4 (E)</td><td rowspan=1 colspan=1>3.53E+02 copies/mL</td><td rowspan=1 colspan=1>1.08E-02 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=2>Type 40 (F)</td><td rowspan=1 colspan=1>4.85E+02 copies/mL</td><td rowspan=1 colspan=1>2.29E-02 TCID5o/mL</td></tr><tr><td rowspan=4 colspan=1>Human Coronavirus(HKU1, 229E, NL63,0C43)</td><td rowspan=1 colspan=2>229E</td><td rowspan=1 colspan=1>4.00E+02 copies/mL</td><td rowspan=1 colspan=1>9.15E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>HKU1</td><td rowspan=1 colspan=1>1.67E+03 copies/mL</td><td rowspan=1 colspan=1>N/A{2</td></tr><tr><td rowspan=1 colspan=2>NL63</td><td rowspan=1 colspan=1>7.64E+01 copies/mL</td><td rowspan=1 colspan=1>1.34E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>OC43</td><td rowspan=1 colspan=1>9.48E+03 copies/mL</td><td rowspan=1 colspan=1>9.58E-01 TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=2>(hMPV-9) A1</td><td rowspan=1 colspan=1>2.13E+03 copies/mL</td><td rowspan=1 colspan=1>2.09E-01 TCID5o/mL</td></tr><tr><td rowspan=1 colspan=2>(hMPV-27) A2</td><td rowspan=1 colspan=1>2.04E+03 copies/mL</td><td rowspan=1 colspan=1>2.14E-01 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=2>(hMPV-3) B1</td><td rowspan=1 colspan=1>5.00E+03 copies/mL</td><td rowspan=1 colspan=1>4.31E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>(hMPV-8) B2</td><td rowspan=1 colspan=1>1.50E+04 copies/mL</td><td rowspan=1 colspan=1>1.66E+00 TCID5o/mL</td></tr><tr><td rowspan=4 colspan=1>Influenza A | Influenza A(subtype H1)</td><td rowspan=2 colspan=1>Brisbane/02/18</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1.35E+04 copies/mL</td><td rowspan=1 colspan=1>3.97E+00 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>H1 Subtype</td><td rowspan=1 colspan=1>1.50E+03 copies/mL</td><td rowspan=1 colspan=1>4.41E-01 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Guangdong-Maonan/SWL/1536/19</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1.37E+04 copies/mL</td><td rowspan=1 colspan=1>5.86E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1 Subtype</td><td rowspan=1 colspan=1>1.37E+04 copies/mL</td><td rowspan=1 colspan=1>5.86E+00 TCID50/mL</td></tr><tr><td rowspan=6 colspan=1>Influenza A | Influenza A(subtype H3)</td><td rowspan=2 colspan=1>HongKong/2671/19</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1.59E+05 copies/mL</td><td rowspan=1 colspan=1>1.50E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3 Subtype</td><td rowspan=1 colspan=1>5.30E+04 copies/mL</td><td rowspan=1 colspan=1>4.98E+00 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>A/Kansas/14/2017</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1.96E+03 copies/mL</td><td rowspan=1 colspan=1>5.58E+00 TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>H3 Subtype</td><td rowspan=1 colspan=1>1.96E+03 copies/mL</td><td rowspan=1 colspan=1>5.58E+00 TCIDso/mL</td></tr><tr><td rowspan=2 colspan=1>Singapore/INFUMH-16-0019/16</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>4.55E+03 copies/mL</td><td rowspan=1 colspan=1>1.10E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3 Subtype</td><td rowspan=1 colspan=1>4.55E+03 copies/mL</td><td rowspan=1 colspan=1>1.10E+01 TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>Influenza B</td><td rowspan=1 colspan=2>Alabama/2/17 (Victoria Lineage)</td><td rowspan=1 colspan=1>3.35E+02 copies/mL</td><td rowspan=1 colspan=1>7.30E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=2>Washington/02/19 (Victoria Lineage)</td><td rowspan=1 colspan=1>3.02E+03 copies/mL</td><td rowspan=1 colspan=1>2.79E+01 TCIDs0/mL</td></tr><tr><td rowspan=1 colspan=2>Colorado/6/17 (Victoria Lineage)</td><td rowspan=1 colspan=1>3.02E+03 copies/mL</td><td rowspan=1 colspan=1>6.64E-01 TCID5o0/mL</td></tr><tr><td rowspan=1 colspan=2>Wisconsin/1/10 (Yamagata Lineage)</td><td rowspan=1 colspan=1>1.01E+03 copies/mL</td><td rowspan=1 colspan=1>3.23E-01 TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>7.61E+02 copies/mL</td><td rowspan=1 colspan=1>1.06E+01 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>8.46E+03 copies/mL</td><td rowspan=1 colspan=1>1.55E+01 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>1.93E+03 copies/mL</td><td rowspan=1 colspan=1>3.18E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=2>A</td><td rowspan=1 colspan=1>5.76E+03 copies/mL</td><td rowspan=1 colspan=1>6.65E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus A</td><td rowspan=1 colspan=2>A (2006 Isolate)</td><td rowspan=1 colspan=1>3.83E+03 copies/mL</td><td rowspan=1 colspan=1>1.11E+00 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus B</td><td rowspan=1 colspan=2>B (3/2015 Isolate #1)</td><td rowspan=1 colspan=1>1.61E+04 copies/mL</td><td rowspan=1 colspan=1>7.48E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus / Rhinovirus</td><td rowspan=1 colspan=2>Human Rhinovirus 1A</td><td rowspan=1 colspan=1>8.19E+03 copies/mL</td><td rowspan=1 colspan=1>4.99E-01 TCID50/mL</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Target Organism</td><td rowspan=1 colspan=1>Strain / Isolate</td><td rowspan=1 colspan=2>Concentration at LoD1</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>Human Rhinovirus B14</td><td rowspan=1 colspan=1>8.18E+03 copies/mL</td><td rowspan=1 colspan=1>1.10E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus C1</td><td rowspan=1 colspan=1>1.92E+04 copies/mL</td><td rowspan=1 colspan=1>N/A{2</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusEchovirus Type 6</td><td rowspan=1 colspan=1>2.25E+04 copies/mL</td><td rowspan=1 colspan=1>3.00E+01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>8.00E+03 copies/mL</td><td rowspan=1 colspan=1>4.04E+01 TCID5o/mL</td></tr></table>

1Concentrations in copies/mL were obtained by digital‐droplet PCR. 2Testing for Coronavirus HKU1 and Rhinovirus 1C utilized a clinical specimen due to the lack of availability of a cultured isolate. Viral concentration was determined in RN copies/mL by digital droplet PCR.

# Limit of Detection Testing with the WHO International Standard for SARS‐CoV‐2 (NIBSC, 20/146)

An LoD study was performed to evaluate the analytical sensitivity of the Respiratory Flex Assay with the World Health Organization (WHO) Internal Standard for SARS‐CoV‐2 (Table 6). The WHO International SARS‐CoV‐2 standard was reconstituted then serially diluted in NPS matrix. As with the LoD study described above, testing was broken into two parts: preliminary and confirmatory LoD testing. For the preliminary LoD study, testing at multiple concentrations in triplicate was performed. The preliminary LoD was defined as the lowest concentration at which $100 \%$ of replicates were positive for SARS‐CoV‐2. The confirmed LoD was determined by testing a 3‐fold dilution series of multiple concentrations around the preliminary LoD in replicates of 20. The confirmed LoD was defined as the lowest concentration at which $\geq 9 5 \%$ of the replicates were positive for the intended reportable target. To confirm the LoD, at least one dilution below the LoD was required to result in less than $9 5 \%$ positivity. The confirmed LoD for the Respiratory Flex Assay with the WHO International Standard was $7 . 7 { \times } 1 0 ^ { 5 } 1 0 / \mathrm { m L }$ .

Table 6. Limit of Detection Results for WHO International Standard for SARS‐CoV‐2 (NIBSC, 20/146) Target   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Concentration Tested (IU/mL)</td><td rowspan=1 colspan=1>SARS-CoV-2 Positivity</td></tr><tr><td rowspan=2 colspan=1>International WHOSARS-CoV-2Standard</td><td rowspan=1 colspan=1>7.70E+05</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>2.57E+05</td><td rowspan=1 colspan=1>65.0% (13/20)</td></tr></table>

e. Analytical Reactivity (Inclusivity) Laboratory (Wet Testing)

The analytical reactivity (inclusivity) of the LIAISON PLEX Respiratory Flex Assay was evaluated by using a collection of 181 isolates and clinical samples (34 bacteria and 147 viruses), representing the genetic diversity of the analytes in the LIAISON PLEX Respiratory Flex Assay. The organisms were diluted to a final concentration of 3X the target LoD in simulated NPS matrix and each diluted organism was tested in triplicate. In cases where $100 \%$ positivity was not achieved at 3X LoD, samples were reprepared at the same concentration and retested in triplicate. If $100 \%$ positivity was obtained during retesting, no additional testing was performed. If less than $100 \%$ positivity was obtained during retesting, the organism was prepared at a higher concentration and tested until $100 \%$ positivity was obtained.

Of the 181 strains tested, a total of 176 strains were detected with $100 \%$ positivity at 3X LoD in the

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission laboratory. Four strains (influenza A Brisbane (02/18), coronavirus NL63 (NR‐470), coronavirus 229E (VR‐ 740) and SARS‐CoV‐2 (Stanford)) were detected at 9X LoD and one strain (B. holmesii (CIP 104396)) was detected at 27X LoD. Results of laboratory (wet testing) are shown in Table 7.

Three influenza A variant strains (H1N1v, H1N2v, and H3N2v) and two influenza A strains (H2N2 and H7N9) were included in the evaluation. The influenza A variant strains were tested in the laboratory using synthetic DNA.  Based on this analysis, the three influenza A variant strains (H1N1v, H1N2v, and H3N2v) strains are expected to be detected as influenza A positive and subtype H1, H1, and H3 positive, respectively. Influenza H2N2 and H7N9 are predicted to be detected as influenza A, ${ \sf H } 1 / { \sf H } 3$ negative.

Table 7. Inclusivity of LIAISON PLEX Respiratory Flex Assay   

<table><tr><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td rowspan=1 colspan=1>y 7 7eX Hssay </td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>ReportableTarget &amp; Subtype</td><td rowspan=2 colspan=1>Serovars/Group/Species</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=2 colspan=1>% Detected (#Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>copies/mL</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=4 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>F061</td><td rowspan=1 colspan=1>2.60E+02</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.19E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CDC F5101 [CDC 84-013939]</td><td rowspan=1 colspan=1>1.95E+02</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.19E+04</td><td rowspan=1 colspan=1>83.3% (5/6)</td></tr><tr><td rowspan=1 colspan=1>CIP 104396</td><td rowspan=1 colspan=1>N/A1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.97E+056</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CIP 104395 [G7702; 92A2997]</td><td rowspan=1 colspan=2>N/A1</td><td rowspan=1 colspan=1>2.19E+04</td><td rowspan=1 colspan=1>80.0% (4/5)</td></tr><tr><td rowspan=8 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>NCTC 5952 [522]</td><td rowspan=1 colspan=1>1.05E+02</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>508 &amp; 344 [NCTC 10853]</td><td rowspan=1 colspan=1>9.15E+00</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>517</td><td rowspan=1 colspan=1>3.12E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>12822</td><td rowspan=1 colspan=1>7.68E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>509 and 609</td><td rowspan=1 colspan=1>6.02E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PT28G</td><td rowspan=1 colspan=1>3.32E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PT 26/28G</td><td rowspan=1 colspan=1>2.73E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C510</td><td rowspan=1 colspan=1>6.18E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>2.37E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=9 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>18323 [NCTC 10739]</td><td rowspan=1 colspan=1>4.12E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CNCTC Hp 12/63 [623]</td><td rowspan=1 colspan=1>4.87E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>10-536</td><td rowspan=1 colspan=1>2.47E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>5 [17921]</td><td rowspan=1 colspan=1>2.99E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Tohama I</td><td rowspan=1 colspan=1>5.71E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>MN2531</td><td rowspan=1 colspan=1>N/A{</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PT9/28G [W28]</td><td rowspan=1 colspan=1>1.99E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>589</td><td rowspan=1 colspan=1>1.95E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>N/A1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.14E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=4 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>1.16E+02</td><td rowspan=1 colspan=1>IFU/mL</td><td rowspan=1 colspan=1>1.71E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>1.93E+02</td><td rowspan=1 colspan=1>IFU/mL</td><td rowspan=1 colspan=1>1.71E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>J-21</td><td rowspan=1 colspan=1>9.48E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.71E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>2023</td><td rowspan=1 colspan=1>9.32E+01</td><td rowspan=1 colspan=1>IFU/mL</td><td rowspan=1 colspan=1>1.71E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=9 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>1.27E+02</td><td rowspan=1 colspan=1>CCU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>15531-TTR</td><td rowspan=1 colspan=1>1.23E+03</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Mac</td><td rowspan=1 colspan=1>1.45E+02</td><td rowspan=1 colspan=1>CCU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>1.56E+02</td><td rowspan=1 colspan=1>CCU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Bru</td><td rowspan=1 colspan=1>N/A1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>M52</td><td rowspan=1 colspan=1>1.83E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>UTMB-10P</td><td rowspan=1 colspan=1>7.83E-01</td><td rowspan=1 colspan=1>CCU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Mutant 22</td><td rowspan=1 colspan=1>9.47E+01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>1.33E-01</td><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>3.89E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>A 31</td><td rowspan=1 colspan=1>1.79E-02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>5.28E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 3</td><td rowspan=1 colspan=1>5.08E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 7A</td><td rowspan=1 colspan=1>7.05E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 21</td><td rowspan=1 colspan=1>3.60E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 11</td><td rowspan=1 colspan=1>1.43E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td></td><td></td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=14 colspan=1>ReportableTarget &amp; Subtype</td><td rowspan=2 colspan=1>Serovars/Group/Species</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=2 colspan=1>% Detected (#Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>copies/mL</td></tr><tr><td rowspan=1 colspan=1>B 14</td><td rowspan=1 colspan=1>5.68E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 34</td><td rowspan=1 colspan=1>1.19E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>83.3% (5/6)</td></tr><tr><td rowspan=1 colspan=1>B 35</td><td rowspan=1 colspan=1>9.52E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.06E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C1</td><td rowspan=1 colspan=1>2.69E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>3.35E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C2</td><td rowspan=1 colspan=1>1.30E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>3.35E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C5</td><td rowspan=1 colspan=1>1.10E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>3.35E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C6</td><td rowspan=1 colspan=1>4.81E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>3.35E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>D 26</td><td rowspan=1 colspan=1>3.51E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.24E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>D 37</td><td rowspan=1 colspan=1>1.43E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.24E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>E4</td><td rowspan=1 colspan=1>3.23E-02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.06E+03</td><td rowspan=1 colspan=1>83.3% (5/6)</td></tr><tr><td rowspan=1 colspan=1>F 40-Dugan</td><td rowspan=1 colspan=1>7.60E-02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.45E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>F41-Tak</td><td rowspan=1 colspan=1>8.97E-03</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.45E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=8 colspan=1>Human Coronavirus</td><td rowspan=1 colspan=1>HKU1</td><td rowspan=2 colspan=2>N/A2</td><td rowspan=1 colspan=1>5.00E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>5.00E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>NL63Source #: NR-4705</td><td rowspan=1 colspan=1>5.41E-03</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.88E+026</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>NL63Source #: 0810228CF5</td><td rowspan=1 colspan=1>4.02E-02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.29E+02</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>OC43Source #: 0810024CF5</td><td rowspan=1 colspan=1>2.87E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.84E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>OC43Source #: VR-15585</td><td rowspan=1 colspan=1>1.02E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.84E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>229ESource #: 0810229CF5</td><td rowspan=1 colspan=1>2.74E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.20E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>229ESource #: VR-7405</td><td rowspan=1 colspan=1>1.51E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>3.60E+036</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=16 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus A10</td><td rowspan=1 colspan=1>3.53E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus 71 (2003)</td><td rowspan=1 colspan=1>1.18E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus A9</td><td rowspan=1 colspan=1>1.98E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus B3</td><td rowspan=1 colspan=1>1.79E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus B4</td><td rowspan=1 colspan=1>1.04E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusEchovirus 6</td><td rowspan=1 colspan=1>9.01E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusEchovirus 9</td><td rowspan=1 colspan=1>5.21E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusEchovirus 11</td><td rowspan=1 colspan=1>1.02E+03</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusEchovirus 30</td><td rowspan=1 colspan=1>1.20E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus A21</td><td rowspan=1 colspan=1>2.02E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human EnterovirusCoxsackievirus A24</td><td rowspan=1 colspan=1>8.50E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Enterovirus68</td><td rowspan=1 colspan=1>1.05E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>6.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A16</td><td rowspan=1 colspan=1>1.45E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A2</td><td rowspan=1 colspan=1>4.89E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A34</td><td rowspan=1 colspan=1>1.69E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A57</td><td rowspan=1 colspan=1>1.04E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=15 colspan=1>ReportableTarget &amp; Subtype</td><td rowspan=2 colspan=1>Serovars/Group/Species</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=2 colspan=1>% Detected (#Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>copies/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A7</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A77</td><td rowspan=1 colspan=1>2.15E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus A85</td><td rowspan=1 colspan=1>7.60E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.46E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus B14</td><td rowspan=1 colspan=1>3.31E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus B17</td><td rowspan=1 colspan=1>1.75E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus B27</td><td rowspan=1 colspan=1>5.09E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus B3</td><td rowspan=1 colspan=1>5.68E+01</td><td rowspan=1 colspan=1>PFU/mL</td><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus B42</td><td rowspan=1 colspan=1>8.64E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus B83</td><td rowspan=1 colspan=1>1.13E+01</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>2.45E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus C1</td><td rowspan=1 colspan=1>NA^{2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus C1</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus C1</td><td rowspan=1 colspan=1>NA{2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus C1</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.75E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=9 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>(hMPV-9) A1</td><td rowspan=1 colspan=1>6.26E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>6.40E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-16) A1</td><td rowspan=1 colspan=1>4.01E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.40E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-20) A2</td><td rowspan=1 colspan=1>NA^{2$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6.12E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-27) A2</td><td rowspan=1 colspan=1>6.41E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>6.12E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-3) B1</td><td rowspan=1 colspan=1>1.29E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.50E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-5) B1</td><td rowspan=1 colspan=1>9.50E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.50E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-4) B2</td><td rowspan=1 colspan=1>1.37E+03</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>4.50E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-8) B2</td><td rowspan=1 colspan=1>4.99E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>4.50E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>(hMPV-18) B2</td><td rowspan=1 colspan=1>2.78E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>4.50E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=30 colspan=1>Influenza A  H1N1</td><td rowspan=2 colspan=1>A/Wisconsin/588/2019</td><td rowspan=2 colspan=1>4.06E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Hawaii/66/2019 X-345A</td><td rowspan=2 colspan=1>2.45E+03</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>H1: 83.3% (5/6)</td></tr><tr><td rowspan=2 colspan=1>A/Indiana/02/2020</td><td rowspan=2 colspan=1>2.06E+03</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Michigan/272/2017</td><td rowspan=2 colspan=1>1.86E+01</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Idaho/07/2018</td><td rowspan=2 colspan=1>8.73E-01</td><td rowspan=2 colspan=1>TCID5o/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Wisconsin/505/2018</td><td rowspan=2 colspan=1>5.40E+00</td><td rowspan=2 colspan=1>TCID5o/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>H1: 83.3% (5/6)</td></tr><tr><td rowspan=2 colspan=1>Guangdong-Maonan/SWL1536/19</td><td rowspan=2 colspan=1>1.93E+00</td><td rowspan=2 colspan=1>TCID5o/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>H1: 83.3% (5/6)</td></tr><tr><td rowspan=2 colspan=1>Brisbane/02/18</td><td rowspan=2 colspan=1>3.97E+00</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=2 colspan=1>1.35E+046</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3</td></tr><tr><td rowspan=2 colspan=1>A/St.Petersburg/61/2015</td><td rowspan=2 colspan=1>1.82E+03</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Bangladesh/3002/2015</td><td rowspan=2 colspan=1>7.78E+02</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Denver/1/57</td><td rowspan=2 colspan=1>6.84E+02</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>New Caledonia/20/99</td><td rowspan=2 colspan=1>8.71E+00</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>PR/8/34</td><td rowspan=2 colspan=1>1.58E+00</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Singapore/63/04</td><td rowspan=2 colspan=1>6.23E-01</td><td rowspan=2 colspan=1>TCID5o/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Solomon Islands/03/06</td><td rowspan=2 colspan=1>5.00E-01</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=2>tableSubtype</td><td rowspan=2 colspan=1>Serovars/Group/Species</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=2 colspan=1>% Detected (#Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>copies/mL</td></tr><tr><td></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Taiwan/42/06</td><td rowspan=2 colspan=1>3.58E-01</td><td rowspan=2 colspan=1>TCID5o/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td></td><td rowspan=4 colspan=1>H1N1v</td><td rowspan=2 colspan=1>A/Ohio/09/2015 (SubtypeSynthetic DNA)</td><td rowspan=2 colspan=2>N/A3</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 0% (0/3)7</td></tr><tr><td></td><td rowspan=1 colspan=1>H1: 100% (3/3)7</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Ohio/09/2015 (Influenza ASynthetic DNA)</td><td rowspan=2 colspan=2>N/A3</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)7</td></tr><tr><td></td><td rowspan=1 colspan=1>H1: 0% (0/3)7</td></tr><tr><td></td><td rowspan=2 colspan=1>H1N2</td><td rowspan=2 colspan=1>A/swine/Ohio/09SW1484E/2009</td><td rowspan=2 colspan=1>4.15E+03</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H1: 100% (3/3)</td></tr><tr><td></td><td rowspan=4 colspan=1>H1N2v</td><td rowspan=2 colspan=1>A/Minnesota/19/2011(Subtype Synthetic DNA)</td><td rowspan=2 colspan=2>N/A3</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 0% (0/3)7</td></tr><tr><td></td><td rowspan=1 colspan=1>H1: 100% (3/3)7</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Minnesota/19/2011(Influenza A Synthetic DNA)</td><td rowspan=2 colspan=2>N/A3</td><td rowspan=2 colspan=1>4.50E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)7</td></tr><tr><td></td><td rowspan=1 colspan=1>H1: 0% (0/3)7</td></tr><tr><td></td><td rowspan=20 colspan=1>H3N2</td><td rowspan=2 colspan=1>A/Kansas/14/2017 NYMC X-327</td><td rowspan=2 colspan=1>1.87E+03</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Texas/71/2017</td><td rowspan=2 colspan=1>4.35E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Wisconsin/04/2018</td><td rowspan=2 colspan=1>2.38E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Arizona/45/2018</td><td rowspan=2 colspan=1>8.27E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Hong Kong/45/2019</td><td rowspan=2 colspan=1>6.09E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Tasmania/503/2020</td><td rowspan=2 colspan=1>1.99E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Delaware/01/2021</td><td rowspan=2 colspan=1>7.01E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Singapore/INFIMH-16-0019/2016</td><td rowspan=2 colspan=1>8.93E+02</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/California/55/2020</td><td rowspan=2 colspan=1>8.55E+01</td><td rowspan=2 colspan=1>FFU/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Alaska/232/2015</td><td rowspan=2 colspan=1>1.14E+03</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)</td></tr><tr><td></td><td rowspan=4 colspan=1>H3N2v</td><td rowspan=2 colspan=1>A/Hawaii/28/2020 (SubtypeSynthetic DNA)</td><td rowspan=2 colspan=2>N/A3</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 0% (0/3)7</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 100% (3/3)7</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Hawaii/28/2020 (Influenza ASynthetic DNA)</td><td rowspan=2 colspan=2>N/A3</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)7</td></tr><tr><td></td><td rowspan=1 colspan=1>H3: 0% (0/3)7</td></tr><tr><td></td><td rowspan=6 colspan=1>H5N1</td><td rowspan=2 colspan=1>A/Egypt/N03072/2010</td><td rowspan=2 colspan=1>4.70E-03</td><td rowspan=2 colspan=1>HA4</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)8</td></tr><tr><td></td><td rowspan=1 colspan=1>Subtype: 0% (0/3)8</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Hubei/1/2010</td><td rowspan=2 colspan=1>3.97E-03</td><td rowspan=2 colspan=1>HA4</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)8</td></tr><tr><td></td><td rowspan=1 colspan=1>Subtype: 0% (0/3)8</td></tr><tr><td></td><td rowspan=2 colspan=1>A/Anhui/01/2005</td><td rowspan=2 colspan=1>1.54E-02</td><td rowspan=2 colspan=1>HA4</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)8</td></tr><tr><td></td><td rowspan=1 colspan=1>Subtype: 0% (0/3)8</td></tr><tr><td></td><td rowspan=1 colspan=1>H7N2</td><td rowspan=1 colspan=1>A/turkey/Virginia/4529/2002</td><td rowspan=1 colspan=1>4.40E-02</td><td rowspan=1 colspan=1>$HA^4}$</td><td rowspan=1 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)8</td></tr></table>

# Diasorin

<table><tr><td rowspan=2 colspan=2>ReportableTarget &amp; Subtype</td><td rowspan=2 colspan=1>Serovars/Group/Species</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=2 colspan=1>% Detected (#Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>copies/mL</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Subtype: 0% (0/3)8</td></tr><tr><td rowspan=2 colspan=1>H7N7</td><td rowspan=2 colspan=1>A/mallard/Netherlands/12/2000</td><td rowspan=2 colspan=1>8.03E-03</td><td rowspan=2 colspan=1>HA4</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)8</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)8</td></tr><tr><td rowspan=2 colspan=1>H9N2</td><td rowspan=2 colspan=1>A/Hong Kong/33982/2009</td><td rowspan=2 colspan=1>5.06E+03</td><td rowspan=2 colspan=1>CEID50/mL</td><td rowspan=2 colspan=1>5.89E+03</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)8</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)8</td></tr><tr><td rowspan=15 colspan=2>Influenza B</td><td rowspan=1 colspan=1>B/Washington/02/2019 (VictoriaLineage)</td><td rowspan=1 colspan=1>1.97E+02</td><td rowspan=1 colspan=1>CEID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/New Hampshire/01/2021(Victoria Lineage)</td><td rowspan=1 colspan=1>9.28E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Missouri/12/2018 (NA D197E)(Victoria Lineage)</td><td rowspan=1 colspan=1>1.35E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/10/2016 (NA1221V) (Yamagata Lineage)</td><td rowspan=1 colspan=1>1.81E+03</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Indiana/17/2017 (NA I221T)(Yamagata Lineage)</td><td rowspan=1 colspan=1>1.79E+03</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Hawaii/01/2018 (NA D197N(Victoria Lineage)</td><td rowspan=1 colspan=1>4.56E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Oklahoma/10/2018 (NAD197N) (Yamagata Lineage)</td><td rowspan=1 colspan=1>1.85E+03</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Michigan/01/2021 (VictoriaLineage)</td><td rowspan=1 colspan=1>7.91E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/286/2017 (VictoriaLineage)</td><td rowspan=1 colspan=1>2.80E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/6/2017 (VictoriaLineage)</td><td rowspan=1 colspan=1>2.33E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Texas/43/2019 (VictoriaLineage)</td><td rowspan=1 colspan=1>1.55E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/10 (YamagataLineage)</td><td rowspan=1 colspan=1>3.23E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Florida/02/06 (YamagataLineage)</td><td rowspan=1 colspan=1>1.60E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/04 (YamagataLineage)</td><td rowspan=1 colspan=1>3.71E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/13 (YamagataLineage)</td><td rowspan=1 colspan=1>6.53E-01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=2>Parainfluenza 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3.19E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.28E+03</td><td rowspan=1 colspan=1>83.3% (5/6)</td></tr><tr><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>2.62E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.28E+03</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=2>Parainfluenza 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4.65E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.54E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>4.81E+00</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>2.54E+04</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=2>Parainfluenza 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>9.54E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>5.79E+03</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC-2011-5</td><td rowspan=1 colspan=1>1.78E+02</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>5.79E+03</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>8.83E+02</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>5.79E+03</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NIH 47885</td><td rowspan=1 colspan=1>3.57E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>5.79E+03</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=2>Parainfluenza 4</td><td rowspan=1 colspan=1>4A</td><td rowspan=1 colspan=1>6.63E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.73E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>4A M-25</td><td rowspan=1 colspan=1>5.94E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.73E+04</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>4B</td><td rowspan=1 colspan=1>5.40E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>1.73E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>4B CH 19503</td><td rowspan=1 colspan=1>8.79E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.73E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=7 colspan=2>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>A 2006 Isolate</td><td rowspan=1 colspan=1>3.34E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>1.15E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>1.10E+03</td><td rowspan=1 colspan=1>PFU/mL</td><td rowspan=1 colspan=1>1.15E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>A Long</td><td rowspan=1 colspan=1>1.11E+03</td><td rowspan=1 colspan=1>PFU/mL</td><td rowspan=1 colspan=1>1.15E+04</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B CH93(18)-18</td><td rowspan=1 colspan=1>1.46E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>4.82E+04</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B WV/14617/85</td><td rowspan=1 colspan=1>2.12E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>4.82E+04</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B 18537</td><td rowspan=1 colspan=1>3.16E+00</td><td rowspan=1 colspan=1>PFU/mL</td><td rowspan=1 colspan=1>4.82E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B1</td><td rowspan=1 colspan=1>1.30E+03</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>4.82E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=1>ReportableTarget &amp; Subtype</td><td rowspan=2 colspan=1>Serovars/Group/Species</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=2 colspan=1>% Detected (#Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>copies/mL</td></tr><tr><td rowspan=13 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>1.37E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.1.529 BA.1: USA/MD-HP20874/2021 (Omicron)</td><td rowspan=1 colspan=1>9.59E-01</td><td rowspan=1 colspan=1>TCID5o0/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Italy-INMI1</td><td rowspan=1 colspan=1>1.44E+02</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Hong Kong/VM20001061/2020</td><td rowspan=1 colspan=1>2.37E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1_2020: USA/NY-Wadsworth-103677-01/2020</td><td rowspan=1 colspan=1>9.74E-01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.1.7:England/204820464/2020(Alpha)</td><td rowspan=1 colspan=1>1.53E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.1.7:USA/CA_CDC_5574/2020 (Alpha)</td><td rowspan=1 colspan=1>6.61E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.351: South Africa/KRISP-K005325/2020 (Beta)</td><td rowspan=1 colspan=1>5.84E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>P1: Japan/TY7-503/2021(Gamma)</td><td rowspan=1 colspan=1>5.65E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>P2_2021: NY-Wadsworth-21006055-01/2021 (Zeta)</td><td rowspan=1 colspan=1>8.82E+00</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.526_2021: USA/NY-Wadsworth-21025952-01/2021Isolate 1 (Lota)</td><td rowspan=1 colspan=1>1.39E+01</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.617.1: USA/CA-Stanford-15_S02/2021 (Kappa)</td><td rowspan=1 colspan=1>1.22E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>7.20E+046</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.617.2: USA/PHC658/2021(Delta)</td><td rowspan=1 colspan=1>9.42E+00</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>2.40E+04</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 No supplier concentration listed. 2 Clinical samples quantified in copies/mL based on digital droplet PCR (ddPCR) analysis. 3 Synthetic DNA quantified in copies/mL based on spectrophotometric analysis and optical density measurements. 4 Titer determined through hemagglutination assay using $0 . 5 \%$ turkey red blood cells. $^ 5 { \mathsf { N o } }$ additional strain information was provided by the supplier so different samples tested within the same species were distinguished using the vendor catalog number. 6 Four strains (influenza A Brisbane (02/18), coronavirus NL63 (NR‐470), coronavirus 229E (VR‐740) and SARS‐CoV‐2 (Stanford)) were detected at 9X LoD and one stra (B. holmesii (CIP 104396)) was detected at 27X LoD. 7 Subtype synthetic DNA and influenza A synthetic DNA were expected to yield positive results for the subtype and influenza A targets, respectively. 8 Matrix positive, subtype negative results for influenza A strains H5N1, H7N2, H7N7, and H9N2 are expected results based on the assay design.

# Diasorin

# In Silico Inclusivity for SARS‐CoV‐2 and Influenza

In addition to wet testing, in silico analyses for SARS‐CoV‐2 and Influenza were performed.

# SARS‐CoV‐2:

For SARS‐CoV‐2, 5,622,325 sequences in GISAID (as of July 31, 2023) were included in the analysis. The LIAISON PLEX Respiratory Flex Assay targets three SARS‐CoV‐2 gene regions (E, ORF1ab, and ORF3a). The Respiratory Flex Assay result logic states that if at least 1 of these targets is detected, SARS‐CoV‐2 is positive. This same result logic was implemented for the in silico inclusivity assessment. Of the sequences included in this evaluation, $9 9 . 9 4 \%$ (5,619,069/5,622,325) have no mismatch in at least one gene oligo set and thus are predicted to be detected by the Respiratory Flex Assay. Of the $0 . 0 6 \%$ (3,256/5,622,325) of sequences with mismatches in at least one oligo binding region in all 3 SARS‐CoV‐2 target genes, a Tm analysis revealed that amplification/hybridization were expected to occur. Thus, it is expected that $100 \%$ of SARS‐CoV‐2 sequences evaluated in this study will be detected by the assay.

# Influenza:

For influenza A, influenza A H1, influenza A H3, and influenza B, sequences uploaded to GISAID between September 1, 2015 and July 5, 2023 were included in the analysis. The following number of sequences were included in the evaluation of influenza A, A H1, A H3, and influenza B: 112,056, 54,364, 104,428, 26,470. Of the 26,470 influenza B sequences, $6 6 . 1 \%$ (17,509/26,470) were Victoria lineage, $30 . 9 \%$ (8,167/26,470) were Yamagata lineage, and $3 . 0 \%$ (794/26,470) were of unknown/unclassified lineage. The influenza in silico inclusivity analysis results are shown in Table 8. Based on the reactivity criteria $5 9 0 \%$ homology), $9 9 . 9 \%$ (112,034/112,056) of influenza A sequences are expected to be detected, $9 8 . 9 \%$ (53,778/54,364) of influenza A H1 sequences are expected to be detected, $9 9 . 9 \%$ (104,315/104,428) of influenza A H3 sequences are expected to be detected, and $9 9 . 9 \%$ (26,433/26,470) of influenza B sequences are expected to be detected.

Table 8. Influenza In Silico Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>ReportableTarget</td><td rowspan=1 colspan=1>Target Gene</td><td rowspan=1 colspan=1>of Sequences inAlignment</td><td rowspan=1 colspan=1># of Sequences withPercent Oligo Identify&gt;90%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Matrix protein (MP)</td><td rowspan=1 colspan=1>112,056</td><td rowspan=1 colspan=1>112,0341</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>HA</td><td rowspan=1 colspan=1>54,364</td><td rowspan=1 colspan=1>53,778</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>HA</td><td rowspan=1 colspan=1>104,428</td><td rowspan=1 colspan=1>104,315</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Non-structural protein (NS)</td><td rowspan=1 colspan=1>26,470</td><td rowspan=1 colspan=1>26,4332</td></tr></table>

1 Analysis included influenza A subtype H0, H1, H3, H5, H7, H9, and H10 strains. 2 Analysis included 17,509 Victoria lineage strains, 8,167 Yamagata lineage strains, and 794 strains of unknown lineage.

# Diasorin

f. Analytical specificity

Cross‐Reactivity and Microbial Interference:

# Off-Panel Cross Reactivity

# Laboratory (Wet Testing)

Cross‐reactivity was assessed in the laboratory by wet testing 60 off‐panel viral, fungal, and bacterial organisms that may be found in a respiratory tract clinical specimen. The potential cross‐reacting organisms were spiked into simulated NPS matrix that was negative for all targets on the assay. Bacteria were tested at concentrations $\ge 1 { \times } 1 0 ^ { 6 } \mathsf { C F U / m L }$ (or equivalent) and viruses were tested at $\geq 1 \mathsf { x } 1 0 ^ { 5 } \mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ (or equivalent), or the highest available concentration. For Mycobacterium tuberculosis, genomic DNA (quantified in $\mathsf { n g / u L }$ ) was evaluated to minimize pathogenic exposure to the test operator.

Of the 60 potential cross‐reacting off‐panel organisms tested in the laboratory (Table 9), 59 organisms yielded negative results for all targets and are considered non‐reactive with the LIAISON PLEX Respiratory Flex Assay. Mycoplasma genitalium cross‐reacted with the Mycoplasma pneumoniae assay at a concentration of $4 { \times } 1 0 ^ { 6 } \mathsf { C C U / m L }$ . No cross‐reactivity was observed when Mycoplasma genitalium was tested at a lower concentration of $1 { \times } 1 0 ^ { 6 } \thinspace { \mathsf { C C U / m L } }$ .

Table 9. Organisms Tested for Potential Cross-Reactivity (Off-Panel)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>4x105 CFU/mL³3</td></tr><tr><td rowspan=1 colspan=1>Aspergillus flavus</td><td rowspan=1 colspan=1>1x10CFU/mL</td><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>4x105}$CFU/mL3</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella avium</td><td rowspan=1 colspan=1>1x106CFU/mL</td><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>1x105 TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>1x10CFU/mL</td><td rowspan=1 colspan=1>MERS-CoV</td><td rowspan=1 colspan=1>NA{2</td></tr><tr><td rowspan=1 colspan=1>Bordetella hinzii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella petrii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>1x105 TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella trematum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Mycobacteriumtuberculosis(H37Rv gDNA)</td><td rowspan=1 colspan=1>2.88 ng/uL</td></tr><tr><td rowspan=3 colspan=1>Bordetella parapertussisBpp5 (synthetic DNA)1</td><td rowspan=3 colspan=1>1x106 copies/mL</td><td rowspan=3 colspan=1>Mycoplasmagenitalium3</td><td rowspan=1 colspan=1>4x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>1x106 CcCU/mL</td></tr><tr><td rowspan=1 colspan=1>4x105 ccU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Nasal Wash (pooled)</td><td rowspan=1 colspan=1>NA{2</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatisSerovar D</td><td rowspan=1 colspan=1>1x106 IFU/mL</td><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus-SARS</td><td rowspan=1 colspan=1>NA{2</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumdiphtheriae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumpseudodiphtheriticum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1x105TCID5so/mL</td><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>1x10^ CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>1x10^5 copies/mL</td><td rowspan=1 colspan=1>Pneumocystis jiroveci</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fluoribacter bozemanae</td><td rowspan=1 colspan=1>4x106 CFU/mL</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacteriumnecrophorum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilusparainfluenzae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex VirusType 1</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcuspyogenes</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella anisa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcussalivarius</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella feeleii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Ureaplasmaurealyticum</td><td rowspan=1 colspan=1>1x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella longbeachae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Varicella-Zoster Virus</td><td rowspan=1 colspan=1>2.34x104TCID5o0/mL³3</td></tr></table>

CFU $=$ Colony Forming Units; $\complement { \complement } =$ Colony Changing Units; IFU $=$ Inclusion Forming Units; TCID50 $=$ Median Tissue Culture Infectious Dose. $^ 1 \mathsf { A }$ portion of the B. parapertussis Bpp5 genome was identified by in‐silico analysis as potentially cross‐reactive with B. pertussis. Synthetic DNA was tested that matched the region of high homology in the assay. Testing was included in the off‐panel testing since the targeted sequence was not expected to be detected as B. parapertussis by the assay. ${ } ^ { 2 } \mathsf { N o }$ concentration provided by the supplier. 3 The highest possible concentration was tested.

# In Silico Cross‐Reactivity

In silico analysis of assay specificity/exclusivity was performed by conducting a BLAST comparison of the assay’s oligos sequences to the GenBank nt sequence database, as of July 14, 2023. Sequences for 83 off‐panel organisms (68 bacteria/fungi and 15 viruses) that can be found in a respiratory specimen were included. Additionally, sequences for all on‐panel organisms were included to evaluate intra‐panel cross‐reactivity. A summary of the results from the analysis is provided in Table 10. The LIAISON PLEX Respiratory Flex assays were shown to be specific for their respective analytes with the following exceptions:

 Cross‐reaction of the Adenovirus assays with closely related Adenovirus G (serotype 52) strains.   
 Cross‐reaction of the SARS‐CoV‐2 assays with closely related bat and pangolin coronavirus sequences;   
 Cross‐reaction of the B. parapertussis assay with strains of B. bronchiseptica that carry IS1001;   
 Cross‐reaction of the influenza A H1 subtyping assay with 3 swine H3N2 strains and 1 avian H6N1 strain;   
 Cross‐reaction of the influenza A H3 subtyping assay with 59 swine H1N1 and swine H1N2 strains, 1 duck H5N2 strain, 1 ostrich H7N1 strain, 1 avian H7N9 strain, 1 avian H8N4 strain, and 1 avian H11N9 strain.

# Diasorin

Table 10. Organisms Predicted by In Silico Analyses to Cross‐React with the Respiratory Flex Assay.   

<table><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>Predicted Cross-Reaction</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenovirus G (serotype 52) - strains</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Bat coronavirus and Bat SARS-like coronavirus (accessions MG772933,MG772934, and MN996532)1</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Bordetella bronchiseptica containing IS1001 element (accessions JX013523to JX013527 and CP022962)2</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>H5N1 (accession CY110922)3;swine H3N2 (accessions KM110061, KM110062, KM110063, andOM935891);avian H6N1 (accession OP888980)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>swine H1N1 and swine H1N2 - 59 strains;duck H5N2 (accession OK103962);ostrich H7N1 (accession AF202244);avian H7N9 (accession KP413675);avian H8N4 (accession OK103964);avian H11N9 (accession OK103956)</td></tr></table>

1 It is unlikely that Bat coronavirus and Bat SARS‐like coronavirus would be present in human clinical NPS specimens; but if present, the cross‐reactive product(s) produced by the LIAISON PLEX Respiratory Flex Assay will be reported as SARS‐CoV‐2. 2 The LIAISON PLEX Respiratory Flex Assay contains primers designed to target the B. parapertussis IS1001 insertion sequence. Some strains of Bordetella bronchiseptica, which is rarely isolated from humans, carry the same sequence that is targeted by LIAISON PLEX Respiratory Flex primers for B. parapertussis. If present, the LIAISON PLEX Respiratory Flex Assay will report these specimens as B. parapertussis.

3 This H5N1 human strain sequence is a chimeric sequence containing H1N1 sequence fragments. Therefore, detection of this sequence by the H1 oligos is not considered a cross‐reaction.

# On‐Panel Cross Reactivity

Potential intra‐panel cross‐reactivity was evaluated using 28 on‐panel organisms. The potential cross‐reacting on‐panel organisms were spiked into simulated NPS matrix that was negative for all targets on the assay. Bacteria were tested at concentrations $\ge 1 { \times } 1 0 ^ { 6 } \mathsf { C F U / m L }$ (or equivalent) and viruses were tested at $\geq 1 \mathsf { x } 1 0 ^ { 5 } \mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ (or equivalent), or the highest available concentration.

All 28 on‐panel organisms tested (Table 11) yielded the expected on‐panel result and did not cross‐react with other target assays of the LIAISON PLEX Respiratory Flex Assay.

Table 11. Organisms Tested for Potential Cross-Reactivity (On-Panel)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1x105TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1x106IFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus 229E</td><td rowspan=1 colspan=1>1x105TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus HKU1</td><td rowspan=1 colspan=1>6.62x104 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus NL63</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus OC43</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr></table>

# Diasorin

LIAISON $\mathsf { P l E X } ^ { \mathrm { \otimes } }$ Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td></tr><tr><td rowspan=1 colspan=1>Echovirus(Enterovirus/Rhinovirus)</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B(Washington/02/2019/VictoriaLineage</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B(Phuket/3073/13/YamagataLineage)</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1x10^5 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H5N11</td><td rowspan=1 colspan=1>2x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H7N21</td><td rowspan=1 colspan=1>6x106 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H7N71</td><td rowspan=1 colspan=1>2x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H9N21</td><td rowspan=1 colspan=1>4x108 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N22</td><td rowspan=1 colspan=1>3x10&#x27; copies/mL</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td></tr></table>

CFU $=$ Colony Forming Units; IFU $=$ Inclusion Forming Units; TCID50 $=$ Median Tissue Culture Infectious Dose; PFU $=$ Plaque Forming Units 1 These influenza A non‐subtype ${ \sf H } \bar { \sf 1 } / { \sf H } \bar { \sf 3 }$ strains are expected to be inclusive to the influenza A matrix target, only (i.e., are expected to be reported as influenza A positive, subtype H1/subtype H3 negative). All were negative for both subtype H1 and subtype H3, as anticipated. 2 This influenza A H1N2 strain is expected to be inclusive to the influenza A matrix target and influenza A subtype H1 target. This strain was positive for both the influenza A matrix target and influenza A subtype H1 target, as expected.

# Microbial Interference

The impact of 16 potentially interfering non‐panel microbial organisms commonly found in nasopharyngeal swab samples were tested (Table 12) in the presence of representative assay panel targets. Each potentially interfering non‐panel organism was tested at a high concentration of $\geq 1 \times 1 0 ^ { 6 }$ $\mathsf { C F U / m L }$ (or equivalent) for bacteria and $1 \times 1 0 ^ { 5 } \mathrm { \ : T C l { D } _ { 5 0 } / m L }$ (or equivalent) for viruses, or the highest concentration available, in the presence of a multi‐analyte panel consisting of five assay targets (B. pertussis, Adenovirus, influenza B, human metapneumovirus, and SARS‐CoV‐2) at a low concentration (3X LoD).

None of the potentially interfering organisms tested at high concentration interfered with detection for panel targets at a low concentration, except for Streptococcus pyogenes at $1 { \times } 1 0 ^ { 6 } \thinspace { \mathsf { C F U / m L } }$ and Legionella pneumophilia at $4 { \times } 1 0 ^ { 5 } \thinspace \mathrm { C F U / m L }$ , which both interfered with detection of Adenovirus in 1 of 6 replicates $( 8 3 . 3 \%$ positivity).

# Diasorin

Table 12. Organisms Tested for Potential Microbial Interference   

<table><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>SARS1</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus 1</td><td rowspan=1 colspan=1>Legionella pneumophilia²</td></tr><tr><td rowspan=1 colspan=1>MERS1</td><td rowspan=1 colspan=1>Measles Virus</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Mumps</td></tr></table>

1 No concentration was provided for material. Material was tested at the highest available concentration by diluting stock material directly into representative multi‐analyte panel preparation. 2 Tested at the highest available concentration of $4 { \times } 1 0 ^ { 5 } \thinspace \mathrm { C F U / m L }$

# Interfering Substances:

The potential inhibitory effect of non‐microbial substances (endogenous and exogenous) expected to be found in nasopharyngeal swab (NPS) specimens or introduced during sample handling, were evaluated for the LIAISON PLEX Respiratory Flex Assay. Potential interference from 36 interfering substances were evaluated in the presence and absence of a multi‐analyte panel of targets (Table 13). Each interfering substance tested was diluted to a clinically relevant concentration and tested in the presence of a positive and negative target in triplicate. The positive target was a multi‐ analyte panel consisting of five assay targets (B. pertussis, Adenovirus, influenza B, human metapneumovirus, and SARS‐CoV‐2) at a low concentration (3X LoD), while the negative target was a simulated NPS matrix. Interference was observed for the substances/concentrations shown in Table 14.

Table 13: Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Substance/Class</td><td rowspan=1 colspan=1>Description/Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Nasal Corticosteroid</td><td rowspan=1 colspan=1>Beclomethasone dipropionate</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Anesthetic</td><td rowspan=1 colspan=1>Benzocaine</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Nasal Corticosteroid</td><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>3.4x10-²2 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Nasal Corticosteroid</td><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>30.6 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Nasal Corticosteroid</td><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>FLONASE Sensimist Allergy Relief</td><td rowspan=1 colspan=1>Fluticasone furoate</td><td rowspan=1 colspan=1>2.84x10-³3μmol/L</td></tr><tr><td rowspan=1 colspan=1>Fluticasone Propionate NasalSpray</td><td rowspan=1 colspan=1>Fluticasone propionate</td><td rowspan=1 colspan=1>2.84x10-3 μmol/L</td></tr><tr><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>20 ng/μL</td></tr><tr><td rowspan=1 colspan=1>Nasal Wash</td><td rowspan=1 colspan=1>Human Nasal Wash</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=2 colspan=1>Sputum/Mucus</td><td rowspan=2 colspan=1>Human Sputum/Mucus</td><td rowspan=1 colspan=1>1 swab/1mL sample1</td></tr><tr><td rowspan=1 colspan=1>1 swab/2mL sample²</td></tr><tr><td rowspan=3 colspan=1>Human Blood</td><td rowspan=3 colspan=1>Human Whole Blood</td><td rowspan=1 colspan=1>5.0% v/v</td></tr><tr><td rowspan=1 colspan=1>4.5% v/v</td></tr><tr><td rowspan=1 colspan=1>4.0% v/v</td></tr><tr><td rowspan=3 colspan=1>Human Cells</td><td rowspan=3 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>1000 cells/μL</td></tr><tr><td rowspan=1 colspan=1>666.7 cells/μL</td></tr><tr><td rowspan=1 colspan=1>333.3 cells/μL</td></tr><tr><td rowspan=1 colspan=1>Oral Anesthetic and Analgesic</td><td rowspan=1 colspan=1>Menthol</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=1 colspan=1>Nasal Corticosteroid</td><td rowspan=1 colspan=1>Mometasone furoate</td><td rowspan=1 colspan=1>8.63x10-4 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin, bovine submaxillary Type I-S</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin, porcine stomach Type II</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin, porcine stomach Type III</td><td rowspan=1 colspan=1>100 μg/mL</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Substance/Class</td><td rowspan=1 colspan=1>Description/Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, Nasal Ointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>3.0 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Anti-viral</td><td rowspan=1 colspan=1>Oseltamivir Phosphate</td><td rowspan=1 colspan=1>1.28 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Afrin Nasal Spray</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Nasal Decongestant</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>1.79x10-1 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>Sodium Chloride</td><td rowspan=1 colspan=1>1%v/v</td></tr><tr><td rowspan=1 colspan=1>Nasal Corticosteroid</td><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Antibiotic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>76.0 μmol/L</td></tr><tr><td rowspan=1 colspan=1>Anti-viral</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Anti-viral</td><td rowspan=1 colspan=1>Zinc</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=4 colspan=1>ZICAM Nasal Spray</td><td rowspan=1 colspan=1>Galphimia Glauca</td><td rowspan=4 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Histaminum Hydrochloricum</td></tr><tr><td rowspan=1 colspan=1>Luffa operculata</td></tr><tr><td rowspan=1 colspan=1>Sulfur</td></tr><tr><td rowspan=1 colspan=1>NPS Swab</td><td rowspan=1 colspan=1>Nylon swab (Copan)</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Transport Media</td><td rowspan=1 colspan=1>Universal Transport Medium (Copan)</td><td rowspan=1 colspan=1>100%</td></tr></table>

1 A nylon nasopharyngeal swab was fully coated with human sputum/mucus and then eluted into 1 mL of simulated NPS matrix, containing 5 representative target organisms at $_ { 3 \times }$ LoD. The eluent was subsequently tested with the Respiratory Flex Assay. $^ 2 \mathsf { A }$ nylon nasopharyngeal swab was fully coated with human sputum/mucus and then eluted into $2 \mathsf { m l }$ of simulated NPS matrix, containing 5 representative target organisms at 3x LoD. The eluent was subsequently tested with the Respiratory Flex Assay.

Table 14 Substances that Interfered with Detection of at Least One Target Organism   

<table><tr><td rowspan=2 colspan=1>ActiveIngredient</td><td rowspan=2 colspan=2>Test Conc.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>% Positivity (# Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=2 colspan=1>HumanSputum/Mucus</td><td rowspan=1 colspan=2>1 swab/1mLsample</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>33.3%(2/6)1</td><td rowspan=1 colspan=1>66.7%(4/6)1</td><td rowspan=1 colspan=1>83.3%(5/6)1</td></tr><tr><td rowspan=1 colspan=2>1 swab/2mLsample</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=3 colspan=1>Human WholeBlood</td><td rowspan=1 colspan=2>5.0% v/v</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>83.3%(5/6)1</td><td rowspan=1 colspan=1>66.7% (4/6)1</td><td rowspan=1 colspan=1>83.3%(5/6)1</td><td rowspan=1 colspan=1>100%(6/6)</td></tr><tr><td rowspan=1 colspan=2>4.5% v/v</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>66.7% (2/3)1</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=2>4.0% v/v</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=3 colspan=1>Leukocytes</td><td rowspan=1 colspan=2>1000 cells/μL</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>33.3%(1/3)²</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>66.7%(2/3)²2</td></tr><tr><td rowspan=1 colspan=2>666.7 cells/μL</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td></tr><tr><td rowspan=1 colspan=2>333.3 cells/μL</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=2>3.0 μmol/L</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>83.3%(5/6)3</td><td rowspan=1 colspan=1>100%(6/6)</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=2>76.0 μmol/L</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>80%(4/5)4</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>80%(4/5)4</td></tr></table>

1 Unexpected negative results were obtained during original and repeat testing, therefore testing was performed at more dilute concentrations until $100 \%$ detection occurred. 2 The original three replicates tested resulted in $3 3 . 3 \%$ (1/3) replicates being invalid and $50 \%$ $( 1 / 2 )$ positivity for hMPV and $100 \%$ positivity (2/2) for Adenovirus, B. pertussis, Flu B, and SARS‐CoV‐2. New test material was prepared and tested, resulting in $6 6 . 6 \%$ (2/3) invalid results, and $0 \% ( 0 / 1 )$ positivity for hMPV and SARS‐CoV‐2. Therefore, testing was performed at more dilute concentrations until $100 \%$ detection occurred. 3 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for influenza B. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for influenza B. 4 The original three replicates tested resulted in $3 3 . 3 \%$ (1/3) replicates being invalid and $50 \%$ $( 1 / 2 )$ positivity for hMPV and SARS‐CoV‐2. New

# Diasorin

est material was prepared and tested, resulting in $100 \%$ (3/3) positivity for hMPV and SARS‐CoV‐2.

# Competitive Inhibition/Co‐infection:

Competitive inhibition of the LIAISON PLEX Respiratory Flex Assay was assessed by testing 27 pairings of clinically prevalent co‐infections, as listed in Table 12. A single pairing consisted of one target at a high concentration with another target at a low concentration (Table 15). All low concentration targets were tested at 3X LoD. All high concentration targets were tested at $1 \times 1 0 ^ { 5 } \mathrm { \top C l D _ { 5 0 } / m L }$ (or equivalent), or the highest available concentration. Testing of each combination was performed in triplicate, at a minimum.

Of the 54 combinations tested, 48 combinations did not show evidence of inhibition and generated the expected results for both targets tested. Interference was observed for the following co‐infections:

Parainfluenza 3 (low concentration) in the presence of human coronavirus OC43 (high concentration of $1 \times 1 0 ^ { 5 } \mathrm { T C l D } _ { 5 0 } / \mathrm { m L } )$ . Competitive interference was no longer observed when the human coronavirus OC43 concentration was decreased to $5 { \times } 1 0 ^ { 4 } \mathrm { T C l D } _ { 5 0 } / \mathrm { m L }$ .   
Parainfluenza 3 (low concentration) in the presence of adenovirus 37D (high concentration of $\mathsf { 1 } \mathsf { x } \mathsf { 1 0 } ^ { \mathsf { 5 } }$ $\mathsf { T C l D } _ { 5 0 } / \mathsf { m L } )$ .   
RSV A (low concentration) in the presence of adenovirus 37D (high concentration of $1 \times 1 0 ^ { 5 } \mathrm { T C l } \mathsf { D } _ { 5 0 } / \mathsf { m L } )$ . Competitive interference was no longer observed when the adenovirus 37D concentration was decreased to $5 { \times } 1 0 ^ { 4 } \mathrm { T C l D } _ { 5 0 } / \mathrm { m L }$ .   
Flu A H3N2 (low concentration) in the presence of adenovirus 37D (high concentration of $\mathsf { 1 } \mathsf { x } \mathsf { 1 0 } ^ { \mathsf { 5 } }$ $\mathsf { T C l D } _ { 5 0 } / \mathsf { m L } )$ . Specifically, detection of influenza A (matrix) was decreased in the presence of adenovirus 37D at a high concentration of $1 \times 1 0 ^ { 5 } \mathrm { \ : T C l { D } _ { 5 0 } / m L }$ . Competitive interference was no longer observed when the adenovirus 37D concentration was decreased to $5 { \times } 1 0 ^ { 4 } \mathrm { T C l D } _ { 5 0 } / \mathrm { m L }$ .   
Human coronavirus 229E (low concentration) in the presence of SARS‐CoV‐2 (high concentration of $1 \times 1 0 ^ { 5 } \mathrm { T C l } \mathsf { D } _ { 5 0 } / \mathsf { m L } )$ .   
SARS‐CoV‐2 (low concentration) in the presence of human coronavirus OC43 (high concentration of $1 \times 1 0 ^ { 5 } \mathrm { T C l D } _ { 5 0 } / \mathrm { m L } )$ .

Table 15. Summary of Competitive Inhibition Results   

<table><tr><td rowspan=1 colspan=2>Target 1 (High Conc.)</td><td rowspan=1 colspan=1>Target 2(Low Conc.)1</td><td rowspan=1 colspan=2>% Detected(# Detected/ # Tested)</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc. (TCID5o/mL)²</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>$1x10{}$</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>$1x10{}$</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x105}$</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>1x10^}$</td><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>66.7% (4/6)3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>5x104</td><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>$1x10}$</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>1x10}$</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x105</td><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Matrix:100% (3/3)Subtype H3:100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>85.7% (6/7)4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>$1x10}$</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

510(k) Summary Page 29 of 45

# Diasorin

<table><tr><td rowspan=1 colspan=2>Target 1 (High Conc.)</td><td rowspan=1 colspan=1>Target 2(Low Conc.)1</td><td rowspan=1 colspan=2>% Detected(# Detected/ # Tested)</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc. (TCIDso/mL)²</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x10}$</td><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>66.7% (4/6)5</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>5x104</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x10}$</td><td rowspan=1 colspan=1>PIV-4</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x105}$</td><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>1x105</td><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>Matrix:66.7% (4/6)6Subtype H3:100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>5x104</td><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Matrix:100% (3/3)Subtype H3:100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x105</td><td rowspan=1 colspan=1>Flu A H1N1</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Matrix:100% (3/3)Subtype H1:100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1x105</td><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>Matrix:100% (3/3)Subtype H3:100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10^r{}$</td><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>83.3% (5/6)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1x10^fr{}$</td><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1x10^{}$</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Adenovirus 3B</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Adenovirus 4E</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1x10^{}$</td><td rowspan=1 colspan=1>Adenovirus 7A</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1x10{}$</td><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1x10^5 (PFU/mL)</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>1x10^fr{}$</td><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>1x105 (CEID5o/mL)</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>1x10fr}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=2>Target 1 (High Conc.)</td><td rowspan=1 colspan=1>Target 2(Low Conc.)1</td><td rowspan=1 colspan=2>% Detected(# Detected/ # Tested)</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc. (TCID5o/mL)²</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1x10^5 (PFU/mL)</td><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PIV-4</td><td rowspan=1 colspan=1>1x105}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>$1x10{}$</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>1x105 (CEID50/mL)</td><td rowspan=1 colspan=1>Adenovirus 37D</td><td rowspan=1 colspan=1>Matrix: 100%(3/3)Subtype H3:100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu A H1N1</td><td rowspan=1 colspan=1>1x105 (CEID50/mL)</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Matrix: 100%(3/3)Subtype H1:100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3N2</td><td rowspan=1 colspan=1>1x105 (CEID5o/mL)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Matrix: 100%(3/3)Subtype H3:100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1x105}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>1x10}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>$1x10^fr{}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>1x10}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>83.3% (5/6)8</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>1.31x104 (copies/mL)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1x10^ (PFU/mL)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3B</td><td rowspan=1 colspan=1>$1x10^r}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 4E</td><td rowspan=1 colspan=1>1x105$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7A</td><td rowspan=1 colspan=1>$1x10^}$</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 Low concentration target organisms were prepared at 3x LoD. 2 Concentrations are in $\mathtt { T C l D 5 0 } / \mathtt { m L }$ , unless otherwise noted. 3 Unexpected negative results were obtained during original and repeat testing for parainfluenza 3, therefore testing was performed at more dilute coronavirus concentrations until $100 \%$ detection of parainfluenza 3 occurred.

4 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for parainfluenza 3. New test material was prepared and tested, resulting in $100 \%$ (4/4) positivity for parainfluenza 3. Four replicates were performed during resting, rather than three because a single false positive SARS‐CoV‐2 result was obtained during retesting. 5 Unexpected negative results were obtained during original and repeat testing for RSV A, therefore testing was performed at more dilute adenovirus concentrations until $100 \%$ detection of RSV A occurred.

Unexpected negative results were obtained during original and repeat testing for influenza A (matrix), therefore testing was performed at more dilute adenovirus oncentrations until $100 \%$ detection of influenza A (matrix) occurred.

The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for coronavirus. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity fo coronavirus.

8 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for SARS‐CoV‐2. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity fo SARS‐CoV‐2.

# Carry‐Over and Cross‐Contamination:

Carry‐over and cross contamination for the LIAISON PLEX Respiratory Flex Assay was evaluated by testing positive and negative samples in an alternating series. A multi‐analyte panel consisting of five assay targets (B. pertussis, Adenovirus, influenza B, human metapneumovirus, and SARS‐CoV‐2) at a high concentration (B. pertussis at $1 { \times } 1 0 ^ { 6 } \thinspace { \mathsf { C F U / m L } }$ and the viral targets at $1 \times 1 0 ^ { 5 } \mathrm { T C l D } _ { 5 0 } / \mathrm { m L } )$ was prepared in simulated NPS matrix  and used for positive samples. Simulated negative NPS matrix was used for negative samples. Positive samples were tested in modules adjacent to negative samples in order to evaluate possible cross contamination. Immediately following the testing of a positive sample, a negative sample was run in the

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission

same module to evaluate possible carry‐over contamination. This alternating series of positive and negative samples was continued across five consecutive runs using two LIAISON PLEX Systems for a total of 30 positive and 30 negative tests. No carry‐over or cross contamination was observed.

g. Assay cut‐off

The specific assay parameters for the LIAISON PLEX® Respiratory Flex Assay are considered confidential and proprietary.

Comparison Studies:

h. Method comparison with predicate device: Refer to Section 2 Clinical Performance.

i. Matrix Comparison: Not applicable

2. Clinical Performance:

# Prospective Clinical Evaluation

A multi‐site prospective clinical study established the clinical performance of the LIAISON PLEX Respiratory Flex Assay for the detection and identification of bacteria and viral targets from nasopharyngeal swab (NPS) specimens transported in Copan Universal Transport Medium™ or BD™ Universal Viral Transport Media, collected from patients exhibiting clinical signs and symptoms of respiratory tract infection (RTI). The clinical performance of the LIAISON PLEX Respiratory Flex Assay was evaluated using NPS clinical specimens prospectively collected between October 2022 to April 2023 from six geographically diverse clinical sites within the United States. The clinical study included remnant, de‐identified specimens collected from pediatric and adult patients exhibiting clinical signs and symptoms of respiratory tract infections. Specimens were stored refrigerated at ${ 2 } { - } 8 ^ { \circ } \mathrm { C }$ for up to 72‐hours before testing (i.e., Category I specimens) or if they could not be tested within 72‐hours, after freezing at ${ } _ { - 7 0 } \circ _ { \mathrm { C } }$ (Category II specimens).

A total of 1911 unique prospective specimens that met the pre‐determined inclusion criteria were enrolled in the study. Clinical runs and re‐runs using LIAISON PLEX Respiratory Flex Assay were tested on the LIAISON PLEX System by trained operators at four clinical sites. Out of the 1911 specimens enrolled in the prospective study, 68 specimens were disqualified and removed from further analysis. Most of the specimen exclusions were due to non‐compliance with the study protocol or due to not meeting the inclusion criteria after enrollment. This left 1843 clinical specimens for evaluation. Of these 1843 specimens, $6 6 . 3 \%$ (1221/1843) were tested fresh, while $3 3 . 7 \%$ (622/1843) were tested frozen. Patient demographic information for the 1843 prospectively collected NPS specimens is presented in Table 16.

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># Specimens (%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Gender</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>839 (45.5%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>1004 (54.5%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1843 (100.0%)</td></tr><tr><td rowspan=1 colspan=2>Age (years)</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>350 (19.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>274 (14.9%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>447 (24.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>535 (29.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65</td><td rowspan=1 colspan=1>237 (12.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1843 (100.0%)</td></tr><tr><td rowspan=1 colspan=2>Subject Status</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>590 (32.0%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>317 (17.2%)</td></tr><tr><td rowspan=1 colspan=1>Emergency Room</td><td rowspan=1 colspan=1>913 (49.5%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>23 (1.2%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1843 (100.0%)</td></tr></table>

The LIAISON PLEX Respiratory Flex Assay was evaluated for prospective clinical performance by comparing to an FDA‐ cleared molecular respiratory panel for all analytes, except the following: SARS‐CoV‐2, B. holmesii, B. parapertussis, and B. pertussis. Performance for SARS‐CoV‐2 was evaluated by comparing to an FDA‐cleared molecular SARS‐CoV‐2 assay. Performance for the denoted Bordetella species was based on comparison to well‐validated Fragment Analysis (FA) assays followed by PCR/Bi‐Directional Sequencing (PCR/BDS) assays (see Table 17).

# Diasorin

Table 17. Comparator Methods for the LIAISON PLEX Respiratory Flex Assay Clinical Study   

<table><tr><td rowspan=1 colspan=1>LIAISON PLEX Respiratory Flex Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus (inclusive to A, B, C , D, E, and F)</td><td rowspan=11 colspan=1>FDA-Cleared MolecularRespiratory Panel</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus (inclusive to HKU1, NL63, OC43, and 229E)</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H1</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H3</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td></tr><tr><td rowspan=1 colspan=1>RSV (inclusive to RSV A and RSV B)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>FDA-Cleared Molecular SARS-CoV-2 Assay, Cleared Under 21CFR 866.3981</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=3 colspan=1>Analytically Validated FragmentAnalysis Assays Followed byPCR/Bi-Directional Sequencing</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td></tr></table>

Out of the 1843 prospective clinical specimens included in the prospective study analysis, $9 5 . 2 \%$ (1755/1843) generated valid Respiratory Flex Assay results (i.e., detected or not detected) on the first attempt, for an initial invalid rate of $4 . 8 \%$ (88/1843). Of the 88 specimens with initial invalid results, 77 produced valid results on repeat, 6 specimens remained invalid on repeat, and 5 specimens were not retested, resulting in a final invalid rate of $0 . 6 \%$ (11/1843). This left 1832 specimens with valid Respiratory Flex Assay results. The final number of evaluable results varied by target based on the number of valid comparator method results obtained.

Clinical Performance (Positive Percent Agreement, Negative Percent Agreement, and $9 5 \%$ confidence interval) of the LIAISON PLEX Respiratory Flex Assay vs the comparator method(s)is summarized in Table 18 for prospective specimens. Positive Percent Agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathsf { T P } / \left( \mathsf { T P } + \mathsf { F N } \right) \right)$ . True positive (TP) indicates that both the Respiratory Flex Assay and the comparator method had a positive result for the specific analyte, and false negative (FN) indicates that the Respiratory Flex Assay was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathsf { T N } / \left( \mathsf { T N } + \mathsf { F P } \right) \right)$ . True negative (TN) indicates that both the Respiratory Flex Assay and the comparator method had negative results, and false positive (FP) indicates that the Respiratory Flex Assay was positive while the comparator result was negative. Specimens that obtained discordant results underwent additional testing with either an FDA‐cleared molecular respiratory panel or PCR/BDS for investigation.

# Diasorin

Table 18. Prospective Clinical Performance of the LIAISON PLEX Respiratory Flex Assay with NPS Specimens   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>75/75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1-100</td><td rowspan=1 colspan=1>1074/1129</td><td rowspan=1 colspan=1>95.1</td><td rowspan=1 colspan=1>93.7-96.2</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>83.2-100</td><td rowspan=1 colspan=1>578/597</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>95.1-98.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>94/94</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.1-100</td><td rowspan=1 colspan=1>1652/17261</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>94.7-96.6</td></tr><tr><td rowspan=3 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1127/1127</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>603/603</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1730/1730</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>1161/1163</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0-79.3</td><td rowspan=1 colspan=1>604/605</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.1-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>37.6-96.4</td><td rowspan=1 colspan=1>1765/1768²</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.5-99.9</td></tr><tr><td rowspan=3 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1146/1146</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>607/607</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1753/1753</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1204/1204</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>616/616</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1820/1820</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>Human Coronavirus</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>90/97</td><td rowspan=1 colspan=1>92.8</td><td rowspan=1 colspan=1>85.8-96.5</td><td rowspan=1 colspan=1>1100/1107</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.7-99.7</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>27/33</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>65.6-91.4</td><td rowspan=1 colspan=1>582/583</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.0-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>117/130^³3</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>83.6-94.1</td><td rowspan=1 colspan=1>1682/16904</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>99.1-99.8</td></tr><tr><td rowspan=3 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>230/242</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>91.5-97.1</td><td rowspan=1 colspan=1>937/962</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>96.2-98.2</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>81/90</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>82.1-94.6</td><td rowspan=1 colspan=1>518/526</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>97.0-99.2</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>311/3325</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>90.5-95.8</td><td rowspan=1 colspan=1>1455/14886</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>96.9-98.4</td></tr><tr><td rowspan=3 colspan=1>hMPV</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>113/118</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>90.5-98.2</td><td rowspan=1 colspan=1>1080/1086</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.8-99.7</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7-98.6</td><td rowspan=1 colspan=1>603/603</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>125/131</td><td rowspan=1 colspan=1>95.4</td><td rowspan=1 colspan=1>90.4-97.9</td><td rowspan=1 colspan=1>1683/16898</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.2-99.8</td></tr><tr><td rowspan=3 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>1185/1186</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>111/111</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7-100</td><td rowspan=1 colspan=1>490/505</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>95.2-98.2</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>129/129</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.1-100</td><td rowspan=1 colspan=1>1675/1691</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>98.5-99.4</td></tr><tr><td rowspan=3 colspan=1>Influenza ASubtype H1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100</td><td rowspan=1 colspan=1>1187/1188</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>84.5-100</td><td rowspan=1 colspan=1>595/595</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100</td><td rowspan=1 colspan=1>1782/178310</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=3 colspan=1>Influenza ASubtype H3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7</td><td rowspan=1 colspan=1>20.8-93.9</td><td rowspan=1 colspan=1>1200/1201</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>102/104</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>93.3-99.5</td><td rowspan=1 colspan=1>509/512</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.3-99.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>104/10711</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>92.1-99.0</td><td rowspan=1 colspan=1>1709/171312</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.4-99.9</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>1200/1200</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>612/612</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100</td><td rowspan=1 colspan=1>1812/1812</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1204/1204</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>616/616</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>1820/1820</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>52.9-97.8</td><td rowspan=1 colspan=1>1196/1196</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>612/612</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>11/1213</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>64.6-98.5</td><td rowspan=1 colspan=1>1808/1808</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>59.6-98.2</td><td rowspan=1 colspan=1>1194/1194</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100</td><td rowspan=1 colspan=1>613/613</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7-98.6</td><td rowspan=1 colspan=1>1807/1807</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>37/39</td><td rowspan=1 colspan=1>94.9</td><td rowspan=1 colspan=1>83.1-98.6</td><td rowspan=1 colspan=1>1164/1165</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>37.6-96.4</td><td rowspan=1 colspan=1>611/611</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>41/4414</td><td rowspan=1 colspan=1>93.2</td><td rowspan=1 colspan=1>81.8-97.7</td><td rowspan=1 colspan=1>1775/177615</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>1199/1200</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>37.6-96.4</td><td rowspan=1 colspan=1>611/611</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>8/916</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>56.5-98.0</td><td rowspan=1 colspan=1>1810/181117</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=3 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>37/38</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>86.5-99.5</td><td rowspan=1 colspan=1>1166/1166</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>81/85</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>88.5-98.2</td><td rowspan=1 colspan=1>531/531</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>118/12318</td><td rowspan=1 colspan=1>95.9</td><td rowspan=1 colspan=1>90.8-98.3</td><td rowspan=1 colspan=1>1697/1697</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>178/183</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>93.8-98.8</td><td rowspan=1 colspan=1>996/1000</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.0-99.8</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>68/72</td><td rowspan=1 colspan=1>94.4</td><td rowspan=1 colspan=1>86.6-97.8</td><td rowspan=1 colspan=1>521/525</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>98.1-99.7</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>246/25519</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>93.4-98.1</td><td rowspan=1 colspan=1>1517/152520</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>99.0-99.7</td></tr></table>

P – true positive; FN – false negative; TN – true negative; FP – false positive; NE – not evaluable 1Of the 74 specimens with false positive adenovirus results by the Respiratory Flex Assay, 21 were positive by an FDA‐cleared molecular respiratory panel, 21 were negative, and 32 were not tested.

the 3 specimens with false positive Bordetella parapertussis results by the Respiratory Flex Assay, 1 was negative by an FDA‐cleared molecular respiratory panel nd 2 were not tested.

3 Of the 13 specimens with false negative coronavirus results by the Respiratory Flex Assay, 3 were negative by PCR/BDS, 9 were positive, and 1 was not tested. $^ 4 { \cal O } \dagger$ the 8 specimens with false positive coronavirus results by the Respiratory Flex Assay, 5 were positive by PCR/BDS, 2 were negative, and 1 was not tested. 5 Of the 21 specimens with false negative enterovirus/rhinovirus results by the Respiratory Flex Assay, 9 were positive by PCR/BDS, 8 were negative, and 4 were no tested.

6 Of the 33 specimens with false positive enterovirus/rhinovirus results by the Respiratory Flex Assay, 4 were positive by PCR/BDS, 27 were negative, and 2 were not tested.

7 Of the 6 specimens with false negative hPMV results by the Respiratory Flex Assay, 4 were positive by PCR/BDS and 2 were negative.   
$^ 8 { \cal O } \mathsf { f }$ the 6 specimens with false positive hPMV results by the Respiratory Flex Assay, 4 were positive by PCR/BDS and 2 were negative.

$^ { 9 } { \mathsf { O } } { \mathsf { f } }$ the 16 specimens with false positive influenza A results by the Respiratory Flex Assay, 7 were positive by PCR/BDS and 9 were negative specimen with a false positive influenza A subtype H1 result by the Respiratory Flex Assay was negative by PCR/BDS.

11The 3 specimens with false negative influenza A subtype H3 results by the Respiratory Flex Assay were all negative by PCR/BDS.

13The 1 specimen with a false negative parainfluenza 1 result by the Respiratory Flex Assay was positive by PCR/BDS.

$^ { 1 4 } { \sf O } { \sf f }$ the 3 specimens with false negative parainfluenza 3 results by the Respiratory Flex Assay, 2 were negative by PCR/BDS and 1 was not tested.

5The 1 specimen with a false positive parainfluenza 3 result by the Respiratory Flex Assay was negative by PCR/BDS.

The 1 specimen with a false negative parainfluenza 4 result by the Respiratory Flex Assay was negative by PCR/BDS.

$^ { 1 8 } { \sf O } { \sf f }$ the 5 specimens with false negative RSV results by the Respiratory Flex Assay, 1 was negative by PCR/BDS, and 3 were negative by an FDA‐cleared molecular Flu/RSV assay.

$^ { 1 9 } { \sf O } { \sf f }$ the 9 specimens with false negative SARS‐CoV‐2 results by the Respiratory Flex Assay, 5 were positive by PCR/BDS, 2 were negative, and 2 were not tested.   
$^ { 2 0 } { \sf O } { \sf f }$ the 8 specimens with false positive SARS‐CoV‐2 results by the Respiratory Flex Assay, 5 were positive by PCR/BDS, 2 were negative, and 1 was not tested.

The LIAISON PLEX Respiratory Flex Assay reported multiple organism detections in a total of 176 prospective specimens as shown in Table 19. Of these 176 specimens, comparator results were unavailable for at least 1 of the organisms identified in the co‐infection for 3 specimens, which were excluded from further analysis. The remaining 173 co‐ infections represent $1 4 . 6 \%$ (173/1187) of positive prospective specimens and $9 . 4 \%$ (173/1840) of all prospective specimens. The majority of co‐infections, $8 7 . 3 \%$ (151/173), contained two organisms, while $1 1 . 6 \%$ (20/173) of coinfections contained three organisms, and $1 . 2 \%$ (2/173) contained 4 organisms. Out of the 173 specimens with multipl

# Diasorin

detections, $4 5 . 5 \%$ (77/173) contained one or more organisms that were not detected by the comparator method(s) (Table 19). Co‐infections identified by the comparator methods which were not reported by the LIAISON PLEX Respiratory Flex Assay are illustrated in Table 20.

Table 19. Co-infections Reported by the LIAISON PLEX Respiratory Flex Assay in the Prospective Study   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte 4</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISONPLEXRespiratoryFlex AssayFalsePositives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalsePositiveAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (2),Bordetellaparapertussis (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Parainfluenza3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Adenovirus (17),Enterovirus/Rhinovirus (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus,Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Adenovirus</td></tr></table>

# Diasorin

<table><tr><td></td><td></td><td></td><td></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte 4</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISONPLEXRespiratoryFlex AssayFalsePositives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalsePositiveAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Parainfluenza2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Parainfluenza1</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Parainfluenza2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Parainfluenza3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>BordetellaparapertussiS</td><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Bordetellaparapertussis</td></tr><tr><td rowspan=1 colspan=1>BordetellaparapertussiS</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Bordetellaparapertussis</td></tr><tr><td rowspan=1 colspan=1>BordetellaparapertussiS</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>BordetellaparapertussiS</td><td rowspan=1 colspan=1>Parainfluenza3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>HumanCoronavirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

# Diasorin

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte 4</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISONPLEXRespiratoryFlex AssayFalsePositives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalsePositiveAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>HumanCoronavirus</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Parainfluenza3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HumanCoronavirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Influenza A(subtype H3)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H3,Influenza A,Enterovirus/Rhinovirus, SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3,Influenza A</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Parainfluenza 4</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

# Diasorin

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte 4</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISONPLEXRespiratoryFlex AssayFalsePositives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalsePositiveAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>Influenza A&amp; InfluenzaA (subtypeH1</td><td rowspan=1 colspan=1>Influenza A(subtype H3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Influenza A&amp; InfluenzaA (subtypeH1)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Influenza A&amp; InfluenzaA (subtypeH3</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=4>Total</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Total Double Infections</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Total Triple Infections</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Total Quadruple Infections</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr></table>

# Table 20. Co-infections Identified by the Comparator Methods which were Not Reported by the LIAISON PLEX Respiratory Flex Assay in the Prospective Study

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalseNegatives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory Flex AssayFalse Negative Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Human Coronavirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Enterovirus/</td></tr></table>

# Diasorin

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalseNegatives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory Flex AssayFalse Negative Analyte(s)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Parainfluenza 2</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Human Coronavirus (1),Parainfluenza 3 (1)</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Parainfluenza 4</td></tr><tr><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>BordetellaParapertussis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Bordetellaparapertussis</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H1)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A(subtype H3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A (subtype H3)</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>Influenza A &amp;Influenza A(subtype H3)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus(2), SARS-CoV-2 (1)</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3>Number of Specimens</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory FlexAssay FalseNegatives</td><td rowspan=1 colspan=1>LIAISON PLEXRespiratory Flex AssayFalse Negative Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Parainfluenza 3 (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Double Infections</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Triple Infections</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td></tr></table>

# Testing of Preselected Archived Specimens

A number of analytes on the LIAISON PLEX Respiratory Flex Assay were of low prevalence during the prospective study and were not encountered in large enough numbers to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective NPS specimens was performed.

A total of 256 pre‐selected left‐over frozen, de‐identified specimens (Category III specimens) sourced from four sites/vendors in the United States were obtained and tested at three US sites. Pre‐selected specimen collection dates ranged from November 2013 through June 2023. Pre‐ selected specimens were characterized by the same comparator methods as the prospective study (described above). The pre‐selected specimens were tested in a randomized, blinded manner with negative specimens. A summary of the available demographic information of the tested specimens is provided in Table 21. Out of the 256 specimens included in the pre‐selected study analysis, 241 $( 9 4 . 1 \% )$ generated valid Respiratory Flex Assay results (i.e., Detected or Not Detected) on the first attempt. There were 15 specimens $( 5 . 9 \% )$ with invalid results on the initial run that required retesting. Of the specimens with initial invalid results, all 15 specimens generated valid Respiratory Flex Assay results after retest for a final success rate of $100 \%$ (256/256). The results of the LIAISON PLEX Respiratory Flex Assay performance for these archived specimens are shown in Table 22.

Table 21. Archived Specimen Demographic Details $( N = 2 5 6 )$   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># Specimens (%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Gender</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>117(45.7%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>124(48.4%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>15(5.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>256(100.0%)</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=2>Age (years)</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>44(17.2%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>53(20.7%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>69(27.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>44(17.2%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65</td><td rowspan=1 colspan=1>32(12.5%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>14(5.5%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>256(100.0%)</td></tr><tr><td rowspan=1 colspan=2>Subject Status</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>EmergencyRoom</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>256 (100.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>256 (100.0%)</td></tr></table>

Table 22. LIAISON PLEX Respiratory Flex Assay Archived Performance Summary for NPS Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>NegativePercent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100</td><td rowspan=1 colspan=1>241/2501</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>93.3-98.1</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>234/234</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100</td><td rowspan=1 colspan=1>233/236</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>96.3-99.6</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>214/217</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.0-99.5</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>68.5-98.7</td><td rowspan=1 colspan=1>241/242</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.7-99.9</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>249/252</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>96.6-99.6</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>24/272</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>71.9-96.1</td><td rowspan=1 colspan=1>223/2293</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>94.4-98.8</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>255/255</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>254/2554</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.8-99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>254/2554</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.8-99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>256/256</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>232/233</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.6-99.9</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>79.8-99.3</td><td rowspan=1 colspan=1>226/232</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>94.5-98.8</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>237/238</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.7-99.9</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>76.4-99.1</td><td rowspan=1 colspan=1>235/236</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.6-99.9</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>254/254</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>230/2335</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>96.3-99.6</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100</td><td rowspan=1 colspan=1>246/247</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.7-99.9</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; NE – not evaluable

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission

1Of the 9 specimens with false positive adenovirus results by the Respiratory Flex Assay, seven were negative by PCR/BDS and two were not tested.

2Of the 3 specimens with false negative enterovirus/rhinovirus results by the Respiratory Flex Assay, one was negative by PCR/BDS, one was positive by PCR/BDS, and one was not tested.

3Of the 6 specimens with false positive enterovirus/rhinovirus results by the Respiratory Flex Assay, four were negative by PCR/BDS and two were not tested.

4The 1 specimen with a false positive influenza A/influenza A H1 result by the Respiratory Flex Assay was negative for influenza A by PCR/BDS and not tested for influenza A H1.

5Of the 3 specimens with false positive parainfluenza 4 results, one was negative by PCR/BDS and two were not tested.

# Contrived Specimen Testing

Contrived specimens were tested to supplement the positive clinical specimens in the prospective and pre‐selected study cohorts for low prevalence targets, including Bordetella holmesii, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae. Positive contrived specimens for influenza A H1N1 pdm09 were prepared and tested prior to completion of the prospective clinical study, in anticipation of potentially low prevalence for influenza A H1. The prospective clinical study ended up yielding an adequate number of influenza A H1 positive specimens to demonstrate performance, however since the contrived data was already acquired, it’s presented here . A total of 300 specimens were contrived, blinded, randomized and tested along with negative specimens at two testing sites during August 2023.

Out of the 300 specimens included in the contrived study analysis, 291 specimens $( 9 7 . 0 \% )$ generated valid RSP Flex Assay results (i.e., Detected or Not Detected) on the first attempt. There were 9 specimens $( 3 . 0 \% )$ with an invalid result on the initial run.  Of the 9 specimens retested, all 9 generated a valid result after a single retest for a final success rate of $100 \%$ (300/300).

Results from contrived specimen testing with the LIAISON PLEX Respiratory Flex Assay are shown in Table 23.

Table 23. LIAISON PLEX Respiratory Flex Assay Performance with Contrived Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>TargetConc.(xLoD)</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=4 colspan=1>Bordetellaholmesii</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.4-100</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100</td><td rowspan=1 colspan=1>250/250</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=4 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7-98.6</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.4-100</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.5-99.6</td><td rowspan=1 colspan=1>250/250</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=4 colspan=1>Bordetellapertussis</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.4-100</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100</td><td rowspan=1 colspan=1>250/250</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Chlamydia</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr></table>

# Diasorin

LIAISON PLEX® Respiratory Flex Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>TargetConc.(xLoD)</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=3 colspan=1>pneumoniae</td><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.4-100</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100</td><td rowspan=1 colspan=1>250/250</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=4 colspan=1>Influenza AH1N1 pdm09</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>80.5-99.3</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.4-100</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.5-99.6</td><td rowspan=1 colspan=1>250/250</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=4 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>80.5-99.3</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100</td><td rowspan=1 colspan=1>64/65</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>91.8-99.7</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.5-99.6</td><td rowspan=1 colspan=1>249/250</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.8-99.9</td></tr></table>

# N. Proposed Labeling:

The labeling provided in the submission satisfies the requirements of 21 CFR 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.