# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K233100   
B Applicant QIAGEN GmbH   
C Proprietary and Established Names QIAstat-Dx Respiratory Panel Plus   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

QIAGEN is formally submitting this Traditional $5 1 0 ( \mathrm { k } )$ Premarket Notification for the QIAstatDx Respiratory Panel Plus, which is a modified version of the previously cleared QIAstat-Dx Respiratory Panel (K183597). The modified panel received an EUA (EUA200075) and this premarket notification is to receive 510(k) clearance of the EUA assay version.

# B Measurand:

The QIAstat Respiratory Panel Plus detects and identifies nucleic acids from the following pathogens:

Severe Acute Respiratory Syndrome (SARS)-Coronavirus-2 (SARS-CoV-2) [R   
RNA-polymerase gene (RdRp) & Envelope protein gene (E)]   
Adenovirus [Hexon gene]   
Human Coronavirus 229E [Membrane protein gene (M)]   
Human Coronavirus HKU1 [Nucleocapsid protein gene (N)]   
Human Coronavirus NL63 [Nucleocapsid protein gene (N)]   
Human Coronavirus OC43 [Nucleocapsid protein gene (N)]   
Human Metapneumovirus $\mathbf { A } { + } \mathbf { B }$ [A/B Nucleoprotein gene (N)]   
Influenza A [Matrix gene (M)]   
Influenza A H1 [Hemagglutinin gene (HA)]   
Influenza A H1N1 pdm09 [Neuraminidase gene (NA)]   
Influenza A H3 [Hemagglutinin gene (HA)]   
Influenza B [Nucleoprotein gene (M)]   
Parainfluenza virus 1 [Hemagglutinin-neuraminidase glycoprotein gene (HN)]   
Parainfluenza virus 2 [Hemagglutinin-neuraminidase glycoprotein gene (HN)]   
Parainfluenza virus 3 [Phosphoprotein gene (P)]   
Parainfluenza virus 4 [Nucleocapsid protein gene (N)]   
Respiratory Syncytial Virus $\mathbf { A } { + } \mathbf { B }$ [Matrix protein gene (M)]   
Rhinovirus/Enterovirus\* [ $5 '$ -UTR region]   
Bordetella pertussis [Transposase Insertion Sequence (IS481)]   
Chlamydophila pneumoniae [Major outer membrane protein gene (ompA)]   
Mycoplasma pneumoniae [Cytadhesin gene (P1)]

\*Both Rhinovirus and Enterovirus are detected but not differentiated.

# C Type of Test:

Multiplexed real-time polymerase chain reaction (RT-PCR) nucleic acid assay for use with the QIAstat-Dx System for the qualitative detection of viral and bacterial pathogens in individuals with signs and symptoms of respiratory tract infection.

# III Intended Use/Indications for Use: Intended Use(s):

The QIAstat-Dx Respiratory Panel Plus is a multiplexed nucleic acid test intended for use with the QIAstat-Dx system for the simultaneous in vitro qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of respiratory tract infection, including SARS-CoV-2.

The following organism types and subtypes are identified using the QIAstat-Dx Respiratory Panel Plus: Adenovirus, Human Coronavirus 229E, Human Coronavirus HKU1, Human Coronavirus NL63, Human Coronavirus OC43, Human Metapneumovirus, Influenza A, Influenza A H1, Influenza A H1N1 pdm09, Influenza A H3, Influenza B, Parainfluenza virus 1,

Parainfluenza virus 2, Parainfluenza virus 3, Parainfluenza virus 4, Respiratory Syncytial Virus, Human Rhinovirus/Enterovirus (not differentiated), Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2), Bordetella pertussis, Chlamydophila pneumoniae and Mycoplasma pneumoniae.

Nucleic acids from viral and bacterial organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. Detecting and identifying specific viral and bacterial nucleic acids from individuals presenting with signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection, if used in conjunction with other clinical, epidemiological and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Negative results in the presence of a respiratory illness may be due to infection with pathogens that are not detected by the test or due to lower respiratory tract infection that is not detected by a NPS specimen.

Conversely, positive results are indicative of the presence of the identified microorganism, but do not rule out co-infection with other pathogens not detected by the QIAstat-Dx Respiratory Panel Plus. The agent(s) detected by the QIAstat-Dx Respiratory Panel Plus may not be the definite cause of disease.

The use of additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

A Indication(s) for Use: See Intended Use (above)

B Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

For in vitro Diagnostic Use Only

# C Special Instrument Requirements:

The QIAstat-Dx Respiratory Panel Plus is part of the QIAstat-Dx system and works only with the QIAstat-Dx Analyzer 1.0.

# IV Device/System Characteristics:

# A Device Description:

The QIAstat-Dx Respiratory Panel Plus is part of the QIAstat-Dx system and works with the QIAstat-Dx Analyzer 1.0. The QIAstat-Dx Respiratory Panel Plus Cartridge is a single-use cartridge that includes all reagents needed for nucleic acid extraction, nucleic acid amplification, and detection of bacteria and viruses (or their subtypes), including SARSCoV-2, that cause respiratory symptoms. Testing requires a small specimen volume and minimal hands-on time, and the results are available in approximately one hour.

The QIAstat-Dx Respiratory Panel Plus is intended to be used with one (l) nasopharyngeal swab (NPS) eluted in Universal Transport Media (UTM), which is not provided with the QIAstat-Dx Respiratory Panel Plus.

The QIAstat-Dx Respiratory panel initially received FDA clearance in May 2019 (K183597). In March 2020, the QIAstat-Dx Respiratory SARS-CoV-2 Panel received EUA authorization which included the addition of SARS-CoV-2 reagents to an independent reaction chamber (i.e., not multiplexed with other analyte reagents) and a reduction in concentration of a lysis reagent. The QIAstat-Dx Respiratory Panel Plus is identical to the QIAstat-Dx Respiratory SARS-CoV-2 Panel. The other analyte reagents are unmodified from that cleared in K183597.

# B Principle of Operation:

The test starts automatically upon insertion of the test cartridge and runs for approximately 1 hour. When the test is finished, the cartridge is removed by the user and discarded. The QIAstat-Dx Analyzer 1.0 automatically interprets test results and displays a summary on the analyzer display screen. The results can be printed using a connected printer if needed. The detected analytes are displayed in red. All other tested but not detected analytes are listed in green. The analyzer will report if an error occurs during processing, in which case the test will fail, and no results will be provided (screen will show “FAIL”).

All the reagents required for the complete execution of the test are pre-loaded and selfcontained in the QIAstat-Dx Respiratory Panel Plus cartridge. The user does not need to manipulate any reagents. During the test, reagents are handled by pneumatically operated microfluidics without any direct contact with the user or the analyzer actuators.

Within the cartridge, multiple steps are automatically performed in sequence by using pneumatic pressure and a multiport valve to transfer sample and fluids via the Transfer Chamber (TC) to their intended destinations. Following the introduction of the sample from a disposable transfer pipette, the following assay steps occur automatically and sequentially:

Resuspension of air-dried internal control and Proteinase K (ProtK) enzyme using provided buffer and mixing with the liquid sample (IC Cavity and ProtK Cavity).   
• Cell lysis using mechanical (rotation) and chemical (chaotropic and isotonic) means (lysis chamber).   
• Membrane based nucleic acid purification from lysate by: Mixing lysate with binding buffer and capturing on the membrane (purification chamber). First washing of membrane to remove bound proteins (purification chamber and waste chamber). Second washing of membrane to leave only bound nucleic acids (purification chamber and waste chamber). Rinsing of Transfer Chamber (TC) using the rinsing buffer before introduction of the eluate (TC).

Drying of membrane with bound nucleic acids with air flow generated by a high flow vacuum pump (purification chamber). Elution of nucleic acids with elution buffer (purification chamber and TC).

Mixing of the purified nucleic acid (eluate) with lyophilized “Master Mix” reagents (dry chemistry container and TC). Sequential transfer of defined aliquots of mixed eluate/Master Mix from the Transfer Chamber to each of eight Reaction Chambers containing the specified, air dried primers and probes. Note that each target specific probe contains a specific fluorescent dye that permits differentiation of target specific amplification within each Reaction Chamber). • Within each Reaction Chamber, real-time, multiplex PCR (“rtPCR”) testing is performed. Increase in fluorescence (indicative of detection of each target analyte) is detected directly within each Reaction Chamber. The detected signal per fluorescent marker per Reaction Chamber is then used by the system software to generate the assay result.

# C Instrument Description Information:

1. Instrument Name:

QIAstat-Dx Analyzer 1.0, with software version 1.4 or higher.

2. Specimen Identification:

The QIAstat-Dx Analyzer is a fully automated instrument that is bi-directionally interfaced and accepts orders from the LIS system. While test orders can be manually programmed through the attached instrument computer, a LIS-generate barcode can also be scanned to initiate testing.

3. Specimen Sampling and Handling: Nasopharyngeal swab (NPS) specimens collected in transport medium.

4. Calibration:

The QIAstat-Dx Analyzer 1.0 is provided factory calibrated and does not require user calibration.

5. Quality Control:

# Control Materials

# a) External Controls

External Controls are not provided with the QIAstat-Dx Respiratory Panel Plus. The following information on external control testing is included in the Instructions for Use, “External controls are not provided with the QIAstat-Dx Respiratory Panel Plus. Quality control requirements should be performed in conformance with local, state, and/or federal regulations or accreditation requirements and your laboratory’s standard quality control procedures.”

External controls for performance studies were commercially obtained: NATtrol ZeptoMetrix Respiratory Verification Panel (cat. id NATRVP-QIA) and ZeptoMetrix NATtrol SARS-CoV-2 (E/ORF1ab) recombinant (cat. id. 0831043). These were diluted with negative transport medium prior to use. The negative sample was prepared using transport medium only. The controls yielded expected results on all days of testing.

# b) Internal Controls

The QIAstat-Dx Respiratory Panel Cartridge includes an Internal Control (IC), which is a titered lyophilized MS2 bacteriophage that provides verification that all steps of the analysis process including sample resuspension, lysis, nucleic acid purification, reverse transcription, and PCR were successful. The results screen displays a message indicating that the internal controls “Passed” when the test was run successfully. An IC message of “Failed” indicates that the internal control was not amplified; ‘Positive’ test results are still reported as positive, but all ‘Negative’ results are invalid.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s):

DEN200031

# C. Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>Subject Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>QIAstat-Dx Respiratory Panel Plus</td><td rowspan=1 colspan=1>BioFire Respiratory Panel 2.1 (RP2.1)</td></tr><tr><td rowspan=1 colspan=1>De Novo/510(k) No.</td><td rowspan=1 colspan=1>K233100</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Regulation</td><td rowspan=1 colspan=1>21 CFR 866.3981</td><td rowspan=1 colspan=1>21 CFR 866.3981</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>QOF</td></tr><tr><td rowspan=1 colspan=1>Device Class</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>Class II</td></tr><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The QIAstat-Dx Respiratory PanelPlus is a multiplexed nucleic acid testintended for use with the QIAstat-Dxsystem for the simultaneous in vitroqualitative detection andidentification of multiple respiratoryviral and bacterial nucleic acids innasopharyngeal swabs (NPS)obtained from individuals with</td><td rowspan=1 colspan=1>The BioFire Respiratory Panel 2.1(RP2.1) is a PCR-based multiplexednucleic acid test intended for use withthe BioFire FilmArray 2.0 or BioFireFilmArray Torch systems for thesimultaneous qualitative detection andidentification of multiple respiratoryviral and bacterial nucleic acids innasopharyngeal swabs (NPS) obtained</td></tr></table>

Predicate   
from individuals suspected of   
respiratory tract infections, including   
COVID-19.   
The following organism types and   
subtypes are identified using the   
BioFire RP2.1:   
• Adenovirus,   
• Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43,   
• Severe Acute Respiratory   
Syndrome Coronavirus (SARS-CoV  
2),   
• Human Metapneumovirus,   
• Human   
Rhinovirus/Enterovirus,   
• Influenza A, including   
subtypes H1, H1-2009, and H3,   
• Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Respiratory Syncytial Virus,   
• Bordetella parapertussis   
(IS1001),   
• Bordetella pertussis (ptxP),   
• Chlamydia pneumoniae, and Mycoplasma pneumoniae

Nucleic acids from the respiratory viral and bacterial organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection is indicative of the presence of the identified microorganism and aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>Subject Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>The use of additional laboratorytesting (e.g., bacterial and viralculture, immunofluorescence andradiography) may be necessary whenevaluating a patient with possiblerespiratory tract infection.</td><td rowspan=1 colspan=1>coinfection with other organisms. Theagent(s) detected by the BioFire RP2.1may not be the definite cause ofdisease. Additional laboratory testing(e.g. bacterial and viral culture,immunofluorescence, and radiography)may be necessary when evaluating apatient with possible respiratory tractinfection.</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Nasopharyngeal swabs (NPS)</td><td rowspan=1 colspan=1>Nasopharyngeal swabs (NPS)</td></tr><tr><td rowspan=1 colspan=1>Organism Detected</td><td rowspan=1 colspan=1>See analyte list above, RNA/DNA</td><td rowspan=1 colspan=1>See analyte list above, RNA/DNA</td></tr><tr><td rowspan=1 colspan=1>Amplification andDetection Technology</td><td rowspan=1 colspan=1>PCR</td><td rowspan=1 colspan=1>PCR</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>One internal control in each cartridgeto control for sample processing thatis subjected to all nucleic acidextraction and amplification stepssimilar to patient samples.Instructions for Use indicates qualitycontrol requirements should beperformed in conformance withlocal, state, and/or federal regulationsor accreditation requirements and thelaboratory&#x27;s standard quality controlprocedures.</td><td rowspan=1 colspan=1>Two controls are included in eachreagent pouch to control for sampleprocessing and both stages of PCR andmelt analysis. Labeling recommendsthe use of external positive andnegative controls regularly. Use viraltransport medium as the externalnegative control, and previouslycharacterized positive samples ornegative samples spiked with wellcharacterized organisms as externalpositive controls.</td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>Subject Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Bordetella Assay Targets</td><td rowspan=1 colspan=1>The QIAstat-Dx Respiratory PanelPlus detects Bordetella pertussis.</td><td rowspan=1 colspan=1>The BioFire Respiratory Panel 2.1(RP2.1) detects Bordetella pertussis(ptxP) and Bordetella parapertussis(IS1001).</td></tr><tr><td rowspan=1 colspan=1>Nucleic Acid Extraction</td><td rowspan=1 colspan=1>Extraction of nucleic acids using spincolumns</td><td rowspan=1 colspan=1>Extraction of nucleic acids usingmagnetic beads</td></tr><tr><td rowspan=1 colspan=1>Amplification andDetection Technology</td><td rowspan=1 colspan=1>Detection of amplified targets uses anincrease in fluorescence to generatethe assay results.</td><td rowspan=1 colspan=1>Melting curve analysis is used toconfirm the identity of amplifiedtargets to generate assay results.</td></tr><tr><td rowspan=1 colspan=1>Operational</td><td rowspan=1 colspan=1>The sample is loaded straight into thecartridge.</td><td rowspan=1 colspan=1>The pouch is hydrated prior tointroducing the sample.</td></tr><tr><td rowspan=1 colspan=1>Amplification andDetection InstrumentSystem</td><td rowspan=1 colspan=1>QIAstat-Dx Analyzer 1.0</td><td rowspan=1 colspan=1>FilmArray 2.0 or FilmArray TorchSystems</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# Standards

• CLSI EP17-A2 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition • CLSI EP07 Interference Testing in Clinical Chemistry – Third Edition • CLSI EP25-A Evaluation of Stability of In Vitro Diagnostic Reagents – September 2009 / EN 23640:2015 In vitro diagnostic medical devices. Evaluation of stability of in vitro diagnostic reagents

• CLSI EP12-A2 User Protocol for Evaluation and test Performance – Second Edition   
• CLSI EP05 Evaluation of Precision of Quantitative Measurement procedures – Third Edition   
• CLSI MM17 Validation and Verification of Multiplex Nucleic Acid Assays – Second Edition   
• CLSI MM03 Molecular Diagnostic Methods for Infectious Diseases – Third Edition   
• ICH GCP E6(R2) Good Clinical Practice   
• HIPAA - Health Insurance Portability and Accountability Act of 1996   
• IEC 62304 Edition 1.1 2015-06 Medical device software - Software life cycle processes

Special Controls

• Class II Special Controls as per 21 CFR 866.3981

FDA Guidance Documents

• Guidance for Test Developers and FDA Staff, Policy for Evaluating Impact of Viral mutations on COVID-19 tests (February 2021).   
Guidance for Industry and FDA Staff , Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Nucleic Acid Assays (October 2009). Guidance for Industry and Food and Drug Administration Staff, Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices (August 2014).   
• Guidance for Industry and Food and Drug Administration Staff, Content of Premarket Submissions for Management of Cybersecurity in Medical Devices (October 2014).   
• Guidance for Industry and Food and Drug Administration Staff. Content of Premarket Submissions for Device Software Functions (June 2023).

# VII Performance Characteristics:

# A Analytical Performance:

1. Precision/Reproducibility:

a. Within-laboratory precision

Within-laboratory precision was examined at one site using the QIAStat-Dx Respiratory Panel Plus assay using contrived panels in simulated NPS matrix of SARS-CoV-2 and a subset of representative organisms present on the original panel (Table 1). Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. Positive panels consisted of combinations of the target analytes at low (1x LoD) and moderate concentrations ( $3 \mathbf { x }$ LoD). The testing was performed across 5 non-consecutive days using two operators and 3 reagent lots (1 lot per replicate). Each operator performed 3 replicates per analyte at each sample concentration across 3 instruments each day for a total of 90 datapoints per analyte and concentration (2 operators $_ { \textrm { X 3 } }$ instruments x 3 replicates x 5 days). The results are shown in Table 1.

Table 1. Within-laboratory Precision Study – Qualitative Results   

<table><tr><td rowspan=1 colspan=1>Analyte(Strain/Isolate)</td><td rowspan=1 colspan=1>Analyte Level</td><td rowspan=1 colspan=1>% Positive (pos n/ valid n)</td><td rowspan=1 colspan=1>% Agreement with ExpectedResults (95% CI)</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2(England/02/2020)</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>96.7% (87/90)</td><td rowspan=1 colspan=1>96.7% (90.6 - 99.3%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>100% (92/92) a</td><td rowspan=1 colspan=1>100%(96.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100%(96.0- 100%)</td></tr><tr><td rowspan=3 colspan=1>Influenza B(Taiwan/62)</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>98.9%(89/90)</td><td rowspan=1 colspan=1>98.9%(94.0-100%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>98.9% (91/92) a</td><td rowspan=1 colspan=1>98.9%(94.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100%(96.0-100%)</td></tr><tr><td rowspan=3 colspan=1>Human coronavirusHKU1</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>100% (96.0-100%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>100% (92/92) a</td><td rowspan=1 colspan=1>100%(96.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100%(96.0-100%)</td></tr><tr><td rowspan=3 colspan=1>Parainfluenzavirus 3 (C-243)</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>97.8% (88/90)</td><td rowspan=1 colspan=1>97.8%(92.2-99.7%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>100% (92/92) a</td><td rowspan=1 colspan=1>100%(96.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100%(96.0-100%)</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus (A2)</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>100%(96.0-100%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>100% (92/92)a</td><td rowspan=1 colspan=1>100%(96.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100%(96.0-100%)</td></tr><tr><td rowspan=3 colspan=1>Adenovirus (B3)</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>96.7% (87/90)</td><td rowspan=1 colspan=1>96.7%(90.6-99.3)%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>100% (92/92)a</td><td rowspan=1 colspan=1>100%(96.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100%(96.0-100%)</td></tr><tr><td rowspan=3 colspan=1>Mycoplasmapneumoniae(PI 1428)</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>95.6% (86/90)</td><td rowspan=1 colspan=1>95.6%(89.0-98.8%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>97.8%(90/92)a</td><td rowspan=1 colspan=1>97.8%(92.4-99.7%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (90/90)</td><td rowspan=1 colspan=1>100% (96.0-100%)</td></tr></table>

a Two additional runs were performed due to partially invalidated runs.

SARS-CoV-2 samples were detected within performance expectations across all concentrations. All negative and low positive samples for the remaining analytes exhibited expected performance, achieving a percent agreement of $100 \%$ and at least $9 5 . 6 \%$ , respectively. Moderate positive samples achieved $100 \%$ expected positivity except influenza B $( 9 8 . 9 \% )$ and $M .$ pneumoniae $( 9 7 . 8 \% )$ . Notably, two tests were unexpectedly negative for $M .$ . pneumoniae and one test for influenza B. Unexpected negatives occurred across two lots and was attributed to a processing error. Repeat testing using the same cartridge lots and samples produced expected results. Overall, all panels met the acceptability criteria.

The mean and variability analysis between days, instruments, replicates, operators, lots and the overall (total) variability for Ct values for the SARS-CoV-2 analyte is shown in Table 2. Overall, the percent coefficient of variation $( \% \mathrm { C V } )$ ranged from $0 . 0 - 2 . 3 \%$ with the greatest factor of variability attributed to between replicate $\mathrm { C t }$ -variability at the 1x LoD level with a standard deviation of $0 . 8 \mathrm { C t }$ -value and $2 . 2 1 \%$ CV.

Assay variability for the subset of other analytes included in the study did not exceed those in the original 510(k) K183597 clearance (data not shown). Overall, Ct variability was low, and the study demonstrates assay variability within an acceptable range.

Table 2. Within-Laboratory Precision Study – SARS-CoV-2 Ct Signal Variability Analysis   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>AnalyteLevel</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenOperators</td><td rowspan=1 colspan=2>BetweenInstrument</td><td rowspan=1 colspan=2>BetweenLots</td><td rowspan=1 colspan=2>BetweenReplicates</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.21</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.30</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>2.01</td></tr></table>

b. Reproducibility

Reproducibility was assessed in K183597 and thus was not repeated here. The Reproducibility Study results can be found in the publicly available K183597 Decision Summary.

2. Linearity:

Not applicable. This is a qualitative assay.

3. Analytical Specificity/Interference:

Analytical Reactivity (Inclusivity)

The QIAstat-Dx Respiratory Panel Plus configuration of multiple viral and bacterial respiratory pathogens is the same as the previously cleared device (QIAstat-Dx Respiratory Panel, K183597), except for the addition of reagents for SARS-CoV-2 detection and a concentration change to a lysis reagent. Inclusivity wet-testing for the remaining on-panel analytes was performed previously in K183597 and not repeated here.

a. Wet-Testing

During Limit of Detection studies, five variants of SARS-CoV-2 were tested and successfully detected. Therefore, no additional inclusivity wet-testing was performed.

# b. In silico Analysis

QIAGEN routinely evaluates the overall inclusivity of the entire set of SARS-CoV-2 available genomes, including all relevant variants and lineages. This includes (1) Inclusivity Analysis and mismatch detection among SARS-CoV-2 strains and (2) the same analysis focused on specific SARS-CoV-2 variants or lineages, evaluating the possible effect of detected mismatches on the QIAstat-Dx performance. Only critical mismatches were evaluated in the laboratory. Laboratory testing was performed using artificial genomic fragments corresponding to targeted regions and including the corresponding mismatch.

In total, 8,118,241 available genomes between January 1, 2020 and June 5, 2022 from around the globe were analyzed for inclusivity. Of them, 7,932,071 $( 9 7 . 7 1 \% )$ presented no evidence of mismatches in the targeted regions. The remainder of genomes $( 2 . 0 1 \% )$ presented mismatches among the sequence in the binding region, but only 19,045 $( 0 . 2 3 \% )$ presented a mismatch in a critical position (i.e., near the $_ { 3 } ,$ -end of any primer) with a frequency ${ > } 0 . 2 \%$ . Genetic patterns included among those sequences were evaluated in laboratory testing, with no loss of performance at LoD level. Consequently, $100 \%$ of sequences analyzed were predicted to be detected.

Variants of Concern, of Interest and Under Investigation (VOCs, VOIs, VUI respectively) were also evaluated from the 8,118,241 sequences. After classifying all previously analyzed genomes into lineages based on GISAID and PANGO classifications, the following variants were included: Alpha (9 clades), Beta (5 clades), Gamma (23 clades), Lambda (2 clades), Mu (4 clades), Delta (242 clades), Omicron (73 clades). As a result, all variants and lineages described are $100 \%$ predicted to be detected.

For the rest of the analytes detected by the QIAstat-Dx Respiratory Panel Plus, an updated in silico analysis was performed to confirm previous device inclusivity results. All single primers and probes corresponding to targets included in the QIAstat-Dx Respiratory Panel Plus (non-SARS-CoV-2 targets) were analyzed using BLAST.

All QIAstat-Dx Respiratory Panel Plus primers and probes are predicted to be inclusive for all clinical prevalent and relevant strains for each pathogen against all available sequences in the NCBI data base.

QIAGEN has created a dedicated Post Market Surveillance (PMS) team with the objective of periodically analyzing all new genomic sequences of SARS-CoV-2 to detect any mutation pattern that may affect the QIAstat-Dx assay performance and create an action plan accordingly.

Cross-Reactivity (Exclusivity) and Competitive/Microbial Interference

The potential for cross-reactivity between all QIAstat-Dx Respiratory Panel Plus on-panel organisms (including SARS-CoV-2) and various on-panel or off-panel organisms that can occur in clinical respiratory specimens was evaluated with a combination of laboratory (in vitro) wet testing and in silico characterization. For laboratory testing, on-panel (intra-panel cross-reactivity) and off-panel tested organisms (not covered by the panel content and, therefore, not intended to be detected by the QIAstat-Dx Respiratory Panel Plus) complement the organisms evaluated in the previously cleared device (QIAstat-Dx Respiratory Panel), and additional organisms were selected to assess the specificity of the new SARS-CoV-2 assays included in the Panel. Additionally, all single primers and probes for the SARS-CoV2 detection were analyzed in silico to complement the overall specificity prediction of the device.

# a. Wet-Testing

# i. On-Panel Cross-Reactivity Testing (Analytical Specificity)

Potential intra-panel cross-reactivity was evaluated with twenty-two (22) on-panel microorganisms (Table 3). The on-panel organisms were evaluated by spiking each independently into simulated NPS matrix at ${ \mathrm { \bar { > } } } 1 0 ^ { 5 }$ $\mathrm { \Delta T C I D _ { 5 0 } / m L }$ (or equivalent) for viruses and ${ > } 1 0 ^ { 6 }$ CFU/mL (or equivalent) for bacteria, or the highest concentration available. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2. Matrix Equivalency Study, later in this document. To evaluate potential intra-panel cross-reactivity, each organism was evaluated in triplicate. No intra-panel cross-reactivity was observed. The results of testing are shown in Table 3.

Table 3. On-Panel Organisms Tested for QIAstat-Dx Respiratory Panel Plus   

<table><tr><td colspan="1" rowspan="1">Organism (Strain)</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">Detectionrate(positivereplicates)</td><td colspan="1" rowspan="1">Cross-ReactivityDetected</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae(UTMB-10P)</td><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">1.00E+05CCU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis (E431)</td><td colspan="1" rowspan="1">ZeptoMetrix0801460</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">3/3a</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae(TWAR strain TW-183)</td><td colspan="1" rowspan="1">ATCC VR-2282</td><td colspan="1" rowspan="1">1.00E+06IFU/mL</td><td colspan="1" rowspan="1">3/3b</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1(A/New Jersey/8/76)</td><td colspan="1" rowspan="1">ATCC VR-897</td><td colspan="1" rowspan="1">1.00E+05CEID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2(A/Switzerland/971529/2013)</td><td colspan="1" rowspan="1">ATCC VR-1837</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009/H1N1(A/California/07/2009NYMC X)</td><td colspan="1" rowspan="1">ATCC VR-1884</td><td colspan="1" rowspan="1">1.00E+05CEID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B(B/Florida/4/2006)</td><td colspan="1" rowspan="1">ATCC VR-1804</td><td colspan="1" rowspan="1">1.00E+05CEID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">ZeptoMetrix0810229CFHI</td><td colspan="1" rowspan="1">1.00E+04U/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">ZeptoMetrix0810024CFHI</td><td colspan="1" rowspan="1">1.00E+04U/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus NL63</td><td colspan="1" rowspan="1">BEI NR-470</td><td colspan="1" rowspan="1">1.00E+05copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus HKU1(Clinical sample)</td><td colspan="1" rowspan="1">Stat-Dx S506</td><td colspan="1" rowspan="1">1.00E+05copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1(C35)</td><td colspan="1" rowspan="1">ATCC VR-94</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2(Greer)</td><td colspan="1" rowspan="1">ATCC VR-92</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3 (C243)</td><td colspan="1" rowspan="1">ATCC VR-93</td><td colspan="1" rowspan="1">1.00E+05CEID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">ZeptoMetrix0810060BCFHI</td><td colspan="1" rowspan="1">1.00E+05U/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">RSV (A2)</td><td colspan="1" rowspan="1">ATCC VR-1540</td><td colspan="1" rowspan="1">1.00E+05CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">hMPV (IA10-2003)</td><td colspan="1" rowspan="1">ZeptoMetrix0810161CFHI</td><td colspan="1" rowspan="1">1.00E+05U/mL</td><td colspan="1" rowspan="1">3/3c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C (Adenoid71)</td><td colspan="1" rowspan="1">ATCC VR-1</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B (Gomen)</td><td colspan="1" rowspan="1">ATCC VR-7</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (/US/IL/14-18952)</td><td colspan="1" rowspan="1">ATCC VR-1824</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (2060)</td><td colspan="1" rowspan="1">ATCC VR-1559</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (Clinicalsample</td><td colspan="1" rowspan="1">Hospital Clinic(Barcelona) S243</td><td colspan="1" rowspan="1">1.00E+05copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">None</td></tr></table>

a B. pertussis initial testing resulted in only two positive replicates with unexpected high Ct value. Investigation concluded that testing was performed with the incorrect dilution and it was repeated with the correct dilution, obtaining expected results. b Two cartridges resulted in Positive with Warning (IC fail) and were repeated to confirm obtained results. c One cartridge fail was repeated.

# ii. Off-Panel Cross-Reactivity Testing (Analytical Specificity)

This study evaluated the analytical specificity (cross-reactivity) of the QIAstat-Dx Respiratory Panel Plus in the presence of non-targeted microorganisms that may be found in a respiratory tract clinical specimen. Forty-nine (49) non-target microorganisms (Table 4) were evaluated in the study. Panel members were composed of one individual non-target microorganism spiked into simulated NPS matrix at ${ > } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (or equivalent) for viruses, ${ > } \mathrm { { i } } 0 ^ { 6 } \mathrm { { C F U / m L } }$ (or equivalent) for bacteria/fungi, or the highest concentration available. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document.

To evaluate cross-reactivity, each microorganism was evaluated in triplicate in the absence of the target organisms. No cross-reactivity was observed at the concentrations tested except for Bordetella bronchiseptica at a concentration of $1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ and Bordetella holmesii at a concentration of $1 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , which both cross-reacted with the Bordetella pertussis primer/probes in $100 \%$ of the replicates tested for each bacterium. As described for the previously cleared QIAstatDx Respiratory Panel (K183597), the targeted gene for Bordetella pertussis (IS481) is a mobile transposon also present in $B .$ . holmesii (detected in $6 / 6$ replicates) and some clusters of $B$ . bronchiseptica (detected in 3/3 replicates), so observed crossreaction with $B .$ . pertussis was expected in both cases. A limitation to this affect has been included in the Instructions for Use). The results are summarized in Table 4.

Table 4. Off-Panel Organisms Tested for QIAstat-Dx Respiratory Panel Plus   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">Detection rate(#Detected/#Tested)</td><td colspan="1" rowspan="1">Cross-ReactivityDetected</td></tr><tr><td colspan="1" rowspan="1">Acinetobactercalcoaceticus</td><td colspan="1" rowspan="1">ZeptoMetrix0804096</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/5a</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella avium</td><td colspan="1" rowspan="1">ZeptoMetrix0804316</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetellabronchiseptica</td><td colspan="1" rowspan="1">ATCC 4617</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">Bordetellapertussisb</td></tr><tr><td colspan="1" rowspan="1">Bordetella hinzii</td><td colspan="1" rowspan="1">VircellMC089</td><td colspan="1" rowspan="1">1.00E+06copies/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">ATCC 51541</td><td colspan="1" rowspan="1">1.00E+05CFU/mL</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">Bordetellapertussisb</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphteriae</td><td colspan="1" rowspan="1">ATCC 11913</td><td colspan="1" rowspan="1">5.00E+03CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">ATCC 49766</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">ATCC 8142</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC700324</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusacidophilus</td><td colspan="1" rowspan="1">ATCC 4356</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella feelei</td><td colspan="1" rowspan="1">VircellMC092</td><td colspan="1" rowspan="1">1.00E+06copies/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionellamicdadei</td><td colspan="1" rowspan="1">ZeptoMetrix0801576</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">1.00E+05CFU/mL</td><td colspan="1" rowspan="1">0/5c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">ZeptoMetrix0801645</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">ATCC 33156</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">ATCC25177DQ</td><td colspan="1" rowspan="1">1.00E+06copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmagenitalium</td><td colspan="1" rowspan="1">ZeptoMetrix0804094-I</td><td colspan="1" rowspan="1">1.00E+06CCU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmahominis</td><td colspan="1" rowspan="1">ATCC 27545</td><td colspan="1" rowspan="1">1.00E+05CFU/mL</td><td colspan="1" rowspan="1">1/9</td><td colspan="1" rowspan="1">Noned</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">ATCC 23714</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">ZeptoMetrix0801510</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseriagonorrhoeae</td><td colspan="1" rowspan="1">ZeptoMetrix0801482</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">ATCCCRM6538</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">ATCC 14990</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">ATCC 13637</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">ZeptoMetrix0801439</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">ZeptoMetrix0801896</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Ureaplasmaurealyticum</td><td colspan="1" rowspan="1">ATCC 27618</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(0157)</td><td colspan="1" rowspan="1">ZeptoMetrix0801622</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">ATCC 25240</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitidis</td><td colspan="1" rowspan="1">ATCC 35561</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/5a</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ZeptoMetrix0801544</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">ATCC 15442</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">ATCC 14289</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">2/6</td><td colspan="1" rowspan="1">Noned</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">ZeptoMetrix0801545</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">University ofKansasMedicalCenter,Kansas, USA(ClinicalSample</td><td colspan="1" rowspan="1">1.00E+05copies/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ZeptoMetrix0810499CFHI</td><td colspan="1" rowspan="1">1.00E+05U/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">ATCC VR-1492PQ</td><td colspan="1" rowspan="1">1.00E+05copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">ATCC VR-24</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">MERS Coronavirus</td><td colspan="1" rowspan="1">VircellMC121</td><td colspan="1" rowspan="1">1.00E+05copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">ATCC VR-106</td><td colspan="1" rowspan="1">1.00E+04TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Herpes SimplexVirus 1</td><td colspan="1" rowspan="1">ATCC VR-1778</td><td colspan="1" rowspan="1">1.00E+05PFU/mL</td><td colspan="1" rowspan="1">0/3c</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC VR-1789</td><td colspan="1" rowspan="1">1.00E+05PFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="2">Herpes SimplexVirus 2</td><td colspan="1" rowspan="1">ATCC VR-1779</td><td colspan="1" rowspan="1">1.00E+05PFU/mL</td><td colspan="1" rowspan="1">0/5a</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-734</td><td colspan="1" rowspan="1">1.00E+05PFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC CRM-10231</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">ZeptoMetrix0801598</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Aspergillusfumigatus</td><td colspan="1" rowspan="1">ZeptoMetrix0801716</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">1/6</td><td colspan="1" rowspan="1">Noned</td></tr><tr><td colspan="1" rowspan="1">Cryptococcusneoformans</td><td colspan="1" rowspan="1">ATCC 32045</td><td colspan="1" rowspan="1">1.00E+06CFU/mL</td><td colspan="1" rowspan="1">0/6</td><td colspan="1" rowspan="1">None</td></tr></table>

a. Two reliability errors were repeated. Repeats were valid and no cross-reactivity was observed (Reliability errors is a summary term for a class of possible errors that occur during the execution of the cartridge on the instrument).

b. As described for the previously cleared device, targeted gene for Bordetella pertussis (IS481) is a mobile transposon also present in $B .$ . holmesii (detected in $6 / 6$ replicates) and some clusters of B. bronchiseptica (detected in 3/3 replicates), so observed cross-reaction with $B$ . pertussis was expected in both cases.

c. One cartridge fail was repeated.

d. B. pertussis was detected with large Ct value (low concentration) for A. fumigatus (1/6 replicates), Mycoplasma hominis (1/9 replicates) and $S .$ . pyogenes (2/6 replicates). Proper investigation confirmed B. pertussis contamination in all cases. In silico analysis confirmed lack of cross-reactivity of $B$ . pertussis assay with all three organisms.

# iii. Competitive Interference

A competitive interference study was done to determine if the presence of a subset of representative on-panel organisms at clinically relevant and challenging concentrations ( $1 . 0 0 \mathrm { E } { + } 0 6 \mathrm { C F U / m L }$ for bacteria, $1 . 0 0 \mathrm { E } { + } 0 5 \ : \mathrm { P F U / m L }$ for viruses) would impede the detection of SARS-CoV-2 at low concentrations ( $3 \mathbf { x }$ LoD) in simulated NPS matrix. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. Testing was performed in triplicate with SARS-CoV-2 detection in all samples and replicates (Table 5).

Table 5. On-panel Targets and SARS-CoV-2 Competitive Interference Study Results   

<table><tr><td colspan="1" rowspan="1">High ConcentrationTarget Organism</td><td colspan="1" rowspan="1">Strain/Isolate</td><td colspan="1" rowspan="1">Source/ Catalog #</td><td colspan="1" rowspan="1"># SARS-CoV-2detected/ validruns</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">ZeptoMetrix0810229CFHI</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">ZeptoMetrix0810024CFHI</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus A12</td><td colspan="1" rowspan="1">Huie</td><td colspan="1" rowspan="1">ATCC VR-863</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">C243</td><td colspan="1" rowspan="1">ATCC VR-93</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">18323</td><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68</td><td colspan="1" rowspan="1">US/IL/14-18952</td><td colspan="1" rowspan="1">ATCC VR-1824</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">2060 (Type 1A)</td><td colspan="1" rowspan="1">ATCC VR-1559</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6a</td><td colspan="1" rowspan="1">D-1 (Cox)</td><td colspan="1" rowspan="1">ATCC VR-241</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">Long</td><td colspan="1" rowspan="1">ATCC VR-26</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">IA10-2003</td><td colspan="1" rowspan="1">ZeptoMetrix0810161CFHI</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1</td><td colspan="1" rowspan="1">New Cal/20/99</td><td colspan="1" rowspan="1">ZeptoMetrix0810036CFHI</td><td colspan="1" rowspan="1">3/3</td></tr></table>

a Echovirus is an Enterovirus and expected to report a positive Rhinovirus/Enterovirus result, accordingly.

# iv. Microbial Interference

A microbial interference study was conducted to assess the inhibitory effects of select non-target organisms on the ability to detect SARS-CoV-2. Clinically relevant and challenging concentrations $\mathrm { 1 . 0 0 E + 0 6 C F U / m L }$ for bacteria/fungi, $1 . 0 0 \mathrm { E } { + } 0 5$ PFU/mL for viruses unless otherwise noted) of non-target organisms were individually mixed with SARS-CoV-2 at 3x LoD in simulated NPS matrix. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. Testing was performed in triplicate with two additional tests performed if SARS-CoV-2 was not detected in any one of the original three replicates. All combinations and replicates successfully detected SARS-CoV-2 except for two samples containing Legionella pneumophila, one Streptococcus salivarius sample, and one S. epidermidis sample (Table 6). For these, additional replicates successfully detected SARS-CoV-2. Where available, at least one additional strain of L. pneumophila or S. salivarius was also tested in triplicate with all samples successfully detecting SARS-CoV-2. See Table 6 for a list of the strains tested and the result summary.

Table 6. Microbial Interference Study Results   

<table><tr><td colspan="1" rowspan="1">Non-Target Organism</td><td colspan="1" rowspan="1">Strain/Isolate</td><td colspan="1" rowspan="1">Source/ Catalog #</td><td colspan="1" rowspan="1"># SARS-CoV-2detected/ validruns</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusa</td><td colspan="1" rowspan="1">FDA 209</td><td colspan="1" rowspan="1">ATCC CRM-6538</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Z022 19F</td><td colspan="1" rowspan="1">ZeptoMetrix 0801439</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">C699 [S30D]</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">Z127</td><td colspan="1" rowspan="1">ZeptoMetrix 0801896</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">AMC 36-A-7</td><td colspan="1" rowspan="1">ATCC 8142</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">AMC 36-A-1</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">CBS 562</td><td colspan="1" rowspan="1">ATCC 18804</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus 1</td><td colspan="1" rowspan="1">ATCC-2011-9</td><td colspan="1" rowspan="1">ATCC VR-1789</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">Fussel</td><td colspan="1" rowspan="1">ATCC 14990</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">PRD-10</td><td colspan="1" rowspan="1">ATCC 15442</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Philadelphiab</td><td colspan="1" rowspan="1">ZeptoMetrix 0801645</td><td colspan="1" rowspan="1">3/5</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Philadelphia-1b</td><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Los Angeles-1</td><td colspan="1" rowspan="1">ATCC 33156</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidisa</td><td colspan="1" rowspan="1">serogroup A</td><td colspan="1" rowspan="1">ATCC 13077</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriaea</td><td colspan="1" rowspan="1">48255</td><td colspan="1" rowspan="1">ATCC 11913</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Cytomegalovirus(CMV)a</td><td colspan="1" rowspan="1">Towne</td><td colspan="1" rowspan="1">Zeptometrix 0810499CFHI</td><td colspan="1" rowspan="1">3/3</td></tr></table>

a S. aureus evaluated at $4 . 5 { \mathrm { x } } 1 0 ^ { 8 } \mathrm { C F U } / { \mathrm { m L } } , N .$ meningitidis $\overline { { 1 . 0 \mathrm { x } 1 0 ^ { 3 } } }$ CFU/mL, C. diphtheriae $\overline { { 1 . 0 \mathrm { x } 1 0 ^ { 3 } } }$ CFU/mL, and CMV at $1 . 0 { \mathrm { x } } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . b Philadelphia and Philadelphia-1 are both designations of strain Philadelphia serogroup-1, differences in naming are due to supplier.

# b. In silico

The QIAstat-Dx Respiratory Panel Plus differs from the previously cleared device (QIAstat-Dx Respiratory Panel, K183597) by the addition of primers and probes for the detection of SARS-CoV-2 and a decrease in the concentration of a lysis reagent. Evaluation of the two SARS-CoV-2 assays was carried out to corroborate the lack of cross-reactivity.

Exclusivity BLAST-based in silico screening was performed on April $2 0 ^ { \mathrm { t h } } 2 0 2 2$ , including all available sequences in the GenBank database. BLAST homology characterization was performed excluding SARS-CoV-2 sequences from the analysis to optimize BLAST results to non-target organisms. Results are summarized in Table 7.

Table 7. Summary of BLAST Homology Analysis of the SARS-CoV-2 Assays   

<table><tr><td rowspan=1 colspan=1>Targetgene</td><td rowspan=1 colspan=1>Sequencetype</td><td rowspan=1 colspan=1>BLASThits</td><td rowspan=1 colspan=1>&gt;70%homologysequences</td><td rowspan=1 colspan=1>Finalsequencesa</td><td rowspan=1 colspan=1>SARS-CoV-2 relatedsequences</td><td rowspan=1 colspan=1>OtherOrganisms</td></tr><tr><td rowspan=3 colspan=1>RdRp(Orflab)gene</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>8651</td><td rowspan=1 colspan=1>1439</td><td rowspan=3 colspan=1>35</td><td rowspan=3 colspan=1>14 Syntheticconstructs orcloningvectors. 9SARS-CoV-2-relatedcoronavirus</td><td rowspan=3 colspan=1>12 BatCoronaviruses</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>5503</td><td rowspan=1 colspan=1>1405</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>6774</td><td rowspan=1 colspan=1>1046</td></tr><tr><td rowspan=3 colspan=1>E gene</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>5462</td><td rowspan=1 colspan=1>1880</td><td rowspan=3 colspan=1>130</td><td rowspan=3 colspan=1>14 Syntheticconstructs orcloningvectors. 11SARS-CoV-2relatedcoronavirus</td><td rowspan=3 colspan=1>9 Pangolincoronavirus and96 Batcoronaviruses</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>5034</td><td rowspan=1 colspan=1>811</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>3477</td><td rowspan=1 colspan=1>630</td></tr></table>

a Final sequences matching Forward primer, Probe and Reverse primer with right amplicon size.

Only Thirty-five (35) and 130 evaluated sequences (RdRp and E genes, respectively) presented possible cross-amplification. After removing synthetic constructs, recombinant, cloning vectors, or SARS-CoV-2 related coronavirus, only potential nonspecific signals could be produced by a cross-reaction of the SARS-CoV-2 assays with a few coronaviruses specific for bats and/or pangolins. Since these organisms are not known to currently circulate in the human population, the risk of a false positive SARS-CoV-2 result is low. A limitation on potential cross-reactivity of bat and pangolin sequences has been added to the Instructions for Use. No GenBank sequences corresponding to humaninfective pathogens were predicted to be positive for the primer/probe sets included in the QIAstat-Dx system to detect SARS-CoV-2.

# Interfering Substances

The performance of the QIAStat-Dx Respiratory Panel Plus for the detection of SARS-CoV-2 was evaluated in the presence of medically and/or physiologically relevant concentrations of potentially interfering substances that may be present in an NPS specimen. In addition, several technique-specific substances (e.g., disinfectants and various brands of swabs or VTM transport mediums) were also tested as potential sources of assay interference. Samples were tested in triplicate with and without addition of the potentially inhibitory substance in simulated NPS matrix using SARS-CoV-2 (NIBSC cat no. 20/146) at $3 \mathbf { x }$ LoD. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. No interference was observed at the concentrations listed for any of the substance tested (Table 8).

Table 8. Potential Interfering Substances Concentrations and Results   

<table><tr><td rowspan=1 colspan=2>SubstanceType</td><td rowspan=1 colspan=1>Substance / Pathogen</td><td rowspan=1 colspan=1>Concentrationtested</td><td rowspan=1 colspan=1>Positive NegativeSamples Samples</td><td rowspan=1 colspan=1>Positive NegativeSamples Samples</td></tr><tr><td rowspan=3 colspan=2>EndogenousSubstances</td><td rowspan=1 colspan=1>Human genomic DNA 200 ng/μL</td><td rowspan=1 colspan=1>20 ng/μL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Human Blood (+Na Citrate)</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Mucin from bovine submaxillary glands</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/4</td></tr><tr><td rowspan=7 colspan=2>Exogenoussubstances</td><td rowspan=1 colspan=1>Tobramycin (systemic antibiotic)</td><td rowspan=1 colspan=1>0.6 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>2 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>OCEAN Saline Nasal Spray</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Afrin Original Pump Mist</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Chiroflu Influenza Vaccine (surface antigeninactivated)</td><td rowspan=1 colspan=1>1.00E-05 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Analgesic ointment (Vicks VapoRub)</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Petroleum Jelly (Vaseline)</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=11 colspan=2>TechniqueSpecificSubstances</td><td rowspan=1 colspan=1>Bleach</td><td rowspan=1 colspan=1>5 %</td><td rowspan=1 colspan=1>5/5*</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Meliseptol Wipes (disinfecting wipes)</td><td rowspan=1 colspan=1>2.50E-04 in2/μ1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>DNAZap</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>ProtectRNA RNase Inhibitor 500xConcentrate</td><td rowspan=1 colspan=1>1 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>5 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Swab Copan 168C (rayon/twistedaluminum shaft)</td><td rowspan=1 colspan=1>1 swab/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Swab Copan FloQ (flocked nylon/plasticshaft)</td><td rowspan=1 colspan=1>1 swab/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Swab Copan 175KS01(polyester/aluminum shaft)</td><td rowspan=1 colspan=1>1 swab/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Swab Puritan</td><td rowspan=1 colspan=1>1 swab/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>VTM Remel M4-RT</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>VTM Sigma Virocult</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr></table>

<table><tr><td rowspan=1 colspan=1>SubstanceType</td><td rowspan=1 colspan=1>Substance / Pathogen</td><td rowspan=1 colspan=1>Concentrationtested</td><td rowspan=1 colspan=1>Positive Negative</td><td rowspan=1 colspan=1>Positive NegativeSamples Samples</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>VTM-RT</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>BD Universal Viral Transport</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>DeltaSwab Virus</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\*Two additional cartridges were run in error, results from these runs were valid and are reported here.

All other targets of the QIAstat-Dx Respiratory Panel Plus have been evaluated in interfering substance studies during the previous device clearance with the results remaining applicable (see QIAstat-Dx Respiratory Panel, K183597). The interference of SARS-CoV-2 detection with a live, intranasal influenza vaccine (i.e., FluMist or Fluenz) was not assessed. Results from the previously cleared device on other panel analytes determined that the live intranasally administered influenza vaccine did not interfere at concentrations of $0 . 0 0 0 0 0 1 \% \mathrm { v } / \mathrm { v }$ or less, however, interference was noted at a ten-fold higher concentration (A limitation to this effect has been added to the Instructions for Use).

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# a. Controls

This assay contains an internal control within each cartridge. External positive and negative controls are not provided with the assay but were run during the performance studies. A description of the external controls used in these studies is presented under section External Controls, earlier in this document.

# b. Sample Stability

Sample stability testing demonstrated that NPS specimens in Universal Transport Medium (UTM) may be stored at the conditions listed below.

Room temperature up to 4 hours at $1 5 { - } 2 5 ^ { \circ } \mathrm { C }$ • Refrigerated up to 3 days at approximately $4 \mathrm { { } ^ { \circ } C }$ • Frozen up to 14 days at $- 2 0 ^ { \circ } \mathrm { C }$

# c. Cartridge Package Open stability

In a study comprising a subset of panel analytes, including SARS-CoV-2, it was determined that after the cartridge package is opened, sample should be introduced into the QIAstat-Dx Respiratory Panel Plus Cartridge and loaded into the QIAstat-Dx Analyzer 1.0 within 120 minutes.

A limit of detection study (LoD) was performed to evaluate the analytical sensitivity of the SARS-CoV-2 assays of the QIAstat-Dx Respiratory Panel Plus. In addition, a subset of the original panel analytes (QIAstat-Dx Respiratory Panel, K183597) was tested sideby-side with the candidate device and the previously cleared device (QIAstat-Dx Respiratory Panel, K183597) to demonstrate that the addition of t h e SARS-CoV-2 assays and a minor modification to the lysis buffer does not impact the performance of other analyte assays in other reaction chambers.

# a. Limit of Detection for SARS-CoV-2

For this study, five (5) SARS-CoV-2 strains (Table 9) were evaluated individually by testing serial dilutions prepared in NPS matrix. Testing was broken into two parts: preliminary and confirmatory LoD testing. For the preliminary LoD study, a serial dilution series consisting of four concentrations was tested in replicates of four per dilution. The preliminary LoD for each target was defined as the lowest concentration at which $100 \%$ of replicates were positive for SARS-CoV-2. The confirmed LoD was established by testing twenty (20) replicates at the concentration determined from the preliminary LoD for each strain. The LoD for each strain was confirmed if $\ge 9 5 \%$ of the replicates were positive. To further confirm the LoD, at least one dilution below the LoD was also tested for each strain in twenty (20) replicates and was required to result in less than $9 5 \%$ positivity. The confirmed LoD for SARS-CoV-2 is summarized in Table 10.

Table 9. SARS-CoV-2 strains used in the Limit of Detection Study   

<table><tr><td rowspan=1 colspan=1>Strain Information</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Stockconcentration</td><td rowspan=1 colspan=1>Live/inactivated</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>STAT-Dx Life,S.L. Clinical sample</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1,40E+07copies/mLa</td><td rowspan=1 colspan=1>Clinical sample, noinactivation</td></tr><tr><td rowspan=1 colspan=1>USA-WA1-2020</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810587CFH</td><td rowspan=1 colspan=1>324163</td><td rowspan=1 colspan=1>4,25E+10copies/mLa</td><td rowspan=1 colspan=1>SARS-CoV-2 (Isolate: USA-WA1/2020) Culture Fluid(Heat Inactivated)</td></tr><tr><td rowspan=1 colspan=1>England/02/2020</td><td rowspan=1 colspan=1>NIBSC</td><td rowspan=1 colspan=1>First WHOInternationalStandard for SARS-CoV-2 RNA 20/146</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1,40E+07copies/mL a</td><td rowspan=1 colspan=1>The first WHO InternationalStandard for SARS-CoV-2RNA for Nucleic acidAmplification Technique(NAT)-based assays consistsof acid-heat inactivatedEngland/02/2020 isolate ofSARS-CoV-2.</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Vall d&#x27;Hebronhospital Clinicalsample</td><td rowspan=1 colspan=1>S1229</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>4,60E+06copies/mLa</td><td rowspan=1 colspan=1>Clinical sample, noinactivation</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Vall d&#x27;Hebronhospital  Clinicalsample</td><td rowspan=1 colspan=1>S1231</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1,90E+07copies/mLa</td><td rowspan=1 colspan=1>Clinical sample, noinactivation</td></tr></table>

a Titer determined in molecular units (copies $/ \mathrm { m L }$ ) by in-house developed and validated qPCR assay.

Table 10. Confirmatory LoD Results for SARS-CoV-2   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(copies/mL)b</td><td rowspan=1 colspan=1>Detection rate</td><td rowspan=1 colspan=1>Mean Ct SARSCoV-2</td></tr><tr><td rowspan=5 colspan=1>STAT-Dx Life, S.L. Clinical sample (ID 243)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>20/20a (100%)</td><td rowspan=1 colspan=1>35.8</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>17/20 (85%)</td><td rowspan=1 colspan=1>35.7</td></tr><tr><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>17/20 (85%)</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>6/20 (30%)</td><td rowspan=1 colspan=1>36.2</td></tr><tr><td rowspan=2 colspan=1>USA-WA1-2020,Zeptometrix</td><td rowspan=1 colspan=1>3,160</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>35.5</td></tr><tr><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>1/20 (5%)</td><td rowspan=1 colspan=1>35.7</td></tr><tr><td rowspan=2 colspan=1>England/02/2020, NIBSC</td><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>19/20a(95%)</td><td rowspan=1 colspan=1>35.9</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>11/20 (55%)</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=3 colspan=1>Vall d&#x27;Hebron hospital —Clinical Sample (ID S1229)</td><td rowspan=1 colspan=1>19,000</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>6,000</td><td rowspan=1 colspan=1>18/20a (90%)</td><td rowspan=1 colspan=1>34.9</td></tr><tr><td rowspan=1 colspan=1>1,900</td><td rowspan=1 colspan=1>17/20(85%)</td><td rowspan=1 colspan=1>34.9</td></tr><tr><td rowspan=3 colspan=1>Vall d&#x27;Hebron hospital —Clinical sample (ID S1231)</td><td rowspan=1 colspan=1>19,000</td><td rowspan=1 colspan=1>20/20aa (100%)</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>6,000</td><td rowspan=1 colspan=1>14/20 (70%)</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>1,900</td><td rowspan=1 colspan=1>10/20 (50%)</td><td rowspan=1 colspan=1>35.4</td></tr></table>

a One “failed” (invalid) result was obtained. Valid result was obtained upon retesting. b Titer determined in molecular units (copies $/ \mathrm { m L }$ ) by in-house developed and validated $\mathsf { q P C R }$ assay.

# b. LoD Comparison Study

A subset of six (6) representative analytes (one from each Reaction Chamber) as listed in Table 11 was included in the LoD comparison study to demonstrate that the addition of SARS-CoV-2 assays and changes to the lysis reagent concentration do not impact the performance of assays in other chambers. Twenty (20) replicates of the subset of original panel analytes were prepared as co-formulated samples using pooled negative clinical matrix at the LoD concentrations established for the previously cleared device (QIAstatDx Respiratory Panel, K183597). Additionally, twenty (20) replicates were tested at $0 . 1 \mathrm { x }$ LoD to further verify LoD equivalency. The panel of representative analytes was tested in parallel with the previously cleared device (QIAstat-Dx Respiratory Panel, K183597) and the candidate device. The expected detection rates for both devices were $2 9 5 \%$ of replicates tested $( \geq 1 9 / 2 0 )$ at the LoD concentration, and ${ < } 9 5 \%$ $( \leq 1 8 / 2 0 )$ at the $0 . 1 \mathbf { x }$ LoD concentration. As summarized in Table 12, the LoD for a subset of original panel analytes was confirmed on both devices in side-by-side testing, which confirmed that the addition of the SARS-CoV-2 assays and changes to the lysis reagent do not impact the performance of assays in other chambers. Moreover, the LoDs as determined in K183597 for single-spiked analytes are equivalent to the LoDs determined for individual analytes under multi-spiked conditions. Multi-spiked LoD studies were not repeated here.

Table 11. Representative strains used in the Limit of Detection Comparison Study   

<table><tr><td colspan="1" rowspan="1">TargetOrganism/Analyte</td><td colspan="1" rowspan="1">Straininformation</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">CatalogueID</td><td colspan="1" rowspan="1">Lot</td><td colspan="1" rowspan="1">Stockconcentration</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-295</td><td colspan="1" rowspan="1">60517845</td><td colspan="1" rowspan="1">1.58E+07CEID50/mL</td></tr><tr><td colspan="1" rowspan="1">CoronavirusHKU1</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">STAT-Dx Life,S.L-Clinical sample</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">S510</td><td colspan="1" rowspan="1">4.00E+08copies/mLa</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus 3</td><td colspan="1" rowspan="1">C 243</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-93</td><td colspan="1" rowspan="1">59380357</td><td colspan="1" rowspan="1">2.32E+06TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">HGP (RhinovirusA2)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-482</td><td colspan="1" rowspan="1">58524058</td><td colspan="1" rowspan="1">2.81E+07TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">GB (AdenovirusB3)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-3</td><td colspan="1" rowspan="1">59490909</td><td colspan="1" rowspan="1">1.58E+10TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">PI 1428</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">29085</td><td colspan="1" rowspan="1">70010561</td><td colspan="1" rowspan="1">1.00E+06CCU/mL</td></tr></table>

a Titer determined in molecular units (copies $/ \mathrm { m L }$ ) by in-house developed and validated qPCR assay.

Table 12. Summary of LoD Comparison Study with a Subset of Panel Analytes   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>QIAstat-DxRespiratoryPanel Plus(# Detected/#Tested, Mean Ct)</td><td rowspan=1 colspan=2>QIAstat-DxRespiratory Panel(# Detected/#Tested, Mean Ct)</td></tr><tr><td rowspan=1 colspan=1>TargetOrganism/Analyte</td><td rowspan=1 colspan=1>Isolate</td><td rowspan=1 colspan=1>LoD Concentration</td><td rowspan=1 colspan=1>1xLoD</td><td rowspan=1 colspan=1>0.1xLoD</td><td rowspan=1 colspan=1>1xLoD</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>5000.0CEID50/mL</td><td rowspan=1 colspan=1>20/20(28.3)</td><td rowspan=1 colspan=1>15/20(30.8)</td><td rowspan=1 colspan=1>20/20(27.7)</td><td rowspan=1 colspan=1>17/20(30.7)</td></tr><tr><td rowspan=1 colspan=1>CoronavirusHKU1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>4.00E+04copies/mLa</td><td rowspan=1 colspan=1>20/20(33.8)</td><td rowspan=1 colspan=1>4/20(34.2)</td><td rowspan=1 colspan=1>20/20(33.4)</td><td rowspan=1 colspan=1>5/20(34.1)</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.3TCID50/mL</td><td rowspan=1 colspan=1>20/20(32.9)</td><td rowspan=1 colspan=1>7/20(32.1)</td><td rowspan=1 colspan=1>20/20(33.1)</td><td rowspan=1 colspan=1>12/20(33.0)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>HGP(Rhinovirus A2)</td><td rowspan=1 colspan=1>8.9TCID50/mL</td><td rowspan=1 colspan=1>20/20(32.3)</td><td rowspan=1 colspan=1>7/20(31.7)</td><td rowspan=1 colspan=1>20/20(31.8)</td><td rowspan=1 colspan=1>5/20(31.9)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>GB(Adenovirus B3)</td><td rowspan=1 colspan=1>4993.0TCID50/mL</td><td rowspan=1 colspan=1>20/20(32.1)</td><td rowspan=1 colspan=1>8/20(34.1)</td><td rowspan=1 colspan=1>20/20(32.4)</td><td rowspan=1 colspan=1>10/20(33.8)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>1.0CCU/mL</td><td rowspan=1 colspan=1>20/20(33.4)</td><td rowspan=1 colspan=1>9/20(34.9)</td><td rowspan=1 colspan=1>20/20(33.1)</td><td rowspan=1 colspan=1>8/20(35.0)</td></tr></table>

a Titer determined in molecular units (copies $/ \mathrm { m L }$ ) by in-house developed and validated qPCR assay.

The QIAStat Dx Respiratory Panel Plus includes defined Ct value cutoffs for SARS-CoV-2, which were established by testing two SARS-CoV-2 isolates. Ct cutoffs for other assays

present on the panel and the internal control assay were established in the original K183597 clearance studies and no changes were made with addition of SARS-CoV-2 detection. The Ct cutoffs are included as automatic calculations in the assay definition file (ADF).

8. Accuracy (Instrument):

Not Applicable

# 9. Carry-Over:

A carryover study was performed using the QIAstat-Dx Respiratory Panel Plus to evaluate the potential occurrence of cross-contamination between consecutive runs and carryover between cartridge chambers on the QIAstat-Dx Analyzer 1.0. Two panels consisting of high concentrations of influenza A, parainfluenza virus 3, and M. pneumoniae (panel A) or coronavirus NL63, RSV, and SARS-CoV-2 (panel B) in simulated NPS matrix were each tested multiple times between runs of negative (no analyte) cartridges for a total of 62 runs across several days. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. No carryover between cartridges or chambers within the cartridges was observed.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not Applicable

2. Matrix Equivalency Study:

Equivalent performance of simulated NPS matrix and natural clinical NPS matrix was established by testing SARS-CoV-2 isolate England/02/2020 at either 1x LoD or 0.1x LoD concentrations, co-spiked with a subset of on-panel organisms at a low concentration (1x LoD). SARS-CoV-2 was also tested alone. Testing of 20 replicates per each combination was performed with a SARS-CoV-2 detection rate of $\ge 9 5 \%$ for all samples at 1x LoD. Below the LoD testing performed as expected and, thus, overall matrix equivalency was established.

Additionally, this study demonstrates that the LoDs of single-spiked SARS-CoV-2 samples are equivalent to the LoDs determined for SARS-CoV-2 under multi-spiked conditions.

# C Clinical Studies:

# 1. Clinical Performance

The clinical performance of the SARS-CoV-2 assays in the QIAstat-Dx Respiratory Panel Plus was established through a multi-center prospective (i.e., all comers) clinical study conducted at five (5) geographically diverse study sites in the U.S. [(1) Northridge, CA; (2) Redondo Beach, CA; (3) Nashville, TN; and (4) Easley, SC; and (5) Milwaukee, WI]. Each site was tasked with prospective sample collection and testing using the candidate device and the comparator method. Site 5 was further tasked with discordant testing.

Note: Clinical performance for the non-SARS-CoV-2 analytes was established in K183597 and not repeated here.

During this study, nasopharyngeal swab (NPS) specimens were prospectively collected in UTM from individuals with signs and symptoms of respiratory infection, from February to May 2023 and during February 2024.

The clinical performance of the SARS-CoV-2 assays in the QIAstat-Dx Respiratory Panel Plus was established by comparing results to an FDA-cleared molecular respiratory panel under 21 CFR 866.3981 that includes SARS-CoV-2.

A total of 616 prospective NPS specimens were enrolled and tested in this clinical study. One specimen was excluded due to failure to meet the inclusion criteria. Overall, 615 evaluable specimens were included in the performance analysis.

# Study demographics are summarized in Table 13.

Table 13. Prospective Specimens: Demographic Summary of Evaluable Subjects   

<table><tr><td colspan="1" rowspan="2">Sex</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Percentage</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">391</td><td colspan="1" rowspan="1">63.58</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">224</td><td colspan="1" rowspan="1">36.42</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">615</td><td colspan="1" rowspan="1">100.00</td></tr><tr><td colspan="1" rowspan="2">Age Group</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Percentage</td></tr><tr><td colspan="1" rowspan="1">&lt;5</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">3.74</td></tr><tr><td colspan="1" rowspan="1">5 - 9</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">3.74</td></tr><tr><td colspan="1" rowspan="1">10 - 13</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.65</td></tr><tr><td colspan="1" rowspan="1">14 - 18</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">5.04</td></tr><tr><td colspan="1" rowspan="1">19 - 29</td><td colspan="1" rowspan="1">208</td><td colspan="1" rowspan="1">33.82</td></tr><tr><td colspan="1" rowspan="1">30 - 49</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1">33.50</td></tr><tr><td colspan="1" rowspan="1">50 - 69</td><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">16.59</td></tr><tr><td colspan="1" rowspan="1">70 - 85</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">2.93</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">615</td><td colspan="1" rowspan="1">100.00</td></tr><tr><td colspan="1" rowspan="2">Race</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Percentage</td></tr><tr><td colspan="1" rowspan="1">American Indian or AlaskanNative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.33</td></tr><tr><td colspan="1" rowspan="1">American Indian or AlaskanNative, Other</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">3.09</td></tr><tr><td colspan="1" rowspan="1">Asian, White</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.49</td></tr><tr><td colspan="1" rowspan="1">Black or African American</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">9.11</td></tr><tr><td colspan="1" rowspan="1">Black or African American,Native Hawaiian or OtherPacific Islander, White</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">Black or African American,White</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">Native Hawaiian or OtherPacific Islander</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">Native Hawaiian or OtherPacific Islander, White</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">Not Reported</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">13.82</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">8.29</td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">391</td><td colspan="1" rowspan="1">63.58</td></tr><tr><td colspan="1" rowspan="1">White, Other</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.49</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">615</td><td colspan="1" rowspan="1">100.00</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2 VaccinationStatus</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Percentage</td></tr><tr><td colspan="1" rowspan="1">Not vaccinated</td><td colspan="1" rowspan="1">132</td><td colspan="1" rowspan="1">21.46</td></tr><tr><td colspan="1" rowspan="1">Prefer not to say/Noinformation</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">11.22</td></tr><tr><td colspan="1" rowspan="1">Vaccinated</td><td colspan="1" rowspan="1">414</td><td colspan="1" rowspan="1">67.32</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">615</td><td colspan="1" rowspan="1">100.00</td></tr></table>

The overall performance of the QIAstat-Dx Respiratory Panel Plus SARS-CoV-2 assays only is shown in Table 14.

Table 14. QIAstat-Dx Respiratory Panel Plus Prospective Clinical Performance Summary for SARS-CoV-2 Assays   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>61 / 631</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>89.0%-99.6%</td><td rowspan=1 colspan=1>551 / 5522</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.0%-100.0%</td></tr></table>

TP-True Positive, FP-False Positive, TN-True Negative, FN-False Negative 1 The two samples with false negative SARS-CoV-2 results by the QIAstat-Dx Respiratory Panel Plus were both positive by two FDA-EUA molecular SARS-CoV-2 assays. 2 The single sample with a false positive SARS-CoV-2 result by the QIAstat-Dx Respiratory Panel Plus was positive by two FDA-EUA molecular SARS-CoV-2 assays.

The PPA for SARS-CoV-2 is $9 6 . 8 \%$ with a two-sided $9 5 \%$ CI of $8 9 . 0 \% 0 - 9 9 . 6 \%$ . The NPA for SARS-CoV-2 is $9 9 . 8 \%$ with a two-sided $9 5 \%$ CI of $9 9 . 0 \% - 1 0 0 \%$ .

# Co-infection Summary

There were two specimens that yielded a co-infection including SARS-CoV-2 by the QIAstat-Dx Respiratory Panel Plus. One was positive for Human Metapneumovirus and the second was positive for Rhinovirus/Enterovirus. Both results were true positive based on the comparator result.

To confirm the clinical performance of the original panel assays and to demonstrate clinical performance equivalency between the two devices (the previously cleared QIAstat-Dx Respiratory Panel and the candidate device, QIAstat-Dx Respiratory Panel Plus), a subset of original panel analytes was chosen as a representative panel of analytes for a comparison study. The representative panel covers all target analyte types (i.e., RNA and DNA viruses and a bacterium). Additionally, the panel was constructed by choosing one analyte from each Reaction Chamber within the test cartridge. The final list includes the following organisms:

1. Influenza B   
2. Coronavirus OC43   
3. Parainfluenza virus 3   
4. Rhinovirus/enterovirus   
5. Adenovirus   
6. Bordetella pertussis

A total of 190 de-identified clinical NPS specimens, both positive and negative as per Standard of Care, were obtained and tested for the representative panel analytes. Each specimen was run in parallel on the QIAstat-Dx Respiratory Panel and the QIAstat-Dx Respiratory Panel Plus. Performance estimates were calculated using the comparator method (QIAstat-Dx Respiratory Panel) as the reference result. The performance data, calculated in positive percent (PPA) and negative percent agreement (NPA) are summarized in Table 15.

Clinical performance of the QIAstat-Dx Respiratory Panel Plus demonstrated that the addition of SARS-CoV-2 and a decrease in the lysis reagent concentration had no impact to the performance of the original panel assays.

Table 15. PPA and NPA of the QIAstat-Dx Respiratory Panel Plus with Representative Panel Analytes   

<table><tr><td rowspan=1 colspan=2>Grouping Variable(s)</td><td rowspan=1 colspan=2>Proportion</td><td rowspan=1 colspan=2>Two-Sided 95% Confidence Limit</td></tr><tr><td rowspan=1 colspan=1>Analysis</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Fractiona</td><td rowspan=1 colspan=1>Percentage</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=6 colspan=1>PPA</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.2%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>22 / 22</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>84.6%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>24 / 24</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.8%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus / Enterovirus</td><td rowspan=1 colspan=1>43 / 43</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.8%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38 / 40</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>83.1%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>24 / 24</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.8%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=6 colspan=1>NPA</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>167 / 168</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>166 / 166</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>163 / 164</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus / Enterovirus</td><td rowspan=1 colspan=1>144 / 145</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>96.2%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>144 / 148</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>93.2%</td><td rowspan=1 colspan=1>99.3%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>163 / 164</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

a The performance of the QIAstat-Dx Respiratory Panel Plus was determined by comparing against results obtained from the QIAstat-Dx Respiratory Panel.

2. Clinical Specificity: See section “Clinical Performance” above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not Applicable

# C Clinical Cut-Off:

Not Applicable

# D Expected Values/Reference Range:

Table 16. Summary of Expected Values of SARS-CoV-2 by age based on QIAstat-Dx Respiratory Panel Plus Result

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Overall (n=615)</td><td rowspan=1 colspan=2>2-21 yrs. (n=108)</td><td rowspan=1 colspan=2>22-49 yrs. (n=387)</td><td rowspan=1 colspan=2>&gt;49 yrs. (n=120)</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>NPositive</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>NPositive</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>NPositive</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>NPositive</td><td rowspan=1 colspan=1>%Positive</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>10.1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20.0</td></tr></table>

# E Other Supportive Instrument Performance Characteristics Data: Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.