# February 8, 2024

T2 Biosystems, Inc.   
Rachel Gilbert   
Manager, Regulatory Affairs   
101 Hartwell Avenue   
Lexington, Massachusetts 02421

Re: K233184 Trade/Device Name: T2Bacteria Panel Regulation Number: 21 CFR 866.3960 Regulation Name: Nucleic Acid-Based Device For The Amplification, Detection, And Identification Of Microbial Pathogens Directly From Whole Blood Specimens Regulatory Class: Class II Product Code: QBX, NSU Dated: September 28, 2023 Received: September 28, 2023

Dear Rachel Gilbert:

We have reviewed your section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

# Ribhi Shawar -S

Ribhi Shawar, Ph.D. (ABMM)   
Branch Chief   
General Bacteriology and Antimicrobial Susceptibility   
Branch   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K233184

Indications for Use (Describe)

The T2Bacteria Panel run on the T2Dx Instrument is a qualitative T2 Magnetic Resonance (T2MR) test for the direct detection of bacterial species in K2EDTA human whole blood specimens from patients with suspected bacteremia. The T2Bacteria Panel identifies six species of bacteria: Acinetobacter baumannii, Enterococcus faecium, Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa and Staphylococcus aureus.

The T2Bacteria Panel is indicated as an aid in the diagnosis of bacteremia and results should be used in conjunction with other clinical and laboratory data. Concomitant blood cultures are necessary to recover organisms for susceptibility testing or further identification and for organisms not detected by the T2Bacteria Panel.

Results from the T2Bacteria Panel are not intended to be used as the sole basis for diagnosis, treatment, or other patient management decisions in patients with suspected bacteremia.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# 510(k) Summary

<table><tr><td colspan="2"></td></tr><tr><td>Date of Summary</td><td>February 5, 2024</td></tr><tr><td>Product Name_</td><td>T2Bacteria® Panel</td></tr><tr><td>Sponsor</td><td>T2Biosystems, Inc.</td></tr><tr><td></td><td>101 Hartwell Avenue</td></tr><tr><td></td><td>Lexington, MA 02421</td></tr><tr><td>Correspondent</td><td>Rachel Gilbert</td></tr><tr><td></td><td>Associate Director, Regulatory Affairs</td></tr><tr><td></td><td>781-226-2767, 1970</td></tr><tr><td></td><td>rgilbert@t2biosystems.com</td></tr><tr><td>Device Trade or Proprietary Name</td><td>T2Bacteria® Panel</td></tr><tr><td>Regulation</td><td>21 CFR 866.3960</td></tr><tr><td>Common Name</td><td>Nucleic acid-based device for the amplification, detection and identification of microbial pathogens directly from whole blood</td></tr><tr><td></td><td>specimens</td></tr><tr><td>Product Code</td><td>QBX, NSU</td></tr><tr><td>Classification</td><td>Class II</td></tr></table>

The purpose of this pre-market 510(k) submission is to amend the T2Bacteria Panel ini�ally cleared under K172708. The following changes to the Panel are addressed within the submission:

• Addi�on of Acinetobacter baumannii to the device intended use/indica�ons for use • Removal of the warnings related to false posi�ves for E. coli and P. aeruginosa from the panel labeling

No changes have been made to the device components or technology since the ini�al clearance of the T2Bacteria Panel.

# Intended Use

The T2Bacteria® Panel run on the ${ \mathsf { T } } 2 { \mathsf { D } } \mathsf { x } ^ { \circledast }$ Instrument is a qualita�ve T2 Magne�c Resonance $\mathopen { } \mathclose \bgroup \left( \mathrm { T } 2 \mathsf { M } \mathsf { R } ^ { \circledast } \aftergroup \egroup \right)$ test for the direct detec�on of bacterial species in $\mathsf { K } _ { 2 } \mathsf { E D T A }$ human whole blood specimens from pa�ents with suspected bacteremia. The T2Bacteria Panel iden�fies six species of bacteria: Acinetobacter baumannii, Enterococcus faecium, Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa and Staphylococcus aureus.

The T2Bacteria Panel is indicated as an aid in the diagnosis of bacteremia and results should be used in conjunc�on with other clinical and laboratory data. Concomitant blood cultures are necessary to recover organisms for suscep�bility tes�ng or further iden�fica�on and for organisms not detected by the T2Bacteria Panel.

Results from the T2Bacteria Panel are not intended to be used as the sole basis for diagnosis, treatment, or other pa�ent management decisions in pa�ents with suspected bacteremia.

# Limita�ons

For prescrip�on use only.

Please refer to the T2Bacteria Panel labeling for a more complete list of warnings, precau�ons, and contraindica�ons.

# Methodology

The T2Bacteria Panel detects and iden�fies six bacterial target species directly from whole blood specimens and independent of blood culture using nucleic acid amplifica�on and proprietary T2MR detec�on technology. The assay is performed on the proprietary T2Dx pla�orm. The whole blood specimen, drawn into a blood collec�on tube containing $\mathsf { K } _ { 2 } \mathsf { E D T A }$ is used for the test. The blood collec�on tube containing a minimum of $3 \mathsf { m l }$ of blood is loaded directly onto the T2Dx instrument as part of the assembled Cartridge, a single use self-contained unit that contains all of the reagents and disposables required to run a single test. Fully automated on the T2Dx, the blood specimen is mixed with the Lysis Reagent to lyse the red blood cells and the bacterial cells are concentrated by centrifuga�on. The Internal Control is added to the concentrated bacterial cells, which then undergo a bead bea�ng step to lyse the bacteria cells. The supernatant containing the DNA from the lysed bacterial cells and the Internal Control are amplified with the target and Internal Control specific primers. The generated amplicon is then aliquoted into individual tubes containing target specific conjugated par�cles for Acinetobacter baumannii, Enterococcus faecium, Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa, Staphylococcus aureus, and the Internal Control. These individual tubes are read in the MR reader and a signal is generated.

Up to seven specimens can be loaded onto the T2Dx Instrument in parallel. When running a single specimen, the first result is reported in 3.5 hours from the �me the specimen is loaded onto the instrument. The results are interpreted using the T2Dx applica�ons so�ware as valid or invalid, and if valid, target specific results are reported as Posi�ve or Target not Detected. For one target, the Escherichia coli channel, results are reported as Posi�ve, Indeterminate, or Target not Detected. An Indeterminate result is a valid result, but the presence or absence of Escherichia coli cannot be defini�vely assessed, and the indeterminate status applies only to the Escherichia coli channel. Results are displayed on the T2Dx touchscreen and can be printed. Raw T2MR data are not available to the end user.

# Summary of Analy�cal Performance (Acinetobacter baumannii)

Limit of Detection (LoD)

The limit of detec�on (LoD) for was determined by spiking whole blood specimens with A. baumannii. The LoD is defined as the lowest concentra�on $( C F U / \mathsf { m L } )$ of A. baumannii. that can be detected at a rate $\geq$ $9 5 \%$ . The LoD was established by tes�ng a minimum of twenty replicates each of two strains of A. baumannii. at mul�ple concentra�ons. The LoD established for A. baumannii. in the T2Bacteria Panel is shown in Table 1.

Table 1: T2Bacteria Confirmed Limit of Detection – Acinetobacter baumannii   

<table><tr><td>Species</td><td>Strain 1 LoD</td><td>Strain 2 LoD</td><td>Final Confirmed LoD</td></tr><tr><td>Acinetobacter baumannii</td><td>3 CFU/mL</td><td>3 CFU/mL</td><td>3 CFU/mL</td></tr></table>

# Reproducibility

A mul�center reproducibility study was performed to determine the run to run, reagent lot, day to day and site to site reproducibility. Tes�ng was performed at three sites (two external and one internal) with a panel of five target species, each tested in triplicate at two concentra�ons (1-2X LoD and 3-4X LoD) using two reagent lots. Tes�ng was performed for six non-consecu�ve days with at least two operators per site for a total of 36 replicates per sample per site.

The reproducibility panel was comprised of each target species spiked in fresh human whole blood specimens in triple-spiked samples (A. baumannii / K. pneumoniae / S. aureus or E. coli / P. aeruginosa / E. faecium). Bacterial levels were confirmed by colony count tes�ng of the original suspension used for spiking. A total of 108 nega�ve blood samples were included in reproducibility panel.

Table 2: Summary of Reproducibility Results – Acinetobacter baumannii   

<table><tr><td>Sample Type</td><td colspan="2">Concentration</td><td>Ab</td></tr><tr><td rowspan="5">A.baumannii, S. aureus and K. pneumoniae Spike</td><td rowspan="3">1-2 x LoD</td><td>N Pos / N Total</td><td>108/108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td>95% Cl</td><td>96.6-100</td></tr><tr><td rowspan="3">3-4 x LoD</td><td>N Pos / N Total</td><td>108/108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td>95% CI</td><td>96.6-100</td></tr><tr><td rowspan="5">E. coli, P. aeruginosa, and E. faecium Spike</td><td rowspan="3">1-2 x LoD</td><td>N Pos / N Total</td><td>1/108</td></tr><tr><td>% Accurate</td><td>99.1</td></tr><tr><td>95% CI</td><td>94.9-100</td></tr><tr><td rowspan="2">3-4 x LoD</td><td>N Pos / N Total</td><td>0/108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td rowspan="4">Negatives</td><td rowspan="4">N/A</td><td>95% CI</td><td>96.6-100 108 /</td></tr><tr><td>N Neg / N Total</td><td>108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td>95% CI</td><td>96.6-100</td></tr></table>

# Analytical Reactivity (Inclusivity)

Analy�cal reac�vity tes�ng was conducted to ensure that the T2Bacteria Panel is capable of detec�ng mul�ple strains of A. baumannii. Clinical isolates were chosen based on resistance, phylogene�c, temporal, and geographic diversity and spiked into whole blood at 2-3x the established LoD.

A total of 14 organisms were evaluated for inclusivity of A. baumannii on the T2Bacteria Panel. Tes�ng was performed in triplicate. In the event of a false nega�ve result, tes�ng was repeated with 20 replicates and 19/20 replicates had to generate a posi�ve result to be considered passing. Test results summarized in Table 3 demonstrate that the T2Bacteria Panel is able to detect mul�ple strains of A. baumannii.

Table 3: Results Summary for T2Bacteria Reactivity (Inclusivity) Testing – Acinetobacter baumannii   

<table><tr><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>N=3</td><td rowspan=1 colspan=1>N=20</td><td rowspan=1 colspan=1>Study Pass/Fail</td></tr><tr><td rowspan=1 colspan=1>17961</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>BAA-2093</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>BAA-1878</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>BAA-747</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>BAA-1797</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>51432</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>17904</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>BAA-1605</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>13421</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>LMG 10542</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>LMG 10551</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>LMG 1157</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>LMG 22454</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>CCUG 6644</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>Pass</td></tr></table>

# Analytical Specificity (Exclusivity)

Analy�cal exclusivity tes�ng of the T2Bacteria Panel was conducted to assess the cross-reac�vity of A. baumannii to non-panel species at 1,000 units/mL concentra�ons (CFU, ${ \mathsf { T C l D } } _ { 5 0 } ,$ or copies $/ \mathsf { m L }$ where applicable) of pathogenically, phylogene�cally, or environmentally relevant organisms in whole blood. Species that were shown to be poten�ally cross-reac�ve at the ini�al test concentra�on were further evaluated at lower target concentra�ons using a pre-defined �tra�on scheme (100, 33, and 10 units/mL). Analy�cal tes�ng of the T2Bacteria Panel included 128 different organisms comprised of the T2Bacteria Panel members themselves, viruses, and pathogenically, phylogene�cally, or environmentally relevant bacterial and fungal species. The test results establish the specificity of A. baumannii on the T2Bacteria Panel in the presence of all organisms tested at 1,000 units / mL.

Table 4: Bacteria Species for which no reactivity was detected at 1,000 CFU/mL – Acinetobacter baumannii   

<table><tr><td colspan="3">Non-Reactive Bacteria Species</td></tr><tr><td>Acinetobacter calcoaceticus</td><td>Escherichia fergusonii</td><td>Raoultella ornithinolytica</td></tr><tr><td>Acinetobacter Iwoffi</td><td>Finegoldia magna</td><td>Raoultella planticola</td></tr><tr><td>Acinetobacter nosocomialis</td><td>Fusobacterium necrophorum</td><td>Salmonella enterica Enteritidis</td></tr><tr><td>Acinetobacter pitti</td><td>Fusobacterium nucleatum</td><td>Salmonella enterica Typhimurium</td></tr><tr><td>Acinetobacter radioresistans</td><td>Klebsiella oxytoca</td><td>Serratia marcescens</td></tr><tr><td>Actinomyces israelii</td><td>Klebsiella variicola</td><td>Shewanella putrefaciens</td></tr><tr><td>Aeromonas hydrophila</td><td>Lactobacillus acidophilus</td><td>Shigella boydii</td></tr><tr><td>Bacteroides fragilis</td><td>Lactococcus lactis</td><td>Shigella dysenteriae</td></tr><tr><td>Burkholderia cepacia</td><td>Leptotrichia trevisanii</td><td>Shigella flexneri</td></tr><tr><td>Chryseobacterium indologenes</td><td>Leuconostoc mesenteroides</td><td>Shigella sonnei</td></tr><tr><td>Citrobacter koseri</td><td>Listeria monocytogenes</td><td>Staphylococcus auricularis</td></tr><tr><td>Clostridium sphenoides</td><td>Moraxella catarrhalis</td><td>Staphylococcus capitis</td></tr><tr><td>Corynebacterium jeikeium</td><td>Morganella morganii</td><td>Staphylococcus epidermidis</td></tr><tr><td>Cupriavidus pauculus</td><td>Myroides odoratus</td><td>Staphylococcus haemolyticus</td></tr><tr><td>Enterobacter aerogenes</td><td>Ochrobactrum anthropi</td><td>Staphylococcus hominis</td></tr><tr><td>Enterobacter cloacae</td><td>Oligella urethralis</td><td>Staphylococcus lugdunensis</td></tr><tr><td>Enterobacter hormaechei</td><td>Pantoea agglomerans</td><td>Staphylococcus saprophyticus</td></tr><tr><td>Enterococcus avium</td><td>Parvimonas micra</td><td>Staphylococcus warneri</td></tr><tr><td>Enterococcus caccae</td><td>Pediococcus pentosaceus</td><td>Staphylococcus xylosus</td></tr><tr><td>Enterococcus casseliflavus</td><td>Peptostreptococcus anaerobius</td><td>Stenotrophomonas maltophilia</td></tr><tr><td>Enterococcus cecorum</td><td>Peptoniphilus harei</td><td>Streptococcus agalactiae</td></tr><tr><td>Enterococcus dispar</td><td>Plesiomonas shigelloides</td><td>Streptococcus anginosus</td></tr><tr><td>Enterococcus durans</td><td>Propionibacterium acnes</td><td>Streptococcus bovis</td></tr><tr><td>Enterococcus faecalis</td><td>Proteus mirabilis</td><td>Streptococcus constellatus</td></tr><tr><td>Enterococcus gallinarum</td><td>Proteus vulgaris</td><td>Streptococcus dysgalactiae</td></tr><tr><td>Enterococcus gilvus</td><td>Providencia stuartii</td><td>Streptococcus oralis</td></tr><tr><td>Enterococcus hirae</td><td>Pseudomonas alcaligenes</td><td>Streptococcus mutans</td></tr><tr><td>Enterococcus italicus</td><td>Pseudomonas fluorescens</td><td>Streptococcus pneumoniae</td></tr><tr><td>Enterococcus malodoratus</td><td>Pseudomonas luteola</td><td>Streptococcus pyogenes</td></tr><tr><td>Enterococcus mundtii</td><td>Pseudomonas oryzihabitans</td><td>Streptococcus salivarius</td></tr></table>

Table 5: Fungal Species for which no reactivity was detected at 1,000 CFU/mL – Acinetobacter baumannii   

<table><tr><td colspan="3">Non-Reactive Bacteria Species</td></tr><tr><td>Enterococcus pallens</td><td>Pseudomonas pseudoalcaligenes (oleovorans)</td><td>Weeksella virosa</td></tr><tr><td>Enterococcus pseudoavium</td><td>Pseudomonas putida</td><td>Yersinia pseudotuberculosis</td></tr><tr><td>Enterococcus raffinosus</td><td colspan="2">Pseudomonas stutzeri</td></tr><tr><td>Escherichia albertii</td><td colspan="2">Ralstonia pickettii</td></tr></table>

Table 6: Viral Species for which no reactivity was detected at 1,000 units – Acinetobacter baumannii   

<table><tr><td colspan="3">Non-Reactive Fungal Species</td></tr><tr><td>Aspergillus fumigatus</td><td>Candida parapsilosis</td><td>Fusarium oxysporum</td></tr><tr><td>Aspergillus niger</td><td>Candida tropicalis</td><td>Rhizomucor miehei</td></tr><tr><td>Candida albicans</td><td>Cryptococcus albidus</td><td>Rhizopus microsporus</td></tr><tr><td>Candida glabrata</td><td>Cryptococcus neoformans</td><td>Rhizopus oryzae</td></tr><tr><td>Candida krusei</td><td>Fusarium moniliforme</td><td>Rhodotorula glutinis</td></tr></table>

<table><tr><td colspan="3">Non-Reactive Viral Species</td></tr><tr><td>Adenovirus1</td><td>Epstein-Barr Virus²</td><td>Herpes Simplex Virus 11</td></tr><tr><td>Cytomegalovirus1</td><td>Hepatitis A Virus²</td><td>Herpes Simplex Virus 21</td></tr><tr><td>Enterovirus Type 681</td><td>Hepatitis B Virus1</td><td>Varicella-Zoster Virus2</td></tr></table>

1Units = TCID50/mL; 2Units= Copies/mL

# Interfering Substances

Studies were conducted to evaluate the impact of poten�al endogenous and exogenous interfering substances on the performance of A. baumannii on the T2Bacteria Panel. These substances were added to nega�ve whole blood samples or to whole blood samples mul�-spiked with either A. baumannii at 2- 3x LoD. Three replicate samples were run for each interfering substance tested.

All of the substances were tested in excess of standard reference or physiological levels and did not interfere with the performance of A. baumannii with the excep�on of Ferumoxytol (Feraheme). Ini�ally, Ferumoxytol was tested at $6 1 8 \mu \mathrm { g / m L }$ , which is threefold higher than its tmax of $2 0 6 ~ { \mu \xi } / { \ m } L$ , but was found to be inhibitory to the performance of A. baumannii on the T2Bacteria Panel. Dilu�ons of Ferumoxytol were performed and it was determined that concentra�ons $\geq 2 1 \mu \mathrm { g } / \mathrm { m L }$ interfere with the performance of A. baumannii on the T2Bacteria Panel.

Table 7: Substances Tested for Interference with the T2Bacteria Panel: No Interference Observed – Acinetobacter baumannii   

<table><tr><td colspan="2">Endogenous Substances &amp; Concentrations</td><td colspan="4">Exogenous Substances &amp; Concentrations</td></tr><tr><td>Albumin</td><td>60 g/L</td><td>Amphotericin B Trihydrate</td><td>240 μg/mL</td><td>Gentamicin sulfate</td><td>21 μmol/L</td></tr><tr><td>ALT</td><td>120 U/liter</td><td>Ampicillin</td><td>152 μmol/L</td><td>Heparin</td><td>3,000 U/L</td></tr><tr><td>AST</td><td>144 U/liter</td><td>Azithromycin</td><td>15.3 μmol/L</td><td>Isovue 370</td><td>180 μL per 4mL vacutainer</td></tr><tr><td>Bilirubin (conjugated)</td><td>342 μmol/L</td><td>Caspofungin</td><td>52.8 μg/mL</td><td>Linezolid</td><td>55.8 μg/mL</td></tr><tr><td>Bilirubin (unconjugated)</td><td>342 μmol/L</td><td>Cefepime Hydrochloride</td><td>492 μg/mL</td><td>Lisinopril</td><td>0.74 μmol/L</td></tr><tr><td>Creatinine</td><td>50 mg/L</td><td>Cefazolin Sodium Salt</td><td>2.643 mmol/L</td><td>Magnevist (gadopentetate dimeglumine)</td><td>1.5 mM</td></tr><tr><td>Gamma Globulin</td><td>60 g/L</td><td>Cefoxitin Sodium Salt</td><td>180 μg/mL</td><td>Meropenem trihydrate</td><td>186 μg/mL</td></tr><tr><td>Hemoglobin</td><td>22.8 - 23.9 g/dL</td><td>Ceftazidime Pentahydrate</td><td>487 μg/mL</td><td>Metronidazole</td><td>701 μmol/L</td></tr><tr><td>Human DNA</td><td>2.2 μg/mL</td><td>Ciprofloxacin</td><td>30.2 μmol/L</td><td>Micafungin</td><td>90 mg/L</td></tr><tr><td>Intralipid (to mimic triglycerides)</td><td>3270 mg/dL</td><td>Clindamycin HCI</td><td>89.1 μmol/L</td><td>Piperacillin/Pipril</td><td>117 μg/mL</td></tr><tr><td>Lactoferrin</td><td>7.5 μmol/L</td><td>Cytarabine</td><td>32.4 μg/mL</td><td>Primaxin, 50:50 ratio of Imipenem:</td><td>528 μg/mL</td></tr><tr><td>Urea</td><td>42.9 mmol/L</td><td>Dexamethasone</td><td>1.53 μg/mL</td><td>Cilastatin Tazobactam (Tazobac)</td><td>18.9 μg/mL</td></tr><tr><td rowspan="3">White Blood Cells</td><td>2.08 x 107 - 2.48 x 107 cells/mL</td><td>EDTA</td><td>5.4 mg/mL</td><td>Vancomycin</td><td>103 μg/mL</td></tr><tr><td></td><td>Fluconazole</td><td>245 μmol/L</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

Competitive Inhibition

A Compe��ve Inhibi�on Study was conducted on the T2Bacteria Panel to evaluate A. baumannii performance in the presence of other Panel bacterial target species at high and low concentra�ons as well as selected bacterial and fungal non-Panel target organisms. The condi�ons tested included: co-infec�on with A. baumannii and an addi�onal Panel target species both at or near the LoD; co-infec�on with A. baumannii and an addi�onal Panel target species where one species is at high �ter $( 1 , 0 0 0 ~ { \mathsf { C F U / m L } } )$ ) and the other is at or near the LoD; and co-infec�on with A. baumannii at or near the LoD and a non-Panel species at high �ter $( 1 , 0 0 0 ~ { \mathsf { C F U / m L } } )$ . Four replicates were tested and if any false nega�ve results were generated, the test was repeated with 20 replicates.

No compe��ve effects were observed in samples containing compe�ng species at ≤1,000 CFU/mL for A.   
baumannii on the T2Bacteria Panel.

# Summary of Clinical Performance (Acinetobacter baumannii)

The performance of A. baumannii on the T2Bacteria Panel was evaluated at eleven sites within the US and compared to the reference method of blood culture. Pa�ents were enrolled prospec�vely and two paired sample collec�ons, one for blood culture and one for tes�ng by the T2Bacteria Panel were drawn from each subject. The blood culture systems used in the study were BD Bactec™ FX, bioMerieux BacT/ALERT™, and Thermo Fisher VersaTREK®. Species iden�fica�on was performed on all posi�ve bacteria cultures and methods included Gram stain, bioMerieux Vitek® 2, bioMerieux or Bruker MALDI TOF, and PCR. The T2Bacteria Panel A. baumannii result was compared against results from these blood culture systems for Posi�ve Percent Agreement (PPA) and Nega�ve Percent Agreement (NPA). A total of 1,427 subjects were tested prospec�vely. Due to the low prevalence of the organisms contained in the Panel, an addi�onal 50 contrived specimens were evaluated at three sites. Contrived specimens were prepared by spiking A. baumannii at defined concentra�ons $( C F U / \mathsf { m L } )$ into healthy donor whole blood. Further, an addi�onal 300 blood samples not spiked with A. baumannii members were also evaluated as part of the contrived arm of the study. 250 of these blood samples were spiked with the addi�onal 5 bacterial species available on the T2Bacteria Panel (50 each).

Table 8 summarizes the overall PPA (sensi�vity) and NPA (specificity) of A. baumannii from the prospec�ve and contrived arms of the study.

Table 8: Overall Performance of the T2Bacteria Panel in the Clinical Study – Acinetobacter baumannii   

<table><tr><td rowspan="2">Species</td><td colspan="2">Sensitivity (PPA)</td><td colspan="2">Specificity (NPA)</td></tr><tr><td>PPA (TP / (TP + FN))</td><td>95% CI</td><td>NPA (TN / (FP + TN))</td><td>95% CI</td></tr><tr><td>A. baumannii</td><td>97.5% (39/40)</td><td>87.1% - 99.6%</td><td>99.2% (1713/1727)</td><td>98.6% - 99.5%</td></tr></table>

• Sensi�vity (PPA) calculated against samples with �ter levels at or above limit of detec�on (LoD) in Contrived Arm and blood culture posi�ves in Prospec�ve Arm

• Specificity (NPA) calculated from all samples (including below LoD and unspiked nega�ve samples) as the total number of nega�ve channels divided by total number of non-spiked channels in Contrived Arm and blood culture nega�ves in Prospec�ve Arm.

Prospective Arm: T2Bacteria Panel Performance vs. Blood Culture

In the prospec�ve arm of the study, the specificity (NPA) of A. baumannii on the T2Bacteria Panel against blood culture from all included subjects was evaluated. The sensi�vity (PPA) could not be calculated for A. baumannii as there were no posi�ve specimens iden�fied by blood culture.

Table 9: T2Bacteria Panel Performance Characteristics for the Prospective Arm of the Clinical Study – Acinetobacter baumannii   

<table><tr><td rowspan="2">Target Species</td><td colspan="2">PPA</td><td colspan="2">NPA</td></tr><tr><td>Sensitivity</td><td>95% CI</td><td>Specificity</td><td>95% Cl</td></tr><tr><td>A. baumannii</td><td>--- (0/0)</td><td></td><td>99.1% (1414/1427)</td><td>98.4 - 99.5%</td></tr></table>

Table 10: T2Bacteria Panel vs. Blood Culture Contingency table, All Included Subjects – Acinetobacter baumannii   

<table><tr><td rowspan=1 colspan=5>Ab</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Blood culture</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>T2</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1414</td><td rowspan=1 colspan=1>1414</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1427</td><td rowspan=1 colspan=1>1427</td></tr><tr><td rowspan=1 colspan=5>Value   - 95% CI + 95% CI</td></tr><tr><td rowspan=1 colspan=1>PPANPA</td><td rowspan=1 colspan=4>---      ---      ---99.1%  98.4%  99.5%</td></tr></table>

# Contrived Arm

Results for the Contrived arm were further analyzed based on A. baumannii concentra�on (either above or below the LoD).

Table 11: PPA for Contrived Specimens Above and Below the LoD – Acinetobacter baumannii   

<table><tr><td rowspan="2">Target Species</td><td rowspan="2">LoD (CFU/mL)</td><td colspan="2">≥ LoD</td><td colspan="2">&lt; LoD</td></tr><tr><td>PPA</td><td>95% Cl</td><td>PPA</td><td>95% CI</td></tr><tr><td>A. baumannii</td><td>3</td><td>97.5% (39/40)</td><td>87.1 - 99.6%</td><td>40.0% (4/10)</td><td>16.8 - 68.7%</td></tr></table>

Evaluation of False Positive Results

Overall, in the prospec�ve study, there were 13 $\scriptstyle { \mathsf { T } } 2 + / { \mathsf { B } } { \mathsf { C } } -$ poten�al false posi�ve A. baumannii posi�ve results. consis�ng of 3 $5 \tau 2 + / \mathsf { B C } +$ concordant results and 155 $\scriptstyle { \mathsf { T } } 2 + / { \mathsf { B } } { \mathsf { C } } -$ poten�al false posi�ve results. Of the 13 poten�al false posi�ve results, 1 represented a result for which an addi�onal blood specimen (drawn at the same �me as the original posi�ve T2 specimen) was posi�ve by an amplifica�on and gene sequencing method.

Table 12: Summary of T2 $( + ) / \mathsf { B C } \left( - \right)$ Results in Prospective Arm – Acinetobacter baumannii   

<table><tr><td rowspan="2">Species</td><td rowspan="2">T2(+) / BC(-) total</td><td rowspan="2">Other Blood Culture positive1</td><td rowspan="2">Sequencing positive²</td><td rowspan="2">T2(+) / BC(-) associated with strong evidence of infection³</td><td rowspan="2">T2(+) /BC(-) associated with other evidence of infection Non-Blood</td><td rowspan="2">T2(+) / BC(-) associated with no evidence of infection</td></tr><tr><td>Matrices Culture Positive4</td></tr><tr><td>A. baumannii</td><td>13</td><td>0</td><td>1</td><td>7.7% (1/13)</td><td>0.0% (0/13)</td><td>92.3 %(12/13)</td></tr></table>

1 Blood cultures posi�ve for the T2 species iden�fied other than the paired blood culture and processed within $\pm \nobreakspace 1 4 \nobreakspace$ days of collec�on of the T2 sample. 2 Sequencing from blood samples drawn at the same �me as collec�on of the T2 sample and posi�ve for the T2 species iden�fied, where this sequencing assay was only run on subjects without posi�ve evidence from other sample sources (footnote 1 and 4). 3 Strong evidence defined as a T2 posi�ve result associated with a blood culture posi�ve from a different draw than T2 draw or a sequencing posi�ve result from a blood sample drawn concurrently with the T2 draw.

# Evalua�on of A. baumannii, E. coli, and P. aeruginosa Detec�on in Nega�ve Blood Samples

Following manufacturing process and facility improvements undertaken by T2 Biosystems to improve the cleanliness of the reagents in the T2Bacteria Panel, data from the tes�ng for ten (10) different lots of T2Bacteria Panel reagents with nega�ve human whole blood was retrospec�vely pulled and evaluated for percent false posi�vity. A total of 980 valid samples were evaluated, 98 replicates for each reagent lot. Tes�ng demonstrated A. baumannii had an overall fall posi�ve percentage of $0 . 2 \%$ , E. coli had an overall false posi�ve percentage of $0 . 5 \%$ , and $P .$ aeruginosa had an overall false posi�ve percentage of $0 . 4 \%$ .

# A. baumannii

Tes�ng nega�ve whole blood with ten different lots of T2Bacteria Panel reagents resulted in two (2) lots that contained a single A. baumannii false posi�ve each. Across the different lots the NPA ranged from $9 9 \%$ to $100 \%$ . Overall results from tes�ng 980 samples demonstrated a $9 9 . 8 \%$ NPA and a false posi�ve percentage of $0 . 2 \%$ .

# E. coli

Tes�ng nega�ve whole blood with ten different lots of T2Bacteria Panel reagents resulted in single false posi�ve detec�on of E. coli in 5 different lots. Across the different lots the NPA ranged from $9 9 \%$ to $100 \%$ . Overall results from tes�ng 980 samples demonstrated a $9 9 . 5 \%$ NPA and a false posi�ve percentage of $0 . 5 \%$ .

# P. aeruginosa

Tes�ng nega�ve whole blood with ten different lots of T2Bacteria Panel reagents resulted in three (3) lots that contained false posi�ves for P. aeruginosa. Across the different lots the NPA ranged from $9 8 \%$ to $100 \%$ . Overall results from tes�ng 980 samples demonstrated a $9 9 . 6 \%$ NPA and a false posi�ve percentage of $0 . 4 \%$ .

Table 13: Percent False Positive – A. baumannii, E. coli, P. aeruginosa   

<table><tr><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>% False Positive</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.1% - 0.7%</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.2% - 1.2%</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.2% - 0.4%</td></tr></table>

# Addi�onal Tes�ng of A. baumannii Contrived Samples

Tes�ng was completed on thirty-three unique strains of Acinetobacter baumannii that were spiked individually into human whole blood at $2 - 3 x$ LoD $( 6 { - } 9 \mathsf { C F U / m L } )$ . Tes�ng was conducted across four (4) different T2Dx devices, u�lized two (2) lots of cartridges, two (2) lots of reagents and blood from five (5) unique donors over 9 tes�ng days. Tes�ng demonstrated posi�ve detec�on of all strains except for one (32/33).

Table 14: PPA for A. baumannii   

<table><tr><td rowspan=1 colspan=1>A. baumanniiConcentrationTested</td><td rowspan=1 colspan=1>PPA(TP/(TP+FN))</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>6-9 CFU/mL</td><td rowspan=1 colspan=1>97.0% (32/33)</td><td rowspan=1 colspan=1>84.7 - 99.5%</td></tr></table>

# Predicate Comparison

Table 15: Comparison Between T2Bacteria Panel and Predicate Device   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">T2Bacteria Panel(New Device)</td><td colspan="1" rowspan="1">T2Bacteria Panel (K172708)(Predicate Device)</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">FDA ProductCode</td><td colspan="1" rowspan="1">QBX, NSU</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">RegulatoryClassification</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">RegulationNumber</td><td colspan="1" rowspan="1">21 CFR 866.3960</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="2">IntendedUse/Indicationsfor Use</td><td colspan="1" rowspan="1">The T2Bacteria Panel run on the T2Dx® Instrumentis a qualitative T2 magnetic resonance (T2MR®) testfor the direct detection of bacterial species inK2EDTA human whole blood specimens frompatients with suspected bacteremia. The T2BacteriaPanel identifies six species of bacteria:Acinetobacter baumannii, Enterococcus faecium,Escherichia coli, Klebsiella pneumoniae,Pseudomonas aeruginosa, and Staphylococcusaureus.The T2Bacteria Panel is indicated as an aid in thediagnosis of bacteremia and results should be usedin conjunction with other clinical and laboratorydata. Concomitant blood cultures are necessary torecover organisms for susceptibility testing orfurther identification and for organisms notdetected by the T2Bacteria Panel.Results from the T2Bacteria Panel are not intended</td><td colspan="1" rowspan="2">The T2Bacteria Panel run on the T2Dx® Instrument isa qualitative T2 magnetic resonance (T2MR®) test forthe direct detection of bacterial species in K2EDTAhuman whole blood specimens from patients withsuspected bacteremia. The T2Bacteria Panel identifiesfive species of bacteria: Enterococcus faecium,Escherichia coli, Klebsiella pneumoniae, Pseudomonasaeruginosa, and Staphylococcus aureus.The T2Bacteria Panel is indicated as an aid in thediagnosis of bacteremia and results should be used inconjunction with other clinical and laboratory data.Concomitant blood cultures are necessary to recoverorganisms for susceptibility testing or furtheridentification and for organisms not detected by theT2Bacteria Panel.Results from the T2Bacteria Panel are not intended tobe used as the sole basis for diagnosis, treatment, orother patient management decisions in patients withsuspected bacteremia.</td></tr><tr><td colspan="1" rowspan="1">to be used as the sole basis for diagnosis,treatment, or other patient management decisionsin patients with suspected bacteremia.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">4 ml whole blood collected in a blood collectiontube with K2EDTA anticoagulant</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Platform</td><td colspan="1" rowspan="1">T2Dx Instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Reagent Trays</td><td colspan="1" rowspan="1">T2Bacteria Test Reagents for detection of bacteria</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test CartridgeFormat</td><td colspan="1" rowspan="1">T2Bacteria Test Cartridge and disposables</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Nucleic acid amplification followed by T2 magneticresonance detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Six (6) different species of bacteria commonlyimplicated in bacteremia: Acinetobacter baumannii,Enterococcus faecium, Escherichia coli, Klebsiellapneumoniae, Pseudomonas aeruginosa, andStaphylococcus aureus</td><td colspan="1" rowspan="1">Five (5) different species of bacteria commonlyimplicated in bacteremia: Enterococcus faecium,Escherichia coli, Klebsiella pneumoniae, Pseudomonasaeruginosa, and Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">WarningsStatements -False-Positives</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">During prospective clinical studies, false positiveresults were observed for E. coli and P. aeruginosa inprospectively collected specimens. Users should beaware of the possibility of occurrence of false positiveresults, especially for E. coli and P. aeruginosa andshould closely monitor QCheck negative controlresults for any trends and determine the need foraction.</td></tr></table>