# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K233410   
B Applicant Luminex Corporation   
C Proprietary and Established Names LIAISON PLEX Respiratory Flex Assay   
D Regulatory Information

<table><tr><td colspan="1" rowspan="1">ProductCode(s)</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">RegulationSection</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">QOF1</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OEM</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OTG</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OZE</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OZX</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panel</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">multiplex nucleic acidassay</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">OZY</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OZZ</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 -Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980 Respiratory viral panelmultiplex nucleic acidassay</td><td colspan="1" rowspan="1">MI - Microbiology</td></tr><tr><td colspan="1" rowspan="1">NSU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570 Instrumentation forclinical multiplex testsystems</td><td colspan="1" rowspan="1">CH - Clinical Chemistry</td></tr></table>

1 Primary Product Code

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to show that the LIASION PLEX Respiratory Flex Assay is substantially equivalent to the BioFire Respiratory Panel 2.1 (RP2.1) (DEN200031) and to obtain clearance for the LIASION PLEX Respiratory Flex Assay.

# B Measurand:

Adenovirus, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza B, Parainfluenza virus 1, Parainfluenza virus 2, Parainfluenza virus 3, Parainfluenza virus 4, Human Enterovirus/Rhinovirus (not differentiated), Respiratory Syncytial Virus (RSV), Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2), Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated), Bordetella holmesii, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae nucleic acid target sequences.

# C Type of Test:

A multiplexed nucleic acid test intended for use with the automated LIAISON PLEX instrument for the qualitative in vitro detection and identification of multiple respiratory pathogen nucleic acids in nasopharyngeal swabs (NPS) collected in BD Universal Transport Media (UVT) or Copan Universal Transport Media (UTM) and obtained from individuals with signs and symptoms of respiratory tract infections.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The LIAISON PLEX Respiratory Flex (RSP Flex) Assay is a multiplexed qualitative test for the simultaneous in vitro detection and identification of multiple bacterial and viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of

respiratory tract infection, including SARS-CoV-2. The test is performed on the automated LIAISON PLEX System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific nucleic acid gene sequences of the following organism types and subtypes:

Viruses:   
Adenovirus   
Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated)   
Human Enterovirus/Rhinovirus (not differentiated)   
Human Metapneumovirus,   
Influenza A   
Influenza A (subtype H1)   
Influenza A (subtype H3)   
Influenza B   
Parainfluenza 1   
Parainfluenza 2   
Parainfluenza 3   
Parainfluenza 4   
Respiratory Syncytial Virus   
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2)

Bacteria: Bordetella holmesii Bordetella parapertussis Bordetella pertussis Chlamydia pneumoniae Mycoplasma pneumoniae

Nucleic acids from the bacterial and viral organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. Detecting and identifying specific bacterial and viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory infection, if used in conjunction with other clinical, epidemiological, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or patient management decisions.

Negative results in the presence of a respiratory illness may be due to infection with pathogens that are not detected by this test or due to lower respiratory tract infection that is not detected by an NPS specimen. Conversely, positive results do not rule out infection or co-infection with organisms not detected by the LIAISON PLEX Respiratory Flex (RSP Flex) Assay. The agent(s) detected may not be the definite cause of disease.

The use of additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography), may be necessary when evaluating a patient with possible respiratory tract infection.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only For in vitro diagnostic use only

D Special Instrument Requirements: For use with the LIAISON PLEX System, only.

# A Device Description:

The LIAISON PLEX Respiratory Flex Assay is performed on the LIAISON PLEX System. The LIAISON PLEX System consists of a touchscreen user interface that includes the software for running and analyzing assay results, one to six processing/imaging LIAISON PLEX modules, and a handheld barcode reader. Each LIAISON PLEX module processes one sample at a time under the control of the LIAISON PLEX System software. The LIAISON PLEX Respiratory Flex Assay components required to perform the test include the following single-use, disposables:

• LIAISON PLEX Respiratory Flex Assay Test Cartridge LIAISON PLEX Respiratory Flex Assay Transfer Pipettes (or equivalent)

Prior to initiating a test on the LIAISON PLEX System, a $3 0 0 \mu \mathrm { L }$ aliquot of NPS in Viral Transport Media (VTM) is pipetted by the user into the Sample Port within the Sample Prep Tray (recommended, but not required), followed by closing the Sample Port Closure. Next, the Sample ID barcode on the sample tube is scanned with the hand-held barcode reader, or the Sample ID is manually entered using the touchscreen keyboard. The user then scans the assay cartridge ID barcode with the hand-held barcode scanner. Lastly, the Respiratory Flex Assay Cartridge is inserted into the module by the user. The LIAISON PLEX System identifies the assay being run and automatically initiates the proper testing protocol to process the sample, analyze the data, and generate test results. Once the test is finished running $\sim 2$ hours), the user ejects the assay cartridge by selecting the green check mark or the eject icon on the touchscreen.

The Respiratory Flex Assay has 19 different reportable targets (organisms and influenza A subtypes H1 and H3). Reporting of these targets is based on detection of one or more of the nucleic acid targets. For each intended Respiratory Flex Assay target, four sequence components, referred to as oligonucleotides (Oligos), consisting of one or more Capture probe(s), Mediator probe(s), forward primer(s), and reverse primer(s), are required.

The LIAISON PLEX Respiratory Flex Assay can be run in Fixed mode, which will return the results for all target analytes. To allow testing flexibility, the LIAISON PLEX Respiratory Flex Assay can be run in Flex mode. This feature allows laboratories to generate custom panels, which include only a subset of analytes selected by the laboratory. No pre-set Flex-panels are defined, and laboratories can choose to create custom panels of specific targets for certain populations and/or seasons, if they desire to incorporate the Flex reporting feature. Target assays that are initially masked can later be unmasked using Flex credits when ordered by a physician. When Flex mode is implemented, all target assays are performed, however the raw data is not analyzed for masked targets until they are unmasked.

# Interpretation of Results

The Respiratory Flex Assay provides a qualitative result for the presence (Detected) or absence (Not Detected) of the Respiratory Flex target nucleic acid gene sequences. The image analysis of the Substrate provides light signal intensities from the target-specific capture spots as well as the internal processing controls, negative control, background, and imaging control spots. The mean signal intensity of a target, after background subtraction, is compared to the assay’s signal detection threshold to make a determination. Table 2 below lists the possible test results generated by the Respiratory Flex Assay representing identification of viral and bacterial nucleic acid sequences/targets.

Table 2. Respiratory Flex Assay Calls for Valid Tests   

<table><tr><td rowspan=1 colspan=1>Test Result Reported as &quot;Detected&quot;</td><td rowspan=1 colspan=1>Reported Target</td></tr><tr><td rowspan=1 colspan=1>Viral Targets</td><td rowspan=1 colspan=1>Viral Targets</td></tr><tr><td rowspan=1 colspan=1>Adenovirus(inclusive to A, B, C, D, E, and F)</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Human Parainfluenza Virus 1</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Human Parainfluenza Virus 2</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Human Parainfluenza Virus 3</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Human Parainfluenza Virus 4</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus(inclusive to HKU1, NL63, OC43, and 229E)</td><td rowspan=1 colspan=1>Human Coronavirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A*</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H1**</td><td rowspan=1 colspan=1>Influenza A subtype H1</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H3**</td><td rowspan=1 colspan=1>Influenza A subtype H3</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=2 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus(inclusive to RSV A and RSV B)</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Bacterial Targets</td><td></td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>Bordetella holmesii</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis (Toxin Promoter Region)</td><td rowspan=1 colspan=1>Bordetella pertussis</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis (IS1001)</td><td rowspan=1 colspan=1>Bordetella parapertussis</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=2>Test Result Reported as &quot;Not Detected</td></tr><tr><td rowspan=1 colspan=2>All Analytes Not Detected</td></tr></table>

\*Detection of Influenza A without an Influenza A/H1 or Influenza A/H3 subtype may occur at low titer of the virus in the specimen or may indicate a false positive due to contamination. The result could also indicate a novel Influenza A strain. In these cases, the sample should be retested. If an Influenza A detected result is obtained without detection of an Influenza A/H1 or $\mathrm { { A / H 3 } }$ subtype upon retesting, contact local or state public health authorities for confirmatory testing.

\*\*Detection of Influenza $\mathrm { A } / \mathrm { H } 1$ or Influenza A/H3 subtypes without an Influenza A “Detected” result may occur at low titer of the virus in the specimen or may indicate a false positive due to contamination. The result could also indicate potential genetic mutations in the Matrix protein gene among circulating seasonal Influenza A viruses. In these cases, the sample should be retested. If an Influenza A/H1 or A/H3 subtype detected result is obtained again without detection of Influenza A upon repeat testing, further investigations may be warranted.

Reasons for invalid (no call) results, together with the appropriate recourse which should be taken by the user, are described in Table 3.

Table 3. LIAISON PLEX Respiratory Flex Assay Invalid Calls and Recourse   

<table><tr><td colspan="1" rowspan="1">Call</td><td colspan="1" rowspan="1">Reason</td><td colspan="1" rowspan="1">Recourse</td></tr><tr><td colspan="1" rowspan="2">No Call</td><td colspan="1" rowspan="1">The hybridization internal control (IC) is not detected1</td><td colspan="1" rowspan="2">Retest from the primarysample beginning withthe assay package insert</td></tr><tr><td colspan="1" rowspan="1">The amplification IC or extraction IC are not detected ANDno DNA or RNA target pathogen is detected, respectively1</td></tr><tr><td colspan="1" rowspan="4">b</td><td colspan="1" rowspan="1">Signal in regions of the microarray which do not containcapture oligos is too high</td><td colspan="1" rowspan="4">section Procedure, usinga new cartridge</td></tr><tr><td colspan="1" rowspan="1">Signal in regions of the microarray containing oligomerspots to ensure proper stringency</td></tr><tr><td colspan="1" rowspan="1">The coefficient of variation of intensities for spots within atleast one spot group is high</td></tr><tr><td colspan="1" rowspan="1">The overall signal across all spot groups, excludingNegative Control and Background, is too high</td></tr></table>

1 Additional information on the ICs (hybridization, amplification, and extraction) is provided in Section IV.C.Instrument Descriptive Information.5.Quality Control.Internal Controls, below.

# B Principle of Operation:

The Respiratory Flex Assay is a multiplexed molecular assay with automated nucleic acid isolation, amplification, and detection of unique genomic sequences of target pathogens. The Respiratory Flex Assay is performed using the LIAISON PLEX System, which is a bench-top sample-to-result molecular diagnostics workstation consisting of a touchscreen user interface that includes the software for running and analyzing assay results, one to six processing/imaging LIAISON PLEX modules, and a handheld barcode reader. The LIAISON PLEX System automates the Respiratory Flex Assay sample analysis steps, which occur within the cartridge, including: (1) Specimen Extraction – Chemical and mechanical RNA/DNA extraction from nasopharyngeal swab specimens obtained from symptomatic patients; (2) Target Amplification - Multiplex RT-PCR- and PCR-based amplification of the extracted nucleic acids to generate target-specific amplicons; (3) Hybridization - Amplicon hybridization to target specific capture DNA in a microarray format and mediator and gold-nanoparticle probe hybridization to captured amplicons. Silver enhancement of the gold nanoparticle probes bound at the capture sites results in gold-silver aggregates that are imaged optically with high efficiency by the LIAISON PLEX System. The user can monitor the status of the assay via the touch screen on the instrument, which displays the run time.

# Contamination Control

The Respiratory Flex Assay includes an Uracil DNA Glycosylase (UDG) enzyme-based strategy to eliminate amplicon contamination. Briefly, the lyophilized amplification master mix formulation contains deoxyuridine triphosphate (dUTP) in place of the standard deoxythymidine triphosphate (dTTP), and during the multiplexed RT-PCR step dUTP is incorporated into the amplicons. Prior to the start of an amplification step, the UDG enzyme renders any dUTP-containing previously generated amplicons non-amplifiable by selectively hydrolyzing at the uracil base, while not impacting the integrity of dTTP containing target RNA. The assay also uses a thermolabile version of UDG enzyme which is inactivated by heat prior to the RT step and does not interfere with the newly generated cDNA and/or the amplicon from the test. While the UDG-based strategy mitigates false positive risk due to lab-based carryover and cross-contamination, incomplete hydrolysis of uracilcontaining amplicons may lead to amplification and detection of a contaminant. Additionally, this strategy does not address genomic contamination during the preparation of the samples. Strict adherence to the prescribed handling/preparation of samples and laboratory/system cleaning protocols and careful disposal of the used consumables can reduce the likelihood of contamination from user-based sources.

# End-Point Detection and Analysis

The target-specific amplicon is detected in an endpoint assay that utilizes a microarray format. For each of the bacterial or viral nucleic acid sequences/analytes detected by the Respiratory Flex Assay, two types of oligonucleotides are required for the endpoint gold nanoparticle probe-based detection: (1) Capture oligonucleotides (or captures) and (2) Mediator oligonucleotides (or mediators). The

Capture oligonucleotides are arrayed on the surface of a substrate (a microarray) within the test cartridge and are designed to specifically bind to one part of the analyte-specific target amplicon. The Mediator oligonucleotides bind to a different portion of the same amplicon and enable binding of gold nanoparticle probes. Notably, in a multiplexed detection system, numerous unique targetspecific mediators can coexist and form unique hybridizations at the different captures on the microarray. Since all the mediators have a target specific region and a poly-A tail region, a single, universal gold nanoparticle poly-T probe is sufficient for target/mediator labeling. Silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency. Light scatter from the capture spots is imaged by the LIAISON PLEX System and intensities from the microarray spots are processed by a decision algorithm to make calls regarding the presence (Detected) or absence (Not Detected) of a nucleic acid sequence/analyte.

# C Instrument Description Information:

1. Instrument Name: LIAISON PLEX System, software version 1.0.0.144.

2. Specimen Identification: Specimen identification information is entered either manually or via barcode.

3. Specimen Sampling and Handling: Nasopharyngeal swab (NPS) specimens collected in BD UVT or Copan UTM.

LIAISON PLEX modules are calibrated during the manufacturing process; calibration is not performed by the user.

5. Quality Control:

Internal Controls

The Assay contains three sets of internal controls to check to ensure performance of sample preparation amplification, and detection. They are described in more detail, below:

1. Extraction control. The extraction control verifies the presence of an amplicon for Bacteriophage MS2, which is added to the sample prior to the nucleic acid extraction step. The extracted control product is amplified and subsequently detected by unique spots on the hybridization array, thereby confirming successful nucleic acid extraction, Reverse Transcription, PCR amplification of RNA targets, and detection.

2. Amplification Control. The amplification control verifies the presence of an amplicon for a synthetic DNA oligonucleotide sequence in the lyophilized PCR master mix. The product is detected by a unique spot on the hybridization array, thereby confirming successful PCR amplification and detection of DNA targets.

3. Hybridization Control. The hybridization control target and mediator oligonucleotide are contained within the Sample Buffer and added to the post-amplification product prior to hybridization. The hybridization control is detected by a unique spot on the hybridization array, thereby confirming successful processing of hybridization and signal enhancement steps.

Internal controls results are reported as Pass, Fail, or N/A (see Table 4 for detailed explanations of each control result). Internal controls must either (1) generate a signal above threshold in each internal reaction for the system to report a valid test result, or (2) the amplification or extraction control result can be below the signal threshold if a DNA or RNA target pathogen is detected, respectively.

Table 4. Interpretation of Internal Control Results for the LIAISON PLEX Respiratory Flex Assay   

<table><tr><td>Internal Control Result</td><td colspan="2">Explanation</td><td>Suggested Action</td></tr><tr><td>Pass</td><td>•</td><td>The hybridization control was detected, indicating successful hybridization. The amplification control was detected, indicating successful amplification. The extraction control was detected, indicating successful extraction.</td><td>Review and report results</td></tr><tr><td>N/A</td><td></td><td>The hybridization control was detected, indicating successful hybridization. A DNA pathogen target was detected, indicating successful amplification. If a DNA pathogen target is detected, the amplification control result is ignored. The extraction control was detected, indicating successful extraction.</td><td>Review and report results</td></tr><tr><td>N/A</td><td></td><td>The hybridization control was detected, indicating successful hybridization. The amplification control was detected, indicating successful amplification. An RNA pathogen target was detected, indicating successful extraction. If an RNA pathogen target is detected, the extraction control result is ignored.</td><td>Review and report results</td></tr><tr><td>Fail</td><td>•</td><td>The hybridization control was not detected, indicating hybridization was not successful. The amplification control, or a DNA pathogen was detected, indicating successful amplification. The extraction control, or an RNA pathogen was detected, indicating successful extraction.</td><td>Repeat test with a new cartridge</td></tr><tr><td>Fail</td><td>• •</td><td>The hybridization control was detected indicating successful hybridization. The amplification control, or a DNA pathogen was not detected, indicating amplification was not successful. The extraction control, or a RNA pathogen was</td><td>Repeat test with a new cartridge</td></tr><tr><td>Fail</td><td></td><td>detected, indicating successful extraction. The hybridization control was detected indicating successful hybridization.</td><td>Repeat test with a new cartridge</td></tr><tr><td></td><td>amplifcation. successful.</td><td>The amplfication control, or a DNA pathogen was detected, indicating successful The extraction control, or a RNA pathogen, was not detected, indicating extraction was not</td><td></td></tr></table>

# External Controls

External controls are not provided with the Respiratory Flex Assay. However, five external control mixes (see Table 5 below) were provided to the clinical study sites for daily testing during the prospective clinical study. External controls were tested on each day of testing, utilizing one external negative control and one of four external positive controls (tested on a rotating basis) representing all Respiratory Flex targets.

Table 5. External Controls Utilized in the Clinical Studies   

<table><tr><td rowspan=1 colspan=1>External Control</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>Positive Run Control - Pool 1</td><td rowspan=1 colspan=1>Adenovirus, Chlamydia pneumoniae, human coronavirus,hMPV, Influenza B, Parainfluenza 1, Parainfluenza 2, RSV</td></tr><tr><td rowspan=1 colspan=1>Positive Run Control - Pool 2</td><td rowspan=1 colspan=1>Bordetella holmesii, Bordetella pertussis,Enterovirus/Rhinovirus, Influenza A, Influenza A (subtypeH1), Influenza A (subtype H3), Mycoplasma pneumoniae,Parainfluenza 3, Parainfluenza 4</td></tr><tr><td rowspan=1 colspan=1>Positive Run Control - Pool 3</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Positive Run Control - Pool 4</td><td rowspan=1 colspan=1>Bordetella parapertussis, Bordetella pertussis</td></tr><tr><td rowspan=1 colspan=1>Negative Run Control</td><td rowspan=1 colspan=1>NA</td></tr></table>

The sponsor is also including the following in the product package insert, “Positive and negative external controls should be tested with each new lot or shipment of reagents, or monthly, (whichever occurs first), or in accordance with updated local, regional, state, and/or federal guidelines. Verified negative nasopharyngeal swab (NPS) specimens can be used as the negative control. Previously characterized positive samples or verified negative NPS specimens spiked with well characterized organisms may be used as the external positive control. External controls should be used in accordance with laboratory protocols and in accordance with local, state, and federal accrediting organizations, as applicable.”

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s): DEN200031

C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K233410</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>LIAISON Plex Respiratory FlexAssay</td><td rowspan=1 colspan=1>BioFire Respiratory Panel 2.1(RP2.1)</td></tr><tr><td rowspan=1 colspan=1>Regulation Number andName</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>21 CFR 866.3981; Devices todetect and identify nucleic acidtargets in respiratory samples from</td></tr></table>

<table><tr><td></td><td></td></tr><tr><td>Product Code</td><td>Same The LIAISON PLEX Respiratory</td></tr><tr><td>Intended Use/Indications For Use</td><td>Flex (RSP Flex) Assay is a multiplexed qualitative test for the simultaneous in vitro detection and identification of multiple bacterial and viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of respiratory tract infection, including SARS-CoV-2. The test is performed on the automated LIAISON PLEX System utilizing reverse transcription (RT), polymerase chain reaction (PCR), and array hybridization to detect specific nucleic acid gene sequences of the following organism types and subtypes: Viruses: Adenovirus Human Coronavirus (HKU1, NL63, OC43, and 229E not differentiated) Human Enterovirus/Rhinovirus (not differentiated) Human Metapneumovirus, Influenza A Influenza A (subtype H1) Influenza A (subtype H3) Influenza B Parainfluenza 1 Parainfluenza 2 Parainfluenza 3 Parainfluenza 4 Respiratory Syncytial Virus Severe Acute Respiratory Syndrome Coronavirus (SARS- CoV-2) Bacteria: Bordetella holmesii Bordetella parapertussis Bordetella pertussis Chlamydia pneumoniae Mycoplasma pneumoniae Nucleic acids from the bacterial and viral organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. Detecting and</td></tr></table>

he following organism types and   
ubtypes are identified using the   
ioFire RP2.1:   
• Adenovirus,   
• Coronavirus 229E,   
• Coronavirus HKU1,   
• Coronavirus NL63,   
• Coronavirus OC43,   
• Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), Human Metapneumovirus, Human Rhinovirus/Enterovirus,   
• Influenza A, including subtypes H1, H3 and H1- 2009, Influenza B,   
• Parainfluenza Virus 1,   
• Parainfluenza Virus 2, Parainfluenza Virus 3,   
• Parainfluenza Virus 4, Respiratory Syncytial Virus, Bordetella parapertussis (IS1001), Bordetella pertussis (ptxP), Chlamydia pneumoniae, and • Mycoplasma pneumoniae   
Nucleic acids from the respiratory   
viral and bacterial organisms   
dentified by this test are generally   
detectable in NPS specimens   
during the acute phase of infection.   
he detection and identification of   
pecific viral and bacterial nucleic

<table><tr><td colspan="3"></td></tr><tr><td>results do not rule out infection or co-infection with organisms not detected by the LIAISON PLEX Respiratory Flex (RSP Flex) Assay. The agent(s) detected may not be the definite cause of disease.</td><td>identifying specific bacterial and viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory infection, if used in conjunction with other clinical, epidemiological, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or patient management decisions. Negative results in the presence of a respiratory illness may be due to infection with pathogens that are not detected by this test or due to lower respiratory tract infection that is not detected by an NPS specimen. Conversely, positive organisms. The agent(s) detected</td><td>signs and/or symptoms of respiratory infection is indicative of the presence of the identified microorganism and aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Negative results in the setting of respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by an NPS specimen. Positive results do not rule out co-infection with other</td></tr><tr><td>The use of additional laboratory</td><td>testing (e.g., bacterial and viral culture, immunofluorescence, and radiography), may be necessary when evaluating a patient with</td><td></td></tr><tr><td>Measurand</td><td>possible respiratory tract infection. Same</td><td>Nucleic acids from target organisms</td></tr><tr><td>Sample Type</td><td>Same LIAISON PLEX System</td><td>Nasopharyngeal swab (NPS) BioFire FilmArray 2.0 or BioFire FilmArray Torch Systems</td></tr><tr><td>Instrumentation Technological Principles</td><td>Highly multiplexed nucleic acid PCR and RT-PCR test with</td><td>Highly multiplexed nested nucleic acid amplification with melt</td></tr><tr><td>Internal Controls</td><td>microarray detection. Multiple internal controls contained in the cartridge monitor</td><td>analysis. Two controls are included in each reagent pouch to control for sample processing and both stages</td></tr><tr><td>Automated Test Processes</td><td>sample processing and RT and PCR functions. Same</td><td>of PCR and melt analysis. Nucleic Acid Extraction and Amplification, Detection and</td></tr><tr><td>Bordetella Species Detected</td><td>Bordetella parapertussis • Bordetella pertussis</td><td>Results Interpretation Bordetella parapertussis • Bordetella pertussis</td></tr><tr><td>Human Coronavirus Result</td><td>Bordetella holmesii The human coronavirus target species (i.e., HKU1, OC43, 229E,</td><td>Each target human coronavirus species (i.e., HKU1, OC43, 229E,</td></tr><tr><td>Reporting</td><td>NL63) are not differentiated.</td><td>NL63) is reported independently.</td></tr><tr><td>Influenza A Subtyping</td><td>Influenza A subtypes H1 and H3 detected/reported.</td><td>Influenza A subtypes H1, H1- 2009, and H3 detected/reported.</td></tr><tr><td>Time to Result</td><td>~ 2 hours</td><td>~45 minutes</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# Standards

ISTA 3A. Packaged-Products for Parcel Delivery System Shipment $7 0 \mathrm { k g } ( 1 5 0 \mathrm { l b } )$ or Less. (2018). ITSA 7D. Temperature Test for Transport Packaging. CLSI EP07. Interference Testing in Clinical Chemistry; Third Edition.   
• CLSI EP37. Supplemental Tables for Interference Testing in Clinical Chemistry; First Edition.   
• CLSI EP25-A. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline.   
• CLSI EP17-A2. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition.   
• CLSI EP05-A3. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline – Third Edition.   
• CLSI EP12-A2. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition.   
• CLSI EP24-A2. Assessment of the Diagnostic Accuracy of Laboratory Testing Using Receiver Operating Characteristic Curves; Approved Guideline – Second Edition. ISO 14971. Medical Devices – Application of Risk Management to Medical Devices. Third Edition (2019-12). ISO 23640. In Vitro Diagnostic Medical Devices – Evaluation of Stability of In Vitro Diagnostic Reagents. ISO 15223-1. Medical Devices – Symbols to be Used with Information to be Supplied by the Manufacturer-Part 1: General Requirements. Fourth Edition (2021-07). ISO 3864-1. Graphic Symbols – Safety Colors and Safety Signs – Part 1. Design Principles for Safety Signs and Safety Markings (2011). IEC 61010-1 Edition 3.1, Consolidated Version. Safety Requirements for Electrical Equipment for Measurement Control and Laboratory Use – Part 1: General Requirements, Including Corrigendum 1. (2017-01). IEC 61326-2-6 Edition 3.0. Electrical Equipment for Measurement Control and Laboratory Use – EMC Requirements – Part 2-6: Particular Requirements – In Vitro Diagnostic (IVD) Medical Equipment (2010-10). IEC 60601-1-2 Edition 4.0. Medial Electrical Equipment – Part 1-2: General Requirements for Basic Safety and Essential Performance – Collateral Standard: Electromagnetic Disturbances-Requirements and Tests (2014-02). IEC 61000-3-2. Electromagnetic Compatibility (EMC) – Part 3-2: Limits for Harmonic Current Emissions, Input Current Up to & Including 16A Per Phase (2014). IEC 61000-3-3. Electromagnetic Compatibility (EMC) – Part 3-3: Limits – Limitation of Voltage Changes, Voltage Fluctuations and Flicker in Public Low – Voltage Supply Systems for Equipment (2013).

# Special Controls

• Class II Special Controls as per 21 CFR 866.3981

Electronic Submission Template for Medical Device 510(k) Submissions - Guidance for Industry and Food and Drug Administration Staff (October 2, 2023). Respiratory Viral Panel Multiplex Nucleic Acid Assay - Class II Special Controls Guidance for Industry and FDA Staff (October 9, 2009). Content of Premarket Submissions for Device Software Functions - Guidance for Industry and Food and Drug Administration Staff (June 14, 2023). Cybersecurity in Medical Devices: Quality System Considerations and Content of Premarket Submissions - Guidance for Industry and Food and Drug Administration Staff (September 23, 2023). • Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests - Guidance for Industry and FDA Staff (March 13, 2007).

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

# a. Within-Laboratory Precision

Within-laboratory precision was evaluated at a single site using the Respiratory Flex Assay run on the LIAISON PLEX System. A total of three contrived panels containing known quantities of the target analytes were prepared in simulated NPS matrix, consisting of HeLa cells at a concentration of $2 \mathrm { x } 1 0 ^ { 3 }$ cells/mL in UTM. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. The viral and bacterial materials used to generate the positive panel members are denoted in Table 6. The contrived positive panels consisted of five representative target organisms co-spiked at a low positive concentration (1.5x LoD) and moderate concentration (5x LoD). A negative panel was also included in the study (see Table 7). The study was conducted with one operator and three cartridge lots over the course of 5 non-consecutive days on two LIAISON PLEX Systems. Each panel member was tested in triplicate once per day on 5 different days generating a total of 45 replicates per panel member (1 Site x 1 Operator x 3 Lots x 5 Days X 1 Run per Day x 3 Replicates per Run).

Table 6. Viral and Bacterial Strains Used in the Within-Laboratory Precision Study   

<table><tr><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>OrganismType</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog Number</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis (9797)</td><td rowspan=1 colspan=1>Bacterium</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9797</td></tr><tr><td rowspan=1 colspan=1>Adenovirus (4E)</td><td rowspan=1 colspan=1>DNA virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810070CF</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Colorado/06/2017)</td><td rowspan=1 colspan=1>RNA virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810573CF</td></tr><tr><td rowspan=1 colspan=1>hMPV (27A2)</td><td rowspan=1 colspan=1>RNA virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810164CF</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA-WA1/2020)</td><td rowspan=1 colspan=1>RNA virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810587CFHI</td></tr></table>

Table 7. Precision Study Sample Panel   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Bordetella pertussis, Adenovirus, Influenza B, hMPV, SARS-CoV-2 (Lowpositive; all analytes at 1.5x LoD)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Bordetella pertussis, Adenovirus, Influenza B, hMPV, SARS-CoV-2(Moderate positive; all analytes at 5x LoD)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Negative</td></tr></table>

The qualitative results (i.e., $\%$ agreement with expected results) from the study are illustrated in Table 8.

Table 8. Within-Laboratory Precision Study – Qualitative Results   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Panel Conc.</td><td rowspan=1 colspan=1>% Positive(pos n/ valid n)</td><td rowspan=1 colspan=1>% Agreement withExpected Results/(95% CI)</td></tr><tr><td rowspan=3 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>93.3% (42/45)</td><td rowspan=1 colspan=1>93.3% (82.1-97.7%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>97.8% (44/45)</td><td rowspan=1 colspan=1>97.8%(88.4-99.6%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>97.8% (44/45)</td><td rowspan=1 colspan=1>97.8%(88.4-99.6%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100%(45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>97.8% (44/45)</td><td rowspan=1 colspan=1>97.8%(88.4-99.6%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>100%(92.1-100%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>100% (92.1-100%)</td></tr></table>

Note: Results are shown only for the intended targets. Panel members co-spiked with 5 different targets are presented 5 times.

All low positive $( 1 . 5 \mathrm { x } \mathrm { L o D } )$ panel members were positive $5 9 7 . 8 \%$ , except for Bordetella pertussis, which had a positivity of $9 3 . 3 \%$ (42/45). All moderate positive $\left( 5 \mathbf { x } \ L \mathbf { o } \mathbf { D } \right)$ panel members were $100 \%$ positive for the spiked target analytes, except for adenovirus and hMPV, which yielded $9 7 . 8 \%$ positivity (44/45). The negative panel member was $100 \%$ negative. There was no lot-to-lot variability observed in the study. The results of the study demonstrate acceptable assay variability.

# b. Reproducibility

A reproducibility study was conducted at three testing sites using the Respiratory Flex Assay run on the LIAISON PLEX System. The study incorporated potential sources of variation introduced by site (three testing sites), day (5 different days), operator (two operators per site), and instrument (six LIAISON PLEX Systems). One lot of Respiratory Flex Assay cartridges was tested at three sites by two operators per site over five days.

The same three contrived panels used to evaluate precision (see Table 7) were included in the reproducibility study. Three replicates of each panel member were tested by each operator at each site on all 5 days of testing generating a total of 90 replicates per panel member. The qualitative results of the study are illustrated in Table 9.

Table 9. Reproducibility Study – Qualitative Results   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">PanelID</td><td colspan="1" rowspan="2">Panel Conc.</td><td colspan="4" rowspan="1">% Agreement with Expected Results</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Overall/(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Bordetellapertussis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Low positive</td><td colspan="1" rowspan="1">93.3%(28/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">96.7%(29/30)</td><td colspan="1" rowspan="1">96.7%(87/90)(90.7-98.9%)</td></tr><tr><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Mod. positive</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Low positive</td><td colspan="1" rowspan="1">96.7%(29/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">96.7%(29/30)</td><td colspan="1" rowspan="1">97.8%(88/90)(92.3-99.4%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Mod. positive</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="3">Influenza B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Low positive</td><td colspan="1" rowspan="1">93.3%(28/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">96.7%(29/30)</td><td colspan="1" rowspan="1">96.7%(87/90)(90.7-98.9%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Mod. positive</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="3">hMPV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Low positive</td><td colspan="1" rowspan="1">90.0%(27/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">93.3%(28/30)</td><td colspan="1" rowspan="1">94.4%(85/90)(87.6-97.6%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Mod. positive</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">93.3%(28/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">97.8%(88/90)(92.3-99.4%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="3">SARS-CoV-2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Low positive</td><td colspan="1" rowspan="1">96.7%(29/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">98.9%(89/90)(94.0-99.8%)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Mod. positive</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100%)</td></tr></table>

Mod – moderate; Note: Results are shown only for the intended targets. Panel members co-spiked with 5 different targets are presented 5 times.

The negative panel was negative $100 \%$ of the time. The moderate positive $( 5 \mathrm { x } \mathrm { L o D } )$ target gave expected results $100 \%$ of the time for adenovirus, Bordetella pertussis, influenza B, and SARS-CoV-2. The moderate positive for human metapneumovirus yielded $9 7 . 8 \%$ (88/90) positive results.

For the low positive $( 1 . 5 \mathrm { x } \mathrm { L o D } )$ target, all targets except for human metapneumovirus yielded at least $9 6 . 7 \%$ (87/90) positivity. The overall positivity for the human metapneumovirus low positive sample was $9 4 . 4 \%$ (85/90).

2. Linearity: Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

Analytical Reactivity (Inclusivity)   
The inclusivity of the LIAISON PLEX Respiratory Flex Assay was evaluated using a combination of in silico analysis of publicly available sequence information and laboratory testing of contrived specimens containing viral and bacterial isolates that were selected to represent phylogenetic, geographic, and temporal diversity.

# a. Wet-Testing

This study was performed to determine the analytical reactivity of the Respiratory Flex Assay with clinically relevant strains, serotypes, or subtypes of the target species. The inclusivity panel was prepared by spiking various target microorganism strains/serotypes/subtypes encompassing temporal and geographical diversity into simulated NPS matrix at a concentration of ${ \sim } 3 \mathbf { x }$ LoD and testing in triplicate. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. Strains that did not yield $100 \%$ reactivity at $3 \mathbf { x }$ LoD were reprepared at the same concentration and retested in triplicate. If $100 \%$ reactivity was obtained during retesting, reactivity was reached. In this situation, the original and retesting results were pooled for performance calculations. If less than $100 \%$ reactivity was observed during retesting, the strain was prepared at a higher concentration and tested until $100 \%$ reactivity was achieved. The strains evaluated and the lowest concentration that met the reactivity criteria outlined above are shown in Table 10 - Table 23, below.

Table 10. Inclusivity Testing – Adenovirus Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=6>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=5>xLoD</td></tr><tr><td rowspan=1 colspan=1>A 31</td><td rowspan=1 colspan=1>5.28x103</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 3</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 7A</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=1 colspan=3></td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 21</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td></td></tr><tr><td rowspan=1 colspan=1>B 11</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 14</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=12 colspan=5>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B 34</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=1 colspan=1>83.3% (5/6)1</td></tr><tr><td rowspan=1 colspan=1>B 35</td><td rowspan=1 colspan=1>2.06x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C 1</td><td rowspan=1 colspan=1>3.35x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C2</td><td rowspan=1 colspan=1>3.35x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C5</td><td rowspan=1 colspan=1>3.35x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>C 6</td><td rowspan=1 colspan=1>3.35x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>D 26</td><td rowspan=1 colspan=1>2.24x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>D 37</td><td rowspan=1 colspan=1>2.24x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>E 4</td><td rowspan=1 colspan=1>1.06x103</td><td rowspan=1 colspan=1>83.3% (5/6)1</td></tr><tr><td rowspan=1 colspan=1>F 40-Dugan</td><td rowspan=1 colspan=1>1.45x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>F 41-Tak</td><td rowspan=1 colspan=1>1.45x103</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for adenovirus. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for adenovirus.

Table 11. Inclusivity Testing – Bordetella holmesii Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>FA061</td><td rowspan=1 colspan=1>2.19x104</td><td rowspan=3 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CDC F5101 [CDC 84-013939]</td><td rowspan=1 colspan=1>2.19x104</td><td rowspan=1 colspan=1>83.3% (5/6)1</td></tr><tr><td rowspan=1 colspan=1>CIP 104395[G7702; 92A2997]</td><td rowspan=1 colspan=1>2.19x104</td><td rowspan=1 colspan=1>80.0% (4/5)2</td></tr><tr><td rowspan=1 colspan=1>CIP 104396</td><td rowspan=1 colspan=1>1.97x105</td><td rowspan=1 colspan=1>27x3</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for $B$ . holmesii. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for $B .$ . holmesii. 2 The original three replicates tested resulted in $33 \%$ (1/3) invalid replicates and $50 \%$ (1/2) positivity for $B .$ . holmesii. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for $B$ . holmesii.

3 Testing at lower concentrations (i.e., $3 \mathbf { x }$ and $9 \mathbf { x }$ LoD) failed to yield $100 \%$ detection. The lowest concentration that yielded $100 \%$ detection was $2 7 \mathbf { x }$ LoD.

Table 12. Inclusivity Testing – Bordetella parapertussis Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>NCTC 5952 [522]</td><td rowspan=8 colspan=1>2.37x103</td><td rowspan=5 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>508 and 344[NCTC10853]</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>517</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>12822</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>509 and 609</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PT28G</td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)1</td></tr><tr><td rowspan=1 colspan=1>PT 26/28G</td><td rowspan=1 colspan=1>100% (3/3)1</td></tr><tr><td rowspan=1 colspan=1>C510</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 Testing results in positivity for $B$ . parapertussis and $B$ . pertussis. CoAs from the vendor confirm that these are genetically modified strains, engineered to contain DNA sequences for both $B$ . parapertussis and $B$ . pertussis toxins.

Table 13. Inclusivity Testing – Bordetella pertussis Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=3>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=2>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>18323 [NCTC 10739]</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CNCTC Hp 12/63 [623]</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>10-536</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5 [17921]</td><td></td><td></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Tohama I</td><td rowspan=1 colspan=2>1.14x104</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>MN2531</td><td rowspan=4 colspan=2></td><td rowspan=4 colspan=1>−</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>PT9/28G [W28]</td><td rowspan=1 colspan=1>100%b (3/3)</td></tr><tr><td rowspan=1 colspan=1>589</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Table 14. Inclusivity Testing – Chlamydia pneumoniae Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>CWL-029</td><td rowspan=4 colspan=1>1.71x103</td><td rowspan=4 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>J-21</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>2023</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Table 15. Inclusivity Testing – Human Coronavirus Results   

<table><tr><td rowspan=2 colspan=2>Species/Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=2 colspan=1>HKU1</td><td rowspan=1 colspan=1>CS-Lum2020-Resp-1146</td><td rowspan=2 colspan=1>5.0x103</td><td rowspan=2 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CS-83254</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>Source #:0810228CF</td><td rowspan=1 colspan=1>2.29x102</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Source#:NR-470</td><td rowspan=1 colspan=1>6.88x102</td><td rowspan=1 colspan=1>9x1</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=2 colspan=1>OC43</td><td rowspan=1 colspan=1>Source#:0810024CF</td><td rowspan=2 colspan=1>2.84x104</td><td rowspan=2 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Source#:VR-1558</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=2 colspan=1>229E</td><td rowspan=1 colspan=1>Source#:0810229CF</td><td rowspan=1 colspan=1>1.20x103</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Source #:VR-740</td><td rowspan=1 colspan=1>3.60x103</td><td rowspan=1 colspan=1>9x1</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

CS-Clinical sample 1 Testing at a lower concentration (i.e., 3x LoD) failed to yield $100 \%$ detection. The lowest concentration that yielded $100 \%$ detection was $9 \mathbf { x }$ LoD.

Table 16. Inclusivity Testing – Enterovirus/Rhinovirus Results   

<table><tr><td rowspan=2 colspan=2>Species/Strain</td><td rowspan=1 colspan=5>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=4>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=2 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>Coxsackievirus A10</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus 71</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td></td><td></td><td rowspan=9 colspan=4>Enterovirus B</td><td rowspan=1 colspan=2>Coxsackievirus A9</td></tr><tr><td></td><td></td><td rowspan=3 colspan=5>Coxsackievirus B3</td><td rowspan=3 colspan=1></td></tr><tr><td></td><td></td></tr><tr><td></td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=7 colspan=4>6.75x104</td><td rowspan=2 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td></td><td rowspan=1 colspan=1>Echovirus 30</td><td rowspan=4 colspan=1>−</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>CoxsackievirusA21</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=7 colspan=1>Rhinovirus A</td><td rowspan=1 colspan=1>16</td><td rowspan=4 colspan=4>2.46x104</td><td rowspan=4 colspan=1>3x</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=4 colspan=4>2.46x104</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=6 colspan=1>Rhinovirus B</td><td rowspan=1 colspan=1>14</td><td rowspan=6 colspan=4>2.45x104</td><td rowspan=6 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=4 colspan=1>Rhinovirus C</td><td rowspan=1 colspan=1>CS-75029 (H7-3)</td><td rowspan=4 colspan=4>5.75x104</td><td rowspan=4 colspan=1>3x</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>CS-75466 (J3-3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>CS-NPS/UTU NEG 178</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>CS-NEG004 (SAR-4)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

CS-Clinical sample

Table 17. Inclusivity Testing –Human Metapneumovirus Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>hMPV-9 (Type A1)</td><td rowspan=2 colspan=1>6.40x103</td><td rowspan=2 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-16 (Type A1)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-20 (Type A2)</td><td rowspan=2 colspan=1>6.12x103</td><td rowspan=2 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-27 (Type A2)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-3 (TypeB1)</td><td rowspan=2 colspan=1>1.50x104</td><td rowspan=2 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-5 (TypeB1)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-4 (TypeB2)</td><td rowspan=3 colspan=1>4.50x104</td><td rowspan=3 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-8 (Type B2)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hMPV-18 (Type B2)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Table 18. Inclusivity Testing – Influenza A Results   

<table><tr><td rowspan=2 colspan=2>Species/Strain</td><td rowspan=1 colspan=3>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=2>xLoD</td></tr><tr><td rowspan=32 colspan=1>H1N1</td><td rowspan=2 colspan=1>A/Wisconsin/588/2019</td><td rowspan=6 colspan=1></td><td rowspan=6 colspan=2></td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Hawaii/66/2019 X-345A</td><td rowspan=1 colspan=1>Matrix: 100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 83.3% (5/6)1</td></tr><tr><td rowspan=2 colspan=1>A/Indiana/02/2020</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Michigan/272/2017</td><td rowspan=8 colspan=1>4.5x103</td><td rowspan=8 colspan=2>3x</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Idaho/07/2018</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Wisconsin/505/2018</td><td rowspan=1 colspan=1>Matrix: 100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 83.3% (5/6)1</td></tr><tr><td rowspan=2 colspan=1>Guangdong-Maonan /SWL1536/19</td><td rowspan=1 colspan=1>Matrix:: 100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 83.3% (5/6)1</td></tr><tr><td rowspan=2 colspan=1>Brisbane/02/18</td><td rowspan=2 colspan=1>1.35x104</td><td rowspan=2 colspan=2>9x 3</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/St.Petersburg/61/2015</td><td rowspan=16 colspan=1>4.5x103</td><td rowspan=19 colspan=2>3x</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Bangladesh/3002/2015</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>New Caledonia/20/99</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>PR/8/34</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Singapore/63/04</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Solomon Islands/03/06</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Taiwan/42/06</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>H1N1v</td><td rowspan=2 colspan=1>A/Ohio/09/2015(Subtype Synthetic DNA)</td><td rowspan=3 colspan=1>4.5x103</td><td rowspan=1 colspan=1>Matrix: 0% (0/3)2</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>A/Ohio/09/2015</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Species/Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(Matrix Synthetic DNA)</td><td rowspan=1 colspan=1></td><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>Subtype H1: 0% (0/3)2</td></tr><tr><td rowspan=2 colspan=1>H1N2</td><td rowspan=2 colspan=1>A/swine/Ohio/09SW1484E/2009</td><td rowspan=2 colspan=1>4.5x103</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=4 colspan=1>H1N2v</td><td rowspan=2 colspan=1>A/Minnesota/19/2011(Subtype Synthetic DNA)</td><td rowspan=4 colspan=1>4.5x103</td><td rowspan=1 colspan=1>Matrix: 0% (0/3)2</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Minnesota/19/2011(Matrix Synthetic DNA)</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H1: 0% (0/3)2</td></tr><tr><td rowspan=20 colspan=1>H3N2</td><td rowspan=2 colspan=1>A/Kansas/14/2017 NYMC X-327</td><td rowspan=20 colspan=1>5.89x103</td><td rowspan=36 colspan=1>3x</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Texas/71/2017</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Wisconsin/04/2018</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Arizona/45/2018</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Hong Kong/45/2019</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Tasmania/503/2020</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Delaware/01/2021</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Singapore/INFIMH-16-0019/2016</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>/California/55/2020</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Alaska/232/2015</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=4 colspan=1>H3N2v</td><td rowspan=2 colspan=1>A/Hawaii/28/2020(Subtype Synthetic DNA)</td><td rowspan=4 colspan=1>5.89x103</td><td rowspan=1 colspan=1>Matrix: 0% (0/3)2</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>A/Hawaii/28/2020(Matrix Synthetic DNA)</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype H3: 0% (0/3)2</td></tr><tr><td rowspan=6 colspan=1>H5N1</td><td rowspan=2 colspan=1>A/Egypt/N03072/2010</td><td rowspan=6 colspan=1>5.89x103</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)2</td></tr><tr><td rowspan=2 colspan=1>A/Hubei/1/2010</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)2</td></tr><tr><td rowspan=2 colspan=1>A/Anhui/01/2005</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)2</td></tr><tr><td rowspan=2 colspan=1>H7N2</td><td rowspan=2 colspan=1>A/turkey/Virginia/4529/2002</td><td rowspan=2 colspan=1>5.89x103</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)2</td></tr><tr><td rowspan=2 colspan=1>H7N7</td><td rowspan=2 colspan=1>A/mallard/Netherlands/12/2000</td><td rowspan=2 colspan=1>5.89x103</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)2</td></tr><tr><td rowspan=2 colspan=1>H9N2</td><td rowspan=2 colspan=1>A/Hong Kong/33982/2009</td><td rowspan=2 colspan=1>5.89x103</td><td rowspan=1 colspan=1>Matrix: 100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Subtype: 0% (0/3)2</td></tr></table>

1 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for the influenza A H1 subtype. New test material was prepared and tested, resulting in $100 \%$ $( 3 / 3 )$ positivity for the influenza A H1 subtype. $^ { 2 } \mathrm { N o }$ positivity was expected based on the strain and/or type of material being tested. 3 Testing at a lower concentration (i.e., $3 \mathbf { x }$ LoD) failed to yield $100 \%$ detection. The lowest concentration that yielded $100 \%$ detection was $9 \mathbf { x }$ LoD.

Table 19. Inclusivity Testing – Influenza B Results   

<table><tr><td rowspan=2 colspan=2>Lineage/Strain</td><td rowspan=1 colspan=7>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=4>Copies/mL</td><td rowspan=1 colspan=3>xLoD</td></tr><tr><td rowspan=8 colspan=1>VictoriaLineage</td><td rowspan=1 colspan=1>B/Washington/02/2019</td><td rowspan=2 colspan=4></td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/New Hampshire/01/2021</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Missouri/12/2018 (NAD197E)</td><td rowspan=3 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Hawaii/01/2018 (NAD197N)</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Michigan/01/2021</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/286/2017</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=2></td><td></td></tr><tr><td rowspan=1 colspan=1>B/Colorado/6/2017</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Texas/43/2019</td><td rowspan=8 colspan=4>1.01x103</td><td rowspan=8 colspan=3>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=7 colspan=1>YamagataLineage</td><td rowspan=1 colspan=1>B/Wisconsin/1/10</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/13</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/10/2016 (NAI221V)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Indiana/17/2017 (NAI221T)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B/Oklahoma/10/2018 (NAD197N</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Table 20. Inclusivity Testing – Mycoplasma pneumoniae Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=4>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=3>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>15531-TTR</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Mac</td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>PI 1428</td><td rowspan=3 colspan=2></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2></td><td></td><td></td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Bru</td><td rowspan=1 colspan=3>3.89x103</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>M52</td><td rowspan=4 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>UTMB-10P</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Mutant 22</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Table 21. Inclusivity Testing – Parainfluenza Results   

<table><tr><td colspan="2" rowspan="2">Species/Strain</td><td colspan="2" rowspan="1">Concentration</td><td colspan="1" rowspan="2">% Detected(# Detected/#Tested)</td></tr><tr><td colspan="1" rowspan="1">Copies/mL</td><td colspan="1" rowspan="1">xLoD</td></tr><tr><td colspan="1" rowspan="2">PIV-1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="2">2.28x103</td><td colspan="1" rowspan="10">3x</td><td colspan="1" rowspan="1">83.3% (5/6)1</td></tr><tr><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="2">PIV-2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="2">2.54x104</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="4">PIV-3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="4">5.79x103</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC-2011-5</td><td colspan="1" rowspan="1">83.3%(5/6)2</td></tr><tr><td colspan="1" rowspan="1">C243</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">NIH 47885</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="2">PIV-4a</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="2">1.73x104</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">M-25</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="2">PIV-4b</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CH 19503</td><td colspan="1" rowspan="1">100% (3/3)</td></tr></table>

1 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for parainfluenza 1. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for parainfluenza 1. 2 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for parainfluenza 3. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for parainfluenza 3.

Table 22. Inclusivity Testing – RSV Results   

<table><tr><td rowspan=2 colspan=2>Species/Strain</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=3 colspan=1>RSV-A</td><td rowspan=1 colspan=1>2006 Isolate</td><td rowspan=3 colspan=1>1.15x104</td><td rowspan=7 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=4 colspan=1>RSV-B</td><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=4 colspan=1>4.82x104</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>B WV/14617/85</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>B1</td><td rowspan=1 colspan=1>100%(3/3)</td></tr></table>

Table 23. Inclusivity Testing – SARS-CoV-2 Results   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=1 colspan=3>Concentration</td><td rowspan=2 colspan=1>% Detected(# Detected/#Tested)</td></tr><tr><td rowspan=1 colspan=2>Copies/mL</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=1 colspan=1>B.1.617.1: Isolate: USA/CA-Stanford-15 S02/2021 (Kappa)</td><td rowspan=1 colspan=2>7.20x104</td><td rowspan=1 colspan=1>9x1</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.1.529 BA.1: Isolate: USA/MD-HP20874/2021 (Omicron)</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Isolate: Italy-INMI1</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Isolate: Hong Kong/VM20001061/2020</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1_2020: Isolate: USA/NY-Wadsworth-103677-01/2020</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.1.7: Isolate: England/204820464/2020(Alpha)</td><td rowspan=8 colspan=2>2.4x104</td><td rowspan=8 colspan=1>3x</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.1.7: Isolate: USA/CA_CDC_5574/2020(Alpha)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.351: Isolate: South Africa/KRISP-K005325/2020 (Beta)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>P1: Isolate: Japan/TY7-503/2021 (Gamma)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>P2_2021: Isolate: NY-Wadsworth-21006055-01/2021 (Zeta)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.526_2021: Isolate: USA/NY-Wadsworth-21025952-01/2021 Isolate 1(Lota)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>B.1.617.2: Isolate: USA/PHC658/2021(Delta)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 Testing at a lower concentration (i.e., $3 \mathrm { x } \mathrm { L o D }$ ) failed to yield $100 \%$ detection. The lowest concentration that yielded $100 \%$ detection was $9 \mathbf { x }$ LoD.

The results from this study demonstrate that the Respiratory Flex Assay is capable of detecting multiple clinically relevant strains of each target analyte.

# In silico

The inclusivity of the LIAISON PLEX Respiratory Flex Assay was evaluated using in silico analysis of the oligonucleotides (i.e., forward primer(s), reverse primer(s), capture and mediator probe(s)) for all assay targets in relation to sequences available in GISAID (for SARS-CoV-2, influenza A, influenza A/H1, influenza A/H3, and influenza B) and sequences in NCBI GenBank for all other target organisms.

Sequence alignments were generated using a publicly available sequence alignment program. Sequences with full coverage of all four oligo-binding regions (forward primer, reverse primer, mediator probe and capture probe) were included in the analyses. Partial target sequences and sequences with ambiguous or degenerate bases in an oligo binding region were excluded. A match (i.e., predicted reactivity) was based on the following criteria for all organisms other than SARS-CoV-2: $2 9 0 \%$ homology between the oligo and reference sequences. If $- 9 0 \%$ homology was observed, a melting temperature (Tm) analysis was performed using a publicly available Tm calculator and applying assay specific conditions. The Tm analysis involved comparing the calculated mismatch Tm values against the assay’s PCR annealing temperature for the primers, against the RT temperature for the RT primers, and against the target hybridization temperature for the capture and mediator probes. If the Tm of the mismatch oligo:sequence pair was above the assay’s temperature for the pertinent RT-PCR step (i.e., annealing, RT, or target hybridization), then reactivity was predicted. The $2 9 0 \%$ homology threshold was chosen because the Respiratory Flex Assay’s oligos are designed to have a much higher Tm than the annealing/RT temperature for primers and the hybridization temperature for the capture and mediator probes. As such, the longest oligos in the Respiratory Flex Assay are more tolerant to mismatches.

For SARS-CoV-2, the criteria were stricter due to the propensity of this organism to acquire point mutations. Specifically, $100 \%$ homology was expected between the oligo and reference sequences for at least one gene oligo set. If $< 1 0 0 \%$ homology was observed for all gene oligo sets, the same melting temperature analysis and reactivity criteria described above were employed.

For SARS-CoV-2, 5,622,325 sequences in GISAID (as of July 31, 2023) were included in the analysis. These sequences included all variants of concern or variants of interest defined as of July 31, 2023. The LIAISON PLEX Respiratory Flex Assay targets three SARS-CoV-2 gene regions. The Respiratory Flex Assay result logic states that if at least 1 of these targets is detected, SARS-CoV-2 is positive. This same result logic was implemented for the in silico inclusivity assessment. Of the sequences included in this evaluation, $9 9 . 9 4 \%$ (5,619,069/5,622,325) have no mismatch in at least one gene oligo set and thus are predicted to be detected by the Respiratory Flex Assay. Of the $0 . 0 4 \%$ (3,256/5,622,325) of sequences with mismatches in at least one oligo binding region in all 3 SARS-CoV-2 target genes, a Tm analysis revealed that amplification/hybridization were expected to occur. Thus, it is expected that $100 \%$ of SARS-CoV-2 sequences evaluated in this study will be detected by the assay.

For influenza A, influenza A H1, influenza A H3, and influenza B, sequences uploaded to GISAID between September 1, 2015, and July 7, 2023, were included in the analysis. The following number of sequences were included in the evaluation of influenza A, A H1, A H3, and influenza B: 112,056, 54,364, 104,428, 26,470. Of the 26,470 influenza B sequences,

$6 6 . 1 \%$ (17,509/26,470) were Victoria lineage, $3 0 . 9 \%$ (8,167/26,470) were Yamagata lineage, and $3 . 0 \%$ (794/26,470) were of unknown/unclassified lineage. The influenza in silico inclusivity analysis results are shown in Table 24. Based on the reactivity criteria $( > 9 0 \%$ homology), $9 9 . 9 \%$ (112,034/112,056) of influenza A (matrix gene) sequences are expected to be detected, $9 8 . 9 \%$ (53,778/54,364) of influenza A H1 sequences are expected to be detected, $9 9 . 9 \%$ (104,315/104,428) of influenza A H3 sequences are expected to be detected, and $9 9 . 9 \%$ (26,433/26,470) of influenza B sequences are expected to be detected.

Table 24. Influenza In Silico Inclusivity Results   

<table><tr><td rowspan=1 colspan=1>ReportableTarget</td><td rowspan=1 colspan=1>Target Gene</td><td rowspan=1 colspan=1>of Sequences inAlignment</td><td rowspan=1 colspan=1># of Sequences withPercent Oligo Identify≥90%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Matrix protein (MP)</td><td rowspan=1 colspan=1>112,056</td><td rowspan=1 colspan=1>112,0341</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>HA</td><td rowspan=1 colspan=1>54,364</td><td rowspan=1 colspan=1>53,778</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>HA</td><td rowspan=1 colspan=1>104,428</td><td rowspan=1 colspan=1>104,315</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Non-structural protein (NS)</td><td rowspan=1 colspan=1>26,470</td><td rowspan=1 colspan=1>26,4332</td></tr></table>

1 Analysis included influenza A subtype H0, H1, H3, H5, H7, H9, and H10 strains. 2 Analysis included 17,509 Victoria lineage strains, 8,167 Yamagata lineage strains, and 794 strains of unknown lineage.

For all other target organisms, in silico inclusivity analyses was performed using sequences available from the GenBank nt database as of July 7, 2023. The majority of sequences evaluated for each target organism are expected to be detected by the Respiratory Flex Assay.

# Cross-Reactivity

# a. Cross-Reactivity Wet-Testing

# i. Off-Panel Cross-Reactivity

This study evaluated the analytical specificity (cross-reactivity) of the Respiratory Flex Assay in the presence of non-targeted microorganisms that may be found in a respiratory tract clinical specimen. Sixty (60) non-target microorganisms (Table 25) were evaluated in the study. Panel members were composed of one individual non-target microorganism spiked into simulated NPS matrix at $\ge \bar { 1 } 0 ^ { 5 } \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ (or equivalent) for viruses, ${ \underline { { \underline { { > } } } } } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ (or equivalent) for bacteria/fungi, or the highest concentration available. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. For Mycobacterium tuberculosis, genomic DNA was evaluated to minimize pathogenic exposure to the test operator. Additionally, DNA was evaluated for Bordetella parapertussis Bpp5, which was identified by in silico analysis to contain a sequence that was between $8 0 . 0 \% - 9 5 . 8 \%$ identical with the primers, mediators, or capture oligos of the Respiratory Flex Bordetella pertussis assay. Wet testing was performed with synthetic DNA consisting of the sequence for the matching region of the $B$ . parapertussis genome to determine if cross-reactivity with the Bordetella pertussis assay occurred.

To evaluate cross-reactivity, each panel was evaluated in triplicate in the absence of the target organisms. No cross-reactivity was observed at the concentrations tested except for Mycoplasma genitalium at a concentration of $4 \mathrm { x } 1 0 ^ { 6 } \mathrm { C C U / m L }$ , which cross-reacted with the Mycoplasma pneumoniae assay in $50 \%$ of replicates (3/6). The concentration of

Mycoplasma genitalium was lowered to $1 \mathrm { x } 1 0 ^ { 6 }$ and $4 \mathrm { x } 1 0 ^ { 5 } \mathrm { C C U / m L }$ and tested; crossreactivity no longer occurred at either of these lower concentrations.

Table 25. Off-Panel Organisms Evaluated for Cross-Reactivity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Conc./Unit</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>4x105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Aspergillus flavus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>4x105 CFU/mL</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella avium</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>1x105TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>MERS-CoV</td><td rowspan=1 colspan=1>NA{2</td></tr><tr><td rowspan=1 colspan=1>Bordetella hinzii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella petrii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella trematum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Mycobacteriumtuberculosis(H37Rv gDNA)</td><td rowspan=1 colspan=1>2.88 ng/uL</td></tr><tr><td rowspan=3 colspan=1>Bordetella parapertussisBpp5 (synthetic DNA)</td><td rowspan=3 colspan=1>1x106 copies/mL</td><td rowspan=3 colspan=1>Mycoplasma genitalium3</td><td rowspan=1 colspan=1>4x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>1x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>4x105 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Nasal Wash (pooled)</td><td rowspan=1 colspan=1>NA2</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatisSerovar D</td><td rowspan=1 colspan=1>1x106 IFU/mL</td><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus-SARS</td><td rowspan=1 colspan=1>NA2</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumdiphtheriae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumpseudodiphtheriticum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>1x105 copies/mL</td><td rowspan=1 colspan=1>Pneumocystis jiroveci</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fluoribacter bozemanae</td><td rowspan=1 colspan=1>4x106 CFU/mL</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacteriumnecrophorum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilusparainfluenzae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus Type</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella anisa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella feeleii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Ureaplasma urealyticum</td><td rowspan=1 colspan=1>1x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella longbeachae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>Varicella-Zoster Virus</td><td rowspan=1 colspan=1>2.34x104 TCID50/mL</td></tr></table>

$\mathrm { C F U } = \mathrm { C o l o n y }$ Forming Units; $\mathrm { C C U } =$ Colony Changing Units; $\mathrm { I F U } =$ Inclusion Forming Units; $\Gamma \mathrm { C I D } 5 0 =$ Median Tissue Culture Infectious Dose.

$^ 1 \mathrm { A }$ portion of the $B$ . parapertussis Bpp5 genome was identified by in-silico analysis as potentially cross-reactive with $B$ . pertussis. Synthetic DNA was tested that matched the region of high homology in the assay. Testing was included in the off-panel cross-reactivity study since the targeted sequence was not expected to be detected as $B$ . parapertussis by the assay.   
$^ { 2 } \mathrm { N o }$ concentration provided by the supplier.

$^ 3 \mathrm { A t } 4 \mathrm { x } 1 0 ^ { 6 } \mathrm { C C U / m L }$ , Mycoplasma genitalium was cross-reactive with the Mycoplasma pneumoniae assay in $5 0 . 0 \%$ (3/6) of replicates. The Mycoplasma genitalium concentration was lowered to $1 \mathrm { x } 1 0 ^ { 6 }$ $\mathrm { C C U / m L }$ and $4 \mathrm { x } 1 0 ^ { 5 } \mathrm { C C U / m L }$ and had $0 \%$ (0/3) positivity for both concentrations.

# ii. On-Panel Cross-Reactivity

Potential intra-panel cross-reactivity was evaluated with twenty-eight (28) on-panel microorganisms (Table 26). The on-panel organisms were evaluated by spiking each independently into simulated NPS matrix at $\ge 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (or equivalent) for viruses and ${ \dot { \geq } } 1 0 ^ { 6 } \mathrm { C F U / m L }$ (or equivalent) for bacteria, or the highest concentration available.

Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. To evaluate potential intra-panel cross-reactivity, each panel was evaluated in triplicate. The results of testing are shown in Table 26.

Table 26. On-Panel Organisms Evaluated for Cross-Reactivity   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc./Unit</td><td colspan="1" rowspan="1">ExpectedTargetPositivity</td><td colspan="1" rowspan="1">UnexpectedTargetPositivity</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1x106IFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus HKU1(Lum2020-Resp-1528)</td><td colspan="1" rowspan="1">6.62x104 copies/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus NL63</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Echovirus(Enterovirus/Rhinovirus)</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza B(Washington/02/2019/VictoriaLineage</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza B(Phuket/3073/13/Yamagata Lineage)</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1x106 CCU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1x105PFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subtype H1:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subtype H3:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H5N11</td><td colspan="1" rowspan="1">2x107 copies/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H7N21</td><td colspan="1" rowspan="1">6x106 copies/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H7N71</td><td colspan="1" rowspan="1">2x107 copies/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H9N21</td><td colspan="1" rowspan="1">4x108 copies/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N22</td><td colspan="1" rowspan="1">3x107 copies/mL</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subtype H1:100% (3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr></table>

CFU $=$ Colony Forming Units; $\mathrm { I F U } =$ Inclusion Forming Units; $\mathrm { T C I D } 5 0 = \mathrm { N }$ Median Tissue Culture Infectious Dose; PFU $=$ Plaque Forming Units 1 These influenza A non-subtype H1/H3 strains are expected to be inclusive to the influenza A matrix target, only (i.e., are expected to be reported as influenza A positive, subtype H1/subtype H3 negative). All were negative for both subtype H1 and subtype H3, as anticipated.

2 This influenza A H1N2 strain is expected to be inclusive to the influenza A matrix target and influenza A subtype H1 target. This strain was positive for both the influenza A matrix target and influenza A subtype H1 target, as expected.

# b. In silico

In silico analysis of assay specificity/exclusivity was performed by conducting a BLAST comparison of the assay’s oligos sequences to the GenBank nt sequence database, as of July 14, 2023. Sequences for 83 off-panel organisms (68 bacteria/fungi and 15 viruses) that can be found in a respiratory specimen were included. Additionally, sequences for all on-panel organisms were included to evaluate intra-panel cross-reactivity. A summary of the results from the analysis is provided in Table 27. The LIAISON PLEX Respiratory Flex assays were shown to be specific for their respective analytes with the following exceptions, which are noted in the device labeling:

• Cross-reaction of the Adenovirus assays with closely related Adenovirus G (serotype 52) strains.   
• Cross-reaction of the SARS-CoV-2 assays with closely related bat and pangolin coronavirus sequences;   
• Cross-reaction of the $B$ . parapertussis assay with strains of B. bronchiseptica that carry IS1001;   
• Cross-reaction of the influenza A H1 subtyping assay with 3 swine H3N2 strains and 1 avian H6N1 strain;

Cross-reaction of the influenza A H3 subtyping assay with 59 swine H1N1 and swine H1N2 strains, 1 duck H5N2 strain, 1 ostrich H7N1 strain, 1 avian H7N9 strain, 1 avian H8N4 strain, and 1 avian H11N9 strain.

Table 27. Organisms Predicted by In Silico Analyses to Cross-React with the Respiratory Flex Assay.   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Predicted Cross-Reaction</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenovirus G (serotype 52) - strains</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Bat coronavirus and Bat SARS-like coronavirus (accessionsMG772933, MG772934, and MN996532)</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Bordetella bronchiseptica containing IS1001 element(accessions JX013523 to JX013527 and CP022962)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>H5N1 (accession CY110922)1;swine H3N2 (accessions KM110061, KM110062,KM110063, and OM935891);avian H6N1 (accession OP888980)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>swine H1N1 and swine H1N2 - 59 strains;duck H5N2 (accession OK103962);ostrich H7N1 (accession AF202244);avian H7N9 (accession KP413675);avian H8N4 (accession OK103964);avian H11N9 (accession OK103956)</td></tr></table>

1 This H5N1 human strain sequence is a chimeric sequence containing H1N1 sequence fragments. Therefore, detection of this sequence by the H1 oligos is not considered a cross-reaction.

# Microbial Interference

To determine if non-target organisms can interfere with detection of on-panel organisms in the same sample, a microbial interference study was conducted. The study evaluated 16 non-target organisms spiked into simulated NPS matrix at $\ge 1 0 ^ { 5 } \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ for viruses and ${ \geq } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria/fungi, or the highest concentration available. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. In addition to each non-target organism, each sample was cospiked with five representative on-panel targets (i.e., $B .$ . pertussis, adenovirus, influenza B, hMPV, and SARS-CoV-2) at $3 \mathbf { x }$ LoD. To evaluate microbial interference, testing was performed in triplicate. If on-panel organisms were detected at $< 1 0 0 \%$ , samples were reprepared at the same concentration and retested in triplicate. If $100 \%$ detection of the on-panel targets was obtained during retesting, no additional testing was performed. When this situation was encountered, the original and retesting results were pooled for performance calculations. Results of the study are shown in Table 28. No microbial interference was observed except for Streptococcus pyogenes at $1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ and Legionella pneumophilia at $4 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , which interfered with detection of adenovirus in $1 6 . 7 \%$ (1/6) of replicates.

Table 28. Microbial Interference Study Results   

<table><tr><td colspan="1" rowspan="2">Off-Panel Organism</td><td colspan="1" rowspan="2">Conc./Unit</td><td colspan="5" rowspan="1">% Positivity (# Detected/# Tested)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">B.pertussis</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus 1</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">MERS</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">83.3%(5/6)1</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(6/6)</td></tr><tr><td colspan="1" rowspan="1">SARS</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">4x105 CFU/mL</td><td colspan="1" rowspan="1">83.3%(5/6)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(6/6)</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="2">Moraxella catarrhalis</td><td colspan="1" rowspan="2">1x106 CFU/mL</td><td colspan="1" rowspan="2">100%(3/3)</td><td colspan="1" rowspan="2">100%(3/3)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="2">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">(3/3)</td><td colspan="1" rowspan="1">(3/3)</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr></table>

1 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for adenovirus. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for adenovirus.

# Interfering Substances

An analytical study was performed to assess the potential inhibitory effects of exogenous and endogenous substances that may be commonly found in NPS specimens. A representative panel of five Respiratory Flex target organisms - Bordetella pertussis, adenovirus, influenza B, hMPV, and SARS-CoV-2 were co-spiked at 3x LoD in simulated NPS matrix and challenged with the 34 substances illustrated in Table 29. Testing was performed in triplicate. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document.

Table 29. Substances Evaluated for Interference   

<table><tr><td colspan="1" rowspan="1">Substance/Class</td><td colspan="1" rowspan="1">Description/Active Ingredient</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Beclomethasone dipropionate</td><td colspan="1" rowspan="1">25 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Anesthetic</td><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="1">10% w/v</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">3.4x10-2 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">30.6 µmol/L</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">25 μg/mL</td></tr><tr><td colspan="1" rowspan="1">FLONASE Sensimist AllergyRelief</td><td colspan="1" rowspan="1">Fluticasone furoate</td><td colspan="1" rowspan="1">2.84x10-3 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Fluticasone Propionate NasalSpray</td><td colspan="1" rowspan="1">Fluticasone propionate</td><td colspan="1" rowspan="1">2.84x10-3 μmol/L</td></tr><tr><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">20 ng/μL</td></tr><tr><td colspan="1" rowspan="1">Nasal Wash</td><td colspan="1" rowspan="1">Human Nasal Wash</td><td colspan="1" rowspan="1">9.1%</td></tr><tr><td colspan="1" rowspan="1">Sputum/Mucus</td><td colspan="1" rowspan="1">Human Sputum/Mucus</td><td colspan="1" rowspan="1">swab/1mL sample</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 swab/2mL sample2</td></tr><tr><td colspan="1" rowspan="3">Human Blood</td><td colspan="1" rowspan="3">Human Whole Blood</td><td colspan="1" rowspan="1">5.0% v/v</td></tr><tr><td colspan="1" rowspan="1">4.5% v/v</td></tr><tr><td colspan="1" rowspan="1">4.0% v/v</td></tr><tr><td colspan="1" rowspan="3">Human Cells</td><td colspan="1" rowspan="3">Leukocytes</td><td colspan="1" rowspan="1">1000 cells/μL</td></tr><tr><td colspan="1" rowspan="1">666.7 cells/μL</td></tr><tr><td colspan="1" rowspan="1">333.3 cells/μL</td></tr><tr><td colspan="1" rowspan="1">Oral Anesthetic and Analgesic</td><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Mometasone furoate</td><td colspan="1" rowspan="1">8.63x10-4 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin, bovine submaxillary Type I-S</td><td colspan="1" rowspan="1">100 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin, porcine stomach Type II</td><td colspan="1" rowspan="1">100 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin, porcine stomach Type III</td><td colspan="1" rowspan="1">100 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, Nasal Ointment</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">3.0 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Anti-viral</td><td colspan="1" rowspan="1">Oseltamivir Phosphate</td><td colspan="1" rowspan="1">1.28 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Afrin Nasal Spray</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Nasal Decongestant</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">1.79x10-1 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">Sodium Chloride</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Triamcinolone acetonide</td><td colspan="1" rowspan="1">25 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibiotic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">76.0 μmol/L</td></tr><tr><td colspan="1" rowspan="1">Anti-viral</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">100 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Anti-viral</td><td colspan="1" rowspan="1">Zinc</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="4">ZICAM Nasal Spray</td><td colspan="1" rowspan="1">Galphimia glauca</td><td colspan="1" rowspan="4">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Histaminum Hydrochloricum</td></tr><tr><td colspan="1" rowspan="1">Luffa operculata</td></tr><tr><td colspan="1" rowspan="1">Sulfur</td></tr><tr><td colspan="1" rowspan="1">NPS Swab</td><td colspan="1" rowspan="1">Nylon swab (Copan)</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Transport Media</td><td colspan="1" rowspan="1">Universal Transport Medium (Copan)</td><td colspan="1" rowspan="1">100%</td></tr></table>

1 A nylon nasopharyngeal swab was fully coated with human sputum/mucus and then eluted into $1 \mathrm { m L }$ of simulated NPS matrix, containing 5 representative target organisms at $3 \mathbf { x }$ LoD. The eluent was subsequently tested with the Respiratory Flex Assay.

$^ 2 \mathrm { A }$ nylon nasopharyngeal swab was fully coated with human sputum/mucus and then eluted into $2 \mathrm { m L }$ of simulated NPS matrix, containing 5 representative target organisms at $3 \mathbf { x }$ LoD. The eluent was subsequently tested with the Respiratory Flex Assay.

If an on-panel organism in the panel was detected at $< 1 0 0 \%$ , samples were reprepared at the same concentration (for both the on-panel organisms and the potential interfering substance) and retested in triplicate. If $100 \%$ detection of the on-panel targets was obtained during retesting, no additional testing was performed. When this situation was encountered, the original and retesting results were pooled for performance calculations. If less than $100 \%$ detection was observed during retesting, the sample was reprepared at a lower concentration of the interfering substance and tested until $100 \%$ detection was achieved.

No interference was observed except for the substances illustrated in Table 30 and described in more detail below:

Human sputum/mucus (prepared by coating a nylon NPS swab with human sputum/mucus, eluting the swab in $1 \mathrm { m L }$ of simulated NPS matrix containing 5 representative target organisms at $3 \mathbf { x }$ LoD, and then testing the eluate) interfered with detection of hMPV, influenza B, and SARS-CoV-2. For hMPV and influenza B, interference was observed during both initial testing and retesting, so per the study protocol, testing was performed at a less challenging concentration until interference was no longer observed (i.e., eluting the

swab into $2 \mathrm { m L }$ of simulated NPS matrix containing 5 representative target organisms at $3 \mathbf { x }$ LoD). Interference was no longer observed when human sputum/mucus was eluted into $2 \mathrm { m L }$ of simulated NPS matrix containing 5 representative analytes.

Human whole blood at $5 . 0 \% \mathrm { v } / \mathrm { v }$ interfered with detection of Bordetella pertussis, hMPV, and influenza B. For hMPV, interference was observed during both initial testing and retesting, so per the study protocol, testing was performed at a less challenging concentration until interference was no longer observed. Interference caused by human whole blood no longer occurred at a concentration of $4 . 0 \% \ \mathrm { v / v }$ . • Leukocytes at $1 0 0 0 \mathrm { c e l l s } / \mu \mathrm { L }$ interfered with detection of hMPV and SARS-CoV-2. The original three replicates tested resulted in $3 3 . 3 \%$ (1/3) replicates being invalid and $50 \%$ (1/2) positivity for hMPV and $100 \%$ positivity (2/2) for SARS-CoV-2. New test material was prepared and tested, resulting in $6 6 . 6 \%$ (2/3) invalid results and $0 \% ( 0 / 1 )$ positivity for hMPV and SARS-CoV-2. Per the study protocol, testing was performed at more dilute concentrations until $100 \%$ detection occurred. Interference caused by leukocytes was no longer observed at a concentration of $3 3 3 . 3 \mathrm { c e l l s } / \mu \mathrm { L }$ . Mupirocin at $3 . 0 \mu \mathrm { m o l / L }$ interfered with detection of influenza B in $3 3 \%$ (1/3) of replicates during initial testing. Upon repeat testing, no interference was observed. Tobramycin at ${ 7 6 . 0 \ \mu \mathrm { m o l / L } }$ interfered with detection of hMPV and SARS-CoV-2 in $50 \%$ (1/2) of replicates during initial testing. In addition, one replicate was invalid during original testing. Upon repeat testing, no interference was observed.

Table 30. Substances that Interfered with Detection of at Least One Target Organism   

<table><tr><td rowspan=2 colspan=1>ActiveIngredient</td><td rowspan=2 colspan=1>Test Conc.</td><td rowspan=1 colspan=5>% Positivity (# Detected/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>B.pertussis</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=2 colspan=1>HumanSputum/Mucus</td><td rowspan=1 colspan=1>1 swab/1mLsample</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>33.3%(2/6)1</td><td rowspan=1 colspan=1>66.7%(4/6)1</td><td rowspan=1 colspan=1>83.3%(5/6)</td></tr><tr><td rowspan=1 colspan=1>1 swab/2mLsample</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=3 colspan=1>Human WholeBlood</td><td rowspan=1 colspan=1>5.0% v/v</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>83.3%(5/6)</td><td rowspan=1 colspan=1>66.7%(4/6)1</td><td rowspan=1 colspan=1>83.3%(5/6)</td><td rowspan=1 colspan=1>100%(6/6)</td></tr><tr><td rowspan=1 colspan=1>4.5% v/v</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>66.7%(2/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>4.0% v/v</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=3 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>1000 cells/μL</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>33.3%(1/3)²2</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>66.7%(2/3)²2</td></tr><tr><td rowspan=1 colspan=1>666.7 cells/μL</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td></tr><tr><td rowspan=1 colspan=1>333.3 cells/μL</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>3.0 μmol/L</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>83.3%(5/6)3</td><td rowspan=1 colspan=1>100%(6/6)</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>76.0 μmol/L</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>80%(4/5)4</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>80%(4/5)4</td></tr></table>

1 Unexpected negative results were obtained during original and repeat testing, therefore testing was performed at more dilute concentrations until $100 \%$ detection occurred. 2 The original three replicates tested resulted in $3 3 . 3 \%$ (1/3) replicates being invalid and $50 \%$ (1/2) positivity for hMPV and $100 \%$ positivity (2/2) for SARS-CoV-2. New test material was prepared and tested, resulting in $6 6 . 6 \%$ (2/3) invalid results, and $0 \%$ $( 0 / 1 )$ positivity for hMPV and SARS-CoV-2. Therefore, testing was performed at more dilute concentrations until $100 \%$ detection occurred.

3 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for influenza B. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for influenza B.

4 The original three replicates tested resulted in $3 3 . 3 \%$ (1/3) replicates being invalid and $50 \%$ (1/2) positivity for hMPV and SARS-CoV-2. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for hMPV and SARS-CoV-2.

# Competitive Interference (Co-Infection)

The impact of competitive interference, caused by co-infections with on-target analytes, was evaluated for the Respiratory Flex Assay by testing contrived samples consisting of an on-target analyte at a low concentration (3x LoD) in the presence of a different target organism at a high concentration $\mathrm { ( \ge 1 x 1 0 ^ { 5 } T C I D 5 0 / m L }$ , $\mathrm { P F U / m L }$ , or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ ) in simulated NPS matrix. Data supporting the use of simulated matrix can be found in section VII Performance

Charactestics.B.2.Matrix Equivalency Study, later in this document. For high concentration Coronavirus HKU1 testing, the highest concentration available $( 1 . 3 1 \mathrm { x } 1 0 ^ { 4 }$ copies $/ \mathrm { m L }$ ) was used. Testing for each target organism (at low concentration) and each potential competitive strain (at high concentration) was performed in triplicate for the 54 co-infection combinations illustrated in Table 31. If competitive interference was observed during initial testing, samples were reprepared at the same concentration and retested in triplicate. If competitive interference was not observed during retesting, no additional testing was performed. When this situation was encountered, the original and retesting results were pooled for performance calculations. If competitive interference was observed during retesting, the high concentration analyte concentration was reduced, and additional testing was performed. Results of the study are shown in Table 31.

Table 31. Competitive Interference Study Sample Panel Composition & Study Results   

<table><tr><td colspan="2" rowspan="1">Target 1 (High Conc.)</td><td colspan="1" rowspan="1">Target 2(Low Conc.')</td><td colspan="2" rowspan="1">% Detected(# Detected/#Tested)</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc. (TCID50/mL²)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">66.7% (4/6)3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">5x104</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subtype H3:100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">100% (7/7)</td><td colspan="1" rowspan="1">85.7% (6/7)4</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">66.7% (4/6)5</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">5x104</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">PIV-4</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">PIV-1</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">Matrix:66.7% (4/6)6</td></tr><tr><td></td><td></td><td colspan="1" rowspan="1">EY</td><td></td><td></td></tr><tr><td colspan="2" rowspan="1">Target 1 (High Conc.)</td><td colspan="1" rowspan="1">Target 2Low Conc.)</td><td colspan="2" rowspan="1">% Detected(# Detected/#Tested)</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc. (TCID50/mL²)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Subtype H3:100%(6/6)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">5x104</td><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subtype H3:100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Flu A H1N1</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subtype H1:100 (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">Matrix:100% (3/3)Subype H3:100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">83.3% (5/6)7</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">CoronavirusHKU11</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 3B</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 4E</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 7A</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">1x105 (PFU/mL)</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">1x105 (CEID50/mL)</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1x105 (PFU/mL)</td><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">PIV-4</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100%% (3/3)</td></tr><tr><td colspan="1" rowspan="1">PIV-1</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">1x105 (CEID50/mL)</td><td colspan="1" rowspan="1">Adenovirus 37D</td><td colspan="1" rowspan="1">Matrix:100% (/3)Subtype H3:100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu A H1N1</td><td colspan="1" rowspan="1">1x105 (CEID50/mL)</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Matrix:10 (3/3)Subbtype H:100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu A H3N2</td><td colspan="1" rowspan="1">1x105 (CEID50/mL)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Matrix:100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="2" rowspan="1">Target 1 (High Conc.)</td><td colspan="1" rowspan="1">Target 2(Low Conc.')</td><td colspan="2" rowspan="1">% Detected(# Detected/#Tested)</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc. (TCID50/mL²)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Subtype H3:100% (3/3)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">83.3% (5/6)8</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">1.31x104 (copies/mL)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1x105 (PFU/mL)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 3B</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 4E</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7A</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr></table>

1 Low concentration target organisms were prepared at 3x LoD. 2 Concentrations are in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , unless otherwise noted.

3 Unexpected negative results were obtained during original and repeat testing for parainfluenza 3, therefore testing was performed at more dilute coronavirus OC43 concentrations until $100 \%$ detection of parainfluenza 3 occurred.

4 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for parainfluenza 3. New test material was prepared and tested, resulting in $100 \%$ (4/4) positivity for parainfluenza 3. Four replicates were performed during resting, rather than three because a single false positive SARSCoV-2 result was obtained during retesting.

5 Unexpected negative results were obtained during original and repeat testing for RSV A, therefore testing was performed at more dilute adenovirus concentrations until $100 \%$ detection of RSV A occurred.

6 Unexpected negative results were obtained during original and repeat testing for influenza A (matrix), therefore testing was performed at more dilute adenovirus concentrations until $100 \%$ detection of influenza A (matrix) occurred.

7 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for coronavirus. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for coronavirus.

8 The original three replicates tested resulted in $6 6 . 7 \%$ (2/3) positivity for SARS-CoV-2. New test material was prepared and tested, resulting in $100 \%$ (3/3) positivity for SARS-CoV-2.

Interference was observed for the following co-infections:

Parainfluenza 3 (low concentration) in the presence of human coronavirus OC43 (high concentration of $\mathrm { 1 x 1 0 ^ { 5 } \ T C I D _ { 5 0 } / m L }$ ). Competitive interference was no longer observed when the human coronavirus OC43 concentration was decreased to $5 \mathrm { x } 1 0 ^ { 4 } \mathrm { \bar { T } C I D } _ { 5 0 } / \mathrm { m L }$ .   
• Parainfluenza 3 (low concentration) in the presence of adenovirus 37D (high concentration of $\mathrm { i x } 1 0 ^ { 5 }$ $\mathrm { \Delta C I D _ { 5 0 } / m L }$ ).   
• RSV A (low concentration) in the presence of adenovirus 37D (high concentration of $\mathrm { 1 x 1 0 ^ { 5 } \ T C I D _ { 5 0 } / m L }$ ). Competitive interference was no longer observed when the adenovirus 37D concentration was decreased to $5 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .   
• Flu A H3N2 (low concentration) in the presence of adenovirus 37D (high concentration of $\mathrm { 1 x 1 0 ^ { 5 } T C I D _ { 5 0 } / m L } \mathrm { \Omega }$ ). Specifically, detection of influenza A (matrix) was decreased in the presence of adenovirus 37D at a high concentration of $\mathrm { 1 x 1 0 ^ { 5 } T C I D 5 0 / m L }$ . Competitive interference was no longer observed when the adenovirus 37D concentration was decreased to $5 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .   
• Human coronavirus 229E (low concentration) in the presence of SARS-CoV-2 (high concentration of $1 \mathrm { x } 1 0 ^ { 5 }$ T $\mathrm { \ C I D } _ { 5 0 } / \mathrm { m L }$ ).

SARS-CoV-2 (low concentration) in the presence of human coronavirus OC43 (high concentration of $\mathrm { 1 x 1 0 ^ { 5 } T C I D 5 0 / m L } \mathrm { \Omega }$ ).

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a. Controls See Section IV.C.Instrument Descriptive Information.5.Quality Control.

# b. Sample Stability

An analytical study was performed to establish the recommended transport and storage conditions for nasopharyngeal swab (NPS) specimens eluted in UVT /UTM that will be tested using the Respiratory Flex Assay. A representative panel of five Respiratory Flex target organisms (i.e., Bordetella pertussis, adenovirus, influenza B, hMPV, and SARS-CoV2) was evaluated. Positive samples were prepared by co-spiking the five organisms at concentrations of $2 \mathbf { x }$ and 5x LoD in natural clinical NPS matrix. Negative samples, consisting of natural clinical NPS matrix, were also included in the study. Initial testing was performed to establish the baseline time point $\scriptstyle { \mathrm { t } } = 0$ hours) for the study, and additional aliquots of the samples were stored at each of the following temperature conditions: (1) room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ , (2) refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ , and (3) Frozen $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ .

At the designated time points shown in Table 32 below, one positive sample at each concentration and a negative sample was tested with the Respiratory Flex Assay. The number of replicates performed for each storage condition and sample concentration is illustrated in Table 33.

Table 32. Overview of Specimen Stability Storage Conditions and Time Points   

<table><tr><td rowspan=2 colspan=1>Test Time Point</td><td rowspan=1 colspan=3>Storage Temperature</td></tr><tr><td rowspan=1 colspan=1>15-30°C</td><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>&lt;-70°C</td></tr><tr><td rowspan=1 colspan=1>Baseline (0-hours)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>5-hours</td><td rowspan=1 colspan=1>X</td><td rowspan=2 colspan=1>NA</td><td rowspan=5 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>9-hours</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>27-hours</td><td rowspan=4 colspan=1>NA</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>72-hours</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>80-hours</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>36-days</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>X</td></tr></table>

“X” indicates the testing time point for each target. “NA” indicates the time point/storage condition as not part of the testing protocol.

Table 33. Number of Replicates Collected for Each Storage Condition and Sample Concentration   

<table><tr><td colspan="6" rowspan="1">Refrigerated Storage</td></tr><tr><td colspan="1" rowspan="1">StorageTemperature</td><td colspan="1" rowspan="1">SampleConcentration</td><td colspan="1" rowspan="1">Baseline(0-hours)</td><td colspan="1" rowspan="1">27-Hours</td><td colspan="1" rowspan="1">72-Hours</td><td colspan="1" rowspan="1">80-Hours</td></tr><tr><td colspan="1" rowspan="3">2°C</td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">8°C</td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="6" rowspan="1">Room Temperature Storage</td></tr><tr><td colspan="1" rowspan="1">StorageTemperature</td><td colspan="1" rowspan="1">SampleConcentration</td><td colspan="1" rowspan="1">Baseline(0-hours)</td><td colspan="1" rowspan="1">5-Hours</td><td colspan="1" rowspan="1">9-Hours</td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="3">30°C</td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="4" rowspan="1">Frozen Storage</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">StorageTemperature</td><td colspan="1" rowspan="1">SampleConcentration</td><td colspan="1" rowspan="1">Baseline(0-hours)</td><td colspan="1" rowspan="1">36-Days</td><td colspan="2" rowspan="4"></td></tr><tr><td colspan="1" rowspan="3">≤-70°C</td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr></table>

The results of this specimen stability study support the stability claims for the Respiratory Flex Assay of clinical NPS specimens in UTM/UVT at the following conditions:

Up to 8-hours at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ • Up to 72-hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ Up to 30 days at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$

# Freeze/Thaw Stability

The performance of the Respiratory Flex Assay with fresh and frozen specimens was evaluated by testing 4 sample panels, each consisting of 5-6 target analytes co-spiked into natural clinical NPS matrix (see Table 34). For each of the 19 Respiratory Flex Assay reportable targets, at least one organism was included in the study. Each positive panel was prepared at both $2 \mathbf { x }$ and ${ 5 } \mathrm { x }$ LoD. In addition, a negative sample was included the study.

Table 34. Positive Sample Panels Evaluated in the Freeze/Thaw Study   

<table><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=5 colspan=1>A</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=5 colspan=1>C</td><td rowspan=1 colspan=1>Bordetella parapertussis</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV A</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Parainfluenza 2</td></tr><tr><td rowspan=1 colspan=1>hMPV (A2)</td><td rowspan=1 colspan=1>Influenza A (subtype H3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>hMPV (B2)</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=5 colspan=1>D</td><td rowspan=1 colspan=1>RSVB</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Parainfluenza 1</td></tr><tr><td rowspan=1 colspan=1>Influenza A (subtype H1)</td><td rowspan=1 colspan=1>Human coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Human coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus A</td><td rowspan=1 colspan=2></td></tr></table>

Positive panels spiked at $2 \mathbf { x }$ were tested in replicates of 40 at $\mathrm { T 0 }$ (fresh) and 20 replicates following 1, 2, and 3 freeze/thaw $\mathrm { ( F / T ) }$ cycles after storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ . Positive panels spiked at 5x LoD and the negative sample were tested in replicates of 10 at $\mathrm { T 0 }$ (fresh) and following 1, 2, and $3 \ \mathrm { F } / \mathrm { T }$ cycles after storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

The results of the study support that NPS specimens in UVT/UTM may undergo up to 2 F/T cycles following storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

c. In-Use Stability

A ‘loaded sample’ study was evaluated to define the duration that the Respiratory Flex Assay cartridges can remain loaded with sample prior to being run on a LIAISON PLEX instrument. The loaded sample stability study evaluated 40 cartridges from a single reagent lot using a multi-analyte co-spiked positive target pool, that contained five targets (adenovirus, $B .$ . pertussis, human metapneumovirus, influenza B, and SARS-CoV-2) each at a final concentration of 5x LoD (for that target) in simulated NPS matrix. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document. Twenty valid cartridges were tested immediately following sample addition to the cartridge. Twenty valid cartridges were tested after adding the sample into the cartridge and being allowed to sit at room temperature (between $1 5 \mathrm { { ^ \circ C } }$ - $3 0 ^ { \circ } \mathrm { C } )$ ) for a minimum of three hours. The Respiratory Flex Assay run time is approximately two hours so performance of a loaded cartridge after three hours will allow a user to delay testing a prepared and loaded cartridge for approximately the duration of a single Respiratory Flex Assay run. All cartridges loaded with sample and tested immediately and loaded with sample and tested after three hours at room temperature resulted in $100 \%$ positivity for all targets. Results demonstrated that RSP Flex Assay cartridges loaded with sample are stable for up to three hours.

# 6. Detection Limit:

A limit of detection study (LoD) was performed to evaluate the analytical sensitivity of the Respiratory Flex Assay. For this study, thirty-eight (38) strains and isolates (Table 35) that represent the 19 reportable targets of the Respiratory Flex Assay were tested individually by serially diluting each target in NPS matrix. Testing was broken into two parts: preliminary and confirmatory LoD testing. For the preliminary LoD study, a 3-fold dilution series consisting of at least 3 concentrations was tested in replicates of six per dilution. The preliminary LoD for each target was defined as the lowest concentration at which $100 \%$ of replicates were positive for the intended reportable target. The confirmed LoD was determined by testing a 3-fold dilution series of concentrations around the preliminary LoD in replicates of 20. The confirmed LoD for each organism was defined as the lowest concentration at which $\ge 9 5 \%$ of the replicates were positive for the intended reportable target. To confirm the LoD, at least one dilution below the LoD was required to result in less than $9 5 \%$ positivity. The confirmed LoDs for the Respiratory Flex Assay bacterial and viral analytes are illustrated in Table 36 and Table 37, respectively.

Table 35. Strains Included in the LoD Study   

<table><tr><td colspan="1" rowspan="1">Organism (Strain/Isolate)</td><td colspan="1" rowspan="1">Vendor</td><td colspan="1" rowspan="1">Catalog Number</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii (F061)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0801464</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis (C510)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0801643</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis (18323 NCTC 10739)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9797</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae (CM-1)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1360</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae (M129)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0801579</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1 (C)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810050CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 3 (B)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810062CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 31 (A)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810073CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 26 (D)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810117CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 40 (F)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810084CF</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 4 (E)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810070CF</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus HKU1</td><td colspan="2" rowspan="1">NA1</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus NL63</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810228CF</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810024CF</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810229CF</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (USA-WA1/2020)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810587CFHI</td></tr><tr><td colspan="1" rowspan="1">Echovirus Type 6</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810076CF</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (1A)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810012CFN</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (B14)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810284CF</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (C1)</td><td colspan="2" rowspan="1">NA1</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus 3 (Type B1)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810156CF</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus 8 (Type B2)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810159CF</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus 9 (Type A1)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810160CF</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus 27 (Type A2)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810164CF</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (A/Kansas/14/2017)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810586CF</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Hongkong/2671-19)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810609CF</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Singapore/INFIMH-160019/16)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810574CF</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (A/Brisbane/02/2018)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810585CF</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Guangdong-Maonan/SWL 1536/19)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810610CF</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Alabama/2/17, Victoria Lineage)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810572CF</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Washington/02/19, Victoria Lineage)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810611CF</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Colorado/06/2017, Victoria Lineage)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810573CF</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Wisconsin/1/10, Yamagata Lineage)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810241CF</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza (Type 1)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810014CF</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza (Type 2)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810015CF</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza (Type 3)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810016CF</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza (Type 4A)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810060CF</td></tr><tr><td colspan="1" rowspan="1">RSV A (2006 Isolate)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810040ACF</td></tr><tr><td colspan="1" rowspan="1">RSV B (3/2015 Isolate #1)</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810479CF</td></tr></table>

1 Clinical sample

Table 36. Confirmed LoD Results for Bacterial Targets   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=2>Concentration at LoD</td><td rowspan=2 colspan=1>Percent Positive(# Positive/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL1</td><td rowspan=1 colspan=1>CFU/mL{2</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>F061</td><td rowspan=1 colspan=1>7.29x103</td><td rowspan=1 colspan=1>86.8</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>18323 NCTC10739</td><td rowspan=1 colspan=1>3.80x103</td><td rowspan=1 colspan=1>1.98x103</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>C510</td><td rowspan=1 colspan=1>7.90x102</td><td rowspan=1 colspan=1>20.6</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>CM-1</td><td rowspan=1 colspan=1>5.68x102</td><td rowspan=1 colspan=1>1.04x102IFU/mL</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>1.30x103</td><td rowspan=1 colspan=1>42.4CCU/mL</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr></table>

1 Concentrations in copies $\mathrm { \Delta m L }$ were obtained by digital-droplet PCR. 2 Concentrations in CFU/mL unless otherwise noted.

Table 37. Confirmed LoD Results for Viral Targets   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=2>Concentration at LoD</td><td rowspan=2 colspan=1>Percent Positive(# Positive/# Tested)</td></tr><tr><td rowspan=1 colspan=1>Copies/mL1</td><td rowspan=1 colspan=1>TCID50/mL2</td></tr><tr><td rowspan=6 colspan=1>Adenovirus(inclusive to A, B,C, D, E, and Fserotypes)</td><td rowspan=1 colspan=1>31 (A)</td><td rowspan=1 colspan=1>1.76x103</td><td rowspan=1 colspan=1>1.09x10-2</td><td rowspan=1 colspan=1>100% (20/20</td></tr><tr><td rowspan=1 colspan=1>3 (B)</td><td rowspan=1 colspan=1>6.86x102</td><td rowspan=1 colspan=1>1.69x10-1</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>1 (C)</td><td rowspan=1 colspan=1>1.12x103</td><td rowspan=1 colspan=1>89.7</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>26 (D)</td><td rowspan=1 colspan=1>7.48x102</td><td rowspan=1 colspan=1>1.10x10-2</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>4 (E)</td><td rowspan=1 colspan=1>3.53x102</td><td rowspan=1 colspan=1>1.08x10-2</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>40 (F)</td><td rowspan=1 colspan=1>4.85x102</td><td rowspan=1 colspan=1>2.29x10-2</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Human Coronavirus(inclusive to HKU1,</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>4.00x102</td><td rowspan=1 colspan=1>9.15x10-2</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>1.67x103</td><td rowspan=1 colspan=1>NA3</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>76.4</td><td rowspan=1 colspan=1>1.34x10-2</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr></table>

<table><tr><td rowspan=1 colspan=1>229E, NL63, andOC43)</td><td rowspan=1 colspan=2>OC43</td><td rowspan=1 colspan=1>9.48x103</td><td rowspan=1 colspan=1>9.58x10-1</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=4 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=2>hMPV-9 (A1)</td><td rowspan=1 colspan=1>2.13x103</td><td rowspan=1 colspan=1>2.09x10-1</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=2>hMPV-27 (A2)</td><td rowspan=1 colspan=1>2.04x103</td><td rowspan=1 colspan=1>2.14x10-1</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=2>hMPV-3 (B1)</td><td rowspan=1 colspan=1>5.00x103</td><td rowspan=1 colspan=1>4.31x10-1</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=2>hMPV-8 (B2)</td><td rowspan=1 colspan=1>1.50x104</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=4 colspan=1>Influenza A /Subtype H1</td><td rowspan=2 colspan=1>Brisbane</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>1.35x104</td><td rowspan=1 colspan=1>3.97</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>1.50x103</td><td rowspan=1 colspan=1>4.41x10-1</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=2 colspan=1>Guangdong</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>1.37x104</td><td rowspan=1 colspan=1>5.86</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>1.37x104</td><td rowspan=1 colspan=1>5.86</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=6 colspan=1>Influenza A /Subtype H3</td><td rowspan=2 colspan=1>Hong Kong</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>1.59x105</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>5.30x104</td><td rowspan=1 colspan=1>4.98</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=2 colspan=1>Kansas</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>1.96x103</td><td rowspan=1 colspan=1>5.58</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>1.96x103</td><td rowspan=1 colspan=1>5.58</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=2 colspan=1>Singapore</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>4.55x103</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>4.55x103</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=4 colspan=1>Influenza B</td><td rowspan=1 colspan=2>Alabama/2/17(Victoria lineage)</td><td rowspan=1 colspan=1>3.35x102</td><td rowspan=1 colspan=1>7.30x10-1</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=2>Washington/02/19(Victoria lineage)</td><td rowspan=1 colspan=1>3.02x103</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=2>Colorado/6/17(Victoria lineage)</td><td rowspan=1 colspan=1>3.02x103</td><td rowspan=1 colspan=1>6.64x10-1</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=2>Wisconsin/1/10(Yamagata lineage)</td><td rowspan=1 colspan=1>1.01x103</td><td rowspan=1 colspan=1>3.23x10-1</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>7.61x102</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>8.46x103</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>1.93x103</td><td rowspan=1 colspan=1>3.18</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=2>A</td><td rowspan=1 colspan=1>5.76x103</td><td rowspan=1 colspan=1>66.5</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=2 colspan=1>RSV (inclusive toRSV A and RSV B)</td><td rowspan=1 colspan=2>A (2006 Isolate)</td><td rowspan=1 colspan=1>3.83x103</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=2>B (3/2015 Isolate #1)</td><td rowspan=1 colspan=1>1.61x104</td><td rowspan=1 colspan=1>7.48x10-1</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=4 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=2>Human Rhinovirus 1A</td><td rowspan=1 colspan=1>8.19x103</td><td rowspan=1 colspan=1>4.99x10-1</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=2>Human Rhinovirus B14</td><td rowspan=1 colspan=1>8.18x103</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=2>Human Rhinovirus C1</td><td rowspan=1 colspan=1>1.92x104</td><td rowspan=1 colspan=1>NA3</td><td rowspan=1 colspan=1>100.0% (20/20)</td></tr><tr><td rowspan=1 colspan=2>Human EnterovirusEchovirus Type 6</td><td rowspan=1 colspan=1>2.25x104</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=2>USA-WA1/2020</td><td rowspan=1 colspan=1>8.00x103</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>95.0% (19/20)</td></tr></table>

1 Concentrations in copies $\mathrm { \Delta m L }$ were obtained by digital-droplet PCR. 2 Concentrations in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ unless otherwise noted. 3 Clinical sample without a titer in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

The LoD for co-analyte spiked samples was also evaluated and shown to be equivalent to single analyte spiked samples.

LoD Testing with the WHO International Standard for SARS-CoV-2 (NIBSC, 20/146) An LoD study was performed to evaluate the analytical sensitivity of the Respiratory Flex Assay with the World Health Organization (WHO) Internal Standard for SARS-CoV-2. The WHO SARS-CoV-2 standard was reconstituted then serially diluted in NP matrix. As with the LoD study described above, testing was broken into two parts: preliminary and confirmatory LoD testing. For the preliminary LoD study, a 10-fold dilution series consisting of nine concentrations was tested in triplicate per dilution. The preliminary LoD was defined as the lowest concentration at which $100 \%$ of replicates were positive for SARS-CoV-2. The confirmed LoD was determined by testing a 3-fold dilution series of multiple concentrations around the preliminary LoD in replicates of 20. The confirmed LoD was defined as the lowest concentration at which $\ge 9 5 \%$ of the replicates were positive for the intended reportable target. To confirm the LoD, at least one dilution below the LoD was required to result in less than $9 5 \%$ positivity. The confirmed LoD for the Respiratory Flex Assay with the WHO International Standard was $\bar { 7 } . 7 \mathrm { x } 1 0 ^ { 5 } \mathrm { I U / m L }$ .

# 7. Assay Cut-Off:

The Respiratory Flex Assay determines the presence/absence of each target analyte (and control) by comparing the mean intensity of capture spots to pre-defined thresholds. In addition to the capture spot thresholds, a set of assay-dependent parameters are used to evaluate the validity of the run. To determine the threshold and parameter values, a sequential, multi-step approach was employed.

In the first step, a threshold training set consisting of 1905 samples from analytical testing was used to estimate the thresholds and assay-dependent parameters. Proposed threshold and assaydependent parameter values were determined by their ability to distinguish between true and false target calls while minimizing the no call rate. The proposed thresholds were determined by the negative signal distribution, where the target is absent from the samples.

In the second step of threshold determination, the performance of the proposed parameter and threshold values from step 1 were evaluated using the threshold test set, which consisted of 736 contrived/clinical samples. The clinical samples included in the Assay Cut-Off Study are independent from those used to demonstrate clinical performance of the Respiratory Flex Assay. The true call vs. no call rate for performance parameters and controls, and reportable target sensitivity and specificity were tabulated for presentation to a technical review team.

In the third step, a technical review team consisting of subject matter experts of the assay and chemistry determined the final threshold values through an assessment of the results for each performance parameter, control, and reportable target. As part of this analysis, an ROC curve was generated for each spot group with the final established thresholds marked to illustrate the expected clinical performance of the Respiratory Flex Assay. The specific assay parameters and thresholds are not reported in this document, as they are considered confidential and proprietary.

8. Accuracy (Instrument): Not applicable.

# 9. Carryover:

An analytical study was performed to assess potential carryover or cross-contamination in the Respiratory Flex Assay by testing high positive and negative samples in an alternating fashion on the LIAISON PLEX instrument. The high positive samples consisted of a representative panel of five assay targets (i.e., $B$ . pertussis, adenovirus, influenza B, hMPV, and SARS-CoV-2). Contrived positive samples were prepared by spiking $B .$ . pertussis at $1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ and the viral targets at $\mathrm { i \bar { x } 1 0 ^ { 5 } \ T C I D { 5 0 } / m L }$ into simulated NPS matrix. Data supporting the use of simulated matrix can be found in section VII Performance Charactestics.B.2.Matrix Equivalency Study, later in this document.

The study was performed by two operators, each using one of two LIAISON PLEX instruments, each containing six modules. Alternating high positive and negative samples were evaluated per the format summarized in Table 38. In total, 30 high positive and 30 negative samples were evaluated. All high positive samples yielded positive results for the five analytes in the sample, while all negative samples were negative. These results confirm that there is no evidence of carryover from samples tested with the Respiratory Flex Assay.

Table 38. Overview of Carryover/Cross-contamination Study Design   

<table><tr><td rowspan=2 colspan=1>Operator</td><td rowspan=2 colspan=1>LIAISONPLEXInstrument</td><td rowspan=2 colspan=1>TestRound</td><td rowspan=1 colspan=6>Module</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=5 colspan=1>1</td><td rowspan=5 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=5 colspan=1>2</td><td rowspan=5 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td></tr></table>

POS-Positive; NEG-Negative

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison:

# Matrix Equivalency Study

This study was designed to demonstrate the equivalence of simulated NPS matrix used in analytical studies, with clinically-derived natural NPS matrix. Simulated NPS matrix was formulated to resemble the content of a clinical NPS specimen collected in VTM, such that the simulated matrix would not artificially alter the performance of the test. The simulated NPS matrix consisted of $2 \mathbf { x } 1 0 ^ { 3 }$ HeLa cells/mL in UTM. The natural clinical NPS matrix consisted of NPS samples collected in UTM, which were negative for all Respiratory Flex Assay targets.

For this study, a representative panel of five assay targets (i.e., B. pertussis, adenovirus, influenza B, hMPV, and SARS-CoV-2) were co-spiked into pooled negative natural clinical NPS matrix and simulated NPS matrix at ${ < } 1 \mathbf { x }$ LoD, $2 \mathbf { x }$ LoD, and ${ 5 } \mathbf { x }$ LoD. Aliquots of negative matrices were also included in the evaluation. Positive samples spiked at ${ < } 1 \mathrm { x } \mathrm { L o D }$ and ${ 5 } \mathrm { x }$ LoD were tested in replicates of 20 per condition, while positive samples prepared at $2 \mathbf { x }$ LoD were tested in replicates of 60 per condition. Negative samples in natural clinical NPS matrix were tested in replicates of 22 per condition, while negative samples in simulated NPS matrix were tested in replicates of 20 per condition. The results of the study are shown in Table 39.

Table 39. Matrix Equivalency Study Results   

<table><tr><td rowspan="2">Target</td><td colspan="8">% Positivity (# Tested/# Positive)</td></tr><tr><td colspan="2">5x LoD</td><td colspan="2">2x LoD</td><td colspan="2">&lt;1x LoD</td><td colspan="2">Negative</td></tr><tr><td></td><td>Simulated 95.0%</td><td>Clinical 95.0%</td><td>Simulated 95.0%</td><td>Clinical 96.7%</td><td>Simulated 30.0%</td><td>Clinical 35.0%</td><td>Simulated 0.0%</td><td>Clinical 0.0%</td></tr><tr><td rowspan="2">Adenovirus</td><td>(19/20)</td><td>(19/20)</td><td>(57/60)</td><td>(58/60)</td><td>(6/20)</td><td>(7/20)</td><td>(0/20)</td><td>(0/22)</td></tr><tr><td>100%</td><td>100%</td><td>100%</td><td>98.3%</td><td>65.0%</td><td>70.0%</td><td>0.0%</td><td>0.0%</td></tr><tr><td rowspan="2">B. pertussis</td><td>(20/20)</td><td>(20/20)</td><td>(60/60)</td><td>(59/60)</td><td>(13/20)</td><td>(14/20)</td><td>(0/20)</td><td>(0/22)</td></tr><tr><td></td><td></td><td></td><td>98.3%</td><td></td><td>60.0%</td><td></td><td></td></tr><tr><td rowspan="2">Influenza B</td><td>100%</td><td>100%</td><td>100%</td><td></td><td>60.0%</td><td></td><td>0.0%</td><td>0.0%</td></tr><tr><td>(20/20)</td><td>(20/20)</td><td>(60/60)</td><td>(59/60)</td><td>(12/20)</td><td>(12/20)</td><td>(0/20)</td><td>(0/22)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(60/60)</td><td colspan="1" rowspan="1">100%(60/60)</td><td colspan="1" rowspan="1">55.0%(11/20)</td><td colspan="1" rowspan="1">30.0%(6/20)</td><td colspan="1" rowspan="1">0.0%(0/20)</td><td colspan="1" rowspan="1">0.0%(0/22)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(60/60)</td><td colspan="1" rowspan="1">98.3%(59/60)</td><td colspan="1" rowspan="1">70.0%(14/20)</td><td colspan="1" rowspan="1">80.0%(16/20)</td><td colspan="1" rowspan="1">0.0%(0/20)</td><td colspan="1" rowspan="1">0.0%(0/22)</td></tr></table>

All samples spiked at 5x LoD and $2 \mathbf { x }$ LoD met the acceptance criteria of $100 \%$ agreement and $9 5 \%$ agreement with expected results, respectively, except for adenovirus at ${ 5 } \mathrm { x }$ LoD, which was detected in $9 5 \%$ (19/20) of replicates in both simulated and natural clinical matrices. Since the results were adenovirus at ${ 5 } \mathrm { x }$ LoD were equivalent between matrices, they were deemed acceptable. As expected, detection varied for samples prepared at ${ < } 1 \mathbf { x }$ LoD. All negative samples were negative, as expected. The results from the study indicate that performance of the Respiratory Flex Assay is equivalent with a representative panel of analytes seeded into natural clinical NPS matrix in UTM and simulated NPS matrix in UTM and thus support use of simulated NPS matrix in select analytical validation studies.

# C Clinical Studies:

# Prospective Clinical Study

The clinical performance of the LIAISON PLEX Respiratory Flex Assay was established in a multi-center study conducted with nasopharyngeal swab (NPS) specimens in BD UVT and Copan UTM collected from individuals with signs and symptoms of respiratory infection. Specimens were prospectively collected (i.e., all comers between two time points that met the clinical study inclusion criteria) during the 2022-2023 respiratory illness season (i.e., October 2022 thru April 2023). Specimens were stored refrigerated at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 72-hours before testing (i.e., Category I specimens) or if they could not be tested within 72-hours, after freezing at - ${ } . 7 0 \mathrm { { } ^ { \circ } C }$ (Category II specimens). In total, six geographically distinct sites in the U.S. were involved in the prospective collection of NPS specimens. Of these six sites, three sites conducted LIAISON PLEX Respiratory Flex Assay testing. A fourth site (Luminex), which was not involved in the prospective collection, also performed Respiratory Flex Assay testing.

A total of 1911 NPS specimens were enrolled in the study. Of these 1911 specimens, 68 were excluded due to protocol deviations (e.g., specimen not stored appropriately, insufficient volume, etc.). This left 1843 clinical specimens for evaluation. Of these 1843 specimens, $6 6 . 3 \%$ (1221/1843) were tested fresh (Category I specimens), while $3 3 . 7 \%$ (622/1843) were tested frozen (Category II specimens). Patient demographic information for the 1843 prospectively collected NPS specimens is presented in Table 40.

Table 40. Demographic Data for Prospectively Collected Specimens   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">Site 5</td><td colspan="1" rowspan="1">Site 6</td></tr><tr><td colspan="1" rowspan="4">Sex</td><td colspan="1" rowspan="2">Male</td><td colspan="1" rowspan="2">839(45.5%)</td><td colspan="1" rowspan="2">271(52.9%)</td><td colspan="1" rowspan="2">43(37.7%)</td><td colspan="1" rowspan="1">143</td><td colspan="1" rowspan="1">354</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">(43.2%)</td><td colspan="1" rowspan="1">(44.5%)</td><td colspan="1" rowspan="1">(35.3%)</td><td colspan="1" rowspan="1">(25.0%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">1004(54.5%)</td><td colspan="1" rowspan="1">241(47.1%)</td><td colspan="1" rowspan="1">71(62.3%)</td><td colspan="1" rowspan="1">188(56.8%)</td><td colspan="1" rowspan="1">441(55.5%)</td><td colspan="1" rowspan="1">33(64.7%)</td><td colspan="1" rowspan="1">30(75.0%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1843(100%)</td><td colspan="1" rowspan="1">512(100%)</td><td colspan="1" rowspan="1">114(100%)</td><td colspan="1" rowspan="1">331(100%)</td><td colspan="1" rowspan="1">795(100%)</td><td colspan="1" rowspan="1">51(100%)</td><td colspan="1" rowspan="1">40(100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Age(Years)</td><td colspan="1" rowspan="1">0-1</td><td colspan="1" rowspan="1">350(19.0%)</td><td colspan="1" rowspan="1">205(40.0%)</td><td colspan="1" rowspan="1">4(3.5%)</td><td colspan="1" rowspan="1">62(18.7%)</td><td colspan="1" rowspan="1">75(9.4%)</td><td colspan="1" rowspan="1">4(7.8%)</td><td colspan="1" rowspan="1">0(0.0%)</td></tr><tr><td colspan="1" rowspan="1">&gt;1-5</td><td colspan="1" rowspan="1">274(14.9%)</td><td colspan="1" rowspan="1">146(28.5%)</td><td colspan="1" rowspan="1">7(6.1%)</td><td colspan="1" rowspan="1">49(14.8%)</td><td colspan="1" rowspan="1">61(7.7%)</td><td colspan="1" rowspan="1">11(21.6%)</td><td colspan="1" rowspan="1">0(0.0%)</td></tr><tr><td colspan="1" rowspan="1">&gt;5-21</td><td colspan="1" rowspan="1">447</td><td colspan="1" rowspan="1">158</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">165</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="4">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(24.3%)</td><td colspan="1" rowspan="1">(30.9%)</td><td colspan="1" rowspan="1">(31.6%)</td><td colspan="1" rowspan="1">(17.2%)</td><td colspan="1" rowspan="1">(20.8%)</td><td colspan="1" rowspan="1">(47.1%)</td><td colspan="1" rowspan="1">(17.5%)</td></tr><tr><td colspan="1" rowspan="1">&gt;21-65</td><td colspan="1" rowspan="1">535(29.0%)</td><td colspan="1" rowspan="1">3(0.6%)</td><td colspan="1" rowspan="1">63(55.3%)</td><td colspan="1" rowspan="1">126(38.1%)</td><td colspan="1" rowspan="1">303(38.1%)</td><td colspan="1" rowspan="1">12(23.5%)</td><td colspan="1" rowspan="1">28(70.0%)</td></tr><tr><td colspan="1" rowspan="1">&gt;65</td><td colspan="1" rowspan="1">237(12.9%)</td><td colspan="1" rowspan="1">0(0.0%)</td><td colspan="1" rowspan="1">4(3.5%)</td><td colspan="1" rowspan="1">37(11.2%)</td><td colspan="1" rowspan="1">191(24.0%)</td><td colspan="1" rowspan="1">0(0.0%)</td><td colspan="1" rowspan="1">5(12.5%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1843(100%)</td><td colspan="1" rowspan="1">512(100%)</td><td colspan="1" rowspan="1">114(100%)</td><td colspan="1" rowspan="1">331(100%)</td><td colspan="1" rowspan="1">795(100%)</td><td colspan="1" rowspan="1">51(100%)</td><td colspan="1" rowspan="1">40(100%)</td></tr></table>

The LIAISON PLEX Respiratory Flex Assay was evaluated for performance by comparing to an FDA-cleared molecular respiratory panel for all analytes, except the following: SARS-CoV-2, $B$ . holmesii, $B$ . parapertussis, and $B$ . pertussis. Performance for SARS-CoV-2 was evaluated by comparing to an FDA-cleared molecular SARS-CoV-2 assay. Performance for the denoted Bordetella species was based on comparison to well-validated Fragment Analysis (FA) assays followed by PCR/Bi-Directional Sequencing (PCR/BDS) assays (see Table 41). The FDAcleared comparator assays were performed in accordance with their respective package inserts.

Table 41. Comparator Methods for the LIAISON PLEX Respiratory Flex Assay Clinical Study   

<table><tr><td rowspan=1 colspan=1>LIAISON PLEX Respiratory Flex Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus (inclusive to A, B, C , D, E, and F)</td><td rowspan=15 colspan=1>FDA-Cleared Molecular RespiratoryPanel</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus (inclusive to HKU1, NL63, OC43,and 229E)</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H1</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H3</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td></tr><tr><td rowspan=1 colspan=1>RSV (inclusive to RSV A and RSV B)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>FDA-Cleared Molecular SARS-CoV-2 Assay, Cleared Under 21 CFR866.3981</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=3 colspan=1>Analytically Validated FragmentAnalysis Assays Followed byPCR/Bi-Directional Sequencing</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>sis Assay</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td></tr></table>

Out of the 1843 prospective clinical specimens included in the prospective study analysis, $9 5 . 2 \%$ (1755/1843) generated valid Respiratory Flex Assay results (i.e., detected or not detected) on the first attempt, for an initial invalid rate of $4 . 8 \%$ (88/1843). Of the 88 specimens with initial invalid results, 77 produced valid results on repeat testing, 6 specimens remained invalid on repeat, and 5 specimens were not retested, resulting in a final invalid rate of $0 . 6 \%$ (11/1843). This left 1832 specimens with valid Respiratory Flex Assay results. The final number of evaluable results varied by target based on the number of valid comparator method results obtained. The final number of evaluable results for each assay target is shown in Table 42.

Table 42. Total Evaluable Results for Prospective Specimens, Stratified by Analyte   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=4>ComparatorMethod</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>RespiratoryFlexInvalidResults1</td><td rowspan=1 colspan=1>RespiratoryFlex ValidANDComparatorResultsUnavailable2</td><td rowspan=1 colspan=1>TotalEvaluableResults</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=4>FDA-clearedmolecularrespiratorypanel</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=3 colspan=4>FA &amp;PCR/BDS</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>1730</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>1773</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>1753</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=3 colspan=1>Human Coronavirus</td><td rowspan=3 colspan=2>−</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=2></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=2 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=2 colspan=2>−</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>1843</td><td rowspan=2 colspan=1>11</td><td rowspan=2 colspan=1>12</td><td rowspan=2 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>hMPV</td><td rowspan=3 colspan=2>−</td><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=2></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=2 colspan=2>−</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>1843</td><td rowspan=2 colspan=1>11</td><td rowspan=2 colspan=1>12</td><td rowspan=2 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H1</td><td rowspan=1 colspan=4>FDA-cleared</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H3</td><td rowspan=1 colspan=4>molecular</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=7 colspan=4>respiratorypanel</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1820</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=4>FDA-clearedmolecularSARS-CoV-2 assay</td><td rowspan=1 colspan=1>1843</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>1780</td></tr></table>

1 Eleven specimens were invalid after repeat testing with the Respiratory Flex Assay. Of these 11 specimens, 6 specimens remained invalid on repeat, while 5 specimens were not retested with the Respiratory Flex Assay. 2 Unavailable indicates a specimen that was invalid by the comparator method or not tested with the comparator method.

A summary of the LIAISON PLEX Respiratory Flex Assay prospective clinical study performance is provided in Table 43. Positive Percent Agreement (PPA) was calculated as $100 \%$ $\times \left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the Respiratory Flex Assay and the comparator method had a positive result for the specific analyte, and false negative (FN) indicates that the Respiratory Flex Assay was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathrm { T N / \left( T N + F P \right) } \right)$ . True negative (TN) indicates that both the Respiratory Flex Assay and the comparator method had negative results, and false positive (FP) indicates that the Respiratory Flex Assay was positive while the comparator result was negative. Specimens that obtained discordant results underwent additional testing with an FDA-cleared molecular respiratory panel or PCR/BDS for investigation.

Table 43. Prospective Clinical Performance of the LIAISON PLEX Respiratory Flex Assay with NPS Specimens   

<table><tr><td colspan="2" rowspan="1">Ii S Specicls</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="2" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/(TN+FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">75/75</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.1-100</td><td colspan="1" rowspan="1">107/1129</td><td colspan="1" rowspan="1">95.1</td><td colspan="1" rowspan="1">93.7-96.2</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">83.2-100</td><td colspan="1" rowspan="1">578/597</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">95.1-98.0</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">94/94</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.1-100</td><td colspan="1" rowspan="1">1652/17261</td><td colspan="1" rowspan="1">95.7</td><td colspan="1" rowspan="1">94.7-96.6</td></tr><tr><td colspan="1" rowspan="3">Bordetellaholmesi</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1127/1127</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">603/603</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1730/1730</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">Bordetellaparapertussis</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">1161/1163</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0-79.3</td><td colspan="1" rowspan="1">604/605</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">37.6-96.4</td><td colspan="1" rowspan="1">1765/17682</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.5-99.9</td></tr><tr><td colspan="1" rowspan="3">Bordetellapertusis</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1146/1146</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">607/607</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1753/1753</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1204/1204</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">616/616</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1820/1820</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">HumanCoronavirus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">90/97</td><td colspan="1" rowspan="1">92.8</td><td colspan="1" rowspan="1">85.8-96.5</td><td colspan="1" rowspan="1">1100/1107</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.7-99.7</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">27/33</td><td colspan="1" rowspan="1">81.8</td><td colspan="1" rowspan="1">65.6-91.4</td><td colspan="1" rowspan="1">582/583</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.0-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">117/1303</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">83.6-94.1</td><td colspan="1" rowspan="1">1682/16904</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">99.1-99.8</td></tr><tr><td colspan="1" rowspan="3">Enterovirus/Rhiovirus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">230/242</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">91.5-97.1</td><td colspan="1" rowspan="1">937/962</td><td colspan="1" rowspan="1">97.4</td><td colspan="1" rowspan="1">96.2-98.2</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">81/90</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">82.1-94.7</td><td colspan="1" rowspan="1">518/526</td><td colspan="1" rowspan="1">98.5</td><td colspan="1" rowspan="1">97.0-99.2</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">311/3325</td><td colspan="1" rowspan="1">93.7</td><td colspan="1" rowspan="1">90.5-95.8</td><td colspan="1" rowspan="1">1455/14886</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">96.9-98.4</td></tr><tr><td colspan="1" rowspan="3">hMPV</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">113/118</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">90.5-98.2</td><td colspan="1" rowspan="1">1080/1086</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.8-99.7</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">66.7-98.6</td><td colspan="1" rowspan="1">603/603</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">125/1317</td><td colspan="1" rowspan="1">95.4</td><td colspan="1" rowspan="1">90.4-97.9</td><td colspan="1" rowspan="1">1683/16898</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">99.2-99.8</td></tr><tr><td colspan="1" rowspan="3">Influenza A</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">82.4-100</td><td colspan="1" rowspan="1">1185/1186</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">111/111</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.7-100</td><td colspan="1" rowspan="1">490/505</td><td colspan="1" rowspan="1">97.0</td><td colspan="1" rowspan="1">95.2-98.2</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">129/129</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.1-100</td><td colspan="1" rowspan="1">1675/16919</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.5-99.4</td></tr><tr><td colspan="1" rowspan="3">Influenza ASubtype H1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td><td colspan="1" rowspan="1">1187/1188</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100</td><td colspan="1" rowspan="1">595/595</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100</td><td colspan="1" rowspan="1">1782/178310</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">Influenza ASubtype H3</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">20.8-93.9</td><td colspan="1" rowspan="1">1200/1201</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">102/104</td><td colspan="1" rowspan="1">98.1</td><td colspan="1" rowspan="1">93.3-99.5</td><td colspan="1" rowspan="1">509/512</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.3-99.8</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">104/10711</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">92.1-99.0</td><td colspan="1" rowspan="1">1709/171312</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-99.9</td></tr><tr><td colspan="1" rowspan="3">Influenza B</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">1200/1200</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">612/612</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100</td><td colspan="1" rowspan="1">1812/1812</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1204/1204</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">616/616</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">1820/1820</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">52.9-97.8</td><td colspan="1" rowspan="1">1196/1196</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">612/612</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">11/1213</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">64.6-98.5</td><td colspan="1" rowspan="1">1808/1808</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 2</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">59.6-98.2</td><td colspan="1" rowspan="1">1194/1194</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">613/613</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">66.7-98.6</td><td colspan="1" rowspan="1">1807/1807</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">37/39</td><td colspan="1" rowspan="1">94.9</td><td colspan="1" rowspan="1">83.1-98.6</td><td colspan="1" rowspan="1">1164/1165</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100</td></tr><tr><td colspan="2" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/(TN+FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">37.6-96.4</td><td colspan="1" rowspan="1">611/611</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">41/4414</td><td colspan="1" rowspan="1">93.2</td><td colspan="1" rowspan="1">81.8-97.7</td><td colspan="1" rowspan="1">1775/177615</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 4</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">1199/1200</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">37.6-96.4</td><td colspan="1" rowspan="1">611/611</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">8/916</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">56.5-98.0</td><td colspan="1" rowspan="1">1810/181117</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">RespiratorySyncytial Virus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">37/38</td><td colspan="1" rowspan="1">97.4</td><td colspan="1" rowspan="1">86.5-99.5</td><td colspan="1" rowspan="1">1166/1166</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">81/85</td><td colspan="1" rowspan="1">95.3</td><td colspan="1" rowspan="1">88.5-98.2</td><td colspan="1" rowspan="1">531/531</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">118/12318</td><td colspan="1" rowspan="1">95.9</td><td colspan="1" rowspan="1">90.8-98.3</td><td colspan="1" rowspan="1">1697/1697</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">SARS-CoV-2</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">178/183</td><td colspan="1" rowspan="1">97.3</td><td colspan="1" rowspan="1">93.8-98.8</td><td colspan="1" rowspan="1">996/1000</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">99.0-99.8</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">68/72</td><td colspan="1" rowspan="1">94.4</td><td colspan="1" rowspan="1">86.6-97.8</td><td colspan="1" rowspan="1">521/525</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.1-99.7</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">246/25519</td><td colspan="1" rowspan="1">96.5</td><td colspan="1" rowspan="1">93.4-98.1</td><td colspan="1" rowspan="1">1517/152520</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">99.0-99.7</td></tr></table>

$^ { 3 } \mathrm { O f }$ the 13 specimens with false negative coronavirus results by the Respiratory Flex Assay, 3 were negative by PCR/BDS, 9 were positive, and 1 was not tested.

4 Of the 8 specimens with false positive coronavirus results by the Respiratory Flex Assay, 5 were positive by PCR/BDS, 2 were negative, and 1 was not tested.

5 Of the 21 specimens with false negative enterovirus/rhinovirus results by the Respiratory Flex Assay,   
9 were positive by PCR/BDS, 8 were negative, and 4 were not tested.   
6 Of the 33 specimens with false positive enterovirus/rhinovirus results by the Respiratory Flex Assay,   
4 were positive by PCR/BDS, 27 were negative, and 2 were not tested.

7 Of the 6 specimens with false negative hPMV results by the Respiratory Flex Assay, 4 were positive by PCR/BDS and 2 were negative.

$^ { 8 } \mathrm { O f }$ the 6 specimens with false positive hPMV results by the Respiratory Flex Assay, 4 were positive by PCR/BDS and 2 were negative.

9 Of the 16 specimens with false positive influenza A results by the Respiratory Flex Assay, 7 were positive by PCR/BDS and 9 were negative.

10The 1 specimen with a false positive influenza A subtype H1 result by the Respiratory Flex Assay was negative by PCR/BDS.

11The 3 specimens with false negative influenza A subtype H3 results by the Respiratory Flex Assay were all negative by PCR/BDS.

12The 4 specimens with false positive influenza A subtype H3 results by the Respiratory Flex Assay were all negative by PCR/BDS.

13The 1 specimen with a false negative parainfluenza 1 result by the Respiratory Flex Assay was positive by PCR/BDS.

$^ { 1 4 } \mathrm { O f }$ the 3 specimens with false negative parainfluenza 3 results by the Respiratory Flex Assay, 2 were negative by PCR/BDS and 1 was not tested.

15The 1 specimen with a false positive parainfluenza 3 result by the Respiratory Flex Assay was negative by PCR/BDS.

16The 1 specimen with a false negative parainfluenza 4 result by the Respiratory Flex Assay was negative by PCR/BDS.

17The 1 specimen with a false positive parainfluenza 4 result by the Respiratory Flex Assay was negative by PCR/BDS.

${ } ^ { 1 8 } \mathrm { O f }$ the 5 specimens with false negative RSV results by the Respiratory Flex Assay, 1 was negative by PCR/BDS, and 3 were negative by an FDA-cleared molecular Flu/RSV assay.

$^ { 1 9 } \mathrm { O f }$ the 9 specimens with false negative SARS-CoV-2 results by the Respiratory Flex Assay, 5 were positive by PCR/BDS, 2 were negative, and 2 were not tested.

$^ { 2 0 } \mathrm { O f }$ the 8 specimens with false positive SARS-CoV-2 results by the Respiratory Flex Assay, 5 were positive by PCR/BDS, 2 were negative, and 1 was not tested.

Positive Percent Agreement (PPA) ranged from $8 0 . 0 \% 1 0 0 \%$ , depending on the analyte. The only analytes for which PPA was lower than expected were $B$ . parapertussis with a PPA of $8 0 . 0 \%$ (4/5) and parainfluenza 4 with a PPA of $8 8 . 9 \%$ (8/9). The limited number of positive NPS specimens for these pathogens required supplementation with archived (Category III specimens) to obtain accurate PPA point estimates.

Negative Percent Agreement (NPA) ranged from $9 5 . 7 \mathrm { - } 1 0 0 \%$

The Respiratory Flex Assay reported multiple organism detections (coinfections) in a total of 176 prospectively collected NPS specimens. Of these 176 specimens, 3 specimens lacked comparator results for least one organism identified in the coinfection, and thus were excluded from further analysis. The remaining 173 specimens with coinfections represent $1 4 . 6 \%$ (173/1187) of all positive prospective specimens and $9 . 4 \%$ (173/1840) of all prospective specimens. Most coinfections contained two organisms $( 8 7 . 3 \%$ , 151/173) while $1 1 . 6 \%$ (20/173) contained three organisms and $1 . 2 \%$ (2/173) contained four organisms. Out of the 173 specimens with coinfections, $4 4 . 5 \%$ (77/173) contained one or more organisms that were not detected by the comparator methods (Table 44). Coinfections identified by the comparator methods which were not reported by the LIAISON PLEX Respiratory Flex Assay are illustrated in Table 45.

Table 44. Coinfections Reported by the LIAISON PLEX Respiratory Flex Assay in the Prospective Study   

<table><tr><td colspan="1" rowspan="2">Analyte 1</td><td colspan="1" rowspan="2">Analyte 2</td><td colspan="1" rowspan="2">Analyte 3</td><td colspan="1" rowspan="2">Analyte 4</td><td colspan="2" rowspan="1">Number ofSpecimens</td><td colspan="1" rowspan="2">Respiratory FlexAssay FalsePositive Analyte(s)1</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">FalsePositive</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (2),Bordetellaparapertussis (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Adenovirus (5)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Adenovirus (17),Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">PIV-1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Adenovirus (4)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">PIV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV-1</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Adenovirus (4)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Adenovirus (3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Bordetellaparapertussis (1)</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Bordetellaparapertussis (1)</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Coronavirus (1),Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Coronavirus (2)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Coronavirus (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">Influenza A(subtype H3)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (1),Influenza A(subtype H3) (1),SARS-CoV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Influenza A (2),Influenza A(subtype H3) (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">PIV-1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">PIV-4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV-4 (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (2)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1">Influenza A(subtype H3)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SARS-CoV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Influenza A (1)</td></tr><tr><td colspan="1" rowspan="1">PIV-2</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SARS-CoV-2 (1)</td></tr><tr><td colspan="4" rowspan="1">Total</td><td colspan="1" rowspan="1">173</td><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="4" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">62</td></tr><tr><td colspan="4" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="4" rowspan="1">Total Quadruple Infections</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr></table>

1 Based on comparator test results

Thirty-four (34) specimens were positive for two or more analytes (i.e., coinfected) by the comparator methods but were reported as negative for one of the analytes using the LIAISON PLEX Respiratory Flex Assay (Table 45).

Table 45. Coinfections Identified by the Comparator Methods which were Not Reported by the LIAISON PLEX Respiratory Flex Assay in the Prospective Study   

<table><tr><td colspan="1" rowspan="2">Analyte 1</td><td colspan="1" rowspan="2">Analyte 2</td><td colspan="1" rowspan="2">Analyte 3</td><td colspan="2" rowspan="1">Number of Specimens</td><td colspan="1" rowspan="2">Respiratory Flex AssayFalse NegativeAnalyte(s)1</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">FalseNegative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Coronavirus (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">PIV-2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SARS-CoV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SARS-CoV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Coronavirus (1)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Enterovirus/</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Coronavirus (1)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Coronavirus (2)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Coronavirus (1)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Coronavirus (1),PIV-3 (1)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Coronavirus (1)</td></tr><tr><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1">PIV-4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV-4 (1)</td></tr><tr><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV (1)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">BordetellaParapertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Bordetellaparapertussis (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (3)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A(subtype H3)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Influenza A (subtype H3)(1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">Influenza A &amp;Influenza A(subtype H3)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">SARS-CoV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (3)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus (2),SARS-CoV-2 (1)</td></tr><tr><td colspan="1" rowspan="1">PIV-3</td><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV-3 (1), Enterovirus/Rhinovirus (1)</td></tr><tr><td colspan="3" rowspan="1">Total</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="3" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">29</td></tr><tr><td colspan="3" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td></tr></table>

1 Based on comparator test results

# Retrospective Clinical Study

Several analytes included on the Respiratory Flex Assay were not encountered during the Prospective Clinical Study in sufficient numbers to demonstrate assay performance. Therefore, the Prospective Clinical Study was supported by additional testing that was performed at three U.S. study sites (including one internal) on frozen, archived specimens (Category III specimens) obtained from four clinical laboratories in the U.S. These specimens were selected for inclusion in the study based solely on the historic qualitative result.

A total of 256 archived NPS specimens were evaluated, of which 66 negatives were included for blinding and randomization purposes. A summary of the demographic information of the tested retrospective specimens is provided in Table 46 below.

Table 46. Demographic Data for Retrospective Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=4 colspan=1>Sex</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>117 (45.7%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>124 (48.4%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>15 (5.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>256 (100%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=7 colspan=1>Age</td><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>44 (17.2%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>53 (20.7%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>69 (27.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>44 (17.2%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>32 (12.5%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>14 (5.5%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>256 (100%)</td></tr></table>

The performance of the LIAISON PLEX Respiratory Flex Assay was determined by comparing to an FDA-cleared molecular respiratory panel for all analytes, except the following: B. holmesii, $B .$ . parapertussis, and $B$ . pertussis. As noted previously in this Decision Summary, performance for the denoted Bordetella species was based on comparison to analytically validated Fragment Analysis (FA) assays followed by PCR/Bi-Directional Sequencing (PCR/BDS) assays.

Out of the 256 specimens included in the pre-selected study analysis, 241 $( 9 4 . 1 \% )$ generated valid Respiratory Flex Assay results (i.e., Detected or Not Detected) on the first attempt. There were 15 specimens $( 5 . 9 \% )$ with invalid results on the initial run that required retesting. Of the specimens with initial invalid results; all 15 specimens generated valid Respiratory Flex Assay results after retest for a final success rate of $100 \%$ (256/256).

For the 256 pre-selected specimens, the final number of evaluable results varied by target based on valid comparator results obtained. The final number of evaluable results for each target is shown in Table 47.

Table 47. Total Evaluable Results for Retrospective Specimens, Stratified by Analyte   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">ComparatorMethod</td><td colspan="1" rowspan="1">TotalSpecimens</td><td colspan="1" rowspan="1">RespiratoryFlexInvalidResults1</td><td colspan="1" rowspan="1">RespiratoryFlex ValidANDComparatorResultsUnavailable1</td><td colspan="1" rowspan="1">TotalEvaluableResults</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">FDA-clearedmolecularrespiratorypanel</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="3">FA&amp;PCR/BDS</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">234</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">244</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">240</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="6">FDA-clearedmolecularrespiratorypanel</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Influenza A subtype H1</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Influenza A subtype H3</td><td colspan="1" rowspan="8">b</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">256</td></tr></table>

1 Unavailable indicates a specimen that was invalid by the comparator method or not tested with the comparator method.

A summary of the LIAISON PLEX Respiratory Flex Assay retrospective clinical study performance, expressed as positive percent and negative percent agreements against the comparator method, are presented in Table 48.

Table 48. Retrospective Clinical Performance of the Respiratory Flex Assay vs. the Comparator Assay   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive PercentAgreement</td><td rowspan=1 colspan=3>NegativePercent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100</td><td rowspan=1 colspan=1>241/2501</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>93.3-98.1</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>234/234</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100</td><td rowspan=1 colspan=1>233/236</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>96.3-99.6</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>214/217</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.0-99.5</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>68.5-98.7</td><td rowspan=1 colspan=1>241/242</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.7-99.9</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>249/252</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>96.6-99.6</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td><td rowspan=1 colspan=1>24/272</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>71.9-96.1</td><td rowspan=1 colspan=1>223/2293</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>94.4-98.8</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>255/255</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>254/2554</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.8-99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>254/2554</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.8-99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza A subtype H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>NE</td><td rowspan=1 colspan=1>256/256</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>232/233</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.6-99.9</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>79.8-99.3</td><td rowspan=1 colspan=1>226/232</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>94.5-98.8</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>237/238</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.7-99.9</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>76.4-99.1</td><td rowspan=1 colspan=1>235/236</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.6-99.9</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>254/254</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>230/2335</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>96.3-99.6</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100</td><td rowspan=1 colspan=1>246/247</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.7-99.9</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; NE – not evaluable 1 Of the 9 specimens with false positive adenovirus results by the Respiratory Flex Assay, seven were negative by PCR/BDS and two were not tested.

$^ { 2 } \mathrm { O f }$ the 3 specimens with false negative enterovirus/rhinovirus results by the Respiratory Flex Assay, one was negative by PCR/BDS, one was positive by PCR/BDS, and one was not tested.

$^ { 3 } \mathrm { O f }$ the 6 specimens with false positive enterovirus/rhinovirus results by the Respiratory Flex Assay, four were negative by PCR/BDS and two were not tested.

4 The 1 specimen with a false positive influenza A/influenza A H1 result by the Respiratory Flex Assay was negative for influenza A by PCR/BDS and not tested for influenza A H1.

$^ 5 \mathrm { O f }$ the 3 specimens with false positive parainfluenza 4 results, one was negative by PCR/BDS and two were not tested.

# Contrived Clinical Study

Some respiratory pathogens are so rare that both prospective and archived specimen collection efforts were insufficient to demonstrate clinical performance. To supplement the prospective clinical study and retrospective archived study data, an evaluation of contrived specimens was performed for five pathogens: Bordetella holmesii, Bordetella parapertussis, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae. Positive contrived specimens for influenza A H1N1 pdm09 were prepared and tested prior to completion of the prospective clinical study, in anticipation of potentially low prevalence for influenza A H1. The prospective clinical study ended up yielding an adequate number of influenza A H1 positive specimens to demonstrate performance, however since the contrived data was already acquired, it’s presented here. These contrived clinical specimens were prepared using unique negative clinical NPS specimens. Contrived specimens were prepared by spiking representative strains at concentrations of 2x, 10x, and $1 0 0 \mathrm { x }$ LoD. Fifty total positive samples for each pathogen were prepared, interspersed with negative samples, and randomized before testing at two clinical sites.

A total of 300 contrived positive samples were tested along with pre-selected archived specimens in a randomized, blinded fashion. Out of the 300 specimens included in the contrived study analysis, 291 specimens $( 9 7 . 0 \% )$ generated valid Respiratory Flex Assay results on the first attempt. There were 9 specimens $( 3 . 0 \% )$ with an invalid result on the initial run. Of the 9 specimens retested, all 9 generated a valid result after a single retest for a final success rate of $100 \%$ (300/300). The results of contrived specimen testing with the Respiratory Flex Assay are presented in Table 49 below.

Table 49. LIAISON PLEX Respiratory Flex Assay Performance with Contrived Specimens   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">TargetConc.(xLoD)</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/(TN+FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">Bordetellaholmesii</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.7-100</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.4-100</td></tr><tr><td colspan="1" rowspan="1">100x</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.9-100</td><td colspan="1" rowspan="1">250/250</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.5-100</td></tr><tr><td colspan="1" rowspan="4">Bordetellaparapertussis</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.7-100</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">66.7-98.6</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.4-100</td></tr><tr><td colspan="1" rowspan="1">100x</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0</td><td colspan="1" rowspan="1">89.5-99.6</td><td colspan="1" rowspan="1">250/250</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.5-100</td></tr><tr><td colspan="1" rowspan="4">Bordetellapertussis</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.7-100</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.4-100</td></tr><tr><td colspan="1" rowspan="1">100x</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.9-100</td><td colspan="1" rowspan="1">250/250</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.5-100</td></tr><tr><td colspan="1" rowspan="4">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.7-100</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">100x</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.9-100</td><td colspan="1" rowspan="1">250/250</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.5-100</td></tr><tr><td colspan="1" rowspan="4">Influenza AH1N1 pdm09</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">80.5-99.3</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.4-100</td></tr><tr><td colspan="1" rowspan="1">100x</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0</td><td colspan="1" rowspan="1">89.5-99.6</td><td colspan="1" rowspan="1">250/250</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.5-100</td></tr><tr><td colspan="1" rowspan="3">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">80.5-99.3</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100</td></tr><tr><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td><td colspan="1" rowspan="1">64/65</td><td colspan="1" rowspan="1">98.5</td><td colspan="1" rowspan="1">91.8-99.7</td></tr><tr><td colspan="1" rowspan="1">100x</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0</td><td colspan="1" rowspan="1">89.5-99.6</td><td colspan="1" rowspan="1">249/250</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">97.8-99.9</td></tr></table>

# 1. Clinical Specificity:

See section C.Clinical Studies above.

2. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The LIAISON PLEX Respiratory Flex Assay prospective clinical study included a total of 1843 prospectively collected NPS specimens, of which 1780 were evaluable for SARS-CoV-2, 1730 were evaluable for Bordetella holmesii, 1773 were evaluable for Bordetella parapertussis, 1753 were evaluable for Bordetella pertussis, and 1820 were evaluable for all remaining target analytes. The number and percentage of cases positive for each analyte, as determined by LIAISON PLEX Respiratory Flex Assay, are presented below, stratified by collection site and age group.

Table 50. LIAISON PLEX Respiratory Flex Assay - Expected Values Stratified by Specimen Collection Site   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="7" rowspan="1">Expected Values</td></tr><tr><td colspan="1" rowspan="1">Site 1(N=508)</td><td colspan="1" rowspan="1">Site 02(N=113)</td><td colspan="1" rowspan="1">Site 03(N=326)</td><td colspan="1" rowspan="1">Site 4(N=789</td><td colspan="1" rowspan="1">Site 5(N=51)</td><td colspan="1" rowspan="1">Site 16(N=39)</td><td colspan="1" rowspan="1">Overall(N=1820)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">14.8%(75/508)</td><td colspan="1" rowspan="1">1.8%(2/112)</td><td colspan="1" rowspan="1">12.1%(39/321)</td><td colspan="1" rowspan="1">4.6%(36/789)</td><td colspan="1" rowspan="1">27.5%(14/51)</td><td colspan="1" rowspan="1">5.1%(2/39)</td><td colspan="1" rowspan="1">9.2%(168/1820)</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">0.0%(0/447)</td><td colspan="1" rowspan="1">0.0%(0/113)</td><td colspan="1" rowspan="1">0.0%(0/320)</td><td colspan="1" rowspan="1">0.0%(0/778)</td><td colspan="1" rowspan="1">0.0%(0/34)</td><td colspan="1" rowspan="1">0.0%(0/38)</td><td colspan="1" rowspan="1">0.0%(0/1730)</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">0.0%(0/477)</td><td colspan="1" rowspan="1">0.0%(0/113)</td><td colspan="1" rowspan="1">0.6%(2/322)</td><td colspan="1" rowspan="1">0.6%(5/788)</td><td colspan="1" rowspan="1">0.0%(0/34)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.4%(7/1773)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">0.0%(0/463)</td><td colspan="1" rowspan="1">0.0%(0/113)</td><td colspan="1" rowspan="1">0.0%(0/320)</td><td colspan="1" rowspan="1">0.0%(0/784)</td><td colspan="1" rowspan="1">0.0%(0/34)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.0%(0/1753)</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">0.0%(0/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">0.0%(0/321)</td><td colspan="1" rowspan="1">0.0%(0/789)</td><td colspan="1" rowspan="1">0.0%(0/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.0%(0/1820)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus</td><td colspan="1" rowspan="1">10.4%(53/508)</td><td colspan="1" rowspan="1">3.6%(4/112)</td><td colspan="1" rowspan="1">2.8%(9/321)</td><td colspan="1" rowspan="1">6.3%(50/789)</td><td colspan="1" rowspan="1">9.8%(5/51)</td><td colspan="1" rowspan="1">10.3%(4/39)</td><td colspan="1" rowspan="1">6.9%(125/1820)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus/Rhinovirus</td><td colspan="1" rowspan="1">31.1%(158/508)</td><td colspan="1" rowspan="1">6.3%(7/112)</td><td colspan="1" rowspan="1">19.0%(61/321)</td><td colspan="1" rowspan="1">12.0%(95/789)</td><td colspan="1" rowspan="1">33.3%(17/51)</td><td colspan="1" rowspan="1">15.4%(6/39)</td><td colspan="1" rowspan="1">18.9%(344/1820)</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">14.8%(75/508)</td><td colspan="1" rowspan="1">1.8%(2/112)</td><td colspan="1" rowspan="1">4.4%(14/321)</td><td colspan="1" rowspan="1">3.4%(27/789)</td><td colspan="1" rowspan="1">15.7%(8/51)</td><td colspan="1" rowspan="1">12.8%(5/39)</td><td colspan="1" rowspan="1">7.2%(131/1820)</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">2.2%(11/508)</td><td colspan="1" rowspan="1">0.9%(1/112)</td><td colspan="1" rowspan="1">3.1%(10/321)</td><td colspan="1" rowspan="1">15.3%(121/789)</td><td colspan="1" rowspan="1">3.9%(2/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">8.0%(145/1820)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">0.8%(4/508)</td><td colspan="1" rowspan="1">0.9%(1/112)</td><td colspan="1" rowspan="1">1.2%(4/321)</td><td colspan="1" rowspan="1">3.4%(27/789)</td><td colspan="1" rowspan="1">3.9%(2/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">2.1%(38/1820)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">1.6%(8/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">1.9%(6/321)</td><td colspan="1" rowspan="1">11.9%(94/789)</td><td colspan="1" rowspan="1">0.0%(0/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">5.9%(108/1820)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">0.0%(0/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">2.5%(8/321)</td><td colspan="1" rowspan="1">0.0%(0/789)</td><td colspan="1" rowspan="1">0.0%(0/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.4%(8/1820)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">0.0%(0/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">0.0%(0/321)</td><td colspan="1" rowspan="1">0.0%(0/789)</td><td colspan="1" rowspan="1">0.0%(0/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.0%(0/1820)</td></tr><tr><td colspan="1" rowspan="1">Site 1(N=508)</td><td colspan="1" rowspan="1">Site 02(N=113)</td><td colspan="1" rowspan="1">Site 03(N=326)</td><td colspan="1" rowspan="1">Site 4(N=789)</td><td colspan="1" rowspan="1">Site 5(N=51)</td><td colspan="1" rowspan="1">Site 16(N=39)</td><td colspan="1" rowspan="1">Overall(N=1820)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">1.2%(6/508)</td><td colspan="1" rowspan="1">0.9%(1/112)</td><td colspan="1" rowspan="1">0.6%(2/321)</td><td colspan="1" rowspan="1">0.3%(2/789)</td><td colspan="1" rowspan="1">0.0%(0/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.6%(11/1820)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">0.2%(1/508)</td><td colspan="1" rowspan="1">1.8%(2/112)</td><td colspan="1" rowspan="1">0.0%(0/321)</td><td colspan="1" rowspan="1">1.0%(8/789)</td><td colspan="1" rowspan="1">0.0%(0/51)</td><td colspan="1" rowspan="1">2.6%(1/39)</td><td colspan="1" rowspan="1">0.7%(12/1820)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">4.5%(23/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">2.8%(9/321)</td><td colspan="1" rowspan="1">0.4%(3/789)</td><td colspan="1" rowspan="1">11.8%(6/51)</td><td colspan="1" rowspan="1">2.6%(1/39)</td><td colspan="1" rowspan="1">2.3%(42/1820)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">0.6%(3/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">0.3%(1/321)</td><td colspan="1" rowspan="1">0.5%(4/789)</td><td colspan="1" rowspan="1">2.0%(1/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">0.5%(9/1820)</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus (RSV)</td><td colspan="1" rowspan="1">6.1%(31/508)</td><td colspan="1" rowspan="1">0.0%(0/112)</td><td colspan="1" rowspan="1">5.0%(16/321)</td><td colspan="1" rowspan="1">8.7%(69/789)</td><td colspan="1" rowspan="1">3.9%(2/51)</td><td colspan="1" rowspan="1">0.0%(0/39)</td><td colspan="1" rowspan="1">6.5%(118/1820)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">5.5%(28/507)</td><td colspan="1" rowspan="1">40.0%(42/105)</td><td colspan="1" rowspan="1">16.9%(55/326)</td><td colspan="1" rowspan="1">15.6%(119/765)</td><td colspan="1" rowspan="1">7.3%(3/41)</td><td colspan="1" rowspan="1">19.4%(7/36)</td><td colspan="1" rowspan="1">14.3%(254/1780)</td></tr></table>

Table 51. LIAISON PLEX Respiratory Flex Assay - Expected Values Stratified by Age Group   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Expe</td><td rowspan=1 colspan=3>Expected Values</td></tr><tr><td rowspan=1 colspan=1>0-1 years(N=347)</td><td rowspan=1 colspan=1>&gt;1-5 years(N=273)</td><td rowspan=1 colspan=1>&gt;5-21 years(N=439</td><td rowspan=1 colspan=1>&gt;21-65 years(N=528)</td><td rowspan=1 colspan=1>&gt; 65 years(N=237</td><td rowspan=1 colspan=1>Overall(N=1820)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>15.3%(53/347)</td><td rowspan=1 colspan=1>22.0%(60/273)</td><td rowspan=1 colspan=1>10.0%(44/439)</td><td rowspan=1 colspan=1>2.1%(11/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>9.2%(168/1820)</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>0.0%(0/316)</td><td rowspan=1 colspan=1>0.0%(0/247)</td><td rowspan=1 colspan=1>0.0%(0/411)</td><td rowspan=1 colspan=1>0.0%(0/522)</td><td rowspan=1 colspan=1>0.0%(0/234)</td><td rowspan=1 colspan=1>0.0%(0/1730)</td></tr><tr><td rowspan=1 colspan=1>Bordetellaparapertussis</td><td rowspan=1 colspan=1>0.6%(2/329)</td><td rowspan=1 colspan=1>1.6%(4/257)</td><td rowspan=1 colspan=1>0.2%(1/422)</td><td rowspan=1 colspan=1>0.0%(0/528)</td><td rowspan=1 colspan=1>0.0%(0/237)</td><td rowspan=1 colspan=1>0.4%(7/1773)</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>0.0%(0/322)</td><td rowspan=1 colspan=1>0.0%(0/254)</td><td rowspan=1 colspan=1>0.0%(0/416)</td><td rowspan=1 colspan=1>0.0%(0/525)</td><td rowspan=1 colspan=1>0.0%(0/236)</td><td rowspan=1 colspan=1>0.0%(0/1753)</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>0.0%(0/347)</td><td rowspan=1 colspan=1>0.0%(0/273)</td><td rowspan=1 colspan=1>0.0%(0/439)</td><td rowspan=1 colspan=1>0.0%(0/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>0.0%(0/1820)</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus</td><td rowspan=1 colspan=1>10.7%(37/347)</td><td rowspan=1 colspan=1>10.3%(28/273)</td><td rowspan=1 colspan=1>6.6%(29/439)</td><td rowspan=1 colspan=1>4.5%(24/528)</td><td rowspan=1 colspan=1>3.0%(7/233)</td><td rowspan=1 colspan=1>6.9%(125/1820)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>30.5%(106/347)</td><td rowspan=1 colspan=1>32.6%(89/273)</td><td rowspan=1 colspan=1>20.3%(89/439)</td><td rowspan=1 colspan=1>8.7%(46/528)</td><td rowspan=1 colspan=1>6.0%(14/233)</td><td rowspan=1 colspan=1>18.9%(344/1820)</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>11.5%(40/347)</td><td rowspan=1 colspan=1>13.9%(38/273)</td><td rowspan=1 colspan=1>6.4%(28/439)</td><td rowspan=1 colspan=1>3.8%(20/528)</td><td rowspan=1 colspan=1>2.1%(5/233)</td><td rowspan=1 colspan=1>7.2%(131/1820)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>4.3%(15/347)</td><td rowspan=1 colspan=1>2.2%(6/273)</td><td rowspan=1 colspan=1>13.2%(58/439)</td><td rowspan=1 colspan=1>7.8%(41/528)</td><td rowspan=1 colspan=1>10.7%(25/233)</td><td rowspan=1 colspan=1>8.0%(145/1820)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>1.4%(5/347)</td><td rowspan=1 colspan=1>0.4%(1/273)</td><td rowspan=1 colspan=1>2.7%(12/439)</td><td rowspan=1 colspan=1>2.7%(14/528)</td><td rowspan=1 colspan=1>2.6%(6/233)</td><td rowspan=1 colspan=1>2.1%(38/1820)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>3.2%(11/347)</td><td rowspan=1 colspan=1>1.8%(5/273)</td><td rowspan=1 colspan=1>10.7%(47/439)</td><td rowspan=1 colspan=1>4.9%(26/528)</td><td rowspan=1 colspan=1>8.2%(19/233)</td><td rowspan=1 colspan=1>5.9%(108/1820)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0.0%(0/347)</td><td rowspan=1 colspan=1>1.1%(3/273)</td><td rowspan=1 colspan=1>0.7%(3/439)</td><td rowspan=1 colspan=1>0.4%(2/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>0.4%(8/1820)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>0.0%(0/347)</td><td rowspan=1 colspan=1>0.0%(0/273)</td><td rowspan=1 colspan=1>0.0%(0/439)</td><td rowspan=1 colspan=1>0.0%(0/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>0.0%(0/1820)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>1.7%(6/347)</td><td rowspan=1 colspan=1>1.1%(3/273)</td><td rowspan=1 colspan=1>0.2%(1/439)</td><td rowspan=1 colspan=1>0.2%(1/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>0.6%(11/1820)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>0.6%(2/347)</td><td rowspan=1 colspan=1>0.7%(2/273)</td><td rowspan=1 colspan=1>0.5%(2/439)</td><td rowspan=1 colspan=1>0.9%(5/528)</td><td rowspan=1 colspan=1>0.4%(1/233)</td><td rowspan=1 colspan=1>0.7%(12/1820)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>4.0%(14/347)</td><td rowspan=1 colspan=1>6.2%(17/273)</td><td rowspan=1 colspan=1>1.1%(5/439)</td><td rowspan=1 colspan=1>1.1%(6/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>2.3%(42/1820)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>0.3%(1/347)</td><td rowspan=1 colspan=1>0.7%(2/273)</td><td rowspan=1 colspan=1>1.1%(5/439)</td><td rowspan=1 colspan=1>0.2%(1/528)</td><td rowspan=1 colspan=1>0.0%(0/233)</td><td rowspan=1 colspan=1>0.5%(9/1820)</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus (RSV)</td><td rowspan=1 colspan=1>17.3%(60/347)</td><td rowspan=1 colspan=1>8.4%(23/273)</td><td rowspan=1 colspan=1>2.5%(11/439)</td><td rowspan=1 colspan=1>2.1%(11/528)</td><td rowspan=1 colspan=1>5.6%(13/233)</td><td rowspan=1 colspan=1>6.5%(118/1820)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>8.4%(29/345)</td><td rowspan=1 colspan=1>5.2%(14/270)</td><td rowspan=1 colspan=1>9.2%(39/426)</td><td rowspan=1 colspan=1>21.9%(112/512)</td><td rowspan=1 colspan=1>26.4%(60/227)</td><td rowspan=1 colspan=1>14.3%(254/1780)</td></tr></table>

F Other Supportive Instrument Performance Characteristics Data: Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.