# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

# B Applicant

Life Technologies Corporation

C Proprietary and Established Names

Applied Biosystems TaqPath COVID-19 Diagnostic PCR Kit

D Regulatory Information

<table><tr><td>Product Code(s)</td><td>Classification</td><td>Regulation Section 21 CFR 866.3981 - Device</td><td>Panel</td></tr><tr><td>QQX</td><td>Class II</td><td>To Detect And Identify Nucleic Acid Targets In Respiratory Specimens From Microbial Agents That Cause The Sars-Cov- 2 Respiratory Infection And Other Microbial Agents When In A Multi- Target Test</td><td>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain market clearance for the Applied Biosystems TaqPath COVID-19 Diagnostic PCR Kit (referred to as TaqPath COVID-19 Diagnostic PCR Kit below).

# B Measurand:

SARS-CoV-2 RNA

# C Type of Test:

The TaqPath COVID-19 Diagnostic PCR Kit is a real-time reverse transcription polymerase chain reaction (RT-PCR) test.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The TaqPath COVID-19 Diagnostic PCR Kit is a real-time reverse transcription polymerase chain reaction (RT-PCR) test intended for the qualitative detection of nucleic acid from SARSCoV-2 in nasopharyngeal and anterior nasal specimens from individuals with signs and symptoms of respiratory tract infection.

The TaqPath COVID-19 Diagnostic PCR Kit is intended for use as an aid in the diagnosis of COVID-19 if used in conjunction with other clinical observations, epidemiological information, and laboratory findings. The SARS-CoV-2 RNA is generally detectable in upper respiratory (anterior nasal and nasopharyngeal swabs) specimens during the acute phase of infection. Positive results are indicative of the presence of SARS-CoV-2 RNA. Positive results do not rule out bacterial infection or co-infection with other pathogens. The agent detected may not be the definite cause of disease.

Negative results do not preclude SARS-CoV-2 infection and should not be used as the sole basis for patient management decisions.

The TaqPath COVID-19 Diagnostic PCR Kit is intended for use by qualified clinical laboratory personnel specifically instructed and trained in the techniques of real-time PCR and in vitro diagnostic procedures.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

IVD - For In Vitro Diagnostic Use Only

D Special Instrument Requirements:

This assay is to be used with one of the following real-time PCR instruments only:

• Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument Applied Biosystems QuantStudio 5 Dx Real-Time PCR Instrument

# IV Device/System Characteristics:

# A Device Description:

The TaqPath COVID-19 Diagnostic PCR Kit is a multiplex real-time RT-PCR test for the qualitative detection of RNA from SARS-CoV-2 in nasopharyngeal and anterior nasal specimens. Each TaqPath COVID-19 Diagnostic PCR Kit includes:

TaqPath COVID-19 Diagnostic PCR Assay Multiplex: Primer and probe sets to detect RNA from three specific genomic regions in SARS-CoV-2: Open Reading Frame (ORF) 1ab, Spike (S) protein, Nucleocapsid (N) protein and primers/probes for MS2   
bacteriophage.   
MS2 bacteriophage, an internal processing control (IPC) for nucleic acid extraction. TaqPath COVID-19 Diagnostic PCR Control as a positive RNA control that contains targets specific to the SARS-CoV-2 genomic regions targeted by the assays.   
TaqPath COVID-19 Diagnostic PCR Control Dilution Buffer for diluting the TaqPath COVID-19 Diagnostic PCR Control to a working concentration.

Patient samples and the IPC are manually added to KingFisher 96 Deep-Well plates and loaded onto the KingFisher Flex Purification System along with reagents from the MagMAX Viral/Pathogen II Nucleic Acid Isolation Kit for automated processing. In this process, nucleic acids from the patient samples are recovered using magnetic-bead technology. Unbound substances and impurities are removed with automated wash steps and purified nucleic acid is eluted from the beads with Elution Buffer.

The extracted sample RNA is subsequently mixed with TaqPath COVID-19 Diagnostic PCR Assay Multiplex and the TaqPath 1‑Step Multiplex Master Mix (No ROX) in 96-well reaction plates and loaded onto one of the following instruments for reverse transcription into cDNA followed by amplification of the SARS-CoV-2 target sequences and the MS2 IPC target sequence:

• Applied Biosystems 7500 Fast Dx Real Time PCR instrument Applied Biosystems QuantStudio 5 Dx Real-Time PCR instrument

Following amplification, the data from the test run is imported into COVID-19 Interpretive Software IVD Edition for analysis and interpretation. The COVID-19 Interpretive Software IVD Edition analyzes the run data, performs quality check analysis, and calculates the interpretive results for each sample and control based on the cycle threshold (Ct) value obtained for each of the targets. The imported data and interpretive results for each run are saved as a batch in the software. Results can be exported as CSV files and reports can be generated in PDF format.

# B Principle of Operation:

The TaqPath COVID-19 Diagnostic PCR Kit contains primer and probe sets to detect RNA from three specific genomic regions in SARS-CoV-2: Open Reading Frame (ORF) 1ab, Spike (S) protein, Nucleocapsid (N) protein and primers/probes for MS2 bacteriophage IPC. During PCR amplification, the probes anneal to the target sequences located between the forward and reverse primers for the target genes. During the PCR extension phase, the $5 '$ nuclease activity of the Taq polymerase degrades the probe, causing the reporter dye to separate from the quencher dye, generating a fluorescent signal. With each cycle, additional reporter dye molecules are cleaved from their respective probes, increasing the fluorescence intensity, which is measured at each PCR cycle by the real-time PCR instrument. Validation of the results is performed automatically by the COVID-19 Interpretive Software IVD Edition based on the performance of positive and negative controls (see Table 1 below). The patient sample results are automatically generated using the calling rules, plate validity and the Ct cutoff values for assay targets as shown in Table 2 below.

Table 1: Interpretation of Positive and Negative Controls   

<table><tr><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=2>Status</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=8>Negative Control</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=2>VALID</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>REPORTReport the results to the healthcareprovider.</td></tr><tr><td rowspan=6 colspan=4>All other scenarios</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=6 colspan=1>RETEST1. In the Samples pane of the Homescreen, ensure that none of thepatient samples are incorrectlyassigned as a control sample. If apatient sample has been incorrectlyassigned: In the instrumentsoftware, correct the Taskassignment, for EDS files changethe experiment name, save the filewith a new file name, import thecorrected file into the interpretivesoftware, then review the results.2.If the patient sample assignmentsare correct and the status is stillINVALID, check the target calls:•  If MS2 is reported asNEGATIVE, the extractionfailed. Repeat the extraction,ensuring that MS2 is correctlyadded to the negative control.If any viral gene targets are∞reported as POSITIVE,contamination occurred.Decontaminate the equipment,replace the extraction reagentsand qPCR reagents, then repeatthe extraction and PCR.</td></tr><tr><td rowspan=2 colspan=2>INVALID</td><td rowspan=2 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=8>Positive Control</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=2>VALID</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>REPORTReport the results to the healthcareprovider.</td></tr></table>

<table><tr><td rowspan=6 colspan=1>All other scenarios</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=6 colspan=1>RETESTIn the Samples pane of the Homescreen, ensure that none of thepatient samples are incorrectlyassigned as a control sample. If apatient sample has been incorrectlyassigned: In the instrumentsoftware, correct the Taskassignment, for EDS files changethe experiment name, save the filewith a new file name, import thecorrected file into the interpretivesoftware, then review the results.2. If the patient sample assignmentsare correct and the status is stillINVALID, check the target calls:If MS2 is reported asPOSITIVE, contaminationoccurred. Decontaminate theequipment, replace the qPCRreagents, then repeat the PCR.If any of the viral gene targetsare reported as NEGATIVE,repeat the PCR. Consider usingnew qPCR reagents andcontrols.</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>INVALID</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 2: Result Interpretation for Patient Samples and Required Action   

<table><tr><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">Status</td><td colspan="1" rowspan="1">Result</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">INVALID</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">RETESTRepeat test by re-extracting theoriginal sample and repeatingthe RT-PCR. If the repeat resultremains invalid, considercollecting a new specimen.</td></tr><tr><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">VALID</td><td colspan="1" rowspan="1">SARS-CoV-2Not Detected</td><td colspan="1" rowspan="1">REPORTReport the results to thehealthcare provider.</td></tr><tr><td colspan="3" rowspan="1">Only one SARS-CoV-2 target= POS</td><td colspan="1" rowspan="1">POSorNEG</td><td colspan="1" rowspan="1">VALID</td><td colspan="1" rowspan="1">SARS-CoV-2Inconclusive</td><td colspan="1" rowspan="1">RETEST/REPORT1.Repeat the test by re-extracting the originalsample and repeating theRT-PCR.IMPORTANT! Sampleswith a result of SARS-CoV-2 Inconclusive shall beretested one time.2.After retesting one time,report the results to thehealthcare provider.3.If the repeat result remainsinconclusive, the healthcare</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">provider should conductadditional confirmationtesting with a new specimen,if clinically indicated.</td></tr><tr><td colspan="1" rowspan="1">Two or more SARS-CoV-2targets = POS</td><td colspan="1" rowspan="1">POSorNEG</td><td colspan="1" rowspan="1">VALID</td><td colspan="1" rowspan="1">PositiveSARS-CoV-2</td><td colspan="1" rowspan="1">REPORTReport the results to thehealthcare provider.</td></tr></table>

# C Instrument Description Information:

1. Instrument Name:

Applied Biosystems 7500 Fast Dx Real Time PCR Instrument Applied Biosystems QuantStudio 5 Dx Real-Time PCR Instrument

2. Specimen Identification: Specimen identification can be configured in an automated fashion or entered manually.

3. Specimen Sampling and Handling: Nasopharyngeal or anterior nasal swab specimen collected in transport media.

4. Quality Control:

Following controls are included in the TaqPath COVID-19 Diagnostic PCR Kit:

a) MS2 bacteriophage internal processing control (IPC) for the nucleic acid extraction. MS2 IPC is added to the sample wells before RNA extraction.   
b) External Positive Control (TaqPath COVID-19 Diagnostic PCR Control) is required for each plate run and contains targets specific to the SARS-CoV-2 genomic regions targeted by the assays.   
c) A Negative Control (molecular-grade, nuclease-free, non-DEPC-treated water) is required for each plate run to monitor non-specific amplification, cross-contamination during experimental setup and nucleic acid contamination of reagents.

V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire COVID-19 Test 2   
B Predicate 510(k) Number(s): K211079   
C Comparison with Predicate(s):

Table 3: TaqPath COVID-19 Diagnostic Kit Substantial Equivalence Comparison   

<table><tr><td colspan="1" rowspan="1">Device &amp; Predicate Device</td><td colspan="1" rowspan="1">K233453</td><td colspan="1" rowspan="1">K211079</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">TaqPath COVID-19 Diagnostic Kit</td><td colspan="1" rowspan="1">BioFire COVID-19 Test 2</td></tr><tr><td colspan="1" rowspan="1">Regulation Number</td><td colspan="1" rowspan="1">21 CFR 866.3981</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Regulation Name</td><td colspan="1" rowspan="1">Multi-Target Respiratory Specimen NucleicAcid Test Including Sars-Cov-2 And OtherMicrobial Agents</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">QQX</td><td colspan="1" rowspan="1">QQX</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">RNA</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">RT-PCR</td><td colspan="1" rowspan="1">RT-PCR</td></tr><tr><td colspan="1" rowspan="1">Condition for use</td><td colspan="1" rowspan="1">For prescription useFor in vitro diagnostic use only.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">nasopharyngeal swabs and anterior nasalswabs</td><td colspan="1" rowspan="1">nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation</td><td colspan="1" rowspan="1">Automated test interpretation</td></tr><tr><td colspan="1" rowspan="1">Target Genes</td><td colspan="1" rowspan="1">ORF1ab, S gene, N Gene</td><td colspan="1" rowspan="1">ORF1ab, S gene, N Gene, ORF8</td></tr><tr><td colspan="1" rowspan="1">Intended Use/ IndicationsFor Use</td><td colspan="1" rowspan="1">The TaqPath COVID-19 Diagnostic PCRKit is a real-time reverse transcriptionpolymerase chain reaction (RT-PCR) testintended for the qualitative detection ofnucleic acid from SARS-CoV-2 innasopharyngeal and anterior nasalspecimens from individuals with signs andsymptoms of respiratory tract infection.The TaqPath COVID-19 Diagnostic PCRKit is intended for use as an aid in thediagnosis of COVID-19 if used inconjunction with other clinicalobservations, epidemiological information,and laboratory findings. The SARS-CoV-2RNA is generally detectable in upperrespiratory (anterior nasal andnasopharyngeal swabs) specimens duringthe acute phase of infection. Positive resultsare indicative of the presence of SARS-CoV-2 RNA. Positive results do not ruleout bacterial infection or co-infection withother pathogens. The agent detected maynot be the definite cause of disease.Negative results do not preclude SARS-CoV-2 infection and should not be used asthe sole basis for patient managementdecisions.The TaqPath COVID-19 Diagnostic PCRKit is intended for use by qualified clinicallaboratory personnel specifically instructedand trained in the techniques of real-timePCR and in vitro diagnostic procedures.</td><td colspan="1" rowspan="1">The BioFire COVID-19 Test 2 is aqualitative nested multiplexed RT-PCR in vitro diagnostic. test intendedfor use with the BioFire FilmArray2.0 and BioFire FilmArray TorchSystems.The BioFire COVID-19 Test 2detects nucleic acids from severeacute respiratory syndromecoronavirus 2 (SARS- CoV-2) innasopharyngeal swabs (NPS) fromindividuals suspected of COVID-19by their healthcare provider. Resultsare for the identification of SARS-CoV-2 RNA. The SARS-CoV-2RNA isgenerally detectable in NPSspecimens during the acute phase ofinfection. Positive results areindicative of the presence of SARS-CoV- 2 RNA; clinical correlationwith patient history and otherdiagnostic information is necessary todetermine patient infection status.Positive results do not rule out co-infection with other pathogens.Results are meant to be used inconjunction with other clinical,epidemiologic, and laboratory data, inaccordance with the guidelinesprovided by the relevant public healthauthorities.The BioFire COVID-19 Test 2 isintended for use by trained medicaland laboratory professionals in alaboratory setting or under thesupervision of a trained laboratoryprofessional.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">7500 Fast Dx, orQuantStudio 5 Dx</td><td colspan="1" rowspan="1">FilmArray 2.0 or FilmArray Torch</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">One negative control and one positivecontrol are run for each assay.</td><td colspan="1" rowspan="1">Two internal controls are included ineach reagent pouch for quality controlof sample processing and both PCRstages and melt analysis.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981.

# VII Performance Characteristics:

# A Analytical Performance:

# 1. Precision/Reproducibility

The reproducibility of the TaqPath COVID-19 Diagnostic PCR Kit was assessed across three sites utilizing both the 7500 Fast Dx Real-Time PCR (RT-PCR) instrument and the QuantStudio 5 Dx RT-PCR instrument at each site. A panel of three samples was created using inactivated SARS-CoV-2 virus (USA-WA1/2020) spiked into pooled negative nasopharyngeal (NP) matrix at concentrations to target 5x limit of detection (LoD), $1 . 5 \mathrm { x }$ LoD, and 0x LoD (i.e., no analyte). Each site conducted two runs per day involving two operators each performing one run per day for five non-consecutive days for each of the three lots of assay reagents. Each run tested three replicates of each sample, resulting in a total of 270 observations per panel member for each RT-PCR instrument (3 sites $\times 2$ operators/site $\times$ 3 lots/operator $\times 5$ days/lot $\times 3$ replicates/day $= 2 7 0$ observations). Samples were blinded and randomized before testing. All samples were extracted using the KingFisher Flex and the MagMAX Viral/Pathogen II Nucleic Acid Isolation Kit.

Qualitative results obtained from testing TaqPath COVID-19 Diagnostic PCR Kit across the three sites are summarized in Table 4. There was $100 \%$ agreement with the expected results observed across all three testing sites on both the 7500 Fast Dx RT-PCR instrument and the QuantStudio 5 Dx RT-PCR instrument.

The reproducibility of TaqPath COVID-19 Diagnostic PCR Kit for the panel members was also assessed based on the standard deviation (SD) and coefficient of variation $( \% C V )$ of the mean cycle threshold (Ct) value for each potential source of variability. This assessment, presented in Tables 5 and 6 for each of the RT-PCR instruments, is stratified by the assay target. An analysis will have less than 270 replicates if there are samples tested that do not have amplification detected for an assay target. The within-laboratory precision of the TaqPath COVID-19 Diagnostic PCR Kit by site was also evaluated and is presented in Table 7 and 8 for each of the RT-PCR instruments.

Table 4: Summary of the Qualitative Results for TaqPath COVID-19 Diagnostic PCR Kit   

<table><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">PanelMember</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">N[1]</td><td colspan="1" rowspan="1">Agreement with theExpected Results</td></tr><tr><td colspan="1" rowspan="7">7500 Fast Dx RT-PCRInstrument</td><td colspan="1" rowspan="3">1.5x LoD</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">5x LoD</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">No analyte</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">QuantStudio 5 Dx RT-PCRInstrument</td><td colspan="1" rowspan="3">1.5x</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">5x</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">No analyte</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100%</td></tr></table>

[1]Total number of valid tests

Table 5: Summary of Reproducibility for 7500 Fast Dx RT-PCR Instrument (Ct signal analysis)   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Detected </td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenLots</td><td rowspan=1 colspan=2>BetweenSites</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenOperators/Run</td><td rowspan=1 colspan=2>Repeatability(ihin-Run)</td><td rowspan=1 colspan=2>TotalReproducibility</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>1.5x LoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>31.82</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>2.54</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>32.27</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.33</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>264/270</td><td rowspan=1 colspan=1>32.27</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.47</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>2.61</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>24.48</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>2.04</td></tr><tr><td rowspan=4 colspan=1>5x LoD</td><td rowspan=1 colspan=1>ORFlab</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>30.14</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>2.07</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.50</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>30.57</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>2.01</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>30.41</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.53</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>24.46</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>2.42</td></tr><tr><td rowspan=1 colspan=1>0x LoD</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>270/270</td><td rowspan=1 colspan=1>24.50</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.97</td></tr></table>

[1]n is number of tests which contribute Ct values to the analysis. N is the total number of valid tests $\mathrm { C t } =$ cycle threshold, LoD $=$ limit of detection, $\mathrm { S D = }$ standard deviation, $\% { \mathrm { C V } } =$ percent coefficient of variation

Table 6: Summary of Reproducibility for QuantStudio 5 Dx RT-PCR Instrument (Ct signal analysis)   

<table><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Detected(n/N)</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">BetweenLots</td><td colspan="2" rowspan="1">BetweenSites</td><td colspan="2" rowspan="1">BetweenDays</td><td colspan="2" rowspan="1">BetweenOperators/Run</td><td colspan="2" rowspan="1">Repeatability(Within-Ruum)</td><td colspan="2" rowspan="1">TotalReproducibility</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="4">1.5x LoD</td><td colspan="1" rowspan="1">ORFlab</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">32.56</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.56</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.71</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">269/270</td><td colspan="1" rowspan="1">33.07</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.84</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">2.04</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">264/270</td><td colspan="1" rowspan="1">33.35</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.12</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">2.31</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">2.60</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">25.55</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">1.17</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">1.58</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">2.13</td></tr><tr><td colspan="1" rowspan="3">5x LoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">30.94</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.13</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">31.34</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">1.25</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">31.35</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.16</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">25.54</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">1.25</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">1.24</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">2.10</td></tr><tr><td colspan="1" rowspan="1">Ox LoD</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">270/270</td><td colspan="1" rowspan="1">25.52</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">1.23</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">1.13</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">2.01</td></tr></table>

[1]n is number of tests which contribute Ct values to the analysis. N is the total number of valid tests $\mathrm { C t } =$ cycle threshold, LoD $=$ limit of detection, $\mathrm { S D = }$ standard deviation, $\% { \mathrm { C V } } =$ percent coefficient of variation

Table 7: Summary of Within-Laboratory Precision by Site for 7500 Fast Dx RT-PCR Instrument (Ct signal analysis)   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=2>PanelMember</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Detected(n/N)</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenLots</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenOperators/Runs</td><td rowspan=1 colspan=2>Repeatability(Within-run)</td><td rowspan=1 colspan=2>Within-labPrecision</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=9 colspan=1>Site 1</td><td rowspan=4 colspan=2>1.5x LoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>32.19</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.69</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.80</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>32.56</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.61</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>32.38</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.81</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.64</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.70</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>2.12</td></tr><tr><td rowspan=4 colspan=2>5x LoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.39</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.24</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.32</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.90</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.26</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.65</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.26</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.55</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>2.26</td></tr><tr><td rowspan=1 colspan=2>Ox LoD</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.48</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>2.08</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>2.43</td></tr><tr><td rowspan=9 colspan=1>Site 2</td><td rowspan=4 colspan=2>1.5x LoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>31.17</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.71</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.82</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>31.76</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.83</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.95</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>86/90</td><td rowspan=1 colspan=1>32.18</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.30</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.30</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.17</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.58</td></tr><tr><td rowspan=3 colspan=2>5x LoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>29.42</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.48</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.66</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.02</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.29</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.12</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.26</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.09</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.83</td></tr><tr><td rowspan=1 colspan=2>Ox LoD</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.30</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.71</td></tr><tr><td rowspan=9 colspan=1>Site 3</td><td rowspan=4 colspan=2>1.5xLoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>32.11</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.75</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.92</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>32.47</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>2.07</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.18</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>32.27</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>3.13</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>3.55</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.64</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.69</td></tr><tr><td rowspan=4 colspan=2>5x LoD</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.60</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.45</td></tr><tr><td rowspan=1 colspan=1>N gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.80</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.23</td></tr><tr><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>30.44</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.39</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.75</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>2.21</td></tr><tr><td rowspan=1 colspan=2>0x LoD</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>24.71</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.72</td></tr></table>

[1]n is number of tests which contribute Ct values to the analysis. N is the total number of valid tests $\mathrm { C t } =$ cycle threshold, $\mathrm { L o D = }$ limit of detection, $\mathrm { S D = }$ standard deviation, $\% { \mathrm { C V } } =$ percent coefficient of variation

Table 8: Summary of Within-Laboratory Precision Results by Site for QuantStudio 5 Dx RT-PCR Instrument   

<table><tr><td colspan="1" rowspan="2">Site</td><td colspan="1" rowspan="2">PanelMember</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Detected(n/N)</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">BetweenLots</td><td colspan="2" rowspan="1">BetweenDays</td><td colspan="2" rowspan="1">BetweenOperators/Runs</td><td colspan="2" rowspan="1">Repeatability(Within-run)</td><td colspan="2" rowspan="1">Within-labPrecision</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="5">Site 1</td><td colspan="1" rowspan="4">1.5x LoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">32.50</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.58</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">1.75</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">33.11</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.69</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">1.88</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">33.20</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">1.59</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">1.77</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.36</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.62</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">2.00</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">30.93</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">31.41</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">1.21</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">31.47</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">1.03</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.31</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.77</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">2.00</td></tr><tr><td colspan="1" rowspan="1">Ox LoD</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.21</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.81</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">2.11</td></tr><tr><td colspan="1" rowspan="9">Site 2</td><td colspan="1" rowspan="4">1.5x LoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">32.69</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">1.61</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">33.22</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.84</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">1.97</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">86/90</td><td colspan="1" rowspan="1">33.43</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">1.02</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">2.55</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">2.85</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.63</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.40</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.65</td></tr><tr><td colspan="1" rowspan="4">5x LoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">30.98</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">1.01</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">31.33</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.17</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">31.24</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">1.07</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.58</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.61</td></tr><tr><td colspan="1" rowspan="1">Ox LoD</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.70</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.38</td></tr><tr><td colspan="1" rowspan="9">Site 3</td><td colspan="1" rowspan="4">1.5xLoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">32.48</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">1.78</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">1.96</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">32.89</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">1.97</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">2.28</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">33.44</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.82</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">2.65</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">3.22</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.65</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">2.27</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">2.68</td></tr><tr><td colspan="1" rowspan="4">5x LoD</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">30.91</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.33</td></tr><tr><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">31.28</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.49</td></tr><tr><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">31.34</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">1.22</td></tr><tr><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.72</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.74</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">2.60</td></tr><tr><td colspan="1" rowspan="1">Ox LoD</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">25.66</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.39</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">2.09</td></tr></table>

[1]n is number of tests which contribute Ct values to the analysis. N is the total number of valid tests $\mathrm { C t } =$ cycle threshold, $\mathrm { L o D = }$ limit of detection, $\mathrm { S D = }$ standard deviation, $\% { \mathrm { C V } } =$ percent coefficient of variation

# 2. Linearity:

Not applicable; this is a qualitative assay.

# 3. Analytical Specificity/Interference:

a) Cross-Reactivity and Microbial Interference

This study evaluated the analytical specificity of the TaqPath COVID-19 Diagnostic PCR Kit by testing samples containing organisms that are commonly found in the respiratory tract and might cause similar clinical symptoms as SARS-CoV-2. A panel of 46 potentially crossreactive organisms (viruses, bacteria and fungi) were spiked into pooled negative nasopharyngeal (NP) clinical matrix at a high, clinically relevant concentration $\mathrm { 1 0 ^ { 6 } C F U / m L }$ or higher for bacteria/fungi and $1 0 ^ { 5 } \ : \mathrm { P F U / m L }$ or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or higher for viruses) in the absence or presence of SARS-CoV-2 target (spiked at ${ \sim } 3 \mathbf { x }$ LoD). Organisms that did not meet the target testing concentration above were tested at the highest possible concentration. Each microorganism was tested in triplicate. For all organisms tested, no cross-reactivity or microbial interference was observed at the concentrations tested, as shown below.

Table 9: Summary of the Microorganisms and Concentrations Tested in CrossReactivity Studies   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Human coronavirus 229E[1]</td><td colspan="1" rowspan="1">2.8×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Human coronavirus NL631]</td><td colspan="1" rowspan="1">2.3×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">MERS-CoV</td><td colspan="1" rowspan="1">29.4 Ct</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Coronavirus-SARS</td><td colspan="1" rowspan="1">28.8 Ct</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Mastadenovirus B Type 7</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV)</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Influenza A[]</td><td colspan="1" rowspan="1">8.3×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Influenza B[]</td><td colspan="1" rowspan="1">8.3×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Respiratory syncytial virus A</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Respiratory syncytial virus B</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Rhinovirus]</td><td colspan="1" rowspan="1">2.8×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1.0×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Cytomegalovirus[1]</td><td colspan="1" rowspan="1">3.2×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Epstein-Barr virus</td><td colspan="1" rowspan="1">1.6×107 copies/mL (cross-reactivity)5.9×106 copies/mL (microbialinterference)</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Human Immunodeficiency Virus type 1[]</td><td colspan="1" rowspan="1">1.0×104 International Units (IU)/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Human coronovirus (HKU1)[1[2]</td><td colspan="1" rowspan="1">9.99 × 104 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Enterovirus (EV-D68)[1[2]</td><td colspan="1" rowspan="1">9.88 × 104 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1.0×106 Infectious Units (IFU)/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">5.4×107 GCE/mL (cross-reactivity)8.6×107 GCE/mL (microbialinterference)</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Staphylococcus epidermis, MRSE</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Staphylococcus aureus, MRSA</td><td colspan="1" rowspan="1">1.00×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.0×106 Colony Changing Units(CCU)/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Mycoplasma genitalium[1]</td><td colspan="1" rowspan="1">4.0×105 Bacteria/mL</td></tr><tr><td colspan="1" rowspan="1">Fungi</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Fungi</td><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">1.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Fungi</td><td colspan="1" rowspan="1">Pneumocystis carinii</td><td colspan="1" rowspan="1">1.0×106 Cells/mL</td></tr><tr><td colspan="1" rowspan="1">Biofluid</td><td colspan="1" rowspan="1">Pooled human nasal wash[3]</td><td colspan="1" rowspan="1">Neat</td></tr></table>

[1]A lower concentration was tested due to inability to obtain stock material with high titer. [2]Microbial interference was not evaluated. [3]Used to represent diverse microbial flora in the human respiratory tract.

In silico cross-reactivity analysis of the primer sequences was conducted in March 2023 for the SARS-CoV-2 and MS2 assay components of the TaqPath COVID-19 Diagnostic PCR Kit and common respiratory microorganisms to demonstrate specificity of the primers to their targeted sequences.

A total of 40,811 genome sequences (31,011 virus isolates, 9,996 bacterial isolates, and 14 fungal isolates) were downloaded from GenBank using the National Center for Biotechnology Information (NCBI) Genome (https://www.ncbi.nlm.nih.gov/datahub/genome) and NCBI Virus (https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/). As complete genomes for Candida albicans and Pneumocystis jirovecii where not on NCBI, their reference sequences (RefSeq) were tested. Rhinovirus Type 1A did not have any complete genomes or a RefSeq, so all 18 partial genomes downloaded from NCBI were tested. These sequences were aligned to the TaqPath COVID-19 Diagnostic PCR Kit assay primer and probe components using Basic Local Alignment Search Tool (BLAST) software v2.12.0 (see table below). Microorganisms were considered non cross-reactive if at least one of the SARS-CoV-2 and MS2 assay primers and probes have ${ \leq } 8 0 \%$ homology or is predicted to produce a qPCR amplicon of ${ \geq } 1 0 { , } 0 0 0$ bases.

None of the isolates analyzed in the study are predicted to cross-react with the TaqPath COVID-19 Diagnostic PCR Kit assay.

Table 10: Microorganisms Assessed for in silico Cross-Reactivity Analysis   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Respiratory Pathogen</td><td colspan="1" rowspan="1">Number of IsolatesTested</td></tr><tr><td colspan="1" rowspan="23">Bacteria</td><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">830</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp</td><td colspan="1" rowspan="1">445</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">3368</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">103</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2053</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">210</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">117</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">351</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">80</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">127</td></tr><tr><td colspan="1" rowspan="1">Neisseria sp.</td><td colspan="1" rowspan="1">310</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">665</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MRSA</td><td colspan="1" rowspan="1">605</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">154</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">151</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">268</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="3">Fungi</td><td colspan="1" rowspan="1">Aspergillus sp.</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="19">Viruses</td><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">284</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">341</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">6556</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus</td><td colspan="1" rowspan="1">485</td></tr><tr><td colspan="1" rowspan="1">Hepatitis B Virus</td><td colspan="1" rowspan="1">10963</td></tr><tr><td colspan="1" rowspan="1">Hepatitis C Virus</td><td colspan="1" rowspan="1">1091</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus-1</td><td colspan="1" rowspan="1">109</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">56</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">81</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">251</td></tr><tr><td colspan="1" rowspan="1">Human Immunodeficiency Virus type 1</td><td colspan="1" rowspan="1">6740</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">385</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">759</td></tr><tr><td colspan="1" rowspan="1">MERS coronavirus</td><td colspan="1" rowspan="1">657</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1332</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">88</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">74</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">415</td></tr><tr><td rowspan="5"></td><td>Parainfluenza virus 4</td><td>48</td></tr><tr><td>Rhinovirus Type 1A</td><td>18</td></tr><tr><td></td><td></td></tr><tr><td>SARS coronavirus</td><td>14</td></tr><tr><td>Varicella-zoster virus</td><td>149</td></tr></table>

# b) Interfering Substances Study:

The potential for interference for the TaqPath COVID-19 Diagnostic PCR Kit was evaluated with substances that may be present in respiratory specimens. A total of 27 potentially interfering endogenous and exogenous substances were tested at or above clinically relevant levels in negative pooled nasopharyngeal clinical matrix in absence (contrived negative sample) and presence (contrived positive sample) of SARS-CoV-2 target (spiked at ${ \sim } 3 \mathbf { x }$ LoD). Each sample was tested in triplicate. The FluMist nasal vaccine was not tested.

Two of the substances tested in the study, nasal corticosteroid Triamcinolone and nasal spray or drops containing Oxymetazoline (e.g., Afrin - No Drip, Extra Moisturizing) gave false results at concentrations greater than $5 \%$ (v/v). Additional testing at lower test substance concentrations were performed to determine the concentration where interference is no longer observed. None of the 27 substances tested were determined to be inhibitory to the TaqPath COVID-19 Diagnostic PCR Kit at the concentrations shown in Table 11 below.

Table 11: Potential Interfering Substances Tested on the TaqPath COVID-19 Diagnostic PCR Kit   

<table><tr><td colspan="1" rowspan="1">Interfering Substance</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Leukocytes (human)</td><td colspan="1" rowspan="1">1% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Mucin: bovine submaxillary gland, type I-S</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">1% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Throat lozenges: Benzocaine (7.5 mg), Dextromethorphan HBr (5 mg)</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Throat lozenges: Menthol (5.4 mg)</td><td colspan="1" rowspan="1">2.2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal sprays or drops: Phenylephrine</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal sprays or drops: Oxymetazoline (Afrin - Allergy Sinus)</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal sprays or drops: Oxymetazoline (Afrin - No Drip, ExtraMoisturizing)</td><td colspan="1" rowspan="1">5% (v/v)[]</td></tr><tr><td colspan="1" rowspan="1">Nasal sprays or drops: Sodium chloride with preservatives</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Bronchodilator: Albuterol</td><td colspan="1" rowspan="1">0.83 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Beclomethasone</td><td colspan="1" rowspan="1">2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Dexamethasone</td><td colspan="1" rowspan="1">1.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Flunisolide</td><td colspan="1" rowspan="1">2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Budesonide</td><td colspan="1" rowspan="1">1% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Mometasone</td><td colspan="1" rowspan="1">1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Triamcinolone</td><td colspan="1" rowspan="1">5% (v/v)[]</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids: Fluticasone</td><td colspan="1" rowspan="1">5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal gel: Luffa opperculata, sulfur</td><td colspan="1" rowspan="1">1% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Zinc Acetate</td><td colspan="1" rowspan="1">7.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Anti-viral drug: Remdesivir</td><td colspan="1" rowspan="1">6.7 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Anti-viral drug: Zanamivir</td><td colspan="1" rowspan="1">5.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Anti-viral drug: Oseltamivir</td><td colspan="1" rowspan="1">33 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, nasal ointment: Mupirocin</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibacterial, systemic: Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Homeopathic allergy medicine: Galphimia glauca, Histaminumhydrochloricum</td><td colspan="1" rowspan="1">10% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Tobacco product</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Analgesic (e.g., ibuprofen)</td><td colspan="1" rowspan="1">21.9 mg/dL</td></tr></table>

[1]Interference was observed at concentrations $5 5 \%$ $\mathbf { \Delta } ^ { \prime } \mathbf { v } / \mathbf { v } )$ .

# 4. Analytical Reactivity (Inclusivity):

The inclusivity of the TaqPath COVID-19 Diagnostic PCR Kit for detection of SARS-CoV-2 was confirmed by testing fifteen variants of SARS-CoV-2. Positive samples were contrived by spiking each SARS-CoV-2 virus strain into negative pooled nasopharyngeal clinical matrix at a concentration of $3 \mathbf { x }$ LoD (150 GCE/mL), then tested in triplicate in a blinded and randomized fashion.

All 15 strains of SARS-CoV-2 tested generated $100 \%$ positivity rate demonstrating analytical inclusivity. Results are summarized in the table below.

Table 12: TaqPath Diagnostic PCR Kit Inclusivity Study Results   

<table><tr><td colspan="1" rowspan="1">Variant</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Catalog Number</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Positivity Rate(#Positive/#Tested</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">BEIResources</td><td colspan="1" rowspan="1">NR-52287</td><td colspan="1" rowspan="1">USA-WA1/2020</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="3">ZeptoMetrix</td><td colspan="1" rowspan="1">0810589CFHI</td><td colspan="1" rowspan="1">Italy-INMI</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0810590CFHI</td><td colspan="1" rowspan="1">HongKong/VM20001061/2020</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Alpha; B.1.1.7</td><td colspan="1" rowspan="1">NATSars(CoV2)- VP</td><td colspan="1" rowspan="1">England/204820464/2020</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Beta; B.1.351</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NATSars(CoV2)- VP</td><td colspan="1" rowspan="1">South Africa/KRISP-K005325/2020</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Gamma; P.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NATSars(CoV2)- VP</td><td colspan="1" rowspan="1">Japan/TY7-503/2021</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Delta; B.1.617.2</td><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="1">NATSars(CoV2)- VP</td><td colspan="1" rowspan="1">USA/PHC658/2021</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Lambda; C.37</td><td colspan="1" rowspan="1">0810640CFHI</td><td colspan="1" rowspan="1">Peru/UN-CDC-2-4069945/2021</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Kappa; B.1.617.1</td><td colspan="1" rowspan="1">0810623CFHI</td><td colspan="1" rowspan="1">USA/CA-Stanford-15_S02/2021</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Omicron;B.1.1.529</td><td colspan="1" rowspan="1">0810642CFHI</td><td colspan="1" rowspan="1">USA/MD-HP20874/2021</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Iota; B.1.526</td><td colspan="1" rowspan="1">0810619CFHI</td><td colspan="1" rowspan="1">USA/NY-WADSWORTH-21025952-01/2021Isolate 1</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">B.1</td><td colspan="1" rowspan="1">0810621CFHI</td><td colspan="1" rowspan="1">USA/NY-Wadsworth-103677-01/2020</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">B.1.595</td><td colspan="1" rowspan="1">0810622CFHI</td><td colspan="1" rowspan="1">USA/NY-Wadsworth-33126-01/2020</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeta; P2</td><td colspan="1" rowspan="1">0810618CFHI</td><td colspan="1" rowspan="1">USA/NY-Wadsworth-21006055-01/2021</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Omicron; BA.2.3</td><td colspan="1" rowspan="1">0810643CFHI</td><td colspan="1" rowspan="1">USA/MD-HP24556/2022</td><td colspan="1" rowspan="1">100% (3/3)</td></tr></table>

In silico analysis was performed to determine inclusivity (reactivity) of the TaqPath COVID19 Diagnostic PCR Kit primer/probe sequences with all known strains/isolates of SARS$\mathrm { C o V - } 2$ from GISAID and GenBank databases from March 2020 to April 2024. Mismatch and melting temperature analyses were performed and genomes with $100 \%$ identity and/or melting temperature $\mathrm { ( T m ) }$ greater than the annealing temperature were considered reactive. Analysis indicated that $5 9 9 \%$ of the sequences are reactive based on $100 \%$ homology or Tm greater than annealing temperature to at least two of the three assay gene targets of the TaqPath COVID-19 Diagnostic PCR Kit. Evaluation of assay components that do not match $100 \%$ to target sequences indicates that most primer or probe mismatches are unlikely to affect assay function.

Sequences from major SARS-CoV-2 variants that were evaluated and found to be reactive with the assay include but are not limited to Alpha, Beta, Delta, Epsilon, Eta, Gamma, Iota, Kappa, Lambda, Mu, Omicron, Theta, and Zeta. Specific Omicron pango lineages evaluated and found to be reactive with the assay include but are not limited to BA.2.86, XBB.1.9.1, XBB.1.9.2, XBB.2.3, XBB.1.16, XBB.1.5, CH.1.1, JN.1, and KP.2.

5. Assay Reportable Range: Not applicable; this is a qualitative assay.

6. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a) Quality Controls

Please refer to Instrument Description Information (Section C.4) above for assay controls.

The stability of SARS-CoV-2 targets was evaluated using nasopharyngeal (NP) swabs collected in viral transport media (VTM). Positive contrived samples were prepared by spiking inactivated SARS-CoV-2 virus into pooled NP swab clinical matrix collected in VTM. Samples were tested fresh (T0) to establish a baseline and then aliquoted and stored at: 6 hours at ambient temperature ( $1 5 \mathrm { { ^ \circ C } }$ to $3 0 ^ { \circ } \mathrm { C }$ ), 6 days refrigerated $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ), 35 days frozen ( $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ ), and 35 days frozen at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ . To establish stability, samples from each condition were withdrawn and tested at intermediate timepoints including the final stated timepoint and one time point beyond the final stability claim. The results of the specimen stability study support the claims listed in Table 13.

Table 13: Stability of Swab Specimens in Viral Transport Media   

<table><tr><td rowspan=1 colspan=1>Storage Temperature</td><td rowspan=1 colspan=1>Stability</td></tr><tr><td rowspan=1 colspan=1>Ambient: 15°C to 30°</td><td rowspan=1 colspan=1>4 hours</td></tr><tr><td rowspan=1 colspan=1>Refrigerated: 2° to 8°</td><td rowspan=1 colspan=1>3 days (72 hours)</td></tr><tr><td rowspan=1 colspan=1>Frozen: -30°C to -10°</td><td rowspan=1 colspan=1>30 days; stable up to 1 freeze-thaw cycles</td></tr><tr><td rowspan=1 colspan=1>Frozen: ≤ -70°</td><td rowspan=1 colspan=1>30 days; stable up to 3 freeze-thaw cycles</td></tr></table>

# c) Fresh vs. Frozen Studies

A fresh vs. frozen study was performed to demonstrate equivalency between fresh and frozen SARS-CoV-2 nasopharyngeal (NP) swabs in viral transport media (VTM). Positive contrived samples were prepared by spiking inactivated SARS-CoV-2 virus in negative pooled NP swab matrix at a concentration of $0 \mathbf { x }$ , 1.5x and 5x LoD. Samples were tested at baseline and subsequently after $1 \mathbf { x }$ and $2 \mathbf { x }$ freeze-thaw cycles when stored at $- 1 0 ^ { \circ } \mathrm { C }$ to $- 3 0 ^ { \circ } \mathrm { C }$ , and after $1 \mathbf { x }$ , 2x, 3x and $4 \mathbf { x }$ freeze-thaw cycles when stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ . Ten replicates were tested for samples prepared at $0 \mathbf { x }$ and 5x LoD and 40 replicates were tested for samples at $1 . 5 \mathrm { x }$ LoD. All samples were negative at $0 \mathrm { { x } \ L o D }$ and all samples spiked at $1 . 5 \mathrm { x }$ and ${ 5 } \mathbf { x }$ LoD were positive. The results demonstrated that freeze-thaw up to two cycles when stored at $- 1 0 ^ { \circ } \mathrm { C }$ to $- 3 0 ^ { \circ } \mathrm { C }$ , and freeze-thaw up to four cycles when stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ , does not impact the performance of the TaqPath COVID-19 Diagnostic PCR Kit.

# d) Transport Media Comparison

The following six different transport media were evaluated for compatibility with the TaqPath COVID-19 Diagnostic PCR Kit:

• Remel MicroTest M4RT Multi-Microbe Media • Remel MicroTest M5 Multi-Microbe Media • Remel MicroTest M6 Multi-Microbe Media • Copan Universal Transport Medium (UTM-RT) • Bartels FlexTrans Transport Medium • BD Universal Viral Transport Medium

Comparable qualitative and quantitative results were obtained with all the six transport media types in the study.

# 7. Detection Limit:

The Limit of Detection (LoD) of the TaqPath COVID-19 Diagnostic PCR Kit was determined by serially diluting gamma-irradiated SARS‑CoV‑2 virus (USA‑WA1/2020) into pooled negative nasopharyngeal (NP) swab matrix. Samples were randomized and blinded prior to testing on two PCR instruments: QuantStudio 5 Dx RT-PCR instrument and 7500 Fast Dx RT-PCR instrument. LoD is defined as the lowest concentration of SARS-CoV-2 RNA that can be detected at a rate of at least $9 5 \%$ .

A preliminary LoD was determined by testing five replicates of three-fold dilutions of quantified SARS-CoV-2 inactivated virus. Subsequently, the LoD was confirmed by testing 20 replicates to demonstrate a $2 9 5 \%$ detection rate at the LoD.

The study established the LoD of the TaqPath COVID-19 Diagnostic PCR Kit to be 50 genomic copy equivalent $\mathrm { ( G C E ) / m L }$ when used with both, the QuantStudio 5 Dx RT-PCR instrument and the 7500 Fast Dx RT-PCR instrument. Table 14 below summarizes the results for each SARS-CoV-2 target, including mean cycle threshold (Ct), standard deviation (SD), COVID-19 positivity, and $\%$ assay target hit rate.

Table 14: Summary of LoD Results and Mean Ct Values for SARS-CoV-2 for the TaqPath COVID-19 Diagnostic PCR Kit   

<table><tr><td colspan="1" rowspan="2">VirusConcentration(GCE/mL)</td><td colspan="1" rowspan="2">COVID-19Positivity(#Detected/#Tested)</td><td colspan="4" rowspan="1">Assay Target Hit Rate (%))(#Detected/#Tested)</td><td colspan="4" rowspan="1">Mean Ct(SD)2</td></tr><tr><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">MS2</td></tr><tr><td colspan="10" rowspan="1">7500 Fast Dx RT-PCR Instrument</td></tr><tr><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">32.21(0.45)</td><td colspan="1" rowspan="1">32.70(0.32)</td><td colspan="1" rowspan="1">32.29(0.54)</td><td colspan="1" rowspan="1">25.53(0.22)</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">80%(16/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">33.74(0.84)</td><td colspan="1" rowspan="1">34.20(0.87)</td><td colspan="1" rowspan="1">33.91(0.65)</td><td colspan="1" rowspan="1">25.45(0.25)</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">70%(14/20)</td><td colspan="1" rowspan="1">80%(16/20)</td><td colspan="1" rowspan="1">85%(17/20)</td><td colspan="1" rowspan="1">45%(9/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">34.70(0.78)</td><td colspan="1" rowspan="1">35.10(0.91)</td><td colspan="1" rowspan="1">35.21(1.22)</td><td colspan="1" rowspan="1">25.33(0.24)</td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">25.21(0.20)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="9" rowspan="1">QuantStudio 5 Dx RT-PCR Instrument</td></tr><tr><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">32.92(0.55)</td><td colspan="1" rowspan="1">33.00(0.38)</td><td colspan="1" rowspan="1">32.76(0.59)</td><td colspan="1" rowspan="1">26.05(0.17)</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">34.24(0.95)</td><td colspan="1" rowspan="1">34.82(0.84)</td><td colspan="1" rowspan="1">34.15(1.02)</td><td colspan="1" rowspan="1">26.10(0.14)</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">85%(17/20)</td><td colspan="1" rowspan="1">75%(15/20)</td><td colspan="1" rowspan="1">85%(17/20)</td><td colspan="1" rowspan="1">70%(14/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">35.00(0.54)</td><td colspan="1" rowspan="1">36.06(0.85)</td><td colspan="1" rowspan="1">35.17(0.82)</td><td colspan="1" rowspan="1">26.10(0.26)</td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">0%(0/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">25.77(0.20)</td></tr></table>

[1]Assay target hit rate is calculated by the number of assay targets that are called positive divided by the number of replicates tested. Based on the calling rules of the assay, COVID‑19 positive is called if two or more of the three assay targets (Orf1ab, N gene, S gene) are called positive. Samples can have different combinations of positive targets, thus contributing to a less than $100 \%$ hit rate on the assay target level but $1 0 0 \%$ hit rate on the COVID‑19 positivity call.   
[2]For concentration levels that did not produce a positive result, no Ct statistics is displayed and it is denoted as 'N/A'.

# LoD Determination with WHO International Standard for SARS-CoV-2 RNA

The LoD of the TaqPath COVID-19 Diagnostic PCR Kit was determined using contrived positive specimens that were created by spiking the reconstituted WHO International Standard for SARS‑CoV‑2 RNA (NIBSC code: 20/146) in pooled negative nasopharyngeal (NP) swab matrix in viral transport media (VTM). The study was conducted in two phases. In Phase I, a preliminary LoD was determined by testing serial dilutions of contrived positive samples at nine concentrations. In Phase II, the LoD was confirmed by testing 20 replicates at three concentrations based around the preliminary LoD.

The LoD of the TaqPath COVID-19 Diagnostic PCR Kit using the WHO International Standard for SARS‑CoV‑2 RNA (NIBSC code: 20/146) was $1 5 0 \ : \mathrm { I U / m L }$ for the 7500 Fast Dx RT-PCR instrument and $5 0 \ \mathrm { I U / m L }$ for the QuantStudio 5 Dx RT-PCR instrument. The LoD results for each SARS-CoV-2 target, including mean Ct, standard deviation (SD), $\%$ COVID-19 positivity, and $\%$ assay target hit rate are summarized in Table 15 below.

Table 15: Summary of LoD Results and Mean Ct Values for SARS-CoV-2 for the TaqPath COVID-19 Diagnostic PCR Kit Using WHO International Standard   

<table><tr><td colspan="1" rowspan="2">Concentration(IU/mL)</td><td colspan="1" rowspan="2">COVID-19Positivity(#Detected/#Tested)</td><td colspan="4" rowspan="1">Assay Target Hit Rate (%)1](#Detected/#Tested)</td><td colspan="4" rowspan="1">Mean Ct(SD) 2</td></tr><tr><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">MS2</td><td colspan="1" rowspan="1">ORF1ab</td><td colspan="1" rowspan="1">N gene</td><td colspan="1" rowspan="1">S gene</td><td colspan="1" rowspan="1">MS2</td></tr><tr><td colspan="10" rowspan="1">7500 Fast Dx RT-PCR Instrument</td></tr><tr><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">32.36(2.40)</td><td colspan="1" rowspan="1">33.75(0.57)</td><td colspan="1" rowspan="1">32.66(0.49)</td><td colspan="1" rowspan="1">24.99(0.20)</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">90%(18/20)</td><td colspan="1" rowspan="1">90%(18/20)</td><td colspan="1" rowspan="1">75%(15/20)</td><td colspan="1" rowspan="1">95%(19/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">34.49(0.73)</td><td colspan="1" rowspan="1">35.58(0.69)</td><td colspan="1" rowspan="1">34.30(0.84)</td><td colspan="1" rowspan="1">25.01(0.16)</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">50%(10/20)</td><td colspan="1" rowspan="1">55%(11/20)</td><td colspan="1" rowspan="1">50%(10/20)</td><td colspan="1" rowspan="1">55%(11/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">35.01(0.76)</td><td colspan="1" rowspan="1">35.80(0.70)</td><td colspan="1" rowspan="1">35.26(1.06)</td><td colspan="1" rowspan="1">25.18(0.21)</td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">25.31(0.04)</td></tr><tr><td colspan="10" rowspan="1">QuantStudio 5 Dx RT-PCR Instrument</td></tr><tr><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">33.52(0.59)</td><td colspan="1" rowspan="1">34.27(0.61)</td><td colspan="1" rowspan="1">33.52(0.70)</td><td colspan="1" rowspan="1">25.22(0.18)</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">95%(19/20)</td><td colspan="1" rowspan="1">90%(18/20)</td><td colspan="1" rowspan="1">80%(16/20)</td><td colspan="1" rowspan="1">90%(18/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">35.09(0.83)</td><td colspan="1" rowspan="1">35.64(0.71)</td><td colspan="1" rowspan="1">34.91(0.94)</td><td colspan="1" rowspan="1">25.44(0.16)</td></tr><tr><td>17</td><td>50% (10/20)</td><td>50% (10/20)</td><td>45% (9/20)</td><td>50% (10/20)</td><td>100% (20/20)</td><td>35.33 (0.93)</td><td>36.64 (0.35)</td><td>35.73 (0.55)</td><td>25.60 (0.12)</td></tr><tr><td>0</td><td>0% (0/3)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>100% (3/3)</td><td>N/A</td><td>N/A</td><td>N/A</td><td>25.46 (0.03)</td></tr></table>

[1]Assay target hit rate is calculated by the number of assay targets that are called positive divided by the number of replicates tested. Based on the calling rules of the assay, COVID‑19 positive is called if two or more of the three assay targets (Orf1ab, $_ \mathrm { N }$ gene, S gene) are called positive. Samples can have different combinations of positive targets, thus contributing to a less than $100 \%$ hit rate on the assay target level but $1 0 0 \%$ hit rate on the COVID‑19 positivity call.

[2]For concentration levels that did not produce a positive result, no Ct statistics is displayed and it is denoted as 'N/A'.

# 8. Assay Cut-Off:

Once a TaqPath COVID-19 Diagnostic PCR Kit test run is completed, the data from the RT-PCR instrument's data collection software is imported into COVID-19 Interpretive Software IVD Edition for analysis and interpretation. The COVID-19 Interpretive Software calculates the interpretive results for each sample and control based on the cycle threshold (Ct) value obtained for each of the targets. The following Ct cutoff values for the assay targets is used during result interpretations.

Table 16: Ct Cutoff Values for Assay Targets in TaqPath COVID-19 Diagnostic PCR Kit   

<table><tr><td rowspan=1 colspan=1>Sample or Control</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Ct Cutoff</td></tr><tr><td rowspan=2 colspan=1>Positive Control</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>Valid Ct values are &gt;37</td></tr><tr><td rowspan=1 colspan=1>Viral targets</td><td rowspan=1 colspan=1>Valid Ct values are ≤37</td></tr><tr><td rowspan=2 colspan=1>Negative Control</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>Valid Ct values are ≤32</td></tr><tr><td rowspan=1 colspan=1>Viral targets</td><td rowspan=1 colspan=1>Valid Ct values are &gt;37</td></tr><tr><td rowspan=2 colspan=1>Clinical samples</td><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>Valid Ct values are ≤32[1]</td></tr><tr><td rowspan=1 colspan=1>Viral targets</td><td rowspan=1 colspan=1>Positive Ct values are ≤37</td></tr></table>

[1]If any of the viral targets is positive, the Ct for MS2 can be ${ > } 3 2$

# 9. Carry-Over:

Carry-over/cross-contamination rate of the TaqPath COVID-19 Diagnostic PCR Kit was evaluated by testing contrived SARS‑CoV‑2 high-positive (formulated at approximately $1 \times 1 0 ^ { 5 }$ plaque forming units $( \mathrm { P F U } ) / \mathrm { m L }$ in pooled negative nasopharyngeal swab matrix) and SARSCoV-2 negative samples in a checkerboard pattern. Ten runs using two KingFisher Flex purification instruments, two operators, and one of each real time PCR instruments–7500 Fast Dx RT-PCR instrument and QuantStudio 5 Dx RT-PCR instrument, were performed. Each run tested 47 replicates of contrived SARS-CoV-2 high positive samples and 47 replicates of SARSCoV-2 negative samples.

Out of a total of 470 negative samples tested, four false positive SARS‑CoV‑2 results were obtained. Two false positive results were obtained on 7500 Fast Dx RT-PCR instrument and two false positive results on QuantStudio 5 Dx RT-PCR instrument. The total carry-over/crosscontamination rate of the TaqPath COVID-19 Diagnostic PCR Kit was determined to be $0 . 8 5 \%$ (4/470).

# B Comparison Studies:

1. Method Comparison with Predicate Device: Refer to the Clinical Studies Section of this document.

2. Matrix Comparison:

Not applicable.

# C Clinical Studies:

The clinical performance of the TaqPath COVID-19 Diagnostic PCR Kit was established in a multi-site prospective study with three external testing sites evaluating nasopharyngeal (NP) and anterior nasal (AN) swab specimens in BD universal viral transport (UVT) medium from individuals with signs and symptoms of respiratory tract infection. One NP swab and one AN swab was collected from each participant enrolled in the all-comer prospective study from 11 geographically diverse locations in the US. The swab collection order was alternated between the NP and the AN swab. Testing of the TaqPath COVID-19 Diagnostic PCR Kit was conducted on the 7500 Fast Dx RT-PCR instrument and QuantStudio 5 Dx RT-PCR instrument. A composite comparator approach, using three highly sensitive FDA cleared/authorized SARS‑CoV‑2 molecular assays was used to evaluate the performance of the TaqPath COVID-19 Diagnostic PCR Kit. The composite comparator result was defined as the concordant results from two comparator assays. In case of discordance between the initial two comparator assays, the sample was tested by a third assay and the result of the third test determined the composite comparator result.

From April to August 2023, a total of 1,076 subjects were enrolled in the prospective clinical study. After exclusion of samples due to withdrawal from the study, not meeting the inclusion criteria or incomplete consent forms, shipment delays at collection sites or specimen leakage during transit, a total of 1,055 NP swab specimens and 1,052 AN swab specimens were tested in the study. Of the 1,055 NP swab specimens tested, two NP swab specimens with inconclusive/invalid results on the 7500 Fast Dx and QuantStudio 5 Dx RT-PCR instruments were excluded from data analysis, resulting in 1,053 evaluable NP swab specimens on each instrument. Of the 1,052 AN swab specimens, three had inconclusive result on the 7500 Fast Dx Real‑Time PCR instrument and were excluded from data analysis, resulting in 1,049 evaluable AN swab specimens on the 7500 Fast Dx RT-PCR instrument and 1,052 evaluable AN swab specimens on the QuantStudio $5 \mathrm { D x }$ RT-PCR instrument. Specimens evaluated using the TaqPath COVID-19 Diagnostic PCR Kit were tested either fresh (tested within 72 hours of collection when stored at $2 ^ { \circ } \mathrm { C } { - } 8 ^ { \circ } \mathrm { C }$ ) or frozen (stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ and tested within 30 days of collection).

# a) Clinical Performance of TaqPath COVID-19 Diagnostic PCR Kit for Nasopharyngeal Swabs

Out of $1 { , } 0 5 3 \mathrm { N P }$ swab specimens evaluated, 89 were positive and 964 were negative according to the composite comparator method. On the 7500 Fast Dx RT-PCR instrument, the TaqPath COVID19 Diagnostic PCR Kit had a positive percent agreement (PPA) of $9 8 . 9 \%$ (88/89) and a negative

percent agreement (NPA) of $9 8 . 4 \%$ (949/964). On the QuantStudio $5 \mathrm { D x }$ RT-PCR instrument, the TaqPath COVID-19 Diagnostic PCR Kit had a PPA of $9 8 . 9 \%$ (88/89) and a NPA of $9 8 . 7 \%$ (951/964). Performance is summarized in the tables below.

Table 17: Clinical Performance Estimates of the TaqPath COVID-19 Diagnostic PCR Kit on 7500 Fast Dx RT-PCR Instrument with NP Swabs Specimens   

<table><tr><td rowspan=2 colspan=1>TaqPath COVID-19 Diagnostic PCR Kit on7500 Fast Dx RT-PCR Instrument</td><td rowspan=1 colspan=3>Composite Comparator</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>949</td><td rowspan=1 colspan=1>950</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>964</td><td rowspan=1 colspan=1>1,053</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>95% CI[1]</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=1>88/89</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>93.9%99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=1>949/964</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>97.4%99.1%</td></tr></table>

[1] Wilson score

Table 18: Clinical Performance Estimates of the TaqPath COVID-19 Diagnostic PCR Kit on QuantStudio 5 Dx RT-PCR Instrument with NP Swabs Specimens   

<table><tr><td rowspan=2 colspan=1>TaqPath COVID-19 Diagnostic PCR Kit onQuantStudio 5 Dx RT-PCR Instrument</td><td rowspan=1 colspan=3>Composite Comparator</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>951</td><td rowspan=1 colspan=1>952</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>964</td><td rowspan=1 colspan=1>1,053</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>95% CI[1]</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=1>88/89</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>93.9%99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=1>951/964</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>97.7%99.2%</td></tr></table>

[1]Wilson score

# b) Clinical Performance of TaqPath COVID-19 Diagnostic PCR Kit for Anterior Nasal Swabs

Out of 1,049 AN swab specimens evaluated on the 7500 Fast Dx RT-PCR instrument, 85 were positive and 964 were negative according to the composite comparator method. On the 7500 Fast Dx RT-PCR instrument, the TaqPath COVID-19 Diagnostic PCR Kit had a PPA of $9 8 . 8 \%$ (84/85) and a NPA of $9 7 . 8 \%$ (943/964). Out of 1,052 AN swab specimens evaluated on the QuantStudio 5 Dx RT-PCR instrument, 85 were positive and 967 were negative according to the composite comparator method. On the QuantStudio $5 \mathrm { D x }$ RT-PCR instrument, the TaqPath COVID-19 Diagnostic PCR Kit had a PPA of $9 8 . 8 \%$ (84/85) and a NPA of $9 8 . 0 \%$ (948/967). Performance is summarized the tables below.

Table 19: Clinical Performance Estimates of the TaqPath COVID-19 Diagnostic PCR Kit on 7500 Fast Dx RT-PCR Instrument with AN Swabs Specimens   

<table><tr><td>TaqPath COVID-19 Diagnostic PCR Kit on</td><td>Composite Comparator</td></tr></table>

<table><tr><td rowspan=1 colspan=1>7500 Fast Dx RT-PCR Instrument</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>943</td><td rowspan=1 colspan=1>944</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>964</td><td rowspan=1 colspan=1>1,049</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>95% CI!I</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=1>84/85</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>93.6% 99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=1>943/964</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>96.7%98.6%</td></tr></table>

[1]Wilson score

Table 20: Clinical Performance Estimates of the TaqPath COVID-19 Diagnostic PCR Kit on QuantStudio 5 Dx RT-PCR Instrument with AN Swabs Specimens   

<table><tr><td rowspan=2 colspan=1>TaqPath COVID-19 Diagnostic PCR Kit onQuantStudio 5 Dx RT-PCR Instrument</td><td rowspan=1 colspan=3>Composite Comparator</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>948</td><td rowspan=1 colspan=1>949</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>967</td><td rowspan=1 colspan=1>1,052</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>95% CI]</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=1>84/85</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>93.6%99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=1>948/967</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>97.0%98.7%</td></tr></table>

[1]Wilson score Other Clinical Supportive Data: Not applicable

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

Samples were collected from 11 geographically diverse collection sites across the US. The SARS-CoV-2 positivity rates, as determined by theTaqPath COVID-19 Diagnostic PCR Kit during the clinical study, per collection site and overall, by specimen type, are shown in the table below.

Table 21: SARS-CoV-2 Positivity Rates by Collection Site Based on the TaqPath COVID19 Diagnostic PCR Kit   

<table><tr><td rowspan="2">Collection Site</td><td colspan="2">QuantStudio 5 Dx RT-PCR Instrument</td><td colspan="2">7500 Fast Dx RT-PCR Instrument</td></tr><tr><td>NP</td><td>AN</td><td>NP</td><td>AN</td></tr><tr><td rowspan="2">D&amp;H, Florida</td><td>Positivity Rate</td><td>Positivity Rate</td><td>Positivity Rate</td><td>Positivity Rate</td></tr><tr><td>28.2% (61/216)</td><td>29.0% (63/217)</td><td>28.2% (61/216)</td><td>30.1% (65/216)</td></tr><tr><td colspan="1" rowspan="1">KUR-B, New York</td><td colspan="1" rowspan="1">3.9% (3/76)</td><td colspan="1" rowspan="1">3.9% (3/77)</td><td colspan="1" rowspan="1">3.9% (3/77)</td><td colspan="1" rowspan="1">3.9% (3/77)</td></tr><tr><td colspan="1" rowspan="1">KUR-E, South Carolina</td><td colspan="1" rowspan="1">12.7% (13/102)</td><td colspan="1" rowspan="1">12.9% (13/101)</td><td colspan="1" rowspan="1">13.9% (14/101)</td><td colspan="1" rowspan="1">11.9% (12/101)</td></tr><tr><td colspan="1" rowspan="1">KUR-M, Texas</td><td colspan="1" rowspan="1">2.5% (2/81)</td><td colspan="1" rowspan="1">2.5% (2/81)</td><td colspan="1" rowspan="1">2.5% (2/81)</td><td colspan="1" rowspan="1">2.5% (2/81)</td></tr><tr><td colspan="1" rowspan="1">KUR_MB, California</td><td colspan="1" rowspan="1">7.7% (1/13)</td><td colspan="1" rowspan="1">7.7% (1/13)</td><td colspan="1" rowspan="1">7.7% (1/13)</td><td colspan="1" rowspan="1">7.7% (1/13)</td></tr><tr><td colspan="1" rowspan="1">KUR-P, South Carolina</td><td colspan="1" rowspan="1">5.8% (3/52)</td><td colspan="1" rowspan="1">5.8% (3/52)</td><td colspan="1" rowspan="1">5.8% (3/52)</td><td colspan="1" rowspan="1">5.8% (3/52)</td></tr><tr><td colspan="1" rowspan="1">KUR-R, California</td><td colspan="1" rowspan="1">0.0% (0/2)</td><td colspan="1" rowspan="1">0.0% (0/2)</td><td colspan="1" rowspan="1">0.0% (0/2)</td><td colspan="1" rowspan="1">0.0% (0/2)</td></tr><tr><td colspan="1" rowspan="1">WR-C, Tennessee</td><td colspan="1" rowspan="1">3.3% (2/60)</td><td colspan="1" rowspan="1">1.7% (1/59)</td><td colspan="1" rowspan="1">3.3% (2/60)</td><td colspan="1" rowspan="1">1.7% (1/58)</td></tr><tr><td colspan="1" rowspan="1">WR-L, Florida</td><td colspan="1" rowspan="1">6.7% (8/120)</td><td colspan="1" rowspan="1">5.0% (6/119)</td><td colspan="1" rowspan="1">6.7% (8/120)</td><td colspan="1" rowspan="1">5.1% (6/118)</td></tr><tr><td colspan="1" rowspan="1">WR-S, California</td><td colspan="1" rowspan="1">2.9% (7/240)</td><td colspan="1" rowspan="1">2.9% (7/240)</td><td colspan="1" rowspan="1">3.3% (8/240)</td><td colspan="1" rowspan="1">2.9% (7/240)</td></tr><tr><td colspan="1" rowspan="1">WR-V, Nevada</td><td colspan="1" rowspan="1">1.1% (1/91)</td><td colspan="1" rowspan="1">4.4% (4/91)</td><td colspan="1" rowspan="1">1.1% (1/91)</td><td colspan="1" rowspan="1">5.5% (5/91)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">9.6% (101/1053)</td><td colspan="1" rowspan="1">9.8% (103/1052)</td><td colspan="1" rowspan="1">9.8% (103/1053)</td><td colspan="1" rowspan="1">10.0% (105/1049)</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.