# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

# B Applicant

T2 Biosystems, Inc.

# C Proprietary and Established Names

T2Bacteria Panel

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QBX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3960 -Nucleic Acid-BasedDevice For TheAmplification, Detection,And Identification OfMicrobial PathogensDirectly From WholeBlood Specimens</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>NSU</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 -Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

Addition of Acinetobacter baumannii to the T2Bateria Panel.   
Removal of the warnings related to false positives for Escherichia coli and Pseudomonas aeruginosa from the T2Bacteria Panel device labeling.

# B Measurand:

The assay amplifies and detects nucleic acids of the following species: Acinetobacter baumannii

Enterococcus faecium Escherichia coli Klebsiella pneumoniae Pseudomonas aeruginosa Staphylococcus aureus

# C Type of Test:

The T2Bacteria Panel, performed on the T2Dx Instrument, is a molecular diagnostic assay for the detection of the above listed bacterial species from whole blood specimens obtained from patients with signs and symptoms of bacterial bloodstream infections.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The T2Bacteria Panel run on the T2Dx Instrument is a qualitative T2 magnetic resonance (T2MR) test for the direct detection of bacterial species in K2EDTA human whole blood specimens from patients with suspected bacteremia. The T2Bacteria Panel identifies six species of bacteria: Acinetobacter baumannii, Enterococcus faecium, Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa and Staphylococcus aureus.

The T2Bacteria Panel is indicated as an aid in the diagnosis of bacteremia and results should be used in conjunction with other clinical and laboratory data. Concomitant blood cultures are necessary to recover organisms for susceptibility testing or further identification, and for organisms not detected by the T2Bacteria Panel.

Results from the T2Bacteria Panel are not intended to be used as the sole basis for diagnosis, treatment, or other patient management decisions in patients with suspected bacteremia.

# C Special Conditions for Use Statement(s):

• Rx - For Prescription Use Only The Panel has been validated for use only with whole blood collected in $4 \mathrm { m L }$ tubes with K2EDTA anticoagulant. An indeterminate result for E. coli cannot be considered positive or negative, and no antimicrobial therapy decisions should be based on this result. Feraheme at a concentration of $\geq 2 1 \mu \mathrm { g / m L }$ was shown to interfere with detection of organisms with the T2Bacteria Panel. Interference by substances other than those described in the Interfering Substances section above could lead to erroneous results. The following cross reactivity was noted with the T2Bacteria Panel: o E. faecium probe – Enterococcus durans; o E. coli probe – Escherichia albertii, Escherichia fergusonii, Shigella boydii, Shigella dysenteriae, Shigella flexneri, Shigella sonnei; o K. pneumoniae probe – Klebsiella variicola, Klebsiella quasipneumoniae; $\bigcirc$ S. aureus probe – Staphylococcus argenteus High concentrations of $E$ . coli in the blood specimen may prevent the detection of low concentrations of $P .$ . aeruginosa when present in the same specimen.

# D Special Instrument Requirements:

The T2Bacteria Panel is performed on the T2Dx Instrument.

# IV Device/System Characteristics:

# A Device Description:

The T2Bacteria panel is a qualitative molecular diagnostic assay that employs a whole blood compatible PCR amplification followed by T2 magnetic resonance (T2MR) detection. The T2Bacteria Panel is performed on the T2Dx Instrument which executes all steps after specimen loading including red blood cell lysis, concentration of bacterial cells, supernatant removal, lysis of bacterial cells, amplification of bacterial DNA and detection of target specific amplicons via hybridization to target-specific particles.

The panel includes both the T2Bacteria Cartridge and the Reagent Tray. The T2Bacteria Cartridge contains the blood cell lysing reagent, beads, and a preservative in an aqueous solution. Additionally, pipettes and tubes are included in the cartridge. The Reagent Tray contains the internal control, reaction buffer, the enzyme solution (polymerase and dNTPs), and target specific oligonucleotide probes coated in Superparamagnetic Particles.

A $\mathrm { K } _ { 2 }$ EDTA whole blood specimen containing a minimum of $3 ~ \mathrm { m L }$ is loaded directly onto the T2Bacteria Sample Inlet, which is then placed on the T2Bacteria Cartridge along with the T2Bacteria Reagent Tray. The blood specimen is mixed with the red blood cell lysing reagent and the bacterial cells are concentrated by centrifugation. The internal control is added to the concentrated bacterial cells and is carried though the entire lysis, amplification, and detection process. A bead-beating process lyses the bacterial cells and the supernatant containing the DNA from the lysed bacterial cells and the internal control are amplified using the target and internal control-specific primers. The generated amplicons are aliquoted into individual tubes for detection of A. baumannii, E. faecium, E. coli, K. pneumoniae, $P .$ . aeruginosa, S. aureus and the internal control. Bacterial and internal control amplicons are hybridized to target-specific probes attached to superparamagnetic particles; hybridization results in clustering of the particles. The hybridization occurring in individual tubes is analyzed in the MR reader and a signal for each target is generated and is detected by T2 Magnetic Resonance (T2MR) indicating the presence of the target organism.

Up to seven specimens can be loaded onto the T2Dx instrument at the same time. When running the first specimen in a series or a single specimen, the result is reported in 3.5 hours from the time the specimen is loaded onto the instrument. The results are interpreted by the device software as valid or invalid (based on the result of the internal control or target detections), and if valid, results are reported as “Positive” or “Target not Detected” for each specific target. For $E$ . coli, results are reported as “Positive”, “Indeterminate” or “Target not Detected”. An Indeterminate result is a valid result, but the presence or absence of $E .$ . coli in the specimen cannot be definitively assessed.

# B Principle of Operation:

During processing on the T2Dx instrument, pathogens are concentrated directly in whole blood, then lysed to release the target DNA. Bacterial DNA is amplified with target-specific primers and amplicons are hybridized to target-specific probes attached to superparamagnetic particles causing clustering of the particles. A signal is detected by T2 Magnetic Resonance (T2MR)

indicating the presence of the target organism. The Internal Control on the Panel monitors performance for each patient sample or control.

# V Substantial Equivalence Information:

A Predicate Device Name(s): T2Bacteria Panel   
B Predicate 510(k) Number(s): K172708   
C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K233184Candidate Device</td><td colspan="1" rowspan="1">K172708Predicate Device</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">T2Bacteria Panel</td><td colspan="1" rowspan="1">T2Bacteria Panel</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The T2Bacteria Panelrun on the T2DxInstrument is aqualitative T2 magneticresonance (T2MR) testfor the direct detection ofbacterial species inK2EDTA human wholeblood specimens frompatients with suspectedbacteremia.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">4 mL whole bloodcollected in a bloodcollection tube withEDTA anticoagulant</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Platform</td><td colspan="1" rowspan="1">T2Dx Instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">Nucleic acidamplification followedby T2 magneticresonance detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Throughput</td><td colspan="1" rowspan="1">Single cartridge test withrandom access withseven drawers on T2Dx.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Cartridge Format</td><td colspan="1" rowspan="1">T2Bacteria TestCartridge anddisposables</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Acinetobacterbaumannii, Enterococcusfaecium, Escherichiacoli, Klebsiellapneumoniae,Pseudomonasaeruginosa andStaphylococcus aureus</td><td colspan="1" rowspan="1">Enterococcus faecium,Escherichia coli,Klebsiella pneumoniae,Pseudomonasaeruginosa andStaphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Warning Statements -False-positives</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">During prospectiveclinical studies, falsepositive results wereobserved for E. coli andP. aeruginosa inprospectively collectedspecimens. Usersshould be aware of thepossibility ofoccurrence of falsepositive results,especially for E. coliand P. aeruginosa andshould closely monitorQCheck negativecontrol results for anytrends and determinethe need for action.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

ISO 15223-1. Medical devices – Symbols to be used with information supplied by the manufacturer. Part 1 – General Requirements. $4 ^ { \mathrm { t h } }$ Ed. 2021-07

Refer to K172708 for complete list of standards/special controls/ guidance documents.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

The reproducibility study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. A multicenter reproducibility study was performed to determine the run to run, reagent lot, day to day and site to site reproducibility of the T2Bacteria panel for detection of A. baumannii. Testing was performed at three sites (two external and one internal) with a panel of target organisms, each tested in triplicate at two concentrations (1-2X LoD and 3-4X LoD) using two reagent lots. Testing was performed for six non-consecutive days with at least two operators per site for a total of 36 replicates per sample per site.

The reproducibility panel included A. baumannii spiked in fresh human whole blood specimens in triple-spiked samples (A. baumannii, $K .$ . pneumoniae and S. aureus). (The applicability of using multi-spiked samples was evaluated in a separate study, see below.) Bacterial levels were confirmed by colony count testing of the original suspension used for spiking. A total of 108 negative blood samples were included in the reproducibility panel. Reproducibility results were acceptable with $\%$ agreement with expected results for detection of A. baumannii. (Table 1)

Table 1. Summary of Reproducibility Results for A. baumannii Detection Across Sites, Reagents and Operators   

<table><tr><td>Sample Type</td><td>Concentration</td><td></td><td>Ab</td></tr><tr><td rowspan="5">A.baumannii, S. aureus and K. pneumoniae Spike</td><td rowspan="3">1-2 x LoD</td><td>N Pos / N Total</td><td>108/108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td>95% CI</td><td>96.6-100</td></tr><tr><td rowspan="3">3-4 x LoD</td><td>N Pos / N Total</td><td>108/108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td>95% CI</td><td>96.6-100</td></tr><tr><td rowspan="5">E. coli, P. aeruginosa, and E. faecium Spike</td><td rowspan="3">1-2 x LoD</td><td>N Pos / N Total</td><td>1/108</td></tr><tr><td>% Accurate</td><td>99.1</td></tr><tr><td>95% CI</td><td>94.9-100</td></tr><tr><td rowspan="3">3-4 x LoD</td><td>N Pos / N Total</td><td>0/108</td></tr><tr><td rowspan="2"></td><td>% Accurate</td><td>100</td></tr><tr><td>95% CI</td><td>96.6-100</td></tr><tr><td rowspan="3">Negatives</td><td rowspan="3">N/A</td><td>N Neg / N Total</td><td>108 / 108</td></tr><tr><td>% Accurate</td><td>100</td></tr><tr><td>95% CI</td><td>96.6-100</td></tr></table>

2. Linearity:

Not Applicable

3. Analytical Specificity/Interference:

A. Cross reactivity. The cross-reactivity study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. Analytical specificity testing was conducted to assess the cross-reactivity of the T2Bacteria Panel to pathogens at concentrations of $1 0 ^ { 3 } \mathrm { C F U / m L }$ for bacteria and fungi and $1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 }$ for viral pathogens. Species tested represented 128 different pathogenically, phylogenetically, or environmentally relevant organisms contrived in human whole blood. No cross reactivity was noted for any species with $A$ . baumannii.

During in silico analysis, five Acinetobacter species, other than A. baumannii, with ≥ $9 6 \%$ primer homology and $\geq 8 4 \%$ probe homology were identified. Three of these species were tested in the cross-reactivity study and were not detected by the assay. The remaining two species, A. seifertii and A. dijkshoorniae are rare species and were not tested in the analytical study. As both species show low percent homology for the capture probe, based on the in silico data, (similar to the three common Acinetobacter species tested and shown to be not detected), it is expected that these species are unlikely to cross react.

This cross-reactivity study was acceptable.

B. Co-infection. The co-infection study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. A competitive inhibition study was performed to evaluate the ability of the T2Bacteria panel to detect $A$ . baumannii present at 1-2X LoD (low concentration) in the presence of other clinically relevant organisms (on- and off-panel) that may be present in a co-infection. Three combinations of organism concentrations were tested in human whole blood samples: 1) samples containing A. baumannii and a second T2Bacteria panel target species, each at a concentration of 1-2X LoD (low concentrations); 2) samples containing A. baumannii at a low concentration and a second T2Bacteria target species at a concentration of 1000 CFU/mL (high concentration); and 3) samples containing A. baumannii at a low concentration and a second non-target species at a concentration of $1 0 0 0 \mathrm { C F U / m L }$ . Testing was performed using four lots of T2Bacteria Cartridges and three lots of reagent trays.

On-panel targets. Samples containing A. baumannii in a low concentration were tested with low and high concentrations of the following on-panel species: E. coli, E. faecium, K. pneumoniae, $P .$ . aeruginosa and S. aureus. A. baumannii was detected in all samples.

Off-panel targets. Samples containing A. baumannii in a low concentration were tested with high concentrations of the following off-panel species: Staphylococcus epidermidis, Staphylococcus haemolyticus, Streptococcus pneumoniae, Streptococcus salivarius, Enterobacter cloacae and Candida albicans. A. baumannii was detected in all samples.

The co-infection studies were acceptable for the detection of A. baumannii.

C. Interfering substances. The interfering substances study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. The study was performed to determine and characterize the effects of potential endogenous and exogenous interfering substances commonly found in blood on the performance of the T2Bacteria panel for detection of A. baumannii. Interferents were screened as pools of interferents in human blood samples; potential interferents were added in high concentrations to represent worst case scenarios; samples were tested in triplicate. Samples were inoculated with A. baumannii at a concentration of 2-3X LoD and samples without interferents were included as controls. Pooled interferents that caused a difference in signal when compared to a sample without interferents were further analyzed individually in blood samples.

Evaluation of potential endogenous substances included the following 13 substances: human DNA (buffy coat), bilirubin (conjugated and unconjugated), elevated ALT/AST, hemoglobin, intralipid (to mimic triglycerides), albumin/IgG, creatinine/urea, circulating human DNA, and lactoferrin. None of the endogenous substances tested showed interference with the detection of A. baumannii with the T2Bacteria panel.

A total of 28 potentially interfering substances were tested at high concentrations. Potential interferents included the following substances: EDTA, Feraheme, heparin, iopamidol, MRI contrast agent (gadopentetate dimeglumine), dexamethasone, lisinopril, cytarabine, amphotericin B trihydrate, caspofungin, fluconazole, metronidazole, micafungin, ampicillin, azithromycin, cefazolin (sodium salt), cefepime hydrochloride, cefoxitin sodium salt, ceftazidime pentahydrate, ciprofloxacin, clindamycin, gentamycin sulfate, imipenem/cilastatin, linezolid, meropenem trihydrate, piperacillin, tazobactam, vancomycin. None of the above exogenous interferents affected the detection of $A$ . baumannii. However, concentrations of Feraheme at $\geq 2 1 \mu \mathrm { g / m L }$ were previously shown to interfere with the performance of the T2Bacteria Panel in K172708. The interference of Feraheme with the T2Bacteria Panel was noted as a limitation in the device labeling.

4. Assay Reportable Range: Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

The quality control for establishment of the clinical sensitivity of the T2Bacteria panel for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. The T2Bacteria Panel utilizes two types of controls, the internal control and the external controls. The Internal Control (IC) is automatically introduced into each specimen during sample processing on the T2Dx instrument and is carried through the lysis, amplification and detection steps of the assay. The IC monitors the amplification and detection process and monitors for the presence of inhibitors in the specimen.

The external controls include the QCheck Positive Kit and the QCheck Negative Kit. The QCheck Positive Kit contains two sample blends of bacterial species, the ASaKp Positive Control (which contains cells of A. baumannii, S. aureus and $K$ . pneumoniae) and the EcPaEfm Positive Control (which contains cells of $E .$ . coli, $P .$ . aeruginosa and $E .$ . faecium) suspended in a buffer-based solution. These positive controls individually interrogate the respective, complementary probe-coated particles in the detection reactions. Results for the positive controls are reported for only the detection channels for species included in the control. The QCheck Negative Kit contains vials of a buffer-based solution which are used to detect contamination of the T2Bacteria regents or the instrument. Results of the negative control are reported for all six detection channels.

QC testing for prospective arm and contrived arm for A. baumannii

At least one Positive and one Negative EC were run at each site on each instrument that was utilized for each day of testing. The QCheck testing with both positive and negative samples was required to pass prior to use of the T2Dx on that day.

A total of 430 results were obtained with the ASaKp positive control during the clinical study. The positivity rate was $9 9 . 3 \%$ with a $9 5 \%$ confidence interval $9 7 . 9 \%$ to $9 9 . 8 \%$ . QCheck Negative controls also passed with $9 9 . 0 \%$ accuracy.

For each day of testing, each T2Dx device had a positive and negative QCheck sample run. Along with daily QCheck testing, additional QCheck Positive-AbSaKp testing was conducted to produce 20 test results with this control.

Testing of 20 QCheck Positive–AbSaKp controls resulted in a pass result for all samples ( $100 \%$ detection), which would indicate that A. baumannii was detected in all control samples. All QCheck Negative controls also passed, indicating no false positive or invalid results with these samples.

# 6. Detection Limit:

The detection limit study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. Limit of detection (LoD) testing was performed using two strains of A. baumannii inoculated into human whole blood. The determination of the LoD included an initial screening phase and a confirmatory phase. In the initial phase, triple spiked samples were prepared at (1, 3, 9 and 10 CFU/mL) and a preliminary LoD was identified. (The applicability of using multi-spiked samples was evaluated in a separate study, see below). The preliminary LoD for A. baumannii was confirmed using two reagent lots and was determined to be $9 \mathrm { C F U / m L }$ for Strain 1 and 3 CFU/mL for Strain 2.

To confirm the preliminary LoD, a minimum of 20 replicates of both strains of each species was tested at increasing concentrations until a positivity rate of $9 5 \%$ was achieved. A minimum of 20 replicates of both strains of A. baumannii were tested. Strain 1 was tested at both 3 CFU/mL and 9 CFU/mL. The LoD of Strain 2 was confirmed at $3 \mathrm { C F U / m L }$ based on the results of the preliminary and replicate testing. The overall LoD was confirmed to be 3 $\mathrm { C F U / m l }$ for A. baumannii.

Table 2. Confirmatory LoD Testing for A. baumannii   

<table><tr><td rowspan=2 colspan=1>CFU/mL</td><td rowspan=1 colspan=2>Strain 1</td><td rowspan=1 colspan=2>Strain 2</td><td rowspan=2 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>Reagent Lot 1No. Tested/No.Correct (%)</td><td rowspan=1 colspan=1>Reagent Lot 2No. Tested/No.Correct (%)</td><td rowspan=1 colspan=1>Reagent Lot 1No. Tested/No.Correct (%)</td><td rowspan=1 colspan=1>Reagent Lot 2No. Tested/No.Correct (%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>42/42 (100) *</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=2 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>NT</td></tr></table>

\* Extra replicates were run for A. baumannii Strain 2 to obtain matched fresh and frozen samples following a loss of frozen samples due to instrument/control errors.

# 7. Assay Cut-Off:

The assay cut-off study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. To establish the limit of blank (LoB) and cut-off values for the A. baumannii target, negative blood specimens from healthy donors and blood specimens from patients suspected of bacteremia were tested using multiple instruments and reagent lots across multiple days. T2MR signals were evaluated to establish the upper limit of the signal distribution for negative samples; the T2MR signal values encompassing $2 9 9 \%$ of all signals from negative samples was determined to be the LoB.

The assay cut-off values were determined for A. baumannii using spiked positive samples; the lower limit of the signal distribution for the A. baumannii target channel and for the internal control measurements determined the assay cut-off. The established cut-off was reevaluated utilizing data obtained with contrived samples evaluated during the clinical study to validate the cut-off with a larger specimen pool; analysis verified the cut-off for $A$ . baumannii to be 130 milliseconds (ms). The cut-off value was established to produce $\geq$ $9 5 . 6 \%$ specificity and $\geq 9 0 . 0 \%$ sensitivity.

# 8. Single species spike, multi-species spike equivalence

The single species/multi species spike equivalency study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. A comparison of the equivalence of the T2MR signal results obtained for human blood samples spiked with a single bacterial species vs. samples spiked with three bacterial species was performed to determine the applicability of testing multiple species in a single blood specimen for the analytical studies. Testing was conducted in healthy donor blood at 1–2X LoD and 3-4X LoD for each species.

Results obtained using a single strain of A. baumannii were compared to results obtained from samples spiked with the combination of A. baumannii/K. pneumoniae/S. aureus. Signals obtained from samples containing the single-spiked samples of A. baumannii gave signals equivalent to the signals obtained from the multi-spiked samples.

# 9. Analytical Sensitivity/Inclusivity

The inclusivity study for detection of A. baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. A total of $1 4 A$ . baumannii isolates were evaluated for inclusivity in the T2Bacterial panel. Isolates were selected to represent variations in antimicrobial resistance, as well as phylogenic, temporal, and geographic diversity (Table 3). Isolates were inoculated into human whole blood at a concentration of 2-3X LoD; testing of each sample was performed in triplicate. For 12/13 samples, the three replicates gave positive results for A. baumannii. For 1/13 samples, two of the three replicates gave positive results for A. baumannii; repeat testing with 20 replicates of this strain gave positive results for all replicates.

Table 3. Results Summary for T2Bacteria Reactivity (Inclusivity) Testing – A. baumannii   

<table><tr><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">N=3</td><td colspan="1" rowspan="1">N=20</td><td colspan="1" rowspan="1">Pass/Fail</td></tr><tr><td colspan="1" rowspan="1">17961</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT*</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">BAA-2093</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">BAA-1878</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">BAA-747</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">BAA-1797</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">51432</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">17904</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">BAA-1605</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">13421</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">LMG10542</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">LMG10551</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">LMG1157</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">LMG22454</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr><tr><td colspan="1" rowspan="1">CCUG 6644</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NT</td><td colspan="1" rowspan="1">Pass</td></tr></table>

$^ { * } \mathrm { N T } =$ not tested

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not Applicable

2. Matrix Comparison: Not applicable

# C Clinical Studies:

# 1. Clinical Sensitivity:

The evaluation of the clinical sensitivity of the T2Bacteria panel for detection of $A$ . baumannii was performed during the initial evaluation of the T2Bacteria Panel, as described in K172708. The performance of A. baumannii on the T2Bacteria Panel was evaluated at eleven sites within the US and results were compared to standard of care (SOC) blood culture. Patients were enrolled prospectively, and two paired sample collections were obtained from each patient, one for blood culture and one for testing by the T2Bacteria Panel. Three FDA cleared blood culture systems were used in the study. The T2Bacteria Panel $A$ . baumannii result was compared against results from standards of care identification methods for Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA). A total of 1,427 samples were tested prospectively. There were no blood samples that tested positive for A. baumannii by standard of care methods during the prospective arm. However, there were 13 positives detected by the T2Bacteria panel, for an NPA of $9 9 . 1 \%$ (1414/1427). Of these 13 potential false positive results, one was positive for $A$ . baumannii by an amplification and gene sequencing.

Due to the low prevalence of positive blood cultures for A. baumannii, evaluation of the T2Bacteria panel was performed using contrived samples. Contrived sample testing was performed at three clinical sites. A total of $5 0 A$ . baumannii-spiked contrived samples and 50 negative samples were prepared using human whole blood specimens collected in $\mathrm { K } _ { 2 } \mathrm { E D T A }$ tubes from healthy adults that were verified to be negative for A. baumannii. Each contrived sample was prepared using a single unique A. baumannii strain; a total of 50 unique strains were tested at concentrations within the following ranges: $< 1$ CFU/mL, 1 – 10 CFU/mL, 11- 30 CFU/mL and 31-100 CFU/mL.

All 50 negative samples showed no detection of A. baumannii. Testing for A. baumannii at concentrations greater than the LoD showed a PPA of $9 7 . 5 \%$

Table 4. Detection of A. baumannii with Contrived Samples   

<table><tr><td rowspan="2">Organism</td><td colspan="4">Spike Concentration</td></tr><tr><td>&lt; 1 CFU/mL (below LoD)</td><td>4 - 9 CFU/mL</td><td>12 − 27 CFU/mL</td><td>31 - 59 CFU/mL</td></tr><tr><td rowspan="3">A. baumannii</td><td>PPA (%)</td><td>PPA (%)</td><td>PPA (%)</td><td>PPA (%)</td></tr><tr><td>95% CI</td><td>95% CI</td><td>95% CI</td><td>95% CI</td></tr><tr><td>4/10 (40.0)</td><td>17/18 (94.4%) *</td><td>17/17 (100)</td><td>5/5 (100)</td></tr><tr><td>LoD = 3 CFU/mL</td><td>16.8 - 68.7</td><td>74.3 - 99.0</td><td>81.6 -100</td><td>56.6 - 100</td></tr></table>

\* One isolate spiked at 8.7 CFU/mL was not detected

# 2. Clinical Specificity:

Based on the observed A. baumannii false positive results during the prospective arm of the clinical testing, several manufacturing and facility process changes and improvements were made.

Following these changes, healthy donor blood testing with the T2Bacteria Panel was performed to evaluate the specificity of the panel for A. baumannii when testing negative blood samples. Data was retrospectively collected from QC clean testing of T2Bacteria Panel reagents with negative blood samples. A total of 980 valid T2Bacteria results from the analysis of ten different lots of reagents that were tested between February 2022 through March 2023 were analyzed. Two (2) lots had a single A. baumannii false positive each for an overall positivity of $0 . 2 \%$ .

In addition, thirty-three unique strains of Acinetobacter baumannii were spiked individually into human whole blood at 2-3x LoD (6-9 CFU/ml). All samples were tested within 24 hours of production and were stored at $2 { - } 8 \mathrm { { ^ \circ C } }$ until tested. Testing was conducted over nine (9) days, across four (4) different T2Dx devices, using two (2) lots of cartridges, two (2) lots of reagents and blood from five (5) unique donors.

Results from testing thirty-three (33) unique strains of A. baumannii demonstrate $9 7 . 0 \%$ (32/33) positive detection (Table 5). One strain was not detected by the T2Bacteria Panel. Ten (10) additional samples were produced with this strain and tested with the T2Bacteria Panel. From this testing, one (1) of ten (10) samples were detected. This data suggests that the T2Bacteria Panel may be able to detect this strain, but sensitivity is substantially reduced.

Table 5. PPA for A. baumannii from additional testing after process changes   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>6-9 CFU/ml</td><td rowspan=1 colspan=1>97.0 % (32/33)</td><td rowspan=1 colspan=1>84.7 - 99.5%</td></tr></table>

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

Based on FDA review of the observed $E .$ . coli and $P .$ . aeruginosa channel false positive results described in K172708, the T2Bacteria Panel labeling included a warning indicating that any $E$ . coli and $P .$ . aeruginosa positive detection result should be presented with a

statement concerning the potential for false positive results. Several manufacturing and facility process changes and improvements were made to reduce the false positive rate.

After implementation of these changes, data was retrospectively collected from QC clean testing of T2Bacteria Panel reagents with negative blood samples. Analysis consisted of 980 valid T2Bacteria Panel results from analysis of 10 different lots of reagents that were tested between February 2022 and March 2023. Testing demonstrated five (5) lots had a single $E$ . coli false positive for an overall $E$ . coli false-positivity rate of $0 . 5 \%$ . The other five lots demonstrated no false positive results. This performance is acceptable and supported the removal of the warning regarding the risk of false positives for $E .$ . coli. Testing demonstrated three (3) lots out of ten (10) had a $P _ { - }$ . aeruginosa false positive (one lot has two (2) false positives) for a total of four (4) false positives and an overall $P .$ . aeruginosa false-positivity rate of $0 . 4 \%$ . This performance is acceptable and supported the removal of the warning regarding the risk of false positives for $P .$ . aeruginosa.

# D Clinical Cut-Off:

Not Applicable

# E Expected Values/Reference Range:

In the prospective clinical study submitted as part of K172708, a total of 1427 paired blood cultures and T2Bacteria specimens were evaluated. There was a total of 36 positive blood cultures and 199 positive T2Bacteria results. Positive T2Bacteria results were evaluated as noted above.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.