# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

# B Applicant

Luminex Corporation

# C Proprietary and Established Names

LIAISON PLEX Yeast Blood Culture Assay

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PEO</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3365 -Multiplex Nucleic AcidAssay For IdentificationOf Microorganisms AndResistance Markers FromPositive Blood Cultures</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>NSU</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 -Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to obtain a substantial equivalence determination for the LIAISION PLEX Yeast Blood Assay.

# B Measurand:

Nucleic acids from the following organisms: Candida albicans, Candida auris, Candida dubliniensis, Candida famata, Candida glabrata, Candida guilliermondii, Candida kefyr, Candida krusei, Candida lipolytica, Candida lusitaniae, Candida parapsilosis, Candida tropicalis, Candida haemulonii/duobushaemulonii, and Cryptococcus neoformans/gattii

# C Type of Test:

A multiplexed nucleic acid test intended for use with the automated LIAISON PLEX instrument for the qualitative in vitro detection and identification of nucleic acid from yeast in a positive blood culture media sample that demonstrates the presence of fungi as determined by Gram stain.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The LIAISON PLEX Yeast Blood Culture (BCY) Assay is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on the LIAISON PLEX System for simultaneous detection and identification of multiple potentially pathogenic fungal organisms in positive blood culture. The LIAISON PLEX BCY Assay is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain fungal organisms as determined by Gram Stain.

The LIAISON PLEX BCY Assay detects and identifies the following fungal organisms:

Candida albicans   
Candida auris   
Candida dubliniensis   
Candida famata   
Candida glabrata   
Candida guilliermondii   
Candida kefyr   
Candida krusei   
Candida lipolytica   
Candida lusitaniae   
Candida parapsilosis   
Candida tropicalis   
Candida haemulonii / duobushaemulonii   
Cryptococcus neoformans / gattii

The detection and identification of specific fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from LIAISON PLEX BCY Assay are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by LIAISON PLEX BCY Assay may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by LIAISON PLEX BCY Assay, susceptibility testing and differentiation of mixed growth) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only D Special Instrument Requirements: For use with the LIAISON PLEX System

# IV Device/System Characteristics:

# A Device Description:

The LIAISON PLEX Yeast Blood Assay (BCY Assay) is an automated test for the detection and identification of nucleic acids from yeast in a positive blood culture media sample. The LIAISON PLEX BCY Assay is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system, and which contain a fungal organism, as determined by Gram stain. The LIAISON PLEX BCY Assay is performed on the LIAISON PLEX System. The LIAISON PLEX System is a fully automated, benchtop device that performs sample preparation, polymerase chain reaction (PCR), and microarraybased hybridization for the detection of target-specific nucleic acids. The LIAISON PLEX BCY Assay components required to perform the test include the following single-use, disposables:

• LIAISON PLEX BCY Assay Test Cartridge LIAISON PLEX BCY Assay Transfer Pipettes (or equivalent)

The test reagents are supplied in a single, disposable test cartridge. The LIAISON PLEX BCY Assay has 14 different reportable targets. Reporting of these targets is based on detection of one or more of the nucleic acid targets. Results can be:

• reviewed by the user on the touch screen,   
• exported to a USB flash drive,   
• printed, or submitted to a Laboratory Information System (LIS) for reporting.

The LIAISON PLEX System automates the BCY Assay sample analysis through the following steps: 1) Sample Preparation, 2) Amplification, and 3) Hybridization., and 4) Signal Analysis. The image analysis of the microarray provides light signal intensities from the target-specific capture spots, as well as the negative control, background, and imaging control spots.

# B Principle of Operation:

The assay workflow is as follows:

1. The user adds the sample into the cartridge via a pipette and closes the cartridge sample port.   
2. The user creates a new order and scans or manually enters the sample ID.   
3. The user scans the assay cartridge barcode with the hand-held barcode reader.

4. The user inserts the assay cartridge into the open bay that the instrument identifies based on the number of runs, and then initiates the run.   
5. This is repeated, as needed, for up to six cartridges.   
6. Next, the system scans the barcode located on the top of the cartridge and automatically configures to perform the appropriate assay based on the unique barcode on the cartridge.

The following steps take place within the closed cartridge:

1. Sample Preparation - The sample is mixed with reagents and nucleic acid extraction occurs via mechanical and chemical cell lysis and magnetic bead-based nucleic acid isolation. The sample then goes through a series of washes to capture and purify the nucleic acid. The nucleic acid is then transferred into the PCR module.   
2. Amplification - The LIAISON PLEX System initiates an amplification process, which heats and cools the PCR tubes for replication of the extracted nucleic acid to generate target specific amplicons.   
3. Hybridization - Nucleic acid is transferred by the LIAISON PLEX System to the hybridization unit. Here, amplified DNA hybridizes to specific capture DNA arrayed on a glass slide in a microarray format and the bound target DNA, in turn, hybridizes with mediator and gold-nanoparticle probes. An image of the hybridization unit is taken and the LIAISON PLEX Software performs data analysis.   
4. Signal Analysis - Gold nanoparticle probes bound specifically to target-containing spots in the microarray are silver-enhanced, and light scatter from the spots is measured and further analyzed to determine the presence (Detected) or absence (Not Detected) of a target.

The system completes all necessary steps for the assay. No user intervention is necessary once the run is initiated. Once the run is complete, the user removes the cartridge(s) and disposes of them in the appropriate waste container.

Following a series of quality control checks a final call is made for the Target: Detected if the Target signal is above threshold and Not Detected otherwise. Failure of any quality control checks results in a “No Call” result.

# V Substantial Equivalence Information:

A Predicate Device Name(s): ePlex Blood Culture Identification Panel Fungal Pathogen (BCID-FP) Panel

B Predicate 510(k) Number(s): K182690

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K240627</td><td colspan="1" rowspan="1">K182690</td></tr><tr><td colspan="1" rowspan="1">Device TradeName</td><td colspan="1" rowspan="1">LIAISON PLEX Yeast BloodCulture Assay</td><td colspan="1" rowspan="1">ePlex Blood Culture IdentificationPanel Fungal Pathogen (BCID-FP)Panel</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td>Similarities</td><td></td><td></td></tr><tr><td>Intended Use/Indications For Use</td><td>The LIAISON PLEX Yeast Blood Culture (BCY) Assay is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on the LIAISON PLEX System for simultaneous detection and identification of multiple potentially pathogenic fungal organisms in positive blood culture. The LIAISON PLEX BCY Assay is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain fungal organisms as determined by Gram Stain. The LIAISON PLEX BCY Assay detects and identifies the following fungal organisms: • Candida albicans Candida auris •Candida dubliniensis • Candida famata Candida glabrata •Candida guilliermondi •Candida kefyr Candida krusei •Candida lipolytica •Candida lusitaniae Candida parapsilosis •Candida tropicalis •Candida haemulonii/ duobushaemulonii •Cryptococcus neoformans/gattii The detection and identification of</td><td>The GenMark ePlex Blood Culture Identification Fungal Pathogen (BCID-FP) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark's ePlex Instrument for simultaneous detection and identification of multiple potentially pathogenic fungal organisms in positive blood culture. The ePlex BCID-FP Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contains fungal organisms. The following fungal organisms are identified using the ePlex BCID-FP Panel: Candida albicans Candida auris Candida dubliniensis Candida famata Candida glabrata Candida guilliermondii Candida kefyr Candida krusei Candida lusitaniae Candida parapsilosis Candida tropicalis Cryptococcus gattii Cryptococcus neoformans Fusarium Rhodotorula The detection and identification of specific fungal nucleic acids from</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">results from the LIAISON PLEXBCY Assay are intended to beinterpreted in conjunction with Gramstain results and should not be usedas the sole basis for diagnosis,treatment, or other patientmanagement decisions.Negative results in the setting of asuspected bloodstream infection maybe due to infection with pathogensthat are not detected by this test.Positive results do not rule out co-infection with other organisms; theorganism(s) detected by LIAISONPLEX BCY Assay may not be thedefinite cause of disease. Additionallaboratory testing (e.g., sub-culturingof positive blood cultures foridentification of organisms notdetected by LIAISON PLEX BCYAssay, susceptibility testing anddifferentiation of mixed growth) andclinical presentation must be takeninto consideration in the finaldiagnosis of bloodstream infection.</td><td colspan="1" rowspan="1">The results from the ePlex BCID-FPPanel are intended to be interpreted inconjunction with Gram stain resultsand should not be used as the solebasis for diagnosis, treatment, orother patient management decisions.Negative results in the setting of asuspected bloodstream infection maybe due to infection with pathogensthat are not detected by this test.Positive results do not rule out co-infection with other organisms; theorganism(s) detected by the ePlexBCID-FP Panel may not be thedefinite cause of disease. Additionallaboratory testing (e.g., sub-culturingof positive blood cultures foridentification of organisms notdetected by ePlex BCID-FP Panel,susceptibility testing anddifferentiation of mixed growth) andclinical presentation must be takeninto consideration in the finaldiagnosis of bloodstream infection.</td></tr><tr><td colspan="1" rowspan="1">Measurand</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">DNA from detected organisms</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Blood Culture samples identified aspositive by a continuous monitoringblood culture system and whichcontain fungal organisms.</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Highly multiplexed nucleic acid PCRtest on an array.</td></tr><tr><td colspan="1" rowspan="1">Level ofAutomation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Automated test interpretation andreporting.</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Internal controls contained in thecartridge monitor performance ofeach step of the testing process.</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Target Analytes</td><td colspan="1" rowspan="1">Same except:the LIAISON PLEX BCYhas the additional targetsCandida lipolytica andCandida haemulonii/</td><td colspan="1" rowspan="1">Candida albicans, Candida auris,Candida dubliniensis, Candidafamata, Candida glabrata, Candidaguilliermondii, Candida kefyr,Candida krusei, Candida lusitaniae,</td></tr><tr><td></td><td>duobushaemulonii, the LIAISON PLEX BCY targets do not include Fusarium or Rhodotorula</td><td>Candida parapsilosis, Candida tropicalis, Cryptococcus gattii, Cryptococcus neoformans, Fusarium, Rhodotorula</td></tr><tr><td>Instrumentation</td><td>LIAISON PLEX</td><td>ePlex Instrument</td></tr><tr><td>Time to Result</td><td>107 minutes</td><td>About 1.5 hours</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# Standards

ISTA 3A. Packaged-Products for Parcel Delivery System Shipment $7 0 \mathrm { k g }$ (150 lbs.) or Less. (2018). ITSA 7D. Temperature Test for Transport Packaging. CLSI EP07. Interference Testing in Clinical Chemistry; Third Edition. CLSI EP25-A. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline.   
• CLSI EP17-A2. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition.   
• CLSI EP37. Supplemental Tables for Interference Testing in Clinical Chemistry; First Edition.   
• CLSI EP12-A2. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition. CLSI EP24-A2. Assessment of the Diagnostic Accuracy of Laboratory Testing Using Receiver Operating Characteristic Curves; Approved Guideline – Second Edition.   
• ANSI AAMI ISO 14971:2019. Medical devices – Applications of risk management to medical devices. ISO 23640. In Vitro Diagnostic Medical Devices – Evaluation of Stability of In Vitro Diagnostic Reagents. ISO 15223-1. Medical Devices – Symbols to be Used with Information to be Supplied by the Manufacturer-Part 1: General Requirements. Fourth Edition (2021-07). ISO 18113-1:2011. In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling). Terms, definition and general requirements.   
• ISO 18113-2:2011. In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use. ISO 18113-3:2011. In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling) – Part 3: In vitro diagnostic instruments for professional use. IEC 61010-1 Edition 3.1, Consolidated Version. Safety Requirements for Electrical Equipment for Measurement Control and Laboratory Use – Part 1: General Requirements, Including Corrigendum 1. (2017-01). IEC 61326-2-6 Edition 3.0. Electrical Equipment for Measurement Control and Laboratory Use – EMC Requirements – Part 2-6: Particular Requirements – In Vitro Diagnostic (IVD) Medical Equipment (2010-10).

IEC 60601-1-2 Edition 4.0. Medial Electrical Equipment – Part 1-2: General Requirements for Basic Safety and Essential Performance – Collateral Standard: Electromagnetic Disturbances-Requirements and Tests (2014-02). ISO 3864-1. Graphic Symbols – Safety Colors and Safety Signs – Part 1. Design Principles for Safety Signs and Safety Markings (2011).

# Special Controls

Class II Special Controls Guideline. Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures (May 27, 2015)

# Guidance Documents

Electronic Submission Template for Medical Device 510(k) Submissions - Guidance for Industry and Food and Drug Administration Staff (October 2, 2023).   
Content of Premarket Submissions for Device Software Functions - Guidance for Industry and Food and Drug Administration Staff (June 14, 2023).   
Cybersecurity in Medical Devices: Quality System Considerations and Content of Premarket Submissions - Guidance for Industry and Food and Drug Administration Staff (September 23, 2023).   
Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests - Guidance for Industry and FDA Staff (March 13, 2007).

# VII Performance Characteristics:

# A Analytical Performance:

1. Reproducibility/Precision:

a. Site-to-Site Reproducibility

Reproducibility of the LIAISON PLEX BCY assay was evaluated at three different sites (one internal, two external) using replicates of four blinded panels (Table 1) representing a subset of targets detected by the assay. The four panels consisted of a negative sample, a negative sample contrived with an off-panel organism, a multi-analyte panel of targets at ring positive low positive concentrations, and a multi-analyte panel of targets at ring positive plus 8 hour blood culture moderate positive concentrations. “Ring Positive” represents when a blood culture bottle is indicated as positive when incubating in an automated blood culture system. “Ring Positive $+ 8$ hour” represents when a blood culture bottle is allowed to incubate in an automated blood culture system for at least 8 hours beyond ring positivity (See Growth and Detection Study for more information). All panels were diluted into negative blood matrix. Negative blood matrix is media from a blood culture bottle spiked with whole blood and incubated for at least five days in an automated blood culture system and confirmed to be negative by the absence of growth on an appropriate nutrient agar plate. The study incorporated potential sources of variation introduced by site, day, and operator. Three replicates of each panel were evaluated once per day over five non-consecutive days for a total of 90 replicates per panel member (3 sites $\texttt { X 5 }$ days x 2 Operators x 3 replicates $= 9 0$ replicates). A single lot of LIAISON PLEX BCY assay cartridges was used for all testing. Results from the reproducibility study are shown in Table 2.

Table 1. Reproducibility Panel   

<table><tr><td>Panel Name</td><td>Organisms</td><td>Expected Targets</td><td>Growth Type</td></tr><tr><td>Ring Positive</td><td>Candida albicans Candida tropicalis Cryptococcus neoformans</td><td>Candida albicans Candida tropicalis Cryptococcus neoformans</td><td>Ring Positive</td></tr><tr><td>Ring Positive +8</td><td>Candida albicans Candida tropicalis Cryptococcus neoformans</td><td>Candida albicans Candida tropicalis Cryptococcus neoformans</td><td>Ring Positive + 8 hours</td></tr><tr><td>Negative Negative</td><td>Negative Blood Culture</td><td>None</td><td>Ring Negative</td></tr><tr><td>contrived with off- panel organism</td><td>Escherichia coli</td><td>None</td><td>Ring Positive + 8 hours</td></tr></table>

Table 2. Reproducibility Study Summary   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Target Type</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Agreement with Expected Results</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=3 colspan=1>C. albicans</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>96.7%(29/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>98.9%(89/90)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive +8Hours</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative BloodMatrix</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>C. tropicalis</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive +8Hours</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative BloodMatrix</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>C. neoformans</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive +8Hours</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative BloodMatrix</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Off-panelNegative</td><td rowspan=1 colspan=2>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>100%</td></tr></table>

No false positives were observed in either of the two negative panels for any of the evaluated organisms. The Ring Positive concentration gave expected results $100 \%$ of the time for Candida tropicalis and Candida neoformans and $9 8 . 9 \%$ of the time for Candida albicans. The Ring Positive $+ 8$ hour concentration yielded expected results $100 \%$ of the tested organisms. The results of the reproducibility study are acceptable.

# b. Lot-to-Lot Reproducibility

Lot-to-lot reproducibility of the LIAISON PLEX BCY assay was evaluated using one operator at a single site testing three unique lots over five non-consecutive days. The same four panels used to evaluate site-to-site reproducibility were used in the lot-to-lot reproducibility assessments. Each panel member was tested in triplicate once per day on five different days generating a total of 45 replicates per panel member (1 Site x 1

Operator x 3 Lots x 5 Days X 1 Run per Day x 3 Replicates per Run). The agreement with expected results from the lot-to-lot reproducibility study is presented in Table 3.

Table 3. Lot-to-Lot Reproducibility Summary   

<table><tr><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>ReportableTarget</td><td rowspan=1 colspan=4>Agreement with Expected Results</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=3 colspan=1>C. albicans</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 hours</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NegativeBlood Matrix</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>C. tropicalis</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 hours</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NegativeBlood Matrix</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>C. neoformans</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 hours</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NegativeBlood Matrix</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ContrivedNegative(E. coli)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Lot-to-lot Reproducibility testing demonstrated $100 \%$ positive agreement with expected results for evaluated LIAISON PLEX BCY assay targets (Candida albicans, Candida tropicalis, and $C i$ ryptococcus neoformans/gattii) at ring positivity and ring positivity $+ 8$ hours, as well as $100 \%$ negative agreement with expected results for the negative sample with no spiked organism (i.e., negative blood matrix) and the contrived negative sample containing an off-panel analyte (i.e., Escherichia coli). The results of these studies are acceptable.

# c. Within-Laboratory Precision

The within-laboratory precision/repeatability of the LIAISON PLEX BCY assay was assessed using two operators, two instruments, and a single lot of reagents over five nonconsecutive days. The sample panel members described in the site-to-site reproducibility section above were used. Data from this study are presented in Table 4.

Table 4. Precision/Repeatability Summary   

<table><tr><td colspan="1" rowspan="3">ReportableTarget</td><td colspan="1" rowspan="3">Panel</td><td colspan="5" rowspan="1">Agreement with Expected results</td><td colspan="2" rowspan="1">95% C.I.</td></tr><tr><td colspan="2" rowspan="1">Operator 1</td><td colspan="2" rowspan="1">Operator 2</td><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="2">Lower</td><td colspan="1" rowspan="2">Upper</td></tr><tr><td colspan="1" rowspan="1">Instrument1</td><td colspan="1" rowspan="1">Instrument2</td><td colspan="1" rowspan="1">Instrument1</td><td colspan="1" rowspan="1">Instrument2</td></tr><tr><td colspan="1" rowspan="3">C. albicans</td><td colspan="1" rowspan="1">RingPositive</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(9/9)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RingPositive+8 hours</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">NegativeBloodMatrix</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">C.tropicalis</td><td colspan="1" rowspan="1">RingPositive</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(9/9)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RingPositive+8 hours</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">NegativeBloodMatrix</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">C.neoformans</td><td colspan="1" rowspan="1">RingPositive</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(9/9)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RingPositive+8 hours</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">NegativeBloodMatrix</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">100%(7/7)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">ContrivedNegative(E. coli)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(9/9)</td><td colspan="1" rowspan="1">100%(9/9)</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">100%(30/30)</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr></table>

Within-laboratory precision/repeatability testing demonstrated $100 \%$ positive agreement with expected results for the LIAISON PLEX BCY assay targets evaluated (Candida albicans, Candida tropicalis, and Cryptococcus neoformans/gattii) at ring positivity and ring positivity $+ 8$ hours, as well as $100 \%$ negative agreement with expected results for the unspiked negative sample (i.e. negative blood matrix) in addition to the contrived negative sample containing off-panel analytes (i.e. Escherichia coli). The results of these studies are acceptable.

2. Linearity:

Not applicable, the assay is qualitative.

3. Analytical Specificity/Interference:

a. Analytical Specificity (Cross-Reactivity)

i. Laboratory Testing

This study evaluated the analytical specificity (cross-reactivity) of the LIAISON PLEX BCY assay in the presence of non-targeted microorganisms that may be found in clinical blood culture samples. A total of 77 off-panel organisms (40 offpanel bacterial species and 37 off-panel yeast species) were evaluated in the study. The potentially cross-reacting organisms were spiked into negative blood matrix for all targets on the assay and tested in triplicate in triplicate in the absence of the target organisms. Bacterial organisms were tested at concentrations $\ge 1 . 0 0 \mathrm { E } \mathrm { + } 0 8 \mathrm { C F U / m L }$ (or equivalent) and yeast organisms were tested at $\geq$ $1 . 0 0 \mathrm { E } { + 0 7 }$ CFU/mL (or equivalent), or the highest available concentration. Of the 40 bacterial species tested no cross-reactivity was observed at the concentrations tested (Table 5). One of three replicates was positive for the on-panel organisms

Candida albicans and Candida dubliniensis when the bacterium Klebsiella pneumoniae (ATCC-BAA 2146) was present. Additional testing with six replicates of Klebsiella pneumoniae did not result in any positive results which resulted in $1 1 \%$ positivity (1/9 total replicates) of $K .$ . pneumoniae at a concentration of $1 . 0 \mathrm { E } { + 0 8 }$ CFU/mL.

Of the 37 yeast species tested, 35 organisms had no cross-reactivity with on-panel organisms (Table 6). Trichosporon asteroides (CBS-KNAW) generated positive results for on-panel organisms Candida glabrata (3/3 replicates) and Candida haemulonii/duobushaemulonii (2/3 replicates) during initial testing. Root cause analysis determined the stock material used to prepare the $T .$ . asteroides samples were contaminated with $C .$ glabrata and $C .$ haemulonii/duobushaemulonii, and additional testing with 3 replicates from a freshly prepared stock of T. asteroides did not produce any positive results. Yarrowia deformans (NRRL Y-63659) cross-reacted and had $100 \%$ detection for the BCY target Candida lipolytica when tested at a concentration of $1 . 0 \mathrm { E } { + 0 7 }$ CFU/mL. The concentration of Y. deformans was lowered to $1 . 0 \mathrm { E + } 0 4 \ : \mathrm { C F U / m L }$ and tested, cross-reactivity was not observed at this lower concentration. Candida pseudohaemulonii (CBS-KNAW 10004) cross-reacted and had $100 \%$ detection for the BCY target Candida haemulonii/duobushaemulonii when tested at $1 . 0 \mathrm { E } { + } 0 7 \mathrm { C F U / m L }$ . Six additional replicates of $C .$ . pseudohaemulonii were tested and demonstrated $100 \%$ detection of the on-panel target $C .$ . haemulonii/duobushaemulonii. The concentration of $C .$ . pseudohaemulonii was lowered to $1 . 0 \mathrm { E } { + } 0 2 \mathrm { C F U / m L }$ and tested, cross-reactivity was not observed at this lower concentration. During the first round of retesting of C. pseudohaemulonii 1/6 replicates was positive for the on-panel target Candida lusitaniae. Root cause analysis determined the use of $C .$ . lusitaniae prior to testing in the same area contributed to the positive result. Cross-reactive organisms identified in this assessment are noted in the device labeling.

Table 5. Bacterial Species Evaluated for Cross-Reactivity Results Summary   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ConcentrationTested (CFU/mL)</td><td colspan="1" rowspan="1">Source &amp; SourceID</td><td colspan="1" rowspan="1">Positivity % (Organism)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">IHMA 128307</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 15309</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 700406</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 14579</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 33179</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 13124</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 43735</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 35029</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 14025</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 35030</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">NCTC 13846</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 29212</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 25559</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 43165</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC-BAA 2146</td><td colspan="1" rowspan="1">11% (C. albicans) (1/9)11% (C. dubliniensis)(1/9)</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 19115</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 53103</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">8.9E+06</td><td colspan="1" rowspan="1">ATCC 10054</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">IHMA 576602</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Prevotella denticola</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">BEI ResourcesHM-1173</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 12401</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 33397</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 25923</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 9993</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 700567</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 27844</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 29663</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 43809</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">8.9E+07</td><td colspan="1" rowspan="1">ATCC 17666</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">NCTC 13920</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 15914</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 6315</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 12964</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.0E+08</td><td colspan="1" rowspan="1">ATCC 15305</td><td colspan="1" rowspan="1">0% (0/3)</td></tr></table>

1 After initial false positive observation (1/3 replicates) additional testing with six replicates of Klebsiella pneumoniae did not result in any positive results.

Table 6. Yeast Species Evaluated for Cross-Reactivity Results Summary   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ConcentrationTested (CFU/mL)</td><td colspan="1" rowspan="1">Source &amp; SourceID</td><td colspan="1" rowspan="1">Positivity %(Organism)</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 1022</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Acremonium kiliense</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 20337</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida bracarensis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">JH 47</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida carpophila</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">CBS-KNAW5256</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida inconspicua</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-2029</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida intermedia</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 14439</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida metapsilosis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-48470</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida nivariensis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">CBS-KNAW9983</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida orthopsilosis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 20504</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida pseudohaemulonii</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">CBS-KNAW10004</td><td colspan="1" rowspan="1">100% (C. haemulonii)(9/9)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11% (C. lusitaniae)(1/9)</td></tr><tr><td colspan="1" rowspan="1">Candida pseudohaemulonii</td><td colspan="1" rowspan="1">1.0E+06</td><td colspan="1" rowspan="1">CBS-KNAW10004</td><td colspan="1" rowspan="1">100% (C. haemulonii)(//3)</td></tr><tr><td colspan="1" rowspan="1">Candida pseudohaemulonii</td><td colspan="1" rowspan="1">1.0E+05</td><td colspan="1" rowspan="1">CBS-KNAW10004</td><td colspan="1" rowspan="1">100% (C. haemulonii)(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida pseudohaemulonii</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="1">CBS-KNAW10004</td><td colspan="1" rowspan="1">100% (C. haemulonii)(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida pseudohaemulonii</td><td colspan="1" rowspan="1">1.0E+03</td><td colspan="1" rowspan="1">CBS-KNAW10004</td><td colspan="1" rowspan="1">33% (C. haemulonii)(1/3)</td></tr><tr><td colspan="1" rowspan="1">Candida pseudohaemulonii</td><td colspan="1" rowspan="1">1.0E+02</td><td colspan="1" rowspan="1">CBS-KNAW10004</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida rugosa</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-95</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida sake</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-1622</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida solani</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-2224</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Candida utilis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 9950</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Exophiala lecanii-corni</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 12734</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Filobasidium elegans</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-6486</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Filobasidium globisporum</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-17828</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Kluyveromyces lactis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 24207</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Kodamaea ohmeri</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-1932</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Meyerozyma caribbica</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-27274</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Malassezia furfur</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 14521</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Malassezia globosa</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC-MYA4612</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Metschnikowia pulcherrima</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 22032</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Malassezia restricta</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC-MYA4611</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Malassezia sympodialis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 96803</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Mucor velutinosus</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC-MYA4766</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Pichia fermentans</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-1619</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Pichia norvegensis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL YB-3904</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 66034</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 18824</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Scedosporium prolificans</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 64913</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Sporidiobolus salmonicolor</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-5483</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Sporothrix schenckii</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 58251</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Trichosporon asteroides</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">CBS-KNAW6183</td><td colspan="1" rowspan="1">50% (C. glabrata) (3/6)33% (C. haemulonii)(2/6)</td></tr><tr><td colspan="1" rowspan="1">Talaromyces marneffei</td><td colspan="1" rowspan="1">1.45E+06spores/mL</td><td colspan="1" rowspan="1">ATCC 18224</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Wickerhamomyces anomalus</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 10262</td><td colspan="1" rowspan="1">0% (0/3)</td></tr><tr><td colspan="1" rowspan="1">Yarrowia deformans</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">NRRL Y-63659</td><td colspan="1" rowspan="1">100% (C. lipolytica)(9/9)</td></tr><tr><td colspan="1" rowspan="1">Yarrowia deformans</td><td colspan="1" rowspan="1">1.0E+06</td><td colspan="1" rowspan="1">NRRL Y-63659</td><td colspan="1" rowspan="1">100% (C. lipolytica)(3/3)</td></tr><tr><td colspan="1" rowspan="1">Yarrowia deformans</td><td colspan="1" rowspan="1">1.0E+05</td><td colspan="1" rowspan="1">NRRL Y-63659</td><td colspan="1" rowspan="1">100% (C. lipolytica)(3/3)</td></tr><tr><td colspan="1" rowspan="1">Yarrowia deformans</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="1">NRRL Y-63659</td><td colspan="1" rowspan="1">0% (0/3)</td></tr></table>

# ii. In silico Testing

In silico analysis of assay specificity/exclusivity was performed by conducting a BLAST comparison of assay oligo sequences to the GenBank nt sequence database, as of December 28, 2023. Sequences for the organisms listed in Table 7 were evaluated for potential cross-reactivity. Sequences for all on-panel organisms were included to evaluate intra-panel cross-reactivity. In silico exclusivity analysis of potential non-specific amplification and detection indicates the following potential cross-reactivity against organisms listed in Table 7:

• The Candida lipolytica oligo designs are predicted to detect Yarrowia deformans strains resulting in false positive results. The Candida haemulonii/duobushamulonii oligo designs are predicted to detect Candida pseudohaemulonii strains resulting in false positive results.

Table 7. Potential Cross-Reactive Organisms Tested in In Silico Exclusivity Analysis   

<table><tr><td rowspan="2">On-Panel Organisms</td><td colspan="2">Off-Panel Organisms</td></tr><tr><td>Bacteria</td><td>Fungi</td></tr><tr><td>Candida albicans</td><td>Acinetobacter baumanii</td><td>Acremonium kiliense</td></tr><tr><td>Candida auris</td><td>Acinetobacter Iwoffii</td><td>Aspergillus fumigatus</td></tr><tr><td>Candida dubliniensis</td><td>Aerococcus viridans</td><td>Candida bracarensis</td></tr><tr><td>Candida duobushamulonii</td><td>Bacillus cereus</td><td>Candida carpophila</td></tr><tr><td>Candida famata</td><td>Bacteroides fragilis</td><td>Candida inconspicua</td></tr><tr><td>Candida glabrata</td><td>Bordetella pertussis</td><td>Candida intermedia</td></tr><tr><td>Candida guilliermondii</td><td>Citrobacter freundii</td><td>Candida lambica</td></tr><tr><td>Candida haemulonii</td><td>Clostridium perfringens</td><td>Candida metapsilosis</td></tr><tr><td>Candida kefyr</td><td>Corynebacterium striatum</td><td>Candida nivariensis</td></tr><tr><td>Candida krusei</td><td>Cutibacterium (Propionibacterium)</td><td>Candida norvegensis</td></tr><tr><td>Candida lipolytica</td><td>acnes Eggerthella lenta</td><td>Candida orthopsilosis</td></tr><tr><td>Candida lusitaniae</td><td>Enterobacter</td><td>Candida pelliculosa</td></tr><tr><td>Candida parapsilosis</td><td>aerogenes Enterobacter</td><td>Candida pseudohaemulonii</td></tr><tr><td>Candida tropicalis</td><td>cloacae Enterococcus avium</td><td>Candida sake</td></tr><tr><td>Cryptococcus gattii</td><td>Enterococcus</td><td>Candida utilis</td></tr><tr><td>Cryptococcus neoformans</td><td>faecalis Enterococcus</td><td>Diutina (Candida) rugosa</td></tr><tr><td></td><td>faecium Escherichia coli</td><td></td></tr><tr><td></td><td>Klebsiella oxytoca</td><td>Exophiala jeanselmei Filobasidium elegans</td></tr><tr><td rowspan="14"></td><td>Klebsiella pneumoniae</td><td>Filobasidium globisporum</td></tr><tr><td>Lactobacillus</td><td></td></tr><tr><td>rhamnosus</td><td>Fusarium solani</td></tr><tr><td>Listeria monocytogenes</td><td>Kluyveromyces lactis</td></tr><tr><td>Micrococcus luteus</td><td>Kodamaea ohmeri</td></tr><tr><td>Morganella</td><td>Malassezia furfur</td></tr><tr><td>morganii Prevotella denticola</td><td></td></tr><tr><td>Proteus mirabilis</td><td>Malassezia globose Malassezia restricta</td></tr><tr><td>Pseudomonas</td><td></td></tr><tr><td>aeruginosa Salmonella enterica</td><td>Malassezia sympodialis</td></tr><tr><td>(Typhi)</td><td>Metschnikowia (Candida) pulcherrima</td></tr><tr><td>Serratia marcescens</td><td>Meyerozyma caribbica (Candida fermentati)</td></tr><tr><td>Staphylococcus aureus</td><td>Mucor velutinosus</td></tr><tr><td>Staphylococcus epidermidis</td><td></td></tr><tr><td>Staphylococcus</td><td>Penicillium marneffei</td></tr><tr><td>hominis</td><td>Rhodotorula mucilaginosa</td></tr><tr><td>Staphylococcus intermedius</td><td>Saccharomyces cerevisiae</td></tr><tr><td>Staphylococcus</td><td></td></tr><tr><td>lugdunensis Staphylococcus</td><td>Scedosporium prolificans</td></tr><tr><td>saprophyticus Stenotrophomonas</td><td>Sporidiobolus salmonicolor</td></tr><tr><td>maltophilia</td><td>Sporothrix schenckii</td></tr><tr><td>Streptococcus agalactiae</td><td>Trichosporon spp.</td></tr><tr><td>Streptococcus</td><td>Yarrowia deformans</td></tr><tr><td>anginosus Streptococcus mitis</td><td>Protozoa</td></tr><tr><td>Streptococcus pneumoniae</td><td>Plasmodium falciparum</td></tr><tr><td></td><td></td></tr><tr><td>Streptococcus pyogenes</td><td>Trypanosoma cruzi</td></tr></table>

Organisms in bold were cross reactive with on-panel targets during laboratory exclusivity testing.

Cross-reactivity of Y. deformans and C. pseudohaemulonii with LIAISON PLEX BCY assay target oligos identified in the in silico analyses was confirmed by wet testing in the Analytical Specificity study and identified cross-reactive organisms are noted in the device labeling.

# b. Microbial Interference/Competitive Inhibition

To determine if non-target organisms can interfere with detection of on-panel organisms in the same sample, a microbial interference study was conducted. To simulate coinfection the study evaluated 33 pair-wise combinations of a low concentration (ring positive) on-panel target and potentially interfering off-panel organisms at high

concentration $\mathrm { 1 . 0 E + 0 8 \ C F U / m L } )$ . To evaluate competitive inhibition pair-wise combinations of a low-concentration on-panel target and a second on-panel target at high concentration (ring positive $+ 8$ hours) were also evaluated. The three on-panel targets selected for these evaluations are representative of the LIAISON PLEX BCY assay targets and chosen based on global prevalence. Off-panel organisms were also selected for their clinical prevalence and likelihood of being present in blood samples. Testing for all studies was performed in triplicate and results of the microbial interference study and competitive inhibition study are presented in Tables 8 and 9, respectively. Competitive Inhibition was not observed for the LIAISON PLEX BCY assay when low concentration on-panel organisms are tested in the presence of high concentration interfering microbes and other on-panel organisms and the results are acceptable.

Table 8. Microbial Interference Results Summary   

<table><tr><td colspan="1" rowspan="1">Off-Panel High ConcentrationInterfering Microbe</td><td colspan="1" rowspan="1">On-Panel LowConcentration Target</td><td colspan="1" rowspan="1">TargetPoositivity</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus aureus(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Enterococcus faecalis(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus epidermidis(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Off-Panel High ConcentrationInterfering Microbe</td><td colspan="1" rowspan="1">On-Panel LowConcentration Target</td><td colspan="1" rowspan="1">TargetPositivity</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Candida glabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter baumannii(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candidaalbicans(4.4E+05CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candidaglabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pneumoniae(1.0E+08 CFU/mL)</td><td colspan="1" rowspan="1">Candida albicans(4.4E+05CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candidaglabrata(3.8E+06 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis(1.5E+07 CFU/mL)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr></table>

Table 9. Competitive Inhibition Results Summary   

<table><tr><td rowspan=1 colspan=1>On Panel HighConcentration Target</td><td rowspan=1 colspan=1>High ConcentrationTarget Positivity</td><td rowspan=1 colspan=1>On Panel LowConcentration Target</td><td rowspan=1 colspan=1>Low ConcentrationTarget Positivity</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(5.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(5.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(1.6E+08 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(1.6E+08 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# c. Interfering Substances

An analytical study was performed to assess the potential inhibitory effects of 18 exogenous and endogenous substances that may be commonly found in blood samples (Table 10). Representative target organisms (Candida albicans, Cryptococcus neoformans, Candida tropicalis, Candida glabrata, Candida guilliermondii, and Candida kefyr) were tested at ring positive concentrations in negative blood matrix. A panel containing negative blood matrix was also tested to assess the risk of false positive results in the presence of potentially interfering substances. All organisms were tested using five replicates. No interference was observed except for the $C .$ .

albicans/Fluconazole $2 5 \mathrm { m g / L }$ combination. During initial testing of Fluconazole one replicate (1/5) did not detect Candida albicans. Dilution of Fluconazole from $2 5 \mathrm { m g / L }$ to $8 . 3 \mathrm { m g / L }$ resulted in $100 \%$ detection of $C .$ . albicans. Potential LIAISON PLEX BCY assay interference by Fluconazole at $2 5 \mathrm { m g / L }$ is noted in the device labeling. These results from the interfering substances study are acceptable.

Table 10. List of Potentially Interfering Substances   

<table><tr><td colspan="1" rowspan="1">Interfering Substance</td><td colspan="1" rowspan="1">TestingConcentration</td></tr><tr><td colspan="1" rowspan="1">Unconjugated Bilirubin</td><td colspan="1" rowspan="1">20 mg/dL</td></tr><tr><td colspan="1" rowspan="1">Conjugated Bilirubin</td><td colspan="1" rowspan="1">20 mg/dL</td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">14 g/L</td></tr><tr><td colspan="1" rowspan="1">γ-globulin</td><td colspan="1" rowspan="1">6 g/dL</td></tr><tr><td colspan="1" rowspan="1">SodiumPolyanetholsulfonate</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Amoxicillin clavulanate</td><td colspan="1" rowspan="1">3.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Amphotericin B</td><td colspan="1" rowspan="1">2 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Caspofungin</td><td colspan="1" rowspan="1">5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">0.23 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">3 mg/L</td></tr><tr><td colspan="1" rowspan="2">Fluconazole</td><td colspan="1" rowspan="1">25 mg/L1</td></tr><tr><td colspan="1" rowspan="1">8.3 mg/L1</td></tr><tr><td colspan="1" rowspan="1">Flucytosine</td><td colspan="1" rowspan="1">90 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Gentamicin sulfate</td><td colspan="1" rowspan="1">3 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Heparin</td><td colspan="1" rowspan="1">0.9 U/mL</td></tr><tr><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">83 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Tetracycline</td><td colspan="1" rowspan="1">5 mg/L</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">30 mg/L</td></tr><tr><td colspan="1" rowspan="1">Intralipid/Triglycerides</td><td colspan="1" rowspan="1">3000 mg/dL</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">0.9% NaCl SolventBlank</td><td colspan="1" rowspan="1">N/A</td></tr></table>

1 During initial testing Fluconazole had (4/5) positivity for $C$ . albicans at a concentration of 25 $\mathrm { m g / L }$ , retesting at $8 . 3 \mathrm { m g / L }$ resulted in $100 \%$ detection.

4. Assay Reportable Range: Not applicable, this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a. Assay Controls

# i. Internal Controls:

Each BCY Assay cartridge includes internal controls that monitor performance of each step of the testing process, including sample preparation, amplification, and detection of targets. The internal control is added to the sample automatically prior to initiation of sample preparation. A post-amplification hybridization control serves as a proxy for hybridization success. Internal control results are reported as Pass or Fail on printed result reports (Table 11).

Table 11. Internal Control Results   

<table><tr><td rowspan=1 colspan=1>InternalControl Result</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Suggested Action</td></tr><tr><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>Test was completed and internal controlswere successful, indicating that validresults were generated.</td><td rowspan=1 colspan=1>Review and report results</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>One or more internal controls failed.</td><td rowspan=1 colspan=1>Repeat test with a newcartridge</td></tr></table>

ii. Recommended External Control:

External controls are not provided with the BCY Assay but are recommended in the package insert. Positive and negative external controls should be tested with each new lot of reagents or monthly, whichever occurs first. Blood culture medium can be used as the negative control. Previously characterized positive samples or blood culture medium spiked with well characterized organisms can be used as the external positive control. External controls should be run in accordance with laboratory protocols and accrediting organizations, as applicable.

# b. Sample Stability

A sample stability study was performed to establish the recommended storage conditions for positive blood cultures prior to testing with the LIAISON PLEX BCY assay. Two representative panels, each containing three LIAISON PLEX BCY assay target organisms (Panel A - Candida albicans, Cryptococcus neoformans, and Candida tropicalis and Panel B - Candida glabrata, Candida guilliermondii, and Candida kefyr) were prepared at concentrations approximating ring positivity and were stored at -80 $\pm 1 5 ^ { \circ } \mathrm { C }$ , $2 { - } 8 ^ { \circ } \mathrm { C }$ , and $3 3 { - } 3 7 ^ { \circ } \mathrm { C }$ temperature over various time points (Table 12).

Additionally, sample stability was compared at ring positivity and 12 hours after ring positivity. Panel organisms were selected to represent both genera of yeast detected by the LIAISON PLEX BCY assay and a diverse fungal target panel. Twenty replicates were tested at the beginning of the study using freshly made panels (time point T0). Ten replicates were tested for each subsequent temperature and time point. The results of the stability study are presented in Tables 13-16 below.

Table 12. Sample Stability Storage Conditions and Time Points   

<table><tr><td rowspan=1 colspan=1>StorageCondition</td><td rowspan=1 colspan=8>Time Points</td></tr><tr><td rowspan=1 colspan=1>Incubator(33-37C)</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>25 hours</td><td rowspan=1 colspan=1>49 hours</td><td rowspan=1 colspan=1>73 hours</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Refrigerated(2-8°C)</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>25 hours</td><td rowspan=1 colspan=1>49 hours</td><td rowspan=1 colspan=1>73 hours</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>14 days</td><td rowspan=1 colspan=1>30days</td></tr><tr><td rowspan=1 colspan=1>Frozen(-80 ±15°C)</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>14 days</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>BottleIncubator</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>12 hourspost-RingPositive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 13. $3 3 { - } 3 7 ^ { \circ } \mathrm { C }$ Incubator Stability Storage Results Summary   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=1 colspan=6>Target (% Positive)</td></tr><tr><td rowspan=1 colspan=1>C. albicans</td><td rowspan=1 colspan=1>C. tropicalis</td><td rowspan=1 colspan=1>C. neoformans</td><td rowspan=1 colspan=1>C. glabrata</td><td rowspan=1 colspan=1>C. guilliermondii</td><td rowspan=1 colspan=1>C. kefyr</td></tr><tr><td rowspan=1 colspan=1>TOn= 20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>25 hoursn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>49 hoursn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>73 hoursn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 14. $2 { - } 8 ^ { \circ } \mathrm { C }$ Refrigerated Stability Storage Results Summary   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=1 colspan=6>Target (% Positive)</td></tr><tr><td rowspan=1 colspan=1>C. albicans</td><td rowspan=1 colspan=1>C. tropicalis</td><td rowspan=1 colspan=1>C. neoformans</td><td rowspan=1 colspan=1>C. glabrata</td><td rowspan=1 colspan=1>C. guilliermondii</td><td rowspan=1 colspan=1>C. kefyr</td></tr><tr><td rowspan=1 colspan=1>TOn= 20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>25 hoursn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>49 hoursn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>73 hoursn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>7 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>14 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 15. $- 8 0 \pm 1 5 ^ { \circ } \mathrm { C }$ Incubator Stability Storage Results Summary   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=1 colspan=6>Target (% Positive)</td></tr><tr><td rowspan=1 colspan=1>C. albicans</td><td rowspan=1 colspan=1>C. tropicalis</td><td rowspan=1 colspan=1>C. neoformans</td><td rowspan=1 colspan=1>C. glabrata</td><td rowspan=1 colspan=1>C. guilliermondii</td><td rowspan=1 colspan=1>C. kefyr</td></tr><tr><td rowspan=1 colspan=1>TOn= 20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>7 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>14 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 daysn=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 16. Ring Positive and 12 Hours Post-Ring Positive Stability Storage Results Summary   

<table><tr><td rowspan=2 colspan=1>Time Point</td><td rowspan=1 colspan=2>Target (% Positive)</td></tr><tr><td rowspan=1 colspan=1>C. neoformans</td><td rowspan=1 colspan=1>C. guilliermondii</td></tr><tr><td rowspan=1 colspan=1>Ring Positiven= 20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>12 hrs PostRing Positiven=10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

The results of the sample stability study support the positive blood culture stability claims for the LIAISON PLEX BCY assay at the following conditions:

• Up to 48 hours at $3 3 { - } 3 7 ^ { \circ } \mathrm { C }$ Up to 14 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ or $\mathrm { \leq } \mathrm { - } 7 0 \mathrm { ‰}$ Samples can also be tested when incubated up to 12 hours after ring positivity in a continuously monitoring blood culture device.

# c. Freeze-Thaw Study

The performance of the LIAISON PLEX BCY assay with fresh and frozen samples was evaluated by testing ten organisms at the following concentrations and replicates in negative blood matrix: three replicates at ring positive (RP), three replicates at ring

positive $+ 8$ hours, and six replicates at ring positive -10-fold dilution (representative of a near-LoD sample concentration). Representative organisms were selected to encompass all three PCR pools of the LIAISON PLEX BCY assay cartridge and represented the majority of the reportable targets detectable by the assay. Each target organism was evaluated at three conditions, fresh, first freeze thaw, and second freeze thaw. The positive agreement with the fresh condition at each freeze-thaw cycle demonstrated $100 \%$ positive agreement at each condition (Table 17).

Table 17. Freeze-Thaw Contrived Sample Stability Results Summary   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Testing Condition</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>Target Positivity</td></tr><tr><td rowspan=9 colspan=1>Candida auris</td><td rowspan=1 colspan=1>RP Fresh1</td><td rowspan=3 colspan=1>1.45E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh1</td><td rowspan=3 colspan=1>5.30E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Fresh</td><td rowspan=3 colspan=1>2.90E+06</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=9 colspan=1>Candidadubliniensis</td><td rowspan=1 colspan=1>RP Fresh1</td><td rowspan=3 colspan=1>3.20E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh1</td><td rowspan=3 colspan=1>4.20E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Fresh</td><td rowspan=3 colspan=1>1.16E+06</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Frozen 1 st thawed</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=9 colspan=1>Candidaguilliermondii</td><td rowspan=1 colspan=1>RP Fresh1</td><td rowspan=3 colspan=1>3.60E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh1</td><td rowspan=3 colspan=1>3.40E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Fresh</td><td rowspan=3 colspan=1>3.10E+06</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>RP -10 fold Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=5 colspan=1>Candidahaemulonii</td><td rowspan=1 colspan=1>RP Fresh1</td><td rowspan=3 colspan=1>9.70E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP Frozen 2nd thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh1</td><td rowspan=2 colspan=1>4.30E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen 1st thawed</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Testing Condition</td><td colspan="2" rowspan="1">Concentration(CV/mL)</td><td colspan="1" rowspan="1">Target Positivity</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="2" rowspan="2">1.29E+06</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="9">Candida kefyr</td><td colspan="1" rowspan="1">RP Fresh1</td><td colspan="2" rowspan="2">1.20E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 1st thawed</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh1</td><td colspan="2" rowspan="3">2.34E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="2" rowspan="2">4.20E+05</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="9">Candida krusei</td><td colspan="1" rowspan="1">RP Fresh1</td><td colspan="2" rowspan="2">1.24E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh1</td><td colspan="2" rowspan="2">4.00E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 1 st thawed</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="2" rowspan="3">7.20E+05</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1 thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="9">Candidalusitaniae</td><td colspan="1" rowspan="1">RP Fresh1</td><td colspan="2" rowspan="3">2.23E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 1 st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh1</td><td colspan="2" rowspan="2">1.32E+08</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="2" rowspan="2">4.00E+06</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1st thawed</td><td colspan="2" rowspan="2">4.00E+06</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="9">Candidaparapsilosis</td><td colspan="1" rowspan="1">RP Fresh1</td><td colspan="2" rowspan="3">1.54E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh1</td><td colspan="2" rowspan="2">3.90E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 1 thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="2" rowspan="2">6.10E+05</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1t thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Testing Condition</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">Target Positivity</td></tr><tr><td colspan="1" rowspan="9">Candidatropicalis</td><td colspan="1" rowspan="1">RP Fresh1</td><td colspan="1" rowspan="3">1.20E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 1 st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh1</td><td colspan="1" rowspan="3">3.30E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="1" rowspan="3">7.00E+05</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="9">Cryptococcusneoformans</td><td colspan="1" rowspan="1">RP Fresh1</td><td colspan="1" rowspan="3">3.40E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 1 st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh1</td><td colspan="1" rowspan="3">5.00E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Fresh</td><td colspan="1" rowspan="3">2.70E+05</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 1st thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">RP -10 fold Frozen 2nd thawed</td><td colspan="1" rowspan="1">100% (6/6)</td></tr></table>

Results from the freeze-thaw study are acceptable and demonstrate that the LIAISON PLEX BCY assay can detect organisms in samples that have been frozen and thawed up to two times.

# d. Carryover

An analytical study was performed to assess potential carryover or cross-contamination in the LIAISON PLEX BCY assay by testing high positive and negative samples in an alternating pattern on the LIAISON PLEX instrument. The high positive sample consisted of Candida lusitaniae grown to ring positive $^ +$ eight hours at a concentration of $1 . 3 \mathrm { E } { + } 0 8 $ CFU/mL; the negative control samples were negative blood matrix. The study was performed by two operators, each using one of two LIAISON PLEX instruments, each containing six modules. Alternating high positive and negative samples were evaluated per the format summarized in Table 18. A total of 30 high positive and 30 negative samples were evaluated. All high positive samples produced positive results, while all negative samples were negative. These results support there is no evidence of carryover from samples tested with the LIAISON PLEX BCY assay.

Table 18. Carryover Study Design   

<table><tr><td colspan="1" rowspan="2">Operator</td><td colspan="1" rowspan="2">LIAISONPLEXInstrument</td><td colspan="1" rowspan="2">TestRound</td><td colspan="6" rowspan="1">Module</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr></table>

6. Detection Limit:

a. Limit of Detection

A limit of detection (LoD) study was performed to assess the analytical sensitivity of the LIAISON PLEX BCY assay. Two strains for each of the 14 target organisms were evaluated in negative blood matrix. The preliminary LoD for each strain was determined using a 6-point 3-fold dilution series and four replicates of each dilution. The preliminary LoD for each target was defined as the lowest concentration at which $100 \%$ of replicates were positive for the intended target. The LoD was confirmed using 20 replicates of a 3- point 3-fold dilution series around the preliminary LoD for each strain. The confirmed LoD was defined as the lowest concentration at which $2 9 5 \%$ (19/20) of replicates tested positive for the intended reportable target. If $2 9 5 \%$ (19/20) of replicates at a particular concentration tested positive, a sample dilution with a three-fold lower concentration was tested until the detection rate was ${ < } 9 5 \%$ . The confirmed LoDs for the LIAISON PLEX BCY assay targets are presented in Table 19.

Table 19. Limit of Detection Results Summary   

<table><tr><td rowspan=1 colspan=1>Reported Target</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source &amp; ID</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>Candida albicans</td><td rowspan=2 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 10231</td><td rowspan=1 colspan=1>3.14E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 14053</td><td rowspan=1 colspan=1>2.83E+05</td></tr><tr><td rowspan=2 colspan=1>Candida auris</td><td rowspan=2 colspan=1>Candida auris</td><td rowspan=1 colspan=1>CBS-KNAW10913</td><td rowspan=1 colspan=1>3.50E+03</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW12766</td><td rowspan=1 colspan=1>3.50E+03</td></tr><tr><td rowspan=2 colspan=1>Candida dubliniensis</td><td rowspan=2 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>ATCC-MYA578</td><td rowspan=1 colspan=1>3.16E+03</td></tr><tr><td rowspan=1 colspan=1>ATCC-MYA577</td><td rowspan=1 colspan=1>3.16E+03</td></tr><tr><td rowspan=2 colspan=1>Candida famata</td><td rowspan=2 colspan=1>Candida famata</td><td rowspan=1 colspan=1>ATCC 20278</td><td rowspan=1 colspan=1>7.77E+02</td></tr><tr><td rowspan=1 colspan=1>ATCC 60229</td><td rowspan=1 colspan=1>6.99E+03</td></tr><tr><td rowspan=2 colspan=1>Candida glabrata</td><td rowspan=2 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15545</td><td rowspan=1 colspan=1>9.99E+03</td></tr><tr><td rowspan=1 colspan=1>ATCC 15126</td><td rowspan=1 colspan=1>1.00E+04</td></tr><tr><td rowspan=2 colspan=1>Candida guilliermondii</td><td rowspan=2 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>ATCC 22017</td><td rowspan=1 colspan=1>3.33E+03</td></tr><tr><td rowspan=1 colspan=1>ATCC 34134</td><td rowspan=1 colspan=1>1.00E+04</td></tr><tr><td rowspan=2 colspan=1>Candida haemulonii /duobushaemulonii</td><td rowspan=1 colspan=1>Candida haemulonii</td><td rowspan=1 colspan=1>CBS-KNAW7375</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=1 colspan=1>Candidaduobushaemulonii</td><td rowspan=1 colspan=1>CBS-KNAW7798</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=2 colspan=1>Candida kefyr</td><td rowspan=2 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>ATCC 8553</td><td rowspan=1 colspan=1>3.33E+03</td></tr><tr><td rowspan=1 colspan=1>ATCC 4135</td><td rowspan=1 colspan=1>3.33E+03</td></tr><tr><td rowspan=2 colspan=1>Candida krusei</td><td rowspan=2 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 6258</td><td rowspan=1 colspan=1>9.83E+03</td></tr><tr><td rowspan=1 colspan=1>ATCC 28870</td><td rowspan=1 colspan=1>2.95E+04</td></tr><tr><td rowspan=2 colspan=1>Candida lipolytica</td><td rowspan=2 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>ATCC 20460</td><td rowspan=1 colspan=1>3.18E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 20177</td><td rowspan=1 colspan=1>3.19E+04</td></tr><tr><td rowspan=2 colspan=1>Candida lusitaniae</td><td rowspan=2 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>ATCC 42720</td><td rowspan=1 colspan=1>1.11E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 34449</td><td rowspan=1 colspan=1>3.33E+04</td></tr><tr><td rowspan=2 colspan=1>Candida parapsilosis</td><td rowspan=2 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 28474</td><td rowspan=1 colspan=1>2.85E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 28475</td><td rowspan=1 colspan=1>9.50E+03</td></tr><tr><td rowspan=2 colspan=1>Candida tropicalis</td><td rowspan=2 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 13803</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=1 colspan=1>ATCC 201380</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=2 colspan=1>Cryptococcus neoformans/ gattii</td><td rowspan=1 colspan=1>Cryptococcusneoformans</td><td rowspan=1 colspan=1>ATCC 208821</td><td rowspan=1 colspan=1>9.01E+04</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC-MYA4871</td><td rowspan=1 colspan=1>9.00E+04</td></tr></table>

# b. Analytical Reactivity (Inclusivity)

The inclusivity of the LIAISON PLEX BCY assay was evaluated using contrived samples containing fungal isolates that were selected to represent phylogenetic, geographic, and temporal diversity of the organisms detected by the panel. The inclusivity panel was comprised of 80 strains representing all 14 reportable targets. Assay performance at bacterial concentrations representative of bottle positivity (ring positive) was assessed with five strains. Bottle positivity concentrations for each strain were determined in the Growth & Detection study (see section 6.d.). The inclusivity panel strains and the concentration for which they were detected are presented in Table 20.

Table 20. Inclusivity Results Summary   

<table><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">Organism Genus andSpecies</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">TargetPositivity</td></tr><tr><td colspan="1" rowspan="5">Candida albicans</td><td colspan="1" rowspan="5">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td><td colspan="1" rowspan="1">4.4E+05</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="1" rowspan="1">4.4E+05</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 90029</td><td colspan="1" rowspan="1">4.4E+05</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4780</td><td colspan="1" rowspan="1">4.4E+05</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801504</td><td colspan="1" rowspan="1">4.4E+05</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida auris</td><td colspan="1" rowspan="5">Candida auris</td><td colspan="1" rowspan="1">CBS-KNAW 12373</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 12767</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 12768</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CDC AR-0383</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 804386</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida dubliniensis</td><td colspan="1" rowspan="5">Candida dubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-582</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-579</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 44508</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-2975</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801915</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida famata</td><td colspan="1" rowspan="5">Debaryomyces hanseniil</td><td colspan="1" rowspan="1">ATCC 9365</td><td colspan="1" rowspan="1">3.2E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 10619</td><td colspan="1" rowspan="1">3.2E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 36239</td><td colspan="1" rowspan="1">3.2E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 1961</td><td colspan="1" rowspan="1">3.2E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 14266</td><td colspan="1" rowspan="1">3.2E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida glabrata</td><td colspan="1" rowspan="5">Candida glabrata</td><td colspan="1" rowspan="1">ATCC MYA-2950</td><td colspan="1" rowspan="1">3.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 34138</td><td colspan="1" rowspan="1">3.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 66032</td><td colspan="1" rowspan="1">3.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 90876</td><td colspan="1" rowspan="1">3.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801535</td><td colspan="1" rowspan="1">3.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida guilliermondii</td><td colspan="1" rowspan="4">Meyerozyma guilliermondii</td><td colspan="1" rowspan="1">ATCC 56822</td><td colspan="1" rowspan="1">1.9E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 14242</td><td colspan="1" rowspan="1">1.9E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 6260</td><td colspan="1" rowspan="1">1.9E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 90197</td><td colspan="1" rowspan="1">1.9E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">Zeptometrix 801602</td><td colspan="1" rowspan="1">1.9E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="10">Candidahaemulonii/duobushaemulonii</td><td colspan="1" rowspan="1">Candida haemulonii</td><td colspan="1" rowspan="1">CDC AR-0395</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="2">Candida haemulonis</td><td colspan="1" rowspan="1">CBS-KNAW 10970</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 10973</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="2">Candida haemulonii var.vulnera</td><td colspan="1" rowspan="1">CBS-KNAW 12437</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 12438</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida duobushaemulonii</td><td colspan="1" rowspan="1">CBS-KNAW 6915</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 7098</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CDC AR-0392</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CDC AR-0394</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CDC AR-0391</td><td colspan="1" rowspan="1">9.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="2">Candida kefyr</td><td colspan="1" rowspan="2">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 2512</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 66028</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">ATCC 204093</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 4135</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801837</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida krusei</td><td colspan="1" rowspan="5">Candida krusei</td><td colspan="1" rowspan="1">ATCC 32196</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 14243</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 22985</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 34135</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801536</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida lipolytica</td><td colspan="1" rowspan="1">Candida lipolytica</td><td colspan="1" rowspan="1">ATCC 8662</td><td colspan="1" rowspan="1">2.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="4">Yarrowia lipolytica</td><td colspan="1" rowspan="1">ATCC 201242</td><td colspan="1" rowspan="1">2.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 90812</td><td colspan="1" rowspan="1">2.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-2613</td><td colspan="1" rowspan="1">2.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 32338</td><td colspan="1" rowspan="1">2.4E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida lusitaniae</td><td colspan="1" rowspan="3">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC 200953</td><td colspan="1" rowspan="1">2.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">CBS-KNAW 4870</td><td colspan="1" rowspan="1">2.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 66035</td><td colspan="1" rowspan="1">2.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="2">Clavispora lusitaniae</td><td colspan="1" rowspan="1">CBS-KNAW 6936</td><td colspan="1" rowspan="1">2.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801603</td><td colspan="1" rowspan="1">1.9E+072</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida parapsilosis</td><td colspan="1" rowspan="5">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 34136</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 96137</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 200954</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 38291</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801537</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Candida tropicalis</td><td colspan="1" rowspan="5">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC MYA-2734</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 1369</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 90874</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC 18807</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801538</td><td colspan="1" rowspan="1">1.2E+07</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="10">Cryptococcus neoformans/gattii</td><td colspan="1" rowspan="1">Cryptococcus neoformansvar. grubii</td><td colspan="1" rowspan="1">ATCC 14116</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="4">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC 90112</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4564</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4566</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801803</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="5">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC MYA-4563</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4873</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4560</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4094</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 801917</td><td colspan="1" rowspan="1">1.8E+06</td><td colspan="1" rowspan="1">100% (3/3)</td></tr></table>

1 Debaryomyces hansenii is a synonymous taxon name for Candida famata. 2 The stock concentration of Clavispora lusitaniae (Zeptometrix 801603) was not concentrated enough to achieve the representative bottle positivity concentration. This preparation was prepared as stock material diluted 1:1 in negative blood matrix which is allowed per protocol directive.

The results from this study demonstrate the LIAISON PLEX BCY assay is capable of detecting multiple clinically relevant strains of each target organism.

# c. In silico Analytical Reactivity (Inclusivity)

The inclusivity of the LIAISON PLEX BCY assay was further assessed using in silico analysis of the oligonucleotides (i.e., forward primer(s), reverse primer(s), capture and mediator probe(s)) for all assay targets in relation to sequences available in the NCBI GenBank nt database. Sequence alignments were generated using a publicly available sequence alignment program. The in silico analysis determined the percent homology of each oligo sequence to its binding region on each target sequence retrieved from public databases. A match (i.e., predicted reactivity) was based on the following criteria, $5 9 0 \%$ homology between the oligo and reference sequences. To determine oligo homology, the highest percent homology of each oligo in the same component was assessed and then the lowest percent identity of the four components (forward primer, reverse primer, mediator, and capture) was used to characterize the inclusivity of the sequence. A summary of insilico analysis results is provided below in Table 21.

d.   
Table 21. In silico Inclusivity Results Summary   

<table><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>Inclusive Organism/Target</td><td rowspan=1 colspan=1>Total #Sequences inAlignment</td><td rowspan=1 colspan=1># Sequences withPercent OligoIdentity ≥ 90%</td><td rowspan=1 colspan=1>PredictedInclusivityPercentage (%)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>181 *</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>286</td><td rowspan=1 colspan=1>286</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candidadubliniensis</td><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>13 *</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>32 *</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candidaguilliermondii</td><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>24 *</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>Candidahaemulonii/duobushamulonii</td><td rowspan=1 colspan=1>Candida haemulonii</td><td rowspan=1 colspan=1>17*</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida duobushamulonii</td><td rowspan=1 colspan=1>9*</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>30 *</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>48 *</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>82 *</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>75 *</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candidaparapsilosis</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>59 *</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>86 *</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=2 colspan=1>Cryptococcusneoformans/gattii</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>3243</td><td rowspan=1 colspan=1>3243</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>100</td></tr></table>

\* Includes WGS (whole genome shotgun) sequences.

All 80 strains evaluated by laboratory testing were detected with $100 \%$ positivity. Insilico analyses performed demonstrate the LIAISON PLEX BCY assay exhibits $2 9 9 \%$ inclusivity to sequences available in the GenBank nt database as of December 30, 2023. The results of the analytical reactivity/inclusivity studies are acceptable.

# d. Growth and Detection Study

A growth and detection study was conducted to establish the range of expected organism concentrations present in incubated blood cultures at bottle positivity (i.e., ring positive) and eight hours after ring positive. Fourteen organisms representing the LIAISON PLEX BCY assay reportable targets were cultivated to ring positive and eight hours after ring positivity using a continuous monitoring blood culture system. Three unique blood bottles were used to assess ring positive growth and ring positive $^ +$ eight hours for a total of six bottles per organism. For each growth condition, all three blood bottles were removed from the incubator within two hours of being identified as ring positive or between 8 and 12 hours after Ring Positivity and counted in CFU/mL. In addition to $\mathrm { C F U / m L }$ quantitation, a minimum of one ring positive bottle and one ring positive $^ +$ eight hours bottle from each target organism were tested in triplicate on the LIAISON PLEX BCY assay. For several organisms a minimum of one ring positive bottle and one ring positive $^ +$ eight hours bottle was frozen at $- 8 0 ^ { \circ } \mathrm { C }$ prior to testing on the LIAISON PLEX BCY assay. Results from the growth and detection study are presented in Table 22 and represent approximate organism levels that may be observed in a clinical setting. All quantified bottle positivity concentrations are equivalent or greater than the established limit of detection for each the targets of the LIAISON PLEX BCY assay and the results are acceptable.

Table 22. Ring Positivity Growth and Detection Results Summary   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">RepresentativeStrain</td><td colspan="1" rowspan="1">Testing Condition</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">TargetPositivity</td></tr><tr><td colspan="1" rowspan="6">Candida albicans</td><td colspan="1" rowspan="6">Candidaalbicans</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">2.21E+04</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">4.30E+05</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">8.60E+05</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">1.83E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">1.54E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">6.90E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida auris</td><td colspan="1" rowspan="6">Candida auris</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.88E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.45E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">2.02E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">5.30E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">4.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">4.80E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candidadubliniensis</td><td colspan="1" rowspan="6">Candidadubliniensis</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.47E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.98E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">3.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">5.10E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="4">Candida famata</td><td colspan="1" rowspan="4">Debarymoycesfabryi</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">7.30E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">4.90E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">3.20E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">1.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">1.80E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">5.90E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida glabrata</td><td colspan="1" rowspan="6">Candidaglabrata</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">8.00E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">3.80E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">6.10E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">1.44E+08</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">1.36E+08</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">1.60E+08</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candidaguilliermondii</td><td colspan="1" rowspan="6">Meyerozymaguilliermondii</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.91E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">3.60E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.86E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.60E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.80E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.40E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candidahaemuloniilduobushaemulonii</td><td colspan="1" rowspan="6">Candidahaemulonis var.haemulonis</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">9.70E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">9.40E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.07E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.50E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.30E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida kefyr</td><td colspan="1" rowspan="6">Candida kefyr</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">8.20E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">6.30E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">2.34E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">2.07E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">2.15E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida krusei</td><td colspan="1" rowspan="6">Candida krusei</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.24E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">1.66E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">2.09E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.00E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.50E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida lipolytica</td><td colspan="1" rowspan="6">Yarrowialipolytica</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">2.44E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">2.50E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">2.42E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">3.90E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.60E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.80E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida lusitaniae</td><td colspan="1" rowspan="6">Candidalusitaniae</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">2.23E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">5.10E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Frozen</td><td colspan="1" rowspan="1">3.50E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">1.33E+08</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">1.22E+08</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">1.32E+08</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candidaparapsilosis</td><td colspan="1" rowspan="6">Candidaparapsilosis</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.54E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.61E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.76E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">4.10E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.90E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">5.50E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Candida tropicalis</td><td colspan="1" rowspan="6">Candidatropicalis</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.31E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.20E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.35E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.00E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.90E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">3.30E+07</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Cryptococcusneoformans/gattii</td><td colspan="1" rowspan="6">Cryptococcusneoformans var.grubii</td><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">1.84E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">3.40E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP Fresh</td><td colspan="1" rowspan="1">2.09E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Fresh</td><td colspan="1" rowspan="1">5.00E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">5.50E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RP + 8 Frozen</td><td colspan="1" rowspan="1">4.40E+06</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Negative BloodMatrix</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0%</td></tr></table>

RP - Ring Positive; $\mathrm { R P } + 8$ - Ring Positive $+ 8$ hours

# 7. Assay Cut-Off:

The LIAISON PLEX BCY assay determines the presence/absence of each target analyte (and control) by comparing the mean signal intensity of capture spots to pre-defined thresholds. In addition to the capture spot thresholds, a set of assay-dependent parameters are used to evaluate the validity of the run. To determine the threshold and parameter values, a sequential, multi-step approach was employed.

In the first step, a threshold training set consisting of 1293 samples from analytical testing was used to estimate the thresholds and assay-dependent parameters. Proposed threshold and assay-dependent parameter values were determined by their ability to distinguish between true and false target calls while minimizing the no call rate. The proposed thresholds were determined by the negative signal distribution, where no target is present in the samples.

In the second step of threshold determination, the performance of the proposed parameter and threshold values from step 1 were evaluated using the threshold test set, which consisted of 92 samples from analytical testing which did not overlap with the threshold training set. The true call vs. no call rate for performance parameters and controls, and reportable target

sensitivity and specificity were tabulated. Final assay thresholds were established using ROC analysis.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable

2. Matrix Comparison:

Bottle Type Equivalency Study

This study was designed to evaluate equivalence of 12 different blood culture bottle matrices from three different blood culture systems on LIAISON PLEX BCY assay performance. Ten percent of bottles from two lots of each bottle type were screened for contamination prior to inclusion in the study. After screening, each bottle matrix was inoculated with either of two positive contrived panels, bacterial strains (Acinetobacter baumannii, Escherichia coli, Klebsiella pneumonia, Staphylococcus aureus, and Pseudomonas aeruginosa; selected for their clinical prevalence in blood infections) or incubated without inoculation for 5-7 days to make negative blood matrix (NBM) (Table 23). Five replicates of each organism were tested in each of two bottle lots. Yeasts for each positive contrived sample were prepared at a concentration approximating blood culture bottle positivity in a clinical sample. Study results are presented in Table 24.

Table 23. Positive Control and Bacterial Concentrations   

<table><tr><td rowspan=1 colspan=1>Control/Sample Type</td><td rowspan=1 colspan=1>Organism Name</td><td rowspan=1 colspan=1>Concentration (CFU/mL)</td></tr><tr><td rowspan=3 colspan=1>Positive Control 1</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>4.40E+05</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>1.20E+07</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>1.84E+06</td></tr><tr><td rowspan=3 colspan=1>Positive Control 2</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>3.80E+06</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>1.86E+07</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>6.30E+06</td></tr><tr><td rowspan=1 colspan=2>All Bacteria</td><td rowspan=1 colspan=1>1.00E+08</td></tr></table>

Table 24. Sample Matrix Equivalency (Bottle Evaluation) Results Summary   

<table><tr><td colspan="1" rowspan="1">Manufacturer</td><td colspan="1" rowspan="1">BottleManufacturerPart Number</td><td colspan="1" rowspan="1">BottleName</td><td colspan="1" rowspan="1">Study Outcome</td></tr><tr><td colspan="1" rowspan="2">ThermoScientific</td><td colspan="1" rowspan="1">7106-44</td><td colspan="1" rowspan="1">VersaTrekRedox 1</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">7107-44</td><td colspan="1" rowspan="1">VersaTrekRedox 2</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="3">Becton,Dickinson, andCompany(BD)</td><td colspan="1" rowspan="1">442022</td><td colspan="1" rowspan="1">Bactec PlusAnaerobic/F</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">442024</td><td colspan="1" rowspan="1">Bactec PlusStandardAnaerobic/F</td><td colspan="1" rowspan="1">2 of 3 lots tested gave false positive results forCandida tropicalis</td></tr><tr><td colspan="1" rowspan="1">442021</td><td colspan="1" rowspan="1">BactecLytic/10Anaerobic/F</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="3">−</td><td colspan="1" rowspan="1">442023</td><td colspan="1" rowspan="1">Bactec PlusAerobic/F</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">442020</td><td colspan="1" rowspan="1">Bactec PedsPlus</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">442027</td><td colspan="1" rowspan="1">BactecStandard/10Aerobic</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="4">BioMérieux</td><td colspan="1" rowspan="1">259789</td><td colspan="1" rowspan="1">BACT/AlertSA</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">410852</td><td colspan="1" rowspan="1">BACT/AlertFN Plus</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">259790</td><td colspan="1" rowspan="1">BACT/AlertSN</td><td colspan="1" rowspan="1">No interference detected</td></tr><tr><td colspan="1" rowspan="1">410853</td><td colspan="1" rowspan="1">BACT/AlertPF Plus</td><td colspan="1" rowspan="1">No interference detected</td></tr></table>

Eleven bottle types tested demonstrated no detectable interference for any of the targets evaluated. Two lots of the Bactec Plus Standard Anaerobic/F bottle type gave false positive results for Candida tropicalis during the screen phase of the study. An additional lot was screened, and negative results were observed for all on-panel organisms, but showed a low, sub-threshold signal for $C .$ . tropicalis. Subsequent testing in the bottle type equivalency study showed $C .$ . tropicalis contamination was detected in negative blood matrix testing (1/5 replicates). No growth occurred on the check plates and root cause analysis identified the bottle media was contaminated with $C .$ . tropicalis nucleic acids. The 12 bottle types evaluated in the bottle type equivalency study demonstrated correct positive and negative agreement with the exception of the Bactec Plus Standard Anaerobic/F bottle type which resulted in one C. tropicalis false positive. The results of this study are acceptable.

# C Clinical Studies:

# 1. Prospective Clinical Study:

The clinical performance of the LIAISON PLEX BCY assay to detect and identify target fungal pathogens from blood samples identified as 1) positive by a continuous monitoring blood culture system, and 2) a Gram stain indicative of a fungal organism, was assessed in a multi-site study using prospective samples. Device testing occurred at four geographically diverse sites within the U.S. representative of the clinical laboratory intended use environment. Prospective samples meeting study inclusion criteria were collected between June 2023 and October 2023. Samples were stored either refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ if tested within 24 hours, or frozen $( { \leq } - 7 0 ^ { \circ } \mathrm { C } )$ if not tested within 24 hours of collection. Due to the low prevalence of prospective positive fungal blood cultures, the study was supplemented with retrospective and contrived samples. LIAISON PLEX BCY assay clinical performance was compared with standard of care culture followed by MALDI-TOF mass spectrometry results. Discordant results analysis for applicable targets were performed by an FDA-cleared molecular assay testing and reported as footnotes in the clinical performance tables.

A total of 3447 prospective blood samples identified as positive by a continuously monitored blood culture system. Of these, 70 samples were identified as exhibiting yeast morphology following Gram stain and were enrolled in the study. One sample was excluded following enrollment due to inconclusive Gram stain results leaving 69 clinical samples for subsequent analysis of the prospective clinical data set. Prospective samples were tested either fresh

(Category I samples) (38/69, $5 5 . 1 \%$ or frozen (Category II samples) (31/69, $4 4 . 9 \%$ ). Patient demographic information for the 69 prospective samples is presented in Table 25.

Table 25. Demographic Data for Prospectively Collected Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>s. Delloglapll</td><td rowspan=1 colspan=1>IC Data Tol 1</td><td rowspan=1 colspan=1>rospeetlvely</td><td rowspan=1 colspan=1>Colcctca spe</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Site 01</td><td rowspan=1 colspan=1>Site 02</td><td rowspan=1 colspan=1>Site 03</td><td rowspan=1 colspan=1>Site 04</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>24(57.1%)</td><td rowspan=1 colspan=1>6(54.5%)</td><td rowspan=1 colspan=1>5(45.5%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>36(52.2%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>17(40.5%)</td><td rowspan=1 colspan=1>5(45.5%)</td><td rowspan=1 colspan=1>6(54.5%)</td><td rowspan=1 colspan=1>4(80.0%)</td><td rowspan=1 colspan=1>32(46.4%)</td></tr><tr><td rowspan=1 colspan=1>Gender Unknown</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>5(100%)</td><td rowspan=1 colspan=1>69(100%)</td></tr><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>2(4.8%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>3(4.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(9.1%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>21(50.0%)</td><td rowspan=1 colspan=1>9(81.8%)</td><td rowspan=1 colspan=1>8(72.7%)</td><td rowspan=1 colspan=1>3(60.0%)</td><td rowspan=1 colspan=1>41(59.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>18(42.9%)</td><td rowspan=1 colspan=1>1(9.1%)</td><td rowspan=1 colspan=1>3(27.3%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>23(33.3%)</td></tr><tr><td rowspan=1 colspan=1>Age Unknown</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>5(100%)</td><td rowspan=1 colspan=1>69(100%)</td></tr><tr><td rowspan=1 colspan=1>Subject Status</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Emergency Room</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(9.1%)</td><td rowspan=1 colspan=1>4(36.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>5(7.2%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>10(90.9%)</td><td rowspan=1 colspan=1>7(63.6%)</td><td rowspan=1 colspan=1>5(100%)</td><td rowspan=1 colspan=1>22(31.9%)</td></tr><tr><td rowspan=1 colspan=1>Status Unknown</td><td rowspan=1 colspan=1>42(100%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>42(60.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>5(100%)</td><td rowspan=1 colspan=1>69(100%)</td></tr><tr><td rowspan=1 colspan=1>Blood CultureBottle type</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>BD BACTEC LyticAnaerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>3(27.3%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>3(4.3%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTEC PlusAerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>8(72.7%)</td><td rowspan=1 colspan=1>6(54.5%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>14(20.3%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTECStandard Aerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>5(45.5%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>5(7.2%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT FAPlus</td><td rowspan=1 colspan=1>40(95.2%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>40(58.0%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT FNPlus</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT PFPlus</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>2(2.9%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT SAStandard Aerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>3(60.0%)</td><td rowspan=1 colspan=1>3(4.3%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT SNStandard Anaerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>Bottle Type Total</td><td rowspan=1 colspan=1>42(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>11(100%)</td><td rowspan=1 colspan=1>5(100%)</td><td rowspan=1 colspan=1>69(100%)</td></tr></table>

Of the 69 clinical samples included in the prospective study analysis, 67 $( 9 7 . 1 \% )$ generated valid LIAISON PLEX BCY assay results (i.e., Detected or Not Detected) on the first attempt. Two samples $( 2 . 9 \% )$ with initial invalid results generated valid LIAISON PLEX BCY assay results after a single retest for a final success rate of $100 \%$ (69/69).

The positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result was defined as a sample where the detected LIAISON PLEX BCY assay result matched the detected comparator method result, while a TN result was one where a negative LIAISON PLEX BCY assay result matched a negative comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated. Table 26 provides a summary of the LIAISON PLEX BCY assay prospective clinical study performance.

Table 26. Prospective Sample Clinical Performance of the LIAISON PLEX BCY Assay   

<table><tr><td colspan="2" rowspan="2">Pathogen Target</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP /(TP+FN)</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="3">Candida albicans</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">72.2%-100%</td><td colspan="1" rowspan="1">27/28</td><td colspan="1" rowspan="1">96.4%</td><td colspan="1" rowspan="1">82.3%-99.4%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">64.6%-100%</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">86.2%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.6%-100%</td><td colspan="1" rowspan="1">51/521</td><td colspan="1" rowspan="1">98.1%</td><td colspan="1" rowspan="1">89.9%-99.7%</td></tr><tr><td colspan="1" rowspan="3">Candida auris</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">43.9%-100%</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.1%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7%-100%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.6%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">51.0%-100%</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.4%-100%</td></tr><tr><td colspan="1" rowspan="3">Candidadubliniensis</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.0%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida famata</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.0%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida glabrata</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74.1%-100%</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">87.5%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">78.5%-100%</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.6%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">86.7%-100%</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92.0%-100%</td></tr><tr><td colspan="1" rowspan="3">Candidaguilliermondii</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.0%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida haemulonii/Candidaduobushaemulonii</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.0%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida kefyr</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7%-100%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.6%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7%-100%</td><td colspan="1" rowspan="1">68/68</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida krusei</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.2%-100%</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.4%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7%-100%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.6%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">43.9%-100%</td><td colspan="1" rowspan="1">66/66</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.5%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida lipolytica</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.0%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr><tr><td colspan="1" rowspan="3">Candida lusitaniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7%-100%</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.6%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7%-100%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.6%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.2%-100%</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.6%-100%</td></tr><tr><td colspan="1" rowspan="3">Candidaparapsilosis</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">70.1%-100%</td><td colspan="1" rowspan="1">28/29</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">82.8%-99.4%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.2%-100%</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.3%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74.1%-100%</td><td colspan="1" rowspan="1">57/582</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">90.9%-99.7%</td></tr><tr><td colspan="1" rowspan="3">Candida tropicalis</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34.2%-100%</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">85.8%-99.5%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">51.0%-100%</td><td colspan="1" rowspan="1">25/27</td><td colspan="1" rowspan="1">92.6%</td><td colspan="1" rowspan="1">76.6%-97.9%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">61.0%-100%</td><td colspan="1" rowspan="1">60/633</td><td colspan="1" rowspan="1">95.2%</td><td colspan="1" rowspan="1">86.9%-98.4%</td></tr><tr><td colspan="1" rowspan="2">Cryptococcusneformans/</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.8%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.0%-100%</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.7%-100%</td></tr></table>

1 The one Candida albicans False Positive sample was also positive by an FDA-cleared molecular assay. 2 The one Candida parapsilosis False Positive sample was also positive by an FDA-cleared molecular assay. 3 One out of the three Candida tropicalis False Positive samples was also positive by an FDA-cleared molecular assay.

The PPA of the LIAISON PLEX BCY assay was $100 \%$ across all analytes in prospectively collected samples. The limited number of positive blood culture samples which contained a fungal organism as determined by Gram stain required supplementation with archived (Category III samples) and contrived (Category IV samples) to obtain more robust performance estimates. The NPA for all organisms was $59 8 \%$ in prospectively collected samples with the exception of Candida tropicalis which had an NPA of $9 5 . 2 \%$ . Discordant analysis testing provided evidence for the presence of $C .$ . tropicalis in one out of three $C .$ . tropicalis false positive results.

The LIAISON PLEX BCY assay reported multiple organism detections (co-infections) in a total of six prospective samples. This represents $8 . 7 \%$ (6/69) of all prospective samples. All six co-infections contained two organisms. Out of the six samples with multiple detections, $1 6 . 7 \%$ (1/6) was concordant with the reference method and $8 3 . 3 \%$ (5/6) contained one organism that was not detected by the reference method (Table 27).

Table 27. Coinfections Reported in the Prospective Study   

<table><tr><td rowspan=2 colspan=1>Analyte 1</td><td rowspan=2 colspan=1>Analyte 2</td><td rowspan=1 colspan=2>Overall(N=69)</td></tr><tr><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>(%)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.45%(1/69)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.45%(1/69)</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.45%(1/69)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.90%(2/69)</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.45%(1/69)</td></tr></table>

# 2. Retrospective Sample Evaluation:

Several targets detected by the LIAISON PLEX BCA assay were not encountered during the prospective clinical study in sufficient numbers to demonstrate assay performance. Therefore, the prospective clinical study was supplemented with additional testing using retrospective positive samples characterized previously by a standard of care molecular assay. To minimize bias, negative retrospective samples $( \mathrm { N } { = } 1 3 )$ ) were tested concurrently with positive retrospective samples $( \mathrm { N } { = } 5 0 )$ ) obtained from six sources within the U.S. for a total of 63 retrospective samples evaluated. All samples were randomized and blinded by an individual not directly involved in the study testing. A summary of the demographic information available for retrospective samples is presented in Table 28.

Table 28. Demographic Data for Retrospective Samples   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1># Samples (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>24(38.1%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>25(39.7%)</td></tr><tr><td rowspan=1 colspan=1>Gender Unknown</td><td rowspan=1 colspan=1>14(22.2%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100%)</td></tr><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>1(1.6%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>27(42.9%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>19(30.2%)</td></tr><tr><td rowspan=1 colspan=1>Age Unknown</td><td rowspan=1 colspan=1>16(25.4%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100%)</td></tr><tr><td rowspan=1 colspan=1>Subject Status</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>2(3.2%)</td></tr><tr><td rowspan=1 colspan=1>Status Unknown</td><td rowspan=1 colspan=1>61(96.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100%)</td></tr><tr><td rowspan=1 colspan=1>Blood Culture Bottle Type</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>BD BACTEC Lytic Anaerobic</td><td rowspan=1 colspan=1>2(3.2%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTEC Plus Aerobic</td><td rowspan=1 colspan=1>17(27.0%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTEC Standard Aerobic</td><td rowspan=1 colspan=1>29(46.0%)</td></tr><tr><td rowspan=1 colspan=1>Bottle Type Unknown</td><td rowspan=1 colspan=1>15(23.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100%)</td></tr></table>

The performance of the LIAISON PLEX BCY assay was determined by comparing to an FDA-cleared molecular assay for all retrospective samples and summarized in Table 29. All retrospective samples generated a valid result on the first attempt.

Table 29. Retrospective Sample Clinical Performance of the LIAISON PLEX BCY Assay   

<table><tr><td rowspan=2 colspan=1>Pathogen Target</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN /(TN+FP)</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>81.6%-100%</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>84.5%-100%</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.6%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida haemulonii /C. duobushaemulonii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7%-100%</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>61.0%-100%</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.7%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans/Cryptococcus gattii</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6%-100%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8%-100%</td></tr></table>

# 3. Contrived Sample Evaluation:

Due to the low prevalence of some of the LIAISON PLEX BCY assay target organisms encountered during prospective and retrospective sample collection, contrived samples were used to supplement clinical performance. Contrived samples were prepared from multiple strains of each of the yeast organisms detected by the assay (Table 30). Blood culture bottles containing whole blood from unique patients were inoculated with low inoculum concentrations of individual isolates and grown until flagged positive (ring positive) by a continuous monitoring blood culture system. Samples were collected within eight hours of ring positivity and stored frozen until testing. Fifty contrived samples were evaluated for each LIAISON PLEX BCY assay target. A total of 750 contrived samples and 79 negative samples were randomized, blinded, and interspersed at the four clinical sites for testing with prospective and retrospective samples.

Of the 829 contrived samples evaluated, 803 $( 9 6 . 9 \% )$ generated valid LIAISON PLEX BCY assay results on the first attempt. Invalid results were obtained for 26 $( 3 . 1 \% )$ samples at first attempt. Twenty-four of these samples generated a valid result after retest leading to a final success rate of $9 9 . 7 \%$ (827/829). The results of contrived sample testing with the LIAISON PLEX BCY assay are presented in Table 31.

Table 30. Contrived Sample Summary   

<table><tr><td colspan="1" rowspan="1">Sample Name</td><td colspan="1" rowspan="1">Strain orSource PN</td><td colspan="1" rowspan="1">Culture CollectionManufacturer</td><td colspan="1" rowspan="1"># of Samplestested</td></tr><tr><td colspan="1" rowspan="5">Candida albicans</td><td colspan="1" rowspan="1">NCCLS 11</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">3147</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AR-0761</td><td colspan="1" rowspan="2">CDC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AR-0762</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">NSBCY0205</td><td colspan="1" rowspan="1">NSBCY</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida albicans Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candida glabrata</td><td colspan="1" rowspan="1">NCCLS 84</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">NSBCY0235</td><td colspan="1" rowspan="1">North ShoreUniversity HealthSystem</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">303542</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AmMS 231</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">DLS0084334</td><td colspan="1" rowspan="1">Discovery LifeScience</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida glabrata Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candida auris</td><td colspan="1" rowspan="1">KCTC 17809</td><td colspan="1" rowspan="4">Westerdijk Institute</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">VPCI669/P/12</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 10913</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">VPCI671/P/12</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">MYA-5003</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida auris Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="3">Candidadubliniensis</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">H12</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Wu272</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">75/043</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">MAD045</td><td colspan="1" rowspan="1">Tampa GeneralHospital</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida dubliniensis Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candida famata</td><td colspan="1" rowspan="1">ATCC 60229</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 20278</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 14260</td><td colspan="1" rowspan="3">Westerdijk Institute</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 14261</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 14994</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida famata Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candidaguilliermondii</td><td colspan="1" rowspan="1">CBS 6021</td><td colspan="1" rowspan="4">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">848</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 7350,CBS 566,DBVPG 6140,IFO 10279,IGC 2730, JM1539, NRRL Y-324</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 14242</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AR-0590</td><td colspan="1" rowspan="1">CDC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida guilliermondii Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="4">Candida kefyr</td><td colspan="1" rowspan="1">[BI CZAS652/6, CCY 29-8-9</td><td colspan="1" rowspan="4">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AmMS 226</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">14 Type B</td><td colspan="1" rowspan="1">14</td></tr><tr><td colspan="1" rowspan="1">CBS 607</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="3" rowspan="1">Candida kefyr Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candida krusei</td><td colspan="1" rowspan="1">M 298</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 14243</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 749,CBS 573,CCRC 20514,IFO 1064, IFO1395, JCM1609, NBRC1395, NRRL Y-413, NRRL Y-7179</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">MAD 062</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">UTMB0817</td><td colspan="1" rowspan="1">UTMB</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="2" rowspan="1">Candida krusei Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Yarrowia /Candidalipolytica</td><td colspan="1" rowspan="1">NRRL Y-1094ATCC 9948,CCY 29-26-47,FMJ 1278, IFO1209]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">PO1f</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CP 602</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 6125</td><td colspan="1" rowspan="1">Westerdijk Institute</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 6317</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Yarrowia /Candida lipolytica</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candida lusitaniae</td><td colspan="1" rowspan="1">Y534</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AmMS 233</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">45090</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">MAD158</td><td colspan="1" rowspan="1">Tampa GeneralHospital</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 4870</td><td colspan="1" rowspan="1">Westerdijk Institute</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida lusitaniae Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candida tropicalis</td><td colspan="1" rowspan="1">API 90 01 105</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">FDA PCI M-59</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AR-0345</td><td colspan="1" rowspan="1">CDC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">MAD069.2</td><td colspan="1" rowspan="1">Tampa GeneralHHospital</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">DLS0082284</td><td colspan="1" rowspan="1">Discovery LifeScience</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida tropicalis Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Candidaparapsilosis</td><td colspan="1" rowspan="1">CDC 317</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CL524</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">UTMB074</td><td colspan="1" rowspan="1">UTMB</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AR-0339</td><td colspan="1" rowspan="1">CDC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">MAD004</td><td colspan="1" rowspan="1">Tampa GeneralHospital</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida parapsilosis Total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="5">Cryptococcus gattii</td><td colspan="1" rowspan="1">RV 48370</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">A1M R272</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">WM779</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">NR-49162</td><td colspan="1" rowspan="2">BEI</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">NR 50422</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="3">Cryptococcusneoformans</td><td colspan="1" rowspan="1">613</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">WM629</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">LACNY0148</td><td colspan="1" rowspan="1">LaboratoryAlliance of CentralNew York</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="2">Cryptococcusneoformans var.grubi</td><td colspan="1" rowspan="1">3756</td><td colspan="1" rowspan="2">ATCC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">H99</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Cryptococcus neoformans/Cryptococcus gattii Total</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="2">Candidaduobushaemulonii</td><td colspan="1" rowspan="1">CBS 7798</td><td colspan="1" rowspan="1">Westerdijk Institute</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">AR-0394</td><td colspan="1" rowspan="1">CDC</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Candida haemulonis</td><td colspan="1" rowspan="1">CBS 10970</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Candida haemuloniivar. vulnera</td><td colspan="1" rowspan="1">CBS 12436</td><td colspan="1" rowspan="2">Westerdijk Institute</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Candidapseudohaemulonii</td><td colspan="1" rowspan="1">CBS 10004</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="3" rowspan="1">Candida haemulonii / C. duobushaemulonii Total</td><td colspan="1" rowspan="1">50</td></tr></table>

Table 31. Contrived Sample Clinical Performance of the LIAISON PLEX BCY Assay   

<table><tr><td rowspan=1 colspan=1>Pathogen Target</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>TP /(TP+FN)</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>776/7771</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.7%-100%</td><td rowspan=1 colspan=1>778/778</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candidaguilliermondii</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida haemulonii /C. duobushaemulonii</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida Krusei</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9%-100%</td><td rowspan=1 colspan=1>776/7772</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3%-99.9%</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.7%-100%</td><td rowspan=1 colspan=1>775/778³3</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.9%-100%</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans/Cryptococcus gattii</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.3%-100%</td><td rowspan=1 colspan=1>727/727</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr></table>

1 The C. famata False Positive sample was also negative by an FDA-cleared molecular assay. 2 The C. parapsilosis False Positive sample was also negative by an FDA-cleared molecular assay. 3 The three C. tropicalis False Positive samples were also negative by an FDA-cleared molecular assay.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The LIAISON PLEX BCY assay prospective clinical study was conducted to evaluate clinical performance of the device using positive blood culture samples. A total of 3447 blood culture samples identified as positive by a continuous monitoring blood culture system were collected across four geographically diverse clinical sites within the U.S. between June and October of 2023. Prospective samples were tested with the LIAISON PLEX BCY assay either fresh or frozen. Of the 3447 samples, 69 samples met inclusion criteria and exhibited yeast morphology after Gram stain and were enrolled in the study. The expected values of each target organism in the 69 prospective samples are presented by site and age in Tables 32 and 33, respectively.

Table 32. Expected Values by Clinical Site (Prospective Samples)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>Site 01(N=42</td><td rowspan=1 colspan=2>Site 02(N=11)</td><td rowspan=1 colspan=2>Site 03(N=11</td><td rowspan=1 colspan=2>Site 04(N=5)</td><td rowspan=1 colspan=2>Total(N=69</td></tr><tr><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>(%)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>28.6%(12/42)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%(3/11)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%(2/11)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%(1/5)</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>26.1%(18/69)</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.8%(2/42)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%(2/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.8%(4/69)</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>35.7%(15/42)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>36.4%(4/11)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>45.5%(5/11)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%(1/5)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>36.2%(25/69)</td></tr><tr><td rowspan=1 colspan=1>Candidaguilliermondii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr><tr><td rowspan=1 colspan=1>Candida haemulonii/C. duobushaemulonii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.4%(1/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.4%(1/69)</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.1%(3/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.3%(3/69)</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.4%(1/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/11)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%(1/11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.9%(2/69)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14.3%(6/42)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%(2/11)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%(2/11)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40.0%(2/5)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>17.4%(12/69)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.1%(3/42)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%(2/11)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%(3/11)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%(1/5)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>13.0%(9/69)</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans/Cryptococcus gattii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/42)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% /11)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/1)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/5)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr></table>

Table 33. Expected Values by Age Group (Prospective Samples)   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="2" rowspan="1">0-1 year(N=3)3</td><td colspan="2" rowspan="1">&gt;2-5 years(N=0)</td><td colspan="2" rowspan="1">&gt;6-21 years(N=1)</td><td colspan="2" rowspan="1">&gt;22-65 years(N=41)</td><td colspan="2" rowspan="1">&gt; 65 years(N=23)</td><td colspan="2" rowspan="1">Unknownyears(N=1)</td><td colspan="2" rowspan="1">Overall(N=69)</td></tr><tr><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">33.3%(1/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">31.7%(13/41)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">17.4%(4/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">26.1%(18/69)</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7.3%(3/41)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.3%(1/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">5.8%(4/69)</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/69)</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/69)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">33.3%(1/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%(1/1)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">24.4%(10/41)</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">52.2%(12/23)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%(1/1)</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">36.2%(25/69)</td></tr><tr><td colspan="1" rowspan="1">Candidaguilliermondii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/69)</td></tr><tr><td colspan="1" rowspan="1">Candida haemulonii/C. duobushaemulonii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/69)</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.3%(1/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.4%(1/69)</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.9%(2/41)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.3%(1/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4.3%(3/69)</td></tr><tr><td colspan="1" rowspan="1">Candida lipolytica</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/69)</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2.4%(1/41)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.3%(1/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2.9%(2/69)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">33.3%(1/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">24.4%(10/41)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.3%(1/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">17.4%(12/69)</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">12.2%(5/41)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">17.4%(4/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">13.0%(9/69)</td></tr><tr><td colspan="1" rowspan="1">Cryptococcusneoformans/Cryptococcus gattii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/0)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/41)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/23)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/1)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/69)</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.