Luminex Corporation Sarah Herzog Sr. Regulatory Affairs Associate 4088 Commercial Avenue Northbrook, Illinois 60062

Re: K240627 Trade/Device Name: LIAISON PLEX Yeast Blood Culture Assay Regulation Number: 21 CFR 866.3365 Regulation Name: Multiplex Nucleic Acid Assay For Identification Of Microorganisms And Resistance Markers From Positive Blood Cultures Regulatory Class: Class II Product Code: PEO, NSU Dated: March 5, 2024 Received: March 6, 2024

Dear Sarah Herzog:

We have reviewed your section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Additional information about changes that may require a new premarket notification are provided in the FDA guidance documents entitled "Deciding When to Submit a 510(k) for a Change to an Existing Device" (https://www.fda.gov/media/99812/download) and "Deciding When to Submit a 510(k) for a Software Change to an Existing Device" (https://www.fda.gov/media/99785/download).

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

# Sincerely, Noel J. Gerald -S

Noel J. Gerald, Ph.D.   
Branch Chief   
Bacterial Respiratory and Medical Countermeasures Branch Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K240627

Device Name LIAISON PLEX Yeast Blood Culture Assay

Indications for Use (Describe)   
The LIAISON PLEX Yeast Blood Culture (BCY) Assay is a qualitative nucleic acid multiplex in vitro diagnostic test   
intended for use on the LIAISON PLEX System for simultaneous detection and identification of multiple potentially   
pathogenic fungal organisms in positive blood culture. The LIAISON PLEX BCY Assay is performed directly on blood   
culture samples identified as positive by a continuous monitoring blood culture system and which contain fungal   
organisms as determined by Gram Stain. The LIAISON PLEX BCY Assay detects and identifies the following fungal   
organisms:   
Candida albicans   
Candida auris   
Candida dubliniensis   
Candida famata   
Candida glabrata   
Candida guilliermondii   
Candida kefyr   
Candida krusei   
Candida lipolytica   
Candida lusitaniae   
Candida parapsilosis   
Candida tropicalis   
Candida haemulonii / duobushaemulonii   
Cryptococcus neoformans / gattii

The detection and identification of specific fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from LIAISON PLEX BCY Assay are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by LIAISON PLEX BCY Assay may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by LIAISON PLEX BCY Assay, susceptibility testing and differentiation of mixed growth) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# Luminex

complexity simplified.

# 510(k) Summary

This Summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

Preparation date: 05 March 2024

A. 510(k) Number: K240627   
B. Purpose for Submission: Traditional 510(k), New Device

# C. Measurand:

Candida albicans, Candida auris, Candida dublinensis, Candida famata, Candida glabrata, Candida guilliermondii, Candida kefyr, Candida krusei, Candida lipolytica, Candida lusitaniae, Candida parapsilosis, Candida tropicalis, Candida haemulonii/duobushaemulonii, and Cryptococcus neoformans/gatti

# D. Type of Test:

Qualitative Real Time Polymerase Chain Reaction (PCR)

# E. Applicant:

Sarah Herzog, Luminex Corporation   
4088 Commercial Avenue   
Northbrook, IL 60062   
(847) 400-9000

# F. Proprietary and Established Names:

LIAISON PLEX® Yeast Blood Culture Assay

# G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>ProductCode</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PEO</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>21 CFR 866.3365 - Multiplex Nucleic AcidAssay for Identification of Microorganismsand Resistance Markers from PositiveBlood Cultures</td><td rowspan=1 colspan=1>83 (Microbiology)</td></tr></table>

# Luminex

complexity simplified.

# H. Intended Use:

# 1. Intended use(s):

The LIAISON PLEX® Yeast Blood Culture (BCY) Assay is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on the LIAISON $\mathsf { P L E X } ^ { \otimes }$ System for simultaneous detection and identification of multiple potentially pathogenic fungal organisms in positive blood culture. The LIAISON PLEX® BCY Assay is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain fungal organisms as determined by Gram Stain.

The LIAISON PLEX® BCY Assay detects and identifies the following fungal organisms:

<table><tr><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td></tr><tr><td rowspan=1 colspan=1>Candida haemulonii/duobushaemulonii</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans/gattii</td></tr></table>

The detection and identification of specific fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from LIAISON $\mathsf { P L E X } ^ { \otimes }$ BCY Assay are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by LIAISON PLEX® BCY Assay may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by LIAISON PLEX® BCY Assay, susceptibility testing and differentiation of mixed growth) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

# 2. Indication(s) for use:

Same as intended use.

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission

3. Special conditions for use statement(s): For prescription use only. For in vitro diagnostic use only.

4. Special instrument requirements: For use with LIAISON PLEX ® Systems.

# I. Device Description:

The LIAISON PLEX® Yeast Blood Culture Assay (BCY Assay) is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system, and which contain a fungal organism, as determined by a Gram stain. The system consists of an instrument, a single-use disposable test cartridge, and a transfer pipette. The user loads the sample into the sample port of the LIAISON PLEX Yeast Blood Culture Assay Cartridge. Next, the user sets up the sample order on the LIAISON PLEX System by first entering the sample information or scanning the barcode ID located on the sample tube, then scanning the barcode ID located on the test cartridge. Last, the user inserts the test cartridge into the processing module to initiate the test. The LIAISON PLEX System identifies the assay being run and automatically initiates the proper testing protocol to process the sample, analyze the data, and generate test results.

The LIAISON PLEX System automates the BCY Assay sample analysis through the following steps: a) Sample Preparation: Nucleic acid extraction via mechanical and chemical cell lysis and magnetic bead-based nucleic acid isolation; b) Amplification: Multiplex PCR based amplification of the extracted nucleic acid to generate target specific amplicons; c) Hybridization: Amplified DNA hybridizes to specific capture DNA arrayed on a glass slide in a microarray format and the bound target DNA, in turn, hybridizes with mediator and gold-nanoparticle probes; d) Signal Analysis: Gold nanoparticle probes bound specifically to target-containing spots in the microarray are silver-enhanced, and light scatter from the spots is measured and further analyzed to determine the presence (Detected) or absence (Not Detected) of a target.

# J. Substantial Equivalence Information:

1. Predicate device name(s): GenMark ePlex Blood Culture Identification Fungal Pathogen (BCID-FP)

2. Predicate 510(k) number(s): K182690

3. Comparison with predicate:

The following tables compare Luminex’s LIAISON® Yeast Blood Culture Assay to the ePlex Blood Culture Identification Fungal Pathogen (BCID-FP).

# Luminex

complexity simplified.

Comparison to Predicate   

<table><tr><td rowspan=1 colspan=1>Comparisonto PredicateDevice</td><td rowspan=1 colspan=3>Predicate Device:GenMark ePlex Blood Culture IdentificationFungal Pathogen (BCID-FP), K182690</td><td rowspan=1 colspan=1>Candidate Device:LIAISON PLEX® Yeast Blood Culture Assay</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=3>PEO</td><td rowspan=1 colspan=1>PEO</td></tr><tr><td rowspan=1 colspan=1>Regulation Number</td><td rowspan=1 colspan=3>21 CFR 866.3365</td><td rowspan=1 colspan=1>21 CFR 866.3365</td></tr><tr><td rowspan=1 colspan=1>Organism Detected</td><td rowspan=1 colspan=3>Candida albicans, Candida auris, Candidadubliniensis, Candida famata, Candidaglabrata, Candida guilliermondii, Candidakefyr, Candida krusei, Candida lusitaniae,Candida parapsilosis, Candida tropicalis,Cryptococcus gattii, Cryptococcusneoformans, Fusarium, Rhodotorula</td><td rowspan=1 colspan=1>Candida albicans, Candida auris, Candidadublinensis,Candida famata, Candida glabrata,Candida guilliermondii, Candida kefyr,Candida krusei, Candida lipolytica,Candida lusitaniae, Candida parapsilosis,Candida tropicalis, Candida haemulonii/duobushaemulonii, Cryptococcusneoformans/gatti</td></tr><tr><td rowspan=1 colspan=1>Measurand</td><td rowspan=1 colspan=3>Nucleic acid from Organisms detected</td><td rowspan=1 colspan=1>Nucleic acid from Organisms detected</td></tr><tr><td rowspan=2 colspan=1>Intended Use</td><td rowspan=2 colspan=3>The GenMark ePlex Blood CultureIdentification Fungal Pathogen (BCID-FP)Panel is a qualitative nucleic acid multiplexin vitro diagnostic test intended for use onGenMark&#x27;s ePlex Instrument forsimultaneous detection and identification ofmultiple potentially pathogenic fungalorganisms in positive blood culture. TheePlex BCID-FP Panel is performed directly onblood culture samples identified as positiveby a continuous monitoring blood culturesystem and which contains fungalorganisms.The following fungal organisms are identifiedusing the ePlex BCID-FP Panel:Candida albicansCandida aurisCandida dubliniensisCandida famataCandida glabrataCandida guilliermondiiCandida kefyrCandida kruseiCandida lusitaniaeCandida parapsilosisCandida tropicalisCryptococcus gattiiCryptococcus neoformansFusariumRhodotorula</td><td rowspan=2 colspan=1>The LIAISON PLEX Yeast Blood Culture(BCY) Assay is a qualitative nucleic acidmultiplex in vitro diagnostic test intendedfor use on the LIAISON PLEX System forsimultaneous detection and identificationof multiple potentially pathogenic fungalorganisms in positive blood culture. TheLIAISON PLEX BCY Assay is performeddirectly on blood culture samplesidentified as positive by a continuousmonitoring blood culture system andwhich contain fungal organisms asdetermined by Gram Stain.The LIAISON PLEX BCY Assay detects andidentifies the following fungal organisms:Candida albicansCandida aurisCandida dublinensisCandida famataCandida glabrataCandida guilliermondiiCandida kefyrCandida kruseiCandida lipolyticaCandida lusitaniaeCandida parapsilosisCandida tropicalisCandida haemulonii/duobushaemuloniiCryptococcus neoformans/gatti</td></tr><tr><td rowspan=1 colspan=1>• Cryp</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=10 colspan=1></td><td rowspan=1 colspan=1>The detection and identification of specificfungal nucleic acids from individualsexhibiting signs and/or symptoms ofbloodstream infection aids in the diagnosisof bloodstream infection when used inconjunction with other clinical information.The results from the ePlex BCID-FP Panelare intended to be interpreted inconjunction with Gram stain results andshould not be used as the sole basis fordiagnosis, treatment, or other patientmanagement decisions.Negative results in the setting of asuspected bloodstream infection may bedue to infection with pathogens that are not</td><td rowspan=10 colspan=1>The detection and identification ofspecific fungal nucleic acids fromindividuals exhibiting signs and/orsymptoms of bloodstream infection aidsin the diagnostic of bloodstreaminfection when used in conjunction withother clinical information. The resultsfrom the LIAISON PLEX BCY Assay areintended to be interpreted inconjunction with Gram stain results andshould not be used as the sole basis fordiagnosis, treatment, or other patientmanagement decisions.Negative results in the setting of asuspected bloodstream infection may bedue to infection with pathogens that arenot detected by this test. Positive resultsdo not rule out co-infection with otherorganisms; the organism(s) detected byLIAISON PLEX BCY Assay may not be thedefinite cause of disease. Additionallaboratory testing (e.g. sub-culturing ofpositive blood cultures for identificationof organisms not detected by LIAISONPLEX BCY Assay, susceptibility testingand differentiation of mixed growth) andclinical presentation must be taken intoconsideration in the final diagnosis ofbloodstream infection.</td></tr><tr><td rowspan=1 colspan=1>detected by this test. Positive results do not</td></tr><tr><td rowspan=1 colspan=1>rule out co-infection with other organisms;</td><td rowspan=1 colspan=1>do not rule out co-infection with other</td></tr><tr><td rowspan=1 colspan=1>the organism(s) detected by the ePlex BCID-</td><td rowspan=1 colspan=1>organisms; the organism(s) detected by</td></tr><tr><td rowspan=1 colspan=1>FP Panel may not be the definite cause of</td><td rowspan=1 colspan=1>LIAISON PLEX BCY Assay may not be the</td></tr><tr><td rowspan=1 colspan=1>disease. Additional laboratory testing (e.g.sub-culturing of positive blood cultures for</td></tr><tr><td rowspan=1 colspan=1>identification of organisms not detected by</td></tr><tr><td rowspan=1 colspan=1>ePlex BCID-FP Panel, susceptibility testing</td><td rowspan=1 colspan=1>of organisms not detected by LIAISON</td></tr><tr><td rowspan=1 colspan=1>and differentiation of mixed growth) and</td></tr><tr><td rowspan=1 colspan=1>clinical presentation must be taken intoconsideration in the final diagnosis ofbloodstream infection.</td></tr><tr><td rowspan=1 colspan=1>Automated System(Sample to Answer)</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>ePlex Instrument</td><td rowspan=1 colspan=1>LIAISON PLEX</td></tr><tr><td rowspan=1 colspan=1>Sample Types</td><td rowspan=1 colspan=1>Blood Culture</td><td rowspan=1 colspan=1>Same</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission

# K. Standards/Guidance Documents Referenced:

Class II Special Controls Guideline: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures (May 2015)

• Electronic Submission Template for Medical Device 510(k) Submissions - Guidance for Industry and Food and Drug Administration Staff (October 2, 2023).

Content of Premarket Submissions for Device Software Functions - Guidance for Industry and Food and Drug Administration Staff (June 14, 2023).

Cybersecurity in Medical Devices: Quality System Considerations and Content of Premarket Submissions - Guidance for Industry and Food and Drug Administration Staff (September 27, 2023).

Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests - Guidance for Industry and FDA Staff (March 13, 2007).

CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition. CLSI document EP12-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

• CLSI. Interference Testing in Clinical Chemistry. 3rd ed. CLSI guideline EP07. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

• ISO 14971:2019 Medical devices - Application of risk management to medical devices

IEC 62366-1:2015 Medical devices - Part 1: Application of usability engineering to medical devices

• ISO 62304:2006 Medical device software - Software life-cycle processes

ISO 15223-1:2016: Medical Devices - Symbols to be used with medical device labels, labeling and information to be supplied - Part 1: General requirements

IEC 61010-1 Ed. 3.0 2010: Safety requirements for electrical equipment for measurement, control, and laboratory use - Part 1: General requirements

EN 61010-2-101:2002/IEC 61010-2-101:2015: Safety requirements for electrical equipment for measurement, control and laboratory use - Part 2-101: Particular requirements for in vitro diagnostic (IVD) medical equipment.

IEC 60601-1-2:2014 (Edition 4.0): Medical electrical equipment - Part 1-2: General requirements for basic safety and essential performance - Collateral Standard: Electromagnetic disturbances - Requirements and tests

ISO 13485:2016/EN ISO 13485:2016; Medical devices - Quality Management System - Requirements for regulatory purposes

ISO 20916:2019; In vitro diagnostic medical devices. Clinical performance studies using specimens from human subjects. Good study practice

EN ISO 18113-1:2011; In vitro diagnostic medical devices - Information supplied by the manufacturer (labeling). Terms, definition and general requirements

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission

EN ISO 18113-2:2011; In vitro diagnostic medical devices - Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use

EN ISO 18113-3:2011; In vitro diagnostic medical devices - Information supplied by the manufacturer (labeling) – Part 3: In vitro diagnostic instruments for professional use

• EN ISO 23640:2015; In vitro diagnostic medical devices - Evaluation of stability of in vitro diagnostic reagents

IEC 61326-1:2012; Electrical equipment for measurement control and laboratory use - EMC requirements - Part 1: General requirements

EN 61326-2-6:2006/IEC 61326-2-6:2012; Electrical equipment for measurement control and laboratory use - EMC requirements - Part 2-6: Particular requirements - In vitro diagnostic (IVD) medical equipment

# L. Test Principle:

The LIAISON PLEX® Yeast Blood Culture Assay (BCY Assay) is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system, and which contain a fungal organism, as determined by a Gram stain. The system consists of an instrument and a single-use, disposable test cartridge and a transfer pipette. The user loads the sample into the sample port of the LIAISON PLEX® Yeast Blood Culture Assay Cartridge. Next, the user sets up the sample order on the LIAISON PLEX System by first entering the sample information or scanning the barcode ID located on the sample tube, then scanning the barcode ID located on the test cartridge. Last, the user inserts the test cartridge into the processing module to initiate the test. The LIAISON PLEX® System identifies the assay being run and automatically initiates the proper testing protocol to process the sample, analyze the data, and generate test results.

The LIAISON $\mathsf { P l E X } ^ { \otimes }$ System automates the LIAISON PLEX® BCY Assay sample analysis through the following steps: a) Sample Preparation: Nucleic acid extraction via mechanical and chemical cell lysis and magnetic bead-based nucleic acid isolation; b) Amplification: Multiplex PCR based amplification of the extracted nucleic acid to generate target specific amplicons; c) Hybridization: Amplified DNA hybridizes to specific capture DNA arrayed on a glass slide in a microarray format and the bound target DNA, in turn, hybridizes with mediator and goldnanoparticle probes; d) Signal Analysis: Gold nanoparticle probes bound specifically to target-containing spots in the microarray are silver-enhanced, and light scatter from the spots is measured and further analyzed to determine the presence (Detected) or absence (Not Detected) of a target.

# M. Performance Characteristics:

1. Analytical performance:

a. Analytical Sensitivity (Limit of Detection)

A limit of detection study (LoD) was performed to evaluate the analytical sensitivity of t h e LIAISON Plex® BCY Assay. Twenty-eight (28) strains and isolates that represent the 14 reportable targets of the LIAISON Plex BCY Assay were tested individually by serially diluting each target in a simulated blood culture sample matrix. Testing was broken into two parts; Preliminary LoD and Confirmation LoD. The Preliminary LoD concentrations were evaluated using a 6 point 3-fold dilution series and testing of at least four replicates per dilution. The Preliminary LoD for each target was defined as the lowest concentration at which $100 \%$ of four replicates were positive for the intended reportable target. With the preliminary LoD results, a 3-point 3 fold dilution series was made. Twenty replicates of each applicable dilution series were tested. Two concentrations were required with 20 replicates each to help determine the confirmed LoD for each strain. The confirmed LoD for each organism was defined as the lower concentration at which $\geq 9 5 \%$ of the 20 replicates were positive for the intended reportable target. The confirmed LoD for each target tested is listed in the following Table 1.

Table 1. LIAISON® PLEX BCY Assay Limit of Detection Results Summary   

<table><tr><td rowspan=1 colspan=1>LIAISON PLEX Target</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source &amp; ID</td><td rowspan=1 colspan=1>ConfirmedLoD (CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 10231</td><td rowspan=1 colspan=1>3.14E+04</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 14053</td><td rowspan=1 colspan=1>2.83E+05</td></tr><tr><td rowspan=2 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>ATCC 8553</td><td rowspan=1 colspan=1>3.33E+03</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>ATCC 4135</td><td rowspan=1 colspan=1>3.33E+03</td></tr><tr><td rowspan=2 colspan=1>Candida famata</td><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>ATCC 20278</td><td rowspan=1 colspan=1>7.77E+02</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>ATCC 60229</td><td rowspan=1 colspan=1>6.99E+03</td></tr><tr><td rowspan=2 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>ATCC MYA-578</td><td rowspan=1 colspan=1>3.16E+03</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>ATCC MYA-577</td><td rowspan=1 colspan=1>3.16E+03</td></tr><tr><td rowspan=2 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15545</td><td rowspan=1 colspan=1>9.99E+03</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 15126</td><td rowspan=1 colspan=1>1.00E+04</td></tr><tr><td rowspan=2 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 6258</td><td rowspan=1 colspan=1>9.83E+03</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 28870</td><td rowspan=1 colspan=1>2.95E+04</td></tr><tr><td rowspan=2 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>ATCC 22017</td><td rowspan=1 colspan=1>3.33E+03</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>ATCC 34134</td><td rowspan=1 colspan=1>1.00E+04</td></tr><tr><td rowspan=2 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>ATCC 20460</td><td rowspan=1 colspan=1>3.18E+04</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>ATCC 20177</td><td rowspan=1 colspan=1>3.19E+04</td></tr><tr><td rowspan=2 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>ATCC 42720</td><td rowspan=1 colspan=1>1.11E+04</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>ATCC 34449</td><td rowspan=1 colspan=1>3.33E+04</td></tr></table>

# Luminex

complexity simplified.

<table><tr><td rowspan=1 colspan=1>LIAISON PLEX Target</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source &amp; ID</td><td rowspan=1 colspan=1>ConfirmedLoD (CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>Candida haemulonii/duobushaemulonii</td><td rowspan=1 colspan=1>Candida haemulonii</td><td rowspan=1 colspan=1>CBS 7375</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=1 colspan=1>Candida duobushaemulonii</td><td rowspan=1 colspan=1>CBS 7798</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=2 colspan=1>Candida auris</td><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>CBS 10913</td><td rowspan=1 colspan=1>3.50E+03</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>CBS 12766</td><td rowspan=1 colspan=1>3.50E+03</td></tr><tr><td rowspan=2 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 28474</td><td rowspan=1 colspan=1>2.85E+04</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 28475</td><td rowspan=1 colspan=1>9.50E+03</td></tr><tr><td rowspan=2 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 13803</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 201380</td><td rowspan=1 colspan=1>3.51E+03</td></tr><tr><td rowspan=2 colspan=1>Cryptococcus neoformans/gattii</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>ATCC 208821</td><td rowspan=1 colspan=1>9.01E+04</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC MYA-4871</td><td rowspan=1 colspan=1>9.00E+04</td></tr></table>

# b. Precision/Reproducibility:

# Precision/Repeatability

Within-laboratory precision/repeatability for the LIAISON PLEX® BCY Assay was evaluated across a minimum of five non-consecutive days utilizing two operators, two LIAISON PLEX® Systems and one lot of the LIAISON PLEX® BCY Assay cartridges. The targets were randomized so that the target being tested was blinded to the operators performing the testing. Each target panel (Bottle/Ring Positive (RP), Bottle/Ring Positive $^ { + 8 }$ hours $( \mathsf { R P + 8 } )$ , Negative Blood Matrix (N), and Contrived-negative containing Escherichia coli $^ { + 8 }$ hours (CN)) was run in triplicate by each operator for five non-consecutive days, totaling 15 replicates per target per operator.

The results of the within-laboratory precision / repeatability study are shown in Table 2.The results show the repeatability of the LIAISON PLEX® BCY Assay with an overall percent agreement of $100 \%$ , meaning the expected results were obtained for all analytes at all test levels across all samples, instruments, operators, and days.

Table 2. Within Laboratory Precision / Repeatability Confidence Interval   

<table><tr><td rowspan=3 colspan=1>ReportableTarget</td><td rowspan=3 colspan=1>Panel</td><td rowspan=3 colspan=1>CartridgeLot</td><td rowspan=1 colspan=5>Target Positivity</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=2>Operator 1</td><td rowspan=1 colspan=2>Operator 2</td><td rowspan=2 colspan=1>Overall</td><td rowspan=2 colspan=1>Lower</td><td rowspan=2 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>Instrument1</td><td rowspan=1 colspan=1>Instrument2</td><td rowspan=1 colspan=1>Instrument1</td><td rowspan=1 colspan=1>Instrument2</td></tr><tr><td rowspan=3 colspan=1>C. albicans</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>100% (8/8)</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>100% (9/9)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 hours</td><td rowspan=1 colspan=1>100% (8/8)</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NegativeBlood Matrix</td><td rowspan=1 colspan=1>0% (0/7)</td><td rowspan=1 colspan=1>0% (0/8)</td><td rowspan=1 colspan=1>0% (0/8)</td><td rowspan=1 colspan=1>0% (0/7)</td><td rowspan=1 colspan=1>0%(0/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=3 colspan=1>ReportableTarget</td><td rowspan=3 colspan=1>Panel</td><td rowspan=3 colspan=1>CartridgeLot</td><td rowspan=1 colspan=5>Target Positivity</td><td rowspan=1 colspan=2>95% C.I.</td></tr><tr><td rowspan=1 colspan=2>Operator 1</td><td rowspan=1 colspan=2>Operator 2</td><td rowspan=2 colspan=1>Overall</td><td rowspan=2 colspan=1>Lower</td><td rowspan=2 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>Instrument1</td><td rowspan=1 colspan=1>Instrument2</td><td rowspan=1 colspan=1>Instrument1</td><td rowspan=1 colspan=1>Instrument2</td></tr><tr><td rowspan=3 colspan=1>C. tropicalis</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>100% (8/8)</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>100% (9/9)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 hours</td><td rowspan=1 colspan=1>100% (8/8)</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NegativeBlood Matrix</td><td rowspan=1 colspan=1>0% (0/7)</td><td rowspan=1 colspan=1>0% (0/8)</td><td rowspan=1 colspan=1>0% (0/8)</td><td rowspan=1 colspan=1>0% (0/7)</td><td rowspan=1 colspan=1>0%(0/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>C.neoformans</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>100% (8/8)</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>100% (9/9)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>C.neoformans</td><td rowspan=1 colspan=1>Ring Positive+8 hours</td><td rowspan=1 colspan=1>100% (8/8)</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NegativeBlood Matrix</td><td rowspan=1 colspan=1>0% (0/7)</td><td rowspan=1 colspan=1>0% (0/8)</td><td rowspan=1 colspan=1>0% (0/8)</td><td rowspan=1 colspan=1>0% (0/7)</td><td rowspan=1 colspan=1>0%(0/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ContrivedNegative(E. coli)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0% (0/6)</td><td rowspan=1 colspan=1>0% (0/9)</td><td rowspan=1 colspan=1>0% (0/9)</td><td rowspan=1 colspan=1>0% (0/6)</td><td rowspan=1 colspan=1>0%(0/30)</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Lot-to-lot Reproducibility

Lot-to-lot reproducibility of the LIAISON PLX BCY Assay was evaluated by testing three different lots of LIAISON PLEX BCY Assay cartridges with one operator over five non-consecutive days, and with two runs per lot. The same target panels used for Precision/Repeatability were used for Lot-to-Lot reproducibility. The blinded reproducibility panels were tested in triplicate for each sample type by operator. Results of the Lot-to-Lot reproducibility are summarized below in Table 3. Overall percent agreement for Lot-to-Lot reproducibility was $100 \%$ .

Table 3 Lot-to-Lot Reproducibility Confidence Intervals   

<table><tr><td rowspan=2 colspan=1>ReportableTarget</td><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>Cartridge Lot</td><td rowspan=2 colspan=1>TargetPositivity</td><td rowspan=1 colspan=2>95% confidence interval</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=8 colspan=1>C. albicans</td><td rowspan=4 colspan=1>RP</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=4 colspan=1>RP+8</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Luminex

complexity simplified.

<table><tr><td rowspan=2 colspan=1>ReportableTarget</td><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>Cartridge Lot</td><td rowspan=2 colspan=1>TargetPositivity</td><td rowspan=1 colspan=2>95% confidence interval</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=8 colspan=1>C. tropicalis</td><td rowspan=4 colspan=1>RP</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=4 colspan=1>RP+8</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=8 colspan=1>C.neoformans</td><td rowspan=4 colspan=1>RP</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=4 colspan=1>RP+8</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>100% (15/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=8 colspan=1>Negative</td><td rowspan=4 colspan=1>CN</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>0% (0/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>0% (0/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>0% (0/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=4 colspan=1>N</td><td rowspan=1 colspan=1>Lot 1 (051823106ADEVIUO)</td><td rowspan=1 colspan=1>0% (0/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2 (032823106ADEVIUO)</td><td rowspan=1 colspan=1>0% (0/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3 (061923106ADEVIUO)</td><td rowspan=1 colspan=1>0% (0/15)</td><td rowspan=1 colspan=1>79.6%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0% (0/45)</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Site-to-site Reproducibility

Site-to-site reproducibility of the LIAISON PLEX BCY Assay was evaluated by testing one lot of LIAISON PLEX BCY Assay cartridges with two operators at each of three sites over five nonconsecutive days. Four target panels were prepared and tested across all sites and operators to evaluate site-to-site reproducibility. The same target panels used for Precision/Repeatability and Lot-to-Lot reproducibility were used for Site-to-Site reproducibility. The blinded reproducibility panels were tested in triplicate for each sample type by each operator on each testing day.

Results of the site-to-site reproducibility study are summarized in Table 4. Reproducibility of

# Luminex

complexity simplified.

the LIAISON PLEX® BCY Assay across three sites was $9 9 . 7 \%$ .

Table 4. LIAISON® PLEX BCY Assay Site to Site Reproducibility Results   

<table><tr><td rowspan=1 colspan=1>Panel Type</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Results (% Agreement)</td><td rowspan=1 colspan=1>95% C.I.</td></tr><tr><td rowspan=4 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>96.7% (29/30)</td><td rowspan=1 colspan=1>83.3% - 99.4%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>98.9% (89/90)</td><td rowspan=1 colspan=1>94.0% - 99.8%</td></tr><tr><td rowspan=4 colspan=1>Ring Positive + 8 hours</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>100%6 (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>100% 90/90)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=4 colspan=1>Negative Blood Matrix</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>100%6 (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>100%6 (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=4 colspan=1>Contrived Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>88.6% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>100%6 (90/90)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>99.7% (359/360)</td><td rowspan=1 colspan=1>98.4% - 100%</td></tr></table>

# Luminex

complexity simplified.

c. Linearity/assay reportable range: Not applicable. The LIAISON® PLEX Yeast Blood Culture Assay is a qualitative assay. d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls:

Several controls are built into the assay and system to ensure identification of processing errors and to establish validity of test results.

Internal Controls

Each LIAISON® PLEX BCY Assay cartridge includes internal controls to ensure performance of sample preparation, amplification, and detection. Internal control is automatically added to the sample prior to initiation of sample preparation and assesses extraction, nucleic acid recovery, amplification, and detection. Finally, a post-amplification hybridization control serves as an indicator of successful hybridization. Internal control results are reported as Pass or Fail on the printed reports (see the following Table 5 for detailed explanations of each control result). Internal controls must generate a signal above the threshold in each internal reaction for the system to report a valid test result.

Table 5. Interpretation of Controls on the LIAISON® PLEX BCY Assay Report   

<table><tr><td>Internal Control Result</td><td>Explanation</td><td>Suggested Action</td></tr><tr><td>Pass</td><td>Test was completed and internal controls were successful, indicating that valid results were generated.</td><td>Review and report results</td></tr><tr><td>Fail</td><td>One or more internal controls failed.</td><td>Repeat test with a new cartridge</td></tr></table>

# External Controls

Positive and negative external controls should be tested with each new lot or shipment of reagents, or monthly, (whichever occurs first), or in accordance with updated local, regional, state, and/or federal guidelines. Verified negative blood matrix can be used as the negative control. Previously characterized positive samples or verified negative blood matrix spiked with well characterized organisms may be used as the external positive control. External controls should be used in accordance with laboratory protocols and in accordance with local, state, and federal accrediting organizations, as applicable.

# Stability:

Specimen Stability

Contrived specimen stability at incubated storage $( 3 3 ^ { \circ } \mathsf { C } - 3 7 ^ { \circ } \mathsf { C } )$ , refrigerated $( 2 ^ { \circ } \mathsf { C } - 8 ^ { \circ } \mathsf { C } )$ , and frozen $\left( - 8 0 ^ { \circ } \mathsf { C } \pm 1 5 \right)$ storage was evaluated for use with the LIAISON® PLEX BCY Assay. Two reference panels, each made of three organisms, were tested at concentrations representing bottle positivity. The panels were then stored at incubated, refrigerated, or frozen temperature conditions. Testing occurred at various time points up to 1 month for

# Luminex

complexity simplified.

LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Yeast Blood Culture Assay Traditional 510(k) Submission the frozen storage, up to 1 month for refrigerated storage, and up to 72 hours for incubated storage. Specimen stability was also evaluated for specimens grown in a continuous blood monitoring system by testing fresh blood bottle ring positive (RP) and blood bottle ring positive plus 12 hours $\left( \mathsf { R P } + 1 2 \right)$ samples.

The results of this study demonstrated that specimens stored frozen $\left( < - 8 0 ^ { \circ } \mathsf { C } \right)$ are stable for up to 1 month, specimens stored refrigerated $( 2 ^ { \circ } \mathsf { C } - 8 ^ { \circ } \mathsf { C } )$ are stable for up to 1 month and specimens stored at incubation $( 3 3 ^ { \circ } \mathsf { C } - 3 7 ^ { \circ } \mathsf { C } )$ are stable for up to 72 hours. Specimens incubated up to 12 hours after bottle ring positive were $100 \%$ detected.

Device Stability

A shelf-life study was conducted to evaluate the real-time stability of the LIAISON PLEX BCY Assay at the recommended storage conditions of room temperature $\big ( 1 5 ^ { \circ } \mathsf { C } - 3 0 ^ { \circ } \mathsf { C } \big )$ . Real-time stability was assessed using three Positive Control panels which each consisted of representative on-panel fungal organisms, and one Negative Control panel which consisted of negative blood matrix. Results of real-time stability demonstrated the LIAISON PLEX BCY Assay is stable for at least 3 months when stored at $1 5 ^ { \circ } \mathsf { C } - 3 0 ^ { \circ } \mathsf { C }$ .

Open-box stability of the LIAISON PLEX BCY Assay was also evaluated by testing four lots of LIAISON PLEX BCY Assay after removal from their foil pouches and stored at room temperature. The same Positive Control panels and Negative Control panel used for real-time stability were used for open-box stability. Results of open-box stability indicate the cartridges are stable for up to eight hours after cartridges are removed from their foil pouches and stored at room temperature.

# Fresh vs. Frozen Specimen Stability

A study was performed to assess tolerance of the LIAISON® PLEX BCY Assay to correctly detect specimens containing fungal organisms that were tested fresh (i.e. unfrozen) which then underwent one or two freeze-thaw cycles and were tested after each freeze-thaw cycle. This was executed by testing three replicates of each target organism at each of the following concentrations: Blood bottle ring positive (RP), Blood bottle ring positive $^ { + 8 }$ hours $( \mathsf { R P + 8 } )$ and Blood bottle ring positive at a ten-fold dilution concentration (RP – 10).

Each target demonstrated an overall positive agreement of $100 \%$ at all three testing conditions of fresh, 1 freeze-thaw, and 2 freeze-thaws. Data supports the stability of specimens for up to two freeze-thaw cycles.

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission e. Growth and Detection Study:

A Growth and Detection study was performed to evaluate expected organism concentrations in blood cultures at bottle positivity (Ring Positive) and eight hours after bottle positivity $( \mathsf { R P + 8 } )$ , using one representative strain for each organisms.

A minimum of one ring positive bottle and one 8 hour plus ring positive bottle from each BCY organism were tested in triplicate on the LIAISON $\mathsf { P l E X } ^ { \otimes }$ BCY assay fresh, meaning the sample was retrieved from the automated blood culture system within the specified timeframe and was tested the same day. A minimum of two ring positive bottles and two 8 hour plus ring positive bottles from each BCY organism were tested in triplicate on the LIAISON PLEX® BCY assay either fresh or after being stored frozen at - $6 5 ^ { \circ } \mathsf { C }$ to ${ \tt - 9 5 ^ { \circ } C } .$ . In addition, one negative blood bottle was tested fresh in triplicate on the LIAISON PLEX® BCY assay. Concentrations shown below in Table 6 represent approximate levels that may be observed in a clinical setting.

Table 6 - Growth and Detection Summary   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>RepresentativeStrain</td><td rowspan=1 colspan=1>Testing Condition(RP/RP+8)</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=6 colspan=1>Candida albicans</td><td rowspan=6 colspan=1>Candidaalbicans</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>2.21E+04</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>4.30E+05</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>8.60E+05</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>1.83E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>1.54E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>6.90E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida auris</td><td rowspan=6 colspan=1>Candida auris</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.88E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.45E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>2.02E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>5.30E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>4.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>4.80E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candidadubliniensis</td><td rowspan=6 colspan=1>Candidadubliniensis</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.47E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.98E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>3.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>5.10E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida famata</td><td rowspan=6 colspan=1>Debarymoycesfabryi</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>7.30E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>4.90E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>3.20E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>1.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>1.80E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>5.90E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida glabrata</td><td rowspan=6 colspan=1>Candidaglabrata</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>8.00E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>3.80E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>6.10E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>1.44E+08</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>1.36E+08</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>1.60E+08</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td rowspan=1 colspan=1>" Yeast Blood Culture :</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>RepresentativeStrain</td><td rowspan=1 colspan=1>Testing Condition(RP/RP+8)</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=6 colspan=1>Candidaguilliermondii</td><td rowspan=6 colspan=1>Meyerozymaguilliermondii</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.91E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>3.60E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.86E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.60E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.80E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.40E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candidahaemulonii/duobushaemulonii</td><td rowspan=6 colspan=1>Candidahaemulonisvar.haemulonis</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>9.70E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>9.40E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.07E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.50E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.30E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida kefyr</td><td rowspan=6 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>8.20E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>6.30E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>2.34E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>2.07E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>2.15E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida krusei</td><td rowspan=6 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.24E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>1.66E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>2.09E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.00E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.50E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida lipolytica</td><td rowspan=6 colspan=1>Yarrowialipolytica</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>2.44E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>2.50E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>2.42E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>3.90E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.60E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.80E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candida lusitaniae</td><td rowspan=6 colspan=1>Candidalusitaniae</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>2.23E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>5.10E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Frozen</td><td rowspan=1 colspan=1>3.50E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>1.33E+08</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>1.22E+08</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>1.32E+08</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Candidaparapsilosis</td><td rowspan=6 colspan=1>Candidaparapsilosis</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.54E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.61E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.76E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>4.10E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.90E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>5.50E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=4 colspan=1>Candida tropicalis</td><td rowspan=4 colspan=1>Candidatropicalis</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.31E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.20E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.35E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.00E+07</td><td rowspan=1 colspan=1>100%</td></tr></table>

Confidential & Restricted 510(k) Summary

# Luminex

complexity simplified.

LIAISON PL $\mathbf { \mu } _ { \mathbf { \mu } } [ \mathbf { \vec { E } } \mathbf { X } ^ { \circledast }$ Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>RepresentativeStrain</td><td rowspan=1 colspan=1>Testing Condition(RP/RP+8)</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.90E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>3.30E+07</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=6 colspan=1>Cryptococcusneoformans/gattii</td><td rowspan=6 colspan=1>Cryptococcusneoformansvar. grubii</td><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>1.84E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>3.40E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP Fresh</td><td rowspan=1 colspan=1>2.09E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Fresh</td><td rowspan=1 colspan=1>5.00E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>5.50E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP + 8 Frozen</td><td rowspan=1 colspan=1>4.40E+06</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>NBM</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0%</td></tr></table>

# f. Analytical Reactivity (Inclusivity)

The analytical reactivity (inclusivity) of the LIAISON PLEX BCY Assay was performed to evaluate inclusive detection of multiple strains of each organism, representing all 14 reportable targets of 16 yeast species, for a total of 80 strains. Assay performance at the representative concentration of bottle positivity was assessed with five strains of each organism diluted in Negative Blood Matrix and tested in triplicate. Bottle positivity represents clinically relevant titers of each organism strain and was determined in execution of the Growth & Detection study for the LIAISON $\mathsf { P l E X } ^ { \mathbb { \left( \mathsf { B } \right) } }$ BCY Assay. The results, presented in the Table 7, demonstrated $100 \%$ positivity for each strain.

Table 7. LIAISON PLEX® BCY Inclusivity Summary   

<table><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>Organism Strain</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=5 colspan=1>Candida albicans</td><td rowspan=5 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 90028</td><td rowspan=1 colspan=1>4.4E+05</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 24433</td><td rowspan=1 colspan=1>4.4E+05</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 90029</td><td rowspan=1 colspan=1>4.4E+05</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-4780</td><td rowspan=1 colspan=1>4.4E+05</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801504</td><td rowspan=1 colspan=1>4.4E+05</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida auris</td><td rowspan=5 colspan=1>Candida auris</td><td rowspan=1 colspan=1>CBS-KNAW 12373</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 12767</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 12768</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0383</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 804386</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida dubliniensis</td><td rowspan=5 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>ATCC MYA-582</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-579</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 44508</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-2975</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801915</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida famata</td><td rowspan=5 colspan=1>Debaryomyces hansenii1</td><td rowspan=1 colspan=1>ATCC 9365</td><td rowspan=1 colspan=1>3.2E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 10619</td><td rowspan=1 colspan=1>3.2E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 36239</td><td rowspan=1 colspan=1>3.2E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 1961</td><td rowspan=1 colspan=1>3.2E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 14266</td><td rowspan=1 colspan=1>3.2E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC MYA-2950</td><td rowspan=1 colspan=1>3.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Confidential & Restricted

# Luminex

complexity simplified.

<table><tr><td rowspan=1 colspan=5>LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission</td></tr><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>Organism Strain</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>ATCC 34138</td><td rowspan=1 colspan=1>3.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 66032</td><td rowspan=1 colspan=1>3.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 90876</td><td rowspan=1 colspan=1>3.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801535</td><td rowspan=1 colspan=1>3.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida guilliermondii</td><td rowspan=4 colspan=1>Meyerozymaguilliermondii</td><td rowspan=1 colspan=1>ATCC 56822</td><td rowspan=1 colspan=1>1.9E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 14242</td><td rowspan=1 colspan=1>1.9E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 6260</td><td rowspan=1 colspan=1>1.9E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 90197</td><td rowspan=1 colspan=1>1.9E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>Zeptometrix 801602</td><td rowspan=1 colspan=1>1.9E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=10 colspan=1>Candida haemulonii/duobushaemulonii</td><td rowspan=1 colspan=1>Candida haemulonii</td><td rowspan=1 colspan=1>CDC AR-0395</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Candida haemulonis</td><td rowspan=1 colspan=1>CBS-KNAW 10970</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 10973</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Candida haemulonii var.vulnera</td><td rowspan=1 colspan=1>CBS-KNAW 12437</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 12438</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candidaduobushaemulonii</td><td rowspan=1 colspan=1>CBS-KNAW 6915</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 7098</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0392</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0394</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0391</td><td rowspan=1 colspan=1>9.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida kefyr</td><td rowspan=5 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>ATCC 2512</td><td rowspan=1 colspan=1>6.3E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 66028</td><td rowspan=1 colspan=1>6.3E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 204093</td><td rowspan=1 colspan=1>6.3E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 4135</td><td rowspan=1 colspan=1>6.3E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801837</td><td rowspan=1 colspan=1>6.3E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida krusei</td><td rowspan=5 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 32196</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 14243</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 22985</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 34135</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801536</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>ATCC 8662</td><td rowspan=1 colspan=1>2.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=4 colspan=1>Yarrowia lipolytica</td><td rowspan=1 colspan=1>ATCC 201242</td><td rowspan=1 colspan=1>2.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 90812</td><td rowspan=1 colspan=1>2.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-2613</td><td rowspan=1 colspan=1>2.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 32338</td><td rowspan=1 colspan=1>2.4E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida lusitaniae</td><td rowspan=3 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>ATCC 200953</td><td rowspan=1 colspan=1>2.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>CBS-KNAW 4870</td><td rowspan=1 colspan=1>2.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 66035</td><td rowspan=1 colspan=1>2.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Clavispora lusitaniae</td><td rowspan=1 colspan=1>CBS-KNAW 6936</td><td rowspan=1 colspan=1>2.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801603</td><td rowspan=1 colspan=1>1.9E+07²2</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>Candida parapsilosis</td><td rowspan=2 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 34136</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 96137</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

Confidential & Restricted

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>Organism Strain</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>ATCC 200954</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 38291</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801537</td><td rowspan=1 colspan=1>1.5E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Candida tropicalis</td><td rowspan=5 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC MYA-2734</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 1369</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 90874</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC 18807</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801538</td><td rowspan=1 colspan=1>1.2E+07</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=10 colspan=1>Cryptococcusneoformans/gattii</td><td rowspan=1 colspan=1>Cryptococcus neoformansvar. grubii</td><td rowspan=1 colspan=1>ATCC 14116</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=4 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>ATCC 90112</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-4564</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-4566</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801803</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=5 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC MYA-4563</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-4873</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-4560</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-4094</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 801917</td><td rowspan=1 colspan=1>1.8E+06</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

1 Debaryomyces hansenii is a synonymous taxon name for Candida famata. 2 The stock concentration of Clavispora lusitaniae (Zeptometrix 801603) was not concentrated enough to achieve the representative bottle positivity concentration. This preparation was prepared as stock material diluted 1:1 in Negative Blood Matrix which is allowed per protocol directive.

# Predicted (in silico) Reactivity (Inclusivity) Results

In-silico inclusivity analyses of the assay oligo sequences were performed against sequences available in the GenBank nt database as of December 30, 2023. The inclusivity analyses involved assessing the precent homology of each oligo sequence to its binding region on each target sequence retrieved from the public databases. The predicted inclusivity based on sequence homology is the percentage of sequences with at least $90 \%$ oligo identity. A summary of in-silico inclusivity analysis results are summarized below in Table 8.

# Luminex

complexity simplified.

Table 8. In-silico Analysis Results   

<table><tr><td rowspan=1 colspan=1>Reportable Target</td><td rowspan=1 colspan=1>Inclusive Organism/Target</td><td rowspan=1 colspan=1>Total #SequencesinAlignment</td><td rowspan=1 colspan=1># Sequences withPercent OligoIdentity≥ 90%</td><td rowspan=1 colspan=1>PredictedInclusivityPercentage(%)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>181 *</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>286</td><td rowspan=1 colspan=1>286</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>13 *</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>32 *</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>24 *</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>Candida haemulonii/duobushamulonii</td><td rowspan=1 colspan=1>Candida haemulonii</td><td rowspan=1 colspan=1>17*</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida duobushamulonii</td><td rowspan=1 colspan=1>9*</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>30 *</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>48 *</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>82 *</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>75 *</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>59 *</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>86 *</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=2 colspan=1>Cryptococcusneoformans/gattii</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>3243</td><td rowspan=1 colspan=1>3243</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>100</td></tr></table>

\* Includes WGS (whole genome shotgun) sequences.

# g. Analytical Specificity (Exclusivity)

Cross-Reactivity:

A Cross-Reactivity study was performed to evaluate the LIAISON $\mathsf { P l E X } ^ { \mathbb { \exp } }$ BCY Assay’s analytical specificity through the detection of off-panel and potentially cross-reactive organisms, including those phylogenetically related to the on-panel organisms and organisms likely to be present in blood culture samples.

Out of the 40 off-panel bacterial species tested, there was no cross-reactivity with any of the on-panel organisms. Out of the 37 off-panel yeast species tested, 35 organisms had no crossreactivity with on-panel organisms. Yarrowia deformans (NRRL Y-63659) cross-reacted and had $100 \%$ detection for the BCY target Candida lipolytica when tested at a concentration of $1 . 0 \mathsf { E } { + } 0 7 \mathsf { C F U / m L }$ . Candida pseudohaemulonii (CBS-KNAW 10004) cross-reacted and had $100 \%$ detection for the BCY target Candida haemulonii/duobushaemulonii when tested at $1 . 0 \mathsf { E } { + } 0 7 \mathsf { C F U / m L }$ .

One bacterial organism, Klebsiella pneumoniae (ATCC-BAA 2146), had $1 1 \%$ positivity (1 of 9) for the on-panel organisms Candida albicans and Candida dubliniensis at a concentration of $1 . 0 \mathsf { E } \mathrm { + } 0 8 \mathsf { C F U / m L } .$ . Candida pseudohaemulonii (CBS-KNAW 10004) also had $1 1 \%$ detection (1 of 9) for the BCY target Candida lusitaniae when tested at $1 . 0 \mathsf { E } { + } 0 7 \mathsf { C F U / m L }$ . Trichosporon asteroides (CBS-KNAW 6183) had $50 \%$ positivity (3 of 6) for Candida glabrata and $33 \%$ (2 of 6) positivity for Candida haemulonii/duobushaemulonii at a concentration of $1 . 0 \mathsf { E } { + } 0 7 $ $\mathsf { C F U / m L }$ . In-silico analysis and wet testing support that cross-reactivity is not expected to occur with these targets.

# Luminex

complexity simplified.

Refer to the Tables below which summarizes cross reactivity testing performed with Bacteria strains (Table 9) and Yeast strains (Table 10).

Table 9. LIAISON PLEX® BCY Assay Analytical Specificity – Cross Reactivity Bacteria Summary   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ConcentrationTested (CFU/mL)</td><td rowspan=1 colspan=1>Source &amp;Source ID</td><td rowspan=1 colspan=1>Positivity % (Organism)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>IHMA 128307</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Iwoffii</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 15309</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Aerococcus viridans</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 700406</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 14579</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 25285</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium (Propionibacterium) acnes</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 33179</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 8090</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 13124</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 43735</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 35029</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 14025</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 35030</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>NCTC 13846</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 29212</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Eggerthella lenta</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 25559</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 43165</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC-BAA2146</td><td rowspan=1 colspan=1>11% (C. albicans) (1/9)11% (C. dubliniensis) (1/9)</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 19115</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus rhamnosus</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 53103</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>8.9E+06</td><td rowspan=1 colspan=1>ATCC 10054</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 25830</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>IHMA 576602</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Prevotella denticola</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>BEI ResourcesHM-1173</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 12453</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 12401</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 33397</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 25923</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 9993</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 700567</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus hominis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 27844</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus intermedius</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 29663</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 43809</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>8.9E+07</td><td rowspan=1 colspan=1>ATCC 17666</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>NCTC 13920</td><td rowspan=1 colspan=1>0% (0/3)</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ConcentrationTested (CFU/mL)</td><td rowspan=1 colspan=1>Source &amp;Source ID</td><td rowspan=1 colspan=1>Positivity % (Organism)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 15914</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 6315</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 12964</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td><td rowspan=1 colspan=1>1.0E+08</td><td rowspan=1 colspan=1>ATCC 15305</td><td rowspan=1 colspan=1>0% (0/3)</td></tr></table>

Table 10. LIAISON PLEX® BCY Assay Analytical Specificity – Cross Reactivity Yeast Summary   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ConcentrationTested (CFU/mL)</td><td rowspan=1 colspan=1>Source &amp; SourceID</td><td rowspan=1 colspan=1>Positivity % (Organism)</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 1022</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Acremonium kiliense</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 20337</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida bracarensis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>JH 47</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida carpophila</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>CBS-KNAW 5256</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida inconspicua</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-2029</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida intermedia</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 14439</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida metapsilosis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-48470</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida nivariensis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>CBS-KNAW 9983</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida orthopsilosis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 20504</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida pseudohaemulonii</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>CBS-KNAW 10004</td><td rowspan=1 colspan=1>100% (C. haemulonii) (9/9)11% (C. lusitaniae) (1/9)</td></tr><tr><td rowspan=1 colspan=1>Candida pseudohaemulonii</td><td rowspan=1 colspan=1>1.0E+06</td><td rowspan=1 colspan=1>CBS-KNAW 10004</td><td rowspan=1 colspan=1>100% (C. haemulonii) (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida pseudohaemulonii</td><td rowspan=1 colspan=1>1.0E+05</td><td rowspan=1 colspan=1>CBS-KNAW 10004</td><td rowspan=1 colspan=1>100% (C. haemulonii) (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida pseudohaemulonii</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>CBS-KNAW 10004</td><td rowspan=1 colspan=1>100% (C. haemulonii) (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida pseudohaemulonii</td><td rowspan=1 colspan=1>1.0E+03</td><td rowspan=1 colspan=1>CBS-KNAW 10004</td><td rowspan=1 colspan=1>33% (C. haemulonii) (1/3)</td></tr><tr><td rowspan=1 colspan=1>Candida pseudohaemulonii</td><td rowspan=1 colspan=1>1.0E+02</td><td rowspan=1 colspan=1>CBS-KNAW 10004</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida rugosa</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-95</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida sake</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-1622</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida solani</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-2224</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Candida utilis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 9950</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Exophiala lecanii-corni</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 12734</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Filobasidium elegans</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-6486</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Filobasidium globisporum</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-17828</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Kluyveromyces lactis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 24207</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Kodamaea ohmeri</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-1932</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Meyerozyma caribbica</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-27274</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Malassezia furfur</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 14521</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Malassezia globosa</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC-MYA 4612</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Metschnikowia pulcherrima</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 22032</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Malassezia restricta</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC-MYA 4611</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Malassezia sympodialis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 96803</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Mucor velutinosus</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC-MYA 4766</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Pichia fermentans</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-1619</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Pichia norvegensis</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL YB-3904</td><td rowspan=1 colspan=1>0% (0/3)</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ConcentrationTested (CFU/mL)</td><td rowspan=1 colspan=1>Source &amp; SourceID</td><td rowspan=1 colspan=1>Positivity % (Organism)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 66034</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 18824</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Scedosporium prolificans</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 64913</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Sporidiobolus salmonicolor</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-5483</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Sporothrix schenckii</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 58251</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Trichosporon asteroides</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>CBS-KNAW 6183</td><td rowspan=1 colspan=1>50%(C. glabrata) (3/6)33%(C. haemulonii) (2/6)</td></tr><tr><td rowspan=1 colspan=1>Talaromyces marneffei</td><td rowspan=1 colspan=1>1.45E+06spores/mL</td><td rowspan=1 colspan=1>ATCC 18224</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Wickerhamomyces anomalus</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>ATCC 10262</td><td rowspan=1 colspan=1>0% (0/3)</td></tr><tr><td rowspan=1 colspan=1>Yarrowia deformans</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>NRRL Y-63659</td><td rowspan=1 colspan=1>100% (C. lipolytica) (9/9)</td></tr><tr><td rowspan=1 colspan=1>Yarrowia deformans</td><td rowspan=1 colspan=1>1.0E+06</td><td rowspan=1 colspan=1>NRRL Y-63659</td><td rowspan=1 colspan=1>100% (C. lipolytica) (3/3)</td></tr><tr><td rowspan=1 colspan=1>Yarrowia deformans</td><td rowspan=1 colspan=1>1.0E+05</td><td rowspan=1 colspan=1>NRRL Y-63659</td><td rowspan=1 colspan=1>100% (C. lipolytica) (3/3)</td></tr><tr><td rowspan=1 colspan=1>Yarrowia deformans</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>NRRL Y-63659</td><td rowspan=1 colspan=1>0% (0/3)</td></tr></table>

In silico cross-reactivity

In silico exclusivity assessment was performed using sequences available in the GenBank nt database as of December 28, 2023. Based on analysis of the sequences available in the GenBank nt database, the following potential cross-reactivity scenarios are predicted:

The Candida lipolytica oligo designs are predicted to detect Yarrowia deformans strains resulting in false positive results. • The Candida haemulonii/duobushamulonii oligo designs are predicted to detect Candida pseudohaemulonii strains resulting in false positive results.

The potential cross-reactive organisms assessed for in silico exclusivity analysis are provided below in Table 11.

Table 11. Potential Cross-Reactive Organisms assessed in the In Silico Exclusivity Analysis   

<table><tr><td rowspan="2">On-Panel Organisms</td><td colspan="2">Off-Panel Organisms</td></tr><tr><td>Bacteria</td><td>Fungi</td></tr><tr><td>Candida albicans</td><td>Acinetobacter baumanii</td><td>Acremonium kiliense</td></tr><tr><td>Candida auris</td><td>Acinetobacter Iwoffii</td><td>Aspergillus fumigatus</td></tr><tr><td>Candida dubliniensis</td><td>Aerococcus viridans</td><td>Candida bracarensis</td></tr><tr><td>Candida duobushamulonii</td><td>Bacillus cereus</td><td>Candida carpophila</td></tr><tr><td>Candida famata</td><td>Bacteroides fragilis</td><td>Candida inconspicua</td></tr><tr><td>Candida glabrata</td><td>Bordetella pertussis</td><td>Candida intermedia</td></tr><tr><td>Candida guilliermondii</td><td>Citrobacter freundii</td><td>Candida lambica</td></tr><tr><td>Candida haemulonii</td><td>Clostridium perfringens</td><td>Candida metapsilosis</td></tr><tr><td>Candida kefyr</td><td>Corynebacterium striatum</td><td>Candida nivariensis</td></tr><tr><td>Candida krusei</td><td>Cutibacterium (Propionibacterium) acnes</td><td>Candida norvegensis</td></tr><tr><td>Candida lipolytica</td><td>Eggerthella lenta</td><td>Candida orthopsilosis</td></tr></table>

# Luminex

complexity simplified.

LIAISON PL $\mathbf { \vec { E } } \mathbf { X } ^ { \circledast }$ Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan="2">On-Panel Organisms</td><td colspan="2">Off-Panel Organisms</td></tr><tr><td>Bacteria</td><td>Fungi</td></tr><tr><td>Candida lusitaniae</td><td>Enterobacter aerogenes</td><td>Candida pelliculosa</td></tr><tr><td>Candida parapsilosis</td><td>Enterobacter cloacae</td><td>Candida pseudohaemulonii</td></tr><tr><td>Candida tropicalis</td><td>Enterococcus avium</td><td>Candida sake</td></tr><tr><td>Cryptococcus gattii</td><td>Enterococcus faecalis</td><td>Candida utilis</td></tr><tr><td>Cryptococcus neoformans</td><td>Enterococcus faecium</td><td>Diutina (Candida) rugosa</td></tr><tr><td rowspan="10"></td><td>Escherichia coli</td><td>Exophiala jeanselmei</td></tr><tr><td>Klebsiella oxytoca</td><td>Filobasidium elegans</td></tr><tr><td>Klebsiella pneumoniae</td><td>Filobasidium globisporum</td></tr><tr><td>Lactobacillus rhamnosus</td><td>Fusarium solani</td></tr><tr><td>Listeria monocytogenes</td><td>Kluyveromyces lactis</td></tr><tr><td>Micrococcus luteus</td><td>Kodamaea ohmeri</td></tr><tr><td>Morganella morganii</td><td>Malassezia furfur</td></tr><tr><td>Prevotella denticola</td><td>Malassezia globose</td></tr><tr><td>Proteus mirabilis</td><td>Malassezia restricta</td></tr><tr><td>Pseudomonas aeruginosa</td><td>Malassezia sympodialis</td></tr><tr><td></td><td>Salmonella enterica (Typhi)</td><td>Metschnikowia (Candida) pulcherrima</td></tr><tr><td>Serratia marcescens</td><td></td><td>Meyerozyma caribbica (Candida fermentati)</td></tr><tr><td></td><td>Staphylococcus aureus</td><td>Mucor velutinosus</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>Staphylococcus epidermidis</td><td>Penicillium marneffei</td></tr><tr><td></td><td>Staphylococcus hominis</td><td>Rhodotorula mucilaginosa</td></tr><tr><td></td><td>Staphylococcus intermedius</td><td>Saccharomyces cerevisiae</td></tr><tr><td></td><td>Staphylococcus lugdunensis</td><td>Scedosporium prolificans</td></tr><tr><td></td><td>Staphylococcus saprophyticus</td><td>Sporidiobolus salmonicolor</td></tr><tr><td></td><td>Stenotrophomonas maltophilia</td><td>Sporothrix schenckii</td></tr><tr><td></td><td>Streptococcus agalactiae</td><td>Trichosporon spp.</td></tr><tr><td>Streptococcus mitis</td><td>Streptococcus anginosus</td><td>Yarrowia deformans</td></tr><tr><td></td><td></td><td>Protozoa</td></tr><tr><td></td><td>Streptococcus pneumoniae Streptococcus pyogenes</td><td>Plasmodium falciparum Trypanosoma cruzi</td></tr></table>

# h. Interference

# Microbial Interference

The impact of nine potentially interfering microbial organisms commonly found in blood culture samples were tested in the presence of on-panel organisms. C. albicans, C. glabrata, and C. parapsilosis were selected as the three representative on-panel organisms used throughout this study. Each potentially interfering organism was tested at a high concentration of $\ge 1 . 0 0 { + } \mathsf { E 0 8 } \mathsf { C F U / m L }$ for bacteria, in the presence of the on-panel organisms which were tested at low concentrations (ring positive), which were determined during the Growth and Detection Study. Table 12 summarizes the results from the Microbial Interference studies.

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   
Table 12. LIAISON PLEX® BCY Microbial Interference Summary   

<table><tr><td rowspan=1 colspan=1>Off-Panel High ConcentrationInterfering Microbe</td><td rowspan=1 colspan=1>On-Panel LowConcentration Target</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=3 colspan=1>Escherichia coli(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Staphylococcus aureus(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Klebsiella pneumoniae(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Pseudomonas aeruginosa(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecalis(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Staphylococcus epidermidis(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecium(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Off-Panel High ConcentrationInterfering Microbe</td><td rowspan=1 colspan=1>On-Panel LowConcentration Target</td><td rowspan=1 colspan=1>TargetPositivity</td></tr><tr><td rowspan=3 colspan=1>(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>(4.4E+05CFU/mL)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candidaglabrata(3.8E+06CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candidaparapsilosis(1.5E+07CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>Streptococcus pneumoniae(1.0E+08 CFU/mL)</td><td rowspan=1 colspan=1>Candidaalbicans(4.4E+05CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candidaglabrata(3.8E+06CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# Luminex

complexity simplified.

# Interfering Substances:

An Interfering Substance study was performed to evaluate non-microbial interference of 18 substances which may be present during collection or within clinical blood samples using the LIAISON PLEX® BCY Assay. Two positive panels containing three (3) on-panel organisms each (C. albicans, C. tropicalis, C. neoformans and C. glabrata, C. guilliermondii, and C. kefyr) and negative blood matrix were used to test each interfering substance at clinically relevant concentrations. The potentially interfering substances and their testing concentrations are listed in Table 15, the substances do not interfere with the LIAISON PLEX BCY Assay. All positive panel results were $100 \%$ positive. However, Fluconazole at the testing concentration of $2 5 \mathsf { m g / L }$ showed interference and resulted in a false negative for C. albicans.At the testing concentrations listed in Table 13, the substances do not interfere with the LIAISON PLEX BCY Assay. All positive panel results were $100 \%$ positive. However, Fluconazole at the testing concentration of $2 5 \mathrm { m g / L }$ showed interference and resulted in a false negative for C. albicans.

Table 13 - List of Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Interfering SubstanceConcentration</td></tr><tr><td rowspan=1 colspan=1>Unconjugated Bilirubin</td><td rowspan=1 colspan=1>20 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Conjugated Bilirubin</td><td rowspan=1 colspan=1>20 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>14 g/L</td></tr><tr><td rowspan=1 colspan=1>Y-globulin</td><td rowspan=1 colspan=1>6 g/dL</td></tr><tr><td rowspan=1 colspan=1>SodiumPolyanetholsulfonate</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin clavulanate</td><td rowspan=1 colspan=1>3.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Amphotericin B</td><td rowspan=1 colspan=1>2 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Caspofungin</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>0.23 mg/mL</td></tr></table>

# Luminex

complexity simplified.

<table><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Interfering SubstanceConcentration</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>3 mg/L</td></tr><tr><td rowspan=2 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>25 mg/L</td></tr><tr><td rowspan=1 colspan=1>8.3 mg/L</td></tr><tr><td rowspan=1 colspan=1>Flucytosine</td><td rowspan=1 colspan=1>90 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamicin sulfate</td><td rowspan=1 colspan=1>3 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>0.9 U/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>83 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>5 mg/L</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>30 mg/L</td></tr><tr><td rowspan=1 colspan=1>Intralipid/Triglycerides</td><td rowspan=1 colspan=1>3000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>0.9% NaCl Solvent Blank</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# Luminex

complexity simplified.

Competitive Interference/Co-infection:

Competitive inhibition of the LIAISON PLEX® BCY was assessed by testing six pairings of clinically prevalent co-infections. A single pairing consisted of one target at a high concentration with another target at a low concentration. Low concentration targets were tested at a concentration representative of Ring Positivity, as determined during the Growth and Detection study. All high concentration targets were tested at a concentration representative of Ring Positivity $^ { + 8 }$ hours, as determined during the Growth and Detection study. Testing of each combination was performed in triplicate.

Table 14. LIAISON PLEX® BCY Co-Infection Summary   

<table><tr><td rowspan=1 colspan=1>On-Panel HighConcentration Target</td><td rowspan=1 colspan=1>High ConcentrationTarget Positivity</td><td rowspan=1 colspan=1>On-Panel LowConcentration Target</td><td rowspan=1 colspan=1>Low ConcentrationTarget Positivity</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(5.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis(5.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(1.6E+08 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata(1.6E+08 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida albicans(4.4E+05 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida glabrata(3.8E+06 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>Candida parapsilosis(1.5E+07 CFU/mL)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

# Carry-Over/Cross-Contamination:

Carry-over and cross contamination for the LIAISON $\mathsf { P l E X } ^ { \otimes }$ BCY Assay was evaluated by testing positive and negative samples in an alternating series. Two operators tested 30 high concentration C. lusitaniae positive samples and 30 negative samples across two instruments over the course of two days. The samples were loaded into cartridges alternating between positive and negative samples, six cartridges at a time using sample prep trays. The six cartridges were loaded into a LIAISON PLEX® instrument in the same order they were loaded with sample. The instruments set-ups made it so the sample loaded and ran on each blade alternated between positive samples and negative samples for all runs executed in this study. The results demonstrated $100 \%$ agreement between expected and observed results, indicating that no carry-over or cross contamination occurred between and during runs.

# i. Assay Cut-off

The specific assay parameters for the LIAISON PLEX® BCY Assay are considered confidential and proprietary.

# Luminex

complexity simplified.

# 2. Comparison Studies:

a. Method comparison with predicate device:

Refer to Section 3 Clinical Performance.

b. Matrix Comparison: Testing of Blood Culture Bottle Types / Matrix Equivalency

The LIAISON PLEX® BCY Assay was evaluated to confirm equivalent performance for target detection in twelve different blood bottle matrixes from three different blood culture systems (BacT/Alert, BACTEC, and VersaTREK). The matrix was tested with negative blood matrix, two positive control panels, and with representative bacteria strains. Positive Control Panel 1 was comprised of Candida albicans, Cryptococcus neoformans, and Candida tropicalis. Positive Control Panel 2 was comprised of Candida glabrata, Candida guilliermondii, and Candida kefyr. The Positive Control Panels were tested at concentration representing blood culture bottle positivity, as determined during the Growth and Detection study. The bacteria strains used to test the assay were Acinetobacter baumannii, Escherichia coli, Klebsiella pneumonia, Straphylococcus aureus, and Pseudomonas aeruginosa. Bacteria strains were all tested at the concentration of $1 . 0 0 \mathsf { E } \mathsf { + } 0 8 \mathsf { C F U / m L }$ . A negative sample, Negative Blood Matrix, was also tested. A summary of blood bottle equivalency results is explained in Table 15.

Table 15 – LIAISON PLEX® BCY Assay Matrix Equivalency / Universal Blood Bottle Summary   

<table><tr><td rowspan=2 colspan=1>Manufacturer</td><td rowspan=2 colspan=1>BottleManufacturerPart Number</td><td rowspan=2 colspan=1>BottleName</td><td rowspan=2 colspan=1>Bottle Lot</td><td rowspan=1 colspan=4>Percent Positivity</td></tr><tr><td rowspan=1 colspan=1>NBM</td><td rowspan=1 colspan=1>PositiveControl11</td><td rowspan=1 colspan=1>PositiveControl2²</td><td rowspan=1 colspan=1>OffPanelTargets³</td></tr><tr><td rowspan=4 colspan=1>ThermoScientific</td><td rowspan=2 colspan=1>7106-44</td><td rowspan=2 colspan=1>VersaTrekRedox 1</td><td rowspan=1 colspan=1>593815</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>593817</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>7107-44</td><td rowspan=2 colspan=1>VersaTrekRedox 2</td><td rowspan=1 colspan=1>576419</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>626823</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>Becton,Dickinson,and Company(BD)</td><td rowspan=2 colspan=1>442022</td><td rowspan=2 colspan=1>Bactec PlusAnaerobic/F</td><td rowspan=1 colspan=1>3010406</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>3041161</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=1>Manufacturer</td><td rowspan=2 colspan=1>BottleManufacturerPart Number</td><td rowspan=2 colspan=1>BottleName</td><td rowspan=2 colspan=1>Bottle Lot</td><td rowspan=1 colspan=4>Percent Positivity</td></tr><tr><td rowspan=1 colspan=1>NBM</td><td rowspan=1 colspan=1>PositiveControl11</td><td rowspan=1 colspan=1>PositiveControl22</td><td rowspan=1 colspan=1>OffPanelTargets³3</td></tr><tr><td rowspan=10 colspan=1></td><td rowspan=2 colspan=1>442024</td><td rowspan=2 colspan=1>Bactec PlusStandardAnaerobic/F</td><td rowspan=1 colspan=1>2186992</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>2326552</td><td rowspan=1 colspan=1>20%(1/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>442021</td><td rowspan=2 colspan=1>BactecLytic/10Anaerobic/F</td><td rowspan=1 colspan=1>2227861</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>3046048</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>442023</td><td rowspan=2 colspan=1>Bactec PlusAerobic/F</td><td rowspan=1 colspan=1>2178710</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>3041879</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>442020</td><td rowspan=2 colspan=1>Bactec PedsPlus</td><td rowspan=1 colspan=1>2263255</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>3067576</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>442027</td><td rowspan=2 colspan=1>BactecStandard/10Aerobic</td><td rowspan=1 colspan=1>2227933</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>3060893</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=7 colspan=1>BioMérieux</td><td rowspan=2 colspan=1>259789</td><td rowspan=2 colspan=1>BACT/AlertSA</td><td rowspan=1 colspan=1>0001059400</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>0001059708</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>410852</td><td rowspan=2 colspan=1>BACT/AlertFN PluS</td><td rowspan=1 colspan=1>0004059160</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>0004059844</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=2 colspan=1>259790</td><td rowspan=2 colspan=1>BACT/AlertSN</td><td rowspan=1 colspan=1>0001059860</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>0001059900</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr><tr><td rowspan=1 colspan=1>410853</td><td rowspan=1 colspan=1>BACT/AlertPF Plus</td><td rowspan=1 colspan=1>0004101300</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr></table>

# Luminex

complexity simplified.

LIAISON PLEX® Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=1>Manufacturer</td><td rowspan=2 colspan=1>BottleManufacturerPart Number</td><td rowspan=2 colspan=1>BottleName</td><td rowspan=2 colspan=1>Bottle Lot</td><td rowspan=1 colspan=4>Percent Positivity</td></tr><tr><td rowspan=1 colspan=1>NBM</td><td rowspan=1 colspan=1>PositiveControl11</td><td rowspan=1 colspan=1>PositiveControl2²</td><td rowspan=1 colspan=1>OffPanelTargets³3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0004101532</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>0%(0/10)</td></tr></table>

# 3. Clinical Performance:

A multi-site clinical study established the diagnostic accuracy of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \exp } }$ BCY Assay for the detection and identification of multiple potentially pathogenic fungal organisms in positive blood culture from blood culture media identified as positive by a continuous monitoring blood culture system and which contain fungal organisms as determined by Gram Stain. The clinical performance of the LIAISON PLEX BCY Assay was evaluated using a combination of prospectively collected de-identified remnant samples, preselected positive samples, and contrived samples.

A total of 3447 unique prospectively collected specimens were evaluated for their eligibility across four geographically diverse US clinical sites. Of these, 69 (1 specimen was excluded due to inconclusive gram stain result) prospective specimens that met the pre-determined inclusion criteria and exhibited yeast morphology following gram stain were enrolled in the study. Clinical runs and re-runs using the LIAISON PLEX® BCY Assay were tested on the LIAISON $\mathsf { P l E X } ^ { \otimes }$ System by trained operators at four sites. Prospective specimen testing occurred between June 2023 and October 2023.

As all targets exhibited low prevalence rates in the prospective study, the prospective specimen set was supplemented with pre-selected and contrived specimens. The 63 pre-selected specimens were remnant, de-identified specimens sourced from 6 different sites/vendors in the United States. The pre-selected specimens were characterized by an FDA cleared molecular assay prior to enrollment in the study. The pre-selected specimens were tested in a randomized, blinded manner with negative specimens at one site during October 2023.

Out of the 132 clinical specimens included in the prospective and pre-selected study analysis, 130 $( 9 8 . 5 \% )$ generated valid BCY Assay results (i.e., Detected or Not Detected) on the first attempt. Two $( 1 . 5 \% )$ specimens with initial invalid results generated valid BCY results after a single retest. 132 specimens generated valid BCY results for a final success rate of $1 0 0 . 0 \%$ (132/132).

Table 16 provides a summary of the general demographic information of the 69 prospectively collected specimens that were included in the clinical study.

# Luminex

complexity simplified.

LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Yeast Blood Culture Assay Traditional 510(k) Submission Table 16 - Demographic Information for Prospectively collected specimens   

<table><tr><td rowspan=1 colspan=1>Tabie I6 - Deme</td><td rowspan=1 colspan=1>grapnic ihior</td><td rowspan=1 colspan=1>Iation Tor Pro</td><td rowspan=1 colspan=1>spectively </td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Site 01</td><td rowspan=1 colspan=1>Site 02</td><td rowspan=1 colspan=1>Site 03</td><td rowspan=1 colspan=1>Site 04</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>24(57.1%)</td><td rowspan=1 colspan=1>6(54.5%)</td><td rowspan=1 colspan=1>5(45.5%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>36(52.2%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>17(40.5%)</td><td rowspan=1 colspan=1>5(45.5%)</td><td rowspan=1 colspan=1>6(54.5%)</td><td rowspan=1 colspan=1>4(80.0%)</td><td rowspan=1 colspan=1>32(46.4%)</td></tr><tr><td rowspan=1 colspan=1>Gender Unknown</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>5(100.0%)</td><td rowspan=1 colspan=1>69(100.0%)</td></tr><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>2(4.8%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>3(4.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(9.1%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>21(50.0%)</td><td rowspan=1 colspan=1>9(81.8%)</td><td rowspan=1 colspan=1>8(72.7%)</td><td rowspan=1 colspan=1>3(60.0%)</td><td rowspan=1 colspan=1>41(59.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>18(42.9%)</td><td rowspan=1 colspan=1>1(9.1%)</td><td rowspan=1 colspan=1>3(27.3%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>23(33.3%)</td></tr><tr><td rowspan=1 colspan=1>Age Unknown</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>5(100.0%)</td><td rowspan=1 colspan=1>69(100.0%)</td></tr><tr><td rowspan=1 colspan=1>Subject Status</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Emergency Room</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(9.1%)</td><td rowspan=1 colspan=1>4(36.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>5(7.2%)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>10(90.9%)</td><td rowspan=1 colspan=1>7(63.6%)</td><td rowspan=1 colspan=1>5(100.0%)</td><td rowspan=1 colspan=1>22(31.9%)</td></tr><tr><td rowspan=1 colspan=1>Status Unknown</td><td rowspan=1 colspan=1>42(100.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>42(60.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>5(100.0%)</td><td rowspan=1 colspan=1>69(100.0%)</td></tr><tr><td rowspan=1 colspan=1>Blood CultureBottle type</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>BD BACTEC LyticAnaerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>3(27.3%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>3(4.3%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTEC PluSAerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>8(72.7%)</td><td rowspan=1 colspan=1>6(54.5%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>14(20.3%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTECStandard Aerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>5(45.5%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>5(7.2%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT FAPlus</td><td rowspan=1 colspan=1>40(95.2%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>40(58.0%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT FNPlus</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT PFPlus</td><td rowspan=1 colspan=1>1(2.4%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>2(2.9%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT SAStandard Aerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>3(60.0%)</td><td rowspan=1 colspan=1>3(4.3%)</td></tr><tr><td rowspan=1 colspan=1>BacT/ALERT SNStandard Anaerobic</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>1(20.0%)</td><td rowspan=1 colspan=1>1(1.4%)</td></tr><tr><td rowspan=1 colspan=1>Bottle Type Total</td><td rowspan=1 colspan=1>42(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>11(100.0%)</td><td rowspan=1 colspan=1>5(100.0%)</td><td rowspan=1 colspan=1>69(100.0%)</td></tr></table>

Table 17 provides a summary of the general demographic information of the 63 pre-selected collected specimens that were included in the clinical study.

# Luminex

complexity simplified.

LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Yeast Blood Culture Assay Traditional 510(k) Submission Table 17 - Demographic Information for Pre-Selected Clinical Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># Samples (%)</td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>24(38.1%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>25(39.7%)</td></tr><tr><td rowspan=1 colspan=1>Gender Unknown</td><td rowspan=1 colspan=1>14(22.2%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100.0%)</td></tr><tr><td rowspan=1 colspan=1>Age (years)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>1(1.6%)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>0(0.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>27(42.9%)</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>19(30.2%)</td></tr><tr><td rowspan=1 colspan=1>Age Unknown</td><td rowspan=1 colspan=1>16(25.4%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100.0%)</td></tr><tr><td rowspan=1 colspan=1>Subject Status</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>2(3.2%)</td></tr><tr><td rowspan=1 colspan=1>Status Unknown</td><td rowspan=1 colspan=1>61(96.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100.0%)</td></tr><tr><td rowspan=1 colspan=1>Blood Culture Bottle Type</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>BD BACTEC Lytic Anaerobic</td><td rowspan=1 colspan=1>2(3.2%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTEC Plus Aerobic</td><td rowspan=1 colspan=1>17(27.0%)</td></tr><tr><td rowspan=1 colspan=1>BD BACTEC Standard Aerobic</td><td rowspan=1 colspan=1>29(46.0%)</td></tr><tr><td rowspan=1 colspan=1>Bottle Type Unknown</td><td rowspan=1 colspan=1>15(23.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63(100.0%)</td></tr></table>

A total of 829 specimens were contrived and tested. To minimize bias, contrived specimens were blinded, randomized, and tested along with negative clinical specimens at all four testing sites during June 2023. Results from contrived specimens were analyzed separately from the prospective and pre-selected data sets.

Out of the 829 specimens included in the contrived study analysis, 803 specimens $( 9 6 . 9 \% )$ generated valid BCY Assay results (i.e., Detected or Not Detected) on the first attempt. There were 26 specimens $( 3 . 1 \% )$ with an invalid result on the initial run. Of the 26 specimens retested, 24 specimens generated a valid result after a single retest for a final success rate of $9 9 . 7 \%$ (827/829).

The invalid rate for prospective, pre-selected, and contrived specimens combined was $2 . 9 \%$ (28/961) after the initial run. After allowing for a single retest of any initial invalid results, 26 $( 2 . 7 \% )$ specimens generated valid BCY results. Two $( 0 . 2 \% )$ specimens remained invalid on repeat for an overall success rate of $9 9 . 8 \%$ (959/961).

For each target in the BCY Assay Panel, the diagnostic performance of the BCY Assay was determined using Sensitivity/PPA and Specificity/NPA, along with the associated $9 5 \%$ confidence intervals, $9 5 \%$ CI as compared to the reference method). The results of the combined prospective and pre-selected specimen analysis are summarized in Table 18. The performance of the BCY Assay for contrived specimens is presented separately in Table 19.

# Luminex

complexity simplified.

Table 18. LIAISON PLEX BCY Assay Performance with Prospective and Pre-Selected Specimens   

<table><tr><td rowspan=1 colspan=8>Table 18. LIAISON PLEX BCY Assay Performance with Prospective and Pre-Selected Specimens</td></tr><tr><td rowspan=2 colspan=2>Pathogen Target</td><td rowspan=1 colspan=3>Sensitivity / PPA1</td><td rowspan=1 colspan=3>Specificity / NPA²</td></tr><tr><td rowspan=1 colspan=1>TPI(TP+FN)</td><td rowspan=1 colspan=1>Sensitivity/PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN /(TN+FP)</td><td rowspan=1 colspan=1>SpecificityNPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>81.6%-100.0%</td><td rowspan=1 colspan=1>51/52</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>89.9%-99.%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>81.6%-100.0%</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.8%-100.0%</td><td rowspan=1 colspan=1>97/983</td><td rowspan=1 colspan=1>99.0%</td><td rowspan=1 colspan=1>94.4%-99.8%</td></tr><tr><td rowspan=3 colspan=1>Candida auris</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0%-100.0%</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.4%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0%-100.0%</td><td rowspan=1 colspan=1>128/128</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candidadubliniensis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>132/132</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candida famata</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>132/132</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.7%-100.0%</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.0%-10.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>84.5%-110.0%</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.3%-100.0%</td><td rowspan=1 colspan=1>86/86</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.7%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candidaguilliermondii</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>132/132</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candidahaemuloni/C.duobushaemulonii</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>132/132</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2%-100.0%</td></tr><tr><td rowspan=2 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>20.7%-100.0%</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr></table>

# Luminex

complexity simplified.

LIAISON PL $\mathbf { \vec { E } } \mathbf { X } ^ { \circledast }$ Yeast Blood Culture Assay Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=2>Pathogen Target</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Sensitivity / PPA1</td><td rowspan=1 colspan=3>Specificity / NPA²</td></tr><tr><td rowspan=1 colspan=2>TP I(TP+FN)</td><td rowspan=1 colspan=1>Sensitivity/PPA1</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN /(TN+FP)</td><td rowspan=1 colspan=1>Specificity/ NPA²2</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>1/1</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>20.7%-100.0%</td><td rowspan=1 colspan=1>131/131</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=2>3/3</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43.9%-100.0%</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=2>1/1</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>20.7%-100.0%</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.2%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0%-100.0%</td><td rowspan=1 colspan=1>128/128</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>132/132</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2%- 10.0%</td></tr><tr><td rowspan=3 colspan=1>Candidalusitaniae</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=2>2/2</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2%-100.0%</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>2/2</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2%-100.0%</td><td rowspan=1 colspan=1>130/130</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1%-100.0%</td></tr><tr><td rowspan=3 colspan=1>Candidaparapsilosis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=2>11/11</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>74.1%-100.0%</td><td rowspan=1 colspan=1>57/58</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>90.9%-99.7%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=2>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>61.0%-100.0%</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.7%-0.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>17/17</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>81.6%-100.0%</td><td rowspan=1 colspan=1>114/1154</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>95.2%-99.8%</td></tr><tr><td rowspan=3 colspan=1>Candidatropicalis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=2>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>61.0%-100.0%</td><td rowspan=1 colspan=1>60/63</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>86.9%-98.4%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.3%-100.%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>61.0%-100.0%%</td><td rowspan=1 colspan=1>123/1265</td><td rowspan=1 colspan=1>97.6%</td><td rowspan=1 colspan=1>93.2%-99.2%</td></tr><tr><td rowspan=3 colspan=1>Cryptococcusneoformans/Cryptococcus gattii</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Pre-selected</td><td rowspan=1 colspan=2>5/5</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>56.6%-100.0%</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.8%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=2>5/5</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>56.6%-100.0%</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1%-100.0%</td></tr></table>

1Sensitivity is designated for prospective sample testing, while PPA is designated for pre-selected sample testing. 2 Specificity is designated for prospective sample testing, while NPA is designated for pre-selected sample testing. 3The one Candida albicans False Positive was positive by a cleared molecular assay. 4The one Candida parapsilosis False Positive was positive by a cleared molecular assay. 5One out of three Candida tropicalis False Positives was positive by a cleared molecular assay.

# Luminex

complexity simplified.

LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Yeast Blood Culture Assay Traditional 510(k) Submission   
Table 19 – LIAISON PLEX BCY Assay Performance with Contrived Specimens   

<table><tr><td rowspan=1 colspan=1>Pathogen Target</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>TP I(TP+FN)</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN /(TN+FP)</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-1000%</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>776/7771</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.7%-100.0%</td><td rowspan=1 colspan=1>778/778</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-1000%</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-1000%</td></tr><tr><td rowspan=1 colspan=1>Candida haemulonii/C.duobushaemulonii</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%-100.0%</td><td rowspan=1 colspan=1>776/7772</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3%-99.9%</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.7%-100.0%</td><td rowspan=1 colspan=1>775/7783</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans/Cryptococcus gattii</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.3%-100.0%</td><td rowspan=1 colspan=1>727/727</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr></table>

The clinical performance of the LIAISON $\mathsf { P l E X } ^ { \otimes }$ BCY Assay was compared to the standard of care culture followed by identification by Matrix Assisted Laser Desorption/Ionization coupled to time-of-flight Mass Spectrometry (MALDI-TOF MS) for all targets, as summarized below in Table 20.

# Luminex

complexity simplified.

Table 20 - Reference Method for the LIAISON PLEX BCY Assay Clinical Study   

<table><tr><td rowspan=1 colspan=1>LIAISON PLEX® BCY Assay Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=14 colspan=1>Standard of Care culture followed byidentification by MALDI-TOF MS</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td></tr><tr><td rowspan=1 colspan=1>Candida lipolytica</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td></tr><tr><td rowspan=1 colspan=1>Candida haemulonii/duobushaemulonii (1)</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans / gattii</td></tr></table>

The pre-defined acceptance criteria for the clinical study were defined as follows:

• The assay shall achieve a target Sensitivity of $\geq 9 0 \%$ for all targets. The Specificity for each target should be established at a level of $2 9 5 \%$ . Failure rate shall be $\leq 1 5 \%$ .

Acceptance Criteria are based on Arms 1, 2, and 3 combined. All acceptance criteria were met successfully.

The study results demonstrate that the diagnostic accuracy of the BCY Assay is acceptable for the safe and effective detection and identification of fungal pathogens from blood culture media identified as positive by a continuous monitoring blood culture system and which contain fungal organisms as determined by Gram Stain from patients exhibiting clinical signs and symptoms of bloodstream infection.

# Luminex

complexity simplified.

# M. Proposed Labeling:

The labeling provided in the submission satisfies the requirements of 21 CFR 809.10.

# N. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.