# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

# B Applicant

Life Technologies Corporation

# C Proprietary and Established Names

Applied Biosystems TaqPath COVID-19, Flu A, Flu B, RSV Select Panel

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The Sars-Cov-2 Respiratory InfectionAnd Other MicrobialAgents When In A Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to obtain market clearance for the Applied Biosystems TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel (referred to as TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel below).

# B Measurand:

The TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel detects and identifies RNA from: SARS-CoV-2, influenza A virus, influenza B virus, and respiratory syncytial virus (RSV).

# C Type of Test:

The TaqPath COVID-19, Flu A, Flu B, RSV Select Panel is a multiplex, real-time reverse transcription polymerase chain reaction (RT-PCR) test.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Applied Biosystems TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel is a multiplex, real-time reverse transcription polymerase chain reaction (RT-PCR) in vitro diagnostic test for the qualitative detection and differentiation of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A virus, influenza B virus, and respiratory syncytial virus (RSV). Nucleic acids are isolated and purified from nasopharyngeal (NP) swab and anterior nasal (AN) swab specimens obtained from individuals exhibiting signs and symptoms of a respiratory tract infection. Clinical signs and symptoms of respiratory viral infection due to SARS-CoV-2, influenza, and RSV can be similar. This test is intended to aid in the differential diagnosis of SARS-CoV-2, influenza A, influenza B, and RSV A/B (undifferentiated) infections in humans and is not intended to detect influenza C virus infections.

Nucleic acids from the viral organisms identified by this test are generally detectable in NP and AN swab specimens during the acute phase of infection. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory tract infection are indicative of the presence of the identified virus and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out coinfection with other organisms. The organism(s) detected by the Applied Biosystems TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel may not be the definite cause of disease. Negative results do not preclude SARSCoV-2, influenza A virus, influenza B virus, or RSV infections.

The Applied Biosystems TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel is intended for use by qualified and trained clinical laboratory personnel specifically instructed and trained in the techniques of real-time PCR and in vitro diagnostic procedures.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only IVD - For In Vitro Diagnostic Use Only

# D Special Instrument Requirements:

This assay is to be used with the KingFisher Apex Dx Purification System and tested on the Applied Biosystems QuantStudio 5 Dx Real-Time PCR Instrument only.

# A Device Description:

The TaqPath COVID-19, Flu A, Flu B, RSV Select Panel is a multiplex, real-time reverse transcription polymerase chain reaction (RT-PCR) test. The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A, influenza B, respiratory syncytial virus (RSV) A/B and RNase P primer and probe sets are designed to detect viral RNA in nasopharyngeal (NP) and anterior nasal (AN) swab specimens from individuals exhibiting signs and symptoms of a respiratory tract infection. Each TaqPath COVID-19, Flu A, Flu B, RSV Select Panel includes the following components:

TaqPath COVID‑19, Flu A, Flu B, RSV Select Assay—Multiplex assays that contain primer and probe sets specific to the following targets:

o Three SARS-CoV-2 targets (Orf1a, Orf1b, and N genes);   
$\bigcirc$ Two influenza A virus targets (PB1 and M genes);   
$\bigcirc$ Two influenza B virus targets (M and NS genes);   
o Three RSV targets (NP, M, and L protein genes);   
O RNase P (internal human sample collection control).

TaqPath COVID‑19, Flu A, Flu B, RSV Select Positive Control—Inactivated viral control that contains SARS-CoV-2, influenza A, influenza B, and RSV. TaqPath COVID‑19, Flu A, Flu B, RSV Select Negative Control—MS2 packaged RNA control that contains targets specific to RNase P genomic regions targeted by the assay. TaqPath 1-Step Select Master Mix (No ROX)—Ready-to-use PCR mix, including reverse transcriptase, polymerase, deoxyribonucleotide triphosphates (dNTPs), salts, and buffer.

In addition to the SARS-CoV-2, influenza A, influenza B and RSV viral assay targets, the assay portion of the panel includes RNase P, which serves as an endogenous internal process control to monitor extraction and amplification of each clinical sample. The TaqPath COVID-19, Flu A, Flu B, RSV Select Panel also contains external process positive and negative controls. The positive control (PC) component included is an inactivated viral control that contains SARSCoV-2, influenza A, influenza B, and RSV viruses. The PC monitors extraction and real-time RT-PCR by demonstrating that each of the four viruses can be detected when present and that RNase P is not detected when absent. The negative control (NC) component included is an MS2 packaged RNA control that contains targets specific to RNase P genomic regions targeted by the assay. The NC also monitors extraction and real-time RT-PCR by demonstrating RNase P can be detected when present and that the four viruses are not detected when absent. The TaqPath 1- Step Select Master Mix (No ROX) included as a component of the kit is a ready-to-use PCR mix which contains a deoxyribonucleotide triphosphate mix (dNTPs), enzymes, and other components to permit reverse transcription and amplification of the assay targets. The TaqPath 1-Step Select Master Mix (No ROX) also contains ribonuclease (RNase) inhibitors as well as deoxyuridine triphosphate (dUTP) and uracil N-glycosylase (known as UNG or UDG),.

The TaqPath COVID-19, Flu A, Flu B, RSV Select Panel workflow begins with nucleic acid extraction from nasopharyngeal and anterior nasal swab specimens stored in viral transport media (VTM) or universal transport medium (UTM). Nucleic acids are isolated and purified from the specimens using the MagMAX Dx Viral/Pathogen NA Isolation Kit with the

KingFisher Apex Dx Purification System. In this process, nucleic acids from the patient samples are recovered using magnetic-bead technology.

The purified nucleic acid is reverse transcribed into cDNA and amplified using the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and the Applied Biosystems QuantStudio 5 Dx Real-Time PCR instrument. The Applied Biosystems Diomni Software is used for the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel results interpretation and is supplied with the QuantStudio 5 Dx Real-Time PCR instrument.

# B Principle of Operation:

Once the reaction plate is loaded onto the real-time PCR instrument, RNA is reverse transcribed into cDNA. After heat inactivation of the reverse transcriptase enzyme, thermal cycling for PCR amplification commences.

During the anneal/extension phase of a PCR cycle, primers and probes specific to the target genes bind to corresponding DNA sequences in the indicated targets. The $5 '$ nuclease activity of Taq polymerase degrades the probe, causing the reporter dye to separate from the quencher dye, generating a fluorescent signal. With each cycle, additional reporter dye molecules are cleaved from their respective probes, increasing the overall fluorescence intensity. Fluorescence intensity is monitored at each PCR cycle by the real-time PCR instrument.

The Diomni Software implements the thermal cycling program, secondary analysis and clinical interpretation of the results using assay-specific parameters from the Assay Definition File (ADF) that is installed in the software. When the software analyzes the run data, validation is performed automatically via quality check (QC) analysis. For a plate run, QC is based on the performance of the positive and negative controls; results interpretation of controls is described in Table 1. Individual samples are also subject to QC checks according to the rules provided in Table 2. A Positive call for RNase P is only required for samples that are negative for all four viral targets; samples with no amplification are called Invalid. The data and interpretive results for each run are saved as a batch in the software.

Table 1. Interpretation of Positive and Negative Controls   

<table><tr><td rowspan=1 colspan=5>Negative Control (NC)</td><td rowspan=2 colspan=1>Result(Well Call)</td><td rowspan=2 colspan=1>Status</td></tr><tr><td rowspan=1 colspan=1>C19(SARS-CoV-2)</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV AB</td><td rowspan=1 colspan=1>RNase P</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Passed</td></tr><tr><td rowspan=1 colspan=5>All other scenarios</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Failed</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5>Positive Control (PC)</td><td rowspan=2 colspan=1>Result(Well Call)</td><td rowspan=2 colspan=1>Status</td></tr><tr><td rowspan=1 colspan=1>C19(SARS-CoV-2)</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV AB</td><td rowspan=1 colspan=1>RNase P</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Passed</td></tr></table>

<table><tr><td>All other scenarios</td><td>Invalid</td><td>Failed</td></tr></table>

Table 2. Result Interpretation for Patient Samples   

<table><tr><td rowspan=1 colspan=5>Target</td><td rowspan=2 colspan=1>Call</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>C19(SARS-CoV-2</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSVAB</td><td rowspan=1 colspan=1>RNase P</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu A, Flu B,RSVAB Detected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu A, Flu B Detected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu A, RSVABDetected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu A Detected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu B, RSVABDetected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu B Detected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2, RSVAB Detected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SARS-CoV-2 Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Flu A, Flu B, RSVAB Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Flu A, Flu B Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Flu A, RSVAB Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Flu A Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Flu B, RSVAB Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Flu B Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS ORNEG</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>RSVAB Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>SARS-CoV-2, Flu A, Flu B,RSVAB Not Detected</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>RETEST[1]</td></tr></table>

[1] Retesting must be performed by re-extracting the original sample and repeating the real-time PCR. If the repeat result remains invalid, collection of a new specimen should be considered. POS-Positive, NEG-Negative

# C Instrument Description Information:

1. Instrument Name:

# KingFisher Apex Dx Purification System Applied Biosystems QuantStudio 5 Dx Real-Time PCR Instrument

2. Specimen Identification: Specimen identification can be configured in an automated fashion or entered manually.

3. Specimen Sampling and Handling: Nasopharyngeal and anterior nasal swab specimens eluted in viral transport media (VTM) or universal transport medium (UTM).

4. Quality Control:

Following controls are included in the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel:

a) RNase P, which serves as an endogenous internal process control to monitor extraction and amplification of each clinical sample.   
b) External Positive Control (PC) is an inactivated viral control that contains SARSCoV-2, influenza A, influenza B, and RSV viruses. The PC is required for each plate run and monitors extraction and real-time RT-PCR by demonstrating that each of the four viruses can be detected when present and that RNase P is not detected when absent.   
c) A Negative Control (NC) is an MS2 packaged RNA control that contains targets specific to RNase P genomic regions targeted by the assay. The NC is required for each plate run and also monitors extraction and real-time RT-PCR by demonstrating RNase P can be detected when present and that the four viruses are not detected when absent.

V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1) B Predicate 510(k) Number(s): DEN200031 C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice</td><td rowspan=1 colspan=1>K241806</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>TaqPath COVID-19, Flu A, Flu B, RSVSelect Panel</td><td rowspan=1 colspan=1>BioFire Respiratory Panel 2.1 (RP2.1)</td></tr><tr><td rowspan=1 colspan=1>Regulation Number</td><td rowspan=1 colspan=1>21 CFR 866.3981</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Regulation Name</td><td rowspan=1 colspan=1>Device To Detect And Identify NucleicAcid Targets In Respiratory SpecimensFrom Microbial Agents That Cause TheSARS-Cov-2 Respiratory Infection AndOther Microbial Agents When In A Multi-Target Test</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td>COVID-19, Flu A, Flu B, RSV Select Panel is a multiplex, real-time reverse infections. ns For findings.</td><td>nucleic acid test intended for use with the BioFire FilmArray 2.0 or BioFire transcription polymerase chain reaction FilmArray Torch systems for the (RT-PCR) in vitro diagnostic test for the simultaneous qualitative detection and qualitative detection and differentiation of identification of multiple respiratory severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained virus, influenza B virus, and respiratory from individuals suspected of syncytial virus (RSV). Nucleic acids are respiratory tract infections, including isolated and purified from nasopharyngeal COVID-19. (NP) swab and anterior nasal (AN) swab The following organism types and specimens obtained from individuals exhibiting signs and symptoms of a subtypes are identified using the respiratory tract infection. Clinical signs BioFire RP2.1: and symptoms of respiratory viral infection Adenovirus, due to SARS-CoV-2, influenza, and RSV Coronavirus 229E, can be similar. This test is intended to aid in Coronavirus NL63, the differential diagnosis of SARS-CoV-2, Coronavirus OC43, influenza A, influenza B, and RSV A/B Coronavirus HKU1, (undifferentiated) infections in humans and Severe Acute Respiratory Syndrome is not intended to detect influenza C virus Coronavirus (SARS-CoV-2), Human Metapneumovirus, Human Rhinovirus/Enterovirus, Nucleic acids from the viral organisms Influenza A, including subtypes H1, identified by this test are generally H1-2009, and H3, detectable in NP and AN swab specimens Influenza B, during the acute phase of infection. The Parainfluenza Virus 1, detection and identification of specific viral Parainfluenza Virus 2, nucleic acids from individuals exhibiting Parainfluenza Virus 3, signs and symptoms of respiratory tract Parainfluenza Virus 4, infection are indicative of the presence of Respiratory Syncytial Virus, the identified virus and aids in diagnosis if Bordetella parapertussis (IS1001), used in conjunction with other clinical and Bordetella pertussis (ptxP), epidemiological information, and laboratory Chlamydia pneumoniae, and Mycoplasma pneumoniae The results of this test should not be used as Nucleic acids from the respiratory vira the sole basis for diagnosis, treatment, or and bacterial organisms identified by other patient management decisions. this test are generally detectable in NPS Positive results do not rule out coinfection specimens during the acute phase of with other organisms. The organism(s) infection. The detection and detected by the Applied Biosystems identification of specific viral and TaqPath COVID-19, Flu A, Flu B, RSV bacterial nucleic acids from individual Select Panel may not be the definite cause exhibiting signs and/or symptoms of of disease. Negative results do not preclude respiratory infection is indicative of the</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B virus, or RSV infections.The Applied Biosystems TaqPathCOVID-19, Flu A, Flu B, RSV Select Panelis intended for use by qualified and trainedclinical laboratory personnel specificallyinstructed and trained in the techniques ofreal-time PCR and in vitro diagnosticprocedures.</td><td colspan="1" rowspan="1">microorganism and aids in the diagnosisof respiratory infection if used inconjunction with other clinical andepidemiological information. Theresults of this test should not be used asthe sole basis for diagnosis, treatment,or other patient management decisions.Negative results in the setting of arespiratory illness may be due toinfection with pathogens that are notdetected by this test, or lowerrespiratory tract infection that may notbe detected by an NPS specimen.Positive results do not rule outcoinfection with other organisms. Theagent(s) detected by the BioFire RP2.1may not be the definite cause ofdisease.Additional laboratory testing (e.g.bacterial and viral culture,immunofluorescence, and radiography)may be necessary when evaluating apatient with possible respiratory tractinfection.</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">SARS-CoV-2, Influenza A, Influenza B,Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Anterior Nasal Swabs and Nasopharyngealswabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Transport Media</td><td colspan="1" rowspan="1">Universal Transport Medium (UTM) / ViralTransport Medium (VTM)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Sample Prep/Nucleic Acid Purification:• MagMAX Dx Viral/Pathogen NAIsolation KitKingFisher Apex Dx Purification SystemReal-time PCR instrument:• QuantStudio 5 Dx</td><td colspan="1" rowspan="1">Sample Prep/Nucleic Acid Purification:•Magnetic bead-based chemistry iscontained within the BioFire RP2.1pouchInstruments:FilmArray 2.0FilmArray Torch</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Real-time Reverse Transcription PCR (RT-PCR)</td><td colspan="1" rowspan="1">Reverse Transcription PCR (RT-PCR)</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">One negative control and one positivecontrol are run for each plate (externalprocess controls). The assay detectsendogenous RNase P present in the</td><td colspan="1" rowspan="1">Two internal controls are included ineach reagent pouch for quality controlof sample processing and both PCRstages and melt analysis.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">specimen as an internal positive/processcontrol.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation</td><td colspan="1" rowspan="1">Same</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI EP37. Supplemental Tables for Interference Testing in Clinical Chemistry; First Edition.   
CLSI EP05-A3. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline – Third Edition.   
CLSI EP07. Interference Testing in Clinical Chemistry; Third Edition.   
CLSI EP24-A2. Assessment of the Diagnostic Accuracy of Laboratory Testing Using Receiver Operating Characteristic Curves; Approved Guideline – Second Edition.   
CLSI EP25-A. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI EP35. Assessment of Equivalence or Suitability of Specimen Types for Medical Laboratory Measurement Procedures- First edition   
CLSI MM06-A2. Quantitative Molecular Methods for Infectious Diseases; Approved Guideline - Second Edition   
CLSI MM17. Verification and Validation of Multiplex Nucleic Acid Assays- Second Edition   
ISO 23640:2011. In vitro diagnostic medical devices. Evaluation of stability of in vitro diagnostic reagents   
ASTM D4169-22. Standard Practice for Performance Testing of Shipping Containers and Systems   
ASTM F1980-21. Standard Guide for Accelerated Aging of Sterile Barrier Systems for Medical Devices   
ISTA 7E. Testing Standard for Thermal Transport Packaging Used in Parcel Delivery System Shipment   
AAMI TIR62366-2:2016. Medical devices-Part 2: Guidance on the application of usability engineering to medical devices   
ISO 20916:2019. In vitro diagnostic medical devices. Clinical performance studies using specimens from human subjects. Good study practices.

Special Controls Class II Special Controls as per 21 CFR 866.3981

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

# a. Within-laboratory precision

Within-laboratory precision was evaluated at a single site using the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel on the QuantStudio $5 \mathrm { D x }$ (QS5Dx) instrument. A total of nine contrived samples containing known either a low $\left( \sim \mathrm { l x } \ \mathrm { L o D } \right)$ or moderate $( { \sim } 3 \mathrm { x } \ L \mathrm { o D } )$ concentrations of each target analyte (Table 3) and a negative sample were prepared in pooled negative NP specimens in VTM/UTM and extracted using the ThermoFisher MagMAX Dx Viral/Pathogen Nucleic Acid Isolation and the ThermoFisher KingFisher Apex Purification System. The study was conducted with two operators and one reagent lot over 12 non-consecutive days. Each panel member was tested in duplicate by each operator twice per day generating a total of 96 replicates per panel member (1 Site x 2 Operators x 1 Lot x 12 Days x 2 runs/day x 2 replicates/run). An extracted positive and negative control was included in each run and there were no invalid runs during precision studies.

Table 3. Precision Study Sample Panel   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>~1x LoD</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>~3x LoD</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>H3N2Texas/50/12</td><td rowspan=1 colspan=1>~1x LoD</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>H3N2Texas/50/12</td><td rowspan=1 colspan=1>~3x LoD</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Victoria/504/00</td><td rowspan=1 colspan=1>~1x LoD</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Victoria/504/00</td><td rowspan=1 colspan=1>~3x LoD</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>B CH93(18)-18</td><td rowspan=1 colspan=1>~1x LoD</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>B CH93(18)-18</td><td rowspan=1 colspan=1>~3x LoD</td></tr></table>

The percent agreement with expected results and Cq variability analysis results from the study are shown in Table 4 and Table 5, respectively.

All 96 replicates of each target analyte were positive $100 \%$ of the time for both low and moderate concentrations. The negative panel member generated negative results, as expected, for all samples (Table 4).

Table 4. Within-laboratory Precision Study Results   
Table 5. Within-laboratory Precision Study - Cq Analysis Results   

<table><tr><td rowspan=1 colspan=1>PanelID</td><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>% Positive(pos n/valid n)</td><td rowspan=1 colspan=1>% Agreement withExpected Results(95% CI)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0% (0/96)</td><td rowspan=1 colspan=1>100% (96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~1x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100% (96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~3x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%6 (96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>~1x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>~3x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>~1x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>~3x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>~1x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100% (96.15-100%)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>~3x LoD</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100% (96.15-100%)</td></tr></table>

The mean and variability analysis between operators, days, runs, within test, and overall are shown in Table 5.

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>LoDConc.</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCq</td><td rowspan=1 colspan=2>BetweenOperators</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>Within Test</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>25.16</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.87</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>∼1x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>35.52</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.47</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>1.59</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~3x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>34.11</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.08</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>∼ 1x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>33.05</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.82</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>~3x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>32.48</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.69</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>∼1x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>34.03</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.83</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>~3x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>32.35</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.59</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>~1x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>34.28</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.30</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>~3x</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>32.30</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.28</td></tr></table>

# b. Reproducibility

A blinded, multi-site reproducibility study was conducted to assess the total variability of the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel across operators, study sites, testing days, runs, instruments, and reagent lots.

The same nine contrived panels evaluated in the Within-laboratory Precision Study (Table 3) were also tested in the Reproducibility Study. The study was performed across three independent sites, each with two operators per site across five non-consecutive days using the QS5Dx instrument. Each panel member was tested with each of the three (3) real-time PCR reagent lots in triplicate, for a total of nine (9) replicates per panel member in every run. At least one (1) Negative Control (NC) and one (1) Positive Control (PC) was included per run. For each panel member, a total of 270 replicates were generated per instrument (3 replicates/lot x 3 lots $_ { \textrm { X 2 } }$ operators x 3 sites x 5 days).

The percent agreement with expected results and $\mathrm { C q }$ variability analysis results from the study are summarized in Table 6 and Table 7.

Table 6. Reproducibility Study Results   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Concentration</td><td colspan="3" rowspan="1">Agreement/Expected(% Agreement)</td><td colspan="1" rowspan="1">Overall Agreement(95% Score CI)</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">89/90(98.89%)[1]</td><td colspan="1" rowspan="1">99.63%(97.93%, 99.93%)</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">~1x LoD</td><td colspan="1" rowspan="1">85/90(94.44%)</td><td colspan="1" rowspan="1">82/90(91.11%)</td><td colspan="1" rowspan="1">83/90(92.22%)</td><td colspan="1" rowspan="1">92.59%(88.84%, 95.15%)</td></tr><tr><td colspan="1" rowspan="1">~3x LoD</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">100%(98.60%, 100%)</td></tr><tr><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">~1x LoD</td><td colspan="1" rowspan="1">88/90(97.78%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">89/90(98.89%)</td><td colspan="1" rowspan="1">98.89%(96.78%, 99.62%)</td></tr><tr><td colspan="1" rowspan="1">~3x LoD</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">100%(98.60%, 100%)</td></tr><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">~1x LoD</td><td colspan="1" rowspan="1">86/90(95.56%)</td><td colspan="1" rowspan="1">89/90(98.89%)</td><td colspan="1" rowspan="1">81/90(90.00%)</td><td colspan="1" rowspan="1">94.81%(91.49%, 96.89%)</td></tr><tr><td colspan="1" rowspan="1">~3x LoD</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">100%(98.60%, 100%)</td></tr><tr><td colspan="1" rowspan="2">RSV</td><td colspan="1" rowspan="1">~1x LoD</td><td colspan="1" rowspan="1">87/90(96.67%)</td><td colspan="1" rowspan="1">87/90(96.67%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">97.78%(95.24%, 98.98%)</td></tr><tr><td colspan="1" rowspan="1">~3x LoD</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">100%(98.60%, 100%)</td></tr></table>

[1]One replicate was called positive for Flu A.

The mean and variability analyte between sites, operators, lots, days, runs, within-runs, and overall, for Ct values is shown in Table 7. A total of 270 replicates were performed for each panel member and variability was calculated based on samples that generated positive results.

Table 7. Reproducibility Study - Cq Analysis Results   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Testlevel</td><td colspan="1" rowspan="2">MeanCq</td><td colspan="2" rowspan="1">Betweensites</td><td colspan="2" rowspan="1">Betweenlots</td><td colspan="2" rowspan="1">Betweendays</td><td colspan="2" rowspan="1">BetweenInstruments/operators</td><td colspan="2" rowspan="1">Repeatability</td><td colspan="2" rowspan="1">Within-labprecision</td><td colspan="2" rowspan="1">Reproducibility(total)</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">25.351]</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.99</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">2.04</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">2.32</td></tr><tr><td colspan="1" rowspan="1">SARS-</td><td colspan="1" rowspan="1">1xLoD</td><td colspan="1" rowspan="1">36.09</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">1.09</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">1.82</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">2.14</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">2.29</td></tr><tr><td colspan="1" rowspan="1">CoV-2</td><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">34.50</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.77</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">2.05</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">2.18</td></tr><tr><td colspan="1" rowspan="2">InfluenzaA</td><td colspan="1" rowspan="1">1xLoD</td><td colspan="1" rowspan="1">34.75</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.46</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">2.25</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">2.69</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">2.70</td></tr><tr><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">33.05</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.51</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">1.65</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">2.23</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">2.24</td></tr><tr><td colspan="1" rowspan="2">InfluenzaB</td><td colspan="1" rowspan="1">1xLoD</td><td colspan="1" rowspan="1">35.01</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">2.51</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">2.65</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">2.81</td></tr><tr><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">33.50</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.40</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.50</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">2.08</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">2.56</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">2.92</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1xLoD</td><td colspan="1" rowspan="1">35.43</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">1.24</td><td colspan="1" rowspan="1">3.50</td><td colspan="1" rowspan="1">1.33</td><td colspan="1" rowspan="1">3.76</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">3.85</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">33.81</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">1.93</td><td colspan="1" rowspan="1">1.13</td><td colspan="1" rowspan="1">3.34</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">3.88</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">3.88</td></tr></table>

[1] Mean Cq, SD, and $\% \mathrm { C V }$ calculated for RNase P only.

2. Linearity:

Linearity is not applicable, as the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel is a qualitative test.

# 3. Analytical Specificity/Interference:

# a. Interfering Substances

Potential interference of the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel was assessed by testing contrived positive samples consisting of one strain each of SARSCoV-2 (USA-WA1/2020), influenza A (H1N1pdm, Michigan/45/15), influenza B (Yamagata, Florida/04/06) and RSV (RSVB/CH93(18)-18)) at $3 \mathbf { x }$ their respective limits of detection (LoDs) as well as negative samples in the presence or absence of the twentysix (26) potentially interfering substances, two (2) solvents, one (1) no interferent control, and five (5) viral transport media (VTM). Additionally, potential interference due to extraction reagent carryover was assessed with four (4) carryover conditions and one (1) no-carryover control. For all potential interferents other than VTM, contrived samples were prepared in pooled negative NP swab specimens in VTM/UTM. VTM testing used pooled negative AN swab specimens that had been collected in the VTM . Each substance was tested in triplicate $\scriptstyle ( \mathrm { N } = 3 )$ ) for each positive and negative sample through the full workflow with one (1) lot of reagents on the QS5Dx instrument.

Detection of both Flu A and Flu B in all samples tested with FluMist at $0 . 0 5 \%$ , $0 . 0 1 \%$ and $0 . 0 0 5 \%$ v/v was expected since it is a live-attenuated virus vaccine. The presence of FluMist did not result in false positive or false negative results for the other targets at any of the three concentrations tested. None of the other exogenous or endogenous substances, transport media or extraction kit reagents tested interfered with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel at the concentrations tested. The results from the study are presented in Table 8.

Table 8. Summary of Interfering Substances Study Results   

<table><tr><td colspan="1" rowspan="2">SubstanceDescription</td><td colspan="1" rowspan="2">Conc.</td><td colspan="1" rowspan="1">NEG</td><td colspan="2" rowspan="1">Influenza A</td><td colspan="2" rowspan="1">Influenza B</td><td colspan="2" rowspan="1">SARS-CoV-2</td><td colspan="2" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">%Positive(nPositive/ Total)</td><td colspan="1" rowspan="1">% Positive(nPositiveTotal</td><td colspan="1" rowspan="1">MeanCq</td><td colspan="1" rowspan="1">%Positive(nPositive/Total)</td><td colspan="1" rowspan="1">MeanCq</td><td colspan="1" rowspan="1">%Positive(nPositive/Total</td><td colspan="1" rowspan="1">MeanCq</td><td colspan="1" rowspan="1">%Positive(nPositive/Total)</td><td colspan="1" rowspan="1">MeanCq</td></tr><tr><td colspan="1" rowspan="1">No interferentcontrol (NP)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0% (0/9)</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">33.20</td><td colspan="1" rowspan="1">100% (9/9)</td><td colspan="1" rowspan="1">33.32</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">34.20</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">33.42</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.94</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.23</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">35.20</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.23</td></tr><tr><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">5% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.76</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.08</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.09</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.78</td></tr><tr><td colspan="1" rowspan="1">Jurkat cell gDNA</td><td colspan="1" rowspan="1">7 μg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.65</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.09</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.02</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.15</td></tr><tr><td colspan="1" rowspan="1">Nasal decongestant Phenylephrine(Equat</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.23</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.30</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.72</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.60</td></tr><tr><td colspan="1" rowspan="1">Nasal decongestant Oxymetazoline(Afrin)</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">30.50</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">31.20</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.42</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.83</td></tr><tr><td colspan="1" rowspan="1">Saline nasal spray(Equate)</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.69</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.04</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.04</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.29</td></tr><tr><td colspan="1" rowspan="1">Bronchodilator(Ventolin Evohaler)</td><td colspan="1" rowspan="1">0.83mg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.67</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.35</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.26</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.51</td></tr><tr><td colspan="1" rowspan="1">Nasalcorticosteroids Dexamethasone</td><td colspan="1" rowspan="1">1.5 mg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.60</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.87</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.44</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.48</td></tr><tr><td colspan="1" rowspan="1">NasalcorticosteroidsFunisolide</td><td colspan="1" rowspan="1">2 mg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.16</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.93</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.78</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">31.27</td></tr><tr><td colspan="1" rowspan="1">NasalcorticosteroidsBudesonide(Apotex)</td><td colspan="1" rowspan="1">1% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.71</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.00</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.10</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.73</td></tr><tr><td colspan="1" rowspan="1">Nasalcorticosteroids Fluticasone(Equate)</td><td colspan="1" rowspan="1">1% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.17</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.57</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.74</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.11</td></tr><tr><td colspan="1" rowspan="1">Nasal gel(NeilMed Nasogel)</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.38</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.27</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.60</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.29</td></tr><tr><td colspan="1" rowspan="1">Homeopathicallergy reliefmedicine (Zicam)</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.36</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.84</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.74</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.12</td></tr><tr><td colspan="1" rowspan="1">Throat lozenges(Chloraseptic Max)</td><td colspan="1" rowspan="1">2.2 mg/mLmethanol;3.3 mg/mLbenzocaine</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.71</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.46</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.33</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.25</td></tr><tr><td colspan="1" rowspan="1">Zinc lozenges(Life Extension)</td><td colspan="1" rowspan="1">7.5 mg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.69</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.17</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.06</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.20</td></tr><tr><td colspan="1" rowspan="1">Antiviral Zanamivir</td><td colspan="1" rowspan="1">5.5 mg/mL</td><td colspan="1" rowspan="1">0% (0/6)</td><td colspan="1" rowspan="1">100% (4/4)</td><td colspan="1" rowspan="1">33.46</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">32.95</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.33</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.39</td></tr><tr><td colspan="1" rowspan="1">Antiviral -Oseltamivirphosphate</td><td colspan="1" rowspan="1">33 μg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.25</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">31.93</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.52</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.38</td></tr><tr><td colspan="1" rowspan="1">Antiviral -Remdesivir</td><td colspan="1" rowspan="1">6.7 μg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.45</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.48</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.88</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.03</td></tr><tr><td colspan="1" rowspan="1">Topical antibiotic(seudomonicAcid</td><td colspan="1" rowspan="1">3.3 mg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.09</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.06</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.56</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.43</td></tr><tr><td colspan="1" rowspan="1">Systemic antibiotic(Tobramycin)</td><td colspan="1" rowspan="1">4 μg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.33</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.24</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.40</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.92</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.05%(v/v</td><td colspan="1" rowspan="1">100%(3/)[]</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">21.59</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">20.85</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.19</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.82</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Flu vaccine:FlluMist</td><td colspan="1" rowspan="1">0.01%(v/v)</td><td colspan="1" rowspan="1">100%(3/3)]</td><td colspan="1" rowspan="1">100% (3/3)23.89</td><td colspan="1" rowspan="1">23.89</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">23.36</td><td colspan="1" rowspan="1">23.36 100% (3/3)</td><td colspan="1" rowspan="1">32.85</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.99</td></tr><tr><td colspan="1" rowspan="1">0.005%(v/v)</td><td colspan="1" rowspan="1">100%(3/) (11]</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">24.86</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">24.24</td><td colspan="1" rowspan="1">24.24 100% (3/3) 33.37</td><td colspan="1" rowspan="1">33.37</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.88</td></tr><tr><td colspan="1" rowspan="1">Analgesic(Iprofen)</td><td colspan="1" rowspan="1">220 μg/mL</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.30</td><td colspan="1" rowspan="1">33.30</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.69</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.00</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.48</td></tr><tr><td colspan="1" rowspan="1">Tobacco (CamelSnus)</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)32.93</td><td colspan="1" rowspan="1">32.93</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.99</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.85</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.82</td></tr><tr><td colspan="1" rowspan="1">Solvent DMSO</td><td colspan="1" rowspan="1">20% (v/v)</td><td colspan="1" rowspan="1">0% (0/6)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.53</td><td colspan="1" rowspan="1">100% (6/6)</td><td colspan="1" rowspan="1">32.74</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.69</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">31.27</td></tr><tr><td colspan="1" rowspan="1">Solvent Ethanol</td><td colspan="1" rowspan="1">20% (v/v)</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)32.34</td><td colspan="1" rowspan="1">32.34</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.79</td><td colspan="1" rowspan="1">32.79 100% (3/3) 34.01</td><td colspan="1" rowspan="1">34.01</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.60</td></tr><tr><td colspan="1" rowspan="1">VTM Remel M4RT</td><td colspan="1" rowspan="1">Neat[2]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.35</td><td colspan="1" rowspan="1">33.35</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.37</td><td colspan="1" rowspan="1">100% (3/3)34.73</td><td colspan="1" rowspan="1">34.73</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.10</td></tr><tr><td colspan="1" rowspan="1">VTM Remel M5</td><td colspan="1" rowspan="1">Neat{2]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.00</td><td colspan="1" rowspan="1">33.00</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.19</td><td colspan="1" rowspan="1">33.19 100% (3/3) 34.76</td><td colspan="1" rowspan="1">34.76</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.06</td></tr><tr><td colspan="1" rowspan="1">VTM Remel M6</td><td colspan="1" rowspan="1">Neat{2]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.31</td><td colspan="1" rowspan="1">33.31</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.05</td><td colspan="1" rowspan="1">33.05 100% (3/3)</td><td colspan="1" rowspan="1">35.42</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.15</td></tr><tr><td colspan="1" rowspan="1">VTM Copan UTM-RT</td><td colspan="1" rowspan="1">Neat{2]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.88</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">31.92</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.57</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.76</td></tr><tr><td colspan="1" rowspan="1">VTM BD UVT</td><td colspan="1" rowspan="1">Neat[2]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3) 31.77</td><td colspan="1" rowspan="1">31.77</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.40</td><td colspan="1" rowspan="1">33.40 100% (3/3)34.47</td><td colspan="1" rowspan="1">34.47</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.96</td></tr><tr><td colspan="1" rowspan="1">No ExtractionReagent CarryoverControl</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.43</td><td colspan="1" rowspan="1">33.43</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.97</td><td colspan="1" rowspan="1">100% (3/3)33.94</td><td colspan="1" rowspan="1">33.94</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.26</td></tr><tr><td colspan="1" rowspan="1">Extraction ReagentCarryover - SamplePlate</td><td colspan="1" rowspan="1">1% (v/v)[3]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.47</td><td colspan="1" rowspan="1">33.47</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.06</td><td colspan="1" rowspan="1">33.06 100% (3/3) 34.59</td><td colspan="1" rowspan="1">34.59</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.52</td></tr><tr><td colspan="1" rowspan="1">Extraction ReagentCarryover - Wash 1</td><td colspan="1" rowspan="1">1% (v/v)[3]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.67</td><td colspan="1" rowspan="1">33.67</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">33.44 100% (3/3)34.25</td><td colspan="1" rowspan="1">34.25</td><td colspan="1" rowspan="1">5 100% (3/3)</td><td colspan="1" rowspan="1">33.33</td></tr><tr><td colspan="1" rowspan="2">Extraction ReagentCarryover - Wash 2</td><td colspan="1" rowspan="1">6% (v/v)[3]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)33.90</td><td colspan="1" rowspan="1">33.90</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.13</td><td colspan="1" rowspan="1">3100% (3/3)34.02</td><td colspan="1" rowspan="1">34.02</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.54</td></tr><tr><td colspan="1" rowspan="1">3% (v/v)[3]</td><td colspan="1" rowspan="1">0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.32</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">32.79</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">34.41</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">33.33</td></tr></table>

[1] Negative samples containing FluMist were positive for Flu A and Flu B and negative for SARS-CoV-2 and RSV [2] Neat $=$ undiluted; AN samples originally collected in indicated VTM [3] Interferent added to eluates (post-extraction), and concentrations are per volume of eluate

# b. Competitive Interference

In this study, the ability to detect a co-infection when one targeted virus is present at a high concentration $( 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ and another targeted virus is present at a low concentration $( 3 \mathrm { x } \ \mathrm { L o D } )$ was assessed using contrived NP samples in VTM/UTM containing all possible pairings of influenza A, influenza B, SARS-CoV-2 and RSV. In parallel, viruses were tested at both levels individually to serve as controls and serve to demonstrate within-panel specificity of each of the target channels. Sample extraction was performed using the KingFisher Apex, and real-time RT-PCR was performed on the QS5Dx with one (1) lot of TaqPath COVID-19, Flu A, Flu B, RSV Select Panel reagents.

No off-target false-positive results were observed in these test conditions, which demonstrates within-panel specificity for each of the target channels. Negative, pooled NP specimens were tested alone (with no virus) as a negative control condition and yielded negative test results. As shown in Table 9, no competitive interference was observed at the concentrations tested.

Table 9. Summary of Competitive Interference Study Results   

<table><tr><td colspan="1" rowspan="2">Viral Test ConditionCombination</td><td colspan="2" rowspan="1">Low Target</td><td colspan="2" rowspan="1">High Target</td></tr><tr><td colspan="1" rowspan="1">Virus(Concentration)</td><td colspan="1" rowspan="1">nPositive/Total(% Positivity)</td><td colspan="1" rowspan="1">Virus(Concentration)</td><td colspan="1" rowspan="1">nPositive/Total(% Positivity)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (low),Influenza A (high)</td><td colspan="1" rowspan="1">SARS-CoV-2(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">Influenza A_(1.0 × 10sTCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (low),Influenza B (high)</td><td colspan="1" rowspan="1">SARS-CoV-2(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">Influenza B_(1.0 × 105$TCID50/mL</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (low),RSV (high)</td><td colspan="1" rowspan="1">SARS-CoV-2(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">RSV (1.0 × 105TCID50/mL</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (low),SARS-CoV-2 (high)</td><td colspan="1" rowspan="1">Influenza A(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">SARS-CoV-2(1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (low),Influenza B (high)</td><td colspan="1" rowspan="1">Influenza A(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">Influenza B(1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (low),RSV (high)</td><td colspan="1" rowspan="1">Influenza A(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">RSV (1.0 × 105TCID50/mL</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (low),SARS-CoV-2 (high)</td><td colspan="1" rowspan="1">Influenza B(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">SARS-CoV-2(1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (low),Influenza A (high)</td><td colspan="1" rowspan="1">Influenza B(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">Influenza A(1.0 × 10$$TCID50/mL</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (low),RSV (high)</td><td colspan="1" rowspan="1">Influenza B(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">RSV (1.0 × 105TCID50/mL</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">RSV (low), SARS-CoV-2 (high)</td><td colspan="1" rowspan="1">RSV (3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">SARS-CoV-2(1.0 × 10s TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">RSV (low), InfluenzaA (high)</td><td colspan="1" rowspan="1">RSV (3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">Influenza A_(1.0 × 10s$TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">RSV (low), InfluenzaB (high)</td><td colspan="1" rowspan="1">RSV (3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">Influenza B_(1.0 × 105$TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (low)</td><td colspan="1" rowspan="1">SARS-CoV-2(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (high)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">SARS-CoV-2(1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (low)</td><td colspan="1" rowspan="1">Influenza A(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Influenza A (high)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Influenza A(1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (low)</td><td colspan="1" rowspan="1">Influenza B(3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Influenza B (high)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Influenza B(1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">RSV (low)</td><td colspan="1" rowspan="1">RSV (3x LoD)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">RSV (high)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">RSV (1.0 × 105TCID50/mL)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Negative (NP alone)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

# c. Cross-Reactivity and Microbial Interference Wet-Testing

Cross-reactivity and microbial interference was assessed for the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel. Testing included microbes that are commonly found in upper respiratory samples, including commensal flora and pathogenic microbes. In the cross-reactivity portion of the study, contrived samples were prepared by spiking microbes into negative pooled nasopharyngeal (NP) swab samples in UTM/VTM to determine whether false positive results may be produced for any of the targets of the test. In addition to contrived samples, results from samples tested during the Clinical Validation study that had been called positive for coronavirus HKU1 (CoV-HKU1) by the comparator test were assessed for evidence of cross-reactivity with SARS-CoV-2 (USA-WA1/2020), influenza A (Michigan/45/15), influenza B (Florida/04/06) or RSV (CH93(18)-18). In the microbial interference portion of the study, the same microorganisms were tested in the presence of one representative viral strain for each of SARS-CoV-2, influenza A, influenza B, and RSV at 3x LoD to determine whether false negative results may be produced for the spiked targets. Each sample was tested in triplicate through the full workflow with one lot of reagents on the QS5Dx instrument. Bacteria and fungi were tested at $1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses were tested at $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , or the highest concentration possible. As shown in Table 10, no cross-reactivity or microbial interference was observed at the concentrations tested.

Table 10. Summary of Cross-Reactivity and Microbial Interference Study Results   

<table><tr><td colspan="1" rowspan="3">Organismtype</td><td colspan="1" rowspan="3">Species</td><td colspan="1" rowspan="3">TestConcentration</td><td colspan="1" rowspan="1">NegativeSamples</td><td colspan="4" rowspan="2">Number positive / total(% Positive)</td></tr><tr><td colspan="1" rowspan="2">NumberNegative /TTotal (%Negative)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">InfluenzaA</td><td colspan="1" rowspan="1">InfluenzaB</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="3">b</td><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Bordetellapertussis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="19">BacteriaBacteria</td><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Fusobacteriumnecrophorum</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae type b</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusacidophilus</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">5.4 x 107copies/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmagenitalium</td><td colspan="1" rowspan="1">4 x 105 cells/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata 1 x 106 CFU/mL</td><td colspan="1" rowspan="1">Neisseria elongata 1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitidisserogroup A</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus MRSA</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">StaphylococcusepidermidisMRSE</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">)3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="3">Fungi</td><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Pneumocystiscarinii</td><td colspan="1" rowspan="1">1 x 106 Cells/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="16">VirusVirus</td><td colspan="1" rowspan="1">Adenovirus type1, species </td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Mastadenovirus Btype 7, Gomen</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus(HHV-5)</td><td colspan="1" rowspan="1">3.2 x 104TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus type68</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus(HHV-4)</td><td colspan="1" rowspan="1">1.57 x 107copies/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3(100%)</td></tr><tr><td colspan="1" rowspan="1">Hepatitis B virus</td><td colspan="1" rowspan="1">4.2 x 107 IU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Hepatitis C virus</td><td colspan="1" rowspan="1">3.3 x 107 IU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Herpes simplexvirus 1 (HSV-1)</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Humancoronavirus 229E</td><td colspan="1" rowspan="1">2.82 x 104TCID5mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">HumancoronavirusHK1</td><td colspan="1" rowspan="1">2 x 106copies/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">HumancoronavirusHKU1</td><td colspan="1" rowspan="1">39 positiveclinicalspecimens</td><td colspan="1" rowspan="1">38/39(97.4%)2]</td><td colspan="1" rowspan="1">Not tested</td><td colspan="1" rowspan="1">Not tested</td><td colspan="1" rowspan="1">Not tested</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">Humancoronavirus NL63</td><td colspan="1" rowspan="1">2.34 x 104TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Humancoronavirus OC43</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Humanimmunodeficiencyvirus type IIIB</td><td colspan="1" rowspan="1">1 x 104 IU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (1100%)</td><td colspan="1" rowspan="1">3/3 100%)</td><td colspan="1" rowspan="1">3/3 (00%)</td></tr><tr><td colspan="1" rowspan="1">Humanmetapneumovirus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="11"></td><td colspan="1" rowspan="1">MERScoronavirus strainFlorida/USA-2Saudi Arabia 2014</td><td colspan="1" rowspan="1">Not provided [3]</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3 10%)</td><td colspan="1" rowspan="1">3 (110%)</td><td colspan="1" rowspan="1">3 (100%)</td></tr><tr><td colspan="1" rowspan="1">MERScoronavirus strainEMC/2012</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus type 1</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus type 2</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus type 3</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus type 4A</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%) 3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%) 3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus type1A</td><td colspan="1" rowspan="1">2.82 x 104TCID50/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">SARS coronavirusstrain 2003-00592</td><td colspan="1" rowspan="1">Not provided [3]</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">SARS coronavirusstrain Urbani</td><td colspan="1" rowspan="1">1 x 105 PFU/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Varicella-zostervirus (HHV-3)</td><td colspan="1" rowspan="1">1.94 x 108copies/mL</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%) 3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%) 3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Biofluid</td><td colspan="1" rowspan="1">Pooled humannasal wash</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)3/3 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr></table>

[1] Subgenomic synthetic control (ATCC VR-3262SD) [2] One sample produced a positive for SARS-CoV-2. [3] Concentration not provided by the vendor.

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# a. Quality Controls

Please refer to Instrument Description Information (Section C.4) above for assay controls.

# b. Specimen Stability

Specimen stability testing was performed on pooled, negative NP and AN swab specimens in VTM/UTM, contrived with influenza A (H1N1pdm, California/07/09), influenza B (Yamagata, Florida/04/06), SARS-CoV-2 (USA-WA1/2020) and RSV (RSVA, 2013 Isolate) individually spiked in each respective matrix at 3x LoD. Samples were stored at room temperature $( 3 0 ^ { \circ } \mathrm { C } )$ , refrigerated $( 8 ^ { \circ } \mathrm { C } )$ and frozen ( $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ and $- 7 0 ^ { \circ } \mathrm { C }$ ) temperatures for specified time periods prior to testing with the TaqPath SARS-CoV-2, Flu A, Flu B, RSV Select Panel. The test condition for samples stored at $3 0 \mathrm { { } ^ { \circ } C }$ included an additional storage of those samples at $8 ^ { \circ } \mathrm { C }$ for 80 hours after completion of the specified time periods at $3 0 \mathrm { { } ^ { \circ } C }$ . Five (5) replicates of samples for each storage condition in each specimen matrix were tested. Two (2) replicates of pooled negative NP and AN swab specimens in UTM/VTM were also tested at all storage conditions. Samples were extracted using KingFisher Apex Dx Purification System, and real-time PCR was performed on the QS5Dx with one lot of assay reagents.

All targets for both NP and AN specimens in VTM/UTM produced $100 \%$ positivity at all the time points tested for the $8 \mathrm { { } ^ { \circ } C }$ , $3 0 ^ { \circ } \mathrm { C }$ and the $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ temperatures. The AN specimens also demonstrated $100 \%$ agreement with the expected results for all the targets at all time points for the $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ storage condition. The specimen stability results are summarized in Tables 11-13 below.

These data support the claimed specimen stability in the Instruction for Use of 24 hours at room temperature $( 1 5 - 2 5 ^ { \circ } \mathrm { C } )$ , 72 hours refrigerated $( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } )$ and 30 days at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ .

Table 11. Results for Sample Stability at the $8 \mathrm { { } ^ { \circ } C }$ Storage Condition   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>StorageCondition</td><td rowspan=2 colspan=1>Timepoint</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>% Positive(nPositive/Total</td><td rowspan=1 colspan=1>% Positive(nPositive/Total</td><td rowspan=1 colspan=1>% Positive(nPositive/Total</td><td rowspan=1 colspan=1>% Positive(nPositive/Total</td><td rowspan=1 colspan=1>% Positive(nPositive/Total)</td></tr><tr><td rowspan=5 colspan=1>NP</td><td rowspan=5 colspan=1>8</td><td rowspan=1 colspan=1>0 Hour</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>48 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>80 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(14/14)*</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=5 colspan=1>AN</td><td rowspan=5 colspan=1>8</td><td rowspan=1 colspan=1>0 Hour</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%b (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>48 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>80 Hours</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr></table>

\*Includes a total of 4 replicates from the original run (1 replicate was excluded) and 10 replicates from Repeat 2.

Table 12. Results for Sample Stability at the $3 0 \mathrm { { ^ \circ C } }$ Storage Condition   

<table><tr><td colspan="1" rowspan="2">Matrix</td><td colspan="1" rowspan="2">StorageCondition</td><td colspan="1" rowspan="2">Timepoint</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">InfluenzaA</td><td colspan="1" rowspan="1">InfluenzaB</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">% Positive(nPositive/Total</td><td colspan="1" rowspan="1">% Positive(nPositive/Total</td><td colspan="1" rowspan="1">% Positive(nPositive/Total</td><td colspan="1" rowspan="1">% Positive(nPositive/Total</td><td colspan="1" rowspan="1">% Positive(nPositive/Total)</td></tr><tr><td colspan="1" rowspan="3">NP</td><td colspan="1" rowspan="3">30°</td><td colspan="1" rowspan="1">0 Hour</td><td colspan="1" rowspan="1">0% (0/2)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td></tr><tr><td colspan="1" rowspan="1">24 Hours</td><td colspan="1" rowspan="1">0% (0/2)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100%b (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td></tr><tr><td colspan="1" rowspan="1">27 Hours</td><td colspan="1" rowspan="1">0% (0/2)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td></tr><tr><td colspan="1" rowspan="2">AN</td><td colspan="1" rowspan="2">30</td><td colspan="1" rowspan="1">0 Hour</td><td colspan="1" rowspan="1">0% (0/2)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td></tr><tr><td colspan="1" rowspan="1">24 Hours</td><td colspan="1" rowspan="1">0% (0/2)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td><td colspan="1" rowspan="1">100% (5/5)</td></tr><tr><td></td><td></td><td>27 Hours</td><td>0% (0/2)</td><td>100% </td><td></td><td>575</td><td>100% (5/5)</td></tr></table>

Table 13. Positivity values for the $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ storage condition   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>StorageCondition</td><td rowspan=2 colspan=1>Timepoint</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>% Positive(nPositive/Total</td><td rowspan=1 colspan=1>% Positive(nPositive/Total)</td><td rowspan=1 colspan=1>% Positive(nPositive/Total)</td><td rowspan=1 colspan=1>% Positive(nPositive/Total</td><td rowspan=1 colspan=1>% Positive(nPositive/Total)</td></tr><tr><td rowspan=4 colspan=1>NP</td><td rowspan=4 colspan=1>≤-70°</td><td rowspan=1 colspan=1>0 day</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>15 days</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>33 days</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=4 colspan=1>AN</td><td rowspan=4 colspan=1>≤-70°</td><td rowspan=1 colspan=1>0 day</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>15 days</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr><tr><td rowspan=1 colspan=1>33 days</td><td rowspan=1 colspan=1>0% (0/2)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td><td rowspan=1 colspan=1>100% (5/5)</td></tr></table>

# c. Extraction Eluate Freeze-thaw Stability

The eluate stability study evaluated the performance of stored and frozen-and-thawed nucleic acid eluates with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel. NP and AN specimens in VTM/UTM were contrived with one strain individually of SARSCoV-2 (USA-WA1/2020), influenza A (California/07/09), influenza B (Florida/04/06) and RSV (RSV A, 2013 isolate) to $3 \mathbf { x }$ LoD prior to being extracted with the MagMAX Dx Viral/Pathogen Nucleic Acid Isolation Kit on the KingFisher Apex to collect the eluates. Fifteen replicates each of NP and AN specimen eluates were either tested fresh or tested after storage at $2 \mathrm { ^ { \circ } C }$ to $8 \mathrm { { } ^ { \circ } C }$ for up to 27 hours followed by storage at $\le - 7 0 ^ { \circ } \mathrm { C }$ for more than 6 days and undergoing two (2) freeze/thaw cycles. Stability was determined from three (3) time points. Testing was performed with one lot of the TaqPath COVID19, Flu A, Flu B, RSV Select Panel on the QS5 Dx instrument.

$100 \%$ of contrived positive samples produced the expected results at all three (3) time points evaluated. The results demonstrate that extracted sample eluates from contrived NP and AN swab samples containing SARS-CoV-2, influenza A, influenza B or RSV were stable when held for 27 hours at $8 \mathrm { { } ^ { \circ } C }$ followed by more than 6 days at $\le - 7 0 ^ { \circ } \mathrm { C }$ while undergoing two (2) freeze/thaw cycles. Therefore, the results support eluates storage claims at $2 \mathrm { ^ { \circ } C }$ to $8 \mathrm { { } ^ { \circ } C }$ for up to 24 hours, storage at $\le - 7 0 ^ { \circ } \mathrm { C }$ for up to 5 days and/or up to one (1) freeze/thaw cycle.

# d. Fresh vs Frozen Specimen Study

Fresh vs. frozen equivalency was assessed by testing contrived positive samples consisting of one representative strain of influenza A (H1N1pdm, Michigan/45/15), influenza B (Yamagata, Florida/04/06), SARS-CoV-2 (USA-WA1/2020) and RSV (RSVB, CH93(18)-18) at $2 \mathbf { x }$ and ${ 5 } \mathrm { x }$ their respective LoDs. Contrived samples were prepared in pooled, negative NP and AN specimens in VTM/UTM, respectively. Each sample for each condition was tested with twenty replicates $( \mathrm { N } { = } 2 0 )$ ) for the $2 \mathbf { x }$ LoD concentration and with ten replicates $( \mathrm { N } { = } 1 0$ ) for the 5x LoD concentration through the full TaqPath COVID-19, Flu A, Flu B, RSV Select Panel workflow. Testing was performed on freshly contrived samples and samples that had undergone two (2), three (3) and six (6) freeze/thaw cycles with one lot of reagents on the QS5Dx instrument. Additionally, negative samples were also tested for all the conditions with five replicates $\left( \mathrm { N } { = } 5 \right)$ . The results of this study support the use of frozen specimens in the clinical performance study and the claim in the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel Instructions for Use that specimens collected in Copan UTM Universal Transport Medium or BD Universal Viral Transport (UVT) Medium may undergo up to 2 freeze/thaw cycles. Do not exceed 1 freeze/thaw cycle for specimens collected in other transport media types.

# 6. Detection Limit:

The Limit of Detection (LoD) of the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel was determined for two (2) strains of influenza A H1N1, two (2) strains of influenza A H3N2, one (1) strain of influenza B of Victoria lineage, two (2) strains of influenza B of Yamagata lineage, one (1) strain of SARS-CoV- 2, one (1) strain of RSV A and one (1) strain of RSV B. Each virus was spiked into pooled, negative nasopharyngeal (NP) swab specimens in VTM/UTM, extracted on the KingFisher Apex Dx Purification System, then PCR was performed on the QS5Dx instrument across three (3) days, with two (2) reagent lots. LoDs for all viruses except the influenza B Victoria lineage strain were also determined in anterior nasal (AN) swab specimens in VTM/UTM.

The preliminary LoD was determined by probit analysis separately for each virus in each specimen type with each reagent lot, and the highest (i.e., worst-case) LoD value across reagent lots was selected as the preliminary LoD for that virus in the respective specimen type.

The preliminary LoDs determined by probit analysis were confirmed by testing at least 24 replicates at, above, and below the preliminary LoDs. Contrived specimens were prepared with negative clinical matrix (NP and AN swab specimens in VTM/UTM) and tested on the QS5Dx instrument. The LoD for the TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel is shown in Table 14 below.

Table 14. Limit of Detection for the TaqPath COVID‑19, Flu A, Flu B, RSV Select Panel   

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Subtype orLineage</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LoD in GCE/mL</td><td colspan="1" rowspan="1">LoD in TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA-WA1/2020</td><td colspan="1" rowspan="1">107 GCE/mL</td><td colspan="1" rowspan="1">0.158 TCID50/mL</td></tr><tr><td colspan="1" rowspan="4">Influenza A</td><td colspan="1" rowspan="2">H1N1</td><td colspan="1" rowspan="1">Michigan/45/15</td><td colspan="1" rowspan="1">340 GCE/mL</td><td colspan="1" rowspan="1">0.045 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">California/07/09</td><td colspan="1" rowspan="1">282 GCE/mL</td><td colspan="1" rowspan="1">0.026 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">H3N2</td><td colspan="1" rowspan="1">SouthAustralia/55/14</td><td colspan="1" rowspan="1">281 GCE/mL</td><td colspan="1" rowspan="1">0.034 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Texas/50/12</td><td colspan="1" rowspan="1">268 GCE/mL</td><td colspan="1" rowspan="1">0.079 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="2">Yamagata</td><td colspan="1" rowspan="1">Victoria/504/00</td><td colspan="1" rowspan="1">1405 GCE/mL</td><td colspan="1" rowspan="1">0.120 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Florida/04/06</td><td colspan="1" rowspan="1">464 GCE/mL</td><td colspan="1" rowspan="1">0.048 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Victoria</td><td colspan="1" rowspan="1">Malaysia/2506/2004</td><td colspan="1" rowspan="1">3500 GCE/mL</td><td colspan="1" rowspan="1">0.075 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">2013 Isolate</td><td colspan="1" rowspan="1">1614 GCE/mL</td><td colspan="1" rowspan="1">0.013 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">CH93(18)-18</td><td colspan="1" rowspan="1">1478 GCE/mL</td><td colspan="1" rowspan="1">0.046 TCID50/mL</td></tr></table>

# a. Limit of Detection with WHO Standard

The study was performed using two-phase approach to determine the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel LoD using the WHO Standard on the QS5Dx instrument. Contrived specimens were formulated with the First WHO International Standard for SARS-CoV-2 RNA spiked into pooled, negative anterior nasal (AN) swab specimens in VTM/UTM at various concentrations.

Once formulated, contrived specimens were extracted and detected using the TaqPath COVID- 19, Flu A, Flu B, RSV Select Panel in Phase I (range finding). The results of Phase I were used to establish the preliminary LoD (which was the lowest concentration to generate $100 \%$ positivity when tested in triplicate). In Phase II, the preliminary LoD was confirmed as the lowest concentration that results in at least $9 5 \%$ positive detection $( { \geq } 9 5 \%$ positive) when tested with 20 replicates. Phase II testing was performed in a blinded format.

The results of Phase I testing produced a preliminary LoD of $5 . 0 1 2 \times 1 0 ^ { 1 } \mathrm { I U / m L }$ , which was the lowest (most dilute) concentration that yielded $100 \%$ positivity. In Phase II, the LoD was confirmed at $1 5 0 . 3 6 \mathrm { I U / m L }$ as the lowest concentration that resulted $\mathrm { i n } \geq 9 5 \%$ positive detection as summarized in Table 15 below.

Table 15. Limit of Detection with SARS-CoV-2 WHO Standard   

<table><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Positive / TotalReplicates(% Positive)</td></tr><tr><td rowspan=1 colspan=1>451.08 IU/mL (9x Preliminary LoD)</td><td rowspan=1 colspan=1>20 / 20 (100%)</td></tr><tr><td rowspan=1 colspan=1>150.36 IU/mL (3x Preliminary LoD)</td><td rowspan=1 colspan=1>20 / 20 (100%)</td></tr><tr><td rowspan=1 colspan=1>50.12 IU/mL (Preliminary LoD)</td><td rowspan=1 colspan=1>18 /20(90%)</td></tr><tr><td rowspan=1 colspan=1>16.71 IU/mL (0.33x Preliminary LoD)</td><td rowspan=1 colspan=1>12 / 20(60%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0 /3 (0%)</td></tr></table>

# 7. Analytical Reactivity (Inclusivity)

# a. In-silico Inclusivity

Table 16 summarizes the genome sequences for SARS-CoV-2, influenza A virus, influenza B virus and respiratory syncytial virus that were downloaded from GISAID and GenBank on the indicated dates and analyzed for identity and melting temperature (Tm). Genomes with $100 \%$ identity and/or Tm greater than annealing temperature (Ta) for at least one assay were designated as reactive. For influenza A, wet testing with a predominant mismatch demonstrated no significant difference in detection between the mismatched sequence and reference sequence, therefore genomes with that mismatch were also predicted to be reactive.

Table 16. In silico Inclusivity (Reactivity) for the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Specimen collectiondate range</td><td rowspan=1 colspan=1>Sequences with 100%identity to ≥ 1 assay</td><td rowspan=1 colspan=1>Sequences with Tm &gt;Ta for ≥ 1 assay</td><td rowspan=1 colspan=1>Predictedinclusivity</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>All through 25 March2024 (GISAID andGenBank)</td><td rowspan=1 colspan=1>&gt;99.99%(11,485,741/11,486,294)</td><td rowspan=1 colspan=1>&gt;99.99%(11,486,283/11,486,294)</td><td rowspan=1 colspan=1>&gt;99.9%</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>1 January 2009through25 March 2024(GenBank) and 23April 2024 (GISAID)</td><td rowspan=1 colspan=1>83.76%(110,802/132,283)</td><td rowspan=1 colspan=1>89.6%(118,481/132,283)</td><td rowspan=1 colspan=1>95.27%[1]</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>1 January 2009through25 March 2024(GenBank) and 23April 2024 (GISAID)</td><td rowspan=1 colspan=1>98.56%(31,480/31,939)</td><td rowspan=1 colspan=1>&gt;99.9%(31,935/31,939)</td><td rowspan=1 colspan=1>&gt;99.9%</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>All through 25 March2024 (GISAID andGenBank)</td><td rowspan=1 colspan=1>87.07%(8,460/9,716)</td><td rowspan=1 colspan=1>99.49%(9,666/9,716)</td><td rowspan=1 colspan=1>99.49%</td></tr></table>

[1] Mismatch substitution was wet tested and showed no significant reduction in sensitivity compared with reference. Including strains with this mismatch in the counts produces a predicted inclusivity of $9 5 . 2 7 \%$ .

# b. Inclusivity Wet-Testing

Reactivity (inclusivity) was determined for eleven (11) strains of influenza A (H1N1), twelve (12) strains of influenza A (H3N2), five (5) strains of influenza B (two (2) of Victoria lineage and three (3) of Yamagata lineage), six (6) strains of RSV (three (3) each of RSV A and B), five (5) strains of SARS-CoV-2 and five (5) avian influenza strains (one (1) each of H5N1, H5N2, H7N2, H7N7 and H7N9). Each virus was introduced at near-LoD levels into negative, pooled NP swab specimens in VTM/UTM, then extracted and detected with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel. Sample extraction was performed using the KingFisher Apex, and real-time RTPCR was performed with the QS5Dx using one (1) lot of TaqPath COVID-19, Flu A, Flu B, RSV Select Panel reagents. Each sample was formulated with live virus, inactivated virus or genomic RNA, depending upon availability and tested in triplicate $( \mathrm { N } { = } 3 )$ . Virus strains that did not produce $100 \%$ detection at approximately 3x LoD were further tested at progressively higher concentrations until $100 \%$ detection was achieved. The results are summarized in Table 17.

Table 17. Reactivity (Inclusivity) Study Results   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">PositiveReplicates /Total (%Positivity</td><td colspan="2" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">GCE/mL</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Georgia/M5081/2012)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">42.83</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (New Caledonia/20/99)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.44</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (NY/03/09) [1],[2]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">4080</td><td colspan="1" rowspan="1">6.88</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Singapore/63/04)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.05</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Brisbane/02/2018) [2]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.11</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (SolomonIslands/3/2006)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">1.67</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Puerto Rico/08/1934)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.11</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Mexico/4108/09) [2]</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">7.75</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Taiwan/42/06)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.99</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Brisbane/59/07)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.07</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Victoria/2570/19)2]</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1020</td><td colspan="1" rowspan="1">0.03</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Wisconsin/67/2005)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.20</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2(Switzerland/9715293/13)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.007</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Kansas/14/2017)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.69</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Singapore/INFIMH-16-0019/16)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.57</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Perth/16/09)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.07</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Victoria/361/2011)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.02</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Hong Kong/8/68)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Brisbane/10/07)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.20</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (California/7/04)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">0.72</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Hong Kong/4801/14)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">2.32</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Hong Kong/2671/19)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1280</td><td colspan="1" rowspan="1">1.07</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Macha/O1237/2021)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">1280</td><td colspan="1" rowspan="1">2.62</td></tr><tr><td colspan="1" rowspan="1">Influenza B Victoria (Colorado/06/2017)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">4215[4]</td><td colspan="1" rowspan="1">0.08</td></tr><tr><td colspan="1" rowspan="1">Influenza B Victoria (Victoria/2/87)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">4215[4]</td><td colspan="1" rowspan="1">1.74</td></tr><tr><td colspan="1" rowspan="1">Influenza B Yamagata(Massachusetts/02/2012)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1392</td><td colspan="1" rowspan="1">0.02</td></tr><tr><td colspan="1" rowspan="1">Influenza B Yamagata (Phuket/3073/13)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1392</td><td colspan="1" rowspan="1">0.004</td></tr><tr><td colspan="1" rowspan="1">Influenza B Yamagata(Wisconsin/01/2010)</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1392</td><td colspan="1" rowspan="1">0.06</td></tr><tr><td colspan="1" rowspan="1">RSV A (Long)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">4842</td><td colspan="1" rowspan="1">4.56</td></tr><tr><td colspan="1" rowspan="1">RSV A (2014 Isolate 342)</td><td colspan="1" rowspan="1">3 /3 (100%)</td><td colspan="1" rowspan="1">4842</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">RSV A (2006 isolate)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">4842</td><td colspan="1" rowspan="1">0.05</td></tr><tr><td colspan="1" rowspan="1">RSV B (9320)[5]</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">4434</td><td colspan="1" rowspan="1">Unknown[3]</td></tr><tr><td colspan="1" rowspan="1">RSV B (3/2015 Isolate #2)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">4434</td><td colspan="1" rowspan="1">2.07</td></tr><tr><td colspan="1" rowspan="1">RSV B (12/2014 Isolate #1)</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">4434</td><td colspan="1" rowspan="1">0.56</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (Alpha Variant (B.1.1.7)England/204820464/2020)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">321</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (Beta (B.1.351) SouthAfrica/KRISP- K005325/2020)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">321</td><td colspan="1" rowspan="1">0.09</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (Gamma (P1) Japan/TY7-</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">321</td><td colspan="1" rowspan="1">0.03</td></tr><tr><td colspan="1" rowspan="1">503/2021)[5]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (Delta (B.1.617.2)USA/PHC658/2021)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">321</td><td colspan="1" rowspan="1">0.02</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (Omicron (BA.2.3)USA/MD-HP24556/2022)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">321</td><td colspan="1" rowspan="1">0.02</td></tr><tr><td colspan="1" rowspan="1">Avian influenza H5N1 (India/NIV/2006)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">Unknown[3]</td></tr><tr><td colspan="1" rowspan="1">Avian influenza H5N2 (pheasant/NewJersey/1355/1998)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">Unknown[3]</td></tr><tr><td colspan="1" rowspan="1">Avian influenza H7N2(turkey/Virginia/4529/2002)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">Unknown[3]</td></tr><tr><td colspan="1" rowspan="1">Avian influenza H7N7(mallard/Netherlands/12/2000)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">Unknown[3]</td></tr><tr><td colspan="1" rowspan="1">Avian influenza H7N9 (Anhui/1/2013)[5]</td><td colspan="1" rowspan="1">3 / 3 (100%)</td><td colspan="1" rowspan="1">1278</td><td colspan="1" rowspan="1">Unknown[3]</td></tr></table>

[1] Reactive with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel at $1 2 \mathrm { x } \mathrm { L o D }$ . [2] H1N1pdm09 strain. [3] $\mathrm { T C I D } 5 0 / \mathrm { m L }$ concentrations unknown (not reported by supplier). [4] $1 . 2 \mathbf { x }$ LoD relative to LoD of $3 5 0 0 \mathrm { G C E / m L }$ determined for influenza B Victoria lineage strain Malaysia/2506/2004. [5] Viral genomic RNA or inactivated virus.

# 8. RNase P Internal Control Cutoff Confirmation

In the RNase P Internal Control Cutoff Confirmation study, viral target-negative “used” (i.e., collected from individual donors and containing RNase P) nasopharyngeal (NP) swab specimens and “unused” (i.e., unopened and not containing RNase P) NP swabs were tested according to the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel Instructions for Use. The IC cutoff was confirmed at $\mathrm { C q } = 3 3 . 0$ while testing individually collected NP swabs. The results demonstrated an overall positive percent agreement (PPA) of $9 8 . 0 7 \%$ ( $9 5 \%$ CI of $9 7 . 3 1 { - } 9 8 . 6 3 \% )$ and negative percent agreement (NPA) of $100 \%$ $9 5 \%$ CI of $9 7 . 4 9 \mathrm { - } 1 0 0 \% $ ) for RNase $\mathrm { P }$ detection in NP swabs. For AN swabs, PPA for RNase $\mathrm { P }$ detection was $9 2 . 3 0 \%$ ( $9 5 \%$ Score CI of $9 0 . 8 3 \substack { - 9 3 . 5 4 \% } )$ and NPA was $100 \%$ $9 5 \%$ Score CI of $9 7 . 5 0 \mathrm { - } 1 0 0 \%$ ).

Due to the lower than expected PPA observed for AN specimens in the RNase P Cutoff confirmation study, an additional study was conducted utilizing an improved AN swab collection technique. All of the evaluated samples produced valid calls. Two (2) out of the one hundred thirty-eight (138) AN samples were positive for COVID-19, while the rest were negative for all viral targets. RNase P was detected in $100 \%$ of both negative and positive samples. The results are summarized in Table 18.

Table 18. Invalid Rate in AN Specimen Swab Collection Study Results   

<table><tr><td rowspan=2 colspan=1>Sample Call</td><td rowspan=2 colspan=1>RNase PCall</td><td rowspan=2 colspan=1>NumberofSamples</td><td rowspan=2 colspan=1>PercentValidity</td><td rowspan=1 colspan=3>RNase P (Cq)</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>2.1</td></tr></table>

This study has demonstrated that the improved AN swab collection technique reduces the invalid rate for AN swabs when tested with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel. The TaqPath COVID-19, Flu A, Flu B, RSV Select Panel Instructions for Use include the improved AN swab collection instructions.

# 9. Carry-Over:

The study was conducted using negative and contrived specimens of pooled nasopharyngeal (NP) swabs collected in VTM/UTM in a modified checkerboard pattern so that 12 positive samples per extraction event were included amongst 82 negative samples. Extraction was completed using contrived specimens with high-titer inactivated SARS-CoV-2 viral material and downstream real-time PCR with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel assay and workflow. The entire workflow was evaluated from sample extraction to real-time PCR results.

Six plates containing 12 high-positive samples (SARS-CoV-2 at $1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { P F U / m L }$ in pooled NP specimens in VTM/UTM) surrounded by negative NP samples were extracted by two operators on two different KingFisher Apex Dx Purification System instruments and tested sequentially with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel.

None of the 492 negative sample replicates tested produced a positive result for SARS-CoV2, demonstrating a carryover/cross-contamination rate of $0 \%$ .

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Refer to Clinical Studies Section of this document.

# 2. Matrix Comparison:

This study was conducted to establish matrix equivalency for the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel among nasopharyngeal (NP) and anterior nasal (AN) swab specimens collected in VTM/UTM.

Contrived samples prepared with one strain each of SARS-CoV-2, influenza A, influenza B and RSV. Forty (40) replicates at 5x LoD, 20 replicates at 2x LoD and 20 negative replicates were tested in NP and AN sample matrices with one lot of the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel reagents on the QS5Dx instrument. Matrix equivalency was demonstrated with a percent positivity of $100 \%$ at ${ 5 } \mathrm { x }$ LoD and $2 \mathbf { x }$ LoD for both matrices. All negative replicates gave negative results, producing a $0 \%$ rate of positivity (Table 19).

Table 19. Matrix Equivalency Study Result Summary   

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Virus Strain</td><td colspan="1" rowspan="1">Condition</td><td colspan="1" rowspan="1">Concentration(GCE/mL)</td><td colspan="1" rowspan="1">Matrix</td><td colspan="1" rowspan="1"># Positive/# Tested(% Positivity)</td></tr><tr><td colspan="1" rowspan="4">SARS-CoV-2</td><td colspan="1" rowspan="4">USA-WA1/2020</td><td colspan="1" rowspan="2">5x LoD</td><td colspan="1" rowspan="2">535</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="2">2x LoD</td><td colspan="1" rowspan="2">214</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="2">Influenza AH1N1</td><td colspan="1" rowspan="2">California/07/09</td><td colspan="1" rowspan="2">5x LoD</td><td colspan="1" rowspan="2">1410</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">2x LoD</td><td colspan="1" rowspan="2">564</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="4">Influenza B</td><td colspan="1" rowspan="4">Florida/04/06</td><td colspan="1" rowspan="2">5x LoD</td><td colspan="1" rowspan="2">2320</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="2">2x LoD</td><td colspan="1" rowspan="2">928</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="4">RSVA</td><td colspan="1" rowspan="4">2013 IsolateCulture Fluid</td><td colspan="1" rowspan="2">5x LoD</td><td colspan="1" rowspan="2">8070</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">20/20 (100%)</td></tr><tr><td colspan="1" rowspan="2">2x LoD</td><td colspan="1" rowspan="2">3228</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">40/40 (100%)</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="2">N/A</td><td colspan="1" rowspan="2">0x LoD</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">0/20 (0%)</td></tr><tr><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">0/20 (0%)</td></tr></table>

# C Clinical Studies:

# 1. Prospective Clinical Study

One NP swab and one AN swab was prospectively collected from each subject enrolled in the study from 14 diverse sites in the US. NP and AN swabs were collected in UTM, and the collection order (first specimen collected) was alternated between NP and AN swabs. Specimens were collected by trained healthcare professionals between February 20, 2023 and February 29, 2024. AN swabs were collected by swabbing both nares with the same swab, and NP swabs were collected from either of the two nostrils.

One thousand eight hundred and forty (1,840) subjects who were experiencing signs and symptoms of respiratory viral infection were enrolled into prospective collection of this study. Data collected from one duplicate enrollment was excluded from the study and two additional subjects withdrew from the study before samples could be collected. AN and NP samples were collected from each of the remaining 1,837 subjects. In addition, 54 AN swabs were excluded due to collection site protocol deviations, four AN swabs were excluded due to testing site protocol deviations, 132 AN swabs were excluded due to a contamination event at one of the testing sites, one AN swab was excluded due to invalid comparator test result, 105 AN swabs were excluded due to invalid TaqPath COVID-19, Flu A, Flu B, RSV Select Panel results. These exclusions resulted in 1,541 AN swabs that were included in final performance analysis, of which 1,192 specimens were tested fresh (Category I specimens) with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and 349 were tested after freezing (Category II specimens). Two (2) of the 1,541 AN specimens observed inconclusive comparator results for influenza A, leaving 1,539 evaluable AN specimens for influenza A target. Of the 1,840 subjects enrolled in the prospective study, 1,647 had their AN swab specimens evaluated with the candidate device. One hundred seventy nine (179) of these specimens were invalid by the candidate device during testing, for an initial invalid rate of $1 0 . 9 \%$ (179/1,647). Upon retesting, the invalid rate for AN specimens decreased to $6 . 4 \%$ (105/1,647).

Of the $1 , 8 3 7 \mathrm { N P }$ swabs that were collected, 55 were excluded due to collection site protocol deviations, three due to testing site protocol deviations, 132 due to contamination event at one of the testing sites and 27 due to invalid TaqPath COVID-19, Flu A, Flu B, RSV Select Panel results. A total of $1 { , } 6 2 0 \mathrm { N P }$ swabs were included in final performance analysis, of which 1,246 specimens were tested fresh (Category I specimens) with the TaqPath COVID19, Flu A, Flu B, RSV Select Panel and 374 were tested after freezing (Category II

specimens). Four (4) of the $1 { , } 6 2 0 \mathrm { N P }$ specimens observed inconclusive comparator results for Flu A, leaving 1,616 evaluable NP specimens for Flu A. Of the 1,840 subjects enrolled in the prospective study, 1,647 had their NP swab specimens evaluated with the candidate device. Forty five (45) of these specimens were invalid by the candidate device during testing, for an initial invalid rate of $2 . 7 \%$ (45/1,647). Upon retesting, the invalid rate for NP specimens decreased to $1 . 6 \%$ (27/1,647).

Demographics of subjects included in the prospective study that were included in the final performance analysis are summarized in Table 20.

Table 20. Demographics of Subjects from Prospective Cohort Study Included in the Final Performance Analysis   

<table><tr><td colspan="5" rowspan="1">Perlormance Analysis</td></tr><tr><td colspan="1" rowspan="1">Category</td><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">AN%</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">NP%</td></tr><tr><td colspan="5" rowspan="1">Sex</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">882</td><td colspan="1" rowspan="1">57.2%</td><td colspan="1" rowspan="1">924</td><td colspan="1" rowspan="1">57.0%</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">655</td><td colspan="1" rowspan="1">42.5%</td><td colspan="1" rowspan="1">692</td><td colspan="1" rowspan="1">42.7%</td></tr><tr><td colspan="1" rowspan="1">Prefer Not to Answer</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="5" rowspan="1">Ethnicity</td></tr><tr><td colspan="1" rowspan="1">Hispanic or Latino</td><td colspan="1" rowspan="1">363</td><td colspan="1" rowspan="1">23.6%</td><td colspan="1" rowspan="1">370</td><td colspan="1" rowspan="1">22.8%</td></tr><tr><td colspan="1" rowspan="1">Not Hispanic or Latino</td><td colspan="1" rowspan="1">1133</td><td colspan="1" rowspan="1">73.5%</td><td colspan="1" rowspan="1">1204</td><td colspan="1" rowspan="1">74.3%</td></tr><tr><td colspan="1" rowspan="1">Prefer not to answer</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">2.8%</td></tr><tr><td colspan="5" rowspan="1">Race</td></tr><tr><td colspan="1" rowspan="1">American Indian or Alaska Native</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1.0%</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1.0%</td></tr><tr><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">2.4%</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">2.4%</td></tr><tr><td colspan="1" rowspan="1">Black or African American</td><td colspan="1" rowspan="1">434</td><td colspan="1" rowspan="1">28.2%</td><td colspan="1" rowspan="1">470</td><td colspan="1" rowspan="1">29.0%</td></tr><tr><td colspan="1" rowspan="1">Native Hawaiian or Other Pacific Islander</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0.7%</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">1.5%</td></tr><tr><td colspan="1" rowspan="1">Prefer not to answer</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">3.8%</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">3.8%</td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">961</td><td colspan="1" rowspan="1">62.4%</td><td colspan="1" rowspan="1">996</td><td colspan="1" rowspan="1">61.5%</td></tr><tr><td colspan="1" rowspan="1">Age (yrs.)</td><td colspan="4" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Std Dev</td><td colspan="1" rowspan="1">19.9</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">19.9</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Min</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Max</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">N Categories</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="5" rowspan="1">Age Distribution</td></tr><tr><td colspan="1" rowspan="1">0-4 years</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">3.3%</td></tr><tr><td colspan="1" rowspan="1">5-19 years</td><td colspan="1" rowspan="1">252</td><td colspan="1" rowspan="1">16.4%</td><td colspan="1" rowspan="1">254</td><td colspan="1" rowspan="1">15.7%</td></tr><tr><td colspan="1" rowspan="1">20-39 years</td><td colspan="1" rowspan="1">447</td><td colspan="1" rowspan="1">29.0%</td><td colspan="1" rowspan="1">464</td><td colspan="1" rowspan="1">28.6%</td></tr><tr><td colspan="1" rowspan="1">40-59 year</td><td colspan="1" rowspan="1">504</td><td colspan="1" rowspan="1">32.7%</td><td colspan="1" rowspan="1">541</td><td colspan="1" rowspan="1">33.4%</td></tr><tr><td colspan="1" rowspan="1">60-79 years</td><td colspan="1" rowspan="1">274</td><td colspan="1" rowspan="1">17.8%</td><td colspan="1" rowspan="1">290</td><td colspan="1" rowspan="1">17.9%</td></tr><tr><td colspan="1" rowspan="1">≥80 years</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">1.1%</td></tr></table>

Results from the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel were compared against an FDA cleared molecular comparator test. The PPA and NPA for SARS-CoV-2, influenza A, influenza B and RSV for NP and AN specimens, along with the two-sided $9 5 \%$ confidence intervals were calculated using the Wilson Score method and are shown in Table 21. Samples that produced a discordant call between the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and the comparator test were further tested with another FDA cleared molecular test. Results from discordant sample testing are presented as footnotes below the table but were not used in performance calculations.

Table 21. Performance Estimates of TaqPath COVID-19, Flu A, Flu B, RSV Select Panel   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">SwabType</td><td colspan="1" rowspan="1">SpecimenCollection</td><td colspan="1" rowspan="1">Number ofSpecimens</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">FP</td><td colspan="1" rowspan="1">PPA(%)</td><td colspan="1" rowspan="1">95% Two-SidedConfidenceInterval</td><td colspan="1" rowspan="1">NPA(%)</td><td colspan="1" rowspan="1">95% Two-SidedConfidenceInterval</td></tr><tr><td colspan="1" rowspan="6">SARS-CoV-2</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1246</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1114</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">90.6% -98.9%</td><td colspan="1" rowspan="1">96.3%</td><td colspan="1" rowspan="1">95.0% -97.2%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">374</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">307</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">88.3% -99.1%</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">94.7% -98.5%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1620</td><td colspan="1" rowspan="1">142</td><td colspan="1" rowspan="1">5a</td><td colspan="1" rowspan="1">1421</td><td colspan="1" rowspan="1">52b</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">92.3% -98.5%</td><td colspan="1" rowspan="1">96.5%</td><td colspan="1" rowspan="1">95.4% -97.3%</td></tr><tr><td colspan="1" rowspan="3">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1192</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1068</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">96.4%</td><td colspan="1" rowspan="1">89.9% -98.8%</td><td colspan="1" rowspan="1">96.3%</td><td colspan="1" rowspan="1">95.0% -97.3%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">349</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">296</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">93.2%</td><td colspan="1" rowspan="1">81.8% -97.7%</td><td colspan="1" rowspan="1">97.1%</td><td colspan="1" rowspan="1">94.5% -98.4%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1541</td><td colspan="1" rowspan="1">121</td><td colspan="1" rowspan="1">6c</td><td colspan="1" rowspan="1">1364</td><td colspan="1" rowspan="1">50d</td><td colspan="1" rowspan="1">95.3%</td><td colspan="1" rowspan="1">90.1% -97.8%</td><td colspan="1" rowspan="1">96.5%</td><td colspan="1" rowspan="1">95.4% -97.3%</td></tr><tr><td colspan="1" rowspan="6">InfluenzaA</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1242</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1191</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">97.7%</td><td colspan="1" rowspan="1">88.2% -99.6%</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98.8% -99.7%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">374</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">356</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">86.7%</td><td colspan="1" rowspan="1">62.1% -96.3%</td><td colspan="1" rowspan="1">99.2%</td><td colspan="1" rowspan="1">97.6% -99.7%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1616</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">3e</td><td colspan="1" rowspan="1">1547</td><td colspan="1" rowspan="1">10f</td><td colspan="1" rowspan="1">94.9%</td><td colspan="1" rowspan="1">86.1% -98.3%</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">98.8% -99.7%</td></tr><tr><td colspan="1" rowspan="3">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1191</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1140</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">91.0% -100.0%</td><td colspan="1" rowspan="1">99.0%</td><td colspan="1" rowspan="1">98.2% -99.4%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">348</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">339</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">85.7%</td><td colspan="1" rowspan="1">48.7% -97.4%</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">97.9% -99.8%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1539</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">1g</td><td colspan="1" rowspan="1">1479</td><td colspan="1" rowspan="1">14h</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.7% -99.6%</td><td colspan="1" rowspan="1">99.1%</td><td colspan="1" rowspan="1">98.4% -99.4%</td></tr><tr><td colspan="1" rowspan="5">InfluenzaB</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1246</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1213</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">96.9%</td><td colspan="1" rowspan="1">84.3% -99.5%</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% -100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">374</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">355</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4% -100.0%</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.4% -100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1620</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">li</td><td colspan="1" rowspan="1">1568</td><td colspan="1" rowspan="1">2j</td><td colspan="1" rowspan="1">98.0%</td><td colspan="1" rowspan="1">89.5% -99.7%</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% -100.0%</td></tr><tr><td colspan="1" rowspan="2">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1192</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1160</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.7% -100.0%</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.4% -100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">349</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">332</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.6% -100.0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% -100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1541</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1492</td><td colspan="1" rowspan="1">2k</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92.4% -100.0%</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5% -100.0%</td></tr><tr><td colspan="1" rowspan="6">RSV</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1246</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1216</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">96.2%</td><td colspan="1" rowspan="1">81.1% -99.3%</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99.2% -99.9%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">374</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">368</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">83.3%</td><td colspan="1" rowspan="1">43.7% -97.0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.0% -100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1620</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">1584</td><td colspan="1" rowspan="1">4m</td><td colspan="1" rowspan="1">93.8%</td><td colspan="1" rowspan="1">79.9% -98.3%</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.4% -99.9%</td></tr><tr><td colspan="1" rowspan="3">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1192</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1163</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">86.2% -100.0%</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.0% -99.8%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">349</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">56.6% -100.0%</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.4% -100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1541</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1506</td><td colspan="1" rowspan="1">6n</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.3% -00.0%</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.1% -99.8%</td></tr></table>

$\mathrm { T P = }$ True Positive, $\mathrm { F N = }$ False Negative, $\mathrm { T N } =$ True Negative, $\mathrm { F P = }$ False Positive, $\mathrm { P P A } =$ Positive Percent Agreement, NP Negative Percent Agreement a Discordant test results based on another FDA cleared test: 3/5 SARS-CoV-2 Positive and 2/5 SARS-CoV-2 Negative b Discordant test result: 38/52 SARS-CoV-2 Positive and 14/52 SARS-CoV-2 Negative c Discordant test result: 5/6 SARS-CoV-2 Positive and 1/6 SARS-CoV-2 Negative d Discordant test result: 37/50 SARS-CoV-2 Positive and 13/50 SARS-CoV-2 Negative e Discordant test result: 1/3 Influenza A Positive and 2/3 Influenza A Negative f Discordant test result: 3/10 Influenza A Positive and 7/10 Influenza A Negative g Discordant test result: 1/1 Influenza A Negative h Discordant test result 4/14 Influenza A Positive and 10/14 Influenza A Negative I Discordant test result: 1/1 Influenza B Negative j Discordant test result: 1/2 Influenza B Positive and 1/2 Influenza B Negative k Discordant test result: 2/2 Influenza B Positive l Discordant test result: 2/2 RSV Negative m Discordant test result: 4/4 RSV Negative n Discordant test result: 1/6 RSV Positive and 5/6 RSV Negative

Number of samples with positive results for more than one target observed in the prospective cohort as detected by the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and comparator method are listed in Table 22.

Table 22. Number of NP and AN Swabs Samples that Detected Co-infections in the Prospective Sample Cohort   

<table><tr><td rowspan=2 colspan=1>Co-Infection Detected</td><td rowspan=1 colspan=2>Number of NP Samples</td><td rowspan=1 colspan=2>Number of AN Samples</td></tr><tr><td rowspan=1 colspan=1>SubjectDevice</td><td rowspan=1 colspan=1>ComparatorDevice</td><td rowspan=1 colspan=1>SubjectDevice</td><td rowspan=1 colspan=1>ComparatorDevice</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, Influenza A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, Influenza B</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, RSV</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Influenza A, RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Influenza B, RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, Influenza A, RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Influenza A, Influenza B, RSV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV2, Influenza A,Influenza B, RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total samples with co-infectionsdetected</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Samples positive for single target</td><td rowspan=1 colspan=1>312</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=1 colspan=1>Total Number of Samples</td><td rowspan=1 colspan=1>1,620</td><td rowspan=1 colspan=1>1,620</td><td rowspan=1 colspan=1>1,541</td><td rowspan=1 colspan=1>1,541</td></tr></table>

# 2. Enrichment Phase

After one thousand (1,000) subjects were enrolled in the prospective study and the desired minimum number of SARS-CoV-2 positive samples were collected, an enrichment phase of the study was initiated to supplement the prospective data for influenza A, influenza B and RSV. Three (3) of the fourteen (14) collection sites enrolled subjects for the enrichment phase. Subjects who were experiencing signs and symptoms of respiratory viral infection with a positive PCR test result for influenza A, influenza B and/or RSV within three (3) days prior to enrollment were included in the study.

A total of 69 subjects were enrolled during the enrichment phase of this study, one AN and one NP swab was collected from each of the 69 subjects. One AN swab was excluded from the performance analysis due to a collection site protocol deviation. Sixty-eight (68) AN and $6 9 \mathrm { N P }$ swabs were included in the performance analysis for the enriched sample cohort. Sixty (60) specimens were tested fresh with the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and nine (9) NP swabs and eight (8) AN swabs specimens were tested after freezing. No invalid results were observed in the enrichment phase for an invalid rate of $0 . 0 \%$ for both AN and NP specimens.

Demographics of subjects included in the enrichment phase are summarized in Table 23.

Table 23. Demographics of Subjects from the Enrichment Phase Cohort Included in the Performance Analysis   

<table><tr><td colspan="1" rowspan="1">Category</td><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">AN%</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">NP%</td></tr><tr><td colspan="5" rowspan="1">Sex</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">57.4%</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">58.0%</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">42.7%</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">42.0%</td></tr><tr><td colspan="1" rowspan="1">Prefer Not to Answer</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="5" rowspan="1">Ethnicity</td></tr><tr><td colspan="1" rowspan="1">Hispanic or Latino</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">80.9%</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">79.7%</td></tr><tr><td colspan="1" rowspan="1">Not Hispanic or Latino</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">19.1%</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">20.3%</td></tr><tr><td colspan="1" rowspan="1">Prefer not to answer</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="5" rowspan="1">Race</td></tr><tr><td colspan="1" rowspan="1">American Indian or Alaska Native</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Black or African American</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">11.8%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">11.6%</td></tr><tr><td colspan="1" rowspan="1">Native Hawaiian or Other Pacific Islander</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5%</td></tr><tr><td colspan="1" rowspan="1">Prefer not to answer</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">86.8%</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">Age (yrs.)</td><td colspan="4" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">43.6</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">43.9</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Std Dev</td><td colspan="1" rowspan="1">19.9</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">20.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Min</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Max</td><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">N Categories</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="5" rowspan="1">Age Distribution</td></tr><tr><td colspan="1" rowspan="1">0-4 years</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4.4%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4.4%</td></tr><tr><td colspan="1" rowspan="1">5-19 years</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7.4%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7.3%</td></tr><tr><td colspan="1" rowspan="1">20-39 years</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">25.0%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">24.6%</td></tr><tr><td colspan="1" rowspan="1">40-59 year</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">42.7%</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">42.0%</td></tr><tr><td colspan="1" rowspan="1">60-79 years</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">17.7%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">18.8%</td></tr><tr><td colspan="1" rowspan="1">≥80 years</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2.9%</td></tr></table>

Results from the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel were compared against an FDA cleared molecular comparator test. The PPA and NPA for Flu A, Flu B and RSV for NP and AN specimens along with the two-sided $9 5 \%$ confidence intervals were calculated using the Wilson Score method and are shown in Table 24. Samples that produced a discordant call between the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and the comparator test were further tested with another FDA cleared molecular test. Results from discordant sample testing are presented as footnotes in each table but were not used in performance calculations.

Table 24. Agreement of TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and Comparator test in the Enrichment Phase Cohort   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">SwabType</td><td colspan="1" rowspan="1">SpecimenCollection</td><td colspan="1" rowspan="1">Number ofSpecimens</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">FP</td><td colspan="1" rowspan="1">PPA(%)</td><td colspan="1" rowspan="1">95% TwoSidedConfidencInterval</td><td colspan="1" rowspan="1">NPA(%)</td><td colspan="1" rowspan="1">95% TwoSidedConfidencInterval</td></tr><tr><td colspan="1" rowspan="6">InfluenzaA</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.7%-99.6%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">78.5%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">34.2%-100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">64.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">1a</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">97.9%</td><td colspan="1" rowspan="1">89.1%-99.6%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">84.5%-100.0%</td></tr><tr><td colspan="1" rowspan="3">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">92.1%100.0%</td><td colspan="1" rowspan="1">93.3%</td><td colspan="1" rowspan="1">70.2%98.8%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">20.7%100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">64.6%100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">1b</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">92.3%100.0%</td><td colspan="1" rowspan="1">95.5%</td><td colspan="1" rowspan="1">78.2%99.2%</td></tr><tr><td colspan="1" rowspan="5">InfluenzaB</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">88.9%</td><td colspan="1" rowspan="1">56.5%98.0%</td><td colspan="1" rowspan="1">98.0%</td><td colspan="1" rowspan="1">89.7%99.7%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">51.0%100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">56.6%100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">1d</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">66.7%98.6%</td><td colspan="1" rowspan="1">98.2%</td><td colspan="1" rowspan="1">90.6%99.7%</td></tr><tr><td colspan="1" rowspan="2">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">88.9%</td><td colspan="1" rowspan="1">56.5%98.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.0%100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">51.0%100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">51.0%100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">1e</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">66.7%98.6%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.5%-100.0%</td></tr><tr><td colspan="1" rowspan="6">RSV</td><td colspan="1" rowspan="3">NPSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">64.6%100.0%</td><td colspan="1" rowspan="1">98.1%</td><td colspan="1" rowspan="1">90.1%99.7%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">43.9%100.0%</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">30.0%90.3%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">3f</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">72.3%-100.0%</td><td colspan="1" rowspan="1">94.9%</td><td colspan="1" rowspan="1">86.1%-98.3%</td></tr><tr><td colspan="1" rowspan="3">ANSwab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">64.6%100.0%</td><td colspan="1" rowspan="1">98.1%</td><td colspan="1" rowspan="1">90.1%99.7%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">43.9%100.0%</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">37.6%96.4%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">2g</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">72.3%-00.0%</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">88.3%-99.1%</td></tr></table>

$\mathrm { T P = }$ True Positive, $\mathrm { F N = }$ False Negative, $\mathrm { T N } =$ True Negative, $\mathrm { F P = }$ False Positive, $\mathrm { P P A } =$ Positive Percent Agreement, NPA= Negative Percent Agreement a Discordant test results: 1/1 Influenza A negative. b Discordant test result: 1/1 Influenza A negative. c Discordant test result: 1/1 Influenza B negative. d Discordant test result: 1/1 Influenza B positive. e Discordant test result: 1/1 Influenza B positive. f Discordant test result: 1/3 RSV positive and 2/3 RSV negative. g Discordant test result: 1/2 RSV positive and 1/2 RSV negative

Number of samples with positive results for more than one target observed in the enrichment phase as detected by the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel and comparator method are listed in Table 25.

Table 25. Number of NP and AN Swabs Samples That Detected Co-infections in the Enrichment Phase   

<table><tr><td rowspan=2 colspan=1>Co-Infection Detected</td><td rowspan=1 colspan=2>Number of NP Samples</td><td rowspan=1 colspan=2>Number of AN Samples</td></tr><tr><td rowspan=1 colspan=1>SubjectDevice</td><td rowspan=1 colspan=1>ComparatorDevice</td><td rowspan=1 colspan=1>SubjectDevice</td><td rowspan=1 colspan=1>ComparatorDevice</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, Influenza A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Influenza A, Influenza B</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Influenza A, RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total samples with co-infectionsdetected</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Samples positive for single target</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Total Number of Samples</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>68</td></tr></table>

# D Clinical Cut-Off:

The clinical cut-off study for target analytes was established in two phases. The first phase was a preliminary benchmarking investigation into the performance of the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel against two comparator devices to assess performance near the analytical limits of the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel. The second testing phase, the Receiver Operating Characteristic (ROC) justification study, was designed to further evaluate the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel with clinical and contrived samples against each comparator device for the purpose of defining the clinical cut-off prior to design validation. Based on these analyses, the assigned Cq cutoffs for the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel are shown in Table 26.

Table 26. Cq cutoff Values for Assay Targets   

<table><tr><td rowspan=1 colspan=1>Sample or Control</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Cq Cutoff</td></tr><tr><td rowspan=5 colspan=1>Positive Control</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Valid Cq values are ≤38</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Valid Cq values are ≤38</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Valid Cq values are ≤37</td></tr><tr><td rowspan=1 colspan=1>RSV A and RSV B</td><td rowspan=1 colspan=1>Valid Cq values are ≤38</td></tr><tr><td rowspan=1 colspan=1>RNase P</td><td rowspan=1 colspan=1>Valid Cq values are &gt;36</td></tr><tr><td rowspan=5 colspan=1>Negative Control</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Valid Cq values are &gt;38</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Valid Cq values are &gt;38</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Valid Cq values are &gt;37</td></tr><tr><td rowspan=1 colspan=1>RSV A and RSV B</td><td rowspan=1 colspan=1>Valid Cq values are &gt;38</td></tr><tr><td rowspan=1 colspan=1>RNase P</td><td rowspan=1 colspan=1>Valid Cq values are ≤33</td></tr><tr><td rowspan=5 colspan=1>Clinical Samples</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Valid Cq values are ≤38</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Valid Cq values are ≤38</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Valid Cq values are ≤37</td></tr><tr><td rowspan=1 colspan=1>RSV A and RSV B</td><td rowspan=1 colspan=1>Valid Cq values are ≤38</td></tr><tr><td rowspan=1 colspan=1>RNase P</td><td rowspan=1 colspan=1>Valid Cq values are ≤33</td></tr></table>

# E Expected Values/Reference Range:

The TaqPath COVID-19, Flu A, Flu B, RSV Select Panel prospective clinical study included a total of 1,840 prospectively collected NP and AN swab specimens, of which 1,541 AN swab specimens and $1 { , } 6 2 0 \mathrm { N P }$ swab specimens were evaluable. The number and percentage of cases positive for SARS-CoV-2, influenza A, influenza B, and RSV, as determined by the TaqPath COVID-19, Flu A, Flu B, RSV Select Panel, are presented in Table 27, stratified by collection site.

Table 27. TaqPath COVID-19, Flu A, Flu B, RSV Select Panel – Expected Values by Specimen Collection Site for NP and AN Swab Specimens   

<table><tr><td colspan="1" rowspan="2">Site</td><td colspan="2" rowspan="1">SARS-CoV-2</td><td colspan="2" rowspan="1">Influenza A</td><td colspan="2" rowspan="1">Influenza B</td><td colspan="2" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">AN</td><td colspan="1" rowspan="1">NP</td><td colspan="1" rowspan="1">AN</td></tr><tr><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">14.2%(23/162)</td><td colspan="1" rowspan="1">13.5%(20/148)</td><td colspan="1" rowspan="1">0.0%(0/162)</td><td colspan="1" rowspan="1">0.0%(0/148)</td><td colspan="1" rowspan="1">0.0%(0/162)</td><td colspan="1" rowspan="1">0.0%(0/148)</td><td colspan="1" rowspan="1">0.0%(0/162)</td><td colspan="1" rowspan="1">0.0%(0/148)</td></tr><tr><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">6.7%(9/135)</td><td colspan="1" rowspan="1">9.5%(12/126)</td><td colspan="1" rowspan="1">0.0%(0/135)</td><td colspan="1" rowspan="1">0.0%(0/126)</td><td colspan="1" rowspan="1">0.0%(0/135)</td><td colspan="1" rowspan="1">0.0%(0/126)</td><td colspan="1" rowspan="1">0.7%(1/135)</td><td colspan="1" rowspan="1">0.8%(1/126)</td></tr><tr><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">13.8%(49/356)</td><td colspan="1" rowspan="1">10.9%(38/348)</td><td colspan="1" rowspan="1">2.2%(8/356)</td><td colspan="1" rowspan="1">1.7%(6/348)</td><td colspan="1" rowspan="1">0.8%(3/356)</td><td colspan="1" rowspan="1">0.9%(3/348)</td><td colspan="1" rowspan="1">2.0%(7/356)</td><td colspan="1" rowspan="1">2.0%(7/348)</td></tr><tr><td colspan="1" rowspan="1">Site4</td><td colspan="1" rowspan="1">11.0%(9/82)</td><td colspan="1" rowspan="1">18.2%(14/77)</td><td colspan="1" rowspan="1">0.0%(0/82)</td><td colspan="1" rowspan="1">0.0%(0/77)</td><td colspan="1" rowspan="1">0.0%(0/82)</td><td colspan="1" rowspan="1">0.0%(0/77)</td><td colspan="1" rowspan="1">1.2%(1/82)</td><td colspan="1" rowspan="1">1.3%(1/77)</td></tr><tr><td colspan="1" rowspan="1">Site5</td><td colspan="1" rowspan="1">9.0%(17/188)</td><td colspan="1" rowspan="1">3.9%(7/178)</td><td colspan="1" rowspan="1">0.0%(0/188)</td><td colspan="1" rowspan="1">0.0%(0/178)</td><td colspan="1" rowspan="1">0.0%(0/188)</td><td colspan="1" rowspan="1">0.0%(0/178)</td><td colspan="1" rowspan="1">0.0%(0/188)</td><td colspan="1" rowspan="1">0.0%(0/178)</td></tr><tr><td colspan="1" rowspan="1">Site6</td><td colspan="1" rowspan="1">7.1%(13/182)</td><td colspan="1" rowspan="1">6.4%(11/171)</td><td colspan="1" rowspan="1">2.8%(5/182)</td><td colspan="1" rowspan="1">2.3%(4/171)</td><td colspan="1" rowspan="1">0.0%(0/182)</td><td colspan="1" rowspan="1">0.0%(0/171)</td><td colspan="1" rowspan="1">0.5%(1/182)</td><td colspan="1" rowspan="1">0.6%(1/171)</td></tr><tr><td colspan="1" rowspan="1">Site7</td><td colspan="1" rowspan="1">21.7%(10/46)</td><td colspan="1" rowspan="1">15.6%(7/45)</td><td colspan="1" rowspan="1">10.9%(5/46)</td><td colspan="1" rowspan="1">6.7%(3/45)</td><td colspan="1" rowspan="1">15.2%(7/46)</td><td colspan="1" rowspan="1">15.6%(7/45)</td><td colspan="1" rowspan="1">2.2%(1/46)</td><td colspan="1" rowspan="1">2.2%(1/45)</td></tr><tr><td colspan="1" rowspan="1">Site8</td><td colspan="1" rowspan="1">8.3%(1/12)</td><td colspan="1" rowspan="1">8.3%(1/12)</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">0.0%(0/12)</td></tr><tr><td colspan="1" rowspan="1">Site9</td><td colspan="1" rowspan="1">0.0%(0/10)</td><td colspan="1" rowspan="1">0.0%(0/7)</td><td colspan="1" rowspan="1">0.0%(0/10)</td><td colspan="1" rowspan="1">0.0%(0/7)</td><td colspan="1" rowspan="1">0.0%(0/10)</td><td colspan="1" rowspan="1">0.0%(0/7)</td><td colspan="1" rowspan="1">0.0%(0/10)</td><td colspan="1" rowspan="1">0.0%(0/7)</td></tr><tr><td colspan="1" rowspan="1">Site10</td><td colspan="1" rowspan="1">13.9%(28/202)</td><td colspan="1" rowspan="1">14.7%(29/197)</td><td colspan="1" rowspan="1">7.9%(16/202)</td><td colspan="1" rowspan="1">8.1%(16/197)</td><td colspan="1" rowspan="1">12.4%(25/202)</td><td colspan="1" rowspan="1">12.7%(25/197)</td><td colspan="1" rowspan="1">6.4%(13/202)</td><td colspan="1" rowspan="1">5.6%(11/197)</td></tr><tr><td colspan="1" rowspan="2">Site11</td><td colspan="1" rowspan="1">8.0%</td><td colspan="1" rowspan="1">6.6%</td><td colspan="1" rowspan="1">28.0%</td><td colspan="1" rowspan="1">30.3%</td><td colspan="1" rowspan="1">12.0%</td><td colspan="1" rowspan="1">11.8%</td><td colspan="1" rowspan="1">9.3%</td><td colspan="1" rowspan="1">11.8%</td></tr><tr><td colspan="1" rowspan="1">(6/75)</td><td colspan="1" rowspan="1">(5/76)</td><td colspan="1" rowspan="1">(21/75)</td><td colspan="1" rowspan="1">(23/76)</td><td colspan="1" rowspan="1">(9/75)</td><td colspan="1" rowspan="1">(9/76)</td><td colspan="1" rowspan="1">(7/75)</td><td colspan="1" rowspan="1">(9/76)</td></tr><tr><td colspan="1" rowspan="1">Site12</td><td colspan="1" rowspan="1">18.8%(15/80)</td><td colspan="1" rowspan="1">18.8%(15/80)</td><td colspan="1" rowspan="1">1.3%(1/80)</td><td colspan="1" rowspan="1">2.5%(2/80)</td><td colspan="1" rowspan="1">1.3%(1/80)</td><td colspan="1" rowspan="1">0.0%(0/80)</td><td colspan="1" rowspan="1">1.3%(1/80)</td><td colspan="1" rowspan="1">0.0%(0/80)</td></tr><tr><td colspan="1" rowspan="1">Site13</td><td colspan="1" rowspan="1">22.7%(5/22)</td><td colspan="1" rowspan="1">16.7%(3/18)</td><td colspan="1" rowspan="1">9.1%(2/22)</td><td colspan="1" rowspan="1">11.1%(2/18)</td><td colspan="1" rowspan="1">9.1%(2/22)</td><td colspan="1" rowspan="1">11.1%(2/18)</td><td colspan="1" rowspan="1">4.5%(1/22)</td><td colspan="1" rowspan="1">5.6%(1/18)</td></tr><tr><td colspan="1" rowspan="1">Site14</td><td colspan="1" rowspan="1">13.2%(9/68)</td><td colspan="1" rowspan="1">15.5%(9/58)</td><td colspan="1" rowspan="1">14.7%(10/68)</td><td colspan="1" rowspan="1">8.6%(5/58)</td><td colspan="1" rowspan="1">5.9%(4/68)</td><td colspan="1" rowspan="1">5.2%(3/58)</td><td colspan="1" rowspan="1">1.5%(1/68)</td><td colspan="1" rowspan="1">5.2%(3/58)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">12.0%(194/1,620)</td><td colspan="1" rowspan="1">11.1%(171/1,541)</td><td colspan="1" rowspan="1">4.2%(68/1,620)</td><td colspan="1" rowspan="1">4.0%(61/1,541)</td><td colspan="1" rowspan="1">3.1%(51/1,620)</td><td colspan="1" rowspan="1">3.2%(49/1,541)</td><td colspan="1" rowspan="1">2.1%(34/1,620)</td><td colspan="1" rowspan="1">2.3%(35/1,541)</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not Applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.