# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K241580   
B Applicant Abbott Molecular   
C Proprietary and Established Names Alinity m SARS-CoV-2   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QQX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - Deviceto Detect and IdentifyNucleic Acid Targets inRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-CoV-2 RespiratoryInfection and OtherMicrobial Agents When inA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to show that the Alinity m SARS-CoV-2 assay is substantially equivalent to the Roche cobas SARS-CoV-2 for use on the cobas 6800/8800 Systems assay (K213804) and to obtain clearance for the Alinity m SARS-CoV-2 assay.

# B Measurand:

Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) RNA

C Type of Test: Qualitative Real Time Polymerase Chain Reaction (RT-PCR)

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

Alinity m SARS-CoV-2 is a real-time in vitro reverse transcription polymerase chain reaction (RT-PCR) assay for use with the automated Alinity m System for the qualitative detection of nucleic acid from SARS-CoV-2 from patients with signs and symptoms of COVID-19 in nasopharyngeal (NP) swab and anterior nasal swab (ANS) specimens.

Results are for the detection and identification of SARS-CoV-2 RNA. Alinity m SARS-CoV-2 assay is intended for use as an aid in the diagnosis of COVID-19 if used in conjunction with other clinical, epidemiologic, and laboratory findings. Positive results are indicative of the presence of SARS-CoV-2 RNA; clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out bacterial infection or co-infection with other viruses.

Negative results do not preclude SARS-CoV-2 infection and should not be used as the sole basis for patient management decisions. Negative results must be combined with clinical observations, patient history, and epidemiological information.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D Special Instrument Requirements: All steps of the Alinity m SARS-CoV-2 assay procedure are executed on the Alinity m System

# IV Device/System Characteristics:

# A Device Description:

The Alinity m SARS-CoV-2 assay consists of 2 reagent kits:

• Alinity m SARS-CoV-2 AMP Kit (09N78-096) is comprised of 2 types of multi-well trays: Alinity m SARS-CoV-2 AMP TRAY 1 and Alinity m SARS-CoV-2 ACT TRAY 2. The intended storage condition for the Alinity m SARS-CoV-2 AMP Kit is $- 1 5 ^ { \circ } \mathrm { C }$ to $- 2 5 ^ { \circ } \mathrm { C }$ .

• Alinity m SARS-CoV-2 CTRL Kit (09N78-086) consists of negative controls and positive controls, each supplied as liquid in single-use tubes. The intended storage condition for the Alinity m SARS-CoV-2 Control Kit is $- 1 5 ^ { \circ } \mathrm { C }$ to $- 2 5 ^ { \circ } \mathrm { C }$ .

The Alinity m SARS-CoV-2 assay may utilize the following for collection and transport of anterior nasal swab specimens:

• Abbott Universal Collection Kit (09N92-030) consists of one Transport Tube with a solid cap containing $1 . 6 5 \mathrm { m L }$ Specimen Transport Buffer and one sterile Specimen Collection Swab. The intended storage condition for the Abbott Universal Collection Kit is $1 5 \mathrm { { ^ \circ C } }$ to $3 0 \mathrm { { } ^ { \circ } C }$ .

• Abbott Universal Collection Kit II (09N92-040) consists of one Transport Tube with a pierceable cap containing $1 . 6 5 \mathrm { m L }$ Specimen Transport Buffer, one sterile Specimen Collection Swab, and one absorbent pad. The intended storage condition for the Abbott Universal Collection Kit is $1 5 \mathrm { { ^ \circ C } }$ to $3 0 \mathrm { { } ^ { \circ } C }$ .

Alinity m SARS-CoV-2 is a real-time in vitro reverse transcription polymerase chain reaction (RT-PCR) assay for use with the automated Alinity m System for the qualitative detection of nucleic acid from SARS-CoV-2 in specimens collected from patients with signs and symptoms of COVID-19.

The steps of the Alinity m SARS-CoV-2 assay consist of sample preparation, RT-PCR assembly, amplification/detection, and result reporting. All stages of the Alinity m SARS-CoV-2 assay procedure are executed automatically by the Alinity m System. No intermediate processing or transfer steps are performed by the user. The Alinity m System is designed to be a randomaccess analyzer that can perform the Alinity m SARS-CoV-2 assay in parallel with other Alinity m assays on the same instrument.

SARS-CoV-2 RNA from specimens are extracted automatically on-board the Alinity m System using the Alinity m Sample Prep Kit 2, Alinity m Lysis Solution, and Alinity m Diluent Solution. The Alinity m System employs magnetic microparticle technology to facilitate nucleic acid capture, wash and elution. The resulting purified nucleic acids are then combined with the liquid unit-dose activation reagent, liquid unit-dose amplification reagents, and Alinity m Vapor Barrier Solution, and transferred by the instrument to an amplification/detection module for reverse transcription, PCR amplification, and real-time fluorescence detection.

The Alinity m SARS-CoV-2 amplification reagents include primers and probes that amplify and detect an exogenous internal control (containing an armored RNA sequence). Amplification and detection of the internal control demonstrates proper sample processing. The internal control is used to demonstrate assay validity.

Assay controls are tested to help ensure that instrument and reagent performance remain satisfactory. During each control event, a negative control and a positive control are processed through sample preparation and RT-PCR procedures that are identical to those used for specimens. Assay controls are used to demonstrate proper sample processing and assay validity. Each Alinity m SARS-CoV-2 CTRL kit contains 12 vials ( $1 . 3 ~ \mathrm { m L }$ fill volume) of Negative Control and 12 vials ( $1 . 3 ~ \mathrm { m L }$ fill volume) of Positive Control.

# Specimen Collection

Nasopharyngeal or anterior nasal swab specimens collected in viral transport media (BD UVT or Copan UTM). The Alinity m SARS-CoV-2 assay may utilize the following for collection and transport of anterior nasal swab specimens:

• Abbott Universal Collection Kit (09N92-030) • Abbott Universal Collection Kit II (09N92-040).

# B Principle of Operation:

The Alinity m SARS-CoV-2 assay utilizes real-time polymerase chain reaction (PCR) to amplify and detect genomic RNA sequences of SARS-CoV-2 from nasopharyngeal and anterior nasal swab specimens. The Alinity m SARS-CoV-2 is designed to target two highly conserved sequences within the SARS-CoV-2 genome: the RNA-dependent RNA polymerase (RdRp) and Nucleocapsid (N) genes. The RdRp and N probes utilized in the SARS-CoV-2 dual-target design are labeled with the same fluorophore. Therefore, amplification of both SARS-CoV-2 targets is detected as a single fluorescent signal during real-time PCR. The steps of the Alinity m SARSCoV-2 assay consist of sample preparation, PCR assembly, amplification/detection, and result calculation and reporting. All steps of the Alinity m SARS-CoV-2 assay procedure are executed automatically by the Alinity m System. The Alinity m System is designed to be a random-access analyzer that can perform the Alinity m SARS-CoV-2 assay in parallel with other Alinity m assays on the same instrument.

# C Instrument Description Information:

1. Instrument Name: Alinity m System.

2. Specimen Identification: Specimen identification information is entered either manually or via barcode.

3. Specimen Sampling and Handling:

The samples may be loaded on the system in any order. The system pipettor robot dispenses and aspirates liquids, as appropriate for each reaction. Sample handling and reagent transport is performed by a handler robot.

4. Calibration:

Not Applicable

5. Quality Control:

Alinity assays utilize quality controls to periodically monitor the performance of assay reagents and of the Alinity m System. A QC is defined as the set of assay controls that are, when valid, necessary to allow reporting of specimen results. A QC in this context consists of a negative control and positive control(s). Each assay control is processed through the same sample extraction and PCR procedure used for specimens.

# V Substantial Equivalence Information:

A Predicate Device Name(s): cobas SARS-CoV-2 Qualitative for use on the cobas 6800/8800 Systems.

B Predicate 510(k) Number(s): K213804

# C Comparison with Predicate(s):

The Alinity m SARS-CoV-2 assay has the same general intended uses as the predicate device cobas SARS-CoV-2 Qualitative for use on the cobas 6800/8800 Systems.

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=2>K241580</td><td rowspan=1 colspan=1>K213804</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=2>Alinity m SARS-CoV-2Assay</td><td rowspan=1 colspan=1>Roche cobas SARS-CoV-2Qualitative for use on thecobas 6800/8800 Systems</td></tr><tr><td rowspan=1 colspan=1>Regulation Number andProduct Code</td><td rowspan=1 colspan=2>21 CFR 866.3981; QQX</td><td rowspan=1 colspan=1>21 CFR 866.3981; QQX</td></tr><tr><td rowspan=1 colspan=1>Device Class</td><td rowspan=1 colspan=2>Ⅱ</td><td rowspan=1 colspan=1>Ⅱ</td></tr><tr><td rowspan=1 colspan=1>Technology/Detection</td><td rowspan=1 colspan=2>Real-Time ReverseTranscription- PolymeraseChain Reaction (RT-PCR)</td><td rowspan=1 colspan=1>Real-Time ReverseTranscription- PolymeraseChain Reaction (RT-PCR)</td></tr><tr><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=2>Alinity m System</td><td rowspan=1 colspan=1>cobas 6800/8800 Systems</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Intended Use/Indications forUse</td><td rowspan=2 colspan=2>Alinity m SARS-CoV-2 isa real-time in vitro reversetranscription polymerasechain reaction (RT-PCR)assay for use with theautomated Alinity mSystem for the qualitativedetection of nucleic acidfrom SARS-CoV-2 frompatients with signs andsymptoms of COVID-19 innasopharyngeal (NP) swaband anterior nasal swab(ANS) specimens.Results are for the detectionand identification of SARS-CoV-2 RNA. Alinity mSARS-CoV-2 assay isintended for use as an aid inthe diagnosis of COVID-19if used in conjunction withother clinical, epidemiologic,and laboratory findings.Positive results are indicativeof the presence of SARS-CoV-2 RNA; clinicalcorrelation with patienthistory and other diagnostic</td><td rowspan=2 colspan=1>cobas SARS-CoV-2Qualitative for use on thecobas 6800/8800 Systemsis a real-time RT-PCR testintended for the qualitativedetection of nucleic acidsfrom SARS-CoV-2 innasal and nasopharyngealspecimens collected fromsymptomatic individualssuspected of COVID-19by their healthcareprovider. Results are forthe detection of SARS-CoV-2 RNA. Positiveresults are indicative of thepresence of SARS-CoV-2RNA; clinical correlationwith patient history andother diagnosticinformation is necessary todetermine patient infectionstatus. Positive results donot rule out bacterialinfection or co- infectionwith other pathogens.Negative results do notpreclude SARS-CoV-2infection and should not beused as the sole basis for</td></tr><tr><td rowspan=1 colspan=1>SARS-</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>information is necessary todetermine patient infectionstatus. Positive results do notrule out bacterial infection orco-infection with otherviruses.Negative results do notpreclude SARS-CoV-2infection and should not beused as the sole basis forpatient managementdecisions. Negative resultsmust be combined withclinical observations, patienthistory, and epidemiologicalinformation.</td><td rowspan=1 colspan=1>patient managementdecisions. Results are meantto be used in conjunctionwith clinical observations,patient history, recentexposures andepidemiological information,and laboratory data, inaccordance with theguidelines provided by therelevant public healthauthorities. cobas SARS-CoV- 2 is intended for useby qualified clinicallaboratory personnelspecifically instructed andtrained in the techniques ofreal-time PCR and on the useof the cobas 6800/8800Systems.</td></tr><tr><td rowspan=1 colspan=1>Conditions for use</td><td rowspan=1 colspan=1>For prescription use</td><td rowspan=1 colspan=1>For prescription use</td></tr><tr><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>Nasopharyngeal swabAnterior nasal swab</td><td rowspan=1 colspan=1>Nasopharyngeal swabNasal swab</td></tr><tr><td rowspan=1 colspan=1>Assay Targets</td><td rowspan=1 colspan=1>SARS-CoV-2 RNA</td><td rowspan=1 colspan=1>SARS-CoV-2 RNA</td></tr><tr><td rowspan=1 colspan=1>Assay Steps</td><td rowspan=1 colspan=1>All steps of the AlinitySARS-CoV-2 assayprocedure are executedautomatically by theAlinity m System.No intermediate processingor transfer steps areperformed by the user.</td><td rowspan=1 colspan=1>All steps of the cobasSARS-CoV-2 qualitativeassay procedure areexecuted automatically bythe cobas 6800/8800Systems.No intermediate processingor transfer steps areperformed by the user.</td></tr><tr><td rowspan=1 colspan=1>Principles of the Procedure</td><td rowspan=1 colspan=1>Alinity m SARS-CoV-2 isa real-time in vitro reversetranscription polymerasechain reaction (RT-PCR)assay for use with theautomated Alinity mSystem for the qualitativedetection of nucleic acidfrom SARS-CoV-2 inNPS and ANS specimens.The steps of the Alinity mSARS-CoV-2 assayconsist of samplepreparation, RT-PCRassembly,amplification/detection,and result reporting. Allstages of the Alinity mSARS-CoV-2 assayprocedure are executed</td><td rowspan=1 colspan=1>Selective amplification ofRNA Internal Control isachieved by the use ofnon- competitive sequencespecific forward andreverse primers whichhave no homology withthe coronavirus genome. Athermostable DNApolymerase enzyme isused for amplification.The cobas SARS-CoV-2master mix containsdetection probes which arespecific for the coronavirustype SARS-CoV-2,members of theSarbecovirus subgenus,and the RNA InternalControl nucleic acid. Thecoronavirus and RNA</td></tr></table>

<table><tr><td>instrument. and elution. detection.</td><td>automatically by the Alinity m System. No intermediate processing or transfer steps are performed by the user. The Alinity m System is designed to be a random- access analyzer that can perform the Alinity m SARS-CoV-2 assay in parallel with other Alinity m assays on the same SARS-CoV-2 RNA from specimens are extracted automatically on-board the Alinity m System using the Alinity m Sample Prep Kit 2, Alinity m Lysis Solution, and Alinity m Diluent Solution. The Alinity m System employs magnetic microparticle technology to facilitate nucleic acid capture, wash The resulting purified nucleic acids are then combined with the liquid unit-dose activation reagent, liquid unit-dose amplification reagents, and Alinity m Vapor Barrier Solution, and transferred by the instrument to an amplification/detection module for reverse transcription, PCR amplification, and real- time fluorescence</td></tr></table>

<table><tr><td>Alinity m System. No intermediate processing or transfer steps are performed by the user. The Alinity m System is designed to be a random- access analyzer that can perform the Alinity m SARS-CoV-2 assay in parallel with other Alinity m assays on the same instrument. SARS-CoV-2 RNA from specimens are extracted automatically on-board the Alinity m System using the Alinity m Sample Prep Kit 2, Alinity m Lysis Solution, and Alinity m Diluent Solution. The Alinity m System employs magnetic microparticle technology to facilitate nucleic acid capture, wash and elution. The resulting purified nucleic acids are then combined with the liquid unit-dose activation reagent, liquid unit-dose amplification reagents, and Alinity m Vapor Barrier Solution, and transferred by the instrument to an amplification/detection module for reverse transcription, PCR amplification, and real- time fluorescence detection.</td><td>Internal Control detection probes are each labeled with unique fluorescent dyes that act as a reporter. Each probe also has a second dye which acts as a quencher. When not bound to the target sequence, the fluorescent signals of the intact probes are suppressed by the quencher dye. During the PCR amplification step, hybridization of the probes to the specific single- stranded DNA template results in cleavage of the probe by the 5&#x27; to 3&#x27; exonuclease activity of the DNA polymerase resulting in separation of the reporter and quencher dyes and the generation of a fluorescent signal. With each PCR cycle, increasing amounts of cleaved probes are generated and the cumulative signal of the reporter dye increases concomitantly. Each reporter dye is measured at defined wavelengths, which enables simultaneous detection and discrimination of the amplified coronavirus target and the RNA Internal Control. The master mix includes deoxyuridine triphosphate (dUTP), instead of deoxythimidine triphosphate (dTTP), which is incorporated into the newly synthesized DNA (amplicon). Any contaminating amplicons from previous PCR runs are destroyed by the AmpErase enzyme [uracil-N-glycosylase], which is included in the PCR mix, when heated in the first thermal cycling step. However, newly</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">inactivated once exposedto temperatures above55°C.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation SystemComponents</td><td colspan="1" rowspan="1">Alinity m System:High-throughput, fullyintegrated laboratoryautomation system whichutilize real-time PCRtechnology</td><td colspan="1" rowspan="1">cobas 6800/8800 Systems:High-throughput, fullyintegrated laboratoryautomation systems whichutilize real-time PCRtechnology</td></tr><tr><td colspan="1" rowspan="1">Sample Preparation InstrumentComponents</td><td colspan="1" rowspan="1">Automated liquid handlingand robotic manipulationplatform.</td><td colspan="1" rowspan="1">Automated liquid handlingand robotic manipulationplatform</td></tr><tr><td colspan="1" rowspan="1">Amplification and DetectionInstrument Components</td><td colspan="1" rowspan="1">The Amp-Detect units of theAlinity m System arecomprised of PCR thermalcycler/fluorescence readermodules that automate thesteps for real-time PCR.</td><td colspan="1" rowspan="1">The Analytic Modules of thecobas 6800/8800 Systemsare used for amplificationand detection of nucleic acidusing real-time PCR, whichis carried out by employingfluorescence spectroscopy.</td></tr><tr><td colspan="1" rowspan="1">Sample ExtractionTechnology</td><td colspan="1" rowspan="1">Non-specific nucleic acidcapture with magneticmicroparticlesMagnetic microparticlesare washed to removeunbound sample componentsincluding potential inhibitorsThe bound nucleic acidsare eluted and transferred tothe lyophilized master mix.An internal control (IC) istaken through the entiresample preparation and real-time PCR procedure alongwith the specimens,calibrators, and controls todemonstrate proper sampleprocessing and IC validity</td><td colspan="1" rowspan="1">Nucleic acid capture withmagnetic glass particlesMagnetic glassparticles are washed toremove unboundsubstances andimpurities, such asdenatured protein,cellular debris andpotential PCRinhibitors.The purified nucleicacids are eluted from theglass particles withelution buffer.An internal control (i.e.,DNA-QS) is taken throughthe sample preparationprocedure along with thespecimens for monitoring thesample preparation and PCRamplification process.</td></tr><tr><td colspan="1" rowspan="1">Amplification Controls</td><td colspan="1" rowspan="1">Assay specific IC. The ICprimer/probe set amplifiesand detects an exogenousRNA sequence unrelated tothe SARS-CoV-2 targetsequences</td><td colspan="1" rowspan="1">Selective amplification ofRNA Internal Control isachieved by the use of non-competitive sequencespecific forward and reverseprimers which have nohomology with thecoronavirus genome.</td></tr><tr><td colspan="1" rowspan="1">Detection Procedure</td><td colspan="1" rowspan="1">Optical detection ofstimulated fluorescence.The fluorescence readermonitors real-time</td><td colspan="1" rowspan="1">Optical detection ofstimulated fluorescence.The Analytic Modulemonitors real-time</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">fluorescence during everyPCR amplification cycle.</td><td colspan="1" rowspan="1">fluorescence during everyPCR amplification cycle.</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Fluorescence labeled,single stranded, target-specific probes.Detection of the DNAamplicon is achievedusing nucleic acid(DNA:DNA)hybridization.SARS-CoV-2 and Internalcontrol (IC) probes are labeledwith different fluorophoresallowing SARS- CoV-2 to bedistinguished from the ICamplicons.</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes(TaqMan Technology) usingfluorescence resonanceenergy transfer (FRET)</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Internal Control (IC)Negative ControlPositive Control</td><td colspan="1" rowspan="1">•Sample processing Control(IC)Negative controlPositive control</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen Collection andTransport</td><td colspan="1" rowspan="1">Nasopharyngeal samplesmust be collected in viraltransport media. Anteriornasal specimens must becollected in viral transportmedium, universal transportmedia, or Abbott UniversalCollection Kit, or AbbottUniversal Collection Kit II.</td><td colspan="1" rowspan="1">Nasopharyngeal swab andnasal swab, specimenscollected in viral transportmedium or universaltransport media.Nasal swab specimens mayalso be collected in cobasPCR Media Uni SwabSample Kit, cobas PCRMedia Dual Swab SampleKit, cobas PCR Media Kit(and 100 tube PCR MediaKit), or 0.9% PhysiologicalSaline.</td></tr><tr><td colspan="1" rowspan="1">Results Reporting</td><td colspan="1" rowspan="1">Not Detected, Negativexx.xx CN, Positive</td><td colspan="1" rowspan="1">Target 1 and Target 2Positive, DetectedTarget 1 Positive andTarget 2 Negative, DetectedTarget I Negative andTarget 2 Positive,Presumptive PositiveTargets 1 and 2 Negative,Not Detected</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981.

VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

a. Within-Laboratory Precision

Within-laboratory precision was evaluated at a single site for the Alinity m SARS-CoV-2 assay using Alinity m SARS-CoV-2 reagents and the Alinity m System with a total of three panels (negative samples, positive samples spiked with SARS-CoV-2 at 2X LoD, and 5X LoD) in a negative Simulated Nasal Matrix (SNM). Positive panel members were prepared by diluting gamma-irradiated SARS-CoV-2 virus in SNM. Un-spiked SNM was used as the SARS-CoV-2 negative panel member. Three operator/instrument/lot combinations were tested for each panel member using Alinity m SARS-CoV-2 Amplification Reagent Kits on five days with two runs per day and four replicates per run, resulting in a total of 120 replicates per panel member (40 replicates per operator/instrument/lot combination). Each Alinity m SARS-CoV-2 AMP Kit reagent lot was assigned to a specific Alinity m System and each operator was assigned to one lot/Alinity m System combination. Precision study results are summarized in Table 1.

Table 1. Within-Laboratory Precision Study Results   

<table><tr><td rowspan=1 colspan=1>TargetConcentration</td><td rowspan=1 colspan=1>Tested</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Rate</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative a</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(96.9%,100.0%)</td></tr><tr><td rowspan=1 colspan=1>2X LoD b(Low positive)</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>(95.4%,99.9%)</td></tr><tr><td rowspan=1 colspan=1>5X LoD(ModeratePositive</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(96.9%,100.0%)</td></tr></table>

a One replicate of negative sample was invalid and excluded from analysis. The sample was not retested as the minimum sample size of 90 was achieved. b One replicate of low positive sample was invalid and excluded from analysis. The sample was not retested as the minimum sample size of 90 was achieved.

Low positive panel members were positive at $9 9 . 2 \%$ (118/119) while moderate positive panel members were positive at $100 \%$ (120/120) for the spiked analyte. The negative panel members were $100 \%$ negative.

b. Within-Laboratory Precision – Testing Specimens in Universal Collection Kits Buffer

An additional precision study was carried out to assess precision by testing specimens in SNM in the Abbott Universal Collection Kits (UCKs) transport buffer. Precision was evaluated for the Alinity mSARS-CoV-2 assay by testing three panel members (negative samples, positive samples spiked with SARS-CoV-2 at 2X LoD, and 5X LoD) prepared in a negative matrix simulating clinical specimens collected in the Universal Collection

Kit or Universal Collection Kit II that contains transport buffer. Positive Panel Members were prepared by diluting gamma irradiated SARS-CoV-2 virus in SARS-CoV-2 negative Simulated Nasal Matrix (SNM) described previously. Un-spiked SNM in the UCKs transport buffer was used as the SARS-CoV-2 negative panel member. Target concentrations for each panel member and testing outline are summarized in Table 2.

Table 2. Precision Panel Members (Specimens in SNM in the Universal Collection Kits Transport Buffer) and Testing Plan   

<table><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>TargetConcentration</td><td rowspan=1 colspan=1>Number ofInstruments/Operators</td><td rowspan=1 colspan=1>Numberof Days</td><td rowspan=1 colspan=1>RunsperDay</td><td rowspan=1 colspan=1>Replicatesper Run</td><td rowspan=1 colspan=1>TotalReplicates</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2X LoD(Near LoD)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5X LoD(ModeratePositive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90</td></tr></table>

The precision study design comprised one sample preparation reagent and one amplification reagent pair on three instrument pairs by three operators with a minimum of six replicates per run and one testing run each day for a minimum of five days on each instrument/operator combination to ensure a minimum of 90 valid replicates per panel member. Results are summarized in Table 3.

Table 3. Precision Study Results for Specimens in SNM in the Universal Collection Kit Transport Buffer   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>TargetConcentration</td><td rowspan=1 colspan=1>Tested</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Rate</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>(95.9%,100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2X LoD(Near LoD)</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>(95.9%,100%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5X LoD(Moderate Positive)</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>(95.9%,100%)</td></tr></table>

All spiked panel members were at $100 \%$ positivity (90/90). The negative panel members were $100 \%$ negative. The results of the study demonstrate acceptable assay variability testing samples collected in Universal Collection Kit buffer.

# c. Reproducibility

Reproducibility of the Alinity m SARS-CoV-2 assay was evaluated at three external testing sites by testing a three-member panel prepared in SNM. The two positive panel members consisted of one moderate positive panel member containing gamma irradiated SARS-CoV-2 virus at approximately 5X LoD and one low positive panel member containing gamma irradiated SARS-CoV-2 virus at approximately 2X LoD. Negative SNM was used as a negative panel member.

A total of three Alinity m SARS-CoV-2 AMP Kit lots were used. Each of the three external sites tested two Alinity m SARS-CoV-2 AMP Kit lots, on five non-consecutive days for each lot. Four replicates of each panel member were tested on each of five days. Each of the three external sites used different lots of Alinity m SARS-CoV-2 CTRL Kits and Alinity m Sample Prep Kit 2. Reproducibility results are summarized in Table 4.

Table 4. Reproducibility Study Summary Results   

<table><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Between Day</td><td rowspan=1 colspan=2>WithinLaboratory</td><td rowspan=1 colspan=2>BetweenSite/Instrument</td><td rowspan=1 colspan=2>Totald</td></tr><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Na</td><td rowspan=1 colspan=1>Nb</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>MeanCN</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Moderate Psitive(5x LoD)</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.535</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>LowPositive(2x LoD)</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>35.78</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.681</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

a Total number of valid replicates. b Replicates with positive result interpretation for positive panels and negative result interpretation for negative panel. Number of replicates used in the Mean and SD calculation for the positive panel members. c Within-laboratory includes Within-Run, Between-Run, and Between-Day Components. d Total includes Within-Run, Between-Run, Between-Day, and Between-Instrument Components.

Both the Low Positive and Moderate Positive Panels were positive $100 \%$ of the time. The negative panel was negative $100 \%$ of the time. The results of the study demonstrate acceptable assay reproducibility.

2. Linearity:

A Linearity Study is not applicable because the Alinity m SARS-CoV-2 assay is a qualitative assay.

# 3. Analytical Reactivity (Inclusivity):

The inclusivity of the Alinity m SARS-CoV-2 Assay for the detection of SARS-CoV-2 was evaluated using a combination of in silico analysis of publicly available sequence information and laboratory (wet) testing of seven isolates of SARS-CoV-2 from six different geographical regions. Seven different strains/isolates were tested at 3X the assay’s LoD and were prepared in negative nasopharyngeal matrix with either purified genomic RNA from the SARS-CoV-2 isolates or inactivated virus. Each strain/isolate was tested in five replicates and results are presented in Table 5.

Table 5. SARS-CoV-2 Inclusivity Wet Testing Results.   

<table><tr><td colspan="1" rowspan="1">SARS-CoV-2 Strain (isolate)</td><td colspan="1" rowspan="1">Concentration a</td><td colspan="1" rowspan="1">Tested(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">DetectionRate</td></tr><tr><td colspan="1" rowspan="1">USA-AZ1/2020</td><td colspan="1" rowspan="1">165 GE/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr><tr><td colspan="1" rowspan="1">USA-CA3/2020</td><td colspan="1" rowspan="1">165 GE/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr><tr><td colspan="1" rowspan="1">Hong Kong/VM20001061/2020</td><td colspan="1" rowspan="1">165 GE/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr><tr><td colspan="1" rowspan="1">USA-IL1/2020</td><td colspan="1" rowspan="1">165 GE/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr><tr><td colspan="1" rowspan="1">Italy-INMI1</td><td colspan="1" rowspan="1">165 GE/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr><tr><td colspan="1" rowspan="1">USA/CA_CDC_5574/2020(B.1.1.7)</td><td colspan="1" rowspan="1">0.027TCID50/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr><tr><td colspan="1" rowspan="1">hCoV-19/South Africa/KRISP-K005325/2020 (B.1.351)</td><td colspan="1" rowspan="1">0.027TCID50/mL</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(5/5)</td></tr></table>

a GE/mL $=$ Genome Equivalent/mL or $\mathrm { T C I D } { \scriptstyle 5 0 } / \mathrm { m L } =$ Median Tissue Culture Infectious $\mathrm { D o s e / m L }$

# In Silico

The inclusivity of the Alinity m SARS-CoV-2 Assay was evaluated using in silico analysis of the primers and probes for the ability to detect various strains of the SARSCoV-2 virus. Primer and probe sequences for SARS-CoV-2 detection were analyzed for homology with sequences available in the Global Initiative on Sharing All Influenza Data (GISAID, as of October 11, 2023) and National Center for Biotechnology Information (NCBI, available as of October 10, 2023) databases and the potential impact of any mismatches and the ability to detect SARS-CoV-2 were evaluated.

Inclusivity was demonstrated by analyzing the sequences of the RdRp and N primer/probe sets for homology with 14,818,776 unambiguous SARS-CoV-2 sequences available in the GISAID database. In addition, an analysis was performed for the sequences that have variant of concern (VOC) and variant of interest (VOI) variant designation. Of the 14,818,776 total available sequences, 13,680,493 have current (as of October 10, 2023) or former VOC, VOI, and variant under monitoring (VUM) designation including Omicron EG.5, Omicron B.1.640, Omicron XBB.1.16, Omicron XBB.1.5, Omicron BA.2.75, Omicron BA.2.86, Omicron CH.1.1, Omicron XBB.1.9.1, Omicron XBB.1.9.2, Omicron XBB.2.3, Omicron XBB, Alpha, Beta, Delta, Gamma, Epsilon, Eta, Iota, Kappa, Lambda, Mu, Theta, and Zeta, among others.

In silico analyses were also performed with the full-length SARS-CoV-2 sequences available in the NCBI database A total of 7,590,332 unambiguous sequences in the NCBI database were analyzed for SARS-CoV-2 inclusivity.

The sequences were evaluated for a mismatch in either RdRp or N gene target regions. The impact of any mismatches on the amplification and detection of each target region was assessed.

# GSAID Database

14,764,951 $( 9 9 . 6 4 \% )$ available sequences in the GISAID database either had no mismatches or had mismatches in one of the assay target regions. These sequences were predicted to have no impact on the detection of SARS-CoV-2. The remaining 53,825 $( 0 . 3 6 \% )$ sequences contained at least one mismatch in both of the assay target regions. 53,754 of these sequences were predicted to be unlikely to impact the detection of SARSCoV-2.

Among the 13,680,493 sequences available in the GISAID database that had VOC and VOI designation, 13,628,903 sequences $( 9 9 . 6 \% )$ either had no mismatches in the assay target regions or had mismatches in one of the target regions. One VUM Omicron (XBB. $1 . 9 . 1 \mathrm { + } \mathrm { X B B } . 1 . 9 . 1 . ^ { \ast } )$ , one former VOC Alpha, six former VOC Omicron $( \mathrm { B } . 1 . 1 . 5 2 9 + \mathrm { B A } . ^ { * } )$ ) variants, and 59 former VOC Delta variants are potentially impacted. Considering the extremely low prevalence of those potentially impacted, and de-escalated status of most impacted variants, the analysis predicts no impact on the detection of SARS-CoV-2 among former or current VOC, VOI and VUM.

# NCBI Database

Among the 7,590,332 unambiguous sequences, 7,559,687 $( 9 9 . 6 \% )$ have either no mismatches in the assay target regions (RdRp and $_ \mathrm { N }$ gene targets) or have mismatches in only one of the target regions. These sequences are predicted not to impact the detection of SARS-CoV-2. The remaining 30,645 $( 0 . 4 \% )$ isolates contained at least one mismatch in both the RdRp and N gene target regions.

The overall potential impact on SARS-CoV-2 detection is considered likely only when both RdRp and N genes are potentially impacted. Among the 30,645 sequences containing at least one mismatch in both target regions, 30,602 were predicted unlikely to impact the detection of SARS-CoV-2. Among the SARS-CoV-2 sequences in the NCBI database, 43 isolates are predicted to be potentially impacted by mismatches.

The majority of sequences evaluated for SARS-CoV-2 are expected to be detected by the Alinity m SARS-CoV-2 Assay.

A summary of the in-silico analyses is presented in Table 6.

Table 6. Summary of In-Silico Analysis of SARS-CoV-2 Oligo Designs   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>NCBI</td><td rowspan=1 colspan=2>GISAID</td></tr><tr><td rowspan=1 colspan=1>Total Number of Sequences</td><td rowspan=1 colspan=2>7,590,332</td><td rowspan=1 colspan=2>14,818,776</td></tr><tr><td rowspan=1 colspan=1>Mismatch in Both RdRp and N Gene</td><td rowspan=1 colspan=1>30645</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=1>53825</td><td rowspan=1 colspan=1>0.36%</td></tr><tr><td rowspan=1 colspan=1>Potentially Impacted a</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>0.00%</td></tr></table>

a Sequences are considered potentially impacted only when both targets (RdRp and N gene) are likely impacted.

# 4. Analytical Specificity/Interference:

# Cross-Reactivity

This study evaluated the analytical specificity (cross-reactivity) of the Alinity m SARS-CoV-2 Assay in the presence of non-targeted microorganisms that may be found in a respiratory tract clinical specimen. Fifty-three microorganisms listed in Table 7 were evaluated in the study and were inoculated into SARS-CoV-2 negative samples to achieve a final titer of $1 0 ^ { 5 }$ units/mL for viruses and fungi, $1 0 ^ { 6 }$ units/mL for bacteria, and $10 \%$ of pooled human nasal wash. Whole microorganisms were used in this study whenever possible. Purified nucleic acid was utilized in cases where the microorganism was not available. The SARS-CoV-2 negative sample was also tested without addition of any potential cross reactants as the control condition. The negative sample was prepared by pooling SARS-CoV-2 negative nasopharyngeal swabs (NPS) in UTM/VTM. Each test condition and the control condition were evaluated in triplicate with the Alinity m SARS-CoV-2 assay. The results of the study are summarized in Table 8.

Table 7. List of Potential Cross-Reactive Organisms Tested   

<table><tr><td colspan="1" rowspan="1">Organism Tested</td><td colspan="1" rowspan="1">Testing Concentration(Units/mL)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E 1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1 3</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL631</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus 3</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus 3</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 52</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV) 2</td><td colspan="1" rowspan="1">1.00E+05 U/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 42</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1) 1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2) 1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus Type B 1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus EV68 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus Type A 1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae 4</td><td colspan="1" rowspan="1">1.00E+06 IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii (PJP) (2%) 4</td><td colspan="1" rowspan="1">N/Aa</td></tr><tr><td colspan="1" rowspan="1">Pooled human nasal wash (10%) 5</td><td colspan="1" rowspan="1">N/Ab</td></tr><tr><td colspan="1" rowspan="1">Candida albicans 4</td><td colspan="1" rowspan="1">1.00E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermis 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii (PJP) (100%) 4</td><td colspan="1" rowspan="1">N/Aa</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus 4</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus 4</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Synthetic Human; bocavirus DNA 6</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV) 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV) 2</td><td colspan="1" rowspan="1">1.00E+05 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Measles 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parechovirus Type 3 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetti 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus (plantarum 17-5) 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae, Long Beach44</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus 4</td><td colspan="1" rowspan="1">1.00E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila psittaci 4</td><td colspan="1" rowspan="1">1.00E+06 IFU/mL</td></tr></table>

a The concentration provided by the vendor is in terms of Ct Range, 23 – 25. This sample was tested neat and as a dilution $100 \%$ and $2 \%$ ) in this study. b The pooled human nasal wash was tested at 1 concentration, $10 \%$ , in this study.

1 Viral particles   
2 Viral lysate   
3 Viral RNA   
4 Whole microorganism   
5 Nasal wash   
6 Synthetic nucleic acid

Table 8. Cross-Reactivity Study Results   

<table><tr><td colspan="1" rowspan="1">Organism Tested</td><td colspan="1" rowspan="1">Tested(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">Invalid(N)</td></tr><tr><td colspan="1" rowspan="1">Control (no cross-reactant)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus Type B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Enterovirus EV68</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus Type A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii (2%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Pooled human nasal wash</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr></table>

None of the potentially cross-reactive viruses or microorganisms evaluated in this study generated a false positive SARS-CoV-2 result when tested with the Alinity m SARS-CoV-2 assay.

# Microbial Interference

The Alinity m SARS-CoV-2 assay was also evaluated for interference with the organisms listed in Table 9. The organisms were spiked into SARS-CoV-2 positive samples to achieve a final titer of $1 0 ^ { 5 }$ units/mL for viruses and fungi, $1 0 ^ { 6 }$ units/ml for bacteria, and $10 \%$ of pooled human nasal wash. A SARS-CoV-2 positive sample was also tested without addition of any potential interfering organisms as the control condition.

The SARS-CoV-2 positive samples at 3X LoD were prepared by spiking gamma irradiated SARS-CoV-2 virus into pooled SARS-CoV-2 negative clinical nasopharyngeal swab (NPS) specimens in UTM/VTM. Three replicates of each test condition and the control condition were tested with the Alinity m SARS-CoV-2 assay. The results are summarized in Table 9.

Table 9. Microbial Interference Study Results   

<table><tr><td colspan="1" rowspan="1">Organism Tested</td><td colspan="1" rowspan="1">Tested(N)</td><td colspan="1" rowspan="1">Positive(N</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">Invalid(N)</td></tr><tr><td colspan="1" rowspan="1">Control (no cross-reactant)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus Type B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Enterovirus EV68</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus Type A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii (2%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Pooled human nasal wash</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii (100%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus (Whole org)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus (Whole org)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Control (no cross-reactant)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Synthetic Human bocavirus DNA</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Parechovirus Type 3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetti</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus (plantarum 17-5)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae,Long Beach 4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila psittaci</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr></table>

None of the potentially interfering organisms evaluated in this study impacts the detection of SARS-CoV-2 virus when tested with the Alinity m SARS-CoV-2 assay.

Interfering Substances – Nasopharyngeal Swab (NPS) Specimens in UVT/UTM

An analytical study was performed to assess the potential inhibitory effects of exogenous and endogenous substances that may be found in NPS specimens, on the performance of the Alinity m SARS-CoV-2 assay. A panel of 34 potentially inhibitory exogenous and endogenous substances were added into SARS-CoV-2 low positive samples (3XLoD) and tested using the Alinity m SARS-CoV-2 assay. A SARS-CoV-2 positive sample was also tested without the

addition of any potentially interfering endogenous and exogenous substances (Control Condition).

The SARS-CoV-2 low positive samples at 3X LoD were prepared by spiking gamma irradiated SARS-CoV-2 virus into pooled SARS-CoV-2 negative clinical nasopharyngeal swab (NPS) specimens in UVT/UTM. Three replicates of each test condition and the control condition were tested with the Alinity m SARS-CoV-2 assay. The interfering substances study results are summarized in Table 10.

Table 10. Interfering Substances Study Results – NPS Specimens in UVT/UTM   

<table><tr><td colspan="1" rowspan="1">Potentially InterferingSubstance</td><td colspan="1" rowspan="1">ActiveIngredient</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">IncludedSamples (N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">Invalid IC(N)</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">6a</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenges, OralAnesthetic and AnalgesicCepacol</td><td colspan="1" rowspan="1">Benzocaine,Menthol</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Purified Bovinemucin protein</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Purified Porcinemucin protein</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">6b</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, NasalOintment - Bactroban</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Spray-Afrin</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Relenza</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Veklury</td><td colspan="1" rowspan="1">Remdesivir</td><td colspan="1" rowspan="1">27.0 μM</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">AntibacterialSystemic(Tobramycin)</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Gel /HomeopathicAllergy Relief Medicine -Zicam</td><td colspan="1" rowspan="1">Galphimia glauca,Histaminum,hydrochloricum,Luffa operculata,Sulfur</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">FluMist d</td><td colspan="1" rowspan="1">Live intranasalinfluenza virus</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid-Flonase Sensimist</td><td colspan="1" rowspan="1">FluticasoneFuroate</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid-QVAR</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Corticosteroid -Dexamethasone</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.2 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">NasalCorticosteroid -Flunisolide</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">NasalCorticosteroid -Triamcinolone</td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">NasalCorticosteroid -Budesonide</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">NasalCorticosteroid -Mometasone</td><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">Fluoride</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Tobacco product</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">0.1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Oral rinse Listerine CoolMint</td><td colspan="1" rowspan="1">Ethanol, essentialoil</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">1.1E6 cells/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Decongestant -Phenylephrine</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Saline nasal mist</td><td colspan="1" rowspan="1">Sodium chloride</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nicotine Product</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">0.1% (w/v)equalto 0.05 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Spray-Afrin</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Chloroseptic Sore ThroatSpray</td><td colspan="1" rowspan="1">Phenol</td><td colspan="1" rowspan="1">5% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Cough Syrup (Wal-Tussin)</td><td colspan="1" rowspan="1">Dextromethorphanand guaifenesin</td><td colspan="1" rowspan="1">5% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mucin - Bovine</td><td colspan="1" rowspan="1">Purified mucinprotein</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Lidocaine andPhenylephrine(e.g., LiposomalNUMB250 Spray)</td><td colspan="1" rowspan="1">Lidocaine andPhenylephrine</td><td colspan="1" rowspan="1">2.68 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Tamiflu- Oseltamivir</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Petroleum Jelly (e.g.,Vaseline</td><td colspan="1" rowspan="1">Petroleum Jelly</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Analgesic Ointment(e.g., Vicks VapoRub)</td><td colspan="1" rowspan="1">Camphor-synthetic,eucalyptus oil andmenthol ointment</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenge, OralAnesthetic andAnalgesic (e.g., ColdEeze)</td><td colspan="1" rowspan="1">ZincuumGluconicum</td><td colspan="1" rowspan="1">2.5% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.02 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Saliva (Human)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr></table>

a, b One replicate was valid and not detected. Per protocol, the sample was retested in triplicate and all retest samples were valid and detected. c One replicate was a “no test” (instrument error) and per protocol, the sample was retested in triplicate and all retest samples were valid and detected. d Flumist was not tested for negative panel due to material availability

An analytical study was performed to evaluate the performance of the Alinity m SARS-CoV2 assay when testing nasal swab samples prepared using the Abbott Universal Collection Kit or the Abbott Universal Collection Kit II buffer and containing potentially interfering endogenous and exogenous substances.

The SARS-CoV-2 positive sample at 3X LoD was prepared by spiking gamma irradiated SARSCoV-2 virus into pooled negative clinical NS specimens prepared using the Abbott Universal Collection Kit or the Abbott Universal Collection Kit II. A minimum of three replicates of each test condition were tested along with the control condition on the Alinity m SARS-CoV-2 assay. The results from this study are presented in Table 11.

Table 11. Interfering Substances Study Results – NS Specimens in UCKs Transport Buffer   

<table><tr><td></td><td colspan="1" rowspan="1">uostalcCs Stady RC:</td><td colspan="1" rowspan="1">uits — Ts Spc</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">port Dur.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Potentially InterferingSubstance</td><td colspan="1" rowspan="1">ActiveIngredient</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">IncludedSamples(N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">InvalidIC (N)</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenges, OralAnesthetic andAnalgesic - Cepacol</td><td colspan="1" rowspan="1">Benzocaine, Menthol</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Mucin - Bovine</td><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, Nasal Ointment -Bactroban</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Spray-Afrin</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Relenza</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Veklury</td><td colspan="1" rowspan="1">Remdesivir</td><td colspan="1" rowspan="1">27.0 μM</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Antibacterial,Systemic(Tobramycin)</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Gel/Homeopathic AllergyRelief Medicine - Zicam</td><td colspan="1" rowspan="1">Galphimia glauca,Histaminum,hydrochloricum, Luffaoperculata, Sulfur</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">FluMist</td><td colspan="1" rowspan="1">Live intranasalinfluenza virus</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">NasalCorticosteroid-FlonaseSensimist</td><td colspan="1" rowspan="1">Fluticasone Furoate</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -QVAR</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Corticosteroid -Dexamethasone</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.2 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Flunisolide</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Triamcinolone</td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Budesonide</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Mometasone</td><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">Fluoride</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">5a</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Tobacco product</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">0.1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Oral rinse- Listerine CoolMint</td><td colspan="1" rowspan="1">Ethanol, essential oil</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">1.1E6 cells/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nasal Decongestant -Phenylephrine</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Saline nasal mist</td><td colspan="1" rowspan="1">Sodium chloride</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Nicotine Product</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">0.03 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Chloroseptic Sore ThroatSpray</td><td colspan="1" rowspan="1">Phenol</td><td colspan="1" rowspan="1">5% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Cough Syrup (e.g.,Robitussin)</td><td colspan="1" rowspan="1">Dextromethorphan andguaifenesin</td><td colspan="1" rowspan="1">5% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Lidocaine &amp; Phenylephrine(e.g., Liposomal NUMB250Spray)</td><td colspan="1" rowspan="1">Lidocaine andphenylephrine</td><td colspan="1" rowspan="1">2.68 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Tamiflu -Oseltamivir</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Petroleum Jelly (e.g.,Vaseline)</td><td colspan="1" rowspan="1">Petroleum Jelly</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Analgesic Ointment(e.g., Vicks VapoRub)</td><td colspan="1" rowspan="1">Camphor-syntheticeucalyptus oil &amp;menthol ointment</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenge, OralAnesthetic &amp; Analgesic(e.g., Cold Eeze)</td><td colspan="1" rowspan="1">Zincuum gluconicum</td><td colspan="1" rowspan="1">2.5% (w/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0.02 mg/mL</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Saliva (Human)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr></table>

a Three replicates from retest and 2 valid “Positive” from initial testing were included in the analysis per the protocol.

No interference with the Alinity m SARS-CoV-2 assay was observed in this analytical study when testing samples prepared using the Abbott Universal Collection Kit or Abbott Universal Collection Kit II buffer.

5. Assay Reportable Range: Not applicable; this is a qualitative assay.

6. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# See Section IV.C.Instrument Descriptive Information.5.Quality Control.

b. Sample Stability

Sample Stability – Nasopharyngeal Swabs in UVT/UTM

An analytical study was performed to establish the stability for nasopharyngeal swab (NPS) specimens eluted in UVT/UTM for testing using the Alinity m SARS-CoV-2 assay. Positive samples were prepared by the dilution of gamma-irradiated SARS-CoV-2 viruses at $3 \mathrm { \Delta x }$ LoD in pooled SARS-CoV-2 negative clinical nasopharyngeal swab specimens collected in UVT/UTM. Aliquots of the positive sample were tested immediately (control condition) and at later time points after storage as indicated in Table 12.

Table 12. Sample Stability – Nasopharyngeal Swabs   

<table><tr><td colspan="1" rowspan="1">StorageCondition</td><td colspan="1" rowspan="1">Storage Condition</td><td colspan="1" rowspan="1">Tested(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">PositiveRate</td></tr><tr><td colspan="1" rowspan="1">A (Control)</td><td colspan="1" rowspan="1">Tested immediately</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">B (Test)</td><td colspan="1" rowspan="1">A minimum of 80 hours at 5°C ± 3°plus a minimum of 4 hours onboardstorage before testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">C (Test C)</td><td colspan="1" rowspan="1">A minimum of 7 days -70°C or colder(with a minimum of 1 freeze/thaw) plus aminimum of 4 hours onboard storagebefore testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">D (Test)</td><td colspan="1" rowspan="1">A minimum of 53 hours at roomtemperature followed by a minimum of80 hours at 5° ± 3°C plus a minimumof 4 hours onboard storage beforetesting</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">E (Test)</td><td colspan="1" rowspan="1">A minimum of 7 days at 5°C ± 3°plus a minimum of 4 hours onboardstorage before testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">F (Test)</td><td colspan="1" rowspan="1">A minimum of 80 hours at 5°C ± 3°followed by a minimum of 7 days at-70°C or colder (with a minimum of 1freeze/thaw) plus a minimum of 4 hoursonboard storage before testing.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">A (Control)</td><td colspan="1" rowspan="1">Tested immediately</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">G (Test)</td><td colspan="1" rowspan="1">A minimum of 90 days at -70°C or colderplus a minimum of 4 hours onboardstorage before testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">A (Control)</td><td colspan="1" rowspan="1">Tested immediately</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">H (Test)</td><td colspan="1" rowspan="1">A minimum of 2 days at 15°C plus aminimum of 4 hours onboard storagebefore testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">I (Test)</td><td colspan="1" rowspan="1">A minimum of 2 days at 26°C plus aminimum of 4 hours onboard storagebefore testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100% (6/6)</td></tr></table>

# Sample Stability – Nasal Swabs in UVT/UTM

An analytical study was performed to establish stability for nasal swabs (NS) specimens eluted in UVT/UTM for testing using the Alinity m SARS-CoV-2 assay. Positive samples were prepared by the dilution of gamma-irradiated SARS-CoV-2 viruses at 1.5X LoD in pooled SARS-CoV-2 negative clinical nasal swab specimens collected in UVT/UTM. Aliquots of the positive sample were tested immediately (control condition) and at later time points after storage as indicated in Table 13.

Table 13. Stability Storage Conditions – Nasal Swabs in UVT/UTM   

<table><tr><td rowspan=1 colspan=1>StorageCondition</td><td rowspan=1 colspan=1>Storage Condition</td><td rowspan=1 colspan=1>Tested (N)</td><td rowspan=1 colspan=1>Positive(N)</td><td rowspan=1 colspan=1>Negative(N)</td><td rowspan=1 colspan=1>PositiveRate</td></tr><tr><td rowspan=1 colspan=1>A (Control)</td><td rowspan=1 colspan=1>Tested immediately</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>B (Test)</td><td rowspan=1 colspan=1>A minimum of 80 hours at 5° ± 3°C plusa minimum of 4 hours onboard storagebefore testing</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>C (Test)</td><td rowspan=1 colspan=1>A minimum of 7 days -70°C or colder(with a minimum of 1 freeze/thaw) plus aminimum of 4 hours onboard storage beforetesting</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>D (Test)</td><td rowspan=1 colspan=1>A minimum of 53 hours at 23.9°Cfollowed by a minimum of 80 hours at5° ± 3°C plus a minimum of 4 hoursonboard storage before testing</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>E (Test)</td><td rowspan=1 colspan=1>A minimum of 7 days at 5°C ± 3°C plus aminimum of 4 hours onboard storage beforetesting</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>F (Test)</td><td rowspan=1 colspan=1>A minimum of 80 hours at 5°C ± 3°followed by a minimum of 7 days at -70°C or colder (with a minimum of 1freeze/thaw) plus a minimum of 4 hoursonboard storage before testing.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>A (Control)</td><td rowspan=1 colspan=1>Tested immediately</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>G (Test)</td><td rowspan=1 colspan=1>A minimum of 90 days at -70°C or colderplus a minimum of 4 hours onboard storagebefore testing</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>A (Control)</td><td rowspan=1 colspan=1>Tested immediately</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>H (Test)</td><td rowspan=1 colspan=1>A minimum of 2 days at 15°C plus aminimum of 4 hours onboard storage beforetesting</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr><tr><td rowspan=1 colspan=1>I (Test)</td><td rowspan=1 colspan=1>A minimum of 2 days at 26°C plus aminimum of 4 hours onboard storage beforetesting</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100% (6/6)</td></tr></table>

An analytical study was performed to establish stability for nasal swabs (NS) specimens collected in the Abbott Universal Collection Kits transport buffer. Positive samples were prepared by the dilution of gamma-irradiated SARS-CoV-2 viruses at $3 \mathbf { x }$ LoD in pooled SARSCoV-2 negative clinical nasal swab specimens collected in Universal Collection Kits transport buffer.

Aliquots of the positive sample were tested immediately (control condition) and at later time points after storage as indicated in Table 14.

Table 14. Stability Storage Conditions—Universal Collection Kit Transport Buffer   

<table><tr><td colspan="1" rowspan="1">StorageCondition</td><td colspan="1" rowspan="1">Storage Condition</td><td colspan="1" rowspan="1">Tested(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">PositiveRate</td></tr><tr><td colspan="1" rowspan="1">A (Control)</td><td colspan="1" rowspan="1">Tested immediately</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">B (Test)</td><td colspan="1" rowspan="1">A minimum of 80 hours at 5°C ± 3°plus a minimum of 4 hours onboardstorage</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">C (Test)</td><td colspan="1" rowspan="1">A minimum of 7 days at 70°C or colderplus a minimum of 4 hours onboardstorage</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">D (Test)</td><td colspan="1" rowspan="1">A minimum of 53 hours at roomtemperature (15°C to 30°C) followed by aminimum of 80 hours at 5° ± 3°C plus aminimum of 4 hours onboard storagebefore testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">E (Test)</td><td colspan="1" rowspan="1">A minimum of 7 days at 5°C ± 3°C plus aminimum of 4 hours onboard storage</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">F (Test)</td><td colspan="1" rowspan="1">A minimum of 80 hours at 5°C ± 3°followed by a minimum of 7 days at70°C or colder plus a minimum of 4hours onboard storage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">G (Test)</td><td colspan="1" rowspan="1">A minimum of 7 days at 20°C or colderplus a minimum of 4 hours onboardstorage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">H (Test)</td><td colspan="1" rowspan="1">A minimum of 1 day at 40°C plus aminimum of 4 hours onboard storage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">I (Test)</td><td colspan="1" rowspan="1">A minimum of 1 day at 45°C plus aminimum of 4 hours onboard storage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">J (Test)</td><td colspan="1" rowspan="1">A minimum of 40°C for 6 hours, 22°for 16 hours, 40°C for 2 hours, 35°C for22 hours, and 40°C for 4 hours plus aminimum of 4 hours onboard storage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">K (Test)</td><td colspan="1" rowspan="1">A minimum of 2 days at 40°C plus aminimum off 4 hours onboard storage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">L (Test)</td><td colspan="1" rowspan="1">A minimum of 2 days at 45°C plus aminimum of 4 hours onboard storage.</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">A (Control)</td><td colspan="1" rowspan="1">Tested immediately</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">M (Test)</td><td colspan="1" rowspan="1">A minimum of 90 days at -70°C orcolder plus a minimum of 4 hoursonboard storage before testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">A (Control)</td><td colspan="1" rowspan="1">Tested immediately</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">N (Test)</td><td colspan="1" rowspan="1">A minimum of 2 days at 15°C plusa minimum of 4 hours onboardstorage before testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr><tr><td colspan="1" rowspan="1">O (Test)</td><td colspan="1" rowspan="1">A minimum of 2 days at 26°C plusa minimum of 4 hours onboardstorage before testing</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100% (6/6)</td></tr></table>

Stability study results for pre-collection storage conditions for the Abbott Universal Collection Kit supports claims for storage at $- 2 0 \mathrm { { ^ \circ C } }$ , $2 \mathrm { ^ { \circ } C }$ to $8 \mathrm { ^ { o } C }$ and $3 1 ^ { \circ } \mathrm { C }$ to $5 5 \mathrm { { ^ \circ C } }$ for 84 days and storage at $1 5 \mathrm { { ^ \circ C } }$ to $3 0 \mathrm { { ^ \circ C } }$ until the expiration date.

# Freeze-Thaw – Nasopharyngeal Swabs in UVT/UTM

A freeze-thaw study was conducted to evaluate specimen stability for the Alinity m SARS$\mathrm { C o V - } 2$ assay using pooled SARS-CoV-2 negative nasopharyngeal swab specimens in UVT/UTM. Three panel members as shown in Table 15 were tested. Positive panel members were prepared by diluting gamma irradiated SARS-CoV-2 viruses in nasopharyngeal swab specimens collected in UVT/UTM. The results of the study are presented in Table 16.

Table 15. Nasopharyngeal Swab Specimens in UVT/UTM   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Target Concentration</td><td rowspan=1 colspan=1>Replicates per condition</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2X LoD</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5X LoD</td><td rowspan=1 colspan=1>10</td></tr></table>

Table 16. Freeze-Thaw Study – Nasopharyngeal Swab in UVT/UTM Results Summary   

<table><tr><td colspan="1" rowspan="1">PanelMember</td><td colspan="1" rowspan="1">Condition a</td><td colspan="1" rowspan="1">Tested(N)</td><td colspan="1" rowspan="1">Positive(N)</td><td colspan="1" rowspan="1">Negative(N)</td><td colspan="1" rowspan="1">Positive Rate</td></tr><tr><td colspan="1" rowspan="3">Negative</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0.0% (0/10)</td></tr><tr><td colspan="1" rowspan="1">One freeze/thaw</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0.0% (0/10)</td></tr><tr><td colspan="1" rowspan="1">Two freeze/thaws</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0.0% (0/10)</td></tr><tr><td colspan="1" rowspan="3">2X LoD</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0% (40/40)</td></tr><tr><td colspan="1" rowspan="1">One freeze/thaw</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0% (40/40)</td></tr><tr><td colspan="1" rowspan="1">Two freeze/thaws</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0% (40/40)</td></tr><tr><td colspan="1" rowspan="3">5X LoD</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0% (10/10)</td></tr><tr><td colspan="1" rowspan="1">One freeze/thaw</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0% (10/10)</td></tr><tr><td colspan="1" rowspan="1">Two freeze/thaws</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0% (10/10)</td></tr></table>

a Freezing at ${ } _ { - 7 0 ^ { \circ } \mathrm { C } }$ .

All control nasopharyngeal swab in UVT/UTM samples were $100 \%$ negative (0/10). At 2X LoD, $100 \%$ of the samples were positive for SARS-CoV-2 detection (40/40). At 5X LoD, $100 \%$ of the samples were positive for SARS-CoV-2 detection (10/10).

# Freeze-Thaw – Nasal Swabs in UVT/UTM

A freeze-thaw study was conducted to evaluate specimen stability for the Alinity m SARSCoV-2 assay using pooled SARS-CoV-2 negative nasal swab specimens in UVT/UTM. The 1X LoD and 5X LoD positive panel members were prepared by diluting gamma irradiated SARS-CoV-2 viruses in pooled SARS-CoV-2 negative nasal swab specimens in UVT/UTM. Three panel members as shown in Table 17 were tested and results are presented in Table 18.

Table 17. Nasal Swab Specimens in UVT/UTM   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Target Concentration</td><td rowspan=1 colspan=1>Replicates per condition</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1X LoD</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5X LoD</td><td rowspan=1 colspan=1>10</td></tr></table>

Table 18. Freeze-Thaw Study – Nasal Swab in UVT/UTM Results Summary   

<table><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Condition a</td><td rowspan=1 colspan=1>IncludedSamples (N)</td><td rowspan=1 colspan=1>Positive(N)</td><td rowspan=1 colspan=1>Negative(N)</td><td rowspan=1 colspan=1>PositiveRate</td><td rowspan=1 colspan=1>NegativeRate</td></tr><tr><td rowspan=3 colspan=1>Negative</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0%(0/10)</td><td rowspan=1 colspan=1>100.0%(10/10)</td></tr><tr><td rowspan=1 colspan=1>One freeze/thaw</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0%(0/10)</td><td rowspan=1 colspan=1>100.0%(10/10)</td></tr><tr><td rowspan=1 colspan=1>Two freeze/thaws</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0%(0/10)</td><td rowspan=1 colspan=1>100.0%(10/10)</td></tr><tr><td rowspan=3 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>0.0% (0/40)</td></tr><tr><td rowspan=1 colspan=1>One freeze/thaw</td><td rowspan=1 colspan=1>42b</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.2%(40/42)</td><td rowspan=1 colspan=1>0.0% (0/40)</td></tr><tr><td rowspan=1 colspan=1>Two freeze/thaws</td><td rowspan=1 colspan=1>42c</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.6%(41/42)</td><td rowspan=1 colspan=1>0.0% (0/40)</td></tr><tr><td rowspan=3 colspan=1>5 x LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>0.0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>One freeze/thaw</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>0.0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>Two freeze/thaws</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>0.0% (0/10)</td></tr></table>

# Freeze-Thaw – Nasal Swabs Collected in the Abbott Universal Collection Kits Transport Buffer

A freeze-thaw study was conducted to evaluate specimen stability for the Alinity m SARSCoV-2 assay using pooled SARS-CoV-2 negative nasal swab specimens in the Abbott Universal Collection Kits transport buffer. The positive panel members were prepared by diluting gamma irradiated SARS-CoV-2 viruses in pooled SARS-CoV-2 negative nasal swab specimens collected in the Abbott Universal Collection Kits transport buffer. Three panel members as shown in Table 19 were tested. The results are presented in Table 20.

Table 19. Nasal Swab Specimens in Abbott Universal Collection Kits Transport Buffer   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Target Concentration</td><td rowspan=1 colspan=1>Replicates per condition</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2X LoD</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5X LoD</td><td rowspan=1 colspan=1>10</td></tr></table>

Table 20. Freeze-Thaw Study – Nasal Swab in Abbott Universal Collection Kits Transport Buffer Results Summary   

<table><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>Condition a</td><td rowspan=1 colspan=1>IncludedSamples(N)</td><td rowspan=1 colspan=1>Positive(N)</td><td rowspan=1 colspan=1>Negative(N)</td><td rowspan=1 colspan=1>Positive Rate</td><td rowspan=1 colspan=1>Negative Rate</td></tr><tr><td rowspan=3 colspan=1>Negative</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0% (0/10)</td><td rowspan=1 colspan=1>100.0% (10/10)</td></tr><tr><td rowspan=1 colspan=1>One freeze/thaw</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0% (0/10)</td><td rowspan=1 colspan=1>100.0% (10/10)</td></tr><tr><td rowspan=1 colspan=1>Two freeze/thaws</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0% (0/10)</td><td rowspan=1 colspan=1>100.0% (10/10)</td></tr><tr><td rowspan=3 colspan=1>2X LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>0.0% (0/40)</td></tr><tr><td rowspan=1 colspan=1>One freeze/thaw</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>0.0% (0/40)</td></tr><tr><td rowspan=1 colspan=1>Two freeze/thaws</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>0.0% (0/40)</td></tr><tr><td rowspan=3 colspan=1>5X LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>0.0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>One freeze/thaw</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>0.0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>Two freeze/thaws</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>0.0% (0/10)</td></tr></table>

a Freezing at - $. 7 0 ^ { \circ } \mathrm { C }$ .

7. Detection Limit:

Limit of detection (LoD) studies were performed to evaluate the analytical sensitivity of the Alinity m SARS-CoV-2 Assay.

The LoD of the Alinity m SARS-CoV-2 assay was determined with the analysis of a dilution series of inactivated SARS-CoV-2 virus in nasopharyngeal swab specimens in UVT/UTM. Dilutions of gamma irradiated SARS-CoV-2 virus were prepared in pooled SARS-CoV-2 negative clinical nasopharyngeal swab (NPS) specimens in UVT/UTM. The preliminary LoD assessment was carried out by testing replicates of three at the following SARS-CoV-2 target concentrations $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ : 0.0180, 0.0090, 0.0045, and 0. The preliminary LoD of $0 . 0 0 9 0 \mathrm { T C I D } { \scriptstyle 5 0 / \mathrm { m L } }$ was confirmed by testing 21 replicates. The results are presented in Table 21.

Table 21. LoD in Clinical Nasopharyngeal Swabs in UVT/UTM.   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2Concentration(TCID50/mL)a</td><td rowspan=1 colspan=1>Number ofReplicatesTested</td><td rowspan=1 colspan=1>Number ofReplicatesDetected</td><td rowspan=1 colspan=1>Detection Rate(%)</td></tr><tr><td rowspan=1 colspan=1>0.0045</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>33.3%</td></tr><tr><td rowspan=1 colspan=1>0.0090</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>0.0180</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.2%</td></tr></table>

a $\mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L } =$ Median Tissue Culture Infectious Dose/mL

# Limit of Detection (LoD) - Nasal Swabs in UVT/UTM

The LoD of the Alinity m SARS-CoV-2 assay was determined with the analysis of a dilution series of inactivated SARS-CoV-2 virus with nasal swab specimens in UVT/UTM. Dilutions of gamma irradiated SARS-CoV-2 virus were prepared in pooled SARS-CoV-2 negative clinical nasal swab (NS) specimens in UVT/UTM. The preliminary LoD assessment was conducted by testing replicates of three at the following SARS-CoV-2 target concentrations $\mathrm { ( T C I D 5 0 / m L ) }$ ): 0.0180, 0.0090, 0.0045, and 0. The preliminary LoD was $0 . 0 0 9 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . LoD confirmation was carried out by testing replicates of 21 at the preliminary LoD and two additional levels. The LoD was confirmed to be $0 . 0 1 8 0 \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ . Results are presented in Table 22.

Table 22. LoD in Clinical Nasal Swabs in UVT/UTM   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2Concentration(TCID50/mL)a</td><td rowspan=1 colspan=1>Number ofReplicatesTested</td><td rowspan=1 colspan=1>Number ofReplicatesDetected</td><td rowspan=1 colspan=1>DetectionRate (%)</td></tr><tr><td rowspan=1 colspan=1>0.0045</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>42.9%</td></tr><tr><td rowspan=1 colspan=1>0.0090</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>81.0%</td></tr><tr><td rowspan=1 colspan=1>0.0180</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

a $\mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L } =$ Median Tissue Culture Infectious $\mathrm { D o s e / m L }$

Limit of Detection (LoD) - Nasal Swabs in the Abbott Universal Collection Kits Transport Buffer

The LoD of the Alinity m SARS-CoV-2 assay was determined with the analysis of a dilution series of inactivated SARS-CoV-2 virus with nasal swab specimens collected in the Abbott Universal Collection Kits. Dilutions of gamma irradiated SARS-CoV-2 virus were prepared in pooled SARS-CoV-2 negative clinical nasal swab (NS) specimens collected in the Abbott Universal Collection Kits transport buffer. A five-member panel with the following target concentrations was tested in replicates of three in the range finding study: 0, 13.5, 27, 55, and 109 GE/mL. The three highest concentrations were repeated with 20 replicates each. The lowest concentration with a detection rate of $9 5 \%$ or greater was $5 5 \ : \mathrm { G E / m L }$ . Results are presented in Table 23.

Table 23. LoD in Clinical Nasal Swabs in Abbott Universal Collection Kits Transport Buffer.   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2Concentration(GE/mL)a</td><td rowspan=1 colspan=1>Number ofReplicatesTested</td><td rowspan=1 colspan=1>Number ofReplicatesDetected</td><td rowspan=1 colspan=1>Detection Rate(%)</td></tr><tr><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20.0%</td></tr><tr><td rowspan=1 colspan=1>55.0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>109.0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

a $\mathrm { { G E / m L = } }$ Genome Equivalent/mL

# Limit of Detection (LoD) – SARS-CoV-2 WHO International Standard in SNM

An LoD study was performed to evaluate the LoD of the Alinity m SARS-CoV-2 Assay with the World Health Organization (WHO) Internal Standard for SARS-CoV-2. The WHO $1 ^ { \mathrm { s t } }$ International Standard for SARS-CoV-2 RNA (NIBSC code: 20/146) was diluted in negative Simulated Nasal Matrix (SNM). The dilution panel of the SARSCoV-2 WHO standard comprised of 8 panel members at the following concentrations: 125, 100, 75, 50, 25, 10, 5 and $1 \ \mathrm { I U / m L }$ . Twenty-four replicates per panel member were tested. The detection rates observed for each panel are summarized in Table 24. The LoD for the Alinity m SARS-CoV-2 assay with the WHO International Standard was 100 $\mathrm { I U / m L }$ .

Table 24. LoD Confirmation Results - WHO SARS-CoV-2 International Standard in SNM.   

<table><tr><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>TargetConcentration(IU/mL)</td><td rowspan=1 colspan=1>Number ofReplicatesDetected</td><td rowspan=1 colspan=1>Number of ReplicatesTested</td><td rowspan=1 colspan=1>DetectionRate (%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>87.5</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.0</td></tr></table>

# 8. Assay Cut-Off:

The assay was designed to report results through all 42 PCR cycles—the maximum number of cycles for this assay. When an amplification of specimen occurred between PCR cycles 1 to 42 (CN) and met the MR threshold (PCR efficiency-related maximum ratio (MR)) the specimen was interpreted as Positive initially. Subsequently, a CN cutoff for the Alinity m SARS-CoV-2 assay was selected at 39.57.

# 9. Carry-Over:

An analytical study was performed to assess potential carry-over or cross-contamination in the Alinity m SARS-CoV-2 assay by testing high positive and negative samples in an alternating fashion on the Alinity m System. Testing with each AMP Tray consisted of 24 replicates of high-positive sample prepared at $2 . 0 \mathrm { { x } 1 0 ^ { 9 } \mathrm { { C o p i e s / \mathrm { { m L } } } } }$ of SARS-CoV-2 material and 24 replicates of negative sample. Sample replicates were arranged such that high-positive and negative samples are processed in alternating reaction vessels. The carryover contamination study tested an array of 363 high-positive samples and 361 negative samples across five Alinity m SARS-CoV-2 AMP Trays on each of three Alinity m systems. Results are presented in Table 25.

Table 25. Alinity m System Carry-Over Rate.   

<table><tr><td>Sample Type</td><td>Tested (N)</td><td>Negative (N)</td><td>Positive (N)</td><td>Carry-over Rate</td><td>95% Confidence Interval</td></tr><tr><td>Negative</td><td>361</td><td>361</td><td>0</td><td>0.0% (0/361)</td><td>(0.0%,1.1%)</td></tr></table>

None of the 361 negative samples exhibited any evidence of carry-over contamination from the high-positive samples to SARS-CoV-2 negative samples (0/361).

# B Comparison Studies:

See Clinical Studies section below.

1. Matrix Comparison:

Simulated Nasal Matrix (SNM) and Clinical Matrix Equivalency Study

To evaluate the equivalence of SNM used in analytical studies with natural clinical matrix, SNM was formulated and tested side by side with clinical nasopharyngeal swab (NPS) specimens collected in UVT/UTM, clinical nasal swab (NS) specimens in collected UVT/UTM, and clinical NS specimens collected in the Universal Collection Kits (UCK) transport buffer.

Each matrix was evaluated by testing four panel members: a negative panel member and three positive panel members which were prepared by the spiking cultured SARS-CoV-2 virus in each type of matrix at $0 . 5 \mathrm { x }$ LoD, 1x LoD, and 3x LoD. For each matrix and panel member 21 replicates were tested to achieve a minimum of 20 replicates. The testing was performed using Alinity m SARS-CoV-2 assay reagents and the Alinity m System. Results are summarized Table 26.

Table 26. Alinity m SARS-CoV-2 Matrix Equivalency Study Results   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>Number of Replicates</td><td rowspan=2 colspan=1>Positive Rate(%)</td></tr><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>TargetLevel</td><td rowspan=1 colspan=1>TotalTested</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=4 colspan=1>Simulated NasalMatrix(SNM)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>71.4%</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.2%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=4 colspan=1>Pooled ClinicalNasopharyngealSwabs in UVT/UTM</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>81.0%</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=4 colspan=1>Pooled ClinicalNasal Swabs inUVT/UTM</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.2%</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95.0%</td></tr><tr><td rowspan=4 colspan=1>Pooled ClinicalNasal Swabs inUniversalCollection Kits(UCK) TransportBuffer</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>76.2%</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.2%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.2%</td></tr></table>

The results demonstrate the equivalency of all matrices evaluated, including the SNM.

# C Clinical Studies:

# Prospective Clinical Studies

The clinical performance of the Alinity m SARS-CoV-2 assay was established in two clinical studies that tested prospective clinical specimens in viral transport media (UVT or UTM) and in Universal Collection Kit (UCK) from individuals with signs and symptoms of respiratory infection.

# Composite Comparator (CC) Definition

A minimum of two and up to three highly sensitive SARS-CoV-2 molecular EUA assays were used as comparator assays to establish the composite comparator (CC) for assessing the performance of the Alinity m SARS-CoV-2 assay. A specimen was categorized as CC positive if a minimum of two comparator positive results were reported. A specimen was categorized as CC negative if two or more comparator results were negative. A specimen was categorized as indeterminate (IND) if CC could not be determined due to missing results from the comparator assays. Specimens with an Indeterminate CC result were excluded from analyses.

# I. Prospective Clinical Study – Nasopharyngeal Swab Specimens

Nasopharyngeal swabs (NPS) were prospectively collected at eight geographically distinct sites in the U.S. Specimens were collected in BD Universal Viral Transport (UVT) with a flocked swab between January 2021 and February 2021.Of the 627 NPS specimens from symptomatic subjects tested, 580 samples were Category I specimens, while 47 were Category II specimens. Of the initial 627 specimens tested, 16 were excluded as invalid with no result obtained and 76 specimens were excluded for having no composite comparator result. Five hundred thirty-five NPS specimens were included for performance analysis. Patient demographic information for the 535 specimens with valid CC results that were included in performance analysis in the prospective clinical study are presented in Table 27.

Table 27 Subject Demographics – Nasopharyngeal Specimen Prospective Clinical Study   

<table><tr><td colspan="1" rowspan="1">Demographic Characteristic</td><td colspan="1" rowspan="2">Statistic</td></tr><tr><td colspan="1" rowspan="1">Age</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">535</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">39</td></tr><tr><td colspan="1" rowspan="1">Minimum</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Maximum</td><td colspan="1" rowspan="1">87</td></tr><tr><td colspan="1" rowspan="1">Age Group</td><td colspan="1" rowspan="1">n (%)</td></tr><tr><td colspan="1" rowspan="1">Birth to 5 years</td><td colspan="1" rowspan="1">4 (0.7%)</td></tr><tr><td colspan="1" rowspan="1">6 to 21 years</td><td colspan="1" rowspan="1">67 (12.5%)</td></tr><tr><td colspan="1" rowspan="1">22 to 59 years</td><td colspan="1" rowspan="1">380 (71.0%)</td></tr><tr><td colspan="1" rowspan="1">≥ 60 years</td><td colspan="1" rowspan="1">84 (15.7%)</td></tr></table>

A summary performance of the Alinity m SARS-CoV-2 assay with prospectively collected nasopharyngeal swab specimen in UVT is provided in Table 28.

Table 28. Summary of Performance of the Prospective Fresh (Category I) and Frozen (Category II) Nasopharyngeal Swab Specimens in UVT   

<table><tr><td rowspan=1 colspan=1>Fresh or Frozen</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>PPA (%)(95% CI)</td><td rowspan=1 colspan=1>NPA (%)(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>96.2 (92.0-98.3)</td><td rowspan=1 colspan=1>95.5 (92.7-97.2)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0 (34.2-100.0)</td><td rowspan=1 colspan=1>93.2 (81.8-97.7)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>535</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>357</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>96.3 (92.1-98.3)</td><td rowspan=1 colspan=1>95.2 (92.5-96.9)</td></tr></table>

TP $=$ true positive; $\mathrm { F N = }$ false negative; $\mathrm { T N } =$ true negative; $\mathrm { F P = }$ false positive, $\mathrm { P P A } =$ Positive Percent Agreement, $\mathrm { \Delta N P A = }$ Negative Percent Agreement.

Of the 627 specimens tested in the nasopharyngeal swab prospective study, 609 gave valid results with 18 initial invalid samples. The initial invalid rate was $2 . 9 \%$ $9 5 \%$ CI: $1 . 8 \% - 4 . 5 \% )$ . Upon retest, only one sample gave a valid result with 17 samples remaining invalid for a final invalid rate of $2 . 7 \%$ ( $9 5 \%$ CI: $1 . 7 \% - 4 . 3 \%$ ).

# II. Prospective Clinical Study – Anterior Nasal Swab Specimens

This prospective clinical study used anterior nasal swab (ANS) specimens (two per patient) that were self-collected under HCP supervision in the Abbott Universal Collection Kit II (UCK) and in the Universal Viral Transport (UVT). ANS samples were collected between September 2021 and January 2022. Adult and pediatric subjects, presenting signs and symptoms consistent with respiratory tract infection and/or COVID-19 were enrolled for specimen collection. Subject demographics are presented in Table 29.

Table 29. Summary of Subject Demographics - Anterior Nasal Specimen Prospective Clinical Study   

<table><tr><td colspan="1" rowspan="1">Demographic Characteristic</td><td colspan="1" rowspan="2">Statistic</td></tr><tr><td colspan="1" rowspan="1">Age</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">785</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">36</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">34</td></tr><tr><td colspan="1" rowspan="1">Minimum</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Maximum</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Age Group</td><td colspan="1" rowspan="1">n (%)</td></tr><tr><td colspan="1" rowspan="1">6 to 21 years</td><td colspan="1" rowspan="1">132 (16.8%)</td></tr><tr><td colspan="1" rowspan="1">22 to 59 years</td><td colspan="1" rowspan="1">591 (75.3%)</td></tr><tr><td colspan="1" rowspan="1">≥ 60 years</td><td colspan="1" rowspan="1">62 (7.9%)</td></tr></table>

Of the 759 ANS UVT specimens included in the analysis with CC, 599 were tested fresh (Category I) and 160 were tested after being stored frozen (Category II). Of the 766 ANS UCK specimens included in the analysis with CC, 603 were tested fresh (Category I) and 163 were tested after being stored frozen (Category II). Performance results are presented in Table 30 for specimens collected in UVT and Table 31 for specimens collected in UCK.

Table 30. Summary of the Clinical Performance of Prospective Fresh (Category I) and Frozen (Category II) Self-Collected ANS Samples (under HCP supervision) in UVT   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Fresh orFrozen</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>PPA (%) (95%CH</td><td rowspan=1 colspan=1>NPA (%) (95%CH</td></tr><tr><td rowspan=2 colspan=1>Anterior Nasal Swab (Self-Collected, under HCPsupervision) UVT</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>599</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>516</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100 (95.5-100)</td><td rowspan=1 colspan=1>99.8 (98.9-100.)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>99.3 (96.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>759</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>661</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100 (96.2-100)</td><td rowspan=1 colspan=1>99.7 (98.9-99.9)</td></tr></table>

TP $=$ true positive; FN $=$ false negative; TN $=$ true negative; $\mathrm { F P = }$ false positive, PPA $=$ Positive Percent Agreement, NPA $=$ Negative Percent Agreement.

Table 31. Summary of the Clinical Performance of Prospective Fresh (Category I) and Frozen (Category II) Self-Collected ANS Samples (under HCP supervision) using UCK   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Fresh orFrozen</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>PPA (%) (95% CI)</td><td rowspan=1 colspan=1>NPA (%) (95%CH</td></tr><tr><td rowspan=2 colspan=1>Anterior Nasal Swab (Self-Collected under HCPsupervision) UCK</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>603</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>511</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>97.6 (91.5-99.3)</td><td rowspan=1 colspan=1>98.1 (96.5-99.0)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>97.3 (93.3-99.0)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>766</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>656</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>97.9 (92.7-99.4)</td><td rowspan=1 colspan=1>97.9 (96.5-98.8)</td></tr></table>

TP $=$ true positive; $\mathrm { F N = }$ false negative; $\mathrm { T N } =$ true negative; $\mathrm { F P = }$ false positive, PPA $=$ Positive Percent Agreement, NPA $=$ Negative Percent Agreement.

Of the initial 787 anterior nasal specimens collected in UVT, 21 gave an invalid result. With a total of 766 valid specimens remaining, the initial invalid rate was $2 . 7 \%$ $9 5 \%$ CI: $1 . 8 \% - 4 . 0 \%$ . Upon retesting of the 21 invalid specimens, 20 were found to be valid with 1 specimen remaining invalid. The final invalid rate was $0 . 1 \%$ ( $9 5 \%$ CI: $0 . 0 \% - 0 . 7 \%$ ).

Of the 792 anterior nasal specimens collected in UCK, 52 specimens gave an invalid result with 740 samples providing valid test results for an initial invalid rate of $6 . 6 \%$ ( $9 5 \%$ CI: $5 . 0 \% - 8 . 5 \%$ ). Upon retesting of the 52 invalid specimens, 51 gave a valid result for a final invalid rate of $0 . 1 \%$ ( $9 5 \%$ CI: $0 . 0 \% - 0 . 7 \%$ ).

# E Expected Values/Reference Range:

A summary of the positivity rate for detection of SARS-CoV-2 by the Alinity m SARS-CoV-2 assay for the nasopharyngeal swab specimens by collection site is presented in Table 32. The SARS-CoV-2 positivity ranged from $6 . 5 \%$ to $6 3 . 6 \%$ .

Table 32. Positivity as Determined by the Alinity m SARS-CoV-2 Assay for Nasopharyngeal Swab Specimens in UVT.   

<table><tr><td colspan="2">Site No. City and State</td><td rowspan="2">Geographic Region</td><td colspan="6"></td></tr><tr><td></td><td></td><td>N</td><td>TP</td><td>FN</td><td>TN</td><td>FP</td><td>Positivity Rate (%)</td></tr><tr><td>1</td><td>Miami, FL</td><td>South</td><td>139</td><td>44</td><td>2</td><td>89</td><td>4</td><td>34.5</td></tr><tr><td>2</td><td>Decatur, GA</td><td>South</td><td>46</td><td>2</td><td>0</td><td>43</td><td>1</td><td>6.5</td></tr><tr><td>3</td><td>Jackson, TN</td><td>South</td><td>97</td><td>15</td><td>0</td><td>82</td><td>0</td><td>15.5</td></tr><tr><td>4</td><td>Tampa, FL</td><td>South</td><td>25</td><td>11</td><td>0</td><td>13</td><td>1</td><td>48.0</td></tr><tr><td>5</td><td>Cincinnati, OH</td><td>Midwest</td><td>64</td><td>20</td><td>0</td><td>38</td><td>6</td><td>40.6</td></tr><tr><td>6</td><td>Yucaipa, CA</td><td>West</td><td>33</td><td>18</td><td>1</td><td>11</td><td>3</td><td>63.6</td></tr><tr><td>7</td><td>Dayton, OH</td><td>Midwest</td><td>55</td><td>21</td><td>1</td><td>32</td><td>1</td><td>40.0</td></tr><tr><td>8</td><td>Austin, TX</td><td>South</td><td>76</td><td>23</td><td>2</td><td>49</td><td>2</td><td>32.9</td></tr><tr><td>All</td><td></td><td></td><td>535</td><td>154</td><td>6</td><td>357</td><td>18</td><td>32.1</td></tr></table>

A summary of the positivity rate for detection of SARS-CoV-2 by the Alinity m SARS-CoV-2 assay for anterior nasal specimens collected in UVT by collection site is presented in Table 33. The SARS-CoV-2 positivity ranged from $0 . 0 \%$ to $4 3 . 6 \%$ .

Table 33. Positivity as Determined by the Alinity m SARS-CoV-2 Assay for Anterior Nasal Swab Specimens in UVT.   

<table><tr><td colspan="2">Site</td><td colspan="8">Geographic Region</td></tr><tr><td>No.</td><td>City and State Huntington Park, CA</td><td>West</td><td>N 83</td><td>TP 0</td><td>FN 0</td><td>TN 83</td><td>FP</td><td></td><td>Rate (%)</td></tr><tr><td>1</td><td>Austin, TX</td><td>South</td><td>112</td><td>10</td><td>0</td><td>102</td><td>0 0</td><td></td><td>0.0 8.9</td></tr><tr><td>2 3</td><td></td><td>South</td><td>65</td><td>6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>4</td><td>Cape Coral, FL Round Rock, TX</td><td>South</td><td>70</td><td>9</td><td>0</td><td>59</td><td></td><td>0</td><td>9.2</td></tr><tr><td>5</td><td>San Antonio, TX</td><td>South</td><td>141</td><td>14</td><td>0 0</td><td>61</td><td></td><td>0</td><td>12.9</td></tr><tr><td>6</td><td>San Antonio, TX</td><td>South</td><td>158</td><td></td><td></td><td>127</td><td></td><td>0</td><td>9.9</td></tr><tr><td></td><td></td><td></td><td></td><td>4</td><td>0</td><td>152</td><td>2</td><td></td><td>3.8</td></tr><tr><td>7 8</td><td>Albuquerque, NM</td><td>West</td><td>101</td><td>44</td><td>0</td><td>57</td><td></td><td>0</td><td>43.6</td></tr><tr><td>All</td><td>Albuquerque, NM</td><td>West</td><td>29 759</td><td>9 96</td><td>0 0</td><td>20 661</td><td>2</td><td>0</td><td>31.0 12.9</td></tr></table>

A summary of the positivity rate for detection of SARS-CoV-2 by the Alinity m SARS-CoV-2 assay for anterior nasal specimens collected in UCK by collection site is presented in Table 34. The SARS-CoV-2 positivity ranged from $1 . 2 \%$ to $4 6 . 5 \%$ .

Table 44. Positivity as Determined by the Alinity m SARS-CoV-2 Assay for Anterior Nasal Swab Specimens in UCK.   

<table><tr><td rowspan="2">Site No.</td><td colspan="8">Geographic</td></tr><tr><td>City and State</td><td>Region</td><td>N</td><td>TP</td><td>FN</td><td>TN</td><td>FP</td><td>Positivity Rate (%)</td></tr><tr><td>1</td><td>Huntington Park, CA</td><td>West</td><td>85</td><td>0</td><td>0</td><td>84</td><td>1</td><td>1.2</td></tr><tr><td>2</td><td>Austin, TX</td><td>South</td><td>112</td><td>10</td><td>0</td><td>98</td><td>4</td><td>12.5</td></tr><tr><td>3</td><td>Cape Coral, FL</td><td>South</td><td>69</td><td>6</td><td>0</td><td>62</td><td>1</td><td>10.1</td></tr><tr><td>4</td><td>Round Rock, TX</td><td>South</td><td>70</td><td>9</td><td>0</td><td>60</td><td>1</td><td>14.3</td></tr><tr><td>5</td><td>San Antonio, TX</td><td>South</td><td>141</td><td>13</td><td>1</td><td>126</td><td>1</td><td>9.9</td></tr><tr><td>6</td><td>San Antonio, TX</td><td>South</td><td>159</td><td>3</td><td>1</td><td>153</td><td>2</td><td>3.1</td></tr><tr><td>7</td><td>Albuquerque, NM</td><td>West</td><td>101</td><td>44</td><td>0</td><td>54</td><td>3</td><td>46.5</td></tr><tr><td>8</td><td>Albuquerque, NM</td><td>West</td><td>29</td><td>9</td><td>0</td><td>19</td><td>1</td><td>34.5</td></tr><tr><td>All</td><td></td><td></td><td>766</td><td>94</td><td>2</td><td>656</td><td>14</td><td>14.1</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not Applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.