# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

# B Applicant

Nuclein, LLC

# C Proprietary and Established Names

DASH SARS-CoV-2 & Flu A/B Test

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain market clearance for the DASH SARS-CoV-2 & Flu A/B Test.

# B Measurand:

SARS-CoV-2 RNA Influenza A RNA Influenza B RNA

# C Type of Test:

The DASH SARS-CoV-2 & Flu A/B Test is a reverse transcription polymerase chain reaction (RT-PCR) test.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The DASH SARS-CoV-2 & Flu A/B Test is a rapid reverse transcription polymerase chain reaction (RT-PCR) assay performed on the DASH Rapid PCR Instrument and is intended for the simultaneous in vitro qualitative detection and differentiation of SARS-CoV-2, influenza A and influenza B virus ribonucleic acid (RNA) in anterior nasal swab specimens from patients with signs and symptoms of respiratory tract infection. The test is intended to aid in the differential diagnosis of SARS-CoV-2, influenza A, and influenza B in humans in conjunction with other clinical, epidemiologic and laboratory findings.

Positive results of a specific target are indicative of the presence of that viral RNA and may not be the definite cause of disease. Positive results do not rule out co-infection with other pathogens. Negative results do not preclude SARS-CoV-2, influenza A or influenza B infection and should not be used as the sole basis for patient management decisions

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

IVD - For In Vitro Diagnostic Use Only

D Special Instrument Requirements: DASH Rapid PCR Instrument (DASH Instrument)

IV Device/System Characteristics:

# A Device Description:

The DASH SARS-CoV-2 & Flu A/B Test is a reverse transcription polymerase chain reaction (RT-PCR) assay for rapid qualitative detection and differentiation of SARS-CoV-2, influenza A (Flu A) and influenza B (Flu B) from human anterior nasal swab (ANS) specimens. The test combines the technologies of sequence specific capture sample preparation and RT-PCR amplification. An anterior nasal swab with a $3 0 \mathrm { - m m }$ breakpoint is used to collect a specimen. The ANS specimen is added directly to the DASH SARS-CoV-2 & Flu A/B Test cartridge sample chamber. The DASH SARS-CoV-2 & Flu A/B Test cartridge contains all reagents necessary to perform the test. The cartridge is capped and inserted into the DASH Instrument to initiate the test, and all subsequent test steps are performed automatically by the DASH Instrument.

Materials included with the DASH SARS-CoV-2 & Flu A/B Test consist of the following: • Ten individually pouched DASH SARS-CoV-2 & Flu A/B Cartridges per kit

Ten individually packaged anterior nares sample collection swabs ( $3 0 \mathrm { m m }$ breakpoint, sterile elongated polyester flock) per kit • DASH SARS-CoV-2 & Flu A/B Test Quick Reference Instructions (QRI).

The DASH Instrument, DASH SARS-CoV-2 & Flu A/B Positive Control Swabs Kit, and DASH Negative Control Swabs Kit are provided separately.

# B Principle of Operation:

The DASH SARS-CoV-2 & Flu A/B Test combines the technologies of sequence specific capture sample preparation and RT-PCR amplification. The DASH SARS-CoV-2 & Flu A/B Test cartridge contains all reagents necessary to perform the test to detect RNA from SARSCoV-2, influenza A and influenza B.

An ANS sample is collected and added directly to the DASH SARS-CoV-2 & Flu A/B Test cartridge. The Test cartridge is capped and inserted into the DASH Instrument to initiate the test. The Test cartridge sample chamber contains lysis buffer which releases the RNA target from the virus. The target RNA molecules are isolated from specimens by use of biotinylated capture oligomers (capture oligomers) which are in turn captured by streptavidin-coated paramagnetic particles. The biotinylated capture oligomers contain sequences complementary to specific regions of the RNA target molecules. During the hybridization step, the sequence specific regions of the capture oligomers bind to specific regions of the RNA target molecules. The capture oligomer:target complex is then concentrated by the addition of streptavidin-coated paramagnetic particles.

The paramagnetic particles, including the bound target molecules, are washed to remove residual specimen matrix that may contain amplification reaction inhibitors. After the target capture steps are completed, the target RNA is ready for RT-PCR amplification. Forty amplification cycles are performed, during which the fluorescence is measured with a four-color LED fluorometer. Amplification of any assay targets, Flu A, Flu B, and SARS-CoV-2, leads to fluorescent signal. An algorithm determines whether the result is positive or negative for each viral target assay. Additionally, a process control is included in the cartridge to control for adequate processing of the target virus through the steps of sample purification, nucleic acid amplification and to monitor for the presence of inhibitors in the RT-PCR reaction. Total run time, from swab insertion into the Test cartridge to display of the test results is approximately 15 minutes.

The DASH Instrument contains firmware and software that operate the instrument and analyze the assay data. Test results are displayed on the DASH Instrument screen and can be printed (optional) after the test has been completed. The procedural control will display “Valid” on the results screen for all samples, both negative and positive (see Table 1 below), to indicate that the test results may be reported.

Table 1. DASH SARS CoV-2 & Flu A/B Results Interpretation   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Result</td><td colspan="1" rowspan="1">Interpretation</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">The target nucleic acids of SARS-CoV-2 are detected.</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">The target nucleic acids of SARS-CoV-2 are not detected.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">The target nucleic acids of Flu A are detected.</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">The target nucleic acids of Flu A are not detected.</td></tr><tr><td colspan="1" rowspan="2">Flu B</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">The target nucleic acids of Flu B are detected.</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">The target nucleic acids of Flu B are not detected.</td></tr><tr><td colspan="1" rowspan="2">Procedural Control</td><td colspan="1" rowspan="1">Valid</td><td colspan="1" rowspan="1">Amplification meets acceptance criteria. Test results may bereported.</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Presence or absence of the target nucleic acids cannot bedetermined. Repeat test with a new cartridge.</td></tr><tr><td colspan="2" rowspan="1">Error</td><td colspan="1" rowspan="1">Presence or absence of the target nucleic acids cannot bedetermined. Instrument detected an error and aborted the currenttest.Repeat test with a new cartridge.</td></tr></table>

# C Instrument Description Information:

1. Instrument Name: DASH Rapid PCR Instrument (DASH Instrument)

2. Specimen Identification: DASH Instrument is connected to a barcode reader which scans specimen and Test cartridge labels.

3. Specimen Sampling and Handling:

The anterior nasal swab specimen is collected and added directly to the DASH SARS-CoV-2 & Flu A/B Test cartridge sample chamber. No collection media is used to store the swab. Each sample is processed individually.

4. Calibration: Not applicable.

5. Quality Control:

# • Procedural Control

An assay targeting the bacteriophage MS2 is built-in the Test cartridge as the Procedural Control (PRC) that undergoes identical processing to the patient specimen which provides confirmation that the test procedure was executed correctly. If the test fails for any reason, the PRC result will be reported as an INVALID and the operator is instructed in the labelling to test the sample again with another test device.

# External Controls

The DASH SARS-CoV-2 & Flu A/B Positive Control and DASH Negative Control swabs are available separately for use as external controls. External controls are tested using the same procedure that is used to test a patient specimen.

The DASH SARS-CoV-2 & Flu A/B Positive Control Swabs Kit is comprised of five single use, individually packaged swabs with $3 0 \mathrm { m m }$ breakpoint, containing a noninfectious solution of Flu A, Flu B, and SARS-CoV-2 viruses. The DASH Negative Control Swabs Kit is comprised of five single-use, individually packaged swabs with 30 mm breakpoint that are void of any added media.

V Substantial Equivalence Information:

A Predicate Device Name(s): BIOFIRE SPOTFIRE Respiratory (R) Panel Mini   
B Predicate 510(k) Number(s): K230719

C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K241652</td><td rowspan=1 colspan=1>K230719</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>DASH SARS-CoV-2 &amp;Flu A/B Test</td><td rowspan=1 colspan=1>BIOFIRE SPOTFIRERespiratory (R) PanelMini</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Regulation Number andName</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>21 CFR 866.3981;Devices to detect andidentify nucleic acidtargets in respiratorysamples from microbialagents that cause theSARS-CoV-2respiratory infectionand other microbialagents when in a multi-analyte test.</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>QOF</td></tr><tr><td rowspan=1 colspan=1>Technology/Detection</td><td rowspan=1 colspan=1>PCR with streptavidin-coated paramagneticparticles andfluorescence detection</td><td rowspan=1 colspan=1>PCR with DNA meltinganalysis</td></tr><tr><td rowspan=1 colspan=1>Intended Use/IndicationsFor Use</td><td rowspan=1 colspan=1>The DASH SARS-CoV-2 &amp; Flu A/B Testis a rapid reversetranscriptionpolymerase chainreaction (RT-PCR)assay performed on theDASH Rapid PCRInstrument and is</td><td rowspan=1 colspan=1>The BIOFIRESPOTFIRE RespiratoryPanel Mini (SPOTFIRER Panel Mini) is amultiplexed polymerasechain reaction (PCR)test intended for usewith the BIOFIRESPOTFIRE System for</td></tr></table>

<table><tr><td colspan="2">intended for the the simultaneous, simultaneous in vitro qualitative detection</td></tr><tr><td>qualitative detection and differentiation of SARS-CoV-2, influenza A and influenza B virus ribonucleic acid (RNA) in anterior nasal swab specimens from patients with signs and symptoms of respiratory tract infection. The test is intended to aid in the differential diagnosis of SARS-CoV-2, influenza A, and influenza B in humans in conjunction with other clinical, epidemiologic and laboratory findings. Positive results of a specific target are indicative of the presence of that viral RNA and may not be the definite cause of disease. Positive results do not rule out co- infection with other pathogens. Negative results do not preclude SARS-CoV-2, influenza A or influenza B infection and should not be used as the sole basis for patient management decisions.</td><td>and identification of multiple respiratory viral nucleic acids in nasopharyngeal swab (NPS) specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19. The following organism types are identified and differentiated using the SPOTFIRE R Panel Mini: Coronavirus SARS- CoV-2 Human rhinovirus Influenza A virus Influenza B virus Respiratory syncytial virus Nucleic acids from the viral organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection are indicative of the presence of the identified microorganism and aids</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">clinical andepidemiologicalinformation, andlaboratory findings. Theresults of this testshould not be used asthe sole basis fordiagnosis, treatment, orother patientmanagement decisions.Negative results in thesetting of a respiratoryillness may be due toinfection withpathogens that are notdetected by this test, orlower respiratory tractinfection that may notbe detected by an NPSspecimen. Positiveresults do not rule outcoinfection with otherorganisms. The agent(s)detected by theSPOTFIRE R PanelMini may not be thedefinite cause ofdisease.Additional laboratorytesting (e.g., bacterialand viral culture,immunofluorescence,and radiography) maybe necessary whenevaluating a patientwith possiblerespiratory tractinfection.</td></tr><tr><td colspan="1" rowspan="1">Test Format</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Single use</td></tr><tr><td colspan="1" rowspan="1">Results Reported</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">~15 minutes</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Target viruses</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Coronavirus SARS-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza AInfluenza B</td><td colspan="1" rowspan="1">CoV-2Human rhinovirusInfluenza A virusInfluenza B virusRespiratory syncytialvirus</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Anterior Nasal Swab</td><td colspan="1" rowspan="1">Nasopharyngeal swabin transport media</td></tr><tr><td colspan="1" rowspan="1">Automated Test Processes</td><td colspan="1" rowspan="1">Sample extraction,amplification, anddetection</td><td colspan="1" rowspan="1">Amplification anddetection</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal proceduralControl included in thecartridgeExternal Controls(purchased separately)</td><td colspan="1" rowspan="1">Two controls areincluded in each reagentpouch to control forsample processing andboth stages of PCR andmelt analysis.External Controls areprovided separately (seeK230868)</td></tr><tr><td colspan="1" rowspan="1">Instrument Systems</td><td colspan="1" rowspan="1">DASH Rapid PCRInstrument</td><td colspan="1" rowspan="1">SPOTFIRE System</td></tr></table>

# VI Standards/Guidance Documents Referenced:

IEC 60601-1-2 Ed. 4.1:2020 Medical electrical equipment - Part 1-2: General requirements for basic safety and essential performance - Collateral Standard: Electromagnetic disturbances - Requirements and tests IEC 61010-1, 3rd edition, May 11, 2012, revision 2023-06-06: Safety requirements for electrical equipment for measurement, control, and laboratory use - Part 1: general requirements IEC 61010-2-010:2019 Safety requirements for electrical equipment for measurement, control, and laboratory use - Part 2-010: Particular requirements for laboratory equipment for the heating of materials IEC 61010-2-101:2018 Safety requirements for electrical equipment for measurement, control, and laboratory use - Part 2-101: Particular requirements for in vitro diagnostic (IVD) medical equipment CLSI EP12 3rd Edition, Evaluation of Qualitative, Binary Output Examination Performance CLSI EP12-A2, User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition. • CLSI MM03-3rd Edition, Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline CLSI EP07 3rd Edition, Interference Testing in Clinical Chemistry CLSI EP37 1st Edition, Supplemental Tables for Interference Testing in Clinical Chemistry • CLSI EP25 2nd Edition, Evaluation of Stability of In Vitro Medical Laboratory Test Reagents • CLSI EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

# a) Precision/Repeatability

Precision of the DASH SARS-CoV-2 & Flu A/B Test was evaluated over a 12-day period, conducted by two operators at a single site, and resulted in 96 replicates per concentration of the SARS-CoV-2, Flu A and Flu B targets. Test samples were contrived in simulated clinical nasal matrix, co-spiked with SARS-CoV-2, Flu A, and Flu B to target concentrations. Three panel members were prepared: true negative (unspiked), low positive (2x LoD (near Limit of Detection)), and moderate positive (5x LoD). The test samples were prepared by applying the contrived viral dilutions onto the swabs. The samples were randomized and blind-coded. Each operator tested two replicates of each sample in each run (two runs per day), for a total of 96 replicates per panel member. Four DASH Instruments were utilized during the execution of this study. Results are summarized in the table below.

Table 2: Summary of Qualitative Results for Precision/Repeatability Study Results   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>SampleConcentration</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>n/N[1]</td><td rowspan=1 colspan=1>Agreement withExpected Results (%)</td></tr><tr><td rowspan=4 colspan=1>SARS-CoV-2</td><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr><tr><td rowspan=4 colspan=1>Flu A</td><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr><tr><td rowspan=4 colspan=1>Flu B</td><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr><tr><td rowspan=2 colspan=1>Negative</td><td rowspan=2 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td></tr></table>

$[ 1 ] _ { \mathtt { n } }$ is number of tests with expected results. N is the total number of valid tests

Table 3: Summary of Precision/Repeatability Results (Cq Variability Analysis)   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">SampleConc.</td><td colspan="1" rowspan="1">n/N[1]</td><td colspan="1" rowspan="1">MeanCq1{2]</td><td colspan="1" rowspan="1">BetweenOperator</td><td colspan="1" rowspan="1">BetweenDay</td><td colspan="1" rowspan="1">BetweenRuns</td><td colspan="1" rowspan="1">Within Run/Repeatability</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">(WithinOperator</td><td colspan="2" rowspan="1">(WithinDay</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">29.4</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.7</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">28.2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">2.0</td></tr><tr><td colspan="1" rowspan="2">Flu A</td><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">24.0</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">5.2</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">5.5</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">22.7</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">3.0</td></tr><tr><td colspan="1" rowspan="2">Flu B</td><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">25.9</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">3.6</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">3.6</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">24.8</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.7</td></tr></table>

$[ 1 ] _ { \mathtt { n } }$ is number of tests with expected results. N is the total number of valid tests $\mathsf { \Pi } ^ { [ 2 ] } \mathrm { D A S H }$ software identifies the cycle at which an amplification curve meets a predefined mathematical criterion as Cq. $\mathrm { C q } =$ quantification cycle, $\mathrm { S D } =$ standard deviation, $\% { \mathrm { C V } } =$ percent coefficient of variation

# b) Reproducibility

Another study was conducted to evaluate the reproducibility of the DASH SARS-CoV-2 & Flu A/B Test at three external CLIA-waived sites with a total of nine untrained operators. A total of three lots of the Test cartridges were used. Operators tested a randomized blindcoded panel of contrived samples at three concentration levels (prepared in the same manner as above for the precision study), with three replicates per level tested by each operator over five days. There were 270 replicates of each concentration tested over the study period (3 replicates $_ { \textrm { X 2 } }$ runs $\textbf { X } 3$ operators x 5 days x 3 sites). Each operator ran one set of external controls (i.e., positive control and negative control) on their assigned instrument prior to testing reproducibility samples daily.

The results are summarized below in Table 4 and 5. No significant differences between sites, or between operators were observed.

Table 4: Summary of Qualitative Results for Reproducibility Study Results   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">SampleConcentration</td><td colspan="4" rowspan="1">% Agreement with Expected Results[1](n/N|21)(95% Confidence Interval)</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Overall</td></tr><tr><td colspan="1" rowspan="3">SARS-CoV-2</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">97.8%(88/90)(92.3% - 99.4%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">99.3%(268/270)(97.3% -99.8%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">98.9%(89/90)(94.0% - 99.9%)</td><td colspan="1" rowspan="1">100.0%(89/89)[3](95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">99.6%(268/269)(97.9% -100.0%)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(270/270)(98.6% -100.0%)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">97.8%(88/90)</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">99.3%(268/270)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(95.9% -100.0%)</td><td colspan="1" rowspan="1">(92.3% - 99.4%)</td><td colspan="1" rowspan="1">(95.9% -100.0%)</td><td colspan="1" rowspan="1">(97.3% -99.8%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(89/89)3](95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(269/269)(98.6% -100.0%)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(270/270)(98.6% -100.0%)</td></tr><tr><td colspan="1" rowspan="3">Flu B</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">97.8%(88/90)(92.3% - 99.4%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">99.3%(268/270)(97.3% -99.8%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(89/89)[3](95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(269/269)(98.6% -100.0%)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(90/90)(95.9% -100.0%)</td><td colspan="1" rowspan="1">100.0%(270/270)(98.6% -100.0%)</td></tr></table>

${ } ^ { [ 1 ] } \mathrm { A }$ total of 15 samples generated invalid results on the first attempt; 14 samples generated valid results upon repeat and were included in the data analysis. $[ 2 ] _ { \mathrm { n } }$ is number of tests with expected results. N is the total number of valid tests [3]One moderate positive sample produced an error/no result and was excluded from the analysis, yielding a total of 89 results for this panel member at Site 2.

Table 5: Summary of Reproducibility Results (Cq Variability Analysis)   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>SampleConc.</td><td rowspan=2 colspan=1>n/N]</td><td rowspan=2 colspan=1>MeanCqq2}</td><td rowspan=1 colspan=2>BetweenSite</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>268/270</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>9.4</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>268/269</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.9</td></tr><tr><td rowspan=2 colspan=1>Flu A</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>268/270</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>9.6</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>269/269</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>268/270</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>9.1</td></tr><tr><td rowspan=1 colspan=1>Moderateositive</td><td rowspan=1 colspan=1>269/269</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.3</td></tr></table>

$[ 1 ] _ { \mathtt { n } }$ is number of tests with expected results. $_ \mathrm { N }$ is the total number of valid tests [2]DASH software identifies the cycle at which an amplification curve meets a predefined mathematical criterion as $\operatorname { C q }$ ${ \mathrm { C q } } =$ quantification cycle, $\mathrm { S D } =$ standard deviation, $\% { \mathrm { C V } } =$ percent coefficient of variation

2. Linearity: Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

a) Cross-Reactivity Study

# Wet Testing

Cross-Reactivity was evaluated for the DASH SARS-CoV-2 & Flu A/B Test on the DASH Instrument by wet testing of 50 different viruses, bacteria, and fungi that could potentially interfere with the assay targets. All samples were prepared in simulated clinical nasal matrix and tested individually, without the presence of the test viral targets. The testing concentrations for potentially interfering microorganisms were ${ \overset { - } { \geq } } 1 0 ^ { 5 }$ units/mL for viruses and ${ \geq } 1 0 ^ { 6 }$ units/mL for other microorganisms, unless otherwise noted in the table below.

None of the evaluated organisms demonstrated cross-reactivity with the assay at the tested concentrations as shown in the table below.

Table 6: Cross-Reactivity Testing Results   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">MicroorganismConcentration(per Swab)</td><td colspan="1" rowspan="1">MicroorganismConcentration(per mL)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type C1</td><td colspan="1" rowspan="1">1.44E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.6E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (also known as HHV-5)</td><td colspan="1" rowspan="1">2.82E+03 TCID50/swab</td><td colspan="1" rowspan="1">7.05E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus (also known as HHV-4)</td><td colspan="1" rowspan="1">1.51E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.78E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">4.00E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">4.75E+03 TCID50/swab</td><td colspan="1" rowspan="1">2.19E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">1.10E+04 Copies/swab</td><td colspan="1" rowspan="1">2.75E+05 Copies/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">4.00E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">1.19E+04 TCID50/swab</td><td colspan="1" rowspan="1">2.98E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">1.54E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.85E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Middle East Respiratory Syndrome coronavirus(MERS-CoV)</td><td colspan="2" rowspan="1">N/A[]</td></tr><tr><td colspan="1" rowspan="1">Genomic RNA Middle East RespiratorySyndrome coronavirus (MERS-CoV)EMC/2012</td><td colspan="1" rowspan="1">1.35 ng RNA/swab</td><td colspan="1" rowspan="1">33.8 ng RNA/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1.19E+04 TCID50/swab</td><td colspan="1" rowspan="1">2.98E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">4.75E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.19E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">4.00E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1.44E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.6E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">2.82E+03 TCID50/swab</td><td colspan="1" rowspan="1">7.05E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus</td><td colspan="1" rowspan="1">2.82E+03 TCID50/swab</td><td colspan="1" rowspan="1">7.05E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus type 1A</td><td colspan="1" rowspan="1">2.82E+03 TCID50/swab</td><td colspan="1" rowspan="1">7.05E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus type 68</td><td colspan="1" rowspan="1">3.40E+03 TCID50/swab</td><td colspan="1" rowspan="1">8.5E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Severe Acute Respiratory Syndromecoronavirus (2003) RNA</td><td colspan="1" rowspan="1">9.80E+03 Copies/swab</td><td colspan="1" rowspan="1">2.45E+05 Copies/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis (heat-inactivated)</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata subsp. glycolytica)</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Nocardia asteroides</td><td colspan="1" rowspan="1">1.74E+05 CFU/swab</td><td colspan="1" rowspan="1">4.35E+06 CFU/swab</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">5.00E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr></table>

[1]Concentration not available from vendor. Sample is qualitative non-infectious purified intact viral particles.

# In silico

In silico cross-reactivity of the DASH SARS-CoV-2 & Flu A/B Test was explored by evaluating genomes representing the viral, bacterial, and fungal organisms in the wet testing cross-reactivity study (see table above). $10 \%$ of complete genomes of these microorganisms were randomly selected and downloaded from GenBank to be included in this analysis. Any microorganism showing ${ > } 8 0 \%$ homology to a single primer, probe, or capture oligonucleotide was further evaluated for homology for the other target primers and probes for amplification. DASH SARS-CoV-2 & Flu A/B Test’s forward and reverse primers for the SARS-CoV-2 target exhibited ${ > } 8 0 \%$ homology to SARS-coronavirus Tor2 isolate and therefore may be amplified but will not be detected due to inability of the test target probes to bind. Performance of DASH SARS-CoV-2 & Flu A/B Test with SARS-coronavirus Tor2 has not been confirmed by wet testing analysis.

# b) Microbial Interference Study

Microbial Interference was evaluated for the DASH SARS-CoV-2 & Flu A/B Test on the DASH Instrument by wet testing over 50 different viruses, bacteria, and fungi that are common causes of respiratory infections. All samples were prepared in simulated clinical nasal matrix and tested with individual microorganisms or with pooled groups of up to four microorganisms in the presence of the test viral targets, SARS-CoV-2, influenza A and influenza B, co-spiked at $3 \mathbf { x }$ LoD. Microbial non-interference was determined if the target had a $100 \%$ positive call rate for all three replicates tested (3/3).

None of the evaluated microorganisms demonstrated interference with the assay at the tested concentrations as shown in the table below.

Table 7: Microbial Interference Testing Results   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">MicroorganismConcentration(per Swab)</td><td colspan="1" rowspan="1">MicroorganismConcentration(per mL)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C1</td><td colspan="1" rowspan="1">1.44E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.6E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (also known as HHV-5)</td><td colspan="1" rowspan="1">2.48E+03 TCID50/swab</td><td colspan="1" rowspan="1">6.2E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus (also known as HHV-4)</td><td colspan="1" rowspan="1">1.51E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.78E+05TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">4.00E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">4.75E+03 TCID50/swab</td><td colspan="1" rowspan="1">2.19E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">9.68E+03 Copies/swab</td><td colspan="1" rowspan="1">2.41E+05 Copies/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63[]</td><td colspan="1" rowspan="1">4.0E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Measles virus[1]</td><td colspan="1" rowspan="1">1.54E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.85E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Genomic RNA Middle East RespiratorySyndrome coronavirus (MERS-CoV)</td><td colspan="1" rowspan="1">1.35 ng RNA/swab</td><td colspan="1" rowspan="1">33.8 ng RNA/mL</td></tr><tr><td colspan="1" rowspan="1">Middle East Respiratory Syndromecoronavirus (MERS-CoV)</td><td colspan="2" rowspan="1">N/A{2]</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1.19E+04 TCID50/swab</td><td colspan="1" rowspan="1">2.98E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">1.19E+04 TCID50/swab</td><td colspan="1" rowspan="1">2.98E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">4.75E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.19E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">4.0E+03 TCID50/swab</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1.44E+04 TCID50/swab</td><td colspan="1" rowspan="1">3.6E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">2.48E+03 TCID50/swab</td><td colspan="1" rowspan="1">6.20E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus</td><td colspan="1" rowspan="1">2.48E+03 TCID50/swab</td><td colspan="1" rowspan="1">6.2E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus type 1A</td><td colspan="1" rowspan="1">2.48E+03 TCID50/swab</td><td colspan="1" rowspan="1">6.2E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus type 68</td><td colspan="1" rowspan="1">2.99E+03 TCID50/swab</td><td colspan="1" rowspan="1">7.48E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Severe Acute RespiratorySyndrome coronavirus (2003) RNA</td><td colspan="1" rowspan="1">8.62E+03 Copies/swab</td><td colspan="1" rowspan="1">2.16E+05 Copies/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Fusobacteriumnecrophorum</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis (heat- inactivated)</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata subsp.glycolytica</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">5.0E+04 CFU/swab</td><td colspan="1" rowspan="1">1.25E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Nocardia asteroides</td><td colspan="1" rowspan="1">1.74E+05 CFU/swab</td><td colspan="1" rowspan="1">4.35E+06 CFU/mL</td></tr></table>

Note: Thicker outer border indicates groups of pooled microorganisms that were tested. [1]Tested individually as interference was observed when tested as a group. [2]Concentration not available from vendor. Sample is qualitative non-infectious purified intact viral particles.

# c) Competitive Interference Study

The impact of competitive interference, caused by co-infections with on-target analytes, was evaluated for the DASH SARS-CoV-2 & Flu A/B Test on the DASH Instrument by testing contrived samples consisting of individual SARS-CoV-2, Flu A or Flu B strains at high concentrations in the presence of the other two target strains at $3 \mathbf { x }$ LoD. For this study, competitive interference was assessed using one strain each of SARS-CoV-2, influenza A and influenza B. Testing for each target strain (at 3x LoD concentration) and each potential competitive strain (at high concentration) was performed in triplicate. No competitive interference was observed if all replicates for the low concentration (3x LoD) target yielded positive results.

The study showed that SARS-CoV-2 at concentrations ${ > } 1 . 4 1 \mathrm { E } { + } 0 6 $ copies/mL inhibited detection of Flu A at $3 \mathbf { x }$ LoD, and at SARS-CoV-2 concentrations of ${ > } 5 . 6 5 \mathrm { E } { + } 0 6$ copies/mL inhibited detection of Flu B at $3 \mathbf { x }$ LoD. Flu A at concentration of $1 . 3 8 \mathrm { E + 0 6 } \mathrm { T C I D 5 0 } / \mathrm { m L }$ inhibited detection of Flu B at $3 \mathbf { x }$ LoD. In addition, Flu B at concentrations above $1 . 0 1 \mathrm { E } { + 0 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ inhibited detection of both SARS-CoV-2 and Flu A at 3x LoD. The highest coinfection concentration of the competing virus, where the remaining two targets are detected for three replicates at $3 \mathbf { x }$ LoD, are reported in the table below.

Table 8: Competitive Interference Study Results   

<table><tr><td colspan="1" rowspan="2">CompetingVirus</td><td colspan="3" rowspan="1">Concentration at which All 3 Targets are Positive</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1.41E+06 copies/mL~47x LoD</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">1.38E+06 TCID50/mL~3405x LoD</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td>Flu B</td><td>3x LoD</td><td>3x LoD</td><td>1.01E+03 TCID50/mL ~60x LoD</td></tr></table>

# d) Exogenous/Endogenous Interfering Substances

The potential for interference for the DASH SARS-CoV-2 & Flu $\mathrm { A } / \mathrm { B }$ Test was evaluated with substances that may be present in respiratory specimens. A total of 22 potentially interfering endogenous and exogenous substances were tested at or above clinically relevant levels in simulated clinical nasal matrix co-spiked at ${ \sim } 3 \mathbf { x }$ LoD with SARS-CoV-2, Flu A and Flu B inactivated viral particles. Each sample was tested in triplicate. The FluMist nasal vaccine was not tested.

Two of the substances tested in the study, Biotin (vitamin B-7) at $4 . 5 8 ~ \mu \mathrm { g / m L }$ and Flonase (active ingredient Fluticasone Propionate) at $5 \%$ (v/v) gave false negative results with SARSCoV-2 target. Additional testing at lower test substance concentrations were performed to determine the concentration where interference is no longer observed. No interference was observed for any of the substances tested at the concentrations noted in the table below.

Table 9: Exogenous/Endogenous Interfering Substances Study Results   

<table><tr><td colspan="1" rowspan="1">Interfering Substance</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Afrin Original Nasal Spray</td><td colspan="1" rowspan="1">15% v/v</td></tr><tr><td colspan="1" rowspan="1">Chloroseptic Sore Throat spray</td><td colspan="1" rowspan="1">20% v/v</td></tr><tr><td colspan="1" rowspan="1">Flonase</td><td colspan="1" rowspan="1">2.5% v/v[1]</td></tr><tr><td colspan="1" rowspan="1">Relenza</td><td colspan="1" rowspan="1">0.3 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">10 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">0.1% w/v</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">2% v/v</td></tr><tr><td colspan="1" rowspan="1">Biotin</td><td colspan="1" rowspan="1">2.29 μg/mL []</td></tr><tr><td colspan="1" rowspan="1">Viral Transport Medium</td><td colspan="1" rowspan="1">50% v/v</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine Nasal Spray</td><td colspan="1" rowspan="1">15% v/v</td></tr><tr><td colspan="1" rowspan="1">Walgreens Saline Nasal Spray</td><td colspan="1" rowspan="1">15% v/v</td></tr><tr><td colspan="1" rowspan="1">Beclomethasone Dipropionate, USP (Micronized)</td><td colspan="1" rowspan="1">0.068 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Flunisolide Nasal Solution, USP 0.025%</td><td colspan="1" rowspan="1">0.04 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasacort Nasal Spray</td><td colspan="1" rowspan="1">0.04 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Walgreens 24 Hour Budesonide Nasal Spray</td><td colspan="1" rowspan="1">0.051 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasonex</td><td colspan="1" rowspan="1">0.04 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Zicam Allergy Relief Nasal Spray</td><td colspan="1" rowspan="1">20% v/v</td></tr><tr><td colspan="1" rowspan="1">Snuff</td><td colspan="1" rowspan="1">0.1% w/v</td></tr><tr><td colspan="1" rowspan="1">Latex glove powder</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td>Mucinex Insta Sooth Kickstart Sore Throat</td><td>1.7 mg/mL</td></tr><tr><td>Zinc gluconate hydrate</td><td>0.1 μg/mL</td></tr></table>

[1]Interference was observed at concentrations higher than these concentrations.

4. Assay Reportable Range: Not applicable.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# a) External Controls

The DASH SARS-CoV-2 & Flu A/B Positive Control and DASH Negative Control swabs are available separately for use as external controls. Please refer to Instrument Description Information (Section C.5) above for assay controls. A study was conducted to evaluate these external controls with DASH SARS-CoV-2 & Flu A/B Test. A total of 45 DASH SARSCoV-2 & Flu A/B Positive Control and 45 DASH Negative Control swabs were tested daily by nine untrained operators at three CLIA-Waived sites across five testing days. Three lots of external controls were distributed evenly across all the three sites. All 45 external positive controls and all 45 external negative controls yielded accurate results.

# b) Sample Stability

A study was conducted to determine the specimen stability on the swab after collection and prior to testing with the DASH SARS-CoV-2 & Flu A/B Test on the DASH Instrument. Specimens included SARS-CoV-2, Flu A and Flu B, co-spiked in pooled negative nasal matrix at a concentration of $2 \mathbf { x }$ LoD. Twenty replicates of the positive samples were tested fresh (T0) and after storage at multiple time points (5, 10, 25, 30 and 40 minutes) at room temperature. Negative samples were also included and were tested in three replicates for each storage condition. Results indicated that positive samples remained stable, and all targets were detected up to twenty-five minutes after collection and storage at room temperature. Negative samples tested correctly for each timepoint.

Per test labeling, patient samples should be tested immediately after collection for optimal test performance.

# c) Matrix Equivalency

The objective of this study was to establish equivalent performance of the DASH SARSCoV-2 & Flu A/B Test on the DASH Instrument between the two matrices used in the analytical studies: clinical pooled nasal matrix (PNM) and simulated clinical nasal matrix (SCNM). SCNM was formulated in phosphate-buffered saline (PBS) with glycerol and mucin, added to mimic the nasal cavity fluid. For this study, SARS-CoV-2 (Isolate hCoV19/USA/GAEHC-2811C/2021), Flu A (A/Victoria/2570/2019) and Flu B (B/Utah/9/2014) were co-spiked in both matrices at $2 \mathbf { x }$ and 5x LoD. Aliquots of negative matrices were also included in the evaluation. For both matrices, ten replicates of the negative samples, 40 replicates of the samples prepared at $2 \mathbf { x }$ LoD and ten replicates of the samples prepared at 5x LoD were tested. The acceptance criteria to demonstrate equivalency was $2 9 5 \%$ detection for samples at 2x LoD, $100 \%$ detection for the samples at 5x LoD for each target, and $0 \%$ detection of the negative samples. The results obtained in this study are summarized below. The data demonstrated equivalent performance of the test with both the matrices.

Table 10: Matrix Equivalency Study Results   

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>n/N[1]</td><td rowspan=1 colspan=1>Detection Rate(% Positive)</td></tr><tr><td rowspan=4 colspan=1>SARS CoV-2</td><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=4 colspan=1>Flu A</td><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=4 colspan=1>Flu B</td><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>Negative Matrix</td><td rowspan=2 colspan=1>N/A</td><td rowspan=1 colspan=1>PNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>SCNM</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0</td></tr></table>

$[ 1 ] _ { \mathtt { n } }$ is number of tests with expected results. N is the total number of valid tests

# 6. Detection Limit:

Limit of detection (LoD) of the DASH SARS-CoV-2 & Flu A/B Test on the DASH Rapid PCR Instrument was determined utilizing five viral strains representing the test targets. All samples were prepared in pooled negative nasal matrix. Initially, three-fold serial dilutions of each strain in pooled negative nasal matrix were prepared and tested in replicates of four to find the lowest level detectable by the DASH SARS-CoV-2 & Flu A/B Test. The final LoD, a concentration producing $9 5 \%$ positivity, was confirmed by testing twenty replicates of each of the strains at the selected preliminary LoD. The final LoD results are summarized in the tables below.

Table 11: Limit of Detection Study Results   

<table><tr><td colspan="1" rowspan="1">Viral Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Source/Product Type</td><td colspan="1" rowspan="1">LoD(copies orTCID50/swab)</td><td colspan="1" rowspan="1">LoD(copies orTCID50/mL)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">hCoV- 19/USA/GA-EHC- 2811C/2021(Lineage B.1.1.529; Omicron Variant)</td><td colspan="1" rowspan="1">BEI Resources/Gamma-Irradiated</td><td colspan="1" rowspan="1">1200copies/swab</td><td colspan="1" rowspan="1">3.0E+04copies/mL</td></tr><tr><td colspan="1" rowspan="2">Flu A</td><td colspan="1" rowspan="1">H1N1 Victoria/2570/19</td><td colspan="1" rowspan="1">Zeptometrics/Culture Fluid</td><td colspan="1" rowspan="1">1.35TCID50/swab</td><td colspan="1" rowspan="1">33.75TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H3N2 Darwin/9/21</td><td colspan="1" rowspan="1">Zeptometrics/Culture Fluid</td><td colspan="1" rowspan="1">0.225TCID50/swab</td><td colspan="1" rowspan="1">5.63TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">Washington/02/19 - Victoria Lineage</td><td colspan="1" rowspan="1">Zeptometrics/Culture Fluid</td><td colspan="1" rowspan="1">0.1TCID50/swab</td><td colspan="1" rowspan="1">2.50TCID50/mL</td></tr><tr><td></td><td>Utah/9/14 - Yamagata Lineage</td><td>Zeptometrics/ Culture Fluid</td><td>0.675 TCID50/swab</td><td>16.88 TCID50/mL</td></tr></table>

# 7. Inclusivity (Analytical Reactivity):

# Wet Testing

The inclusivity of the DASH SARS-CoV-2 & Flu A/B Test on the DASH Instrument was evaluated by testing seven strains of SARS-CoV-2, 21 strains of Flu A, and 10 strains of Flu B. Positive samples were contrived by spiking each virus strain, individually, into negative pooled nasal matrix at concentrations of ${ \sim } 3 \mathbf { x }$ LoD and testing in triplicate in a blinded fashion. Inclusivity was determined if the target had a $100 \%$ detection (3/3) for all three replicates tested. If less than $100 \%$ detection was observed, the strain was prepared at a higher concentration and tested until $100 \%$ detection was achieved. The strains evaluated and the lowest concentration that met the inclusivity criteria outlined above are shown in the table below.

Table 12: Inclusivity Wet Testing Results   

<table><tr><td colspan="1" rowspan="1">Test Target Variant/Subtype</td><td colspan="1" rowspan="1">Virus Concentration(per Swab)</td><td colspan="1" rowspan="1">Virus Concentration(per mL)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2B.1.1.7 (England/204820464/2020)</td><td colspan="1" rowspan="1">0.0828 TCID50/swab[1]</td><td colspan="1" rowspan="1">2.07 TCID50/mL[1]</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2B.1.351 (South Africa/KRISP-K005325/2020)</td><td colspan="1" rowspan="1">0.0414 TCID50/swab</td><td colspan="1" rowspan="1">1.04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2B.1.617.2; Delta Variant (USA/PHC658/2021)</td><td colspan="1" rowspan="1">0.1656 TCID50/swab[2]</td><td colspan="1" rowspan="1">4.14 TCID50/mL [2]</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2P1; Gamma Variant (Japan/TY7-503/2021)</td><td colspan="1" rowspan="1">0.0414 TCID50/swab</td><td colspan="1" rowspan="1">1.04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2B.1.617.1; Kappa Variant (USA/CA-Stanford-15_S02/2021)</td><td colspan="1" rowspan="1">0.0828 TCID50/swab[]</td><td colspan="1" rowspan="1">2.07 TCID50/mL[1]</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2BA 2.3; Omicron Variant</td><td colspan="1" rowspan="1">0.0414 TCID50/swab</td><td colspan="1" rowspan="1">1.04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Gamma-irradiated SARS-CoV-2isolate USA-WA1/2020</td><td colspan="1" rowspan="1">7200 copies/swab</td><td colspan="1" rowspan="1">1.8E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (New Cal/20/99)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (PR/8/34)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Brisbane/59/07)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Taiwan/42/06)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (Singapore/63/04)</td><td colspan="1" rowspan="1">4.05 TCID50/swab[3]</td><td colspan="1" rowspan="1">101.25 TCID50/mL [3]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm (California/07/09)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm (New York/18/09)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm Virus (Guangdong-Maonan/SWL 1536/19)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm (Mexico/4108/09)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm (Victoria/2570/19)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Wisconsin/67/05)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 Virus (HongKong/2671/19)</td><td colspan="1" rowspan="1">8.1 TCID50/swab[1]</td><td colspan="1" rowspan="1">202.50 TCID50/mL[1]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Stockholm/6/14)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Hong Kong/8/68)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Texas/50/12)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Kansas/14/17)</td><td colspan="1" rowspan="1">8.1 TCID50/swab[1]</td><td colspan="1" rowspan="1">202.50 TCID50/mL[]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (California/7/04)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Norway/466/14)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Kumamoto/102/02)</td><td colspan="1" rowspan="1">4.05 TCID50/swab</td><td colspan="1" rowspan="1">101.25 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (Brisbane/10/07)</td><td colspan="1" rowspan="1">8.1 TCID50/swab[1]</td><td colspan="1" rowspan="1">202.5 TCID50/mL [1]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H5N3 (Genomic RNA fromKilbourne F3:A/duck/Singapore/645/1997mutant)</td><td colspan="1" rowspan="1">1.76E+03 ng RNA/swab</td><td colspan="1" rowspan="1">4.4E+04 ng RNA/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Lee/40)</td><td colspan="1" rowspan="1">4.05 TCID50/swab[1]</td><td colspan="1" rowspan="1">101.25 TCID50/mL[1]</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Massachusetts/2/12)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Victoria Lineage(Austria/1359417/21)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Victoria Lineage (Victoria/2/87)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Victoria Lineage (Alabama/2/17)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Yamagata Lineage(Panama/45/90)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Yamagata Lineage (Florida/04/06)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B Yamagata Lineage(Brisbane/9/14)</td><td colspan="1" rowspan="1">2.025 TCID50/swab</td><td colspan="1" rowspan="1">50.63 TCID50/mL</td></tr></table>

[1]Testing at a lower concentration (i.e., 3x LoD) failed to yield $! 0 0 \%$ detection. The lowest concentration that yielded $100 \%$ detection was 6x LoD. [2]Testing at a lower concentration (i.e., 3x LoD) failed to yield $! 0 0 \%$ detection. The lowest concentration that yielded $100 \%$ detection was $1 2 \mathbf { x }$ LoD. [3]Influenza A H1N1 (Singapore/63/04) was detected in $2 / 3$ replicates at $3 \mathbf { x }$ LoD. The strain was reprepared at 3x LoD and tested in 7 replicates for $100 \%$ detection.

# In silico

The inclusivity of the DASH SARS-CoV-2 & Flu A/B Test was also evaluated using in silico analysis of the sample preparation capture oligonucleotides and the PCR primers and probes for SARS-CoV-2 and Flu A and B in relation to sequences available in the National Center for Biotechnology Information (NCBI) SARS-CoV-2 and influenza virus database. Genomic sequences for SARS-CoV-2 were retrieved from the NCBI on May 8, 2024, and included all lineages and variants of concern (VOC) or variants of interest (VOI). The influenza genomic sequences were retrieved from the NCBI on May 13, 2024. Flu A sequences were limited to subtypes H1N1 (excluding Swine flu) and H3N2, based upon completeness. Flu B sequences were required to be full-length, with no subtype limitations applied. All genomic sequences containing ambiguous nucleotides were excluded from the analysis.

The query oligonucleotides for the specific virus target were aligned to the filtered sets of viral genome sequences using the Burrows-Wheeler Aligner (BWA) program, where any difference would be a mismatch or an insertion/deletion. The genomic sequences were ranked according to the number of capture oligonucleotides and PCR primers and probes containing zero, one, and two or more mismatches. Melting temperature (Tm) analysis was performed for any oligonucleotide:genome pairing that contained a mismatch, and the pairings that passed the Tm analysis criteria were combined with pairings containing no mismatches, then grouped into amplifiable sets. The resulting oligonucleotide:genome pairings were counted to determine the number of sequences predicted to be captured and, if captured, predicted to be detected by the DASH SARS-CoV-2 & Flu A/B Test. The results of this predicted-to-detect analysis are summarized in the table below:

Table 13: Sequences Predicted by In Silico Analysis to Be Detected by DASH SARSCoV-2 & Flu A/B Test   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Sequences Predicted to be Detected</td></tr><tr><td rowspan=1 colspan=1>Flu A(Subtypes H1N1 and H3N2)</td><td rowspan=1 colspan=1>≥99.97% (18112 of 18117 sequences analyzed)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>≥98.27% (8136 of 8279 sequences analyzed)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>≥99.99% (994778 of 994846 sequences analyzed)</td></tr></table>

Based on the analysis, $9 9 . 9 9 \%$ (994778/994846) of SARS-CoV-2 sequences, $9 9 . 9 7 \%$ (18112/18117) of influenza A subtype H1N1 and H3N2 sequences and $9 8 . 3 \%$ (8136/8279) of influenza B sequences are expected to be detected by the DASH SARS-CoV-2 & Flu A/B Test.

Inclusivity Information for Non-Seasonal Influenza A Subtypes (in silico Analysis) DASH SARS-CoV-2 & Flu A/B Test in silico inclusivity analysis was performed by aligning with sequences corresponding to the gene target in Flu A subtype H5N3 and H7N7 sequences downloaded from GISAID (Global Initiative on Sharing All Influenza Data). Reactivity was predicted based on the number and location of the mismatches in the targeted region. The results of the BLAST analysis demonstrated that the subtype H5N3 and H7N7 of Flu A are expected to be detected by DASH SARS-CoV-2 & Flu A/B Test. Wet testing for H5N3 strain of Flu A was also performed, confirming inclusivity of DASH SARS-CoV-2 & Flu A/B Test with the H5N3 strain of Flu A (see Table 12 above). The Flu A H7N7 strain was not available for wet testing confirmation.

Additional Flu A in silico inclusivity analysis was performed for Flu A subtypes available from the GISAID database from November 5, 2023 to November 5, 2024 that have infected humans. In this analysis, all possible combinations of hemagglutinin and neuraminidase were considered, and all available sequences were required to contain the Flu A target region, found in humans, and to be full-length as reported by the submitters. All sequences containing ambiguous nucleotides were excluded from the analysis. Based on these criteria, a total of 37 sequences including subtype H5N1 (26), H5N6 (3), H9N2 (6), H10N3 (1), and H10N5 (1) were analyzed in the study. The resulting oligonucleotide:genome pairings were counted to determine the number of sequences predicted to be captured and, if captured, predicted to be detected by the DASH SARS-CoV-2 & Flu A/B Test. Based on in silico

inclusivity analysis of the Flu A capture oligonucleotides and PCR primers and probes, it is predicted that the DASH SARS-CoV-2 & Flu A/B Test will detect $9 7 . 3 \%$ (36/37) of all human host Flu A sequences collected between November 5, 2023 and November 5 2024. The performance characteristics of DASH SARS-CoV-2 & Flu A/B Test with these Flu A subtypes have not been confirmed by wet testing analysis.

# 8. Assay Cut-Off:

The DASH Rapid PCR Instrument has four fluorescence detectors also known as channels and an algorithm that is utilized for detection of the targets. The DASH SARS-CoV-2 & Flu $\mathrm { A } / \mathrm { B }$ Test does not have a simple $\mathrm { C t }$ or Cq threshold below which the target is ‘Detected’, and above which ‘Not Detected’. The $\mathrm { C q }$ threshold for the test is 40, same as the total number of PCR cycles. If $\mathrm { C q }$ is ${ > } 4 0$ , then the target is always ‘Not Detected’.

# 9. Carry-Over:

An analytical study was performed to assess potential carryover or cross-contamination in the DASH SARS-CoV-2 & Flu A/B Test by testing high positive and negative samples in an alternating fashion on the DASH Instrument. The high positive samples consisted of a single viral target (either SARS-CoV-2, Flu A or Flu B), prepared by spiking the viral target at ${ \geq } 1 \mathbf { x } 1 0 ^ { 5 }$ copies $/ \mathrm { m L }$ or ${ \geq } 1 \mathbf { x } 1 0 ^ { 5 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ in simulated clinical nasal matrix. The study utilized eight viral strains representing the test targets (two strains of SARS-CoV-2, and three strains each of Flu A and Flu B).

Samples were tested across five DASH Instruments, where one positive sample was followed by one negative sample. The process was repeated eight times to cover two strains of SARSCoV-2, and three strains each of Flu A and Flu B. In total, 40 high positive and 40 negative samples were evaluated. All high positive samples yielded positive results for the viral targets in the sample, while all negative samples were negative. These results confirm that there is no evidence of carryover from samples tested with the DASH SARS-CoV-2 & Flu A/B Test on the DASH Rapid PCR Instrument.

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Refer to the ‘Clinical Studies’ section of this document.

# C Clinical Studies:

# 1. Clinical Performance:

The clinical performance of the DASH SARS-CoV-2 & Flu A/B Test on DASH Instrument was established in a multi-center study conducted with anterior nasal swab (ANS) specimens collected from individuals with signs and symptoms of respiratory infection. Specimens were prospectively collected (i.e., all comers between two time points that met the clinical study inclusion criteria) from January and March 2024. Seven geographically diverse CLIAwaived testing sites within the United States participated in the study. Informed consent was obtained for all patients prior to testing. AN swabs for comparator testing were collected by the healthcare provider from both sides of the nose and placed into transport media for comparator testing. For testing with the DASH SARS-CoV-2 & Flu A/B Test, AN swabs from both sides of the nose were either healthcare provider-collected or adult-collected from individuals below 14 years, or self-collected from individuals aged 14 years or older. The order of swab collection was alternated between the comparator swab and the DASH SARSCoV-2 & Flu A/B Test. Performance of the DASH SARS-CoV-2 & Flu A/B Test was assessed by a comparison of results with an FDA cleared RT-PCR test for SARS-CoV-2, and a second FDA cleared test for Flu A and Flu B (expressed as positive percent agreement (PPA) and negative percent agreement (NPA)). All discordant results between the DASH SARS-CoV-2 & Flu A/B Test and the comparator were investigated using a third highly sensitive FDA cleared test and are footnoted under the Performance table below.

A total of 819 subjects were enrolled in the study from the seven clinical sites. Of these, two subjects were later found ineligible for enrollment. Of the remaining 817 enrolled subjects, 22 were excluded due to protocol deviations (e.g., specimen shipping issues, comparator sample being unable to be tested within the stability window of the test IFU, etc.), leaving 795 evaluable specimens to be included in the calculations of estimates of assay performance. Of those, additional three specimens were deemed unevaluable for Flu A and B due to unavailability of comparator test results. In total, 795 specimens were included in the calculations of performance for SARS-CoV-2, and 792 specimens for calculations of performance for Flu A and Flu B when tested on the DASH SARS-CoV-2 & Flu A/B Test.

Of the 833 samples tested on DASH SARS-CoV-2 & Flu A/B Test, including 16 repeat samples, there were 24 instances of invalid results $( 2 . 9 \%$ ; $9 5 \%$ CI: $1 . 9 \% - 4 . 3 \% )$ ). The clinical performance of the DASH SARS-CoV-2 & Flu A/B Test is summarized in the table below.

Table 14: Clinical Performance of DASH SARS-CoV-2 & Flu A/B Test   

<table><tr><td rowspan=2 colspan=1>Test Targets</td><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=2>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>160/168[1]</td><td rowspan=1 colspan=1>95.2%(90.9% - 97.6%)</td><td rowspan=1 colspan=1>624/627</td><td rowspan=1 colspan=1>99.5%(98.6% - 99.8%)</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>50/53[2]</td><td rowspan=1 colspan=1>94.3%(84.6% - 98.1%)</td><td rowspan=1 colspan=1>725/739[3]</td><td rowspan=1 colspan=1>98.1%(96.8% - 98.9%)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>36/37[4]</td><td rowspan=1 colspan=1>97.3%(86.2% - 99.9%)</td><td rowspan=1 colspan=1>749/755[5]</td><td rowspan=1 colspan=1>99.2%(98.3% - 99.6%)</td></tr></table>

TP-true positive; FN-false negative; TN-true negative; FP-false positive [1]SARS-CoV-2 (PPA): One (1) sample was negative by another FDA cleared test, agreeing with the DASH. [2]Flu A PPA: Two (2) samples were negative by another FDA cleared test, agreeing with the DASH. [3]Flu A NPA: 13 samples were positive by another FDA cleared test, agreeing with the DASH. [4]Flu B PPA: One (1) sample was negative by another FDA cleared test, agreeing with the DASH. [5]Flu B NPA: Four (4) samples were positive by another FDA cleared test, agreeing with the DASH.

2. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values:

The positivity for SARS-CoV-2, Flu A and Flu B, as determined by DASH SARS-CoV-2 & Flu A/B Test, are shown below, stratified by the study site and by patient age group.

Table 15: Overall Positivity Rates Observed During the Clinical Study   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Overall</td><td rowspan=1 colspan=2>Site 1(N=89)</td><td rowspan=1 colspan=2>Site 2(N=178)</td><td rowspan=1 colspan=2>Site 3(N=152)</td><td rowspan=1 colspan=2>Site 4(N=59)</td><td rowspan=1 colspan=2>Site 5(N=124)</td><td rowspan=1 colspan=2>Site 6(N=93)</td><td rowspan=1 colspan=2>Site 7(N=97)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>795</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>20.1%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11.2%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>10.7%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>16.4%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>10.2%</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>23.4%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>26.9%</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>47.4%</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9.3%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10.1%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>7.9%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0%</td></tr></table>

$\mathrm { N } =$ Total evaluable specimens; $\# =$ Number of Positives; $\% = \%$ positivity rate

Table 16: Positivity Rates by Age Group   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=2 colspan=3>Overall</td><td rowspan=1 colspan=8>Subject Age</td></tr><tr><td rowspan=1 colspan=2>2-&lt;14 years</td><td rowspan=1 colspan=2>14-24 years</td><td rowspan=1 colspan=2>&gt;24-64 years</td><td rowspan=1 colspan=2>≥65 years</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>795</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>20.1%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>11.8%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>17.5%</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>60.6%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>10.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>16.0%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>58.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>41.70%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>44.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

$\mathrm { N } =$ Total evaluable specimens; $\# = \mathrm { N u m b e r }$ of Positives; $\% = \%$ positivity rate

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.