# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K241573   
B Applicant Abbott Molecular Inc   
C Proprietary and Established Names Alinity m Resp-4-Plex   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to show that the Alinity m Resp-4-Plex for use on the Alinity m System is substantially equivalent to the Hologic Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay (K222736) and to obtain clearance for the Alinity m Resp-4-Plex Assay.

# B Measurand:

The Alinity m Resp-4-Plex detects Severe Acute Respiratory Syndrome Coronavirus-2 (SARSCoV-2), influenza A, influenza B, and respiratory syncytial virus (RSV) RNA.

# C Type of Test:

The Alinity m Resp-4-Plex is a multiplexed in vitro reverse transcription polymerase chain reaction (RT-PCR) assay that amplifies and detects genomic RNA sequences of SARS-CoV-2, influenza A, influenza B, and RSV through sample preparation, PCR assembly, amplification, and detection using real-time PCR. All steps of the assay are automated and performed on the Alinity m System.

III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

Alinity m Resp-4-Plex is a multiplexed real-time in vitro reverse transcription polymerase chain reaction (RT-PCR) assay for use with the automated Alinity m System for the qualitative detection and differentiation of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV2), influenza A virus, influenza B virus and Respiratory Syncytial Virus (RSV) in nasopharyngeal swab specimens collected from patients with signs and symptoms of respiratory tract infection. Clinical signs and symptoms of respiratory tract infection due to SARS-CoV-2, influenza A, influenza B, and RSV can be similar.

The Alinity m Resp-4-Plex assay is intended for use in the differential detection of SARS-CoV2, influenza A, influenza B and/or RSV RNA and aids in the diagnosis of COVID-19, influenza and/or RSV infections if used in conjunction with other clinical and epidemiological information, and laboratory findings. SARS-CoV-2, influenza A, influenza B and RSV viral RNA are generally detectable in nasopharyngeal swab specimens during the acute phase of infection. This test is not intended to detect influenza C virus infections.

Positive results are indication of the presence of the identified virus, but do not rule out bacterial infection or co-infection with other pathogens not detected by the test. The agent(s) detected by the Alinity m Resp-4-Plex assay may not be the definite cause of disease.

Negative results do not preclude SARS-CoV-2, influenza A, influenza B and/or RSV infections and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only For in vitro diagnostic use   
D Special Instrument Requirements: For use with the Alinity m System only.

IV Device/System Characteristics:

A Device Description:

The Alinity m Resp-4-Plex assay is a multiplex in vitro reverse transcription polymerase chain reaction (RT-PCR) assay for use with the automated Alinity m System for the qualitative detection and differentiation of SARS-CoV-2, influenza A, influenza B, and RSV in nasopharyngeal (NP) swab specimens from patients with signs and symptoms of respiratory tract infection.

# B Principle of Operation:

The Alinity m Resp-4-Plex assay requires two kits as follows:

Alinity m Resp-4-Plex AMP Kit; 09N79-095, consists of 2 types of multi-well assay trays. The amplification trays (AMP TRAY 1) contain unit-dose liquid PCR amplification/detection reagents and unit-dose liquid Internal Control (IC) in separate wells. The activation trays (ACT TRAY 2) contain liquid unit-dose activation reagent. The intended storage condition for the Alinity m Resp-4-Plex AMP Kit is $- 2 5 ^ { \circ } \mathrm { C }$ to $- 1 5 ^ { \circ } \mathrm { C }$ .

Alinity m Resp-4-Plex CTRL Kit; 09N79-085, consists of negative controls and positive controls, each supplied as liquid in single-use tubes. The intended storage condition for the Alinity m Resp-4-Plex CTRL Kit is $- 2 5 ^ { \circ } \mathrm { C }$ to $- 1 5 ^ { \circ } \mathrm { C }$ .

The Alinity m Resp-4-Plex assay utilizes real-time PCR to amplify and detect genomic RNA sequences of SARS-CoV-2, influenza A, influenza B, and RSV from NP swab specimens. The Alinity m Resp-4-Plex detects assay analytes and internal control (IC) separately, using analyte specific probes. The assay targets two different genes within the SARS-CoV-2 genome: the RNA-dependent RNA polymerase (RdRp) and nucleocapsid (N) gene, the matrix gene of the influenza A genome, the nonstructural 1 gene of the influenza B genome, and the matrix gene of the RSV genome. The fluorescently labeled probes do not generate a detectable signal unless they are specifically bound to the amplified product. All probes are labeled with analyte specific fluorophores, thus allowing for simultaneous detection and differentiation of amplified products of all four viruses and IC in a single reaction vessel. The steps of the Alinity m Resp-4-Plex assay consist of sample preparation, PCR assembly, amplification/detection, and result calculation and reporting. All steps of the Alinity m Resp-4-Plex assay procedure are executed automatically by the Alinity m System. The Alinity m System is designed to be a continuous random-access analyzer that can perform the Alinity m Resp-4-Plex assay in parallel with other Alinity m assays on the same instrument.

The Alinity m System performs automated sample preparation using the Alinity m Sample Prep Kit 2, Alinity m Lysis Solution, and Alinity m Diluent Solution. The Alinity m System employs magnetic microparticle technology to facilitate nucleic acid capture, wash, and elution. The IC is introduced into each specimen at the beginning of the sample preparation process to demonstrate that the process was completed correctly for each specimen and control sample.

The resulting purified RNA is combined with liquid unit-dose Alinity m Resp-4-Plex activation reagent and liquid unit-dose Alinity m Resp-4-Plex amplification/detection reagent and transferred into a reaction vessel. Alinity m Vapor Barrier Solution is then added to the reaction vessel which is then transferred to an amplification/detection unit for reverse transcription, PCR amplification, and real-time fluorescence detection.

A positive control and a negative control are tested at or above an established minimum frequency to help ensure that instrument and reagent performance remain satisfactory. During each control event, a negative control and positive control are processed through sample preparation and PCR procedures that are identical to those used for specimens.

# C Instrument Description Information:

1. Instrument Name: Alinity m System with software version 1.5.4

2. Specimen Identification: Specimen identification information is entered either manually or via barcode.

3. Specimen Sampling and Handling:

The samples may be loaded on the system in any order. The system pipettor robot dispenses and aspirates liquids, as appropriate for each reaction. Sample handling and reagent transport is performed by a handler robot.

4. Calibration: Not applicable.

5. Quality Control:

The Alinity m Resp-4-Plex Control Kit consists of a negative control and a positive control and are used for validity determination of the Alinity m Resp-4-Plex assay on the automated Alinity m System. A Quality Control is defined as the set of assay controls that are, when valid, necessary to allow reporting of specimen results. Each assay control is processed through the same sample extraction and PCR procedure used for specimens.

V Substantial Equivalence Information:

A Predicate Device Name(s): Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay

B Predicate 510(k) Number(s): K222736

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K241573</td><td colspan="1" rowspan="1">K222736</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Alinity m Resp-4-Plex</td><td colspan="1" rowspan="1">Panther Fusion SARS-CoV-2/ FluA/B/RSV assay</td></tr><tr><td colspan="1" rowspan="1">Regulation Number andProduct Code</td><td colspan="1" rowspan="1">21 CFR 866.3981; QOF</td><td colspan="1" rowspan="1">21 CFR 866.3981; QOF</td></tr><tr><td colspan="3" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">Technology/Detection</td><td colspan="1" rowspan="1">Reverse transcriptasemultiplexed polymerase chainreaction test</td><td colspan="1" rowspan="1">Reverse transcriptase multiplexedpolymerase chain reaction test</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">Alinity m System</td><td colspan="1" rowspan="1">Panther System and PantherFusion System</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indications For Use</td><td colspan="1" rowspan="1">Alinity m Resp-4-Plex is amultiplexed real-time in vitroreverse transcriptionpolymerase chain reaction (RT-PCR) assay for use with theautomated Alinity m Systemfor the qualitative detection anddifferentiation of Severe AcuteRespiratory SyndromeCoronavirus 2 (SARS-CoV-2),influenza A virus, influenza Bvirus and Respiratory SyncytialVirus (RSV) in nasopharyngealswab specimens collected frompatients with signs andsymptoms of respiratory tractinfection. Clinical signs andsymptoms of respiratory tractinfection due to SARS-CoV-2,influenza A, influenza B, andRSV can be similar.The Alinity m Resp-4-Plexassay is intended for use in thedifferential detection of SARS-CoV-2, influenza A, influenzaB and/or RSV RNA and aids inthe diagnosis of COVID-19,influenza and/or RSVinfections if used inconjunction with other clinicaland epidemiologicalinformation, and laboratoryfindings. SARS-CoV-2,influenza A, influenza B andRSV viral RNA are generallydetectable in nasopharyngealswab specimens during theacute phase of infection. Thistest is not intended to detect</td><td colspan="1" rowspan="1">The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay is a fullyautomated multiplexed real-timepolymerase chain reaction (RT-PCR)in vitro diagnostic test intended forthe qualitative detection anddifferentiation of severe acuterespiratory syndrome coronavirus 2(SARS-CoV-2), influenza A virus(Flu A), influenza B virus (Flu B),and respiratory syncytial virus(RSV). Nucleic acids are isolatedand purified from nasopharyngeal(NP) specimens obtained fromindividuals exhibiting signs andsymptoms of a respiratory tractinfection. Clinical signs andsymptoms of respiratory viralinfection due to SARS-CoV-2,influenza, and RSV can be similar.This assay is intended to aid in thedifferential diagnosis of SARS-CoV-2, Flu A, Flu B, and RSV infectionsin humans and is not intended todetect influenza C virus infections.Nucleic acids from the viralorganisms identified by this testare generally detectable in NPspecimens during the acute phaseof infection. The detection andidentification of specific viralnucleic acids from individualsexhibiting signs and symptoms ofrespiratory tract infection areindicative of the presence of theidentified virus and aids indiagnosis if used in conjunctionwith other clinical andepidemiological information, andlaboratory findings. The results ofthis test should not be used as thesole basis for diagnosis,</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">influenza C virus infections.Positive results are indicationof the presence of the identifiedvirus, but do not rule outbacterial infection or co-infection with other pathogensnot detected by the test. Theagent(s) detected by the Alinitym Resp-4-Plex assay may notbe the definite cause of disease.Negative results do notpreclude SARS-CoV-2,influenza A, influenza B and/orRSV infections and should notbe used as the sole basis fordiagnosis, treatment or otherpatient management decisions.</td><td colspan="1" rowspan="1">treatment, or other patientmanagement decisions.Positive results do not rule outcoinfection with other organisms.The organism(s) detected by thePanther Fusion SARS-CoV-2/FluA/B/RSV assay may not be thedefinite cause of disease.Negative results do not precludeSARS-CoV-2, influenza A virus,influenza B virus, or RSV infections.This assay is designed for use on thePanther Fusion system.The Hologic RespDirect CollectionKit can be used to collect NPspecimens for testing with thePanther Fusion SARS-CoV-2/FluA/B/RSV Assay. Additionally, otherNP swabs (not provided with theHologic RespDirect Collection Kit)may be used to collect NP specimensin 3mL of VTM or UTM.Ancillary Collection Kit:RespDirect Collection KitThe Hologic RespDirect CollectionKit is intended to be used for thecollection of nasopharyngeal (NP)swab specimens (collected by ahealthcare provider) for testing withthe Panther Fusion SARS-CoV-2/FluA/B/RSV Assay.</td></tr><tr><td colspan="1" rowspan="1">Condition for use</td><td colspan="1" rowspan="1">For prescription use</td><td colspan="1" rowspan="1">For prescription use</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasopharyngeal swabspecimens</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimens</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">SARS-CoV-2, influenza A,influenza B, RSV</td><td colspan="1" rowspan="1">SARS-CoV-2, influenza A,influenza B, RSV</td></tr><tr><td colspan="1" rowspan="1">Influenza A Subtyping</td><td colspan="1" rowspan="1">No</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Internal and external controls</td><td colspan="1" rowspan="1">Internal and external controls</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen collection andtransport</td><td colspan="1" rowspan="1">BD UVT and Copan UTM</td><td colspan="1" rowspan="1">BD UVT, Copan UTM, RemelMicroTest M4RT, M5, or M6,Hardy Diagnostics ViralTransport Media, and HologicRespDirect Collection Kit</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981.

VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

# a. Precision

The Alinity m Resp-4-Plex assay within-laboratory precision was evaluated by testing five panel members: one influenza A/influenza B/RSV/SARS-CoV-2 negative panel member and four positive panel members. Un-spiked negative simulated nasal matrix (NSM) was used as the influenza A/influenza B/RSV/SARS-CoV-2 negative panel member. Positive panel members were prepared by spiking cultured viral stocks of influenza A (H3N2/Switzerland/9715293/13), influenza B (Brisbane/33/08), RSV (B/ West Virginia/14617/85) and gamma irradiated SARS-CoV-2 (USA-WA1/2020) in negative NSM to achieve 2x and 5x the assays claimed LoD for each analyte. The composition of the panels is described in Table 1. Each panel member was tested with four replicates twice each day for five days, on three Alinity m Systems operated by three operators using three reagent lots (one reagent lot and one operator per system). The qualitative and Cycle Number (CN) results for each analyte at each target level are summarized in Table 2 and Table 3, respectively.

Table 1: Panel Composition   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A (2x) / RSV (2x)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A (5x)/ RSV (5x)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B (2x) / SARS-CoV-2 (2x)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza B (5x) / SARS-CoV-2 (5x)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative</td></tr></table>

Table 2: Precision Study Results   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>PanelNumber</td><td rowspan=1 colspan=1>Panel Description</td><td rowspan=1 colspan=1>n Agreement/N Valid Tested</td><td rowspan=1 colspan=1>% Agreement withExpected Results/ NValid Test (95% CI)</td></tr><tr><td rowspan=3 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>117/117a</td><td rowspan=1 colspan=1>100.0% (96.8%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100.0% (96.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>3, 4, and 5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>100.0%(98.9%-100.0%)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100.0%(96.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100.0%(96.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>1, 2, and 5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>357/357</td><td rowspan=1 colspan=1>100.0%(98.9%-100.0%)</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>117/117a</td><td rowspan=1 colspan=1>100.0%(96.8%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100.0%(96.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>3, 4, and 5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>100.0%(98.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2*</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100.0% (96.9%-100.0%)</td></tr></table>

<table><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100.0% (96.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>1, 2, and 5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>357/357</td><td rowspan=1 colspan=1>100.0% (98.9%-100.0%)</td></tr></table>

\*Inactivated virus a One replicate of Panel 1, Instrument 2 was invalid and excluded from analysis. The sample was not retested as the minimum sample size was achieved. One replicate of Panel 1, Instrument 1 was a no test. The sample was not retested as the minimum sample size was achieved. One replicate of Panel 1, Instrument 3 was a no test. The sample was not retested as the minimum sample size was achieved.

All low (2x) and moderate $( 5 \mathbf { x } )$ panel members were $100 \%$ positive for the spiked target analytes. The negative panel members were $0 . 0 \%$ positive for all analytes (Table 2, above).

Table 3: Precision Study - CN Analysis Results   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Conc.</td><td rowspan=2 colspan=1>nb/Na</td><td rowspan=2 colspan=1>MeanCN</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>Within-Laboratory</td><td rowspan=1 colspan=2>BetweenInstruments</td><td rowspan=1 colspan=2>Totale</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=3 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>31.70</td><td rowspan=1 colspan=1>0.501</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.109</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.115</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.526</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.223</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.571</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>33.04</td><td rowspan=1 colspan=1>0.512</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.512</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.231</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.562</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>Negativef</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>28.93</td><td rowspan=1 colspan=1>0.179</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.058</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.188</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.254</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.316</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>30.22</td><td rowspan=1 colspan=1>0.163</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.058</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.180</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.256</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.313</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Negativef</td><td rowspan=1 colspan=1>357/357</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>29.91</td><td rowspan=1 colspan=1>0.554</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.205</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.086</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.597</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.136</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.612</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>31.08</td><td rowspan=1 colspan=1>0.622</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.253</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.671</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.196</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.699</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>Negativef</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2g</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>32.80</td><td rowspan=1 colspan=1>0.291</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.105</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.309</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.216</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.377</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>34.23</td><td rowspan=1 colspan=1>0.392</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.114</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.409</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.275</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.493</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>Negativef</td><td rowspan=1 colspan=1>357/357</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

\* Conc. $=$ Concentration; $\mathrm { C N } { = }$ Cycle Number; $\mathrm { S D = }$ Standard Deviation; $\mathrm { C V } =$ coefficient of variation a $\Nu =$ Total number valid replicates. b $\mathbf { n } =$ Number of replicates with positive result interpretation for positive panel members and number of replicates with negative result interpretation for negative panel member; the number of replicates used in the Mean Cycle Number (CN) and SD calculations for the positive panel members. c Within-laboratory includes Within-Run, Between-Run, and Between-Day Components. d Alinity m System, Alinity m Resp-4-Plex AMP Kit lot, and Operator are confounded, and the confounding effect is represented by Instrument. e Total includes Within-Run, Between-Run, Between-Day, and Between-Instrument Components. f The negative results included 3 panel members negative for the respective analyte. g Inactivated virus

b. Reproducibility The reproducibility of the Alinity m Resp-4-Plex assay was evaluated at three external clinical testing sites. A total of five panels were prepared consisting of one influenza A/influenza B/RSV/SARS-CoV-2 negative panel and four positive panel members. The panel composition is described in Table 4. All panel members were prepared in simulated nasal matrix utilizing cultured viral stocks of influenza A (H3N2/Switzerland/9715291/13), influenza B (Brisbane/33/08), RSV (B/West Virginia/14617/85), and gamma irradiated SARS-CoV-2 (USA-WA1/2020). A total of three Alinity m Resp-4-Plex AMP kit lots were used. Each of the three external sites tested two Alinity m Resp-4-Plex AMP kit lots on five non-consecutive days for each lot. Four replicates of each panel member were tested on each of five days. The qualitative results and Cycle Number (CN) analysis for the reproducibility study are shown in Table 5 and Table 6, respectively.

Table 4: Panel Composition   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A (2x) / RSV (2x)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Influenza A (5x) / RSV (5x)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza B (2x) / SARS-CoV-2 (2x)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Influenza B (5x) / SARS-CoV-2 (5x)</td></tr></table>

Table 5: Reproducibility Study Results   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>PanelNumber</td><td rowspan=2 colspan=1>PanelDescription</td><td rowspan=1 colspan=4>% Agreement with Expected Results (n Agreement/N Valid Tested)(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2*</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>95.0% 38/40(83.5-98.6)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>97.5% (39/40)(87.1-99.6)</td><td rowspan=1 colspan=1>97.5% (117/120)(92.9-99.1)</td></tr><tr><td rowspan=1 colspan=1>1, 2, 3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (120/120)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(91.2-100.0)</td><td rowspan=1 colspan=1>97.5% (119/120)(87.1-99.6)</td><td rowspan=1 colspan=1>99.7% (359/360)(98.4-100.0)</td></tr><tr><td rowspan=3 colspan=1>Influenza A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>100% (39/39)(91.0-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (119/119)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>1, 4,5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (120/120(96.9-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (360/360)(98.9-100.0)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (119/119)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (359/359)(98.9-100.0)</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>100% (39/39)(91.0-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (40/40)(91.2-100.0)</td><td rowspan=1 colspan=1>100% (119/119)(96.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>1,4, 5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (120/120)(96.9-100.0)</td><td rowspan=1 colspan=1>100% (360/360)(98.9-100.0)</td></tr></table>

\*Inactivated virus

The Alinity m Resp-4-Plex assay demonstrated $100 \%$ agreement for SARS-CoV-2, influenza A, influenza B, and RSV moderate positive $( 5 \mathbf { x } )$ panel members. For low positive (2x) panel members, the assay yielded $100 \%$ for influenza A, influenza B, and RSV and a $9 7 . 5 \%$ agreement for SARS-CoV-2. For negative panel members, the assay yielded $100 \%$ agreement for influenza A, influenza B, and RSV and $9 9 . 2 \%$ agreement for SARS-CoV-2.

Table 6: Reproducibility Study – CN Analysis Results   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Conc.</td><td rowspan=2 colspan=1>nbNa</td><td rowspan=2 colspan=1>MeanCN</td><td rowspan=1 colspan=2>Within-Run/Day</td><td rowspan=1 colspan=2>BetweenRun/Day</td><td rowspan=1 colspan=2>Within-Laboratory</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Totald</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=3 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>30.92</td><td rowspan=1 colspan=1>0.568</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.149</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.587</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.207</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.623</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>119/119</td><td rowspan=1 colspan=1>32.18</td><td rowspan=1 colspan=1>0.592</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.208</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.628</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.065</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.631</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>28.88</td><td rowspan=1 colspan=1>0.220</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.119</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.250</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.081</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.081</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.275</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>30.08</td><td rowspan=1 colspan=1>0.590</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.123</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.603</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.188</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.631</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>359/359</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>29.09</td><td rowspan=1 colspan=1>0.736</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.314</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.800</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.800</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>119/119</td><td rowspan=1 colspan=1>30.35</td><td rowspan=1 colspan=1>0.665</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.665</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.120</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.155</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.693</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>SARS-ovV-2f</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>32.56</td><td rowspan=1 colspan=1>0.325</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.091</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.337</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.056</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.029</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.343</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>117/120</td><td rowspan=1 colspan=1>34.00</td><td rowspan=1 colspan=1>0.776</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.278</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.824</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.175</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.843</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>Negativee</td><td rowspan=1 colspan=1>359/360</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

Conc. $=$ Concentration; $\mathrm { C N } =$ Cycle Number; SD $=$ Standard Deviation; $\mathrm { C V } =$ coefficient of variation a $\mathrm { N } =$ Total number valid replicates. b Number of replicates with positive result interpretation for positive panel members and number of replicates with negative result interpretation for negative panel member; the number of replicates used in the Mean Cycle Number (CN) and SD calculations for the positive panel members. c Within-laboratory includes Within-Run/Day and Between-Run/Day Components. d Total includes Within-Run/Day, Between-Run/Day, Between-Lot, and Between-Site/Instrument Variance Components. e The negative results included 3 panel members negative for the respective analyte. f Inactivated virus

# 2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

Analytical Reactivity (Inclusivity)

a. Wet-Testing

This study was performed to determine the analytical reactivity of the Alinity m Resp-4- Plex assay with clinically relevant strains, serotypes, or subtypes of the target species (i.e., SARS-CoV-2, influenza A, influenza B, and RSV). Inclusivity panels were created by spiking virus strains at concentrations of $3 \mathbf { x }$ LoD if units of concentration matched that of the LoD strains, or in a dilution series with a minimum of three concentration levels if units of concentration did not match that of the LoD strains. The dilution series consisted of at least one concentration that gave positive results $100 \%$ of the time and at least one concentration that gave positive results ${ < } 1 0 0 \%$ of the time. Strains that did not yield $100 \%$ reactivity at $3 \mathbf { x }$ LoD or any dilutions of the dilution series were prepared at higher concentrations and tested until the minimum concentration that achieved $100 \%$ reactivity was reached. Each panel was tested in replicates of five. All strains with the same reported unit of concentration as reported for the assays LoD study yielded $100 \%$

detection rate at 3x LoD. For those strains with a different unit of concentration as reported for the assays LoD study, the lowest concentration tested that achieved $100 \%$ is presented. As presented in Table 7, the results from this study demonstrate that the Alinity m Resp-4-Plex assay is capable of detecting multiple clinically relevant strains of SARS-CoV-2, influenza A, influenza B, and RSV.

Table 7: Inclusivity Wet Testing Results   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Strain/Isolate Name</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">RelativeLoD</td><td colspan="1" rowspan="1">n Positive/N Valid</td><td colspan="1" rowspan="1">DetectionRate</td></tr><tr><td colspan="1" rowspan="18">Influenza A</td><td colspan="1" rowspan="1">Wisconsin/588/2019 (H1N1)</td><td colspan="1" rowspan="1">1.37E-06 HA</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Delaware/01/2021 (H3N2)</td><td colspan="1" rowspan="1">3.64E-01 FFU/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Perth/16/2009 (H3N2)</td><td colspan="1" rowspan="1">0.045 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Hong Kong/45/2019 (H3N2)Candidate Vaccine Virus</td><td colspan="1" rowspan="1">2.56E-06 GP</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">California/07/2009 (H1N1)</td><td colspan="1" rowspan="1">3.33E+00CEID50/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Fort Monmouth/1/1947(H1N1)</td><td colspan="1" rowspan="1">1.67E+00CEID50/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">New Jersey/8/1976 (H1N1)</td><td colspan="1" rowspan="1">1.00E+01CEID50/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Tasmania/503/2020 (H3N2)Candidate Vaccine Virus</td><td colspan="1" rowspan="1">2.37E-07 GP</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Puerto Rico/8/1934 (H1N1)</td><td colspan="1" rowspan="1">3.33E+00CEID50/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Vietnam/1203/2004 (H5N1)</td><td colspan="1" rowspan="1">9.00E-05 μg/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Equine/Prague/1/1956 (HA) xA/Aichi/2/1968 (NA) xA/Puerto Rico/8/1934,Reassortant X-33 (H7N2)2</td><td colspan="1" rowspan="1">2.08E-05 pg/μL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Hong Kong/2671/2019(H3N2) - Candidate VaccineVirus</td><td colspan="1" rowspan="1">4.27E-05 GP</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Aichi/2/1968 (H3N2)2</td><td colspan="1" rowspan="1">1.80E-03 pg/μL</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Victoria/2570/2019 (H1N1)Candidate Vaccine Virus</td><td colspan="1" rowspan="1">5.12E-05 HA</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Guangdong-Maonan/1536/2019 (H1N1)Candidate Vaccine Virus</td><td colspan="1" rowspan="1">8.53E-06 HA</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Brisbane/02/2018 (H1N1)</td><td colspan="1" rowspan="1">0.006 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Singapore/INFIMH-16-0019/2016 (H3N2)</td><td colspan="1" rowspan="1">0.045 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Brisbane/10/2007 (H3N2)</td><td colspan="1" rowspan="1">0.045 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="5">Influenza B</td><td colspan="1" rowspan="1">Brisbane/60/2008 (Victorialineage)</td><td colspan="1" rowspan="1">0.06 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Malaysia/2506/2004 (Victorialineage)</td><td colspan="1" rowspan="1">0.006 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Lee/1940</td><td colspan="1" rowspan="1">0.06 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Allen/1945</td><td colspan="1" rowspan="1">2.78E-02CEID50/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Great Lakes/1739/1954(Yamagata lineage)</td><td colspan="1" rowspan="1">1.67E+00CEID50/mL</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="4">b</td><td colspan="1" rowspan="1">Washington/02/2019(Victoria lineage) -Candidate Vaccine Virus</td><td colspan="1" rowspan="1">1.42E-06 HA</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Phuket/3073/2013 (Yamagatalineage)  Candidate VaccineVirus</td><td colspan="1" rowspan="1">2.96E-07 HA</td><td colspan="1" rowspan="1">Unknown1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Colorado/6/2017 (Victorialineage)</td><td colspan="1" rowspan="1">0.02 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Alabama/2/2017 (Victorialineage)</td><td colspan="1" rowspan="1">0.02 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="6">RSV</td><td colspan="1" rowspan="1">3/2015 Isolate 1 (RSV B)</td><td colspan="1" rowspan="1">0.1 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">12/2014 Isolate 1 (RSV B)</td><td colspan="1" rowspan="1">0.03 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1/2015 Isolate 1 (RSV A)</td><td colspan="1" rowspan="1">0.09 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">12/2014 Isolate 12 (RSV A)</td><td colspan="1" rowspan="1">0.9 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">2014 Isolate 341 (RSV A)</td><td colspan="1" rowspan="1">0.09 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">11/2014 Isolate 2 (RSV B)</td><td colspan="1" rowspan="1">0.03 TCID50/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="9">SARS-CoV-2</td><td colspan="1" rowspan="1">USA-IL1/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Germany/BavPat1/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">USA-AZ1/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">USA-CA3/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Italy-INMI12</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">HongKong/VM20001061/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">USA/CA CDC 5574/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">USA/GA-EHC-2811C/20212</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">USA-WI1/20202</td><td colspan="1" rowspan="1">90 GE/mL</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

$\mathrm { C E I D 5 0 / m L }$ - median chicken embryo infectious dose $\mathrm { \Delta } / \mathrm { m L }$ ; HA - Hemagglutination titer, indicates Hemagglutinin levels; FFU - focus-forming units $/ \mathrm { m L }$ ; $\mathrm { G C / m L }$ - Genome Copies $\mathrm { \Delta } / \mathrm { m L }$ ; $\mathrm { G E / m L }$ -Genome Equivalent/mL; GP - Glycoprotein titer, indicates Glycoprotein levels. 1 Strains with units of concentration that were not as reported for the assays LoD were evaluated using at least three concentrations. The lowest concentration with a detection rate of $1 0 0 \%$ for each strain was reported. 2 Genomic RNA

# b. In silico

Inclusivity of the Alinity m Resp-4-Plex assay was also evaluated using in silico analysis of SARS-CoV-2 primers and probes for homology with sequences available in GISAID and NCBI databases.

For SARS-CoV-2, all available sequences up to October 3, 2023, from GISAID and up to October 9, 2023 from NCBI databases were evaluated. Inclusivity was demonstrated by analyzing the sequences of the RdRp and N primer/probe sets for homology with 14,818,776 unambiguous SARS-CoV-2 sequences available in the GISAID database. In addition to the overall GISAID database, an analysis was also performed for the sequences that have variant of concern (VOC) and variant of interest (VOI) variant designation. Of the 14,818,776 total available sequences, 13,680,493 have current or former VOC, VOI, and variant under monitoring (VUM) designation including Omicron EG.5, Omicron B.1.640, Omicron XBB.1.16, Omicron XBB.1.5, Omicron BA.2.75, Omicron BA.2.86, Omicron CH.1.1, Omicron XBB.1.9.1, Omicron XBB.1.9.2, Omicron XBB.2.3, Omicron XBB, Alpha, Beta, Delta, Gamma, Epsilon, Eta, Iota, Kappa, Lambda, Mu, Theta, and Zeta, among others. In silico analyses were also performed with the full-length SARS-CoV-2 sequences available in the NCBI database of October 9, 2023. A total of 7,590,332 unambiguous sequences in the NCBI database were analyzed for SARS-CoV-2 inclusivity. Analysis showed that greater than or equal to $9 9 . 9 9 \%$ of the SARS-CoV-2 sequences evaluated are predicted to be detected by the Alinity m Resp-4- Plex assay.

Cross-Reactivity/Microbial Interference

# Wet-Testing

The impact of potentially cross-reacting microorganisms on the analytical specificity of the Alinity m Resp-4-Plex assay was assessed by adding microorganisms listed in Table 8 into influenza A, influenza B, RSV, and SARS-CoV-2 negative samples to achieve a final titer of $1 0 ^ { 5 }$ units/mL (or higher) for viruses and fungi, $1 0 ^ { 6 }$ units/mL (or higher) for bacteria, $1 0 \% \mathrm { v / v }$ (or higher) of pooled human nasal wash, or the highest concentration available. The negative sample was prepared by pooling negative clinical nasopharyngeal swab specimens prescreened to be negative for all four target analytes. The negative sample was also tested without addition of any potential cross reactants (control condition).

Microbial interference was assessed by preparing cultured viral stocks of influenza A, influenza B, RSV, and gamma irradiated SARS-CoV-2 positive samples in pooled negative nasopharyngeal swab specimens at approximately 3x LoD for each analyte. The positive sample was also tested without addition of any potential cross reactants (control condition). A minimum of four replicates of each test condition and the control condition were tested with the Alinity m Resp-4-Plex assay on the Alinity m System. No cross-reactivity or microbial interference was observed at the concentrations tested.

Table 8: Organisms Evaluated in Cross-Reactivity and Microbial Interference Study by WetTesting   

<table><tr><td colspan="2" rowspan="1">Bacteria</td><td colspan="2" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Adenovirus Type 01g</td><td colspan="1" rowspan="1">Human Metapneumovirus 9Type 1Ag</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Adenovirus Type 02g</td><td colspan="1" rowspan="1">Human Metapneumovirus18 Type 2Ba, g</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila psittaci</td><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">Adenovirus Type 03g</td><td colspan="1" rowspan="1">Influenza A (H1N1)c, g, h</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheria</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Adenovirus Type 04g</td><td colspan="1" rowspan="1">Influenza A (H3N2)c, g</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetti</td><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">Adenovirus Type 05g</td><td colspan="1" rowspan="1">Influenza Bd, g</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Adenovirus Type 07Ag</td><td colspan="1" rowspan="1">Influenza C</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Adenovirus Type 11g</td><td colspan="1" rowspan="1">Measlesg</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Adenovirus Type 14a, g</td><td colspan="1" rowspan="1">MERS-Coronavirus</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Adenovirus Type 31g</td><td colspan="1" rowspan="1">Mumps</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Staphylococcus epidermis</td><td colspan="1" rowspan="1">Adenovirus Type 34g</td><td colspan="1" rowspan="1">Parainfluenza Virus Type 18</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">Bocavirusj</td><td colspan="1" rowspan="1">Parainfluenza Virus Type 2g</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Coxsackievirusg</td><td colspan="1" rowspan="1">Parainfluenza Virus Type 3g</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">Cytomegalovirusg</td><td colspan="1" rowspan="1">Parainfluenza Virus Type 4g</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Echovirusg</td><td colspan="1" rowspan="1">Parechovirus Type 3g</td></tr><tr><td colspan="2" rowspan="1">Fungi</td><td colspan="1" rowspan="1">Enterovirus Type 68g</td><td colspan="1" rowspan="1">Rhinovirus 1Ag, h</td></tr><tr><td colspan="2" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">Enterovirus Eg</td><td colspan="1" rowspan="1">Rhinovirus B42g</td></tr><tr><td colspan="1" rowspan="1">Candida albicansf</td><td colspan="1" rowspan="1">Epstein-Barr Virusg</td><td colspan="1" rowspan="1">RSV Ae, g</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii (PJP)b</td><td colspan="1" rowspan="1">Herpes Simplex VirusType 1g</td><td colspan="1" rowspan="1">RSV Be, g, h</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Human Coronavirus229Eg</td><td colspan="1" rowspan="1">SARS-Coronavirusg</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">Human CoronavirusHKU1</td><td colspan="1" rowspan="1">Varicella-Zoster Virusg</td></tr><tr><td colspan="1" rowspan="1">Pooled Human Nasal Wash (10%)</td><td colspan="1" rowspan="1">Human CoronavirusNL63g</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Human CoronavirusOC43g</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Human Metapneumovirus5 Type B1g</td><td colspan="1" rowspan="1"></td></tr></table>

cps/mL - copies/mL, $\mathrm { T C I D } 5 0 / \mathrm { m L }$ - Median Tissue Culture Infectious Dose $/ \mathrm { m L }$ , $\mathrm { G E / m L }$ - Genome Equivalent $\mathrm { \Delta m L }$ , $\mathrm { G C / m L }$ - Genome Copies $/ \mathrm { m L }$ , $\mathrm { I F U / m L }$ - infectious units $/ \mathrm { m L }$ , $\mathrm { C F U / m L }$ - culture forming units $/ \mathrm { m L }$ , $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ - Median Chicken Embryo Infectious Dose/mL, $\mathrm { G C / m L }$ - genome copies/mL a Tested at $1 . 0 0 \mathrm { E } { + } 0 4$ units/mL. b Pneumocystis jirovecii was not measured in terms of copy number. Instead, it was measured in cycle threshold (Ct). This specimen was tested neat. c Only the cross-reactivity of the non-influenza A signals were evaluated. d Only the cross-reactivity of the non-influenza B signals were evaluated. e Only the cross-reactivity of the non-RSV signals were evaluated. f Tested at $1 . 0 0 \mathrm { E } { + } 0 6 $ units $/ \mathrm { m L }$ g Viral lysate h Viral particles i Viral RNA j Viral DNA

# Interfering Substances

The study assessed the impact of potentially interfering endogenous and exogenous substances commonly encountered in the nasopharyngeal swab specimens on the detection of the Alinity m Resp-4-Plex assay. Each of the potentially interfering substances were evaluated in triplicate in two different positive panel members (PM) each containing multiple analytes at $3 \mathbf { x }$ LoD in pooled negative clinical NP swab matrix. A minimum of three replicates, for each of the two panel members, without any potentially interfering substances were also tested as the control condition. The list of potentially interfering substances is presented in Table 9 below. The panel member (PM) composition is presented below:

PM 1: influenza A (A/Switzerland/9715293/13 (H3N2), influenza B   
(B/Brisbane/33/08), RSV B (West Virginia/14617/85), and SARS-CoV-2 (USA  
WA1/2020 gamma-irradiated)   
PM 2: influenza A (A/Brisbane/59/2007 (H1N1)), influenza B   
(B/Massachusetts/02/2012), and RSV A (Long/MD/56)

In addition, endogenous and exogenous substances were tested with the Alinity m Resp-4- Plex assay in the absence of target analytes. FluMist was not evaluated in the absence of target analytes due to the lack of availability of this substance at the time of testing. Four replicates of each of the potentially interfering substances were evaluated in pooled negative clinical NP swab matrix. A minimum of four replicates of pooled negative clinical NP swab matrix without any potentially interfering substances were also tested as the control condition.

All positive samples reported positive results for influenza A, influenza B, RSV, and SARSCoV-2 in the presence of the potentially interfering substances and all negative samples reported negative for influenza A, influenza B, RSV and SARS-CoV-2. No interference in the performance of Alinity m Resp-4-Plex assay was observed for the substances at the concentrations tested.

Table 9: List of Potentially Interfering Endogenous and Exogenous Substances and Concentrations Tested   

<table><tr><td colspan="1" rowspan="1">Potentially InterferingSubstance</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">ConcentrationTested</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenges, OralAnesthetic and Analgesic -Cepacol</td><td colspan="1" rowspan="1">Benzocaine, Menthol</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin-Porcine</td><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, Nasal Ointment -Mupirocin</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal Spray - PerrigoOxymetazoline HCl</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug - Relenza</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drug  Remdesivir</td><td colspan="1" rowspan="1">Remdesivir</td><td colspan="1" rowspan="1">27.0 μM</td></tr><tr><td colspan="1" rowspan="1">Antibacterial, Systemic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal Gel / HomeopathicAllergy Relief Medicine -Zicam</td><td colspan="1" rowspan="1">Galphimia glauca,Histaminum,hydrochloricum, Luffaoperculata, Sulfur</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">FluMista</td><td colspan="1" rowspan="1">Live intranasal influenzavirus</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid - FlonaseSensimist</td><td colspan="1" rowspan="1">Fluticasone Furoate</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid - QVAR</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Dexamethasone</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.2 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Flunisolide</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Triamcinolone</td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Budesonide</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid -Mometasone</td><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">Fluoride</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Tobacco Pouch</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">0.1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Oral Rinse - Listerine CoolMint</td><td colspan="1" rowspan="1">Ethanol, essential oil</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">1.1E6 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal Decongestant -Phenylephrine</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Mist</td><td colspan="1" rowspan="1">Sodium chloride</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nicotine Pouch</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">0.1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Throat Spray - Chloraseptic</td><td colspan="1" rowspan="1">Phenol</td><td colspan="1" rowspan="1">5%(v/v)</td></tr><tr><td colspan="1" rowspan="1">Cough Syrup - Wal-Tussin</td><td colspan="1" rowspan="1">Dextromethorphan</td><td colspan="1" rowspan="1">5% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Mucin-Bovine</td><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Liposomal NUMB520 Spray</td><td colspan="1" rowspan="1">Lidocaine andPhenylephrine</td><td colspan="1" rowspan="1">2.68 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tamiflu</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Vicks VapoRub</td><td colspan="1" rowspan="1">Camphor-syntheticeucalyptus oil andmenthol ointment</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">Petroleum Jelly</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Cold Eeze (Throat lozenges)</td><td colspan="1" rowspan="1">Zincum gluconicum 2X</td><td colspan="1" rowspan="1">2.5% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">20 ng/uL</td></tr><tr><td colspan="1" rowspan="1">Saliva</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10% (w/v)</td></tr></table>

a Not evaluated in the absence of target analytes due to the lack of availability of this substance at the time of testing. FluMist is predicted to produce positive results for influenza A and influenza B due to the presence of attenuated viruses.

# Competitive Interference

The purpose of this study was to evaluate a potential competitive interference between target analytes for the Alinity m Resp-4-Plex assay. Four panels were prepared for this study, each consisting of three analytes at a low concentration $( \leq 3 \mathbf { x }$ the claimed LoD of the respective analyte) and the fourth analyte at a high concentration of $1 \mathrm { \Delta x 1 0 ^ { 5 } T C I D 5 0 / m L }$ $( > 1 0 0 0 \mathrm { x }$ LoD of the respective analyte). A minimum of 20 replicates of each of the four panels were tested. The viral strains used to prepare the panels are described in Table 10.1.

Table 10.1. Viral Strains and Concentrations Used in the Competitive Interference Study   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>A/Switzerland/9715293/2013 (H3N2)</td><td rowspan=1 colspan=1>0.0075 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>A/HongKong/8/1968 (H3N2)</td><td rowspan=1 colspan=1>105 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>B/Brisbane/33/2008 (Victoria lineage)</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>B/Taiwan/2/1962</td><td rowspan=1 colspan=1>105 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>RSV B/WestVirginia/14617/1985</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>RSV A/2014 Isolate 342</td><td rowspan=1 colspan=1>105 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>SARS-CoV-2, Isolate USA-WA1/2020*</td><td rowspan=1 colspan=1>0.015 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>SARS-CoV-2, Isolate Italy-INMI1(Heat Inactivated)</td><td rowspan=1 colspan=1>105 TCID50/mL</td></tr></table>

\*Based on the vendor’s Certificate of Analysis for SARS-CoV-2, Isolate USA-WA1/2020 (Part# NR-52287, Lot# 70033322), $1 \mathrm { T C I D } 5 0 / \mathrm { m L }$ is equal to 6,071 genome equivalents $\scriptstyle ( \mathrm { G E } ) / \mathrm { m L }$ .

As shown in Table 10.2, no competitive interference was observed for on-panel viruses for the Alinity m Resp-4-Plex assay in this study. None of the analytes present at a high concentration of $1 0 ^ { 5 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ interfered with the detection of any of the other three analytes prepared at $\leq 3 \mathbf { x }$ LoD.

Table 10.2: Competitive Interference Study Sample Panel Composition and Study Results b One Panel 3 sample had delayed Internal Control (IC). The analytes of this sample that tested “Negative” (influenza A, RSV, and SARS-CoV-2) resulted in Internal Control Failed error and were excluded from analysis, while the analyte (influenza B) that tested “Positive” was flagged but included in analysis. No additional testing was needed for Panel 3 as the minimum number of replicates required per the protocol has been met.

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>ConcentrationsTested</td><td rowspan=1 colspan=1>Detection Rate(n Agreement/N Valid Tested)(95% CI)</td></tr><tr><td rowspan=4 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.0075 TCID50/mL</td><td rowspan=1 colspan=1>100.0%6 (20a/20) (83.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.0075 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(21/21) (84.5%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.0075 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(20b/20) (83.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(20%/20) (83.9%-100.0%)</td></tr><tr><td rowspan=4 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.01TCID50/mL</td><td rowspan=1 colspan=1>100.0%(22a/22) (85.1%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.01TCID50/mL</td><td rowspan=1 colspan=1>100.0%(21/21) (84.5%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>105TCID50/mL</td><td rowspan=1 colspan=1>100.0%(21b/21) (84.5%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.01TCID50/mL</td><td rowspan=1 colspan=1>100.0%(20%/20)(83.9%-100.0%)</td></tr><tr><td rowspan=4 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(21a/21) (84.5%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(21/21) (84.5%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(20b/20) (83.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(20%/20) (83.9%-100.0%)</td></tr><tr><td rowspan=4 colspan=1>SARS-CoV-2d</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(23a/23)(85.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.015 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(21/21) (84.5%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.015 TCID50/mL</td><td rowspan=1 colspan=1>100.0%(20b/20))(83.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.015 TCID50/mL</td><td rowspan=1 colspan=1>100.0% (20%/20) (83.9%-100.0%)</td></tr></table>

a Three Panel 1 samples had delayed Internal Control (IC). The analytes of these samples that tested “Negative” resulted in Internal Control Failed error and were excluded from analysis, while the remaining analytes that tested “Positive” were flagged, but included in analysis. Two additional replicates of Panel 1 were tested to meet the minimum number of replicates. Results were valid (for all analytes) and included in the analysis.

c One Panel 4 sample was a “no test” due to instrument error and resulted in Background check on Amp-Detect unit 3 failed error. A maintenance and diagnostic procedure must be performed prior to system usage. The sample (all analytes) was excluded from analysis per the protocol. No additional testing was needed as the minimum number of replicates required per the protocol has been met.   
d Inactivated virus

# 4. Assay Reportable Range:

Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a. Controls See Section IV.C.5. Quality Control.

# b. Sample Stability

An analytical study was performed to establish the stability for nasopharyngeal swab (NPS) specimens collected in viral transport media (VTM). Positive samples were prepared by diluting cultured influenza A, influenza B, RSV, and inactivated SARSCoV-2 viral stocks at 3x LoD in pooled negative NPS specimens collected in VTM. The aliquots of the positive samples were tested immediately (control condition) and at later time points following extended storage at a variety of storage conditions. The test conditions, number of replicates performed for each test condition and positivity rate are presented in Table 11 below.

Table 11: Sample Stability Study Results Summary   

<table><tr><td rowspan=2 colspan=1>Storage Condition</td><td rowspan=1 colspan=4>Detection Rate (n Positive/ N Tested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Tested immediately</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 3 days at 2°C to8° plus a minimum of 4 hoursonboard storage</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 7 days at -70°C orcolder plus a minimum of 4 hoursonboard storagea</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 2 days at 23°followed by a minimum of 3days at 2°C to 8°C plus aminimum of 4 hours onboardstorage</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 7 days at 2°C to8° plus a minimum of 4 hoursonboard storage</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 3 days at 2°C to8°C followed by a minimum of 7days at -70°C or colder plus aminimum of 4 hours onboardstoragea</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 365 days at -70°or colder plus a minimum of 4hours onboard storagea</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>Tested immediately(Control condition)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 2 days at 15°C plusa minimum of 4 hours onboardstorage</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr><tr><td rowspan=1 colspan=1>A minimum of 2 days at 26°Cplus a minimum of 4 hoursonboard storage</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td><td rowspan=1 colspan=1>100.0% (5/5)</td></tr></table>

a All samples were thawed at $1 5 \mathrm { { } ^ { \circ } C }$ to $2 5 ^ { \circ } \mathrm { C }$ after being removed from the $- 7 0 \mathrm { { } ^ { \circ } C }$ storage.

The results of this specimen stability study support the stability claims of the Alinity m Resp-4-Plex assay of clinical NPS specimens in VTM at the following conditions:

Room temperature $( 1 5 . 2 5 ^ { \circ } \mathrm { C } )$ for up to 2 days. • Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 7 days. Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ for up to one year (365 days).

c. Freeze/Thaw Stability

A specimen freeze-thaw stability was conducted to evaluate NPS specimens collected in VTM/UTM. Positive panel members were prepared by spiking cultured viral stocks of influenza A, influenza B, RSV and inactivated SARS-CoV-2 into negative pooled nasopharyngeal swab specimens at 2x, 3x, and ${ 5 } \mathbf { x }$ LoD. The negative panel member

consisted of pooled nasopharyngeal swab specimens, pre-screened to be negative for influenza A, influenza B, RSV, and SARS-CoV-2. Each panel member was tested immediately after preparation (control condition) and after two freeze/thaw cycles. The results of the study are presented in Table 12.

Table 12: Freeze-Thaw Study Results Summary   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>StorageCondition</td><td rowspan=1 colspan=4>Detection Rate(n Agreement with Expected Result/ N Tested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td></tr><tr><td rowspan=1 colspan=1>TwoFreeze/Thaws</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td></tr><tr><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>TwoFreeze/Thaws</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=2 colspan=1>3x LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>TwoFreeze/Thaws</td><td rowspan=1 colspan=1>100%(42/42)</td><td rowspan=1 colspan=1>100%(42/42)</td><td rowspan=1 colspan=1>100%(42/42)</td><td rowspan=1 colspan=1>100%(42/42)</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td></tr><tr><td rowspan=1 colspan=1>TwoFreeze/Thaws</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td></tr></table>

\* Freezing at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

# d. Kit Stability

Stability of the Alinity M Resp-4-Plex AMP Kit components and Alinity m Resp-4-Plex CTRL Kit were evaluated in real time stability study. The study data supports the claimed storage conditions for the Alinity M Resp-4-Plex AMP Kit and Alinity m Resp-4-Plex CTRL Kit.

# e. On-board Reagent Stability

Stability of the Alinity M Resp-4-Plex AMP kit components and Alinity m Resp-4-Plex CTRL Kit while on-board the Alinity n System was evaluated. The study data supports the claimed on-board storage conditions for the Alinity M Resp-4-Plex AMP Kit and Alinity m Resp-4-Plex CTRL Kit.

# 6. Detection Limit:

The Limit of Detection (LoD) of the Alinity m Resp-4-Pleax assay was determined by testing ten (10) cultured viruses, including five (5) influenza A strains (H1N1 and H3N2 subtypes), two (2) influenza B strains (Victoria and Yamagata lineages), two (2) RSV strains (RSV A and RSV B), and one inactivated SARS-CoV-2 strain diluted in pooled negative NP clinical specimens collected in viral transport media.

For each virus, the preliminary LoD was determined by testing a minimum of three levels, each in a minimum of three replicates. The final LoD was confirmed by testing a minimum of 20 replicates of each strain at concentrations bracketing the preliminary LoD. For each strain, LoD was defined as the lowest concentration at which greater than or equal to $9 5 \%$ of all replicates tested positive (positive rate greater or equal to $9 5 \%$ ), as summarized in Table 13.

Table 13: Confirmatory Limit of Detection Study   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>n Positive/NTested(DetectionRate)</td></tr><tr><td rowspan=5 colspan=1>Influenza A</td><td rowspan=1 colspan=1>A/Brisbane/59/2007 (H1N1)</td><td rowspan=1 colspan=1>0.002TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/45/2015 (H1N1)</td><td rowspan=1 colspan=1>0.004TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/Switzerland/9715293/13 (H3N2)</td><td rowspan=1 colspan=1>0.015TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/04/2018 (H3N2)</td><td rowspan=1 colspan=1>0.06 TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/Darwin/9/21(H3N2)</td><td rowspan=1 colspan=1>0.004TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Victoria lineage/Brisbane/33/08</td><td rowspan=1 colspan=1>0.02 TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>B/Yamagata lineage/Massachusetts/2/12</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>RSVA/Long/MD/56</td><td rowspan=1 colspan=1>0.3 TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>RSVB/West Virginia/14617/85</td><td rowspan=1 colspan=1>0.1 TCID50/mL</td><td rowspan=1 colspan=1>21/21 (100%)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Isolate USA-WA1/2020, gammairradiated</td><td rowspan=1 colspan=1>30 GenomeCopies/mL</td><td rowspan=1 colspan=1>20/21 (95.2%)</td></tr></table>

# 7. Assay Cut-Off:

The Abbot m Resp-4-Plex assay was designed to report results through all 42 PCR cycles, the maximum number of cycles for this assay. When an amplification of specimen occurred between PCR cycles 1 to 42 (CN) and met the MR threshold (PCR efficiency-related maximum ratio (MR)) the specimen was interpreted as Positive initially. Subsequently, a CN cutoff for the Alinity m Resp-4-Plex assay was selected at 37.91 for SARS-CoV-2, 40.0 for influenza A and influenza B, and 39.5 for RSV.

8. Accuracy (Instrument): Not applicable.

# 9. Carry-Over:

Potential carryover contamination for the Alinity m Resp-4-Plex assay was evaluated by testing alternating replicates of high positive and negative samples. SARS-CoV-2 (as the representative analyte) high-positive samples were prepared by diluting inactivated SARSCoV-2 into simulated nasal matrix to $2 . 0 \mathrm { E } { + } 0 9 $ copies/mL. Unspiked simulated nasal matrix served as the negative sample. SARS-CoV-2 high-positive and negative samples were tested across a minimum of five Alinity m Resp-4-Plex AMP Trays on each of three Alinity m systems (minimum total of 15 plates). Each AMP Tray tested 24 replicates of high-positive sample and 24 replicates of negative sample arranged such that high-positive and negative samples were processed in alternating reaction vessels. The Alinity m Resp-4-Plex assay did not exhibit carryover from high positive samples to negative samples. The carryover rate was

$0 . 0 \%$ (0 detected out of 360 valid negative samples). Results from the carry-over study are presented in Table 14.

Table 14: Carryover Rate   

<table><tr><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>Tested(N)</td><td rowspan=1 colspan=1>Negative(N)</td><td rowspan=1 colspan=1>Positive(N)</td><td rowspan=1 colspan=1>Carry-overRate</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%(0/360)</td><td rowspan=1 colspan=1>(0.0%,1.1%)</td></tr></table>

N – number of replicates

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison:

# Matrix Equivalency - Simulated Nasal Matrix (SNM) vs. Pooled Clinical Nasopharyngeal Swab Specimens

The purpose of this study was to demonstrate equivalence between simulated nasal matrix (SNM) and pooled clinical nasopharyngeal swab specimens (clinical NP pool) collected in viral transport media (UTM) for Alinity m Resp-4-Plex assay. The equivalence between SNM and clinical NP pool was demonstrated by testing positive samples prepared with cultured viral stocks of influenza A, influenza B, RSV and inactivated SARS-CoV-2 in both matrix types at approximately 1x and 5x the assay’s claimed Limit of Detection (LoD) for the respective analyte. Twenty-one replicates were tested at 1x LoD and 10 replicates were tested at 5x LoD for each matrix type.

Positivity (detection) rates observed in both SNM, and clinical NP Pool were greater than or equal to $9 5 \%$ for all analytes tested targeted at 1x LoD level and equal to $100 \%$ for all analytes at 5x LoD. Results are summarized in Table 15 below. Based on this data, Alinity m Resp-4-Plex assay performance is equivalent in SNM and pooled clinical nasopharyngeal swab specimens.

Table 15: Alinity m Resp-4-Plex Assay Matrix Equivalency Study Results   

<table><tr><td colspan="1" rowspan="2">Viral Target</td><td colspan="1" rowspan="2">Matrix</td><td colspan="3" rowspan="1">1x LoD</td><td colspan="3" rowspan="1">5x LoD</td></tr><tr><td colspan="1" rowspan="1">ReplicatesTested</td><td colspan="1" rowspan="1">ReplicatesPositive</td><td colspan="1" rowspan="1">PositivityRate</td><td colspan="1" rowspan="1">ReplicatesTested</td><td colspan="1" rowspan="1">ReplicatesPositive</td><td colspan="1" rowspan="1">PositivityRate</td></tr><tr><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">SNM</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">ClinicalNP Pool</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">SNM</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">ClinicalNP Pool</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">RSV</td><td colspan="1" rowspan="1">SNM</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">ClinicalNP Pool</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">SNM</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">ClinicalNP Pool</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr></table>

SNM – Simulated Nasal Matrix

# C Clinical Studies:

1. Clinical Sensitivity:

# Prospective Clinical Study

The performance of the Alinity m Resp-4-Plex assay was evaluated in a multicenter study using prospectively collected nasopharyngeal swab specimens in UVT or UTM from individuals presenting with signs and symptoms of respiratory tract infections. A total of four US clinical sites tested the specimens with the Alinity m Resp-4-Plex assay. All four sites tested specimens for influenza A, influenza B, and RSV performance, and three of the four sites tested specimens for SARS-CoV-2 performance.

Specimens collected for SARS-CoV-2 performance evaluation were collected at ten geographically distributed locations in the US over two time periods; between December 2, 2020 and February 10, 2021 and between May 8, 2023 through May 31, 2023. A total of 874 specimens were tested, of which 48 results were excluded due to invalid results of the candidate device and 128 were excluded due to no composite comparator results, leaving 698 valid SARS-CoV-2 results.

The specimens for influenza A, influenza B, and RSV performance evaluation were collected during the 2021-2022 viral respiratory season at seven geographically distributed locations in the US and during the 2020 viral respiratory season at one location in the Southern Hemisphere. All specimens collected in the Southern Hemisphere were frozen and shipped to a US testing site for candidate and comparator device testing. Valid Alinity m Resp-4-Plex results included 2,753 each for influenza A and influenza B analytes (from 2,453 US specimens and 300 specimens from outside the US), and 2,745 for RSV analyte (from 2,445 US specimens and 300 specimens from outside the US). Of these, 2,504 influenza A results, 2,710 influenza B results, and 2,700 RSV results from the Alinity m Resp-4-Plex assay were used in the analysis. Composite comparator results could not be determined for 249 specimens for influenza A, 43 specimens for influenza B, and 45 specimens for RSV.

Demographic characteristics for the subjects included in the analysis from the prospective population are shown in Table 16 for the evaluation of influenza A, influenza B, and RSV and Table 17 for SARS-CoV-2.

Table 16: Demographics of Subjects from Prospective Population that were Included in the Performance Analysis for Influenza A, Influenza B, and RSV   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="2">Statistic</td></tr><tr><td colspan="1" rowspan="1">Age</td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">2,710</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">37.0</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">28.02</td></tr><tr><td colspan="1" rowspan="1">Minimum</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Maximum</td><td colspan="1" rowspan="1">103</td></tr><tr><td colspan="1" rowspan="1">Age Group</td><td colspan="1" rowspan="1">n (%)</td></tr><tr><td colspan="1" rowspan="1">Birth to 5 years</td><td colspan="1" rowspan="1">600 (22.1%)</td></tr><tr><td colspan="1" rowspan="1">6 to 21 years</td><td colspan="1" rowspan="1">359 (13.2%)</td></tr><tr><td colspan="1" rowspan="1">22 to 59 years</td><td colspan="1" rowspan="1">1051 (38.8%)</td></tr><tr><td colspan="1" rowspan="1">≥ 60 years</td><td colspan="1" rowspan="1">700 (25.8%)</td></tr></table>

Table 17: Demographics of Subjects from Prospective Population that were Included in the Performance Analysis for SARS-CoV-2   

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=2 colspan=1>Statistic</td></tr><tr><td rowspan=1 colspan=1>Age</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>698</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>41.3</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>16.88</td></tr><tr><td rowspan=1 colspan=1>Minimum</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Maximum</td><td rowspan=1 colspan=1>87</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>n (%)</td></tr><tr><td rowspan=1 colspan=1>Birth to 5 years</td><td rowspan=1 colspan=1>5 (0.7%)</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>80 (11.5%)</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>497 (71.2%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>116 (16.6%)</td></tr></table>

For the evaluation of influenza A, influenza B, and RSV, the Alinity m Resp-4-Plex results were compared to a composite comparator (CC) established using two to three FDA cleared assays for influenza A, influenza B, and RSV. For evaluation of SARS-CoV-2 performance, the Alinity m Resp-4-Plex results were compared to CC established using two to three highly sensitive EUA SARS-CoV-2 molecular assays. Specimens were tested with up to three comparator assays, volume availability permitting. For each analyte, a minimum of two concordant comparator assays results were needed to establish CC for each subject. In cases where results between two comparator tests differed in qualitative results or one of the comparator tests did not have a valid result, a third comparator test was used as a tiebreaker to determine CC. A specimen was categorized as CC positive if a minimum of two comparator positive results were reported. A specimen was categorized as CC negative if two or more comparator results were negative. A specimen was categorized as indeterminate (IND) if CC could not be determined due to missing results from the comparator assays. Specimens with an Indeterminate (IND) CC were excluded from analyses.

The performance of the Alinity m Resp-4-Plex assay with prospective specimens is summarized in Table 18. Alinity m Resp-4-Plex results were compared to composite comparator by the analysis of positive percent agreement (PPA) and negative percent agreement (NPA). PPA was not calculated for the influenza B analyte because there was no CC positive specimen in the prospective study.

Table 18: Alinity m Resp-4-Plex Performance with Prospective Clinical Samples   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>443</td><td rowspan=1 colspan=1>182/191</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>91.3-97.5</td><td rowspan=1 colspan=1>443/460</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>94.2-97.7</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>34.2-100.0</td><td rowspan=1 colspan=1>42/45</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>82.1-97.7</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>184</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>485</td><td rowspan=1 colspan=1>184/193</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>91.4-97.5</td><td rowspan=1 colspan=1>485/505</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>94.0-97.4</td></tr><tr><td rowspan=3 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2088</td><td rowspan=1 colspan=1>94/94</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.1-100.0</td><td rowspan=1 colspan=1>2088/2097</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.2-99.8</td></tr><tr><td rowspan=1 colspan=1>Frozenb</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>51.0-100.0</td><td rowspan=1 colspan=1>309/309</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2397</td><td rowspan=1 colspan=1>98/98</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.2- 100.0</td><td rowspan=1 colspan=1>2397/2406</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.3-99.8</td></tr><tr><td rowspan=3 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2383</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NCa</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>2383/2384</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Frozenb</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>326</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NCa</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2709</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>2709/2710</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.8-100.0</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2318</td><td rowspan=1 colspan=1>48/49</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.3-99.6</td><td rowspan=1 colspan=1>2318/2325</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.4-99.9</td></tr><tr><td rowspan=1 colspan=1>Frozenb</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>326</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NCa</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2644</td><td rowspan=1 colspan=1>48/49</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.3-99.6</td><td rowspan=1 colspan=1>2644/2651</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.5-99.9</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; NC – Not calculable; CI – Confidence Interval a PPA was not calculated because there was no CC positive specimen in this study. b Performance data for specimens collected outside the US and in the US are similar.

# Retrospective Clinical Study

Influenza B prevalence was very low during the prospective clinical study; there was no comparator positive specimen to evaluate the performance of the Alinity m Resp-4-Plex assay. To supplement the results of the prospective specimen population, retrospective specimen testing was performed using preselected archived influenza B positive NP swab specimens in UVT or UTM collected during the 2017-2018 and 2019-2020 influenza seasons, which were randomly mixed with known negative specimens to target $10 \%$ influenza B prevalence. The specimens were tested with both Alinity m Resp-4-Plex and FDA cleared comparators for influenza A, influenza B, and RSV. A total of 515 valid Alinity m Resp-4-Plex results were obtained for each of the influenza A, influenza B, and RSV analytes. Of these, 506 influenza A results, 504 of influenza B results, and 505 RSV results were used in the analysis. Composite comparator results could not be determined for nine specimens for influenza A, 11 specimens for influenza B, and ten specimens for RSV. Demographic characteristics for the subjects included in the analysis from the retrospective population are shown in Table 19.

Table 19: Summary of Demographic Characteristics for Subjects Included in the Analysis – Retrospective Population.   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="2">Statistic</td></tr><tr><td colspan="1" rowspan="1">Age</td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">506</td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">22.1</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">24.86</td></tr><tr><td colspan="1" rowspan="1">Minimum</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Maximum</td><td colspan="1" rowspan="1">93</td></tr><tr><td colspan="1" rowspan="1">Age Group</td><td colspan="1" rowspan="1">n (%)</td></tr><tr><td colspan="1" rowspan="1">Birth to 5 years</td><td colspan="1" rowspan="1">219 (43.3%)</td></tr><tr><td colspan="1" rowspan="1">6 to 21 years</td><td colspan="1" rowspan="1">99 (19.6%)</td></tr><tr><td colspan="1" rowspan="1">22 to 59 years</td><td colspan="1" rowspan="1">139 (27.5%)</td></tr><tr><td colspan="1" rowspan="1">≥ 60 years</td><td colspan="1" rowspan="1">49 (9.7%)</td></tr></table>

The agreement of the Alinity m Resp-4-Plex assay and the CC with retrospective specimens is summarized in Table 20. PPA was not calculated for influenza A or RSV because there was no CC positive for influenza A and only 1 CC positive for RSV in the retrospective study.

Table 20: Alinity m Resp-4-Plex Agreement with Retrospective Clinical Samples   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>TP</td><td rowspan=2 colspan=1>FP</td><td rowspan=2 colspan=1>FN</td><td rowspan=2 colspan=1>TN</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>503</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>503/506</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.3-99.8</td></tr><tr><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>447</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>92.9-100.0</td><td rowspan=1 colspan=1>447/454</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>96.9-99.3</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>NAa</td><td rowspan=1 colspan=1>NAa</td><td rowspan=1 colspan=1>NAa</td><td rowspan=1 colspan=1>504/504</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.2-100.0</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; NC – Not calculable; CI – Confidence Interval a PPA was not calculated because only one CC positive specimen for RSV in this study

2. Clinical Specificity: See Clinical Sensitivity above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The Alinity m Resp-4-Plex assay included a total of 874 prospectively collected NPS specimens that were tested for SARS-CoV-2, of which 698 results were evaluable. Influenza A, influenza B, and RSV included 2,753 each for influenza A and influenza B analytes and 2,745 for RSV. Of these, 2,504 influenza A analyte results, 2,710 influenza B analyte results, and 2,700 RSV analyte results of the Alinity m Resp-4-Plex assay were evaluable. Positivity, as determined by the Alinity m Resp-4-Plex AMP Kit results are presented in Table 21 – 23 for SARS-CoV-2 stratified by collection site and collection period and influenza A, influenza B, and RSV stratified by collection site.

Table 21: Alinity m Resp-4-Plex Expected Results by Specimen Collection Site for Specimens Collected from December 2020 to February 2021 for SARS-CoV-2   

<table><tr><td colspan="1" rowspan="1">Collection Site</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expected Value(n Positive/N Total)</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">171</td><td colspan="1" rowspan="1">38.6% (66/171)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">21.3% (13/61)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">20.4%(22/108)</td></tr><tr><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">48.0%(12/25)</td></tr><tr><td colspan="1" rowspan="1">Site 5</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">37.5%(24/64)</td></tr><tr><td colspan="1" rowspan="1">Site 6</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">67.6%(23/34)</td></tr><tr><td colspan="1" rowspan="1">Site 7</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">41.1%(23/56)</td></tr><tr><td colspan="1" rowspan="1">Site 8</td><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">27.3%(21/77)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">596</td><td colspan="1" rowspan="1">34.2% (204/596)</td></tr></table>

Table 22: Alinity m Resp-4-Plex Expected Results by Specimen Collection Site for Specimens Collected in May 2023 for SARS-CoV-2   

<table><tr><td rowspan=1 colspan=1>Collection Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expected Value(n Positive/N Total)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0.0% (0/0)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>0.0% (0/0)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>0.0% (0/102)</td></tr></table>

Table 23: Alinity m Resp-4-Plex Expected Results by Specimen Collection Site for Influenza A, Influenza B, and RSV   

<table><tr><td rowspan=2 colspan=1>Collection Site</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=2>Influenza B</td><td rowspan=1 colspan=2>RSV</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>ExpectedValues(n Positive/NTotal)</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>ExpectedValues (nPositive/NTotal)</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expected Values(n Positive/NTotal)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>291</td><td rowspan=1 colspan=1>0.0% (0/291)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0.0% (0/300)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0.0% (0/300)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>1185</td><td rowspan=1 colspan=1>2.2 (26/1185)</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>0.1% (1/1222)</td><td rowspan=1 colspan=1>1221</td><td rowspan=1 colspan=1>2.4% (29/1221)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>6.2% (13/211)</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>0.0% (0/309)</td><td rowspan=1 colspan=1>307</td><td rowspan=1 colspan=1>5.2% (16/307)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>349</td><td rowspan=1 colspan=1>16.0% (56/349)</td><td rowspan=1 colspan=1>404</td><td rowspan=1 colspan=1>0.0% (0/404)</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>1.7% (7/401)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>9.3%(9/97)</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>0.0% (0/97)</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>2.2% (2/93)</td></tr><tr><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.0%(0/15)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.0% (0/15)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.0% (0/15)</td></tr><tr><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>5.4% (3/56)</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0.0% (0/62)</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1.6% (1/62)</td></tr><tr><td rowspan=1 colspan=1>Site 8</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0.0% (0/300)</td><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>0.0% (0/301)</td><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>0.0% (0/301)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2504</td><td rowspan=1 colspan=1>4.3% (107/2504)</td><td rowspan=1 colspan=1>2710</td><td rowspan=1 colspan=1>0.0% (1/2710)</td><td rowspan=1 colspan=1>2700</td><td rowspan=1 colspan=1>2.0% (55/2700)</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data: Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.