Inflammatix, Inc.   
Diana Lane   
VP, Quality Assurance and Regulatory Affairs 540 Oakmead Pkwy.   
Sunnyvale, California 94085

Re: K241676 Trade/Device Name: TriVerity Regulation Number: 21 CFR 866.3215 Regulation Name: Device To Detect And Measure Non-Microbial Analyte(s) In Human Clinical Specimens To Aid In Assessment Of Patients With Suspected Sepsis Regulatory Class: Class II Product Code: PRE, OOI Dated: December 9, 2024 Received: December 9, 2024

Dear Diana Lane:

We have reviewed your section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Additional information about changes that may require a new premarket notification are provided in the FDA guidance documents entitled "Deciding When to Submit a 510(k) for a Change to an Existing Device" (https://www.fda.gov/media/99812/download) and "Deciding When to Submit a 510(k) for a Software Change to an Existing Device" (https://www.fda.gov/media/99785/download).

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

All medical devices, including Class I and unclassified devices and combination product device constituent parts are required to be in compliance with the final Unique Device Identification System rule ("UDI Rule"). The UDI Rule requires, among other things, that a device bear a unique device identifier (UDI) on its label and package (21 CFR 801.20(a)) unless an exception or alternative applies (21 CFR 801.20(b)) and that the dates on the device label be formatted in accordance with 21 CFR 801.18. The UDI Rule (21 CFR 830.300(a) and 830.320(b)) also requires that certain information be submitted to the Global Unique Device Identification Database (GUDID) (21 CFR Part 830 Subpart E). For additional information on these requirements, please see the UDI System webpage at https://www.fda.gov/medical-devices/device-advicecomprehensive-regulatory-assistance/unique-device-identification-system-udi-system.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medicaldevices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn (https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatory

assistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,   
Bryan M. Digitally signed by Bryan M. Grabias -S   
Grabias -S Date: 2025.01.10 15:35:43 -05'00' Bryan Grabias, Ph. D. Acting Branch Chief Bacterial Respiratory and Medical Countermeasures Branch Division of Microbiology Devices OHT7: Office of In Vitro Diagnostics Office of Product Evaluation and Quality Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K241676

Device Name TriVerity

Indications for Use (Describe)

The TriVerity test is an automated, semi-quantitative in vitro diagnostic test that measures the relative expression levels of host response genes in RNA isolated from whole blood collected in the PAXgene Blood RNA tube using reverse transcription loop-mediated isothermal amplification (RT-LAMP) on the Myrna instrument.

The TriVerity test is indicated for use in conjunction with clinical assessments and other laboratory findings as an aid to differentiate bacterial infections, viral infections, and non-infectious illness, as well as to determine the likelihood of 7- day need for mechanical ventilation, vasopressors, and/or renal replacement therapy in adult patients with suspected acute infection or suspected sepsis presenting to the emergency department.

The test generates three scores that each fall within one of five discrete interpretation bands based on the increasing likelihood of

1) bacterial infection,

2) viral infection, and   
3) severe illness, as defined by the need for mechanical ventilation, vasopressors, and/or renal replacement therapy (RRT) within seven days.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# 510(k) SUMMARY

# 1.0 GENERAL INFORMATION

Submission Date: Submitter Information Submitted by:

June 10, 2024

Contact Person:

Inflammatix, Inc.   
540 Oakmead Pkwy.   
Sunnyvale, CA 94085   
Diana Lane   
VP, Quality Assurance and Regulatory Affairs Tel: +1.831.820.7560   
Email: dlane@inflammatix.com

# 2.0 PURPOSE OF SUBMISSION

To obtain a substantial equivalence determination for TriVerity™.

# 3.0 MEASURAND

Measurement of 29 mRNA host response transcripts and 3 housekeeping transcripts

<table><tr><td rowspan=1 colspan=3>Host Response Genes</td></tr><tr><td rowspan=1 colspan=1>ANKRD22</td><td rowspan=1 colspan=1>ARG1</td><td rowspan=1 colspan=1>BATF</td></tr><tr><td rowspan=1 colspan=1>C3AR1</td><td rowspan=1 colspan=1>CD163</td><td rowspan=1 colspan=1>CEACAM1</td></tr><tr><td rowspan=1 colspan=1>CLEC5A</td><td rowspan=1 colspan=1>CTSL1</td><td rowspan=1 colspan=1>DEFA4</td></tr><tr><td rowspan=1 colspan=1>HERC5</td><td rowspan=1 colspan=1>HLA-DMB</td><td rowspan=1 colspan=1>IFI27</td></tr><tr><td rowspan=1 colspan=1>IFI44</td><td rowspan=1 colspan=1>IFI44L</td><td rowspan=1 colspan=1>IL18R1</td></tr><tr><td rowspan=1 colspan=1>IL1R2</td><td rowspan=1 colspan=1>ISG15</td><td rowspan=1 colspan=1>JUP</td></tr><tr><td rowspan=1 colspan=1>KCNJ2</td><td rowspan=1 colspan=1>LY86</td><td rowspan=1 colspan=1>OASL</td></tr><tr><td rowspan=1 colspan=1>OLFM4</td><td rowspan=1 colspan=1>PSMB9</td><td rowspan=1 colspan=1>RSAD2</td></tr><tr><td rowspan=1 colspan=1>S100A12</td><td rowspan=1 colspan=1>TDRD9</td><td rowspan=1 colspan=1>TGFBI</td></tr><tr><td rowspan=1 colspan=1>XAF1</td><td rowspan=1 colspan=1>ZDHHC19</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Housekeeping Genes</td></tr><tr><td rowspan=1 colspan=1>KPNA6</td><td rowspan=1 colspan=1>RREB1</td><td rowspan=1 colspan=1>YWHAB</td></tr></table>

# 4.0 TYPE OF TEST

Quantitative, Reverse Transcription Loop-mediated Isothermal Amplification (qRT-LAMP)

5.0

# APPLICANT

Inflammatix, Inc.

6.0

# PROPRIETARY AND ESTABLISHED NAMES

TriVerity™ TriVerity Test TriVerity Test System

# 7.0 REGULATORY INFORMATION

Trade Name: TriVerity™   
Regulation: 866.3215 – Device to detect and measure non-microbial analyte(s) in human clinical specimens to aid in assessment of patients with suspected sepsis   
Product Code: PRE – RT-qPCR Assay For mRNA Transcript Immune Biomarkers   
Panel: 81 (Microbiology)   
Device Class: Class II

# 8.0 INTENDED USE/INDICATIONS FOR USE

8.1 Intended Use See Indications for Use, below.

8.2 Indications for Use

The TriVerity test is an automated, semi-quantitative in vitro diagnostic test that measures the relative expression levels of host response genes in RNA isolated from whole blood collected in the PAXgene Blood RNA tube using reverse transcription loop-mediated isothermal amplification (RT-LAMP) on the Myrna instrument.

The TriVerity test is indicated for use in conjunction with clinical assessments and other laboratory findings as an aid to differentiate bacterial infections, viral infections, and non-infectious illness, as well as to determine the likelihood of 7- day need for mechanical ventilation, vasopressors, and/or renal replacement therapy in adult patients with suspected acute infection or suspected sepsis presenting to the emergency department.

The test generates three scores that each fall within one of five discrete interpretation bands based on the increasing likelihood of

1) bacterial infection,   
2) viral infection, and   
3) severe illness, as defined by the need for mechanical ventilation, vasopressors, and/or renal replacement therapy (RRT) within seven days.

8.3 Special conditions for use statement(s): Rx – For Prescription Use Only

8.4 Special instrument requirements: Myrna instrument

# 9.0 DEVICE DESCRIPTION

# 9.1 Device Description:

The TriVerity test is an in vitro diagnostic test for simultaneous amplification and detection of 29 informative host response RNA transcripts and 3 housekeeping transcripts (listed in measurands) using total RNA extracted from human blood. The test has been designed, manufactured, and validated for use on the Myrna instrument. The TriVerity test is performed with a TriVerity cartridge, a single-use, disposable, multi-chambered fluidic cartridge that runs on the Myrna instrument. All processing steps are automated and occur within a TriVerity cartridge, including sample extraction/purification and qRT-LAMP for the detection and relative quantification of the 29 informative host response RNA transcripts and 3 housekeeping transcripts (listed in measurands). All cartridge steps in this process, following the addition of the sample, are fully automated and completely integrated. In approximately 30 minutes, the test generates three scores that each fall within one of five discrete interpretation bands based on the increasing likelihood of

1) bacterial infection,   
2) viral infection, and   
3) severe illness, as defined by the need for mechanical ventilation, vasopressors, and/or renal replacement therapy (RRT) within seven days.

The specimen used for the TriVerity test is a sample of whole blood collected by venipuncture using the PAXgene blood collection tubes within the PAXgene Blood RNA System (K042613). The cartridge contains all the necessary reagents to perform RNA isolation and subsequent amplification from the sample.

The TriVerity test quantifies the amount of each transcript in the sample based on the detection of fluorescence by the Myrna instrument. The cartridge includes the reagents for reverse transcription and LAMP. All 32 transcripts (and two in cartridge controls) are amplified and quantified. These values are input to two fixed classifiers which output the three separate scores, each with five discrete interpretation bands. The bands reflect monotonically increasing likelihood of bacterial infection, viral infection, and severe illness, as defined by the need for mechanical ventilation, vasopressors, and/or renal replacement therapy (RRT) within seven days.

# 9.2 Principles of Operation

The TriVerity test performs RNA extraction and isolation, followed by Quantitative Reverse Transcription Loop-mediated Isothermal Amplification (qRT-LAMP) to measure the expression levels of 29 hosts response RNA transcripts and 3 housekeeping RNA transcripts isolated from whole blood. Amplicons generated by the qRT-LAMP process are detected and quantitated by an intercalating fluorescent dye. The test will only run on the Myrna instrument. The expression levels of the 32 RNA transcripts measured are input to two fixed classifiers which output three separate scores, each with five discrete interpretation bands. The bands reflect monotonically increasing likelihood of bacterial infection, viral infection, and severe illness, as defined by the need for mechanical ventilation, vasopressors, and/or renal replacement therapy (RRT) within seven days.

# 10.0 SUBSTANTIAL EQUIVALENCE

Predicate Device Name: SeptiCyte RAPID Predicate 510(k) Number: K203748 Comparison with Predicate:

Table 1: Comparison with Predicate   
Proprietary Information Information herein is not to be disclosed without permission from Inflammatix, Inc.   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TriVerity(Proposed Device)</td><td colspan="1" rowspan="1">SeptiCyte RAPID(K203748) (Predicate)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The TriVerity test is an automated, semi-quantitative in vitro diagnostic test thatmeasures the relative expression levelsof host response genes in RNA isolatedfrom whole blood collected in thePAXgene Blood RNA tube using reversetranscription loop-mediated isothermalamplification (RT-LAMP) on the Myrnainstrument.The TriVerity test is indicated for use inconjunction with clinical assessmentsand other laboratory findings as an aidto differentiate bacterial infections, viralinfections, and non-infectious illness, aswell as to determine the likelihood of 7-day need for mechanical ventilation,vasopressors, and/or renal replacementtherapy in adult patients with suspectedacute infection or suspected sepsispresenting to the emergencydepartment.The test generates three scores thateach fall within one of five discreteinterpretation bands based on theincreasing likelihood of1) bacterial infection,2) viral infection, and3) severe illness, as defined by the needfor mechanical ventilation, vasopressors,and/or renal replacement therapy (RRT)within seven days.</td><td colspan="1" rowspan="1">The SeptiCyte RAPID test is a geneexpression assay using reversetranscription polymerase chain reactionto quantify the relative expression levelsof host response genes isolated fromwhole blood collected in the PAXgeneBlood RNA Tube. The SeptiCyte RAPIDtest is used in conjunction with clinicalassessments and other laboratoryfindings as an aid to differentiateinfection-positive (sepsis) from infection-negative systemic inflammation inpatients suspected of sepsis on their firstday of ICU admission. The SeptiCyteRAPID test generates a score(SeptiScore) that falls within one of fourdiscrete Interpretation Bands based onthe increasing likelihood of infectionpositive systemic inflammation.SeptiCyte RAPID is intended for in-vitrodiagnostic use on the Biocartis IdyllaSystem.</td></tr><tr><td colspan="1" rowspan="1">Indication forUse Population</td><td colspan="1" rowspan="1">Patients presenting with suspected acuteinfection or sepsis in the emergencydepartment</td><td colspan="1" rowspan="1">Patients suspected of sepsis on their firstday of Intensive Care Unit (ICU) admission</td></tr><tr><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">II</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PRE</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SpecimenCollectionTube</td><td colspan="1" rowspan="1">PAXgene Blood RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SpecimenProcessing</td><td colspan="1" rowspan="1">Automated extraction within theinstrument platform</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">AssayPrinciple</td><td colspan="1" rowspan="1">Quantitative gene expression assay,based on real-time generation offluorescence from intercalatingfluorescent dye during RT-LAMPamplification of nucleic acidtemplates</td><td colspan="1" rowspan="1">Quantitative gene expression assay,based on real-time generation offluorescence from hydrolysis of dye-quencher hydrolysis probes during cyclesof PCR amplification of nucleic acidtemplates</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnology</td><td colspan="1" rowspan="1">Reverse transcription loop-mediatedisothermal amplification (RT-LAMP)</td><td colspan="1" rowspan="1">Reverse transcription polymerase chainreaction</td></tr><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">29 mRNA host response genes:ANKRD22, ARG1, BATF, C3AR1, CD163,CEACAM1, CLEC5A, CTSL1, DEFA4, HERC5,HLA-DMB, IFI27, IFI44, IFI44L, IL18R1, IL1R2,ISG15, JUP, KCNJ2, LY86, OASL, OLFM4,PSMB9, RSAD2, S100A12, TDRD9, TGFBI,XAF1, ZDHHC193 housekeeping genes:KPNA6, RREB1, YWHAB</td><td colspan="1" rowspan="1">Two mRNA transcript immunebiomarkers: PLA2G7, PLAC8</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Myrna instrument</td><td colspan="1" rowspan="1">Biocartis Idylla system</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two internal controls were developed,serving as within-cartridge positivecontrols. They serve as sample processingand qRT-LAMP controls. The controls areRNA spike in controls from the ExternalRNA Controls Consortium.External controls are not provided with theassay and are not required for assayperformance; however, labeling describestheir setup and use.</td><td colspan="1" rowspan="1">MS2 bacteriophage particles, serving assample processing control (SPC), i.e., aswithin-cartridge positive control for boththe sample extraction step and thecoupled RT-qPCR step.External controls not provided with theassay but are described in labeling withprotocols available from sponsor.</td></tr><tr><td colspan="1" rowspan="1">Result Output</td><td colspan="1" rowspan="1">TriVerity measures the expression levels of32 mRNA transcripts which are input intotwo fixed classifiers. The result is threeseparate scores, each with five discreteinterpretation bands. The bands reflectmonotonically increasing likelihood ofbacterial infection, viral infection, andsevere illness, as defined by the need formechanical ventilation, vasopressors,and/or renal replacement therapy (RRT)within seven days.</td><td colspan="1" rowspan="1">SeptiScore, calculated from theexpression levels of the two mRNAanalytes PLA2G7, PLAC8.The SeptiScore is placed into fourdiscrete bands that describe amonotonically increasing likelihood ofsepsis vs. Systemic InflammatoryResponse Syndrome (SIRS).</td></tr></table>

# 11.0 STANDARD/GUIDANCE DOCUMENTS REFERENCED

• CLSI EP05-A3 – Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline - Third Edition   
• CLSI EP06 – Evaluation of the Linearity of Quantitative Measurement Procedures: A Statistical Approach; Approved Guideline – Second Edition   
• CLSI EP07 – Interference Testing in Clinical Chemistry; Approved Guideline – Third Edition   
• CLSI EP17-A2 – Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition   
• CLSI EP25-A – Evaluation of Stability of In Vitro Diagnostic Reagents: Approved Guideline   
• CLSI EP37 – Supplemental Tables for Interference Testing in Clinical Chemistry (1st Edition)   
• ASTM D4169-23 – Standard Practice for Performance Testing of Shipping Containers and Systems   
• ASTM F2825-18 – Standard Practice for Climatic Stressing of Packaging Systems for Single Parcel Delivery   
• FDA Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices (issued 11 May 2005)

# 12.0 PERFORMANCE CHARACTERISTICS

12.1 Analytical Performance

# a. Reproducibility

Multi-site Reproducibility

A multi-site reproducibility study was conducted at two external sites and one internal site, each equipped with three instruments and multiple operators. The panel of four contrived samples used in the study spanned the range of scores expected from patient samples. Testing was conducted over five days with two runs per day with each panel in triplicate. Each of the two daily runs was performed by a different operator. The expected sample results are summarized in 2. Standard deviation was calculated for each score (Bacterial/Viral/Severity) and for each band (15 bands). The standard deviation measured in each band, and for each overall score was within acceptance criteria.

Table 2: Overview of the Contrived Panel Members for Reproducibility and Precision   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Expected Results</td></tr><tr><td rowspan=1 colspan=1>Sample A</td><td rowspan=1 colspan=1>Bacterial Result: HighViral Result: ModerateSeverity Result: High</td></tr><tr><td rowspan=1 colspan=1>Sample B</td><td rowspan=1 colspan=1>Bacterial Result: Very HighViral Result: Very LowSeverity Result: Very High</td></tr><tr><td rowspan=1 colspan=1>Sample C</td><td rowspan=1 colspan=1>Bacterial Result: Very LowViral Result: Very HighSeverity Result: Very Low</td></tr><tr><td rowspan=1 colspan=1>Sample F</td><td rowspan=1 colspan=1>Bacterial Result: Very LowViral Result: Very LowSeverity Result: Very Low</td></tr></table>

Table 3: Summary of results for multi-site reproducibility   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>ScoreType</td><td rowspan=1 colspan=1>MeanScore</td><td rowspan=1 colspan=1>BetweenDay SD</td><td rowspan=1 colspan=1>BetweenInstrument SD</td><td rowspan=1 colspan=1>BetweenSite SD</td><td rowspan=1 colspan=1>ReproducibilitySD</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>38.17</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>4.91</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>22.33</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.52</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>5.21</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>37.48</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>3.65</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>47.42</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.75</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>6.24</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.91</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>48.20</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.84</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.51</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>49.10</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.40</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>6.86</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.56</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>3.49</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.84</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>3.17</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.01</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>8.89</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>2.55</td></tr></table>

# Intra-Site Precision

Precision was assessed using a total of 138 evaluable clinical samples covering all score bands and score types, with at least 15 results in every score band. Each clinical sample was measured consecutively in duplicate within the same instrument and using the same cartridge lot in the same laboratory. The study was performed over 5 days using 3 different cartridge lots and 12 different instruments. The standard deviation (SD) was calculated for each score (Bacterial/Viral/Severity) and for each band (15 bands). The SD measured in each band, and for each overall score was within acceptance criteria. The overall SD for bacterial, viral, and severity scores were 2.71, 2.89, and 2.93, respectively:

Table 4: Precision measured using replicates of clinical samples   

<table><tr><td rowspan=2 colspan=1>Band</td><td rowspan=1 colspan=3>Imprecision SD (N=138)</td></tr><tr><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>Severity</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.59</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>1.72</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.21</td><td rowspan=1 colspan=1>4.32</td><td rowspan=1 colspan=1>3.89</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.90</td><td rowspan=1 colspan=1>3.56</td><td rowspan=1 colspan=1>3.61</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.69</td><td rowspan=1 colspan=1>3.58</td><td rowspan=1 colspan=1>2.92</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>1.38</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2.71</td><td rowspan=1 colspan=1>2.89</td><td rowspan=1 colspan=1>2.93</td></tr></table>

# Lot-to-lot Reproducibility

The lot-to-lot reproducibility of the TriVerity test score was assessed using a consistent measurement procedure, instrument, operator, operating conditions, and location to obtain measurements of the same sample using three different cartridge lots (Lot A, Lot B, Lot C) with varied lots of reagent formulation of critical reagents. For each contrived sample level (A, B, C, F), a single Myrna instrument was used to run sample replicates in cartridges with alternating runs for each lot, resulting in a total of six replicates per lot/sample level. This design was applied to all four contrived sample levels to characterize the reproducibility of the different score bands across different cartridge lots. The SDs measured for each score and sample type were within acceptance criteria.

Table 5: Lot-to-Lot Reproducibility Results   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Score</td><td rowspan=1 colspan=1>Mean Score</td><td rowspan=1 colspan=1>Between Lot SD</td><td rowspan=1 colspan=1>Overall SD</td></tr><tr><td rowspan=3 colspan=1>Sample A</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>33.56</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>27.33</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=3 colspan=1>Sample B</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>47.61</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>6.28</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>47.94</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=3 colspan=1>Sample C</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>49.17</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>7.33</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=3 colspan=1>Sample F</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>3.33</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>3.94</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>9.33</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.5</td></tr></table>

# Repeatability

TriVerity test score repeatability was assessed using the same measurement procedure, the same operator, the same measuring system, the same operating conditions, and the same location to obtain replicate measurements of the same object over a period of 12 (nonconsecutive) days. For each contrived sample level (Samples A, B, C, F), a single Myrna instrument was used to measure cartridges from a single cartridge lot per sample level, with duplicate runs in the morning and duplicate runs in the afternoon. This design was applied to all 4 contrived sample levels to fully test repeatability of the different score bands. The results obtained across all runs for all sample/score combinations met the acceptance criteria.

Table 6: Repeatability Study Results   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Score</td><td rowspan=1 colspan=1>Mean Score</td><td rowspan=1 colspan=1>RepeatabilitySD</td></tr><tr><td rowspan=3 colspan=1>Sample A</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>4.99</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3.97</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>4.66</td></tr><tr><td rowspan=3 colspan=1>Sample B</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>45.9</td><td rowspan=1 colspan=1>1.06</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>3.23</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>47.9</td><td rowspan=1 colspan=1>0.89</td></tr><tr><td rowspan=3 colspan=1>Sample C</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.50</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>49.8</td><td rowspan=1 colspan=1>0.38</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>1.21</td></tr><tr><td rowspan=3 colspan=1>Sample F</td><td rowspan=1 colspan=1>Bacterial</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>0.43</td></tr><tr><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.88</td></tr><tr><td rowspan=1 colspan=1>Severity</td><td rowspan=1 colspan=1>8.1</td><td rowspan=1 colspan=1>1.76</td></tr></table>

# b. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Real-time Reagent Stability

Real-time stability studies are ongoing to support product claims for room temperature storage $( 1 5 - 3 0 ^ { \circ } \mathsf { C } )$ .

# PAXgene Sample Collection/Handling

A specimen stability study was conducted to confirm the equivalence of TriVerity results on the Myrna instrument from paired fresh PAXgene Blood RNA samples (run within 1 hour of collection) compared to the PAXgene Blood RNA samples stored at room temperature for 24 hours and the same blood sample stored frozen at ${ } ^ { - 8 0 } { } ^ { \circ } \mathsf { C }$ for 24 hours followed by a thaw cycle. The scores of 60 patient samples were compared when fresh after collection (T0), after 24 hours at room temperature (T1) and when subsequently frozen at ${ } ^ { - 8 0 } { } ^ { \circ } \mathsf { C }$ for 24 hours (T2). The results met acceptance criteria on mean score shift(bias) of T1 and T2 relative to T0.

Additionally, a specimen stability study was conducted to confirm fresh specimen stability between 0 and 12 hours under real-time storage conditions. The study was conducted with fresh PAXgene Blood RNA samples from 16 healthy volunteers. The study compared scores from paired fresh PAXgene Blood RNA samples (run within 1 hour of collection, T0) to the PAXgene Blood RNA samples stored at each of $1 5 ~ ^ { \circ } \mathsf { C }$ and ${ 2 5 ^ { \circ } } \complement$ for 2 hours (T1), 12 hours (T2), and 14 hours (T3) post-collection. The results met acceptance criteria on mean score shift (bias) of T1, T2, and T3 relative to T0 at both tested temperatures.

These data support TriVerity testing immediately after blood draw, up to 12 hours after blood draw (our recommended testing window with a significant safety factor) or after sample processing according to the manufacturer instructions for the PAXgene sample tube.

# Shipping Stability

Shipping studies were conducted for both the TriVerity cartridges and the Myrna instrument. Control tests were run on cartridges prior to conditioning or simulated ship testing. Cartridge shippers were then preconditioned per ASTM F2825-18 prior to ship testing. Cartridge and instrument shippers were subjected to simulated ship testing per ASTM D4169:23, Distribution Cycle 13 (DC-13), with assurance level II. There was no impact on cartridge or instrument performance after testing.

# c. Linearity

Linearity is not applicable to scores because scores are determined by logistic regression models. Individual marker linearity was assessed using serial dilutions of pure in vitro transcribed RNA sequences (IVT) corresponding to each of the 32 markers that are measured in the TriVerity test. The TriVerity test was found to be linear for all 32 markers at all RNA input amounts tested, from $\mathsf { 1 } \mathsf { x } \mathsf { 1 0 } ^ { \mathsf { 6 } }$ copies/mL to $\mathsf { 1 } \mathsf { x } \mathsf { 1 0 } ^ { \mathsf { 9 } }$ copies/mL, considering an Allowable Deviation for Linearity (ADL) of $5 \%$ . The results validated the reportable dynamic range.

Table 7: Linearity Study Results   

<table><tr><td rowspan=1 colspan=5>Linearity testing, 1x106 copies/mL to 1x10° copies/mL</td></tr><tr><td rowspan=1 colspan=1>Gene</td><td rowspan=1 colspan=1>slope</td><td rowspan=1 colspan=1>intercept</td><td rowspan=1 colspan=1>Pearson</td><td rowspan=1 colspan=1>ADL 5%</td></tr><tr><td rowspan=1 colspan=1>ANKRD22</td><td rowspan=1 colspan=1>-1.65</td><td rowspan=1 colspan=1>27.61</td><td rowspan=1 colspan=1>0.989</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>ARG1</td><td rowspan=1 colspan=1>-1.91</td><td rowspan=1 colspan=1>31.19</td><td rowspan=1 colspan=1>0.985</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>BATF</td><td rowspan=1 colspan=1>-2.03</td><td rowspan=1 colspan=1>35.08</td><td rowspan=1 colspan=1>0.988</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>C3AR1</td><td rowspan=1 colspan=1>-1.88</td><td rowspan=1 colspan=1>31.14</td><td rowspan=1 colspan=1>0.993</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>CD163</td><td rowspan=1 colspan=1>-1.61</td><td rowspan=1 colspan=1>27.66</td><td rowspan=1 colspan=1>0.993</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>CEACAM1</td><td rowspan=1 colspan=1>-2.10</td><td rowspan=1 colspan=1>35.61</td><td rowspan=1 colspan=1>0.981</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>CLEC5A</td><td rowspan=1 colspan=1>-1.78</td><td rowspan=1 colspan=1>30.00</td><td rowspan=1 colspan=1>0.979</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>CTSL1</td><td rowspan=1 colspan=1>-2.06</td><td rowspan=1 colspan=1>33.66</td><td rowspan=1 colspan=1>0.997</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>DEFA4</td><td rowspan=1 colspan=1>-1.85</td><td rowspan=1 colspan=1>30.31</td><td rowspan=1 colspan=1>0.996</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>HERC5</td><td rowspan=1 colspan=1>-1.97</td><td rowspan=1 colspan=1>32.63</td><td rowspan=1 colspan=1>0.982</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>HLA-DMB</td><td rowspan=1 colspan=1>-2.36</td><td rowspan=1 colspan=1>39.00</td><td rowspan=1 colspan=1>0.995</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>IFI27</td><td rowspan=1 colspan=1>-1.84</td><td rowspan=1 colspan=1>32.00</td><td rowspan=1 colspan=1>0.980</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>IF144</td><td rowspan=1 colspan=1>-1.85</td><td rowspan=1 colspan=1>30.30</td><td rowspan=1 colspan=1>0.994</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>IFI44L</td><td rowspan=1 colspan=1>-2.55</td><td rowspan=1 colspan=1>41.69</td><td rowspan=1 colspan=1>0.996</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>IL18R1</td><td rowspan=1 colspan=1>-1.85</td><td rowspan=1 colspan=1>30.33</td><td rowspan=1 colspan=1>0.993</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>IL1R2</td><td rowspan=1 colspan=1>-2.95</td><td rowspan=1 colspan=1>40.95</td><td rowspan=1 colspan=1>0.961</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>ISG15</td><td rowspan=1 colspan=1>-1.74</td><td rowspan=1 colspan=1>30.82</td><td rowspan=1 colspan=1>0.996</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>JUP</td><td rowspan=1 colspan=1>-2.03</td><td rowspan=1 colspan=1>34.26</td><td rowspan=1 colspan=1>0.990</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>KCNJ2</td><td rowspan=1 colspan=1>-1.84</td><td rowspan=1 colspan=1>30.55</td><td rowspan=1 colspan=1>0.996</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>KPNA6</td><td rowspan=1 colspan=1>-1.89</td><td rowspan=1 colspan=1>31.37</td><td rowspan=1 colspan=1>0.995</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>LY86</td><td rowspan=1 colspan=1>-1.73</td><td rowspan=1 colspan=1>28.66</td><td rowspan=1 colspan=1>0.995</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>OASL</td><td rowspan=1 colspan=1>-1.57</td><td rowspan=1 colspan=1>27.59</td><td rowspan=1 colspan=1>0.982</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>OLFM4</td><td rowspan=1 colspan=1>-2.41</td><td rowspan=1 colspan=1>37.80</td><td rowspan=1 colspan=1>0.991</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>PSMB9</td><td rowspan=1 colspan=1>-2.06</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>0.989</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>RREB1</td><td rowspan=1 colspan=1>-2.15</td><td rowspan=1 colspan=1>34.02</td><td rowspan=1 colspan=1>0.991</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>RSAD2</td><td rowspan=1 colspan=1>-1.84</td><td rowspan=1 colspan=1>33.40</td><td rowspan=1 colspan=1>0.961</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>S100A12</td><td rowspan=1 colspan=1>-2.52</td><td rowspan=1 colspan=1>40.38</td><td rowspan=1 colspan=1>0.995</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>TDRD9</td><td rowspan=1 colspan=1>-2.06</td><td rowspan=1 colspan=1>32.94</td><td rowspan=1 colspan=1>0.977</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>TGFBI</td><td rowspan=1 colspan=1>-2.10</td><td rowspan=1 colspan=1>35.75</td><td rowspan=1 colspan=1>0.989</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>XAF1</td><td rowspan=1 colspan=1>-2.20</td><td rowspan=1 colspan=1>35.77</td><td rowspan=1 colspan=1>0.986</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>YWHAB</td><td rowspan=1 colspan=1>-2.95</td><td rowspan=1 colspan=1>43.84</td><td rowspan=1 colspan=1>0.943</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>ZDHHC19</td><td rowspan=1 colspan=1>-2.06</td><td rowspan=1 colspan=1>34.41</td><td rowspan=1 colspan=1>0.974</td><td rowspan=1 colspan=1>Pass</td></tr></table>

# d. Analytical Sensitivity

Analytical sensitivity was evaluated using IVT corresponding to each of the 32 markers at concentrations ranging from $\mathsf { 1 } \mathsf { x } \mathsf { 1 0 } ^ { \mathsf { 5 } }$ copies $\mathsf { m L }$ to $\mathsf { 1 } \mathsf { x } \mathsf { 1 0 } ^ { \mathsf { 6 } }$ copies/mL per concentration and evaluating amplification. The study was executed by a single operator over 3 days with 3 instruments used for testing 20 replicates per concentration. Testing was performed in descending order, from the highest concentration sample to the lowest. Analytical sensitivity was determined to be $1 { \times } 1 0 ^ { 6 } \mathrm { c p / m L }$ which was defined as the lowest concentration at which $9 5 \%$ (19/20) samples for each individual marker tested showed measurable amplification.

Table 8: Analytical Sensitivity Results   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Gene</td><td rowspan=1 colspan=3>% Amplification</td></tr><tr><td rowspan=1 colspan=1>T6 1x106</td><td rowspan=1 colspan=1>T7 5x105$</td><td rowspan=1 colspan=1>T8 1x105</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ANKRD22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ARG1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>BATF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>C3AR1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>CD163</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>CEACAM1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>CLEC5A</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>CTSL1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>DEFA4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>ERCC17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ERCC59</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>HERC5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>HLA-DMB</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>IFI27</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>IFI44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>IFI44L</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>IL18R1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>IL1R2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>ISG15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>JUP</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>KCNJ2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>KPNA6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>LY86</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>OASL</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>OLFM4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>PSMB9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>RREB1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>RSAD2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>S100A12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>TDRD9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>TGFBI</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>XAF1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>YWHAB</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ZDHHC19</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PASS</td><td rowspan=1 colspan=1>FAIL</td><td rowspan=1 colspan=1>FAIL</td></tr></table>

Proprietary Information Information herein is not to be disclosed without permission from Inflammatix, Inc.

# e. Detection limits

Two clinical sample pools were created to give adequate representation of TriVerity scores across the score range. The clinical pools were created by combining individual clinical samples stored in PAXgene tubes, with white blood cell (WBC) counts of $1 9 9 5 \ : \mathrm { c e l l s / \mu L }$ for Pool A and $1 2 9 6 { \mathsf { c e l l s } } / { \mu \mathrm { L } }$ for Pool B. The pools were measured undiluted in 10 replicates and then diluted using leukoreduced blood successively down to the equivalent to 125 cells/ $\mu \iota$ or 108 cells/ $\mu \iota$ and measured in 20 replicates.

# Limit of Blank

Leukoreduced blood was used as a blank. Ten replicates of Leukoreduced blood were tested and none provided any valid result (0/10 valid results).

# Limit of Detection

For each pool, the LoD was defined as the lowest WBC concentration for which at least $9 5 \%$ (19/20) replicates generate scores. The final LoD was the higher of the LoDs determined for each individual pool.

# Limit of Quantitation

For each pool, the LoQ was defined as the lowest WBC concentration for which at least $9 5 \%$ of WBC replicates generate scores with a standard deviation of 5.50 or less. The final LoQ was the higher of the LoQs determined for each individual pool. As two different clinical pools were used, the highest LoD/LoQ from each pool was chosen as the overall LoQ/LoQ. The LoQ and the LoD are 500 cells/µL (rounded up from 499 cells $ { \mu }  { \mathsf { L } } )$ .

Table 9: LoD and LoQ results   

<table><tr><td rowspan=2 colspan=1>Sample ID</td><td rowspan=2 colspan=1>Cells/μL</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=3>Mean Score</td><td rowspan=1 colspan=3>Score SD</td><td rowspan=2 colspan=1>% ValidRuns</td><td rowspan=2 colspan=1>LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>Bac</td><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>Sev</td><td rowspan=1 colspan=1>Bac</td><td rowspan=1 colspan=1>Viral</td><td rowspan=1 colspan=1>Sev</td></tr><tr><td rowspan=1 colspan=1>Leukoreducedblood</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Below LoD/LoQ</td></tr><tr><td rowspan=5 colspan=1>Pool A</td><td rowspan=1 colspan=1>1995</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>15.30</td><td rowspan=1 colspan=1>42.30</td><td rowspan=1 colspan=1>12.80</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Above LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>16.10</td><td rowspan=1 colspan=1>38.50</td><td rowspan=1 colspan=1>11.29</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Lod/LoQ</td></tr><tr><td rowspan=1 colspan=1>374</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>13.89</td><td rowspan=1 colspan=1>38.50</td><td rowspan=1 colspan=1>11.50</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>90%</td><td rowspan=1 colspan=1>Below LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>12.82</td><td rowspan=1 colspan=1>36.59</td><td rowspan=1 colspan=1>10.41</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>89%</td><td rowspan=1 colspan=1>Below LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>13.08</td><td rowspan=1 colspan=1>29.08</td><td rowspan=1 colspan=1>10.58</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>60%</td><td rowspan=1 colspan=1>Below LoD/LoQ</td></tr><tr><td rowspan=5 colspan=1>Pool B</td><td rowspan=1 colspan=1>1296</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>45.70</td><td rowspan=1 colspan=1>7.20</td><td rowspan=1 colspan=1>42.00</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Above LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>44.40</td><td rowspan=1 colspan=1>8.35</td><td rowspan=1 colspan=1>39.80</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Above LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>324</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>44.19</td><td rowspan=1 colspan=1>8.95</td><td rowspan=1 colspan=1>40.24</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Above LoD/LoQ</td></tr><tr><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>42.53</td><td rowspan=1 colspan=1>11.37</td><td rowspan=1 colspan=1>36.42</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>Lod/LoQ</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.85</td><td rowspan=1 colspan=1>13.54</td><td rowspan=1 colspan=1>30.38</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>65%</td><td rowspan=1 colspan=1>Below LoD/LoQ</td></tr></table>

# f. Analytical Specificity

# Cross Reactivity

Cross-reactivity and non-specific amplification of genomic DNA (gDNA) as well as no amplification from no-template controls (NTC) were evaluated for the TriVerity test.

Cross-reactivity was assessed by spiking $1 0 \mathrm { \ n g / \mu L }$ of gDNA into two levels of contrived samples with previously characterized score results and comparing them with unspiked samples, with six test runs and six control runs. The gDNA-spiked sample runs were compared with control runs by calculating the mean result for each of the three scores for spiked and control samples. The standard deviation for both conditions and sample levels were calculated, and no cross-reactivity was observed.

Genomic gDNA non-specific amplification was evaluated by spiking $2 0 \ : \mathrm { n g / \mu L }$ of gDNA into RNA stabilization buffer, resulting in a total of 20 runs. None observed.

All 20 test runs $( 1 0 0 \% )$ with no template showed no amplification signal from $2 0 \ : \mathrm { n g / \mu L }$ human genomic DNA template.

# Endogenous and Exogenous Interferents

Based on the CLSI Guidance EP07, “Interference Testing in Clinical Chemistry” the TriVerity test was evaluated in the presence of interfering substances. A specimen panel was prepared by spiking potential interferents into each of two contrived panel members B and C (see Table 2). The potential interferents were dissolved in appropriate solvents, and then spiked into the contrived samples at concentrations higher than the maxima of their normal or expected clinical reference ranges. Each substance was added to a blood sample in a PAXgene Blood RNA tube. Each interfering substance was spiked into four tubes of both panel types and tested with two different cartridge lots, resulting in a total of eight tests per interfering substance. No interference was found for any of the substances in Table 10 at the concentration listed.

Table 10: Endogenous and Exogenous Interferences Tested   

<table><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Concentration in PAXgeneBlood RNA Tube(pre-RNA extraction)</td><td rowspan=1 colspan=1>EquivalentConcentration in Wholeblood (*)</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>400 mg/L</td><td rowspan=1 colspan=1>1504 mg/L</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>10 g/L</td><td rowspan=1 colspan=1>37.6 g/L</td></tr><tr><td rowspan=1 colspan=1>Rheumatoid Factor</td><td rowspan=1 colspan=1>45 U/mL</td><td rowspan=1 colspan=1>169.2 U/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>2000 mg/dL</td><td rowspan=1 colspan=1>7520 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>5 g/dL</td><td rowspan=1 colspan=1>18.8 g/dL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>10 u/mL</td><td rowspan=1 colspan=1>37.6 U/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem/Cilastatin</td><td rowspan=1 colspan=1>100 mg/L</td><td rowspan=1 colspan=1>376 mg/L</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>100 mg/L</td><td rowspan=1 colspan=1>376 mg/L</td></tr><tr><td rowspan=1 colspan=1>Cefotaxime</td><td rowspan=1 colspan=1>400 mg/L</td><td rowspan=1 colspan=1>1504 mg/L</td></tr><tr><td rowspan=1 colspan=1>Dopamine</td><td rowspan=1 colspan=1>500 mg/dL</td><td rowspan=1 colspan=1>1880 mg/dL</td></tr><tr><td rowspan=1 colspan=1>CRP (c-reactive protein)</td><td rowspan=1 colspan=1>60 mg/L</td><td rowspan=1 colspan=1>225.6 mg/L</td></tr><tr><td rowspan=1 colspan=1>Norepinephrine</td><td rowspan=1 colspan=1>670 μmol/L</td><td rowspan=1 colspan=1>2519.2 µmol/L</td></tr><tr><td rowspan=1 colspan=1>Dobutamine</td><td rowspan=1 colspan=1>11.2 mg/L</td><td rowspan=1 colspan=1>42.1 mg/L</td></tr><tr><td rowspan=1 colspan=1>Furosemide</td><td rowspan=1 colspan=1>59.9 mg/L</td><td rowspan=1 colspan=1>225.2 mg/L</td></tr><tr><td rowspan=1 colspan=1>IL-6 (Interleukin-6)</td><td rowspan=1 colspan=1>2000 pg/mL</td><td rowspan=1 colspan=1>7520 pg/mL</td></tr><tr><td rowspan=1 colspan=1>sCD14</td><td rowspan=1 colspan=1>5 μg/mL</td><td rowspan=1 colspan=1>18.8 μg/mL</td></tr><tr><td rowspan=1 colspan=1>LPS (Lipopolysaccharides)</td><td rowspan=1 colspan=1>5 ng/mL</td><td rowspan=1 colspan=1>18.8 ng/mL</td></tr></table>

(\*) Considering a standard mixture of $2 . 5 \mathsf { m l }$ of whole blood in $6 . 9 \mathsf { m l }$ of PAXgene solution.

# Carryover

Carryover contamination was investigated using three contrived Panel members B, C, and F (see Table 2) that were tested with 3 instruments across 2 days with 10 runs per instrument per day. Tests were performed in sequence alternating Panel members and scores were evaluated in terms of bias and imprecision when compared with control conditions. No substantial bias or imprecision were observed and therefore, no carryover or amplicon contamination.

g. Assay Reportable Range See linearity above.

h. Assay Cut-Off See clinical cut-off (Section 3b) below.

12.2 Comparison Studies a. Method Comparison with Predicate Device: N/A

b. Matrix Comparison N/A

Proprietary Information Information herein is not to be disclosed without permission from Inflammatix, Inc.

# a. Clinical Sensitivity and Clinical Specificity

A prospective, multi-center clinical study (SEPSIS-SHIELD; NCT04094818) was conducted to establish the performance characteristics of the TriVerity test on the Myrna instrument. This was a non-interventional, minimal-risk study to analyze gene expression and other laboratory data from biological samples collected from participants with suspected respiratory, urinary, intra-abdominal, skin and soft tissue, meningitis/encephalitis, and at least one vital sign change, and other acute infections, or suspected sepsis of any cause with at least two vital sign changes and a blood culture order.

Study sites included emergency departments (EDs) of community and academic hospitals located in twenty-one geographically diverse locations throughout the United States plus one site in Europe. Enrollment was offered to all patients who met the enrollment criteria at presentation to the ED. Of the 1,441 participants who consented to the study, there were 1,222 evaluable participants.

The primary objective of the study was to evaluate the performance of the TriVerity test for (1) diagnosing bacterial and viral infections using consensus and forced clinical adjudication as the comparator and (2) evaluate the performance of the TriVerity test for predicting severe illness (defined as the need for mechanical ventilation, vasopressors, or renal replacement therapy [RRT] within 7 days) using clinical outcomes as the comparator.

The analysis population consisted of all participants who provided consent, met all inclusion criteria, met no exclusion criteria, and had their PAXgene blood sample collected and successfully processed (resulting in three TriVerity scores). The overall analysis population was further divided into diagnostic and prognostic analysis populations with additional requirements. The diagnostic endpoint analysis population included participants who had a completed adjudication for bacterial and viral infection to determine the ground truth. The diagnostic population was further classified as Consensus (only contains participants with a certain (Yes or No) or Forced adjudication (all patients adjudicated as Yes and/or Probable vs. No and/or Unlikely). The prognostic endpoint analysis population included participants who had information available on the presence of severe illness (acute need for mechanical ventilation, vasopressor and/or renal replacement therapy) within 7 days.

Each of the bands for the TriVerity test were assessed in terms of likelihood ratio, Positive Percent Agreement (PPA), Negative Percent Agreement (NPA), and predictive values (post-test probabilities). The calculation of PPA and NPA estimates considered each cutoff independently to assign True Positive, True Negative, False Positive, and False negative status to each assay result.

Among 1,441 participants who consented, 188 $( 1 3 . 0 \% )$ were excluded due to not meeting the inclusion criteria, meeting exclusion criteria (screen failures), or withdrawals during the study. There were 31 $( 2 . 2 \% )$ participants without valid TriVerity results. A total of 1,222 participants were evaluable for the diagnostic endpoint (accuracy of the TriVerity test for the identification of bacterial and viral infections) and 1,120 participants were evaluable for the prognostic endpoint (prediction of need for mechanical ventilation, vasopressor use, and/or RRT within 7 days).

Using forced adjudication, $5 8 . 1 \%$ bacterial infections, $20 . 0 \%$ viral infections, $2 . 5 \%$ co-infections; $1 9 . 4 \%$ of participants were adjudicated as noninfected. Consensus adjudication yielded $6 1 . 5 \%$ of bacterial infections compared to $2 2 . 6 \%$ viral infections and $1 . 6 \%$ bacterial-viral coinfections; $1 4 . 3 \%$ of participants were adjudicated to not have an infection. In the prognostic population, $1 0 . 9 \%$ of participants required mechanical ventilation, vasopressor use, and/or RRT within 7 days (prognostic endpoint).

Participant demographics at the time of presentation in the ED are shown in Table 11. These results demonstrate that a representative cohort of U.S. emergency department patients presenting with suspected acute infection and suspected sepsis was enrolled. The results also demonstrate similar age, sex, race, and ethnicity between the forced and consensus diagnostic populations, as well as between the diagnostic and prognostic populations.

Table 11: Participant Demographics for the Diagnostic and Prognostic Populations   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Population</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>DiagnosticPopulation:Forced(N=1222)</td><td rowspan=1 colspan=1>DiagnosticPopulation:Consensus(N=729)</td><td rowspan=1 colspan=1>PrognosticPopulation(N=1120)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mean (SD)</td><td rowspan=1 colspan=1>50.9 (17.61)</td><td rowspan=1 colspan=1>50.6 (17.35)</td><td rowspan=1 colspan=1>51.3 (17.62)</td></tr><tr><td rowspan=1 colspan=1>Median (Range)</td><td rowspan=1 colspan=1>52.0 (18.0, 90.0)</td><td rowspan=1 colspan=1>52.0 (18.0, 90.0)</td><td rowspan=1 colspan=1>52.0 (18.0, 90.0)</td></tr><tr><td rowspan=1 colspan=1>Sex at Birth, n (%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>605 (49.5%)</td><td rowspan=1 colspan=1>345 (47.3%)</td><td rowspan=1 colspan=1>544 (48.6%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>617 (50.5%)</td><td rowspan=1 colspan=1>384 (52.7%)</td><td rowspan=1 colspan=1>576 (51.4%)</td></tr><tr><td rowspan=1 colspan=1>Race, n (%)*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>American Indian or AlaskaNative</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (0.1%)</td></tr><tr><td rowspan=1 colspan=1>Asian*</td><td rowspan=1 colspan=1>11 (0.9%)</td><td rowspan=1 colspan=1>6 (0.8%)</td><td rowspan=1 colspan=1>8 (0.7%)</td></tr><tr><td rowspan=1 colspan=1>Black or African American</td><td rowspan=1 colspan=1>360 (29.5%)</td><td rowspan=1 colspan=1>223 (30.6%)</td><td rowspan=1 colspan=1>321 (28.7%)</td></tr><tr><td rowspan=1 colspan=1>Native Hawaiian or OtherPacific Islander</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>2 (0.3%)</td><td rowspan=1 colspan=1>2 (0.2%)</td></tr><tr><td rowspan=1 colspan=1>White*</td><td rowspan=1 colspan=1>786 (64.3%)</td><td rowspan=1 colspan=1>461 (63.2%)</td><td rowspan=1 colspan=1>734 (65.5%)</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>63 (5.2%)</td><td rowspan=1 colspan=1>37 (5.1%)</td><td rowspan=1 colspan=1>56 (5.0%)</td></tr><tr><td rowspan=1 colspan=1>Ethnicity, n (%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hispanic or Latino</td><td rowspan=1 colspan=1>135 (11.0%)</td><td rowspan=1 colspan=1>97 (13.3%)</td><td rowspan=1 colspan=1>128 (11.4%)</td></tr><tr><td rowspan=1 colspan=1>Not Hispanic or Latino</td><td rowspan=1 colspan=1>1064 (87.1%)</td><td rowspan=1 colspan=1>623 (85.5%)</td><td rowspan=1 colspan=1>972 (86.8%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>23 (1.9%)</td><td rowspan=1 colspan=1>9 (1.2%)</td><td rowspan=1 colspan=1>20 (1.8%)</td></tr></table>

\*Note: Two patients gave their race as biracial (Asian, White), added to forced and prognostic population.

Table 12 shows medical history and immune status of participants. Results obtained are representative of a U.S. emergency department cohort presenting with suspected acute infection and suspected sepsis; we did not observe marked differences in the distribution of underlying diseases and immune status in the different study populations (prognostic vs. forced diagnostic vs. consensus diagnostic populations). Importantly, a total of 219 $( 1 7 . 9 \% )$ participants in the forced population were immunosuppressed, among these, 30 fell into more than one of the immunosuppression categories. A total of 206 $( 1 8 . 4 \% )$ participants in the prognostic population were immunosuppressed. Among these, 30 participants fell into more than one immunosuppression category.

Table 12: Medical History (A) and Type of Immunosuppression (B) of Participants   

<table><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=3>Population</td></tr><tr><td rowspan=1 colspan=1>Medical History1</td><td rowspan=1 colspan=1>Diagnostic Population:Forced(N=1222)</td><td rowspan=1 colspan=1>DiagnosticPopulation:Consensus(N=729)</td><td rowspan=1 colspan=1>PrognosticPopulation(N=1120)</td></tr><tr><td rowspan=1 colspan=1>Blood Disorders</td><td rowspan=1 colspan=1>86 (7.0%)</td><td rowspan=1 colspan=1>51 (7.0%)</td><td rowspan=1 colspan=1>80 (7.1%)</td></tr><tr><td rowspan=1 colspan=1>Cardiovascular Diseases</td><td rowspan=1 colspan=1>519 (42.5%)</td><td rowspan=1 colspan=1>303 (41.6%)</td><td rowspan=1 colspan=1>485 (43.3%)</td></tr><tr><td rowspan=1 colspan=1>Kidney Diseases</td><td rowspan=1 colspan=1>224 (18.3%)</td><td rowspan=1 colspan=1>140 (19.2%)</td><td rowspan=1 colspan=1>213 (19.0%)</td></tr><tr><td rowspan=1 colspan=1>Gastrointestinal Tract Diseases</td><td rowspan=1 colspan=1>284 (23.2%)</td><td rowspan=1 colspan=1>167 (22.9%)</td><td rowspan=1 colspan=1>265 (23.7%)</td></tr><tr><td rowspan=1 colspan=1>Metabolic/EndocrinologicalDiseases</td><td rowspan=1 colspan=1>524 (42.9%)</td><td rowspan=1 colspan=1>315 (43.2%)</td><td rowspan=1 colspan=1>490 (43.8%)</td></tr><tr><td rowspan=1 colspan=1>Musculoskeletal Diseases</td><td rowspan=1 colspan=1>177 (14.5%)</td><td rowspan=1 colspan=1>106 (14.5%)</td><td rowspan=1 colspan=1>167 (14.9%)</td></tr><tr><td rowspan=1 colspan=1>Neuropsychiatric Diseases</td><td rowspan=1 colspan=1>282 (23.1%)</td><td rowspan=1 colspan=1>162 (22.2%)</td><td rowspan=1 colspan=1>264 (23.6%)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Tract Diseases</td><td rowspan=1 colspan=1>380 (31.1%)</td><td rowspan=1 colspan=1>229 (31.4%)</td><td rowspan=1 colspan=1>350 (31.3%)</td></tr><tr><td rowspan=1 colspan=1>Skin or Soft Tissue Diseases</td><td rowspan=1 colspan=1>61 (5.0%)</td><td rowspan=1 colspan=1>36 (4.9%)</td><td rowspan=1 colspan=1>58 (5.2%)</td></tr><tr><td rowspan=1 colspan=1>Urogenital Diseases</td><td rowspan=1 colspan=1>110 (9.0%)</td><td rowspan=1 colspan=1>68 (9.3%)</td><td rowspan=1 colspan=1>106 (9.5%)</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>200 (16.4%)</td><td rowspan=1 colspan=1>110 (15.1%)</td><td rowspan=1 colspan=1>186 (16.6%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=3>Population</td></tr><tr><td rowspan=1 colspan=1>Type of Immunosuppression²2</td><td rowspan=1 colspan=1>Diagnostic Population:Forced(N=1222)</td><td rowspan=1 colspan=1>DiagnosticPopulation:Consensus(N=729)</td><td rowspan=1 colspan=1>PrognosticPopulation(N=1120)</td></tr><tr><td rowspan=1 colspan=1>Bone Marrow, Stem Cell and/orOther Cell Transplant</td><td rowspan=1 colspan=1>3 (0.2%)</td><td rowspan=1 colspan=1>2 (0.3%)</td><td rowspan=1 colspan=1>3 (0.3%)</td></tr><tr><td rowspan=1 colspan=1>Malignancies</td><td rowspan=1 colspan=1>125 (10.2%)</td><td rowspan=1 colspan=1>73 (10.0%)</td><td rowspan=1 colspan=1>121 (10.8%)</td></tr><tr><td rowspan=1 colspan=1>HIV/AIDS</td><td rowspan=1 colspan=1>26 (2.1%)</td><td rowspan=1 colspan=1>16 (2.2%)</td><td rowspan=1 colspan=1>22 (2.0%)</td></tr><tr><td rowspan=1 colspan=1>Solid Organ Transplant</td><td rowspan=1 colspan=1>35 (2.9%)</td><td rowspan=1 colspan=1>27 (3.7%)</td><td rowspan=1 colspan=1>34 (3.0%)</td></tr><tr><td rowspan=1 colspan=1>Steroid Treatment</td><td rowspan=1 colspan=1>34 (2.8%)</td><td rowspan=1 colspan=1>17 (2.3%)</td><td rowspan=1 colspan=1>30 (2.7%)</td></tr><tr><td rowspan=1 colspan=1>Other Immunosuppression</td><td rowspan=1 colspan=1>30 (2.5%)</td><td rowspan=1 colspan=1>17 (2.3%)</td><td rowspan=1 colspan=1>29 (2.6%)</td></tr></table>

1 Calculated using provider questionnaire information at time of enrollment 2 Note: Participants may have more than a single type of immunosuppression

Using clinical adjudication as the reference standard, the accuracy of the bacterial result readouts of the TriVerity test by interpretation band is shown in Table 13 for the forced adjudication population.

Table 13: Accuracy of the TriVerity Test for the Diagnosis of Bacterial Infections in the Forced Population   

<table><tr><td rowspan=2 colspan=1>TriVerityBacterialBand</td><td rowspan=1 colspan=2>ClinicallyAdjudicatedBacterialInfection(Forced)</td><td rowspan=2 colspan=1>LR (80% CI)</td><td rowspan=2 colspan=1>PPA(PositivePercentAgreement)</td><td rowspan=2 colspan=1>NPA(NegativePercentAgreement)</td><td rowspan=2 colspan=1>RelativeFrequency ofResult(% in Band)</td><td rowspan=2 colspan=1>Probabilityof BacterialInfection(%)</td><td rowspan=2 colspan=1>Probabilityof NoBacterialInfection(%)</td></tr><tr><td rowspan=1 colspan=1>Yes(N)</td><td rowspan=1 colspan=1>No(N)</td></tr><tr><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>5.24(4.19-6.83)</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>94.2</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>11.0</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>184</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>1.96(1.65-2.37)</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>87.3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>75.1</td><td rowspan=1 colspan=1>24.9</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>0.95(0.82-1.11)</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>19.6</td><td rowspan=1 colspan=1>59.4</td><td rowspan=1 colspan=1>40.6</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>0.61(0.54-0.69)</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>48.5</td><td rowspan=1 colspan=1>51.5</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>0.16(0.12-0.2)</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>80.0</td></tr><tr><td rowspan=1 colspan=9>Prevalence = 60.6%</td></tr></table>

LR, likelihood ratio.

Compared to the bacterial infection status as determined by clinical adjudication, a high NPA and positive likelihood ratio were observed for the Very High interpretation band. Similarly, the High band showed high NPA and likelihood ratio for the diagnosis of a bacterial infection. In addition, TriVerity demonstrated high PPA and low negative likelihood ratio for the Very Low and Low interpretation bands. Lastly, $8 0 . 4 \%$ of results fell into the clinically actionable Very High, High, Low or Very Low interpretation. At a prevalence of $60 . 6 \%$ for bacterial infections, probabilities for having or not having a bacterial infection in the outer Very High and Very Low interpretation bands were $8 9 . 0 \%$ and $80 . 0 \%$ , respectively.

Expectedly, higher accuracy results were observed when applying the consensus adjudication as the reference standard to determine the bacterial infection status (Table 14) due to the exclusion of participants with uncertain (Probable and Unlikely) bacterial infection status.

Table 14: Accuracy of the TriVerity Test for the Diagnosis of Bacterial Infections in the Consensus Population   

<table><tr><td rowspan=2 colspan=1>TriVerityBacterialBand</td><td rowspan=1 colspan=2>ClinicallyAdjudicatedBacterialInfection(Consensus)</td><td rowspan=2 colspan=1>LR (80% CI)</td><td rowspan=2 colspan=1>PPA(PositivePercentAgreement)</td><td rowspan=2 colspan=1>NPA(NegativePercentAgreement)</td><td rowspan=2 colspan=1>RelativeFrequencyof Result(% in Band)</td><td rowspan=2 colspan=1>Probabilityof BacterialInfection(%)</td><td rowspan=2 colspan=1>Probabilityof NoBacterialInfection(%)</td></tr><tr><td rowspan=1 colspan=1>Yes(N)</td><td rowspan=1 colspan=1>No(N)</td></tr><tr><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>8.04(5.66-12.43)</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>93.2</td><td rowspan=1 colspan=1>6.8</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2.5(1.97-3.31)</td><td rowspan=1 colspan=1>23.3</td><td rowspan=1 colspan=1>90.7</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>81.1</td><td rowspan=1 colspan=1>18.9</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>1.14(0.94-1.42)</td><td rowspan=1 colspan=1>19.6</td><td rowspan=1 colspan=1>82.9</td><td rowspan=1 colspan=1>18.7</td><td rowspan=1 colspan=1>66.2</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0.54(0.45-0.63)</td><td rowspan=1 colspan=1>81.5</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>0.08(0.05-0.11)</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>87.9</td></tr><tr><td rowspan=1 colspan=9>Prevalence = 63.1%</td></tr></table>

LR, likelihood ratio.

The accuracy of the viral result readout of the TriVerity test by interpretation band is shown in Tables 15 and 16 for the forced and consensus populations. Compared to the viral infection status as determined by clinical adjudication, a high NPA $( 9 7 . 7 \% )$ and likelihood ratio (20.04) was observed for the Very High interpretation band. Similarly, the High band showed an NPA of $9 3 . 8 \%$ and a likelihood ratio of 2.04 for the diagnosis of a viral infection. TriVerity demonstrated a high PPA $( 8 8 . 7 \% )$ and a low likelihood ratio (0.25) in the Very Low interpretation band. The Low band showed a PPA of $8 5 . 1 \%$ and a likelihood ratio of 0.48. Likelihood ratios ranged from 0.25 to 20.04 and increased monotonically from the Very Low to the Very High interpretation band. Lastly, $8 4 . 9 \%$ of TriVerity viral results fell into the clinically actionable Very High, High, Low and Very Low bands; $20 . 0 \%$ of results fell into the clinically actionable Very High and High interpretation bands whereas $6 5 . 0 \%$ of results fell into the clinically actionable Very Low and Low interpretation bands; only $1 5 . 1 \%$ of results were found in the moderate interpretation band.

At a prevalence of $2 2 . 5 \%$ for viral infections, the positive and negative predictive values of the outer Very High and Very Low interpretation bands were $8 5 . 3 \%$ and $9 3 . 2 \%$ , respectively.

Table 15: Accuracy of the TriVerity Test for the Diagnosis of Viral Infections in the Forced Population   

<table><tr><td rowspan=2 colspan=1>TriVerityViralBand</td><td rowspan=1 colspan=2>ClinicallyAdjudicatedViral Infection(Forced)</td><td rowspan=2 colspan=1>LR(80% Cl)</td><td rowspan=2 colspan=1>PPA(PositivePercentAgreement)</td><td rowspan=2 colspan=1>NPA(NegativePercentAgreement)</td><td rowspan=2 colspan=1>RelativeFrequency ofResult(% in Band)</td><td rowspan=2 colspan=1>Probability ofViralInfection (%)</td><td rowspan=2 colspan=1>Probability ofNo ViralInfection (%)</td></tr><tr><td rowspan=1 colspan=1>Yes(N)</td><td rowspan=1 colspan=1>No(N)</td></tr><tr><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>20.04(15.45-27.29)</td><td rowspan=1 colspan=1>46.5</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>85.3</td><td rowspan=1 colspan=1>14.7</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>2.04(1.57-2.65)</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>62.8</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0.96(0.76-1.18)</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>84.8</td><td rowspan=1 colspan=1>15.1</td><td rowspan=1 colspan=1>21.7</td><td rowspan=1 colspan=1>78.3</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>0.48(0.39-0.57)</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>87.8</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>427</td><td rowspan=1 colspan=1>0.25(0.2-0.31)</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>45.1</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>93.2</td></tr><tr><td rowspan=1 colspan=9>Prevalence = 22.5%</td></tr></table>

LR, likelihood ratio.

Expectedly, higher accuracy TriVerity results were observed when applying the consensus adjudication as the reference standard to determine the viral infection status (Table 16) due to the exclusion of participants with uncertain (Probable and Unlikely) viral infection status.

Table 16: Accuracy of the TriVerity Test for the Diagnosis of Viral Infections in the Consensus Population   

<table><tr><td rowspan=2 colspan=1>TriVerityViralBand</td><td rowspan=1 colspan=2>ClinicallyAdjudicatedViralInfection(Consensus)</td><td rowspan=2 colspan=1>LR (80% CI)</td><td rowspan=2 colspan=1>PPA(PositivePercentAgreement)</td><td rowspan=2 colspan=1>NPA(NegativePercentAgreement)</td><td rowspan=2 colspan=1>RelativeFrequencyof Result(% in Band)</td><td rowspan=2 colspan=1>Probabilityof ViralInfection (%)</td><td rowspan=2 colspan=1>Probabilityof No ViralInfection (%)</td></tr><tr><td rowspan=1 colspan=1>Yes(N)</td><td rowspan=1 colspan=1>No(N)</td></tr><tr><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>40.93(27.73-72.16)</td><td rowspan=1 colspan=1>59.3</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>2.36(1.68-3.25)</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>43.1</td><td rowspan=1 colspan=1>56.9</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>0.87(0.64-1.13)</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1>21.8</td><td rowspan=1 colspan=1>78.2</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>0.32(0.22-0.41)</td><td rowspan=1 colspan=1>90.4</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>25.1</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>90.7</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>0.09(0.05-0.14)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>48.2</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>97.1</td></tr><tr><td rowspan=1 colspan=9>Prevalence = 24.3%</td></tr></table>

LR, likelihood ratio

Taken together, the results shown above demonstrate that the TriVerity test has very high accuracy for the diagnosis of bacterial and viral infections.

Table 17 shows the accuracy of the TriVerity illness severity results for the prediction of the need for mechanical ventilation, vasopressors, and/or RRT within 7 days. The data demonstrate that the TriVerity test was able to predict the need for mechanical ventilation, vasopressors, and/or RRT within 7 days (the prognostic severity endpoint of the study). The NPA of the Very High and High illness severity interpretation bands for the prediction of severe disease was $9 8 . 7 \%$ and $8 6 . 1 \%$ , with likelihood ratios of 11.33 and 2.41, respectively. Conversely, the PPA of the Very Low and Low illness severity bands were $9 1 . 8 \%$ and $8 7 . 7 \%$ with likelihood ratios of 0.22 and 0.43, respectively. $7 9 . 6 \%$ of results were observed in the clinically actionable Very High, High, Very Low, and Low interpretation bands; $1 8 . 8 \%$ of these in the Very High and High bands, $6 0 . 7 \%$ in the Low and Very Low bands. At a prevalence of $1 0 . 9 \%$ severe illness, probabilities of having or not having severe illness were $5 8 . 1 \%$ and $9 7 . 3 \%$ for the Very High and Very Low interpretation bands, respectively.

Table 17: Accuracy of the Severity Result Readout of the TriVerity Test for the Prediction of Mechanical Ventilation, Vasopressor Use, and/or Renal Replacement Therapy within 7 days   

<table><tr><td rowspan=2 colspan=1>TriVerityIllnessSeverityBand</td><td rowspan=1 colspan=2>Need forMechanicalVentilation,Vasopressors,and/or RRTWithin 7 Days</td><td rowspan=2 colspan=1>LR(80% Cl)</td><td rowspan=2 colspan=1>PPA(PositivePercentAgreement)</td><td rowspan=2 colspan=1>NPA(NegativePercentAgreement)</td><td rowspan=2 colspan=1>RelativeFrequencyo Result(% in Band)</td><td rowspan=2 colspan=1>Probabilityof SevereIllness (%)</td><td rowspan=2 colspan=1>ProbabilityofNo SevereIllness (%)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>11.33(7.07-17.75)</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>58.1</td><td rowspan=1 colspan=1>41.9</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>2.41(1.96-2.91)</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>86.1</td><td rowspan=1 colspan=1>16.1</td><td rowspan=1 colspan=1>22.8</td><td rowspan=1 colspan=1>77.2</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>1.63(1.35-1.97)</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>80.9</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>16.6</td><td rowspan=1 colspan=1>83.4</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>0.43(0.3-0.57)</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>367</td><td rowspan=1 colspan=1>0.22(0.14-0.31)</td><td rowspan=1 colspan=1>91.8</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>97.3</td></tr><tr><td rowspan=1 colspan=9>Prevalence = 10.9%</td></tr></table>

LR, likelihood ratio.

We also determined the accuracy of the three TriVerity test results readouts using AUROCs in Table 18.

Table 18: Area Under the Receiver-Operator Characteristics for the TriVerity Bacterial, Viral, and Severe Illness Results   

<table><tr><td rowspan=1 colspan=1>Population</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>AUROC</td><td rowspan=1 colspan=1>Lower 80% CI</td><td rowspan=1 colspan=1>Upper 80% CI</td></tr><tr><td rowspan=1 colspan=1>Bacterial Infection(Forced Adjudication)</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=1 colspan=1>Viral Infection(Forced adjudication)</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>0.85</td></tr><tr><td rowspan=1 colspan=1>Bacterial Infection(Consensus Adjudication)</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>0.85</td></tr><tr><td rowspan=1 colspan=1>Viral Infection(Consensus Adjudication)</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>0.93</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial or Viralvs. Non-Infected(Forced Adjudication)</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.81</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial or Viralvs. Non-Infected(Consensus Adjudication)</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.86</td></tr><tr><td rowspan=1 colspan=1>Severe Infection(Prognostic Endpoint)</td><td rowspan=1 colspan=1>1120</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>0.81</td></tr></table>

These results demonstrate the high AUROCs of the TriVerity bacterial (0.83) and viral (0.91) results when using the consensus adjudication. Expectedly, slightly lower results were achieved using the forced adjudication. Similarly, AUROCs for the illness severity results were high (0.78) for the prognostic endpoint.

# b. Clinical Cut-Off

TriVerity cut-off values were established prior to the registrational clinical trial (INF-04). PPA and NPA of the outside bands were the primary performance targets with a target goal of:

“Very Low” Band PPA $29 5 \%$ : Minimize the number of false negative patients (patients who would go undetected, hence untreated) “Very High” Band NPA $2 9 5 \%$ : Minimize the number of false positive patients (minimize overdiagnosis, leading to unnecessary treatment)

Threshold setting across all bands also considered other clinical performance metrics such as likelihood ratios, moderate band coverage, outer band coverage, and monotonicity and was executed in an iterative fashion.

Illness Severity Score thresholds were set (tuned) using predicted probabilities and ground truths for 399 patient samples collected from Emergency Department settings. Bacterial and Viral Score thresholds were tuned using predicted probabilities and ground truths for 472 patient samples collected from Emergency Department and ICU studies. All tuning studies were independent from the final clinical study (INF-04), were collected in PAXgene blood tubes, and were processed on the Myrna instrument using GMP-grade TriVerity cartridges. See Table 19 for recommended interpretation of results.

A)

Table 19: Recommendations for Interpretation of TriVerity Results: (A) Bacterial Score, (B) Viral Score, (C) Severity Score   

<table><tr><td rowspan=1 colspan=1>Bacterial Score</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>0-10</td><td rowspan=1 colspan=1>Very Low Score</td><td rowspan=1 colspan=1>Very low bacterial infection likelihood</td></tr><tr><td rowspan=1 colspan=1>11-20</td><td rowspan=1 colspan=1>Low Score</td><td rowspan=1 colspan=1>Low bacterial infection likelihood</td></tr><tr><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>Moderate Score</td><td rowspan=1 colspan=1>Moderate bacterial infection likelihood</td></tr><tr><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>High Score</td><td rowspan=1 colspan=1>High bacterial infection likelihood</td></tr><tr><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>Very High Score</td><td rowspan=1 colspan=1>Very high bacterial infection likelihood</td></tr></table>

B)

<table><tr><td rowspan=1 colspan=1>Viral Score</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>0-10</td><td rowspan=1 colspan=1>Very Low Score</td><td rowspan=1 colspan=1>Very low viral infection likelihood</td></tr><tr><td rowspan=1 colspan=1>11-20</td><td rowspan=1 colspan=1>Low Score</td><td rowspan=1 colspan=1>Low viral infection likelihood</td></tr><tr><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>Moderate Score</td><td rowspan=1 colspan=1>Moderate viral infection likelihood</td></tr><tr><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>High Score</td><td rowspan=1 colspan=1>High viral infection likelihood</td></tr><tr><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>Very High Score</td><td rowspan=1 colspan=1>Very high viral infection likelihood</td></tr></table>

C)

<table><tr><td rowspan=1 colspan=1>Severity Score</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>0-10</td><td rowspan=1 colspan=1>Very Low Score</td><td rowspan=1 colspan=1>Very low 7-day severe illness likelihood</td></tr><tr><td rowspan=1 colspan=1>11-20</td><td rowspan=1 colspan=1>Low Score</td><td rowspan=1 colspan=1>Low 7-day severe illness likelihood</td></tr><tr><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>Moderate Score</td><td rowspan=1 colspan=1>Moderate 7-day severe illness likelihood</td></tr><tr><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>High Score</td><td rowspan=1 colspan=1>High 7-day severe illness likelihood</td></tr><tr><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>Very High Score</td><td rowspan=1 colspan=1>Very High 7-day severe illness likelihood</td></tr></table>

# c. Other Clinical Supporting Data

Secondary and subgroup analyses were performed to demonstrate that accuracy results for the TriVerity test presented above are representative across the entire clinical performance study population and defined patient.

Probabilities of bacterial infection by TriVerity test interpretation band are shown below. Results across the three TriVerity (Bacterial, Viral, and Severity) Score readouts are consistent, indicating a clear relationship between TriVerity scores and increasing likelihoods of bacterial infection, viral infection, and illness severity endpoints across TriVerity interpretation bands. The $80 \%$ CIs for nonadjacent bands do not overlap in any of the analyses.

![](images/54f15bb45b212e2d41bea8888e7b179ca611139f82044f24ccb8cc5e313969a5.jpg)  
Figure 1: Probabilities of Bacterial Infection by TriVerity Interpretation Band: (A) Forced Population, (B) Consensus Population

Note: Band $\ b { 1 = }$ Very Low, Band $2 = 1 0 w$ , Band $\ 3 =$ Moderate, Band4 High, Band ${ 5 = }$ Very High.

![](images/856ff98d05888a03afc6c9577a48ad7c9896934efce89e5f27dae2fa65033c1c.jpg)  
Figure 2: Probabilities of Viral Infection by TriVerity Interpretation Band: (A) Forced Population, (B) Consensus Population

Note: Band $\ b { 1 = }$ Very Low, Band $_ { 2 = }$ Low, Band $\ 3 =$ Moderate, Band4 $\therefore =$ High, Band ${ 5 = }$ Very High.

# Figure 3: Probabilities of Need of Mechanical Ventilation, Vasopressors, and Renal Replacement Therapy Within 7 Days

![](images/449420c5f4047a1e2fd67aa48f77472314dd1a8e69e4276adae9da3f7831507d.jpg)

Note: Band 1=Very Low, Band $2 = \mathsf { L o w }$ , Band $\ 3 =$ Moderate, Band4 $=$ High, Band 5=Very High.

Next, we plotted the likelihood ratios and $80 \%$ CIs for each interpretation band for the bacterial, viral, and severity result readouts of the TriVerity test. Results across the three TriVerity (bacterial, viral, and illness severity) result readouts are consistent, indicating a clear relationship between TriVerity scores and increasing likelihoods of bacterial infection, viral infection, and illness severity across TriVerity interpretation bands. CIs for non-adjacent bands do not overlap in any of the analyses. Of importance, the point estimates of the band likelihood ratios are monotonically increasing, and the $80 \%$ CIs of the likelihood ratios in non-adjacent bands are non-overlapping, for all scores.

![](images/81f45ff397d900c9ee67bc1a7d48da6a11a33298e21071f18d0fc03fc3b3abe0.jpg)  
Figure 4: Likelihood Ratios for Bacterial Infection by TriVerity Interpretation Band: (A) Forced Cohort, (B) Consensus Cohort   
LR, likelihood ratio; LR is calculated as nominal likelihood on a log scale Band $\scriptstyle 1 = 1$ Very Low, Band ${ 2 } = \mathsf { L o w }$ , Band $\ 3 =$ Moderate, Band $4 =$ High, Band ${ 5 = }$ Very High.

![](images/92ffee1c90f37db000b348ceac8f6ba8add69c7cc8b731ca370b66b528a22d01.jpg)  
Figure 5: Likelihood Ratios for Viral Infection by TriVerity Interpretation Band: (A) Forced Cohort, (B) Consensus Cohort   
LR, likelihood ratio; LR is calculated as nominal likelihood on a log scale Band $1 =$ Very Low, Band ${ 2 } = \mathsf { L o w }$ , Band $\ 3 =$ Moderate, Band $4 =$ High, Band ${ 5 = }$ Very High.

# Figure 6: Likelihood Ratios for Need of Mechanical Ventilation, Vasopressors and Renal Replacement Therapy Within 7 Days by TriVerity Interpretation Band

![](images/16834b22e4dab6d2a389fdb4eaa3051609dfd2964465d13baf556e49a13f9e23.jpg)  
LR, likelihood ratio; LR is calculated as nominal likelihood on a log scale Band $1 = ^ { \prime }$ Very Low, Band $_ { 2 = }$ Low, Band $\ 3 =$ Moderate, Band $4 =$ High, Band ${ 5 = }$ Very High.

Of importance, the point estimates of the band likelihood ratios are monotonically increasing, and the $80 \%$ CIs of the likelihood ratios in nonadjacent bands are non-overlapping, for all scores.

The diagnostic accuracy of TriVerity appears superior to that of commonly used diagnostic biomarkers. The AUROC gap between the TriVerity bacterial score and PCT is of major clinical importance and statistically significant. In addition, PCT does not detect non-infectious disease nor severe illness in the ED, whereas TriVerity does.

Table 20: Accuracy of TriVerity Test Results Compared to Biomarkers Expressed as Area Under the Receiver-Operating Characteristics   

<table><tr><td rowspan=1 colspan=2>Result</td><td rowspan=1 colspan=1>ComparatorBiomarker</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>AUROC</td><td rowspan=1 colspan=1>80% CI(lower - upper)</td></tr><tr><td rowspan=1 colspan=1>Population/Endpoint</td><td rowspan=1 colspan=5>Diagnostic Results</td></tr><tr><td rowspan=2 colspan=1>Forced Adjudication</td><td rowspan=2 colspan=1>C-ReactiveProtein</td><td rowspan=1 colspan=1>CRP</td><td rowspan=2 colspan=1>1200</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.71-0.75</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>0.75- 0.78</td></tr><tr><td rowspan=2 colspan=1>Forced Adjudication</td><td rowspan=2 colspan=1>Procalcitonin</td><td rowspan=1 colspan=1>PCT</td><td rowspan=2 colspan=1>1186</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.67-0.71</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>0.75- 0.79</td></tr><tr><td rowspan=2 colspan=1>Forced Adjudication</td><td rowspan=2 colspan=1>White BloodCells</td><td rowspan=1 colspan=1>WBC</td><td rowspan=2 colspan=1>1179</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>0.68 - 0.72</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.75- 0.78</td></tr><tr><td rowspan=2 colspan=1>Consensus Adjudication</td><td rowspan=2 colspan=1>C-ReactiveProtein</td><td rowspan=1 colspan=1>CRP</td><td rowspan=2 colspan=1>714</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.72 - 0.77</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.81- 0.85</td></tr><tr><td rowspan=2 colspan=1>Consensus Adjudication</td><td rowspan=2 colspan=1>Procalcitonin</td><td rowspan=1 colspan=1>PCT</td><td rowspan=2 colspan=1>711</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>0.68- 0.73</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.81- 0.85</td></tr><tr><td rowspan=2 colspan=1>Consensus Adjudication</td><td rowspan=2 colspan=1>White BloodCells</td><td rowspan=1 colspan=1>WBC</td><td rowspan=2 colspan=1>709</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.73-0.78</td></tr><tr><td rowspan=1 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.81 - 0.85</td></tr><tr><td rowspan=1 colspan=1>Population/Endpoint</td><td rowspan=1 colspan=5>Prognostic Results</td></tr><tr><td rowspan=2 colspan=1>Prognostic Population</td><td rowspan=2 colspan=1>Lactate</td><td rowspan=1 colspan=1>Lactate</td><td rowspan=1 colspan=1>1058</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.73 - 0.80</td></tr><tr><td rowspan=1 colspan=1>TriVerity Severity</td><td rowspan=1 colspan=1>1058</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.75-0.80</td></tr><tr><td rowspan=2 colspan=1>Prognostic Population</td><td rowspan=2 colspan=1>qSOFA</td><td rowspan=1 colspan=1>qSOFA</td><td rowspan=1 colspan=1>1042</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.75 - 0.81</td></tr><tr><td rowspan=1 colspan=1>TriVerity Severity</td><td rowspan=1 colspan=1>1042</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.76-0.81</td></tr></table>

Table 21 shows the accuracy of TriVerity bacterial results vs. PCT results stratified by race. TriVerity bacterial results demonstrated markedly superior AUROCs compared to PCT across Whites, African American, and other races using forced and consensus adjudication. AUROCs ranged from $0 . 7 6 - 0 . 8 6$ (forced adjudication) and from 0.82 – 0.91 (consensus adjudication) for the TriVerity bacterial result; for PCT they ranged from 0.55 - 0.73 (forced adjudication) and from 0.62 – 0.74 (consensus adjudication) across different races; ranges were narrow (max. 0.10) for the TriVerity bacterial results whereas they were markedly wider (max. 0.18) for PCT potentially indicating an impact of race on accuracy of PCT.

Table 21: Accuracy of TriVerity Bacterial Scores Compared to Procalcitonin by Race   

<table><tr><td rowspan=1 colspan=1>Population</td><td rowspan=1 colspan=1>Test</td><td rowspan=1 colspan=1>Race</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>AUROC</td><td rowspan=1 colspan=1>Lower80% CI</td><td rowspan=1 colspan=1>Upper80% CI</td></tr><tr><td rowspan=6 colspan=1>ForcedAdjudication</td><td rowspan=3 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>Black or AfricanAmerican</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.79</td></tr><tr><td rowspan=1 colspan=1>Other1</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>0.91</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>758</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=3 colspan=1>Procalcitonin</td><td rowspan=1 colspan=1>Black or AfricanAmerican</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.67</td></tr><tr><td rowspan=1 colspan=1>Other1</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>0.64</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>758</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>0.75</td></tr><tr><td rowspan=3 colspan=1>Consensus</td><td rowspan=3 colspan=1>TriVerity Bacterial</td><td rowspan=1 colspan=1>Black or AfricanAmerican</td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0.87</td></tr><tr><td rowspan=1 colspan=1>Other1</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.96</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>449</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.84</td></tr><tr><td rowspan=3 colspan=1>Adjudication</td><td rowspan=3 colspan=1>Procalcitonin</td><td rowspan=1 colspan=1>Black or AfricanAmerican</td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>0.71</td></tr><tr><td rowspan=1 colspan=1>Other1</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>449</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>0.77</td></tr></table>

1 for the calculation of AUROCs, racial groups with ${ < } 1 5$ participants were merged with “Other”.

These results indicate that TriVerity could substantially improve diagnosis and prognosis of suspected acute infection and sepsis, particularly in minorities where procalcitonin underperforms.

Likelihood ratios and AUROCs for the TriVerity results did not differ markedly based on the presence or absence of immunosuppression (Tables 22-24). Likelihood ratios did not differ markedly between the consensus and forced population. These results indicate that the TriVerity diagnostic results are robust across normal immune status and immunosuppression.

Table 22: Accuracy of Bacterial TriVerity Results by Immunosuppression in the Forced (A) and Consensus (B) Population   

<table><tr><td colspan="2" rowspan="1">A: Forced Adjudication</td><td colspan="4" rowspan="1">Clinically AdjudicatedBacterial Infection</td></tr><tr><td colspan="1" rowspan="1">Population</td><td colspan="1" rowspan="1">TriVerityBacterialBand</td><td colspan="1" rowspan="1">Yesn (%)</td><td colspan="1" rowspan="1">Non (%)</td><td colspan="1" rowspan="1">LikelihoodRatio(80% Cl)</td><td colspan="1" rowspan="1">AUROC</td></tr><tr><td colspan="1" rowspan="5">Immunosuppressed Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">57 (89.06%)</td><td colspan="1" rowspan="1">7 (10.94%)</td><td colspan="1" rowspan="1">4.33(2.83-8.13)</td><td colspan="1" rowspan="5">0.76</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">28 (65.12%)</td><td colspan="1" rowspan="1">15 (34.88%)</td><td colspan="1" rowspan="1">0.99(0.7-1.47)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">30 (68.18%)</td><td colspan="1" rowspan="1">14 (31.82%)</td><td colspan="1" rowspan="1">1.14(0.8-1.73)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">22 (51.16%)</td><td colspan="1" rowspan="1">21 (48.84%)</td><td colspan="1" rowspan="1">0.56(0.39-0.8)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">6 (24.00%)</td><td colspan="1" rowspan="1">19 (76.00%)</td><td colspan="1" rowspan="1">0.17(0.08-0.28)</td></tr><tr><td colspan="1" rowspan="5">Immunocompetent Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">169 (88.95%)</td><td colspan="1" rowspan="1">21 (11.05%)</td><td colspan="1" rowspan="1">5.45(4.2-7.37)</td><td colspan="1" rowspan="5">0.77</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">156 (77.23%)</td><td colspan="1" rowspan="1">46 (22.77%)</td><td colspan="1" rowspan="1">2.3(1.91-2.83)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">112 (57.44%)</td><td colspan="1" rowspan="1">83 (42.56%)</td><td colspan="1" rowspan="1">0.91(0.78-1.09)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">135 (48.04%)</td><td colspan="1" rowspan="1">146 (51.96%)</td><td colspan="1" rowspan="1">0.63(0.55-0.71)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">26 (19.26%)</td><td colspan="1" rowspan="1">109 (80.74%)</td><td colspan="1" rowspan="1">0.16(0.12-0.20)</td></tr><tr><td colspan="2" rowspan="1">B: Consensus Adjudication</td><td colspan="4" rowspan="1">Clinically AdjudicatedBacterial Infection</td></tr><tr><td colspan="1" rowspan="1">Population</td><td colspan="1" rowspan="1">TriVerityBacterialBand</td><td colspan="1" rowspan="1">Yesn (%)</td><td colspan="1" rowspan="1">Non (%)</td><td colspan="1" rowspan="1">LikelihoodRatio(80% CI)</td><td colspan="1" rowspan="1">AUROC</td></tr><tr><td colspan="1" rowspan="5">Immunosuppressed Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">42 (93.33%)</td><td colspan="1" rowspan="1">3 (6.67%)</td><td colspan="1" rowspan="1">5.87(3.42-18.76)</td><td colspan="1" rowspan="5">0.80</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">17 (70.83%)</td><td colspan="1" rowspan="1">7 (29.17%)</td><td colspan="1" rowspan="1">1.02(0.62-1.87)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">18 (75.00%)</td><td colspan="1" rowspan="1">6 (25.00%)</td><td colspan="1" rowspan="1">1.26(0.74-2.43)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">13 (54.17%)</td><td colspan="1" rowspan="1">11 (45.83%)</td><td colspan="1" rowspan="1">0.5(0.31-0.82)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">3 (20.00%)</td><td colspan="1" rowspan="1">12 (80.00%)</td><td colspan="1" rowspan="1">0.1(0.03-0.21)</td></tr><tr><td colspan="1" rowspan="5">Immunocompetent Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">123 (93.18%)</td><td colspan="1" rowspan="1">9 (6.82%)</td><td colspan="1" rowspan="1">8.56(5.88-14.52)</td><td colspan="1" rowspan="5">0.83</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">90 (83.33%)</td><td colspan="1" rowspan="1">18 (16.67%)</td><td colspan="1" rowspan="1">3.13(2.36-4.46)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">72 (64.29%)</td><td colspan="1" rowspan="1">40 (35.71%)</td><td colspan="1" rowspan="1">1.13(0.9-1.45)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">72 (47.06%)</td><td colspan="1" rowspan="1">81 (52.94%)</td><td colspan="1" rowspan="1">0.56(0.47-0.67)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">10 (10.87%)</td><td colspan="1" rowspan="1">82 (89.13%)</td><td colspan="1" rowspan="1">0.08(0.05-0.11)</td></tr></table>

# Table 23: Accuracy of Viral TriVerity Results by Immunosuppression in the Forced (A) and Consensus (B) Population

<table><tr><td colspan="2" rowspan="1">A: Forced Adjudication</td><td colspan="4" rowspan="1">Clinically AdjudicatedViral Infection</td></tr><tr><td colspan="1" rowspan="1">Population</td><td colspan="1" rowspan="1">TriVerityViral Band</td><td colspan="1" rowspan="1">Yesn (%)</td><td colspan="1" rowspan="1">Non (%)</td><td colspan="1" rowspan="1">Likelihood Ratio(80% CI)</td><td colspan="1" rowspan="1">AUROC</td></tr><tr><td colspan="1" rowspan="5">Immunosuppressed Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">21 (84.00%)</td><td colspan="1" rowspan="1">4 (16.00%)</td><td colspan="1" rowspan="1">20.3(11.63-51.16)</td><td colspan="1" rowspan="5">0.81</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">7 (30.43%)</td><td colspan="1" rowspan="1">16 (69.57%)</td><td colspan="1" rowspan="1">1.69(0.92-2.8)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">4 (12.12%)</td><td colspan="1" rowspan="1">29 (87.88%)</td><td colspan="1" rowspan="1">0.53(0.22-0.93)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">6 (10.91%)</td><td colspan="1" rowspan="1">49 (89.09%)</td><td colspan="1" rowspan="1">0.47(0.25-0.74)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">7 (8.43%)</td><td colspan="1" rowspan="1">76 (91.57%)</td><td colspan="1" rowspan="1">0.36(0.2-0.54)</td></tr><tr><td colspan="1" rowspan="5">Immunocompetent Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">107 (85.60%)</td><td colspan="1" rowspan="1">18 (14.40%)</td><td colspan="1" rowspan="1">19.98(15.26-28.76)</td><td colspan="1" rowspan="5">0.84</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">28 (39.44%)</td><td colspan="1" rowspan="1">43 (60.56%)</td><td colspan="1" rowspan="1">2.19(1.61-2.93)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">36 (23.84%)</td><td colspan="1" rowspan="1">115 (76.16%)</td><td colspan="1" rowspan="1">1.05(0.83-1.31)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">35 (12.46%)</td><td colspan="1" rowspan="1">246 (87.54%)</td><td colspan="1" rowspan="1">0.48(0.37-0.58)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">24 (6.40%)</td><td colspan="1" rowspan="1">351 (93.60%)</td><td colspan="1" rowspan="1">0.23(0.17-0.28)</td></tr><tr><td colspan="2" rowspan="1">B: Consensus Adjudication</td><td colspan="4" rowspan="1">Clinically AdjudicatedViral Infection</td></tr><tr><td colspan="1" rowspan="1">Population</td><td colspan="1" rowspan="1">TriVerityViral Band</td><td colspan="1" rowspan="1">Yesn (%)</td><td colspan="1" rowspan="1">Non (%)</td><td colspan="1" rowspan="1">LikelihoodRatio(80% CI)</td><td colspan="1" rowspan="1">AUROC</td></tr><tr><td colspan="1" rowspan="5">Immunosuppressed Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">14 (87.50%)</td><td colspan="1" rowspan="1">2 (12.50%)</td><td colspan="1" rowspan="1">29.96(13.91-nan)</td><td colspan="1" rowspan="5">0.89</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">4 (40.00%)</td><td colspan="1" rowspan="1">6 (60.00%)</td><td colspan="1" rowspan="1">2.85(1.19-6.42)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">2 (10.53%)</td><td colspan="1" rowspan="1">17 (89.47%)</td><td colspan="1" rowspan="1">0.5(0.0-1.05)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3 (9.38%)</td><td colspan="1" rowspan="1">29 (90.63%)</td><td colspan="1" rowspan="1">0.44(0.15-0.79)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">2 (3.64%)</td><td colspan="1" rowspan="1">53 (96.36%)</td><td colspan="1" rowspan="1">0.16(0.0-0.3)</td></tr><tr><td colspan="1" rowspan="5">Immunocompetent Cohort</td><td colspan="1" rowspan="1">Very High</td><td colspan="1" rowspan="1">91 (93.81%)</td><td colspan="1" rowspan="1">6 (6.19%)</td><td colspan="1" rowspan="1">44.4(28.72-89.14)</td><td colspan="1" rowspan="5">0.91</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">21 (43.75%)</td><td colspan="1" rowspan="1">27 (56.25%)</td><td colspan="1" rowspan="1">2.28(1.56-3.26)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">20 (24.39%)</td><td colspan="1" rowspan="1">62 (75.61%)</td><td colspan="1" rowspan="1">0.94(0.66-1.28)</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">14 (9.27%)</td><td colspan="1" rowspan="1">137 (90.73%)</td><td colspan="1" rowspan="1">0.3(0.19-0.4)</td></tr><tr><td colspan="1" rowspan="1">Very Low</td><td colspan="1" rowspan="1">6 (2.74%)</td><td colspan="1" rowspan="1">213 (97.26%)</td><td colspan="1" rowspan="1">0.08(0.04-0.12)</td></tr></table>

<table><tr><td rowspan=1 colspan=2>A: Consensus Adjudication</td><td rowspan=1 colspan=4>Clinically Adjudicated Viral Infection</td></tr><tr><td rowspan=1 colspan=1>Population</td><td rowspan=1 colspan=1>TriVerityViral Band</td><td rowspan=1 colspan=1>Yesn (%)</td><td rowspan=1 colspan=1>Non (%)</td><td rowspan=1 colspan=1>LikelihoodRatio</td><td rowspan=1 colspan=1>AUROC</td></tr><tr><td rowspan=5 colspan=1>Immunosuppressed Cohort</td><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>14 (87.50)</td><td rowspan=1 colspan=1>2 (12.50)</td><td rowspan=1 colspan=1>31.7</td><td rowspan=5 colspan=1>0.90</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>3 (33.33)</td><td rowspan=1 colspan=1>6 (66.67)</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>2 (11.11)</td><td rowspan=1 colspan=1>16 (88.89)</td><td rowspan=1 colspan=1>0.57</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>2 (6.90)</td><td rowspan=1 colspan=1>27 (93.10)</td><td rowspan=1 colspan=1>0.33</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>2 (3.64)</td><td rowspan=1 colspan=1>53 (96.36)</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=5 colspan=1>Immunocompetent Cohort</td><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>91 (93.81)</td><td rowspan=1 colspan=1>6 (6.19)</td><td rowspan=1 colspan=1>43.0</td><td rowspan=5 colspan=1>0.91</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>21 (44.68)</td><td rowspan=1 colspan=1>26 (55.32)</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>19 (23.46)</td><td rowspan=1 colspan=1>62 (76.54)</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>14 (9.66)</td><td rowspan=1 colspan=1>131 (90.34)</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>6 (2.87)</td><td rowspan=1 colspan=1>203 (97.13)</td><td rowspan=1 colspan=1>0.08</td></tr></table>

Proprietary Information Information herein is not to be disclosed without permission from Inflammatix, Inc.

Table 24: Likelihood Ratios for the Severity Interpretation bands in the TriVerity Test for the Prediction of Need for Mechanical Ventilation, Vasopressors, and/or Renal Replacement Therapy within 7 Days Stratified by Immunosuppression   

<table><tr><td rowspan=1 colspan=2>B: Forced Adjudication</td><td rowspan=1 colspan=4>Clinically Adjudicated Viral Infection</td></tr><tr><td rowspan=1 colspan=1>Population</td><td rowspan=1 colspan=1>TriVerityViral Band</td><td rowspan=1 colspan=1>Yesn (%)</td><td rowspan=1 colspan=1>Non (%)</td><td rowspan=1 colspan=1>LikelihoodRatio</td><td rowspan=1 colspan=1>AUROC</td></tr><tr><td rowspan=5 colspan=1>Immunosuppressed Cohort</td><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>21 (84.00)</td><td rowspan=1 colspan=1>4 (16.00)</td><td rowspan=1 colspan=1>20.5</td><td rowspan=5 colspan=1>0.81</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>6 (27.27)</td><td rowspan=1 colspan=1>16 (72.73)</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>4 (12.90)</td><td rowspan=1 colspan=1>27 (87.10)</td><td rowspan=1 colspan=1>0.58</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>5 (9.80)</td><td rowspan=1 colspan=1>46 (90.20)</td><td rowspan=1 colspan=1>0.42</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>7 (8.64)</td><td rowspan=1 colspan=1>74 (91.36)</td><td rowspan=1 colspan=1>0.37</td></tr><tr><td rowspan=5 colspan=1>Immunocompetent Cohort</td><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>107 (85.60)</td><td rowspan=1 colspan=1>18 (14.40)</td><td rowspan=1 colspan=1>19.2</td><td rowspan=5 colspan=1>0.83</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>28 (40.58)</td><td rowspan=1 colspan=1>41 (59.42)</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>34 (23.12)</td><td rowspan=1 colspan=1>113 (76.88)</td><td rowspan=1 colspan=1>0.97</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>35 (12.96)</td><td rowspan=1 colspan=1>235 (87.04)</td><td rowspan=1 colspan=1>0.48</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>24 (6.78)</td><td rowspan=1 colspan=1>330 (93.22)</td><td rowspan=1 colspan=1>0.24</td></tr></table>

<table><tr><td rowspan=1 colspan=2>cYanmaGYellowb</td><td rowspan=1 colspan=4>Need for Mechanical Ventilation, Vasopressors, and/or RRT Within7 Days</td></tr><tr><td rowspan=1 colspan=1>Population</td><td rowspan=1 colspan=1>TriVeritySeverity Band</td><td rowspan=1 colspan=1>Yesn (%)</td><td rowspan=1 colspan=1>Non (%)</td><td rowspan=1 colspan=1>Likelihood Ratio(80% Cl)</td><td rowspan=1 colspan=1>AUROC</td></tr><tr><td rowspan=5 colspan=1>Immunosuppressed Cohort</td><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>5 (50.00%)</td><td rowspan=1 colspan=1>5 (50.00%)</td><td rowspan=1 colspan=1>6.63(2.71-15.89)</td><td rowspan=5 colspan=1>0.75</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>10 (22.73%)</td><td rowspan=1 colspan=1>34 (77.27%)</td><td rowspan=1 colspan=1>1.95(1.28-2.8)</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>7 (15.91%)</td><td rowspan=1 colspan=1>37 (84.09%)</td><td rowspan=1 colspan=1>1.25(0.74-1.89)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>4 (7.02%)</td><td rowspan=1 colspan=1>53 (92.98%)</td><td rowspan=1 colspan=1>0.5(0.19-0.82)</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>1 (1.96%)</td><td rowspan=1 colspan=1>50 (98.04%)</td><td rowspan=1 colspan=1>0.13(0.0-0.34)</td></tr><tr><td rowspan=5 colspan=1>Immunocompetent Cohort</td><td rowspan=1 colspan=1>Very High</td><td rowspan=1 colspan=1>13 (61.90%)</td><td rowspan=1 colspan=1>8 (38.10%)</td><td rowspan=1 colspan=1>14.01(7.87-26.5)</td><td rowspan=5 colspan=1>0.79</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>31 (22.79%)</td><td rowspan=1 colspan=1>105 (77.21%)</td><td rowspan=1 colspan=1>2.55(2.01-3.22)</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>31 (16.76%)</td><td rowspan=1 colspan=1>154 (83.24%)</td><td rowspan=1 colspan=1>1.74(1.38-2.13)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>11 (4.47%)</td><td rowspan=1 colspan=1>235 (95.53%)</td><td rowspan=1 colspan=1>0.4(0.26-0.56)</td></tr><tr><td rowspan=1 colspan=1>Very Low</td><td rowspan=1 colspan=1>9 (2.76%)</td><td rowspan=1 colspan=1>317 (97.24%)</td><td rowspan=1 colspan=1>0.24(0.14-0.35)</td></tr></table>

Table 25 shows pre- and post-test probabilities for the need for mechanical ventilation, vasopressor use and/or RRT within 7 days (primary endpoint) overall and stratified by qSOFA scores of 0-1 (indicating low risk) vs. 2-3 (indicating high risk). The sequential use of the clinical score qSOFA and TriVerity markedly increases the risk stratification for the need of mechanical ventilation, vasopressor use and/or RRT within 7 days compared to qSOFA alone. For both the qSOFA low risk 0-1 score and the qSOFA high risk 2-3 score participants could be further stratified into risk categories.

Table 25: Pre- and Post-Test Probabilities for the Need for Mechanical Ventilation, Vasopressor Use and/or Renal Replacement Therapy within 7 Days Overall and Stratified by qSOFA Scores of 0-1 (Low Risk) or 2-3 (High Risk)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>IllnessSeverityScore</td><td rowspan=1 colspan=3>Need for Mechanical Ventilation, Vasopressors, and/or RRT Within 7Days</td></tr><tr><td rowspan=1 colspan=1>qSOFA Day 0</td><td rowspan=1 colspan=1>NO</td><td rowspan=1 colspan=1>YES</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=5 colspan=1>Low (0-1)N = 960Positivity:74/960(7.71%)</td><td rowspan=1 colspan=1>Very high</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>52.17%</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>14.19%</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>13.11%</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.86%</td></tr><tr><td rowspan=1 colspan=1>Very low</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.02%</td></tr><tr><td rowspan=5 colspan=1>High (2-3)N = 82Positivity:38/82(46.34%)</td><td rowspan=1 colspan=1>Very high</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>71.43%</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>72.73%</td></tr><tr><td rowspan=1 colspan=1>Moderate</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>42.86%</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17.65%</td></tr><tr><td rowspan=1 colspan=1>Very low</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25.00%</td></tr></table>

# d. Expected Values/Reference Range:

Blood samples from 120 self-reported healthy volunteers across various ethnicities and races were tested on the TriVerity test to characterize its performance in samples from healthy subjects. A large portion of bacterial $( 9 0 . 8 \% )$ , viral $( 6 4 . 1 \% )$ and severity $( 9 9 . 2 \% )$ scores fell into the “Low” or “Very Low” interpretation bands. We did not observe marked differences in the bacterial, viral, or severity scores between different races or ethnicities.

Table 26: Bacterial Interpretation Band Range by Race and Ethnicity   

<table><tr><td rowspan=2 colspan=2>Race/ Ethnicity</td><td rowspan=1 colspan=5>TriVerity Bacterial Band</td></tr><tr><td rowspan=1 colspan=1>Very Low(0-10)</td><td rowspan=1 colspan=1>Low(11-20)</td><td rowspan=1 colspan=1>Moderate(21-30)</td><td rowspan=1 colspan=1>High(31-40)</td><td rowspan=1 colspan=1>Very High(41-50)</td></tr><tr><td rowspan=2 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>74.1%</td><td rowspan=1 colspan=1>14.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>51.9%</td><td rowspan=1 colspan=1>44.4%</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>81.5%</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>22.5%</td><td rowspan=1 colspan=1>68.3%</td><td rowspan=1 colspan=1>7.5%</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

Table 27: TriVerity Reference Viral Interpretation Band Range by Race and Ethnicity   

<table><tr><td rowspan=2 colspan=2>Race/ Ethnicity</td><td rowspan=1 colspan=5>TriVerity Viral Band</td></tr><tr><td rowspan=1 colspan=1>Very Low(0-10)</td><td rowspan=1 colspan=1>Low(11-20)</td><td rowspan=1 colspan=1>Moderate(21-30)</td><td rowspan=1 colspan=1>High(31-40)</td><td rowspan=1 colspan=1>Very High(41-50)</td></tr><tr><td rowspan=2 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>22.2%</td><td rowspan=1 colspan=1>40.7%</td><td rowspan=1 colspan=1>29.6%</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>40.7%</td><td rowspan=1 colspan=1>22.2%</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>14.8%</td></tr><tr><td rowspan=2 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>41.7%</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>31.5%</td><td rowspan=1 colspan=1>40.7%</td><td rowspan=1 colspan=1>20.4%</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>23.3%</td><td rowspan=1 colspan=1>40.8%</td><td rowspan=1 colspan=1>24.2%</td><td rowspan=1 colspan=1>7.5%</td><td rowspan=1 colspan=1>4.2%</td></tr></table>

Table 28: TriVerity Reference Severity Interpretation Band Range by Race and Ethnicity   

<table><tr><td rowspan=2 colspan=2>Race/ Ethnicity</td><td rowspan=1 colspan=5>TriVerity Severity Band</td></tr><tr><td rowspan=1 colspan=1>Very Low(0-10)</td><td rowspan=1 colspan=1>Low(11-20)</td><td rowspan=1 colspan=1>Moderate(21-30)</td><td rowspan=1 colspan=1>High(31-40)</td><td rowspan=1 colspan=1>Very High(41-50)</td></tr><tr><td rowspan=2 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>81.5%</td><td rowspan=1 colspan=1>14.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>92.6%</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>86.7%</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

# 13.0 INSTRUMENT NAME

Myrna™

# 14.0 SYSTEM DESCRIPTION

14.1 Modes of Operation:

Quantitative Reverse Transcription Loop-Mediated isothermal Amplification (qRT-LAMP) of mRNA

14.2 Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_ __ or No

14.3 Specimen Identification:

This can be completed by the barcode reader or manually.

14.4 Specimen Sampling and Handling:

The Myrna instrument fully automates and integrates RNA isolation, amplification and result output for each specimen used for the TriVerity test. User intervention is required for collecting the whole blood sample by venipuncture using the PAXgene blood collection tubes within the PAXgene Blood RNA System (K042613). The user then mixes the sample by inversion followed by loading the tube into the cartridge and then inserting the cartridge into the Myrna instrument. For sample stability see (12.1.b) above.

# 14.5 Quality Control:

Each cartridge contains two internal process controls for amplification inhibition, assay reagents, and sample processing effectiveness.

# 15.0 PROPOSED LABELING

The labeling is sufficient, and it satisfies the requirements of 21 CFR Part 809.10.

# 16.0 CONCLUSION

The results of these analytical (non-clinical) and clinical studies demonstrate that TriVerity is substantially equivalent to SeptiCyte RAPID (K203748).