# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K242109   
B Applicant Cepheid   
C Proprietary and Established Names Xpert Xpress CoV-2 plus   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QQX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain 510(k) clearance for the Xpert Xpress CoV-2 plus test for use on the GeneXpert Xpress System.

# B Measurand:

Conserved RNA sequences within the genes encoding the nucleocapsid protein (N), envelope protein (E) and RNA-dependent RNA polymerase protein (RdRP) of SARS-CoV-2 viruses.

# C Type of Test:

A multiplexed, real-time reverse transcriptase polymerase chain reaction (RT-PCR) assay for the qualitative detection and differentiation of SARS-CoV-2 viral RNA from nasopharyngeal swab (NPS) specimens and anterior nasal swab (ANS) specimens using the GeneXpert Instrument Systems platform.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Xpert Xpress CoV-2 plus test, performed on the GeneXpert Xpress System, is a rapid realtime RT-PCR test intended for the qualitative detection of SARS-CoV-2 RNA in nasopharyngeal and anterior nasal swab specimens collected from individuals with signs and symptoms of respiratory tract infection.

The Xpert Xpress CoV-2 plus test is intended for use as an aid in the diagnosis of COVID-19 if used in conjunction with other clinical, epidemiologic, and laboratory findings. Positive results are indicative of the presence of SARS-CoV-2 RNA. Positive results do not rule out bacterial infection or co-infection with other pathogens.

Negative results do not preclude SARS-CoV-2 infection. The results of this test should not be used as the sole basis for diagnosis and patient management decisions.

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only For in vitro diagnostic use only

D Special Instrument Requirements:

GeneXpert Xpress System

# IV Device/System Characteristics:

# A Device Description:

The Xpert Xpress CoV-2 plus test, performed on the GeneXpert Xpress System, is an automated in vitro diagnostic test for the qualitative detection and differentiation of SARS-CoV-2 viral RNA in nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens collected from individuals showing signs and symptoms of respiratory viral infection. The test is performed with three steps: 1) transfer liquid sample to the test cartridge with the transfer pipette (provided in the test kit), 2) run the test on the GeneXpert Xpress System, and 3) read the results.

The Hub configuration of the GeneXpert Xpress System includes a GeneXpert IV Instrument, an integrated barcode scanner, and a computer with a touchscreen and preloaded GeneXpert Xpress software designed for running tests and viewing results. The system requires the use of singleuse disposable cartridges that hold the RT-PCR reagents and host sample purification, nucleic acid amplification, and detection of the target sequences.

The Xpert Xpress CoV-2 plus test cartridge includes all the necessary reagents for the detection of SARS-CoV-2 viral RNA from NPS and ANS specimens collected with nylon flocked swabs and placed into either $3 ~ \mathrm { m L }$ viral transport medium (VTM)/ universal transport medium (UTM) or $2 \mathrm { m L }$ eNAT. With a supplied transfer pipette, eluted NPS and ANS swab specimens are loaded into the sample chamber of a single-use, self-contained Xpert Xpress CoV-2 plus test cartridge. Each Xpert Xpress CoV-2 plus test cartridge contains separate chambers for sample loading, sample processing, and target amplification by real-time RT-PCR, and includes all the reagents necessary to carry out these processes.

The primers and probes in the Xpert Xpress CoV-2 plus test are designed to amplify and detect gene sequences for the following:

• SARS-CoV-2 (envelope small membrane protein [E], a sequence in the N nucleocapsid protein [N2] and RNA-dependent RNA polymerase [RdRP]).

A Sample Processing Control (SPC) and a Probe Check Control (PCC) are also included in the cartridge utilized by the GeneXpert instrument. The SPC is present to control for adequate processing of the sample and to monitor for the presence of potential inhibitor(s) in the RT-PCR reaction. The SPC also ensures that the RT-PCR reaction conditions (temperature and time) are appropriate for the amplification reaction and that the RT-PCR reagents are functioning as expected. The PCC verifies reagent rehydration, PCR tube filling, and confirms that all reaction components are present in the cartridge including monitoring for probe integrity and dye stability. Because the cartridges are self-contained, and specimens never contact working parts of the instrument modules, cross-contamination between samples is minimized. The test cartridge containing the patient sample is loaded onto one of four randomly accessible GeneXpert instrument modules, which performs fully automated and integrated sample preparation and real-time RT-PCR for the Xpert Xpress CoV-2 plus test in approximately 30 minutes.

The Xpert Xpress CoV-2 plus test includes an early assay termination (EAT) function which will provide earlier time to results in high titer specimens if the signal from the target nucleic acid reaches a predetermined threshold before all assay cycles have been completed. When the EAT feature is activated, the earliest time to receive a positive result is calculated to be approximately 19 minutes.

# B Principle of Operation:

The Xpert Xpress CoV-2 plus test is a nucleic acid-based test using real-time RT-PCR. After addition of the specimen to the Sample Chamber of the Xpert Xpress CoV-2 plus test cartridge, the cartridge is loaded onto the GeneXpert Xpress System platform. The instrument then performs automated sample processing including RNA extraction, followed by reverse

transcription, amplification, detection, and reporting of results. The results are interpreted automatically by the GeneXpert System and are shown in the View Results window.

# C Instrument Description Information:

1. Instrument Name:

GeneXpert Xpress System

2. Specimen Identification:

Sample ID is a unique identifier that links the sample being processed to the patient that provided the specimen. The Sample ID can be entered into the GeneXpert Xpress System either by scanning a barcode, entering the Sample ID manually using the virtual keyboard or having the system assign a random Sample ID.

To perform a test, the user selects “NEW TEST” icon from the Home Screen. Either patient information must be entered if configured by an administrator or the Sample ID screen appears. If the Patient Information screen appears, manually enter or scan the Patient ID barcode. If the Sample ID screen appears, scan the Sample ID barcode or manually enter the Sample ID for the patient specimen. The user is then instructed to scan the cartridge barcode and select the test to run. The prepared test cartridge is loaded into an available instrument module with the flashing green light to initiate the test.

# 3. Specimen Sampling and Handling:

The Xpert Xpress CoV-2 plus test cartridge includes reagents for the detection of viral RNA from SARS-CoV-2 in nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens. The specimens are collected with a nylon flocked swab and placed into a transport tube containing $3 ~ \mathrm { m L }$ of viral transport medium (VTM)/Universal Transport Medium (UTM) or $2 \mathrm { m L }$ of eNAT. At the test facility, the operator mixes the specimen by rapidly inverting the collection tube 5 times. A disposable transfer pipette supplied with the Xpert Xpress CoV-2 plus is used to transfer an aliquot of the specimen into the Sample Chamber of the Xpert Xpress CoV-2 plus test cartridge. After closing the cartridge lid, the GeneXpert cartridge is loaded onto the GeneXpert Xpress instrument, which automates and integrates sample preparation, nucleic acid extraction and amplification, and detection of target sequences in simple or complex samples using real-time PCR and RT-PCR assays.

4. Calibration:

Calibration of the GeneXpert Xpress instrument is performed at the factory, and recalibration in the field at customer sites is not required during the initial system startup. A GeneXpert operator or Field Service Engineer with Administrator user permissions can perform calibration checks during annual maintenance.

5. Quality Control:

Internal Controls

The GeneXpert Xpress System relies on the built-in internal controls included in each Xpert Xpress CoV-2 plus test cartridge to ensure the test system is performing as expected. These internal controls consisted of Sample Processing Control (SPC) and Probe Check Control (PCC).

Sample Processing Control (SPC) – The SPC is an encapsidated RNA pseudovirus that ensures adequate lysis of target virus, monitors the presence of PCR inhibitors, and verifies the use of proper PCR conditions. The SPC passes if it meets the validated acceptance criteria. The SPC result should be PASS in a sample that is negative for all three SARS-CoV2 target analytes and can be NEGATIVE or POSITIVE in a sample containing detectable levels of one or more of the SARS-CoV-2 target analytes.

Probe Check Control (PCC) – Before the start of the PCR reaction, the GeneXpert System measures the fluorescence signal from the probes to monitor bead rehydration, reaction tube filling, probe integrity, and dye stability. The PCC passes if it meets the validated acceptance criteria. If any of the PCC conditions fails, the result is reported as NO RESULT – REPEAT TEST and the test must be repeated using a new test cartridge.

# External Controls

External controls (ECs) are not required for the end users to obtain a valid Xpert test result. External controls are not provided with the Xpert Xpress CoV-2 plus test and are available from Zeptometrix. Specifically, the following external controls have been validated for use with the Xpert Xpress CoV-2 plus test:

External Positive Control: NATtrol SARS-Related Coronavirus 2 (SARS-CoV-2); Catalog# NATSARS(COV2)-ERC-IVD   
External Negative Control: NATtrol SARS Associated Coronavirus 2 (SARS-CoV-2); Catalog# NATSARS(COV2)-NEG-IVD

All external controls must be used in accordance with local, state, and federal accrediting organizations, as applicable.

It is recommended that external controls be tested following the manufacturer’s instruction at the frequency noted below.

Each time a new lot of Xpert Xpress CoV-2 plus kits is received. Each time a new shipment of Xpert Xpress CoV-2 plus kits is received even if it is the same lot as previously received. Each time a new operator is performing the test (i.e., operator who has not performed the test before). When problems (storage, operator, instrument, or other) are suspected or identified. If otherwise required by your institution’s standard Quality Control (QC) procedures.

A Predicate Device Name(s): Xpert Xpress CoV-2 plus, performed on the GeneXpert Instrument System   
B Predicate 510(k) Number(s): K230440   
C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K242109</td><td colspan="1" rowspan="1">K230440</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Xpert Xpress CoV-2 plus,Performed on the GeneXpertXpress System</td><td colspan="1" rowspan="1">Xpert Xpress CoV-2 plus,Performed on the GeneXpertInstrument Systems</td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indicationsfor Use</td><td colspan="1" rowspan="1">The Xpert Xpress CoV-2 plus test isintended for use as an aid in thediagnosis of COVID-19 if used inconjunction with other clinical,epidemiologic, and laboratoryfindings. Positive results areindicative of the presence of SARS-CoV-2 RNA. Positive results do notrule out bacterial infection or co-infection with other pathogens.Negative results do not precludeSARS- CoV-2 infection. Theresults of this test should not beused as the sole basis fordiagnosis and patient managementdecisions.</td><td colspan="1" rowspan="1">The Xpert Xpress CoV-2 plus test isintended for use as an aid in thediagnosis of COVID-19 if used inconjunction with other clinical,epidemiologic, and laboratoryfindings. Positive results areindicative of the presence of SARS-CoV-2 RNA. Positive results do notrule out bacterial infection or co-infection with other pathogens.Negative results do not precludeSARS- CoV-2 infection. The resultsof this test should not be used as thesole basis fordiagnosis and patient managementdecisions.</td></tr><tr><td colspan="1" rowspan="1">Regulation Name</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">21 CFR 866.3981Devices to detect and identify nucleicacid targets in respiratory samplesfrom microbial agents that cause theSARS-CoV-2 respiratory infectionand other microbial agents when in amulti-analyte test</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">QQXRespiratory Specimen Nucleic AcidSARS-CoV-2 Test</td></tr><tr><td colspan="1" rowspan="1">Technology/ Detection</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Real-time reverse transcriptionpolymerase chain reaction (RT-qPCR)</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">SARS-CoV-2 (E, N2, RdRP)</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Nasopharyngeal swab (NPS)Anterior nasal swab (ANS)</td></tr><tr><td colspan="1" rowspan="1">Transport Media</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">•  Universal Transport Medium(UTM)/Viral TransportMedium (VTM)•  eNAT</td></tr><tr><td colspan="1" rowspan="1">Test Format</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Single Use</td></tr><tr><td colspan="1" rowspan="1">Automation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Automated Nucleic Acid Extraction,Detection and Results Interpretation</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Non-Determinate ResultsTest</td><td colspan="1" rowspan="1">NO RESULT - REPEAT TESTINSTRUMENT ERROR</td><td colspan="1" rowspan="1">INVALIDERRORNO RESULT</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">•  Sample Processing Control (SPC)•  Probe Check Control (PCC)</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">~30 minutes</td></tr><tr><td colspan="1" rowspan="1">Instrument Systems</td><td colspan="1" rowspan="1">Cepheid GeneXpert Xpress System</td><td colspan="1" rowspan="1">Cepheid GeneXpert InstrumentSystems (Dx and Infinity)</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Consensus Standards

<table><tr><td rowspan=1 colspan=1>FDARecognitionNumber</td><td rowspan=1 colspan=1>DocumentNumber</td><td rowspan=1 colspan=1>Title</td><td rowspan=1 colspan=1>Application ofStandards</td></tr><tr><td rowspan=1 colspan=1>5-125</td><td rowspan=1 colspan=1>ISO 14971:2019</td><td rowspan=1 colspan=1>Medical devices — Application of riskmanagement to medical devices</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>7-233</td><td rowspan=1 colspan=1>CLSI EP17-A2</td><td rowspan=1 colspan=1>Evaluation of Detection Capability for ClinicalLaboratory Measurement Procedures; ApprovedGuideline</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>7-251</td><td rowspan=1 colspan=1>CLSI EP05-A3</td><td rowspan=1 colspan=1>Evaluation of Precision Performance ofQuantitative Measurement Methods; ApprovedGuideline  Third Edition</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>7-275</td><td rowspan=1 colspan=1>CLSI EP07 - A3</td><td rowspan=1 colspan=1>Interference Testing in Clinical Chemistry;Approved Guideline  Third Edition.</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>7-152</td><td rowspan=1 colspan=1>CLSI EP12-A2</td><td rowspan=1 colspan=1>User Protocol for Evaluation of Qualitative TestPerformance; Approved Guideline - SecondEdition</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>19-42</td><td rowspan=1 colspan=1>IEC 61326-1Edition 3.0 2020-10</td><td rowspan=1 colspan=1>Electrical equipment for measurement, control andlaboratory use - EMC requirements - Part 1:General requirements</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>19-43</td><td rowspan=1 colspan=1>IEC 61326-2-6Edition 3.0 2020-10</td><td rowspan=1 colspan=1>Electrical equipment for measurement, control andlaboratory use - EMC requirements - Part 2-6:Particular requirements - In vitro diagnostic (IVD)medical equipment</td><td rowspan=1 colspan=1>General Use</td></tr><tr><td rowspan=1 colspan=1>13-79</td><td rowspan=1 colspan=1>IEC 62304 Edition1.1 2015-06</td><td rowspan=1 colspan=1>Medical device software - Software life cycleprocesses</td><td rowspan=1 colspan=1>Declaration ofConformity</td></tr><tr><td rowspan=1 colspan=1>19-36</td><td rowspan=1 colspan=1>IEC 60601-1-2Edition 4.1 2020-09</td><td rowspan=1 colspan=1>Medical electrical equipment - Part 1-2: Generalrequirements for basic safety and essentialperformance - Collateral Standard:Electromagnetic disturbances - Requirements andtests</td><td rowspan=1 colspan=1>General Use</td></tr></table>

The Xpert Xpress CoV-2 plus test was developed in accordance with the special controls for 21 CFR 866.3981 as detailed in the reclassification order DEN200031.

# Guidance Documents

<table><tr><td rowspan=1 colspan=1>Document</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and Staff  Recommendations for Clinical Laboratory Improvement Amendments of 1988(CLIA) Waiver Applications for Manufacturers of In Vitro Diagnostic Devices, issued on February 26, 2020.</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff  Recommendations for Dual 510(k) and CLIA Waiver by ApplicationStudies, issued February 26, 2020.</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff  Appropriate Use of Voluntary Consensus Standards in PremarketSubmissions for Medical Devices, issued September 14, 2018</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff  Design Considerations and Premarket Submission Recommendations forInteroperable Medical Devices, issued September 6, 2017.</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff, Content of Premarket Submissions for Device Software Functions, suedJune 14, 2023</td></tr><tr><td rowspan=1 colspan=1>Guidance or Industry and FDA Staff, Electromagnetic Compatibility (EMC) of Medical Devices, ssued on June6, 2022</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry  Cybersecurity for Networked Medical Devices Containing Off-the-Shelf (OTS)Software, issued January 14, 2005.</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff, Cybersecurity in Medical Devices: Quality System Considerations andContent of Premarket Submissions, issued on September 27, 2023</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff, General Principles of Software Validation, issued January 11, 2002</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff Electronic Submission Template for Medical Device 510(k) Submission,issued October 2, 2023.</td></tr><tr><td rowspan=1 colspan=1>Guidance for Test Developers and FDA Staff  Policy for Evaluating Impact of Viral Mutations on COVID-19Tests (Revised); issued January 12, 2023.</td></tr><tr><td rowspan=1 colspan=1>Guidance for Industry and FDA Staff  Transition Plan for Medical Devices Issued Emergency UseAuthorizations (EUAs) Related to Coronavirus Disease 2019 (COVID-19), issued March 2023</td></tr></table>

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

The studies conducted to support the analytical performance of the Xpert Xpress CoV-2 plus test, performed on the GeneXpert Instrument Systems, were previously reviewed and described in K230440. In this submission, analytical studies data generated on the GeneXpert Instrument Systems were reanalyzed using the latest 6.4a GeneXpert Xpress software to assess the analytical performance on the GeneXpert Xpress System.

The results from the analysis are described in sections below.

1. Precision / Reproducibility a. Within-laboratory Precision:

Within-laboratory precision was previously evaluated as described in K230440. A total of 281 test results (240 samples and 41 controls) were processed using the GeneXpert Xpress 6.4a software. In all cases, the reanalysis gave the same results as described in K230440.

The reanalysis of the within-laboratory precision study data from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System.

# b. Reproducibility:

A blinded, multi-site reproducibility study was conducted to assess the total variability of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System across operators, study sites, testing days, runs and instruments. The reproducibility study design incorporated contrived panel members as indicated in Table 1.

Table 1. Reproducibility Panel   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Level</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~1.5x LoD</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~3x LoD</td></tr></table>

Panel members were contrived using inactivated NATtrol SARS-CoV-2 (ZeptoMetrix, Buffalo, NY, catalog number NATSARS(COV2)-ERC) in simulated nasal/NP swab matrix. The “Negative” sample did not contain any target microorganism or target RNA. The study was evaluated at three (3) external CLIA-Waived sites with three (3) operators at each site. The study was conducted with one (1) lot of Xpert Xpress CoV-2 plus cartridges over five (5) independent days of testing per operator (may be non-consecutive) with one (1) run per operator per day, with a run consisting of two (2) replicates per panel member. Study results are summarized in Table 2.

Table 2. Summary of Reproducibility Results by Site and Operator   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=4>Site 1</td><td rowspan=1 colspan=4>Site 2</td><td rowspan=1 colspan=4>Site 3</td><td rowspan=2 colspan=1>Agreement(%)</td></tr><tr><td rowspan=1 colspan=1>OP1</td><td rowspan=1 colspan=1>OP2</td><td rowspan=1 colspan=1>OP3</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>OP1</td><td rowspan=1 colspan=1>OP2</td><td rowspan=1 colspan=1>OP3</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>OP1</td><td rowspan=1 colspan=1>OP2</td><td rowspan=1 colspan=1>OP3</td><td rowspan=1 colspan=1>Site</td></tr><tr><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>100%(90/90;95.9-100)</td></tr><tr><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2Low Pos</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>100%(90/90;95.9-100)</td></tr><tr><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2Mod Pos</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>100%(90/90;95.9-100)</td></tr><tr><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30/30</td></tr></table>

$\mathrm { O P = }$ Operator

The Xpert Xpress CoV-2 plus test demonstrated $100 \%$ agreement with expected results for all panel members. The results of the study demonstrate acceptable assay reproducibility for the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System.

Of the 270 samples included in the Reproducibility study, 12 samples yielded nondeterminate results upon initial test. Table 3 shows the list of samples with test results from both initial test and re-test. All non-determinate results were resolved upon retest.

Table 3. Samples with Non-determinate Results   

<table><tr><td rowspan=1 colspan=1>Reason for Failure</td><td rowspan=1 colspan=1>NumberofSamples</td><td rowspan=1 colspan=1>Number ofSamples Resolvedon Retest</td></tr><tr><td rowspan=1 colspan=1>SPC failed</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Module was reset</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Insufficient volume</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Probe check failure</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Cartridge integrity test failure</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td></tr></table>

2. Analytical Reactivity (Inclusivity):

# In silico Analysis of Inclusivity

In silico SARS-CoV-2 analysis was previously reviewed and described in K230440. Additionally, the inclusivity of Xpert Xpress CoV-2 plus was evaluated using in-silico analysis of the assay amplicons and probes against the 2,691,091 SARS-CoV-2 sequences available in the GISAID gene database, for variants that were circulating between August $1 ^ { \mathrm { s t } }$ , 2022 and September $4 ^ { \mathrm { t h } }$ , 2023 for three SARS-CoV-2 gene targets, E, N2 and RdRP. The 2,691,091 SARS-CoV-2 sequences were separated into the lineages of interest based on the Pango Lineage assigned to each genome by GISAID, and those with ambiguous nucleotides were removed. The inclusivity analyses focused on the combined, non-ambiguous sequences from the variants of interest and variants of concern between August $1 ^ { \mathrm { s t } }$ , 2022 and September $4 ^ { \mathrm { t h } }$ , 2023. These constituted 2,519,206 sequences for the E target, 2,551,690 sequences for the N2 target and 2,476,229 sequences for the RdRP target. Table 4 shows the number of sequences in each amplicon alignment and summarizes the effective predicted inclusivity for E, N2 and RdRP amplicons for the variants of interest and concern.

Table 4. Predicted Inclusivity for E, N2 and RdRP Amplicons for SARS-CoV-2 Variants of Interests and Concern   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2Target Amplicon</td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch a</td><td rowspan=1 colspan=1>2 or MoreMismatches</td><td rowspan=1 colspan=1>PredictedInclusivity</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>2,503,319 of 2,519,206 total (99.4%)</td><td rowspan=1 colspan=1>15,694 (0.6%)</td><td rowspan=1 colspan=1>193 (0.007%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>2,495,212 of 2,551,690 total (97.8%)</td><td rowspan=1 colspan=1>55,765 (2.2%)</td><td rowspan=1 colspan=1>713 (0.028%)</td><td rowspan=1 colspan=1>99.95%</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>2,448,754 of 2,476,229 total (98.9%)</td><td rowspan=1 colspan=1>27,322 (1.1%)</td><td rowspan=1 colspan=1>153 (0.006%)</td><td rowspan=1 colspan=1>100%</td></tr></table>

a. Single-nucleotide mismatches are predicted to not impact the performance of the test.

The in-silico inclusivity of the Xpert Xpress CoV-2 plus probe oligonucleotides for E, N2 and RdRP was also assessed against the top 20 most frequent matches in the GISAID EpiCoV sequence database between August $1 ^ { \mathrm { s t } }$ , 2022 and September $4 ^ { \mathrm { t h } }$ , 2023, which constituted 2,519,206 for the E target, 2,551,690 for the N2 target and 2,476,229 for the

RdRP target. For each of the probe oligonucleotides used in the Xpert Xpress CoV-2 plus test, Table 5 summarizes the number sequences as well as the corresponding percentage of sequences from this dataset with exact match, 1 mismatch/insertion, and 2 or more mismatches/insertions in the alignment.

Table 5. Predicted Inclusivity for E, N2 and RdRP Probes for SARS-CoV-2 Variants of Interests and Concern   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2TargetProbe</td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch/Insertion a</td><td rowspan=1 colspan=1>2 or MoreMismatches/Insertions</td><td rowspan=1 colspan=1>PredictedInclusivity</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>2,516,153 of 2,519,206 total (99.9%)</td><td rowspan=1 colspan=1>2997(0.1%)</td><td rowspan=1 colspan=1>0(0.0%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>2,528,096 of 2,551,690 total (99.1%)</td><td rowspan=1 colspan=1>12728(0.5%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>99.6%</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2,465,482 of2,476,229 total(99.6%)</td><td rowspan=1 colspan=1>10418(0.4%)</td><td rowspan=1 colspan=1>99.6%</td></tr></table>

a. Single-nucleotide mismatches are predicted to not impact the performance of the test.

Based on the built-in redundancy of the Xpert Xpress CoV-2 plus test’s SARS-CoV-2 amplification system (i.e., three independent targets, only one of three must be detected to assign a positive result), all evaluated sequences are predicted to be detected by the Xpert Xpress CoV-2 plus test.

# Wet-Testing

Inclusivity wet-testing was previously evaluated as described in K230440. Two hundred twenty-seven (227) runs were previously evaluated for the inclusivity study for K230440. Out of the 227 runs, 20 tests were not able to be reanalyzed due to software incompatibility. Two hundred and seven tests from the analytical inclusivity study were reanalyzed. There were 206 valid results and one non-determinate result. All 206 valid tests gave the exact same test results after reanalysis. The number of non-determinate (ND) GeneXpert results was unchanged after reanalysis and the ND test results descriptions remained the same.

The reanalysis of the data from the inclusivity study from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System

# 3. Analytical Specificity/Interference

Analytical Specificity (Cross-Reactivity)

# Cross-Reactivity Wet-Testing

Analytical specificity (cross-reactivity) was previously evaluated as described in K230440. One hundred twenty-five (125) tests from the analytical specificity study were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 125 valid results and zero (0) non-determinate (ND) results. All 125 valid tests gave exactly the same test results after reanalysis.

The reanalysis of the analytical specificity (cross-reactivity) study data from K230440 demonstrates equivalent performance of the Xpert Xpress Cov-2 plus test on the GeneXpert Xpress System.

# Microbial Interference

Microbial interference was previously evaluated as described in K230440. Three hundred twenty (320) tests were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 312 valid results and eight (8) non-determinate results. All 312 valid tests gave the exact same test results after reanalysis. The number of non-determinate (ND) GeneXpert results was unchanged after reanalysis. The ND test results descriptions remained the same.

The reanalysis of the analytical microbial interference study data from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System.

# Potentially Interfering Substances

The impact of potentially interfering substance on the Xpert Xpress CoV-2 plus test was previously evaluated as described in K230440. Four hundred twenty-four (424) tests from the potentially interfering substances study were reanalyzed using the latest $6 . 4 \mathrm { a }$ GeneXpert Xpress software. There were 421 valid results and three (3) non-determinate results. All 421 valid tests gave the exact same test results after reanalysis. The number of non-determinate GeneXpert results was unchanged after reanalysis and the ND test results descriptions remained the same.

The reanalysis of the data from the interfering substances study from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System.

4. Assay Reportable Range: Not Applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a. Controls

# External Control Evaluation

Two hundred and fifty (250) tests from the external control evaluation study in K230440 were reanalyzed. Of the 250 runs tested there were 248 valid tests that gave the exact same test results after reanalysis. The number of non-determinate test results remained unchanged.

The reanalysis of the data from the external control validation study from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System

b. Specimen Stability NPS in UTM/VTM Matrix

Two hundred ninety-two (292) tests from the NPS Specimen Stability in the UTM/VTM Matrix Study in K230440 were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 290 valid results and two (2) non-determinate results. All 290 valid tests gave the exact same test results after reanalysis while the number of non-determinate results was unchanged.

This reanalysis supports specimen storage conditions in NPS UTM/VTM matrix at refrigerated $( 2 \mathrm { - } 8 ^ { \mathrm { ~ \circ } } \mathrm { C } )$ temperatures for up to seven (7) days and at room temperature (15- $3 0 ^ { \circ } \mathrm { C } )$ ) for up to 48 hours until testing is performed on the GeneXpert Xpress Systems.

# NPS in eNAT Matrix

Two hundred ninety-three (293) tests from the NPS Specimen Stability in eNAT Matrix Study in K230440 were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 290 valid results and three (3) non-determinate results. All 290 valid tests gave the exact same test results after reanalysis, the number of non-determinate results remained unchanged. The ND test results descriptions remained the same after reanalysis.

This reanalysis supports specimen storage conditions in NPS eNAT matrix at refrigerated $( 2 - 8 ^ { \circ } \mathrm { C } )$ temperatures for up to seven (7) days and at room temperature $( 1 5 - 3 0 ^ { \circ } \mathrm { C } )$ for up to 48 hours until testing is performed on the GeneXpert Xpress Systems.

# c. Specimen Stability – Freeze-Thaw

Three hundred seventy-four (374) tests from the NPS-UTM/VTM and NPS-eNAT specimens freeze/thaw study conducted at $- 8 0 \mathrm { { } ^ { \circ } C }$ in K230440 were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 366 valid results and eight nondeterminate (ND) results. All 366 valid tests gave the exact same test results after reanalysis. The number of non-determinate GeneXpert results was unchanged after reanalysis. The ND test results descriptions remained the same.

This reanalysis supports storage and freeze-thaw stability claims for NPS and NS samples collected in VTM and eNAT that can undergo one cycle of freeze-thaw (- $8 0 ^ { \circ } \mathrm { C }$ ) before testing with the Xpert Xpress CoV-2 plus test.

# d. Prepared Cartridge Hold Time

Two hundred and sixty-six (266) tests from the cartridge hold time study in K230440 were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 258 valid results and eight non-determinate results. All 258 valid tests gave the exact same test results after reanalysis and the number of non-determinate GeneXpert results was unchanged. The ND test results descriptions remained the same.

The reanalysis of the data from the cartridge hold time study from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System and supports a cartridge hold time of 4.5 hours.

6. Detection Limit:

# LoD in Clinical NPS in UTM/VTM and eNAT Matrix

The LoD in clinical NPS in UTM/VTM was previously established as described in K230440. Eight hundred fifty-two (852) tests from the LoD study for clinical NPS in UTM/VTM matrix and in eNAT were reanalyzed using the latest 6.4a GeneXpert Xpress software. There were 836 valid tests and 16 non-determinate results (ND). All 836 valid tests gave the exact same test results after the reanalysis. The number of non-determinate GeneXpert results was unchanged after reanalysis and the ND descriptions remained the same.

The reanalysis of the data from the LoD in clinical NPS in UTM/VTM matrix studies from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System.

The LoD concentrations and mean Ct values at the verified LoDs for clinical NPS in UTM/VTM or in eNAT specimens tested using two reagent lots are shown in Table 6.

Table 6. Results for Verification of SARS-CoV-2 LoD in Clinical NPS-UTM/VTM and Clinical NPS-eNAT   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Reagent Lot</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>LoD(copies/mL)aa</td><td rowspan=1 colspan=1>PositiveResults out of# ValidReplicates</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>Mean ECt</td><td rowspan=1 colspan=1>MeanN2 Ct</td><td rowspan=1 colspan=1>MeanRdRPCt</td></tr><tr><td rowspan=4 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=2 colspan=1>Clinical NPS-UTM/VTM</td><td rowspan=2 colspan=1>403</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>36.6</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>36.5</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=2 colspan=1>Clinical NPS-eNAT</td><td rowspan=2 colspan=1>403</td><td rowspan=1 colspan=1>20/20b</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>35.5</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>35.5</td></tr></table>

a Genomic copies/ mL b One of 20 replicates reported INVALID. The run was successfully repeated to obtain 20 valid replicates.

# LoD in Clinical ANS in UTM/VTM Matrix

The LoD in clinical ANS in UTM/VTM was previously established as described in K230440. Six hundred sixty-six (666) tests from the LoD in clinical NS in UTM/VTM matrix were reanalyzed. There were 652 valid results and 14 non-determinate (ND) results. All 652 valid tests gave the exact same test results after reanalysis and the number of ND results was unchanged. The ND test result descriptions remained the same.

The LoD concentrations and mean Ct values at the verified LoDs for clinical ANS in UTM/VTM specimens tested using two reagent lots are shown in Table 7.

Table 7. Results for Verification of SARS-CoV-2 LoD in Clinical ANS-UTM/VTM   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Reagent Lot</td><td rowspan=1 colspan=1>LoD(copies/mL)</td><td rowspan=1 colspan=1>PositiveResults outof # ofValidReplicates</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MeanE Ct</td><td rowspan=1 colspan=1>MeanN2 Ct</td><td rowspan=1 colspan=1>MeanRdRP Ct</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>36.8</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>36.1</td></tr></table>

# LoD in Clinical ANS Matrix – WHO Study

One hundred twenty-two (122) tests from the LoD in clinical ANS matrix study testing the $1 ^ { \mathrm { s t } }$ WHO International Standard for SARS-CoV-2 RNA Virus in K230440 were reanalyzed. There were 116 valid results and six (6) non-determinate results. All 116 valid tests gave the exact same test results after reanalysis and the number of non-determinate GeneXpert results was unchanged after reanalysis. The ND test results descriptions remained the same. Table 8 presents the Mean Ct values for gene targets within the WHO international standard for the SARS-CoV-2 virus.

Table 8. LoD Concentration and Mean Ct Values for E, N2 and RdRP for the $\mathbf { 1 ^ { s t } }$ WHO International Standard for SARS-CoV-2 RNA Virus in Clinical NS-UTM/VTM Matrix   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>LoDConcentration</td><td rowspan=1 colspan=1>MeanE Ct</td><td rowspan=1 colspan=1>MeanN2 Ct</td><td rowspan=1 colspan=1>MeanRdRP Ct</td></tr><tr><td rowspan=1 colspan=1>1st WHOInternationalStandard for SARS-CoV-2 RNA Virus</td><td rowspan=1 colspan=1>England/02/2020isolate ofSARS-CoV-2</td><td rowspan=1 colspan=1>1000 IU/mL(3.0 Log10 IU/mL)</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>36.5</td></tr></table>

# 7. Assay Cut-Off:

The Xpert Xpress CoV-2 plus test includes defined cycle threshold $\mathrm { ( C t ) }$ ranges for the SARSCoV-2 targets and SPC. The Ct cut-offs are hard coded values and are included as automatic calculations in the assay definition file (ADF) of the Xpert Xpress CoV-2 plus test. The cutoff values are identical to those reported in K230440.

8. Accuracy (Instrument): Not Applicable

9. Carry-Over:

Ninety (90) tests from the carry-over contamination study in K230440 were reanalyzed. There were 88 valid results and two (2) non-determinate (ND) results. All 88 valid tests gave the exact same test results after reanalysis with GeneXpert Xpress software version 6.4a. The number of ND GeneXpert results was unchanged after reanalysis. The ND test results descriptions remained the same.

The reanalysis of the data from the carry-over contamination study from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System.

Equivalent Xpert Xpress CoV-2 plus performance for the detection of viral targets (SARSCoV-2 – E, N2, RdRP) between clinical ANS-UTM/VTM matrix, clinical NPS- UTM/VTM matrix and the simulated NPS/ANS UTM/VTM matrix was previously evaluated in K230440.

Five hundred thirteen (513) tests from the specimen matrices equivalency study in K230440 were reanalyzed. There were 503 valid results and ten (10) non-determinate (ND) results. All 503 valid tests gave the exact same test results after reanalysis with GeneXpert Xpress software version 6.4a. The number of non-determinate GeneXpert results remained unchanged after reanalysis and the ND test result descriptions remained the same.

The reanalysis of the data from the specimen matrices equivalency study with clinical NPS and clinical ANS in UTM/VTM, and simulated NPS/ANS in UTM/VTM matrices from K230440 demonstrates equivalent performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System when testing natural NPS or ANS collected in UTM/VTM matrix and simulated NPS/ANS in UTM/VTM matrices.

# B Clinical Studies:

1. Clinical Sensitivity:

# Prospective Clinical Study

The performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System was established in a multi-site prospective clinical study that included 22 geographically diverse locations in the United States in specimens collected from individuals showing signs and symptoms of respiratory infection. Of the 22 sites, 21 sites performed GeneXpert Xpress System testing and specimen collection while one (1) site performed comparator and discrepant investigation.

Prospectively collected, fresh nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens in UTM/VTM, from individuals with signs and symptoms of respiratory viral infection, were enrolled from January 24, 2022 to May 16, 2022 in the US in an all-comers fashion. Xpert Xpress CoV-2 plus testing on the GeneXpert Xpress System was performed from January 24, 2022 to May 16, 2022, at 21 testing sites.

There were 2200 prospective subjects/ specimens from symptomatic individuals enrolled for evaluating the performance of the Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System (1094 NPS and 1106 ANS). Of these, ten (10) subjects/ specimens (five (5) NPS and five (5) ANS) were found to be ineligible as not meeting inclusion/exclusion criteria or having incomplete Informed Consent leaving 2190 eligible samples (1089 NPS and 1101 ANS). From these specimens, an additional 256 (128 NPS and 128 ANS) were excluded due to protocol deviations leaving 1934 subjects/ specimens (961 NPS and 973 ANS) in the final dataset. There were an additional 151 specimens excluded $\mathrm { ( N P S = 7 8 }$ and $\mathbf { A N S } = 7 3$ ) due to: unresolved repeat ND for Xpert testing $\mathbf { \tilde { n } = }$ nine (9), unresolved repeat ND and procedural deviation with comparator testing ( $\mathbf { \tilde { n } = }$ two (2) invalid and 138 non-evaluable comparator results $( \mathtt { n } = 1 4 0 )$ . There 1783 samples/ specimens eligible for performance evaluation $( { \mathrm { N P S } } =$ 883 and $\mathbf { A N S } = 9 0 0$ ). Table 9 provides demographic data for the prospective fresh specimens available for performance analyses .

Table 9. Clinical Study Demographics   

<table><tr><td rowspan=1 colspan=1>Prospectively Collected FreshSpecimens from 2022 (Category I)</td><td rowspan=1 colspan=1>NPS(N=961)</td><td rowspan=1 colspan=1>ANS(N=973)</td><td rowspan=1 colspan=1>Overall(N=1934)</td></tr><tr><td rowspan=1 colspan=4>Gender</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>569 (59.2%)</td><td rowspan=1 colspan=1>616 (63.3%)</td><td rowspan=1 colspan=1>1185 (61.3%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>392 (40.8%)</td><td rowspan=1 colspan=1>357 (36.7%)</td><td rowspan=1 colspan=1>749 (38.7%)</td></tr><tr><td rowspan=1 colspan=4>Age Group (Years)</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>8 (0.8%)</td><td rowspan=1 colspan=1>54 (5.5%)</td><td rowspan=1 colspan=1>62 (3.2%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>275 (28.6%)</td><td rowspan=1 colspan=1>285 (29.3%)</td><td rowspan=1 colspan=1>560 (29.0%)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>537 (55.9%)</td><td rowspan=1 colspan=1>505 (51.9%)</td><td rowspan=1 colspan=1>1042 (53.9%)</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>141 (14.7%)</td><td rowspan=1 colspan=1>129 (13.3%)</td><td rowspan=1 colspan=1>270 (14.0%)</td></tr><tr><td rowspan=1 colspan=4>Race</td></tr><tr><td rowspan=1 colspan=1>American Indian or Alaska Native</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>1 (0.1%)</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>27 (2.8%)</td><td rowspan=1 colspan=1>25 (2.6%)</td><td rowspan=1 colspan=1>52 (2.7%)</td></tr><tr><td rowspan=1 colspan=1>Black or African American</td><td rowspan=1 colspan=1>225 (23.4%)</td><td rowspan=1 colspan=1>217 (22.3%)</td><td rowspan=1 colspan=1>442 (22.9%)</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>656 (68.3%)</td><td rowspan=1 colspan=1>661 (67.9%)</td><td rowspan=1 colspan=1>1317 (68.1%)</td></tr><tr><td rowspan=1 colspan=1>Black or African American, White</td><td rowspan=1 colspan=1>4 (0.4%)</td><td rowspan=1 colspan=1>4 (0.4%)</td><td rowspan=1 colspan=1>8 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>Other Mixed (each N≤5)</td><td rowspan=1 colspan=1>3 (0.3%)</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>5 (0.3%)</td></tr><tr><td rowspan=1 colspan=1>Participant Declined to Answer orUnknown</td><td rowspan=1 colspan=1>46 (4.8%)</td><td rowspan=1 colspan=1>63 (6.5%)</td><td rowspan=1 colspan=1>109 (5.6%)</td></tr><tr><td rowspan=1 colspan=2>Ethnicity</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>75 (7.8%)</td><td rowspan=1 colspan=1>81 (8.3%)</td><td rowspan=1 colspan=1>156 (8.1%)</td></tr><tr><td rowspan=1 colspan=1>Non-Hispanic</td><td rowspan=1 colspan=1>861 (89.6%)</td><td rowspan=1 colspan=1>857 (88.1%)</td><td rowspan=1 colspan=1>1718 (88.8%)</td></tr><tr><td rowspan=1 colspan=1>Participant Declined to Answer orUnknown</td><td rowspan=1 colspan=1>25 (2.6%)</td><td rowspan=1 colspan=1>35 (3.6%)</td><td rowspan=1 colspan=1>60 (3.1%)</td></tr><tr><td rowspan=1 colspan=4>COVID-19 Vaccination Status</td></tr><tr><td rowspan=1 colspan=1>Vaccinated</td><td rowspan=1 colspan=1>721 (75.0%)</td><td rowspan=1 colspan=1>717 (73.7%)</td><td rowspan=1 colspan=1>1438 (74.4%)</td></tr><tr><td rowspan=1 colspan=1>Not Vaccinated</td><td rowspan=1 colspan=1>222 (23.1%)</td><td rowspan=1 colspan=1>239 (24.6%)</td><td rowspan=1 colspan=1>461 (23.8%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>18 (1.9%)</td><td rowspan=1 colspan=1>17 (1.7%)</td><td rowspan=1 colspan=1>35 (1.8%)</td></tr></table>

The Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System was evaluated by comparing to a U.S. FDA-cleared molecular respiratory panel that includes SARS-CoV-2. Positive Percent Agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the Xpert Xpress CoV-2 plus and the comparator method had a positive result, and false negative (FN) indicates that the Xpert Xpress CoV-2 plus was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathrm { T N / \left( T N + F P \right) } \right)$ . True negative (TN) indicates that both the Xpert Xpress CoV-2 plus and the comparator method had negative results, and false positive (FP)

indicates that the Xpert Xpress CoV-2 plus was positive while the comparator result was negative. Specimens that obtained discordant SARS-CoV-2 results underwent additional testing with a U.S. FDA EUA SARS-CoV-2 molecular test.

A summary of the Xpert Xpress CoV-2/ plus test on the GeneXpert Xpress System prospective clinical study performance is provided in Table 10 for both NPS and ANS specimens.

Table 10. Xpert Xpress CoV-2 plus test on the GeneXpert Xpress System Prospective Clinical Performance (Category I/Fresh NPS and ANS specimens)   

<table><tr><td>Analyte</td><td>Number of Specimens</td><td>True Positives</td><td>False Negatives</td><td>False Positive</td><td>True Negative</td><td>PPA (%)</td><td>95% CI</td><td>NPA (%)</td><td>95% CI</td></tr><tr><td rowspan="3">SARS-</td><td colspan="8"></td></tr><tr><td>883</td><td>112</td><td>2</td><td>7</td><td>NPS Samples 762</td><td>98.2</td><td>93.8-99.5</td><td>99.1</td><td>98.1-99.6</td></tr><tr><td></td><td></td><td></td><td></td><td>ANS Samples</td><td></td><td></td><td></td><td></td></tr><tr><td>CoV-2</td><td>900</td><td>103</td><td>1</td><td>7</td><td>789</td><td>99.0</td><td>94.8-99.8</td><td>99.1</td><td>98.2-99.6</td></tr></table>

a. Discrepant test results based on U.S. FDA EUA SARS-CoV-2 molecular test: 1/14 SARS-CoV-2 positive; 12/14 SARS-CoV-2 negative; 1/14 invalid. b. Discrepant test results based on U.S. FDA EUA SARS-CoV-2 molecular test: 1/3 SARS-CoV-2 positive; 2/3 SARS-CoV-2 negative.

# Non-determinate (ND) rate

Of the total 1934 prospective specimens evaluated, 81 specimens yielded an initial ND result. After repeat testing, 70 specimens yielded a valid result such that the initial ND rate was $4 . 2 \%$ (81/1934) and final ND rate was $0 . 6 \%$ (11/1934).

Of the 961 NPS specimens evaluated, 39 specimens yielded an initial ND result. After repeat testing, 32 specimens yielded a valid result. The initial ND rate using NPS specimens was $4 . 1 \%$ (39/961) and final ND rate was $0 . 7 \%$ (7/961).

Of the 973 ANS specimens evaluated from the symptomatic population, 42 specimens yielded an initial ND result. After repeat testing, 38 specimens yielded a valid result. The initial ND rate using ANS specimens was $4 . 3 \%$ (42/973) and final ND rate was $0 . 4 \%$ (4/973).

2. Clinical Specificity: See Section “Clinical Sensitivity” above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not Applicable.

# C Clinical Cut-Off:

Not applicable

# D Expected Values/Reference Range:

Expected values as determined by the Xpert Xpress CoV-2 plus test during the prospective clinical study, stratified by specimen type, gender, age, and vaccination status are presented in Table 11.

Table 11. Xpert Xpress CoV-2 plus Positivity Rates Stratified by Specimen Type, Gender, Age and Vaccination Status   

<table><tr><td rowspan=2 colspan=1>Subgroup</td><td rowspan=2 colspan=1>Category</td><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=2>Black or AfricanAmerican</td><td rowspan=1 colspan=3>White</td><td rowspan=2 colspan=1>Awnn iranOANa</td><td rowspan=2 colspan=1>Aran</td><td rowspan=2 colspan=1>Ori ae &#x27;urterg</td><td rowspan=2 colspan=1>pte nie</td><td rowspan=2 colspan=1>PerddPe pennMo Marunoumun</td></tr><tr><td rowspan=1 colspan=1>Hhrstce</td><td rowspan=1 colspan=1>Hurd!ce-uoN</td><td rowspan=1 colspan=1>Hhrs!ce</td><td rowspan=1 colspan=1>Herdstc-uoN</td><td rowspan=1 colspan=1>uoumn</td></tr><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>13.5%(119/883)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>10.8%(21/195)</td><td rowspan=1 colspan=1>9.5%(6/63)</td><td rowspan=1 colspan=1>15%(82/546)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>4.2%(1/24)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>17.8%(8/45)</td></tr><tr><td rowspan=1 colspan=1>ANS</td><td rowspan=1 colspan=1>12.2%(110/900)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>9.5%(19/200)</td><td rowspan=1 colspan=1>5.6%(4/71)</td><td rowspan=1 colspan=1>13.6%(73/537)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>8.7%(2/23)</td><td rowspan=1 colspan=1>0%(0/4)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>19.4%(12/62)</td></tr><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>12.9%(141/1097)</td><td rowspan=1 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>11.1%(29/262)</td><td rowspan=1 colspan=1>3.4%(3/87)</td><td rowspan=1 colspan=1>14.4%(94/653)</td><td rowspan=1 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>7.7%(2/26)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>100%(1/1)</td><td rowspan=1 colspan=1>20.0%(12/60)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>12.8%(88/686)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>8.3%(11/133)</td><td rowspan=1 colspan=1>14.9%(7/47)</td><td rowspan=1 colspan=1>14.2%(61/430)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>4.8%(1/21)</td><td rowspan=1 colspan=1>0%(0/4)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>17.0%(8/47)</td></tr><tr><td rowspan=5 colspan=1>AgeGroup(Years)</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>6.6% (4/61)</td><td rowspan=1 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>4.2%(1/24)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>18.2%(2/11)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>5.0%(1/20)</td></tr><tr><td rowspan=1 colspan=1>6 -21</td><td rowspan=1 colspan=1>9%(46/509)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>8.2%(12/146)</td><td rowspan=1 colspan=1>4.9%(3/61)</td><td rowspan=1 colspan=1>9.3%(22/237)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>16.7%(2/12)</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>14.0%(6/43)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>13.7%(132/962)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>12.1%(25/206)</td><td rowspan=1 colspan=1>7.4%(5/68)</td><td rowspan=1 colspan=1>14.6%(90/617)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>3.3%(1/30)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>29.7%(11/37)</td></tr><tr><td rowspan=2 colspan=1>≥ 60</td><td rowspan=2 colspan=1>18.7%(47/251)</td><td rowspan=2 colspan=1>NA(0/0)</td><td rowspan=2 colspan=1>10.5%(2/19)</td><td rowspan=2 colspan=1>40%(2/5)</td><td rowspan=2 colspan=1>18.8%(41/218)</td><td rowspan=2 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>NA</td><td rowspan=2 colspan=1>0%(0/2)</td><td rowspan=2 colspan=1>NA(0/0)</td><td rowspan=2 colspan=1>NA(0/0)</td><td rowspan=2 colspan=1>28.6%(2/7)</td></tr><tr><td rowspan=1 colspan=1>(0/0)</td></tr><tr><td rowspan=3 colspan=1>VaccinationStatus</td><td rowspan=1 colspan=1>Vaccinated</td><td rowspan=1 colspan=1>13.4%(179/1334)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>11.6%(29/249)</td><td rowspan=1 colspan=1>7.8%(8/102)</td><td rowspan=1 colspan=1>14.4%(124/860)</td><td rowspan=1 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>0%(0/1)</td><td rowspan=1 colspan=1>7.5%(3/40)</td><td rowspan=1 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>25%(1/4)</td><td rowspan=1 colspan=1>19.2%(14/73)</td></tr><tr><td rowspan=1 colspan=1>NotVaccinated</td><td rowspan=1 colspan=1>11.0%(46/417)</td><td rowspan=1 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>7.7%(11/143)</td><td rowspan=1 colspan=1>6.2%(2/32)</td><td rowspan=1 colspan=1>14.4%(30/209)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>0%(0/4)</td><td rowspan=1 colspan=1>0%(0/5)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>13.6%(3/22)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>12.5%(4/32)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>7.1% (1/14)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>0%(0/3)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>NA(0/0)</td><td rowspan=1 colspan=1>25.0%(3/12)</td></tr><tr><td rowspan=2 colspan=2>Overall</td><td rowspan=2 colspan=1>12.8%(229/1783)</td><td rowspan=2 colspan=1>0%(0/3)</td><td rowspan=2 colspan=1>10.1%(40/395)</td><td rowspan=2 colspan=1>7.5%(10/134)</td><td rowspan=2 colspan=1>14.3%(155/1083)</td><td rowspan=2 colspan=1>0%(0/2)</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>6.4%</td><td rowspan=2 colspan=1>0%(0/7)</td><td rowspan=2 colspan=1>25.0%(1/4)</td><td rowspan=2 colspan=1>18.7%(20/107)</td></tr><tr><td rowspan=1 colspan=1>(0/1)</td><td rowspan=1 colspan=1>(3/47)</td></tr></table>

# E Other Supportive Instrument Performance Characteristics Data:

Not Applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.