# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

# C Proprietary and Established Names

QIAstat-Dx Meningitis/Encephalitis (ME) Panel

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PLO</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3970 - DeviceTo Detect And IdentifyMicrobial PathogenNucleic Acids InCerebrospinal Fluid</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission:

The purpose of this submission is to obtain substantial equivalence determination for the QIAstat-Dx ME Panel.

# B Measurand:

Enterovirus, Escherichia coli K1, Haemophilus influenzae, Listeria monocytogenes, Neisseria meningitidis (encapsulated), Streptococcus agalactiae, Streptococcus pneumoniae, Streptococcus pyogenes, and Cryptococcus neoformans/gattii nucleic acid target sequences

# C Type of Test:

A multiplexed nucleic acid real-time PCR test for use with the QIAstat-Dx Analyzer 1.0 instrument for the qualitative in vitro detection and identification of nucleic acid from bacteria, yeast, and viruses in a cerebrospinal fluid sample

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The QIAstat-Dx Meningitis/Encephalitis (ME) Panel is a qualitative multiplexed nucleic acid real-time PCR based in vitro diagnostic test intended for use with the QIAstat-Dx Analyzer 1.0. The QIAstat-Dx ME Panel is capable of simultaneous detection and identification of multiple bacterial, viral, and yeast nucleic acids from cerebrospinal fluid (CSF) specimens obtained via lumbar puncture from individuals with signs and/or symptoms of meningitis and/or encephalitis.

The following organisms are identified using the QIAstat-Dx ME Panel: Enterovirus, Escherichia coli K1, Haemophilus influenzae, Listeria monocytogenes, Neisseria meningitidis (encapsulated), Streptococcus agalactiae, Streptococcus pneumoniae, Streptococcus pyogenes, and Cryptococcus neoformans/gattii\*.

The QIAstat-Dx ME Panel is indicated as an aid in the diagnosis of specific agents of meningitis and/or encephalitis and results must be used in conjunction with other clinical, epidemiological, and laboratory data. Results from the QIAstat-Dx ME Panel are not intended to be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with organisms not included in the QIAstat-Dx ME Panel. The agent or agents detected may not be the definite cause of the disease. Negative results do not preclude central nervous system infection.

Not all agents of central nervous system infection are detected by this test and sensitivity in clinical use may differ from that described in the instructions for use.

The QIAstat-Dx ME Panel is not intended for testing specimens collected from indwelling central nervous system medical devices.

The QIAstat-Dx ME Panel is intended to be used in conjunction with standard of care culture for organism recovery, serotyping, and antimicrobial susceptibility testing.

\*Cryptococcus neoformans and Cryptococcus gattii are not differentiated.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

# D Special Instrument Requirements:

The QIAstat-Dx ME Panel assay is intended for use with the QIAstat-Dx Analyzer 1.0 instrument, which was originally cleared in K183597.

IV Device/System Characteristics:

A Device Description: QIAstat-Dx ME Panel

The QIAstat-Dx ME Panel is a single use plastic device which contains wet and dry reagents to allow automated extraction and reverse-transcription real-time quantitative polymerase chain reaction (RT-qPCR) for use in the QIAstat-Dx Analyzer 1.0 instrument. All reagents required to perform the test are pre-loaded on the cartridge, the microfluidics system is pneumatically operated preventing reagents within the cartridge from contracting the user or analyzer actuators.

Within the cartridge, multiple steps are automatically performed in sequence by using pneumatic pressure and a multiport valve to transfer sample and fluids via the transfer chamber to the intended destinations.

# QIAstat-Dx Analyzer 1.0 Instrument

The QIAstat-Dx Analyzer 1.0 instrument hosts the QIAstat-Dx ME Panel cartridge and runs predefined assay protocols by use of pneumatic pressure and a multiport valve.

Interpretation of Results

The QIAstat-Dx Analyzer 1.0 instrument automatically interprets and saves test results. After ejecting the cartridge, the results summary screen is automatically displayed. Detected analytes (i.e., positive results) are displayed at the top of the list under the category ‘Detected’ in red font with a plus sign $( + )$ next to the name. The last section of the results screen shows all targets tested with either a plus sign if it was detected or a minus sign (-) with the name in green colored font if the analyte was tested but not detected.

# Quality Control

The QIAstat-Dx ME Panel includes a titered lyophilized MS2 bacteriophage internal control (IC) that provides verification that all analysis steps including sample resuspension, lysis, nucleic acid purification, reverse transcription, and PCR were successful. The results screen displays a message indicating that the internal controls “Passed” when the test was run successfully. A message of “Failed” indicates that the internal control was not amplified; ‘Positive’ test results are still reported as positive, but all ‘Negative’ results are invalid. Positive and negative external controls are recommended by the manufacturer but are not provided.

# B Principle of Operation:

Once the cartridge has been inserted into the instrument, the test starts automatically and runs for approximately 80 minutes. When the test is finished, the cartridge is removed by the user and discarded. The QIAstat-Dx Analyzer 1.0 instrument automatically interprets test results and displays a summary on the analyzer display screen. The results can be printed using a connected printer, if needed.

The collection of specimens and their subsequent loading into the QIAstat-Dx ME Panel cartridge should be performed by personnel trained in safe handling of biological samples. The user collects the cerebrospinal fluid (CSF) specimen and loads $2 0 0 \mu \mathrm { L }$ of sample into the main port of the QIAstat-Dx ME Panel. All remaining steps through results interpretation and display are completed automatically by the QIAstat-Dx Analyzer 1.0 instrument.

# V Substantial Equivalence Information:

A Predicate Device Name(s): FilmArray Meningitis/Encephalitis (ME) Panel for use with FilmArray Torch   
B Predicate 510(k) Number(s): K160462

# C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=5>K242256</td><td rowspan=1 colspan=1>K160462</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=5>QIAstat-Dx Meningitis/Encephalitis (ME)Panel</td><td rowspan=1 colspan=1>FilmArrayMeningitis/Encephalitis(ME) Panel</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Intended Use/IndicationsFor Use</td><td rowspan=4 colspan=5>The QIAstat-DxMeningitis/Encephalitis(ME) Panel is aqualitative multiplexednucleic acid real-timePCR based in vitrodiagnostic test intendedfor use with theQIAstat-Dx Analyzer1.0 instrument. TheQIAstat-Dx ME Panelis capable ofsimultaneous detectionand identification ofmultiple bacterial, viral,and yeast nucleic acidsfrom cerebrospinal fluid(CSF) specimensobtained via lumbarpuncture fromindividuals with signsand/or symptoms ofmeningitis and/orencephalitis.The followingorganisms are identifiedusing the QIAstat-DxME Panel: Enterovirus,Escherichia coli K1,Haemophilus</td><td rowspan=4 colspan=1>The FilmArrayMeningitis/Encephalitis (ME)Panel is a qualitativemultiplexed nucleicacid-based in vitrodiagnostic test intendedfor use with FilmArray,FilmArray 2.0, andFilmArray Torchsystems. The FilmArrayME Panel is capable ofsimultaneous detectionand identification ofmultiple bacterial, viral,and yeast nucleic acidsdirectly fromcerebrospinal fluid(CSF) specimensobtained via lumbarpuncture fromindividuals with signsand/or symptoms ofmeningitis and/orencephalitis.The FilmArray MEPanel is indicated as anaid in the diagnosis ofspecific agents ofmeningitis and/or</td></tr><tr><td rowspan=1 colspan=3>individuals</td></tr><tr><td rowspan=1 colspan=3>and/or symptoms of</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>e QIAst</td></tr></table>

<table><tr><td colspan="2">influenzae, Listeria monocytogenes,</td></tr><tr><td rowspan="9"></td><td>(encapsulated), Streptococcus</td></tr><tr><td>Neisseria meningitidis conju clinica</td></tr><tr><td>epider</td></tr><tr><td>agalactiae,</td></tr><tr><td>labora from Panel</td></tr><tr><td>Streptococcus</td></tr><tr><td>pneumoniae,</td></tr><tr><td>Streptococcus pyogenes, and</td></tr><tr><td>basis Cryptococcus treatm neoformans/gattii*. patien decisi</td></tr><tr><td>The QIAstat-Dx ME Panel is indicated as an aid in the diagnosis of specific agents of meningitis and/or encephalitis and results must be used in conjunction with other clinical, epidemiological, and laboratory data. Results</td><td>result co-inf organ in the Panel detect defini diseas result</td></tr><tr><td>from the QIAstat-Dx ME Panel are not intended to be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with organisms not</td><td>centra (CNS agents are de and se clinic from the pa The F Panel testing</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">sensitivity in clinicaluse may differ from thatdescribed in theinstructions for use.The QIAstat-Dx MEPanel is not intended fortesting specimenscollected fromindwelling centralnervous system medicaldevices.The QIAstat-Dx MEPanel is intended to beused in conjunctionwith standard of careculture for organismrecovery, serotyping,and antimicrobialsusceptibility testing.*Cryptococcusneoformans andCryptococcus gattii arenot differentiated.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Bacteria:Escherichia coli K1HaemophilusinfluenzaeListeriamonocytogenesNeisseria meningitidis(encapsulated)StreptococcusagalactiaeStreptococcuspneumoniaeVirus:EnterovirusYeast:Cryptococcusneoformans/gattii</td><td colspan="1" rowspan="1">Bacteria:Escherichia coli K1HaemophilusinfluenzaeListeriamonocytogenesNeisseria meningitidis(encapsulated)StreptococcusagalactiaeStreptococcuspneumoniaeViruses:EnterovirusYeast:Cryptococcusneoformans/gattii</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Cerebrospinal Fluid</td><td colspan="1" rowspan="1">Cerebrospinal Fluid</td></tr><tr><td colspan="1" rowspan="1">Analyte Detected</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">RNA/DNA</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">RT-PCR</td><td colspan="1" rowspan="1">RT-PCR</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">One internal processingcontrol in each cartridgeis subjected to allnucleic acid extractionand amplification steps.</td><td colspan="1" rowspan="1">Two controls areprovided in eachreagent pouch to controlfor sample processingand both stages of PCRand melt analysis.Labeling recommendsuse of negative(transport medium) andpositive (previouslycharacterized positivesample or negativesample spiked withtarget organism)external controls.</td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Bacterium:Streptococcuspyogenes</td><td colspan="1" rowspan="1">Viruses:CytomegalovirusHerpes simplex virusHerpes simplex virus2Human herpesvirus 6Human parechovirusVaricella zoster virus</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">QIAstat-Dx Analyzer1.0</td><td colspan="1" rowspan="1">FilmArray, FilmArray2.0, and FilmArrayTorch systems</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI EP07 $3 ^ { \mathrm { r d } }$ Edition 7-275 Interference Testing in Clinical Chemistry   
CLSI EP17-A2 7-233 Evaluation of Detection Capability for Clinical Laboratory   
Measurement Procedures; Approved Guideline – Second Edition   
CLSI EP25-A (Replaces EP25-P) 7-235 Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline   
CLSI EP12-A2 7-152 User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition   
CLSI EP05-A3 (Reaffirmed: September 2019) 7-251 Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline – Third Edition   
CLSI MM03- $\cdot 3 ^ { \mathrm { r d } }$ Edition (Replaces MM03-A2) 7-260 Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline   
IEC 62304 Edition 1.1 2015-06 CONSOLIDATED VERSION 13-79 Medical device software – Software life cycle processes

# A Analytical Performance:

1. Inclusivity

a. The Inclusivity (analytical reactivity) study extended the list of pathogen strains tested during the QIAstat-Dx ME Panel Limit of Detection (LoD) study to confirm the reactivity of the detection system in the presence of different strains of the same organisms at a concentration near or above the respective LoD.

A variety of clinically relevant strains of each target organism of the QIAstat-Dx ME Panel representing organism sub-types, strains, serotypes, and genotypes of different temporal and geographic diversity of each analyte were included in the study. Analytical reactivity was assessed with both in vitro wet-testing and in silico analysis.

A total of 130 strains covering 10 different analytical strains for each targeted organism or relevant species were tested as described in Table 1 below.

Table 1: Inclusivity Strains Evaluated Wet Testing   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Source (Catalog ID)</td><td colspan="1" rowspan="1">Strain/Serotype</td><td colspan="1" rowspan="1">LowestConcentrationTested</td></tr><tr><td colspan="1" rowspan="11">Escherichia coliK1</td><td colspan="1" rowspan="1">ATCC (700973) a</td><td colspan="1" rowspan="1">Strain C5 [Bort]; O18ac:K1:H7</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (11775) a</td><td colspan="1" rowspan="1">NCTC 9001. Serovar O1:K1:H7</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">NCTC (9007)</td><td colspan="1" rowspan="1">Strain Bi 7509/41; O7:K1:H-</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="2">ATCC (23509)</td><td colspan="1" rowspan="1">NCDC Bi 7509-41 Serotype O7</td><td colspan="1" rowspan="2">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">:K1(L) :NM</td></tr><tr><td colspan="1" rowspan="1">ATCC (23511)</td><td colspan="1" rowspan="1">NCDC F 11119-41</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 17666)</td><td colspan="1" rowspan="1">O-2, U9-41</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 17674)</td><td colspan="1" rowspan="1">O-16, F1119-41</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0801905)</td><td colspan="1" rowspan="1">Z136 CTX-M-15</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">NCTC (11101)</td><td colspan="1" rowspan="1">Sc15 02:K1:H6</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">NCTC (9045)</td><td colspan="1" rowspan="1">Strain H61; O45:K1:H10</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="10">Listeriamonocytogenes</td><td colspan="1" rowspan="1">ZeptoMetrix (0801534) a</td><td colspan="1" rowspan="1">Type 1/2b</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (19115) a</td><td colspan="1" rowspan="1">Type 4b. Strain Li 2</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-2659)</td><td colspan="1" rowspan="1">Type 1/2a. Strain 2011L-2676</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (19111)</td><td colspan="1" rowspan="1">Type 1/2a. Strain Li 20</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0804339)c</td><td colspan="1" rowspan="1">Type 4b</td><td colspan="1" rowspan="1">0.03x LoD c</td></tr><tr><td colspan="1" rowspan="1">ATCC (13932)</td><td colspan="1" rowspan="1">serotype 4b. Strain 1071/53 [LMG21264, NCTC 10527]</td><td colspan="1" rowspan="1">0.011x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (19114)</td><td colspan="1" rowspan="1">Li 23. Serotype 4a</td><td colspan="1" rowspan="1">0.02x LoD</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 13237)</td><td colspan="1" rowspan="1">FSL J2-064</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (7644)</td><td colspan="1" rowspan="1">Gibson</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-679)</td><td colspan="1" rowspan="1">EGDe</td><td colspan="1" rowspan="1">0.026x LoD</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">ATCC (13090) a</td><td colspan="1" rowspan="1">Serotype B. M2092 [CIP 104218,</td><td colspan="1" rowspan="2">1x LoD</td></tr><tr><td colspan="1" rowspan="1">L. Cunningham]</td></tr><tr><td colspan="4" rowspan="10">ATCC (35561) a              Serotype Y. M-112 [BO-6]               1x LoDATCC (13077)          Serogroup A, M1027 [NCTC10025]         0.03x LoDATCC (13102)                  Serogroup C, M1628                0.03x LoDATCC (13113)                Serotype D. M158 [37A]Neisseria                3x LoDmeningitidis     IDT (gBlock 77859371) b       sequence with variant ctrA gene       1:1000 from stock(encapsulated)         ATCC (43744)                          W135                        0.03x LoDATCC (BAA-335)                       MC58                       0.03x LoDATCC (23255)                  79 Eur. Serogroup B             1:1000 from stockATCC (13092)              Serotype B. M997 [S-3250-L]        1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (35561) a</td><td colspan="1" rowspan="1">Serotype Y. M-112 [BO-6]</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (13077)</td><td colspan="1" rowspan="1">Serogroup A, M1027 [NCTC10025]</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (13102)</td><td colspan="1" rowspan="1">Serogroup C, M1628</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (13113)</td><td colspan="1" rowspan="1">Serotype D. M158 [37A]</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">IDT (gBlock 77859371) b</td><td colspan="1" rowspan="1">sequence with variant ctrA gene</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (43744)</td><td colspan="1" rowspan="1">W135</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-335)</td><td colspan="1" rowspan="1">MC58</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (23255)</td><td colspan="1" rowspan="1">79 Eur. Serogroup B</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (13092)</td><td colspan="1" rowspan="1">Serotype B. M997 [S-3250-L]</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="10">Streptococcusagalactiae</td><td colspan="1" rowspan="1">ZeptoMetrix (0801545) a</td><td colspan="1" rowspan="1">Z019</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (13813) a</td><td colspan="1" rowspan="1">G19 group B</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (12403)</td><td colspan="1" rowspan="1">Serotype III. Typing strain D136C(3) [3Cole 106, CIP 82.45]</td><td colspan="1" rowspan="1">1.2x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-611)</td><td colspan="1" rowspan="1">2603 V/R. Serotype V</td><td colspan="1" rowspan="1">0.013x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (31475)</td><td colspan="1" rowspan="1">type III-ST283</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 43898)</td><td colspan="1" rowspan="1">MNZ929</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (12401)</td><td colspan="1" rowspan="1">Typing strain H36B - type Ib</td><td colspan="1" rowspan="1">1.4x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (27591)</td><td colspan="1" rowspan="1">CDC SS700 [A909; 5541], type 1c</td><td colspan="1" rowspan="1">0.014x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (49446)</td><td colspan="1" rowspan="1">3139 [CNCTC 1/82] Serotype IV</td><td colspan="1" rowspan="1">0.0127x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0801556)</td><td colspan="1" rowspan="1">Z023</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="12">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">ZeptoMetrix (0801439) a</td><td colspan="1" rowspan="1">19F</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (33400) a</td><td colspan="1" rowspan="1">Serotype 1. NCTC 7465</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-334)</td><td colspan="1" rowspan="1">Serotype 4. TIGR4 [JNR.7/87]</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="2">ATCC (BAA-341)</td><td colspan="1" rowspan="1">Serotype 5. SPN1439-106</td><td colspan="1" rowspan="2">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">[Colombia 5-19]</td></tr><tr><td colspan="1" rowspan="1">ATCC (10343)</td><td colspan="1" rowspan="1">Serotype 11A. Type 43</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (700672)</td><td colspan="1" rowspan="1">Serotype 14. VH14</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (700673)</td><td colspan="1" rowspan="1">Serotype 19A. Hungary 19A-6[HUN663]</td><td colspan="1" rowspan="1">0.026x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0804016)</td><td colspan="1" rowspan="1">Z319; 12F</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="2">ATCC (6303)</td><td colspan="1" rowspan="1">Diplococcus pneumoniae; Type 3.</td><td colspan="1" rowspan="2">3x LoD</td></tr><tr><td colspan="1" rowspan="1">Strain [CIP 104225]</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-661)</td><td colspan="1" rowspan="1">DCC1476 [Sweden 15A-25]</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="10">Streptococcuspyogenes</td><td colspan="1" rowspan="1">ZeptoMetrix (0804351) a</td><td colspan="1" rowspan="1">Z472; Serotype M1</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (19615) a</td><td colspan="1" rowspan="1">Bruno [CIP 104226]</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0801512)</td><td colspan="1" rowspan="1">Z018; Serotype M58</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (BAA-947)</td><td colspan="1" rowspan="1">Serotype M1. MGAS 5005</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (14289)</td><td colspan="1" rowspan="1">Lancefield's group A / C203 S</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (12203)</td><td colspan="1" rowspan="1">NCTC 8709 (Type 6 glossy)</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (12353)</td><td colspan="1" rowspan="1">Group a, type 12. Typing strain T12 [F.Griffith SF 42]</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (12972)</td><td colspan="1" rowspan="1">Group a, type 14</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (8133)</td><td colspan="1" rowspan="1">Group a, type 23</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (12384)</td><td colspan="1" rowspan="1">C203 -Type 3</td><td colspan="1" rowspan="1">0.029x LoD</td></tr><tr><td colspan="1" rowspan="2">Enterovirus A</td><td colspan="1" rowspan="1">ZeptoMetrix (0810107CF)a</td><td colspan="1" rowspan="1">Coxsackievirus A16</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1801) a</td><td colspan="1" rowspan="1">A6, species A. Strain Gdula</td><td colspan="1" rowspan="1">1x LoD</td></tr></table>

<table><tr><td colspan="1" rowspan="9">b</td><td></td><td></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-168)</td><td colspan="1" rowspan="1">A10. M.K. (Kowalik)</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1432)</td><td colspan="1" rowspan="1">Enterovirus 71. Strain H</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810236CF)</td><td colspan="1" rowspan="1">Species A, Serotype EV-A71 (2003Isolate)</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR-471)</td><td colspan="1" rowspan="1">Tainan/4643/1998</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1550)</td><td colspan="1" rowspan="1">A2 Fl [Fleetwood]</td><td colspan="1" rowspan="1">3x LoD c</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-673)</td><td colspan="1" rowspan="1">A7- 275/58</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-170)</td><td colspan="1" rowspan="1">A12 - Texas 12</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1775)</td><td colspan="1" rowspan="1">EV-A71. Strain BrCr</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="10">Enterovirus B</td><td colspan="1" rowspan="1">ZeptoMetrix (0810019CF)a</td><td colspan="1" rowspan="1">Coxsackievirus B5</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810017CF)a</td><td colspan="1" rowspan="1">Coxsackievirus A9, species B</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-28)</td><td colspan="1" rowspan="1">Species B, Serotype CV-B1, StrainConn-5</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-29)</td><td colspan="1" rowspan="1">Species B, Serotype CV-B2. Strain Ohio-1</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810075CF)</td><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810076CF)</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810077CF)</td><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810074CF)</td><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">1:10000 from stock</td></tr><tr><td colspan="1" rowspan="1">NCPV (0901047v)</td><td colspan="1" rowspan="1">Echovirus 18</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-41)c</td><td colspan="1" rowspan="1">Species B, Serotype E-11</td><td colspan="1" rowspan="1">3x LoD c</td></tr><tr><td colspan="1" rowspan="10">Enterovirus C</td><td colspan="1" rowspan="1">ATCC (VR-1023) a</td><td colspan="1" rowspan="1">Coxsackievirus A17, species C. Strain G-12</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-583) a</td><td colspan="1" rowspan="1">Coxsackievirus A24. Starin DN-19</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-850)c</td><td colspan="1" rowspan="1">Coxsackievirus A21. Strain Kuykendall[V-024-001-012]</td><td colspan="1" rowspan="1">3x LoDc</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-169)</td><td colspan="1" rowspan="1">A11 - Belgium-1</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1488)</td><td colspan="1" rowspan="1">A13 - Flores</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-182)</td><td colspan="1" rowspan="1">A22 - Chulman</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-178)</td><td colspan="1" rowspan="1">A20 - IH Pool 35</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-176)</td><td colspan="1" rowspan="1">A18 -G-13</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">NCTC (0812075v)</td><td colspan="1" rowspan="1">CV-A21. Strain H06452 472</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">NCTC (0812074v)</td><td colspan="1" rowspan="1">CV-A21. Strain H06418 508</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="11">Enterovirus D</td><td colspan="1" rowspan="1">ATCC (VR-836) a</td><td colspan="1" rowspan="1">EV 70, species D, strain J670/71</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1823) a</td><td colspan="1" rowspan="1">Enterovirus D68. Strain US/MO/14-18947</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810237CF)</td><td colspan="1" rowspan="1">Enterovirus 68. 2007 Isolate</td><td colspan="1" rowspan="1">0.03x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1824)</td><td colspan="1" rowspan="1">Enterovirus D68. Strain US/IL/14-18952</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1197)</td><td colspan="1" rowspan="1">D68. Strain F02-3607 Corn</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0810302CF)</td><td colspan="1" rowspan="1">Type 68 Major Group (09/2014 Isolate2)</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1825)</td><td colspan="1" rowspan="1">Enterovirus D68. Strain US/KY/14-18953</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (VR-1826)</td><td colspan="1" rowspan="1">Enterovirus D68. Strain Fermon</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 49130)</td><td colspan="1" rowspan="1">Enterovirus D68. US/MO/14- 18949</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR-</td><td colspan="1" rowspan="2">Enterovirus D68. USA/2018-23089</td><td colspan="1" rowspan="2">3x LoD</td></tr><tr><td colspan="1" rowspan="1">51998)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC (MYA-4567) a</td><td colspan="1" rowspan="1">Serotype D strain WM629, type VNIV</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="9">Cryptococcusneoformans</td><td colspan="1" rowspan="1">ATCC (208821) a</td><td colspan="1" rowspan="1">H99</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (32045)</td><td colspan="1" rowspan="1">type strain, CBS 132</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4564)</td><td colspan="1" rowspan="1">Serotype A strain WM148, type VNI</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (13690)</td><td colspan="1" rowspan="1">M2092</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4566)</td><td colspan="1" rowspan="1">Serotype AD strain WM628, type VNIII</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix (0801803)</td><td colspan="1" rowspan="1">Serotype A</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 50335)</td><td colspan="1" rowspan="1">NIH9hi90</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 50332)</td><td colspan="1" rowspan="1">NIH306</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 48776)</td><td colspan="1" rowspan="1">Var grubiiYL99α</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="10">Cryptococcusgattii</td><td colspan="1" rowspan="1">ATCC (MYA-4094) a</td><td colspan="1" rowspan="1">Serotype B strain R272, type VGIIb</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4877) a</td><td colspan="1" rowspan="1">A6MR38</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4560)</td><td colspan="1" rowspan="1">Serotype B strain WM179, type VGI</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4562)</td><td colspan="1" rowspan="1">Serotype B strain WM161, type VGIII</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4563)</td><td colspan="1" rowspan="1">Serotype C strain WM779, type VGIV</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">ATCC (MYA-4138)</td><td colspan="1" rowspan="1">A1M R265</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">ATCC (14248)</td><td colspan="1" rowspan="1">110 [CBS 883]</td><td colspan="1" rowspan="1">1:1000 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 50184)</td><td colspan="1" rowspan="1">AIR265</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 50195)</td><td colspan="1" rowspan="1">Alg166</td><td colspan="1" rowspan="1">1:100 from stock</td></tr><tr><td colspan="1" rowspan="1">BEI Resources (NR- 50198)</td><td colspan="1" rowspan="1">Alg254</td><td colspan="1" rowspan="1">1:100 from stock</td></tr></table>

a Strains tested and evaluated during the Limit of Detection studies. b Commercial artificial dsDNA fragment (gBlock) including ctrA gene sequence was tested due to unavailability of analytical sample for this variant (GenBank Accession ID HQ156899). c Higher concentration tested to meet $100 \%$ detection rate (30xLoD for ATCC (VR-1550, VR-850 and VR-41) and 3xLoD for ZeptoMetrix (0804339)).

125 out of 130 pathogen strains were successfully detected by the QIAstat-Dx ME Panel when tested. Five strains were not detected by the assay (Table 2).

Table 2: Inclusivity Strains Not Detected by the QIAstat-Dx ME Panel   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain/Serotype</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli K1</td><td rowspan=1 colspan=1>NCDC Bi 7509-41 Serotype O7:K1(L):NM</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli K1</td><td rowspan=1 colspan=1>Z136CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>CV-A21StrainH06452 472</td></tr><tr><td rowspan=1 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>CV-A21, StrainH06418 508</td></tr><tr><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>Serotype III. Typing strain D136C(3) [3 Cole 106,CIP 52.45]</td></tr></table>

To make assay reactivity predictions of all primers-probe oligonucleotide sequences included in the panel against publicly available sequence databases to detect any possible crossreaction or unexpected detection of any primer set, in silico analysis was performed. In addition, strains not available for in vitro testing were included in in silico analysis to confirm the predicted inclusivity of the different strains of the same organisms (Table 3).

Table 3: Clinically Relevant Strains/Serotypes Detected per Pathogen   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Clinically relevant strains/subtypes detected</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis(encapsulated)</td><td rowspan=1 colspan=1>All the encapsulated serotypes (A, B, C, D, E, H, I, K, L,NG, W, W135, X, Y, Z, 29E)</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii/neoformans</td><td rowspan=1 colspan=1>All Cryptococcus spp. serotypes: Serotype A (C.neoformans var neoformans), serotype D (C. neoformansvar grubii), serotypes B and C (C. gattii including allVGI,VGII, VGIII, VGIV molecular types)</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Serotypes 1/2a,1/2b, 1/2c, 3a, 3b, 3c, 4a, 4b, 4c, 4d, 4e, 7</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>All encapsulated serotypes (a, b, c, d, e, f) andunencapsulated strains (nontypeable, NTHi) includingvar. H. aegyptus</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Coxsackievirus A (CV-A1 through CV-A24),coxsackievirus B (CV-B1 through CV-B6), Echovirus (E-1 through E-33), Enterovirus A (EV-A71, EV-A76, EV-A89 through EV-A92, EV-A119, EVA120), EnterovirusB (EV-B69, EV-B73 through EV-B75, EV-B79, EV-B80through EV-B88, EVB93, EV-B97, EV-B98, EV-B100,EV-B101, EV-C99, EV-C102, EV-C104, EV-C105, EV-C109, EV-C116 through EV-C118), Enterovirus EV-B106, EV-B107, EV-B111), Enterovirus C (EVC96, D(EV-D68, EV-D70, EV-D94), Poliovirus (PV-1 throughPV-3)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli Kl</td><td rowspan=1 colspan=1>K1 strains (excluding general E.coli strains)</td></tr><tr><td rowspan=1 colspan=1>Rest of On-Panel organism withno biological subclassification(S. pneumoniae, S. agalactiae, S.pyogenes)</td><td rowspan=1 colspan=1>All genomic sequences available in databases detected</td></tr></table>

2. Precision/Reproducibility:

a. Multi-site Reproducibility Study

Reproducibility of the QIAstat-Dx ME Panel assay was evaluated at three different sites (one internal, two external) using three blinded panels (Table 4) representing a subset of targets detected by the assay. The study evaluated positive samples (artificial CSF spiked with selected pathogens) containing analytes at both 1X LoD and 3X LoD concentrations as well as negative samples (artificial CSF). Testing occurred over 5 non-consecutive days with each sample evaluated in two replicates/site/day. The study also included 13 operators across the three study sites and a total of 4 cartridge lots.

A total of 90 replicates per target were tested at each concentration evaluated. Samples were prepared in bulk, divided into single-use aliquots, and stored frozen at $- 8 0 ^ { \circ } \mathrm { C }$ or below until usage. Samples were evaluated for expected reactivity before inclusion in the study.

Table 4: Reproducibility Study Panels   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Testedconcentration</td></tr><tr><td rowspan=5 colspan=1>1</td><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>MYA-4094</td><td rowspan=1 colspan=1>7730064</td><td rowspan=5 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>13813</td><td rowspan=1 colspan=1>70011789</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>19115</td><td rowspan=1 colspan=1>70029351</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1801</td><td rowspan=1 colspan=1>70002938</td></tr><tr><td rowspan=1 colspan=1>E. coli K1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>700973</td><td rowspan=1 colspan=1>70022532</td></tr><tr><td rowspan=5 colspan=1>2</td><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>MYA-4094</td><td rowspan=1 colspan=1>7730064</td><td rowspan=5 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>13813</td><td rowspan=1 colspan=1>70011789</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>19115</td><td rowspan=1 colspan=1>70029351</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1801</td><td rowspan=1 colspan=1>70002938</td></tr><tr><td rowspan=1 colspan=1>E. coli K1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>700973</td><td rowspan=1 colspan=1>70022532</td></tr><tr><td rowspan=4 colspan=1>3</td><td rowspan=4 colspan=1>Negative (aCSF)</td><td rowspan=4 colspan=1>EcocyteBioscience</td><td rowspan=1 colspan=1>LRE-S-</td><td rowspan=1 colspan=1>aCSF A:</td><td rowspan=4 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>LSG-1000-1</td><td rowspan=1 colspan=1>5002764104</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>aCSF B:</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>181222A</td></tr></table>

Reproducibility was assessed by evaluating the agreement of positive or negative results form the investigational assay with the expected results per for each target and concentration evaluated. The study acceptance criteria for each panel member were the following:

• 3x LoD: the observed proportion of positive calls must be $2 9 6 . 7 \%$ (87 / 90) (95% CI: $9 0 . 6 \% 9 9 . 3 \%$ ).   
• 1x LoD: the observed proportion of positive calls must be $2 9 0 . 0 \%$ (81 / 90) $9 5 \%$ CI: $8 1 . 9 \% - 9 5 . 3 \%$ ). Negative: The proportion of negative results must be $2 9 6 . 7 \%$ (87 / 90) ( $9 5 \%$ CI: $9 0 . 6 \% 9 9 . 3 \%$ ).

Raw data interpretation and results reporting were automatically done by the Operational Module Application Software and the Assay Definition File (ADF) that is installed in the QIAstat-Dx Analyzer 1.0 instrument.

At 3x LoD concentration, all targets gave $100 \%$ correct calls. At 1x LoD concentration, all targets gave $100 \%$ correct calls, except for Listeria monocytogenes (Table 5). All negative samples returned a negative call $100 \%$ of the time. The study results were acceptable and demonstrate appropriate reproducibility of the assay.

Table 5: Reproducibility Study Results Summary   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Concentration</td><td colspan="4" rowspan="1">Agreement with Expected Results</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites[95% CI]</td></tr><tr><td colspan="6" rowspan="1">Bateria</td></tr><tr><td colspan="1" rowspan="2">Escherichia ColiK1</td><td colspan="1" rowspan="1">3x LoD1044 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="1">1x LoD348 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="2">Listeriamonocytogenes</td><td colspan="1" rowspan="1">3x LoD19920 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="1">1x LoD6640 CFU/mL</td><td colspan="1" rowspan="1">96.7%(29/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">98.9%(89/90)[94.0%-99.8%]</td></tr><tr><td colspan="1" rowspan="2">Streptococcusagalactiae</td><td colspan="1" rowspan="1">3x LoD10140 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="1">1x LoD3380 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Enterovirus (EV)</td><td colspan="1" rowspan="1">3x LoD480 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="1">1x LoD160 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="6" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="2">Cryptococcusgattii/Cryptococcusneoformans*</td><td colspan="1" rowspan="1">3x LoD39600 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="1" rowspan="1">1x LoD13200 CFU/mL</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr><tr><td colspan="6" rowspan="1">Negative (no analyte)</td></tr><tr><td colspan="2" rowspan="1">Negative CSF Matrix</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(30/30)</td><td colspan="1" rowspan="1">100.0%(90/90)[95.9%-100.0%]</td></tr></table>

\*Cryptococcus gattii and Cryptococcus neoformans are not differentiated.

3. Linearity: Not applicable. The device is a qualitative assay.

4. Analytical Specificity/Interference: a. Analytical Specificity (Cross-Reactivity)

The analytical specificity of the assay was evaluated using both in vitro wet-testing and in silico analysis to assess the potential cross-reactivity and exclusivity of the QIAstat-Dx ME Panel. On-panel organisms were tested to assess the potential for intra-panel crossreactivity and off-panel organisms were tested to evaluate cross-reactivity with organisms not covered by the panel (panel exclusivity). Off-panel organisms were selected due to being known to colonize the central nervous system or cause meningitis and/or encephalitis symptoms, are common skin flora or laboratory contaminants, are genetically similar to on-panel analytes, or are microorganisms for which much of the population may have been infected.

Samples were prepared by spiking potential cross-reactive organisms (Table 6) into artificial CSF matrix (see Section B.2. for matrix comparison) at $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viral targets, $1 0 ^ { 6 } \mathrm { C F U / m L }$ for fungi/yeast target, and $1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacterial and fungi/yeast target, or the highest concentration possible based on the organism stock.

All on-panel pathogens evaluated in the study resulted in specific detection, and all offpanel pathogens tested showed a negative result with no cross-reactivity was observed except for the pathogens shown in Table 6 below.

Table 6: Cross-Reactivity Organisms Evaluated   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain/Serotype</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain/Serotype</td></tr><tr><td rowspan=4 colspan=1>Ba</td><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Z091</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>LRA 08 01 73 [API SA,DSM 6674]</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>[ATCC 13316, NCTC9750]</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>PRD-10 [CIP 103467,NCIB 10421, PCI 812]</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>CDC F6683</td><td rowspan=1 colspan=1>Salmonella bongori</td><td rowspan=1 colspan=1>CIP 82.33</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumurealyticus</td><td rowspan=1 colspan=1>3 [Garcia strain]</td><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CDC K-1891 [ATCC25928</td></tr></table>

20   

<table><tr><td rowspan=2 colspan=5>CronobacterCDC 4562-70          Serratia marcescens               PCI 1107(Enterobacter) sakazakii</td></tr><tr><td rowspan=1 colspan=1>Cronobacter(Enterobacter) sakazakii</td><td rowspan=1 colspan=1>CDC 4562-70</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td></td><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Z052</td><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>CDC C-123</td></tr><tr><td></td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CDC 442-68</td><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>Z046</td></tr><tr><td></td><td rowspan=1 colspan=1>Escherichia coli (non-K1)</td><td rowspan=1 colspan=1>2003-3055</td><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>AMC 43-GG9</td></tr><tr><td></td><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>Z302</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>FDA 209</td></tr><tr><td></td><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>CDC 980-72</td><td rowspan=1 colspan=1>Staphylococcus capitis</td><td rowspan=1 colspan=1>PRA 360 677</td></tr><tr><td></td><td rowspan=1 colspan=1>Escherichia vulneris</td><td rowspan=1 colspan=1>CDC 875-72</td><td rowspan=1 colspan=1>Staphylococcusepiddermidis</td><td rowspan=1 colspan=1>FDA strain PCI 1200</td></tr><tr><td></td><td rowspan=1 colspan=1>Haemophilus ducreyiDCC1476</td><td rowspan=1 colspan=1>[Sweden 15A-25]</td><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>SM 131</td></tr><tr><td></td><td rowspan=1 colspan=1>Haemophilus haemolyticus</td><td rowspan=1 colspan=1>NCTC 10659</td><td rowspan=1 colspan=1>Staphylococcus hominis</td><td rowspan=1 colspan=1>Z031</td></tr><tr><td></td><td rowspan=1 colspan=1>Haemophilusparahaemolyticus</td><td rowspan=1 colspan=1>536 [NCTC 8479]</td><td rowspan=1 colspan=1>Staphylococcuslugdunensis</td><td rowspan=1 colspan=1>LRA 260.05.79</td></tr><tr><td></td><td rowspan=1 colspan=1>Haemophilusparainfluenzae</td><td rowspan=1 colspan=1>NCTC 7857</td><td rowspan=1 colspan=1>Staphylococcussaprophyticus</td><td rowspan=1 colspan=1>NCTC 7292</td></tr><tr><td></td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NCTC 9633 [NCDC298-53, NCDC 410-68]</td><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>NCTC 10713</td></tr><tr><td></td><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1>SLCC 3379</td><td rowspan=1 colspan=1>Streptococcus bovis</td><td rowspan=1 colspan=1>Z167</td></tr><tr><td></td><td rowspan=1 colspan=1>Listeria ivanovii</td><td rowspan=1 colspan=1>Li 1979</td><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>Grouping strain C74</td></tr><tr><td></td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>AM-15</td><td rowspan=1 colspan=1>Streptococcus intermedius</td><td rowspan=1 colspan=1>Z126</td></tr><tr><td></td><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>M30</td><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>(tigurinus) ClinicalIsolate</td></tr><tr><td></td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Z017</td><td rowspan=1 colspan=1>Streptococcus mutans</td><td rowspan=1 colspan=1>LRA 28 02 81</td></tr><tr><td></td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>NCDC A7515</td><td rowspan=1 colspan=1>Streptococcus oralis</td><td rowspan=1 colspan=1>Z307</td></tr><tr><td></td><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>AmMS 138</td><td rowspan=1 colspan=1>Streptococcuspseudopneumoniae</td><td rowspan=1 colspan=1>CDC-SS-1757</td></tr><tr><td></td><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>AMC 14-D-1</td><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>C699</td></tr><tr><td></td><td rowspan=1 colspan=1>Pantoea agglomerans</td><td rowspan=1 colspan=1>Enterobacteragglomerans</td><td rowspan=1 colspan=1>Streptococcus sanguinis</td><td rowspan=1 colspan=1>DSS-10</td></tr><tr><td></td><td rowspan=1 colspan=1>Proprionibacterium acnes</td><td rowspan=1 colspan=1>NCTC 737</td><td rowspan=1 colspan=2>Y</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>Z014</td><td rowspan=1 colspan=1>Cryptococcus adeliensisCryptococcus adeliaeNaganishia adelienses</td><td rowspan=1 colspan=1>Cryptococcus adeliae</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>CBS 562</td><td rowspan=1 colspan=1>Cryptococcus albidus</td><td rowspan=1 colspan=1>AmMS 228</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>Z145</td><td rowspan=1 colspan=1>Cryptococcus amylolentus</td><td rowspan=1 colspan=1>NRRY Y-7784</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>CBS 138</td><td rowspan=1 colspan=1>CryptococcusdepauperatusAspergillus depauperatusFilobasidiella depauperate</td><td rowspan=1 colspan=1>K [ARSEF 2058, CBS7842]</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Cryptococcus flavescensPapiliotrema flavescens</td><td rowspan=1 colspan=1>Cryptococcus alurentiivar.Flavescens (Saito)Lodder et Kerger-vanRij</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>Z010</td><td rowspan=1 colspan=1>Cryptococcus laurentii</td><td rowspan=1 colspan=1>CBS 139</td></tr></table>

<table><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>Candida metapsilosis</td><td rowspan=1 colspan=1>MC0429</td><td rowspan=1 colspan=1>Cryptococcusuniguttulatus</td><td rowspan=1 colspan=1>AmMS 234</td></tr><tr><td rowspan=1 colspan=1>Candida orthopsilosis</td><td rowspan=1 colspan=1>MCO471</td><td rowspan=1 colspan=1>Cryptococcus wingfieldiiTsuchiyaea wingfieldii</td><td rowspan=1 colspan=1>OTU 26</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>CBS 604</td><td rowspan=1 colspan=1>Filobasidiumcapsuligenum</td><td rowspan=1 colspan=1>ML-186</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Vitek #8935</td><td rowspan=1 colspan=1>Naegleria fowleri</td><td rowspan=1 colspan=1>Genomic DNA fromNaegleria fowleri</td></tr><tr><td rowspan=1 colspan=1>Candida viswanathii</td><td rowspan=1 colspan=1>PK 233 [NCYC 997,pK233]</td><td rowspan=1 colspan=1>Toxoplasma gondii</td><td rowspan=1 colspan=1>Haplogroup 2</td></tr><tr><td rowspan=21 colspan=1>Sins</td><td rowspan=1 colspan=1>Adenovirus A12</td><td rowspan=1 colspan=1>Huie</td><td rowspan=1 colspan=1>Human Rhinovirus A16</td><td rowspan=1 colspan=1>11757</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C2</td><td rowspan=1 colspan=1>Adenoid 6 (NIAID202-001-014)</td><td rowspan=1 colspan=1>Human Rhinovirus A1b</td><td rowspan=1 colspan=1>2060</td></tr><tr><td rowspan=1 colspan=1>Adenovirus D20</td><td rowspan=1 colspan=1>A.A</td><td rowspan=1 colspan=1>Human Rhinovirus B3</td><td rowspan=1 colspan=1>FEB</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E4</td><td rowspan=1 colspan=1>RI-67</td><td rowspan=1 colspan=1>Human Rhinovirus B83</td><td rowspan=1 colspan=1>Barlor 7 [V-190-001-021</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F41</td><td rowspan=1 colspan=1>Tak</td><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/Florida/3/2006</td></tr><tr><td rowspan=1 colspan=1>BK polyoma virus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Influenza A H1N1-2009</td><td rowspan=1 colspan=1>A/California/08/2009</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NL63 (Amsterdam I)</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Virginia/ATCC4/2009</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>JC polyoma virus</td><td rowspan=1 colspan=1>MAD-4</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Davis</td><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>Edmonston</td></tr><tr><td rowspan=1 colspan=1>Dengue virus (Type 2)</td><td rowspan=1 colspan=1>New Guinea C</td><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>Jones</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr Virus</td><td rowspan=1 colspan=1>B95-8</td><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td></tr><tr><td rowspan=1 colspan=1>Hepatitis B virus (HBV)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Hepatitis C virus (HCV)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Parvovirus B19</td><td rowspan=1 colspan=1>B19</td></tr><tr><td rowspan=1 colspan=1>Herpes simplex virus 1</td><td rowspan=1 colspan=1>Macintyre</td><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>A2</td></tr><tr><td rowspan=1 colspan=1>Herpes simplex virus 2</td><td rowspan=1 colspan=1>HSV-2. (Strain: MS)</td><td rowspan=1 colspan=1>Rotavirus RRV</td><td rowspan=1 colspan=1>(Rhesus Rotavirus)</td></tr><tr><td rowspan=1 colspan=1>Human herpes virus 6</td><td rowspan=1 colspan=1>HHV-6B. (Strain:Z29)</td><td rowspan=1 colspan=1>Rubella Virus</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Human herpes virus 7</td><td rowspan=1 colspan=1>SB</td><td rowspan=1 colspan=1>St. Louis EncephalitisVirus</td><td rowspan=1 colspan=1>Parton</td></tr><tr><td rowspan=1 colspan=1>Human herpes virus 8</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Varicella-zoster virus</td><td rowspan=1 colspan=1>Ellen</td></tr><tr><td rowspan=1 colspan=1>Human Immunodeficiency</td><td rowspan=1 colspan=1>Quatitative SyntheticHumanImmunodeficiencyVirus (HIV-1) RNA</td><td rowspan=1 colspan=1>West Nile Virus</td><td rowspan=1 colspan=1>1986</td></tr><tr><td rowspan=1 colspan=1>Human parechovirus</td><td rowspan=1 colspan=1>Serotype 3</td><td rowspan=1 colspan=2>Y</td></tr><tr><td rowspan=1 colspan=3>Yeast</td><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1>NRRL Y-567</td></tr></table>

Table 7: Cross-Reactive Pathogens Summary   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx ME PanelTarget</td><td rowspan=1 colspan=1>Potential Cross-ReactiveOrganism</td><td rowspan=1 colspan=1>Cross ReactiveConcentration</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Haemophilus haemolyticus</td><td rowspan=1 colspan=1>≥1.00E+03 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Cryptococcusneoformans/gattii</td><td rowspan=1 colspan=1>Cryptococcus wingfieldiiTsuchiyaea wingfieldii</td><td rowspan=1 colspan=1>≥1.00E+01 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus flavescensPapiliotrema flavescens</td><td rowspan=1 colspan=1>≥4.00E+03 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans/gattii</td><td rowspan=1 colspan=1>Cryptococcus amylolentus</td><td rowspan=1 colspan=1>≥1.00E+01 CFU/mL</td></tr></table>

In silico analysis was also performed for the QIAstat-Dx ME Panel primer/probe designs in two steps to further characterize the sequence specificity of the RT-PCR assays to detect possible unspecific homologies and/or unspecific cross-reactions.

The result of the analysis identified six potential cross-reactive off-panel targets (Table 8).

Table 8: In silico Cross-Reactivity Analysis Summary   

<table><tr><td rowspan=1 colspan=1>Off-Panel Organism</td><td rowspan=1 colspan=1>On-Panel Signal</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pseudopneumoniae*</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Listeria innocua*</td><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus amylolentus</td><td rowspan=3 colspan=1>Cryptococcus neoformans/gattii</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus depauperatus*</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus wingfieldii</td></tr></table>

\*Potential cross-reactivity was not confirmed by in vitro wet-testing.

# b. Interfering Substances:

An interference study was conducted to evaluate whether assay performance was affected by commonly encountered interfering substances (Table 9). The substances tested in the study (21) included endogenous as well as exogenous substances that are commonly found and/or introduced into CSF specimens during collection.

All QIAstat-Dx ME Panel target organisms were tested at 3x LoD in artificial CSF matrix and testing was performed in triplicate. Potential interfering substances were spiked into the samples at high concentrations intended to reflect worst case concentrations encountered in clinical samples.

Table 9: Interference Testing Results Summary   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">InterferenceObserved?</td></tr><tr><td colspan="1" rowspan="4">Btpn</td><td colspan="1" rowspan="1">Human blood</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">20 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">D(+)Glucose</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">L-lactate (Na)</td><td colspan="1" rowspan="1">2.2 mg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Immuniglobulin G (human)</td><td colspan="1" rowspan="1">20 mg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Albumin (human)</td><td colspan="1" rowspan="1">30 mg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Peripheral blood mononuclear cells</td><td colspan="1" rowspan="1">10,000 cells/μL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="16">Souox</td><td colspan="1" rowspan="1">Chlorhexidine</td><td colspan="1" rowspan="1">0.4% (w/v)</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">7% (v/v)</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">1% (v/v)</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">0.1% (v/v)</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">0.01% (v/v)</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Acyclovir</td><td colspan="1" rowspan="1">69 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Amphotericin B</td><td colspan="1" rowspan="1">5.1 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">210 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">840 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Cefotaxime</td><td colspan="1" rowspan="1">645 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Ganciclovir</td><td colspan="1" rowspan="1">25 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">30 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">339 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">180 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Voriconazole</td><td colspan="1" rowspan="1">11 μg/mL</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">0.399μg/mL</td><td colspan="1" rowspan="1">No</td></tr></table>

Most evaluated endogenous and exogenous substances have been confirmed not to interfere with any of the panel target assays at concentrations potentially found in clinical samples. However, interference was observed for Bleach present at concentrations above $0 . 0 1 \%$ .

# c. Microbial Interference/Competitive Inhibition:

A study was conducted to evaluate whether high concentrations of viruses ( $1 0 ^ { 5 }$ units/mL) or bacteria $\mathrm { 1 0 ^ { 6 } C F U / m L }$ ) could impact assay performance. Briefly, non-target organisms (Table 10) were individually mixed with artificial CSF matrix containing one of the four spiked targeted QIAstat Dx ME Panel organism mixes ( $3 \mathbf { x }$ LoD) (Table 11). Testing was performed in triplicate.

Table 10: Microbial Interference Organisms Tested   

<table><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr virus</td><td rowspan=1 colspan=1>105 cp/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1-2009</td><td rowspan=1 colspan=1>$10}$CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (non-K1)</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aures</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Measles virus</td><td rowspan=1 colspan=1>105 TCID50/mL</td></tr></table>

Table 11: On Panel Organisms Evaluated in Microbial Interference Testing   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Testedconcentration</td></tr><tr><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>Cryptococcus neoformans/gattii</td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>Enterovirus A</td></tr><tr><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=4 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli K1</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Negative (CSF)</td><td rowspan=1 colspan=1>N/A</td></tr></table>

All QIAstat-Dx ME Panel organisms were successfully detected when tested in combination with the potential microbial interferents.

5. Assay Reportable Range: Not applicable. The device is a qualitative assay.

6. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods): a. Assay Controls

# i. Internal Controls:

The QIAstat-Dx ME Panel includes an Internal Control, which is titered Schizosaccharomyces pombe, a yeast (fungi) that is included in the cartridge in dried form and is rehydrated upon sample loading. This Internal Control material verifies all steps of the analysis process, including sample homogenization, lysis of viral and cellular structures (by means of chemical and mechanical disruption), nucleic acid purification, reverse transcription, and real-time PCR.

A positive signal for the Internal Control indicates that all processing steps performed by the QIAstat-Dx ME Panel were successful.

A negative signal of the Internal Control does not negate any positive results for detected and identified targets, but it does invalidate all negative results in the analysis. Therefore, the test should be repeated if the Internal Control signal is negative.

# ii. Recommended External Control:

All external quality control testing should be performed in accordance with local, state, and federal regulations or accreditation organizations and should follow the laboratory standard quality control procedures.

Control materials are not provided with the QIAstat-Dx ME Panel.

b. Sample Stability

A sample stability study was conducted to demonstrate that clinical cerebrospinal fluid specimens may be stored at room temperature (15 to $2 5 \mathrm { ^ \circ C }$ ) for up to twenty-four hours, at refrigerated conditions (2 to $8 ^ { \mathrm { o } } \mathrm { C }$ ) for up to seven days or in the freezer (-15 to $- 2 5 \mathrm { ^ \circ C }$ ) and (- $- 6 0 \mathrm { { ^ \circ C } }$ to $- 9 0 ^ { \mathrm { o } } \mathrm { C }$ ) for up to two and four months, respectively, before testing with the QIAstat-Dx ME Panel without affecting the performance.

Positive samples used in this study were prepared in clinical negative CSF matrix spiked with targets at $2 . 5 \mathrm { x }$ LoD concentration (Table 12). At least three small pools of clinical matrix were used for sample preparation. Pathogen-negative samples without added pathogens were not assessed in this study since negative results from the non-spiked targets have been analyzed. Samples were prepared in bulk and divided into single aliquots that were tested after specific storage conditions.

Table 12: Sample Stability Study Organism Panels   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>MYA-4877</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>13813</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>801534</td></tr><tr><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33400</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis (encapsulated)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>13090.00</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>10211</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli K1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>700973.000</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>804351</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810107CF</td></tr></table>

Right after sample preparation, at least twenty replicates of each sample were tested as a referenced condition (T0). The rest of the prepared aliquots were then stored and tested.

Each storage condition was considered suitable for storage if all spiked pathogens at $2 . 5 \mathrm { x }$ LoD generated a detection rate of $2 9 5 \%$ $( \geq 1 9 / 2 0 )$ of the tested aliquots per storage condition and, additionally, all pathogens not included in the respective sample resulted in a negative signal.

All panel analytes included in Sample Mixes 1, 2 and 3 passed the study acceptance criteria indicating a true positive agreement of at least $9 5 \%$ for all time points assessed except for Neisseria meningitidis (ATCC; 13090) in Sample Mix 2, which showed a detection rate ${ < } 9 5 \%$ (18/20) at 121 days (- $- 6 0 \mathrm { { ^ \circ C } }$ to $- 9 0 ^ { \mathrm { o } } \mathrm { C }$ ). An investigation was conducted to assess the $\mathrm { C t }$ drift of Neisseria meningitidis (ATCC; 13090) across the different timepoints (including T0) for the storage condition $- 6 0 \mathrm { { ^ \circ C } }$ to $- 9 0 \mathrm { { ^ \circ C } }$ . Results of this analysis showed that the regression line for this target was well within the allowable drift limit and was deemed to be statistically stable at all the timepoints assessed at the storage condition $- 6 0 \mathrm { { ^ \circ C } }$ to $- 9 0 \mathrm { { ^ \circ C } }$ . Therefore, Neisseria meningitidis test results at 121 days (- $- 6 0 \mathrm { { ^ \circ C } }$ to $- 9 0 \mathrm { { ^ \circ C } }$ ) were accepted.

A detection rate of ${ < } 9 5 \%$ was observed for Enterovirus at 31 days. Investigation concluded cartridges used for Enterovirus exceeded their shelf life. Results obtained during subsequent timepoints passed acceptance criteria.

Data from the study was adequate to support the storage claims included in the labeling. The results of this study support claims for storage of cerebrospinal fluid for 24 hours at $1 5 { - } 2 5 ^ { \circ } \mathrm { C }$ or refrigerated for up to seven days at $2 { - } 8 ^ { \circ } \mathrm { C }$ prior to testing samples with the QIAstat-Dx ME Panel. No claim for use of frozen specimens was included in the Package Labeling.

c. Freeze/Thaw Study

A fresh vs frozen study was performed to demonstrate equivalent assay performance when samples are frozen at $- 6 0 ^ { \circ } \mathrm { C }$ to $- 9 0 ^ { \circ } \mathrm { C }$ for at least 12 hours and subjected to 3 freezethaw cycles prior testing and to support the use of frozen samples during the validation studies.

For the fresh vs. frozen study, analytical samples were manufactured using organisms from commercial suppliers (Table 15). Samples consisted of mixtures of organisms spiked into pre-screened negative clinical CSF matrix at a final concentration of 5x LoD, 1x LoD and 0.1x LoD as defined by the Limit of Detection in Combined Samples.

Sample mixes were prepared in pools, divided into aliquots, and either immediately tested (fresh) or frozen at $- 6 0 ^ { \circ } \mathrm { C }$ to $- 9 0 ^ { \circ } \mathrm { C }$ for at least 12 hours. Samples were frozen twice with at least two hours between freeze/thaw cycles and then thawed again before testing at each timepoint with the QIAstat-Dx ME Panel for a total of 3 freeze/thaw cycles. Testing was conducted by a minimum of one operator using at least one lot of QIAstatDx ME Panel on 3 or more QIAstat-Dx Analyzer 1.0 instruments and replicated as described in Table 13.

Table 13: Number of Replicates Tested and Expected Outcome of Fresh vs Frozen Study   

<table><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Minimum Number of Replicatesper Strain per Storage Condition</td><td rowspan=1 colspan=1>Expected Results(% postive)</td></tr><tr><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>≥95</td></tr><tr><td rowspan=1 colspan=1>0.1x</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>&gt;0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td></tr></table>

The results (Table 14) of the freeze/thaw study support sample stability through three freeze/thaw cycles prior to testing with the QIAstat-Dx ME Panel.

Table 14: Freeze/Thaw Study Results Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">PercentAgreementFresh</td><td colspan="1" rowspan="1">PercentAgreementFrozen</td></tr><tr><td colspan="1" rowspan="3">Streptococcusagalactiae</td><td colspan="1" rowspan="3">ZeptoMetrix</td><td colspan="1" rowspan="3">801545</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">70.0%</td><td colspan="1" rowspan="1">80.0%</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">13813</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="6">Listeriamonocytogenes</td><td colspan="1" rowspan="3">ZeptoMetrix</td><td colspan="1" rowspan="3">801534</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">80.0%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">19115</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">70.0%</td><td colspan="1" rowspan="1">60.0%</td></tr><tr><td colspan="1" rowspan="6">Streptococcuspneumoniae</td><td colspan="1" rowspan="3">ZeptoMetrix</td><td colspan="1" rowspan="3">801439</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">30.0%</td><td colspan="1" rowspan="1">50.0%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">33400</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">9.00%</td><td colspan="1" rowspan="1">80.0%</td></tr><tr><td colspan="1" rowspan="6">Neisseriameningitidis(encapsulated)</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">13090</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">90.9%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">35561</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.0%</td></tr><tr><td colspan="1" rowspan="6">Haemophilusinfluenzae</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">10211</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">8142</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="6">Escherichiacoli K1</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">700973</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">60.0%</td><td colspan="1" rowspan="1">90.9%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">11775</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">70.0%</td><td colspan="1" rowspan="1">90.0%</td></tr><tr><td colspan="1" rowspan="6">Streptococcuspyogenes</td><td colspan="1" rowspan="3">ZeptoMetrix</td><td colspan="1" rowspan="3">804351</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">20.0%</td><td colspan="1" rowspan="1">40.0%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">19615</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Enterovirus A</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810107CF</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">40.0%</td><td colspan="1" rowspan="1">50.0%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">VR-1801</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">86.7%</td><td colspan="1" rowspan="1">83.3%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">20.0%</td><td colspan="1" rowspan="1">50.0%</td></tr><tr><td colspan="1" rowspan="6">Cryptococcusneoformans</td><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">MYA-4567</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">40.0%</td><td colspan="1" rowspan="1">40.0%</td></tr><tr><td colspan="1" rowspan="3">ATCC</td><td colspan="1" rowspan="3">208821</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">0.1x LoD</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="4" rowspan="1">Negative CSF Matrix</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

Carryover

A carryover study was performed to evaluate the potential occurrence of crosscontamination between consecutive runs when using the QIAstat-Dx ME Panel on the QIAstat-Dx Analyzer 1.0. Alternating CSF samples containing either high-positive $( 1 0 ^ { 5 } -$ $1 0 ^ { 6 }$ organism $/ \mathrm { m L }$ ) or no organisms were evaluated on two QIAstat-Dx Analyzer 1.0 instruments.

No carryover between samples was observed in the QIAstat-Dx ME Panel, demonstrating that the system design and recommended sample handling and testing practices are effective in preventing unexpected results due to carryover or cross-contamination between samples.

7. Detection Limit: Individual Limit of Detection (LoD)

The LoD was defined as the lowest concentration at which $2 9 5 \%$ of samples generate a positive result. Dilutions of 26 commercial pathogen strains (Table 15) were prepared using artificial cerebrospinal fluid. Equivalence between the artificial CSF matrix and native CSF matrix was established in a separate matrix equivalency study (see Section B.2).

Table 15: Individual Spiked Organism Limit of Detection Results   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">IndividualLoD</td><td colspan="1" rowspan="1">DetectionRate</td></tr><tr><td colspan="5" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli K1</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700973</td><td colspan="1" rowspan="1">348 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli K1</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">11775</td><td colspan="1" rowspan="1">786 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10211</td><td colspan="1" rowspan="1">316 CFU/mL</td><td colspan="1" rowspan="1">32/32</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8142</td><td colspan="1" rowspan="1">2540 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801534</td><td colspan="1" rowspan="1">1860 CFU/mL</td><td colspan="1" rowspan="1">21/21</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">19115</td><td colspan="1" rowspan="1">6640 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis(encapsulated)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13090</td><td colspan="1" rowspan="1">0.0828 CFU/mL</td><td colspan="1" rowspan="1">31/32</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis(encapsulated)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">35561</td><td colspan="1" rowspan="1">13.3 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801545</td><td colspan="1" rowspan="1">1750 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13813</td><td colspan="1" rowspan="1">3380 CFU/mL</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801439</td><td colspan="1" rowspan="1">714 CFU/mL</td><td colspan="1" rowspan="1">29/30</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">33400</td><td colspan="1" rowspan="1">0.622 CFU/mL</td><td colspan="1" rowspan="1">29/29</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">804351</td><td colspan="1" rowspan="1">1800 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">19615</td><td colspan="1" rowspan="1">91 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="5" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Enterovirus A</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810107CF</td><td colspan="1" rowspan="1">3.79 TCID50/mL</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="1">Enterovirus A</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1801</td><td colspan="1" rowspan="1">160 TCID50/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Enterovirus B</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810019CF</td><td colspan="1" rowspan="1">89.1 TCID50/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Enterovirus B</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810017CF</td><td colspan="1" rowspan="1">43.6 TCID50/mL</td><td colspan="1" rowspan="1">28/29</td></tr><tr><td colspan="1" rowspan="1">Enterovirus C</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1023</td><td colspan="1" rowspan="1">15.8 TCID50/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Enterovirus C</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-583</td><td colspan="1" rowspan="1">4.99 TCID50/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-836</td><td colspan="1" rowspan="1">49.9 TCID50/mL</td><td colspan="1" rowspan="1">30/31</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4567</td><td colspan="1" rowspan="1">506 TCID50/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="5" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4567</td><td colspan="1" rowspan="1">2210 CFU/mL</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">208821</td><td colspan="1" rowspan="1">164 CFU/mL</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4094</td><td colspan="1" rowspan="1">13200 CFU/mL</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4877</td><td colspan="1" rowspan="1">2600 CFU/mL</td><td colspan="1" rowspan="1">29/29</td></tr></table>

# i. Multi-Spiked Limit of Detection (LoD)

The LoD for each pathogen was assessed in multiple-spiked samples containing up to five organisms to determine if multiple analytes in a single CSF sample during the analytical studies for potential loss of detection when multiple targets are present in clinical specimens (co-infections). Multiplexed samples were tested at 5x LoD, 1x LoD, and 0.1x LoD concentrations following the LoDs established in the individual LoD Study.

Samples consisting of three different organism mixes with up to five targeted organisms each were compared to contrived single-spike samples for the following representative panel analytes: bacterium (E. coli K1), yeast (C. neoformans), DNA virus (HSV-1), and RNA virus $\mathrm { ( H P e V ) }$ . All samples were prepared in a CSF matrix. Serial dilutions were prepared for both single-spike samples and multiple spiked samples containing the corresponding analyte for comparison.

All sample dilutions were prepared using artificial CSF. Testing was conducted over 3 days by different operators using at least 5 different QIAstat-Dx ME Panel lots on 3 or more QIAstat-Dx Analyzer 1.0 instruments.

In addition to meeting the acceptance criteria for qualitative performance, there should be no significant differences in assay metrics (i.e., Ct values) between samples prepared in the individual and combined samples.

Table 16: Multiplexed Limit of Detection Expected Results   

<table><tr><td rowspan=1 colspan=1>Concentration*</td><td rowspan=1 colspan=1>Minimum Number ofReplicates per Strain</td><td rowspan=1 colspan=1>Expected Results(% postive)</td></tr><tr><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1x</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>≥95</td></tr><tr><td rowspan=1 colspan=1>0.1x</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10-90</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td></tr></table>

\*LoD as determined in individual LoD study.

No significant differences that negatively impacted assay performance were observed.

8. Assay Cut-Off: Not applicable.

# B Comparison Studies:

1. Method Comparison with Predicate Device: N/A

2. Matrix Comparison:

A matrix equivalency study was conducted to compare the performance of analytical samples prepared in negative clinical cerebrospinal fluid (cCSF) matrix to sample prepared in artificial CSF matrix (aCSF). A total of 26 pathogen strains (at least two different strains per panel target) were tested in combined samples spiked in true-negative cCSF. Clinical CSF was sourced from commercial suppliers and was tested prior to its use in mix manufacture for true-negativity with either the QIAstat-Dx ME Panel or an alternative method.

Samples were prepared by spiking pathogens in multiplex sample mixes containing up to 5 organisms per sample in cCSF at 5x LoD, 1x LoD and 0.1x LoD concentrations following the Limit of Detection in Combined Samples study (See Section 7.ii.). The concentration of the pathogen strain that achieved a result around the detection limit (1x LoD) was assessed by testing a minimum of 30 replicates; 5x LoD and $0 . 1 \mathbf { x }$ LoD concentrations were assessed by testing a minimum of 10 replicates. Some pathogens required an additional round of testing to establish the dilution to achieve LoD, in addition to the initial testing described.

Testing for each concentration was executed during at least 3 different days by different operators using four different lots of QIAstat-Dx ME Panel cartridges executed on 3 or more QIAstat-Dx Analyzers. Negative samples consisted of unspiked cCSF, and a minimum of 10 replicates were tested.

To demonstrate equivalence, in addition to meeting the acceptance criteria for qualitative performance, there should be no significant differences in assay metrics (i.e., Ct values). The difference in mean target Ct values between the two sample matrix types, evaluated with an ANOVA fitted test, will not be statistically significant when p-value ${ > } 0 . 0 5$ . If statistically significant, then the mean difference in Ct value of the combined sample in cCSF compared to the combined samples in aCSF should be within $\pm 2 \mathbf { x }$ SD of the combined sample in cCSF to be deemed equivalent.

Of the 26 pathogen strains evaluated, the LoD concentration was equivalent between combined (multi-spiked) samples in aCSF and cCSF in 17 strains. Nine pathogen strains (Streptococcus pneumoniae (ZeptoMetrix 0801439), Streptococcus pyogenes (ZeptoMetrix 0804351), Enterovirus B (ZeptoMetrix 0810019CF), Enterovirus C (ATCC VR-583), Enterovirus D (ATCC VR-1823)) were not equivalent and had a new LoD concentration determined in cCSF.

# C Clinical Studies:

# 1. Prospective Clinical Sensitivity:

The clinical performance of the Qiagen QIAstat-Dx ME Panel to detect and identify target bacteria, fungi, and viruses from cerebrospinal fluid samples was assessed in a multi-site study using residual de-identified CSF samples. Testing was conducted across 13 geographically diverse sites (10 U.S. and 3 European) using residual cerebrospinal fluid specimens from patients with signs and symptoms of meningitis and/or encephalitis. Performance for the QIAstat-Dx ME Panel was compared to the predicate device, BioFire FilmArray ME Panel except for the target not detected by the predicate device, Streptococcus pyogenes, which were compared to PCR followed by bidirectional sequencing.

Testing of prospectively collected residual specimens occurred between March 2022 – March 2023. A total of 1737 specimens were enrolled with 205 withdrawn, most commonly for ineligibility (not meeting inclusion/exclusion criteria). To supplement results of the prospective clinical study, frozen archived positive specimens were blinded and mixed with prospective specimens at clinical sites. Contrived samples were assigned a unique blinded specimen ID prior to being randomized and shipped to investigator sites along with negative contrived samples.

Prior to testing at the clinical sites, prospective cerebrospinal fluid specimens were stored up to 24 hours at room temperature $( 1 5 . 2 5 ^ { \circ } \mathrm { C } )$ , up to 7 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ , or frozen $( \leq - 7 0 ^ { \circ } \mathrm { C } \pm 2 0 ^ { \circ } \mathrm { C } )$ prior to testing. After comparator testing at the study site, all specimens were frozen $( { \le } { - } 7 0 ^ { \circ } \mathrm { C }$ $\pm 2 0 ^ { \circ } \mathrm { C } )$ and stored before being shipped to central testing sites for additional comparator testing or discordance testing, if required. Discordant results were analyzed using Standard of Care (SoC) culture when sufficient sample volume remained.

The study inclusion criteria included the following:

Sites were advised to make best effort to provide unique specimens (only one sample per patient).   
Specimens must have met the laboratory testing criteria for individuals with signs and/or symptoms of meningitis and/or encephalitis. Specimens consisting of only CSF obtained via lumbar puncture were collected.   
Specimens were residual and should not have undergone more than three (3) freeze/thaw cycles.   
Minimum $5 0 0 \mu \mathrm { L }$ of residual volume.   
Fresh specimens should be stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7 days. Ship within to testing sites within 48 hours of collection (for testing at sponsor site / testing site within total 72 hours post collection). Frozen specimens should be stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ $( \pm 2 0 ^ { \circ } \mathrm { C } )$ up to 2 months.

The study exclusion criteria included the following:

• Specimens not fitting inclusion criteria outlined above   
• Cerebrospinal fluid obtained from an external ventricular drain or shunt source   
• Specimen has been centrifuged   
• Obvious physical damage of banked residual specimen Repeat specimens from the same subject

Table 17: Demographic Summary for Evaluable Prospective Samples   

<table><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Variable</td><td colspan="1" rowspan="1">Subgroup</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">(%)</td></tr><tr><td colspan="1" rowspan="9">Prospective Fresh</td><td colspan="1" rowspan="7">Age Group</td><td colspan="1" rowspan="1">&lt;1 year</td><td colspan="1" rowspan="1">136</td><td colspan="1" rowspan="1">14.0%</td></tr><tr><td colspan="1" rowspan="1">1-17 years old</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">9.0%</td></tr><tr><td colspan="1" rowspan="1">18-44 years old</td><td colspan="1" rowspan="1">284</td><td colspan="1" rowspan="1">29.2%</td></tr><tr><td colspan="1" rowspan="1">45-64 years old</td><td colspan="1" rowspan="1">266</td><td colspan="1" rowspan="1">27.4%</td></tr><tr><td colspan="1" rowspan="1">65-84 years old</td><td colspan="1" rowspan="1">187</td><td colspan="1" rowspan="1">19.2%</td></tr><tr><td colspan="1" rowspan="1">≥85 years old</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.1%</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td></tr><tr><td colspan="1" rowspan="2">Gender</td><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">498</td><td colspan="1" rowspan="1">51.2%</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">474</td><td colspan="1" rowspan="1">48.8%</td></tr><tr><td colspan="1" rowspan="10">Prospective Frozen</td><td colspan="1" rowspan="7">Age Group</td><td colspan="1" rowspan="1">&lt;1 year</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">4.9%</td></tr><tr><td colspan="1" rowspan="1">1-17 years old</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">7.4%</td></tr><tr><td colspan="1" rowspan="1">18-44 years old</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">24.1%</td></tr><tr><td colspan="1" rowspan="1">45-64 years old</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">31.5%</td></tr><tr><td colspan="1" rowspan="1">65-84 years old</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="1">28.3%</td></tr><tr><td colspan="1" rowspan="1">≥85 years old</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">3.6%</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td></tr><tr><td colspan="1" rowspan="3">Gender</td><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">271</td><td colspan="1" rowspan="1">49.1%</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">280</td><td colspan="1" rowspan="1">50.7%</td></tr><tr><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td></tr><tr><td colspan="1" rowspan="2">Combined</td><td colspan="1" rowspan="2">Age Group</td><td colspan="1" rowspan="1">&lt;1 year</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">10.7%</td></tr><tr><td colspan="1" rowspan="1">1-17 years old</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">8.4%</td></tr><tr><td colspan="1" rowspan="8">−</td><td colspan="1" rowspan="5">−</td><td colspan="1" rowspan="1">18-44 years old</td><td colspan="1" rowspan="1">417</td><td colspan="1" rowspan="1">27.4%</td></tr><tr><td colspan="1" rowspan="1">45-64 years old</td><td colspan="1" rowspan="1">440</td><td colspan="1" rowspan="1">28.9%</td></tr><tr><td colspan="1" rowspan="1">65-84 years old</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">22.5%</td></tr><tr><td colspan="1" rowspan="1">≥85 years old</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">2.0%</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.1%</td></tr><tr><td colspan="1" rowspan="3">Gender</td><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">769</td><td colspan="1" rowspan="1">50.5%</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">754</td><td colspan="1" rowspan="1">49.5%</td></tr><tr><td colspan="1" rowspan="1">Not Available</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td></tr></table>

Table 18: Prospective Clinical Performance Summary   

<table><tr><td colspan="8" rowspan="1">lable lo.Prospective Clinical Periormance Summary</td></tr><tr><td colspan="2" rowspan="2">Target</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="3">Escherichia Coli K1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/3 a</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">20.8%-93.9%</td><td colspan="1" rowspan="1">969/969</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/1 a</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%-79.3%</td><td colspan="1" rowspan="1">551/551</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">15.0%-85.0%</td><td colspan="1" rowspan="1">1520/1520</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.7%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/1 b</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.0%-79.3%</td><td colspan="1" rowspan="1">970/971b</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.4%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">51.0%-100.0%</td><td colspan="1" rowspan="1">546/548b</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">98.7%-99.9%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">37.6%-96.4%</td><td colspan="1" rowspan="1">1516/1519</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.4%-99.9%</td></tr><tr><td colspan="1" rowspan="3">Listeriamonocytogenes</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">20.7%-100.0%</td><td colspan="1" rowspan="1">971/971</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">3/4c</td><td colspan="1" rowspan="1">75.0%</td><td colspan="1" rowspan="1">30.1%-95.4%</td><td colspan="1" rowspan="1">548/548</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">37.6%-96.4%</td><td colspan="1" rowspan="1">1519/1519</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.7%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Neisseriameningitidis(encapsulated)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">20.7%-100.0%</td><td colspan="1" rowspan="1">971/971</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">551/552d</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.0%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">20.7%-100.0%</td><td colspan="1" rowspan="1">1522/1523</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.6%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Streptococcusagalactiae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">34.2%-100.0%</td><td colspan="1" rowspan="1">970/970</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">20.7%-100.0%</td><td colspan="1" rowspan="1">551/551</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">43.9%-100.0%</td><td colspan="1" rowspan="1">1521/1521</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.7%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">20.7%-100.0%</td><td colspan="1" rowspan="1">845/848e</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.0%-99.9%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">64.6%-100.0%</td><td colspan="1" rowspan="1">515/517</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">98.6%-99.9%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">67.6%-100.0%</td><td colspan="1" rowspan="1">1360/1365</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99.1%-99.8%</td></tr><tr><td colspan="1" rowspan="3">Streptococcuspyogenes</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">778/778</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.5%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">513/513</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1291/1291</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.7%-100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="3">Enterovirus (EV)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">18/20f</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">69.9%-97.2%</td><td colspan="1" rowspan="1">951/952f</td><td colspan="1" rowspan="1">99.9.%</td><td colspan="1" rowspan="1">99.4%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">51.0%-100.0%</td><td colspan="1" rowspan="1">548/548</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">22/24</td><td colspan="1" rowspan="1">91.7%</td><td colspan="1" rowspan="1">74.2%-97.7%</td><td colspan="1" rowspan="1">1499/1500</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.6%-100.0%</td></tr><tr><td colspan="4" rowspan="1">Fungi / Yeast</td><td colspan="4" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Cryptococcus gattii /Cryptococcusneoformans (notdifferentiated)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/58</td><td colspan="1" rowspan="1">40.0%</td><td colspan="1" rowspan="1">11.8%-76.9%</td><td colspan="1" rowspan="1">965/967 g</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.2%-99.9%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">34.2%-100.0%</td><td colspan="1" rowspan="1">550/550</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/7</td><td colspan="1" rowspan="1">57.1%</td><td colspan="1" rowspan="1">25.0%-84.2%</td><td colspan="1" rowspan="1">1515/1517</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.5%-100.0%</td></tr></table>

a For the prospective fresh Escherichia coli $K I$ discordant sample, no organisms were detected with PCR/BDS. For the frozen Escherichia coli K1 discordant sample no organisms were detected with bacterial culture.

b For the prospective fresh Haemophilus influenzae discordant sample, no organisms were detected by the SoC bacterial culture and testing with PCR/BDS. Of the three (3) false positive Haemophilus influenzae samples, no organisms were detected in one fresh and one frozen by SoC culture and PCR/BDS was also negative. No additional testing results associated with the final frozen false positive sample were available.

c For the frozen Listeria monocytogenes discordant sample the negative result was confirmed positive with SoC culture and LDT result was positive.

d For the frozen Neisseria meningitidis (encapsulated) sample, no organisms were detected by SoC culture and LDT testing with PCR/BDS also returned a negative result for this sample.

e Of the five (5) false positive Streptococcus pneumoniae samples, no organisms were detected in four (3 fresh and 1 frozen) prospective samples with SoC culture. One prospective frozen sample had no SoC result available. However, an FDA cleared method conducted as part of the study also produced a negative result.

f For the prospective fresh Enterovirus discordant samples, no organisms were detected in one sample by two independent SoC LDT assays. The negative result for the second sample returned a negative result with PCR/BDS. For the prospective fresh false positive Enterovirus sample, a negative result was returned when tested with PCR/BDS.

g Of the three false negative Cryptococcus gattii / Cryptococcus neoformans (not differentiated) results, no organisms were detected in two fresh samples with fungal culture and PCR/BDS. The remaining false negative fresh sample was confirmed negative for Cryptococcus gattii / Cryptococcus neoformans (not differentiated) with PCR/BDS. Of the two false positive results, no organisms were detected for one fresh sample with PCR/BDS. No organisms were detected in the second fresh sample with SoC fungal culture.

# 2. Retrospective Samples

Several targets detected by the QIAstat-Dx ME Panel were not encountered during the prospective clinical study in sufficient numbers to demonstrate assay performance.

Therefore, the prospective clinical study was supplemented with additional testing using retrospective positive samples characterized previously by a standard of care molecular assay. A summary of the demographic information for retrospective samples is presented in Table 19.

Table 19: Retrospective Samples Demographic Data   

<table><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Subgroup</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>(%)</td></tr><tr><td rowspan=7 colspan=1>Archived</td><td rowspan=5 colspan=1>Age Group</td><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>26.8%</td></tr><tr><td rowspan=1 colspan=1>1-17 years old</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>22.0%</td></tr><tr><td rowspan=1 colspan=1>18-44 years old</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>29.3%</td></tr><tr><td rowspan=1 colspan=1>45-64yearssold</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>12.2%</td></tr><tr><td rowspan=1 colspan=1>65-84 years old</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9.8%</td></tr><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>46.3%</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>53.7%</td></tr></table>

The performance of the QIAstat-Dx ME Panel was determined by comparing to an FDAcleared molecular assay for all retrospective samples and summarized in Table 20. All retrospective samples generated a valid result in the first attempt.

Table 20: Retrospective Clinical Performance Summary   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP /(TP+FN)</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN /(TN+FP)</td><td rowspan=1 colspan=1>(%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Escherichia ColiK1</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34.2%-100.0%</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.0%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>61.0%-100.0%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.1%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Listeriamonocytogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.4%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Neisseriameningitidis(encapsulated)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43.9%-100.0%</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.8%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>70.1%-100.0%</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.36%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>51.0%-100.0%</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>82.4%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspyogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.7%-100.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterovirus (EV)</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>70.1%-100.0%</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.3%-100.0%</td></tr><tr><td rowspan=1 colspan=4>Fungi / Yeast</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Cryptococcusgattii /Cryptococcusneoformans (notdifferentiated)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>67.6%-100.0%</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.6%-100.0%</td></tr></table>

Contrived specimens were prepared for each target. Streptococcus pyogenes was not observed in the prospective study and is supported solely with contrived samples. Contrived samples were prepared by spiking five quantified strains representative of the genetic diversity of each pathogen (Table 21) at a concentration of $2 \mathbf { x }$ and 5x LoD in negative cerebrospinal fluid. Contrived specimens were mixed with prospective specimens, stored at - $7 0 \mathrm { { } ^ { \circ } C }$ $( \pm 2 0 ^ { \circ } \mathrm { C } )$ , and tested in batches as received by the clinical testing sites. It was not possible to test contrived samples alongside prospective samples for 676 samples. An equal number of negative samples were randomly mixed with the prepared samples to maintain blinded conditions.

Table 21: Contrived Target Strains   

<table><tr><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalogue ID</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii /Cryptococcus neoformans</td><td colspan="1" rowspan="1">WM629 [CBS 10079]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4567</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii /Cryptococcus neoformans</td><td colspan="1" rowspan="1">A1M R272</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4094</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii /Cryptococcus neoformans</td><td colspan="1" rowspan="1">C. neoformans H99</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">208821</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii /Cryptococcus neoformans</td><td colspan="1" rowspan="1">Alg254</td><td colspan="1" rowspan="1">BEI Resources</td><td colspan="1" rowspan="1">NR-50198</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii /Cryptococcus neoformans</td><td colspan="1" rowspan="1">A6MR38 [CBS 11545]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">MYA-4877</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Coxsackievirus A16</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810107CF</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Coxsackievirus B5</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810019CF</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">G-12 NIAID V-020-002-062</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1023</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">US/M0/14-18947</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1823</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">EV 70, species D, strain J670/71</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-836</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli Kl</td><td colspan="1" rowspan="1">Strain C5 [Bort]; O18ac:K1:H7</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700973</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli Kl</td><td colspan="1" rowspan="1">O-16, F1119-41</td><td colspan="1" rowspan="1">BEI Resources</td><td colspan="1" rowspan="1">NR-17674</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">K1 O-2, U9-41</td><td colspan="1" rowspan="1">BEI Resources</td><td colspan="1" rowspan="1">NR-17666</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli K1</td><td colspan="1" rowspan="1">NCDC F 11119-41</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">23511</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli Kl</td><td colspan="1" rowspan="1">NCTC 9001. Serovar O1:K1:H7</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">11775</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">type b (cap)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10211</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Type a [strain AMC 36-A-3]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9006</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Type f [strain GA-1264]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700223</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">AMC 36-A-7</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8142</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Type c [strain C 9007]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">49699</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Strain Li2</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">19115</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Serotype 1/2b</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">801534</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Strain Li 20</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">19111</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">2011L-2676</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2659</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Serotype 4b</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">804339</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M2092</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13090</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M-112</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">35561</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Serotype B. M997 [S-3250-L]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13092</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Serotype D. M158 [37A]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13113</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M-1574 [199/W135]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43744</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">G19 group B</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13813</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Z019</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801545</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">type III-ST283</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">31475</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Z023</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801556</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">2603 V/R. Serotype V</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-611</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">19F</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801439</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Serotype 1. NCTC 7465</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">33400</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Serotype 5. SPN1439-106[Colombia 5-19]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-341</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Serotype 11A. Type 43</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">10343</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Serotype 14. VH14</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700672</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Z472; Serotype M1</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">804351</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Bruno [CIP 104226]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">19615</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Z018; Serotype M58</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">801512</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Lancefield's group A / C203 S</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">14289</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Serotype M1. MGAS 5005</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-947</td></tr></table>

Table 22: Contrived Samples Performance Summary   

<table><tr><td colspan="2" rowspan="2">Target</td><td colspan="3" rowspan="1">Positive Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP /(TP+FN)</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="5" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="3">Escherichia Coli K1</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">92.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.6%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">85/85</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">95.7%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Haemophilus influenzae</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">57/57</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.7%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.4%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">93/93</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.0%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Listeria monocytogenes</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">47/49</td><td colspan="1" rowspan="1">95.9%</td><td colspan="1" rowspan="1">86.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.8%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">85/87</td><td colspan="1" rowspan="1">97.7%</td><td colspan="1" rowspan="1">92.0%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Neisseria meningitidis(encapsulated)</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">46/48</td><td colspan="1" rowspan="1">95.8%</td><td colspan="1" rowspan="1">86.0%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">87.1%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">85/88</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">90.5%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Streptococcus agalactiae</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">92.7%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">91.0%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">88/88</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">95.8%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">55/57</td><td colspan="1" rowspan="1">96.5%</td><td colspan="1" rowspan="1">88.1%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">91.0%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">94/96</td><td colspan="1" rowspan="1">97.9%</td><td colspan="1" rowspan="1">92.7%-100.0%</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">47/49</td><td colspan="1" rowspan="1">95.9%</td><td colspan="1" rowspan="1">86.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">91.2%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">87/89</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">92.2%-100.0%</td></tr><tr><td colspan="5" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="3">Enterovirus (EV)</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">48/49</td><td colspan="1" rowspan="1">98.0%</td><td colspan="1" rowspan="1">89.3%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">91.0%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">87/88</td><td colspan="1" rowspan="1">98.9%</td><td colspan="1" rowspan="1">93.8%-100.0%</td></tr><tr><td colspan="5" rowspan="1">Fungi / Yeast</td></tr><tr><td colspan="1" rowspan="3">Cryptococcus gattii /Cryptococcus neoformans(not differentiated)</td><td colspan="1" rowspan="1">2xLoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">91.4%-100.0%</td></tr><tr><td colspan="1" rowspan="1">5xLoD</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.8%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">79/79</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">95.4%-100.0%</td></tr></table>

D Clinical Cut-Off: Not applicable.

# E Expected Values/Reference Range:

The QIAstat-Dx ME Panel assay prospective clinical study was conducted to evaluate clinical performance of the device using cerebrospinal fluid samples. The number and percentage of positive results as determined by the QIAstat-Dx ME Panel, stratified by age group are presented in Table 23. Overall, 1527 specimens were included in the prevalence assessment and the QIAstat-Dx ME Panel detected at least one organism in a total of 65 prospective specimens analyzed in the study $4 . 3 \%$ positivity rate).

Table 23: Expected Values by Age Group (Prospective Samples)   

<table><tr><td colspan="1" rowspan="2">Pathogen</td><td colspan="1" rowspan="2">Overall</td><td colspan="7" rowspan="1">Age (years)</td></tr><tr><td colspan="1" rowspan="1">&lt;1</td><td colspan="1" rowspan="1">1-17</td><td colspan="1" rowspan="1">18-44</td><td colspan="1" rowspan="1">45-64</td><td colspan="1" rowspan="1">65-84</td><td colspan="1" rowspan="1">&gt;85</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="9" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli K1</td><td colspan="1" rowspan="1">0.1%(2)</td><td colspan="1" rowspan="1">100.0%(2/2)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">0.5%(7)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.2%(1)</td><td colspan="1" rowspan="1">1.1%(5)</td><td colspan="1" rowspan="1">0.3%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="1" rowspan="1">Listeriamonocytogenes</td><td colspan="1" rowspan="1">0.3%(4)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.5%(2)</td><td colspan="1" rowspan="1">0.3%(1)</td><td colspan="1" rowspan="1">3.2%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitidies(encapsulated)</td><td colspan="1" rowspan="1">0.1%(2)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.2%(1)</td><td colspan="1" rowspan="1">0.2%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">0.2%(3)</td><td colspan="1" rowspan="1">0.6%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.5%(2)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1.1%(17)</td><td colspan="1" rowspan="1">1.8%(3)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.7%(3)</td><td colspan="1" rowspan="1">1.6%(7)</td><td colspan="1" rowspan="1">1.2%(4)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">0.1%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.8%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="9" rowspan="1">Virus</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">1.5%(23)</td><td colspan="1" rowspan="1">7.4%(12)</td><td colspan="1" rowspan="1">2.3%(3)</td><td colspan="1" rowspan="1">1.2%(5)</td><td colspan="1" rowspan="1">0.2%(1)</td><td colspan="1" rowspan="1">0.3%(1)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">50.0%(1)</td></tr><tr><td colspan="9" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="1">Cryotococcus gattii/ Cryptococcusneoformans</td><td colspan="1" rowspan="1">0.4%(6)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.2%(1)</td><td colspan="1" rowspan="1">1.1%(5)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td><td colspan="1" rowspan="1">0.0%(0)</td></tr><tr><td colspan="9" rowspan="1">Overall Panel Result</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">95.7%(1462)</td><td colspan="1" rowspan="1">89.0%(145)</td><td colspan="1" rowspan="1">96.9%(124)</td><td colspan="1" rowspan="1">97.4%(407)</td><td colspan="1" rowspan="1">94.8%(419)</td><td colspan="1" rowspan="1">98.0%(336)</td><td colspan="1" rowspan="1">96.8%(30)</td><td colspan="1" rowspan="1">50.0%(1)</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">4.3%(65)</td><td colspan="1" rowspan="1">11.0%(18)</td><td colspan="1" rowspan="1">3.1%(4)</td><td colspan="1" rowspan="1">2.6%(11)</td><td colspan="1" rowspan="1">5.2%(23)</td><td colspan="1" rowspan="1">2.0%(7)</td><td colspan="1" rowspan="1">3.2%(1)</td><td colspan="1" rowspan="1">50.0%(1)</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.