# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K242526   
B Applicant Visby Medical, Inc.   
C Proprietary and Established Names Visby Medical Respiratory Health Test   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The Sars-Cov-2 Respiratory InfectionAnd Other MicrobialAgents When In A Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To demonstrate that the performance of the Visby Medical Respiratory Health Test is equivalent to the performance of the FDA cleared DiaSorin Molecular Simplexa COVID-19 & Flu A/B Direct (K220963).

# B Measurand:

• Influenza A RNA Influenza B RNA Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) RNA

C Type of Test: Qualitative RT-PCR

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Visby Medical Respiratory Health Test is a single-use (disposable), fully integrated, automated Reverse Transcription Polymerase Chain Reaction (RT-PCR) in vitro diagnostic test intended for the simultaneous qualitative detection and differentiation of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A, and influenza B RNA in nasopharyngeal swab and anterior nasal swab specimens from individuals with signs and symptoms of respiratory tract infection. Clinical signs and symptoms of respiratory tract infection due to SARS-CoV-2, influenza A, and influenza B can be similar.

The Visby Medical Respiratory Health Test is intended for use as an aid in the differential diagnosis of SARS-CoV-2, influenza A, and influenza B infection if used in conjunction with other clinical and epidemiological information, and laboratory findings. SARS-CoV-2, influenza A, and influenza B viral RNA are generally detectable in nasopharyngeal swab and anterior nasal swab specimens during the acute phase of infection. This test is not intended to detect influenza C virus infections.

Positive results are indicative of the presence of the identified virus, but do not rule out bacterial infection or co-infection with other organisms not detected by the test. The agent(s) detected by the Visby Medical Respiratory Health Test may not be the definitive cause of disease. Negative results do not preclude SARS-CoV-2, influenza A, or influenza B infection. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only For In Vitro Diagnostic Use Only

D Special Instrument Requirements: Single-use test without special instrument requirement.

# IV Device/System Characteristics:

# A Device Description:

The Visby Medical Respiratory Health Test (referred to as “Visby Test” hereafter) is a single-use (disposable), fully integrated, compact device containing a reverse transcription polymerase chain reaction (RT-PCR) based assay for qualitative detection and differentiation of RNA from influenza A, influenza B, and SARS-CoV-2 viruses from nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens.

The test system includes the Visby Medical Respiratory Health Test, the reusable Visby Medical power adaptor, the Visby Medical Respiratory Health Buffer Tube, and fixed-volume transfer pipette. Operators are instructed to collect the sample and elute it into the Visby Respiratory Health Buffer.

A test operator transfers the collection media (containing patient specimen) into the sample port of the device using the provided fixed-volume pipette and then slides a purple switch on the front of the device to both close the sample port and initiate the fully automated testing process. The device processes NPS and ANS samples by automatically performing all steps required to complete lysis, reverse transcription, polymerase chain reaction, and amplicon detection.

There are LED lights on the lower left corner of the device that are used to communicate the test status. Stable white light in the shape of a power symbol will appear indicating the test is plugged in. In addition, three progress lights to the right of the power light will each blink for ${ \sim } 1 0$ minutes in sequence and then provide a stable white light to indicate the progression of the test. At the conclusion of the test, a green LED under “DONE” next to the white lights signals that the test run completed successfully.

When the run has completed, the operator visually interprets the status lights on the front of the device and the colorimetric output in the detection window. A valid test will have both an illuminated green LED under “DONE” and a purple spot adjacent to “RESULTS VALID” in the detection window. For valid tests, a purple spot next to the name of a pathogen indicates a positive result for that target.

In the case of a lack of a purple “RESULTS VALID” spot, the test is invalid, and the operator is instructed to repeat the test. If the subsequent test is also invalid, the operator is instructed to collect a new sample and repeat the test with a new device.

The device has built-in procedural controls. These include an internal process control and built-in electronic control. The internal control is designed to ensure that all steps in the testing process including reverse transcription, amplification of target sequences, and amplicon detection are working properly. The electronic control monitors if device is being operated outside of its temperature range or if there are any other device errors. The result of the internal process control is displayed in the detection window while the results of the electronic controls are displayed using the status lights.

Third-party external positive and negative controls are recommended for use with this test. These controls are not provided with the device and must be purchased separately by the customer directly from Microbiologics. The labeling states that the external controls “must be tested once with each new shipment received and once for each untrained operator.”

# B Principle of Operation:

The device houses all the reagents to perform the testing process. The sample enters a lysis module and rehydrates the RT enzyme and RT primers. The mixture then moves through a sample preparation module where viruses and human cells are simultaneously lysed, and RNA is reverse transcribed. The resulting fluid (containing cDNA) is then mixed with lyophilized PCR reagents containing the DNA polymerase enzyme and PCR primers. The PCR mixture (containing cDNA template and reagents) is then thermal cycled to amplify the targets, including human beta-2 microglobulin (B2M) RNA, which serves as an internal process control. If present, the amplified pathogen target (SARS-CoV-2, influenza A, and/or influenza B) and the amplified internal process control hybridize to specific probes located on a flow cell. For SARS-CoV-2, primer sets are included targeting the nucleocapsid (N) gene and the open reading frame 1ab (ORF-1ab) gene. The primer sets for influenza A target the matrix protein 1 (M1) gene. For influenza B, the primer set targets the nuclear export protein (NEP) and nonstructural protein 1 (NS1) genes. Detection of the target-specific PCR product is accomplished via an enzyme-linked colorimetric assay using streptavidin bound horseradish peroxidase (HRP) and a colorimetric substrate that forms a purple precipitate. Test results can be expected in approximately 30 minutes and are visually interpreted by the operator.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Simplexa COVID-19 & Flu A/B Direct

B Predicate 510(k) Number(s): K220963

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K242526</td><td colspan="1" rowspan="1">K220963</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Visby Medical RespiratoryHealth Test</td><td colspan="1" rowspan="1">DiaSorin Molecular SimplexaCOVID-19 &amp; Flu A/B Direct</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1">K242526</td><td colspan="1" rowspan="1">K220963</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">QOF</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">21 CFR 866.3981</td></tr><tr><td colspan="1" rowspan="1">Prescription Use Only</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Real-time Reverse TranscriptionPolymerase Chain Reaction (RT-PCR)</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The Visby Medical RespiratoryHealth Test is a single-use(disposable), fully integrated,automated Reverse TranscriptionPolymerase Chain Reaction (RT-PCR) in vitro diagnostic testintended for the simultaneousqualitative detection anddifferentiation of severe acuterespiratory syndrome coronavirus2 (SARS-CoV-2), influenza A,and influenza B RNA in</td><td colspan="1" rowspan="1">The DiaSorin Molecular SimplexaCOVID-19 &amp; Flu A/B Direct is areal-time RT-PCR assay intendedfor use on the LIAISON MDXinstrument for the in vitroqualitative detection anddifferentiation of nucleic acid fromsevere acute respiratory syndromecoronavirus 2 (SARS-CoV-2),influenza A virus and influenza Bvirus in nasopharyngeal swabs(NPS) from individuals with signs</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">from individuals with signs andsymptoms of respiratory tractinfection. Clinical signs andsymptoms of respiratory tractinfection due to SARS-CoV-2,influenza A, and influenza B canbe similar.The Visby Medical RespiratoryHealth Test is intended for use asan aid in the differentialdiagnosis of SARS-CoV-2,influenza A, and influenza Binfection if used in conjunctionwith other clinical andepidemiological information, andlaboratory findings. SARS-CoV-2, influenza A, and influenza Bviral RNA are generallydetectable in nasopharyngealswab and anterior nasal swabspecimens during the acute phaseof infection. This test is notintended to detect influenza Cvirus infections.Positive results are indicative ofthe presence of the identifiedvirus, but do not rule out bacterialinfection or co-infection withother organisms not detected bythe test. The agent(s) detected bythe Visby Medical RespiratoryHealth Test may not be thedefinitive cause of disease.Negative results do not precludeSARS-CoV-2, influenza A, orinfluenza B infection. The resultsof this test should not be used asthe sole basis for diagnosis,treatment, or other patientmanagement decisions.</td><td colspan="1" rowspan="1">The Simplexa COVID-19 &amp; FluA/B Direct assay is intended foruse as an aid in the differentialdiagnosis of SARS-CoV-2,influenza A and influenza Binfection. Negative results do notpreclude SARS-CoV-2, influenzaA or influenza B infection andshould not be used as the sole basisfor patient management decisions.Positive results do not rule outcoinfections with other organisms.Results should be combined withclinical observations, patienthistory, and epidemiologicalinformation.The Simplexa COVID-19 &amp; FluA/B Direct assay is intended foruse by qualified and trained clinicallaboratory personnel specificallyinstructed and trained in thetechniques of real-time PCR and invitro diagnostic procedures.</td></tr><tr><td colspan="1" rowspan="1">Specimen</td><td colspan="1" rowspan="1">Nasopharyngeal Swab;Anterior Nasal Swab</td><td colspan="1" rowspan="1">Nasopharyngeal Swab</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">SARS-CoV-2, influenza A virusand influenza B virus</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Internal processing control (IPC),External controls available.</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1">K242526</td><td colspan="1" rowspan="1">K220963</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">Visual</td><td colspan="1" rowspan="1">Automatic</td></tr></table>

# VI Standards/Guidance Documents Referenced:

1. 21 CFR 866.3981 - Special Controls   
2. ANSI AAMI IEC 60601-1-2:2014 [Including AMD 1:2021] 19-36   
3. IEC 60601-1-2: 2014 +Al: 2020   
4. IEC TR 60601-4-2 Edition 1.0 2016-05 19-19   
5. IEC 61010-1:2010   
6. IEC 61010-1:2010/AMD1:2016   
7. IEC 61010-1 Edition 3.1 2017-01 CONSOLIDATED VERSION 19-34   
8. IEC 61010-2-101:2018   
9. IEC 61010-2-010:2019   
10. ANSI AAMI IEC 62304:2006/Al:2016 13-79   
11. ANSI AAMI ISO 14971:2019 5-125

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

A precision/reproducibility study was conducted with the Visby Medical Respiratory Health Test at three CLIA-waived external sites by a total of six untrained operators (two operators at each site). Each operator tested a panel (Table 1) in triplicate each day across six non-consecutive days. The panel consisted of seven members (live virus for influenza A and influenza B, and inactivated virus for SARS-CoV-2) prepared by spiking organisms in negative pooled clinical swab matrix and applied to individual dry nasopharyngeal swabs. Three reagent lots were used in this study. Results were generated from a total of 108 replicates of each panel member (3 sites x 6 days x 2 operators/site x 3 replicates/operator/day).

Table 1. Precision/Reproducibility Study Sample Panel   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Concentration(copies/swab)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Influenza A, H1N1pdm, Brisbane/02/18</td><td rowspan=1 colspan=1>Low (1x LoD)</td><td rowspan=1 colspan=1>106</td></tr><tr><td rowspan=1 colspan=1>Influenza A, H1N1pdm, Brisbane/02/18</td><td rowspan=1 colspan=1>Moderate (4x LoD)</td><td rowspan=1 colspan=1>424</td></tr><tr><td rowspan=1 colspan=1>Influenza B Victoria, Washington/02/19</td><td rowspan=1 colspan=1>Low (1x LoD)</td><td rowspan=1 colspan=1>728</td></tr><tr><td rowspan=1 colspan=1>Influenza B Victoria,Washington/02/19</td><td rowspan=1 colspan=1>Moderate (4x LoD)</td><td rowspan=1 colspan=1>2912</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 USA-WA1/2020</td><td rowspan=1 colspan=1>Low (1x LoD)</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 USA-WA1/2020</td><td rowspan=1 colspan=1>Moderate (4x LoD)</td><td rowspan=1 colspan=1>400</td></tr></table>

A summary of percent agreement with expected results for each panel member by site and overall is presented in the following table. This study demonstrates that untrained users could perform the Visby Test accurately including testing samples with organism concentrations near the assay LoD.

Table 2. Precision/Reproducibility Study – Site to Site Results   

<table><tr><td rowspan=2 colspan=1>Panel Member</td><td rowspan=1 colspan=3>% Agreement withExpected Results (Count)</td><td rowspan=1 colspan=2>Overall Agreement</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>% Agreement(count)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A1x LoD</td><td rowspan=1 colspan=1>94.4 %(34/36)</td><td rowspan=1 colspan=1>100 %(36/36) a</td><td rowspan=1 colspan=1>88.9%(32/36)</td><td rowspan=1 colspan=1>94.4%(102/108) a</td><td rowspan=1 colspan=1>88.4%-97.4%</td></tr><tr><td rowspan=1 colspan=1>Influenza A4x LoD</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100.0%(108/108)</td><td rowspan=1 colspan=1>96.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B1x LoD</td><td rowspan=1 colspan=1>97.2 %(35/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>91.7%(33/36)</td><td rowspan=1 colspan=1>96.3%(104/108)</td><td rowspan=1 colspan=1>90.9%-98.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza B4x LoD</td><td rowspan=1 colspan=1>97.2 %(35/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>99.1%(107/108)</td><td rowspan=1 colspan=1>94.9%-99.8%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-21x LoD</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>88.9%(32/36)</td><td rowspan=1 colspan=1>95.4%(103/108)</td><td rowspan=1 colspan=1>89.6%-98.0%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-24x LoD</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100.0%(108/108)</td><td rowspan=1 colspan=1>96.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100 %(36/36)</td><td rowspan=1 colspan=1>100.0%(108/108)</td><td rowspan=1 colspan=1>96.6%-100.0%</td></tr></table>

a One low positive (1x LoD) influenza A sample was an unexpected positive for SARS-CoV-2.

Table 3. Precision/Reproducibility Study – Lot-to-Lot Results   

<table><tr><td rowspan=2 colspan=1>Panel Member</td><td rowspan=1 colspan=3>% Agreement withExpected Results (Count)</td><td rowspan=1 colspan=2>Overall Agreement</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>% Agreement(count)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A1x LoD</td><td rowspan=1 colspan=1>88.9%(32/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>94.4%(34/36) a</td><td rowspan=1 colspan=1>94.4%(102/108) a</td><td rowspan=1 colspan=1>88.4%-97.4%</td></tr><tr><td rowspan=1 colspan=1>Influenza A4x LoD</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(108/108)</td><td rowspan=1 colspan=1>96.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B1x LoD</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>91.7%(33/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>96.3%(104/108)</td><td rowspan=1 colspan=1>90.9%-98.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza B4x LoD</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>99.1%(107/108)</td><td rowspan=1 colspan=1>94.9%-99.8%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-21x LoD</td><td rowspan=1 colspan=1>88.9%(32/36)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>95.4%(103/108)</td><td rowspan=1 colspan=1>89.6%-98.0%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-24x LoD</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(108/108)</td><td rowspan=1 colspan=1>96.6%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(108/108)</td><td rowspan=1 colspan=1>96.6%-100.0%</td></tr></table>

a One low positive (1x LoD) influenza A sample was an unexpected positive for SARS-CoV-2.

Table 4. Precision/Reproducibility Study – Operator-to-Operator Results   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=3>Site 1: % Agreement withExpected Results (Count)</td><td rowspan=1 colspan=3>Site 2: % Agreement withExpected Results (Count)</td><td rowspan=1 colspan=3>Site 3: % Agreement withExpected Results (Count)</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Influenza A1x LoD</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>88.9%(16/18)</td><td rowspan=1 colspan=1>94.4%(34/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18) a</td><td rowspan=1 colspan=1>100.0%(36/36) a</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>83.3%(15/18)</td><td rowspan=1 colspan=1>88.9%(32/36)</td></tr><tr><td rowspan=1 colspan=1>Influenza A4x LoD</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td></tr><tr><td rowspan=1 colspan=1>Influenza B1x LoD</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>88.9%(16/18)</td><td rowspan=1 colspan=1>91.7%(33/36)</td></tr><tr><td rowspan=1 colspan=1>Influenza B4x LoD</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-21x LoD</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>88.9%(16/18)</td><td rowspan=1 colspan=1>88.9%16/18)</td><td rowspan=1 colspan=1>88.9%(32/36)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-24x LoD</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(18/18)</td><td rowspan=1 colspan=1>100.0%(36/36)</td></tr></table>

a One low positive (1x LoD) influenza A sample was an unexpected positive for SARS-CoV-2.

2. Linearity: Not applicable. This is a qualitative assay.

3. Analytical Specificity/Interference:

a. Analytical Reactivity (Inclusivity):

SARS-CoV-2 In Silico Analysis:

The inclusivity of the Visby Medical Respiratory Health Test was evaluated using in silico analysis of the forward primers, reverse primers, and probes for the SARS-CoV-2 target systems in relation to sequences available in the GISAID gene database. SARS-CoV-2 in silico analysis was performed using all available SARS-CoV-2 sequences in the GISAID database up to August 2, 2024 $( > 1 6 . 8$ million sequences). The SARS-CoV-2 sequences included the following lineages and variants of concern (VOC) or variants of interest (VOI) that may have important epidemiological, immunological, or pathogenic properties from a public health perspective: Delta, Alpha, Omicron BA.1, Omicron BA.2, Omicron BA.4, Omicron BA.5, Omicron BQ.1, Omicron BQ.1.1, Omicron BF.7, Gamma, Epsilon, Iota, Beta, Mu, Zeta, Kappa, Eta, Lamba, and Theta, and others.

All mismatches present in $5 5 \%$ of SARS-CoV-2 sequences (either overall or in the last 30, 60, 90, or 120 days prior to analysis) underwent further in silico analysis. None of these mismatches analyzed are predicted to significantly reduce the percent of primer/probe bound at the nominal annealing/detection temperatures of the device; the maximum predicted reduction in binding across all analyzed mismatches is $< 1 8 \%$ in only the N1 primer/probe set. These minor reductions in primer/probe binding, in combination with the redundancy of the N1 or Orf1ab gene assay targets is expected to have a minimal impact on the ability of the Visby Medical Respiratory Health Test to detect variants for SARS-CoV-2.

# Wet Testing:

An analytical reactivity study was conducted to evaluate the ability of the Visby Medical Respiratory Health Test to detect multiple SARS-CoV-2, influenza A, and influenza B strains that are temporally and geographically diverse. This study evaluated ten H1N1 (pandemic 2009) strains, $1 3 \mathrm { H } 3 \mathrm { N } 2$ strains, 12 influenza B strains (five each of Victoria and Yamagata lineages and two additional strains), and six SARS-CoV-2 strains as shown in Table 5 below. Inactivated strains were used for SARS-CoV-2 and cultured virus were used for influenza strains, unless noted. Each virus was spiked into negative pooled clinical swab matrix at analyte concentrations of 3x LoD. This spiked material was then transferred onto a nasopharyngeal (NP) swab and tested on three devices.

Of the 41 viruses tested, 40 were successfully detected in 3/3 devices for the expected strain (influenza A, influenza B or SARS-CoV-2) on the Visby Medical Respiratory Health Test at $\leq 3 \mathbf { x }$ LoD. The Omicron (BA.2.3) isolate (USA/MD-HP24556/2022) was detected in 3/3 devices at 6x LoD. The results from this study demonstrate that the Visby Medical Respiratory Health Test is capable of detecting multiple strains of influenza A, influenza B and SARS-CoV2 that are circulating in humans at the concentrations tested.

Table 5. Analytical Reactivity (Inclusivity) Wet-Testing Study Results   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=6>ConcentrationTested(copies/swab)</td><td rowspan=1 colspan=1>DetectionRate</td></tr><tr><td rowspan=11 colspan=1>Influenza AH1N1(pandemic2009)</td><td rowspan=1 colspan=1>A/Brownsville/39H/2009</td><td rowspan=2 colspan=6></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/H090-761-V1(0)/2009</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>A/Netherlands/2629/2009</td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A/Massachusetts/15/2013</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Bangladesh/3002/2015</td><td rowspan=2 colspan=6>318 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/45/2015</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/St. Petersburg/61/2015</td><td rowspan=1 colspan=6></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/66/2019</td><td rowspan=1 colspan=6></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/02/2020</td><td rowspan=2 colspan=6></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/588/2019</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>Influenza AH3N2</td><td rowspan=1 colspan=1>A/Netherlands/22/2003</td><td rowspan=4 colspan=6>393 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/New York/55/2004</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Uruguay/716/2007</td><td rowspan=1 colspan=1>3/3</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=5>ConcentrationTested(copies/swab)</td><td rowspan=1 colspan=1>DetectionRate</td></tr><tr><td rowspan=9 colspan=1></td><td rowspan=1 colspan=1>A/Hong Kong/H090-756-V1(0)/2009</td><td rowspan=4 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/2011</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Switzerland/9715293/2013</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Alaska/232/2015</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/California/55/2020</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/2671/2019</td><td></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/04/2018</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=7 colspan=1>Influenza BVictoriaLineage</td><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>B/St. Petersburg/14/2006</td><td rowspan=3 colspan=4></td><td></td><td></td></tr><tr><td rowspan=3 colspan=5>2,184 (3x LoD)</td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=2 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/New Jersey/1/2012</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=5 colspan=1>Influenza BYamagataLineage</td><td rowspan=1 colspan=1>B/New York/1061/2004</td><td rowspan=3 colspan=5>2,334 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Florida/4/2006</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Texas/06/2011</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013</td><td rowspan=2 colspan=5></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Guangdong-Liwan/1133/2014</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Lee/1940</td><td rowspan=2 colspan=5>2,334 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/1959</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=12 colspan=1>SARS-CoV-2</td><td rowspan=2 colspan=1>Alpha (B.1.1.7)England/204820464/2020</td><td rowspan=1 colspan=5>100 (1x LoD)</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=5>300 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Beta (B.1.351)South Africa/KRISP- K005325/2020</td><td rowspan=1 colspan=5>100 (1x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Gamma (P.1)Japan/TY7-503/2021</td><td rowspan=1 colspan=5>100 (1x LoD)</td><td rowspan=1 colspan=1>2/3</td></tr><tr><td rowspan=1 colspan=5>300 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Delta (B.1.617.2)USA/PHC658/2021</td><td rowspan=1 colspan=5>100 (1x LoD)</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=5>300 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>B.1.1.519 (BA.1)Isolate: USA/MD-HP20874/2021</td><td rowspan=1 colspan=5>100 (1x LoD)</td><td rowspan=1 colspan=1>2/3</td></tr><tr><td rowspan=1 colspan=5>300 (3x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Omicron (BA.2.3)Isolate: USA/MD-HP24556/2022</td><td rowspan=1 colspan=5>100 (1x LoD)</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=5>300 (3x LoD)</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=5>600 (6x LoD)</td><td rowspan=1 colspan=1>3/3</td></tr></table>

b. Cross-reactivity:

Wet Testing:

The analytical specificity (cross-reactivity) of the Visby Medical Respiratory Health Test was evaluated by testing a cohort of forty-three non-targeted microorganisms (including pooled human nasal wash) that may be found in a respiratory tract clinical specimen (Table 6). Panels were composed of up to five different non-target microorganisms spiked into negative pooled clinical swab matrix at $\ge 1 \mathrm { x } 1 0 ^ { 5 } \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ (for viruses) and ${ \bf \bar { \tau } } ^ { \mathrm { } } \geq 1 \mathrm { x } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ (for bacteria and fungi), with the exception of Cytomegalovirus, Measles, Mumps, which were tested at ${ \geq } 1 \mathrm { x } 1 0 ^ { 4 }$ T $\mathrm { \dot { C } I D } { \scriptstyle 5 0 } / { \scriptstyle \mathrm { m L } }$ or genome copies/mL as shown in Table 6. Each panel was tested in triplicate.

All sample groups returned 3/3 devices with the expected negative results. Overall, the data show that none of the 43 organisms (including pooled human nasal wash) cause false positive results when tested using the Visby Medical Respiratory Health Test at the concentrations tested.

Table 6. Cross-reactivity Wet-Testing Study Results   

<table><tr><td colspan="1" rowspan="1">cross-Tcactiviy wct-Testing Stuay Rcsuits</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration tested</td><td colspan="1" rowspan="1">Detection Rate</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus HKU1**</td><td colspan="1" rowspan="1">1.1 x 105 genome copies/mL</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus NL63*†</td><td colspan="1" rowspan="1">1.1 x 105 genome copies/mL</td></tr><tr><td colspan="1" rowspan="1">SARS-Coronavirus*†</td><td colspan="1" rowspan="1">1.1 x 105 genome copies/mL</td></tr><tr><td colspan="1" rowspan="1">MERS-Coronavirus**</td><td colspan="1" rowspan="1">1.1 x 105 genome copies/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus strain 1, C1 Ad 71</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus strain 7</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.1 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.1 x 105 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus (hMPV)</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza virus 1</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza virus 2</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza virus 3</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza virus 4b</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.1 x 104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps*</td><td colspan="1" rowspan="1">1.1 x 104 genome copies/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus (Strain B)</td><td colspan="1" rowspan="1">1.1 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human rhinovirus 1A (strain 2060)</td><td colspan="1" rowspan="1">1.1 x 105 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium xerosis</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration tested</td><td colspan="1" rowspan="1">Detection Rate</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella (Branhamella) catarrhalis</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td><td colspan="1" rowspan="4">0/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis serogroup a 1.1 x 106 genomic copies/mL</td><td colspan="1" rowspan="1">Neisseria meningitidis serogroup a 1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td><td colspan="1" rowspan="5">0/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii also calledPneumocystis carinii Delanoe andDelanoe</td><td colspan="1" rowspan="1">1.1 x 106 nuclei/mL</td><td colspan="1" rowspan="3">0/3</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1.1 x 106 cfu/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1.1 x 106 genomic copies/mL</td></tr><tr><td colspan="1" rowspan="1">Pooled human nasal wash</td><td colspan="1" rowspan="1">100% (no dilution)</td><td colspan="1" rowspan="1">0/3</td></tr></table>

\* Purified RNA was tested for these organisms. † Synthetic RNA was tested for these organisms.

# c. Microbial Interference:

A microbial interference study was conducted to assess potential inhibitory effects of 43 nontarget microorganisms (including pooled human nasal wash) that may be found in a human respiratory specimen (Table 6) on performance of the Visby Medical Respiratory Health Test. Panels were composed of up to five different non-target microorganisms spiked into pooled clinical swab matrix that contained low positive (i.e., $3 \mathrm { x } \ \mathrm { L o D } )$ concentrations of each target organism SARS-CoV-2 (USA-WA1/2020, inactivated virus), influenza A (2009 H1N1, Brisbane/02/18, live virus), and influenza B (Victoria, Washington/02/19, live virus). Non-target organisms were spiked at $\ge 1 \mathrm { x } 1 0 ^ { 5 } \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ (for viruses) and $\ge 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ (for bacteria and fungi), with the exception of Cytomegalovirus, Measles, Mumps, which were tested at $\geq 1 \mathrm { x } 1 0 ^ { 4 } \mathrm { T C I D } \varsigma _ { 0 } / \mathrm { m L }$ or genome copies $/ \mathrm { m L }$ as shown in Table 6. Each panel was tested in triplicate.

All sample groups returned $3 / 3$ devices with the expected negative results. Overall, the data shows that none of the 43 organisms (including pooled human nasal wash) cause false negative results when tested using the Visby Medical Respiratory Health Test at the concentrations tested.

The potential for high concentrations (i.e., ${ \mathrm { > } } 1 \mathbf { x } 1 0 ^ { 5 }$ copies/swab) of one target organism to interfere with detection of low concentrations (i.e., 3x LoD) of another target organism (i.e., co-infections with on-target analytes) was evaluated for the Visby Medical Respiratory Health Test.

Influenza A (2009 H1N1, Brisbane/02/18, live virus), influenza B (Victoria, Washington/02/19, live virus), and SARS-CoV-2 (USA-WA1/2020, inactivated virus) viruses were spiked into negative pooled clinical swab matrix at varying concentrations and then tested in triplicate. Low concentrations were prepared at $3 \mathbf { x }$ LoD for the respective viruses, and high concentrations were prepared as described in Table 7. No competitive interference was observed for any of the three target viruses at the concentrations tested.

Table 7. Competitive Interference Results Summary   

<table><tr><td rowspan=1 colspan=3>Viral Targets in Sample</td><td rowspan=1 colspan=3>Detection Rate</td></tr><tr><td rowspan=1 colspan=1>Influenza Acopies/swab)</td><td rowspan=1 colspan=1>Influenza B(copies/swab)</td><td rowspan=1 colspan=1>SARS-CoV-2(copies/swab)</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>1.06 x 105</td><td rowspan=1 colspan=1>2184</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>1.06 x 105</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>318</td><td rowspan=1 colspan=1>7.28 x 105</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.28 x 10 5$</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>318</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.0 x 10 5$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2184</td><td rowspan=1 colspan=1>1.0 x 105</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# e. Interfering Substances:

The performance of the Visby Medical Respiratory Health Test was evaluated in the presence of medically and/or physiologically relevant concentrations of potentially interfering substances that may be present in a respiratory specimen. For this study, negative swab clinical sample matrix pools were individually spiked with the potential interfering substances at the concentrations denoted in Table 8. To determine if these potential interfering substances impact test performance in the presence and absence of target analytes, both target-spiked and target-non-spiked samples were generated, respectively. Target-spiked samples were created by triple spiking influenza A (2009 H1N1, Brisbane/02/18, live virus), influenza B (Victoria, Washington/02/19, live virus), and SARS-CoV-2 (USA-WA1/2020, inactivated virus) at $3 \mathrm { x } \ \mathrm { L o D }$ into negative pooled swab clinical sample matrix, containing a single potential interfering substance. Target-non-spiked samples consisted of negative pooled nasal swab clinical sample matrix and a single potential interfering substance. For each potentially interfering substance, three replicates were tested for the target-spiked and target-non-spiked samples.

All target-non-spiked samples were $100 \%$ negative for influenza A, influenza B, and SARSCoV-2. All target-spiked samples were $100 \%$ positive for influenza A, influenza B, and SARS-CoV-2, except for cotton and rayon swabs. For these swab materials, two of the three target-spiked samples were positive for all targets, while one of three target-spiked samples was negative for SARS-CoV-2. Based on the results for cotton and rayon swabs, these swab materials are not acceptable swab types for use with the Visby Medical Respiratory Health Test. None of the other evaluated substances interfered with performance of the device at the concentrations tested.

Table 8. Substances and Swab Materials Evaluated for Interference   

<table><tr><td rowspan=2 colspan=1>Potential Interferant</td><td rowspan=2 colspan=1>ActiveIngredient</td><td rowspan=2 colspan=1>TestConcentration</td><td rowspan=1 colspan=2>Detection Rate</td></tr><tr><td rowspan=1 colspan=1>NegativeSamples</td><td rowspan=1 colspan=1>3x LoDSamples</td></tr><tr><td rowspan=1 colspan=1>Afrin (Nasal Spray)</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>25% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Saline (Nasal Spray)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>25% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>NeoSynephrine Cold &amp; SinusExtra Strength Spray (Nasal Spray)</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>25% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Flonase (Nasal Corticosteroid)</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>25% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Nasacort (Nasal Corticosteroid)</td><td rowspan=1 colspan=1>Triamcinolone</td><td rowspan=1 colspan=1>25% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Blood, Human</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Bovine Submaxillary Gland,Type I-S</td><td rowspan=1 colspan=1>Purified MucinProtein</td><td rowspan=1 colspan=1>1% (w/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Hand Lotion</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (w/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Hand Sanitizer</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (w/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Method All Purpose Cleaner</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Seventh Generation Disinfectant</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Softsoap Moisturizing Hand Soap</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (w/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, Nasal Ointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>12 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, Systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>2.43 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Antiviral</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Biotin</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3.5 ug/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Zicam Cold RemedyNo Drip Nasal Spray(Homeopathic Allergy Relief)</td><td rowspan=1 colspan=1>Luffa Opperculata,Galphimia Glauca,Sabadilla</td><td rowspan=1 colspan=1>25% (v/v)</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Swab Material Type</td><td rowspan=1 colspan=1>ActiveIngredient</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>NegativeSamples</td><td rowspan=1 colspan=1>3x LoDSamples</td></tr><tr><td rowspan=1 colspan=1>Cotton Swab</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>2/3a</td></tr><tr><td rowspan=1 colspan=1>Flocked Polyester Swab</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Foam Swab</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Rayon Swab</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>2/3a</td></tr></table>

a One device returned a positive result for influenza A and influenza B, but negative for SARS-CoV-2.

# 4. Assay Reportable Range:

Not applicable.

This is a qualitative colorimetric test with binary output (positive/negative). The intensity of the color is not proportional to the concentration of the target organisms.

5. Specimen Stability:

Data provided support the following stability claims for specimens eluted in Visby Respiratory Health Buffer:

• Room temperature $( 1 5 - 3 0 ^ { \circ } \mathrm { C } )$ storage for up to 120 minutes (2 hours) • Refrigerated $\left( 2 - 8 ^ { \circ } \mathrm { C } \right)$ storage for up to 48 hours (2 days)

6. Detection Limit:

# a. Limit of Detection (Individual Analytes):

The purpose of this study was to determine the limit of detection (LoD) of the Visby Medical Respiratory Health Test for the detection of influenza A, influenza B, and SARS-CoV-2 as single analytes. The study assessed two live strains of influenza A (influenza A 2009 H1N1, Brisbane/02/18 and influenza A H3N2, Kansas/14/2017) and influenza B (influenza B Victoria, Washington/02/19 and influenza B Yamagata, Oklahoma/10/2018), and one inactivated strain of SARS-CoV-2 (USA-WA1/2020). Each organism was individually spiked into pooled negative clinical swab matrix and applied at the appropriate final concentration to nasopharyngeal swabs for testing per the instructions for use for the Visby Medical Respiratory Health Test. Each virus was tested in a range finding study that included five concentrations of a three-fold dilution series, where each concentration was tested with five replicates. The lowest concentration with $100 \%$ detection was estimated to be the LoD. The estimated LoD for each strain was verified by testing 20 replicates at the estimated LoD concentration and demonstrating that at least 19/20 replicates were positive. If the criteria of at least 19 out of 20 positive replicates were not met, testing was repeated with a higher concentration until at least 19/20 replicates gave a positive result.

The LoD of the Visby Medical Respiratory Health Test for influenza A, influenza B, and SARS-CoV-2 are summarized in Table 9.

Table 9. LoD Values of the Visby Medical Respiratory Health Test   

<table><tr><td rowspan=2 colspan=1>Virus</td><td rowspan=1 colspan=2>Analytical Limit of Detection (LoD)</td></tr><tr><td rowspan=1 colspan=1>Copies/swab</td><td rowspan=1 colspan=1>TCID50/swab or FFU/swab</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1, Brisbane/02/18</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>4.89 TCID50/swab</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2, Kansas/14/2017</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>2.01 FFU/swab</td></tr><tr><td rowspan=1 colspan=1>Influenza B Victoria, Washington/02/19</td><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>9.20 TCID50/swab</td></tr><tr><td rowspan=1 colspan=1>Influenza B Yamagata, Oklahoma/10/2018</td><td rowspan=1 colspan=1>778</td><td rowspan=1 colspan=1>88.37 TCID50/swab</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA-WA1/2020)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td></tr></table>

FFU: Fluorescent Focus Units. Titer by Fluorescent Focus Assay in MDCK-SIAT1 Cells.

7. Assay Cut-Off:

Not applicable. This is a visually interpreted test where any shade of purple indicates a positive reaction.

# 8. Carry-Over:

Not applicable. This is a single-use test and is not subject to carryover from previously tested specimens.

# B Comparison Studies:

# 1. Method Comparison with Predicate Device:

See Clinical Studies below.

# 2. Matrix Comparison:

An LoD equivalency study was conducted to compare side-by-side testing of the same viral stocks prepared in nasal and nasopharyngeal swab matrices and eluted in Visby Respiratory Buffer. Testing was performed with a minimum of three dilutions and until the concentration was found for each virus in each matrix with at least 19/20 positive results where the dilution below had $< 1 9 / 2 0$ positive results. The LoD was considered equivalent between the nasal and nasopharyngeal swab matrices if the lowest concentration of virus with at least 19/20 positives is within one three-fold dilution for each other.

Influenza A, influenza B, and inactivated SARS-CoV-2 virus were spiked into each matrix at the specified concentration for each target to prepare a triple positive sample. Three triple positive samples were made in each matrix, and they were tested with 20 devices.

The results of the testing are summarized in Table 10 below. The bolded results indicate the lowest concentrations for each virus with at least 19/20 positive results. These results support that the LoD for detection of influenza A, influenza B, and SARS-CoV-2 are equivalent for nasal and nasopharyngeal swab matrices.

Table 10. LoD Equivalency Results for Nasal and Nasopharyngeal Swab Matrices   

<table><tr><td rowspan=2 colspan=2>Virus</td><td rowspan=2 colspan=1>Concentration(copies/swab, xLoD)</td><td rowspan=1 colspan=2>Detection Rate</td></tr><tr><td rowspan=1 colspan=1>NasalSwab Matrix</td><td rowspan=1 colspan=1>NasopharyngealSwab Matrix</td></tr><tr><td rowspan=3 colspan=2>Influenza A 2009H1N1,Brisbane/02/18</td><td rowspan=1 colspan=1>318 (3x)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>106 (1x)</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>35 (1/3x)</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>16/20</td></tr><tr><td rowspan=3 colspan=2>Influenza BVictoria,Washington/02/19</td><td rowspan=1 colspan=1>728 (1x)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>243 (1/3x)</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>81 (1/9x)</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>15/20</td></tr><tr><td rowspan=3 colspan=2>SARS-CoV-2 (USAWA1/2020)</td><td rowspan=1 colspan=1>333 (3.3x)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>33 (1/3x)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>11 (1/9x)</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>10/20</td></tr></table>

# C Clinical Studies:

The clinical performance of the Visby Medical Respiratory Health Test was evaluated in a multicenter prospective study conducted at five geographically distinct clinical sites representative of CLIA-waived testing facilities. The sites were distributed across the United States and were described as urgent care and family care clinics. A total of 18 untrained operators, representative of CLIA waived users, participated in the study. Reference testing took place at a single laboratory.

The study prospectively enrolled subjects with signs and symptoms of respiratory tract infection. Subjects were enrolled from May 2022 through March 2023 and from November 2023 through

February 2024. The average age among study participants was 34 years, with a range between one month to 89 years of age.

One NP swab specimens or dual nostril AN swab specimen was collected from each eligible subject. The AN specimens could be self-collected under health care provider (HCP) observation (for subjects $\geq 1 4$ years of age), or health care provider (HCP) collected, and all NP swabs were HCP collected. The swab specimen was placed directly in the Visby Respiratory Health Buffer and tested on-site by study operators using the Visby Medical Respiratory Health Test. The participating operators conducted the test by following the instructions in the Quick Reference Guide (QRG). The study operators were representative of operators conducting testing at a CLIA Waived setting and did not receive any training on the use of the Visby Medical Respiratory Health Test.

The specimen was also tested at the reference laboratory for comparator testing using an FDAcleared nucleic acid amplification test (NAAT) for influenza A and influenza B targets and a highly sensitive FDA-authorized laboratory-based RT-PCR EUA assay for the SARS-CoV-2 target.

A total of 1,592 subjects were initially enrolled. Study samples were excluded from the data analysis due to lack of a valid comparator result $\left( \mathrm { n } \mathrm { = } 5 3 \right)$ ), lack of a valid Visby Medical Respiratory Health Test result $( \mathrm { n } { = } 2 1 )$ ), the subject was asymptomatic $( \mathrm { n } { = } 1 0 )$ ), or for protocol deviations (i.e., failure to complete a required procedure $( \mathrm { n } { = } 4 )$ or incorrectly filled out informed consent form $( \mathrm { n } { = } 2 )$ ). Additionally, one subject withdrew from the study. This left 1,501 subjects for performance analysis.

The performance estimates for the Visby Medical Respiratory Health Test for the detection of SARS-CoV-2, influenza A, and influenza B were calculated as positive percent agreement (PPA) and negative percent agreement (NPA) with the comparator result, which are shown in Tables 11-13 stratified by specimen type. These results are stratified by site in Tables 14-16.

Table 11. SARS-CoV-2 Clinical Performance for the Visby Test vs. Comparator, by Specimen Type   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>687</td><td rowspan=1 colspan=1>5c</td><td rowspan=1 colspan=1>96.3%(91.6-98.4%)</td><td rowspan=1 colspan=1>99.0%(97.9-99.5%)</td></tr><tr><td rowspan=1 colspan=1>ANS</td><td rowspan=1 colspan=1>672</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>5b</td><td rowspan=1 colspan=1>549</td><td rowspan=1 colspan=1>2d</td><td rowspan=1 colspan=1>98.3%(94.0-99.5%)</td><td rowspan=1 colspan=1>99.1%(97.9-99.6%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1501</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>97.2%(94.4-98.7%)</td><td rowspan=1 colspan=1>99.0%(98.3-99.5%)</td></tr></table>

PPA $\circleddash$ Positive Percent Agreement with comparator; NPA $\circleddash$ Negative Percent Agreement with comparator; $\mathrm { { T P } = }$ true positive; $\mathrm { F P = }$ false positive; $\mathrm { T N } { = }$ true negative; $\mathrm { F N = }$ false negative a 5 of 7 false positive NPS specimens tested positive and 2 tested negative with an alternate EUA molecular assay. $^ { \mathsf { b } } 4$ of 5 false positive AN specimens tested positive and 1 tested negative with an alternate EUA molecular assay. $^ \mathrm { ~ c ~ } 3$ of 5 false negative NPS specimens tested negative and 2 tested positive with an alternate EUA molecular assay. $^ \mathrm { d } \ 1$ of 2 false negative AN specimens tested negative and 1 tested positive with an alternate EUA molecular assay.

Table 12. Influenza A Clinical Performance for the Visby Test vs. Comparator, by Specimen Type   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>97.1%(85.1-99.5%)</td><td rowspan=1 colspan=1>99.5%(98.7-99.8%)</td></tr><tr><td rowspan=1 colspan=1>ANS</td><td rowspan=1 colspan=1>672</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>5b</td><td rowspan=1 colspan=1>604</td><td rowspan=1 colspan=1>2d</td><td rowspan=1 colspan=1>96.8%(89.1-99.1%)</td><td rowspan=1 colspan=1>99.2%(98.1-99.7%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1501</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1395</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.9%(91.3-98.9%)</td><td rowspan=1 colspan=1>99.4%(98.8-99.7%)</td></tr></table>

PPA $\circleddash$ Positive Percent Agreement with comparator; NPA $\circleddash$ Negative Percent Agreement with comparator; $\mathrm { T P = }$ true positive; $\mathrm { F P = }$ false positive; $\mathrm { T N } =$ true negative; $\mathrm { F N = }$ false negative a 3 of 4 false positive NPS specimens tested positive and 1 tested negative with an alternate EUA molecular assay. $^ { \textrm { b } } 5$ of 5 false positive AN specimens tested positive with an alternate EUA molecular assay. $^ \mathrm { ~ c ~ } 1$ false negative NPS specimen tested positive with an alternate EUA molecular assay. $^ { \textrm { d } } 2$ false negative AN specimens tested positive with an alternate EUA molecular assay.

Table 13. Influenza B Clinical Performance for the Visby Test vs. Comparator, by Specimen Type   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(79.6-100%)</td><td rowspan=1 colspan=1>99.8%(99.1-99.9%)</td></tr><tr><td rowspan=1 colspan=1>ANS</td><td rowspan=1 colspan=1>672</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>656</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(79.6-100%)</td><td rowspan=1 colspan=1>99.9%(99.1-100%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1501</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1468</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(88.7-100%)</td><td rowspan=1 colspan=1>99.8%(99.4-99.9%)</td></tr></table>

PPA $\underline { { \underline { { \mathbf { \Pi } } } } }$ Positive Percent Agreement with comparator; NPA $\circleddash$ Negative Percent Agreement with comparator; $\mathrm { T P = }$ true positive; $\mathrm { F P = }$ false positive; $\mathrm { T N } { = }$ true negative; $\mathrm { F N = }$ false negative $^ \mathrm { a } 2$ of 2 false positive NPS specimens tested positive with an alternate EUA molecular assay. $^ { \textsf { b } } 1$ false positive AN specimen tested positive with an alternate EUA molecular assay.

Table 14. Influenza A Clinical Performance for the Visby Test vs. Comparator, by Site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1501</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1395</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.9%(91.3%-98.9%)</td><td rowspan=1 colspan=1>99.4%(98.8%-99.7%)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>732</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>96.5%(88.1%-99.0%)</td><td rowspan=1 colspan=1>99.7%(99.0%-99.9%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7%(20.8%-93.9%)</td><td rowspan=1 colspan=1>98.7%(93.2%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>221</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(83.9%-100%)</td><td rowspan=1 colspan=1>97.8%(94.9%-99.1%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>355</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(78.5%-100%)</td><td rowspan=1 colspan=1>99.7%(98.4%-100%)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(43.9%-100%)</td><td rowspan=1 colspan=1>100%(86.2%-100%)</td></tr></table>

Table 15. Influenza B Clinical Performance for the Visby Test vs. Comparator, by Site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1501</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1468</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(88.7%-100%)</td><td rowspan=1 colspan=1>99.8%(99.4%-99.9%)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>777</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(78.5%-100%)</td><td rowspan=1 colspan=1>100%(99.5%-100%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(43.9%-100%)</td><td rowspan=1 colspan=1>100%(95.4%-100%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>245</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(20.7%-100%)</td><td rowspan=1 colspan=1>100%(98.5%-100%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>355</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(74.1%-100%)</td><td rowspan=1 colspan=1>99.4%(97.9%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(20.7%-100%)</td><td rowspan=1 colspan=1>96.2%(81.1%-99.3%)</td></tr></table>

Table 16. SARS-CoV-2 Clinical Performance for the Visby Test vs. Comparator, by Site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA(95% CI)</td><td rowspan=1 colspan=1>NPA(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1501</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1236</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>97.2%(94.4%-98.7%)</td><td rowspan=1 colspan=1>99.0%(98.3%-99.5%)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>621</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.1%(94.7%-99.4%)</td><td rowspan=1 colspan=1>98.6%(97.3%-99.3%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>85.7%(48.7%-97.4%)</td><td rowspan=1 colspan=1>98.7%(92.8%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>229</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(81.6%-100%)</td><td rowspan=1 colspan=1>100.00%(98.4%-100%)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>355</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.2%(86.9%-98.4%)</td><td rowspan=1 colspan=1>99.3%(97.5%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(56.6%-100%)</td><td rowspan=1 colspan=1>100%(85.1%-100%)</td></tr></table>

Of the 1,575 tests performed on the Visby Medical Respiratory Health Test, 79 $( 5 . 0 \%$ , $9 5 \%$ CI = $4 . 0 { - } 6 . 2 \% )$ had an initial invalid test result of which 21 ( $1 . 3 \%$ , $9 5 \% \mathrm { C I } = 0 . 8 – 2 . 0 \% )$ were invalid when retested. Invalid results were more frequent for ANS ( $6 . 7 \%$ initial with $9 5 \% \mathrm { C I } = 5 . 0 $ - $8 . 7 \%$ , $2 . 2 \%$ upon repeat with $9 5 \% \mathrm { C I } = 1 . 3 { - } 3 . 5 \% )$ than for NPS ( $3 . 6 \%$ initial with $9 5 \% \mathrm { C I } = 2 . 5 \AA$ - $5 . 1 \%$ , $0 . 6 \%$ upon retest with $9 5 \% \mathrm { C I } = 0 . 3 \ – 1 . 4 \% )$ .

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

Table 17. Positivity Rate of the Visby Medical Respiratory Health Test by Age for Detection of influenza A, influenza B and SARS-CoV-2 in Respiratory Swab Specimens (NP and AN) During the Prospective Clinical Study (Enrollment from May 2022-March 2023)   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=2 colspan=1>N (%)</td><td rowspan=1 colspan=3>% Positive (# Positive / # Tested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>SARS-COV-2</td></tr><tr><td rowspan=1 colspan=1>0-5</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>8.3%</td></tr></table>

Table 18. Positivity Rate of the Visby Medical Respiratory Health Test by Age for Detection of influenza A, influenza B and SARS-CoV-2 in Respiratory Swab Specimens (NP and AN) During the Prospective Clinical Study (Enrollment from November 2023- February 2024)   

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=2 colspan=1>N (%)</td><td rowspan=1 colspan=3>% Positive (# Positive / # Tested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>SARS-COV-2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(6.5%)</td><td rowspan=1 colspan=1>(0/72)</td><td rowspan=1 colspan=1>(0/72)</td><td rowspan=1 colspan=1>(6/72)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>247(22.2%)</td><td rowspan=1 colspan=1>8.9%(22/247)</td><td rowspan=1 colspan=1>0.0%(0/247)</td><td rowspan=1 colspan=1>8.1%(20/247)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>649(58.2%)</td><td rowspan=1 colspan=1>4.8%(31/649)</td><td rowspan=1 colspan=1>0.002%(1/649)</td><td rowspan=1 colspan=1>20.5%(133/649)</td></tr><tr><td rowspan=1 colspan=1>≥ 60</td><td rowspan=1 colspan=1>147(13.2%)</td><td rowspan=1 colspan=1>0.01%(1/147)</td><td rowspan=1 colspan=1>0.0%(0/147)</td><td rowspan=1 colspan=1>25.9%(38/147)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1115</td><td rowspan=1 colspan=1>4.8%(54/1115)</td><td rowspan=1 colspan=1>0.00%(1/1115)</td><td rowspan=1 colspan=1>17.7%(197/1115)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Age</td><td rowspan=2 colspan=1>N (%)</td><td rowspan=1 colspan=3>% Positive (# Positive / # Tested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>SARS-COV-2</td></tr><tr><td rowspan=1 colspan=1>0-5</td><td rowspan=1 colspan=1>13(3.4%)</td><td rowspan=1 colspan=1>15.4%(2/13)</td><td rowspan=1 colspan=1>7.7%(1/13)</td><td rowspan=1 colspan=1>0.0%(0/13)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>78(20.1%)</td><td rowspan=1 colspan=1>19.2%(15/78)</td><td rowspan=1 colspan=1>12.8%(10/78)</td><td rowspan=1 colspan=1>9.0%(7/78)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>234(60.6%)</td><td rowspan=1 colspan=1>11.5%(27/234)</td><td rowspan=1 colspan=1>9.0%(21/234)</td><td rowspan=1 colspan=1>16.2%(38/234)</td></tr><tr><td rowspan=1 colspan=1>≥ 60</td><td rowspan=1 colspan=1>61(15.8%)</td><td rowspan=1 colspan=1>8.2%(5/61)</td><td rowspan=1 colspan=1>0.0%(0/61)</td><td rowspan=1 colspan=1>26.2%(16/61)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>12.7%(49/386)</td><td rowspan=1 colspan=1>8.3%(32/386)</td><td rowspan=1 colspan=1>15.8%(61/386)</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.