# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K242367   
B Applicant BioFire Diagnostics, LLC   
C Proprietary and Established Names BIOFIRE FILMARRAY Gastrointestinal (GI) Panel   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PCH</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3990 -GastrointestinalMicroorganism MultiplexNucleic Acid-Based Assay</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To update labeling of the cleared BIOFIRE FILMARRAY Gastrointestinal Panel to include new clinical data and analytical cross-reactivity testing evaluating potential risk of false positive results for the Norovirus GI/GII assay target.

# B Measurand:

Campylobacter (C. jejuni/C. coli/C. upsaliensis), Clostridium difficile (C. difficile) toxin A/B, Plesiomonas shigelloides, Salmonella, Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), Yersinia enterocolitica, Enteroaggregative Escherichia coli (EAEC), Enteropathogenic Escherichia coli (EPEC), Enterotoxigenic Escherichia coli (ETEC) lt/st, Shiga-like toxinproducing Escherichia coli (STEC) stx1/stx2, Shigella/ Enteroinvasive Escherichia coli (EIEC), Cryptosporidium, Cyclospora cayetanensis, Entamoeba histolytica, Giardia lamblia, Adenovirus F 40/41, Astrovirus, Norovirus GI/GII, Rotavirus A,Sapovirus (Genogroups I, II, IV, and V)

# C Type of Test:

A multiplexed nucleic acid-based test intended for use with the FilmArray 2.0 System or the Torch System for the qualitative in vitro detection and identification of multiple bacteria, viruses, and parasites. The BIOFIRE FILMARRAY GI Panel assay is performed directly from stool samples in Cary Blair transport media.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The BIOFIRE FILMARRAY Gastrointestinal (GI) Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with BIOFIRE FILMARRAY Systems. The BIOFIRE GI Panel is capable of the simultaneous detection and identification of nucleic acids from multiple bacteria, viruses, and parasites directly from stool samples in Cary Blair transport media obtained from individuals with signs and/or symptoms of gastrointestinal infection. The following bacteria (including several diarrheagenic E. coli/Shigella pathotypes), parasites, and viruses are identified using the BIOFIRE GI Panel:

• Campylobacter (C. jejuni/C. coli/C. upsaliensis)

• Clostridium difficile (C. difficile) toxin A/B • Plesiomonas shigelloides

• Salmonella

• Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), including specific identification of Vibrio cholerae

• Yersinia enterocolitica • Enteroaggregative Escherichia coli (EAEC)

• Enteropathogenic Escherichia coli (EPEC)

• Enterotoxigenic Escherichia coli (ETEC) lt/st

• Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the $E .$ . coli O157 serogroup within STEC)

• Shigella/ Enteroinvasive Escherichia coli (EIEC)

• Cryptosporidium • Cyclospora cayetanensis • Entamoeba histolytica

• Giardia lamblia (also known as $G .$ . intestinalis and G. duodenalis)

• Adenovirus F 40/41 • Astrovirus • Norovirus GI/GII • Rotavirus A

• Sapovirus (Genogroups I, II, IV, and V)

The BIOFIRE GI Panel is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out co-infection with organisms not included in the BIOFIRE GI Panel. The agent detected may not be the definite cause of the disease.

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.

This device is not intended to monitor or guide treatment for $C .$ . difficile infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for $E .$ . coli O157, Plesiomonas shigelloides, Yersinia enterocolitica, Astrovirus, and Rotavirus A were established primarily with retrospective clinical specimens.

Performance characteristics for Entamoeba histolytica, and Vibrio (V. parahaemolyticus, V.   
vulnificus, and Vibrio cholerae) were established primarily using contrived clinical specimens.

Negative BIOFIRE GI Panel results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or noninfectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

A gastrointestinal microorganism multiplex nucleic acid-based assay also aids in the detection and identification of acute gastroenteritis in the context of outbreaks.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

For in vitro diagnostic use only

# D Special Instrument Requirements:

The BIOFIRE FILMARRAY Gastrointestinal (GI) Panel is to be used with the BIOFIRE 2.0 System or BIOFIRE Torch System.

# IV Device/System Characteristics:

# A Device Description:

The BIOFIRE FILMARRAY Gastrointestinal Panel (BIOFIRE FILMARRAY GI Panel) was originally cleared in K140407.

The BIOFIRE FILMARRAY GI Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with BIOFIRE FILMARRAY systems. The BIOFIRE FILMARRAY GI Panel is capable of the simultaneous detection and identification of nucleic acids from multiple bacteria, viruses, and parasites directly from stool samples in Cary Blair transport media obtained from individuals with signs and/or symptoms of gastrointestinal infection.

# B Principle of Operation:

The principle of operation remains unchanged from the original clearance (K140407). Refer to the original published decision summary for specific details on the principle of operation of the device.

In response to a perceived elevated rate of BIOFIRE FILMARRAY GI Panel Norovirus GI/GII false positive results from customers, post-market analytical and clinical investigations were conducted to assess potential cross-reactivities of the Norovirus GI/GII assay and to provide an updated assessment of the Norovirus assay clinical performance.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BIOFIRE FILMARRAY Gastrointestinal (GI) Panel

B Predicate 510(k) Number(s): K230404

C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K242367</td><td rowspan=1 colspan=1>K230404</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>BIOFIREFILMARRAYGastrointestinal (GI)Panel</td><td rowspan=1 colspan=1>BIOFIREFILMARRAYGastrointestinal (GI)Panel</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intended Use/IndicationsFor Use</td><td rowspan=1 colspan=1>The BIOFIREFILMARRAYGastrointestinal (GI)Panel is a qualitativemultiplexed nucleic</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td>acid-based in vitro cholerae (EAEC)</td><td>diagnostic test intended for use with BIOFIRE FILMARRAY Systems. The BIOFIRE GI Panel is capable of the simultaneous detection and identification of nucleic acids from multiple bacteria, viruses, and parasites directly from stool samples in Cary Blair transport media obtained from individuals with signs and/or symptoms of gastrointestinal infection. The following bacteria (including several diarrheagenic E. coli/Shigella pathotypes), parasites, and viruses are identified using the BIOFIRE GI Panel: Campylobacter (C. jejuni/C. coli/C. upsaliensis) Clostridium difficile (C. difficile) toxin A/B Plesiomonas shigelloides Salmonella Vibrio (V. parahaemolyticus/V. vulnificus/ V. cholerae), including specific identification of Vibrio Yersinia enterocolitica Enteroaggregative Escherichia coli Enteropathogenic Escherichia coli (EPEC)</td></tr><tr><td></td><td>Enterotoxigenic Escherichia coli (ETEC) lt/st Shiga-like toxin- producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the E. coli 0157 serogroup within STEC) Shigellal Enteroinvasive Escherichia coli (EIEC) Cryptosporidium Cyclospora cayetanensis Entamoeba histolytica •Giardia lamblia (also known as G. intestinalis and G. duodenalis) Adenovirus F 40/41 Astrovirus Norovirus GI/GII Rotavirus A Sapovirus (Genogroups I, II, IV, and V) The BIOFIRE GI Panel is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not</td></tr></table>

<table><tr><td colspan="3"></td></tr><tr><td></td><td>the disease. Concomitant culture is necessary for organism recovery and further</td><td></td></tr><tr><td></td><td>typing of bacterial agents. This device is not intended to monitor or guide treatment for C.</td><td></td></tr><tr><td></td><td>difficile infection. Due to the small number of positive specimens collected for certain organisms during the prospective clinical study,</td><td></td></tr><tr><td></td><td>performance characteristics for E. coli O157, Plesiomonas shigelloides, Yersinia enterocolitica, Astrovirus, and Rotavirus A were established primarily with retrospective clinical specimens. Performance</td><td></td></tr><tr><td></td><td>characteristics for Entamoeba histolytica, and Vibrio (V.</td><td></td></tr><tr><td></td><td>parahaemolyticus, V. vulnificus, and Vibrio cholerae were established primarily using contrived clinical specimens. Negative BIOFIRE GI Panel results in the setting of clinical illness compatible with gastroenteritis may be</td><td></td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">due to infection bypathogens that are notdetected by this test ornon-infectious causessuch as ulcerativecolitis, irritable bowelsyndrome, or Crohn'sdisease.A gastrointestinalmicroorganismmultiplex nucleic acid-based assay also aids inthe detection andidentification of acutegastroenteritis in thecontext of outbreaks.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Human stool samplecollected in Cary Blairtransport media.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Nested multiplex PCRfollowed by highresolution meltinganalysis to confirm theidentity of amplifiedproduct.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">BIOFIRE 2.0 System orBIOFIRE Torch System</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">About 1 hour</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated testinterpretation and reportgeneration. User cannotaccess raw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Preparation Method</td><td colspan="1" rowspan="1">Sample Processing isautomated in theBIOFIRE System.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Reagents are stored atroom temperature.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Shelf-Life</td><td colspan="1" rowspan="1">12 months from Date ofManufacture</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls areincluded in each reagentpouch to control forsample processing and</td><td colspan="1" rowspan="1">Same</td></tr><tr><td></td><td>both stages of PCR and melt analysis.</td><td></td></tr></table>

# VI Standards/Guidance Documents Referenced:

# FDA-recognized Standards

• ISO 14971:2019 ‘Medical devices – Application of risk management to medical devices’ ISO 15223-1:2021 ‘Medical devices – Symbols to be used with information to be supplied by the manufacturer – Part 1: General requirements

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

A reproducibility study was previously performed for the BIOFIRE FILMARRAY GI Panel performance in the original clearance (K140407) and in additional subsequent regulatory submissions (K143005, K160459, and K230404). Please refer to the published decision summaries for additional information. Since there is no change to the assay, instrument, or software, and a new reproducibility study was not conducted.

# 2. Linearity:

The BIOFIRE FILMARRAY GI Panel is a qualitative assay and therefore linearity studies are not applicable.

# 3. Analytical Specificity/Interference:

Analytical specificity and interfering substances studies were performed for the original BIOFIRE FILMARRAY GI Panel performance evaluation (K140407). Please refer to the published decision summary for additional information.

Additional analytical specificity testing was performed to evaluate the reactivity of the BIOFIRE FILMARRAY GI Panel Norovirus GI/GII assay (and other Panel assays) with multiple bacterial organisms identified in post-market investigations of false positive Norovirus GI/GII results in clinical specimens. Sequence data recovered from BIOFIRE FILMARRAY GI Panel pouch arrays suspected of false positive Norovirus GI/GII results were analyzed and aligned to sequences of a variety of commensal bacteria from the human gastrointestinal tract, including Mediterraneibacter (Ruminococcus) gnavus, Enterobacter hormaechei, Anaerostipes hadrus, Parabacteroides merdae, Parabacteroides distasonis, Blautia wexlerae, Prevotella sp. ( $P .$ . bivia, P. copri, $P .$ . histicola, and $P$ . intermedia) and Bifidobacterium pseudocatenulatum. Both wet-testing of isolates and in silico analysis approaches were used to evaluate cross reactivity of these organisms with the Norovirus GI/GII assay and other Panel assays.

Wet testing of organism isolates followed the general procedure used for Analytical Specificity testing in the original clearance for the BIOFIRE FILMARRAY GI Panel (K140407). Isolates were prepared and tested at concentrations ${ > } 1 . 0 \mathrm { E } { + } 0 8$ cells/mL in triplicate (one replicate using one of three different reagent lots). Initial positive results for any replicate were followed by at least two additional replicates to determine whether the result was reproducible. Norovirus GI/GII Detected results (Noro 1 assay positive) were reported in at least 1/3 or 2/5 replicates at the initial high concentration for Mediterraneibacter (Ruminococcus) gnavus, Parabacteroides merdae, Anaerostipes hadrus (one of two isolates), and Enterobacter hormaechei (one of two isolates). Norovirus GI/GII Not Detected results were reported for all replicates of Parabacteroides distasonis, Bifidobacterium pseudocatenulatum and Blautia wexlerae, all Prevotella sp., as well as one of two isolates of each A. hadrus and $E$ . hormaechei.

# In silico Analysis

Amplicon sequences recovered from BIOFIRE FILMARRAY FILMARRAY GI Panel pouches with suspected false positive Norovirus GI/GII results were matched to a reference sequence for the best match species. The reference sequence was then used to search for regions flanking the amplicon which might be used as sites for the Norovirus GI/GII assay primer binding. Homology of assay primers with the predicted binding site was used to predict the likelihood of amplification and detection.

Testing and in silico analysis concluded that Mediterraneibacter (Ruminococcus) gnavus and Parabacteroides merdae are potentially cross-reactive species and present a risk of false positive Norovirus GI/GII results if present in a specimen at very high concentration. Additionally, predicted cross-reactive sequences were identified in a subset of genomes of Anaerostipes hadrus and Enterobacter hormaechei $( < 3 5 \%$ and $< 2 5 \%$ of genomes, respectively). Sequence in approximately $50 \%$ of the Parabacteroides distasonis genomes can be amplified by a combination of outer and inner reaction primers, but in silico analyses suggest a false positive result by the BIOFIRE FILMARRAY GI Panel is unlikely due to exclusion by the assay Tm range. Tm range-based exclusion was also demonstrated for offtarget amplification of sequence in Bifidobacterium pseudocatenulatum mitigating the risk of false results due to cross-reactivity with this organism. No risk of cross-reactivity/off-target amplification could be identified for Blautia wexlerae based on sequence analysis or testing.

The analytical specificity study to support the original clearance of the BIOFIRE FILMARRAY GI Panel evaluated Prevotella melaninogenica (a human commensal and opportunistic pathogen). No cross-reactivity with $P$ . melaninogenica was observed, however internal investigations into potentially false positive Norovirus GI/GII results recovered sequence from non-specific amplicons which, when submitted for BLAST analysis, aligned to atypical Prevotella sp. In silico analyses of four additional Prevotella sp. ( $P .$ . bivia, $P$ . copri, $P _ { - }$ . histicola, and $P .$ . intermedia) did not indicate non-specific amplification or interaction with the Norovirus GI/GII assay. These results are consistent with the lack of any observed cross-reactivity from wet-testing of these isolates. Results from the additional cross-reactivity testing and in silico analyses (Tables 1 & 2) will be included in the updated device labeling.

Table 1. Observed or Predicted Cross-Reactivity with Off-Panel Organisms   

<table><tr><td rowspan=1 colspan=1>BIOFIRE GI Panel Test Result</td><td rowspan=1 colspan=1>Cross-Reactive Organism(s)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Entamoeba dispar</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Bifidobacterium spp aRuminococcus spp a</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E.coli (ETEC)It/st</td><td rowspan=1 colspan=1>Citrobacter koseriCitrobacter sedlakiiHafnia alvei aCedecea davisiae a</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII b</td><td rowspan=1 colspan=1>Prevotella spp. (sequences from unculturable/uncharacterizedspecies) cMediterraneibacter (Ruminococcus) gnavusParabacteroides spp. (P. merdae, P. acidifaciens d, P. distasonis e)Anaerostipes hadrus (select sequences) fEnterobacter hormaechei (select sequences) g</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>E. coli with variant type III secretion protein h</td></tr><tr><td rowspan=1 colspan=1>Vibrio(V. parahaemolyticus/V.vulnificus/V. cholerae)</td><td rowspan=1 colspan=1>Vibrio alginolyticusVibrio fluvialis iVibrio mimicus iGrimontia (formerly Vibrio) hollisae</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>Yersinia frederiksenitajYersinia kristensenii j</td></tr></table>

a Cross-reactivity was not observed when tested at high concentration $\overline { { ( 1 . 5 { \times } 1 0 ^ { 9 } \mathrm { c e l l s } / \mathrm { m L } ) } }$ ). However, cross-reactivity was suspected or confirmed in clinical specimens and/or the potential for cross-reactivity is supported by in silico predictions.   
b Cross-reactivity was identified by post-market investigation of suspected false positive Norovirus GI/GII results in clinical specimens. Cross-reactivity with the species listed was confirmed by analytical testing at high concentration $( > 2 . 4 \mathrm { x } 1 0 ^ { \bar { 8 } }$ cells/mL) and/or is supported by sequence analysis.   
c Cross-reactive sequences are inconsistent with other Prevotella sequence data, suggesting non-specific interaction with atypical or uncharacterized species and/or sequences.   
d P. acidifaciens was not tested but was determined by sequence analysis to have a similar risk of cross-reactivity as $P .$ . merdae.   
e Norovirus GI/GII Not Detected was reported when $P .$ . distasonis was tested at high concentration $\mathbf { \ } ^ { ( 3 . 1 \mathbf { x } 1 0 ^ { 9 } }$ cells/mL). However, non-specific amplification products with Tm values close to the assay specific Tm range have been observed and the potential for false positive Norovirus GI/GII test results exists.   
f The risk of false positive Norovirus GI/GII results due to cross-reactivity with A. hadrus is associated with only a subset of A. hadrus RefSeq genome sequences ( $< 3 5 \%$ as of June 2024).   
g The risk of false positive Norovirus GI/GII results due to cross-reactivity with $E .$ . hormaechei is associated with only a subset of $E$ . hormaechei RefSeq genome sequences $< 2 5 \%$ as of June 2024).   
h Cross-reactivity resulting in false positive Salmonella results has not been observed in analytical or clinical testing. However, non-specific amplification products with Tm values close to the assay specific Tm range have been observed and the potential for false positive Salmonella test results exists.   
i Detected at concentrations near the Vibrio assay LoD.   
j Y. kristensenii and Y. fredericksenii are difficult to distinguish from Y. enterocolitica by standard laboratory methods. a Anerostipes hadrus isolates (DSM 23942 and ATCC 29173) were tested at ${ > } 2 . 4 \times 1 0 ^ { 8 }$ cells/mL. Norovirus GI/GII Detected results were only observed with DSM 23942. The ATCC 29173 isolate does not carry the cross-reactive sequence.   
b Though not observed in analytical testing, cross-reactivity of the Giardia lamblia assay with one or more Bifidobacterium and Ruminococcus species was observed in the clinical evaluation (see Table 1).   
c Though not observed in analytical testing, possible cross-reactivity of the ETEC 2 assay with Hafnia alvei and Cedecea davisiae was observed in the clinical evaluation or predicted by in silico analysis (see Table 1).   
d Two isolates of this species were tested for analytical specificity.   
e Enterobacter hormaechei isolates (ATCC BAA-2082 and ATCC 49162) were tested at ${ > } 5 . 0 { \times } 1 0 ^ { 8 }$ cells/mL. Norovirus GI/GII Detected results were only observed with ATCC 49162. The ATCC BAA-2082 sequence is not predicted to be cross-reactive by in silico analysis.   
f Though not observed in analytical testing, cross-reactivity of the Norovirus GI/GII assay with a sequence identified in roughly half $( \sim 5 0 \% )$ of the $P .$ . distasonis genomes evaluated could occur at high concentration.   
${ \bf g _ { \alpha A } }$ similar risk of cross-reactivity was identified with sequences annotated as Parabacteroides sp. and $P .$ . acidifaciens. $^ \mathrm { h } \mathrm { N } _ { 0 }$ cross-reactivity with high concentrations of various Prevotella species (commensal and pathogenic) was observed in analytical testing, but the potential for weak cross-reactivity between the Norovirus GI/GII assay and unique variant sequences annotated as unculturable Prevotella sp. has been identified via investigation of discrepant results in clinical specimens. i Though not observed in analytical testing, in silico analysis indicates that, similar to Y. kristensenii, cross-reactivity between the Yersinia entercolitica assay and Yersinia fredericksenii is possible at high concentrations (see Table 1).

Table 2. Off-Panel Organisms Evaluated with Wet-testing or by in silico Analyses   

<table><tr><td colspan="3">Bacteria</td></tr><tr><td colspan="3">Tested</td></tr><tr><td>Abiotrophia defectivia</td><td>Clostridium novyi</td><td>Morganella morganii</td></tr><tr><td>Acinetobacter baumannii</td><td>Clostridium perfringens</td><td>Parabacteroides distasonis f</td></tr><tr><td>Acinetobacter lwoffii</td><td>Clostridium ramosum</td><td>Parabacteroides merdae g</td></tr><tr><td>Aeromonas hydrophila</td><td>Clostridium septicum</td><td>Peptoniphilus asaccharolyticus</td></tr></table>

<table><tr><td>Anaerococcus tetradius</td><td>Clostridium tetani</td><td>eplosireplocOccas a Photobacterium damselae</td></tr><tr><td></td><td></td><td></td></tr><tr><td>Anaerostipes hadrus a,d</td><td>Collinsella aerofaciens</td><td>Porphyromonas asaccharolytica</td></tr><tr><td>Arcobacter butzleri</td><td>Corynebacterium genitalium</td><td>Prevotella bivia h</td></tr><tr><td>Arcobacter cryaerophilus</td><td>Desulfovibrio piger</td><td>Prevotella copri h</td></tr><tr><td>Bacillus cereus</td><td>Diffusely adherent E.coli</td><td>Prevotella intermedia h</td></tr><tr><td>Bacteroides fragilis</td><td>Escherichia blattae</td><td>Prevotella histicola h</td></tr><tr><td>Bacteroides thetaiotaomicron</td><td>Escherichia fergusonii</td><td>Prevotella melaninogenica h</td></tr><tr><td>Bacteroides vulgatus</td><td>Escherichia hermannii</td><td>Proteus mirabilis</td></tr><tr><td>Bifidobacterium adolescentis b</td><td>Escherichia vulneris</td><td>Proteus penneri</td></tr><tr><td>Bifidobacterium bifidum b</td><td>Edwardsiella tarda</td><td>Proteus vulgaris</td></tr><tr><td>Bifidobacterium longum b</td><td>Egglerthella lenta</td><td>Providencia alcalifaciens</td></tr><tr><td>Bifidobacterium pseudocatenulatum</td><td>Enterobacter cloacae</td><td>Pseudomonas aeruginosa</td></tr><tr><td>Blautia (Ruminococcus) obeum</td><td>Enterobacter hormaechei d,e</td><td>Ruminococcus bromii b</td></tr><tr><td>Blautia wexlerae</td><td>Enterococcus faecalis</td><td>Ruminococcus flavefaciens b</td></tr><tr><td>Campylobacter concisus</td><td>Enterococcus faecium</td><td>Selenomonas ruminantium</td></tr><tr><td>Campylobacter curvus</td><td>Eubacterium cylindroides</td><td>Serratia liquefaciens</td></tr><tr><td>Campylobacter fetus</td><td>Eubacterium rectale</td><td>Serratia marcescens</td></tr><tr><td>Campylobacter gracilis</td><td>Faecalibacterium prausnitzii</td><td>Shewanella algae</td></tr><tr><td>Campylobacter helveticus</td><td>Fusobacterium varium</td><td>Staphylococcus aureus</td></tr><tr><td>Campylobacter hominis</td><td>Gardnerella vaginalis</td><td>Staphylococcus epidermidis</td></tr><tr><td>Campylobacter hyointestinalis</td><td>Gemella morbillorum</td><td>Stenotrophomonas maltophilia</td></tr><tr><td>Campylobacter lari</td><td>Grimontia (Vibrio) hollisae</td><td>Streptococcus agalactiae</td></tr><tr><td>Campylobacter mucosalis</td><td>Haemophilus influenzae</td><td>Streptococcus intermedius</td></tr><tr><td>Campylobacter rectus</td><td>Hafnia alveic</td><td>Streptococcus pyogenes</td></tr><tr><td>Campylobacter showae</td><td>Helicobacter fennelliae</td><td>Streptococcus salivarius</td></tr><tr><td>Campylobacter sputorum</td><td>Helicobacter pylori</td><td>Trabulsiella guamensis</td></tr><tr><td>Campylobacter ureolyticus</td><td>Klebsiella (Enterobacter) aerogenes</td><td>Veillonella parvula</td></tr><tr><td>Cedecea davisae</td><td>Klebsiella oxytoca</td><td>Vibrio alginolyticus</td></tr><tr><td>Chlamydia trachomatis</td><td>Klebsiella pneumoniae</td><td>Vibrio fluvialis</td></tr><tr><td>Citrobacter amalonaticus</td><td>Lactobacillus acidophilus</td><td>Vibrio mimicus</td></tr><tr><td>Citrobacter freundii</td><td>Lactobacillus reuteri</td><td>Yersinia bercovieri</td></tr><tr><td>Citrobacter koseri d</td><td>Lactococcus lactis</td><td>Yersinia frederiksenii </td></tr><tr><td>Citrobacter sedlakii</td><td>Leminorella grimontii</td><td>Yersinia intermedia</td></tr><tr><td>Clostridium acetobutylicum</td><td>Listeria monocytogenes</td><td>Yersinia kristensenii</td></tr><tr><td>Clostridium botulinum</td><td>Mediterraneibacter (Ruminococcus) gnavus</td><td>Yersinia mollaretii</td></tr><tr><td>Clostridium difficile non-toxigenic d</td><td>Megamonas hypermegale</td><td>Yersinia pseudotuberculosis</td></tr><tr><td>Clostridium histolyticum</td><td>Megasphaeara elsdenii</td><td>Yersinia rohdei</td></tr><tr><td>Clostridium methylpentosum</td><td>Methanobrevibacter smithii</td><td>Eu</td></tr></table>

<table><tr><td colspan="2">Protozoa/Parasites</td><td>Fungi</td></tr><tr><td>Tested</td><td>In silico Analysis Only</td><td>Tested</td></tr><tr><td>Babesia microti</td><td>Ancylostoma duodenale</td><td>Aspergillus fumigatus</td></tr><tr><td>Blastocystis hominis</td><td>Ascaris lumbricoides</td><td>Candida albicans</td></tr><tr><td>Conidiobolus lachnodes</td><td>Balantidium coli</td><td>Candida catenulate</td></tr><tr><td colspan="1" rowspan="2">Conidiobolus lobatusEncephalitozoon hellemEncephalitozoon intestinalisEntamoeba disparEntamoeba gingivalisEntamoeba moshkovskiiGiardia murisPentatrichomonas hominisSchistosoma mansoniToxoplasma gondiiTrichomonas tenax</td><td colspan="2" rowspan="2">Chilomastix mesniliDientamoeba fragilisEndolimax nanaEntamoeba coliEntamoeba hartmanniEntamoeba poleckiEnterobius vermicularisEnteromonas hominisIsospora belliNecator americanus</td><td colspan="1" rowspan="2">Penicillium marneffeiSaccharomyces boulardiSaccharomyces cerevisiae</td></tr><tr><td colspan="1" rowspan="1">Dientamoeba fragilisEndolimax nana</td></tr><tr><td colspan="4" rowspan="1">Viruses</td></tr><tr><td colspan="3" rowspan="1">Tested</td><td colspan="1" rowspan="1">In silico Analysis Only</td></tr><tr><td colspan="3" rowspan="1">Adenovirus A:31                             Coronavirus 229EAdenovirus B:34                             Coxsackievirus B3Adenovirus C:2                              Cytomegalovirus (CMV)Adenovirus D:37                             Echovirus 6Adenovirus E:4a                             Enterovirus 68Astrovirus variant VA1                      Hepatitis AAstrovirus variant MLB                     Herpes Simplex Type 2Bocavirus Type 1                             Rhinovirus 1A</td><td colspan="1" rowspan="1">Adenovirus G52Norovirus GIVRotavirus BRotavirus C</td></tr></table>

# 4. Assay Reportable Range:

The BIOFIRE FILMARRAY GI Panel is a qualitative assay thus the assay reportable range is not applicable.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

No changes were made to the assay, assay reagents, instrument, or assay software. For additional information, please refer to the original published decision summary of K143005.

6. Detection Limit:

A limit of detection (LoD) study was performed to support the original BIOFIRE FILMARRAY GI Panel (K140407). Since no changes were made to the assay, assay reagents, instrument, or assay software, no new LoD study was conducted.

# 7. Assay Cut-Off:

The BIOFIRE system evaluates DNA melting curve data rather than real-time PCR curves or a measurement value (e.g. Ct values or RFU) to determine positive and negative results; hence, the system does not utilize cut-offs as traditionally defined. For additional information regarding the assay cut-offs refer to the published decision summary for the BIOFIRE FILMARRAY GI Panel (K140407).

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison: Not applicable.

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable.

2. Clinical Specificity:

The BIOFIRE FILMARRAY GI Panel Norovirus GI/GII negative percent agreement (NPA) was originally established in a prospective clinical study performed in 2013 to support the original clearance of the device (K140407). These clinical study results for the Norovirus GI/GII assay are summarized in Table 3 below.

Table 3. BIOFIRE FILMARRAY GI Panel Norovirus GI/GII Clinical Performance Summary (2013)   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>52/55</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>84.9-98.9%</td><td rowspan=1 colspan=1>1483/1501</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.1-99.3%</td></tr></table>

To evaluate potential risk of false positive results with the Norovirus GI/GII target, an additional supplemental clinical study was conducted. The study was conducted with 872 fresh stool specimens collected and tested at three geographically distinct clinical sites within the U.S. Patient demographic information from this study is summarized in Table 4 below.

Table 4. Demographic Data for Prospectively Collected Specimens   

<table><tr><td>Study Specimens</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Total Specimens</td><td rowspan=1 colspan=1>872</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>394 (45.2%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>478 (54.8%)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number of Specimens (%)</td></tr><tr><td rowspan=1 colspan=1>&lt; 1 year</td><td rowspan=1 colspan=1>70 (8.0%)</td></tr><tr><td rowspan=1 colspan=1>1-5 years</td><td rowspan=1 colspan=1>120 (13.8%)</td></tr><tr><td rowspan=1 colspan=1>6-12years</td><td rowspan=1 colspan=1>64 (7.3%)</td></tr><tr><td rowspan=1 colspan=1>13-21years</td><td rowspan=1 colspan=1>127 (14.6%)</td></tr><tr><td rowspan=1 colspan=1>22-64 years</td><td rowspan=1 colspan=1>283 (32.5%)</td></tr><tr><td rowspan=1 colspan=1>65+ years</td><td rowspan=1 colspan=1>208 (23.9%)</td></tr></table>

Frozen aliquots of each specimen were tested at bioMerieux using the CDC CaliciNet Norovirus GI/GII assay, the same Norovirus comparator assay used in the 2013 clinical study. The use of frozen specimens was supported by a fresh vs. frozen equivalency study conducted in the original submission (K140407). Of note, the current CDC CaliciNet Norovirus GI/GII assay has been updated from the version used as the comparator in the 2013 clinical study. The updated assay includes new probe designs and quantification cycle cut-off thresholds. Specimens used for comparator testing were randomized and blinded to test operators. Testing with the BIOFIRE FILMARRAY GI Panel resulted in an apparent detection rate for the Norovirus GI/GII assay of $7 . 2 \%$ (63/872). A comparison of the BIOFIRE FILMARRAY GI Panel Norovirus GI/GII assay results with the comparator method are summarized in Table 5.

Table 5. BIOFIRE FILMARRAY GI Panel Norovirus GI/GII Performance   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Comparator</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>GI PanelPerformance</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Cand c</td><td rowspan=1 colspan=1>Re</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>34/35(97.1%)</td><td rowspan=1 colspan=1>85.1-99.9%</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>808</td><td rowspan=1 colspan=1>808/837(96.5%)</td><td rowspan=1 colspan=1>95.1-97.7%</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>837</td><td rowspan=1 colspan=2></td></tr></table>

Samples with discrepant results were further analyzed by an independent molecular method and bidirectional sequencing. Investigation of the single false negative result from Table 3 showed evidence of Norovirus using the independent molecular method with bidirectional sequencing. Additional investigation of the 29 false positives observed in the study indicated evidence of Norovirus in three specimens. Sequences from various non-viral targets (including Anaerostipes hadrus, Escherichia coli, Eubacterium sp., Homo sapiens, Jutongia huaianensis, Klebsiella pneumoniae, Parabacteroides distasonis, Parabacteroides merdae, Prevotella sp., and Ruminococcus gnavus) was observed in 20 false positive specimens. Several of these organisms were evaluated for cross-reactivity with BIOFIRE FILMARRAY GI Panel by wet testing or in silico methods (see Tables 1 & 2). The cause of the six remaining false positive specimens was suspected to be variable detection near or below the LoD of the comparator assay or unexpected primer dimer amplification. While the observed NPA results of in this supplemental clinical study indicated a minor decrease in the lower

bound of the $9 5 \%$ confidence interval when compared to the original performance evaluation of the device, the results are acceptable.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

Expected Values for all analytes on the BIOFIRE FILMARRAY GI Panel were obtained in the 2013 prospective clinical study and are summarized in the published decision summary for K140407. Expected Values for the supplemental study presented in this submission have been stratified by study site and are summarized in Table 6 below. The prevalence of positive Norovirus GI/GII assay results in this study stratified by age group are presented in Table 7.

Table 6. Expected Values by Clinical Site   

<table><tr><td rowspan=2 colspan=1>GI PanelResult</td><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td></tr><tr><td rowspan=1 colspan=1>#/SAa</td><td rowspan=1 colspan=1>EVb</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>63/872</td><td rowspan=1 colspan=1>7.2%</td><td rowspan=1 colspan=1>8/253</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>43/337</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>12/282</td><td rowspan=1 colspan=1>4.3%</td></tr></table>

${ } ^ { \mathrm { a } } \# / \mathrm { S A } =$ number of GI Panel positives divided by the number of specimens analyzed (i.e., with valid comparator results) for that analyte ${ } ^ { \mathsf { b } } \operatorname { E V } =$ Expected Value

Table 7. Expected Values by Age Group   

<table><tr><td rowspan=2 colspan=1>GI PanelResult</td><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=2>&lt;1 year</td><td rowspan=1 colspan=2>1-5 years</td><td rowspan=1 colspan=2>6-12 years</td><td rowspan=1 colspan=2>13-21years</td><td rowspan=1 colspan=2>22-64 years</td><td rowspan=1 colspan=2>65+years</td></tr><tr><td rowspan=1 colspan=1>#/SAa</td><td rowspan=1 colspan=1>EVb</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#/SA</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>63/872</td><td rowspan=1 colspan=1>7.2%</td><td rowspan=1 colspan=1>13/70</td><td rowspan=1 colspan=1>18.6%</td><td rowspan=1 colspan=1>16/120</td><td rowspan=1 colspan=1>13.3%</td><td rowspan=1 colspan=1>7/64</td><td rowspan=1 colspan=1>10.9%</td><td rowspan=1 colspan=1>11/127</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>12/283</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>4/207</td><td rowspan=1 colspan=1>1.9%</td></tr></table>

${ } ^ { \mathrm { a } } \# / \mathrm { S A } =$ number of GI Panel positives divided by the number of specimens analyzed (i.e., with valid comparator results) for that analyte ${ } ^ { \mathsf { b } } \operatorname { E V } =$ Expected Value

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.