Luminex Corporation   
Sheri Calderon   
Senior Regulatory Affairs Associate   
4088 Commercial Avenue   
Northbrook, Illinois 60062

Re: K243013 Trade/Device Name: LIAISON PLEX Gram-Negative Blood Culture Assay Regulation Number: 21 CFR 866.3365 Regulation Name: Multiplex Nucleic Acid Assay For Identification Of Microorganisms And Resistance Markers From Positive Blood Cultures Regulatory Class: Class II Product Code: PEN, NSU Dated: March 21, 2025 Received: March 21, 2025

Dear Sheri Calderon:

We have reviewed your section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Additional information about changes that may require a new premarket notification are provided in the FDA guidance documents entitled "Deciding When to Submit a 510(k) for a Change to an Existing Device" (https://www.fda.gov/media/99812/download) and "Deciding When to Submit a 510(k) for a Software Change to an Existing Device" (https://www.fda.gov/media/99785/download).

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

All medical devices, including Class I and unclassified devices and combination product device constituent parts are required to be in compliance with the final Unique Device Identification System rule ("UDI Rule"). The UDI Rule requires, among other things, that a device bear a unique device identifier (UDI) on its label and package (21 CFR 801.20(a)) unless an exception or alternative applies (21 CFR 801.20(b)) and that the dates on the device label be formatted in accordance with 21 CFR 801.18. The UDI Rule (21 CFR 830.300(a) and 830.320(b)) also requires that certain information be submitted to the Global Unique Device Identification Database (GUDID) (21 CFR Part 830 Subpart E). For additional information on these requirements, please see the UDI System webpage at https://www.fda.gov/medical-devices/device-advicecomprehensive-regulatory-assistance/unique-device-identification-system-udi-system.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medicaldevices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn (https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatory

Sincerely,

# Noel J. Gerald -S

Noel J. Gerald, Ph.D.   
Deputy Division Director   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K243013

Device Name LIAISON PLEX $\textsuperscript { \textregistered }$ Gram-Negative Blood Culture Assay

Indications for Use (Describe)   
The LIAISON PLEX $\textsuperscript { \textregistered }$ Gram-Negative Blood Culture (BCN) Assay, performed using the automated, sample-to-result LIAISON PLEX $\textsuperscript { \textregistered }$ System, is a qualitative multiplexed in vitro diagnostic test for the simultaneous detection and identification of selected gram-negative pathogens and/or selected genetic determinants associated with antimicrobial resistance in positive blood culture bottles. LIAISON PLEX $\textsuperscript { \textregistered }$ BCN Assay is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system which contain gramnegative bacteria as determined by Gram stain.

The LIAISON PLEX $\textsuperscript { \textregistered }$ BCN Assay detects and identifies the following:

Resistance Markers: CTX-M (blaCTX-M) IMP (blaIMP) KPC (blaKPC) NDM (blaNDM) OXA (blaOXA) VIM (blaVIM) MCR SME (blaSME)

Gram Negative Genera and Species: Enterobacteriaceae / Morganellaceae Acinetobacter baumannii   
Acinetobacter spp.   
Citrobacter spp.   
Enterobacter spp. (1)   
Escherichia coli (2)   
Haemophilus influenzae   
Klebsiella oxytoca   
Klebsiella pneumoniae   
Klebsiella variicola   
Morganella morganii   
Neisseria meningitidis   
Proteus spp.   
Pseudomonas aeruginosa   
Pseudomonas spp.   
Salmonella spp.   
Serratia marcescens   
Stenotrophomonas maltophilia (1) Due to reclassification, Klebsiella aerogenes will be reported Enterobacter spp.   
(2) LIAISON PLEX $^ \mathrm { \textregistered }$ BCN Assay will not distinguish between Escherichia coli and Shigella spp. (S. dysenteriae, S. boydii, S. flexneri and S. sonnei)

LIAISON PLEX $\textsuperscript { \textregistered }$ BCN Assay contains targets for the detection of genetic determinants associated with resistance to carbapenems (blaCTX-M, blaIMP, blaKPC, blaNDM, blaOXA48-like, blaVIM, blaSME) to aid in the identification of potentially antimicrobial-resistant organisms in positive blood culture samples. In addition, the panel includes an assay for the detection of the mobilized genetic determinant MCR, an emerging marker of public health importance. The antimicrobial resistance gene or marker detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene and marker assays do not indicate susceptibility, as multiple mechanisms of resistance to ß-lactams and colistin exist.

LIAISON PLEX $\textsuperscript { \textregistered }$ BCN Assay is indicated for use in conjunction with other clinical and laboratory findings to aid in the diagnosis of bacterial bloodstream infections (BSI). LIAISON PLEX $\textsuperscript { \textregistered }$ BCN Assay is not intended to monitor treatment of these infections. Sub-culturing of positive blood cultures is necessary to recover organisms for antimicrobial susceptibility testing (AST), for identification of organisms not detected by LIAISON PLEX $^ \mathrm { \textregistered }$ BCN Assay, to detect mixed infections that may not be detected by LIAISON PLEX $^ \mathrm { \textregistered }$ BCN Assay, for association of antimicrobial resistance marker genes to a specific organism, or for epidemiological typing.

Type of Use (Select one or both, as applicable)

Prescription Use (Part 21 CFR 801 Subpart D)

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# Diasorin

# 510(k) Summary

This Summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

Preparation date: 16 April 2025

A. 510(k) Number: K243013   
B. Purpose for Submission: Traditional 510(k), New Device

# C. Measurand:

Nucleic acid sequences for the following organisms: Enterobacteriaceae / Morganellaceae, Acinetobacter baumannii, Acinetobacter spp., Citrobacter spp., Enterobacter spp., Escherichia coli, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella pneumoniae, Klebsiella variicola, Morganella morganii, Neisseria meningitidis, Proteus spp., Pseudomonas aeruginosa, Pseudomonas spp., Salmonella spp., Serratia marcescens, and Stenotrophomonas maltophilia. Nucleic acid sequences for the following resistance markers: CTX‐M (blaCTX‐M), IMP (blaIMP), KPC (blaKPC), NDM (blaNDM), OXA (blaOXA), VIM (blaVIM), MCR, and SME (blaSME).

# D. Type of Test:

Qualitative Multiplexed Direct Detection Hybridization Assay

# E. Applicant:

Sheri Calderon, Luminex Corporation   
4088 Commercial Avenue   
Northbrook, IL 60062   
(847) 400‐9000

# F. Proprietary and Established Names:

LIAISON PLEX® Gram‐Negative Blood Culture Assay

# G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PEN</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>21 CFR 866.3365 - Multiplex Nucleic Acid Assayfor Identification of Microorganisms andResistance Markers from Positive Blood Cultures</td><td rowspan=1 colspan=1>83 (Microbiology)</td></tr></table>

# Diasorin

# H. Intended Use:

# 1. Intended use(s):

The LIAISON PLEX® Gram‐Negative Blood Culture (BCN) Assay, performed using the automated, sample‐to‐result LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ System, is a qualitative multiplexed in vitro diagnostic test for the simultaneous detection and identification of selected gram‐negative pathogens and/or selected genetic determinants associated with antimicrobial resistance in positive blood culture bottles. LIAISON PLEX® BCN Assay is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system which contain gram‐negative bacteria as determined by Gram stain.

The LIAISON PLEX® BCN Assay detects and identifies the following:

<table><tr><td rowspan=1 colspan=1>Resistance Markers</td><td rowspan=1 colspan=1>Gram Negative Genera and Species</td></tr><tr><td rowspan=1 colspan=1>CTX-M (blaCTX-M)</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td></tr><tr><td rowspan=1 colspan=1>IMP (blaIMP)</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td></tr><tr><td rowspan=1 colspan=1>KPC (blaKPC)</td><td rowspan=1 colspan=1>Acinetobacter spp.</td></tr><tr><td rowspan=1 colspan=1>NDM (blaNDM)</td><td rowspan=1 colspan=1>Citrobacter spp.</td></tr><tr><td rowspan=1 colspan=1>OXA (blaOXA)</td><td rowspan=1 colspan=1>Enterobacter spp.1</td></tr><tr><td rowspan=1 colspan=1>VIM (blaVIM)</td><td rowspan=1 colspan=1>Escherichia coli 2</td></tr><tr><td rowspan=1 colspan=1>MCR</td><td rowspan=1 colspan=1>Haemophilus influenzae</td></tr><tr><td rowspan=1 colspan=1>SME (blaSME)</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td></tr><tr><td rowspan=10 colspan=1></td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas spp.</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr></table>

1 Due to reclassification, Klebsiella aerogenes will be reported Enterobacter spp. 2 LIAISON PLEX BCN Assay will not distinguish between Escherichia coli and Shigella spp. (S. dysenteriae, S. boydii, S. flexneri and S. sonnei)

LIAISON PLEX® BCN Assay contains targets for the detection of genetic determinants associated with resistance to carbapenems $( b I a _ { \mathrm { C T X - M } } , b I a _ { \mathrm { 1 M P } } , b I a _ { \mathrm { K P C } } , b I a _ { \mathrm { N D M } } , b I a _ { \mathrm { O X A 4 8 - l i k e } } , b I a _ { \mathrm { V I M } } , b I a _ { \mathrm { S M E } } )$ to aid in the identification of potentially antimicrobial‐resistant organisms in positive blood culture samples. In addition, the panel includes an assay for the detection of the mobilized genetic determinant MCR, an emerging marker of public health importance. The antimicrobial resistance gene or marker detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene and marker assays do not indicate susceptibility, as multiple mechanisms of resistance to ß‐lactams and colistin exist.

LIAISON PLEX® BCN Assay is indicated for use in conjunction with other clinical and laboratory

# Diasorin

findings to aid in the diagnosis of bacterial bloodstream infections (BSI). LIAISON PLEX® BCN Assay is not intended to monitor treatment of these infections. Sub‐culturing of positive blood cultures is necessary to recover organisms for antimicrobial susceptibility testing (AST), for identification of organisms not detected by LIAISON PLEX® BCN Assay, to detect mixed infections that may not be detected by LIAISON PLEX® BCN Assay, for association of antimicrobial resistance marker genes to a specific organism, or for epidemiological typing.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): For prescription use only. For in vitro diagnostic use only.

4. Special instrument requirements: For use with LIAISON PLEX® Systems.

# I. Device Description:

The LIAISON PLEX® Gram‐Negative Blood Culture (BCN) Assay is an automated test for the detection and identification of nucleic acid from gram‐negative bacteria in a positive blood culture media sample. The BCN Assay is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system, and which contain gram‐negative bacteria, as determined by a Gram stain.

The LIAISON PLEX® System is a fully automated, bench‐top "sample‐to‐answer" device that performs sample preparation, polymerase chain reaction (PCR) and microarray‐based hybridization for the detection of target‐specific nucleic acids. The test reagents are supplied as a single, disposable test cartridge. PCR is not performed on the LIAISON PLEX® BCN Assay, as it is a non‐ amplified, direct detection test performed on the LIAISON PLEX® System.

# J. Substantial Equivalence Information:

1. Predicate device name(s): VERIGENE Blood Culture Gram‐Negative (BC‐GN) Nucleic Acid Test

2. Predicate 510(k) number(s): K132843

3. Comparison with predicate: The following tables compare the LIAISON PLEX® Gram‐Negative Blood Culture Assay to the VERIGENE Gram‐Negative Blood Culture (BC‐GN) Nucleic Acid Test.

<table><tr><td rowspan=1 colspan=1>Comparisonto PredicateDevice</td><td rowspan=1 colspan=1>Predicate Device: K132843VERIGENE Gram-Negative Blood Culture(BC-GN) Nucleic Acid Test, K132843</td><td rowspan=1 colspan=1>Candidate Device: K243013LIAISON PLEX® Gram-Negative BloodCulture Assay</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>PEN</td><td rowspan=1 colspan=1>PEN</td></tr><tr><td rowspan=1 colspan=1>Regulation Number</td><td rowspan=1 colspan=1>21 CFR 866.3365</td><td rowspan=1 colspan=1>21 CFR 866.3365</td></tr><tr><td rowspan=1 colspan=1>Organism Detected</td><td rowspan=1 colspan=1>Organisms: Acinetobacter spp.,Citrobacter spp., Enterobacter spp.,Proteus spp., Escherichia coli, Klebsiellapneumoniae, Klebsiella oxytoca, andPseudomonas aeruginosaResistance Markers: CTX-M (blaCTX-M),KPC (blaKPC), NDM (blaNDM), VIM(blaVIM), IMP (blaIMP), and OXA (blaOXA)</td><td rowspan=1 colspan=1>Organisms: Enterobacteriaceae /Morganellaceae, Acinetobacterbaumannii, Acinetobacter spp.,Citrobacter spp., Enterobacter spp.,Escherichia coli, Haemophilus influenzae,Klebsiella oxytoca, Klebsiella pneumoniae,Klebsiella variicola, Morganella morganii,Neisseria meningitidis, Proteus spp.,Pseudomonas aeruginosa, Pseudomonasspp., Salmonella spp., Serratiamarcescens, and StenotrophomonasmaltophiliaResistance markers: CTX-M (blaCTX-M),IMP (blaIMP), KPC (blaKPC), NDM(blaNDM), OXA (blaOXA), VIM (blaVIM),MCR, and SME (blaSME).</td></tr><tr><td rowspan=1 colspan=1>Measurand</td><td rowspan=1 colspan=1>Nucleic acid from Organisms detected</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=3 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The Verigene® Gram Negative Blood CultureNucleic Acid Test (BC-GN), performed usingthe sample-to-result Verigene System, is aqualitative multiplexed in vitro diagnostic testfor the simultaneous detection andidentification of selected gram-negativebacteria and resistance markers. BC-GN isperformed directly on blood culture mediausing blood culture bottles identified aspositive by a continuous monitoring bloodculture system and which contain gram-negative bacteria as determined by gramstain.BC-GN detects and identifies the following:</td><td rowspan=1 colspan=1>The LIAISON PLEX® Gram-Negative BloodCulture (BCN) Assay, performed using theautomated, sample-to-result LIAISONPLEX® System, is a qualitative multiplexedin vitro diagnostic test for the simultaneousdetection and identification of selectedgram-negative pathogens and/or selectedgenetic determinants associated withantimicrobial resistance in positive bloodculture bottles. BCN is performed directlyon blood culture media using blood culturebottles identified as positive by acontinuous monitoring blood culturesystem and which contain gram-negativebacteria as determined by Gram stain.The BCN Assay detects and identifies thefollowing:</td></tr><tr><td rowspan=2 colspan=2>Bacterial Genera and Species               bacteria as determined by Gram stain.Acinetobacter spp.                           The BCN Assay detects and identifies theCitrobacter spp.                              following:Enterobacter spp.                              Gram Negative Genera and SpeciesProteus spp.                                     Enterobacteriaceae /Escherichia coli1                                    MorganellaceaeKlebsiella pneumoniae                         Acinetobacter baumanniiKlebsiella oxytoca                              Acinetobacter spp.Pseudomonas aeruginosa                      Citrobacter spp.Resistance Markers                            Enterobacter spp.1CTX-M (blaCTX-M)                              Escherichia coli²KPC (blaKPC)                                    Haemophilus influenzaeNDM (blaNDM)                                 Klebsiella oxytocaVIM (blaVIM)                                   Klebsiella pneumoniaeIMP (blaIMP)                                    Klebsiella variicola</td></tr><tr><td rowspan=1 colspan=1>Resistance MarkersCTX-M (blaCTX-M)KPC (blaKPC)NDM (blaNDM)VIM (blaVIM)IMP (blaIMP)</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="3" rowspan="1">Diasorin             LIAISON PLEX® Gram-Negative Blood Culture Assay Traditional 510(k)</td></tr><tr><td colspan="1" rowspan="1">Comparisonto PredicateDevice</td><td colspan="1" rowspan="1">Predicate Device: K132843VERIGENE Gram-Negative Blood Culture(BC-GN) Nucleic Acid Test, K132843</td><td colspan="1" rowspan="1">Candidate Device: K243013LIAISON PLEX® Gram-Negative BloodCulture Assay</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">OXA (blaOXA)</td><td colspan="1" rowspan="2">Morganella morganiiNeisseria meningitidisProteus spp.Pseudomonas aeruginosaPseudomonas spp.Salmonella spp.Serratia marcescensStenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="3">1BC-GN will not distinguish Escherichia coli fromSla . .e,S. and S. sonnei)BC-GN is indicated for use in conjunction withother clinical and laboratory findings to aid inthe diagnosis of bacterial bloodstreaminfections; however, it is not used to monitorthese infections. Sub-culturing of positiveblood cultures is necessary to recoverorganisms for antimicrobial susceptibilitytesting (AST), for identification of organismsnot detected by BC-GN, to detect mixedinfections that may not be detected by BC-GN, for association of antimicrobial resistancemarker genes to a specific organism, or forepidemiological typing.</td></tr><tr><td colspan="1" rowspan="1">Resistance MarkersCTX-M (blaCTX-M)IMP (blalMP)KPC (blaKPC)NDM (blaNDM)OXA (blaOXA)VIM (blaVIM)MCRSME (blaSME)</td></tr><tr><td colspan="1" rowspan="1">1 Due to reclassification, Klebsiella aerogeneswill be reported Enterobacter spp.2 LIAISON PLEX BCN will not distinguishbetween Escherichia coli and Shigella spp. (S.dysenteriae, S. boydii, S. flexneri and S.sonnei)LIAISON PLEX® BCN Assay contains targetsfor the detection of genetic determinantsassociated with resistance to carbapenems(blaCTX-M, blaIMP, blaKPC, blaNDM,blaOXA48-like, blaVIM, blaSME) to aid inthe identification of potentiallyantimicrobial-resistant organisms inpositive blood culture samples. In addition,the panel includes an assay for thedetection of the mobilized geneticdeterminant MCR, an emerging marker ofpublic health importance. Theantimicrobial resistance gene or markerdetected may or may not be associatedwith the agent responsible for disease.Negative results for these selectantimicrobial resistance gene and markerassays do not indicate susceptibility, asmultiple mechanisms of resistance to -lactams and colistin exist.LIAISON PLEX® BCN Assay is indicated foruse in conjunction with other clinical andlaboratory findings to aid in the diagnosisof bacterial bloodstream infections (BSI).LIAISON PLEX® BCN Assay is not intended tomonitor treatment of these infections.</td></tr><tr><td colspan="1" rowspan="1">Comparisonto PredicateDevice</td><td colspan="1" rowspan="1">Predicate Device: K132843VERIGENE Gram-Negative Blood Culture(BC-GN) Nucleic Acid Test, K132843</td><td colspan="1" rowspan="1">Candidate Device: K243013LIAISON PLEX® Gram-Negative BloodCulture Assay</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Sub-culturing of positive blood cultures isnecessary to recover organisms forantimicrobial susceptibility testing (AST),for identification of organisms notdetected by LIAISON PLEX® BCN Assay, todetect mixed infections that may not bedetected by LIAISON PLEX® BCN Assay, forassociation of antimicrobial resistancemarker genes to a specific organism, or forepidemiological typing.</td></tr><tr><td colspan="1" rowspan="1">Automated System(Sample to Answer)</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">VERIGENE</td><td colspan="1" rowspan="1">LIAISON PLEX</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Positive Blood Culture</td><td colspan="1" rowspan="1">Same</td></tr></table>

# Diasorin

# K. Standards/Guidance Documents Referenced:

Class II Special Controls Guideline: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures (May 2015)

Electronic Submission Template for Medical Device 510(k) Submissions ‐ Guidance for Industry and Food and Drug Administration Staff (October 2, 2023).

 Content of Premarket Submissions for Device Software Functions ‐ Guidance for Industry and Food and Drug Administration Staff (June 14, 2023).

Cybersecurity in Medical Devices: Quality System Considerations and Content of Premarket Submissions ‐ Guidance for Industry and Food and Drug Administration Staff (September 23, 2023).

Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests ‐ Guidance for Industry and FDA Staff (March 13, 2007).

CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline ‐ Second Edition. CLSI document EP12‐A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25‐A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

 ISO 14971:2019 Medical devices ‐ Application of risk management to medical devices

IEC 62366‐1:2015 $^ +$ A1:2020 Medical devices ‐ Part 1: Application of usability engineering to medical devices

 ISO 62304:2006 Medical device software ‐ Software life‐cycle processes

ISO 15223‐1:2021: Medical Devices ‐ Symbols to be used with medical device labels, labeling and information to be supplied ‐ Part 1: General requirements

IEC 61010‐1 Ed. 3.1 2017‐01: Safety requirements for electrical equipment for measurement, control, and laboratory use ‐ Part 1: General requirements

EN 61010‐2‐101:2002/IEC 61010‐2‐101:2015: Safety requirements for electrical equipment for measurement, control and laboratory use  ‐  Part 2‐101: Particular requirements for in vitro diagnostic (IVD) medical equipment.

IEC 60601‐1‐2:2014 (Edition 4.0): Medical electrical equipment ‐ Part 1‐2: General requirements for basic safety and essential performance ‐ Collateral Standard: Electromagnetic disturbances ‐ Requirements and tests

ISO 13485:2016/EN ISO 13485:2016; Medical devices ‐ Quality Management System ‐ Requirements for regulatory purposes

ISO 20916:2019; In vitro diagnostic medical devices. Clinical performance studies using specimens from human subjects. Good study practice

EN ISO 18113‐1:2011; In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling). Terms, definition and general requirements

EN ISO 18113‐2:2011; In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use

# Diasorin

EN ISO 18113‐3:2011; In vitro diagnostic medical devices ‐ Information supplied by the manufacturer (labeling) – Part 3: In vitro diagnostic instruments for professional use

EN ISO 23640:2015; In vitro diagnostic medical devices ‐ Evaluation of stability of in vitro diagnostic reagents

IEC 61326‐1:2012; Electrical equipment for measurement control and laboratory use ‐ EMC requirements ‐ Part 1: General requirements

EN 61326‐2‐6:2006/IEC 61326‐2‐6:2012; Electrical equipment for measurement control and laboratory use ‐ EMC requirements ‐ Part 2‐6: Particular requirements ‐ In vitro diagnostic (IVD) medical equipment

# L. Test Principle:

The LIAISON PLEX® Gram‐Negative Blood Culture (BCN) Assay is performed directly on blood culture media using blood culture bottles identified as positive by a continuous monitoring blood culture system, and which contain gram‐negative bacteria, as determined by a Gram stain. The system consists of an instrument and a single‐use, disposable test cartridge. The user loads an aliquot of the sample into the sample port of the LIAISON PLEX® Gram‐Negative Blood Culture Assay Cartridge. Next, the user sets up the sample order on the LIAISON PLEX® System by first entering the sample information or scanning the barcode ID located on the sample tube, then scanning the barcode ID located on the test cartridge. Last, the user inserts the test cartridge into the processing module to initiate the test. The LIAISON PLEX® System identifies the assay being run and automatically initiates the proper testing protocol to process the sample, analyze the data, and generate test results.

The LIAISON PLEX® System automates the LIAISON PLEX® BCN Assay sample analysis through the following steps: a) Sample Preparation: Nucleic acid extraction via mechanical and chemical cell lysis and magnetic bead‐based nucleic acid isolation; b) Hybridization: Extracted nucleic acid hybridize to target‐specific capture DNA on a microarray format, and target‐specific mediator and gold nanoparticle probe hybridize to captured nucleic acids; c) Signal Analysis: Gold nanoparticle probes bound specifically to target‐containing spots in the microarray are silver‐enhanced, and light scatter from the spots is measured and further analyzed to determine the presence (Detected) or absence (Not Detected) of a target.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

Site‐to‐site Reproducibility

Site‐to‐site reproducibility of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ BCN Assay was evaluated by testing LIAISON PLEX® BCN Assay cartridges with a reproducibility panel, blinded to operators, consisting of eight blood culture samples; six representative on‐panel organisms

# Diasorin

(Acinetobacter baumannii, Klebsiella pneumoniae, Haemophilus influenzae, Neisseria meningitidis, Escherichia coli, and Serratia marcescens) individually cultured at ring positivity and eight hours after ring positivity, one negative sample contrived with an off‐ panel organism (Staphylococcus aureus), and one negative blood culture matrix (NBM) sample. Each sample of the blinded reproducibility panel was tested in triplicate for each sample type by two operators at three sites, two external and one internal, for five non‐ consecutive testing days. The call agreement results $( \% )$ are presented in Table 1, which demonstrated $100 \%$ call accuracy for ring positive, ring positive plus 8 hours, and negative blood matrix samples, and $9 9 . 4 \%$ call accuracy for the contrived negative sample (Staphylococcus aureus). Overall, results of site‐to‐site reproducibility evaluation demonstrated $9 9 . 9 \%$ reproducibility of the LIAISON PLEX® BCN Assay.

Table 1. LIAISON PLEX® Gram‐Negative Blood Culture Assay Site‐to‐Site Reproducibility Results Summary   

<table><tr><td rowspan=2 colspan=1>Organism ID</td><td rowspan=2 colspan=1>ReportableTargets</td><td rowspan=2 colspan=1>SampleType</td><td rowspan=1 colspan=4>Call Agreement (%) with Expected Results</td><td rowspan=2 colspan=1>Overall95% C.I.</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall (AllSites</td></tr><tr><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=2 colspan=1>Acinetobacterbaumannii |Acinetobacterspp.</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Klebsiellapneumoniae</td><td rowspan=2 colspan=1>Enterobacteriaceae /Morganellaceae | Klebsiellapneumoniae</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(31/31)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(91/91)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Haemophilusinfluenzae</td><td rowspan=2 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Neisseriameningitidis</td><td rowspan=2 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli</td><td rowspan=2 colspan=1>Enterobacteriaceae /Morganellaceae | Escherichiacoli</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens</td><td rowspan=2 colspan=1>Enterobacteriaceae /Morganellaceae |Serratiamarcescens</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr></table>

# Diasorin

<table><tr><td rowspan=2 colspan=1>Organism ID</td><td rowspan=2 colspan=1>ReportableTargets</td><td rowspan=2 colspan=1>SampleType</td><td rowspan=1 colspan=4>Call Agreement (%) with Expected Results</td><td rowspan=2 colspan=1>Overall95% C.I.</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall (AllSites)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>98.3%(59/60)</td><td rowspan=1 colspan=1>100%(60/60)</td><td rowspan=1 colspan=1>100%(60/60)</td><td rowspan=1 colspan=1>99.4%(179/180)</td><td rowspan=1 colspan=1>96.9% -99.9%</td></tr><tr><td rowspan=1 colspan=1>No Target(Negative BloodMatrix)</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>RingNegative</td><td rowspan=1 colspan=1>100%(60/60)</td><td rowspan=1 colspan=1>100%(60/60)</td><td rowspan=1 colspan=1>100%(60/60)</td><td rowspan=1 colspan=1>100%(180/180)</td><td rowspan=1 colspan=1>97.9% -100%</td></tr><tr><td rowspan=1 colspan=1>cYanmaGYellowb</td><td rowspan=1 colspan=7>Resistance Markers</td></tr><tr><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=2 colspan=1>OXA</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Klebsiellapneumoniae</td><td rowspan=2 colspan=1>KPC</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(31/31)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(91/91)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli</td><td rowspan=2 colspan=1>MCR</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens</td><td rowspan=2 colspan=1>SME</td><td rowspan=1 colspan=1>RingPositive</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>RingPositive +8 HR</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=3>Overall Agreement (All Targets / SampleTypes)</td><td rowspan=1 colspan=1>99.8%(479/480)</td><td rowspan=1 colspan=1>100%(481/481)</td><td rowspan=1 colspan=1>100%(480/480)</td><td rowspan=1 colspan=1>99.9%(1440/1441)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=7>Overall (All Sites) 95% Confidence Interval</td><td rowspan=1 colspan=1>99.6% -100%</td></tr></table>

# Precision/Repeatability

Within laboratory (operator‐to‐operator) precision/repeatability of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ BCN Assay was evaluated based on the results generated by two operators testing site‐to‐site reproducibility samples at ring positivity and at 8 hours plus ring positivity, contrived negative, and negative blood culture samples. Within laboratory precision/repeatability of the LIAISON PLEX® BCN Assay was $9 9 . 8 \%$ , and results are summarized in Table 2.

# Diasorin

Table 2. LIAISON PLEX® Gram‐Negative Blood Culture Assay Within‐Laboratory Precision/Repeatability Results Summary   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Target Type</td><td rowspan=1 colspan=2>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Operator 1</td><td rowspan=1 colspan=1>Operator 2</td></tr><tr><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 Hours</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=2 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 Hours</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=2 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 Hours</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=2 colspan=1>Neisseriameningitidis</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 Hours</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 Hours</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>Ring Positive</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Ring Positive+8 Hours</td><td rowspan=1 colspan=1>100% 15/15</td><td rowspan=1 colspan=1>100% 15/15</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Off-panelNegative</td><td rowspan=1 colspan=1>100% 30/30</td><td rowspan=1 colspan=1>96.7% 29/30</td></tr><tr><td rowspan=1 colspan=1>No Target(Negative BloodMatrix</td><td rowspan=1 colspan=1>Off-panelNegative</td><td rowspan=1 colspan=1>100% 30/30</td><td rowspan=1 colspan=1>100% 30/30</td></tr></table>

![](images/0e85f1fcfff32f234c18bc86a9a5ec79a47ad3e11176c883a2b113ada773dc1e.jpg)

![](images/f7e99df3404313223da87ec613e888a885f001a1ffa8c1d79983a03f72ef8239.jpg)

b. Linearity/assay reportable range: Not applicable. The LIAISON® PLEX Gram‐Negative Blood Culture Assay is a qualitative assay.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Controls Several controls are built into the assay and system to ensure identification of processing

# Diasorin

errors and to establish validity of test results.

Internal Controls

Each LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Gram‐Negative Blood Culture Assay cartridge includes internal controls to ensure performance of sample preparation and detection. The internal extraction control is present in the lysis tube when the sample is added.  Sample preparation is initiated and the extraction control assesses extraction, nucleic acid recovery, and detection. Finally, addition of a post‐extraction hybridization control serves as an indicator of successful hybridization. Internal control results are reported as Pass or Fail on the printed reports (see Table 3 for detailed explanations of each control result). Internal controls must generate a signal above threshold in each internal reaction for the system to report a valid test result.

Table 3. Interpretation of Controls on the LIAISON PLEX® Gram‐Negative Blood Culture Assay Report   

<table><tr><td>Internal Control Result</td><td>Explanation</td><td>Suggested Action</td></tr><tr><td>Pass</td><td>Test was completed and internal controls were successful, indicating that valid results were generated.</td><td>Review and report results</td></tr><tr><td>Fail</td><td>One or more internal control failed.</td><td>Repeat test with a new cartridge</td></tr></table>

# External Controls

Positive and negative external controls should be tested with each new lot or shipment of reagents, or monthly, (whichever occurs first), or in accordance with updated local, regional, state, and/or federal guidelines. Verified negative blood matrix can be used as the negative control. Previously characterized positive samples or verified negative blood matrix spiked with well characterized organisms may be used as the external positive control. External controls should be used in accordance with laboratory protocols and in accordance with local, state, and federal accrediting organizations, as applicable.

# Stability

# Specimen Stability

Performance of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Gram‐Negative Blood Culture (BCN) Assay was assessed using specimens tested in a fresh state (at bottle/ring positive and at bottle/ring positive $^ { + 8 }$ hours) and after exposure to various storage conditions. Conditions tested included refrigerated storage $2 ^ { \circ }$ to $8 ^ { \circ } \mathsf { C }$ ) and room temperature storage $1 5 ^ { \circ } \mathsf { C }$ to $3 0 ^ { \circ } \mathsf { C }$ ) to span the typical “fresh specimen” storage conditions across multiple time points. Positive specimens containing four target organisms representing a total of eight gram‐negative targets and select resistance markers were tested as well as negative blood matrix control specimen containing no target organisms. All positives tested yielded $100 \%$ positivity across all time‐points and storage conditions tested. The negative sample demonstrated $0 \%$ positivity across all time‐points and storage conditions tested. The results demonstrated that specimens may be stored under the following temperature conditions

# Diasorin

without impacting the performance of the LIAISON PLEX® BCN cartridge:

Up to 72 hours (3 days) at Refrigerated $( 2 - 8 ^ { \circ } \mathsf { C } )$ or Room Temperature $\cdot ( 1 5 { - } 3 0 ^ { \circ } \mathsf { C } )$ storage conditions.

# Device Stability

A shelf‐life study was conducted to evaluate the real‐time stability of the LIAISON PLEX® BCN Assay at the recommended storage conditions of room temperature $( 1 5 ^ { \circ } \mathsf { C } - 3 0 ^ { \circ } \mathsf { C } )$ . Real‐time stability was assessed using seven Positive Control panels which interrogate all reportable targets in the assay, and one Negative Control which consisted of negative blood matrix. Results of real‐time stability demonstrated the LIAISON PLEX ${ \mathsf { B C N } } ^ { \otimes }$ Assay is stable for at least 3 months when stored at $1 5 ^ { \circ } \mathsf { C } - 3 0 ^ { \circ } \mathsf { C }$ . Shelf‐life will be extended based upon results of on‐going stability testing.

An open box stability study was performed to evaluate the stability of the LIAISON PLEX® BCN Assay cartridges at room temperature once removed from their foil pouch. Testing was performed shortly after kits were manufactured and will be repeated at the end of the product shelf‐life. Non‐aged $( T _ { \mathrm { i n i t i a l } } )$ cartridges were tested at 0 hours (T0), 2 hours (T2), and 9 hours (T9) after removal from their pouch. Each time point included testing of seven Positive Control panels which interrogate all reportable targets in the assay, and one Negative Control which consisted of negative blood matrix. Results of open‐box stability indicate the cartridges are stable for up to eight hours after cartridges are removed from their foil pouches and stored at room temperature.

# Fresh vs. Frozen Specimen Stability

A fresh vs. frozen specimen stability study was performed to evaluate the performance of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Blood Culture Negative (BCN) Assay across specimens that have been prepared “fresh” (at two growth durations referred to as Bottle/Ring Positive and Bottle/Ring Positive $^ { + 8 }$ hours) and subjected to a range of freeze/thaw (F/T) cycles as well as those experiencing a prolonged storage in frozen conditions. The study was performed using four representative organisms detected by the LIAISON $\mathsf { P l E X } ^ { \otimes }$ BCN Assay cultured in blood culture bottles. Performance testing using negative blood matrix served as a control test during the study. Material was tested under 5 different conditions – Initial Testing: Fresh, 1st Freeze‐thaw, 2nd Freeze‐thaw; 1‐Month Testing:  1st Freeze‐thaw and 2nd Freeze‐thaw. Material was frozen for a minimum of 8 hours in between each freeze‐thaw cycle. In the 1‐Month testing, material was frozen for at least 34 days prior to the 1st freeze‐thaw. A total of 455 replicates were included in this study. The results demonstrated $100 \%$ positivity for target positive samples at both growth durations and all freeze‐thaw conditions. The negative blood demonstrated $0 \%$ positivity of all reportable targets for the assay at all freeze‐thaw conditions.

# d. Growth and Detection Study

The Growth and Detection study was performed to evaluate detection of species / targets listed in Table 4 in blood cultures. For the evaluation, 15 organisms representing all 26 reportable targets were tested at Ring Positive and Ring Positive $^ { + 8 }$ hours. Concentrations

# Diasorin

for both conditions are outlined in Table 4. One negative blood bottle was also grown for at least five days in an automated blood culture system until it was flagged as “negative” and was also tested on the LIAISON PLEX® BCN Assay.

A minimum of three Bottle/Ring Positive and three Bottle/Ring Positive $^ { + 8 }$ Hour bottles from each BCN organism were tested in triplicate and one negative blood bottle was tested in triplicate on the LIAISON PLEX® BCN Assay. The LIAISON PLEX $^ { \textregistered }$ BCN Assay results, presented in Table 4 and Table 5, demonstrated $100 \%$ positivity for target positive samples and $0 \%$ positivity when tested with negative blood.

Table 4 – Growth and Detection Summary – Positive Samples   

<table><tr><td colspan="8" rowspan="1">Positive Samples</td></tr><tr><td colspan="1" rowspan="2">Species Tested</td><td colspan="1" rowspan="2">Targets Tested</td><td colspan="3" rowspan="1">At Positivity</td><td colspan="3" rowspan="1">8 Hours After Positivity</td></tr><tr><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1">PositiveAgreement/Total (%Detected)</td><td colspan="1" rowspan="1">Per Bottle(CFU/mL)</td><td colspan="1" rowspan="1">Mean(CFU/mL)</td><td colspan="1" rowspan="1">PositiveAgreement/Total (%Detected)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumannii[OXA] IHMA128307</td><td colspan="1" rowspan="1">Acinetobacterbaumannii|Acinetobacter spp.OXA</td><td colspan="1" rowspan="1">2.13E+082.18E+081.86E+08</td><td colspan="1" rowspan="1">2.06E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.09E+081.06E+081.75E+08</td><td colspan="1" rowspan="1">1.63E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Citrobacterfreundii [VIM]IHMA 549813</td><td colspan="1" rowspan="1">Citrobacter spp. EnterobacteriaceaeMorganellaceaeVIM</td><td colspan="1" rowspan="1">9.63E+081.49E+091.39E+09</td><td colspan="1" rowspan="1">1.28E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.76E+092.16E+091.68E+09</td><td colspan="1" rowspan="1">1.87E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">EnterobactercloacaeATCC 35030</td><td colspan="1" rowspan="1">Enterobacter spp. Enterobacteriaceae /Morganellaceae</td><td colspan="1" rowspan="1">1.46E+091.61E+091.68E+09</td><td colspan="1" rowspan="1">1.58E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.68E+092.07E+092.36E+09</td><td colspan="1" rowspan="1">2.04E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli[MCR]NCTC 13846</td><td colspan="1" rowspan="1">EnterobacteriaceaeMorganellaceaeEscherichia coli |MCR</td><td colspan="1" rowspan="1">1.70E+091.96E+091.95E+09</td><td colspan="1" rowspan="1">1.87E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.45E+091.83E+091.66E+09</td><td colspan="1" rowspan="1">1.65E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca [CTX-M]IHMA 683079</td><td colspan="1" rowspan="1">CTX-M |Enterobacteriaceae /MorganellaceaeKlebsiella oxytoca</td><td colspan="1" rowspan="1">4.53E+081.48E+099.73E+08</td><td colspan="1" rowspan="1">9.69E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">5.3E+082.9E+082.04E+08</td><td colspan="1" rowspan="1">3.41E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae[KPC]IHMA 629630</td><td colspan="1" rowspan="1">EnterobacteriaceaeMorganellaceaeKPC | Klebsiellapneumoniae</td><td colspan="1" rowspan="1">1.33E+091.66E+091.69E+09</td><td colspan="1" rowspan="1">1.56E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.55E+091.61E+091.59E+09</td><td colspan="1" rowspan="1">1.58E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">KlebsiellavariicolaATCC BAA-830</td><td colspan="1" rowspan="1">EnterobacteriaceaeMorganellaceaeKlebsiella variicola</td><td colspan="1" rowspan="1">1.66E+091.82E+091.78E+09</td><td colspan="1" rowspan="1">1.75E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.00E+092.13E+092.05E+09</td><td colspan="1" rowspan="1">2.06E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella entent EnteritidisATCC 13076</td><td colspan="1" rowspan="1">EnterobacteriaceaeMorganellaceaeSalmonella spp.</td><td colspan="1" rowspan="1">4.90E+071.38E+085.80E+08</td><td colspan="1" rowspan="1">2.56E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.88E+092.06E+091.61E+09</td><td colspan="1" rowspan="1">1.85E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">Enterobacteriaceae /</td><td colspan="1" rowspan="1">1.55E+08</td><td colspan="1" rowspan="1">2.38E+08</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">1.82E+09</td><td colspan="1" rowspan="1">1.47E+09</td><td colspan="1" rowspan="1">12/121</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">marcescens[SME]NCTC 13920</td><td colspan="1" rowspan="1">MorganellaceaeSME / Serratiamarcescens</td><td colspan="1" rowspan="1">3.27E+082.32E+08</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">1.39E+091.19E+09</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="1">Morganellamorganii[NDM]IHMA 605873</td><td colspan="1" rowspan="1">Enterobacteriaceae /MorganellaceaeMorganellamorganii | NDM</td><td colspan="1" rowspan="1">1.77E+091.76E+091.52E+09</td><td colspan="1" rowspan="1">1.68E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.75E+092.65E+092.74E+09</td><td colspan="1" rowspan="1">2.71E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa[IMP]IHMA 576602</td><td colspan="1" rowspan="1">IMP | PseudomonasaeruginosaPseudomonas spp.</td><td colspan="1" rowspan="1">8.97E+087.30E+087.70E+08</td><td colspan="1" rowspan="1">7.99E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.33E+091.14E+091.30E+09</td><td colspan="1" rowspan="1">1.26E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophiliaATCC 13636</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">3.83E+085.23E+084.37E+08</td><td colspan="1" rowspan="1">4.48E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.23E+099.47E+088.27E+08</td><td colspan="1" rowspan="1">1.00E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">ProteusmirabilisATCC 12453</td><td colspan="1" rowspan="1">Enterobacteriaceae /MorganellaceaeProteus spp.</td><td colspan="1" rowspan="1">2.14E+092.35E+092.30E+09</td><td colspan="1" rowspan="1">2.26E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">2.28E+091.80E+092.28E+09</td><td colspan="1" rowspan="1">2.12E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">HaemophilusinfluenzaeATCC 9007</td><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">2.12E+092.32E+091.96E+09</td><td colspan="1" rowspan="1">2.13E+09</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.37E+092.42E+091.32E+09</td><td colspan="1" rowspan="1">1.70E+09</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">NeisseriameningitidisATCC 43744</td><td colspan="1" rowspan="1">Neisseriameningitidis</td><td colspan="1" rowspan="1">2.39E+082.36E+088.93E+07</td><td colspan="1" rowspan="1">1.88E+08</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">1.43E+084.10E+083.07E+08</td><td colspan="1" rowspan="1">2.87E+08</td><td colspan="1" rowspan="1">9/9(100%)</td></tr></table>

1 One OXA false positive was observed in initial testing. An additional set of replicates was tested resulting in 12 total replicates.

Table 5 – Growth and Detection Summary – Negative Samples   

<table><tr><td rowspan=1 colspan=5>Negative Samples</td></tr><tr><td rowspan=1 colspan=1>Species Tested</td><td rowspan=1 colspan=1>Targets Tested</td><td rowspan=1 colspan=1>Per Bottle(CFU/mL)</td><td rowspan=1 colspan=1>Mean(CFU/mL)</td><td rowspan=1 colspan=1>Targets Detected/Total (%Detected)</td></tr><tr><td rowspan=1 colspan=1>Negative Blood</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>0.00E+00</td><td rowspan=1 colspan=1>0.00E+00</td><td rowspan=1 colspan=1>0/3(0%)</td></tr></table>

# e. Analytical Reactivity (Inclusivity)

The analytical reactivity study was performed to evaluate the inclusivity of the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ BCN Assay through laboratory testing of multiple strains representing “on‐panel” reportable target bacteria or bacterial species. All on‐panel organisms $[ n = 2 4 6 ]$ were

# Diasorin

tested in triplicate. One hundred percent $( 1 0 0 \% )$ positivity was noted for all targets including expected resistance marker calls for 245 of the 246 on‐panel organisms. A summary of organism results is listed in Table 6 and a summary of resistance marker results is listed in Table 7. Of the 246 on‐panel organisms tested, 17 organisms generated additional target calls that align with known cross‐reactivity based on in silico analysis, or design features. Additionally, one on‐panel organism (S. proteamaculans) only generated the known cross‐reactivity target call, not the expected target call. The 18 cross‐reactive organisms are highlighted in Table 8.

Table 6: LIAISON PLEX® BCN Assay Analytical Reactivity (Inclusivity) Summary by Organism   

<table><tr><td colspan="1" rowspan="1">Reportable Target(Family)</td><td colspan="1" rowspan="1">ReportableTarget(Genus)</td><td colspan="1" rowspan="1">ReportableTarget(Species)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Numberofstrains</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="15">N/A</td><td colspan="1" rowspan="14">Acinetobacterspp.</td><td colspan="1" rowspan="14">N/A</td><td colspan="1" rowspan="1">Acinetobacter guillouiae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterradioresistens</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baylyi</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter bereziniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobactercalcoaceticus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterhaemolyticus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter indicus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter johnsonii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter junii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacternosocomialis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter pittii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter schindleri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterspp.</td><td colspan="1" rowspan="1">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="11">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="11">Citrobacterspp.</td><td colspan="1" rowspan="11">N/A</td><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter farmeri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter gillenii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter murliniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter rodentium</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter werkmanii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="6" rowspan="1">JTasom             LIAISON PLEX® Gram-Negative Blood Culture Assay Traditional 51O(k)</td></tr><tr><td colspan="1" rowspan="1">Reportable Target(Family)</td><td colspan="1" rowspan="1">ReportableTarget(Genus)</td><td colspan="1" rowspan="1">ReportableTarget(Species)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Numberofstrains</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="7">/ Morganellaceae</td><td colspan="1" rowspan="7">spp.</td><td colspan="1" rowspan="7">b</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacterbugandensis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobactercancerogenus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter ludwigii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="5">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="5">N/A</td><td colspan="1" rowspan="5">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Klebsiellavariicola</td><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Morganellamorganii</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Neisseriameningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="5">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="5">Proteus spp.</td><td colspan="1" rowspan="5">N/A</td><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Proteus myxofaciens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="8">N/A</td><td colspan="1" rowspan="8">Pseudomonasspp.</td><td colspan="1" rowspan="8">N/A</td><td colspan="1" rowspan="1">Pseudomonasalcaligenes</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonaschlororaphis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasfluorescens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasmendocina</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas monteilii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas mosselii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Reportable Target(Family)</td><td colspan="1" rowspan="1">ReportableTarget(Genus)</td><td colspan="1" rowspan="1">ReportableTarget(Species)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Numberofstrains</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="8">b</td><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="7">b</td><td colspan="1" rowspan="1">mucidolens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonasoryzihabitans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonaspseudoalcaligenes</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasresinovorans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas stutzeri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas veronii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="6">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="6">Salmonellaspp.</td><td colspan="1" rowspan="6">N/A</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericasubsp. arizonae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericasubsp. diarizonae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericasubsp. houtenae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericasubsp. indica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="16">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="16">N/A</td><td colspan="1" rowspan="16">N/A</td><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Cedecea lapagei</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Cedecea neteri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cronobacter muytjensii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enteric group 137</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia albertii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermanii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Kluyvera cryocrescens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Kluyvera georgiana</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Kluyvera intermedia</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Reportable Target(Family)</td><td colspan="1" rowspan="1">ReportableTarget(Genus)</td><td colspan="1" rowspan="1">ReportableTarget(Species)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Numberofstrains</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="23"></td><td colspan="1" rowspan="23"></td><td colspan="1" rowspan="23"></td><td colspan="1" rowspan="1">Lelliottia nimipressuralis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pluralibacter gergoviae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Providencia acalifaciens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Providencia heimbachae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Providencia rettigeri</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Rahnella aquatillis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Raoultella terrigena</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Xenorhabdus bovienii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Xenorhabdus poinarii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Yokenella regensburgei</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr><tr><td colspan="1" rowspan="1">Serratia proteamaculans</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0%^</td></tr><tr><td colspan="1" rowspan="1">Serratia rubidaea</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%^</td></tr></table>

^ See Table 8. Known Cross‐Reactive Organisms

Table 7: LIAISON PLEX BCN Assay Analytical Reactivity (Inclusivity) Resistance Marker Summary   

<table><tr><td colspan="1" rowspan="1">Reportable Target(Resistance Marker)</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1"># of strains</td><td colspan="1" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="9">CTX-M</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Kluyvera georgiana</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="4">IMP</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="5">KPC</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="4">MCR</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="4">NDM</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="5">OXA</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter radioresistens</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter indicus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">VIM</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100%</td></tr></table>

Table 8: LIAISON PLEX® BCN Assay Known Analytical Reactivity (Inclusivity) Cross Reactive Organisms   

<table><tr><td colspan="1" rowspan="1">Cross-ReactiveOrganism</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Expected Target</td><td colspan="1" rowspan="1">Additional TargetDETECTED</td><td colspan="1" rowspan="1">NumberDetected</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">Cedecea lapagei</td><td colspan="1" rowspan="1">ATCC 43028</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">Escherichiaalbertii</td><td colspan="1" rowspan="1">ATCC 46494</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">Kluyveraascorbata</td><td colspan="1" rowspan="1">ATCC 33433</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae|CTX-M</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leclerciaadecarboxylata</td><td colspan="1" rowspan="1">ATCC 23216</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Enterobacter spp.</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lelliottianimipressuralis</td><td colspan="1" rowspan="1">ATCC 9912</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Enterobacter spp.</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providenciarettigeri</td><td colspan="1" rowspan="1">ATCC 9250</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1">UTMB GN-403</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">Raoultellaornithinolytica</td><td colspan="1" rowspan="1">ATCC 31898</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Raoultellaplanticola</td><td colspan="1" rowspan="1">ATCC 33531</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Raoultellaterrigena</td><td colspan="1" rowspan="1">ATCC 33257</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Citrobacter spp.</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ATCC 33105</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratiafonticola</td><td colspan="1" rowspan="1">ATCC 29844</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">ATCC 14461</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">Serratialiquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratiaproteamaculans</td><td colspan="1" rowspan="1">ATCC 35475</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae#</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratiarubidaea</td><td colspan="1" rowspan="1">ATCC 27593</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Xenorhabdusbovienii</td><td colspan="1" rowspan="1">ATCC 35271</td><td colspan="1" rowspan="1">Enterobacteriaceae/ Morganellaceae</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">2/3</td></tr></table>

# Expected Target not detected.

# Predicted (in silico) Reactivity (Inclusivity) Results

For all targets, in silico inclusivity analysis was performed using sequences available in the GenBank and WGS (whole genome shotgun) databases from February to April 2024. Alignments of the signal fragment for each target were generated using MAFFT (version 7.490). For all targets, the inclusivity analysis involved assessing the percent homology of each oligo sequence to its binding region on each target sequence retrieved from the public databases. The predicted inclusivity based on sequence homology is the percentage of sequences with at least $90 \%$ oligo identity.  In determining oligo identity, the highest percent identity of each oligo in the same component was assessed and then the lowest percent identity of the two components (capture and mediator probes) is used to characterize the inclusivity of the sequence. In silico analysis results are summarized below in Table 9.

Table 9 – In silico Analysis Results   

<table><tr><td colspan="1" rowspan="1">Reportable Target</td><td colspan="1" rowspan="1">Inclusive Organism/Target</td><td colspan="1" rowspan="1">Total # Sequences inAlignment</td><td colspan="1" rowspan="1"># Sequences withPercent OligoIdentity ≥ 90%</td><td colspan="1" rowspan="1">PredictedInclusivityPercentage (%)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">1456</td><td colspan="1" rowspan="1">1450</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp. a</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">4451</td><td colspan="1" rowspan="1">4445</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="3">Citrobacter spp. b</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">440</td><td colspan="1" rowspan="1">440</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">425</td><td colspan="1" rowspan="1">422</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">other Citrobacter species</td><td colspan="1" rowspan="1">1065</td><td colspan="1" rowspan="1">1065</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="3">Enterobacter spp. c</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">266</td><td colspan="1" rowspan="1">266</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Klebsiella / Enterobacter aerogenes</td><td colspan="1" rowspan="1">1083</td><td colspan="1" rowspan="1">1083</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">other Enterobacter species</td><td colspan="1" rowspan="1">3795</td><td colspan="1" rowspan="1">3777</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="2">Enterobacteriaceae /Morganellaceae</td><td colspan="1" rowspan="1">Enterobacteriaceae</td><td colspan="1" rowspan="1">28524</td><td colspan="1" rowspan="1">28442</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Morganellaceae</td><td colspan="1" rowspan="1">3829</td><td colspan="1" rowspan="1">3715</td><td colspan="1" rowspan="1">97</td></tr><tr><td colspan="1" rowspan="5">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">5083</td><td colspan="1" rowspan="1">5067</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">711</td><td colspan="1" rowspan="1">711</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">277</td><td colspan="1" rowspan="1">277</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">246</td><td colspan="1" rowspan="1">246</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">1061</td><td colspan="1" rowspan="1">1061</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1">236</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca d</td><td colspan="1" rowspan="1">Klebsiella oxytoca species complex</td><td colspan="1" rowspan="1">2117</td><td colspan="1" rowspan="1">2105</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">6163</td><td colspan="1" rowspan="1">6131</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">728</td><td colspan="1" rowspan="1">728</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">179</td><td colspan="1" rowspan="1">179</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Proteus spp. e</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">484</td><td colspan="1" rowspan="1">481</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1341</td><td colspan="1" rowspan="1">1341</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas spp. f</td><td colspan="1" rowspan="1">Pseudomonas spp.</td><td colspan="1" rowspan="1">6480</td><td colspan="1" rowspan="1">6473</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="8">Salmonella spp.</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">420</td><td colspan="1" rowspan="1">420</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. diarizonae</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td><td colspan="1" rowspan="1">2164</td><td colspan="1" rowspan="1">2162</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. houtenae</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. salamae</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. VII</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">262</td><td colspan="1" rowspan="1">262</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">410</td><td colspan="1" rowspan="1">408</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">CTX-M 8</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">12187</td><td colspan="1" rowspan="1">12146</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">IMP h</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">1448</td><td colspan="1" rowspan="1">1441</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">4032</td><td colspan="1" rowspan="1">4031</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="3">MCR</td><td colspan="1" rowspan="1">MCR-1</td><td colspan="1" rowspan="1">1695</td><td colspan="1" rowspan="1">1695</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">MCR-2</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">MCR-3</td><td colspan="1" rowspan="1">211</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NDM </td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">5426</td><td colspan="1" rowspan="1">5420</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="4">OXA</td><td colspan="1" rowspan="1">OXA-23 family</td><td colspan="1" rowspan="1">1721</td><td colspan="1" rowspan="1">1718</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">OXA-24/40/143 family</td><td colspan="1" rowspan="1">403</td><td colspan="1" rowspan="1">403</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">OXA-48 family</td><td colspan="1" rowspan="1">1968</td><td colspan="1" rowspan="1">1962</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">OXA-58 family</td><td colspan="1" rowspan="1">425</td><td colspan="1" rowspan="1">425</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">SME m</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">189</td><td colspan="1" rowspan="1">188</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">VIM n</td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">1438</td><td colspan="1" rowspan="1">1438</td><td colspan="1" rowspan="1">100</td></tr></table>

a Includes sequences for 86 Acinetobacter species.

b Includes sequences for 17 Citrobacter species.

c Includes sequences for 22 Enterobacter species.

d Includes sequences for 6 members of the Klebsiella oxytoca species complex (KoSC).

e Includes sequences for 11 Proteus species.

f Includes sequences for 282 Pseudomonas species.

g Includes sequences for CTX‐M‐1 to CTX‐M‐269; not all types have sequences available.

h Includes sequences for IMP‐1 to IMP‐102; not all types have sequences available.

i Includes sequences for KPC‐1 to KPC‐208; not all types have sequences available.

Includes sequences for MCR‐1.1 to MCR1.37, MCR‐2.1 and MCR‐2.8, and MCR‐3.1 to MCR‐3.42; not all types have sequences available.

k Includes sequences for NDM‐1 to NDM‐61; not all types have sequences available.

l Includes sequences for OXA‐23 family, OXA‐24/40/143 family, OXA‐48 family and OXA‐58 family; not all types have sequences available.

Includes sequences for SME‐1 to SME‐5; not all types have sequences available.

n Includes sequences for VIM‐1 to VIM‐86; not all types have sequences available.

# f. Analytical Specificity (Exclusivity)

# Cross‐Reactivity

The analytical specificity study was performed to evaluate cross‐reactivity of the LIAISON PLEX® BCN Assay through testing “off‐panel” organisms, including both those phylogenetically related to the on‐panel organisms and organisms likely to be present in typical blood culture samples. Out of the 113 off‐panel species tested in triplicate, 17 species generated positive results. Summary results for all 113 species are listed in Table 10a. Of those 17 species that generated positive results, 11 species were cross‐reactive and those cross‐reactive calls are listed in Table 10b.

Table 10a: LIAISON PLEX® BCN Assay Analytical Specificity (Cross‐Reactivity) Summary   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Positivity</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="4" rowspan="1">Gram Positive</td></tr><tr><td colspan="1" rowspan="1">Abitrophia defectiva</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Leuconostoc carnosum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Leuconostoc mesenteroids</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium bernardiae</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium haemolyticum</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Listeria ivanovii</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformins</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Bacillus thuringiensis</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Pediococcus acidilactici</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium pseudodiptheriticum</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Planococcus citreus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Planococcus kocurii</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium avidum</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Rothia dentocariosa</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium freudenreichii</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Rothia (Stomatococcus) mucilaginosa</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavis, VRE, vanC</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterocococcus dispar</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum, vanC</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus mundtii</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Kytococcus sedentarius, Met R</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus equinus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium frankenforstense</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Gram Negative</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Brevundimonas diminuta</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Herbaspirillum huttiense</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Capnocytophaga ochracea</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Cardiobacterium hominis</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Comamonas testosteroni</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Delftia acidovorans</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Parabacteroides distasonis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Elizabethkingia meningoseptica</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Prevotella bivia</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Klebsiella quasivariicola</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">Providencia rustigiani</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">33%</td><td colspan="1" rowspan="1">Providencia vermicola</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter kobei</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas paraeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus aegyptius</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Serratia entomophila</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella grimontii</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Serratia nevei</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella huaxiensis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella michiganensis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Serratia quinivorans</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasipneumoniae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Serratia ureilytica</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Lelliottia amnigena</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">canmaGllowb</td><td colspan="1" rowspan="1"></td></tr></table>

LIAISON PLEX® Gram‐Negative Blood Culture Assay Traditional 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Positivity</td></tr><tr><td rowspan=1 colspan=4>Yeast/Fungi</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Kluyveromyces lactis</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Schizosaccharomyces pombe</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 10b: LIAISON PLEX® BCN Assay Cross‐Reactive Organisms Summary   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=2>Results</td></tr><tr><td rowspan=1 colspan=4>Not Reported to Cause Bloodstream Infections</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Inclusive</td><td rowspan=1 colspan=1>Cross-reacting</td></tr><tr><td rowspan=1 colspan=1>Lelliottia amnigena</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Enterobacter spp.</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas paraeruginosa</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Pseudomonas spp.</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Serratia nevei</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Serratia entomophila</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Serratia ureilytica</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Klebsiella quasivariicola</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Rare/Not Primarily Associated with Bloodstream Infections</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Inclusive</td><td rowspan=1 colspan=1>Cross-reacting</td></tr><tr><td rowspan=1 colspan=1>Haemophilus aegyptius</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Haemophilus influenzae</td></tr><tr><td rowspan=1 colspan=1>Klebsiella michiganensis</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td></tr><tr><td rowspan=1 colspan=4>Rare/Opportunistic with Some Documented Bloodstream Infection Cases</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Inclusive</td><td rowspan=1 colspan=1>Cross-reacting</td></tr><tr><td rowspan=1 colspan=1>Klebsiella grimontii</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td></tr><tr><td rowspan=1 colspan=1>Klebsiella quasipneumoniae</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Enterobacteriaceae / Morganellaceae</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Haemophilusparahaemolyticus</td><td rowspan=1 colspan=1>33%</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Enterobacteriaceae /Morganellaceae</td></tr></table>

# In Silico Cross‐Reactivity

In silico exclusivity assessment of the oligo sequences incorporated in the assay designs was performed against on‐panel and off‐panel organisms listed in Table 11. Based on the analysis of sequences available in the GenBank nucleotide database as of June 7, 2024, the following potential cross‐reactivity is predicted:

Acinetobacter baumannii oligo designs are predicted to detect some strains of Acinetobacter pittii and Klebsiella pneumoniae.   
Some strains of Escherichia species (E. albertii, E. coli, E. fergusonii, E. marmotae), Raoultella terrigena and Shigella species (S. boydii, S. dysenteriae, S. flexneri, S. sonnei) are predicted to produce false positive Citrobacter spp. results.   
Enterobacter spp. oligo designs are predicted to detect some strains of Klebsiella huaxiensis, Leclercia adecarboxylata and Lelliottia species (Lelliottia amnigena, Lelliottia nimipressuralis). Some strains of Aeromonas hydrophila, Aggregatibacter aphrophilus, Corynebacterium frankenforstense, Haemophilus parahaemolyticus, Haemophilus parainfluenzae and

# Diasorin

Pseudomonas wenzhouensis are predicted to produce false positive Enterobacteriaceae / Morganellaceae results.

Escherichia coli oligo designs are predicted to detect some strains of Escherichia albertii and Escherichia marmotae.

Haemophilus influenzae oligo designs are predicted to detect some strains of Haemophilus aegyptius.

Some strains of Cedecea species (C. davisae, C. lapagei, C. neteri), Enterobacter species (E.   
kobei, E. mori), Escherichia coli, Klebsiella species (K. aerogenes, K. electrica, K. grimontii, K.   
huaxiensis, K. michiganensis, K. pasteurii, K. pneumoniae, K. variicola), Kluyvera species (K.   
ascorbate, K. cryocrescens), Raoultella species (R. ornithinolytica, R. planticola) and unspecified Superficieibacter species are predicted to produce false positive Klebsiella oxytoca results.

Klebsiella pneumoniae oligo designs are predicted to detect some strains of Enterobacteriaceae bacterium and Klebsiella species (K. africana, K. grimontii, K. quasipneumoniae, K. quasivariicola, K. variicola).

Some strains of Cronobacter muytjensii, Klebsiella pneumoniae and Klebsiella quasipneumoniae are predicted to produce false positive Klebsiella variicola results.

Some strains of Arsenophonus species (A. apicola, A. endosymbiont, A. nasoniae), Haemophilus parahaemolyticus, Leclercia adecarboxylata, Morganella psychrotolerans, Providencia species (P. hangzhouensis, P. heimbachae, P. huaxiensis, P. manganoxydans, P. rettgeri, P. rustigianii, P. stuartii, P. vermicola), Serratia species (S. grimesii, S. proteamaculans, S. quinivorans) and Xenorhabdus species (X. bovienii, X. budapestensis, X. griffiniae, X. hominickii, X. poinarii) are predicted to produce false positive Proteus spp. results.

Pseudomonas aeruginosa oligo designs are predicted to detect some strains of Pseudomonas paraeruginosa and Pseudomonas fluorescens.

Salmonella spp. oligo designs are predicted to detect one strain of Enterobacteriaceae bacterium.

Some strains of Serratia species (S. bockelmannii, S. entomophila, S. ficaria, S. fonticola, S. grimesii, S. inhibens, S. liquefaciens, S. nematodiphila, S. nevei, S. odorifera, S. proteamaculans, S. quinivorans, S. rhizosphaerae, S. rubidaea, S. surfactantfaciens, S. symbiotica, S. ureilytica) are predicted to produce false positive Serratia marcescens results.

Stenotrophomonas maltophilia oligo designs are predicted to detect some strains of Enterobacter cloacae and other Stenotrophomonas species (S. acidaminiphila, S. nitritireducens, S. rhizophila).   
 Some strains of Serratia liquefaciens are predicted to produce false positive Serratia marcescens and CTX‐M results.   
 Some strains of Serratia liquefaciens and Serratia nevei are predicted to produce false positive Serratia marcescens and IMP results. Some strains of Serratia nevei are predicted to produce false positive Serratia marcescens and OXA results.

Table 11.  Potential Cross‐Reactive Organisms assessed in the In Silico Exclusivity Analysis   

<table><tr><td rowspan="2">On-Panel Organisms</td><td colspan="4">Off-Panel Organisms</td></tr><tr><td>Gram-Positive Bacteria</td><td>Gram-Negative Bacteria</td><td>Resistance Markers</td><td>Yeasts / Viruses / Parasites</td></tr><tr><td>Acinetobacter baumannii</td><td>Actinomyces israelii</td><td>Actinobacillus hominis</td><td>AmpC</td><td>Aspergillus flavus</td></tr><tr><td>Acinetobacter spp.</td><td>Actinomyces naeslundii</td><td>Actinobacillus ureae</td><td>CMY</td><td>Aspergillus</td></tr><tr><td>Citrobacter spp.</td><td>Actinomyces</td><td></td><td></td><td>fumigatus</td></tr><tr><td rowspan="2"></td><td>odontolyticus</td><td>Aeromonas caviae</td><td>mecA</td><td>Aspergillus niger</td></tr><tr><td>Aerococcus</td><td>Aeromonas</td><td></td><td>Aspergillus terreus</td></tr><tr><td>Enterobacter spp.</td><td>sanguinicola</td><td>hydrophila</td><td>mecC</td><td></td></tr><tr><td>Enterobacteriaceae</td><td>Aerococcus urinae</td><td>Aeromonas sobria</td><td>0mpK36</td><td>Blastomyces dermatitidis</td></tr><tr><td>Escherichia coli /</td><td></td><td>Aggregatibacter</td><td></td><td></td></tr><tr><td>Shigella spp.</td><td>Aerococcus viridans</td><td>actinomycetemcomit ans</td><td>RAHN</td><td>Candida albicans</td></tr><tr><td>Haemophilus</td><td>Arcanobacterium</td><td>Aggregatibacter</td><td>SHV</td><td>Candida auris</td></tr><tr><td>influenzae</td><td>bernardiae</td><td>aphrophilus</td><td></td><td>Candida</td></tr><tr><td>Klebsiella oxytoca</td><td>Arcanobacterium haemolyticum</td><td>Bacteroides caccae</td><td>SPM</td><td>dubliniensis</td></tr><tr><td>Klebsiella</td><td>Arthrobacter</td><td></td><td>TEM</td><td>Candida</td></tr><tr><td>pneumoniae</td><td>psychrolactophilus</td><td>Bacteroides fragilis</td><td></td><td>duobushaemulonii</td></tr><tr><td>Klebsiella variicola</td><td>Bacillus spp.</td><td>Bacteroides ovatus</td><td>vanA</td><td>Candida famata</td></tr><tr><td>Morganella morganii</td><td>Brochothrix thermosphacta</td><td>Bacteroides thetaiotaomicron</td><td>vanB</td><td>Candida glabrata</td></tr><tr><td>Morganellaceae</td><td>Carnobacterium</td><td>Bacteroides uniformis</td><td>vanC</td><td>Candida</td></tr><tr><td>Neisseria</td><td>divergens Carnobacterium</td><td></td><td></td><td>guilliermondii Candida</td></tr><tr><td>meningitidis</td><td>maltaromaticum</td><td>Bacteroides vulgatus</td><td>vanD</td><td>haemulonii</td></tr><tr><td>Proteus spp.</td><td>Cellulomonas turbata</td><td>Bacteroides</td><td>vanM</td><td>Candida</td></tr><tr><td>Pseudomonas</td><td>Cellulosimicrobium</td><td>xylanisolvens Bordetella</td><td></td><td>inconspicua</td></tr><tr><td>aeruginosa</td><td>cellulans</td><td>bronchiseptica</td><td></td><td>Candida kefyr</td></tr><tr><td>Pseudomonas spp.</td><td>Clostridioides difficile</td><td>Bordetella parapertussis</td><td></td><td>Candida krusei</td></tr><tr><td>Salmonella spp.</td><td>Clostridium bifermentans</td><td>Bordetella pertussis</td><td></td><td>Candida lipolytica</td></tr><tr><td>Serratia marcescens</td><td>Clostridium clostridioforme</td><td>Brevundimonas</td><td></td><td>Candida lusitaniae</td></tr><tr><td>Stenotrophomonas</td><td>Clostridium perfringens</td><td>diminuta Brevundimonas</td><td></td><td>Candida</td></tr><tr><td>maltophilia</td><td>Clostridium ramosum</td><td>vesicularis</td><td></td><td>metapsilosis</td></tr><tr><td>CTX-M</td><td>(Thomasclavelia</td><td>Burkholderia cepacia</td><td></td><td>Candida multis- gemmis</td></tr><tr><td>IMP</td><td>ramosa) Clostridium septicum</td><td>Burkholderia mallei</td><td></td><td>Candida nivariensis</td></tr><tr><td>KPC</td><td>Clostridium tertium</td><td>Burkholderia</td><td></td><td>Candida</td></tr><tr><td>MCR (MCR-1, MCR-</td><td></td><td>multivorans Burkholderia</td><td></td><td>norvegensis</td></tr><tr><td>2, MCR-3)</td><td>Clostridium tetani</td><td>pseudomallei</td><td></td><td>Candida orthopsilosis</td></tr><tr><td>NDM</td><td>Corynebacterium spp.</td><td>Campylobacter</td><td></td><td>Candida</td></tr><tr><td></td><td></td><td>hominis Capnocytophaga</td><td></td><td>parapsilosis</td></tr><tr><td>OXA</td><td>Cutibacterium acnes</td><td>ochracea</td><td></td><td>Candida sojae</td></tr><tr><td>SME</td><td>Cutibacterium avidum</td><td>Cardiobacterium hominis</td><td></td><td>Candida tropicalis</td></tr><tr><td>VIM</td><td>Cutibacterium</td><td>Chlamydia</td><td></td><td>Candida</td></tr><tr><td></td><td>granulosum</td><td>trachomatis</td><td></td><td>viswanathii</td></tr><tr><td></td><td></td><td>Chlamydophila</td><td></td><td>Coccidioides</td></tr><tr><td></td><td>Enterococcus avium</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td>pneumoniae</td><td></td><td>immitis</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Enterococcus</td><td>Chromobacterium</td><td></td><td></td></tr><tr><td></td><td>casseliflavus</td><td>violaceum</td><td></td><td>Coccidioides posadasii</td></tr><tr><td rowspan="20"></td><td>Enterococcus cecorum</td><td>Comamonas</td><td></td><td>Cryptococcus</td></tr><tr><td>Enterococcus dispar</td><td>testosteroni Delftia acidovorans</td><td></td><td>amylolentus Cryptococcus gattii</td></tr><tr><td></td><td>Eikenella corrodens</td><td></td><td>Cryptococcus</td></tr><tr><td>Enterococcus durans</td><td>Elizabethkingia</td><td></td><td>neoformans Cryptococcus</td></tr><tr><td>Enterococcus faecalis</td><td>meningoseptica</td><td></td><td>uniguttulatus</td></tr><tr><td>Enterococcus faecium</td><td>Fusobacterium necrophorum</td><td></td><td>Cutaneotrichospor on curvatum</td></tr><tr><td>Enterococcus</td><td>Fusobacterium</td><td></td><td>Cyberlindnera</td></tr><tr><td>flavescens</td><td>nucleatum</td><td></td><td>fabianii</td></tr><tr><td>Enterococcus</td><td></td><td></td><td>Geotrichum</td></tr><tr><td>gallinarum</td><td>Haemophilus aegyptius</td><td></td><td>capitatum (Magnusiomyces</td></tr><tr><td></td><td></td><td></td><td>capitatus</td></tr><tr><td>Enterococcus hirae</td><td>Haemophilus ducreyi</td><td></td><td>Histoplasma</td></tr><tr><td>Enterococcus mundtii</td><td>Haemophilus</td><td></td><td>capsulatum Kluyveromyces</td></tr><tr><td></td><td>haemolyticus</td><td></td><td>lactis</td></tr><tr><td>Enterococcus raffinosus</td><td>Haemophilus</td><td></td><td></td></tr><tr><td></td><td>parahaemolyticus</td><td></td><td>Kodamaea ohmeri</td></tr><tr><td>Erysipelothrix</td><td>Haemophilus</td><td></td><td>Lodderomyces</td></tr><tr><td>rhusiopathiae</td><td>parainfluenzae</td><td></td><td>elongisporus</td></tr><tr><td>Finegoldia magna</td><td>Haemophilus</td><td></td><td>Magnusiomyces</td></tr><tr><td></td><td>parasuis</td><td></td><td>capitatus Millerozyma</td></tr><tr><td>Gemella haemolysans</td><td>Haemophilus quentini</td><td></td><td>farinosa</td></tr><tr><td>Gemella morbillorum</td><td>Haemophilus sputorum</td><td></td><td>Naganishia albida</td></tr><tr><td>Granulicatella adiacens</td><td>Herbaspirillum</td><td></td><td>Papiliotrema</td></tr><tr><td></td><td>huttiense</td><td></td><td>laurentii</td></tr><tr><td>Granulicatella elegans</td><td>Kingella denitrificans</td><td></td><td>Penicillium</td></tr><tr><td>Kocuria kristinae</td><td></td><td></td><td>chrysogenum Rhodotorula</td></tr><tr><td></td><td>Kingella kingae</td><td></td><td>mucilaginosa</td></tr><tr><td>Kocuria rhizophila</td><td>Kingella negevensis</td><td></td><td>Saccharomyces</td></tr><tr><td></td><td></td><td></td><td>cerevisiae Schizosaccharomyc</td></tr><tr><td>Kytococcus sedentarius</td><td>Kingella oralis</td><td></td><td>es pombe</td></tr><tr><td>Lactobacillus</td><td>Legionella</td><td></td><td>Talaromyces</td></tr><tr><td>acidophilus</td><td>pneumophila</td><td></td><td>marneffei</td></tr><tr><td>Lactobacillus crispatus</td><td>Leptospira interrogans</td><td></td><td>Trichosporon asahii</td></tr><tr><td>Lactobacillus</td><td></td><td></td><td>Wickerhamomyces</td></tr><tr><td>rhamnosus</td><td>Moraxella catarrhalis</td><td></td><td>anomalus</td></tr><tr><td>Lactococcus garvieae</td><td>Moraxella osloensis</td><td></td><td>BK Virus</td></tr><tr><td>Lactococcus lactis</td><td>Mycobacterium tuberculosis</td><td></td><td>Chikungunya Virus</td></tr><tr><td rowspan="2">On-Panel Organisms</td><td colspan="3">Off-Panel Organisms</td></tr><tr><td>Gram-Positive Bacteria Neisseria</td><td>Gram-Negative Resistance Bacteria Markers</td><td>Yeasts / Viruses / Parasites</td></tr><tr><td></td><td>Leuconostoc carnosum</td><td>gonorrhoeae</td><td>Cytomegalovirus</td></tr><tr><td>Leuconostoc citreum</td><td>Neisseria lactamica</td><td></td><td>Dengue Virus</td></tr><tr><td>Leuconostoc mesenteroides</td><td>Neisseria mucosa</td><td></td><td>Enterovirus</td></tr><tr><td>Listeria spp.</td><td>Neisseria sicca</td><td></td><td>Epstein Barr Virus</td></tr><tr><td>Macrococcus</td><td>Parabacteroides</td><td></td><td>Hepatitis A virus</td></tr><tr><td>caseolyticus</td><td>distasonis Parabacteroides</td><td></td><td></td></tr><tr><td>Micrococcus luteus</td><td>merdae</td><td></td><td>Hepatitis B virus</td></tr><tr><td>Mycobacterium avium complex (MAC)</td><td>Pasteurella</td><td></td><td>Hepatitis C virus</td></tr><tr><td>Mycobacterium</td><td>aerogenes</td><td></td><td>Human</td></tr><tr><td>fortuitum</td><td>Pasteurella canis</td><td></td><td>alphaherpesvirus 1</td></tr><tr><td>Mycobacterium</td><td>Pasteurella multocida</td><td></td><td>Human</td></tr><tr><td>mucogenicum</td><td></td><td></td><td>alphaherpesvirus 2</td></tr><tr><td>Mycoplasma hominis</td><td>Pasteurella stomatis</td><td></td><td>Human</td></tr><tr><td>Mycoplasma</td><td>Prevotella bivia</td><td></td><td>betaherpesvirus 6 Human</td></tr><tr><td>pneumoniae</td><td></td><td></td><td>betaherpesvirus 7 Human</td></tr><tr><td>Nocardia farcinica</td><td>Prevotella buccae</td><td></td><td>Immunodeficiency</td></tr><tr><td>Parvimonas micra</td><td>Prevotella denticola</td><td></td><td>Virus JC Virus</td></tr><tr><td>Pediococcus acidilactici</td><td>Prevotella melaninogenica</td><td></td><td>Measles Virus</td></tr><tr><td>Pediococcus pentosaceus</td><td>Prevotella oralis</td><td></td><td></td></tr><tr><td>Peptostreptococcus</td><td>Psychrobacter</td><td></td><td>Mumps Virus</td></tr><tr><td>anaerobius</td><td>cryohalolentis</td><td></td><td>Parvovirus B19</td></tr><tr><td>Planococcus citreus</td><td>Psychrobacter</td><td></td><td>Rubella Virus</td></tr><tr><td>Planococcus kocurii</td><td>immobilis Ralstonia</td><td></td><td></td></tr><tr><td></td><td>mannitolilytica</td><td></td><td>Varicella Zoster Virus</td></tr><tr><td>Propionibacterium freudenreichii</td><td>Ralstonia pickettii</td><td></td><td>West Nile Virus</td></tr><tr><td>Propionibacterium propionicum (Arachnia</td><td></td><td></td><td></td></tr><tr><td>propionica)</td><td>Serratia spp.</td><td></td><td>Zika Virus</td></tr><tr><td>Rhodococcus equi</td><td>Stenotrophomonas acidaminiphila</td><td></td><td>Plasmodium</td></tr><tr><td>Rothia dentocariosa</td><td>Stenotrophomonas</td><td></td><td>falciparum</td></tr><tr><td></td><td>nitritireducens</td><td></td><td>Trypanosoma cruzi</td></tr><tr><td>Rothia mucilaginosa</td><td>Stenotrophomonas</td><td></td><td></td></tr><tr><td></td><td>rhizophila</td><td></td><td></td></tr><tr><td>Sarcina ventriculi</td><td>Treponema pallidum</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>Solibacillus silvestris</td><td>Veillonella parvula</td><td></td><td></td></tr><tr><td>Staphylococcus spp.</td><td>Vibrio alginolyticus</td><td></td><td></td></tr></table>

<table><tr><td rowspan="2">On-Panel Organisms</td><td colspan="4">Off-Panel Organisms</td></tr><tr><td>Gram-Positive Bacteria</td><td>Gram-Negative Bacteria</td><td>Resistance Markers</td><td>Yeasts / Viruses / Parasites</td></tr><tr><td rowspan="7"></td><td>Streptococcus spp.</td><td>Vibrio parahaemolyticus</td><td></td><td></td></tr><tr><td>Ureaplasma parvum</td><td>Vibrio vulnificus</td><td></td><td></td></tr><tr><td>Ureaplasma</td><td></td><td></td><td></td></tr><tr><td>urealyticum</td><td></td><td></td><td></td></tr><tr><td>Vagococcus fluvialis</td><td></td><td></td><td></td></tr><tr><td>Weissella</td><td></td><td></td><td></td></tr><tr><td>paramesenteroides</td><td></td><td></td><td></td></tr></table>

# g. Interference

Competitive Inhibition / Co‐Infection and Microbial Interference

The competitive inhibition and microbial interference study was executed to evaluate the performance of the LIAISON PLEX® BCN Assay detection of on‐panel organisms in the presence of highly concentrated, potentially interfering microbes (off‐panel) and potential co‐infections with highly concentrated on‐panel organisms.

To evaluate competitive inhibition, a set of ten representative on‐panel strains were tested to assess dual target detection when combined in a pair‐wise fashion with the same on‐panel strains, as listed in Table 12. The ten on‐panel pair‐wise strain mixtures were each tested in triplicate as a low concentration (ring positive) – high concentration (ring positive $^ { + 8 }$ hours) combination. The on‐panel organisms were chosen to be representative of clinically relevant poly‐microbial infections in blood stream pathogens.

To assess microbial interference, seven off‐panel microbes were combined in pairs with the same ten representative on‐panel strains for assay evaluation, as listed in Table 13. These specific organisms were chosen as they are potentially interfering micro‐organisms that are commonly found in sites of blood stream infections that may be present in positive blood culture samples but are not expected to be detected by the LIAISON PLEX® BCN Assay. Off‐panel microbe pair testing with on‐panel strains was performed in triplicate in high‐low concentration combinations, respectively.

As can be observed in Tables 12 and 13, $100 \%$ target detection was achieved for all ten on‐ panel organisms, both in the presence of on‐panel strains and in the presence of off‐panel microbes.

Table 12: LIAISON PLEX® BCN Assay Competitive Inhibition Summary   

<table><tr><td colspan="1" rowspan="1">On-Panel HighConcentration Target</td><td colspan="1" rowspan="1">Positivity</td><td colspan="1" rowspan="1">On-Panel LowConcentration Target</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="5">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Escherichia coli</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="9">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="8">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td></tr></table>

# Diasorin

<table><tr><td rowspan=1 colspan=1>On-Panel HighConcentration Target</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>On-Panel LowConcentration Target</td><td rowspan=1 colspan=1>Positivity</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 13: LIAISON PLEX® BCN Assay Microbial Interference Summary   

<table><tr><td colspan="1" rowspan="1">On-Panel LowConcentration Target</td><td colspan="1" rowspan="1">Positivity</td><td colspan="1" rowspan="1">Off-Panel HighConcentration Target</td><td colspan="1" rowspan="1">Positivity</td></tr><tr><td colspan="1" rowspan="7">Acinetobacter baumannii</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Citrobacter freundii</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Enterobacter cloacae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Escherichia coli</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="5">Haemophilus influenzae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Klebsiella oxytoca</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Neisseria meningitidis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Proteus mirabilis</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="7">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Cutibacterium(Propionibacterium) acnes</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">0%</td></tr></table>

# Diasorin

# Interfering Substances

The interfering substances study was performed to evaluate the performance of the LIAISON PLEX® BCN Assay in the presence of non‐microbial (endogenous and exogenous) interfering substances which may be present in blood culture specimens. A set of four representative “on‐panel” organisms were tested to assess effectiveness of target detection in the presence of six interfering substances. Each interfering substance was tested across five replicates. In addition, a negative control (five replicates) was tested alongside the positive specimens to assess impact of the same interfering agents in specimens containing no target; additionally, a positive control (specimen without interfering substances) for all four targets was also tested to assess for detection capabilities.

All four targets demonstrated $100 \%$ positivity in the presence of all six interfering substances, and $0 \%$ target detection was observed in the negative control sample in the presence of the same interfering substances (Table 14). Additionally, $100 \%$ target detection was observed for the positive control for each target.

Table 14: LIAISON PLEX® BCN Assay Interfering Substances Summary   

<table><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Interfering SubstanceConcentration</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Positivity</td></tr><tr><td rowspan=4 colspan=1>N/A</td><td rowspan=4 colspan=1>N/A</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=5 colspan=1>Unconjugated Bilirubin</td><td rowspan=5 colspan=1>20 mg/dL</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=5 colspan=1>Conjugated Bilirubin</td><td rowspan=5 colspan=1>20 mg/dL</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=5 colspan=1>Hemoglobin</td><td rowspan=5 colspan=1>14 g/L</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=5 colspan=1>Intralipid</td><td rowspan=5 colspan=1>3000 mg/dL</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0%</td></tr></table>

# Diasorin

LIAISON PLEX® Gram‐Negative Blood Culture Assay Traditional 510(k)   

<table><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Interfering SubstanceConcentration</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Positivity</td></tr><tr><td rowspan=5 colspan=1>Y-globulin</td><td rowspan=5 colspan=1>6 g/dL</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=5 colspan=1>Sodiumpolyanetholsulfonate(SPS)</td><td rowspan=5 colspan=1>0.25% w/v</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0%</td></tr></table>

# Carry‐Over/Cross Contamination

This study was performed to evaluate the risk of carry‐over and cross contamination occurring during normal use of the device when highly concentrated positive specimens are processed alongside negative specimens. Two operators tested 30 high concentration positive samples consisting of Morganella morganii at a final concentration of $2 . 7 4 \mathsf { E } { + } 0 9$ $\mathsf { C F U / m L }$ and 30 negative samples consisting of Negative Blood Matrix. This concentration of Morganella morganii was selected as the representative analyte for the high positive control since it was the highest concentration material from the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ Growth and Detection study Ring Positive $^ { + 8 }$ hours growths, thus representing the most challenging scenario for potential cross contamination. Testing was performed on one LIAISON PLEX® instrument containing 6 blades/modules over the course of four days. The samples were loaded into cartridges alternating between positive and negative samples (checkerboard fashion), six specimens at a time using sample prep trays. The results, presented in Table 15, demonstrate $100 \%$ agreement between expected and observed results, indicating that no cross contamination occurred within runs and no carry‐over was observed across runs.

Table 15. Overall Results   

<table><tr><td rowspan=1 colspan=1>Overall Percent Agreement between Expected and Observed Results</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii Positivity in High Positive Control Replicates</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii Positivity in Negative Control Replicates</td><td rowspan=1 colspan=1>0%</td></tr></table>

# h. Assay Cut‐off

The specific assay parameters for the LIAISON PLEX® BCN Assay are considered confidential and proprietary.

# Diasorin

2. Comparison Studies:

a. Method comparison with predicate device:

Refer to Section 3 Clinical Performance.

b. Matrix Comparison: Testing of Blood Culture Bottle Types / Matrix Equivalency

The performance of LIAISON PLEX® Gram‐Negative Blood Culture (BCN) Assay was evaluated across 12 different types of commercially available blood culture media bottles using representative on‐panel BCN targets with resistance markers, off‐panel (gram‐ positive) LIAISON PLEX® BCN Assay targets, and negative blood matrix (NBM). The results, presented in Table 16, demonstrate that all 12 bottle types are compatible with the LIAISON $\mathsf { P l E X } ^ { \mathbb { \exp } }$ Gram‐Negative Blood Culture (BCN) Assay. A total of 650 independent blood culture bottles were evaluated, which included 506 gram‐negative bottle cultures, 120 gram‐positive bottle cultures, and 24 negative blood matrix bottles; see Table 16, final row. The effectiveness of bioMerieux BACT/ALERT® FA Plus blood culture bottle – was established through blood culture growth required for all other analytical studies, such as growth and detection and analytical reactivity/inclusivity verification testing, and was not included in testing performed for the media equivalency study.

Table 16: LIAISON PLEX® Gram‐Negative Blood Culture (BCN) Assay Media Equivalency/ Universal Blood Culture Bottle Result Summary   

<table><tr><td colspan="1" rowspan="2">ManufacturerSystem</td><td colspan="1" rowspan="2">Blood CultureBottleManufacturer</td><td colspan="1" rowspan="2">Blood CultureBottle Type</td><td colspan="2" rowspan="1">Number of InoculatedBottles</td><td colspan="1" rowspan="2">NegativeBloodMatrixb(NegativeDetection)</td></tr><tr><td colspan="1" rowspan="1">Gram-NegativeBacteria(PositiveDetection)</td><td colspan="1" rowspan="1">Gram-positiveBacteria(NegativeDetection)</td></tr><tr><td colspan="1" rowspan="4">BiomerieuxBACT/ALERT® 3DSystem</td><td colspan="1" rowspan="4">BiomerieuxBACT/ALERT</td><td colspan="1" rowspan="1">BACT/ALERT® SA(Aerobic)</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT SN(Anaerobic)</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT FNPlus (Anaerobic)</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT® PFPlus (Pediatric)</td><td colspan="1" rowspan="1">63/63</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="6">N/Aa</td><td colspan="1" rowspan="6">Becton DickinsonaBACTEC™M</td><td colspan="1" rowspan="1">BACTEC™Standard</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACTEC™ PluS(Aerobic)</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACTEC™Standard(Anaerobic)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACTEC™ PluS(Anaerobic)</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACTEC™M PedsPlus (Pediatric)</td><td colspan="1" rowspan="1">63/63</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">BACTECT™M Lytic(Anaerobic)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">N/Aa</td><td colspan="1" rowspan="1">Thermo ScientificaVersaTREK™</td><td colspan="1" rowspan="1">REDOX™ 1 EZDraw™ (Aerobic)</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">REDOX™ 2 EZDraw™(Anaerobic)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="3" rowspan="2">Number of Expected Calls/Total Number ofIndependent Blood Bottles</td><td colspan="1" rowspan="1">506/506 (100%)</td><td colspan="1" rowspan="1">120/120 (100%)</td><td colspan="1" rowspan="1">24/24 (100%)</td></tr><tr><td colspan="3" rowspan="1">650/650 (100%)</td></tr></table>

a For gram‐negative blood culture preparation, BACTEC™ and VersaTREK™ systems were not available and corresponding media bottles were placed in a standard laboratory incubator with a shaker for growth. For gram‐positive blood culture growth, only VersaTREK™ bottles needed to be placed in a standard laboratory incubator with a shaker in lieu of VersaTREK™ system; BACT/ALERT ® and BACTEC™ blood bottles were grown using corresponding automated blood culture systems up to bottle ring positivity. b Negative Blood Matrix (NBM) was tested using two independent blood inoculations per bottle type and each NBM sample was tested in replicates of three (total six replicates per bottle type) per study design; 72 total runs from 24 total bottle inoculations.

The average concentrations of each organism grown in 12 different types of media bottles are shown in Table 17; representing the approximate concentrations obtained from a mono‐bacterial blood culture used in this testing.

Table 17: Concentration of Blood Cultures in Different Media Bottles Used in Testing   

<table><tr><td colspan="1" rowspan="1">Gram-Negative (On-panel) Organism*</td><td colspan="1" rowspan="1">Expected Result</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">AveragePositive BloodCultureConcentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Acinetobacter spp., Acinetobacterbaumanii, OXA</td><td colspan="1" rowspan="1">IHMA 128307</td><td colspan="1" rowspan="1">3.03E+08</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">ATCC 15309</td><td colspan="1" rowspan="1">2.05E+08</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Citrobacter spp., VIM</td><td colspan="1" rowspan="1">IHMA 549813</td><td colspan="1" rowspan="1">1.03E+09</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Citrobacter spp.</td><td colspan="1" rowspan="1">ATCC 25405</td><td colspan="1" rowspan="1">9.87E+09</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Enterobacter spp.</td><td colspan="1" rowspan="1">ATCC 35030</td><td colspan="1" rowspan="1">1.16E+09</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Enterobacter spp.</td><td colspan="1" rowspan="1">ATCC 35029</td><td colspan="1" rowspan="1">9.50E+09</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Proteus spp.</td><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">6.64E+08</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Proteus spp.</td><td colspan="1" rowspan="1">ATCC 29905</td><td colspan="1" rowspan="1">4.54E+08</td></tr><tr><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Salmonella spp.</td><td colspan="1" rowspan="1">ATCC 43975</td><td colspan="1" rowspan="1">1.42E+09</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericasubsp. arizonae</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Salmonella spp.</td><td colspan="1" rowspan="1">ATCC 13314</td><td colspan="1" rowspan="1">1.67E+09</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas spp., Pseudomonasaeruginosa, IMP</td><td colspan="1" rowspan="1">IHMA 576602</td><td colspan="1" rowspan="1">4.88E+08</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas mendocina</td><td colspan="1" rowspan="1">Pseudomonas spp.</td><td colspan="1" rowspan="1">ATCC 25411</td><td colspan="1" rowspan="1">5.24E+08</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Klebsiella oxytoca, CTX-M</td><td colspan="1" rowspan="1">IHMA 683079</td><td colspan="1" rowspan="1">7.29E+08</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Klebsiella pneumoniae, KPC</td><td colspan="1" rowspan="1">IHMA 629630</td><td colspan="1" rowspan="1">6.61E+08</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Klebsiella variicola</td><td colspan="1" rowspan="1">Clinical IsolateV0512</td><td colspan="1" rowspan="1">1.28E+09</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Escherichia coli, MCR</td><td colspan="1" rowspan="1">NCTC 13846</td><td colspan="1" rowspan="1">1.17E+09</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Serratia marcescens, SME</td><td colspan="1" rowspan="1">IHMA 1642209</td><td colspan="1" rowspan="1">1.34E+09</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae,Morganella morganii, NDM</td><td colspan="1" rowspan="1">IHMA 605873</td><td colspan="1" rowspan="1">1.28E+09</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 9007</td><td colspan="1" rowspan="1">6.70E+08</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 43744</td><td colspan="1" rowspan="1">3.07E+08</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 700269</td><td colspan="1" rowspan="1">4.18E+08</td></tr><tr><td colspan="1" rowspan="1">Gram-Positive (Off-panel) Organism*</td><td colspan="1" rowspan="1">Expected Result</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">AveragePositive BloodCultureConcentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">7.51E+08</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">4.12E+08</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus(MRSA)</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">2.07E+08</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis (MRSE)</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">2.26E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 12386</td><td colspan="1" rowspan="1">7.50E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 27823</td><td colspan="1" rowspan="1">5.25E+08</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 19659</td><td colspan="1" rowspan="1">8.50E+07</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereusa</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 10702</td><td colspan="1" rowspan="1">4.4E+08</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 27010</td><td colspan="1" rowspan="1">3.74E+08</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumstriatum</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 43735</td><td colspan="1" rowspan="1">3.48E+08</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">No Target Detected</td><td colspan="1" rowspan="1">ATCC 15313</td><td colspan="1" rowspan="1">7.58E+08</td></tr></table>

a Concentration of Bacillus cereus representing single blood bottle culture in VersaTREK™ REDOX™2 EZ Draw™ Media.

\* Genus and species taxonomies are ever evolving based on the latest research.  The strain ID is a unique identifier that will not be changed by the supplier.  Therefore, the strain ID assigned by the supplier should be the reference

# Diasorin

used for that material.  The genus and species should be for reference purposes only.

# 3. Clinical Performance:

A multi‐site clinical study established the diagnostic accuracy of the LIAISON PLEX® Gram‐ Negative Blood Culture (BCN) Assay for the detection and identification of pathogenic gram‐ negative organisms in positive blood culture. The clinical performance of the LIAISON® PLEX BCN Assay was evaluated using clinical specimens prospectively collected between March 2024 and July 2024 from four geographically diverse clinical sites within the United States. The clinical study utilized remnant, de‐identified blood culture specimens collected from patients exhibiting clinical signs and symptoms of bloodstream infection, evidenced by positive identification by a continuous monitoring blood culture system.

A total of 381 unique prospectively collected specimens that met the pre‐determined inclusion criteria were enrolled in the study. Clinical runs and re‐runs using the LIAISON $\mathsf { P l E X } ^ { \mathbb { \otimes } }$ BCN Assay were tested on the LIAISON PLEX® System by trained operators at four clinical sites. For targets that exhibited low prevalence rates in the prospective study, the prospective specimen set was supplemented with 231 pre‐selected left‐over, de‐identified specimens sourced from seven vendors in the United States and one site in Italy. The pre‐selected specimens were identified by Standard of Care (SoC) testing and confirmed as positive by VITEK 2 and/or PCR followed by bi‐directional sequencing (BDS) according to the reference method algorithm prior to enrollment in the study. To minimize bias, pre‐selected specimens were tested across the five sites in a randomized, blinded manner along with negative specimens.

Out of the 381 specimens enrolled in the prospective arm of the study, 30 prospective specimens were excluded from the analysis (one (1) duplicate patient enrollment, and 29 with incomplete reference testing results due to mixed growth, insufficient growth, or no growth).

Contrived specimens were tested to supplement the positive clinical specimens in the prospective and pre‐selected study cohorts for all targets. A total of 746 specimens were contrived, blinded, randomized, and tested along with negative specimens at four testing sites during April 2024– August 2024.

Table 18 provides a summary of the general demographic information of the 351 prospectively collected and 231 pre‐selected specimens that were included in the study analysis.

Table 18: LIAISON PLEX® BCN Assay Summary of the General Demographic Information   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Prospective (N =351)</td><td colspan="1" rowspan="1">Pre-selected (N=231)</td></tr><tr><td colspan="1" rowspan="1"># Specimens (%)</td><td colspan="1" rowspan="1"># Specimens (%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Gender</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">181 (51.6%)</td><td colspan="1" rowspan="1">136 (58.9%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">170 (48.4%)</td><td colspan="1" rowspan="1">91 (39.4%)</td></tr><tr><td colspan="1" rowspan="1">Gender Unknown</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">4 (1.7%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">351 (100.0%)</td><td colspan="1" rowspan="1">231 (100.0%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Age (years)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">0-1</td><td colspan="1" rowspan="1">10 (2.8%)</td><td colspan="1" rowspan="1">13 (5.6%)</td></tr><tr><td colspan="1" rowspan="1">&gt;1-5</td><td colspan="1" rowspan="1">3 (0.9%)</td><td colspan="1" rowspan="1">2 (0.9%)</td></tr><tr><td colspan="1" rowspan="1">&gt;5-21</td><td colspan="1" rowspan="1">10 (2.8%)</td><td colspan="1" rowspan="1">4 (1.7%)</td></tr><tr><td colspan="1" rowspan="1">&gt;21-65</td><td colspan="1" rowspan="1">142 (40.5%)</td><td colspan="1" rowspan="1">90 (39.0%)</td></tr><tr><td colspan="1" rowspan="1">&gt;65</td><td colspan="1" rowspan="1">181 (51.6%)</td><td colspan="1" rowspan="1">107 (46.3%)</td></tr><tr><td colspan="1" rowspan="1">Age Unknown</td><td colspan="1" rowspan="1">5 (1.4%)</td><td colspan="1" rowspan="1">15 (6.5%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">351 (100.0%)</td><td colspan="1" rowspan="1">351 (100.0%)</td></tr><tr><td colspan="3" rowspan="1">Subject Status</td></tr><tr><td colspan="1" rowspan="1">Emergency Room</td><td colspan="1" rowspan="1">68 (19.4%)</td><td colspan="1" rowspan="1">2 (0.9%)</td></tr><tr><td colspan="1" rowspan="1">Hospitalized</td><td colspan="1" rowspan="1">262 (74.6%)</td><td colspan="1" rowspan="1">18 (7.8%)</td></tr><tr><td colspan="1" rowspan="1">Long-Term Care</td><td colspan="1" rowspan="1">2 (0.6%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Outpatient</td><td colspan="1" rowspan="1">1 (0.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Status Unknown</td><td colspan="1" rowspan="1">18 (5.1%)</td><td colspan="1" rowspan="1">211 (91.3%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">351 (100.0%)</td><td colspan="1" rowspan="1">231 (100.0%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Blood Culture Bottle Type</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">BD BACTEC Lytic Anaerobic</td><td colspan="1" rowspan="1">65(18.5%)</td><td colspan="1" rowspan="1">8 (3.5%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC Plus Aerobic</td><td colspan="1" rowspan="1">61(17.4%)</td><td colspan="1" rowspan="1">20 (8.7%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC Standard Aerobic</td><td colspan="1" rowspan="1">19(5.4%)</td><td colspan="1" rowspan="1">89 (38.5%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT FA Plus</td><td colspan="1" rowspan="1">72(20.5%)</td><td colspan="1" rowspan="1">29 (12.6%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT FN PluS</td><td colspan="1" rowspan="1">77(21.9%) -</td><td colspan="1" rowspan="1">26 (11.3%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT PF Plus</td><td colspan="1" rowspan="1">3 (0.9%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT SA Standard Aerobic</td><td colspan="1" rowspan="1">22(6.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BacT/ALERT SN StandardAnaerobic</td><td colspan="1" rowspan="1">16(4.6%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC Plus Anaerobic</td><td colspan="1" rowspan="1">1 (0.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC Peds Plus</td><td colspan="1" rowspan="1">10 (2.8%)</td><td colspan="1" rowspan="1">1 (0.4%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC (Unknown Aerobic)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">5 (2.2%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC (UnknownAnaerobic)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">3 (1.3%)</td></tr><tr><td colspan="1" rowspan="1">BD BACTEC (Unknown)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">15 (6.5%)</td></tr><tr><td colspan="1" rowspan="1">Unknown Aerobic</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">4 (1.7%)</td></tr><tr><td colspan="1" rowspan="1">Unknown Anaerobic</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">1 (0.4%)</td></tr><tr><td colspan="1" rowspan="1">Bottle Type Unknown</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">30 (13.0%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">351 (100.0%)</td><td colspan="1" rowspan="1">231 (100.0%)</td></tr></table>

The LIASON $\mathsf { P l E X } ^ { \otimes }$ BCN Assay results were compared to culture followed by automated microbiological/biochemical identification using VITEK 2, PCR followed by BDS, or a combination according to the algorithm described in Table 19 below.

Table 19 – Reference Method Algorithm   

<table><tr><td colspan="1" rowspan="1">LIAISON PLEX® BCN AsSay Target</td><td colspan="1" rowspan="1">Comparator Method</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="7">Culture followed by Automatedmicrobiological/biochemical identification using VITEK2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter spp.</td></tr><tr><td colspan="1" rowspan="1">Enterobacter spp.</td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae/Morganellaceae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(1)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="7"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas spp.</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="3">Culture followed by Automatedmicrobiological/biochemical identification using VITEK2 with positive confirmation of the clinical isolate byPCR/BDS</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="9">PCR followed by bi-directional sequencing</td></tr><tr><td colspan="1" rowspan="1">CTX-M (blacTx-M)</td></tr><tr><td colspan="1" rowspan="1">IMP (blaiMP)</td></tr><tr><td colspan="1" rowspan="1">KPC (blakPc)</td></tr><tr><td colspan="1" rowspan="1">NDM (blaNDM)</td></tr><tr><td colspan="1" rowspan="1">OXA (blaoxA)</td></tr><tr><td colspan="1" rowspan="1">VIM (blaviM)</td></tr><tr><td colspan="1" rowspan="1">MCR</td></tr><tr><td colspan="1" rowspan="1">SME</td></tr></table>

(1)The LIAISION PLEX® BCN Assay is not designed to distinguish between Escherichia coli and Shigella spp. (S. dysenteriae, S. boydii, S. flexneri, and S. sonnei).

Of the 582 clinical specimens collectively included in the prospective and pre‐selected study analysis (Arms 1 and 2 combined), 556 samples $( 9 5 . 5 \% )$ generated valid LIAISON PLEX® BCN Assay results (i.e., Detected or Not Detected) on the first attempt. There were 26 specimens $( 4 . 5 \% )$ with an invalid result on the initial run. Of the 26 specimens retested, 24 $( 4 . 1 \% )$ specimens generated valid BCN results after a single retest for a final testing success rate of $9 9 . 7 \%$ (580/582).

For each target in the LIAISON PLEX® BCN Assay Panel, the diagnostic performance of the LIAISON PLEX® BCN Assay was determined using Sensitivity/PPA and Specificity/NPA, along with the associated $9 5 \%$ confidence intervals, $9 5 \%$ CI as compared to the reference method).  The results of the combined prospective and pre‐selected specimen analysis are summarized in Table 20.

Table 20 – Sensitivity/PPA and Specificity/NPA of Combined Prospective and Pre‐Selected Data Set   

<table><tr><td colspan="2" rowspan="2">Pathogen Target</td><td colspan="3" rowspan="1">Sensitivity/PPA</td><td colspan="3" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Sensitivity/PPA (%)</td><td colspan="1" rowspan="1">95% Cl</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">Specificity/NPA (%)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="3">Acinetobacterbaumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">350/350</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">30/31</td><td colspan="1" rowspan="1">96.8%</td><td colspan="1" rowspan="1">83.8% - 99.4%</td><td colspan="1" rowspan="1">199/199</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.1% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">30/31</td><td colspan="1" rowspan="1">96.8%</td><td colspan="1" rowspan="1">83.8% - 99.4%</td><td colspan="1" rowspan="1">549/549</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">3/5</td><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">23.1% - 88.2%</td><td colspan="1" rowspan="1">345/345</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">33/35</td><td colspan="1" rowspan="1">94.3%</td><td colspan="1" rowspan="1">81.4% - 98.4%</td><td colspan="1" rowspan="1">195/195</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.1% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">36/401</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">76.9% - 96%</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Citrobacter spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">61% - 100%</td><td colspan="1" rowspan="1">344/344</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8%</td><td colspan="1" rowspan="1">79.8% - 99.3%</td><td colspan="1" rowspan="1">206/206</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">83.3% - 99.4%</td><td colspan="1" rowspan="1">550/550</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">TP /(TP+FN)</td><td colspan="1" rowspan="1">Sensitivity/PPA (%)</td><td colspan="1" rowspan="1">95% Cl</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">Specificity/NPA (%)</td><td colspan="1" rowspan="1">95% Cl</td></tr><tr><td colspan="1" rowspan="3">Enterobacter spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9% - 100%</td><td colspan="1" rowspan="1">330/330</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">59.6% - 98.2%</td><td colspan="1" rowspan="1">220/220</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">83.3% - 99.4%</td><td colspan="1" rowspan="1">550/550</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Enterobacteriaceae/Morganellaceae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">305/306</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.2% - 99.9%</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">123/123</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97% - 100%</td><td colspan="1" rowspan="1">106/107</td><td colspan="1" rowspan="1">99.1%</td><td colspan="1" rowspan="1">94.9% - 99.8%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">428/429</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">98.7% - 100%</td><td colspan="1" rowspan="1">150/151</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">96.3% - 99.9%</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">178/179</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">96.9% - 99.9%</td><td colspan="1" rowspan="1">169/171</td><td colspan="1" rowspan="1">98.8%</td><td colspan="1" rowspan="1">95.8% - 99.7%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">229/229</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.4% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">179/180</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">96.9% - 99.9%</td><td colspan="1" rowspan="1">398/400²2</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">98.2% - 99.9%</td></tr><tr><td colspan="1" rowspan="3">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">67.6% - 100%</td><td colspan="1" rowspan="1">342/342</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">86.2% - 100%</td><td colspan="1" rowspan="1">206/206</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">32/32</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89.3% - 100%</td><td colspan="1" rowspan="1">548/548</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">70.1% - 100%</td><td colspan="1" rowspan="1">340/341</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">26/27</td><td colspan="1" rowspan="1">96.3%</td><td colspan="1" rowspan="1">81.7% - 99.3%</td><td colspan="1" rowspan="1">203/203</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.1% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">85.8% - 99.5%</td><td colspan="1" rowspan="1">543/5443</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99% - 100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">44/45</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.4% - 99.6%</td><td colspan="1" rowspan="1">304/305</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.2% - 99.9%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">227/229</td><td colspan="1" rowspan="1">99.1%</td><td colspan="1" rowspan="1">96.9% - 99.8%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">45/46</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.7% - 99.6%</td><td colspan="1" rowspan="1">531/534</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98.4% - 99.8%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella variicola</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">67.6% - 100%</td><td colspan="1" rowspan="1">341/341</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">51% - 100%</td><td colspan="1" rowspan="1">197/197</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.1% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">75.8% - 100%</td><td colspan="1" rowspan="1">538/538</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Morganellamorganii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">43.9% - 100%</td><td colspan="1" rowspan="1">347/347</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">56.6% - 100%</td><td colspan="1" rowspan="1">225/225</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">67.6% - 100%</td><td colspan="1" rowspan="1">572/572</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Neisseriameningitidis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">350/350</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">33.3%</td><td colspan="1" rowspan="1">6.1% - 79.2%</td><td colspan="1" rowspan="1">227/227</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/35$</td><td colspan="1" rowspan="1">33.3%</td><td colspan="1" rowspan="1">6.1% - 79.2%</td><td colspan="1" rowspan="1">577/577</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Proteus spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">85.7% - 100%</td><td colspan="1" rowspan="1">327/327</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">78.5% - 100%</td><td colspan="1" rowspan="1">216/216</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.6% - 100%</td><td colspan="1" rowspan="1">543/543</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">28/29</td><td colspan="1" rowspan="1">96.6%</td><td colspan="1" rowspan="1">82.8% - 99.4%</td><td colspan="1" rowspan="1">321/321</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">61% - 100%</td><td colspan="1" rowspan="1">224/224</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">34/35</td><td colspan="1" rowspan="1">97.1%</td><td colspan="1" rowspan="1">85.5% - 99.5%</td><td colspan="1" rowspan="1">545/545</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">83.3% - 99.4%</td><td colspan="1" rowspan="1">320/320</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">7/17</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">64.6% - 100%</td><td colspan="1" rowspan="1">223/223</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.3% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">36/37</td><td colspan="1" rowspan="1">97.3%</td><td colspan="1" rowspan="1">86.2% - 99.5%</td><td colspan="1" rowspan="1">543/543</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">Salmonella spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">349/349</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">80.6% - 100%</td><td colspan="1" rowspan="1">214/214</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.6% - 100%</td><td colspan="1" rowspan="1">563/563</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="2">Serratiamarcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">70.1% - 100%</td><td colspan="1" rowspan="1">340/341</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9% - 100%</td><td colspan="1" rowspan="1">209/210</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">97.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88.3% - 100%</td><td colspan="1" rowspan="1">549/551</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">98.7% - 99.9%</td></tr><tr><td colspan="1" rowspan="3">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">349/349</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">72.2% - 100%</td><td colspan="1" rowspan="1">219/220</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">97.5% - 99.9%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74.1% - 100%</td><td colspan="1" rowspan="1">568/569</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99% - 100%</td></tr><tr><td colspan="8" rowspan="1">Resistance Marker Genes6</td></tr><tr><td colspan="1" rowspan="3">CTX-M (blacTx-M)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">40/41</td><td colspan="1" rowspan="1">97.6%</td><td colspan="1" rowspan="1">87.4% - 99.6%</td><td colspan="1" rowspan="1">295/297</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">97.6% - 99.8%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">56.6% - 100%</td><td colspan="1" rowspan="1">169/169</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">45/46</td><td colspan="1" rowspan="1">97.8%</td><td colspan="1" rowspan="1">88.7% - 99.6%</td><td colspan="1" rowspan="1">464/466</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">98.4% - 99.9%</td></tr><tr><td colspan="1" rowspan="3">IMP (blaiMP)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">338/338</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">174/174</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">512/512</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">KPC (blakpc)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">336/336</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">164/164</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.7% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">500/500</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.2% - 100%</td></tr><tr><td colspan="1" rowspan="3">NDM (blanDM)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">338/338</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">173/173</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.7% - 100%</td><td colspan="1" rowspan="1">511/511</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">OXA (blaoxA)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">332/337</td><td colspan="1" rowspan="1">98.5%</td><td colspan="1" rowspan="1">96.6% - 99.4%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">87.1% - 100%</td><td colspan="1" rowspan="1">137/138</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">96% - 99.9%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">87.1% - 100%</td><td colspan="1" rowspan="1">469/475</td><td colspan="1" rowspan="1">98.7%</td><td colspan="1" rowspan="1">97.3% - 99.4%</td></tr><tr><td colspan="1" rowspan="3">VIM (blaviM)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">338/338</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">174/174</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.8% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">512/512</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr><tr><td colspan="1" rowspan="3">MCR</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">334/334</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">131/131</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">465/465</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.2% - 100%</td></tr><tr><td colspan="1" rowspan="3">SME</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">72.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Pre-selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">84.5% - 100%</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89% - 100%</td></tr></table>

1Clinical isolates from 2/4 Acinetobacter spp. false negative specimens were negative for Acinetobacter spp. by BDS. 2The clinical isolate from $_ { 1 / 2 }$ Escherichia coli false positive specimens was positive for Escherichia coli by BDS. 3The Klebsiella oxytoca false positive specimen was positive for Klebsiella oxytoca by Standard of Care. 4Two of the three (2/3) Klebsiella pneumoniae false positive specimens were positive for Klebsiella pnuemoniae by Standard of Care. 5Both Neisseria meningitidis false negative specimens were positive by another FDA-cleared molecular assay. 6The LIAISON PLEX® BCN assay will report the presence or absence of resistance markers only if an applicable organism is also detected, therefore the total number of evaluable samples for each resistance marker is dependent on the number of applicable organisms enrolled.

Out of the 746 specimens included in the contrived study analysis, 717 specimens $( 9 6 . 1 \% )$ generated valid LIAISON $\mathsf { P l E X } ^ { \otimes }$ BCN Assay results (i.e., Detected or Not Detected) on the first attempt. There were 29 specimens $( 3 . 9 \% )$ with an invalid result on the initial run. Of the 29 specimens retested, 28 $( 3 . 8 \% )$ specimens generated a valid result after a single retest for a final

# Diasorin

testing success rate of $9 9 . 9 \%$ (745/746).

The performance (Sensitivity/PPA and Specificity/NPA) of the LIAISION PLEX® BCN Assay for contrived specimens is presented separately in Table 21.

Table 21 – Sensitivity/PPA and Specificity/NPA for the Contrived Data Set   

<table><tr><td rowspan=1 colspan=1>Pathogen Target</td><td rowspan=1 colspan=3>Sensitivity/PPA</td><td rowspan=1 colspan=3>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>TP /(TP+FN)</td><td rowspan=1 colspan=1>Sensitivity/PPA (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN / (TN+FP)</td><td rowspan=1 colspan=1>Specificity/NPA (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=7>Bacterial Identification</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.7% - 100%</td><td rowspan=1 colspan=1>720/720</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.1% - 100%</td><td rowspan=1 colspan=1>700/700</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Citrobacter spp.</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>89.3% - 100%</td><td rowspan=1 colspan=1>713/713</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter spp.</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.4% - 100%</td><td rowspan=1 colspan=1>691/691</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae/Morganellaceae</td><td rowspan=1 colspan=1>460/460</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2% - 100%</td><td rowspan=1 colspan=1>284/285</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.4% - 100%</td><td rowspan=1 colspan=1>704/704</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0% - 0%</td><td rowspan=1 colspan=1>745/745</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9% - 100%</td><td rowspan=1 colspan=1>725/725</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.3% - 100%</td><td rowspan=1 colspan=1>698/699</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9% - 100%</td><td rowspan=1 colspan=1>695/695</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.2% - 100%</td><td rowspan=1 colspan=1>683/683</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9% - 100%</td><td rowspan=1 colspan=1>695/695</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.6% - 100%</td><td rowspan=1 colspan=1>715/715</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>85.8% -99.5%</td><td rowspan=1 colspan=1>709/709</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas spp.</td><td rowspan=1 colspan=1>57/58</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>90.9% -99.7%</td><td rowspan=1 colspan=1>687/687</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>70/70</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.8% - 100%</td><td rowspan=1 colspan=1>675/675</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.5% - 100%</td><td rowspan=1 colspan=1>690/690</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.7% - 100%</td><td rowspan=1 colspan=1>696/696</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5% - 100%</td></tr><tr><td rowspan=1 colspan=7>Resistance Marker Genes1</td></tr><tr><td rowspan=1 colspan=1>CTX-M (blacTx-M)</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.5% - 100%</td><td rowspan=1 colspan=1>305/306</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.2% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>IMP (blaiMP)</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.4% - 100%</td><td rowspan=1 colspan=1>357/357</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>KPC (blakPcC)</td><td rowspan=1 colspan=1>61/61</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.1% - 100%</td><td rowspan=1 colspan=1>350/350</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>NDM (blaNDM)</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.6% - 100%</td><td rowspan=1 colspan=1>344/344</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>OXA (blaoxA)</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.1% - 100%</td><td rowspan=1 colspan=1>375/376</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.5% - 100%</td></tr><tr><td rowspan=1 colspan=1>VIM (blaviM)</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9% - 100%</td><td rowspan=1 colspan=1>361/361</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>MCR</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9% - 100%</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8% - 100%</td></tr><tr><td rowspan=1 colspan=1>SME</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9% - 100%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6% - 100%</td></tr></table>

1The LIAISON PLEX® BCN assay will report the presence or absence of resistance markers only if an applicable organism is also detected, therefore the total number of evaluable samples for each resistance marker is dependent on the number of applicable organisms enrolled.

# Diasorin

The pre‐defined acceptance criteria for the clinical study were:

The assay shall achieve a target Sensitivity/Positive Percent Agreement of $290 \%$ for all targets except Klebsiella oxytoca and Stenotrophomonas maltophilia which shall achieve $2 8 5 \%$ . The Specificity/Negative Percent Agreement for each target should be established at a level of $2 9 5 \%$ .  Failure rate shall be $\leq 1 0 \%$ .

Acceptance Criteria are based on Prospective, Pre‐selected, and Contrived sets combined.

As all predefined acceptance criteria were met, the study results demonstrate that the diagnostic accuracy of the LIAISON PLEX® BCN Assay is acceptable for the safe and effective detection and identification of gram‐negative pathogens from blood culture media identified as positive by a continuous monitoring blood culture system and which contain gram‐negative organism as determined by Gram stain from patients exhibiting clinical signs and symptoms of bloodstream infection.

# N. Proposed Labeling:

The labeling provided in the submission satisfies the requirements of 21 CFR 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.