# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

A 510(k) Number K243396   
B Applicant Hologic, Inc.   
C Proprietary and Established Names Aptima SARS-CoV-2 Assay   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QQX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The Sars-Cov-2 Respiratory InfectionAnd Other MicrobialAgents When In A Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to show that the Aptima SARS-CoV-2 Assay is substantially equivalent to the BioFire COVID-19 Test 2 (K211079) and to obtain clearance for the Aptima SARS-CoV-2 Assay.

# B Measurand:

The Aptima SARS-CoV-2 Assay detects Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) ribosomal RNA isolated from nasopharyngeal swab specimens and anterior nasal swab specimens from patients with signs and symptoms of respiratory tract infections.

# C Type of Test:

This assay is a nucleic acid amplification test that uses Transcription Mediated Amplification (TMA), and Dual Kinetic Assay (DKA) to detect SARS-CoV-2 RNA.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

Aptima SARS-CoV-2 Assay is a nucleic acid amplification in vitro diagnostic test intended for the qualitative detection of RNA from severe acute respiratory syndrome coronavirus 2 (SARSCoV-2) isolated and purified from nasopharyngeal (NP) swabs and anterior nasal (AN) swabs obtained from patients exhibiting signs and symptoms of COVID-19.

Positive results are indicative of the presence of SARS-CoV-2 RNA. The Aptima SARS-CoV-2 Assay is intended for use as an aid in the diagnosis of COVID-19 if used in conjuction with other clinical, epidemiological, and laboratory findings. Clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out bacterial infection or co-infection with other viruses.

Negative results do not preclude SARS-CoV-2 infection and should not be used as the sole basis for patient management decisions. Negative results must be combined with clinical observations, patient history, and epidemiological information.

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only For in vitro diagnostic use only

D Special Instrument Requirements: For use with the Panther and Panther Fusion System only.

# IV Device/System Characteristics:

# A Device Description:

The Aptima SARS-CoV-2 Assay is an in vitro diagnostic test intended for qualitative detection of RNA from the SARS-CoV-2 virus isolated and purified from nasopharyngeal and anterior nasal swab specimens obtained from individuals exhibiting signs and symptoms of COVID-19.

The Aptima SARS-CoV-2 Assay involves four main steps: target capture, transcription-mediated amplification (TMA), Hybridization Protection Assay (HPA), and dual kinetic assay (DKA) detection. Nucleic acid capture takes place in a single tube of a Multi-Tube Unit (MTU) on Panther systems. The captured nucleic acid is combined with amplification reagents. TMA is then performed for the captured nucleic acid on the Panther system. Acridinium Ester (AE) labeled probe oligos bind to the generated amplicon, resulting in light emission. The Panther

system compares the relative light unit (RLU) to a predetermined cut-off to produce a qualitative result for the presence or absence of the analyte. Dual Kinetic Assay (DKA) technology enables multiplex detection from a single sample.

# Results Reporting

The system software assesses the validity of all samples including controls. The user interprets results generated based on Table 1, which is included in the Aptima SARS-CoV-2 Assay package insert.

Table 1: Results Interpretation   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2 Result</td><td rowspan=1 colspan=1>IC Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>SARS-CoV-2 not detected</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>SARS-CoV-2 detected</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Invalid. There was an error in thegeneration of the result; retest sample.</td></tr></table>

# B Principle of Operation:

The Aptima SARS-CoV-2 Assay combines the technologies of target capture, Transcription Mediated Amplification (TMA), and Dual Kinetic Assay (DKA).

Specimens are collected and transferred into their respective specimen transport tubes. The transport solutions in these tubes release the RNA target and protect them from degradation during storage. When the Aptima SARS-CoV-2 Assay is performed in the laboratory, the target RNA molecules are isolated from specimens by use of capture oligomers via target capture that utilizes magnetic microparticles. The capture oligomers contain sequences complementary to specific regions of the target molecules as well as a string of deoxyadenosine residues. A separate capture oligomer is used for each target. During the hybridization step, the sequence specific regions of the capture oligomers bind to specific regions of the target molecules. The capture oligomer:target complex is then captured out of solution by decreasing the temperature of the reaction to room temperature. This temperature reduction allows hybridization to occur between the deoxyadenosine region on the capture oligomer and the poly-deoxythymidine molecules that are covalently attached to the magnetic particles. The microparticles, including the captured target molecules bound to them, are pulled to the side of the reaction vessel using magnets and the supernatant is aspirated. The particles are washed to remove residual specimen matrix that may contain amplification reaction inhibitors. After the target capture steps are completed, the specimens are ready for amplification.

Target amplification assays are based on the ability of complementary oligonucleotide primers to specifically anneal and allow enzymatic amplification of the target nucleic acid strands. The Aptima SARS-CoV-2 Assay replicates specific regions of the RNA from SARS-CoV-2 virus. Detection of the RNA amplification product sequences (amplicon) is achieved using nucleic acid hybridization. Single-stranded chemiluminescent nucleic acid probes, which are unique and complementary to a region of each target amplicon and Internal Control (IC) amplicon, are labeled with different acridinium ester (AE) molecules. The AE-labeled probes combine with the amplicon to form stable hybrids. The Selection Reagent differentiates hybridized from unhybridized probe, eliminating the generation of signal from the unhybridized probe. During the detection step, light emitted from the labeled hybrids is measured as photon signals in a

luminometer and are reported as Relative Light Units (RLU). In DKA, differences in the kinetic profiles of the labeled probes allow for the differentiation of signal; kinetic profiles are derived from measurements of photon output during the detection read time.

The Aptima SARS-CoV-2 Assay amplifies and detects two conserved regions of the ORF1ab gene in the same reaction, using the “glower” kinetic type. The two regions are not differentiated and amplification of either or both regions lead to RLU signal. The assay results are determined by a cut-off based on the total RLU and the kinetic curve type.

# C Instrument Description Information:

1. Instrument Name: Panther or Panther Fusion system, software version 7.2.7 and 7.2.9

2. Specimen Identification: Specimen identification is entered via barcode.

3. Specimen Sampling and Handling:

Nasopharyngeal swab (NPS) specimens and anterior nasal swab (ANS) specimens collected in transport media.

4. Calibration:

Real Time Fluorometers (RTF) undergo a single calibration during manufacturing. No additional calibration is performed by the end user.

5. Quality Control:

In order to generate valid results, a set of assay controls must be tested. Two types of controls are provided: one negative control; and one positive control. One replicate of the negative assay control and one replicate of positive assay control must be tested each time a new kit is loaded on the system or when the current set of valid controls have expired.

An internal control is added to each sample with the working Target Capture Reagent (wTCR) and is co-extracted and co-amplified with the target nucleic acid. During processing, the internal control acceptance criteria are automatically verified by the Panther system software.

Detection of the internal control is not required for samples that are positive for SARS-CoV2. The internal control must be detected in all samples that are negative for SARS-CoV-2 targets; samples that fail to meet that criteria will be reported as Invalid. Each sample with an Invalid result must be retested. Both types of controls must generate results within predefined specifications to be valid

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire COVID-19 Test 2

# C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K243396</td><td colspan="1" rowspan="1">K211079</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Aptima SARS-CoV-2 Assay</td><td colspan="1" rowspan="1">BioFire COVID-19 Test 2</td></tr><tr><td colspan="1" rowspan="1">Regulation Number</td><td colspan="1" rowspan="1">21 CFR 866.3981</td><td colspan="1" rowspan="1">21 CFR 866.3981</td></tr><tr><td colspan="1" rowspan="1">Product Code(s)</td><td colspan="1" rowspan="1">QQX</td><td colspan="1" rowspan="1">QQX</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The Aptima SARS-CoV-2Assay is a nucleic acidamplification in vitrodiagnostic test intended forthe qualitative detection ofRNÁ from severe acuterespiratory syndromecoronavirus 2 (SARS-CoV-2) isolated and purified fromnasopharyngeal (NP) swabsand anterior nasal (AN)swabs obtained from patientswith signs and symptoms ofCOVID-19.Positive results are indicativeof the presence of SARS-CoV-2 RNA. The AptimaSARS-CoV-2 Assay isintended for use as an aid tothe diagnosis of COVID-19if used in conjunction withother clinical,epidemiological, andlaboratory findings. Clinicalcorrelation with patienthistory and other diagnosticinformation is necessary todetermine patient infectionstatus. Positive results do notrule out bacterial infection orco-infection with otherviruses.Negative results do notpreclude SARS-CoV-2infection and should not beused as the sole basis for</td><td colspan="1" rowspan="1">The BioFire COVID-19 Test2 is a qualitative nestedmultiplexed RT-PCR in vitrodiagnostic test intended foruse with the BioFireFilmArray 2.0 and BioFireFilmArray Torch Systems.The BioFire COVID-19 Test2 detects nucleic acids fromsevere acute respiratorysyndrome coronavirus 2(SARS-CoV-2) innasopharyngeal swabs (NPS)from individuals suspected ofCOVID-19 by their healthcareprovider.Results are for theidentification of SARS-CoV-2RNA. The SARS-CoV-2RNA is generally detectablein NPS specimens during theacute phase of infection.Positive results are indicativeof the presence of SARS-CoV-2 RNA; clinicalcorrelation with patienthistory and other diagnosticinformation is necessary todetermine patient infectionstatus. Positive results do notrule out co-infection withother pathogens.Results are meant to be usedin conjunction with otherclinical, epidemiologic, andlaboratory data, in accordance</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">patient managementdecisions. Negative resultsmust be combined withclinical observations, patienthistory, and epidemiologicalinformation.</td><td colspan="1" rowspan="1">with the guidelines providedby the relevant public healthauthorities. The BioFireCOVID-19 Test 2 is intendedfor use by trained medical andlaboratory professionals in alaboratory setting or under thesupervision of a trainedlaboratory professional.</td></tr><tr><td colspan="1" rowspan="1">Intended User</td><td colspan="1" rowspan="1">Professional use</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasopharyngeal swabs andanterior nasal swabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Technology Principle ofOperation</td><td colspan="1" rowspan="1">Transcription-mediatedamplification NAAT</td><td colspan="1" rowspan="1">Nested multiplex RT-PCRfollowed by high resolutionmelting analysis to confirmidentity of amplified nucleicacids</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Internal and run controls</td><td colspan="1" rowspan="1">Internal and external controls</td></tr><tr><td colspan="1" rowspan="1">Time to Obtain Test Results</td><td colspan="1" rowspan="1">Approximately 2.5 hours</td><td colspan="1" rowspan="1">Approximately 45 minutes</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981 Transition Plan for Medical Devices Issued Emergency Use Authorizations (EUAs) Related to Coronavirus Disease 2019 (COVID-19), Guidance for Industry, Other Stakeholders, and Food and Drug Administration Staff, Document issued on March 27, 2023 • The 510(k) Program: Evaluating Substantial Equivalence in Premarket Notifications [510(k)], Guidance for Industry and Food and Drug Administration Staff, July 2014 Interference Testing in Clinical Chemistry. (CLSI) EP07 3rd Edition Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline - Third Edition (CLSI) EPI5-A3 (Reaffirmed: September 2019) User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition. (CSLI) EPI2-A2 Evaluation of Commutability of Processed Samples; Approved Guideline - Third Edition. (CLSI) EP14-A3 User Verification of Precision and Estimation of Bias; Approved Guideline - Third Edition (CLSI) IIEPI 5-A3 (Reaffirmed: September 2019) Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition. (CLSI) EP17-A2 Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guidelines – Second Edition (CLSI) EP24-A2

Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. (CLSI). EP25-A (Replaces EP25-P) Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline (CLSI). MMO3-3rd Edition (Replaces MM03-A2) Collection Transport Preparation and Storage of Specimens for Molecular Methods (CLSI). MM13 $2 ^ { \mathrm { { \bar { n } d } } }$ Edition Medical devices - Symbols to be used with information to be supplied by the manufacturer - Part 1: General requirements (ISO) 15223- Fourth edition 2021-07 Medical devices - Application of risk management to medical devices (ISO) 14971 Third Edition 2019-12 Medical devices - Part 1: Application of usability engineering to medical devices (IEC) 62366-1 Edition 1.1 2020-06 CONSOLIDATED VERSION Medical device software - Software life cycle processes (IEC) 62304 Edition 1.1 2015-06 CONSOLIDATED VERSION Evaluation of Qualitative Binary Output Examination Performance (CLSI) EP12 3rd Edition   
• Evaluation of Stability of In Vitro Medical Laboratory Test Reagents (CLSI) EP25 2nd Edition Assessment of Equivalence or Suitability of Specimen Types for Medical Laboratory Measurement Procedures (CLSI) EP35 1st Edition   
• Supplemental Tables for Interference Testing in Clinical Chemistry (CLSI) EP37 1st Edition

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

# a. Precision

Aptima SARS-CoV-2 Assay within-lab precision was evaluated with a 4-member panel consisting of inactivated SARS-CoV-2 virus in negative clinical NP swab VTM/UTM matrix. The 4-member panel included a Negative, a High Negative (0.1x LoD), a Low Positive (1x LoD) and a Moderate Positive (5x LoD) panel. The panels were tested by two operators, across three panther systems, using three reagent lots, with two runs per day over six days for a minimum of 36 runs per panel. Each of the four panels was tested in three replicates per run for a total of 108 replicates per panel.

A total of 39 runs were completed for this study. Two runs were invalid due to sample dispense verification failures of the negative control for an invalid rate of $5 . 1 \%$ . Both invalid runs were repeated. Of the total 433 tests completed in the 37 valid runs, there was one invalid test due to a volume verification failure error. The test was repeated in the same run and was valid upon retest. The invalid reaction rate was $0 . 2 \%$ (1/433).

The agreement with expected results was $100 \%$ in the Negative, Low Positive and Moderate Positive panel members. The High Negative panel member was $1 0 \mathrm { x }$ below the assay LoD, therefore a mix of positive and negative results were expected. This panel had

68/108 $( 6 3 \% )$ positive results. Agreement with expected results for all four panels is shown in Table 2.

Table 2: Agreement of Aptima SARS-CoV-2 Assay Results with Expected Results   

<table><tr><td rowspan=1 colspan=1>Panel Description</td><td rowspan=1 colspan=1>PanelConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>MeankRLU</td><td rowspan=1 colspan=1>% Agreement withExpected Results/(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0% (0/108)</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>100% (96.6-100)</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>0.1x LoD (0.001)</td><td rowspan=1 colspan=1>63% (68/108)</td><td rowspan=1 colspan=1>627</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1.0x LoD (0.01)</td><td rowspan=1 colspan=1>100% (108/108)</td><td rowspan=1 colspan=1>1131</td><td rowspan=1 colspan=1>100% (96.6-100)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>5.0x LoD (0.05)</td><td rowspan=1 colspan=1>100% (108/108)</td><td rowspan=1 colspan=1>1147</td><td rowspan=1 colspan=1>100% (96.6-100)</td></tr></table>

The total SARS-CoV-2 signal variability measured as $\% C V$ ranged from $2 . 7 5 \%$ to $3 . 8 4 \%$ in Negative, Low Positive, and Moderate Positive panel members. For the sources of variation all six factors evaluated had $\% \mathrm { C V }$ values $< 3 . 0 \%$ as shown in Table 3. The High Negative panel member is $1 0 \mathrm { x }$ below the assay LoD and the $\% C V$ for this panel is expected to be higher than the others. The highest source of variability for this panel was within-run variability.

Table 3: Signal (kRLU) Variability of the Aptima SARS-CoV-2 Assay by Panel Member   

<table><tr><td rowspan=2 colspan=1>PanelDescription</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenInstruments</td><td rowspan=1 colspan=2>BetweenOperators</td><td rowspan=1 colspan=2>BetweenLots</td><td rowspan=1 colspan=2>BetweenRuns</td><td rowspan=1 colspan=2>Within Runs</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>4.97</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>4.04</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>6.75</td><td rowspan=1 colspan=1>2.33</td><td rowspan=1 colspan=1>9.35</td><td rowspan=1 colspan=1>3.23</td></tr><tr><td rowspan=1 colspan=1>HighNegativ*</td><td rowspan=1 colspan=1>30.45</td><td rowspan=1 colspan=1>4.85</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>244.08</td><td rowspan=1 colspan=1>38.91</td><td rowspan=1 colspan=1>245.97</td><td rowspan=1 colspan=1>39.21</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>6.46</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>6.74</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>28.10</td><td rowspan=1 colspan=1>2.48</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>31.77</td><td rowspan=1 colspan=1>2.81</td><td rowspan=1 colspan=1>43.43</td><td rowspan=1 colspan=1>3.84</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>8.53</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>5.59</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>22.98</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>11.06</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>15.59</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>31.59</td><td rowspan=1 colspan=1>2.75</td></tr></table>

$\mathrm { C V } =$ coefficient of variation, $\mathrm { S D } =$ standard deviation. \*Panel was built to 10x below the assay LoD. Higher variability is expected in this panel. Note: In the event that variability from some factors is numerically negative, SD and CV are shown as 0.0.

# b. Reproducibility

Aptima SARS-CoV-2 Assay reproducibility was evaluated at three US sites using one negative and two positive panel members. Testing was performed using one lot of assay reagents and six operators (two at each site). At each site, testing was performed for at least five days. Each run had three replicates of each panel member. A negative panel member was created using pooled negative clinical NP swab specimens in VTM/UTM processed into tubes containing specimen transport media (STM). Positive panel members were created by spiking 1-2x LoD (low positive) or $3 { - } 5 \mathbf { x }$ LoD (moderate positive) concentrations of SARS-CoV-2 inactivated virus into the clinical negative matrix.

A total of 32 runs were performed with two invalid runs at one site for an invalid run rate of $6 . 3 \%$ . All 270 samples processed in valid runs had valid results for an invalid reaction rate of $0 . 0 \%$ . Table 4 shows the agreement by site and overall agreement with expected results by panel member. Agreements with the expected results were $100 \%$ for all panel member components.

Table 4: Reproducibility Study Results   

<table><tr><td rowspan=2 colspan=1>PanelDescription</td><td rowspan=2 colspan=1>PanelConcentration</td><td rowspan=1 colspan=4>% Agreement with Expected Results/ (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (90/90)(95.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1x-2x LoD</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (90/90)(95.9-100.0)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>3x-5x LoD</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (30/30)(88.7-100.0)</td><td rowspan=1 colspan=1>100% (90/90)(95.9-100.0)</td></tr></table>

The overall analysis results (all sites combined) are summarized in Table 5.

Table 5: Reproducibility Study – RLU Signal Variability Analysis   

<table><tr><td rowspan=2 colspan=1>PanelDescription</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeankRLU</td><td rowspan=1 colspan=2>BetweenSites</td><td rowspan=1 colspan=2>BetweenOperators/Runsa</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>WithinRuns</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>286.0</td><td rowspan=1 colspan=1>27.04</td><td rowspan=1 colspan=1>9.45</td><td rowspan=1 colspan=1>25.42</td><td rowspan=1 colspan=1>8.89</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>6.55</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>37.69</td><td rowspan=1 colspan=1>13.18</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1152.2</td><td rowspan=1 colspan=1>67.79</td><td rowspan=1 colspan=1>5.88</td><td rowspan=1 colspan=1>15.16</td><td rowspan=1 colspan=1>1.32</td><td rowspan=1 colspan=1>25.06</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>53.77</td><td rowspan=1 colspan=1>4.67</td><td rowspan=1 colspan=1>91.35</td><td rowspan=1 colspan=1>7.93</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1163.7</td><td rowspan=1 colspan=1>77.30</td><td rowspan=1 colspan=1>6.64</td><td rowspan=1 colspan=1>36.60</td><td rowspan=1 colspan=1>3.15</td><td rowspan=1 colspan=1>4.10</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>26.67</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>89.68</td><td rowspan=1 colspan=1>7.71</td></tr></table>

$\mathrm { C V } =$ coefficient of variation, $\mathrm { S D = }$ standard deviation, kRLU $=$ relative light unit x 1000 a Between Operator may be confounded with Between Run; therefore, Between Operator and Between Run estimates are combined in Between Operator/Run.

# 2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

Analytical Reactivity (Inclusivity)

a. Wet-Testing

This study was performed to determine the analytical reactivity of the Aptima SARSCoV-2 Assay with clinically relevant strains of the assay target. Panels were prepared by spiking cultured viral stock into pooled negative clinical NP swabs in VTM/UTM processed into tubes containing STM. Panels were prepared at a minimum of 3x LoD for each strain individually and tested in triplicate. Higher concentrations were tested if $< 1 0 0 \%$ detection was observed at $3 \mathbf { x }$ LoD. The strains evaluated and the lowest concentration that achieved $100 \%$ reactivity are shown in Table 6 below.

Table 6: Testing Results for SARS-CoV-2 Inclusivity   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2 Strain /Variant</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>USA-WA1/2020*</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA-CA1/2020</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA-AZ1/2020</td><td rowspan=1 colspan=1>0.101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA-WI1/2020</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/OR-OHSU-PHL00037/2021 | B.1.1.7</td><td rowspan=1 colspan=1>0.03TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Uganda/MUWRP-20200195568/2020 | A.23.1</td><td rowspan=1 colspan=1>0.03TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/PHC658/2021 | B.1.617.2</td><td rowspan=1 colspan=1>0.03TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP05285/2021B.1.617.2</td><td rowspan=1 colspan=1>0.03TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/CA/VRLC009/2021| B.1.427</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/CA/VRLC012/2021| P.2</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP03056/2021 | B.1.525</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/CA-Stanford-15 S02/2021 | B.1.617.1</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Peru/un-CDC-2-4069945/2021 | C.37</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP20874/2021 | B.1.1.529</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/GA-EHC-2811C/2021 | B.1.1.529</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP30386/2022 |BA.4</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/COR-22-063113/2022 | BA.5</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>South Africa/CERI-KRISP-K040013/2022 | BA.5</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP38861/2022BQ.1.1</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP40900/2022XBB.1.5</td><td rowspan=1 colspan=1>0.101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP47865/2023XXB.2.3</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP46933/2023EG.1.2</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP47946/2023EG.5.1</td><td rowspan=1 colspan=1>0.03TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/CA-Stanford-139S35/2023XBB.1.9</td><td rowspan=1 colspan=1>0.101TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/CA-Stanford-139S23/2023XBB.1.16</td><td rowspan=1 colspan=1>0.101TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MI-UM-10052670540/2023BA.2.86</td><td rowspan=1 colspan=1>0.102TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/New York-PV96109/2023 |JN.1</td><td rowspan=1 colspan=1>0.151TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP49152/2023 | HV.13</td><td rowspan=1 colspan=1>0.015 TCID50/mL</td></tr></table>

\*Strain used to establish LoD. $^ { 1 } I n$ silico analysis showed $1 0 0 \%$ homology to amplification regions. $^ 2 I n$ silico analysis identified a single mismatch in the probe oligo for one region. Due to the location of the mismatch and $1 0 0 \%$ homology to the second region, detection is not expected to be impacted.

The results from this study demonstrate that the Aptima SARS-CoV-2 Assay is capable of detecting multiple clinically relevant strains of SARS-CoV-2.

# b. In silico

The inclusivity of the Aptima SARS-C0V-2 Assay was evaluated using in silico analysis of the forward, revers primers, and probes compared to available gene databases (GISAID and NCBI). All available sequences up to January 31, 2024 from the GISAID and NCBI gene databases were evaluated. Due to the high volume ${ \scriptscriptstyle ( > 1 4 }$ million) of SARS-CoV-2 sequences available in the NCBI and GISAID, a subset of at least $10 \%$ of available sequences up to July 31, 2023 and all sequences from August 1, 2023 – January 31, 2024 were analyzed from each database. To reduce sampling bias, each subset was chosen at random from each source. The sequences evaluated include lineages and variants of concern (VOC) or variants under investigation (VUI) that may have important epidemiological, immunological, or pathogenic properties from the public health perspective.

Sequence alignments were generated using the multiple sequence alignment program MAFFT (Multiple Alignment using Fast Fourier Transform). All non-human isolates as well as partial sequences that do not span the amplicon regions of the assay were removed. Additionally, any sequence with missing or ambiguous sequence information for the target region was removed. Due to the dual amplification systems for SARS-CoV2, only sequences with mismatches in both regions were assessed for potential impact to the inclusivity of the assay.

Bases on the in silico analysis of GISAID and NCBI sequences available for SARS-CoV2 up to January 31, 2024, the Aptima SARS-CoV-2 Assay is predicated to detect $9 9 . 9 8 \%$ (2,136,815/2,137,175) of all evaluated sequences.

# Cross-Reactivity/Microbial Interference

# a. Wet-Testing

This study evaluated the analytical specificity (cross-reactivity) and microbial interference for the Aptima SARS-CoV-2 Assay in the presence of closely related or commonly encountered organisms found in a respiratory tract specimen. Panels of 49 organisms were tested in processed negative clinical NP swab VTM/ UTM matrix. To evaluate cross-reactivity, each panel member was evaluated in triplicate in the absence of the target organism. To evaluate microbial interference, each panel was tested in triplicate in the presence of 3x LoD SARS-CoV-2. The organisms evaluated are shown below in Table 7. Bacteria were tested at $1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses were tested at $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , except where noted. No cross-reactivity or microbial interference was observed at the concentrations tested.

Table 7: Aptima SARS-CoV-2 Analytical Specificity and Microbial Interference Microorganisms   

<table><tr><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1">Concentration1</td><td colspan="1" rowspan="1">Bacteria/Fungi</td><td colspan="1" rowspan="1">Concentration1</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7a</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">CMV Strain AD 169</td><td colspan="1" rowspan="1">4.2x104 TCID50/mL</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">EBV</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU12</td><td colspan="1" rowspan="1">1x106 copies/mL</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">1x104 TCID50/mL</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus(hMPV)</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2)</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">4.8x106 TCID50/mL</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">1x104 TCID50/mL</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Pneumocystis jirovecii (PJP)</td><td colspan="1" rowspan="1">1x106 nuclei/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">1.7x107 TCID50/mL</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1x104 TCID50/mL</td><td colspan="1" rowspan="1">Staphylococcus epidermis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus2</td><td colspan="1" rowspan="1">1x106 copies/mL</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">1x104 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pooled human nasal wash3 to represent diversemicrobial flora in humanrespiratory tract</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Streptococcus salivaris</td><td colspan="1" rowspan="1">1x106 CFU/mL</td></tr></table>

1CFU $=$ Colony Forming Units; TCID5 $) =$ Median Tissue Culture Infectious Dose 2Cultured virus and whole genome purified nucleic acid for Human coronavirus HKU1 and SARS-coronavirus were not readily available at the time testing was performed. Human coronavirus HKU1 and SARS-coronavirus IVTs corresponding to the ORF1ab gene regions targeted by the assay were used to evaluate cross-reactivity and microbial interference. $^ 3 \mathrm { I n }$ place of evaluating pooled human nasal wash, testing of 30 individual negative clinical NP swab specimens was performed to represent diverse microbial flora in the human respiratory tract.

# Interfering Substances

This study evaluated the performance of the Aptima SARS-CoV-2 Assay in the presence of endogenous and exogenous substances that may be encountered in clinical specimens. The assay was evaluated with potentially interfering substances in the presence and absence of the target analyte. For analyte negative samples, potential interfering substances were added to pooled clinical negative NP swab specimens in VTM/UTM. For analyte positive samples, potentially interfering substances and SARS-CoV-2 at a concentration of 3x LoD was added to negative NP swab specimens in VTM/UTM. Three replicates were tested in the presence and absence of SARS-CoV-2 for each pool. The substances evaluated are listed in Table 8 below. All targetpositive panels were $100 \%$ positive and all target negative panels were $100 \%$ negative. The study results indicate that none of the evaluated substances interfered with the Aptima SARS-CoV-2 Assay at the concentrations tested.

Table 8: Interfering Substances Testing Panels   

<table><tr><td colspan="1" rowspan="1">PoolNumber</td><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="2">Endogenous</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">60 ug/mL</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">2% v/v</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="4">Nasal sprays ordrops</td><td colspan="1" rowspan="1">Neo-Synephrine</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">15% v/v</td></tr><tr><td colspan="1" rowspan="1">Anefrin</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15% v/v</td></tr><tr><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="1">Saline</td><td colspan="1" rowspan="1">Sodium chloride</td><td colspan="1" rowspan="1">15% v/v</td></tr><tr><td colspan="1" rowspan="1">Ventolin HFA</td><td colspan="1" rowspan="1">Albuterol</td><td colspan="1" rowspan="1">45 ng/mL</td></tr><tr><td colspan="1" rowspan="3">5</td><td colspan="1" rowspan="7">Nasalcorticosteroids</td><td colspan="1" rowspan="1">QVAR Beconase AQ</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">15 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Dexacort</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">12 ug/mL</td></tr><tr><td colspan="1" rowspan="1">Flonase</td><td colspan="1" rowspan="1">Fluticasone</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="1">Nasacort3</td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Rhinocort3</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="1">Nasonex</td><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">0.5 ng/mL</td></tr><tr><td colspan="1" rowspan="1">AEROSPAN</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">9.9 ug/mL</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Nasalgel/homeopathicallergy reliefmedicine</td><td colspan="1" rowspan="1">Zicam (AllergyRelief)</td><td colspan="1" rowspan="1">Luffa opperculata,Galphimia, Glauca,Histaminumhydrochloricum, sulfur</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Throat lozenges,oral anesthetic,and analgesic</td><td colspan="1" rowspan="1">Cepacol ExtraStrength</td><td colspan="1" rowspan="1">Benzocaine, Menthol</td><td colspan="1" rowspan="1">0.7 mg/mL</td></tr><tr><td colspan="1" rowspan="2">10</td><td colspan="1" rowspan="2">Anti-viral drugs</td><td colspan="1" rowspan="1">Relenza</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">TamiFlu</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">399 ng/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Virazole</td><td colspan="1" rowspan="1">Ribavirin</td><td colspan="1" rowspan="1">10.5 ug/mL</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Antibiotic, nasalointment</td><td colspan="1" rowspan="1">Bactroban cream</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">1.6 ug/mL</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Antibacterial,systemic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">33.1 ug/mL</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="2">Solvent Control</td><td colspan="1" rowspan="1">Water</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Dimetyl Sulfoxide(DMSO)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Ctrl</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Zing</td><td colspan="1" rowspan="1">Throat lozenges,oral anesthetic,and analgesic</td><td colspan="1" rowspan="1">Cold-Eeze throatlozenge</td><td colspan="1" rowspan="1">Zinc gluconate</td><td colspan="1" rowspan="1">0.7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Control(2)2}$</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

1 $\mathbf { v } / \mathbf { v } ^ { \prime }$ : volume by volume 2 Control condition tested with Zinc pool 3 Two out of three replicates for the pool containing Nasacort and Rhinocort were positive on initial testing. Substances were separated and retested and all replicates (3/3) repeated as positive.

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a. Controls The assay contains an internal control (IC-S) added to each test specimen and external positive and negative controls. For more information, see Section IV.C.5. Quality Control, above.

b. Sample Stability Sample stability studies have been performed and the data supports the following claims:

# Primary Specimens (NP swab and ANS storage in VTM/UTM)

Specimens can be stored in VTM/UTM (prior to transfer to the Panther Fusion Specimen Lysis Tube (SLT)) under the following conditions:

Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 96 hours before transfer to the Panther Fusion Specimen Lysis Tube or, • Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ . Samples may be freeze/thawed up to 3 times prior to testing.

# Processed Specimens (NP swab and ANS storage in a Panther Specimen Lysis Tube (SLT))

Once a patient specimen in VTM/UTM is transferred to the Panther Fusion SLT, samples can be stored under the following conditions:

Room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ for up to 6 days or, Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 3 months or, • Frozen at $- 2 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Freeze/thaw cycles should be minimized due to potential for sample degradation. Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Freeze/thaw cycles should be minimized due to potential for sample degradation.

# RespDirect Collection Kit (eSTM)

A NPS specimen collected with the RespDirect Collection Kit can be stored under the following conditions:

Room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ for up to 6 days or,   
Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 3 months or,   
Frozen at $- 2 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Samples may be freeze/thawed up to 3 times prior to testing or,   
Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Samples may be freeze/thawed up to 3 times prior to testing.

6. Detection Limit:

# Nasopharyngeal Swab Specimens in VTM/UTM

The analytical sensitivity (Limit of detection or LoD) of the Aptima SARS-CoV-2 Assay was determined by spiking heat inactivated SARS-CoV-2 (USA-WA1/2020; BEI Resources; NR-52281) into pooled negative NP swab matrix to create serial dilutions. To determine the preliminary LoD, each dilution was tested in replicates of ten with two reagent lots. The best performing concentration of the two lots was chosen as the preliminary LoD. The LoD was confirmed at $0 . 0 1 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ by testing at minimum 30 replicates. Confirmed LoD concentrations for heat inactivated SARS-CoV-2 are summarized in Table 9 below.

Additionally, the LoD of inactivated SARS-CoV-2, (NIBSC, 20/146) was determined. The LoD was determined by spiking into pooled negative clinical NP swabs in VTM/UTM matrix at multiple concentrations. Testing was conducted with three reagent lots, with a minimum of eight replicated tested per panel concentration per reagent lot for a minimum of 24 replicates. Probit analysis was then performed for each reagent lot. The final LoD was determined to be the highest value between the three reagent lots tested that produced $2 9 5 \%$ positivity. Confirmed LoD concentrations for inactivated SARS$\mathrm { C o V - } 2$ was determined to be $8 7 . 5 ~ \mathrm { I U / m L }$ .

Table 9: LoD Determination for SARS-CoV-2   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2 Strain</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Inactivated cultured SARS-CoV-2 virus(USA-WA1/2020</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Inactivated SARS-CoV-2, (NIBSC, 20/146)</td><td rowspan=1 colspan=1>87.5 IU/mL</td></tr></table>

# Nasopharyngeal Swab Specimens in RespDirect Collection Kit (eSTM)

This study evaluated the analytical sensitivity (LoD) of the inactivated SARS-CoV-2 (NIBSC, code 20/146) in the RespDirect collection kit containing eSTM. Positive and negative panels were prepared using pooled negative clinical NP swab matrix collected with the RespDirect Collection Kit. Positive panels were prepared by diluting inactivated SARS-CoV-2 to 1x LoD, 0.5 log below the LoD and 1 log below the LoD previously established with the VTM/UTM. A total of 24 replicates were tested for each panel. The established LoD from the Aptima SARS-CoV-2 Assay with VTM/UTM in STM was confirmed with $\ge 9 5 \%$ detection at LoD and $69 5 \%$ at $0 . 5 \log$ below the LoD. Results are presented in Table 10. The results demonstrate that the established LoD in UTM/VTM is equivalent in the RespDirect Collection Kits containing eSTM.

Table 10: LoD Comparison Between VTM/UTM and eSTM   

<table><tr><td rowspan=2 colspan=1>ReagentLot</td><td rowspan=2 colspan=1>Concentration</td><td rowspan=1 colspan=2>VTM/UTM</td><td rowspan=1 colspan=2>eSTM</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=3 colspan=1>ReagentLot 1</td><td rowspan=1 colspan=1>LoD (87.5 IU/mL)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100% (24/24)</td></tr><tr><td rowspan=1 colspan=1>0.5 Log Below LoD (27.7IU/mL)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>79% (19/24)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>71% (17/24)</td></tr><tr><td rowspan=1 colspan=1>1 Log Below LoD(8.75 IU/mL)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>17% (4/24)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29% (7/24)</td></tr><tr><td rowspan=3 colspan=1>ReagentLot 2</td><td rowspan=1 colspan=1>LoD (87.5 IU/mL)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>96% (23/24)</td></tr><tr><td rowspan=1 colspan=1>0.5 Log Below LoD (27.7IU/mL</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>46% (11/24)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>83% (20/24)</td></tr><tr><td rowspan=1 colspan=1>1 Log Below LoD(8.75 IU/mL)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29% (7/24)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>4% (1/24)</td></tr></table>

# 7. Assay Cut-Off:

The Aptima SARS-CoV-2 Assay determines the presence/absence of analyte and control by utilizing the Kinetic Differentiation Test (KDT). The analyte and internal control are assigned fluorescent probes with two different kinetic signatures (glower or flasher respectively). The signal output of these fluorescent probes is measured in Relative Light Units (RLU). RLU output over time generates a kinetic curve and the area under the curve represents the total signal (total RLU). To assign positive or negative status, KDT determines the kinetic curve type and then the total RLU is compared to predetermined RLU cutoffs.

The SARS-CoV-2 RLU cutoffs were determined by processing a range of known samples representing positive and negative samples. For the SARS-CoV-2 analyte, positivity was based on glower or dual kinetic (glower and flasher) interpretation combined with a minimum RLU value. The analyte positive cutoff was approached by ROC analysis, with the value refined based on mean and standard deviation from a known positive population. Validity of the negative sample was based on unambiguous kinetic activity associated with only one fluorescent probe (either glower or flasher) combined with a minimum RLU above background. If the kinetic interpretation or RLU do not meet these standards, the result is “Invalid”. The established SARS-CoV-2 RLU cutoffs were validated in the analytical and clinical studies described in this memo.

8. Accuracy (Instrument): Not applicable

# 9. Carry-Over:

The purpose of this study was to determine the carryover contamination rate of the Aptima SARS-CoV-2 Assay. The carryover contamination rate was established by testing high titer concentrations SARS-CoV-2 panels interspersed with negative panels in a checkerboard pattern. Hight titer panels were prepared by spiking inactivated SARS-CoV-2 virus at a high concentration into pooled negative clinical NP swab VTM/UTM processed into tubes containing STM. The SARS-CoV-2 negative panel consisted of negative STM. Testing was conducted on three Panther systems and one lot of Aptima SARS-CoV-2 Assay reagents, with one valid baseline run and two valid checkerboard runs on each system with.

Checkerboard layouts consisted of alternating positive and negative samples with 49 negative samples and 49 positive samples, for a total of 100 tests including controls. Testing was conducted over three days. All negative samples in all baseline runs tested negative for SARS-CoV-2 with a positive rate of $0 . 0 \%$ . All high titer positive samples tested positive for SARS-CoV-2 and all negative panels tested negative for SARS-CoV-2 indicating no carryover was observed.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable

2. Matrix Comparison:

a. Transport Media Equivalency Study

The purpose of this study was to demonstrate that equivalency between viral transport media (VTM) types when used with the Aptima SARS-CoV-2 Assay. The following media types were included in the study: Remel Micro Test M4RT, Remel Micro Test M5, Remel Micro Test M6, BD Universal Viral Transport Media, Copan Universal Transport Medium, and Hardy Diagnostics Viral Transport Media. To demonstrate equivalency between the viral transport medias, samples were created by spiking inactivated SARSCoV-2, (NIBSC, 20/146) at four concentrations (negative, 0.2x, 1.5x, and 5x LoD) into negative simulated VTM/UTM matrices (ex., Remel M4RT transport media spiked with HeLa cells at $2 \mathrm { x } 1 0 ^ { 4 }$ cells/mL) and negative clinical NP swabs in VTM/UTM. Negative and contrived positive samples generated with natural clinical NP matrix were included to demonstrate equivalency between clinical NP matrix and simulated NP matrix, supporting the use of simulated NP matrix in this study. Twenty (20) replicates per target concentration were tested for each transport media type and analyte concentration. The results are shown in Table 11 and support that the Aptima SARS-CoV-2 Assay can be used with the evaluated viral transport media types.

Table 11: Results Summary of Positivity from Matrix Equivalency   

<table><tr><td colspan="1" rowspan="1">Matrices Tested</td><td colspan="1" rowspan="1">Concentration*</td><td colspan="1" rowspan="1">% Positivity(n pos/n Tested)</td></tr><tr><td colspan="1" rowspan="4">Clinical NP (control)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">90% (18/20)</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="4">Remel M4RT</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">60% (12/20)</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="4">Remel M5</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">60% (12/20)</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Remel M6</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">60% (12/20)</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="4">BD UVT</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0%(0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">90%(18/20)</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">100%(20/20)</td></tr><tr><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100%(20/20)</td></tr><tr><td colspan="1" rowspan="4">Hardy VTM</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0%(0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">40%(8/20)</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">100%(20/20)</td></tr><tr><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100%(20/20)</td></tr><tr><td colspan="1" rowspan="4">Copan UTM</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">0.2x (FIO)</td><td colspan="1" rowspan="1">90% (18/20</td></tr><tr><td colspan="1" rowspan="1">1.5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">100% (20/20)</td></tr></table>

$^ { * } 0 . 2 \mathrm { x }$ LoD tested for information only (FIO) to ensure panels at $1 . 5 \mathrm { x }$ LoD and 5x LoD were not overspiked.

# b. Collection Device Equivalency – RespDirect Collection Kit (eSTM) vs VTM/UTM

The purpose of this study was to demonstrate equivalency of collection methods for NP swabs and ANS swabs. For this study, three contrived specimen types (NP swab collected with RespDirect Collection Kit (eSTM-NP), ANS swab collected with RespDirect Collection Kit (eSTM-NS) and NP swabs collected in VTM/SLT (NP swab collected in VTM/UTM and processed into SLT (VTM/SLT-NP)) were prepared by spiking inactivated SARS-CoV-2, (NIBSC, 20/146) into paired, negative clinical NP eSTM and negative clinical NP VTM/UTM matrix collected from individuals with signs and symptoms of respiratory infection. Contrived panels were generated at both $2 \mathbf { x }$ and 5x LoD. In total, 300 SARS-CoV-2 positives were evaluated. Additionally, 450 negative samples, consisting of negative clinical NP matrix, only, were included in the study. Five false positive results were observed (one VTM/SLT-NP and four eSTM-NP specimens). After retest, all results were negative. These false positive results are potentially due to environmental contamination. Results are shown in Table 12, below and demonstrate that the two collection methods are equivalent.

Table 12: Collection Method Equivalency Study Results   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>SARS-CoV-2Concentration</td><td rowspan=1 colspan=1>VTM/SLT-NP %Agreement (95%CII)</td><td rowspan=1 colspan=1>eSTM-NP %Agreement(95% CI)</td><td rowspan=1 colspan=1>eSTN-NS %Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (150/150)(97.5%-100.0%)</td><td rowspan=1 colspan=1>100% (150/150)(97.5%-100.0%)</td><td rowspan=1 colspan=1>100% (150/150)(97.5%-100.0%)</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>100% (50/50)(92.9%-100.0%)</td><td rowspan=1 colspan=1>100% (50/50)(92.9%-100.0%)</td><td rowspan=1 colspan=1>100% (50/50)(92.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (50/50)(92.9%-100.0%)</td><td rowspan=1 colspan=1>100% (50/50)(92.9%-100.0%)</td><td rowspan=1 colspan=1>100% (50/50)(92.9%-100.0%)</td></tr></table>

# C Clinical Studies:

1. Clinical Sensitivity: The clinical performance of the Aptima SARS-CoV-2 Assay was evaluated using prospective clinical anterior nasal swabs samples collected in UTM/VTM and eSTM in the

RespDirect Collection Kit and prospective clinical nasopharyngeal samples collected in UTM/VTM from individuals with signs and symptoms of respiratory viral infection.

Anterior nasal swab specimens were obtained either through healthcare provider (HCP) or patient self-collection (under HCP direction/observation) in UTM/VTM or eSTM from nine geographically diverse clinical collection sites. The Aptima SARS-CoV-2 Assay was evaluated for SARS-CoV-2 performance by comparing its results from anterior nasal swab specimens in UTM/VTM or in eSTM to a composite comparator algorithm (CCA). The composite comparator algorithm (CCA) consisted of two highly sensitive U.S. FDA EUA molecular tests and a validated PCR followed by bi-directional sequencing (PCR/BDS) assay. A final CCA result was assigned with two of the three composite comparator assays were in concordance. All comparator testing was performed in accordance with the respective package inserts. Subject enrollment was between October 25, 2022 and May 1, 2023. A total of 2295 non withdrawn subjects were tested in the study. Of the 2295 subjects tested, 118 subjects were not evaluable. Of the 118 subjects, 41 subjects had their sample withdrawn, 31 did not have a valid final Aptima SARS-CoV-2 Assay result, and 46 had an unknown CCA result. A total of 2177 subjects were evaluated for the performance analysis, including 1159 with evaluable ANS specimens in UTM/VTM, and 1018 with evaluable ANS specimens in RespDirect/eSTM. Demographic information for the 2177 evaluable individuals is provided in Table 13 below.

Remnant nasopharyngeal swab specimens were collected in UTM/VTM in a multicenter study, procured from clinical specimen supplies. All prospectively collected leftover NP swab samples were collected during two collection periods: June 2020 through July 2020 (prospectively collected frozen) and January 2023 through April 2023 (prospectively collected fresh). The Aptima SARS-CoV-2 Assay was evaluated for SARS-CoV-2 performance by comparing its results from NP swab specimens in UTM/VTM to a CCA. A total of 1646 NP swab specimens were tested in valid Aptima SARS-CoV-2 Assay runs, including 9 with initial invalid results. Upon retest, all 1646 specimens yielded final valid results. Of the 1646 samples obtained, 149 NP swab specimens were excluded from analysis due to mishandling at the site and two samples were not evaluable due to the unknown CC result. The final data set consisted of 1495 evaluable NP swab specimens including 1195 tested fresh and 300 tested after freezing. Demographic information for the 1495 evaluable individuals is provided in Table 14 below.

Table 13: Subject Demographics for Prospectively Collected Anterior Nasal Swab Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>Site 8</td><td rowspan=1 colspan=1>Site 9</td></tr><tr><td rowspan=3 colspan=1>Age (years)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>40.7</td><td rowspan=1 colspan=1>43.9</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>50.5</td><td rowspan=1 colspan=1>45.5</td><td rowspan=1 colspan=1>46.3</td><td rowspan=1 colspan=1>44.2</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>53.0</td><td rowspan=1 colspan=1>47.0</td><td rowspan=1 colspan=1>48.0</td><td rowspan=1 colspan=1>43.0</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>31.0</td></tr><tr><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>0-90</td><td rowspan=1 colspan=1>18-77</td><td rowspan=1 colspan=1>0-17</td><td rowspan=1 colspan=1>18-87</td><td rowspan=1 colspan=1>3-85</td><td rowspan=1 colspan=1>0-85</td><td rowspan=1 colspan=1>11-90</td><td rowspan=1 colspan=1>13-87</td><td rowspan=1 colspan=1>0-69</td><td rowspan=1 colspan=1>7-84</td></tr><tr><td rowspan=4 colspan=1>COVID-19VaccinationStatus</td><td rowspan=1 colspan=1>Fullyvaccinated</td><td rowspan=1 colspan=1>1451</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>372</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>196</td></tr><tr><td rowspan=1 colspan=1>Partiallyvaccinated</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Unvaccinated</td><td rowspan=1 colspan=1>601</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>2177</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>470</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>283</td></tr></table>

Table 14: Summary of Subject Demographics for Evaluable Prospectively Collected Nasopharyngeal Swab Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td></tr><tr><td rowspan=7 colspan=1>Age(years)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>40.7</td><td rowspan=1 colspan=1>18.2</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>48.5</td><td rowspan=1 colspan=1>38.3</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>16.0</td><td rowspan=1 colspan=1>53.0</td><td rowspan=1 colspan=1>38.0</td></tr><tr><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>0  90</td><td rowspan=1 colspan=1>0-98</td><td rowspan=1 colspan=1>0-93</td><td rowspan=1 colspan=1>1-89</td><td rowspan=1 colspan=1>0-91</td></tr><tr><td rowspan=1 colspan=1>&lt;5</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>5-21</td><td rowspan=1 colspan=1>373</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>353</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>1495</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>455</td><td rowspan=1 colspan=1>466</td><td rowspan=1 colspan=1>232</td></tr></table>

Positive Percent Agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathrm { T P / \left( T P + F N \right) } \right)$ . True positive (TP) indicates that both the Aptima SARS-CoV-2 Assay and the CCA had a positive result for SARS-CoV-2, and false negative (FN) indicates that the Aptima SARS-CoV-2 Assay result was negative while the CCA was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathrm { T N / \left( T N + F P \right) } \right)$ . True negative (TN) indicates that both the Aptima SARS-CoV-2 Assay and the CCA had negative results, and false positive (FP) indicates that the Aptima SARS-CoV-2 Assay result was positive while the CCA was negative.

The Aptima SARS-CoV-2 Assay clinical performance data for ANS and NPS specimens, expressed as positive percent and negative percent agreements against the CCA are presented by in Table 15 and 16 below.

Table 15: Aptima SARS-CoV-2 Assay Performance for Anterior Nasal Swab Specimens   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>UTM/VTM</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>138/143</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>92.1-98.5</td><td rowspan=1 colspan=1>992/1016</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>96.5-98.4</td></tr><tr><td rowspan=1 colspan=1>RespDirect</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>892</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.6-100.0</td><td rowspan=1 colspan=1>892/910</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>96.9-98.7</td></tr></table>

Table 16: Aptima SARS-CoV-2 Assay Performance for Nasopharyngeal Swabs Specimens in UTM/VTM   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1107</td><td rowspan=1 colspan=1>80/82</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>91.5-99.3</td><td rowspan=1 colspan=1>1107/1113</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>98.8-99.8</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>251</td><td rowspan=1 colspan=1>44/48</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>80.4-96.7</td><td rowspan=1 colspan=1>251/252</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.8-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1358</td><td rowspan=1 colspan=1>124/130</td><td rowspan=1 colspan=1>95.4</td><td rowspan=1 colspan=1>90.3-97.9</td><td rowspan=1 colspan=1>1358/1365</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>98.9-99.8</td></tr></table>

2. Clinical Specificity: See Section “Clinical Sensitivity” above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

D Clinical Cut-Off: Not applicable.

# E Expected Values/Reference Range:

The Aptima SARS-CoV-2 Assay prospective clinical study included a total of 2241 prospectively collected ANS specimens and a total of 1646 NPS specimens, of which 2177 ANS specimens and 1495 NPS specimens were evaluable for SARS-CoV-2. The number and percentage of cases positive for SARS-CoV-2 for ANS and NPS specimens as determined by the Aptima SARS-CoV-2 Assay are presented below in Table 17 and Table 18.

Table 17: Aptima SARS-CoV-2 Assay Expected Values for Anterior Nasal Swab Specimens by Collection Site   

<table><tr><td rowspan=1 colspan=1>Collection Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expected Value</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>2.2% (2/91)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>9.0% (9/100)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>219</td><td rowspan=1 colspan=1>3.2% (7/219)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>470</td><td rowspan=1 colspan=1>16.0% (75/470)</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>16.9% (85/504)</td></tr><tr><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>10.1% (7/69)</td></tr><tr><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>22.4% (38/170)</td></tr><tr><td rowspan=1 colspan=1>Site 8</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>5.9% (16/271)</td></tr><tr><td rowspan=1 colspan=1>Site 9</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>17.3% (49/283)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2177</td><td rowspan=1 colspan=1>13.2% (288/2177)</td></tr></table>

Table 18: Aptima SARS-CoV-2 Assay Expected Values for Nasopharyngeal Swab Specimens by Collection Site and Collection Periods   

<table><tr><td rowspan=2 colspan=1>Collection Site</td><td rowspan=1 colspan=2>Collection Period06/2020 to 07/2020</td><td rowspan=1 colspan=2>Collection Period01/2023 to 04/2023</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expected Value</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expected Value</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>3.8% (13/342)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>455</td><td rowspan=1 colspan=1>8.1% (37/455)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>7.4% (5/68)</td><td rowspan=1 colspan=1>398</td><td rowspan=1 colspan=1>9.0% (36/398)</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>17.2% (40/232)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>15.0% (45/300)</td><td rowspan=1 colspan=1>1195</td><td rowspan=1 colspan=1>7.2% (86/1195)</td></tr></table>

F Other Supportive Instrument Performance Characteristics Data: Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.