# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K060654   
B. Purpose for Submission: To obtain clearance of a new device   
C. Measurand: Factor II and Factor V   
D. Type of Test: Qualitative   
E. Applicant: AutoGenomics, Inc.   
F. Proprietary and Established Names: INFINITI™ System

# G. Regulatory Information:

1. Regulation section: 864.7280

2. Classification: Class II

3. Product code: NPR (factor II) & NPQ (factor V)

4. Panel:

81 Hematology

# H. Intended Use:

1. Intended use(s):

The INFINITI™ System Assay for Factor II and Factor V is an in vitro diagnostic device that consists of reagents and instrumentation which includes polymerase chain reaction (PCR) primers, hybridization matrices, a thermal cycler, an imager, and software for detection and genotyping of Factor II (Prothrombin) G20210A and Factor V Leiden G1691A point mutations in DNA obtained from human blood samples. The INFINITI™ System Assay for Factor II and Factor V is a qualitative assay for use in clinical laboratories upon prescription by the attending physician.

2. Indication(s) for use:

The INFINITI™ System Assay for detection and genotyping of Factor II (Prothrombin) G20210A and Factor V G1691A mutations is intended to be used as an aid to diagnosis in the evaluation of patients with suspected thrombophilia.

3. Special conditions for use statement(s):

4. Special instrument requirements:

The test is intended to be used on the INFINITI Analyzer.

# I. Device Description:

The INFINITI™ System is an in vitro diagnostic device which utilizes proprietary filmbased microarray technology combined with process automation, reagent management and software technology for the detection and genotyping of the Factor II (Prothrombin)m G20210A mutation and the Factor V Leiden mutation from deoxyribonucleic acid (DNA) isolated from human whole peripheral blood samples. The INFINITI System is comprised of four components:

BioFilmChip™ Microarray Intellipac™ Reagent Module • INFINITI Analyzer Qmatic™ Operating Software

The BioFilmChip Microarray is a film-based microarray consists of multiple layers of porous hydrogel matrix ( $8 { \cdot } 1 0 { \mathrm { u m } }$ in thickness) coated on a polyester solid support. An intermediate layer incorporates a proprietary material to reduce intrinsic fluorescence. The top layer is designed for the immobilization of biomolecules such as oligonucleotides to enable the genomic analysis on the same platform. There can be up to 240 spots per microarray with each spot representing a different allele. The microarrays are designed to be assay specific.

The Intellipac Reagent Module which acts as a communication link contains eight reservoirs that house the test reagents and has an integrated 64K bit memory chip. The assay protocol resides in this memory chip and upon request is loaded to the INFINITI Analyzer.

The INFINITI Analyzer is an instrument used for clinical multiplex systems intended to measure and sort multiple signals from a clinical sample. The INFINITI Analyzer is designed to measure fluorescence signals of labeled DNA target hybridized to BioFilmChip microarrays. The INFINITI Analyzer is completely self-contained and automates the Factor II and Factor V assays. The INFINITI Analyzer integrates all the discrete processes of sample handling, reagent management, hybridization, detection and result analysis. The assays are processed automatically and read by the built-in microscope.

The Qmatic operating software is a proprietary software module built into the INFINITI Analyzer and manages the entire INFINITI System by integrating multiple processes such as the multiplex assay protocol for Factor II and Factor V, fluid handling, robotics, optical detection and results analysis.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Roche Factor V Leiden Kit (LightCycler $\textsuperscript { \textregistered }$ Instrument) Roche Factor II (Prothrombin) G20210A (LightCycler $\textsuperscript { \textregistered }$ Instrument)

2. Predicate 510(k) number(s): K033612 & K033607

3. Comparison with predicate:

<table><tr><td colspan="4" rowspan="1">Similarities (Assay)</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">INFINITITM System</td><td colspan="1" rowspan="1">Roche Factor VLeiden Kit</td><td colspan="1" rowspan="1">Roche Factor IILeiden Kit</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Purified DNA from humanblood samples</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Oligonucleotide probesand primers</td><td colspan="1" rowspan="1">Specific for Factor V Leidenand Factor II (prothrombin)G20210A</td><td colspan="1" rowspan="1">Specific for Factor VLeiden</td><td colspan="1" rowspan="1">Specific for Factor II(prothrombin)G20210A</td></tr><tr><td colspan="1" rowspan="1">Sample preparation</td><td colspan="1" rowspan="1">Performed off-line</td><td colspan="1" rowspan="1">Performed off-line</td><td colspan="1" rowspan="1">Performed off-line</td></tr><tr><td colspan="1" rowspan="1">Sample size</td><td colspan="1" rowspan="1">10-20 μL</td><td colspan="1" rowspan="1">10-20 μL</td><td colspan="1" rowspan="1">10-20 μL</td></tr><tr><td colspan="4" rowspan="1">Differences (Assay)</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">INFINITITM System</td><td colspan="1" rowspan="1">Roche Factor VLeiden Kit</td><td colspan="1" rowspan="1">Roche Factor IILeiden Kit</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">1 ng DNA/test (analytical)</td><td colspan="1" rowspan="1">50 allelecopies/reaction</td><td colspan="1" rowspan="1">50 allelecopies/reaction</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Film-based microarraytechnology combined withprocess automation, reagentmanagement and softwaretechnology</td><td colspan="1" rowspan="1">Real time PCR</td><td colspan="1" rowspan="1">Real time PCR</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Similarities (Instrumentation)</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=2>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>INFINITITM System</td><td rowspan=1 colspan=1>LightCycler Instrument</td><td rowspan=1 colspan=1>Affymetrix GeneChipMicroarrayInstrumentationSystem</td></tr><tr><td rowspan=1 colspan=1>Detection procedure</td><td rowspan=1 colspan=1>Optical detection ofsimulated fluorescence</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Optical detection ofsimulated fluorescence</td></tr><tr><td rowspan=1 colspan=1>Sample preparation</td><td rowspan=1 colspan=1>Performed off-line</td><td rowspan=1 colspan=1>Performed off-line</td><td rowspan=1 colspan=1>Performed off-line</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Differences (Instrumentation)</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=2>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>INFINITITM System</td><td rowspan=1 colspan=1>LightCycler Instrument</td><td rowspan=1 colspan=1>Affymetrix GeneChipMicroarrayInstrumentationSystem</td></tr><tr><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>The INFINITI Analyzeris a multiplex systemintended to measurefluorescence signals oflabeled detection primershybridized tomicroarrays.</td><td rowspan=1 colspan=1>The LightCycler is a fullyautomated amplificationand detection system fornucleic acids usingfluorescence detection.</td><td rowspan=1 colspan=1>The AffymetrixGeneChip MicroarrayInstrumentation Systemis a multiplex systemintended to measurefluorescence signals oflabeled DNA targethybridized to GeneChiparrays.</td></tr><tr><td rowspan=1 colspan=1>Optical Detection</td><td rowspan=1 colspan=1>One fixed wavelength(650 nm)</td><td rowspan=1 colspan=1>Three fixed wavelengths(530 nm, 640 nm, 710nm)</td><td rowspan=1 colspan=1>Two fixed wavelengths(560 nm, 650 nm)</td></tr><tr><td rowspan=1 colspan=1>Detection Chemistry</td><td rowspan=1 colspan=1>Direct fluorescence</td><td rowspan=1 colspan=1>Paired hybridizationprobes using fluorescenceenergy transfer (FRET)</td><td rowspan=1 colspan=1>Direct fluorescence</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

FDA guidance: “Class II Special Controls Guidance Document: Factor V Leiden DNA Mutation Detection Systems”

# L. Test Principle:

The INFINITI System assay for Factor II and Factor V is designed to simultaneously detect mutations of two genes: Factor II (G20210A) and Factor V (G1691A). The assay protocol is based on five major processes:

a) PCR amplification of purified DNA from human genomic DNA   
b) Labeling of the amplified product (allele specific primer extension)   
c) Hybridization of the labeled amplified product to a microarray by signature Tag/Capture interaction iso conditions   
d) Scanning of the microarray   
e) Signal detection and analysis (determination of the Factor II and V genotypes_

Steps (b) through (e) are automated by the INFINITI Analyzer.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Determined at three clinical sites, with three different instruments, three days and three different operators. Five replicates, each, of two genomic DNA samples were tested by each site. Correct calls (first time) were $93 \%$ ; after repeat run, $100 \%$ of calls were correct.

# Assay:

(Within chip): Using the same sample and three different instruments, one heterozygous sample was repeated five scans on the same chip. Also within chip $\%$ CVs, using three spots per mutation, were determined from three chips from one lot and ranged from $0 . 6 - 3 9 . 1 \%$ . Each chip was read five times using one instrument. The average of triplicate spots was provided for FV-W, FV-M, FII-W and FII-M.

(Chip-to-chip): Using the same sample and the same instrument, the assay was run using three microarray chips from one lot of BioFilmChip Microarray five times testing a known genomic sample (Factor II Wild, Factor V Wild). This was repeated two other times, using a different instrument. The CVs using average triplicate spots for each mutation ranged from $9 - 1 2 \%$ for wild-type present calls. All calls were $100 \%$ correct.

(Lot-to-lot): Five replicates, each, of two genomic DNA samples were sent to three different clinical sites (three different instruments and operators) and analyzed. Two lots of microarray chips and two lots of Intellipac reagents were used by the study

sites.

Because lot-to-lot comparison at the study sites involved only two lots of Intellipac reagents and two lots of microarray chips, data from in-house testing was provided. In-house tests were conducted with three lots of BioFilmChip microassays using the same instrument four times, each time using a different sample. Two-way ANOVA on the RFU reading did not detect lot-to-lot difference on three of the four test runs (p $> 0 . 0 5 )$ , and detected lot-to-lot difference on one test run $( 0 . 0 5 > { \tt p } > 0 . 0 1 )$ ). Genotype calls were $100 \%$ correct.

Run 1: FV – Heterozygote; FII – Heterozygote Run 2: FV – Heterozygote; FII – Wild Type Run 3: FV – Wild Type; FII – Heterozygote Run 4: FV – Wild Type; FII – Wild Type

(Day-to-day): To provide a comparison between days, a three-day reproducibility study was performed in-house. A known genomic sample (Factor II Mutant and Factor V Wild) was assayed 12 times on each of three days using one instrument. The RFU signal $\% \mathrm { C V }$ ranged from 1.35 to 14.87 on day 1, 0.77 to 19.72 on day 2 and 0.41 to 21.2 on day 3. Genotype calls were $100 \%$ correct.

# (Site-to-site):

To assess site-to-site reproducibility, three clinical sites representing three different instruments, three different days and three different operators. The same two known genomic DNA samples were tested by the three sites. The genotypes include (Factor V Wild and Factor II Homozygous Mutant – same five replicates) and (Factor V Wild and Factor II Wild - same five replicates). All genotype calls were correct. Results of these studies demonstrate that the INFINITI System Assay for Factor II & Factor V detects the target mutations in the specified sample size/level.

$\% C V s$ for calls present by site:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FV - WT / FII − WT</td><td rowspan=1 colspan=1>FV - WT / FII - Homozygous</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>52.5 - 55</td><td rowspan=1 colspan=1>21.7 - 24.6</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>16.6 - 36.8</td><td rowspan=1 colspan=1>7.1 - 11.7</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>19.0 - 37.7</td><td rowspan=1 colspan=1>7.9 - 27.2</td></tr></table>

The following Table provides a summary of the mutations detected during these studies.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Factor II</td><td colspan="3" rowspan="1">Factor V</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WildType</td><td colspan="1" rowspan="1">Mutant</td><td colspan="1" rowspan="1">Heterozygous</td><td colspan="1" rowspan="1">WildType</td><td colspan="1" rowspan="1">Mutant</td><td colspan="1" rowspan="1">Heterozygous</td></tr><tr><td colspan="1" rowspan="1">ARUP</td><td colspan="1" rowspan="1">29+5*</td><td colspan="1" rowspan="1">7+5*</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">29</td></tr><tr><td colspan="1" rowspan="1">UCLA</td><td colspan="1" rowspan="1">48 + 5*</td><td colspan="1" rowspan="1">5*</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Long Beach</td><td colspan="1" rowspan="1">52 +5*</td><td colspan="1" rowspan="1">5*</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Samplecarry-over</td><td colspan="1" rowspan="1">6*</td><td colspan="1" rowspan="1">6*</td><td colspan="1" rowspan="1">6*</td><td colspan="1" rowspan="1">6*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6*</td></tr></table>

\* known genomic samples

# Instrument:

One DNA sample (Factor II Wild, Factor C Wild) was analyzed using three different INFINITI Analyzers and one lot of BioFilmChips, five times (five runs).

In addition, data on three instruments using a Standard Microarray Chip was provided. Three instruments were tested on three different days. For each instrument tested, each capture probe spot on the standard microchip was read 24 times, then averaged, and a $\%$ CV calculated for that spot. The following lists the ranges for the $\%$ CVs for the three instruments tested.

Reproducibility (Instrument)   

<table><tr><td rowspan=1 colspan=1>Instrument S/N</td><td rowspan=1 colspan=1>Date</td><td rowspan=1 colspan=1>Average %CV</td><td rowspan=1 colspan=1>% CV Range</td></tr><tr><td rowspan=1 colspan=1>013006</td><td rowspan=1 colspan=1>08-11-06</td><td rowspan=1 colspan=1>4.03%</td><td rowspan=1 colspan=1>1.9-7.5</td></tr><tr><td rowspan=1 colspan=1>032106</td><td rowspan=1 colspan=1>08-01-06</td><td rowspan=1 colspan=1>3.99%</td><td rowspan=1 colspan=1>2.7-6.5</td></tr><tr><td rowspan=1 colspan=1>080206</td><td rowspan=1 colspan=1>08-02-06</td><td rowspan=1 colspan=1>3.24%</td><td rowspan=1 colspan=1>1.9-5.3</td></tr></table>

Intra-instrument Reproducibility: The $\%$ CV using a single chip five times on a single instrument ranged from $0 . 9 \% - 2 8 . 3 \%$ . Genotype calls were $100 \%$ reproduced within each instrument.

Inter-instrument Reproducibility: Based on two-way ANOVA for the Factor II and Factor V WT/MUT ratios, there was no significant different detected in 4 of the 5 runs $( \mathtt { p } < 0 . 0 1 )$ . The inter-instrument variability was determined as $0 . 5 \% - 1 2 \% \ C$ V. All genotype calls were correct and reproducible.

An additional inter-instrument reproducibility study was performed with three different instruments, three different chips, over three days using a heterozygous sample for both FII and FV. All genotype calls were correct.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Positive or negative DNA assay controls are not included as part of the assay. The manufacturer recommends that a positive sample (heterozygous for each mutation), a

negative sample, and a “No Template Control” be run with each assay.

Internal Controls: Five negative and three positive control spots are pre-immobilized on each microarray chip. These are procedural hybridization controls. The negative control is a $3 ^ { \circ }$ -biotynlated 24mer to which nothing should bind. The positive control is a $3 ^ { \circ }$ -biotynlated 24mer that hybridizes the complementary Cy5 labeled oligomer supplied in hybridization solution in the Intellipac Reagent Module. The negative control spot values are also used in part of the algorithm calculations. Three Registration (immobilized dyes) spots are also incorporated on each microarray and used to ensure proper alignment/position during the optical reading process.

# d. Detection limit:

The package insert recommends $2 5 { \mathrm { n g } } /$ reaction. A serial dilution (300 – 0.005 ng/reaction) of a genomic sample demonstrated reproducible results down to 1 ng/reaction.

# e. Analytical specificity:

Studies related to specificity were conducted during assay development. PCR primer specificity was determined by amplicon size on a gel and sequencing the amplicon. ASP primer specificity was determined by the correct calls made by the assay using known genomic samples. Capture probe specificity was determined by hybridizing different oligos and demonstrating that correct oligo hybridizes to the known spot.

Interference studies: Interference studies were conducted for bilirubin, cholesterol, and heparin. One blood sample was divided into 7 test samples $( 3 0 0 ~ \mu \mathrm { l }$ each). Each sample was spiked with cholesterol (70 and $7 \mathrm { m g / d l }$ ) bilirubin (8 and $0 . 8 \mathrm { m g / d l }$ ) and heparin (1333 and $1 3 3 \mathrm { U } / \mathrm { d l }$ ) and compared to an unspiked control sample. DNA was extracted and each sample was tested three times. No interference was observed.

No studies were conducted with oral-anticoagulants, therefore, no claims were made.

# f. Assay cut-off:

Ratio cut-off for a homozygous call requires that one analyte of the pair be at least 4x the other. If the Wild Type signal is greater than the mutant signal, the call is Wild Type. If the mutant signal is greater than the Wild Type signal, the call is Homozygous Mutant. The Indeterminate zone is a ratio between 2 and 4. A ratio $\leq 2$ is Heterozygous. A ratio $\geq 4$ is Homozygous.

The algorithm for determining the genotype call and cut-off values were empirically established using known samples and following certain criteria, adjusting for the background, and taking into consideration the relative efficiencies of the PCR amplification and variation in ASP for the different sequences.

The following describes the algorithm for determining the genotype call for the INFINITI System Assay for Factor II & Factor V.

• First, qualify the signals:

o Average the three spots (RFU) for each Analyte and average the five spots (RFU) for the Negative control.   
o Subtract the average Negative signal from the average Analyte signal. If the adjusted Analyte signal is less than 1.0, assign the nominal value 1.0.   
o The adjusted analyte signal should be at least two times the negative signal for at least one of the signals in the pair, otherwise, the call is “Indeterminate”.

• After signals are qualified, for Factor II, multiply the adjusted signals by the “Multiplier” (2 for Wild and 0.75 for Mutant). Scaling using the “Multiplier” is done to balance inequality of the reactions that result in signal generation. The Multiplier is set such that in a heterozygous sample the signals of the Wild and Mutant forms are equal.

For Factor V, the “Multiplier” is 1.0 (no scaling).

• To make the call, the ratio cutoffs are set such that a homozygous call requires that one analyte of the pair to be at least $4 \mathbf { x }$ the other, and a heterozygous call requires that one analyte of the pair to be at most $2 \mathbf { x }$ the other, i.e.,

$\bigcirc$ Determine the ratio of the high signal (Wild or Mutant) to the low signal (Wild or Mutant).   
$\bigcirc$ If the ratio is $\leq 2$ , the call is “Heterozygous”   
$\bigcirc$ If the ratio is $\geq 4$ , the call is “Homozygous”.   
$\gtrdot$ If the Wild signal is greater than the Mutant signal, the call is “Homozygous Wild”   
¾ If the Mutant signal is greater than the Wild signal, the call is “Homozygous Mutant”

o If the ratio is between 2 and 4, the call is “Indeterminate”.

As described above, a call is made based on a qualified signal. Therefore, inaccurate calls are rare. This demonstrates the robustness of the algorithm for making the calls.

An “indeterminate” call is made if any of the following occurs:

(a) The analyte does not have a qualified signal in the wild or mutant forms of the analyte. To be qualified the adjusted signal (after subtracting the negative/background signal) must be greater than $2 \mathbf { x }$ the negative/background signal.   
(b) The cut-off ratio [high signal (wild or mutant) to the low signal (wild or mutant)] is between 2 and 4.

2. Comparison studies:

# a. Method comparison with predicate device:

Method comparison (clinical) studies were conducted at three clinical sites. All three sites compared the INFINITI System Assay for Factor II and Factor V to the 510(k) cleared Roche Factor II and Factor V Kits. Subjects included in the comparison studies were suspected thombophilic patients. In addition, identical sets of known genomic samples were tested at each site, (five samples of known Factor II G20210A mutation sample and five samples from a known Wild Type sample.

For Factor II, overall agreement between the two methods was $9 8 . 6 \%$ for a total of 208 samples. For Factor V, overall agreement between the two methods was $100 \%$ for a total of 175 samples.

The results of the comparison studies are summarized below.

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>NumberTested</td><td rowspan=1 colspan=1>Number ofCorrectCalls onFirst Run</td><td rowspan=1 colspan=1>Number ofInvalidCalls* onFirst Run</td><td rowspan=1 colspan=1>AgreementFirst Run</td><td rowspan=1 colspan=1>Number ofCorrectCallsincludingRepeat Run</td><td rowspan=1 colspan=1>Number ofInvalidCalls* onRepeatRun</td><td rowspan=1 colspan=1>AgreementAfterRepeatRun</td></tr><tr><td rowspan=1 colspan=8>Factor II G20210A</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>146**</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>86.3%</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.6%</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>22**</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>208</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>89.9%</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.6%</td></tr><tr><td rowspan=1 colspan=8>Factor V G1691A</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>89.4%</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>92.0%</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

\* No discordant results. Invalid results refer to “indeterminate” results.

\*\*Samples tested were taken from suspected thrombophilia patients except:

15 of the 146 WT samples (negative for Factor II G20210A mutation) were from a known genomic WT sample which was split into 15 individual samples.   
15 of 22 Factor II G20210A mutation samples were from a known genomic sample which was split into 15 individual samples.

Results by site:

# Site 1:

Study Samples: Fifty-seven (57) study samples were obtained from 60 patients.   
Peripheral blood was drawn into tubes containing EDTA as an anticoagulant.

To test for the Factor II G20210A mutation, ten (10) known genomic DNA samples (5

Factor G20210A mutation samples and 5 Wild Type for Factor II G20210A mutation) were added to the comparison study.

Data and Results: Summary of results is provided in the table below followed by an Excel-based spreadsheet of the data for the site.

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>NumberTested</td><td rowspan=1 colspan=1>Number ofCorrectCalls onFirst Run</td><td rowspan=1 colspan=1>Number ofInvalidCalls* onFirst Run</td><td rowspan=1 colspan=1>AgreementFirst Run</td><td rowspan=1 colspan=1>Number ofCorrectCallsincludingRepeat Run</td><td rowspan=1 colspan=1>Number ofInvalidCalls* onRepeat Run</td><td rowspan=1 colspan=1>AgreementAfter RepeatRun</td></tr><tr><td rowspan=1 colspan=8>Factor II</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>53**</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>75.5%</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>5***</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>80.6%</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Factor V</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>68.4%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>75.9%</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

$^ { * } \mathrm { { N o } }$ discordant results; invalid calls refer to indeterminate results.

\*\* 5 known genomic WT Samples \*\*\* 5 known genomic Factor II Samples

# Site 2:

Study Samples: Fifty-four (54) study samples were obtained from hospital inpatients who had physician orders for Factor II G20210A and Factor V Leiden mutations. Whole blood was collected by routine phlebotomy into BD Vacutainer tubes containing EDTA as an anticoagulant.

To test for the Factor II G20210A mutation, ten (10) known genomic DNA samples (5 Factor G20210A mutation samples and 5 Wild Type for Factor II G20210A mutation) were added to the comparison study.

Data and Results: Summary of results is provided in the table below followed by an Excel-based spreadsheet of the data for the site.

<table><tr><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">NumberTested</td><td colspan="1" rowspan="1">Number ofCorrect Callson First Run</td><td colspan="1" rowspan="1">Number ofInvalidCalls* onFirst Run</td><td colspan="1" rowspan="1">AgreementFirst Run</td><td colspan="1" rowspan="1">Number ofCorrect CallsincludingRepeat Run</td><td colspan="1" rowspan="1">Number ofInvalidCalls* onRepeat Run</td><td colspan="1" rowspan="1">AgreementAfter RepeatRun</td></tr><tr><td colspan="8" rowspan="1">Factor II</td></tr><tr><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">57**</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">94.7%</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">5***</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">95.2%</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td></tr><tr><td></td><td></td><td></td><td></td><td colspan="4" rowspan="1">Factor V</td></tr><tr><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.1%</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td></tr></table>

$^ { * } \mathrm { { N o } }$ discordant results; invalid calls refer to indeterminate results.

\*\* 5 known genomic WT Samples \*\*\* 5 known genomic Factor II Samples

# Site 3:

Study Samples: A total of 69 samples were analyzed. These samples were de-identified DNA extracted from samples submitted to ARUP Laboratories for Factor II and Factor V analysis. Each sample was assayed using the INFINITI System Assay for Factor II & Factor V, and using the Roche ThermaCycler Factor II and Factor V Assays.

To test for the Factor II G20210A mutation, ten (10) known genomic DNA samples (5 Factor G20210A mutation samples and 5 Wild Type for Factor II G20210A mutation) were added to the comparison study.

Data and Results: Summary of results is provided in the table below followed by an Excel-based spreadsheet of the data for the site.

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>NumberTested</td><td rowspan=1 colspan=1>Number ofCorrect Callson First Run</td><td rowspan=1 colspan=1>Number ofInvalidCalls* onFirst Run</td><td rowspan=1 colspan=1>AgreementFirst Run</td><td rowspan=1 colspan=1>Number ofCorrect CallsincludingRepeat Run</td><td rowspan=1 colspan=1>Number ofInvalidCalls* onRepeat Run</td><td rowspan=1 colspan=1>AgreementAfter RepeatRun</td></tr><tr><td></td><td></td><td></td><td></td><td rowspan=1 colspan=4>Factor II</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>36**</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>94.4%</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>12***</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>93.6%</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.2%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Factor V</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100%</td></tr></table>

$^ { * } \mathrm { { N o } }$ discordant results; invalid calls refer to indeterminate results.

\*\* 5 known genomic WT Samples \*\*\* 5 known genomic Factor II Samples b. Matrix comparison:

Not applicable. This test is for use only with human peripheral whole blood collected using EDTA as the anticoagulant.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.   
b. Clinical specificity: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

The Factor V Leiden G1691A mutation is the most common mutation associated with inherited thrombosis and results in resistance to activated protein C. It has a relatively high prevalence in the general population (about $5 \%$ in Caucasians), and accounts for $8 5 \%$ to $9 5 \%$ of activated protein C resistant cases.

The G20210A mutation in the prothrombin (Factor II) gene is the second most common mutation associated with hereditary thrombosis. It is associated with increased plasma prothrombin levels and is present in $1 \%$ to $2 \%$ of the general population. Homozygote carriers are very rare.

N. Instrument Name: INFINITI Analyzer

# O. System Descriptions:

1. Modes of Operation: Simultaneous random and continuous access

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No

3. Specimen Identification: Operator-entered information as part of worklist sample ID

4. Specimen Sampling and Handling:

A unique identification, comprised of operator-entered information plus date and time, is attached to each worklist sample ID in order to distinguish one run from another. Once the worklist is submitted, all pipetting, sample identification, hybridization, incubation, drying, scanning and result interpretation can be completed without manual intervention.

# 5. Calibration:

The INFINITI Analyzer does not require routine calibration. The normalizing factor for the laser is calculated by the instrument using the Standard Microarray Chip. The Standard Microarray Chip is a non-assay array used to calculate a multiplier used to normalize all readings by the INFINITI Analyzer. The Standard Microarray Chip is for use at AutoGenomics during instrument calibration and by AutoGenomics Customer Service to check on instruments that are in the field.

The user does not have the capability to re-calibrate the instrument. Only AutoGenomics Customer Service is authorized to perform this task. An error message is displayed when the signal levels are off by $> 2 0 \%$ .

# 6. Quality Control:

It is recommended that a positive sample (heterozygous and/or homozygous for both genotypes), from a cell line or patient sample, for each mutation; a negative control (a sample that does not contain the mutation of interest, i.e., a wild type sample); and a “No Template Control” (Molecular Grade water) be included with each test run. The user should contact AutoGenomics for recommendations.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.