# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k100496   
B. Purpose for Submission: New device   
C. Measurand: Human 5,10-methylenetetrahydofolate reductase (MTHFR)   
D. Type of Test: Qualitative genotyping test for single nucleotide polymorphism detection   
E. Applicant: Hologic Inc. Third Wave Technologies   
F. Proprietary and Established Names: Invader® MTHFR 1298   
G. Regulatory Information: 1. Regulation section: 21 CFR §864.7280 Factor V Leiden DNA mutation detection systems 2. Classification: Class II 3. Product code: OMM: Test 5,10-Methylenetetrahydrofolate Reductase Mutations, Genomic DNA PCR 4. Panel: 81 Hematology

# H. Intended Use:

1. Intended use(s): The Invader $\textsuperscript { \textregistered }$ MTHFR 1298 test is an in vitro diagnostic test intended for the detection and genotyping of a single point mutation (A to C at position 1298) of the human 5,10-methylenetetrahydrofolate reductase (MTHFR) gene in isolated genomic DNA obtained from whole blood potassium EDTA samples from patients with suspected thrombophilia.   
2. Indication(s) for use: Same as Intended use.   
3. Special conditions for use statement(s): For prescription use only   
4. Special instrument requirements: Fluorometer

I. Device Description:

The Invader MTHFR 1298 test consists of the following components:

• MTHFR 1298 Oligo Mix   
• Universal Buffer   
• Universal Enzyme Mix   
• No DNA Control   
• MTHFR 1298 Wild Type Control

• MTHFR 1298 Heterozygous Control • MTHFR 1298 Mutant Control • Invader Call Reporter™ Software – Version 5.3 • Invader $\textsuperscript { \textregistered }$ MTHFR 1298 Software – Version 2.0

# J. Substantial Equivalence Information:

1. Predicate device name(s) and 510(k) number(s): Verigene® MTHFR Nucleic Acid Test (k070597)

2. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicates</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>An in vitro diagnostic (IVD)device for the detection andgenotyping of a single pointmutation of the human 5,10methylene-tetrahydrofolatereductase gene (MTHFR) inisolated genomic DNA obtainedfrom patients with suspectedthrombophilia</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target Population</td><td rowspan=1 colspan=1>Patients with suspectedthrombophilia</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Purified DNA isolated fromhuman whole peripheral blood</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target AmplificationTechnology</td><td rowspan=1 colspan=1>PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Signal GenerationTechnology</td><td rowspan=1 colspan=1>Fluorescence Resonance EnergyTransfer (FRET)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Output Data</td><td rowspan=1 colspan=1>Assay signal results areinterpreted by a software programand are assigned a genotype thatis presented to the end user in areport format</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Indications for Use</td><td colspan="1" rowspan="1">Detection and genotyping of asingle point mutation (A to C atposition 1298) of the humanMTHFR gene</td><td colspan="1" rowspan="1">Detection and genotyping of asingle point mutation (C to Tat position 677) of the humanMTHFR gene</td></tr><tr><td colspan="1" rowspan="1">Allele Discrimination</td><td colspan="1" rowspan="1">Targeted cleavage of distinctFRET cassette bound to allele-specific primary probe with aunique 5'-flap</td><td colspan="1" rowspan="1">SNP discrimination viaoligonucleotide probes;detection via evanescentwave light scatter withnanoparticles</td></tr><tr><td colspan="1" rowspan="1">Reaction Conditions</td><td colspan="1" rowspan="1">20-μL reaction in multiple plasticmicrotiter wells</td><td colspan="1" rowspan="1">25-µL reaction in unitized testcartridge</td></tr><tr><td colspan="1" rowspan="1">Signal Detection</td><td colspan="1" rowspan="1">End-point detection of amplifiedsequences after PCR</td><td colspan="1" rowspan="1">Single-image sensor wherenanoparticles are illuminated</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">amplification</td><td colspan="1" rowspan="1">using a fixed-wavelengthlight source</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">Non-specified, third-partyfluorometer and thermal cycler</td><td colspan="1" rowspan="1">Verigene Reader(photosensor detection ofsignal enhancednanoparticles) and VerigeneProcessor (Hybridization andWash Station)</td></tr><tr><td colspan="1" rowspan="1">Software Interface</td><td colspan="1" rowspan="1">Java-based software installed on astandalone PC capable ofconverting raw fluorescence datainto genotype calls</td><td colspan="1" rowspan="1">Verigene Software withintegrated graphical userinterface</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff – Class II Special Controls Guidance Document: Factor V Leiden DNA Mutation Detection Systems issued on March 16, 2004   
Guidance for Industry and FDA Staff - Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices issued May 11, 2005 Guidance for Industry and FDA Staff – Format for Traditional and Abbreviated 510(k)s issued on August 12, 2005

# L. Test Principle:

The Invader® MTHFR 1298 test combines target amplification by PCR using a thirdparty thermal cycler with signal generation by the Invader Plus® chemistry and signal detection using interpretative software and a third-party fluorometer for genotyping of a single point mutation (A to C at position 1298) of the human 5,10- methylenetetrahydrofolate reductase (MTHFR).

During the signal generation phase, a discriminatory Primary Probe transiently hybridizes to the amplified target sequence along with an Invader $\textsuperscript { \textregistered }$ oligonucleotide, to form an overlapping structure. The $5 "$ -end of the Primary Probe includes a $5 " \cdot$ -flap that does not hybridize to the target DNA. The $3 "$ -nucleotide of the bound Invader $\textsuperscript { \textregistered }$ oligonucleotide overlaps the Primary Probe, and does not hybridize to the target DNA. The Cleavase $^ \mathrm { \textregistered }$ enzyme recognizes this overlapping structure and cleaves off the unpaired $5 "$ -flap of the Primary Probe, releasing it as a target-specific product. The released $5 "$ -flap transiently hybridizes with a corresponding fluorescent resonance energy transfer (FRET) cassette forming an overlapping structure that is recognized and the fluorophore is cleaved from the FRET cassette by the Cleavase $^ \mathrm { \textregistered }$ enzyme. When the FRET cassette is cleaved, a fluorophore and quencher are separated, generating detectable fluorescence signal.

The Primary Probe and the $5 "$ -flap are designed to have a melting temperature aligned with the Invader $\textsuperscript { \textregistered }$ reaction temperature so that under the isothermal reaction conditions $( { \sim } 6 3 ^ { \circ } \mathrm { C } )$ the Primary Probes cycle on and off the target DNA and the $5 " .$ - flaps cycle on and off of the corresponding FRET cassettes. This allows for multiple rounds of Primary Probe cleavage for each DNA target resulting in an accumulation of released $5 "$ -flaps and multiple rounds of FRET cassette cleavage for each $5 ^ { \circ }$ -flap resulting in an accumulation of released fluorophore.

The Invader® MTHFR 1298 test uses two different discriminatory Primary Probes, one for the mutant allele and one for the wild type allele. Each Primary Probe is assigned a unique 5'-flap, and distinct FRET cassette, with a spectrally distinct fluorophore. The released $5 "$ -flaps will bind only to their respective FRET cassettes to generate a target-specific signal, linking the wild type allele with one fluorophore (Fluorescence 1: FAM) and the mutant allele with the second fluorophore (Fluorescence 2: RED).

The Invader $^ \mathrm { \textregistered }$ MTHFR 1298 software, in combination with Invader Call Reporter™ software, is a data analysis software that provides a working template for the setup of reaction mixes and sample placement, and following the import of fluorescence data, it determines results and validity for controls and samples.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: Repeatability/Reproducibility: Two operators each from 3 different sites (2 external sites and 1 internal site) performed the testing in duplicate twice daily over 5 non-consecutive days using the same testing materials including a panel of nine (9) whole blood samples with each of the three (3) possible genotypes (i.e. wild type (WT), heterozygous (HET), homozygous mutant (MUT)) represented. There were no first-pass No-Calls or Miscalls, and there was no statistical difference due to Site, Operator or Day.

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Operator</td><td rowspan=2 colspan=1>SamplesTested</td><td rowspan=1 colspan=3>First pass</td><td rowspan=1 colspan=2>Final</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>Miscalls</td></tr><tr><td rowspan=2 colspan=1>Site001</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Site002</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Site003</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>521</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Lot-to-Lot Reproducibility: A total of nine (9) genomic DNA samples (three (3) WTs, three (3) HETs and three (3) MUTs) were tested in quadruplicate using three (3) different kit lots of the Invader $^ \mathrm { \textregistered }$ MTHFR 1298 test. The percent agreement between Invader $\textsuperscript { \textregistered }$ MTHFR 1298 test and sequencing was $100 \%$ $\scriptstyle ( \mathrm { n = 1 0 8 } )$ .

<table><tr><td rowspan=2 colspan=1>Lot</td><td rowspan=2 colspan=1>SamplesTested</td><td rowspan=1 colspan=3>First pass</td><td rowspan=1 colspan=2>Final</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>Miscalls</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Genomic DNA Extraction Reproducibility: Four commonly used extraction methods were used to process thirty (30) human whole blood samples and ten (10) leukocyte depleted whole blood (LDWB) spiked with cell lines with the following genotypes:

<table><tr><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=4>Number of Samples</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Human whole blood l samples</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>LDWB spiked with cell lines</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>40</td></tr></table>

The 160 extracted samples were assayed in singlet in a single run using a single kit lot of the Invader® MTHFR 1298 test. With the exception of one sample that was removed from the study due to loss of traceability of the sample identification, all other samples, regardless of extraction methods, were correctly called on the first pass and gave $100 \%$ agreement with DNA sequencing.

b. Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods): Quality Control:

Each test contains positive and negative controls to assure proper functioning of the system. Failure of any controls will be indicated as “Invalid” in the test results section of the report. The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.

Positive Control: The genotype controls (WT, HET, MUT) ensure reagents were assembled correctly and perform according to the specifications.

Negative Control: The No DNA Control is used by the interpretive software to set the “noise” component of the run for “signal-to-noise” calculations.

# Real-Time Stability Study

The ongoing Stability Study tests seven (7) gDNA samples including three (3) WT, two (2) HET and two (2) MUT genotypes using three (3) lots of Invader® MTHFR 1298 product stored under two recommended conditions: (1) $- 3 0 ^ { \circ } \mathrm { C }$ to ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ (Standard Storage of intermediate components) and (2) $4 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ (Standard Storage of Genotype-Specific Controls). Functional testing is performed in quadruplicate at each time point and the interim test results have demonstrated 7 months stability for the device.

<table><tr><td colspan="1" rowspan="2">Sample/Control</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">TO Result</td><td colspan="3" rowspan="1">T4 Result</td><td colspan="3" rowspan="1">T7 Result</td></tr><tr><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td></tr><tr><td colspan="1" rowspan="1">Control 1</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">Control 2</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td></tr><tr><td colspan="1" rowspan="1">Control 3</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td></tr><tr><td colspan="1" rowspan="1">gDNA 1</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">gDNA 2</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td><td colspan="1" rowspan="1">WT</td></tr><tr><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">2Lot 3</td><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td></tr><tr><td colspan="1" rowspan="1">gDNA 3</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td><td colspan="1" rowspan="1">HET</td></tr><tr><td colspan="1" rowspan="1">gDNA 4</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td><td colspan="1" rowspan="1">MUT</td></tr><tr><td colspan="2" rowspan="1">Percent Agreement</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td></tr></table>

# Reagent Freeze-Thaw Stability Study

Product in the final configuration was subject to 15 freeze-thaw cycles prior to the final thaw at the time of testing. Functional testing was performed in replicate of 3-8 using genomic DNA samples isolated from cell lines, representing all possible genotypes including two (2) WT, two (2) HET and two (2) MUT. Results showed that all samples were correctly called using Invader $\textsuperscript { \textregistered }$ Invader® MTHFR 1298 test reagents that underwent repeated freeze-thaw cycles.

d. Detection limit:

Three (3) genomic DNA (gDNA) samples with different genotypes (i.e. WT, HET, MUT) were extracted from whole blood collected in potassium EDTA. Each sample was diluted to eight different concentrations 0.5, 5, 20, 40, 80, 200, 400, $8 0 0 ~ \mathrm { { n g / \mu L } }$ and tested in replicates of forty (40). The recommended range of the assay was determined to be between ${ 5 { - } 8 0 \mathrm { n g / \mu L } }$ of input gDNA, based on $100 \%$ concordance of all tested replicates with bi-directional sequencing.

<table><tr><td rowspan=2 colspan=1>Input DNAConcentration</td><td rowspan=1 colspan=3>Concordance of Genotypes</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>0.5 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>67.5% (27/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>5 ng/μl</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>20 ng/μl</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>40 ng/μl</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>80 ng/μl</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>200 ng/μl</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>400 ng/μl</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td></tr><tr><td rowspan=1 colspan=1>800 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100%(40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr></table>

e. Analytical specificity:

Test performance was not affected by addition of the following substances to nine (9) whole blood samples of different genotype (3 WT, 3 HET, 3 MUT) prior to extraction:

Heparin ( $1 5 0 0 \mathrm { U / d L }$ human whole blood) • Cholesterol $( 3 0 0 \mathrm { m g / d L }$ human whole blood) $\bullet$ Bilirubin $1 0 \mathrm { m g / d L }$ human whole blood) $\bullet$ Hemoglobin (up to $0 . 2 \%$ in whole blood) • Potassium EDTA $1 . 8 ~ \mathrm { m g / m L }$ human whole blood) • Ethanol-based Wash Buffer $5 \%$ in DNA sample)

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison:

A total of 348 gDNA samples extracted from whole blood samples were genotyped using the Invader® MTHFR 1298 test and bi-directional DNA sequencing. The observed agreement between the Invader $^ \mathrm { \textregistered }$ MTHFR 1298 test and bi-directional DNA sequencing was $100 \%$ (347/347). The first run agreement with bi-directional sequencing was $9 9 . 7 1 \%$ (347/348).

Table: Comparison of Invader® MTHFR 1298 Test and DNA Sequencing Results

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>First Pass Results</td><td rowspan=1 colspan=7>Final Results</td></tr><tr><td rowspan=1 colspan=1>Geno-type[&quot;]</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>No-Call</td><td rowspan=1 colspan=1>Mis-Call</td><td rowspan=1 colspan=1>%Agree-ment</td><td rowspan=1 colspan=1>95%LCB[2]</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>No-Call</td><td rowspan=1 colspan=1>Mis-Call</td><td rowspan=1 colspan=1>%Agree-ment</td><td rowspan=1 colspan=1>95%LCB</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.45</td><td rowspan=1 colspan=1>96.99</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.45</td><td rowspan=1 colspan=1>96.99</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>97.63</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>97.63</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>92.78</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>92.78</td></tr></table>

[1]Genotype as determined with DNA sequencing; [2]Lower boundary of the $9 5 \%$ confidence interval

b. Matrix comparison:

Not applicable c. Instrument Equivalency:

Twenty-nine (29) human whole blood samples and ten (10) leukocyte depleted whole blood samples spiked with cell lines were extracted using two (2) commonly used extraction methods. The extracts were tested with the Invader $\textsuperscript { \textregistered }$ MTHFR 1298 test using three (3) commercially available thermal cyclers and the raw fluorescent data acquired on three (3) commercially available fluorometers. Results from the three (3) fluorometers were transferred into the interpretive software and genotype calls compared to bidirectional sequencing. The results showed $100 \%$ concordance with bidirectional sequencing as follows:

<table><tr><td rowspan=1 colspan=4>Concordance by Instrument</td></tr><tr><td rowspan=2 colspan=1>Fluorometer</td><td rowspan=1 colspan=3>Thermal Cycler</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr></table>

3. Clinical studies:

a. Clinical Sensitivity: Not applicable   
b. Clinical specificity: Not applicable   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range: The MTHFR A1298C mutation being genotyped in the Invader® MTHFR 1298

Test is present at a frequency of ${ \sim } 3 3 \%$ in the general population.

N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.