# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k100943   
B. Purpose for Submission: New device   
C. Measurand: Factor II   
D. Type of Test: Qualitative genotyping test for single nucleotide polymorphism detection   
E. Applicant: Hologic Inc.   
F. Proprietary and Established Names: Invader® Factor II

# G. Regulatory Information:

1. Regulation section: 21 CFR §864.7280 Factor V Leiden DNA mutation detection systems   
2. Classification: Class II   
3. Product code: NPR: Test, Factor II G20210A Mutations, Genomic DNA PCR   
4. Panel: 81 Hematology

# H. Intended Use:

1. Intended use(s): The Invader® Factor II test is an in vitro diagnostic test intended for the detection and genotyping of a single point mutation (G to A at position 20210) of the human Factor II gene in isolated genomic DNA obtained from whole blood potassium EDTA samples from patients with suspected thrombophilia.

2. Indication(s) for use: Same as Intended use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Multi-well fluorometer capable of: Multi-Labeling Measurement Parameters Measurement 1 Measurement 2 Read mode Top Top Excitation wavelength 485/20 nm 560/20 nm Emission wavelength 535/25 nm 612/10 nm Gain Manual Manual Number of flashes 10 10 Integration time 20 μs 20 μs

I. Device Description:

The Invader® Factor II test consists of the following components:

Factor II Oligo Mix • Universal Buffer • Universal Enzyme Mix No DNA Control • Factor II Wild Type Control • Factor II Heterozygous Control • Factor II Mutant Control • Invader® Call Reporter™ Software – Version 5.3 • Invader® Factor II Software – Version 2.0

# J. Substantial Equivalence Information:

1. Predicate device name(s) and 510(k) number(s): Roche Factor II (Prothrombin) G20210A Kit (k033612)

2. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>An in vitro diagnostic testintended for the detection andgenotyping of a single pointmutation (G to A at position20210) of the human Factor IIgene in isolated genomic DNAobtained from whole bloodpotassium EDTA samplesfrom patients with suspectedthrombophilia.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target Population</td><td rowspan=1 colspan=1>Patients with suspectedthrombophilia</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Purified DNA isolated fromhuman whole peripheral blood</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target AmplificationTechnology</td><td rowspan=1 colspan=1>PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Signal GenerationTechnology</td><td rowspan=1 colspan=1>Fluorescence ResonanceEnergy Transfer (FRET)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Output Data</td><td rowspan=1 colspan=1>Assay signal results areinterpreted by a softwareprogram and are assigned agenotype that is presented tothe end user in a report format</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Allele Discrimination</td><td colspan="1" rowspan="1">Targeted cleavage of distinctFRET cassette bound to allele-</td><td colspan="1" rowspan="1">Melting curve analysis ofhybridizations to allele-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">specific primary probe with aunique 5'-flap</td><td colspan="1" rowspan="1">specific FRET probes</td></tr><tr><td colspan="1" rowspan="1">Reaction Conditions</td><td colspan="1" rowspan="1">20-µL reaction in multipleplastic microtiter wells</td><td colspan="1" rowspan="1">15 µL reaction in glasscapillaries loaded in asample carousel</td></tr><tr><td colspan="1" rowspan="1">Signal Detection</td><td colspan="1" rowspan="1">End-point detection ofamplified sequences after PCRamplification</td><td colspan="1" rowspan="1">Real-time detection ofamplified sequences duringPCR cycles</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">Non-specified, third-partyfluorometer and thermal cycler</td><td colspan="1" rowspan="1">LightCycler® InstrumentV1.2 including a cycler anda fluorimeter</td></tr><tr><td colspan="1" rowspan="1">Software Interface</td><td colspan="1" rowspan="1">Java-based software installedon a standalone PC capable ofconverting raw fluorescencedata into genotype calls</td><td colspan="1" rowspan="1">LightCycler® SoftwareV3.5</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff – Class II Special Controls Guidance Document: Factor V Leiden DNA Mutation Detection Systems issued on March 16, 2004 Guidance for Industry and FDA Staff - Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices issued May 11, 2005 Guidance for Industry and FDA Staff – Format for Traditional and Abbreviated 510(k)s issued on August 12, 2005

# L. Test Principle:

The Invader® Factor II test combines target amplification by PCR using third-party thermal cycler with signal generation by the Invader Plus® chemistry and signal detection using interpretative software and third-party fluorometer for genotyping of a single point mutation (G to A at position 20210) of the human Factor II gene.

During the signal generation phase, a discriminatory Primary Probe transiently hybridizes to the amplified target sequence along with an Invader® oligonucleotide, to form an overlapping structure. The $5 "$ -end of the Primary Probe includes a $5 "$ -flap that does not hybridize to the target DNA. The $3 "$ -nucleotide of the bound Invader® oligonucleotide overlaps the Primary Probe, and does not hybridize to the target DNA. The Cleavase® enzyme recognizes this overlapping structure and cleaves off the unpaired $5 "$ -flap of the Primary Probe, releasing it as a target-specific product. The Primary Probe is designed to have a melting temperature aligned with the Invader® reaction temperature so that under the isothermal reaction conditions $( { \sim } 6 3 ^ { \circ } \mathrm { C } )$ the Primary Probes cycle on and off the target DNA. This allows for multiple rounds of Primary Probe cleavage for each DNA target resulting in an accumulation of released $5 "$ -flaps.The released $5 "$ -flap transiently hybridizes with a corresponding fluorescent resonance energy transfer (FRET) cassette forming an overlapping structure that is recognized and the fluorophore is cleaved from the FRET cassette by the Cleavase enzyme. The 5'-flap is designed to have a melting temperature aligned with the Invader® reaction temperature, so that the $5 "$ -flaps cycle on and off of the corresponding FRET cassettes. This allows for multiple rounds of FRET cassette cleavage for each $5 "$ -flap, and an accumulation of released fluorophore. When the FRET cassette is cleaved, a fluorophore and quencher are separated, generating detectable fluorescence signal.

The Invader® Factor II test uses two different discriminatory Primary Probes, one for the mutant allele and one for the wild type allele. Each Primary Probe is assigned a unique $5 "$ - flap, and distinct FRET cassette, with a spectrally distinct fluorophore. The released $5 "$ -flaps will bind only to their respective FRET cassettes to generate a target-specific signal, linking the wild type allele with one fluorophore (Fluorescence 1: RED) and the mutant allele with the second fluorophore (Fluorescence 2: FAM).

The Invader® Factor II software, in combination with Invader® Call Reporter™ software, is a data analysis software that provides a working template for the setup of reaction mixes and sample placement, and following the import of fluorescence data, it determines results and validity for controls and samples.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: Repeatability/Reproducibility: Two operators each from 3 different sites (2 external sites and 1 internal site) performed the testing in duplicate twice daily over 5 nonconsecutive days using the same testing materials including a panel of nine (9) whole blood samples, representing all possible genotypes including three (3) wild types (WT), three (3) heterozygous mutants (HET) and three (3) homozygous mutants (MUT).

There were 36/540 $( 6 . 7 \% )$ first pass No Calls due to “Invalid Control” result on two (2) independent runs. Upon retraining of the operator and retesting of the runs, all controls reported “Valid” and 34 samples were found to be in agreement with sequencing. Two (2) samples with Final No Call were called correctly upon reextraction and re-testing. There were zero Miscalls, and there was no statistical difference due to Site, Operator or Day.

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Operator</td><td rowspan=2 colspan=1>SamplesTested</td><td rowspan=1 colspan=3>First pass</td><td rowspan=1 colspan=2>Final</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>Miscalls</td></tr><tr><td rowspan=2 colspan=1>Site001</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>002</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>003</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.78%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>538</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.63%</td></tr></table>

Lot-to-Lot Reproducibility: A total of four (4) genomic DNA samples (three (3) wild type and one (1) heterozygous) were tested in quadruplicate using three (3) different kit lots of the Invader® Factor II test. The percent agreement between Invader® Factor II test and sequencing was $100 \%$ $( \mathrm { n } { = } 4 8 )$ ).

Genomic DNA Extraction Reproducibility: Four commonly used extraction methods were used to process thirty (30) human whole blood samples and ten (10) leukocyte depleted whole blood (LDWB) spiked with cell lines with the following genotypes:

<table><tr><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=4>Number of Samples</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Human whole blood samples</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>LDWB spiked with cell lines</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>40</td></tr></table>

The 160 extracted samples were assayed in singlet in a single run using a single kit lot of the Invader® Factor II test. With the exception of one sample that was removed from the study due to loss of traceability of the sample identification, all other samples, regardless of extraction methods, were correctly called on the first pass and gave $100 \%$ agreement with DNA sequencing.

$b$ . Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Each test contains positive and negative controls to assure proper functioning of the system: Failure of any controls will be indicated as “Invalid” in the test results section of the report. The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.

Positive Control: The genotype controls (WT, HET, MUT) ensure reagents were assembled correctly and perform according to the specifications.

Negative Control: The No DNA Control is used by the interpretive software to set the “noise” component of the run for “signal-to-noise” calculations.

Hardware and Software Controls: The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.

# Real-Time Stability Study

The ongoing Stability Study tests seven (7) gDNA samples including three (3) WT, two (2) HET and two (2) MUT genotypes using three (3) lots of Invader® Factor II product stored under two recommended conditions: 1) ${ - 2 0 ^ { \circ } C }$ (Standard Storage of intermediate components) and 2) $4 ^ { \circ } { - } 8 ^ { \circ } \mathrm { C }$ (Standard Storage of Genotype-Specific Controls). Functional testing is performed in quadruplicate at each time point and the interim test results have demonstrated 7 months stability for the device.

<table><tr><td rowspan=2 colspan=1>Sample/Control</td><td rowspan=2 colspan=1>Genotype</td><td rowspan=1 colspan=3>TO Result</td><td rowspan=1 colspan=3>T4 Result</td><td rowspan=1 colspan=3>T7 Result</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td></tr><tr><td rowspan=1 colspan=1>Control 1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>Control 2</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>Control 3</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=1>gDNA 1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>gDNA 2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>gDNA 3</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>gDNA 4</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

# Reagent Freeze-Thaw Stability Study

Product in the final configuration was subject to 15 freeze-thaw cycles prior to the final thaw at the time of testing. Functional testing was performed in replicate of 3-8 using genomic DNA samples isolated from cell lines, representing all possible genotypes including two (2) WT, two (2) HET and two (2) MUT. Results showed that all samples were correctly called using Invader® Factor II test reagents that underwent repeated freeze-thaw cycles.

d. Detection limit:

Two (2) genomic DNA samples with different genotypes (i.e., one WT and the other HET mutant) were extracted from whole blood collected in potassium EDTA. They were serially diluted and tested 40 times each at eight (8) different input amounts of (0.5, 5, 20, 40, 80, 200, 400, $8 0 0 ~ \mathrm { { n g / \mu L } }$ . The lower and upper limits of input gDNA when all samples were called correctly were determined to be $5 \mathrm { n g / \mu L }$ and $8 0 ~ \mathrm { n g / \mu L }$ , respectively.

e. Analytical specificity:

Interfering Substances

Test performance was not affected by addition of the following substances to four (4) whole blood samples of different genotype (3 WT, 1 HET) prior to extraction:

Heparin ( $1 5 0 0 \mathrm { U / d L }$ human whole blood) • Cholesterol $( 3 0 0 \mathrm { m g / d L }$ human whole blood) • Bilirubin $1 0 \mathrm { m g / d L }$ human whole blood) Hemoglobin (up to $0 . 2 \%$ in whole blood) • Potassium EDTA ( $1 . 8 ~ \mathrm { m g / m L }$ human whole blood) Ethanol-based Wash Buffer $5 \%$ in DNA sample)

Interfering Mutations

The effect of interfering mutations on G20210A genotyping by the Invader® Factor II Test was evaluated with six (6) contrived DNA samples, including one (1) WT sample with no secondary polymorphism, one (1) HET sample and four (4) WT samples each with a known secondary polymorphism (i.e., A20207C, C20209T, A20218G, or C20221T). Forty replicates for each of the six (6) different samples were tested, and no effect of secondary polymorphisms was observed on G20210A genotyping by the Invader® Factor II Test.

<table><tr><td>Non-Panel Mutation or Polymorphism</td><td>Panel Mutation</td></tr><tr><td>20207A&gt;C 20209C&gt;T 20218A&gt;G 20221C&gt;T</td><td>20210G&gt;A</td></tr></table>

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison: A total of 336 gDNA samples extracted from whole blood samples were genotyped using the Invader® Factor II Test and DNA sequencing. All samples gave $100 \%$ agreement with DNA sequencing.

Table: Comparison of Invader® Factor II Test and DNA Sequencing Results

<table><tr><td>Geno-</td><td>First Pass Results</td><td>Final Results</td></tr></table>

<table><tr><td rowspan=1 colspan=1>typel</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>No-Call</td><td rowspan=1 colspan=1>Mis-Call</td><td rowspan=1 colspan=1>%Agree-ment</td><td rowspan=1 colspan=1>95%LCB[2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>No-Call</td><td rowspan=1 colspan=1>Mis-Call</td><td rowspan=1 colspan=1>%Agree-ment</td><td rowspan=1 colspan=1>95%LCB</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>305</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.02</td><td rowspan=1 colspan=1>305</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.02</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>88.27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>88.27</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>65.18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>65.18</td></tr></table>

Note: [1]Genotype as determined with DNA sequencing; [2]Lower boundary of the $9 5 \%$ confidence interval

b. Matrix comparison: Not applicable   
c. Instrument Equivalency: Twenty-nine (29) human whole blood samples and ten (10) leukocyte depleted whole blood samples spiked with cell lines were extracted using two (2) commonly used extraction methods. The extracts were tested with the Invader® Factor II test using three (3) commercially available thermal cyclers and the raw fluorescent data acquired on three (3) commercially available fluorometers. Results from the three (3) fluorometers were transferred into the interpretive software and genotype calls compared to bi-directional sequencing.

<table><tr><td rowspan=1 colspan=4>Concordance by Instrument</td></tr><tr><td rowspan=2 colspan=1>Fluorometer</td><td rowspan=1 colspan=3>Thermal Cycler</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr></table>

3. Clinical studies:

a. Clinical Sensitivity: Not applicable   
$b$ . Clinical specificity: Not applicable   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The Factor II mutation being genotyped in the Invader® Factor II Test is present at a frequency of $1 \%$ in the general population.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.