# Invader® Factor V 510(k) SUMMARY

JUN - 1 2011

A. 510(k) Number: K100980   
B. Purpose for Submission: New Device   
C. Measurand: Factor V   
D. Type of Test: Qualitative genotyping test for single nucleotide polymorphism detection.

E. Applicant: Hologic Inc. Third Wave Technologies 250 Campus Drive Marlborough, MA 01752 508-263-8853 Contact Person: Randall J. Covill, Manager, Regulatory Affairs Date of Submission: April 2010

F. Proprietary and Established Names: Invader Factor V   
G. Regulatory Information: 1. Regulation Section: 21 CFR 864.7280 2 Classification: Class II 3. Product Code: NPQ: Test, Factor V Leiden Mutations, Genomic DNA PCR 4. Panel: Hematology (81)

# H. Intended Use:

I.Intended Use(s): The Invader® Factor V test is an in vitro diagnostic test intended for the detection and genotyping of a single point mutation (G to A at position 1691) of the human Factor V gene in isolated genomic DNA obtained from whole blood potassium EDTA samples from patients with suspected thrombophilia.

2.Indication(s) for use: Same as Intended Use

3. Special Conditions for use statements(s): For prescription use only

4. Special instrument requirements: None

# I. Device Description:

The Invader Factor V test consists of the following components: Factor V Oligo Mix Universal Buffer Universal Enzyme Mix No DNA Control Factor V Wild Type Control Factor V Heterozygous Control Factor V Mutant Control Invader Call Reporter™M Software Invader® Factor V Software

# J. Substantial Equivalence Information:

1.Predicate device name(s): Factor V Leiden Kit, Roche   
2. Predicate 510(k) number(s): Roche, K033607   
3. Comparison with predicate:

Table 1: Comparison with Predicate Device   

<table><tr><td colspan="3" rowspan="1">Iable T: Comparison with Predicate DeviceCharacteristic                  Predicate Device                           Proposed Device</td></tr><tr><td colspan="1" rowspan="1">Product Name(Manufacturer,Submission)</td><td colspan="1" rowspan="1">Factor V Leiden Kit(Roche, K033607)</td><td colspan="1" rowspan="1">Invader® Factor V(Hologic, Inc., N/A)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Factor V Leiden Kit is an in vitrodiagnostic test for the detection andgenotyping of a single point mutation(G to A at position 1691) of the humanFactor V gene, from DNA isolatedfrom human whole peripheral blood.The Factor V Leiden Kit is indicatedas an aid to diagnosis in the evaluationof patients with suspectedthrombophilia. The test is intended tobe used on the LightCycler instrument.The sample preparation must beperformed according to a workflowprocedure described in the packageinsert.</td><td colspan="1" rowspan="1">The Invader® Factor V test is an invitro diagnostic test intended for thedetection and genotyping of a singlepoint mutation (G to A at position1691) of the human Factor V gene inisolated genomic DNA obtainedfrom whole blood potassium EDTAsamples from patients with suspectedthrombophilia.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Purified DNA isolated from humanwhole peripheral blood</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Indications forUse</td><td colspan="1" rowspan="1">Same as Intended Use</td><td colspan="1" rowspan="1">Same as Intended Use</td></tr><tr><td colspan="1" rowspan="1">TargetPopulation</td><td colspan="1" rowspan="1">Patients with suspected thrombophilia</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">The amplicon is detected byfluorescence using a specific pair of Hprobes. The H probes consist of twodifferent oligonucleotides thathybridize to an internal sequence of theamplified fragment during theannealing phase of the PCR cycle. Oneprobe is labeled at the 5'-end withLightCycler® Red 640-N-hydroxy-</td><td colspan="1" rowspan="1">PCR and Invader® usingFluorescence Resonance EnergyTransfer (FRET) chemistry forsignal reporting. Both our device andpredicate device detect signal fromamplicons using FluorescenceResonance Energy Transfer (FRET.)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">succinimide ester (Red 640-NHSester), and to avoid extension,modified at the 3'-end byphosphorylation. The other probe islabeled at the 3'-end withfluorescein.3. Only after hybridizationto the template DNA, do the twoprobes come in close proximity,resulting in fluorescence resonanceenergy transfer (FRET) between thetwo fluorophores. During FRET,fluorescein, the donor fluorophore, isexcited by the light source of theLightCycler® 2.0Instrument, and part of the excitationenergy is transferred to LightCycler®Red 640-NHS ester, the acceptorfluorophore.</td><td colspan="1" rowspan="1">b</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">LightCycler® Instrument using SW 3.5</td><td colspan="1" rowspan="1">Non-specified, third-partyfluorometer and thermal cycler.</td></tr><tr><td colspan="1" rowspan="1">SoftwareInterface</td><td colspan="1" rowspan="1">LightCycler® Instrument using SW3.5. Expro database and macros.</td><td colspan="1" rowspan="1">Java-based software installed on astandalone PC capable of convertingraw fluorescence data into genotypecalls.</td></tr><tr><td colspan="1" rowspan="1">DetectionMethod</td><td colspan="1" rowspan="1">The LightCyler® uses opticaldetection of stimulated fluorescencegenerated by the following chemistry:The H probes are also used todetermine the genotype by performinga melting curve analysis after theamplification cycles are completed andthe amplicon is present at increasedconcentration.•The Red 640-labeled H probehybridizes to a part of the targetsequence that is not mutated andfunctions as an anchor probe.•The Fluorescein-labeled H probespans the mutation site (mutationprobe). During the melting curveanalysis, increasing temperature causes</td><td colspan="1" rowspan="1">PCR and Fluorescence ResonanceEnergy Transfer (FRET) chemistryfor signal reporting.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">the fluorescence to decrease becausethe shorter of the two probes (mutationprobe) dissociates first and the twofluorescent dyes are no longer in closeproximity. If the Factor V Leidenmutation is present, the mismatch ofthe mutation probe with the targetdestabilizes the hybrid so the decreasein fluorescence will occur at a lowertemperature. With the wild-typegenotype, mismatches will not occur,and therefore, the heteroduplex DNAhas a higher melting temperature (Tm).The heterozygous genotype exhibits adistinctive combination of properties.</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Sample Size</td><td colspan="1" rowspan="1">10-20pl in glass capillaries</td><td colspan="1" rowspan="1">20ul reaction containing 0.25-4ng/ulgDNA extracted from humanperipheral whole blood.</td></tr><tr><td colspan="1" rowspan="1">DetectionProcedure</td><td colspan="1" rowspan="1">Optical detection of stimulatedfluorescence using a specific pair ofprobes.</td><td colspan="1" rowspan="1">Multi-well fluorometer to detect rawfluorescence.</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Paired hybridization probes usingfluorescence resonance energy transfer(FRET) followed by melting curveanalysis.</td><td colspan="1" rowspan="1">PCR and Invader® usingFluorescence Resonance EnergyTransfer (FRET) chemistry forsignal reporting.</td></tr><tr><td colspan="1" rowspan="1">Analysis Time</td><td colspan="1" rowspan="1">A multi-step assay with different timesrequired for each step. Detectionoccurs at defined intervals during PCRcycle and can be reviewed in real-time.</td><td colspan="1" rowspan="1">~90 min. amplification followed by1 min signal detection. Softwareanalysis post signal detection.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff Class I Special Controls Guidance   
Document: Factor V Leiden DNA Mutation Detection Systems issued on March 16,   
2004   
Guidance for Industry and FDA Staff- Guidance for the Content of Premarket   
Submissions for Software Contained in Medical Devices issued May 11, 2005   
Guidance for Industry and FDA Staff - Format for Traditional and Abbreviated   
510(k)s issued on August 12, 2005

# L. Test Principle:

The Invader® Factor V test utilizes the Invader Plus® chemistry with DNA isolated from human whole blood, for the detection of the targeted sequence polymorphism. Specifically, the Invader Pus® cheiyiliz a gle-ube, t phase reacion,inclu tart aplii ansal generation (mediated by Invader® chemistry). Invader Plus® reaction mixes are assembled by combining the Factor V Oligo Mix, Universal Enzyme Mix, and Universal Buffer. In a 96-well plate, reaction mix is combined with purified genomic DNA samples, as well as four (4) controls included with the test. The No DNA Control is used by the interpretive software to set the "noise" component of the run for "signal-to-noise" calculations. The genotype-specific controls (WT, HET, MUT) ensure reagents were assembled correctly and perform according to the specifications. The 96-well plate is transferred to an appropriately programmed thermal cycler for target amplification and signal generation. In the target amplification phase of the reaction, amplification is carried out using "two-step" cycling conditions (i.e. denaturation & annealing/extension). Following amplification, Taq polymerase is inactivated by a 10 minute incubation at $9 9 ^ { \circ } \mathrm { C }$ , after which the thermal cycler proceeds to $6 3 ^ { \circ } \mathrm { C }$ to initiate the signal generation (Invader®) phase of the reaction (see Figure 1).

![](images/0aabd636696b6def4c821817de6cda5876e1fd1f7a670b43a3755836715d4f75.jpg)

During the signal generation phase, a discriminatory Primary Probe transiently hybridizes to the amplified target sequence along with an Invader® oligonucleotide, to form an overlapping structure. The $5 "$ -end of the Primary Probe includes a $5 "$ -flap that does not hybridize to the target DNA. The EY $3 ^ { \prime }$ -nucleotide of the bound Invader® oligonucleotide overlaps the Primary Probe, and does not hybridize to the target DNA. The Cleavase® enzyme recognizes this overlapping structure and cleaves off the unpaired $5 ^ { \prime }$ -flap of the Primary Probe, releasing it as a target-specific product. The Primary Probe is designed to have a melting temperature aligned with the Invader® reaction temperature so that under the isothermal reaction conditions $( { \sim } 6 3 ^ { \circ } \mathrm { C } )$ the Primary Probes cycle on and off the target DNA. This allows for multiple rounds of Primary Probe cleavage for each DNA target resulting in an accumulation of the number of released $5 ^ { \prime }$ -flaps.The released $5 ^ { \circ }$ -flap transiently hybridizes with a corresponding FRET cassette forming an overlapping structure that is recognized and the fluorophore is cleaved from the FRET cassette by the Cleavase® enzyme. The EY $5 ^ { \circ }$ -flap is designed to have a melting temperature aligned with the Invader® reaction temperature, so that the $5 '$ -flaps cycle on and off of the corresponding FRET cassettes. This allows for multiple rounds of FRET cassette cleavage for each $5 ^ { \prime }$ -flap, and an accumulation of released fluorophore. When the FRET cassette is cleaved, a fluorophore and quencher are separated, generating detectable fluorescence signal. The format uses two different discriminatory Primary Probes, one for the mutant allele and one for the wild type allele (Figure 1). Each Primary Probe is assigned a unique $5 "$ -flap, and distinct FRET cassette, with a spectrally distinct fluorophore. By design, the released $5 "$ -flaps will bind only to their respective FRET cassettes to generate a target-specific signal, linking the wild type allele with one fluorophore (Fluorescence 1: RED) and the mutant allele with the second fluorophore (Fluorescence 2: FAM).

The Invader® Factor V software, in combination with Invader Call Reporter™ software, is a data analysis software package developed by Hologic for use with the Invader® Factor V test. The software package provides a working template for the setup of reaction mixes and sample placement, and following the import of fluorescence data, it determines results and validity for controls and samples. A summary of the Invader Call ReporterT™M Invader® Factor V package workflow is shown in Figure 2.

![](images/2fb34f8b60562fc532fefcb11175d7d5205ac1dcf4dbd517f994b314d0a35b9f.jpg)  
Figure 2. Invader Call Reporter™M Invader® Factor V Package Workflow

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

External Reproducibility (Study #1): Two operators each from three (3) different sites (2 external sites and 1 internal site) performed the testing, in duplicate, over five (5) non-consecutive days for a ten (10) day period using the same testing materials including a panel of nine (9) unique leukocyte depleted whole blood samples spiked cell lines specific for each of the three (3) possible genotypes (i.e. 3 wild type, 3 heterozygous, 3 homozygous mutant).

Table 2: Inter-laboratory Reproducibility of Invader Factor V Test   

<table><tr><td rowspan="2">Site</td><td rowspan="2">Operator</td><td rowspan="2">Samples tested</td><td colspan="3">First Pass</td><td colspan="3">Final</td><td rowspan="2">Final % Agreement</td></tr><tr><td>Correct Calls</td><td>No Calls (Invalld, EQ)</td><td>Miscalts</td><td>Correct Calls</td><td>No Calls Inalld,</td><td>Miscalls Final Conest CU</td></tr><tr><td>Site</td><td>1</td><td>90</td><td>90</td><td>0</td><td>0</td><td>90</td><td>EQ) 0</td><td>0</td><td>Sam ples Tested 100%</td></tr><tr><td>01</td><td>2</td><td>90</td><td>90</td><td>0</td><td>0</td><td>90</td><td>0</td><td>0</td><td>100%</td></tr><tr><td>Site</td><td>1</td><td>90</td><td>90</td><td>0</td><td>0</td><td>90</td><td>0</td><td>0</td><td>100%</td></tr><tr><td>002</td><td>2</td><td>90</td><td>90</td><td>0</td><td>0</td><td>90</td><td>Q</td><td>0</td><td>100%</td></tr><tr><td>Site</td><td>1</td><td>90</td><td>90</td><td>0</td><td>0</td><td>90</td><td>0</td><td>0</td><td>100%</td></tr><tr><td>003</td><td>2</td><td>90</td><td>71</td><td>19*</td><td>0</td><td>89</td><td>1</td><td>0</td><td>98.89%</td></tr><tr><td>All</td><td>All</td><td>540</td><td>521</td><td>19°</td><td>0</td><td>539</td><td>1</td><td>0</td><td>99.81%</td></tr></table>

and re-testing this sample was found to be in agreement with sequencing.

Lot-to-Lot Reproducibility (Study #9): A total of five (5) genomic DNA samples (tre (3) wild type and two () heterozgous) were tested in quadruplicate sng three ()different kit lots of the Invader® Factor V test. The percent agreement between Invader® Factor V test and sequencing was $100 \%$ EPY $( \mathtt { n } ^ { = 6 0 } )$

<table><tr><td rowspan=1 colspan=7>Table 3: Lot to Lot Reproducibility</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>#SamplesTested</td><td rowspan=1 colspan=1>FirstPassCorrectCalls</td><td rowspan=1 colspan=1>FirstPass NoCalls</td><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>FinalCorrectCalls</td><td rowspan=1 colspan=1>FinalAgreement%</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td></tr></table>

b. Linearity/assay reportable range: Refer to section d below.

C. Traceability, Stability, Expected values (controls, calibrators, or methods): Real-Time Stability Study (Study #5): Three (3) lots of product in the final configuration are being stored under recommended conditions: (1) -30 to $- 1 5 \mathrm { { \circ C } }$ (Standard Storage of intermediate components) as well as (2) $+ 2 ^ { \circ }$ to $+ 8 \textdegree C$ E (Standard Storage of Genotype-Specific Controls). Functional testing is performed with samples representing all three (3) genotypes in quadruplicate at each time point. The interim test results.have demonstrated 6 months stability for the device.

Table 4: Factor V Genotype Results; Real-Time Stability   

<table><tr><td rowspan=1 colspan=1>SamplelControl</td><td rowspan=1 colspan=1>OyExpected Factor V; Genotype</td><td rowspan=1 colspan=3>:ATo Result</td><td rowspan=1 colspan=3>Ty Result</td><td rowspan=1 colspan=3>T6 Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td></tr><tr><td rowspan=1 colspan=1>Control 1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>Control 2</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>Control 3</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=1>gDNA 1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>gDNA 2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>gDNA 3</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>gDNA 4</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=2>Percent Agreement</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

Reagent Freeze-Thaw Stability Study (Study #6): Product in the final configuration was subject to 15 freeze-thaw cycles prior to the final thaw at the time of testing. Functional testing was performed using genomic DNA isolate from cell lines, representing all possible genotypes. The percent agreement between the sequencing result and the Invader® Factor V test were $100 \%$ therefore demonstrating stability for up to fifteen (15) freeze/thaw cycles.

<table><tr><td rowspan=1 colspan=18>Table 5: Freeze/Thaw Stability of Invader Factor V</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=15>           Number of Freeze/Thaw Cycles</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>1·</td><td rowspan=1 colspan=1>·2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Control 1(WT)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Control 2(HET)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Control 3(MUT)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>gDNA(WT)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>gDNA(HET)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>gDNA(MUT)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=18>*Testing with gDNA samples did not occur at this testing point.</td></tr></table>

d. Detection limit/Analytical Sensitivity and Normal Range (Study #3): Two (2) genomic DNA samples with different genotypes (i.e. WT, HET) were extracted from whole blood collected in potassium EDTA. Each sample was diluted to eight different concentrations 0.5, 5, 20, 40, 80, 200, 400, $8 0 0 ~ \mathrm { { n g / \mu L } }$ and tested in replicates of forty (40). The recommend range of the assay was determined to be between $5 { - } 8 0 ~ \mathrm { \mathfrak { n } g / \mu L } ,$ of input gDNA, based on $100 \%$ E concordance of all tested replicates with bi-directional sequencing.

<table><tr><td rowspan=1 colspan=3>Table 6: Analytical Sensitivity and Normal RangePercent Agreement Between ReplicatesSample ID (Genotype based on Sequencing)</td></tr><tr><td rowspan=1 colspan=1>Input Sample Concentration</td><td rowspan=1 colspan=1>03-4542 (HET)</td><td rowspan=1 colspan=1>03-4420 (WT)</td></tr><tr><td rowspan=1 colspan=1>0.5 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>5 ng/μl</td><td rowspan=1 colspan=1>100% (40/40).</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>20 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>40 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>80 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>200 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>400 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>800 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr></table>

# e. Analytical specificity (Interfering Substances (Study #4)):

Test performance was not affected by addition of the following substances to four (4) whole blood samples of different genotype (3 WT, 1 HET) prior to extraction:

Heparin ( $1 5 0 0 \mathrm { U / d L }$ human whole blood) Cholesterol $( 3 0 0 ~ \mathrm { m g / d L }$ human whole blood) $\bullet$ Bilirubin ( $1 0 ~ \mathrm { m g / d L }$ human whole blood) $\bullet$ Hemoglobin (up to $0 . 2 \%$ in whole blood) • Potassium EDTA $\mathrm { \mathrm { K } } _ { 2 } \mathrm { E D T A } )$ $\mathrm { { 1 . 8 \ m g / m L } }$ human whole blood) • Ethanol-based Wash Buffer $5 \%$ in DNA sample)

Table 7: Summary. Comparison of Invader\* Factor V Interfering Substance Results to Sequencing   

<table><tr><td rowspan=1 colspan=1>InterferingSubstanceCode</td><td rowspan=1 colspan=1>Substance Concentration / (in bloodor DNA sample)</td><td rowspan=1 colspan=1>% Agreement withSequencingGenotype</td><td rowspan=1 colspan=1>% Agreement withUntreated SampleInvader Factor VGenotype</td><td rowspan=1 colspan=1>PASS /FAIL</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>No Addition Control (Untreated)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Bilirubin 10mg/dl (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>Cholesterol 300mg/dl (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>K2EDTA 1.8mg/ml (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>Heparin 1500 U/dl (blood)</td><td rowspan=1 colspan=1>100% (8 of8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F1.</td><td rowspan=1 colspan=1>Hemoglobin 0.2% (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>Hemoglobin 0.1% (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F3</td><td rowspan=1 colspan=1>Hemoglobin 0.05% (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F4</td><td rowspan=1 colspan=1>Hemoglobin 0.025% (blood)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>Ethanol-based Wash Buffer 5%(DNA)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>100% (8 of 8)</td><td rowspan=1 colspan=1>PASS</td></tr></table>

f. Pre-Analytical Equivalency Study/Genomic DNA Extraction Reproducibility (Study #7): Thirty (30) human whole blood samples and ten (10) leukocyte depleted whole blood spiked with cell lines were divided and extracted using four (4), commercially available DNA extraction methods (A. Qiagen QIAamp® 96 DNA Blood Kit, B. Qiagen QIAamp® DNA Blood Mini Kit, C. Gentra Generation® Capture Column Kit (Qiagen), D. Roche MagNA Pure LC DNA Isolation Kit I). The 160 extracted DNAs were analyzed in singlicate with one (1) lot of the device. The percent agreement between the Invader® Factor $v$ test for each extraction method and bi-directional sequencing was $100 \%$ $\scriptstyle ( \ n = 4 0 )$

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Table 8: Pre-Analytical Equivalency</td></tr><tr><td rowspan=1 colspan=1>ExtractionMethod</td><td rowspan=1 colspan=1>#SamplesTested</td><td rowspan=1 colspan=1>FirstPassCorrectCalls</td><td rowspan=1 colspan=1>FirstPass NoCalls</td><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>FinalCorrectCalls</td><td rowspan=1 colspan=1>FinalAgreement%</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39*</td><td rowspan=1 colspan=1>100*</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=7>*Sample was removed from study due to loss of traceability of the sampleidentification.</td></tr></table>

g. Instrument Equivalency (Study #8): Twenty-nine (29) human whole blood samples and ten (10) leukocyte depleted whole blood samples spiked with cell lines were extracted using Qiagen QIAamp® DNA Blood Mini Kit and Roche MagNA Pure LC DNA Isolation Kit I. The extracts were tested with the Invader® Factor V test using three (3) commercially available thermal cyclers (1. ABI GeneAmp® PCR System 9700 with 96-well gold block, 2. ABI VeritiTM and 3. MJ Research PTC-100) and the raw fluorescent data acquired on three (3) commercially available fluorometers (A. Tecan Infinite, B. Tecan Genios® and C. BioTek, $\mathsf { F L x } 8 0 0 ^ { \cdot }$ Results from the three (3) fluorometers were transferred into the interpretive software and genotype calls compared to bi-directional sequencing.

<table><tr><td rowspan=2 colspan=4>Table 9: Concordance by InstrumentThermal Cycler</td></tr><tr><td rowspan=2 colspan=1>Fluorometer</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>78 of 78= 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78= 100%</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>77 of 78 = 98.7%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr></table>

h. Secondary Polymorphism Impact (Study #10): Samples tested included one Factor V (G1691A) homozygous normal sample, one Factor V (G1691A) heterozygous sample and three Factor V (G1691A) homozygous normal samples each with a known secondary polymorphism, G1689A, A1692C or A 1696G. Forty replicates for each of the five different samples were tested.

<table><tr><td rowspan=1 colspan=8>Table 10: Invader® Factor V Concordance</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=6>Expected Results - Factor V (G1691A) Genotype</td></tr><tr><td rowspan=1 colspan=1>Sample01</td><td rowspan=1 colspan=1>Sample02</td><td rowspan=1 colspan=1>Sample03</td><td rowspan=1 colspan=1>Sample04</td><td rowspan=1 colspan=1>Sample05</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>Hc</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>200</td></tr></table>

2. Comparison studies:

a. Method comparison: Bi-directional Sequencing (Study #2)

Human whole blood samples $( \mathtt { n } = 3 5 2 )$ underwent DNA extraction and subsequent bi-directional DNA sequence analysis. The same DNA samples were then analyzed using the Invader® Factor $\mathbf { V }$ test. The observed agreement between the Invader® Factor V test and bi-directional DNA sequencing was $100 \%$ (352/352). The overall agreement with bi-directional sequencing was $100 \%$ (352/352).

Table 10: Agrcement between the Invader® Factor V Test and Bi-directional DNA Sequencing   

<table><tr><td rowspan=1 colspan=1>Factor VGenotype*</td><td rowspan=1 colspan=1>Number tested</td><td rowspan=1 colspan=1>Number ofValid Resultson 1t Run</td><td rowspan=1 colspan=1>Number ofCorrectgenotypecalls on FirstRun</td><td rowspan=1 colspan=1>Agreement</td></tr><tr><td rowspan=1 colspan=1>HomozygousWild Type(GG)</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous(GA)</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HomozygousMutant(AA)</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=5>* Genotype determined through bi-directional DNA sequencing</td></tr></table>

3. External Reproducibility studies:

a. Clinical Sensitivity: Refer to section 1d above.   
EPY $b$ Clinical specificity: Refer to section le above.

4. Expected values/Reference range: (Prevalence) Factor V: $5 \%$ E

# N. System Descriptions:

Modes of Operation: Closed System

2. Software: FDA has reviewed applicant's Hazard Analysis and software development processes for this line of product type. Yes__X___or No_

3. Specimen Identification: Manual Labeling

4. Specimen Sampling and Handling:

DNA should be extracted using a validated DNA extraction method that generates DNA concentration greater than $5 \mathrm { n g / \mu l }$ .

5. Quality Control:

Each test contains positive and negative controls to assure proper functioning of the system: Failure of any controls will be indicated as "Invalid" in the test results section of the report. The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.

Positive Control: The genotype controls (WT, HET, MUT) ensure reagents were assembled correctly and perform according to the specifications.

Negative Control: The No DNA Control is used by the interpretive software to set the "noise" component of the run for "signal-to-noise" calculations.

The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.

OProposed Labeling:

The labeling is sufficient and satisfies the requirements of 21 CFR Part 809.10.

# P. Conclusion:

The submitted information in this 510 (k) notification is complete and supports a substantial equivalence decision.

Hologic Inc. c/o Mr. Randall J. Covill Manager, Regulatory Affairs 250 Campus Drive Marlborough, MA 01752

Re: k100980 Trade/Device Name: Invader® Factor V Regulation Number: 21 CFR $\ S 8 6 4 . 7 2 8 0 $ EY Regulation Name: Factor V Leiden DNA Mutation Detection Systems Regulatory Class: Class II Product Code: NPQ Dated: May 19, 2011 Received: May 26, 2011

Dear Mr. Covill:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class II (Special'Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice

requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

![](images/b0f57331bd1e0ae65719cf341f3c71be5473b84bffb6710d74de203392ef66df.jpg)

Maria M. Chan, Ph.D   
Director   
Division of Immunology and Hematology Devices   
Office of In Vitro Diagnostic Device Evaluation and Safety   
Center for Devices and Radiological Health

# Indications for Use Form

510(k) Number (if known): k100980

Device Name: Invader Factor V test

Indications for Use:

The Invader® Factor V test is an in vitro diagnostic test intended for the detection and genotyping of a single point mutation $G$ to A at position 1691) of the human Factor V gene in isolated genomic DNA obtained from whole blood potassium EDTA samples from patients with suspected thrombophilia.

Over-The-Counter Use (21 CFR 801 Subpart C)