A. 510(k) Number: K100987   
B. Purpose for Submission: New Device   
C. Measurand: MTHFR 677   
D.Type of Test: Qualitative genotyping test for single nucleotide polymorphism detection.

E. Applicant: Hologic Inc. Third Wave Technologies 250 Campus Drive Marlborough, MA 01752 508-263-8912 Contact Person: Randall J. Covill, Manager, Regulatory Affairs Date of Submission: April 2010

F. Proprietary and Established Names: Invader® MTHFR 677

G. Regulatory Information: 1. Regulation Sections: 21 CFR 864.7280 Classification: Class II

3. Product Code: OMM: Test 5,10-Methylenetetrahydrofolate Reductase Mutations, Genomic DNA PCR

4. Panel: Hematology (81)

# H. Intended Use:

1.Intended Use(s): The Invader® MTHFR 677 test is an in vitro diagnostic test intended for the detection and genotyping of a single point mutation (C to T at position 677) of the human 5,10-methylenetetrahydrofolate reductase (MTHFR) gene in isolated genomic DNA obtained from whole blood Potassium EDTA samples from patients with suspected thrombophilia.   
2. Indication(s) for use: Same as Intended Use   
3. Special Conditions for use statements(s): For prescription use only   
4. Special instrument requirements: None

# I. Device Description:

The Invader MTHFR 677 test consists of the following components: MTHFR 677 Oligo Mix Universal Buffer Universal Enzyme Mix No DNA Control MTHFR 677 Wild Type Control MTHFR 677 Heterozygous Control MTHFR 677 Mutant Control Invader Call Reporter™M Software

Invader® MTHFR 677 Software

# J. Substantial Equivalence Information:

1. Predicate device name(s): Verigene® MTHFR Nucleic Acid Test (Nanosphere, K070597)   
2. Predicate 510(k) number(s): Nanosphere, K070597   
3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C2oAn 3402'3aalsa2 7 Xl (m0amvdol</td><td colspan="1" rowspan="1">1-3l58e72m? 9</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">"The Verigene MTHFR NucleicAcid Test is an in vitro diagnosticfor the detection and genotyping ofa single point mutation (C to T atposition 677) of the human 5, 10methylene-tetrahydrofolatereductase gene (MTHFR) inpatients with suspectedthrombophilia, from isolatedgenomic DNA obtained fromwhole blood samples. The test isintended to be used on theVerigene System."The Verigene System is a bench-top molecular diagnosticworkstation that automates the invitro diagnostic analysis anddetection of nucleic acids usinggold nanoparticle probetechnology. The Verigene Systemis intended to be used byexperienced laboratoryprofessionals with training on basiclaboratory techniques and on theuse of the system components.</td><td colspan="1" rowspan="1">The Invader® MTHFR 677 test is anin vitro diagnostic test intended forthe detection and genotyping of asingle point mutation (C to T atposition 677) of the human 5,10-methylenetetrahydrofolate reductase(MTHFR) gene in isolated genomicDNA obtained from whole bloodPotassium EDTA samples frompatients with suspectedthrombophilia.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Purified DNA isolated from humanwhole peripheral blood</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Indications for</td><td colspan="1" rowspan="1">Same as Intended Use</td><td colspan="1" rowspan="1">Same as Intended Use</td></tr><tr><td colspan="1" rowspan="1">Use</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TargetPopulation</td><td colspan="1" rowspan="1">Patients with suspectedthrombophilia</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">SNP discrimination viaoligonucleoptide probes; detectionvia evanescent wave light scatterwith nanoparticles.</td><td colspan="1" rowspan="1">PCR followed by Invader® achievingSNP discrimination viaoligonucleotide probes.</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">The Verigene System consists oftwo instruments, the VerigeneProcessor and the Verigene Reader,and utilizes single-use, disposableTest Cartridges to process andgenotype multiple genes in a DNAsamples in approximately 1.5hours.</td><td colspan="1" rowspan="1">Non-specified, third-partyfluorometer and thermal cycler.</td></tr><tr><td colspan="1" rowspan="1">SoftwareInterface</td><td colspan="1" rowspan="1">Embedded software in closedsystem, integrated graphical userinterface.</td><td colspan="1" rowspan="1">Java-based software installed on astandalone PC capable of convertingraw fluorescence data into genotypecalls.</td></tr><tr><td colspan="1" rowspan="1">Detection Method</td><td colspan="1" rowspan="1">Single-image sensor wherenanoparticles are illuminated usinga fixed-wavelength light source.</td><td colspan="1" rowspan="1">PCR and Fluorescence ResonanceEnergy Transfer (FRET) chemistryfor signal reporting.</td></tr><tr><td colspan="1" rowspan="1">Sample Size</td><td colspan="1" rowspan="1">25μL</td><td colspan="1" rowspan="1">20ul reaction containing 0.25-4ng/ulgDNA extracted from humanperipheral whole blood.</td></tr><tr><td colspan="1" rowspan="1">DetectionProcedure</td><td colspan="1" rowspan="1">Single-image sensor wherenanoparticles are illuminated usinga fixed-wavelength light source.</td><td colspan="1" rowspan="1">Multi-well fluorometer to detect rawfluorescence.</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Detection via evanescent wavelight scatter with nanoparticles.</td><td colspan="1" rowspan="1">PCR and Invader® usingFluorescence Resonance EnergyTransfer (FRET) chemistry for signalreporting.</td></tr><tr><td colspan="1" rowspan="1">Analysis Time</td><td colspan="1" rowspan="1">90 min. processing with 2 minanalysis time.</td><td colspan="1" rowspan="1">~90 min. amplification followed by 1min signal detection. Softwareanalysis post signal detection.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Factor V Leiden DNA Mutation Detection Systems issued on   
March 16, 2004   
Guidance for Industry and FDA Staff - Guidance for the Content of   
Premarket Submissions for Software Contained in Medical Devices issued   
May 11, 2005   
Guidance for Industry and FDA Staff - Format for Traditional and   
Abbreviated $5 1 0 ( \mathbf { k } ) \mathbf { s }$ issued on August 12, 2005

# L.Test Principle:

The Invader® MTHFR 677 test utilizes the Invader Plus® chemistry with DNA isolated from human whole blood, for the detection of the targeted sequence polymorphism. Specifically, the Invader $\mathrm { \ P l u s } ^ { \otimes }$ chemistry utilizes a single-tube, two phase reaction, including target amplification and signal generation (mediated by Invader® chemistry). Invader $\mathrm { \sf P l u s } ^ { \otimes }$ reaction mixes are assembled by combining the MTHFR 677 Oligo Mix, Universal Enzyme Mix, and Universal Buffer. In a 96-well plate, reaction mix is combined with purified genomic DNA samples, as well as four (4) controls included with the test. The No DNA Control is used by the interpretive software to set the "noise" component of the run for "signal-to-noise" calculations. The genotype-specific controls (WT, HET, MUT) ensure reagents were assembled correctly and perform according to the specifications. The 96-well plate is transferred to an appropriately programmed thermal cycler for target amplification and signal generation. In the target amplification phase of the reaction, amplification is carried out using "two-step" cycling conditions (i.e. denaturation & annealing/extension). Following amplification, Taq polymerase is inactivated by a 10 minute incubation at $9 9 ^ { \circ } \mathrm { C } ,$ after which the thermal cycler proceeds to $6 3 \textdegree$ to initiate the signal generation (Invader) phase of the reaction (see Figure 1).

![](images/ef87831590f129233de60ca4dcb0eaa88d4bc92e7ac8ac48042d567a13ffb27c.jpg)  
Figure 1. Invader® Signal Generation Phase

During the signal generation phase, a discriminatory Primary Probe transiently hybridizes to the amplified target sequence along with an Invader® oligonucleotide, to form an overlapping structure. The $5 "$ -end of the Primary Probe includes a $5 ^ { \prime }$ -flap that does not hybridize to the target DNA. The $3 ^ { \prime }$ -nucleotide of the bound Invader® oligonucleotide overlaps the Primary Probe, and does not hybridize to the target DNA. The Cleavase® enzyme recognizes this overlapping structure and cleaves off the unpaired $5 "$ -flap of the Primary Probe, releasing it as a target-specific product. The Primary Probe is designed to have a melting temperature aligned with the Invader reaction temperature so that under the isothermal reaction conditions $( { \sim } 6 3 ^ { \circ } \mathrm { C } )$ the Primary Probes cycle on and off the target DNA. This allows for multiple rounds of Primary Probe cleavage for each DNA target resulting in an accumulation of the number of released $5 "$ -flaps. The released $5 ^ { \circ }$ -flap transiently hybridizes with a corresponding FRET cassette forming an overlapping structure that is recognized and the fluorophore is cleaved from the FRET cassette by the Cleavase® enzyme. The $5 '$ -flap is designed to have a melting temperature aligned with the Invader® reaction temperature, so that the $\varsigma ^ { \star }$ -flaps cycle on and off of the corresponding FRET cassettes. This allows for multiple rounds of FRET cassette cleavage for each $5 '$ . flap, and an accumulation of released fluorophore. When the FRET cassette is cleaved, a fluorophore and quencher are separated, generating detectable fluorescence signal. The format uses two different discriminatory Primary Probes, one for the mutant allele and one for the wild type allele (Figure 1). Each Primary Probe is assigned a unique 5'-flap, and distinct FRET cassette, with a spectrally distinct fluorophore. By design, the released $5 "$ - flaps will bind only to their respective FRET cassettes to generate a target-specific signal, linking the wild type allele with one fluorophore (Fluorescence 1: FAM) and the mutant allele with the second fluorophore (Fluorescence 2: RED).

The Invader® MTHFR 677 software, in combination with Invader Call Reporter™M software, is a data analysis software package developed by Hologic for use with the Invader® MTHFR 677 test. The software package provides a working template for the setup of reaction mixes and sample placement, and following the import of fluorescence data, it determines results and validity for controls and samples. A summary of the Invader Call ReporterTM- Invader® MTHFR 677 package workflow is shown in Figure 2.

![](images/a92679c9d3fb21507a3ae7f6df00bf9d6f5ffcf38ccc57e4defe007726ee9959.jpg)  
Figure 2. Invader Call Reporter™- Invader® MTHFR 677 Package Workflow

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: External Reproducibility (Study #1): Two operators each from three (3) different sites (2 external sites and 1 internal site) performed the testing, in duplicate, over five (5) non-consecutive days for a ten (10) day period using the same testing materials including a panel of nine (9) whole blood samples specific for each of the three (3) possible genotypes (i.e. wild type, heterozygous, homozygous mutant).

Table 2: Inter-laboratory Reproducibility of Invader® 677 Tes   

<table><tr><td rowspan=2 colspan=1>#Site7 - c</td><td rowspan=2 colspan=1>K. 2..</td><td rowspan=2 colspan=1>Sape $..-Uζ</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>First Pass</td><td rowspan=1 colspan=1>f.N991</td><td rowspan=1 colspan=1>Fka </td><td rowspan=1 colspan=1>: : Final...</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Final %Agreement3Samples TesledE</td></tr><tr><td rowspan=1 colspan=1>5€</td><td rowspan=1 colspan=1>No Calls3EQ)</td><td rowspan=1 colspan=1>:...Miscalls</td><td rowspan=1 colspan=1>23Calls</td><td rowspan=1 colspan=1>.No Calls-i(Inalid;.EQ)</td><td rowspan=1 colspan=1>“;Miscalls•  -.</td></tr><tr><td rowspan=2 colspan=1>Site001</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Site002</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Site003</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Lot-to-Lot Reproducibility (Study #9): A total of nine (9) genomic DNA samples (three (3) wild type , three (3) heterozygous and three (3) mutants) were tested in quadruplicate using three (3) different kit lots of the Invader® MTHFR 677 test. The percent agreement between Invader® MTHFR 677 test and sequencing was E $100 \%$ $[ n { = } 1 0 8 )$

<table><tr><td rowspan=1 colspan=7>Table 3: Lot to Lot Reproducibility</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>sD.Pass&#x27;CorrectCalls</td><td rowspan=1 colspan=1>FirstPasasss$3</td><td rowspan=1 colspan=1>Miscall</td><td rowspan=1 colspan=1>FinalSCorrect.Calls</td><td rowspan=1 colspan=1>AAgreement8¥</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>100</td></tr></table>

b. Linearity/assay reportable range: Refer to paragraph D below.

C. Traceability, Stability, Expected values (controls, calibrators, or methods): Real-Time Stability Study (Study #5): Three (3) lots of product in the final configuration are being stored under recommended conditions: (1) $\cdot 3 0 ^ { \circ }$ to $- 1 5 \textdegree C$ (Standard Storage of intermediate components) as well as (2) $+ 2 ^ { \circ }$ to $+ 8 \%$ EY (Standard Storage of Genotype-Specific Controls). Functional testing is performed with samples representing all 3 genotypes in quadruplicate at each time point. The interim test results have demonstrated 7 months stability for the device.

<table><tr><td rowspan=2 colspan=11>Table 4: MTHFR 677 Genotype Results; Real-time Stability2v4                             41                                                           *                      1Sequencing/                              f                         x 3:Sample/                                                          s4:2 ..                                     74E&quot;ToesultT2x.57IResultControl2M3H30F.14             7&#x27;                            *              117*AGGenoty4pe      4:                                                                 3S:            **3E&amp;3$713                                 3</td></tr><tr><td rowspan=1 colspan=1>2v4Sample/7 Control43</td><td rowspan=1 colspan=1>41Sequencing/E2M          3A*GGenotypeE&amp;</td></tr><tr><td rowspan=1 colspan=1>J:$23}$</td><td rowspan=1 colspan=1>*     G÷5</td><td rowspan=1 colspan=1>DTLotn8.</td><td rowspan=1 colspan=1>Lot 23&#x27;.  j .</td><td rowspan=1 colspan=1>L63.Xi$</td><td rowspan=1 colspan=1>.: e.?Lot 1:+</td><td rowspan=1 colspan=1>RLot2:</td><td rowspan=1 colspan=1>3rLot 32</td><td rowspan=1 colspan=1>5:a.Lot 1101</td><td rowspan=1 colspan=1>oEotL</td><td rowspan=1 colspan=1>Lot.</td></tr><tr><td rowspan=1 colspan=1>Control 1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>Control 2</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>Control 3</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=1>gDNA 1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>gDNA 2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>gDNA 3</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>gDNA 4</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=2>:473:...</td><td rowspan=1 colspan=1>ES100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>y:</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>B10O</td><td rowspan=1 colspan=1>i. e12</td><td rowspan=1 colspan=1>To.•</td><td rowspan=1 colspan=1>100, ·!</td><td rowspan=1 colspan=1>3::X,¥</td></tr></table>

Reagent Freeze-Thaw Stability Study (Study #6): Product in the final configuration was subject to 15 freeze-thaw cycles prior to the final thaw at the time of testing. Functional testing was performed using genomic DNA isolated from cell lines, representing all possible genotypes. The percent agreement between the sequencing result and the Invader® MTHFR 677 test were $100 \%$ , therefore demonstrating stability for up to fifteen (15) freeze/thaw cycles.

<table><tr><td rowspan=1 colspan=18>Table 5: Freeze/Thaw Stability of Invader MTHFR 677</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=12>Number ofFreeze/Thaw.Cycles.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>HL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sample1</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>374</td><td rowspan=1 colspan=1>14T</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7R</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>UE</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>:13</td><td rowspan=1 colspan=1>L14K</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Control 1sFL W</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Control 2HET</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Control 3(MUT)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>DNA 3\frac }$WT    </td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>gDNA(HET)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>gDNA 2L MUT) </td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total:   &quot;]</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=7></td><td rowspan=1 colspan=1>aDN</td><td rowspan=1 colspan=2>A samples</td><td rowspan=1 colspan=2>did not occu</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5></td></tr></table>

# d. Detection limit/Analytical Sensitivity and Normal Range (Study #3):

Three (3) genomic DNA samples with different genotypes (i.e. WT, HET, MUT) were extracted from whole blood collected in potassium EDTA. Each sample was diluted to eight different concentrations 0.5, 5, 20, 40, 80, 200, 400, ${ \tt 8 0 0 } \mathrm { n g / }$ EY $\mu \mathrm { L }$ and tested in replicates of forty (40). The recommend range of the assay was determined to be between ${ 5 { - } 8 0 \ n g / \mu \mathrm { L } }$ of input gDNA, based on $100 \%$ concordance of all tested replicates with bi-directional sequencing.

<table><tr><td rowspan=1 colspan=4>.•:                                    y                   1Table 6: Percent Agreement Between Replicates     6.          5..C.-114.t.                                                             rSample ID (Genotype based on Sequencing)</td></tr><tr><td rowspan=1 colspan=1>Input SampleConcentration</td><td rowspan=1 colspan=1>03-4520 (MUT)</td><td rowspan=1 colspan=1>03-4525 (HET)</td><td rowspan=1 colspan=1>03-4524 (WT)</td></tr><tr><td rowspan=1 colspan=1>0.5 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>5 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>20 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>40 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>80 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>200 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>400 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr><tr><td rowspan=1 colspan=1>800 ng/μl</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr></table>

# e. Analytical specificity (Interfering Substances) (Study #4):

Test performance was not affected by addition of the following substances to nine (9) whole blood samples of different genotype (3 WT, 3 HET, 3 MUT) prior to extraction:

√ Heparin (1500 U/dL human whole blood) $\bullet$ Cholesterol $( 3 0 0 ~ \mathrm { m g / d L }$ human whole blood) $\bullet$ Bilirubin $1 0 ~ \mathrm { m g / d L }$ human whole blood) • Hemoglobin (up to $0 . 2 \%$ in whole blood) • Potassium EDTA $\mathrm { 1 . 8 \ m g / m L }$ human whole blood) • Ethanol-based Wash Buffer ( $5 \%$ in DNA sample)

Interfering Substance Code

Substance Concentration / (in blood or DNA sample)

% Agreement with Sequencing Genotype % Agreement with Untreated Sample Invader MTHFR 677 Genotype

<table><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>No Addition Control(Untreated)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Bilirubin 10mg/dl (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>Cholesterol 300mg/dl (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>KEDTA 1.8mg/ml (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>Heparin 1500 U/dI (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F1</td><td rowspan=1 colspan=1>Hemoglobin 0.2% (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>Hemoglobin 0.1% (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F3</td><td rowspan=1 colspan=1>Hemoglobin 0.05% (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>F4</td><td rowspan=1 colspan=1>Hemoglobin 0.025% (blood)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>Buffer AW2 5% (DNA)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>100% (18 of 18)</td><td rowspan=1 colspan=1>PASS</td></tr></table>

f. Pre-Analytical Equivalency Study/Genomic DNA Extraction Reproducibility (Study #7): Thirty (30) human whole blood samples and ten (10) leukocyte depleted whole blood spiked with cell lines were divided and extracted using four (4), commercially available DNA extraction methods (A. Qiagen QIAamp® 96 DNA Blood Kit, B. Qiagen QIAamp® DNA Blood Mini Kit, C. Gentra Generation® Capture Column Kit (Qiagen), D. Roche MagNA Pure LC DNA Isolation Kit I). The 160 extracted DNAs were analyzed in singlicate with one (1) lot of the device. The percent agreement between the Invader® MTHFR 677 test for each extraction method and bi-directional sequencing was $100 \%$ $( n { = } 4 0 )$

<table><tr><td rowspan=1 colspan=7>Table 8a: Pre-Analytical Equivalency</td></tr><tr><td rowspan=1 colspan=1>ractioi</td><td rowspan=1 colspan=1>*amHETested$</td><td rowspan=1 colspan=1>FirstPassCCorrectCal</td><td rowspan=1 colspan=1>First•;3a</td><td rowspan=1 colspan=1>[iscal S</td><td rowspan=1 colspan=1>Ca s</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39*</td><td rowspan=1 colspan=1>100*</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=7>*Sample was removed from study due to loss of traceability of the sampleidentification.</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Table 8b: Genotype Distribution Across Sample Type</td></tr><tr><td rowspan=1 colspan=1>Sample Type           7</td><td rowspan=1 colspan=1>EWT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>*      srotal</td></tr><tr><td rowspan=1 colspan=1>Human Whole Blood</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>9*</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>29*</td></tr><tr><td rowspan=1 colspan=1>LDWB Spiked with Cell Lines**</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>9*</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>39*</td></tr><tr><td rowspan=1 colspan=5>*Sample was removed from study due to loss of traceability of the sample identification.**LDWB=Leukocyte Depleted Whole Blood</td></tr></table>

g. Instrument Equivalency (Study #8): Twenty-nine (29) human whole blood samples and ten (10) leukocyte depleted whole blood samples spiked with cell lines were extracted using two (2) commonly used extraction methods. The extracts were tested with the Invader® MTHFR 677 test using three (3) commercially available thermal cyclers (1. ABI GeneAmp® PCR System 9700 with 96-well gold block, 2. ABI VeritiTM and 3. MJ Research PTC-100) and the raw fluorescent data acquired on three (3) commercially available fluorometers Tecan Inite, B.Tecan Genis a C. BiT $\mathbf { F L x 8 0 0 } .$ Results from the three (3) fluorometers were transferred into the interpretive software and genotype calls compared to bi-directional sequencing.

<table><tr><td rowspan=1 colspan=4>Table 9: Concordance by Instrument</td></tr><tr><td rowspan=2 colspan=1>Fhuorometer</td><td rowspan=1 colspan=3>erma      er         </td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>031</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>78 of 78= 100%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>78 of 78= 100%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td><td rowspan=1 colspan=1>78 of 78 = 100%</td></tr></table>

# 2 Comparison studies:

a. Method comparison: Bi-directional Sequencing (Study #2):

Human whole blood samples $( \mathfrak { n } = 3 6 1 )$ underwent DNA extraction and subsequent bi-directional DNA sequence analysis. The same DNA samples were then analyzed using the Invader® MTHFR 677 test. The observed agreement between the Invader® MTHFR 677 test and bidirectional DNA sequencing was $100 \%$ (359/359). The first run agreement with bi-directional sequencing was $9 9 . 4 5 \%$ (359/361).

<table><tr><td rowspan=1 colspan=5>Table 10a: Agreement between the Invader® MTHFR 677 Test and Bi-directional DNA Sequencing</td></tr><tr><td rowspan=1 colspan=1>qX          $ 4xa}MTHFR677Genotype</td><td rowspan=1 colspan=1>¤ifNumberitested  </td><td rowspan=1 colspan=1>6551Number ofvalid resultson.a:Jy7         *</td><td rowspan=1 colspan=1>Number of5 . 5 Correctgenotypecalls on FirstRun7r&#x27;:t             6*</td><td rowspan=1 colspan=1>s5First RunAgreement</td></tr><tr><td rowspan=1 colspan=1>HomozygousWild Type(GG)</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>178**</td><td rowspan=1 colspan=1>178**</td><td rowspan=1 colspan=1>98.89%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous(GA)</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HomozygousMutant(AA)</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>359**</td><td rowspan=1 colspan=1>359**</td><td rowspan=1 colspan=1>99.45%</td></tr><tr><td rowspan=1 colspan=5>* Genotype determined through bi-directional DNA sequencing** Two samples failed to generate valid results. These samples were reported asinvalid (EQ) and no genotype calls were assigned by the interpretive software.The EQ result was used to determine the First Run Agreement.</td></tr></table>

<table><tr><td rowspan=1 colspan=15>Table 10b: Agreement between thc Invader® MTHFR 677 Test andBi-directional DNA Sequencing</td></tr><tr><td rowspan=1 colspan=6>Genotype*</td><td rowspan=1 colspan=2>First Pass Results</td><td rowspan=1 colspan=5>Genotype</td><td rowspan=1 colspan=2>Final Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>No-Call</td><td rowspan=1 colspan=1>Mis-Call</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95%LCB*</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>No-Call</td><td rowspan=1 colspan=1>Mis-Call</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>95%LCB</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.45</td><td rowspan=1 colspan=1>96.04</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.45</td><td rowspan=1 colspan=1>96.04</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>97.16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>97.16</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>96.18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>96.18</td></tr></table>

\*Genotype determined through bi-directional DNA sequencing. \*\*Lower boundary of the $9 5 \%$ confidence interval for each genotype. Calculated across all samples.

3. External Reproducibility studies:

a. Clinical Sensitivity: please refer to section 1d above.   
b. Clinical specificity: please refer to section le above.

4. Expected values/Reference range: (Prevalence) MTHFR 677: 30-40% Caucasians

# N. System Descriptions:

1. Modes of Operation: Closed System

2 Software: FDA has reviewed applicant's Hazard Analysis and software development processes for this line of product type. Yes__X_ _or No

3. Specimen Identification:

Manual Labeling

4. Specimen Sampling and Handling:

DNA should be extracted using a validated DNA extraction method that generates DNA concentration range of greater than $\mathsf { S n g / \mu l }$ .

5. Quality Control:

Each test contains positive and negative controls to assure proper functioning of the system: Failure of any controls will be indicated as "Invalid" in the test results section of the report. The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.   
Positive Control: The genotype controls (WT, HET, MUT) ensure reagents were assembled correctly and perform according to the specifications.   
Negative Control: The No DNA Control is used by the interpretive software to set the "noise" component of the run for "signal-to-noise" calculations.   
Hardware and Software Controls:   
The genotyping test result will not be reported for any sample for which a positive or negative control failure occurs.

Proposed Labeling: The labeling is sufficient and satisfies the requirements of 21 CFR Part 809.10.

# P. Conclusion:

The submitted information in this 510 (k) notification is complete and supports a substantial equivalence decision.

Hologic Inc.   
Third Wave Technologies   
c/o Mr. Randall J. Covill   
Senior Specialist Regulatory Affairs 502 South Rosa Road   
Madison, WI 53719

Re: k100987 Trade/Device Name: Invader $\textsuperscript { \textregistered }$ MTHFR 677 Regulation Number: 21 CFR $\ S 8 6 4 . 7 2 8 0 $ Regulation Name: Factor V Leiden DNA Mutation Detection Systems Regulatory Class: Class II Product Code: OMM Dated: March 31, 2011 Received: April 7, 2011

Dear Mr. Covill:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class II (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice reuirements as et forth in the quality systems (S) regulation (1 CFR Part 820). Thi letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

Tama In than

Maria M. Chan, Ph.D   
Director   
Division of Immunology and Hematology Devices   
Office of In Vitro Diagnostic Device Evaluation and Safety   
Center for Devices and Radiological Health

# Indication for Use Statement

510(k) Number (if known): _k100987

Device Name: Invader MTHFR 677 test

Indication for Use:

The Invader $\textsuperscript { \textregistered }$ MTHFR 677 test is an in vitro diagnostic test intended for the detection and genotyping of a single point mutation (C to T at position 677) of the human 5,10- methylenetetrahydrofolate reductase (MTHFR) gene in isolated genomic DNA obtained from whole blood potassium EDTA samples from patients with suspected thrombophilia.

Prescription Use___X_ AND/OR Over-The-Counter Use (Part 21 CFR 801 Subpart D) (21 CFR 801 Subpart C)

Concurrence of CDRH, Office of In Vitro Diagnostic Devices (OIVD)

Lnn Divislon Sign-Off Office of In Vitro Dlagnostic Device Evaluation and Safety 5101) K100987