# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

LSI D7S486 probe target on chromosome 7q31 and the CEP 7 probe target on chromosome 7p11.1-q11.1

# E. Applicant:

# G. Regulatory Information:

1. Regulation section: 21 CFR 864.1870, Early growth response 1 (EGFR1) gene fluorescence in-situ hybridization (FISH) test system for specimen characterization.

2. Classification: II

3. Product code: PFG - DNA FISH probe kit for specimen characterization, human chromosome, hematological disorders

4. Panel:

Hematology and Pathology Devices Panel

# H. Intended Use:

1. Intended use(s):

The Vysis D7S486/CEP 7 FISH Probe Kit is a device intended for specimen characterization, and detects the LSI D7S486 probe target on chromosome 7q31 and the CEP 7 probe target on chromosome 7p11.1-q11.1 in bone marrow and peripheral blood specimens from patients with acute myeloid leukemia or myelodysplastic syndrome. The assay results are intended to be interpreted by a qualified pathologist or cytogeneticist. This device is not intended for high-risk uses such as selecting therapy, predicting therapeutic response or disease screening. The use of this device for diagnosis, prognosis, monitoring, or risk assessment has not been established.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s):

For prescription use only, with the following labeling warning:

• The assay results are intended to be interpreted only by a qualified pathologist or cytogeneticist.   
• The Vysis D7S486/CEP7 FISH Probe Kit – is not for high-risk uses such as selecting therapy, predicting therapeutic response or disease screening.   
• The use of this product for diagnosis, prognosis, monitoring, or risk assessment has not been established.   
• Caution: Federal law restricts this device to sale by or on the order of a physician or other practitioner licensed by the law of the State in which he practices, to use or order the use of the device.

4. Special instrument requirements:

Fluorescence microscope equipped with appropriate excitation and emission filters.

# I. Device Description:

The Vysis D7S486/CEP 7 FISH Probe Kit is a device intended for specimen characterization. The device uses DNA FISH probe technology to detect the LSI D7S486 probe target on chromosome 7q31 and the CEP 7 probe target on chromosome 7p11.1-q11.1 in bone marrow and peripheral blood specimens. The Vysis D7S486/CEP 7 FISH Probe Kit consists of a mixture of two DNA FISH probes (item 1 below) and four general purpose reagents (items 2 through 5 below) which are sufficient to process 20 assays:

1. Vysis LSI D7S486 SpectrumOrange and CEP 7 SpectrumGreen Probes: a. The SpectrumOrange labeled LSI D7S486 probe is approximately $3 0 8 \mathrm { k b }$ in

length (chr7:115983468-115675366; February 2009 Assembly UCSC Human Genome Browser). b. The SpectrumGreen labeled CEP 7 probe targets the D7Z1 alpha satellite sequence at the centromere of chromosome 7.

2. Vysis LSI/WCP Hybridization Buffer   
3. DAPI II Counterstain   
4. NP-40   
5. 20X SSC

# J. Substantial Equivalence Information:

1. Predicate device name(s): Vysis EGR1 FISH probe Kit- SC (Specimen Characterization)

2. Predicate 510(k) number(s): K123951

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Indications for Use</td><td rowspan=1 colspan=1>For specimencharacterization frompatients with acute myeloidleukemia and myeloplasticsyndrome</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Fluorescence In SituHybridization</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Kit components</td><td rowspan=1 colspan=1>DNA fluorophore labeledProbes,Vysis LSI/WCPHybridization buffer, DAPIII,NP-40 and20XSSC</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Detection of LSI D7S486probe target on chromosome7q31 and the CEP 7 probetarget on chromosome7p11.1-q11.1</td><td colspan="1" rowspan="1">Detection of LSI EGR1probe target onchromosome 5q.</td></tr><tr><td colspan="1" rowspan="1">Upper reference limit</td><td colspan="1" rowspan="1">4.5% 1 Red, 1 Green (1R1G)6.5% 1 Red, 2 Green (1R2G)</td><td colspan="1" rowspan="1">6%</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Bone marrow and peripheralblood</td><td colspan="1" rowspan="1">Bone marrow</td></tr></table>

K. Standard/Guidance Document Referenced (if applicable): Not applicable

# L. Test Principle:

FISH is a technique that allows visualization of specific nucleic acid sequences within a cellular preparation. Specifically, FISH involves precise annealing of a single-stranded, fluorophore-labeled DNA probe to a complementary target sequence. Hybridization of the probe with the cellular DNA is visible using fluorescence microscopy.

Bone marrow and peripheral blood cell specimens from patients are attached to microscope slides using standard cytogenetic procedures. The resulting DNA specimen is denatured to single-stranded form and subsequently allowed to hybridize with the probes of the Vysis D7S486/CEP 7 FISH Probe Kit. Following hybridization, the unbound probe is removed by a series of washes, and the nuclei are counterstained with DAPI II, a DNA-specific stain that fluoresces blue. Hybridization of the Vysis LSI D7S486 SpectrumOrange/CEP 7 SpectrumGreen Probes is viewed using a fluorescence microscope equipped with appropriate excitation and emission filters, allowing visualization of the orange and green fluorescent signals.

In a cell with normal copy numbers of the Vysis LSI D7S486 SpectrumOrange/CEP 7 SpectrumGreen probe targets, two orange (2R) signals (D7S486) and two green (2G) signals (CEP 7) will be expected. In a cell having only one copy of chromosome 7, 1 orange, 1 green (1R1G) pattern is expected, and in a cell with loss of 7q, 1 orange, 2 green (1R2G) pattern is expected.

FISH results are interpreted by utilizing appropriate controls and analytical techniques as well as taking into consideration other clinical and diagnostic test data.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: Repeatability and reproducibility of the Vysis D7S486/CEP 7 FISH Probe Kit were tested as shown in the table below:

<table><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Study Protocol</td><td rowspan=1 colspan=1>Conclusion</td></tr><tr><td rowspan=1 colspan=1>Precision andReproducibility:Intra-Day,Inter-Day andInter-Site</td><td rowspan=1 colspan=1>Testing PanelNumber of bone marrow specimens: 6 panelmembers 2 high positive panel members (High Positive 1</td><td rowspan=1 colspan=1>Acceptancecriteria met</td></tr></table>

<table><tr><td>- 2 low positive panel members (Low Positive 1 and 2) 2 negative panel members (Negative 1 and 2) •Number of peripheral blood specimens: 6 panel members and 2 with &gt;45% positivity) 2 low positive panel members (Low Positive 1 and 2) 2 negative panel members (Negative 1 and 2) Study Design Number of test sites: 3 Number of testing days: 5 non-consecutive Number of replicates: 2 Number of lots tested: 1 Signal Patterns 1 Red, 1 Green (1R1G) 1 Red, 2 Green (1R2G) Number of technologists: a minimum of 2 and a maximum of 3 (if third reader needed per package insert) Number of nuclei evaluated per panel by each technologist: 100 nuclei Pre-specified acceptance criteria:</td><td>2 high positive panel members (High Positive 1 High Positive - ≥95% for the high positive specimen category for each site Negative - ≥90% for the negative specimen category across all sites with no more than 3 discordant results occurring at one site Testing panel same as above Acceptance criteria met Number of lots tested: 3 Number of replicates: 4</td></tr></table>

<table><tr><td></td><td>Pre-specified acceptance criteria: High Positive - ≥ 95% for the high positive specimen category across all lots Negative - ≥ 87% for the negative specimen category across all lots with no more than 2 discordant results occurring for 1 lot</td><td></td></tr></table>

Overall Agreement Site-to-Site for Bone Marrow Specimen   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Number</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Signal</td><td rowspan=1 colspan=1>Agreea</td><td rowspan=1 colspan=1>Disagreeb</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>88%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>97%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr></table>

a Agree is number of concordant slides. Disagree is number of discordant slides.

Overall Agreement Site-to-Site for Peripheral Blood Specimen   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Number</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Signal</td><td rowspan=1 colspan=1>Agreea</td><td rowspan=1 colspan=1>Disagreeb</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>93%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td></tr></table>

a Agree is number of concordant slides. Disagree is number of discordant slides.

Site-to-Site Analysis of Variance Components for Bone Marrow Specimen   

<table><tr><td colspan="1" rowspan="2">Sample</td><td colspan="1" rowspan="2">Signal</td><td colspan="1" rowspan="2">N</td><td colspan="1" rowspan="2">mean</td><td colspan="1" rowspan="1">Within Day(Component)</td><td colspan="1" rowspan="1">Between Day(Component)</td><td colspan="1" rowspan="1">Between Site(Component)</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td></tr><tr><td colspan="1" rowspan="1">Negative 1</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.43</td></tr><tr><td colspan="1" rowspan="1">Negative 2</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.47</td></tr><tr><td colspan="1" rowspan="1">Low Positive 1</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">10.5</td><td colspan="1" rowspan="1">2.11</td><td colspan="1" rowspan="1">1.41</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">2.78</td></tr><tr><td colspan="1" rowspan="1">Low Positive 2</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">10.0</td><td colspan="1" rowspan="1">3.19</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">2.33</td><td colspan="1" rowspan="1">3.95</td></tr><tr><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">43.9</td><td colspan="1" rowspan="1">6.76</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">7,63</td><td colspan="1" rowspan="1">10.19</td></tr><tr><td colspan="1" rowspan="1">High Positive 2</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">4.61</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">5.44</td><td colspan="1" rowspan="1">7.13</td></tr><tr><td colspan="1" rowspan="1">Negative 1</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">0.84</td></tr><tr><td colspan="1" rowspan="1">Negative 2</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.44</td></tr><tr><td colspan="1" rowspan="1">Low Positivel</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">8.9</td><td colspan="1" rowspan="1">2.61</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">1.19</td><td colspan="1" rowspan="1">2.87</td></tr><tr><td colspan="1" rowspan="1">Low Positive2</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">9.3</td><td colspan="1" rowspan="1">2.43</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">2.43</td></tr><tr><td colspan="1" rowspan="1">High Positive 1</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">48.3</td><td colspan="1" rowspan="1">6.20</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">8.06</td><td colspan="1" rowspan="1">10.25</td></tr><tr><td colspan="1" rowspan="1">High Positive 2</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">43.9</td><td colspan="1" rowspan="1">3.78</td><td colspan="1" rowspan="1">3.97</td><td colspan="1" rowspan="1">4.19</td><td colspan="1" rowspan="1">6.90</td></tr></table>

Site-to-Site Analysis of Variance Components for Peripheral Blood Specimen   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Signal</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>mean</td><td rowspan=1 colspan=1>Within Day(Component)</td><td rowspan=1 colspan=1>Between Day(Component)</td><td rowspan=1 colspan=1>Between Site(Component)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=1 colspan=1>Negative 1</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.71</td></tr><tr><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.49</td></tr><tr><td rowspan=1 colspan=1>Low Positive 1</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>2.46</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>2.03</td><td rowspan=1 colspan=1>3.19</td></tr><tr><td rowspan=1 colspan=1>Low Positive 2</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>2.60</td></tr><tr><td rowspan=1 colspan=1>High Positive 1</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>42.1</td><td rowspan=1 colspan=1>3.26</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>5.82</td><td rowspan=1 colspan=1>6.98</td></tr><tr><td rowspan=1 colspan=1>High Positive 2</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>52.8</td><td rowspan=1 colspan=1>4.07</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>4.95</td></tr><tr><td rowspan=1 colspan=1>Negative 1</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.72</td></tr><tr><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.57</td></tr><tr><td rowspan=1 colspan=1>Low Positive1</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1>2.61</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>2.67</td></tr><tr><td rowspan=1 colspan=1>low Positive2</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>2.35</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>2.42</td></tr><tr><td rowspan=1 colspan=1>High Positive 1</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>44.8</td><td rowspan=1 colspan=1>3.77</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>5.35</td><td rowspan=1 colspan=1>6.55</td></tr><tr><td rowspan=1 colspan=1>High Positive 2</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>3.90</td><td rowspan=1 colspan=1>3.17</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>5.29</td></tr></table>

Lot-to-Lot Overall Agreement for Bone Marrow Specimen   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Signal</td><td rowspan=1 colspan=3>Number</td><td rowspan=1 colspan=1>PercentAgreement</td></tr><tr><td rowspan=1 colspan=1>Agree</td><td rowspan=1 colspan=1>Disagree</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>88%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>92%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Lot-to-Lot Overall Agreement for Peripheral Blood Specimen

<table><tr><td colspan="1" rowspan="2">Sample</td><td colspan="1" rowspan="2">Signal</td><td colspan="3" rowspan="1">Number</td><td colspan="1" rowspan="1">PercentAgreement</td></tr><tr><td colspan="1" rowspan="1">Agree</td><td colspan="1" rowspan="1">Disagree</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">1R2G</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">96%</td></tr><tr><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">1R1G</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">100%</td></tr></table>

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Kit stability The kit stability was assessed as detailed in the table below:

<table><tr><td>Study</td><td>Study Protocol</td><td>Conclusion</td></tr><tr><td>Stability</td><td>Attributes evaluated: signal intensity, specificity, background, and cross- hybridization Number of lots of the test device: 3 • Number of specimens: 1 (Normal lymphocyte slides) Number of replicates: 3 Components of the kit: Vysis LSI D7S496/CEP 7 Probe LSI/WCP Hybridization Buffer 20X SSC DAPI II</td><td>Acceptance criteria met for 12 month stability</td></tr><tr><td>Transport and Temperature Extreme (TTE) and</td><td>• Device components were removed from -20°C and placed at 30° (±2) for 4 hours ± 15 minutes and 3 cycles of 24-48 hours on dry ice and then at 30° (±2°C) for 4 hours ±15 minutes 1 lot of the device 3 specimens</td><td>Acceptance criteria met</td></tr><tr><td>In-Use Freeze- Thaw Stability</td><td>A series of 20 freeze-thaw cycles was performed on the probes, hybridization buffer and DAPI II counterstain. 1 lot of the device 3 specimens Attributes evaluated: Same as above Pre-specified acceptance criteria:</td><td>Acceptance criteria met throughout and at the end of 20 freeze-thaw cycles</td></tr><tr><td>hybridization Signal Stability</td><td>• A single slide from each of 3 bone marrow and 3 peripheral blood specimens was tested at baseline and then stored at -20°C ± 10°C protected from light •The samples were tested at the following time points: Day 1, Day 8, Day 21, and Day 28 1 lot of the device • Attributes evaluated: Same as above •Pre-specified acceptance criteria: Same as above</td><td>Post-hybridization signal stability met acceptance criteria at all time points up to Day 28 which support 3 weeks post-hybridized slides stability.</td></tr><tr><td>Probe Photostability</td><td>Procedure: The Vysis D7S486 SpectrumOrange/CEP 7 SpectrumGreen probes were continuously exposed to white fluorescent light (to mimic standard laboratory conditions) at 15-30°C for difference lengths of time. Time intervals: 0 hour, 24 hours, 72 hours, 1 week, 2 weeks, 3 weeks, 4 weeks, 6 weeks, and 8 weeks Number of lots of the test device: 1 Attributes evaluated: Same as above Pre-specified acceptance criteria: Same as above</td><td>Acceptance criteria met for Photo-stability of 8 weeks.</td></tr></table>

Controls:

There are no external calibrators or controls associated with this device. For any FISH probe, the best control to assure the correct hybridization occurs is signal production at the intended target sequence.

# d. Detection limit:

The analytical sensitivity of the Vysis D7S486/CEP 7 Probes was established using interphase nuclei from 25 bone marrow and 25 peripheral blood specimens from either karyotypically normal individuals or patients with a 1R1G signal pattern and 1R2G signal pattern. The orange and green signal patterns of 200 nuclei for each specimen type were evaluated by 2 technologists each scoring 100 nuclei per specimen for a total of 200 nuclei per specimen. A total of 5000 nuclei were scored from normal specimens. The Vysis D7S486/CEP 7 FISH Probe Kit has an analytical sensitivity of $9 8 . 1 \%$ (4903/5000) $( 9 5 \% \mathrm { C I } 9 7 . 6  – 9 8 . 4 \% )$ for bone marrow and $9 8 . 5 \%$ (4923/5000) $9 5 \% \mathrm { C I } 9 8 . 1 { - } 9 8 . 8 \% )$ for peripheral blood.

<table><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Nuclei withExpected SignalPattern</td><td rowspan=1 colspan=1>Total NumberofNuclei Scored</td><td rowspan=1 colspan=1>Analytical Sensitivity(95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>LSID7S486</td><td rowspan=1 colspan=1>Bonemarrow</td><td rowspan=1 colspan=1>4903</td><td rowspan=1 colspan=1>5000</td><td rowspan=1 colspan=1>98.1 (97.6, 98.40)</td></tr><tr><td rowspan=1 colspan=1>LSID7S486</td><td rowspan=1 colspan=1>Peripheralblood</td><td rowspan=1 colspan=1>4923</td><td rowspan=1 colspan=1>5000</td><td rowspan=1 colspan=1>98.5 (98.1, 98.88)</td></tr></table>

# e. Analytical specificity:

Analytical specificity is defined as the percentage of signals that hybridize to the correct locus and no other location. The analytical specificity of the Vysis LSI D7S486 SpectrumOrange/CEP 7 SpectrumGreen Probes for their respective chromosome target loci (7q31 and 7p11.1-q11.1, respectively) was established using metaphase chromosomes prepared from peripheral blood cultures of 4 male and 1 female karyotypically normal specimens. The hybridization location of each FISH signal on chromosomes of 100 consecutive metaphase nuclei was evaluated by 1 technologist for a total of 200 target loci per probe.

For each probe and sample, the number of metaphase chromosome FISH signals hybridized to the correct locus and the number of metaphase chromosome FISH signals hybridized to the incorrect locus were enumerated. The analytical specificity of each probe was calculated as the number of metaphase chromosome FISH signals hybridized to the correct locus divided by the total number of metaphase chromosome FISH signals hybridized and multiplied by 100 to give a percentage.

The analytical specificity of the Vysis D7S486/CEP 7 FISH Probe Kit was considered acceptable if greater than or equal to $9 9 \%$ and the D7S486/CEP 7 probes was $100 \%$ (200/200) $( 9 5 \% \mathrm { C I } 9 8 – 1 0 0 \% )$ .

<table><tr><td rowspan=2 colspan=1>Probe</td><td rowspan=2 colspan=1>CorrectTarget Locus</td><td rowspan=1 colspan=2>Number of MetaphaseChromosome Signals</td><td rowspan=2 colspan=1>AnalyticalSpecificity(95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Hybridized tothe CorrectTarget Locus</td><td rowspan=1 colspan=1>TotalHybridizedSignals</td></tr><tr><td rowspan=1 colspan=1>LSI D7S486</td><td rowspan=1 colspan=1>7q31</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100 %(98, 100)</td></tr><tr><td rowspan=1 colspan=1>CEP 7</td><td rowspan=1 colspan=1>7p11.1-q11.1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100 %(98, 100)</td></tr></table>

f. Assay cut-off: Same as reference range (see #5 below)

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity: Not applicable

b. Clinical specificity: Not applicable

c. Other clinical supportive data (when a. and b. are not applicable):

The sponsor provided two peer-reviewed published papers to support the clinical validity of the device in characterizing bone marrow and peripheral blood specimens from patients with AML or MDS. Sponsor provided additional a third paper that used specimen from patients with MMM. The studies described in the three papers used the Abbott Vysis D7S486/CEP 7 FISH Probe Kit. This information is listed in the table below:

<table><tr><td rowspan=1 colspan=1>Conditions</td><td rowspan=1 colspan=1>Data Source 1Vance et al</td><td rowspan=1 colspan=1>Data Source 2Cherry et al2</td><td rowspan=1 colspan=1>Data Source 3Tefferi et al3</td></tr><tr><td rowspan=1 colspan=1>Was the specific device (probe) under reviewin this submission used in the study?</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Was the specimen type in the studyrepresentative of the claimed specimen type(s)</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Target population(disease status)</td><td rowspan=1 colspan=1>AML</td><td rowspan=1 colspan=1>MDS</td><td rowspan=1 colspan=1>MMM</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Upper referencelimit(percentage andper 200 nuclei)</td><td rowspan=1 colspan=2>(1R1G signal pattern)</td><td rowspan=1 colspan=1>4.5%9/200 nuclei</td><td rowspan=1 colspan=1>4.5%9/200 nuclei</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=2>(1R2G signal pattern)</td><td rowspan=1 colspan=1>6.5%13/200 nuclei</td><td rowspan=1 colspan=1>6.5%13/200 nuclei</td><td rowspan=1 colspan=1>6.5%13/200 nuclei</td></tr><tr><td rowspan=2 colspan=1>Total Numberof specimenstested for eachclaimed type</td><td rowspan=1 colspan=2>Bone Marrow</td><td rowspan=1 colspan=1>179</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=2>Peripheral Blood</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=4 colspan=1>Number ofspecimens witha positive proberesult</td><td rowspan=2 colspan=1>BoneMarrow</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>PeripheralBlood</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=4 colspan=1>Range ofpositive proberesults</td><td rowspan=2 colspan=1>BoneMarrow</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>70-96%</td><td rowspan=1 colspan=1>23 &amp; 87.5%</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>58-93%</td><td rowspan=1 colspan=1>22.5-44%</td><td rowspan=1 colspan=1>9 and 16%</td></tr><tr><td rowspan=2 colspan=1>PeripheralBlood</td><td rowspan=1 colspan=1>1R1G</td><td rowspan=1 colspan=1>11%</td><td rowspan=2 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>1R2G</td><td rowspan=1 colspan=1>37-96%</td><td rowspan=1 colspan=1>27%</td></tr></table>

AML acute myeloid leukemia; ${ \bf M D S } { = }$ myelodysplastic syndrome; MMM $\circeq$ myelofibrosis with myeloid metaplasia.

# REFERENCES

1. Vance GH, Kim H, Hicks GA, et al. Utility of interphase FISH to stratify patients into cytogenetic risk categories at diagnosis of AML in an Eastern Cooperative Oncology Group (ECOG) clinical trial (E1900) Leuk Res 2007; 31:605-09.   
2. Cherry A, Brockman S, Paternoster, S et al. Comparison of interphase FISH and metaphse cytogenetics to study myelodysplastic syndrome: an Eastern Cooperative Oncology Group (ECOG) study. Leukemia Research 2003; 27: 1085–1090.   
3. Tefferi A, Meyer R, Wyatt W, et al. Comparison of peripheral blood interphase cytogenetics with bone marrow karyotype analysis in myelofibrosis with myeloid metaplasia. British J Hematology 2001; 115: 316-319.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The upper reference limit is defined as the maximum quantity of scoreable interphase nuclei with an altered signal pattern from karyotypically normal specimens. The upper reference limit is expressed in terms of a percentage, or the actual number of a specific atypical nuclear FISH signal pattern per the standard number of nuclei tested.

The upper reference limit for $4 . 5 \%$ or nine 1R1G patterns per 200 scoreable interphase nuclei, and the upper reference limit is $6 . 5 \%$ or thirteen 1R2G patterns per 200 scoreable interphase nuclei. The $4 . 5 \%$ and $6 . 5 \%$ upper reference limits were adopted from the publication of Vance et al, who utilized the Vysis D7S486/CEP 7 FISH Probe.1

In order to validate the $4 . 5 \%$ and $6 . 5 \%$ upper reference limits, the Vysis D7S486/CEP 7 FISH assay was performed on interphase nuclei from 25 bone marrow and 25 peripheral blood specimens from either karyotypically normal individuals or patients lwith signal patterns of 1R1G or 1R2G. The orange and green signal patterns of 200 nuclei for each specimen type were evaluated by each of two technologists scoring 100 nuclei per specimen.

Among the 25 karyotypically normal specimens for both peripheral blood and bone marrow, none produced 1R1G and 1R2G signals above the $4 . 5 \%$ and $6 . 5 \%$ upper reference limits, respectively.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.