# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION

A. 510(k) Number: k132978   
B. Purpose for Submission: New Device   
C. Measurand: Factor V Leiden single nucleotide variant G1691A and the Factor II single nucleotide variant G20210A   
D. Type of Test: DNA Genotyping Test   
E. Applicant: Sequenom, Inc.   
F. Proprietary and Established Names: Impact $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test on the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System

# G. Regulatory Information:

Regulation section: 21 CFR§864.7280; Factor V Leiden DNA Mutation Detection Systems

2. Classification: Class II

3. Product code: PHJ; System, Mass Spectrometry, Multiplex Genotyping, Hereditary Thrombophilia Related Mutations

4. Panel: Hematology (81)

# H. Intended Use:

1. Intended use(s):

The IMPACT $\mathrm { D x ^ { T M } }$ Factor V Leiden and Factor II Genotyping Test is a qualitative in vitro diagnostic device intended for use in the detection and genotyping of a single point mutation (G1691A, referred to as the Factor V Leiden mutation or FVL) of the Factor V gene, located on Chromosome 1q23, and a single point mutation (G20210A) of the prothrombin gene (referred to as Factor II or FII), located on Chromosome 11p11-q12, from genomic DNA isolated from EDTA anti-coagulated human whole blood samples. The test is to be performed on the IMPACT $\mathrm { D x ^ { T M } }$ System and is indicated for use as an aid in diagnosis of patients with suspected thrombophilia.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): For Prescription Use Only

4. Special instrument requirements:

The IMPACT $\mathrm { D x ^ { T M } }$ Factor V Leiden and Factor II Genotyping Test, is designed for use with the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System. The IMPACT $\mathrm { D x ^ { T M } }$ System includes the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO (NANO), IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA (MA), TYPER $\mathrm { D x } ^ { \mathrm { T M } }$ Software (TYPER $\mathrm { D x } ^ { \mathbf { T M } }$ ), System consumables (SpectroCHIP $\textsuperscript { \textregistered }$ Arrays (Chips), Clean Resin, 3-Point Calibrant (Calibrant).

# I. Device Description:

The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test is a multiplexed genetic testing system for parallel detection and genotyping of two point mutations: G1691A of the Factor V gene and G20210A of the Factor II gene, from genomic DNA isolated from EDTAanti-coagulated human whole blood specimens. The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II genotyping Test is comprised of the following assay reagents, instruments, software and consumables:

IMPACT $\mathrm { D x ^ { T M } }$ Factor V Leiden and Factor II Primer set (FVL/FII PCR primers; FVL/FII Extend Primers), Impact $\mathrm { D x } ^ { \mathrm { T M } }$ PCR Reagent Set (10X PCR Buffer dUPT/DNTP Mix, $2 5 \mathrm { m M }$ , $\mathrm { M g C l } _ { 2 }$ , $2 5 ~ \mathrm { m M }$ , Uracil-DNA Glycosylase, $5 \mathrm { U } / \mu \mathrm { l }$ , IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ PCR Enzyme, $5 \mathrm { U } / \mu \mathrm { l } \dot { \mu }$ ), and IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Extend Reagent Set (Buffer Plus, 10X, Termination Mix, $1 0 \mathrm { m M }$ , thermosequenase, $3 2 \mathrm { U } / \mu \mathrm { l }$ , Shrimp Alkaline Phosphatase (SAP) Buffer, Shrimp Alkaline Phosphatase (SAP) Enzyme, $1 . 7 \mathrm { U } / \mu \mathrm { l } $ ).

The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO is a self-contained, enclosed instrument that uses computercontrolled robotics to transfer nanoliter volumes of analyte from a 96-well microtiter plate onto the consumable SpectroCHIP $\textsuperscript { \textregistered }$ array (CHIP). The instrument includes integrated computer pre-loaded with the Nanodispenser software.

The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA is a bench top spectrometer that processes the chips by means of matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS) analysis. The instrument includes an integrated computer pre-loaded with TYPER $\mathrm { D x ^ { T M } }$ software, a monitor, and a firewall for communication with the $\mathrm { D x } ^ { \mathrm { T M } }$ NANO. The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA includes Clean Resin Set and 3-point Calibrant.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Roche Factor V Leiden Kit Roche Factor II (Prothrombin) G20210A Kit

2. Predicate 510(k) number(s): k033607 k033612

3. Comparison with predicate:

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Impact DxTM Factor VLeiden and Factor IIGenotyping Test</td><td colspan="1" rowspan="1">Roche Factor VLeiden Kit</td><td colspan="1" rowspan="1">Roche Factor II(Prothrombin)G20210A kit</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Qualitative in vitrodiagnostic genotyping testfor the detection of FactorII and Factor V alleles</td><td colspan="1" rowspan="1">Qualitative in vitrodiagnosticgenotyping test forthe detection ofFactor V alleles</td><td colspan="1" rowspan="1">Qualitative in vitrodiagnosticgenotyping test forthe detection ofFactor II alleles</td></tr><tr><td colspan="1" rowspan="1">Indications forUse</td><td colspan="1" rowspan="1">Aid in the diagnosis ofpatients with suspectedthrombophilia</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Purified DNA fromEDTA anti-coagulatedhuman blood samples</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SamplePreparation</td><td colspan="1" rowspan="1">DNA extraction andpurification</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Alleles detected</td><td colspan="1" rowspan="1">Factor V Leiden(G1691A) and Factor II(G20210A)</td><td colspan="1" rowspan="1">Factor V Leiden(G1691A)</td><td colspan="1" rowspan="1">Factor IIProthrombin(G20210A)</td></tr><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">IMPACT DxTM System</td><td colspan="1" rowspan="1">Roche LightCycler</td><td colspan="1" rowspan="1">Roche LightCycler</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">SNP Discrimination byallele-specific singlenucleotide extensioncoupled with MALDI-TOF mass spectrometry</td><td colspan="1" rowspan="1">Fluorogenicdetection of PCR-amplificationproducts by meltingcurve analysis</td><td colspan="1" rowspan="1">Fluorogenicdetection of PCR-amplificationproducts by meltingcurve analysis</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">IMPACT DxTM System</td><td colspan="1" rowspan="1">Roche LightCycler®</td><td colspan="1" rowspan="1">Roche LightCycler®</td></tr><tr><td colspan="1" rowspan="1">TechnologicalDetectionPrinciples</td><td colspan="1" rowspan="1">Genotyping test forsimultaneous detectionof PCR-amplified DNAsequences (multiplexsystem)</td><td colspan="1" rowspan="1">Genotyping test forPCR-amplifiedDNA sequences(monoplex)</td><td colspan="1" rowspan="1">Genotyping test forPCR-amplified DNAsequences(monoplex)</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal control persample plus externalpositive and negativecontrols per run</td><td colspan="1" rowspan="1">External positiveand negativecontrols per run</td><td colspan="1" rowspan="1">External positive andnegative controls perrun</td></tr><tr><td colspan="1" rowspan="1">Calibrant</td><td colspan="1" rowspan="1">3 point calibrator</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

FDA Class II Special Controls Guidance Document, Factor V Leiden DNA mutation detection system.

CLSI EP9-A2, Method Comparison and Bias Estimation Using Patient Samples; Approved Guideline— $- 2 ^ { \mathrm { n d } }$ Edition.

# L. Test Principle:

The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test (FVL/FII Test) is performed on the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System (IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO and IMPACT $\mathrm { D x ^ { T M } }$ MA) using the TYPER $\mathrm { D x ^ { T M } }$ Software. The overall process involves DNA isolation, PCR amplification, primer extension, desalting, and mass spectrophotometry of the products.

# DNA isolation and PCR amplification

Briefly, genomic DNA is isolated and purified from EDTA whole blood specimens using a commercial DNA extraction kit. Four target-specific polymerase chain reaction (PCR) amplifications of the Factor V and Factor II genes using site-specific forward and reverse primers are conducted [Factor II forward (F2F), Factor II reverse (F2R), Factor V forward (F5F) and Factor V reverse (F5R)], followed by neutralization of unincorporated deoxyribonucleotide triphosphates (dNTPs) in the PCR products using shrimp alkaline phosphatase (SAP) to terminate the primer extension process.

# Primer Extension Reaction

The PCR products then undergo multiplexed allele-specific single base primer extension reactions, using acyclic terminator nucleotides. Target-specific forward and reverse primers bind directly $5 '$ and $_ { 3 } ,$ to the variant loci. During the extension reaction, the extension cocktail (primers, enzyme, buffer and terminator nucleotides) is added to the amplification products. Gene-specific extension primers, each ending at the nucleotide just before the target site, are used for each of the four reactions (F2F, F2R, F5F, and F5R). The reaction mixture is thermocycled to allow the enzymatic addition of terminator nucleotides into the target sites. In the extension reaction mixture, all four terminator nucleotides (i.e., A, U, C and G) are present. During this reaction, each extension primer is extended by only one terminator nucleotide, producing allele-specific (i.e., wild-type vs. mutant) extension products of different masses depending on the sequence variation at the site of the gene being analyzed resulting in gene-specific sets (i.e., Factor V vs. Factor II genes) of extension products.

# Matrix assisted laser desorption ionization –time of flight (MALDI-TOF) Spectrometry

The products of the extension reaction are desalted and dispensed onto a disposable 96-pad SpectroCHIP $\textsuperscript { \textregistered }$ Array by the IMPACT $\mathrm { D x ^ { T M } }$ NANO. The Chip consists of a pre-defined array of pads for sample analysis (96 pads) and calibration (5 pads). In addition, a 2-D barcode is provided on each Chip for identification and tracking purposes. The Chip contains pre-deposited reagents designed to minimize salt adducts (i.e., cationized sample ion species in the mass spectra which reduces signal intensity and mass resolution, while increasing spectra complexity). The analyte solution is applied to the Chip and following exposure to a vacuum, forms a crystalline compound from which the laser desorption can occur in the MALDI-TOF mass spectrometer.

The analytes on the Chip are then interrogated on the IMPACT $\mathrm { D x ^ { T M } }$ MA using MALDITOF MS. Matrix assisted laser desorption ionization (MALDI) is the process used to ionize the sample in to the gas phase. A pulsed laser beam is directed on to the sample. Energy from the laser beam (337-nm laser energy wavelength) causes structural decomposition of the crystallized analytes (desorption), and ionizes the sample into the gas-phase. The gas-phase ions are accelerated by a pulse electric field through the time-of-flight (TOF) vacuum tube. The ions then drift through an electric field-free flight path and are detected by a microchannel plate device inside the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA. Ions hitting the detector cause an electrical signal which is recorded. A MALDI-TOF mass spectral analyzer detects the massto-charge ratio. The ion signal obtained in the mass spectra after detection, essentially reflects the actual mass of the molecules as can be determined after applying a simple calibration procedure. The resulting mass spectra are captured and analyzed by the TYPER $\mathrm { D x } ^ { \mathrm { T M } }$ software. The software then makes genotype calls using a test-specific algorithm that includes calibration steps.

# Genotype calling

The Data Acquirer-Data Caller module of the TYPER $\mathrm { D x } ^ { \mathrm { T M } }$ software processes spectra to extract peaks, and makes genotype calls based on peak properties defined in the established algorithm. Each Sample pad on a Chip, the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA acquires 5 spectra from 5 different spots on the pad. Each spot is queried with 20 laser shots. The data from the 20 laser shots are summed and then averaged to form spectrum for the individual spot. The software performs mass calibration for each spectrum. The spectra for each pad are calibrated. A baseline is then calculated for the average spectrum, excluding regions of expected peaks and unexpected peaks with signal-to-noise ratios (SNRs) above a pre-defined cutoff, and other forms of spectra characterization, to form the final spectrum for the pad. Any peak with a peak probability less than a pre-determined cutoff value is excluded from further data analysis. That is, the corresponding extension product is considered not present. The SNR is the area of the peak above background and the cut-off is 10 for this assay.

For each assay (i.e., each monoplex test with one single extension primer), a call probability is then calculated based on the peak probabilities of all extension products, as well as the extension reaction rate. The extension rate is the percent conversion of the extend primer converted from unextended primer into allele specific analyte. Extension rate is an indication of the quality of the amplification reaction. Unextended primer will also be represented in the spectra. If there is more than one extension product present, the call probability is adjusted by a skewing factor based on peak area ratio. For any assay with a call probability less than a pre-determined cutoff value, no genotype call is reported for the assay. Based on the number of extension products detected, a genotype call is made as homozygous if there is only one extension product present or heterozygous if there are two products present.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Repeatability:

A study was conducted to assess the repeatability (within-run) and between-run precision of the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test. Repeatability was assessed using 3 clinical genomic DNA samples encompassing the six possible genotypes (wild type (GG), heterozygous (GA) and homozygous (AA) for both FVL and FII genes. Three (3) replicates of each of the 3 clinical DNA samples were tested by 1 operator performing the test on 5 non-consecutive days (triplicate x 5 runs $= 1 5$ results per sample) using the same IMPACT $\mathrm { D x ^ { T M } }$ test reagents and IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System. The number of correct genotype calls for each sample was calculated. All of the genotype calls $( 1 0 0 \% )$ were made correctly. Prespecified acceptance criteria were that correct genotypes $( \geq 9 9 \% )$ must be obtained from all clinical DNA samples for each run. Table 1 below summarizes the correct calls, no calls, and mis-calls for all DNA samples for all runs tested in the Repeatability Study.

Table 1. Summary of Genotyping Calls for Repeatability   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>CorrectCall</td><td rowspan=1 colspan=1>MisCall</td><td rowspan=1 colspan=1>No Calls</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=6>FII</td></tr><tr><td rowspan=1 colspan=1>Wild Type(GG)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Homozygous(AA)</td><td rowspan=1 colspan=1>14*</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous(GA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=6>FVL</td></tr><tr><td rowspan=1 colspan=1>Wild Type(GG)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Homozygous(AA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous(GA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

\* One replicate missing due to a dispense failure in Run 2

The average of the extension rates and signal-to-noise ratios (SNRs) and their respective SDs and $\%$ CVs from all clinical DNA samples for all assays of all runs were also calculated. Pre specified criteria for extension Rates (ER) for clinical DNA samples must be ${ \ge } 0 . 8$ and must demonstrate a coefficient of variation (CV) $< 2 0 \%$ within and between runs. The sum of SNRs for clinical DNA samples must be ${ \ge } 1 0 $ and must demonstrate a coefficient of variation (CV) $< 3 0 \%$ within and between runs. The mean extension rates were greater than $9 5 \%$ for all reactions $( \% \mathrm { C V }$ less than 0.7; results not shown). The results demonstrated Total $\% \mathrm { C V }$ less than $1 2 . 5 \%$ for SNR (Tables 2 and 3).

Table 2. Repeatability Signal-to-Noise Ratio (SNR) –Factor II   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=9>FII Forward</td></tr><tr><td rowspan=1 colspan=1>Wild Type(GG)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>65.9</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>6.4</td></tr><tr><td rowspan=1 colspan=1>Homozygous(AA)</td><td rowspan=1 colspan=1>14*</td><td rowspan=1 colspan=1>68.9</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>10.0</td></tr><tr><td rowspan=1 colspan=1>Heterozygous(GA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>78.3</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>9.6</td></tr></table>

<table><tr><td rowspan=1 colspan=9>FII Reverse</td></tr><tr><td rowspan=1 colspan=1>Wild Type (GG)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>49.0</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>9.4</td></tr><tr><td rowspan=1 colspan=1>Homozygous (AA)</td><td rowspan=1 colspan=1>14*</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>8.1</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (GA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>48.7</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>11.2</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>12.3</td></tr></table>

Table 3. Repeatability Signal-to-Noise Ratio (SNR) – Factor V Leiden   

<table><tr><td rowspan=1 colspan=9>FVL Forward</td></tr><tr><td rowspan=1 colspan=1>Wild Type (GG)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>10.7</td></tr><tr><td rowspan=1 colspan=1>Homozygous (AA)</td><td rowspan=1 colspan=1>14*</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>8.1</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>8.8</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (GA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>44.3</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>8.4</td></tr><tr><td rowspan=1 colspan=9>FVL Reverse</td></tr><tr><td rowspan=1 colspan=1>Wild Type (GG)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>40.9</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>12.5</td></tr><tr><td rowspan=1 colspan=1>Homozygous (AA)</td><td rowspan=1 colspan=1>14*</td><td rowspan=1 colspan=1>46.2</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>9.6</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>10.8</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (GA)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>51.5</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>12.8</td></tr></table>

\* One replicate missing due to a dispense failure in Run 2.

# Reproducibility:

The reproducibility of the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Test system was assessed at three external clinical sites. Twelve human genomic DNA samples, including a wild type (GG), heterozygous mutant (GA) and homozygous mutant (AA) for both FV and FII genes were used in this study. Each DNA sample was tested in singlet by each of 3 sites using one IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System at each site. Two operators each conducted 1 run on each of 5 non-consecutive days for a total of 10 testing runs per site. The number of correct calls, defined as the number of samples yielding the expected genotypes for both FV and FII genes, no calls and miscalls were calculated for each operator, and all sites and operators combined. The data are reported separately before repeat testing and after repeat testing of no calls. No call results were retested once per the testing protocol. Pre-specified criteria for agreement between the FVL/FII test and the reference method must be equal or greater than $9 5 \%$ . All operators, except one, across the 3 sites produced $100 \%$ agreement between the genotypes after all no call results were retested. One operator (Operator 2, Site 1) had 2 samples yielding repeated no calls. There were no miscalls. Summary data is shown in Tables 4 and 5 below.

Table 4. Genotype Reproducibility Study Results—Factor II   

<table><tr><td rowspan=1 colspan=2>Sample</td><td rowspan=1 colspan=3>First Run Results</td><td rowspan=1 colspan=3>Final Results</td><td rowspan=1 colspan=3>Agreement</td></tr><tr><td rowspan=2 colspan=1>SampleNumber</td><td rowspan=2 colspan=1>FIIGenotyp</td><td rowspan=2 colspan=1>CorrectCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>IncorrectCalls</td><td rowspan=2 colspan=1>CorrectCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>IncorrectCalls</td><td rowspan=2 colspan=1>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.3</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.3</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr></table>

Table 5. Genotype Reproducibility Study Results—Factor V Leiden   

<table><tr><td rowspan=1 colspan=2>Sample</td><td rowspan=1 colspan=3>First Run Results</td><td rowspan=1 colspan=3>Final Results</td><td rowspan=1 colspan=3>Agreement</td></tr><tr><td rowspan=2 colspan=1>SampleNumber</td><td rowspan=2 colspan=1>FVLGenotyp</td><td rowspan=2 colspan=1>CorrectCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>IncorrectCalls</td><td rowspan=2 colspan=1>CorrectCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>IncorrectCalls</td><td rowspan=2 colspan=1>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.3</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.3</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>100</td></tr></table>

Extension Rate and SNR Reproducibility: The extension rate data from the reproducibility study (between site) from the analysis of the data from each of the four assays was calculated. The results demonstrated the overall mean of all extension rates was greater than 0.94 and less than $2 . 0 \%$ CV (Data not shown). The signal-to-noise ratio data from the reproducibility study (between site) from the analysis of the data from each of the four assays (Factor II forward, Factor II reverse, Factor V forward and Factor V reverse) was calculated. Tables 6-9 below show the mean, $\% C V$ and $9 5 \%$ CI of the mean.

Table 6. Reproducibility Study Signal-to-Noise Ratio’s--Factor II –Forward   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>OverallMean</td><td rowspan=1 colspan=2>Residual(Run, Day, Operator)</td><td rowspan=1 colspan=2>Between Site(N=3)</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>54.1</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>12.7</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>56.9</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>25.0</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>70.1</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>10.2</td><td rowspan=1 colspan=1>14.5</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>51.5</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>23.8</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>25.0</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>55.9</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>15.2</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>15.6</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>60.3</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>21.2</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>13.2</td><td rowspan=1 colspan=1>21.9</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>66.8</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>10.8</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>11.1</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>65.5</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>67.7</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>10.1</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>10.4</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>64.9</td><td rowspan=1 colspan=1>9.4</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>9.6</td><td rowspan=1 colspan=1>14.8</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>64.7</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>65.8</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>12.8</td></tr></table>

Table 7. Reproducibility Study Signal-to-Noise Ratio’s--Factor II -Reverse   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>OverallMean</td><td rowspan=1 colspan=2>Residual(Run, Day, Operator)</td><td rowspan=1 colspan=2>Between Site(N=3)</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>45.5</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>15.3</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>45.7</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>29.3</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>49.5</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1>17.4</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>30.8</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>45.8</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>19.4</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>41.5</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>25.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>25.7</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>55.1</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1>16.5</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>43.8</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>15.3</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>50.7</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1>20.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>20.5</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>51.0</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>24.1</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>51.4</td><td rowspan=1 colspan=1>10.1</td><td rowspan=1 colspan=1>19.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>10.2</td><td rowspan=1 colspan=1>19.8</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>52.4</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>20.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>20.5</td></tr></table>

Table 8. Reproducibility Study Signal-to-Noise Ratio’s-- Factor V Forward   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>OverallMean</td><td rowspan=1 colspan=2>Residual(Run, Day, Operator)</td><td rowspan=1 colspan=2>Between Site(N=3)</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>18.8</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>11.2</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>32.4</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>16.3</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>28.1</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>21.6</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>21.8</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>22.6</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>20.1</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>15.2</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>18.6</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>18.8</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>19.0</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>21.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>21.5</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>23.2</td></tr></table>

Table 9. Reproducibility Study Signal-to-Noise Ratio’s-- Factor V Reverse   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>OverallMean</td><td rowspan=1 colspan=2>Residual(Run,Day, Operator</td><td rowspan=1 colspan=2>Between Site(N=3)</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>41.4</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>14.4</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>46.5</td><td rowspan=1 colspan=1>11.8</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>40.7</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>15.2</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>11.4</td><td rowspan=1 colspan=1>28.5</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>42.5</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>20.8</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>44.4</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>23.9</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>49.7</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1>17.4</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>47.5</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>14.5</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>50.4</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>15.0</td></tr><tr><td rowspan=1 colspan=1>HOM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>48.2</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>13.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>55.2</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>13.5</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>55.6</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>12.6</td></tr></table>

# Reagent-Lot-Operator Reproducibility:

A study was conducted to determine the reproducibility of the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test System among multiple kit lots, operators, and IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ instruments. Three test kit lots, 3 operators and 3 IMPACT $\mathrm { D x ^ { T M } }$ systems were used in this study. A total of 3 clinical samples encompassing each mutation (wild type, heterozygous and homozygous) for each allele were used. To ensure each of the three parameters under testing (i.e., reagent kit lot, operator and IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System) was properly combined with the two other parameters, a testing scheme was designed for this study as shown in Table 10 below. There were a total of 9 different reagent kit lotoperator-system combinations, therefore 9 different runs in the scheme, which included 3 runs for each of the 3 parameters under testing.

Table 10. Testing Scheme for Reproducibility   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IMPACTDxTM</td><td rowspan=1 colspan=1>IMPACTDxTM</td><td rowspan=1 colspan=1>IMPACTDxTM</td></tr><tr><td rowspan=1 colspan=1>Operator 1</td><td rowspan=1 colspan=1>Kit Lot 1</td><td rowspan=1 colspan=1>Kit Lot 2</td><td rowspan=1 colspan=1>Kit Lot 3</td></tr><tr><td rowspan=1 colspan=1>Operator 2</td><td rowspan=1 colspan=1>Kit Lot 3</td><td rowspan=1 colspan=1>Kit Lot 1</td><td rowspan=1 colspan=1>Kit Lot 2</td></tr><tr><td rowspan=1 colspan=1>Operator 3</td><td rowspan=1 colspan=1>Kit Lot 2</td><td rowspan=1 colspan=1>Kit Lot 3</td><td rowspan=1 colspan=1>Kit Lot 1</td></tr></table>

The number of correct genotype calls was calculated for each DNA sample, all DNA samples for each assay and all assays combined. The mean extension rate for each of the 3 replicates of each parameter was calculated for each sample, all samples for each assay and all assays combined. The average of the mean extension rates of the 6 replicates of each parameter under testing and their respective coefficients of variation (CVs) were calculated for each DNA sample, all DNA samples for each assay and all assays combined. Correct genotypes $( \geq 9 9 \% )$ must be obtained for all genomic DNA samples for each run.

For reagent lot-to-reagent lot reproducibility, the mean Extension Rates for all DNA samples for the 4 assays ranged from $9 2 . 2 \%$ to 100.0 among the 3 lots. Mean SNR values ranged from 32.4 to 79.3 for all samples for the 4 assays among the 3 lots. Between lot variance ranged from 0 to $4 . 5 \%$ for Extension Rates, and from $1 4 . 7 \%$ to $2 2 . 0 \%$ for SNR.

For operator-to-operator reproducibility, the mean Extension Rates for all DNA samples for the 4 assays ranged from $9 2 . 6 \%$ to $1 0 0 . 0 \%$ among the three operators. Mean SNR values ranged from 30.5 to 82.4 for all samples for the 4 assays among the 3 operators. Between operator variance ranged from 0 to $4 . 4 \%$ for Extension Rates, and $1 4 . 8 \%$ to $2 1 . 2 \%$ for SNR.

For system-to-system reproducibility, the mean Extension Rates for all DNA samples for the 4 assays ranged from $9 4 . 0 \%$ to $1 0 0 . 0 \%$ among the three systems. Mean SNR values ranged from 33.3 to 78.0 for all samples for the 4 assays among the 3 systems. Between system variance ranged from 0 to $3 . 2 \%$ for Extension Rates, and $8 . 1 \%$ to $1 4 . 0 \%$ for SNR.

# Randomized Reagent Reproducibility Study

A randomized reproducibility study was performed in order to demonstrate that the kit lot and general consumable reagents (SpectroCHIP $\textsuperscript { \textregistered }$ Array, Clean Resin and 3-Point Calibrant) are interchangeable. Three replicates of each of 3 samples were tested for each reagent lot combination. Samples consisted of leukocyte-depleted whole blood samples spiked with cell lines of each genotype for both FII and FVL. Two lots of each of the 3 consumable reagents and the kit lot were combined for testing (in random order) according to the scheme in Table 11. The distribution of results for SNRs and ERs for all 4 assays (F2F, F2R, F5F, and F5R) and 3 clinical samples for the 16 reagent lot combinations are shown below in Table 12. The genotype calls from the 3 clinical samples were concordant with known genotypes.

Table 11. Reagent Lot Combinations for Randomized Reproducibility   

<table><tr><td rowspan=2 colspan=1>Run</td><td rowspan=2 colspan=1>IMPACT DxFVL/FII Kit</td><td rowspan=1 colspan=3>Consumable Reagents</td></tr><tr><td rowspan=1 colspan=1>Clean Resin</td><td rowspan=1 colspan=1>3-Point Calibrant</td><td rowspan=1 colspan=1>SpectroCHIP</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr></table>

Table 12. Reproducibility between Reagents   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>SNR</td><td rowspan=1 colspan=2>ER</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Range ofResults</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Range ofResults</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>Within-Run</td><td rowspan=1 colspan=1>46.1 to 54.0</td><td rowspan=1 colspan=1>5.2 to 9.1</td><td rowspan=1 colspan=1>98.6 to 99.2</td><td rowspan=1 colspan=1>0.8 to 1.7</td></tr><tr><td rowspan=1 colspan=1>Between Kit lots</td><td rowspan=1 colspan=1>31.5 to 78.2</td><td rowspan=1 colspan=1>5.7 to 19.4</td><td rowspan=1 colspan=1>94.8 to 100</td><td rowspan=1 colspan=1>0.1 to 2.5</td></tr><tr><td rowspan=1 colspan=1>BetweenClean Resin lots</td><td rowspan=1 colspan=1>31.9 to 76.2</td><td rowspan=1 colspan=1>6.8 to 19.7</td><td rowspan=1 colspan=1>95.6 to 100</td><td rowspan=1 colspan=1>0.1 to 2.2</td></tr><tr><td rowspan=1 colspan=1>Between 3-PointCalibrant lots</td><td rowspan=1 colspan=1>31.5 to 75.9</td><td rowspan=1 colspan=1>6.4 to 19.2</td><td rowspan=1 colspan=1>95.2 to 100</td><td rowspan=1 colspan=1>0.04 to 2.3</td></tr><tr><td rowspan=1 colspan=1>Between SpectroCHIP®lots</td><td rowspan=1 colspan=1>31.8 to 76.2</td><td rowspan=1 colspan=1>5.2 to 17.2</td><td rowspan=1 colspan=1>95.5 to 100</td><td rowspan=1 colspan=1>0.03 to 2.3</td></tr></table>

# Extraction Study:

To evaluate the effect of the extraction step on sample reproducibility, an additional, separate extraction study was conducted. The impact of the pre-analytic extraction step on results was assessed at a single site, using 3 specimens, representing all variants reported by the assay. Genomic DNA was isolated using three commonly used commercially available kits. Three DNA extraction methods were tested independently by 2 different operators who tested each sample using each of the 3 methods in replicates of 2 each day for 3 non- consecutive days. A total of 36 replicates per sample per method were assessed. The DNA concentration and A260/A280 ratio of the extracted gDNA samples was determined using spectrophotometry and the assays were performed according to the instructions for use. The pre-specified acceptance criteria were that correct genotypes $( \geq 9 9 \% )$ must be obtained from all clinical DNA samples for each run. Extension rates for clinical DNA samples must be ${ \ge } 0 . 8$ and must demonstrate a coefficient of variation (CV) $< 2 0 \%$ between extraction methods. SNRs for clinical DNA samples must be ${ \ge } 1 0 $ and must demonstrate a coefficient of variation (CV) $< 3 0 \%$ between extraction methods. A total of 1294 calls were made, all of the genotypes were $100 \%$ correct with the exception of two no-calls.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Kit Stability, Real Time

Three kit lots were held at the recommended storage conditions $\left( - 2 0 ^ { \circ } \mathrm { C } \right)$ for the following time points: 0, 1 month, 2 months, 4 months, 5 months and 6 months. Seven (7) genomic DNA samples representing the 6 possible genotypes for Factor II and Factor V genes, and the no template control (NTC) using 4 replicates. The number of correct genotype calls, including the expected no-calls from NTC, was calculated for each kit lot and all lots combined for each storage time point. A total of 2,344 genotype calls were made from the replicates of the 7 DNA samples using the 3 kit lots over the 6 month study period. All genotype calls were reported to be correct. All NTC calls were correctly reported. For the genotype calls, the mean extension rates ranged from 98.2 to $9 8 . 9 \%$ , which were above the acceptance criteria of $2 8 0 \%$ . For all the genotype calls, the mean SNRs ranged from 39.1 to 51.9, which exceeded the acceptance criteria of ${ \ge } 1 0 $ . No deterioration of the 3 performance measures (correct genotype call rate, extension rate and SNRs) were observed from the 3 lots over the 6 month study period.

# Kit Freeze-Thaw

Three kit lots were held at $- 2 0 ^ { \circ } \mathrm { C }$ and then 11 freeze-thaw cycles were performed. Seven (7) genomic DNA samples and the NTC were tested in 4 replicates each. All calls across all time points were correct. The extension rates ranged from 97.9 to $9 8 . 9 \%$ , all of which exceeded the acceptance criteria of $2 8 0 \%$ . The mean SNR ranged from 36.6 to 53.8, all of which exceeded the acceptance criteria of ${ \ge } 1 0 $ . No deterioration of the 3 performance measures (correct genotype call rate, extension rate and SNRs) were observed from the 3 lots over the 11 freeze-thaw cycles tested.

# Shipping Stability

Three kit lots were packaged in accordance with the expectations for the marketed test, which includes the outer shipping box packed with dry ice. The packaged kits underwent two 120 hour temperature profiles in series with a single re-icing. The summer temperature profile series was the following: $+ 2 2 ^ { \circ } \mathrm { C }$ for 4 hours; $+ 3 5 ^ { \circ } \mathrm { C }$ for 6 hours; $+ 3 0 ^ { \circ } \mathrm { C }$ for 80 hours; $+ 4 5 ^ { \circ } \mathrm { C }$ for 6 hours and $+ 3 0 ^ { \circ } \mathrm { C }$ for 24 hours. The winter temperature profile series was the following: $+ 1 8 ^ { \circ } \mathrm { C }$ for 4 hours; $- 1 0 ^ { \circ } \mathrm { C }$ for 6 hours; $+ 1 0 ^ { \circ } \mathrm { C }$ for 80 hours; $- 1 0 ^ { \circ } \mathrm { C }$ for 6 hours and $+ 1 0 ^ { \circ } \mathrm { C }$ for 24 hours. Kits were then unpacked and tested using a non-template control (NTC) and 7 clinical genomic DNA samples that encompassed the 6 genotypes for the FII and FV genes. The number of correct genotype calls, including the expected no-calls from NTCs, was calculated for each of the three kit lot and all lots combined for each temperature profile. All genotype calls were correct.

# Controls

The IMPACT $\mathrm { D x ^ { T M } }$ Factor V Leiden and Factor II Genotyping Test includes the following assay-specific controls:

1. Factor II Positive Control: The Factor II Positive Control is a reference human genomic DNA sample with known genotypes. This material serves as a positive control for the Factor II G20210A mutation and a negative control for the Factor V Leiden G1691A mutation. This control is included in a predefined location (well F12) on the test plate for every run. The result from this location on each test plate must be "PASS" (i.e., expected result heterozygous for Factor II and wild-type for Factor V) before any spectra are collected from any sample on the same plate. If the Factor II Positive Control fails, TYPER $\mathrm { D x } ^ { \mathrm { T M } }$ will abort the run.

2. Factor V Leiden Positive Control: The Factor V Leiden Positive Control is a reference human genomic DNA sample obtained from the WHO with known genotypes. This material serves as a positive control for the Factor V Leiden G1691A mutation and a negative control for the Factor II G20210A mutation. This control is included in a pre-defined location (well G12) on the test plate for every run. The result from this location on each test plate must be "PASS" (i.e., expected result heterozygous for Factor V and wild-type for Factor II) before any spectra are collected from any sample on the same plate. If the Factor V Leiden Positive Control fails, TYPERDx™ will abort the run.

3. Negative Control: The Negative Control is a no-template control (purified water) and serves as an assay process negative control. This control is included in a pre-defined location (well H12) on the test plate for every run. The result from this location on each test plate should always be PASS (i.e., no detection of any targeted sequence) before any spectra are collected from any sample on the same plate. If the Negative Control fails, TYPER $\mathrm { D x ^ { T M } }$ will abort the run.

4. Internal Control: The Internal Control is a built-in assay process positive control for each sample being tested. The control detects a specific human gene on a sex chromosome using the same Sequenom biochemistry. The Internal Control result must be "PASS" (i.e., detection of the specific human sex chromosome gene) before any genotype result on either Factor V or Factor II is reported on the sample. If the Internal Control fails, TYPER $\mathrm { D x } ^ { \mathrm { T M } }$ will not report any result on the sample.

# d. Detection limit:

The recommended genomic DNA input for this assay is $2 5 { \mathrm { n g } }$ . A study was conducted to estimate the performance of the assay when the gDNA input is less than the recommended DNA input concentration. Three clinical gDNA samples that encompassed the wild type allele (GG), heterozygous allele (GA) and mutant homozygous allele (AA) for both Factor II and Factor IV genes were used to determine the limit of detection (LoD) for each of the four reactions simultaneously occurring with the test, (F2F, F2R, F5F and F5R). Input DNA levels tested ranged from $2 5 { \mathrm { n g } }$ to $0 . 0 0 1 5 \mathrm { n g }$ , in a two-fold dilution series to include 15 levels. Each level was tested in replicates of six. The limit of detection for the assay is claimed to be $0 . 6 7 { \mathrm { n g } }$ (200 allele copies) which is below the recommended input quantity of $2 5 { \mathrm { n g } }$ per reaction.

To establish an upper limit for input, 3 replicates of 12.5, 25, 50, 100, 200, 400 and $8 0 0 ~ \mathrm { n g }$ of DNA per reaction were tested across 4 different samples encompassing the 6 possible FVL and FII genotypes. Assay performance remained acceptable at all input levels; genotyping calls were accurately made at all levels.

# e. Analytical specificity:

Interfering substances

The effect of potential interfering substances on the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ test was assessed by spiking various substances into whole blood (wild-type) samples, and leukocytedepleted whole blood samples spiked with cell lines that contained FII and FVL variants, designed to mimic naturally-occurring WBC levels. Samples were spiked with the interfering substance prior to DNA extraction. Testing was conducted on 6-8 replicates per sample. Testing was compared to unspiked controls. The following potentially interfering substances were tested: hemoglobin, $2 0 0 \mathrm { m g / d L }$ ; K2-EDTA, $3 . 4 \mu \mathrm { M }$ ; heparin, 3000 units/L; cholesterol, $5 0 0 \mathrm { m g / d L }$ ; and bilirubin, $6 0 \mathrm { m g / d L }$ and ethanol (spiked after DNA extraction to mimic the buffer). The number of correct genotype calls was calculated for each assay for each potential interferent. The mean extension rate (ER) and the mean SNRs were also calculated for each assay for each potential interferent. Genotyping calls for all four assays (F2F, F2R, F5F, F5R) were correct. The mean ER ranged from $9 3 . 9 \%$ to $9 9 . 9 \%$ across all assays, while mean SNR ranged from 27.6 to 77.9 $\% \mathrm { C V }$ range $3 . 8 \%$ to $2 1 . 8 \%$ ) across all four assays and did not differ when compared to the no-treatment controls.

# Interfering mutations

The effect of mutations in close proximity to genotyping at locus 1691 of the FVL gene, as well as locus 20210 on the FII gene was assessed. Synthetic DNA constructs containing individual known variants at position 1689 (G to A), 1692 (A to C), 1696 (A to G) and1690 (C to T) and at positions 20207 (A to C), 20209 (C to T), 20218 (A to G) and 20221 (C to T) were tested for their effect on FVL and FII genotyping results. Constructs were tested in triplicate at input DNA concentrations of 10,000 copies per construct per reaction using the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ test. Correct genotype calls were obtained in the presence of the constructs for all replicates with four exceptions. The presence of rare mutation $1 6 9 2 \mathrm { A } { > } \mathrm { C }$ impeded the genotype calling at the 1691 locus in the reverse FV assay, while its presence had no interference in the forward FVL assay. The presence of rare mutation $1 6 9 0 \mathrm { C } { > } \mathrm { T }$ also impeded the genotype calling at the 1691 locus in the forward FVL assay, but not in the reverse FVL assay. Additionally, the presence of rare mutations at $2 0 2 0 7 \mathrm { A } { > } \mathrm { C }$ or $2 0 2 0 9 \mathrm { C } { > } \mathrm { T }$ impeded the genotype calling at locus 20210 in the forward FII assays, while neither mutation had interference in the reverse FII assays. Since the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test is designed to require concordant forward and reverse assay results in order to report a genotype on a patient specimen, no FVL or FII genotype will be reported on the patient specimens when the Factor V forward assay fails to produce a genotype call due to the presence of an interfering $1 6 9 0 { \mathrm { C } } { \mathrm { > } } { \mathrm { T } } _ { \mathrm { a } }$ , or $1 6 9 2 \mathrm { A } { > } \mathrm { C }$ , or on a patient specimen when the Factor II forward assay fails to produce a genotype call due to the presence of either the $2 0 2 0 7 \mathrm { A } { > } \mathrm { C }$ or $2 0 2 0 9 \mathrm { C } { > } \mathrm { T }$ interfering mutations. A limitation in the instructions for use labeling indicates this interference. Results are summarized in Table 13.

Table 13. Genotype Results at the 1691 locus for the FVL Gene and the 20210 locus of the FII Gene   

<table><tr><td rowspan=1 colspan=1>Construct</td><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Correct GenotypeCalls</td><td rowspan=1 colspan=1>MeanExtension Rate(%)</td><td rowspan=1 colspan=1>Mean SNR</td></tr><tr><td rowspan=2 colspan=1>Control FVL 1691G</td><td rowspan=1 colspan=1>F5F</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>29.6</td></tr><tr><td rowspan=1 colspan=1>F5R</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>40.8</td></tr><tr><td rowspan=2 colspan=1>FVL 1690C&gt;T</td><td rowspan=1 colspan=1>F5F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>No Calls (3)</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>F5R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=2 colspan=1>FVL 1689G&gt;A</td><td rowspan=1 colspan=1>F5F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>37.1</td></tr><tr><td rowspan=1 colspan=1>F5R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>48.5</td></tr><tr><td rowspan=2 colspan=1>FVL 1692A&gt;C</td><td rowspan=1 colspan=1>F5F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>37.5</td></tr><tr><td rowspan=1 colspan=1>F5R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>No Calls (3)</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=2 colspan=1>FVL 1696A&gt;G</td><td rowspan=1 colspan=1>F5F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>36.9</td></tr><tr><td rowspan=1 colspan=1>F5R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>45.2</td></tr><tr><td rowspan=2 colspan=1>Control FII 20210G</td><td rowspan=1 colspan=1>F2F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>55.5</td></tr><tr><td rowspan=1 colspan=1>F2R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>42.6</td></tr><tr><td rowspan=2 colspan=1>FII 20207 A&gt;C</td><td rowspan=1 colspan=1>F2F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>No Calls (3)</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=1 colspan=1>F2R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=2 colspan=1>FII 20209 C&gt;T</td><td rowspan=1 colspan=1>F2F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>No Calls (3)</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>F2R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>43.3</td></tr><tr><td rowspan=2 colspan=1>FII 20218 A&gt;G</td><td rowspan=1 colspan=1>F2F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>67.4</td></tr><tr><td rowspan=1 colspan=1>F2R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>50.8</td></tr><tr><td rowspan=2 colspan=1>FII 20221 C&gt;T</td><td rowspan=1 colspan=1>F2F</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>65.6</td></tr><tr><td rowspan=1 colspan=1>F2R</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>51.5</td></tr></table>

# Carryover Contamination

Two separate studies assessed the potential for cross-contamination of the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System FVL/FII test:

1. The first study assessed the likelihood of random sample cross-over during the test procedure from the initial PCR amplification step through analyte dispensing on the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO. One clinical gDNA sample (GA for FVL and GG for FII) and the NTC were dispensed on one 96-well plate, following an alternating checkerboard pattern. Forty-six replicates of the gDNA sample, 47 replicates of the NTC and 3 positive and negative controls were tested.   
2. The second study assessed the likelihood of sample carryover as a result of potentially ineffective cleaning of the dispensing pin set between transfers during the automated sample dispensing process. Twenty four replicates of clinical gDNA sample (GG for FVL and GA for FII) were dispensed into rows A and E of a 96 well plate using a distribution pattern, which simulates the dispensing pattern of the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO pin tool, which moves in

a 2/3 block from top to bottom, then from left to right. Twenty three replicates of a second gDNA sample (GG for FVL and AA for FII) were dispensed into rows B and F; replicates of the NTC were dispensed into rows C, D, G and H. Combined, 47 replicates of the two clinical gDNA samples, 46 replicates of the NTC and the 3 positive and negative controls were tested.

The number of correct genotype calls, including the no-calls from the NTC, was calculated for each assay and all assays combined. The mean extension rate was also calculated for each assay and all assays combined. Additionally, the mean of SNRs of both allelic (wild type vs. mutant) peaks was calculated for each assay and all assays combined. Correct genotypes $( \geq 9 9 \% )$ must be obtained for all clinical DNA samples. Genotype calls on either the FVL or FII gene must be less than $1 \%$ from all NTC wells. For the clinical gDNA samples, extension rates must be $2 8 0 \%$ and SNR must be ${ \ge } 1 0 $ . A total of 372 genotype calls were made from the clinical gDNA samples for both study designs. $100 \%$ of the genotype calls were made correctly; the mean extension rates on the DNA samples within assays ranged from 97.0 to $100 \%$ and from 98.9 to $9 9 . 1 \%$ in all assays combined. The mean SNR on the DNA samples within assays ranged from 32.1 to 77.8 and from 48.4 to 50 for all assays combined. A total of 372 call attempts were made on the NTC from both study designs and $100 \%$ resulted in no-calls.

f. Assay cut-off: Not Applicable

2. Comparison studies:

a. Method comparison with predicate device:

# Accuracy:

The accuracy of the IMPACT $\mathrm { D x ^ { T M } }$ Factor V Leiden and Factor II Genotyping Test was evaluated by testing banked genomic DNA samples from EDTA anti-coagulated whole blood from patients with suspected thrombophilia, and comparing the results to bi-directional Sanger sequencing. Following a pre-specified sample selection protocol, a total of 860 samples were de-identified and submitted for analysis to three clinical sites. The inclusion/exclusion criteria were in accordance with the American College of Medical Genetics (ACMG) Consensus Statement of Factor V Leiden Mutation Testing, dated October 21, 2000 and updated on October 26, 2004, and are as follows:

Samples were from patients 18 years of age or older with suspected thrombophilia as defined by one or more of the following:

• Age $< 5 0$ , any venous thrombosis.   
• Venous thrombosis, age ${ > } 5 0$ , except when active malignancy is present.   
• Venous thrombosis in unusual sites (such as hepatic, mesenteric and cerebral veins). • Venous thrombosis and a strong family history of thrombotic disease.   
• Venous thrombosis in pregnant women or women taking oral contraceptives.   
• Women with recurrent pregnancy loss or unexplained severe preeclampsia, placental abruption, intrauterine fetal growth retardation or stillbirth.   
• Myocardial infarction in female smokers under age 50.   
• Recurrent venous thrombosis.

Samples were not utilized for this study if they met any of the following criteria:

• Previous specimen donation under this protocol from the same patient. Inadequate sample volume to complete the study-specific testing. Insufficient clinical/medical history information.

The sample set was enriched to ensure that a minimum of $10 \%$ of samples with the FII or FVL variants were included. A summary of the accuracy results are shown in Tables 14 and 15. A total of 18 no calls were observed for FVL, of which 4 wild-type specimen results remained unresolved after repeat testing. A total of 16 no calls were observed for FII, of which 5 calls remain unresolved after repeat testing. One retest was allowed per study protocol. There were no miscalls.

Table 14. IMPACT $\mathrm { D x ^ { T M } }$ FVL Genotyping Test compared to bi-directional sequencing   

<table><tr><td rowspan=2 colspan=1>FVL Genotype</td><td rowspan=2 colspan=1>−</td><td rowspan=1 colspan=5>Before Repeat Testing</td><td rowspan=1 colspan=5>After Repeat Testing</td></tr><tr><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1># NoCalls1</td><td rowspan=1 colspan=1>MissedCalls ls2</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95%LCB </td><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1># NoC als</td><td rowspan=1 colspan=1>MissedCall ls</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95%LCB3</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>710</td><td rowspan=1 colspan=1>697</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>706</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.7</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>92.2</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>95.3</td></tr><tr><td rowspan=1 colspan=1>Homozygous</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>88.0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>88.0</td></tr></table>

1 No Calls $=$ sample failures or results generating a “no call” result 2 Missed Calls $=$ wrong or incorrect calls 3 One-sided $9 5 \%$ lower confidence bound 4 Number of samples requiring repeat testing due to a no call after initial test

Table 15. IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ FII Genotyping Test compared to bi-directional sequencing   

<table><tr><td rowspan=2 colspan=1>FIIGenotype</td><td rowspan=2 colspan=1>−</td><td rowspan=1 colspan=5>Before Repeat Testing</td><td rowspan=1 colspan=5>After Repeat Testing</td></tr><tr><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1># NoCalls1</td><td rowspan=1 colspan=1>MissedCalls ls2</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95%LCB </td><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1># NoCalls</td><td rowspan=1 colspan=1>Missed Calls}</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95%LCB3</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>762</td><td rowspan=1 colspan=1>747</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>757</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>98.6</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.1</td></tr><tr><td rowspan=1 colspan=1>Homozygous</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6</td></tr></table>

1 No Calls $=$ sample failures or results generating a “no call” result 2 Missed Calls $=$ wrong or incorrect calls 3 One-sided $9 5 \%$ lower confidence bound 4 Number of samples requiring repeat testing due to a no call after initial test

b. Matrix comparison:

Not applicable. The assay uses genomic DNA purified from EDTA anti-coagulated whole blood.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.   
b. Clinical specificity: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

Not applicable. Factor II (G20210A) and Factor V Leiden (G1691A) mutations are present in $2 \%$ and $5 \%$ of the general population, respectively.

# N. Instrument Name:

IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System including IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Nano, IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA, IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Software, IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Accessories, and assay specific Panel Definition File (FactorV-II-Assay- $\mathbf { \partial } \cdot \mathbf { V } 1 . 0 . \mathrm { x m l }$ )

# O. System Descriptions:

1. Modes of Operation: Batch via 96-well sample plate and 96-pad SpectroCHIP $\textsuperscript { \textregistered }$ Array

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

The user enters barcode number of the sample plate and specimen name for each well of the plate into a plate definition file. The plate definition file is used by the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Software to identify specimens.

4. Specimen Sampling and Handling:

Specimens are processed according to IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping test assay instructions.

After completing test biochemistry, the user loads the sample plate onto the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO. The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO automatically transfers analytes from the sample plate to a SpectroCHIP $\textsuperscript { \textregistered }$ Array (Chip). The user then removes the Chip from the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO and loads it to the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA. The IMPACT $\mathrm { D x ^ { T M } }$ MA acquires data and reports test results. Refer to IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test Package Insert and IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System User Guide for detail information on specimen sampling and handling.

# 5. Calibration:

The IMPACT $\mathrm { D x ^ { T M } }$ System uses 3-Point Calibrant to perform a calibration procedure for every run and every Chip. The 3-Point Calibrant is a solution mixture of three oligonucleotides of known molecular weights. The IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO transfers 3- Point Calibrant from a single-well reservoir located on the instrument dry station to the five designated calibrant pads of each SpectroCHIP $\textsuperscript { \textregistered }$ Array. The IMPACT $\mathrm { D x ^ { T M } }$ MA acquires and analyzes the spectra from all the calibrant pads before processing any control or sample pad. If the average of the calibrant spectra passes the predetermined calibrant quality control criteria, the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA continues to acquire and analyze spectra from control and sample pads on the Chip; if not, the data acquisition on the Chip is aborted.

During the spectrum acquisition from each pad on the Chip, based on the known masses of the assay-specific extension products stored in the assay-specific panel definition file and their measured positions in the spectrum obtained from the pad, the TYPER $\mathrm { D x } ^ { \mathrm { T M } }$ software finds the 5 to 12 largest peaks in the acquired spectrum corresponding to the known extension products and further adjusts the TOF to mass conversion algorithm coefficients for the Chip pad. These new coefficients are then used to calculate the mass of each peak in the spectrum acquired from that pad on the Chip.

# 6. Quality Control:

The IMPACT $\mathrm { D x ^ { T M } }$ Factor V Leiden and Factor II Genotyping Test includes the following assay-specific controls. Refer to the section on Controls above for a description of assay specific controls.

In addition to the assay-specific controls, the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System imposes additional non-assay-specific control steps to ensure high-quality genotyping results:

1. During the analyte transfer step, the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ NANO monitors the dispensing volume when dispensing on the individual pads of a Chip. If the dispensed volume of a pad falls outside of a pre-determined range, the pad is marked as dispense failure and no spectrum acquisition will be attempted on the pad during the subsequent spectrum acquisition step on the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA.   
2. During the spectrum acquisition step, the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ MA first attempts to acquire data from the 5 calibrant pads. If the spectra from the calibrant pads pass quality control, the instrument continues to the next step. If calibration quality control tests fail, TYPER $\mathrm { D x ^ { T M } }$ will abort the spectrum acquisition run.

# P. Other Supportive Instrument Performance Characteristics Data not covered in the “Performance Characteristics” Section above:

Desalting: Clean ion exchange resin and water are added to the extension products to conduct cationic exchange of salts and remove excess cations that may interfere with the assay from the reaction. The impact of improperly desalted reactions prior to placement on the SpectroCHIP $\textsuperscript { \textregistered }$ Array was assessed. Currently $1 5 \mathrm { m g }$ per reaction is used. Amounts greater than and less than $1 5 \mathrm { m g }$ were tested in replicates of three for each Clean Resin input amount. All 5 concentrations ( $1 2 \mathrm { m g }$ , $1 3 . 5 \mathrm { m g }$ , $1 5 \mathrm { m g } .$ , $1 6 . 5 \mathrm { m g }$ , and $1 8 \mathrm { m g } )$ ) allowed for accurate genotyping calls. An additional evaluation was made on the possibility that the Clean Resin step was omitted completely. The data demonstrated across 18 replicates for each reaction that without Clean Resin, no call results occured, and incorrect genotype results did not occur; demonstrating that the incomplete removal of cations that affect FII or FVL allele detection by MALDI-TOF did not occur.

Extension Primer Specificity: To determine the specificity of the extension primers, 13 human genomic DNA samples including all genotypes (i.e., wild type, heterozygous mutant, and homozygous mutant) for both FV and FII genes, were used following the recommended testing protocol, varying the extension primers in the reactions. The extension primers for the Factor II forward (F2F), Factor II reverse (F2R), Factor V forward (F5F), Factor V reverse (F5R) and the internal control (IC) assays were used individually and together. The number of correct genotype calls, including the expected no-calls from the reactions where select extension primers were omitted, was calculated for each of the assays from each run. Study demonstrated correct genotype calls for all results assessed (one sample was excluded due to failed volume dispensing). The study was repeated using 8 replicates of a sample. $100 \%$ of calls were correct.

Reportable Mass Range: The reportable mass range and mass resolving power of the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System was assessed. A set of 6 pairs of oligonucleotides (oligos), with a mass range from 5012 Da to $8 4 3 0 \mathrm { D a }$ , were used and shown in the Table below. The two oligos for each of the 6 pairs have a mass difference of 14.9 Da (Marker 7), 9.1 Da (Marker 13), 24 Da (Marker 17), 40 Da (Marker 21), 25 Da (Marker 28), and 16.1 Da (Marker 36), respectively. Each pair of oligos consists of one parent oligo and a derivative oligo with a single nucleotide polymorphism (SNP) from the parent oligo. Mixtures of the oligo set were tested at equal molar concentrations for all oligos at $1 . 2 5 \mu \mathrm { M }$ , $0 . 8 3 \mu \mathrm { M }$ , $0 . 5 6 \mu \mathrm { M }$ , $0 . 3 7 \mu \mathrm { M }$ , $0 . 2 5 \mu \mathrm { M }$ , $0 . 1 6 \mu \mathrm { M }$ , $0 . 1 1 \mu \mathrm { M }$ , and $0 . 0 9 \mu \mathrm { M }$ with 14 replicates for each concentration. This test setup was repeated under different environmental conditions as well (Temperature/Relative humidity: $2 6 ^ { \circ } \mathrm { C } / 2 0 \%$ , $2 6 ^ { \circ } \mathrm { C } / 7 5 \%$ , $1 6 ^ { \circ } \mathrm { C } / 2 0 \%$ , and $1 6 \% \%$ ). Among all calls, $9 8 . 8 \%$ of the call attempts resulted in successful peak calls. Among the calls made, all of them $( 1 0 0 \% )$ were made correctly. These results demonstrated that the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ System is sufficient for discriminating the mass differences of the extension products observed with the IMPACT $\mathrm { D x } ^ { \mathrm { T M } }$ Factor V Leiden and Factor II Genotyping Test, which covers a mass range of 5781Da to 6960 Da for the possible extension reaction products.

12-Oligonucleotide Set   

<table><tr><td rowspan=1 colspan=1>Marker</td><td rowspan=1 colspan=1>Mass (Da)</td></tr><tr><td rowspan=1 colspan=1>7Parent</td><td rowspan=1 colspan=1>5027.3</td></tr><tr><td rowspan=1 colspan=1>7sNp</td><td rowspan=1 colspan=1>5012.4</td></tr><tr><td rowspan=1 colspan=1>13Parent</td><td rowspan=1 colspan=1>5704.7</td></tr><tr><td rowspan=1 colspan=1>13sNp</td><td rowspan=1 colspan=1>5713.8</td></tr><tr><td rowspan=1 colspan=1>17Parent</td><td rowspan=1 colspan=1>6141</td></tr><tr><td rowspan=1 colspan=1>17sNp</td><td rowspan=1 colspan=1>6117</td></tr><tr><td rowspan=1 colspan=1>21Parent</td><td rowspan=1 colspan=1>6705.4</td></tr><tr><td rowspan=1 colspan=1>21sNp</td><td rowspan=1 colspan=1>6745.4</td></tr><tr><td rowspan=1 colspan=1>28Parent</td><td rowspan=1 colspan=1>7372.9</td></tr><tr><td rowspan=1 colspan=1>28sNp</td><td rowspan=1 colspan=1>7347.9</td></tr><tr><td rowspan=1 colspan=1>36Parent</td><td rowspan=1 colspan=1>8413.5</td></tr><tr><td rowspan=1 colspan=1>36sNP</td><td rowspan=1 colspan=1>8429.6</td></tr></table>

Dynamic Range of Analyte Concentrations: The ability of the IMPACT Dx System to support the dynamic range needed for the assay was assessed. A mixture of 5 oligonucleotides representing the reportable mass range, and 5 concentrations was assessed ( $1 3 \mathrm { n M } .$ , $1 0 0 \mathrm { n M }$ , $1 5 0 \mathrm { n M }$ , $2 0 0 \mathrm { n M }$ , and $7 2 6 \mathrm { n M } )$ . $1 0 0 \mathrm { n M }$ is the detection sensitivity specification of the System. Sixteen (16) replicates of the mixture were tested using the IMPACT $\mathrm { D x ^ { T M } }$ System. All peaks representing the 5 oligonucleotides were correctly called from all 16 replicates. The results demonstrate that the Sequenom genotyping tests is sufficient for detecting the alleles, where the expected extension products for each extension primer would be either only one present (i.e., homozygous) or two present in approximately equal molar ratio (i.e., heterozygous).

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.