# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM

A. 510(k) Number: K172913   
B. Purpose for Submission: To obtain clearance for a new device   
C. Measurand: Factor II and Factor V   
D. Type of Test: Genotyping test   
E. Applicant: Roche Molecular Systems, Inc.   
F. Proprietary and Established Names: cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test

# G. Regulatory Information:

1. Regulation section: 21 CFR 84.7280; Factor V Leiden DNA Mutation Detection Systems

2. Classification: Class II

3. Product code:

NPR; Test, Factor II G20210A Mutations, Genomic DNA PCR, Factor V Leiden DNA Mutation Detection Systems

NPQ; Test, Factor V Leiden Mutations, Genomics DNA PCR, Factor V Leiden DNA Mutation Detection Systems

4. Panel: Hematology (81)

# H. Intended Use:

1. Intended use(s):

The cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test is an in vitro diagnostic device that uses realtime PCR for the detection and genotyping of the Factor II (Prothrombin) G20210A mutation and the Factor V Leiden G1691A mutation in genomic DNA obtained from $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood specimens as an aid in diagnosis of patients with suspected thrombophilia. The cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test and cobas $\boldsymbol { z } 4 8 0$ analyzer are used together for automated amplification and detection.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): Prescription Use only

4. Special instrument requirements: cobas z 480 analyzer

# I. Device Description:

The cobas Factor II and Factor V Test is a real-time polymerase chain reaction (PCR) test for the qualitative detection and genotyping of a single point mutation in the human Factor V gene (G to A at position 1691, referred to as the Factor V Leiden mutation) and a single point mutation in the human Factor II gene (G to A at position 20210), in genomic DNA isolated from peripheral whole blood, as an aid in diagnosis of patients with suspected thrombophilia.

The assay consists of one reagent kit and a system platform. The reagent kit provides the necessary reagents and controls to perform automated real-time PCR amplification and detection. The system platform consists of real-time PCR thermal cycler (cobas $\boldsymbol { z } 4 8 0$ analyzer) and a control unit (cobas 4800 System CU).

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Roche Factor II (Prothrombin) G20210A Kit Roche Factor V Leiden Kit

2. Predicate 510(k) number(s):

K033612 K033607 (DEN030005)

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device: cobas Factor IIand Factor V Test(K172913)</td><td colspan="1" rowspan="1">Predicate: RocheFactor II(Prothrombin)G20210A KitK033612</td><td colspan="1" rowspan="1">Predicate: RocheFactor V LeidenKit K033607</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The cobas Factor II andFactor V Test is an invitro diagnostic devicethat uses real-time PCRfor the detection andgenotyping of theFactor II (Prothrombin)G20210A mutation andthe Factor V LeidenG1691A mutation ingenomic DNA obtainedfrom K2EDTA wholeblood specimens as anaid in diagnosis ofpatients with suspectedthrombophilia.The test is performedon the cobas z 480analyzer for automatedamplification anddetection.</td><td colspan="1" rowspan="1">Same except thepredicate devicewas cleared on theLightCycler 1.2analyzer.</td><td colspan="1" rowspan="1">Same except thepredicate devicewas cleared on theLightCycler 1.2analyzer.</td></tr><tr><td colspan="1" rowspan="1">Type of Test</td><td colspan="1" rowspan="1">Genotyping test</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Target ofDetection</td><td colspan="1" rowspan="1">Single nucleotidemutation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Indication for Use</td><td colspan="1" rowspan="1">Aid in the diagnosis ofpatients with suspectedthrombophilia</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended User</td><td colspan="1" rowspan="1">Healthcareprofessionals</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Purified DNA from</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">human blood samples</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Differences</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device: cobas Factor IIand Factor V Test(K172913)</td><td colspan="1" rowspan="1">Predicate: RocheFactor II(Prothrombin)G20210A KitK033612</td><td colspan="1" rowspan="1">Predicate: RocheFactor V LeidenKit K033607</td></tr><tr><td colspan="1" rowspan="1">TechnologicalDetectionPrinciples</td><td colspan="1" rowspan="1">Real-time PCR test forsimultaneousamplification of twotargets and detection ofspecific SNPs in thePCR amplified DNAsequences (multiplexsystem)</td><td colspan="1" rowspan="1">Real-time PCR testfor amplification ofone target anddetection of specificSNP in PCR-amplified DNAsequences</td><td colspan="1" rowspan="1">Real-time PCR testfor amplificationof one target anddetection ofspecific SNP inPCR-amplifiedDNA sequences</td></tr><tr><td colspan="1" rowspan="1">Oligonucleotideprobes andprimers</td><td colspan="1" rowspan="1">Specific for Factor VG1691A, Factor IIG20210A and the wild-type Factor II andFactor V sequences</td><td colspan="1" rowspan="1">Specific for FactorII G20210A andFactor II wild-typesequences</td><td colspan="1" rowspan="1">Specific for FactorV Leiden G1691Aand Factor V wild-type sequences</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Fluorogenic detectionof SNPs in each PCRamplification productby allele-specificcleavage of TaqManprobes</td><td colspan="1" rowspan="1">Fluorogenicdetection of SNP inPCR amplificationproduct by meltingcurve analysis</td><td colspan="1" rowspan="1">Fluorogenicdetection of SNPin PCRamplificationproduct by meltingcurve analysis</td></tr><tr><td colspan="1" rowspan="1">AnalyticalSensitivity</td><td colspan="1" rowspan="1">&lt;50 allele copies (0.01ng input DNA/reaction)</td><td colspan="1" rowspan="1">Approximately 50allele copies perreaction</td><td colspan="1" rowspan="1">Approximately 50allele copies perreaction</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">cobas z 480</td><td colspan="1" rowspan="1">Roche LightCyclerversion 1.2</td><td colspan="1" rowspan="1">Roche LightCyclerversion 1.2</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Endogenous internalcontrol (each gene isinternal control forother gene), plusexternal positive andnegative controlsrequired in each run</td><td colspan="1" rowspan="1">External positiveand negativecontrols required ineach run</td><td colspan="1" rowspan="1">External positiveand negativecontrols requiredin each run</td></tr><tr><td colspan="1" rowspan="1">Reference method</td><td colspan="1" rowspan="1">Bi-directional Sangersequencing</td><td colspan="1" rowspan="1">Sanger sequencing</td><td colspan="1" rowspan="1">Sanger sequencing</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI EP7-A2, Interference Testing in Clinical Chemistry; Approved Guideline – $2 ^ { \mathrm { n d } }$ Edition

# L. Test Principle:

DNA is extracted offline from whole blood specimens collected in $\mathrm { K } _ { 2 } \mathrm { E D T A }$ tubes. The userselected DNA extraction method must provide DNA of sufficient concentration. Automated real-time PCR is then performed on the cobas $\boldsymbol { z } 4 8 0$ analyzer. The Factor II and Factor V mutations are detected simultaneously in the same real-time PCR reaction. The amplification reactions generate a Factor II specific amplicon and Factor V specific amplicon in all samples, and utilize four fluorescent-dye labeled oligonucleotide probes for detection of the Factor II and Factor V genotypes. Depending upon the test order, results for one or both mutations are reported for each DNA sample.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Lot-to-Lot Repeatability: The lot-to-lot repeatability of the cobas Factor II and Factor V Test was evaluated by testing genomic DNA samples isolated from seven K2EDTA whole blood samples with three lots of the cobas Factor II and Factor V Test. The study was conducted over five non-consecutive days with two operators, two cobas z 480 analyzers, one run per day per kit lot, and two replicates per run, for a total of 60 replicates of each sample. The cobas Factor II and Factor V Test results were compared to the Factor II and Factor $\mathrm { V }$ genotype by bi-directional Sanger sequencing. The overall agreement between cobas Factor II and Factor V Test results and nucleic acid sequencing was $100 \%$ across all samples and reagent lots (one-sided, lower $9 5 \%$ confidence limit $9 9 . 3 \%$ ). The table below summarizes the study results by individual lots and combined lots.

<table><tr><td rowspan=2 colspan=1>SampleID</td><td rowspan=2 colspan=1>Factor IIGenotype</td><td rowspan=2 colspan=1>Factor VGenotype</td><td rowspan=2 colspan=1>Number ofTests perLot</td><td rowspan=1 colspan=4>Number (%) Correct GenotypeResults</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Lot 4</td></tr><tr><td rowspan=1 colspan=1>S1</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=1>S2</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=1>S3</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=1>S4</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=1>S5</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>Homozygousmutant</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=1>S6</td><td rowspan=1 colspan=1>Homozygousmutant</td><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=1>S7</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>20(100%)</td><td rowspan=1 colspan=1>60(100%)</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>140(100%)</td><td rowspan=1 colspan=1>140(100%)</td><td rowspan=1 colspan=1>140(100%)</td><td rowspan=1 colspan=1>420(100%)</td></tr></table>

Clinical Reproducibility: A panel of nine specimens was tested at three sites, two operators per site, with one instrument per site and three lots of reagent lots with one lot per site. Each site had one operator which performed one run per day using one sample preparation method over five non-consecutive days. Reproducibility was assessed using a nine member panel as demonstrated in the table below: four $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples, three contrived whole blood samples and two extracted genomic (gDNA) samples diluted to $0 . 2 \mathrm { n g / \mu L }$ . Below are tables which summarize the overall agreement results of the reproducibility study for Factor II and Factor V by overall genotype and testing site/sample preparation method.

Description of Specimens   

<table><tr><td rowspan=1 colspan=1>Panel Number</td><td rowspan=1 colspan=2>Genotype of Specimen</td><td rowspan=1 colspan=1>Sample Type</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Heterozygous</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Factor II Heterozygous</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Homozygous</td><td rowspan=1 colspan=1>Contrived Whole Blood</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Factor II Homozygous Mutant</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Contrived Whole Blood</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Factor II Heterozygous</td><td rowspan=1 colspan=1>Factor V Heterozygous</td><td rowspan=1 colspan=1>Contrived Whole Blood</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Heterozygous</td><td rowspan=1 colspan=1>Genomic DNA</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Factor II Heterozygous</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Genomic DNA</td></tr></table>

Factor II – Reproducibility Study Summary by Site   

<table><tr><td rowspan=1 colspan=1>GenotypebySequencinga</td><td rowspan=1 colspan=3>Correct Results/Number ofSamples Tested</td><td rowspan=2 colspan=1>IncorrectResults</td><td rowspan=2 colspan=1>InvalidResults</td><td rowspan=2 colspan=1>CorrectResults/Numberof Valid Results(%)</td><td rowspan=2 colspan=1>95%LCBd</td></tr><tr><td rowspan=1 colspan=1>Factor II</td><td rowspan=1 colspan=1>Site1b/Lot1/MethodA</td><td rowspan=1 colspan=1>Site2b/Lot2/MethodB</td><td rowspan=1 colspan=1>Site3b/Lot3/MethodC</td></tr><tr><td rowspan=1 colspan=1>WTe</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>300/300 (100%)</td><td rowspan=1 colspan=1>98.78</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179 (100%)</td><td rowspan=1 colspan=1>97.96</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>94.04</td></tr></table>

aWT: Wild Type; HET: Heterozygous; MUT: Homozygous Mutant bEach site used a different DNA isolation method (A, B, C) and different reagent lot cInvalid result was not retested dTwo-sided $9 5 \%$ lower confidence bound (LCB) (exact method) eAt each site, 20 results were from gDNA samples, and 20 were from contrived whole blood samples fFrom contrived whole blood samples only

Factor V – Reproducibility Study Summary by Site   

<table><tr><td rowspan=1 colspan=1>GenotypebySequencinga</td><td rowspan=1 colspan=3>Correct Results/Number ofSamples Tested</td><td rowspan=2 colspan=1>IncorrectResults</td><td rowspan=2 colspan=1>InvalidResults</td><td rowspan=2 colspan=1>CorrectResults/Numberof Valid Results(%)</td><td rowspan=2 colspan=1>95%LCBd</td></tr><tr><td rowspan=1 colspan=1>Factor V</td><td rowspan=1 colspan=1>Site1b/Lot1/MethodA</td><td rowspan=1 colspan=1>Site2b/Lot2/MethodB</td><td rowspan=1 colspan=1>Site3b/Lot3/MethodC</td></tr><tr><td rowspan=1 colspan=1>WTe</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>300/300 (100%)</td><td rowspan=1 colspan=1>98.78</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179 (100%)</td><td rowspan=1 colspan=1>97.96</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>94.04</td></tr></table>

aWT: Wild Type; HET: Heterozygous; MUT: Homozygous Mutant bEach site used a different DNA isolation method (A, B, C) and different reagent lot cInvalid result was not retested dTwo-sided $9 5 \%$ lower confidence bound (LCB) (exact method) eAt each site, 20 results were from gDNA samples, and 20 were from contrived whole blood samples fFrom contrived whole blood samples only

Factor II – Reproducibility Study Summary by Operator   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>GenotypePanel1</td><td rowspan=1 colspan=1>TotalSamplesTested</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>MissedCalls</td><td rowspan=1 colspan=1>NoCalls{2</td><td rowspan=1 colspan=1>Percent Agrement% (Lower Bound of95% CI3)</td></tr><tr><td rowspan=1 colspan=1>Site1</td><td rowspan=1 colspan=1>Operator1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Operator2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1>Site2</td><td rowspan=1 colspan=1>Operator1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Operator2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0 (88.06)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1>Site3</td><td rowspan=1 colspan=1>Operator1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Operator2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr></table>

1WT: Wild Type; HET: Heterozygous; MUT: Homozygous Mutant 2Invalid results that were not retested $3 9 5 \%$ CI $=$ Two-sided $9 5 \%$ Exact Binomial Confidence Interval

Factor V – Reproducibility Study Summary by Operator   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>GenotypePanel</td><td rowspan=1 colspan=1>TotalSamplesTested</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>MissedCalls</td><td rowspan=1 colspan=1>NoCalls2</td><td rowspan=1 colspan=1>Percent Agrement% (Lower Bound of95% CI3)</td></tr><tr><td rowspan=1 colspan=1>Site1</td><td rowspan=1 colspan=1>Operator1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Operator2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1>Site2</td><td rowspan=1 colspan=1>Operator1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Operator2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0 (88.06)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1>Site3</td><td rowspan=1 colspan=1>Operator1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Operator2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (92.89)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (88.43)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0 (69.15)</td></tr></table>

1WT: Wild Type; HET: Heterozygous; MUT: Homozygous Mutant 2Invalid results that were not retested $3 9 5 \%$ CI $=$ Two-sided $9 5 \%$ Exact Binomial Confidence Interval

$b$ . Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Whole Blood Stability Study: This study was conducted to determine the stability of $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood specimens for isolation of gDNA for testing with the cobas Factor II and Factor V test. The study also compared the performance of the cobas Factor II and Factor $\mathrm { V }$ test on DNA isolated from freshly drawn whole blood samples to the performance on DNA isolated from whole blood samples frozen and thawed up to three times. DNA was isolated from $\mathrm { K _ { 2 } E D T A }$ whole blood was collected from six donors and stored at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ and $\le - 7 0 ^ { \circ } \mathrm { C }$ . In addition, one $\mathrm { K } _ { 2 } \mathrm { E D T A }$ blood collection tube from each donor was placed at: $2 { - } 8 ^ { \circ } \mathrm { C }$ , $2 5 \mathrm { { ^ \circ C } }$ and $3 2 \mathrm { { } ^ { \circ } C }$ . The cobas Factor II and Factor V test yielded the correct genotype results for all samples, storage conditions and time intervals tested, including up to three freeze/thaw cycles at ${ } - 2 0 \mathrm { { } ^ { \circ } C }$ and $\le - 7 0 ^ { \circ } \mathrm { C }$ . Genomic DNA for testing with the cobas Factor II and Factor V test may be isolated from $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood specimens after storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to seven days, at $9 { - } 3 0 ^ { \circ } \mathrm { C }$ for up to three days. The performance of the cobas Factor $\mathrm { I I }$ and Factor $\mathrm { V }$ test on whole blood samples frozen and thawed up to three times was comparable to the performance on fresh, never frozen samples.

Extracted Genomic DNA Stability Study: The stability of gDNA isolated from whole blood for testing with the cobas Factor II and Factor $\mathrm { V }$ test was determined by isolating gDNA from six $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood specimens using three commercially available genomic DNA isolation kits, and testing the gDNA samples with the cobas Factor II and Factor V test after storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ , ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ and $\le - 7 0 ^ { \circ } \mathrm { C }$ . All tests with the cobas Factor II and Factor $\mathrm { V }$ test for all samples, DNA isolation methods, storage conditions and time intervals yielded Factor II and Factor $\mathrm { V }$ genotype results in agreement with DNA sequencing. Genomic DNA isolated from $\mathrm { K _ { 2 } E D T A }$ whole blood for testing with the cobas Factor II and Factor $\mathrm { V }$ test may be stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to seven days or frozen at ${ } _ { - 2 0 \mathrm { { } ^ { \circ } C } }$ to below $- 7 0 \mathrm { { } ^ { \circ } C }$ for at least nine weeks. Genomic DNA may be frozen and thawed up to three times.

Real-time and accelerated stability study: Three lots of the cobas Factor II and Factor V test kit were subjected to real-time stability testing at $2 \mathrm { - } 8 ^ { \circ } \mathrm { C }$ , and accelerated stability testing at $2 5 \mathrm { { ^ \circ C } }$ and $3 7 ^ { \circ } \mathrm { C }$ . The kit performance was assessed at each time point with the kit function release test and by testing genomic DNA extracted from four whole blood specimens at specified timepoints in the real-time study. The accelerated stability testing at $2 5 \mathrm { { ^ \circ C } }$ and $3 7 ^ { \circ } \mathrm { C }$ is complete. The real-time stability testing at $2 { - } 8 ^ { \circ } \mathrm { C }$ is in progress, however real-time stability testing results through seven months from the date of manufacture is complete. The $2 5 \mathrm { { ^ \circ C } }$ and $3 7 ^ { \circ } \mathrm { C }$ accelerated stability studies on all three lots support a shelf life at $2 { - } 8 ^ { \circ } \mathrm { C }$ of 18 months. The initial shelf life assignment for the cobas Factor II and Factor V test, based on the $2 5 \mathrm { { ^ \circ C } }$ and $3 7 ^ { \circ } \mathrm { C }$ accelerated stability studies is 15 months at $2 { - } 8 ^ { \circ } \mathrm { C }$ . The 15-month shelf-life assignment will be confirmed and extended with real-time stability study data.

Open Reagent Stability Study: The stability of the cobas Factor II and Factor V test, after the reagents and controls have been used once, was determined by removing half the volume from multiple vials of the kit reagents and controls, storing and recapped vials at $2 { - } 8 ^ { \circ } \mathrm { C }$ and testing them after approximately 31, 61 and 91 days of storage. At each time interval, the previously opened reagent and control vials were testing with the kit functional release test and by testing gDNA isolated from four $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood specimens. The study was conducted with one lot of the cobas Factor II and Factor V test. After opening and recapping, the reagents and controls were stable at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 90 days or the expiration date on the vials.

Activated Master Mix Stability Study: The stability of the activated Factor II and Factor V (F2F5) master mix (MMX) was determined by preparing activated F2F5 MMX, and testing the activated F2F5 MMX after storage at $3 2 \mathrm { { } ^ { \circ } C }$ for 0, 1, 2, 2.5, 4 and 4.5 hours. Each time interval was tested with the kit functional test and by testing gDNA isolated from four $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples. The study was conducted with three lots of the cobas Factor II and Factor $\mathrm { V }$ test. Activated F2F5 MMX may be stored at room temperature for up to 4 hours.

# d. Detection limit:

Analytical sensitivity - Lower Limit: To determine the minimum input of genomic DNA necessary to yield correct Factor II and Factor V genotype results, genomic DNA was isolated from three K2EDTA whole blood samples (Factor II heterozygous, Factor V heterozygous, Factor V homozygous mutant) and one cell line (Factor II homozygous mutant), using three different commercial DNA isolation methods for the whole blood samples, and one method for the cell line. Each genomic DNA sample was tested with the cobas Factor II and Factor V Test at 10 concentrations: undiluted (which varied between 6 and $3 8 ~ \mathrm { n g / \mu L }$ ) and nine serial dilutions from 1.0 to $0 . 0 0 0 1 ~ \mathrm { n g / u L }$ . Each dilution was tested with 24 replicates with each of two kit lots, for a total of 48 replicates per concentration per genomic DNA sample. The undiluted genomic DNA samples were tested with six replicates with each kit lot, for a total of 12 replicates per genomic DNA sample.

The rate of correct Factor II and Factor $\mathrm { V }$ genotype results across all four samples, all three DNA isolation methods and both kit lots was $9 8 \%$ at $0 . 0 1 ~ \mathrm { n g / \mu L }$ and $100 \%$ at all higher concentrations (see table below). The cobas Factor II and Factor $\mathrm { V }$ Test is designed to yield Invalid results if the input DNA concentration is too low. There were no incorrect genotype results in the study. The limit of detection is $0 . 0 1 ~ \mathrm { n g / \mu L }$ , which is 10 times lower than the lowest recommended DNA input concentration. The table below summaries the study results.

Lower Limit Detection   

<table><tr><td rowspan=2 colspan=1>Concentration(ng/μL)</td><td rowspan=2 colspan=1>Number</td><td rowspan=1 colspan=4>cobas Factor II and Factor V Test</td></tr><tr><td rowspan=1 colspan=1>Number (%)Correct</td><td rowspan=1 colspan=2>Number (%)Incorrect</td><td rowspan=1 colspan=1>Number (%)Invalid</td></tr><tr><td rowspan=1 colspan=1>Undiluted*</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120 (100%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>473 (98%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>7 (2%)</td></tr><tr><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>86 (18%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>394 (82%)</td></tr><tr><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr><tr><td rowspan=1 colspan=1>0.0003</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr><tr><td rowspan=1 colspan=1>0.0001</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=2>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>480 (100%)</td></tr></table>

$^ { * 6 }$ to $3 8 ~ \mathrm { n g / \mu L }$

Analytical Sensitivity - Upper Limit: To evaluate higher concentrations of DNA input for Factor II and Factor $\mathrm { V }$ Test, genomic DNA was isolated from four K2EDTA whole blood samples using three different commercial DNA isolation methods, and concentrated genomic DNA from cell lines were added to yield total DNA concentrations of $3 0 0 ~ \mathrm { { n g / \mu L } }$ , $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $7 5 { \mathrm { n g / \mu L } }$ . Genomic DNA from Factor II heterozygous, Factor $\mathrm { V }$ heterozygous and Factor $\mathrm { V }$ homozygous mutant cell lines were added to genomic DNA from whole blood samples of the same Factor II and

Factor V genotypes. Factor II homozygous mutant cell line DNA was added to genomic DNA from a leukocyte depleted whole blood (LDWB) sample. The genomic DNA samples at $3 0 0 ~ \mathrm { { n g / \mu L } }$ , $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $7 5 { \mathrm { n g / \mu L } }$ were tested with 24 replicates with each of two kit lots, for a total of 48 replicates per concentration per genomic DNA sample. The genomic DNA samples from whole blood without added cell line DNA were tested with six replicates with each kit lot, for a total of 12 replicates per genomic DNA sample. All samples at $3 0 0 ~ \mathrm { { n g / \mu L } }$ , $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $7 5 { \mathrm { n g / \mu L } }$ yielded the correct Factor II and Factor $\mathrm { V }$ genotype results in all tests (see table below). The LDWB samples without added cell line DNA yielded Invalid results, as expected; the other genomic DNA samples from whole blood yielded correct Factor II and Factor V results. The highest recommended DNA input concentration is $1 5 0 ~ \mathrm { { n g / \mu L } }$ . The table below summarizes the study results.

Upper Limit Detection   

<table><tr><td rowspan=1 colspan=2>cYanmaGYellowb</td><td rowspan=1 colspan=3>cobas Factor II and Factor V Test ResultsNumber (%)</td></tr><tr><td rowspan=1 colspan=1>Concentration (ng/μL)</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Number (%)Correct</td><td rowspan=1 colspan=1>Number (%)Incorrect</td><td rowspan=1 colspan=1>Number (%)Invalid</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>576 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>576 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>576 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Genomic DNA from neat wholeblood samples (6 to 38 ng/μL)</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>108 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Leukocyte depleted whole bloodsample*</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

\*Eluates from LDWB sample yielded “Invalid” results due to the absence of leukocytes. These expected results are considered “correct” for the purpose of this study.

# e. Analytical specificity:

Substances: Six $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples representing Factor II and Factor V wild type and heterozygous genotypes were tested for potential interference by the following substances: triglycerides, bilirubin, cholesterol and hemoglobin, $\mathrm { K } _ { 2 } \mathrm { E D T A }$ , heparin, warfarin (Coumadin), rivaroxaban (Xarelto) and dabigatran etexilate (Pradaxa). Triglycerides $( 3 7 \mathrm { m M } )$ , bilirubin (conjugated and unconjugated, $3 4 2 \mu \mathrm { M } )$ , and cholesterol ( $\mathbf { \Lambda } ^ { \mathrm { 1 3 \ m M } }$ did not interfere with the cobas Factor II and Factor $\mathrm { V }$ Test when added to whole blood at the test concentrations recommended in CLSI guideline, EP07-A2. Hemoglobin did not interfere with the cobas Factor II and Factor V Test when added to whole blood at a concentration of $2 0 ~ \mathrm { g / L }$ . The anticoagulant K2EDTA did not interfere when it was tested at $5 . 7 \ : \mathrm { m g / m L }$ , which is approximately three times the concentration of K2EDTA in whole blood when the blood collection tube was filled to capacity. Heparin $\left( 3 0 0 0 \mathrm { U / L } \right)$ , warfarin $( 3 2 . 5 \ \mathrm { \mu m o l / L } )$ , rivaroxaban $( 1 . 5 \mu \mathrm { m o l / L } )$ ), and dabigatran etexilate $( 0 . 6 \mu \mathrm { m o l / L } )$ did not interfere with the cobas Factor II and Factor V Test. Correct genotype results were obtained for all replicates of all the gDNA samples.

For potential interference with extraction buffer and ethanol, these substances were added to gDNA isolated from $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood. A commercial extraction buffer containing guanidine hydrochloride, a common ingredient in commercial DNA extraction buffers, interfered with the cobas Factor II and Factor V Test, causing invalid results when it was present in genomic DNA samples at a concentration of $2 . 5 \%$ (v/v). Ethanol, a common ingredient in the wash buffers of commercial DNA isolation methods, interfered with the cobas Factor II and Factor V Test when added to genomic DNA samples to a concentration of $5 \%$ (v/v). For both substances, observed interference caused invalid results.

Mutations: To determine the effect of known single nucleotide polymorphisms (SNPs) close to the Factor V Leiden mutation $( 1 6 9 1 \mathrm { G } { > } \mathrm { A } )$ and the Factor II (prothrombin) mutation $( 2 0 2 1 0 \mathrm { G } { > } \mathbf { A } )$ ), plasmid DNAs containing eight known SNPs ( $2 0 2 0 7 \mathrm { A } { > } \mathrm { C }$ , $2 0 2 0 9 \mathrm { C } { > } \mathrm { T } _ { \mathrm { i } }$ , $2 0 2 1 8 \mathrm { A } { > } \mathrm { G }$ , $2 0 2 2 1 \mathrm { C } { > } \mathrm { T }$ , $1 6 8 9 \mathrm { G } \mathrm { > } \mathrm { A }$ , $1 6 9 0 \mathrm { C } { > } \mathrm { T }$ , $1 6 9 2 \mathrm { A } { > } \mathrm { C }$ and $1 6 9 6 \mathrm { A } { > } \mathrm { G } )$ in the probe binding regions of the cobas Factor II and Factor V Test were tested. The Factor II SNP plasmids and the Factor V SNP plasmids were wild type at positions 20210 and 1691, respectively. Each SNP plasmid DNA was tested alone, and in combination with wild type Factor II plasmid DNA, wild type Factor V plasmid DNA, wild type Factor II and wild type Factor V plasmid DNAs, and with genomic DNA from wild type whole blood.

None of the SNP plasmids caused false positive results for the Factor II (prothrombin) or Factor V Leiden mutations, and none of the SNP plasmids interfered with detection of the wild type Factor II or Factor V sequences. All four Factor II SNP plasmids and three of four Factor V SNP plasmids were detected as wild type Factor II or Factor V DNA, respectively. One Factor V SNP plasmid $1 6 8 9 \mathrm { G } { > } \mathrm { A } )$ was not detected by the cobas Factor II and Factor V Test. If this SNP is present on both alleles, the test result would be invalid.

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device:

A total of 300 specimens from patients whose routine medical care called for Factor II and/or Factor V measurements were obtained to represent the intended use population. The commercial vendors that provided these samples indicated that the collection included the genotypes as demonstrated in the table below.

Factor II and Factor V genotypes included in the method comparison study   

<table><tr><td rowspan=1 colspan=1>Factor II (F2)</td><td rowspan=1 colspan=1>Factor V (F5)</td><td rowspan=1 colspan=1>Specimen Type</td></tr><tr><td rowspan=1 colspan=1>Wild Type (WT F2)</td><td rowspan=1 colspan=1>Wild Type (WT F5)</td><td rowspan=1 colspan=1>K2EDTA whole blood</td></tr><tr><td rowspan=1 colspan=1>Wild Type (WT F2)</td><td rowspan=1 colspan=1>Heterozygous (HET F5)</td><td rowspan=1 colspan=1>K2EDTA whole blood</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (HET F2)</td><td rowspan=1 colspan=1>Wild Type (WT F5)</td><td rowspan=1 colspan=1>K2EDTA whole blood</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (HET F2)</td><td rowspan=1 colspan=1>Heterozygous (HET F5)</td><td rowspan=1 colspan=1>K2EDTA whole blood</td></tr><tr><td rowspan=1 colspan=1>Wild Type (WT F2)</td><td rowspan=1 colspan=1>Homozygouss (MUT F5)</td><td rowspan=1 colspan=1>K2EDTA whole blood</td></tr><tr><td rowspan=1 colspan=1>Homozygous (MUT F2)</td><td rowspan=1 colspan=1>Wild Type (WT F5)</td><td rowspan=1 colspan=1>K2EDTA whole blood and gDNA</td></tr></table>

One external site conducted testing with the cobas Factor II and Factor V test. One commercial laboratory performed bi-directional Sanger sequencing. The 300 samples were randomized and tested by two operators. All samples were tested for Factor II and Factor V by both cobas Factor II and Factor V Test and Sanger sequencing. All runs and tests were valid for the cobas test and no repeat tests were performed. A test result was classified as correct for Factor II or Factor V if both the cobas test and Sanger sequencing detected the same genotype.

The table below presents agreement between the cobas test and Sanger sequencing for Factor II results. The Overall Percentage Agreement (OPA) between the two tests was $100 \%$ with a two-sided $9 5 \%$ lower confidence bound (exact method) of $9 8 . 7 8 \%$ . Both Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) were $100 \%$ with lower confidence bounds of $9 7 . 5 9 \%$ for PPA and $9 7 . 5 5 \%$ for NPA. The percent agreement of correct results for heterozygous and homozygous mutant genotypes were both $100 \%$ .

Performance of the cobas Factor II and Factor V Test using Bi-directional Sanger Sequencing as a Reference for the Identification of Factor II Genotype   

<table><tr><td rowspan=1 colspan=1>Genotype bySequencing</td><td rowspan=1 colspan=1>TotalSamplesTested</td><td rowspan=1 colspan=1>CorrectCalls1</td><td rowspan=1 colspan=1>No callsorInvalidResults²</td><td rowspan=1 colspan=1>Missed orIncorrectCalls3</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>95%LCB4</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% NPA</td><td rowspan=1 colspan=1>97.55%</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% PPA</td><td rowspan=1 colspan=1>97.59%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.20%</td></tr><tr><td rowspan=1 colspan=1>Mutant5</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.89%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% OPA</td><td rowspan=1 colspan=1>98.78%</td></tr></table>

1cobas test Factor II genotype results in agreement with the Factor II genotype by Sanger sequencing 2Invalid or failed cobas result (no Factor II genotype call) 3cobas test Factor $\mathrm { I I }$ genotype results discordant with the Factor II genotype by sequencing 4Two-sided $9 5 \%$ lower confidence boundary is calculated using Exact method 5Homozygous mutant

For Factor V, the table below presents agreement between the cobas test and Sanger sequencing for Factor V results. The OPA between the two tests was $100 \%$ with a two-sided $9 5 \%$ lower confidence bound (exact method) of $9 8 . 7 8 \%$ . Both PPA and NPA were $100 \%$ with lower confidence bounds of $9 7 . 6 2 \%$ for PPA and $9 7 . 5 2 \%$ for NPA. The percent agreement of correct results for heterozygous and homozygous mutant genotypes were both $100 \%$ .

Performance of the cobas Factor II and Factor V Test using Bi-Directional Sanger Sequencing as a Reference for the Identification of Factor V Genotype   

<table><tr><td rowspan=1 colspan=1>Genotype bySequencing</td><td rowspan=1 colspan=1>TotalSamplesTested</td><td rowspan=1 colspan=1>CorrectCalls1</td><td rowspan=1 colspan=1>No callsorInvalidResults2</td><td rowspan=1 colspan=1>Missed orIncorrectCalls3</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>95%LCB4</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% NPA</td><td rowspan=1 colspan=1>97.52%</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% PPA</td><td rowspan=1 colspan=1>97.62%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.20%</td></tr><tr><td rowspan=1 colspan=1>Mutant5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85.18%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% OPA</td><td rowspan=1 colspan=1>98.78%</td></tr></table>

1cobas test Factor V genotype results in agreement with the Factor V genotype by Sanger sequencing 2Invalid or failed cobas result (no Factor V genotype call) 3cobas test Factor V genotype results discordant with the Factor V genotype by sequencing 4Two-sided $9 5 \%$ lower confidence boundary is calculated using Exact method 5Homozygous mutant

The table below presents method comparison study results for the combined Factor II and Factor V results.

<table><tr><td rowspan=2 colspan=1>Bi-DirectionalSangerSequencingResult</td><td rowspan=1 colspan=6>cobas Factor II and Factor V Test Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>HETF2/HETF5</td><td rowspan=1 colspan=1>HETF2/WTF5</td><td rowspan=1 colspan=1>MUTF2/WTF5</td><td rowspan=1 colspan=1>WTF2/HETF5</td><td rowspan=1 colspan=1>WTF2/MUTF5</td><td rowspan=1 colspan=1>WTF2/WTF5</td></tr><tr><td rowspan=1 colspan=1>HET F2/HET F5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>HET F2/WT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>MUT F2/WT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>WT F2/HET F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>WT F2/MUT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>WT F2/WT F5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>300</td></tr></table>

b. Matrix comparison:

Matched frozen and fresh $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples were compared and resulted in $100 \%$ correct genotype results.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable

b. Clinical specificity:

Not applicable

c. Other clinical supportive data (when a. and b. are not applicable):

DNA Extraction Method Study: Genomic DNA was isolated from 15 whole blood samples using three commercially available DNA isolation methods according to the manufacturer’s instructions, by two operators for three days, for a total of six DNA isolations per sample with each DNA isolation method. Each isolated gDNA sample was tested in triplicate with the cobas Factor II and Factor V Test. One hundred percent of the results with the cobas Factor II and Factor V Test were in agreement with the Factor II and Factor V genotype by bi-directional Sanger sequencing. One gDNA sample was excluded from the results as it was rust colored and yielded invalid results in all three tests. The table below summaries the results from the DNA extraction method study.

<table><tr><td rowspan=2 colspan=1>DNA IsolationMethod</td><td rowspan=1 colspan=2>Total number of</td><td rowspan=1 colspan=3>Number (%) of</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>DNAIsolations</td><td rowspan=1 colspan=1>Tests</td><td rowspan=1 colspan=2>Correct Results</td><td rowspan=1 colspan=1>IncorrectResults</td><td rowspan=1 colspan=1>InvalidResults</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>267a(98.9%)</td><td rowspan=1 colspan=1>(96.8 99.8)b</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>3a (1.1%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>268c(99.3%)</td><td rowspan=1 colspan=1>(97.4-99.9)b</td><td rowspan=1 colspan=1>1c (0.4%)</td><td rowspan=1 colspan=1>1c (0.4%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270(100%)</td><td rowspan=1 colspan=1>98.6-100)b</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>805(99.4%)</td><td rowspan=1 colspan=1>(98.6 99..8)b</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>4 (0.5%)</td></tr></table>

aOne of 90 DNA samples isolated with method A was rust-colored and generated three invalid results. DNA samples with any appearance other than clear and colorless should not be tested, as they may yield invalid or incorrect results.   
$6 9 5 \%$ two-sided, confidence interval   
cThe triplicate results from one sample isolated with method B were inconsistent: one correct, one incorrect, one invalid. The sample eluate was re-tested in triplicate and all results were correct upon testing.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range: Not applicable

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.