January 12, 2018

Roche Molecular Systems, Inc. Pooja Shah Sr. Regulatory Affairs Specialist 4300 Hacienda Drive Pleasanton, California 94588

Re: K172913 Trade/Device Name: cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test Regulation Number: 21 CFR 864.7280 Regulation Name: Factor V Leiden DNA mutation detection systems Regulatory Class: Class II Product Codes: NPR, NPQ Dated: December 13, 2017 Received: December 14, 2017

Dear Pooja Shah:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR

Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice   
(https://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/) and CDRH Learn   
(http://www.fda.gov/Training/CDRHLearn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (http://www.fda.gov/DICE) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).   
Lea Carrington   
Director   
Division of Immunology and Hematology Devices   
Office of In Vitro Diagnostics and Radiological Health   
Center for Devices and Radiological Health

![](images/9e139ca290e275ab62abb1a96f2aec510c911eeca8f1d3e42f77eb28d4437259.jpg)

Enclosure

Device Name cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# Attachment 3. 510k Summary

# cobas® Factor II and Factor V Test 510(k) Summary

This summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

<table><tr><td rowspan=1 colspan=1>Submitter Name</td><td rowspan=1 colspan=1>Roche Molecular Systems, Inc.</td></tr><tr><td rowspan=1 colspan=1>Address</td><td rowspan=1 colspan=1>4300 Hacienda Drive, Pleasanton, CA 94588-2722 USA</td></tr><tr><td rowspan=1 colspan=1>Contact</td><td rowspan=1 colspan=1>Pooja ShahPhone: 925-596-8342FAX: 925-225-0207Email: pooja.shah.ps1@roche.com</td></tr><tr><td rowspan=1 colspan=1>Date Prepared</td><td rowspan=1 colspan=1>September 13th 2017</td></tr><tr><td rowspan=1 colspan=1>Proprietary Name</td><td rowspan=1 colspan=1>cobas® Factor II and Factor V Test</td></tr><tr><td rowspan=1 colspan=1>Common Name</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Classification Name</td><td rowspan=1 colspan=1>Factor V Leiden DNA mutation detection system</td></tr><tr><td rowspan=1 colspan=1>Product Codes</td><td rowspan=1 colspan=1>NPR,NPQ 21 CFR 864.7280</td></tr><tr><td rowspan=1 colspan=1>Predicate Devices</td><td rowspan=1 colspan=1>K033612 &amp; K033607</td></tr><tr><td rowspan=1 colspan=1>Establishment Registration</td><td rowspan=1 colspan=1>2243471 &amp; 3004141078</td></tr></table>

# 1. DEVICE DESCRIPTION

The cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test is a real-time polymerase chain reaction (PCR) test for the qualitative detection and genotyping of a single point mutation in the human Factor V gene (G to A at position 1691, referred to as the Factor V Leiden mutation) and a single point mutation in the human Factor II gene (G to A at position 20210), in genomic DNA isolated from peripheral whole blood, as an aid in diagnosis of patients with suspected thrombophilia.

DNA is extracted offline from whole blood specimens collected in K2EDTA tubes. The userselected DNA extraction method must provide DNA of sufficient concentration. Automated realtime PCR is then performed on the cobas $\boldsymbol { z } 4 8 0$ analyzer. The Factor II and Factor V mutations are detected simultaneously in the same real-time PCR reaction. Depending upon the test order, results for one or both mutations are reported for each DNA sample.

The assay consists of one reagent kit and a system platform. The reagent kit provides the necessary reagents and controls to perform automated real-time PCR amplification and detection. The system platform consists of a real-time PCR thermal cycler (cobas $\boldsymbol { z } 4 8 0$ analyzer) and a control unit (cobas $^ \mathrm { \textregistered }$ 4800 System CU). The amplification reactions generate a Factor II specific amplicon and a Factor V specific amplicon in all samples, and utilize four fluorescent-dye labeled oligonucleotide probes for detection of the Factor II and Factor V genotypes.

# 2. INDICATIONS FOR USE

The cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test is an in vitro diagnostic device that uses real-time PCR for the detection and genotyping of the Factor II (Prothrombin) G20210A mutation and the Factor V Leiden mutation G1691A in genomic DNA obtained from K2EDTA whole blood specimens as an aid in diagnosis of patients with suspected thrombophilia. The cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test and cobas $\boldsymbol { z } 4 8 0$ analyzer are used together for automated amplification and detection.

# 3. TECHNOLOGICAL CHARACTERISTICS

The cobas® Factor II and Factor V Test has the same general intended use, technological characteristics and principles of operation as the two predicate devices. A substantial equivalence table comparing the similarities and differences between the cobas® Factor II and Factor V Test and its predicate devices is provided in Table 1

# Table 1: Similarities and Differences between the cobas® Factor II and Factor V Test and the Predicate Device

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Submitted Device:cobas® Factor II and Factor VTest</td><td colspan="1" rowspan="1">Predicate Device:Roche LightCyclerFactor II (Prothrombin) TestK033612</td><td colspan="1" rowspan="1">Predicate Device:Roche LightCyclerFactor V Leiden TestK033607</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The cobas® Factor II and FactorV Test is an in vitro diagnosticdevice that uses real-time PCRfor the detection and genotypingof the Factor II (Prothrombin)G20210A mutation and theFactor V Leiden mutation ingenomic DNA obtained fromK2EDTA whole blood specimensas an aid in diagnosis of patientswith suspected thrombophilia.The test is performed on thecobas z 480 analyzer forautomated amplification anddetection.</td><td colspan="1" rowspan="1">Same as the Submitted deviceexcept the predicate devicewas cleared on theLightCycler® 1.2 Instrument.</td><td colspan="1" rowspan="1">Same as the Submitted deviceexcept the predicate devicewas cleared on theLightCycler® 1.2 Instrument.</td></tr><tr><td colspan="1" rowspan="1">Type of Test</td><td colspan="1" rowspan="1">Genotyping Test</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Target ofDetection</td><td colspan="1" rowspan="1">Single nucleotide polymorphism</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Indication forUse</td><td colspan="1" rowspan="1">Aid in the diagnosis of patientswith suspected thrombophilia</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended User</td><td colspan="1" rowspan="1">Health Care Professional</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Purified DNA from human bloodsamples.</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalDetectionPrinciples</td><td colspan="1" rowspan="1">Real-time PCR test forsimultaneous amplification of twotargets and detection of specificSNPs in the PCR amplified DNAsequences (multiplex system).</td><td colspan="1" rowspan="1">Real-time PCR test foramplification of one target anddetection of specific SNP inPCR-amplified DNAsequences</td><td colspan="1" rowspan="1">Real-time PCR test foramplification of one targetand detection of specific SNPin PCR-amplified DNAsequences</td></tr><tr><td colspan="1" rowspan="1">SamplePreparation</td><td colspan="1" rowspan="1">DNA extraction and purificationfrom whole blood, performed off-line</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Submitted Device:cobas® Factor II and Factor VTest</td><td colspan="1" rowspan="1">Predicate Device:Roche LightCyclerFactor II (Prothrombin) TestK033612</td><td colspan="1" rowspan="1">Predicate Device:Roche LightCycler®Factor V Leiden TestK033607</td></tr><tr><td colspan="1" rowspan="1">Oligonucleotideprobes andprimers</td><td colspan="1" rowspan="1">Specific for Factor V G1691A,Factor II G20210A and the wildtype Factor II and Factor Vsequences</td><td colspan="1" rowspan="1">Specific for Factor II G20210Aand Factor II wild typesequences.</td><td colspan="1" rowspan="1">Specific for Factor V LeidenG1691A and Factor V wildtype sequences</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Fluorogenic detection of SNPs ineach PCR amplification productby allele-specific cleavage ofTaqMan probes</td><td colspan="1" rowspan="1">Fluorogenic detection of SNPin PCR amplification productby melting curve analysis</td><td colspan="1" rowspan="1">Fluorogenic detection of SNPin PCR amplification productby melting curve analysis</td></tr><tr><td colspan="1" rowspan="1">AnalyticalSensitivity</td><td colspan="1" rowspan="1">&lt; 50 allele copies (0.01 ng inputDNA/reaction)</td><td colspan="1" rowspan="1">Approximately 50 allelecopies per reaction</td><td colspan="1" rowspan="1">Approximately 50 allelecopies per reaction</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">cobas® 4800 System</td><td colspan="1" rowspan="1">Roche LightCycler® version1.2</td><td colspan="1" rowspan="1">Roche LightCycler® version1.2</td></tr><tr><td colspan="1" rowspan="1">Controls used</td><td colspan="1" rowspan="1">Endogenous Internal control(each gene is internal control forother gene), plus externalpositive and negative controlsrequired in each run.</td><td colspan="1" rowspan="1">External positive and negativecontrols required in each run.</td><td colspan="1" rowspan="1">External positive and negativecontrols required in each run.</td></tr><tr><td colspan="1" rowspan="1">ReferenceMethod</td><td colspan="1" rowspan="1">Bi-directional Sanger sequencing</td><td colspan="1" rowspan="1">Sanger Sequencing</td><td colspan="1" rowspan="1">Sanger Sequencing</td></tr></table>

# 4. NON-CLINICAL PERFORMANCE EVALUATION

# 4.1. Analytical sensitivity (Lower Limit)

To determine the minimum input of genomic DNA necessary to yield correct Factor II and Factor V genotype results, genomic DNA was isolated from three $\mathrm { K _ { 2 } E D T A }$ whole blood samples (Factor II heterozygous, Factor V heterozygous, Factor V homozygous mutant) and one cell line (Factor II homozygous mutant), using three different commercial DNA isolation methods for the whole blood samples, and one method for the cell line. Each genomic DNA sample was tested with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test at 10 concentrations: undiluted (which varied between 6 and $3 8 \mathrm { n g / \mu L }$ ) and nine serial dilutions from 1.0 to $0 . 0 0 0 1 ~ \mathrm { n g / \mu L }$ . Each dilution was tested with 24 replicates with each of two kit lots, for a total of 48 replicates per concentration per genomic DNA sample. The undiluted genomic DNA samples were tested with six replicates with each kit lot, for a total of 12 replicates per genomic DNA sample.

The rate of correct Factor II and Factor $\mathrm { V }$ genotype results across all four samples, all three DNA isolation methods and both kit lots was $9 8 \%$ at $0 . 0 1 ~ \mathrm { n g / \mu L }$ and $100 \%$ at all higher concentrations (Table 2). The cobas $^ \mathrm { \textregistered }$ Factor II and Factor $\mathrm { V }$ Test is designed to yield Invalid results if the input DNA concentration is too low. There were no incorrect genotype results in the study. The limit of detection is $0 . 0 1 ~ \mathrm { n g / \mu L }$ , which is 10 times lower than the lowest recommended DNA input concentration.

Table 2: Analytical sensitivity of the cobas® Factor II and Factor V Test   

<table><tr><td rowspan=2 colspan=1>Concentration(ng/μL)</td><td rowspan=2 colspan=1>Number</td><td rowspan=1 colspan=3>cobas® Factor II and Factor V Test Results</td></tr><tr><td rowspan=1 colspan=1>Number (%) Correct</td><td rowspan=1 colspan=1>Number (%) Incorrect</td><td rowspan=1 colspan=1>Number (%) Invalid</td></tr><tr><td rowspan=1 colspan=1>Undiluted*</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>480 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>473 (98%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>7 (2%)</td></tr><tr><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>86 (18%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>394 (82%)</td></tr><tr><td rowspan=1 colspan=1>0.001</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr><tr><td rowspan=1 colspan=1>0.0003</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr><tr><td rowspan=1 colspan=1>0.0001</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>480 (100%)</td></tr></table>

$^ { \star } 6$ to $3 8 ~ \mathsf { n g } / \mu \mathsf { L }$

# 4.2. Analytical sensitivity (Upper Limit)

To evaluate higher concentrations of DNA input for the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test, genomic DNA was isolated from four $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples using three different commercial DNA isolation methods, and concentrated genomic DNAs from cell lines were added to yield total DNA concentrations of $3 0 0 ~ \mathrm { { n g / \mu L } }$ , $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $7 5 ~ \mathrm { n g / \mu L }$ . Genomic DNAs from Factor II heterozygous, Factor $\mathrm { V }$ heterozygous and Factor $\mathrm { V }$ homozygous mutant cell lines were added to genomic DNAs from whole blood samples of the same Factor II and Factor $\mathrm { V }$ genotypes. Factor II homozygous mutant cell line DNA was added to genomic DNA from a leukocyte depleted whole blood (LDWB) sample. The genomic DNA samples at $3 0 0 ~ \mathrm { { n g / \mu L } }$ , $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $7 5 { \mathrm { n g / \mu L } }$ were tested with 24 replicates with each of two kit lots, for a total of 48 replicates per concentration per genomic DNA sample. The genomic DNA samples from whole blood without added cell line DNA were tested with six replicates with each kit lot, for a total of 12 replicates per genomic DNA sample. All samples at $3 0 0 ~ \mathrm { { n g / \mu L } }$ , $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $7 5 { \mathrm { n g / \mu L } }$ yielded the correct Factor II and Factor $\mathrm { V }$ genotype results in all tests (Table 3). The LDWB samples without added cell line DNA yielded Invalid results, as expected; the other genomic DNA samples from whole blood yielded correct Factor II and Factor V results. The highest recommended DNA input concentration is $1 5 0 ~ \mathrm { { n g / \mu L } }$ .

Table 3: Testing of higher DNA input for the cobas® Factor II and Factor V Test   

<table><tr><td rowspan=1 colspan=2>EY</td><td rowspan=1 colspan=3>cobas® Factor II and Factor V Test Results Number (%)</td></tr><tr><td rowspan=1 colspan=1>Concentration (ng/uL)</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Number (%)Correct</td><td rowspan=1 colspan=1>Number (%)Incorrect</td><td rowspan=1 colspan=1>Number (%)Invalid</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>576 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>576 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>576 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Genomic DNA from neat wholeblood samples(6 to 38 ng/μL)</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>108 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Leukocyte depletedwhole blood sample*</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

\*Eluates from leukocyte-depleted whole blood sample yielded “Invalid” results due to the absence of leukocytes. These expected results are considered “correct” for the purpose of the study.

# 4.3. DNA Extraction Method Study

Genomic DNA was isolated from fifteen whole blood samples using three commercially available DNA isolation methods according to the manufacturer’s instructions, by two operators for three days, for a total of six DNA isolations per sample with each DNA isolation method. Each isolated genomic DNA sample was tested in triplicate with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test (Table 4). One hundred percent of the results with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test were in agreement with the Factor II and Factor V genotype by bi-directional Sanger sequencing. One genomic DNA sample isolated with method A was excluded from the results. It was rust colored and yielded invalid results in all three tests. Isolated DNA samples should appear clear and colorless. DNA samples with any appearance other than clear and colorless should not be tested as they may yield invalid or incorrect results.

Table 4: Results by DNA Isolation Methods   

<table><tr><td rowspan=2 colspan=1>DNA IsolationMethod</td><td rowspan=1 colspan=2>Total Number of</td><td rowspan=1 colspan=4>Number (%) of</td></tr><tr><td rowspan=1 colspan=1>DNAIsolations</td><td rowspan=1 colspan=1>Tests</td><td rowspan=1 colspan=2>Correct Results</td><td rowspan=1 colspan=1>IncorrectResults</td><td rowspan=1 colspan=1>InvalidResults</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>267^(98.9%)</td><td rowspan=1 colspan=1>(96.8 - 99.8)b</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>$3^ (11%)</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>268(99.3%)</td><td rowspan=1 colspan=1>(97.4-99.9)b</td><td rowspan=1 colspan=1>$1(0.4%</td><td rowspan=1 colspan=1>$1(0.4%)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270 (100.0%)</td><td rowspan=1 colspan=1>(98.6 -100)b</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>805 (99.4%)</td><td rowspan=1 colspan=1>(98.6 - 99.8)b</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>4 (0.5%)</td></tr></table>

a One of 90 DNA samples isolated with method A was rust-colored and generated 3 invalid results. Only clear and colorless DNA samples should be tested. DNA samples with any appearance other than clear and colorless should not be tested, as they may yield invalid or incorrect results. $^ \circ \geqslant 5 \%$ two-sided, confidence interval c The triplicate results from one sample isolated with method B were inconsistent: 1 correct, 1 incorrect, 1 invalid. The sample eluate was re-tested in triplicate and all results were correct upon re-testing.

# 4.4. Analytical Specificity (Potentially Interfering Mutation Study)

To determine the effect of known single nucleotide polymorphisms (SNPs) close to the Factor V Leiden mutation $1 6 9 1 \mathrm { G } { > } \mathrm { A } _ { . }$ ) and the Factor II (prothrombin) mutation $2 0 2 1 0 \mathrm { G } { > } \mathrm { A } _ { \it { i } }$ ), plasmid DNAs containing eight known SNPs (20207A ${ } _ { > } \mathrm { C }$ , 20209C $>$ T, 20218A>G, 20221C>T, $1 6 8 9 \mathrm { G } \mathrm { > } \mathrm { A }$ , 1690C>T, $1 6 9 2 \mathrm { A } { > } \mathrm { C }$ and $1 6 9 6 \mathrm { A } { > } \mathrm { G }$ were tested. The Factor II SNP plasmids and the Factor V SNP plasmids were wild type at positions 20210 and 1691, respectively. Each SNP plasmid DNA was tested alone, and in combination with wild type Factor II plasmid DNA, wild type Factor V plasmid DNA, wild type Factor II and wild type Factor V plasmid DNAs, and with genomic DNA from wild type whole blood.

None of the SNP plasmids caused false positive results for the Factor II (prothrombin) or Factor V Leiden mutations, and none of the SNP plasmids interfered with detection of the wild type Factor II or Factor V sequences. All four Factor II SNP plasmids and three of four Factor V SNP plasmids were detected as wild type Factor II or Factor V DNA, respectively. One Factor V SNP plasmid $( 1 6 8 9 \mathrm { G } { > } \mathrm { A } )$ was not detected by the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test. If this SNP is present on both alleles, the test result would be Invalid.

# 4.5. Potential Interfering Substances Study

Triglycerides $( 3 7 { \mathrm { m M } } )$ , bilirubin (conjugated and unconjugated, $3 4 2 \mu \mathrm { M }$ ), and cholesterol ( $\mathbf { \rho } _ { 1 3 \mathrm { m M } }$ did not interfere with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test when added to whole blood at the test concentrations recommended in CLSI guideline, EP07-A2.11 Hemoglobin did not interfere with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor $\mathrm { V }$ Test when spiked into whole blood to yield a total hemoglobin concentration of ${ \sim } 2 5 . 8 - 3 1 ~ \mathrm { g / d L }$ .

The anticoagulant $\mathrm { K } _ { 2 } \mathrm { E D T A }$ did not interfere when it was tested at $5 . 7 \ : \mathrm { m g / m L }$ , which is approximately 3 times the concentration of $\mathrm { K } _ { 2 } \mathrm { E D T A }$ in whole blood when the blood collection tube is filled to capacity. Heparin, coumadin (Warfarin), rivaroxaban (Xarelto), and dabigatran etexilate (Pradaxa) did not interfere with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test.

A commercial extraction buffer containing guanidine hydrochloride, a common ingredient in commercial DNA extraction buffers, interfered with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test, causing invalid results when it was present in genomic DNA samples at a concentration of $2 . 5 \%$ (v/v). Ethanol, a common ingredient in the wash buffers of commercial DNA isolation methods, interfered with the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test when added to genomic DNA samples to a concentration of $5 \%$ (v/v). For both substances, observed interference caused invalid results.

# 4.6. Lot-to-Lot Repeatability

The lot-to-lot repeatability of the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test was evaluated by testing genomic DNA samples isolated from seven $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples with three lots of the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test. The study was conducted over five non-consecutive days with two operators, two cobas $\mathbf { z } 4 8 0$ analyzers, one run per day per kit lot, and two replicates per run, for a total of 60 replicates of each sample (Table 5). The cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test results were compared to the Factor II and Factor V genotype by bi-directional Sanger sequencing. The overall agreement between cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test results and nucleic acid sequencing was $100 \%$ across all samples and reagent lots (one-sided, lower $9 5 \%$ confidence limit $9 9 . 3 \%$ )

Table 5: Lot-to-lot Repeatability   

<table><tr><td rowspan=2 colspan=1>SampleID</td><td rowspan=2 colspan=1>Factor IIGenotype</td><td rowspan=2 colspan=1>Factor VGenotype</td><td rowspan=2 colspan=1>Number ofTests perLot</td><td rowspan=1 colspan=4>Number (%) Correct Genotype Results</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>All Lots</td></tr><tr><td rowspan=1 colspan=1>S1</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=1>S2</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=1>S3</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=1>S4</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=1>S5</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>Homozygousmutant</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=1>S6</td><td rowspan=1 colspan=1>Homozygousmutant</td><td rowspan=1 colspan=1>Wild type</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=1>S7</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>60 (100%)</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>140 (100%)</td><td rowspan=1 colspan=1>140 (100%)</td><td rowspan=1 colspan=1>140 (100%)</td><td rowspan=1 colspan=1>420 (100%)</td></tr></table>

# 5. CLINICAL PERFORMANCE EVALUATION

# 5.1. Method Comparison Study

# 5.1.1. Study Objective

The objective of this investigational protocol was to compare the performance of the cobas® Factor II and Factor V Test to bi-directional Sanger sequencing for the identification of the Factor II and Factor V mutations in DNA isolated from whole blood specimens from patients with suspected thrombophilia.

# 5.1.2. Methodology

# 5.1.2.1. Study Design

A total of 300 specimens from patients whose routine medical care called for Factor II and/or Factor V measurements were obtained to represent the intended use population. The commercial vendors that provided these samples indicated that the collection included: individual Factor II and Factor V heterozygous mutations, individual homozygous Factor II and Factor V mutations, compound heterozygous mutations, and samples that were wild type for Factor II and Factor V Table 6.

There were 284 samples that were whole blood and represented frequently found genotypes for Factor II and Factor V. The majority of rare mutations such as Factor II homozygous/ Factor V Wild Type were acquired as DNA samples without identified extraction methodology.

Table 6: Factor II and Factor V genotypes included in the study   

<table><tr><td rowspan=1 colspan=1>Factor II (F2)</td><td rowspan=1 colspan=1>Factor V (F5)</td><td rowspan=1 colspan=1>Specimen Type</td></tr><tr><td rowspan=1 colspan=1>Wild Type (WT F2)</td><td rowspan=1 colspan=1>Wild Type (WT F5)</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>Wild Type (WT F2)</td><td rowspan=1 colspan=1>Heterozygous (HET F5)</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (HET F2)</td><td rowspan=1 colspan=1>Wild Type (WT F5)</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>Heterozygous (HET F2)</td><td rowspan=1 colspan=1>Heterozygous (HET F5)</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>Wild Type (WT F2)</td><td rowspan=1 colspan=1>Homozygous Mutant (MUT F5)</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>Homozygous Mutant (MUT F2)</td><td rowspan=1 colspan=1>Wild Type (WT F5)</td><td rowspan=1 colspan=1>Whole Blood and DNA</td></tr></table>

# 5.1.2.2. Statistical Methods

The performance of the cobas® Factor II and Factor V Test was evaluated separately for Factor II and Factor V genotypes by using bi-directional Sanger sequencing as the reference method. Positive Percent Agreement (PPA), Negative Percent Agreement (NPA) and Overall Percentage Agreement (OPA) were estimated at the genotype level (Wild Type, Heterozygous, and Homozygous Mutant) separately for Factor II and Factor V.

# 5.1.3. Results

A total of 300 samples were tested for Factor II and Factor V by both cobas® Factor II and Factor V Test and Sanger sequencing. All runs and tests were valid for cobas® test and no repeat tests were performed. A test result was classified as correct for Factor II or Factor V, if both the cobas® test and Sanger sequencing detect the same genotype. A test result was classified as an incorrect for Factor II or Factor $\mathrm { V }$ if the cobas® test and Sanger sequencing detect a different genotype for Factor II or Factor V.

Table 7 presents agreement between the cobas® test and Sanger sequencing for Factor II results. The OPA between the two tests was $100 \%$ with a two-sided $9 5 \%$ lower confidence bound (exact method) of $9 8 . 7 8 \%$ . Both PPA and NPA were $100 \%$ with lower confidence bounds of

$9 7 . 5 9 \%$ for PPA and $9 7 . 5 5 \%$ for NPA. The percent agreement of correct results for Heterozygous and Homozygous Mutant genotypes were both $100 \%$ .

Table 7: Performance of the cobas® Factor II and Factor V Test using Bi-Directional Sanger Sequencing as a Reference for the Identification of Factor II Genotype   

<table><tr><td rowspan=1 colspan=1>Genotype bySequencing</td><td rowspan=1 colspan=1>TotalSamplesTested</td><td rowspan=1 colspan=1>CorrectCalls1</td><td rowspan=1 colspan=1>No calls orInvalidResults2</td><td rowspan=1 colspan=1>Missed orIncorrectCalls3$</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>95%LCB4</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(NPA)</td><td rowspan=1 colspan=1>97.55%</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(PPA)</td><td rowspan=1 colspan=1>97.59%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.20%</td></tr><tr><td rowspan=1 colspan=1>Mutant5</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.89%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(OPA)</td><td rowspan=1 colspan=1>98.78%</td></tr></table>

1 cobas® test Factor II genotype results in agreement with the Factor II genotype by sequencing 2 Invalid or failed cobas® result (no Factor II genotype call) 3 cobas® test Factor II genotype results discordant with the Factor II genotype by sequencing 4 Two-sided $9 5 \%$ lower confidence boundary is calculated using Exact method 5 Homozygous mutant

Table 8 presents agreement between the cobas® test and Sanger sequencing for Factor V results The OPA between the two tests was $100 \%$ with a two-sided $9 5 \%$ lower confidence bound (exact method) of $9 8 . 7 8 \%$ . Both PPA and NPA were $100 \%$ with lower confidence bounds of $9 7 . 6 2 \%$ for PPA and $9 7 . 5 2 \%$ for NPA. The percent agreement of correct results for Heterozygous and Homozygous Mutant genotypes were both $100 \%$ .

Table 8: Performance of the cobas® Factor II and Factor V Test using Bi-Directional Sanger Sequencing as a Reference for the Identification of Factor V Genotype   

<table><tr><td rowspan=1 colspan=1>Genotype bySequencing</td><td rowspan=1 colspan=1>TotalSamplesTested</td><td rowspan=1 colspan=1>CorrectCalls1</td><td rowspan=1 colspan=1>No calls orInvalidResults²</td><td rowspan=1 colspan=1>Missed orIncorrectCalls3</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>95% LCB4</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(NPA)</td><td rowspan=1 colspan=1>97.52%</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(PPA)</td><td rowspan=1 colspan=1>97.62%</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.20%</td></tr><tr><td rowspan=1 colspan=1>Mutant5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85.18%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(OPA)</td><td rowspan=1 colspan=1>98.78%</td></tr></table>

1 cobas® test Factor V genotype results in agreement with the Factor V genotype by sequencing 2 Invalid or failed cobas® result (no Factor V genotype call) 3 cobas® test Factor V genotype results discordant with the Factor V genotype by sequencing 4 Two-sided $9 5 \%$ lower confidence boundary is calculated using Exact method 5 Homozygous mutant

Table 9: Performance of the cobas $^ \mathrm { \textregistered }$ Factor II and Factor V Test Using Bi-Directional Sanger Sequencing as a Reference for the Identification of Combined Factor II and Factor V Result   

<table><tr><td rowspan=2 colspan=1>Bi-DirectionalSangersequencingResult</td><td rowspan=1 colspan=6>cobas®Factor II and Factor V Test Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>HET F2IHET F5</td><td rowspan=1 colspan=1>HET F2IWT F5</td><td rowspan=1 colspan=1>MUT F2IWT F5</td><td rowspan=1 colspan=1>WT F2IHET F5</td><td rowspan=1 colspan=1>WT F2IMUT F5</td><td rowspan=1 colspan=1>WT F2IWT F5</td></tr><tr><td rowspan=1 colspan=1>HET F2/ HET F5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>HET F2/ WT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>MUT F2/ WT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>WT F2/ HET F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>WT F2/ MUT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>WT F2/ WT F5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>300</td></tr></table>

# 5.1.4. Conclusion

The results of this Method Comparison Study support the intended use of the cobas® Factor II and Factor V Test for diagnostic purposes as an aid in the clinical management of patients with suspected thrombophilia.

# 5.2. Clinical Reproducibility Study

# 5.2.1. Study Objective

The objective of this study was to evaluate the reproducibility of the cobas® Factor II and Factor V Test for the detection and genotyping of the Factor II (prothrombin) G20210A and Factor V Leiden (G1691A) mutations.

# 5.2.2. Methodology

# 5.2.2.1. Study Design

The reproducibility study included three manual whole blood sample preparation methods. These three whole blood DNA sample preparation kits were intended to be representative of commercially available sample preparation kits for DNA extraction from whole blood.

Reproducibility of the cobas® Factor II and Factor V Test was evaluated with $\mathrm { K } _ { 2 } \mathrm { E D T A }$ whole blood samples and genomic DNA samples across the following factors:

Sample Preparation Method: One (1) sample preparation method per site   
Reagent Lot: Three (3) reagent lots, 1 lot per site   
Test Site: Three (3) sites (2 external and 1 internal)   
Operator: Two (2) operators per site   
Instrument: One (1) instrument per site   
Run: One (1) run per operator per day   
Days: Five (5) non-consecutive days per reagent lot

Reproducibility was assessed using a nine-member panel: four unique $\mathrm { K } _ { 2 }$ EDTA blood samples, three contrived blood samples and two extracted genomic DNA (gDNA) samples diluted to $0 . 2 \mathrm { n g / \mu L }$ .

Table 10: Description of Panel Members   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=2>Genotype of Panel Member</td><td rowspan=1 colspan=1>Sample Type</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>Factor Il Wild Type</td><td rowspan=1 colspan=1>Factor V Heterozygous</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>Factor II Heterozygous</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Whole Blood</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Homozygous mutant</td><td rowspan=1 colspan=1>Contrived Whole Blood</td></tr><tr><td rowspan=1 colspan=1>06</td><td rowspan=1 colspan=1>Factor II Homozygous mutant</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Contrived Whole Blood</td></tr><tr><td rowspan=1 colspan=1>07</td><td rowspan=1 colspan=1>Factor II Heterozygous</td><td rowspan=1 colspan=1>Factor V Heterozygous</td><td rowspan=1 colspan=1>Contrived Whole Blood</td></tr><tr><td rowspan=1 colspan=1>08</td><td rowspan=1 colspan=1>Factor II Wild Type</td><td rowspan=1 colspan=1>Factor V Heterozygous</td><td rowspan=1 colspan=1>Genomic DNA</td></tr><tr><td rowspan=1 colspan=1>09</td><td rowspan=1 colspan=1>Factor II Heterozygous</td><td rowspan=1 colspan=1>Factor V Wild Type</td><td rowspan=1 colspan=1>Genomic DNA</td></tr></table>

# 5.2.2.2. Statistical Analyses

Data were summarized by the percentage of correct results, and the associated one-sided $9 5 \%$ exact lower confidence bounds (LCBs) or two-sided $9 5 \%$ exact confidence interval (CIs) and data were presented by panel member, genotype, site/sample preparation method and sample type (gDNA or whole blood).

For a valid test, a correct result was defined as agreement between the cobas test result and the panel member genotype as determined by Sanger sequencing; an incorrect result was defined as disagreement between the cobas test result and the panel member genotype as determined by Sanger sequencing. Test results are for both Factor II and Factor V and are either valid for both results or invalid for both results. A ‘No Call’ indicated that the test result was not available due to failed or invalid test result. Percent agreement of correct results between Sanger sequencing and cobas® Factor II and Factor V Test was determined from valid results.

# 5.2.3. Results

A total of 38 runs were performed in this study. A subset of 8 runs performed at site 1 with an expired sample preparation lot were determined as not acceptable and excluded from the statistical analysis. These runs were repeated with in-date reagents. The rest of 30 runs are all valid and meet all protocol requirements. A total of 540 tests were performed on the 9 panel members in the 30 valid runs, with 539 valid results and one invalid result. The following results are based on the dataset from these 30 runs.

# 5.2.3.1. Results for Factor II

Table 11 summarizes the overall agreement results of the reproducibility study for Factor II by overall genotype and testing site/sample preparation method. All results from valid tests were correct calls, leading to $100 \%$ agreement for each genotype. With the invalid result treated as an incorrect result, the percent agreement was $9 9 . 4 \%$ for HET and remained $100 \%$ for WT and MUT.

Table 11: Summary of Reproducibility Study -- Factor II   

<table><tr><td rowspan=1 colspan=1>Genotype bySequencinga</td><td rowspan=1 colspan=3>Correct Results/Number of Samples Tested</td><td rowspan=2 colspan=1>IncorrectResults</td><td rowspan=2 colspan=1>InvalidResultsc</td><td rowspan=2 colspan=1>Correct Results/Number of ValidResults (%)</td><td rowspan=2 colspan=1>95% LCBd</td></tr><tr><td rowspan=1 colspan=1>Factor II</td><td rowspan=1 colspan=1>Site1b/Lot1/Method A</td><td rowspan=1 colspan=1>Site2blLot2/Method B</td><td rowspan=1 colspan=1>Site 3blLot3/Method C</td></tr><tr><td rowspan=1 colspan=1>WTe</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>300/300 (100%)</td><td rowspan=1 colspan=1>98.78</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179 (100%)</td><td rowspan=1 colspan=1>97.96</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>94.04</td></tr></table>

a WT: Wild Type; HET: Heterozygous; MUT: Homozygous Mutant b Each site used different DNA isolation method (A, B, C) and different reagent lot c Invalid result that was not retested d Two-sided $9 5 \%$ Lower Confidence Bound (LCB) (exact method) e At each site, 20 results were from gDNA samples, and 20 from contrived whole blood samples f From contrived whole blood samples only

# 5.2.3.2. Results for Factor V

Table 12 summarizes the overall agreement results of the reproducibility study for Factor V by overall genotype and testing site/reagent lot/sample preparation method. All results from valid tests were correct calls, leading to $100 \%$ agreement for each genotype. With the invalid result treated as an incorrect result, the percent agreement was $9 9 . 4 \%$ for HET and remained $100 \%$ for WT and MUT.

Table 12: Summary of Reproducibility Study -- Factor V   

<table><tr><td rowspan=1 colspan=1>Genotype bySequencinga</td><td rowspan=1 colspan=3>Correct Results/Number of Samples Tested</td><td rowspan=2 colspan=1>IncorrectResults</td><td rowspan=2 colspan=1>InvalidResults</td><td rowspan=2 colspan=1>Correct Results/Number of ValidResults (%)</td><td rowspan=2 colspan=1>95% LCBd</td></tr><tr><td rowspan=1 colspan=1>Factor V</td><td rowspan=1 colspan=1>Site1blLot1/Method A</td><td rowspan=1 colspan=1>Site 2blLot2lMethod B</td><td rowspan=1 colspan=1>Site 3blLot3/Method C</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>300/300 (100%)</td><td rowspan=1 colspan=1>98.78</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179 (100%)</td><td rowspan=1 colspan=1>97.96</td></tr><tr><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60/60 (100%)</td><td rowspan=1 colspan=1>94.04</td></tr></table>

a WT: Wild Type; HET: Heterozygous; MUT: Homozygous Mutant b Each site used different DNA isolation method (A, B, C) and different reagent lot c Invalid result that was not retested d Two-sided $9 5 \%$ Lower Confidence Bound (LCB) (exact method) e At each site, 20 results were from gDNA samples, and 20 from contrived whole blood samples f From contrived whole blood samples only

# 5.3. Conclusions

The submitted information in this $5 1 0 ( \mathrm { k } )$ notification demonstrates that the cobas® Factor II and Factor V Test is as safe and effective as the predicate devices and therefore supports a Substantial Equivalence decision.