# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k070597   
B. Purpose for Submission: New device   
C. Measurand: Factor II (Prothrombin) Factor V Leiden 5, 10-Methylenetetrahydrofolate reductase (MTHFR)   
D. Type of Test: Genotype   
E. Applicant: Nanosphere INC   
F. Proprietary and Established Names: Verigene $\textsuperscript { \textregistered }$ F5 Nucleic Acid Test Verigene $\textsuperscript { \textregistered }$ F2 Nucleic Acid Test Verigene $\textsuperscript { \textregistered }$ MTHFR Nucleic Acid Test

# G. Regulatory Information:

1. Regulation section: 21 CFR 864.7280 Factor V Leiden DNA mutation detection systems   
2. Classification: Class II   
3. Product code: NPR, Test, Factor II G20210A mutations, genomic DNA PCR NPQ, Test, Factor V Leiden mutations, genomic DNA PCR OMM, Test, 5, 10-Methylenetetrahydrofolate reductase mutations, genomic DNA PCR NSU, Instrumentation for Clinical Multiplex Test Systems   
4. Panel: Hematology (81); Chemistry (75)

# H. Intended Use:

1. Intended use(s): The Verigene $\textsuperscript { \textregistered }$ F5 Nucleic Acid Test is an in vitro diagnostic for the detection and genotyping of a single point mutation (G to A at position 1691; also known as Factor V Leiden) of the human Factor V gene (F5; Coagulation Factor V gene) in patients with suspected thrombophilia, from isolated genomic DNA obtained from whole blood samples. The test is intended to be used on the Verigene $\textsuperscript { \textregistered }$ System.

The Verigene $\textsuperscript { \textregistered }$ F2 Nucleic Acid Test is an in vitro diagnostic for the detection and genotyping of a single point mutation (G to A at position 20210of the human Factor II gene (F2; prothrombin gene) in patients with suspected thrombophilia, from isolated genomic DNA obtained from whole blood samples. The test is intended to be used on the Verigene $\textsuperscript { \textregistered }$ System.

The Verigene $\textsuperscript { \textregistered }$ MTHFR Nucleic Acid Test is an in vitro diagnostic for the detection and genotyping of a single point mutation (C to T at position 677) of the human 5, 10- methylenetetrahydrofolate reductase gene (MTHFR) in patients with suspected thrombophilia, from isolated genomic DNA obtained from whole blood samples. The test is intended to be used on the Verigene $\textsuperscript { \textregistered }$ System.

2. Indication(s) for use: Same as Intended Use   
3. Special conditions for use statement(s): For Prescription Use Only.   
4. Special instrument requirements: Verigene $\textsuperscript { \textregistered }$ System

# I. Device Description:

The Verigene $\textsuperscript { \textregistered }$ System consists of two instruments, the Verigene $\textsuperscript { \textregistered }$ Processor and the Verigene $\textsuperscript { \textregistered }$ Reader. It utilizes single-use, disposable test Cartridges to process and genotype multiple genes in a DNA sample in ${ \sim } 1 \ \%$ hrs. Up to 8 Verigene $\textsuperscript { \textregistered }$ Processors may be connected to a single Verigene $\textsuperscript { \textregistered }$ Reader. Each processor contains 4 hybridization modules, and the modules can simultaneously run different tests.

J. Substantial Equivalence Information:

1. Predicate device name(s): Roche Diagnostic Corp. Factor V Leiden Kit Roche Diagnostic Corp. Factor II (Prothrombin) G20210A Kit.   
2. Predicate 510(k) number(s): k033607, k033612   
3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The Verigene® F5 Nucleic Acid Testis an in vitro diagnostic for thedetection and genotyping of a singlepoint mutation (G to A at position1691; also known as Factor VLeiden) of the human Factor V gene(F5; Coagulation Factor V gene) inpatients with suspectedthrombophilia, from isolatedgenomic DNA obtained from wholeblood samples. The test is intendedto be used on the Verigene® System.</td><td colspan="1" rowspan="1">The Factor V Leiden Kit isan in vitro diagnostic testfor the detection andgenotyping of a singlepoint mutation (G to A atposition 1691) of thehuman Factor V gene, fromDNA isolated from humanwhole peripheral blood.The Factor V Leiden Kit isindicated as an aid todiagnosis in the evaluationof patients with suspectedthrombophilia</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The Verigene® F2 Nucleic Acid Testis an in vitro diagnostic for thedetection and genotyping of a singlepoint mutation (G to A at position</td><td colspan="1" rowspan="1">The Factor II(Prothrombin)G202100A Kit is an invitro diagnostic test for thedetection and genotyping</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">20210of the human Factor II gene(F2; prothrombin gene) in patientswith suspected thrombophilia, fromisolated genomic DNA obtained fromwhole blood samples. The test isintended to be used on theVerigene® System.</td><td colspan="1" rowspan="1">of a single point mutations(G to A at position 202 10)of the human Factor IIgene, from DNA isolatedfrom human wholeperipheral blood. TheFactor 1I (Prothrombin)G20210A Kit is indicatedas an aid to diagnosisin the evaluation of patientswith suspectedthrombophilia</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Peripheral whole blood</td><td colspan="1" rowspan="1">same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The Verigene® MTHFR NucleicAcid Test is an in vitro diagnosticfor the detection and genotypingof a single point mutation (C to Tat position 677) of the human 5,10-methylenetetrahydrofolatereductase gene (MTHFR) inpatients with suspectedthrombophilia, from isolatedgenomic DNA obtained fromwhole blood samples. The test isintended to be used on theVerigene® System.</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Detection method</td><td colspan="1" rowspan="1">Signal amplification Chemicalamplification of reporter signal</td><td colspan="1" rowspan="1">DNA amplification (PCR)</td></tr><tr><td colspan="1" rowspan="1">Sample size</td><td colspan="1" rowspan="1">25 μL</td><td colspan="1" rowspan="1">10-20 μL in glass capillaries</td></tr><tr><td colspan="1" rowspan="1">Detectionprocedure</td><td colspan="1" rowspan="1">Single-image sensor wherenanoparticles are illuminatedusing a fixed-wavelength lightsource</td><td colspan="1" rowspan="1">Optical detection ofstimulated fluorescence</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">SNP discrimination viaoligonucleotide probes; diction viaevanescent wave light scatter withnanoparticles</td><td colspan="1" rowspan="1">Paired hybridization probesusing fluorescence resonanceenergy transfer (FRET)</td></tr><tr><td colspan="1" rowspan="1">Analysis time</td><td colspan="1" rowspan="1">1.5 hrs</td><td colspan="1" rowspan="1">Detection occurs at definedintervals during PCR cycleand can e viewed in real-time</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for the Content of Premarket Submissions for Software Contained in Medical   
Devices   
Class II Special Controls Guidance Document: Factor V Leiden DNA Mutation Detection   
Systems   
CLSI EP5-A2; Evaluation of Precision Performance of Quantitative Measurement Methods

# L. Test Principle:

The analysis sequence is the same for each of the three tests. After extracted and purified DNA, mixed with hybridization buffer, is loaded into the sample well of the test cartridge, it is ready for processing. An internal barcode reader reads the cartridge ID and sends the information to the Verigene $\textsuperscript { \textregistered }$ Reader. The reader establishes the hybridization parameters and starts the hybridization process.

The genotyping process occurs with a hybridization of the target analyze to a synthetic genespecific oligonucleotide capture strand on the test cartridge’s substrate. A synthetic mediator target–specific oligonucleotide is included with the test-specific sample buffer to form a hybridization” sandwich” with the gene sequence of interest. After washing, a probe composed of a gold nanoparticle with covalently bound oligonucleotide complementary to a sequence on the intermediate oligonucleotide, is introduced. A signal enhancement reagent is added to the hybridization chamber and reacts with the gold nanoparticle to amplify the signal for the Verigene $\textsuperscript { \textregistered }$ Reader scanning and analysis.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: The reproducibility testing included 4 studies, each using a different DNA samples extracted from whole blood. In the first study, each of the three test sites ran the same sample in duplicate.

<table><tr><td rowspan=1 colspan=1>Sites</td><td rowspan=1 colspan=1>F5</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>MTHFR</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr></table>

In the second study, one operator at Site 1 analyzed the same sample in duplicate each

day for three days.

<table><tr><td rowspan=1 colspan=1>Days</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>F5</td><td rowspan=1 colspan=1>MTHFR</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr></table>

In the third study, three operators at Site 2 each analyzed the same sample in duplicate.

<table><tr><td rowspan=1 colspan=1>Operators</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>F5</td><td rowspan=1 colspan=1>MTHFR</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr></table>

In the fourth study, one operator at Site 3 analyzed the same sample in duplicate using three lots of reagents.

<table><tr><td rowspan=1 colspan=1>Reagent Lots</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>F5</td><td rowspan=1 colspan=1>MTHFR</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr></table>

b. Linearity/assay reportable range: NA   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): NA   
d. Detection limit: In 40 genomic samples diluted to $4 0 \mathrm { n g / \mu L }$ the total call rate for all three assays was $82 \%$ . There were no mis-calls.   
$e$ . Analytical specificity: To simulate sample cross-contamination, a sample of known genotype was spiked with various amounts of a sample with a known but different genotype. The first sample consisted of a F2 (wild-type); F5 (mutant); MTHFR (wild type) and the

contaminating sample’s genotype was: F2 (mutant); F5 (wild-type); MTHFR (hetero). The percent contaminating sample ranged from $0 . 5 0 \%$ . The results indicated that neither the call rate nor accuracy in a homozygous sample was affected in the presence of up to $10 \%$ contamination by volume of a sample with the opposite homozygous genotype. At least $20 \%$ contamination by volume of a sample with a heterozygous genotype is required to affect the call rate.

A study was performed to assess the interference of heparin, hemoglobin or magnetic beads in a purified DNA sample. Interference was observed in the 5X magnetic bead sample where the F5 signal was slightly lower than the control. The genotyping ration results showed slight variations but none led to an increase in no-calls or a miscall.

f. Assay cut-off: NA

2. Comparison studies:

a. Method comparison with predicate device:

Accuracy (percent agreement) was determined by comparison to bi-directional DNA sequencing:

A total of 287 samples, sixty-eight percent $( 6 8 \% )$ from patients undergoing “rule-out thrombophilia” testing, were analyzed at three sites using the Verigene $\textsuperscript { \textregistered }$ $) F 5 / F 2 /$ MTHFR Nucleic Acid Tests and by bi-directional sequencing analysis at an independent reference laboratory. All purified DNA samples were from whole blood collected using EDTA as the anticoagulant. Data comparison details for $F 5$ , $F 2$ , and MTHFR are shown in tables 3-5 below:

Table 3. Verigene $^ \mathrm { \textregistered }$ F5 Nucleic Acid Test method comparison results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Verigene®F5 Nucleic Acid Test</td></tr><tr><td rowspan=4 colspan=1>Seence*SasSs5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Heterozygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td></tr><tr><td rowspan=1 colspan=1>wt</td><td rowspan=1 colspan=1>253</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td></tr></table>

Table 4. Verigene $^ \mathrm { \textregistered }$ F2 Nucleic Acid Test method comparison results   
Table 5. Verigene $^ \mathrm { \textregistered }$ MTHFR Nucleic Acid Test method comparison results   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="4" rowspan="1">VerigeneF2 Nucleic Acid Test</td></tr><tr><td colspan="1" rowspan="4">Seence*aass5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Wild-type(wt)</td><td colspan="1" rowspan="1">Heterozygous(het)</td><td colspan="1" rowspan="1">Mutant(mut)</td></tr><tr><td colspan="1" rowspan="1">wt</td><td colspan="1" rowspan="1">258</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">het</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">mut</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="4" rowspan="1">Verigene® MTHFR Nucleic Acid Test</td></tr><tr><td colspan="1" rowspan="4">Seence2ans5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Wild-type(wt)</td><td colspan="1" rowspan="1">Heterozygous(het)</td><td colspan="1" rowspan="1">Mutant(mut)</td></tr><tr><td colspan="1" rowspan="1">wt</td><td colspan="1" rowspan="1">125</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">het</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">mut</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">26</td></tr></table>

\*Two DNA samples did not return complete bi-directional sequencing results and, therefore, were excluded from the data. Specifically, one sample did not produce results for any genotype reads (i.e., $F 2 , F 5 .$ , and MTHFR) and one sample returned no results for the $F 2$ genotype. These four genotype reads represent $0 . 7 \%$ of the total possible calls in the method comparison study. There were no samples excluded due to discrepancies between sequenced results.

The Verigene $\textsuperscript { \textregistered }$ F5 / F2 / MTHFR Nucleic Acid Tests demonstrated $100 \%$ Positive Percent Agreement $9 5 \% C \mathrm { I } { = } 9 8 . 9$ to $1 0 0 . 0 \%$ and $100 \%$ Negative Percent Agreement $9 5 \% C \mathrm { I } { = } 9 8 . 9$ to $1 0 0 . 0 \%$ ) in these studies based on data from calls made.

b. Matrix comparaison: NA

3. Clinical studies:

a. Clinical Sensitivity: NA

b. Clinical specificity: NA   
c. Other clinical supportive data (when a. and b. are not applicable): NA

4. Clinical cut-off: NA

5. Expected values/Reference range: NA

N. Instrument Name:

Verigene $\textsuperscript { \textregistered }$ System

# O. System Descriptions:

1. Modes of Operation: Closed system

2. Software: FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types: Yes __X_ or No

3. Specimen Identification: Barcode reader

4. Specimen Sampling and Handling: DNA should be extracted using a DNA extraction method that provides DNA with the following characteristics: Average purity: Optimal mean: 1.89 (typical range: 1.60 to 2.00 $\left( \mathbf { A } _ { 2 6 0 } / \mathbf { A } _ { 2 6 0 } \right)$ )

Concentration: $4 0 \mathrm { n g / \mu L }$ to $4 0 0 \mathrm { n g / \mu L }$ DNA elution: water or Tris-EDTA

5. Calibration:

The temperature control system on the Verigene $\textsuperscript { \textregistered }$ Processor device is the only component that requires calibration. This is performed at the time of manufacture or by a Nanosphere service technician.

6. Quality Control:

1. Automated on-line quality misprints that monitor instrument functionality, software performance, fluidics, test conditions, reagent integrity, and procedural steps in each assay each time a test is performed   
2. Liquid materials

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Q. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.