# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K082562   
B. Purpose for Submission: Clearance for new device   
C. Manufacturer and Instrument Name: Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument with the SDS Software version 1.4   
D. Type of Test or Tests Performed: Real-Time PCR   
E. System Descriptions:

#

# 1. Device Description:

The Applied Biosystems (AB) 7500 Fast Dx Real-Time PCR instrument integrates a thermal cycler, a fluorimeter and application specific software. The instrument houses the thermal cycler and the fluorimeter, while the application software is run on a PC that is attached to the instrument. Samples are placed in a tube strip or 96-well low-head space plate that is moved to a Peltier-based thermal block and positioned relative to the optics using a tray loading mechanism.

Excitation for all samples is provided by a halogen tungsten white source that passes through 5 switchable excitation filters prior to reaching the sample. Fluorescence emission is then detected through a 5 color emissions filter wheel to a charge coupled device (CCD) camera. The instrument is designed to complete quantitative RT-PCR runs in about 40 minutes.

The Sequence Detection Software (SDS) version 1.4 for the 7500 Fast Dx Instrument is used for instrument control, data collection and data analysis. The software can measure cycle-by-cycle real-time signals from the sample. The software provides a variety of tools to help the user analyze the data extracted from the samples. The software also provides lamp-life monitoring and other instrument maintenance information. The software runs as an application on Windows® XP platform. Changes to the Dx software are subject to change control in accordance with 21 CFR Part 820.40.

2. Principles of Operation: Device Features Controlled by Software

The Sequence Detection Software (SDS) version 1.4 for the 7500 Fast Dx Instrument is used for instrument control, data collection and data analysis. The software can measure cycle-by-cycle real-time signals from the sample. The software provides a variety of tools to help the user analyze the data extracted from the samples. The software also provides lamp-life monitoring and other instrument maintenance information.

# Operational Environment

The SDS 1.4 software functions on the Microsoft® Windows® XP Pro SP2 operating system.

3. Modes of Operation: Batch via 96 well plate or tube strip

4. Specimen Identification: Entered by user.

5. Specimen Sampling and Handling: Specimens are processed according to assay instructions.

6. Calibration:

Calibration is performed at regular six-month intervals by AB service personnel.

The user performs a background calibration. A background calibration measures the level of background fluorescence in the instrument. During a background calibration run, the instrument:

• Performs continuous reads of a background plate containing PCR buffer for 10 minutes at $6 0 ~ ^ { \circ } \mathrm { C }$ .   
• Averages the spectra recorded during the run and extracts the resulting spectral component to a calibration file.

The software then uses the calibration file during subsequent runs to remove the background fluorescence from the run data. The user is directed to perform this calibration monthly or as often as necessary depending on instrument use, well as after replacing the lamp.

7. Quality Control:

Quality control is addressed for each separately cleared specific assay to be run on

the instrument.

8. Software:

FDA has reviewed applicant’s Hazard Analysis and Software Development processes for this line of product types:

Yes____X____ or No_

# F. Regulatory Information:

1. Regulation section: 862.2570 Instrumentation for clinical multiplex test systems

2. Classification: Class II

3 Product code: NSU

4. Panel: Clinical Chemistry (75)

# G. Intended Use:

1. Indication(s) for Use:

The Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument with the SDS Software version 1.4 is a real-time nucleic acid amplification and detection system that measures nucleic acid signals from reverse transcribed RNA and converts them to comparative quantitative readouts using fluorescent detection of dual-labeled hydrolysis probes. The 7500 Fast Dx is to be used only by technologists trained in laboratory techniques, procedures and on use of the analyzer.

2. Special Conditions for Use Statement(s): For prescription use only

# H. Substantial Equivalence Information:

1. Predicate Device Name(s) and 510(k) numbers:

# Affymetrix GeneChip Microarray Instrumentation System (K042279)

2. Comparison with Predicate Device:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Device</td><td>Predicate</td></tr><tr><td>Multiplex capable</td><td>Able to measure and sort multiple signals generated by an assay from a clinical sample.</td><td>Able to measure and sort multiple signals generated by an assay from a clinical sample.</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Real-Time PCR</td><td rowspan=1 colspan=1>Microarray</td></tr><tr><td rowspan=1 colspan=1>Indications for Use</td><td rowspan=1 colspan=1>The Applied Biosystems7500 Fast Dx Real-TimePCR Instrument with theSDS Software version 1.4is a real-time nucleic acidamplification anddetection system thatmeasures nucleic acidsignals from reversetranscribed RNA andconverts them tocomparative quantitativereadouts usingfluorescent detection ofdual-labeled hydrolysisprobes. The 7500 FastDx is to be used only bytechnologists trained inlaboratory techniques,procedures and on use ofthe analyzer.</td><td rowspan=1 colspan=1>The Affymetrix GeneChipMicroarray InstrumentationSystem consisting ofGeneChip 3000Dx scannerwith autoloader, FS450Dxfluidics station and theGCOSDx software isintended to measurefluorescence signals oflabeled DNA targethybridized to GeneChiparrays for use withseparately clearedGeneChip microarrayassays.</td></tr></table>

# I. Special Control/Guidance Document Referenced (if applicable):

Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems: http://www.fda.gov/cdrh/oivd/guidance/1546.html

# J. Performance Characteristics:

1. Analytical Performance:

a. Accuracy:

Accuracy was assessed during the clearance of the assay (k080570) and will be addressed for each assay to be run on this system.

# b. Precision/Reproducibility:

The rRT-PCR Flu Panel reproducibility and precision studies were performed to evaluate reproducibility of the assay at three separate laboratory sites using the ABI 7500 Fast Dx Real-time PCR instruments and SDS software version 1.4.

The ABI 7500 Fast Dx Real-Time PCR instrument functionality was validated at each site by AB prior to the reproducibility assessment using a panel of nine (9) simulated samples included influenza A/H1N1, A/H3N2, A/H5N1 (reassortant), and Influenza B at two viral RNA concentrations each (a low viral RNA concentration and a 1:10 dilution of the same sample). The low viral RNA concentration generally was one log above the assay cut-off for all analytes, whereas the 1:10 dilution of the same sample approximated a sample at the assay cut-off. Simulated samples in the panel used in the reproducibility evaluation were:

Sample #1 Influenza A/H1N1 at a low viral RNA titer range Sample #2 Influenza A/H1N1 at a 1:10 dilution of sample #1 Sample #3 Influenza A/H3N2 at a low viral RNA titer range • Sample #4 Influenza A/H3N2 at a 1:10 dilution of sample #3 Sample #5 Influenza A/H5N1 WT at a low viral RNA titer range Sample #6 Influenza A/H5N1 WT at a 1:10 dilution of sample #5 Sample #7 Influenza B Yamagata at a low viral RNA titer range Sample #8 Influenza B Yamagata at a 1:10 dilution of sample #7 Sample #9 Influenza Negative (Uninfected A549 cells)

The panels and assay controls were tested at each site by 2 operators on 5 different days within a 10-day period. Each participating clinical site tested one of the four RNA purification methods recommended for use with the assay (k080570) to evaluate reproducibility of the CDC rRT-PCR Flu Panel on the ABI 7500 Fast Dx Real-Time PCR instruments. The manufacturer’s instructions for use provided in the package insert were followed. Results generated for each of the extraction methods are summarized in the tables below.

Reproducibility Study Summary for the CDC rRT-PCR Flu Panel on the ABI 7500 Fast Dx Real-Time PCR Instrument and SDS version 1.4 Software.   

<table><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">Sample and AnalyteTested</td><td colspan="3" rowspan="1">Method 1</td><td colspan="3" rowspan="1">Method 2</td><td colspan="3" rowspan="1">Method 3</td><td colspan="3" rowspan="1">Method 4</td><td colspan="1" rowspan="2">TotalAgreementw/ExpectedResults</td><td colspan="1" rowspan="2">95 %CI</td></tr><tr><td colspan="1" rowspan="1">Aehpeee Resct</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Rhpee /Resct</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Amnhwaee /Rsct</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Aehpee /Resct</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">Sample 1 (low)InfA</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.73</td><td colspan="1" rowspan="1">3.12</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">32.24</td><td colspan="1" rowspan="1">5.65</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.34</td><td colspan="1" rowspan="1">1.94</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.55</td><td colspan="1" rowspan="1">3.69</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 1 (low)H1</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.48</td><td colspan="1" rowspan="1">2.84</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.78</td><td colspan="1" rowspan="1">5.64</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.67</td><td colspan="1" rowspan="1">4.47</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.08</td><td colspan="1" rowspan="1">4.11</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 1 (low)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.62</td><td colspan="1" rowspan="1">4.00</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.08</td><td colspan="1" rowspan="1">5.63</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">27.67</td><td colspan="1" rowspan="1">3.28</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.22</td><td colspan="1" rowspan="1">3.13</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 2(1:10 of Sample 1)InfA</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.59</td><td colspan="1" rowspan="1">3.22</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.30</td><td colspan="1" rowspan="1">5.93</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">37.03</td><td colspan="1" rowspan="1">4.07</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.97</td><td colspan="1" rowspan="1">3.32</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">86.8 - 99.9</td></tr><tr><td colspan="1" rowspan="1">Sample 2(1:10 of Sample 1) H1</td><td colspan="1" rowspan="1">6/10</td><td colspan="1" rowspan="1">37.92</td><td colspan="1" rowspan="1">2.99</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">37.76</td><td colspan="1" rowspan="1">8.31</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">38.28</td><td colspan="1" rowspan="1">4.28</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">37.21</td><td colspan="1" rowspan="1">4.77</td><td colspan="1" rowspan="1">31/40</td><td colspan="1" rowspan="1">61.6 - 89.2</td></tr><tr><td colspan="1" rowspan="1">Sample 2(1:10 of Sample 1)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.51</td><td colspan="1" rowspan="1">3.80</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.13</td><td colspan="1" rowspan="1">9.58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.46</td><td colspan="1" rowspan="1">2.47</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.65</td><td colspan="1" rowspan="1">2.12</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 3 (low)InfA</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.39</td><td colspan="1" rowspan="1">3.74</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.32</td><td colspan="1" rowspan="1">7.09</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.01</td><td colspan="1" rowspan="1">7.16</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">32.29</td><td colspan="1" rowspan="1">1.82</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 3 (low)H3</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.25</td><td colspan="1" rowspan="1">2.52</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.98</td><td colspan="1" rowspan="1">7.60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.07</td><td colspan="1" rowspan="1">2.86</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.28</td><td colspan="1" rowspan="1">1.71</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">86.8 - 99.9</td></tr><tr><td colspan="1" rowspan="1">Sample 3 (low)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.17</td><td colspan="1" rowspan="1">5.32</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.02</td><td colspan="1" rowspan="1">5.33</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.30</td><td colspan="1" rowspan="1">3.05</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.50</td><td colspan="1" rowspan="1">3.34</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 4(1:10 of Sample 3)InfA</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">38.44</td><td colspan="1" rowspan="1">4.05</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">37.03</td><td colspan="1" rowspan="1">5.08</td><td colspan="1" rowspan="1">5/10</td><td colspan="1" rowspan="1">39.05</td><td colspan="1" rowspan="1">3.46</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.18</td><td colspan="1" rowspan="1">2.41</td><td colspan="1" rowspan="1">32/40</td><td colspan="1" rowspan="1">64.4 - 91.0</td></tr><tr><td colspan="1" rowspan="1">Sample 4(1:10 of Sample 3) H3</td><td colspan="1" rowspan="1">6/10</td><td colspan="1" rowspan="1">38.82</td><td colspan="1" rowspan="1">3.34</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">39.90</td><td colspan="1" rowspan="1">6.14</td><td colspan="1" rowspan="1">6/10</td><td colspan="1" rowspan="1">38.01</td><td colspan="1" rowspan="1">4.81</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.82</td><td colspan="1" rowspan="1">3.49</td><td colspan="1" rowspan="1">29/40</td><td colspan="1" rowspan="1">56.1 - 85.4</td></tr><tr><td colspan="1" rowspan="1">Sample 4(1:10 of Sample 3)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.75</td><td colspan="1" rowspan="1">4.32</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.11</td><td colspan="1" rowspan="1">7.89</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.56</td><td colspan="1" rowspan="1">4.60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.68</td><td colspan="1" rowspan="1">3.35</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)InfA</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.68</td><td colspan="1" rowspan="1">2.70</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.90</td><td colspan="1" rowspan="1">6.66</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.07</td><td colspan="1" rowspan="1">3.14</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.80</td><td colspan="1" rowspan="1">1.74</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)H5a</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.69</td><td colspan="1" rowspan="1">3.76</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.65</td><td colspan="1" rowspan="1">5.52</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.58</td><td colspan="1" rowspan="1">4.94</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.03</td><td colspan="1" rowspan="1">4.55</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)H5b</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">37.91</td><td colspan="1" rowspan="1">3.58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.27</td><td colspan="1" rowspan="1">9.56</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.09</td><td colspan="1" rowspan="1">4.49</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.54</td><td colspan="1" rowspan="1">3.11</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">86.8 - 99.9</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)NaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.44</td><td colspan="1" rowspan="1">4.05</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.16</td><td colspan="1" rowspan="1">7.18</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">27.97</td><td colspan="1" rowspan="1">4.84</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.46</td><td colspan="1" rowspan="1">2.78</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)InfA</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">36.90</td><td colspan="1" rowspan="1">6.60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.84</td><td colspan="1" rowspan="1">4.89</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">37.10</td><td colspan="1" rowspan="1">5.15</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.54</td><td colspan="1" rowspan="1">2.61</td><td colspan="1" rowspan="1">36/40</td><td colspan="1" rowspan="1">76.3  97.2</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)Haa</td><td colspan="1" rowspan="1">5/10</td><td colspan="1" rowspan="1">37.90</td><td colspan="1" rowspan="1">3.24</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">4.91</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">37.77</td><td colspan="1" rowspan="1">7.35</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.71</td><td colspan="1" rowspan="1">2.51</td><td colspan="1" rowspan="1">31/40</td><td colspan="1" rowspan="1">61.6 - 89.2</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)H5b</td><td colspan="1" rowspan="1">4/10</td><td colspan="1" rowspan="1">39.29</td><td colspan="1" rowspan="1">ND*</td><td colspan="1" rowspan="1">5/10</td><td colspan="1" rowspan="1">39.43</td><td colspan="1" rowspan="1">3.74</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">38.74</td><td colspan="1" rowspan="1">3.35</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.71</td><td colspan="1" rowspan="1">5.30</td><td colspan="1" rowspan="1">27/40</td><td colspan="1" rowspan="1">50.9  81.4</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.05</td><td colspan="1" rowspan="1">3.58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.12</td><td colspan="1" rowspan="1">3.39</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.59</td><td colspan="1" rowspan="1">4.37</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.66</td><td colspan="1" rowspan="1">3.45</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 7 (low)Inf B</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">34.10</td><td colspan="1" rowspan="1">4.25</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.49</td><td colspan="1" rowspan="1">9.12</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.70</td><td colspan="1" rowspan="1">7.70</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">32.83</td><td colspan="1" rowspan="1">3.61</td><td colspan="1" rowspan="1">38/40</td><td colspan="1" rowspan="1">83.1 - 99.4</td></tr><tr><td colspan="1" rowspan="1">Sample 7 (low)NaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.78</td><td colspan="1" rowspan="1">3.55</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.75</td><td colspan="1" rowspan="1">10.16</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.54</td><td colspan="1" rowspan="1">4.42</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.90</td><td colspan="1" rowspan="1">3.13</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 8(1:10 of Sample 7) InfB</td><td colspan="1" rowspan="1">1/10</td><td colspan="1" rowspan="1">36.76</td><td colspan="1" rowspan="1">ND*</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">5.77</td><td colspan="1" rowspan="1">3/10</td><td colspan="1" rowspan="1">38.95</td><td colspan="1" rowspan="1">ND*</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">37.11</td><td colspan="1" rowspan="1">5.81</td><td colspan="1" rowspan="1">20/40</td><td colspan="1" rowspan="1">33.8  66.2</td></tr><tr><td colspan="1" rowspan="1">Sample 8(1:10 of Sample 7)NaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.48</td><td colspan="1" rowspan="1">4.46</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.58</td><td colspan="1" rowspan="1">6.30</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.71</td><td colspan="1" rowspan="1">3.35</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.92</td><td colspan="1" rowspan="1">4.07</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 9 Influenza (-)RNaseP</td><td colspan="9" rowspan="1">10/1029.79 3.17 10/10 30.08  5.91   10/10 28.27 4.83</td><td colspan="3" rowspan="1">10/10 28.86  3.68</td><td colspan="2" rowspan="1">40/40   91.2100.0</td></tr><tr><td colspan="15" rowspan="1">1161/1320 86.4189.6213 / 250 85.2 %      319 / 330 96.7 %       303 / 33091.8 %      326 / 330 98.8 %88.0 %</td></tr></table>

# Well-to-Well Temperature Uniformity

The sponsor performed a Temperature Non-Uniformity (TNU) Test to verify that temperature set points are maintained in a uniform manner across the entire sample block. The test was carried out at two different temperature set points relevant to the use of the instrument, $9 4 ^ { \circ } \mathrm { C }$ and $6 0 ^ { \circ } \mathrm { C }$ , and the test was deemed passing if the average of the difference of the highest and lowest temperature readings across the eight wells that were tested for each set point was less than $1 ^ { \circ } \mathrm { C }$ . The results are in the table below:

<table><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>TNU Test@94.0°(&lt;1.0°C)</td><td rowspan=1 colspan=1>TNU Test@60.0°C(&lt;1.0°C)</td><td rowspan=1 colspan=1>Day Tested</td><td rowspan=1 colspan=1>Results(Pass/Fail)</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>06</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>07</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>08</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>Day 5</td><td rowspan=1 colspan=1>Pass</td></tr></table>

The sponsor also performed a Temperature Accuracy test to verify that the instrument can ramp to and hold a programmed temperature accurately. The test was carried out at two different temperature set points relevant to the use of the instrument, $8 5 ^ { \circ } \mathrm { C }$ and $4 5 \mathrm { { ^ \circ C } }$ , and the test was deemed passing if the average temperature across the eight wells that were tested for each set point was within $\pm 0 . 5 ^ { \circ } \mathrm { C }$ of that set point. The results are in the table below:

<table><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Temp Accuracy@85.0°C (±0.5C)</td><td rowspan=1 colspan=1>Temp Accuracy@45.0°C (±0.5°C)</td><td rowspan=1 colspan=1>Day Tested</td><td rowspan=1 colspan=1>Results(Pass/Fail)</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>85.12</td><td rowspan=1 colspan=1>44.97</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>85.05</td><td rowspan=1 colspan=1>44.92</td><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>85.11</td><td rowspan=1 colspan=1>45.01</td><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>85.07</td><td rowspan=1 colspan=1>45.01</td><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>45.16</td><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>06</td><td rowspan=1 colspan=1>84.99</td><td rowspan=1 colspan=1>44.91</td><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>07</td><td rowspan=1 colspan=1>84.97</td><td rowspan=1 colspan=1>45.01</td><td rowspan=1 colspan=1>Day 4</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>08</td><td rowspan=1 colspan=1>84.91</td><td rowspan=1 colspan=1>44.76</td><td rowspan=1 colspan=1>Day 5</td><td rowspan=1 colspan=1>Pass</td></tr></table>

The data demonstrate the ability of the ABI 7500 Fast Dx Real-Time PCR instrument to maintain uniform and accurate temperatures across the sample block.

# Well-to-Well Signal Uniformity

The sponsor performed an RNase P verification that addresses well-to-well signal uniformity by measuring the average signal and standard deviation of two unknown sample populations. The study was designed to demonstrate both well-to-well signal uniformity as well as that the instrument is capable of discriminating between two populations of samples with a two-fold difference in copy number.

The RNase P plate was pre-loaded with the reagents necessary for the detection and quantitation of genomic copies of the human RNase P gene (a single-copy gene encoding the RNase moiety of the RNase P enzyme).

Each well contained:

• TaqMan® Universal PCR Master Mix   
• RNase P primers   
• FAM™ dye-labeled probe   
• Known concentration of human genomic DNA template

The figure below illustrates the arrangement of the standard and unknown populations on the RNase P plate. The RNase P plate contained five replicate groups of standards (1250, 2500, 5000, 10,000, and 20,000 copies), two unknown populations (5000 and 10,000 copies), and four no template control (NTC) wells.

# RNase P Plate for 7500 Fast Dx instrument

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=3 colspan=12>RNase P - Population 1</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=4></td></tr><tr><td rowspan=1 colspan=1>C</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=3>NTC</td><td rowspan=1 colspan=5>STD 1250</td><td rowspan=1 colspan=4>STD 2500</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=3>STD 5000</td><td rowspan=1 colspan=5>STD 10000</td><td rowspan=1 colspan=4>STD 20000</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=3 colspan=12>RNase P - Population 2</td></tr><tr><td rowspan=1 colspan=1>G</td></tr><tr><td rowspan=1 colspan=1>H</td></tr></table>

After the run, the SDS software:

• Generated a standard curve from the averaged threshold cycle (CT) values of the replicate groups of standards.   
• Calculated the concentration of the two unknown populations using the standard curve.   
• Calculated the following using the mean quantity and standard   
deviation for the unknown populations to assess the instrument   
performance:

$$
[ \mathrm { ( C o p y U n k _ { 2 } ) } \cdot 3 ( \mathbb { { O } } _ { \mathrm { C o p y U n k 2 } } ) ] > [ \mathrm { ( C o p y U n k _ { 1 } ) } + 3 ( \mathbb { { O } } _ { \mathrm { C o p y U n k 1 } } ) ]
$$

where:

CopyUnk1 $=$ Average copy number of unknown #1 (5,000-copy   
population)   
σCopyUnk1 $=$ Standard deviation of unknown #1 (5,000-copy   
population)   
CopyUnk2 $=$ Average copy number of unknown #2 (10,000-copy   
population)   
σCopyUnk2 $=$ Standard deviation of unknown #2 (10,000-copy   
population)

The instrument passes the verification if the analyzed data demonstrates that the instrument distinguishes between 5,000 and 10,000 genome equivalents (i.e. a two-fold difference in copy number) with a $9 9 . 7 \%$ confidence level. The results are presented in the table below and show the ABI 7500 Fast Dx Real-Time PCR instrument met the passing criteria outlined above. These results also show the uniformity of the readout of the same signal (i.e. 10k copy or $5 \mathrm { k \Omega }$ copy) between different wells (note standard deviations and $\%$ CV of $< 6$ ), and the performance of the standard curve $\mathrm { R } ^ { 2 }$ values $> 0 . 9 9$ ).

<table><tr><td rowspan=2 colspan=1>Instrument</td><td rowspan=1 colspan=3>10K Copy</td><td rowspan=1 colspan=3>5K Copy</td><td rowspan=1 colspan=1>[Mean Qty 10Kcopy - 3 Std Dev(10K copy)] &gt;</td><td rowspan=1 colspan=1>StandardCurve R²</td><td rowspan=1 colspan=2>Number of Outliers(Maximum # : 6 perpopulation)</td><td rowspan=2 colspan=1>ResultsPass/Fail)</td></tr><tr><td rowspan=1 colspan=1>MeanQty</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MeanQty</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>%0</td><td rowspan=1 colspan=1>Mean Qty 5Kcopy + 3 StdDev (5K copy)]</td><td rowspan=1 colspan=1>Value &gt;00.990</td><td rowspan=1 colspan=1>5K Copy</td><td rowspan=1 colspan=1>10KCopy</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>9608</td><td rowspan=1 colspan=1>403</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>4725</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.998</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>9666</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>5003</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.995</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>9711</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>4787</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.998</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>9860</td><td rowspan=1 colspan=1>530</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>4813</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.996</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>9881</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>4721</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.997</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>06</td><td rowspan=1 colspan=1>9822</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>4794</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.998</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>07</td><td rowspan=1 colspan=1>9557</td><td rowspan=1 colspan=1>384</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>4901</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.999</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>08</td><td rowspan=1 colspan=1>9608</td><td rowspan=1 colspan=1>372</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>4729</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>0.999</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pass</td></tr></table>

FAM is a trademark of Applied Biosystems. TaqMan is a registered trademark of Roche Molecular Systems.

# c. Linearity:

Linearity was assessed during the clearance of the assay (k080570) and will be addressed for each assay to be run on this system.

# d. Carryover:

Carryover was assessed during the clearance of the assay (k080570) and will be addressed for each assay to be run on this system. In addition, the sponsor recommends the use of good laboratory practices to minimize crosscontamination during the sample prep process.

# e. Interfering Substances:

Interfering substances was assessed during the clearance of the assay (k080570) and will be addressed for each assay to be run on this system.

# K. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# L. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.