# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k103112   
B. Purpose for Submission: New accessory reagents for separately FDA cleared Affymetrix GeneChip microarray assays   
C. Measurand: mRNA gene expression   
D. Type of Test: Gene expression microarray   
E. Applicant: Affymetrix, Inc.   
F. Proprietary and Established Names: Affymetrix Gene Profiling Reagents

# G. Regulatory Information:

2. Classification: Class II

4. Panel: Clinical Chemistry (75)

# H. Intended Use:

1. Intended use(s): The Affymetrix Gene Profiling Reagents are intended for the preparation of labeled complementary RNA target from purified total RNA from fresh or frozen clinical tissue specimens for hybridization to Affymetrix GeneChip $^ \mathrm { \textregistered }$ microarrays and the measurement of fluorescence signals of labeled RNA target using the Affymetrix GeneChip $^ \mathrm { \textregistered }$ Microarray Instrumentation System.

Intended for use with separately FDA-cleared Affymetrix GeneChip microarray assays specifying the use of the Affymetrix Gene Profiling Reagents.

2. Indication(s) for use: Same as Intended Use.   
3. Special conditions for use statement(s): Prescription use only.   
4. Special instrument requirements: Affymetrix GeneChip $^ \mathrm { \textregistered }$ Microarray Instrumentation System (k080995)

# I. Device Description:

The Affymetrix Gene Profiling (AGP) Reagents consist of three separate kits: RNA Control Kit, Transcript Synthesis and Labeling Kit, and the Transcript Detection Kits. The RNA Control Kit contains a $1 6 \mu \mathrm { L }$ vial of Poly-A control which contains in-vitro synthesized, polyadenylated transcripts for these $B _ { \cdot }$ . subtilis genes lys, phe, and dap, which are pre-mixed at staggered concentrations (1:100,000, 1:50,000, and 1:6,667, respectively) and a $3 . 8 \mathrm { m L }$ bottle of ready-to-use Dilution buffer,. The Transcript Synthesis and Labeling Kit is separated into Kit A and Kit B for appropriate storage. Kit A, stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ consists the following: one vial each of $1 ^ { \mathrm { s t } }$ strand synthesis buffer $( 1 2 8 \mu \mathrm { L } )$ , $2 ^ { \mathrm { n d } }$ strand synthesis buffer $( 5 7 6 \mu \mathrm { L } )$ , and In Vitro transcription buffer $( 7 0 4 \mu \mathrm { L } )$ , and one bottle each of magnetic beads $( 3 . 4 6 \mathrm { m L } )$ , beads wash buffer $( 5 . 4 \mathrm { m L } )$ , and nuclease-free water $ { 1 . 6 5 }  { \mathrm { m L } } )$ ). Kit B, stored at $- 1 5$ to $- 3 0 ^ { \circ } \mathrm { C }$ , contains one vial each of $1 ^ { \mathrm { s t } }$ strand synthesis enzyme mix $( 3 2 \mu \mathrm { L } )$ , $2 ^ { \mathrm { n d } }$ strand synthesis enzyme mix $( 6 4 \mu \mathrm { L } )$ , In-Vitro transcription enzyme mix $( 1 9 2 \mu \mathrm { L } )$ , and RNA label $( 6 4 \mu \mathrm { L } )$ . The Transcript Detection Kit is divided into three kits A, B, and C. Kit A, stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ , contains the hybridization module and the stain module. The hybridization module contains ready-to-use tubes or vials of Pre-Hybridization Mix $( 6 . 4 \mathrm { m L }$ ), 2X hybridization mix $( 4 . 0 \mathrm { m L } )$ , DMSO $( 0 . 8 \mathrm { m L } )$ , nuclease-free water $( 5 . 0 \mathrm { m L } )$ and 5X fragmentation buffer $( 1 9 2 \mu \mathrm { L } )$ . The stain module contains one bottle each of stain cocktail 1 and 2, $1 9 . 2 \mathrm { m L }$ each, and two bottles, $3 0 . 0 \mathrm { m L }$ of array holding buffer. Kit B, stored at 2 to $8 ^ { \circ } \mathrm { C }$ , contains three $8 6 0 ~ \mathrm { m L }$ bottles of wash buffer A and one $6 4 0 ~ \mathrm { m L }$ bottle of wash buffer B. Kit C, stored at – 15 to $- 3 0 ^ { \circ } \mathrm { C }$ , contains one $1 3 4 . 4 \mu \mathrm { L }$ vial of Oligo B2 and one $4 0 0 \mu \mathrm { L }$ vial of 20X hybridization control.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Affymetrix GeneChip $^ \mathrm { \textregistered }$ Microarray Instrumentation System   
2. Predicate K number(s): k080995   
3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceAffymetrix GeneChip® MicroarrayInstrumentation System usingAffymetrix Gene Profiling Reagents</td><td rowspan=1 colspan=1>PredicateAffymetrix GeneChip®Microarray InstrumentationSystem using PathworkExpression reagents</td></tr><tr><td rowspan=1 colspan=1>Specimentype</td><td rowspan=1 colspan=1>Total RNA isolated from fresh orfrozen clinical tumor tissue</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Requiredplatform</td><td rowspan=1 colspan=1>Affymetrix GeneChip®GCS3000Dx Scanner andFS450Dx Fluidics Station</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Microarray</td><td rowspan=1 colspan=1>Pathwork Tissue of Origin Pathchip™M</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceAffymetrix GeneChip®Microarray InstrumentationSystem using AffymetrixGene Profiling Reagents</td><td colspan="1" rowspan="1">PredicateAffymetrix GeneChip®MicroarrayInstrumentation System usingPathwork Expression reagents</td></tr><tr><td colspan="1" rowspan="1">Reagents used</td><td colspan="1" rowspan="1">Affymetrix Gene ProfilingReagents (RNA ControlKit; Transcript Synthesisand Labeling Kit; and the</td><td colspan="1" rowspan="1">Pathwork® Expression 3'-Amplification One-Cycle TargetReagents [includes Pathwork®®Expression 3'-Amplification One-</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceAffymetrix GeneChip®Microarray InstrumentationSystem using AffymetrixGene Profiling Reagents</td><td colspan="1" rowspan="1">PredicateAffymetrix GeneChip® MicroarrayInstrumentation System usingPathwork Expression reagents</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Transcript Detection Kits)</td><td colspan="1" rowspan="1">Cycle cDNA Synthesis Kit,Pathwork® Expression 3'-Amplification for IVT Labeling, andPathwork® Control Kit]</td></tr><tr><td colspan="1" rowspan="1">Performanceestablishedusing</td><td colspan="1" rowspan="1">Pathwork Fresh/FrozenTissue of Origin (TOO) test(k080896) usingAffymetrix Gene ProfilingReagents</td><td colspan="1" rowspan="1">Pathwork Fresh/Frozen Tissue ofOrigin (TO0) test (k080896) usingPathwork® TOO Kit</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

ISO 15223 Medical Devices - Symbols to be used with medical device labels, labeling and   
information to be supplied   
CEN 13640 Stability Testing of In Vitro Diagnostic Reagents

L. Test Principle: Performance of the Affymetrix Gene Expression reagents was established using the Pathwork Fresh/Frozen Tissue of Origin (TOO) test (see k080896).

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: a. Precision/Reproducibility: Precision and reproducibility were performed internally at Affymetrix and at two external sites using previously extracted total RNA specimens and total RNA isolated from frozen tumor specimens using the Pathwork TOO Pathchip™. Precision and reproducibility were measured by determination of $\% \mathrm { C V }$ established from the fold change of the probe set signal Pearson correlation coefficient.

Lot-to-Lot: To demonstrate lot-to-lot reproducibility three lots of AGP Reagent Kits were tested using $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MicroArray Quality Control (MAQC) samples A and B $\mathrm { \Delta A } = 1 0 0 \%$ Universal Human Reference RNA; $\mathrm { B } = 1 0 0 \%$ Human Brain Reference RNA) total RNAs, tested in quadruplicate, and compared to three lots of the Pathwork $^ \mathrm { \textregistered }$ Expression $3 ^ { \circ }$ -Amplification One-Cycle Target Reagents (One-Cycle) using $1 0 0 0 \mathrm { n g }$ of total RNA from the same samples.

Operator-to-Operator: To demonstrate reproducibility of performance between operators, three lots of AGP Reagent Kits were tested using $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNAs tested in quadruplicate by three different operators and compared to three lots of the One-Cycle Reagents using $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNA in quadruplicate by three different operators.

Day-to-Day: To demonstrate reproducibility between days, one lot of the AGP

Reagent Kits was tested using $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNAs in quadruplicate and compared to three lots of the One-Cycle Reagents using $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNA in quadruplicate over three non-consecutive days.

The $\% C V s$ were calculated for between lot, operator (confined within the lot-to-lot variation), and days and are shown in Table 1 below. Since the between-operator precision is confined within the between-lot variation it is not reported separately.

Table 1. Precision for between-days and –lots for MAQC specimens.   

<table><tr><td rowspan=3 colspan=1>Varianceparameter</td><td rowspan=3 colspan=1>Specimen</td><td rowspan=1 colspan=3>MedianCV%</td></tr><tr><td rowspan=2 colspan=1>One-CycleReagents</td><td rowspan=1 colspan=2>GeneProfilingReagents</td></tr><tr><td rowspan=1 colspan=1>1000 ng specimen</td><td rowspan=1 colspan=1>100 ng specimen</td></tr><tr><td rowspan=2 colspan=1>Between Days(within lot)</td><td rowspan=1 colspan=1>MAQC A</td><td rowspan=1 colspan=1>7.97 %</td><td rowspan=1 colspan=1>7.17 %</td><td rowspan=1 colspan=1>7.03 %</td></tr><tr><td rowspan=1 colspan=1>MAQC B</td><td rowspan=1 colspan=1>7.68 %</td><td rowspan=1 colspan=1>7.24 %</td><td rowspan=1 colspan=1>9.33 %</td></tr><tr><td rowspan=2 colspan=1>Between Lots</td><td rowspan=1 colspan=1>MAQC A</td><td rowspan=1 colspan=1>8.70 %</td><td rowspan=1 colspan=1>9.54 %</td><td rowspan=1 colspan=1>8.74 %</td></tr><tr><td rowspan=1 colspan=1>MAQC B</td><td rowspan=1 colspan=1>9.52 %</td><td rowspan=1 colspan=1>9.93 %</td><td rowspan=1 colspan=1>11.72 %</td></tr></table>

![](images/c31c44fec7ef2fb180d714eb0326f417c09397b5c9e4c39b91066b9cdd4363fc.jpg)  
Figures 1 and 2 below depict examples of the graph for the lowest r value and highest r value, respectively. [Fig. 1. $\mathrm { y } { = } 0 . 9 6 8 \mathrm { x } - 0 . 0 5 5$ . $\mathrm { r } = 0 . 9 5 6 2$ ; Fig. 2. $\mathrm { y } { = } 0 . 9 9 3 \mathrm { x } + 0 . 0 8 8$ ; $\mathrm { r } { = } 0 . 9 9 6 5 ]$   
TOO Reagents – 1000 ng – Lot 2 – MAQC B $( \mathrm { n } { = } 4 )$

![](images/e89749219156f3d23fd09fe9ed568ada2abfcb8c8c6d54ffc6b615733f8ec716.jpg)

Table 2 below, shows the results for an analysis comparing day-to-day for the processing of the MAQC A and B total RNA samples, using a single lot of each of the One -Cycle and AGP Reagents. The four replicates were used to calculate the detected probe sets per condition which were used for the calculation of the Pearson correlation coefficient (r) and linear equation between conditions $\mathrm { L } { = } 1 0 \mathrm { t }$ ; $\mathrm { D } { = }$ day).

Table 2. Comparisons between lots and days for MAQC A and day-to-day for MAQC B.   

<table><tr><td colspan="1" rowspan="1">MAQC A</td><td colspan="1" rowspan="1">One-CycleL1/D2</td><td colspan="1" rowspan="1">One-CycleL1/D3</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D1</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D2</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D3</td><td colspan="1" rowspan="1">AGP,10 ng,L3/D1</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D2</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D3</td></tr><tr><td colspan="1" rowspan="1">One-Cycle,1 μg, L1/D1</td><td colspan="1" rowspan="1">0.9945y=0.990x+0.095</td><td colspan="1" rowspan="1">0.9944y=0.988x+0.076</td><td colspan="1" rowspan="1">0.9653y=0.955x+0.116</td><td colspan="1" rowspan="1">0.9673y=0.958x+0.114</td><td colspan="1" rowspan="1">0.9693y=0.956x+0.191</td><td colspan="1" rowspan="1">0.9709y=0.984x+0.018</td><td colspan="1" rowspan="1">0.9709y=0.997x+0.199</td><td colspan="1" rowspan="1">0.9719y=0.978x+0.162</td></tr><tr><td colspan="1" rowspan="1">One-Cycle,1 μg, L1/D2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9939y=0.992x+0.026</td><td colspan="1" rowspan="1">0.9636y=0.958x+0.077</td><td colspan="1" rowspan="1">0.9663y=0.961x+0.070</td><td colspan="1" rowspan="1">0.9692y=0.960x+0.140</td><td colspan="1" rowspan="1">0.9663y=0.983x+0.035</td><td colspan="1" rowspan="1">0.966y=0.977x+0.185</td><td colspan="1" rowspan="1">0.9679y=0.979x+0.141</td></tr><tr><td colspan="1" rowspan="1">One-Cycle,1 μg, L1/D3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9677y=0.964x+0.066</td><td colspan="1" rowspan="1">0.9699y=0.966x+0.063</td><td colspan="1" rowspan="1">0.9719y=0.964x+0.139</td><td colspan="1" rowspan="1">0.9727y=0.991x+0.065</td><td colspan="1" rowspan="1">0.9721y=0.984x+0.158</td><td colspan="1" rowspan="1">0.9740y=0.986x+0.114</td></tr><tr><td colspan="1" rowspan="1">AGP,100 ng, L3/D1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9948y=0.995x+0.052</td><td colspan="1" rowspan="1">0.994y=0.990x+0.150</td><td colspan="1" rowspan="1">0.9852y=0.963x+0.200</td><td colspan="1" rowspan="1">0.9838y=0.968x+0.000</td><td colspan="1" rowspan="1">0.9842y=0.968x+0.024</td></tr><tr><td colspan="1" rowspan="1">AGP,100 ng, L3/D2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9961y=0.991x</td><td colspan="1" rowspan="1">0.9846y=0.963x</td><td colspan="1" rowspan="1">0.9838y=0.968x</td><td colspan="1" rowspan="1">0.9847y=0.969x</td></tr><tr><td colspan="1" rowspan="1">MAQC A</td><td colspan="1" rowspan="1">One-CycleL1/D2</td><td colspan="1" rowspan="1">One-CycleL1/D3</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D1</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D2</td><td colspan="1" rowspan="1">AGP,100 ng,L3/D3</td><td colspan="1" rowspan="1">AGP,1000 ng,L3/D1</td><td colspan="1" rowspan="1">AGP,1000 ng,L3/D2</td><td colspan="1" rowspan="1">AGP,1000 ng,L3/D3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+0.121</td><td colspan="1" rowspan="1">+0.218</td><td colspan="1" rowspan="1">+0.013</td><td colspan="1" rowspan="1">+0.034</td></tr><tr><td colspan="1" rowspan="1">AGP,100 ng, L3/D3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9859y=0.959x+0.300</td><td colspan="1" rowspan="1">0.9852y=0.9665x+0.095</td><td colspan="1" rowspan="1">0.9868y=0.966x+0.111</td></tr><tr><td colspan="1" rowspan="1">AGP,1 μg, L3/D1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9961y=0.990x+0.247</td><td colspan="1" rowspan="1">0.9962y=0.990x+0.217</td></tr><tr><td colspan="1" rowspan="1">AGP, 1 μg,L3/D2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.9963y=0.996x+0.001</td></tr></table>

<table><tr><td rowspan=1 colspan=1>MAQC B</td><td rowspan=1 colspan=1>One-CycleL1/D2</td><td rowspan=1 colspan=1>One-CycleL1/D3</td><td rowspan=1 colspan=1>AGP,100 ng,L3/D1</td><td rowspan=1 colspan=1>AGP,100 ng,L3/D2</td><td rowspan=1 colspan=1>AGP,100 ng,L3/D3</td><td rowspan=1 colspan=1>AGP,1000 ng,L3/D1</td><td rowspan=1 colspan=1>AGP,1000 ng,L3/D2</td><td rowspan=1 colspan=1>AGP,1000 ng,L3/D3</td></tr><tr><td rowspan=1 colspan=1>One-Cycle,1 μg, L1/D1</td><td rowspan=1 colspan=1>0.9936y=0.995x+0.014</td><td rowspan=1 colspan=1>0.9933y=0.992x+0.024</td><td rowspan=1 colspan=1>0.9576y=0.947x+0.063</td><td rowspan=1 colspan=1>0.9636y=0.959x+0.103</td><td rowspan=1 colspan=1>0.9649y=0.957x+0.154</td><td rowspan=1 colspan=1>0.9668y=0.977x+0.018</td><td rowspan=1 colspan=1>0.9684y=0.971x+0.207</td><td rowspan=1 colspan=1>0.9695y=0.982x+0.103</td></tr><tr><td rowspan=1 colspan=1>One-Cycle,1 μg, L1/D2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9944y=0.991x+0.050</td><td rowspan=1 colspan=1>0.9568y=0.945x+0.102</td><td rowspan=1 colspan=1>0.9630y=0.956x+0.140</td><td rowspan=1 colspan=1>0.9646y=0.955+0.190</td><td rowspan=1 colspan=1>0.9638y=0.972x+0.044</td><td rowspan=1 colspan=1>0.966y=0.965x+0.271</td><td rowspan=1 colspan=1>0.9658y=0.976x+0.165</td></tr><tr><td rowspan=1 colspan=1>One-Cycle,1 μg, L1/D3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9611y=0.952x+0.063</td><td rowspan=1 colspan=1>0.9659y=0.962x+0.111</td><td rowspan=1 colspan=1>0.9671y=0.964x+0.164</td><td rowspan=1 colspan=1>0.9699y=0.981x+0.011</td><td rowspan=1 colspan=1>0.9699y=0.973x+0.221</td><td rowspan=1 colspan=1>0.9711y=0.984x+0.117</td></tr><tr><td rowspan=1 colspan=1>AGP,100 ng, L3/D1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9942y=1.000x+0.128</td><td rowspan=1 colspan=1>0.9930y=0.996x+0.198</td><td rowspan=1 colspan=1>0.9852y=0.961x+0.141</td><td rowspan=1 colspan=1>0.9788y=0.966x+0.062</td><td rowspan=1 colspan=1>0.9803y=0.957x+0.020</td></tr><tr><td rowspan=1 colspan=1>AGP,100 ng, L3/D2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9965y=0.993x+0.088</td><td rowspan=1 colspan=1>0.9838y=0.969x+0.210</td><td rowspan=1 colspan=1>0.9826y=0.975x+0.007</td><td rowspan=1 colspan=1>0.9842y=0.967x+0.075</td></tr><tr><td rowspan=1 colspan=1>AGP,100 ng, L3/D3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9859y=0.966x+0.267</td><td rowspan=1 colspan=1>0.9839y=0.973x+0.045</td><td rowspan=1 colspan=1>0.9854y=0.965x+0.128</td></tr><tr><td rowspan=1 colspan=1>AGP,1 μg, L3/D1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9953y=0.987x+0.267</td><td rowspan=1 colspan=1>0.9957y=0.998x+0.169</td></tr><tr><td rowspan=1 colspan=1>AGP,1 μg, L3/D2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.9963y=1.006x+0.069</td></tr></table>

Reproducibility was also evaluated by calculating the probe set signal Pearson correlation coefficients (r) for the probe set signal and the fold change probe set signal Pearson correlation coefficient (r) between all of the conditions for the detected probe set signal intensity.

![](images/afedee345ee38b0307a1cbd9bca5bbb11816e18429ef5c0b64df657184e78253.jpg)  
Fig. 3 and 4 below depict the fold change probe set signal Pearson correlation coefficients (r) comparing single lots (lot #1) of AGP reagents ( $1 0 0 ~ \mathrm { n g }$ specimen) to One-Cycle reagents $\scriptstyle ( 1 = 0 . 9 8 7$ ) and between days of a single lot (lot #3) of AGP reagents $\scriptstyle ( 1 = 0 . 9 9 7$ ), respectively. Solid diagonal lines $= 1 . 0$ fold change and dashed diagonal lines $= 0 . 5$ fold change.

Repeatability was evaluated from the replicates tested in the reproducibility studies within test condition for the probe sets called present. The $\%$ CVs were calculated for all replicates for the 3 lots of the AGP Reagent Kits tested on 3 different days using both $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNAs and are compared to the 3 lots of the One-Cycle Reagent Kits tested on 3 different days using $1 0 0 0 \mathrm { n g }$ of the same specimens. The calculated $\%$ CVs were $< 1 0 \%$ for all conditions and representative values are shown in Table 3 below.

Table 3. Median $\% \mathrm { C V }$ values for comparison of AGP and One-Cycle Reagents between lot and day.   

<table><tr><td rowspan=2 colspan=1>Reagent</td><td rowspan=2 colspan=1>Specimen</td><td rowspan=1 colspan=5>Median %CV</td></tr><tr><td rowspan=1 colspan=1>Lot1 Day1</td><td rowspan=1 colspan=1>Lot1 Day2</td><td rowspan=1 colspan=1>Lot1 Day3</td><td rowspan=1 colspan=1>Lot2 Day1</td><td rowspan=1 colspan=1>Lot3 Day1</td></tr><tr><td rowspan=1 colspan=1>One-Cycle</td><td rowspan=1 colspan=1>MAQC A 1000ng</td><td rowspan=1 colspan=1>5.06%</td><td rowspan=1 colspan=1>4.55%</td><td rowspan=1 colspan=1>4.85%</td><td rowspan=1 colspan=1>3.67%</td><td rowspan=1 colspan=1>4.30%</td></tr><tr><td rowspan=1 colspan=1>AGP</td><td rowspan=1 colspan=1>MAQC A100ng</td><td rowspan=1 colspan=1>5.31%</td><td rowspan=1 colspan=1>5.81%</td><td rowspan=1 colspan=1>5.41%</td><td rowspan=1 colspan=1>4.24%</td><td rowspan=1 colspan=1>4.04%</td></tr><tr><td rowspan=1 colspan=1>One-Cycle</td><td rowspan=1 colspan=1>MAQC B1000ng</td><td rowspan=1 colspan=1>4.57%</td><td rowspan=1 colspan=1>4.11%</td><td rowspan=1 colspan=1>4.35%</td><td rowspan=1 colspan=1>4.88%</td><td rowspan=1 colspan=1>4.64%</td></tr><tr><td rowspan=1 colspan=1>AGP</td><td rowspan=1 colspan=1>MAQC B100ng</td><td rowspan=1 colspan=1>8.77%</td><td rowspan=1 colspan=1>4.42%</td><td rowspan=1 colspan=1>5.15%</td><td rowspan=1 colspan=1>5.46%</td><td rowspan=1 colspan=1>4.79%</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Lot1 Day1</td><td rowspan=1 colspan=1>Lot2 Day1</td><td rowspan=1 colspan=1>Lot3 Day1</td><td rowspan=1 colspan=1>Lot3 Day2</td><td rowspan=1 colspan=1>Lot3 Day3</td></tr><tr><td rowspan=2 colspan=1>AGP</td><td rowspan=1 colspan=1>MAQC A 1000ng</td><td rowspan=1 colspan=1>5.55%</td><td rowspan=1 colspan=1>4.38%</td><td rowspan=1 colspan=1>5.88%</td><td rowspan=1 colspan=1>4.54%</td><td rowspan=1 colspan=1>5.44%</td></tr><tr><td rowspan=1 colspan=1>MAQC B 1000ng</td><td rowspan=1 colspan=1>5.74%</td><td rowspan=1 colspan=1>5.33%</td><td rowspan=1 colspan=1>4.23%</td><td rowspan=1 colspan=1>4.37%</td><td rowspan=1 colspan=1>5.64%</td></tr></table>

# Input Total RNA and Copy RNA (cRNA) Yield:

Repeatability studies, for total RNA input and cRNA yield were also conducted at two external sites. Eight replicates using $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNAs were tested in batches (8 samples at minimum per batch) and the median probe set signal $\% \mathrm { C V }$ from detected probe sets was calculated for the replicates and are shown in Table 4 below.

Table 4. $\% \mathrm { C V s }$ for specimens at external sites.   

<table><tr><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=2>MAQC A</td><td rowspan=1 colspan=2>MAQC B</td></tr><tr><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1000 ng</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.59%</td><td rowspan=1 colspan=1>4.75%</td><td rowspan=1 colspan=1>5.11%</td><td rowspan=1 colspan=1>4.99%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.68%</td><td rowspan=1 colspan=1>4.52%</td><td rowspan=1 colspan=1>6.45%</td><td rowspan=1 colspan=1>5.51%</td></tr></table>

The amount of starting material to be used with the AGP Reagents was between 100 $\boldsymbol { \mathrm { n g } } - 1 0 0 0 \boldsymbol { \mathrm { n g } }$ total RNA and the reagents were tested at two external studies using total RNA at the minimum and maximum amount of the input total RNA range. The defined amount of total RNA was processed with the Transcript Synthesis and Labeling Kit and the resulting amount of cRNA was measured   
spectrophotometrically. The results showed that both $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of input total RNA would yield sufficient cRNA to hybridization at least one microarray $( \geq 2 0$ µg cRNA) (see Table 5).

Table 5. cRNA concentration and yield from MAQC total RNA specimens.   

<table><tr><td rowspan=3 colspan=1>Specimen#</td><td rowspan=3 colspan=1>Startingamount</td><td rowspan=2 colspan=1>cRNAConc.(μg/μL)</td><td rowspan=1 colspan=1>cRNAYield</td><td rowspan=1 colspan=1>cRNAConc.</td><td rowspan=1 colspan=1>cRNAYield</td><td rowspan=2 colspan=1>cRNAConc.(μg/μL)</td><td rowspan=2 colspan=1>cRNAYield(μg)</td></tr><tr><td rowspan=1 colspan=1>(μg)</td><td rowspan=1 colspan=1>(μg/μL)</td><td rowspan=1 colspan=1>(μg)</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2 - Repeats</td></tr><tr><td rowspan=1 colspan=8>MAQC A</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>2.052</td><td rowspan=1 colspan=1>61.55</td><td rowspan=1 colspan=1>2.818</td><td rowspan=1 colspan=3>84.55   4.031    120.94</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.877</td><td rowspan=1 colspan=1>56.32</td><td rowspan=1 colspan=1>2.478</td><td rowspan=1 colspan=3>74.33  3.573     107.2</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.865</td><td rowspan=1 colspan=1>55.96</td><td rowspan=1 colspan=1>2.182</td><td rowspan=1 colspan=3>65.46  4.188    125.63</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.754</td><td rowspan=1 colspan=1>52.62</td><td rowspan=1 colspan=1>3.796</td><td rowspan=1 colspan=3>113.89  5.136    154.09</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.925</td><td rowspan=1 colspan=1>57.76</td><td rowspan=1 colspan=1>4.701</td><td rowspan=1 colspan=3>141.02  5.133    153.98</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.686</td><td rowspan=1 colspan=1>50.58</td><td rowspan=1 colspan=1>5.348</td><td rowspan=1 colspan=3>160.43  3.527     105.8</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.885</td><td rowspan=1 colspan=1>56.56</td><td rowspan=1 colspan=1>4.906</td><td rowspan=1 colspan=3>147.19  5.554    166.63</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.677</td><td rowspan=1 colspan=1>50.30</td><td rowspan=1 colspan=1>4.335</td><td rowspan=1 colspan=3>130.05  2.501     75.03</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.112</td><td rowspan=1 colspan=1>63.35</td><td rowspan=1 colspan=1>2.053</td><td rowspan=1 colspan=3>61.6</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.067</td><td rowspan=1 colspan=1>62.02</td><td rowspan=1 colspan=1>2.096</td><td rowspan=1 colspan=3>62.87</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.009</td><td rowspan=1 colspan=1>60.27</td><td rowspan=1 colspan=1>3.912</td><td rowspan=1 colspan=3>117.37</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.015</td><td rowspan=1 colspan=1>60.46</td><td rowspan=1 colspan=1>1.88</td><td rowspan=1 colspan=3>56.39</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.088</td><td rowspan=1 colspan=1>62.64</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=3>58.8</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.055</td><td rowspan=1 colspan=1>61.66</td><td rowspan=1 colspan=1>1.91</td><td rowspan=1 colspan=3>57.31</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.067</td><td rowspan=1 colspan=1>62.02</td><td rowspan=1 colspan=1>1.912</td><td rowspan=1 colspan=3>57.35</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.969</td><td rowspan=1 colspan=1>59.08</td><td rowspan=1 colspan=1>1.928</td><td rowspan=1 colspan=3>57.83</td></tr><tr><td rowspan=1 colspan=8>MAQC B</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.498</td><td rowspan=1 colspan=1>44.95</td><td rowspan=1 colspan=1>3.412</td><td rowspan=1 colspan=3>102.37  4.611     138.32</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.379</td><td rowspan=1 colspan=1>41.38</td><td rowspan=1 colspan=1>2.304</td><td rowspan=1 colspan=3>69.13  3.484    104.52</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.469</td><td rowspan=1 colspan=1>44.08</td><td rowspan=1 colspan=1>1.282</td><td rowspan=1 colspan=3>38.46  2.036     61.08</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.344</td><td rowspan=1 colspan=1>40.31</td><td rowspan=1 colspan=1>1.176</td><td rowspan=1 colspan=3>35.28   4.904    147.13</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.475</td><td rowspan=1 colspan=1>44.25</td><td rowspan=1 colspan=1>5.36</td><td rowspan=1 colspan=3>160.79  4.989    149.66</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.403</td><td rowspan=1 colspan=1>42.08</td><td rowspan=1 colspan=1>5.578</td><td rowspan=1 colspan=3>167.35  5.431    162.92</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.384</td><td rowspan=1 colspan=1>41.53</td><td rowspan=1 colspan=1>5.775</td><td rowspan=1 colspan=3>173.24 2.638     79.13</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>1.182</td><td rowspan=1 colspan=1>35.46</td><td rowspan=1 colspan=1>4.58</td><td rowspan=1 colspan=3>137.41  3.272     98.17</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.853</td><td rowspan=1 colspan=1>55.59</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=3>54.31</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.829</td><td rowspan=1 colspan=1>54.86</td><td rowspan=1 colspan=1>1.646</td><td rowspan=1 colspan=3>49.37</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.736</td><td rowspan=1 colspan=1>52.09</td><td rowspan=1 colspan=1>2.208</td><td rowspan=1 colspan=3>66.25</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.682</td><td rowspan=1 colspan=1>50.45</td><td rowspan=1 colspan=1>1.895</td><td rowspan=1 colspan=3>56.86</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.832</td><td rowspan=1 colspan=1>54.95</td><td rowspan=1 colspan=1>2.045</td><td rowspan=1 colspan=3>61.34</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.887</td><td rowspan=1 colspan=1>56.60</td><td rowspan=1 colspan=1>3.231</td><td rowspan=1 colspan=3>96.94</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>1.908</td><td rowspan=1 colspan=1>57.23</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=3>45.89</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1000 ng</td><td rowspan=1 colspan=1>2.041</td><td rowspan=1 colspan=1>61.24</td><td rowspan=1 colspan=1>2.004</td><td rowspan=1 colspan=3>60.12</td></tr></table>

The .CEL files for the $1 0 0 ~ \mathrm { n g }$ samples received from Site 2 were analyzed and found the performance of the Poly-A Control below the specifications for lys and phe due to the use of an incorrect dilution of the Poly-A Control in the RNA Control Kit for the $1 0 0 ~ \mathrm { n g }$ of total RNA from MAQC A and B samples. The 16 samples were repeated and the signal intensity of the Poly A controls was found to be adequate.

Reproducibility of cRNA yield using total RNA from human tissue specimens: To demonstrate cRNA yield with the AGP Reagents, testing was conducted internally by Affymetrix. One hundred nanograms $( 1 0 0 \mathrm { n g } )$ of total RNA from ten commercially available human tissues were run in triplicate using the Transcript Synthesis and Labeling Kit (Kit 2). All replicates $( 1 0 0 \% , 3 0 / 3 0 )$ of the samples tested yielded greater than $\geq 2 0 ~ \mu \ g$ cRNA and a cRNA concentration $\geq 0 . 6 2 5 ~ \mu \mathrm { g } / \mu \mathrm { L }$ . Data is provided in Table 6.

Table 6. Tissue specimen cRNA yield values.   

<table><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">cRNAConcentrationμg/μL</td><td colspan="1" rowspan="1">AdjustedcRNAyields (μg)</td><td colspan="1" rowspan="1">AveragecRNA yields(μg)</td></tr><tr><td colspan="1" rowspan="1">Kidney Total RNA R1</td><td colspan="1" rowspan="1">2.250</td><td colspan="1" rowspan="1">69.6</td><td colspan="1" rowspan="3">67.1</td></tr><tr><td colspan="1" rowspan="1">KidneyTotal RNAR2</td><td colspan="1" rowspan="1">2.112</td><td colspan="1" rowspan="1">65.4</td></tr><tr><td colspan="1" rowspan="1">KidneyTotal RNAR3</td><td colspan="1" rowspan="1">2.141</td><td colspan="1" rowspan="1">66.3</td></tr><tr><td colspan="1" rowspan="1">PancreasTotal RNAR1</td><td colspan="1" rowspan="1">1.796</td><td colspan="1" rowspan="1">55.6</td><td colspan="1" rowspan="3">54.2</td></tr><tr><td colspan="1" rowspan="1">PancreasTotalRNAR2</td><td colspan="1" rowspan="1">1.792</td><td colspan="1" rowspan="1">55.5</td></tr><tr><td colspan="1" rowspan="1">PancreasTotal RNAR3</td><td colspan="1" rowspan="1">1.664</td><td colspan="1" rowspan="1">51.5</td></tr><tr><td colspan="1" rowspan="1">Heart Total RNAR1</td><td colspan="1" rowspan="1">2.241</td><td colspan="1" rowspan="1">69.4</td><td colspan="1" rowspan="3">69.2</td></tr><tr><td colspan="1" rowspan="1">Heart Total RNAR2</td><td colspan="1" rowspan="1">2.338</td><td colspan="1" rowspan="1">72.4</td></tr><tr><td colspan="1" rowspan="1">Heart TotalRNAR3</td><td colspan="1" rowspan="1">2.126</td><td colspan="1" rowspan="1">65.8</td></tr><tr><td colspan="1" rowspan="1">MAQCB Total RNA_R1</td><td colspan="1" rowspan="1">2.443</td><td colspan="1" rowspan="1">75.6</td><td colspan="1" rowspan="3">74.4</td></tr><tr><td colspan="1" rowspan="1">MAQCB3 Total RNAR2</td><td colspan="1" rowspan="1">2.441</td><td colspan="1" rowspan="1">75.6</td></tr><tr><td colspan="1" rowspan="1">MAQCB Total RNAR3</td><td colspan="1" rowspan="1">3.009</td><td colspan="1" rowspan="1">72.1</td></tr><tr><td colspan="1" rowspan="1">Liver Total RNAR1</td><td colspan="1" rowspan="1">1.985</td><td colspan="1" rowspan="1">61.4</td><td colspan="1" rowspan="3">57.1</td></tr><tr><td colspan="1" rowspan="1">Liver Total RNAR2</td><td colspan="1" rowspan="1">1.843</td><td colspan="1" rowspan="1">57.0</td></tr><tr><td colspan="1" rowspan="1">Liver Total RNAR3</td><td colspan="1" rowspan="1">1.705</td><td colspan="1" rowspan="1">52.8</td></tr><tr><td colspan="1" rowspan="1">Breast Total RNAR1</td><td colspan="1" rowspan="1">2.161</td><td colspan="1" rowspan="1">66.9</td><td colspan="1" rowspan="3">62.2</td></tr><tr><td colspan="1" rowspan="1">Breast Total RNAR2</td><td colspan="1" rowspan="1">1.844</td><td colspan="1" rowspan="1">58.9</td></tr><tr><td colspan="1" rowspan="1">Breast Total RNAR3</td><td colspan="1" rowspan="1">1.965</td><td colspan="1" rowspan="1">60.8</td></tr><tr><td colspan="1" rowspan="1">Testicle Total RNA R1</td><td colspan="1" rowspan="1">2.207</td><td colspan="1" rowspan="1">68.3</td><td colspan="1" rowspan="3">69.5</td></tr><tr><td colspan="1" rowspan="1">Testicle Total RNA_R2</td><td colspan="1" rowspan="1">2.268</td><td colspan="1" rowspan="1">70.2</td></tr><tr><td colspan="1" rowspan="1">Testicle Total RNA R3</td><td colspan="1" rowspan="1">2.261</td><td colspan="1" rowspan="1">70.0</td></tr><tr><td colspan="1" rowspan="1">HeLa Total RNAR1</td><td colspan="1" rowspan="1">2.401</td><td colspan="1" rowspan="1">74.3</td><td colspan="1" rowspan="3">73.7</td></tr><tr><td colspan="1" rowspan="1">HeLa Total RNAR2</td><td colspan="1" rowspan="1">2.277</td><td colspan="1" rowspan="1">72.8</td></tr><tr><td colspan="1" rowspan="1">HeLa Total RNA R3</td><td colspan="1" rowspan="1">2.390</td><td colspan="1" rowspan="1">74.0</td></tr><tr><td colspan="1" rowspan="1">Thyroid Total RNA R1</td><td colspan="1" rowspan="1">1.689</td><td colspan="1" rowspan="1">52.2</td><td colspan="1" rowspan="3">50.5</td></tr><tr><td colspan="1" rowspan="1">Thyroid Total RNAR2</td><td colspan="1" rowspan="1">1.562</td><td colspan="1" rowspan="1">48.3</td></tr><tr><td colspan="1" rowspan="1">Thyroid Total RNAR3</td><td colspan="1" rowspan="1">1.644</td><td colspan="1" rowspan="1">50.8</td></tr><tr><td colspan="1" rowspan="1">Skeletal Muscle Total RNA_R1</td><td colspan="1" rowspan="1">2.066</td><td colspan="1" rowspan="1">63.9</td><td colspan="1" rowspan="3">62.3</td></tr><tr><td colspan="1" rowspan="1">Skeletal MuscleTotalRNAR2</td><td colspan="1" rowspan="1">2.043</td><td colspan="1" rowspan="1">63.2</td></tr><tr><td colspan="1" rowspan="1">Skeletal Muscle Total RNA R3</td><td colspan="1" rowspan="1">1.931</td><td colspan="1" rowspan="1">59.8</td></tr></table>

b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods): Controls:

The RNA Control Kit provides exogenous positive controls to monitor the entire eukaryotic target labeling process. Each array contains probe sets for several $B$ . subtilis genes that are absent in the samples analyzed (lys, phe, and dap). The RNA Control Kit contains in vitro synthesized, polyadenylated transcripts for these three $B$ . subtilis genes that are premixed at staggered concentrations to allow probe array users to assess the overall success of the assay. The concentrated Poly-A Control can be diluted with the Dilution Buffer and spiked directly into the RNA samples to achieve the final concentrations (i.e., ratio of copy number) of: lys 1:100,000; phe 1:50,000, and dap 1:6,667.

The spiked-in controls are then amplified and labeled together with the samples. Examining the hybridization intensities of these controls on arrays helps users monitor the amplification and labeling processes independently from the quality of the starting RNA samples. An additional synthetic biotinylated control oligo (Oligo B2) is included with the Transcription Detection kit and is added to the hybridization solution. The resulting signals allow for image analysis alignment.

Performance of the Poly-A Control, Oligo B2 and hybridization controls were evaluated by testing conducted internally at Affymetrix and at two external sites. For internal evaluation, three lots of AGP Reagent Kits were tested using $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNAs tested in quadruplicate. The testing conducted at the two external sites included eight replicates using $1 0 0 ~ \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ of MAQC A and B total RNAs in batches of a minimum of 8 samples. In both studies, the Oligo B2 controls allowed all .CEL files to be successfully automatically gridded for all samples. The analysis of the Poly-A Control showed the $3 ^ { \circ }$ AFFX-r2- Bs probe sets for all three spikes (lys, phe, and dap) were present with adequate signal intensities.

Signal intensities and $\mathrm { r } ^ { 2 }$ values for the correlation of the $3 ^ { \circ }$ AFFX-r2-Bs signal intensities with the relative ratio for each spike, followed the relative concentration in the Poly-A Control mixture: $l y s < p h e <$ dap and the $\mathrm { r } ^ { 2 }$ values met the acceptance

criteria of 0.900 for both studies. The $3 ^ { \circ }$ AFFX-r2 probe set for bioB was called present for all samples in both studies. The signal intensity for the hybridization controls (bioB, bioC, bioD and cre) followed the relative concentration in the mixture: $\mathrm { b i o B < b i o C < b i o D < c r \epsilon }$ e for all samples tested in both studies. One hundred percent of the samples passed the acceptance criteria for the performance of the controls in the internal study (80/80) and the external study (64/64).

In addition the performance of the controls was evaluated in the clinical studies. Forty-five total RNA samples were tested at each of the 3 clinical sites and $100 \%$ (135/135) passed the acceptance criteria for the Poly-A Control, Oligo B2 and hybridization controls resulting in successful .CEL files for analysis. Analysis of the results from the testing of 16 total RNA samples isolated from frozen tissues at one site, in duplicate, resulted in valid .CEL files from $100 \%$ (32/32) of the specimens.

# Stability:

Data from ongoing real-time stability studies conducted on the three kits which make up the AGP Reagent kits were provided to support the following:

Table 7. Summary of Stability data results and claims.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>RNA ControlKit</td><td rowspan=1 colspan=1>Transcript Synthesis&amp; Labeling Kit</td><td rowspan=1 colspan=1>Transcript DetectionKit</td></tr><tr><td rowspan=1 colspan=1>Normal Storageconditions</td><td rowspan=1 colspan=1>-15 to -30°C</td><td rowspan=1 colspan=1>Kit A = 2 to 8°CKit B = -15 to -30°C</td><td rowspan=1 colspan=1>Kit A &amp; B = 2 to 8°CKit C = -15 to -30°C</td></tr><tr><td rowspan=1 colspan=1>Stability parameter</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cYanmaGellowb</td></tr><tr><td rowspan=1 colspan=1>Freeze/Thaw</td><td rowspan=1 colspan=1>12 cycles</td><td rowspan=1 colspan=1>7 cycles</td><td rowspan=1 colspan=1>12 cycles</td></tr><tr><td rowspan=1 colspan=1>Freeze/Thaw claimed</td><td rowspan=1 colspan=1>8 cycles</td><td rowspan=1 colspan=1>4 cycles</td><td rowspan=1 colspan=1>8 cycles</td></tr><tr><td rowspan=1 colspan=1>Open vial</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>20 days</td><td rowspan=1 colspan=1>30 days</td></tr><tr><td rowspan=1 colspan=1>Open vial claimed</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>20 days</td><td rowspan=1 colspan=1>30 days</td></tr><tr><td rowspan=1 colspan=1>Shelf-life</td><td rowspan=1 colspan=1>14 months</td><td rowspan=1 colspan=1>14 months</td><td rowspan=1 colspan=1>8 months</td></tr></table>

d. Detection limit: Not applicable.   
e. Analytical specificity: Not applicable.   
f. Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device: Clinical performance of the AGP Reagents was performed in two parts using the Pathwork TOO assay for fresh/frozen tissues at three external sites.

The first study utilized 45 total RNA specimens previously isolated from 15 human tumor type specimens included in the Pathwork TOO test (k080896). Three independent specimens of each tumor type, based on available diagnosis, were included and consisted of bladder, breast, colorectal, gastric, germ cell, hepatocellular, kidney, lung, lymphoma, melanoma, ovary, pancreas, prostate, soft tissue sarcoma, and thyroid. Thirty-nine (39) of the 45 samples were the same or comparable total RNA samples used in the Pathwork Reproducibility Study from k080896 and six samples were received from a separate source. To ensure adequate system performance of the Pathwork Tissue of Origin Test, each site was required to verify and/or establish specimen characteristics and specifications during the study. These characteristics and specifications included: the amount and purity of the total RNA in the amplification reaction was at least 200 ng and $\geq 1 . 7$ (A260/A280); the labeled cRNA yield was at least ${ \tt > } 1 5 \mu \mathrm { g } ;$ ; that $1 5 ~ { \mu \mathrm { g } }$ of labeled cRNA was used in the fragmentation reaction; $1 0 ~ \mu \ g$ labeled cRNA was used for the hybridization reaction and that Oligo B2 performed as expected.

Each site, using different lots of the AGP Reagent kits, processed the total RNA from each specimen according to the step-by-step procedure for processing the total RNA with the AGP Reagent kits. Only 39 of the 45 specimens were processed using the One-Cycle reagents at two sites only. In both cases however, each site followed the Pathwork Specimen Processing Guide for Frozen Specimens. The resulting data files (.CEL files) were forwarded to Pathwork and processed by the TOO algorithm version 21.2, standardization v6.2, an electronic report for clinical interpretation was generated. For the study the input total RNA was 200 ng which produced sufficient cRNA for the hybridization of one PathchipTM microarray $( \geq 1 5 \mu \ g )$ . The clinical sites and Pathwork Diagnostics were both blinded to each specimen’s available diagnosis.

All samples processed with AGP reagents passed provided results. However, all samples processed with One-Cycle reagents did not meet the amount of labeled cRNA generated. Predictions across runs from the two reagents consistently agreed or disagreed with the available clinical diagnosis except for those situations when the Similarity (SS) Scores are close to 30. Table 8 below shows the range of similarity scores generated for the 39 specimens tested with both types of reagents. Agreement with the available clinical diagnosis was not considered as a success criteria for this study as the studies were not intended to redefine the performance of the Pathwork TOO assay. Discrepancies found in this study mostly occurred near the decisionmaking cut-offs (value of 30) which were defined by the assay manufacturer.

Table 8. Reproducibility of SS Scores across sites (total RNA study) for the 39 specimens in common with the Pathwork reproducibility study.   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>SS Range</td><td rowspan=1 colspan=1>N (# Specimens)</td><td rowspan=1 colspan=1>Average SD</td><td rowspan=1 colspan=1>Average %CV</td></tr><tr><td rowspan=5 colspan=1>IVD</td><td rowspan=1 colspan=1>[0,20]</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>9.5%</td></tr><tr><td rowspan=1 colspan=1>[20,40]</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.54</td><td rowspan=1 colspan=1>9.6%</td></tr><tr><td rowspan=1 colspan=1>[40,60]</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.31</td><td rowspan=1 colspan=1>12.2%</td></tr><tr><td rowspan=1 colspan=1>[60,80]</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.81</td><td rowspan=1 colspan=1>4.0%</td></tr><tr><td rowspan=1 colspan=1>[80,100]</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.47</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=5 colspan=1>One Cycle</td><td rowspan=1 colspan=1>[0,20]</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5.96</td><td rowspan=1 colspan=1>55.4%</td></tr><tr><td rowspan=1 colspan=1>[20,40]</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>14.92</td><td rowspan=1 colspan=1>41.9%</td></tr><tr><td rowspan=1 colspan=1>[40,60]</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>22.80</td><td rowspan=1 colspan=1>44.6%</td></tr><tr><td rowspan=1 colspan=1>[60,80]</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13.80</td><td rowspan=1 colspan=1>19.3%</td></tr><tr><td rowspan=1 colspan=1>[80,100]</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3.64</td><td rowspan=1 colspan=1>4.2%</td></tr></table>

The three sites using AGP Reagents agreed on 43/45 samples in each case for an overall concordance of $9 5 . 6 \%$ . The $C V \%$ values were found to be higher for the [0,20], [20,40], and [40,60] SS ranges using the One-Cycle reagents than observed in $\mathrm { k 0 8 0 8 9 6 }$ . This may be due to the number of specimens whose scores fell within each range. The worst comparison between One-Cycle reagents was $8 7 . 5 \%$ . The result of the comparison between the results using the AGP Reagents versus the One-Cycle reagents was $9 1 . 5 \%$ .

The second part of the prospective clinical study was conducted at a single external site and included testing of 16 frozen tissues from RNA extraction to hybridization to the PathchipTM microarray and subsequent analysis by Pathwork. Each of the 15 tumor types included in the Pathwork TOO Fresh/Frozen assay were represented by the 16 specimens. The site and Pathwork were both blinded to the available diagnosis of each tissue specimen. The resulting .CEL files were sent to Pathwork Diagnostics for analysis.

The site extracted total RNA from the frozen tissue following the instructions provided in the Pathwork Specimen Processing Guide for Frozen Specimens. The total RNA extracted from each specimen was split for processing on a single lot of each reagent. Two runs per reagent were performed. An external total RNA control was added to each run. Predictions across runs from two different types of reagents consistently agree or disagree with the available clinical diagnosis except for those situations when SS are close to 30 in which case simple assay variation can cause a SS to switch between being above or below 30. Table 9 below, shows the number and average $\% \mathrm { C V }$ shown for the specimens across runs when each reagent type was used.

Table 9. Reproducibility of SS Scores across runs (from frozen specimens).   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Similarity Score Range</td><td rowspan=1 colspan=1>N (# Specimens)</td><td rowspan=1 colspan=1>Average SD</td><td rowspan=1 colspan=1>Average %CV</td></tr><tr><td rowspan=5 colspan=1>IVD</td><td rowspan=1 colspan=1>[0,20)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>17.2%</td></tr><tr><td rowspan=1 colspan=1>[20,40)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.72</td><td rowspan=1 colspan=1>9.2%</td></tr><tr><td rowspan=1 colspan=1>[40,60)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.66</td><td rowspan=1 colspan=1>13.3%</td></tr><tr><td rowspan=1 colspan=1>[60,80)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>[80,100)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>1.3%</td></tr><tr><td rowspan=5 colspan=1>One-Cycle</td><td rowspan=1 colspan=1>[0,20)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.32</td><td rowspan=1 colspan=1>16.0%</td></tr><tr><td rowspan=1 colspan=1>[20,40)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.73</td><td rowspan=1 colspan=1>9.2%</td></tr><tr><td rowspan=1 colspan=1>[40,60)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>[60,80)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4.36</td><td rowspan=1 colspan=1>6.2%</td></tr><tr><td rowspan=1 colspan=1>[80,100)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>1.1%</td></tr></table>

For four of the tissue samples, (GA, GC, LI and PA), the predictions from all four runs were not identical but in all four cases, the SS that cause the change in prediction are hovering right at the $30 \%$ cutoff. There were three samples for which the prediction disagreed with the available clinical diagnosis for all four predictions, and in all three cases, all four predictions were identical. The percent agreement between

One-Cycle reagent runs was $8 7 . 5 \%$ and between AGP runs was $8 7 . 5 \%$ . A comparison between the combinations of runs and reagents varied from $7 5 . 0 \%$ to $9 3 . 8 \%$ . The overall percent agreement between One-Cycle and AGP Reagents for the 16 samples included in the analysis is $8 2 . 9 \%$ .

b. Matrix comparison: Not applicable.

3. Clinical studies: a. Clinical Sensitivity: Not applicable. b. Clinical specificity: Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range: Not applicable.

N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.