# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: k110786   
B. Purpose for Submission: Modification to a previously cleared device; new sample type   
C. Measurand: Genotype of Cytochrome P450 2C9 (CYP450 2C9) and Vitamin K epoxide reductase complex subunit I (VKORC1)   
D. Type of Test: Qualitative genetic test for single nucleotide polymorphism detection   
E. Applicant: GenMark Diagnostics   
F. Proprietary and Established Names: eSensor Warfarin Sensitivity Saliva Test

# G. Regulatory Information:

1. Regulation section: 21 CFR §862.3360 – Drug Metabolism Enzyme Genotyping Test 21 CFR §864.7750 – Prothrombin Time Test 21 CFR §862.2570 - Instrumentation for Clinical Multiplex Test Systems

2. Classification: Class II

3. Product code: ODW - Cytochrome P450 2C9 (CYP450 2C9) Drug Metabolizing Enzyme Genotyping System ODV - Vitamin K epoxide reductase complex subunit 1 (VKORC1) Genotyping System NSU - Instrumentation for Clinical Multiplex Test Systems

4. Panel: Toxicology (91), Hematology (81), Chemistry (75)

# H. Intended Use:

1. Intended use(s): See Indications for use below.

2. Indication(s) for use:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test is an in vitro diagnostic test for the detection and genotyping of the $^ { * 2 }$ and $^ { * 3 }$ alleles of the cytochrome P450 (CYP450) 2C9 gene locus and the Vitamin K epoxide reductase C1 (VKORC1) gene promoter polymorphism $\left( - 1 6 3 9 \mathrm { G } > \mathrm { A } \right)$ ) from genomic DNA of human saliva samples collected using the Oragene $^ \mathrm { \textregistered }$ Dx Device, as an aid in the identification of patients at risk for increased warfarin sensitivity.

3. Special conditions for use statement(s): For Prescription use only.

4. Special instrument requirements:

Oragene•Dx collection device (k110701) models OGD-500, OGD-575, OXD-525 and OYD-500, eSensor $\textsuperscript { \textregistered }$ XT-8 Instrument (k073720)

# I. Device Description:

The kit consists of the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test cartridge, the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test amplification reagents (including PCR mix and DNA polymerase), the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test detection reagents (including exonuclease, probes and hybridization buffer ingredients) and the eSensor $\textsuperscript { \textregistered }$ XT-8 System. One eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test Kit has sufficient materials for 24 tests.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Osmetech Molecular Diagnostics eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test

2. Predicate K number(s): k073720

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Proposed Device</td><td rowspan=1 colspan=1>Predicate Device(k073720)</td></tr><tr><td rowspan=1 colspan=1>Intended use</td><td rowspan=1 colspan=1>For the detection andgenotyping of the *2 and* 3 alleles of thecytochrome P450(CYP450) 2C9 gene locusand the Vitamin Kepoxide reductase C1(VKORC1) genepromoter polymorphism(-1639G&gt;A)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Indications for use</td><td rowspan=1 colspan=1>As an aid in theidentification of patientsat risk for increasedwarfarin sensitivity.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Device components</td><td rowspan=1 colspan=1>Test cartridge,amplification reagents(including PCR mix andDNA polymerase),detection reagents(including exonuclease,probes and hybridizationbuffer ingredients) andthe eSensor® XT-8System</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Proposed Device</td><td rowspan=1 colspan=1>Predicate Device(k073720)</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Genomic DNA obtainedfrom saliva</td><td rowspan=1 colspan=1>Genomic DNA obtainedfrom a human wholeblood sample</td></tr><tr><td rowspan=1 colspan=1>DNA extraction method</td><td rowspan=1 colspan=1>Manual ethanolextraction method;provided in AttachmentA of the package insert</td><td rowspan=1 colspan=1>Any whole blood DNAextraction methodvalidated by thelaboratory that providesat least 10 ng of DNA ata minimumconcentration of 2ng/μL.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

None cited.

# L. Test Principle:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test uses an electrochemical detection based microarray method for determining the genotype of a defined panel of polymorphisms from purified genomic DNA isolated from human saliva. This method was cleared under k073720 using gDNA from blood as the sample type. In the process, regions of the genome containing the polymorphisms of interest are amplified by PCR, and the resulting double stranded PCR amplicons are digested with exonuclease lambda to generate single stranded target DNA which is then mixed with a hybridization solution containing a pair of allele-specific oligonucleotide signal probes for each polymorphism. Each signal probe within the pair is labeled with a genotype-specific ferrocene derivative.

The mixture of amplified target DNA sample and signal buffer is loaded onto a test cartridge containing single-stranded oligonucleotide capture probes that are covalently bound to gold-plated electrodes. The cartridge is then inserted into the XT8 Instrument. During the hybridization in the XT-8 Instrument, each target DNA binds to a capture probe. Each pair of working electrodes on the array contains a different capture probe.

The signal and capture probes are designed with sequences complementary to immediately adjacent regions on the corresponding target DNA sequence and so both signal and capture probes bind to complementary sequences on the target DNA. In this manner, a three-member complex is formed among capture probe, target, and signal probe based on sequence-specific hybridization. This process brings the end of the signal probe containing electrochemically active ferrocene labels into close proximity to the electrode surface.

Hybridization of the three-member complex at the electrode surface and subsequent application of an excitation voltage causes the ferrous ion in each ferrocene group to undergo cyclic oxidation and reduction at its characteristic redox potential, leading to loss or gain of an electron, and the generation of an alternating current at the electrode surface that is measured using voltammetry. Higher-order harmonic signal analysis also facilitates discrimination of ferrocene-dependent faradaic current from background capacitive current. Signals from the ferrocene labels are detected and measured by instrument software, and the ratio of signals from the different labels allows identification of genotype. Genotyping boundaries and signal threshold for each polymorphism are pre-programmed into instrument software, and genotypes are called by comparison of the signal ratio observed for an unknown sample to the SNPspecific genotyping boundaries and signal threshold. Sequential analysis of each electrode allows genotyping of multiple mutations or polymorphisms.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Performance of DNA from saliva samples collected by the Oragene Dx collection devices on the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test in two reproducibility studies was demonstrated.

1. Device reproducibility using prepared sample panel: a study was performed to evaluate the reproducibility of the performance of the Oragene∙Dx device (OGD-500 format, k110701) with the GenMark Diagnostics eSensor Warfarin Sensitivity Saliva Test across multiple sites and operators.

Ten donors self-collected six saliva samples each (two samples per lot x three lots of OGD-500). Two samples collected using the same OGD-500 lot were pooled to generate three samples per donor. Triplicate aliquots of each sample from each donor were provided to four operators at three sites for DNA extraction (purification)1. Site 2 and 3 each had one operator and Site 1 (internal) had two operators for a study total of four operators.

All purified genomic DNA samples were tested for concentration and $\mathbf { A } _ { 2 6 0 } / \mathbf { A } _ { 2 8 0 }$ ratio at a single external DNA Testing Site, Site 3. The eSensor Warfarin Sensitivity Testing was performed by the same four operators at the three sites where DNA extraction was also performed.

Genotypes of the ten donors:   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Genotype by Sequencing</td></tr><tr><td rowspan=1 colspan=1>Donor ID</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>VKOR</td></tr><tr><td rowspan=1 colspan=1>RP - 1</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>RP - 2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=1>RP - 3</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=1>RP - 4</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>RP - 5</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td></tr><tr><td rowspan=1 colspan=1>RP - 6</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td></tr><tr><td rowspan=1 colspan=1>RP - 7</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>RP - 8</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>RP - 9</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td></tr><tr><td rowspan=1 colspan=1>RP - 10</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td></tr></table>

First run genotyping results:   

<table><tr><td rowspan=2 colspan=1>DonorID</td><td rowspan=1 colspan=3>Genotype by sequencing</td><td rowspan=1 colspan=4>Number of Samples Tested byeSensor</td><td rowspan=1 colspan=4>Number of Correct Calls</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Op 3</td><td rowspan=1 colspan=1>Op 4</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Op 3</td><td rowspan=1 colspan=1>Op 4a</td></tr><tr><td rowspan=1 colspan=1>RP-1</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72.20%</td></tr><tr><td rowspan=1 colspan=1>RP-2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>70.00%</td></tr><tr><td rowspan=1 colspan=1>RP-3</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>86.10%</td></tr><tr><td rowspan=1 colspan=1>RP-4</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>97.20%</td></tr><tr><td rowspan=1 colspan=1>RP-5</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-6</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.00%</td></tr><tr><td rowspan=1 colspan=1>RP-7</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>75.00%</td></tr><tr><td rowspan=1 colspan=1>RP-8</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>83.30%</td></tr><tr><td rowspan=1 colspan=1>RP-9</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>94.40%</td></tr><tr><td rowspan=1 colspan=1>RP-10</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>97.20%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>85.30%</td></tr></table>

a two first-pass runs were invalidated at Site 3 due to contamination in the PCR blank (DCM failures) which resulted in 47 no-calls.

After final run and investigation:   

<table><tr><td rowspan=2 colspan=1>DonorID</td><td rowspan=1 colspan=3>Genotype by sequencing</td><td rowspan=1 colspan=4>Number of Samples Tested byeSensor</td><td rowspan=1 colspan=4>Number of Correct Calls</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Op 3</td><td rowspan=1 colspan=1>Op 4</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Op 3</td><td rowspan=1 colspan=1>Op 4</td></tr><tr><td rowspan=1 colspan=1>RP-1</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-2</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-3</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-4</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-5</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-6</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-7</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-8</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-9</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RP-10</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td></tr></table>

# 2. Reproducibility of Sample Collection, Processing and Testing Procedure:

A study was performed to evaluate the reproducibility of the entire sample collection, processing and testing procedure. Samples were directly shipped from donors to investigational sites for analysis of DNA concentration, $\mathbf { A } _ { 2 6 0 } / \mathbf { A } _ { 2 8 0 }$ ratio and eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test genotyping. Donors were selected based on their naivety to the saliva collection device; all donors had used the product at most once previously.

The selected 15 donors had the following genotypes:

<table><tr><td rowspan=1 colspan=1>Donor</td><td rowspan=1 colspan=1>CYP2C9</td><td rowspan=1 colspan=1>VKORC1</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>G/A</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>G/A</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>A/A</td></tr><tr><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>G/A</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>A/A</td></tr><tr><td rowspan=1 colspan=1>Q</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>A/A</td></tr><tr><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>A/A</td></tr><tr><td rowspan=1 colspan=1>S*</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>A/A</td></tr><tr><td rowspan=1 colspan=1>T</td><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>G/A</td></tr></table>

The study was conducted at three sites: one internal site and two external clinical laboratories. Donors $( \mathrm { n } { = } 1 5 )$ were each shipped four OGD-500 devices. Each donor was asked to provide four samples: one sample was sent directly by the donors to each of the three sites. (One sample was sent to DNA Genotek for remediation purposes if needed.) Each operator extracted one aliquot of DNA from each saliva sample they received and determined the DNA concentration and $\mathrm { A } _ { 2 6 0 } / \mathrm { A } _ { 2 8 0 }$ ratio using their laboratory’s standard procedures, prior to testing the sample on the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test. devices were shipped to 15 donors. One donor did not return any of their samples and thus was excluded from the study. 14 samples were tested once at each site for a study total of 42 samples.

First pass genotyping test results:

<table><tr><td rowspan=2 colspan=1>Donor</td><td rowspan=1 colspan=3>Genotype by sequencing</td><td rowspan=1 colspan=3>Number of SamplesTested by eSensor</td><td rowspan=1 colspan=3>Number of Correct Calls</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.70%</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Q</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>T</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=4>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>90.50%</td></tr></table>

1 This sample, F3, was tested twice (also incorrect in re-testing table below), and both times the results were “low signal for all polymorphisms”. The sample was cloudy suggesting that it was compromised. A new sample was obtained from the donor, tested, and a correct result was obtained.

2 Upon sequencing the reference sample for Donor S it was determined that this donor had an interfering mutation at the site adjacent to the $2 \mathrm { C } 9 ^ { \ast } 2$ mutation: $4 2 9 \mathrm { C } { > } \mathrm { T }$ . This mutation is described in the literature and is known to impact genotyping results. The patient’s $^ { * 2 }$ genotype is heterozygous at $^ { * 2 }$ as determined by sequencing but the eSensor Warfarin Sensitivity Saliva test result was homozygous for $^ { * 2 }$ . This limitation is stated in the GenMark Diagnostics eSensor Warfarin Sensitivity Saliva Test package insert.

After re-testing (Donor S excluded):   

<table><tr><td rowspan=2 colspan=1>Donor</td><td rowspan=1 colspan=3>Genotype by sequencing</td><td rowspan=1 colspan=3>Number of SamplesTested by eSensor</td><td rowspan=1 colspan=3>Number of Correct Calls</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.70%</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Q</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>MUT</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>T</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>HET</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=4>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>97.40%</td></tr></table>

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): See k073720.

# d. Detection limit:

The recommended input for the eSensor Warfarin Sensitivity Saliva Test is 10 ng of genomic DNA $5 \mu \mathrm { L }$ of $2 { \mathrm { n g / \mu L } }$ genomic DNA sample). All four formats of the Oragene•Dx collection device provide genomic DNA with sufficient DNA yield and concentration for the genotyping test (e.g., for the OGD-500 model in the sample volume tolerance study of k110701, median DNA yield was $3 6 . 1 ~ \mu \mathrm { g }$ and median DNA concentration was $4 2 . 1 \ \mathrm { n g / \mu L }$ ). The same study in k110701 also showed that underfilling the Oragene•Dx devices by $50 \%$ less saliva than recommended provided genomic DNA with sufficient DNA yield and concentration for the genotyping test (e.g., for the OGD-500, median DNA yield was $1 8 . 0 \mu \mathrm { g }$ and median DNA concentration was $2 6 . 6 \mathrm { n g / \mu L }$ .

# e. Analytical specificity:

Effect of Endogenous Interfering Substances: Interfering substances including salivary $\mathfrak { a }$ -amylase, hemoglobin, immunoglobulin A (IgA) and total protein were spiked into saliva samples at the highest amounts found in literature. 10 donors provided five saliva samples each which were each spiked with one of the four interfering substances. A control sample was included. Three extractions were performed on each sample. There was $100 \%$ agreement between the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test results and bidirectional DNA sequencing for all test substances in first pass, demonstrating no effect of any interfering substances on genotyping.

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls</td><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Amylase</td><td rowspan=1 colspan=1>260 ± 45U/mL</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>20 mg/mL</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>IgA</td><td rowspan=1 colspan=1>188 ± 80 mg/L</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TotalProtein</td><td rowspan=1 colspan=1>1.46 ± 0.4mg/mL</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Effect of Exogenous Interfering Substances: Potentially interfering exogenous substances (eating, drinking, chewing gum, using mouthwash and smoking) introduced into saliva samples through various activities were tested. Each activity group was composed of five donors who each provided three samples - a baseline/control sample prior to the activity, and samples collected immediately after the activity and then 30 minutes after the activity. Three samples per donor were tested. There was $100 \%$ agreement between the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test results and bidirectional DNA sequencing for all activities tested in first pass, demonstrating no effect of any interfering substances on genotyping.

<table><tr><td rowspan=1 colspan=1>Activity</td><td rowspan=1 colspan=1>Time-point</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls</td><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=3 colspan=1>Eating</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Immediate</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Drinking</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Immediate</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Chewing Gum</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Immediate</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Mouthwash</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Immediate</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Smoking</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Immediate</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

f. Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device:

Method comparison evaluation was performed in order to determine accuracy of the eSensor Warfarin Sensitivity Saliva Test as compared to bi-directional DNA sequencing. Testing was conducted at one site only (internal site). The test panel for the method comparison study consisted of 316 unique human genomic DNA samples. These samples were isolated from saliva specimens collected using the Oragene•Dx OGD-500 device followed by DNA extraction using a manual method using ethanol (this method is provided in Attachment A of the package insert). The study was performed with six lot of eSensor Warfarin Sensitivity Saliva Test, two instruments, four operators over multiple days. The total assay correct call rate is $9 9 . 0 5 \%$ and a $9 5 \%$ confidence lower bound of $9 7 . 5 6 \%$ .

Agreement between eSensor Warfarin Sensitivity Saliva Test and Bi-directional DNA Sequencing (by loci):   

<table><tr><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>NumberTested</td><td rowspan=1 colspan=1>ReplicatesperSample</td><td rowspan=1 colspan=1>Number ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>NumberofIncorrectCalls</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% One-SidedConfidenceLower Limit</td></tr><tr><td rowspan=1 colspan=1>2C9*2wt/wt</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>96.8%</td></tr><tr><td rowspan=1 colspan=1>2C9*2wt/*2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>93.4%</td></tr><tr><td rowspan=1 colspan=1>2C9*2*2/*2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>71.7%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2C9*3wt/wt</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>274</td><td rowspan=1 colspan=1>2d</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>96.3%</td></tr><tr><td rowspan=1 colspan=1>2C9*3wt/*3</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.6%</td></tr><tr><td rowspan=1 colspan=1>2C9*3*3/*3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>36.8%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>VKORC13673 (-1639)GG</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>94.9%</td></tr><tr><td rowspan=1 colspan=1>VKORC13673 (-1639)GA</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>96.5%</td></tr><tr><td rowspan=1 colspan=1>VKORC13673(-1639)AA</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.3%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

a Genotype determined through bi-directional DNA sequencing b Calls produced on first run

c Bi-directional sequencing revealed the sample had a mutation that interfered with the eSensor test. This rare mutation and its effect on the assay is noted in the limitations section of the package insert. d Investigation by the sponsor demonstrated that residual ethanol in the sample can lead to an incorrect call; see below.

During the extraction process, users should ensure that any residual ethanol has been removed prior to DNA rehydration. Excessive $( \geq 1 2 . 5 \% )$ carryover of ethanol from the extracted DNA into the eSensor Warfarin Sensitivity Saliva Test may result in an incorrect call, as seen in the method comparison study for two samples.

Agreement between eSensor Warfarin Sensitivity Saliva Test and Bi-directional DNA Sequencing (by sample):   

<table><tr><td rowspan=1 colspan=3>Sample genotype</td><td rowspan=3 colspan=1>#sampletested</td><td rowspan=1 colspan=4>First time run</td><td rowspan=1 colspan=4>Final result</td></tr><tr><td rowspan=1 colspan=2>2C9</td><td rowspan=2 colspan=1>VKORC13673(-1639)</td><td rowspan=2 colspan=1>#CorrectCallsc</td><td rowspan=2 colspan=1># NoCalls</td><td rowspan=2 colspan=1># In-correctCalls</td><td rowspan=2 colspan=1>CorrectCallRatee(%)</td><td rowspan=2 colspan=1>#CorrectCallsc</td><td rowspan=2 colspan=1># NoCalls</td><td rowspan=2 colspan=1># In-correctCalls</td><td rowspan=2 colspan=1>CorrectCallRatee(%)</td></tr><tr><td rowspan=1 colspan=1>*2</td><td rowspan=1 colspan=1>*3</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>88.9%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.0%</td></tr><tr><td rowspan=1 colspan=3>Total</td><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>99.1%</td></tr></table>

a Genotype determined through bi-directional DNA sequencing b Excludes samples with indeterminate/no calls c A sample with correct call indicates a correct call at all three loci. One incorrect or no call at one out of the three loci for the sample is considered an incorrect or indeterminate call for the whole sample d Final results reflect one time repeat of samples with indeterminate calls e Correct call rate $= \#$ samples with correct calls/# samples tested

b. Matrix comparison:

Performance of DNA from saliva samples collected by the four formats of the Oragene•Dx collection device, OGD-500, OYD-500, OXD-525, and OGD575 on the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test was demonstrated and found to be equivalent; see k110701. The four formats of the Oragene Dx collection device, OGD-500, OYD-500, OXD-525, and OGD-575 can be used with the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.   
$b$ . Clinical specificity: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range: See k073720.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.