# K113731

# 510(k) Summary

Submitter Name: Address:

# Contact:

GenMark Diagnostics   
5964 La Place Court   
Carlsbad, CA 92008   
Joel Centeno   
Vice President, Quality, Regulatory, and Clinical Affairs   
1 (760) 448-4304   
1 (760) 683-6821   
joel.centeno@genmarkdx.com   
December 16, 2011   
eSensor® Respiratory Viral Panel (RVP)   
Respiratory Viral Panel Multiplex Nucleic Acid Assay Syste   
Viral RNA/DNA of the foliowing targets:   
Phone:   
Fax:   
E-mail:   
Date Prepared:   
Device Trade Name:   
Device Common Name   
Measurand:

<table><tr><td>Target</td><td>Abrv.</td><td>Target</td><td>Abrv.</td></tr><tr><td>Influenza A</td><td>Flu A</td><td>Human Rhinovirus</td><td>HRV</td></tr><tr><td>Influenza A H1</td><td>Flu A H1</td><td>Human Metapneumovirus</td><td>hMPV</td></tr><tr><td>Influenza A H3</td><td>Flu A H3</td><td>Adenovirus B/E</td><td>ADV B/E</td></tr><tr><td>Influenza A 2009 H1N1</td><td>2009 H1N1</td><td>Adenovirus C</td><td>ADV C</td></tr><tr><td>Influenza B</td><td>Flu B</td><td>Parainfluenza Virus 1</td><td>PIV 1</td></tr><tr><td>Respiratory Syncytial Virus A</td><td>RSV A</td><td>Parainfluenza Virus 2</td><td>PIV 2</td></tr><tr><td>Respiratory Syncytial Virus B</td><td>RSV B</td><td>Parainfluenza Virus 3</td><td>PIV 3</td></tr></table>

Sample Type: Technology: Device Panel: Classification Name:

Classification Code:

Nasopharyngeal Swab (NPS)   
Polymerase Chain Reaction (PCR)   
OIVD Division of Immunology and Microbiology   
Respiratory Viral Panel Multiplex Nucleic Acid Assay, 866.3980 Instrumentation for clinical multiplex test systems, 862.2570. OCC: Respiratory viral panel multiplex nucleic acid assay   
OEM: Human Metapneumovirus (hMPV) RNA assay system OEP: Influenza A virus subtype differentiation nucleic acid assay OOU: Parainfluenza Multiplex Nucleic Acid Assay   
NSU: Instrumentation for clinical multiplex test systems   
Luminex® xTag™ RVP, 510(k) Number K081483   
Classification Code: OCC, OEM, OEP   
Regulation No. 866.3980   
eSensor® Warfarin Sensitivity Test, 510(k) No. K073720   
Classification Code (applicable): NSU Instrumentation for   
Clinical Multiplex Test Systems

# Predicate Device(s):

# Intended Use:

The eSensor® Respiratory Viral Panel (RVP) is a qualitative nucleic acid multiplex in vitro dianostic  intended or u on he eSen X ${ \boldsymbol { \cdot } } { \boldsymbol { 8 } } ^ { \dagger M }$ system or he simultaneous detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals exhibiting signs and symptoms of respiratory infection.

The following virus types and subtypes are identified using the eSensor RVP: Influenza A, Influenza A H1 Seasonal Subtype, Influenza A H3 Seasonal Subtype; Influenza A 2009 H1N1 subtype, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype BParainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Human Metapneumovirus, Human Rhinovirus, Adenovirus species B/E, and Adenovirus species C.

The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and epidemiological information.

Negative results do not preclude respiratory viral infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Positive results do not rule out bacterial infection, or co-infection with other viruses. The agent detected may not be the definite cause of disease.The use of additional laboratory testing e.g.bacterial and viral culture immunofluorescence and radiography) and clinical presentation must be taken into consideration in the final diagnosis of respiratory viral infection.

Performance characteristics for Influenza A were established during the 2010/2011 influenza season when Influenza A 2009 H1N1 and H3N2 were the predominant Influenza A viruses in circulation. When other Influenza A viruses emerge, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

For prescription use only.

Indication for Use: Same as Intended Use

# Device Description:

The eSensor RVP is a multiplex microarray-based genotyping test system. It is based on the principles of competitive DNA hybridization using a sandwich assay format, wherein a singlestranded target binds concurrently to sequence-specific solution-phase signal probe and solidphase electrode-bound capture probe. The test employs reverse transcription polymerase chain reaction amplification, exonuclease digestion and hybridization of target DNA/RNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate singlestranded DNA suitable for hybridization. Hybridization occurs in the eSensor XT-8 Cartridge (described below) where the single-stranded target DNA is mixed with a hybridization solution containing labeled signal probes.

During hybridization, the single-stranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working electrode surface. Single-stranded signal probes (abeled with electrochemically active ferrocenes) bind to the target adjacent to the capture probe. When inserted into the eSensor XT-8 instrument (described below), simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each pair of working electrodes on the array contains a different capture probe, and sequential analysis of each electrode allows detection of multiple viral targets.

Table 1: Reagents supplied with the kit:   

<table><tr><td rowspan=1 colspan=1>Box</td><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Packaging &amp; Quantity</td><td rowspan=1 colspan=1>Storage</td></tr><tr><td rowspan=2 colspan=1>eSensorRespiratory ViralPanel Cartridges</td><td rowspan=1 colspan=1>eSensor® RVP Cartridges</td><td rowspan=1 colspan=1>6 foil bags with 8 cartridgeseach in each cartridge pack</td><td rowspan=1 colspan=1>.10-25</td></tr><tr><td rowspan=1 colspan=1>eSensor Respiratory Viral PanelProduct Insert</td><td rowspan=1 colspan=1>1 copy</td><td rowspan=1 colspan=1>Dry place(retain for reference)</td></tr><tr><td rowspan=3 colspan=1>eSensor*Respiratory ViralPanel AmplificationReagents</td><td rowspan=1 colspan=1>RVP Enzyme Mix</td><td rowspan=1 colspan=1>2 vials with 40 μL each</td><td rowspan=3 colspan=1>-15 to -30 (in a designated pre-PCR location)</td></tr><tr><td rowspan=1 colspan=1>RVP PCR Mix</td><td rowspan=1 colspan=1>2 vials with 1000 μL each</td></tr><tr><td rowspan=1 colspan=1>MS2 Internal Control</td><td rowspan=1 colspan=1>2 vials with 300 μL each</td></tr><tr><td rowspan=4 colspan=1>eSensorRespiratory ViralPanel DetectionReagents</td><td rowspan=1 colspan=1>RVP Signal Buffer</td><td rowspan=1 colspan=1>2 vials with 2200 μL each</td><td rowspan=4 colspan=1>-15 to -30 (in a designated post-PCR location)</td></tr><tr><td rowspan=1 colspan=1>Exonuclease</td><td rowspan=1 colspan=1>2 vials with 145 µL each.</td></tr><tr><td rowspan=1 colspan=1>Buffer-1</td><td rowspan=1 colspan=1>2 vials with 350 μL each</td></tr><tr><td rowspan=1 colspan=1>Buffer-2</td><td rowspan=1 colspan=1>2 vials with 700 μL each</td></tr></table>

# The Assay Cartridge (eSensor XT-8 Cartridge)

The eSensor XT-8 cartridge device consists of a printed circuit board (PCB) with a multi-layer laminate and a plastic cover that forms a hybridization chamber. The cartridge is fitted with a pump and check valves that circulate the hybridization solution when inserted into the eSensor XT-8 instrument. The PCB chip consists of an array of 72 gold-plated working electrodes, a silver reference electrode, and two gold-plated auxiliary electrodes. Each working electrode has a connector contact pad on the opposite side of the chip for electrical connection to the eSensor XT-8 instrument. Each electrode is modified with a multi-component; self-assembled monolayer that includes oligonucleotide capture probes specific for each polymorphic site on the test panel and insulator molecules. The cartridge also contains an electrically erasable programmable readonly memory component (EEPROM) that stores information related to the cartridge (e.g., assay identifier, cartridge lot number, and expiration date).

# The eSensor XT-8 Instrument

The eSensor XT-8 instrument was previously cleared for IVD use by the FDA under K073720 and K090901.

The eSensor XT-8 is a clinical multiplex instrument that has a modular design consisting of a base module and one, two, or three cartridge-processing towers containing 8, 16, or 24 cartridge slots, respectively. The cartridge slots operate independently of each other. Any number of cartridges can be loaded at one time, and the remaining slots are available for use while the instrument is running.

The base module controls each processing tower, provides power, and stores and analyzes data. The instrument is designed to be operated solely with the touch screen interface. Entering patient accession numbers and reagent lot numbers can be performed by the bar code scanner or the touch screen.

Each processing tower consists of eight cartridge modules, each containing a cartridge connector, a precision-controlled heater, an air pump, and electronics. The air pumps drive the pump and valve system in the cartridge, eliminating fluid contact between the instrument and the cartridge. The pneumatic pumping enables recirculation of the hybridization solution allowing the target DNA and the signal probes to hybridize with the complementary capture probes on the electrodes. The pump in the cartridge is connected to a pneumatic source from the eSensor XT8 instrument and provides unidirectional pumping of the hybridization mixture through the channel during hybridization. Using this process to circulate the hybridization solution minimizes the instirred boundary layer at the electrode surface and continuously replenishes the volume above the electrode that has been depleted of complementary targets and signal probes.

The XT-8 instrument provides electrochemical detection of bound signal probes by ACV and subsequent data analysis and test report generating functions. All hybridization, ACV and analysis parameters are defined by a scanning protocol loaded into the XT-8 Software, and then specified for use by the EEPROM on each cartridge.

Principle of eSensor Technology: eSensor technology uses a solid-phase electrochemical method for determining the presence of one or more of a defined panel of virus target sequences. Purified DNA/RNA is isolated from the patient specimen according to defined laboratory procedures and the extracted nucleic acid is reverse transcribed and/or amplified using virus specific primers with an RT-PCR enzyme mix. The amplified DNA is converted to single-stranded DNA via exonuclease digestion and is then combined with a signal buffer containing ferrocenelabeled signal probes that are specific for the different viral targets. The mixture of amplified sample and signal buffer is loaded onto a cartridge containing single-stranded oligonucleotide capture probes bound to gold-plated electrodes. The cartridge is inserted into the XT-8 instrument where the single-stranded targets hybridize to the complementary sequences of the capture probes and signal probes, as shown in Figure 1. The presence of each target is determined by voltammetry, which generates specific electrical signals from the ferrocene-labeled signal probe.

The eSensor RVP provides a qualitative result based upon the presence (Positive) or absence Target Not Detected) of the viruses contained in the panel along with the internal MS2 control. Positive and negative results are determined based on he electrical signals generated being either above or below specified signal boundaries, respectively.

![](images/6e14a2f322fcb3b9fe2a46a05827f3ef6b33e9b871f187c69dc8ad3af486d7e6.jpg)  
cletoylf i surface using voltammetry.

Substantial Equivalence Discussion: The eSensor Respiratory Viral Panel (RVP) uses the similar fundamental scientific technologies and has the same intended use as that of the predicate device, the Luminex® xTag® RVP and eSensor XT-8 Instrument. The eSensor XT-8 Instrument described in K078720 (eSensor Warfarin Sensitivity Test) is the identical instrument with a unique Assay Analysis Module (AAM) necessary to support the RVP IVD.

<table><tr><td colspan="2">Table 2: Substantial Equivalence Predicate Comparison:</td></tr><tr><td>Element</td><td>GenMark eSensor RVP Luminex xTag RVP</td></tr><tr><td>Subject Device Intended Use The eSensor® Respiratory Viral Panel (RVP) is a qualitative nucleic acid multiplex</td><td>K081483 The xTAG RVP (Respiratory Viral Panel) is a</td></tr><tr><td rowspan="3"></td><td>in vitro diagnostic test intended for use on the eSensor XT-8™ system for the simultaneous detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from</td><td>qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of multiple respiratory virus nucleic acids in nasopharyngeal swabs from individuals suspected of respiratory tract infections. The following virus types and subtypes are identified using RVP: Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory</td></tr><tr><td>individuals exhibiting signs and symptoms of respiratory infection. The following virus types and subtypes are identified using the eSensor RVP: Influenza A, Influenza A H1 Seasonal Subtype, Influenza A H3 Seasonal Subtype, Influenza A 2009 H1N1 subtype, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Human Metapneumovirus, Human Rhinovirus, Adenovirus species B/E, and Adenovirus species C.</td><td>Syncytial Virus subtype B, Parainfluenza 1, Parainfluenza 2, and Parainfluenza 3 virus, Human Metapneumovirus, Rhinovirus, and Adenovirus. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and laboratory findings. It is recommended that specimens found to be negative after examination using RVP be confirmed by cell culture. Negative results do not</td></tr><tr><td>The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and epidemiological information. Negative results do not preclude respiratory viral infection and should not be used as the</td><td>preclude respiratory viral infection and should not be used as the sole basis for diagnosis, treatment or other management decisions. Positive results do not rule out bacterial infection or co-infection with other organisms. The agent detected may not be the definite cause of disease. The use of additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) and clinical presentation must be taken into consideration in order to obtain the final</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>GenMark eSensor RVPSubject Device</td><td rowspan=1 colspan=1>Luminex xTag RVPK081483</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>sole basis for diagnosis, treatment or otherpatient management decisions. Positiveresults do not rule out bacterial infection, orco-infection with other viruses. The agentdetected may not be the definite cause ofdisease. The use of additional laboratorytesting (e.g. bacterial and viral culture,immunofluorescence and radiography) andclinical presentation must be taken intoconsideration in the final diagnosis ofrespiratory viral infection.Performance characteristics for Influenza Awere established during the 2010/2011influenza season when Influenza A 2009H1N1 and H3N2 were the predominantInfluenza A viruses in circulation. Whenother Influenza A viruses emerge,performance characteristics may vary.If infection with a novel Influenza A virus issuspected based on current clinical andepidemiological screening criteriarecommended by public health authorities,specimens should be collected withappropriate infection control precautions fornovel virulent influenza viruses and sent tostate or local health departments for testing.Viral culture should not be attempted inthese cases unless a BSL 3+ facility isavailable to receive and culture specimens.For prescription use only.</td><td rowspan=1 colspan=1>diagnosis of respiratory infection.Due to seasonal prevalence, performancecharacteristics for Influenza A/H1 were establishedprimarily with retrospective specimens.The RVP assay cannot adequately detectAdenovirus species C, or serotypes 7a and 41.The RVP primers for detection of rhinovirus cross-react with enterovirus. A rhinovirus reactive resultshould be confirmed by an alternate method (e.g.cell culture).Performance characteristics for Influenza A Viruswere established when Influenza AH3 and A/H1were the predominant Influenza A viruses incirculation. When other Influenza A viruses areemerging, performance characteristics may vary. Ifinfections with a novel Influenza A virus issuspected based on current clinical andepidemiological screening criteria recommended bypublic health authorities, specimens should becollected with appropriate infection controlprecautions for novel virulent Influenza viruses andsent to a state or local health department fortesting. Viral culture should not be attempted inthese cases unless a BSL 3+ facility is available toreceive and culture specimens.</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>GenMark eSensor RVPSubject Device</td><td rowspan=1 colspan=1>Luminex xTag RVPK081483</td></tr><tr><td rowspan=1 colspan=1>Organisms Detected</td><td rowspan=1 colspan=1>Influenza A, Influenza A H1 SeasonalSubtype, Influenza A H3 Seasonal Subtype,Influenza A 2009 H1N1 strain, Influenza B,Respiratory Syncytial Virus subtype A,Respiratory Syncytial Virus subtype B,Parainfluenza 1, Parainfluenza 2,Parainfluenza 3, Human Metapneumovirus,Human Rhinovirus, Adenovirus B/E andAdenovirus C</td><td rowspan=1 colspan=1>Influenza A, Influenza A subtype H1, Influenza Asubtype H3, Influenza B, Respiratory SyncytialVirus subtype A, Respiratory Syncytial Virussubtype B, Parainfluenza 1, Parainfluenza 2, andParainfluenza 3 virus, Human Metapneumovirus,Rhinovirus, and Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Specimen type</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Nasopharyngeal swabs (NPS)</td></tr><tr><td rowspan=1 colspan=1>Sample preparation</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Up front sample processing is required to extractnucleic acid</td></tr><tr><td rowspan=1 colspan=1>Assay technology</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>PCR-based system for detecting viral nucleic acidsin clinical specimens targeting unique regions ofvirus RNA/DNA</td></tr><tr><td rowspan=1 colspan=1>Detectiontechnology</td><td rowspan=1 colspan=1>Solid phase electrochemical</td><td rowspan=1 colspan=1>Spectrofluorometry</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>1. Same2. eSensor XT-8</td><td rowspan=1 colspan=1>1. EasyMag® extraction system2. Luminex® 100/200™M instrument</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>Application software and embeddedfirmware (controls hardware functions) onXT-8 in addition to Assay Analysis Module(AAM) for RVP-IVD</td><td rowspan=1 colspan=1>IS or xPONENT software; xTAG Data AnalysisSoftware RVP (US)</td></tr><tr><td rowspan=1 colspan=1>Time to results</td><td rowspan=1 colspan=1>Approximately 6 hours</td><td rowspan=1 colspan=1>Approximately 8 hours</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Test interpretation</td><td rowspan=1 colspan=1>Automated test interpretation and reportgeneration. User can access the raw signalsgenerated by the assay.</td><td rowspan=1 colspan=1>Semi-automated test interpretation. User mustreview all &quot;no call&quot; results to determine cause andretesting strategy.</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Internal control added to each sample. Externalcontrol processed with each batch of samples.</td></tr></table>

# NON-CLINICAL PERFORMANCE DATA

# Limit of Detection

# Limit of Detection (LoD)

The Limit of Detection (LoD) was identified and verified for each viral target of the eSensor RVP using samples prepared from regrown and re-titered viral reference strains as detailed in Table 3. Inial estimations involved serially diluting each viral strain in M5 media.The lowest five dilutions prepared from each target were extracted in triplicate and the assay was performed once for each extract. Verification of the LoD was performed by meeting $9 5 \%$ detection or in at least 19 of the 20 extraction replicates on the selected dilution of each culture. Once the LoD was verified for each viral target in M5, verification of the LoD was also performed with the M4 media. Each viral target was diluted in M4 media down to the LoD concentration. The LoD dilution of each culture was extracted 20 times and the eSensor RVP assay was performed on each extract. The final results summary with the verified LoD concentration in $T C l D _ { 5 0 } / m L$ for both M5 and M4 media is shown in Table 3.

Table 3: LoD Results Summary   

<table><tr><td rowspan=1 colspan=1>Viral Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>StartingTCID5o/mL</td><td rowspan=1 colspan=1>LoD SpikedConc.(TCIDso/mL)</td><td rowspan=1 colspan=1>%Positive</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H1N1Brisbane/59/07</td><td rowspan=1 colspan=1>4.17 x 105$</td><td rowspan=1 colspan=1>4.17 x 10-1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>H1N1Brisbane/59/07</td><td rowspan=1 colspan=1>4.17 x 105</td><td rowspan=1 colspan=1>4.17 x 10-1</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58 x 108$</td><td rowspan=1 colspan=1>1.58 x 103</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58 x 108</td><td rowspan=1 colspan=1>1.58 x 103</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009H1N1</td><td rowspan=1 colspan=1>NY/2009</td><td rowspan=1 colspan=1>1.05 x 10</td><td rowspan=1 colspan=1>1.05 x 10-1</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>3.16 x 10}$</td><td rowspan=1 colspan=1>3.16 x 10</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>4.17 x 10{}$</td><td rowspan=1 colspan=1>4.17 x 10</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.58 x 104</td><td rowspan=1 colspan=1>1.58 x 10^$</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>2.81× 104</td><td rowspan=1 colspan=1>2.81 × 10^2</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>2.81 x 10}$</td><td rowspan=1 colspan=1>2.81 x 10$</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.81 x 10&#x27;</td><td rowspan=1 colspan=1>2.81 x 10&#x27;</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>2.81 x 10</td><td rowspan=1 colspan=1>2.81 x 10 </td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>1.58 x 10</td><td rowspan=1 colspan=1>1.58 x 10</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ADV B/E</td><td rowspan=1 colspan=1>Type 4</td><td rowspan=1 colspan=1>1.58 x-10</td><td rowspan=1 colspan=1>1.58 x 10</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>8.89 x 10&#x27;</td><td rowspan=1 colspan=1>8.89 x 101</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Analytical Reactivity (Inclusivity)

# Analytical Inclusivity Testing

The performance of the eSensor RVP with multiple viral target strains representing temporal and geographic diversity for each claimed viral target was evaluated. Each viral target strain was diluted in M5 transport media to a titer of 3X LoD for the corresponding viral target and extracted in triplicate using the bioMerieux NucliSENS easyMAG System. Following extraction, each

replicate was tested once using the eSensor RVP. In the case that a viral target strain is not detected at 3X LoD, 1000-fold serial dilutions were made from the viral stock and then each dilution was extracted in triplicate and tested using the eSensor RVP.

In cases where unexpected False Negative results were obtained, in silico analysis was performed. Table 4 shows the results.

Table 4: Analytical Reactivity (Inclusivity) Results   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">ConcentrationDetected(TCID5o/ml)</td><td colspan="1" rowspan="1">LoD MultipleDetected</td></tr><tr><td colspan="1" rowspan="10">Flu A H1</td><td colspan="1" rowspan="1">A/NewCaledonia/20/1999</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">FM/1/47H1</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Denver/1/57</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/SolomonIslands/3/2006</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Taiwan/42/06</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">1260</td><td colspan="1" rowspan="1">Flu A- 3x;H1- 3000x*</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">Flu A- 3x;H1- not detected*</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">6372</td><td colspan="1" rowspan="1">Flu A- 3xH1- 15172x*</td></tr><tr><td colspan="1" rowspan="1">A/Fort Monmouth/1/1947(H1N1)</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">Flu A- 3xH1-13x*</td></tr><tr><td colspan="1" rowspan="10">Flu A H3</td><td colspan="1" rowspan="1">A/Aichi/2/68 H3N2</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">AVictoria/3/75</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Aisconsin/67/05</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Perth/16/2009</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Alice (vaccine)AEngland/42/72</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">MRC-2 RecombinantStrain</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Nanchang/933/95</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="3">Flu A 2009H1N1</td><td colspan="1" rowspan="1">A/NY/02/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ANew Jersey/8/76</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/California/7/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="7">′</td><td colspan="1" rowspan="1">A/Swine NY/01/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Swine NY/03/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/09</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">AVirginia/ATCC1/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">AVirginia/ATCC2/2009</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">Flu A- 3x2009 H1N1-6x**</td></tr><tr><td colspan="1" rowspan="1">AVirginia/ATCC3/2009</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">Flu A- 3x2009 H1N1-27x**</td></tr><tr><td colspan="1" rowspan="1">A/lowa/15/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">Flu A- 3x;2009 H1N1-1000x**</td></tr><tr><td colspan="1" rowspan="8">Flu B(Victorialineage exceptwhere noted)</td><td colspan="1" rowspan="1">B/Florida/02/06(Yamagata)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/5/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">RSV A</td><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Long</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="3">RSV B</td><td colspan="1" rowspan="1">9320</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">WV/14617/85</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Wash/18537/62</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">PIV1</td><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">0.084</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">0.084</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">.PIV2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">PIV3</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="4">hMPV</td><td colspan="1" rowspan="1">IA3-2002 G, A1</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">IA14-2003 G, A2</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Peru2-2002 G, B1</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Peru6-2003 G, B2</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">ConcentrationDetected(TCIDso/ml)</td><td colspan="1" rowspan="1">LoD MultipleDetected</td></tr><tr><td colspan="1" rowspan="9">HRV A</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">450x*</td></tr><tr><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">569x#</td></tr><tr><td colspan="1" rowspan="1">A7</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A16</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Detected**</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">A34</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A57</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A77</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">277G</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">100x*</td></tr><tr><td colspan="1" rowspan="7">HRV B</td><td colspan="1" rowspan="1">B3</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">80x</td></tr><tr><td colspan="1" rowspan="1">B14</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">14x</td></tr><tr><td colspan="1" rowspan="1">B17</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">253x</td></tr><tr><td colspan="1" rowspan="1">842</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">FO2-2547</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">89x</td></tr><tr><td colspan="1" rowspan="1">B83</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">127x</td></tr><tr><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">HRV C</td><td colspan="1" rowspan="1">$$</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="9">ADV B</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 7A</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 11 (ot 306523)</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">De Wit Type 14</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Ch.79 Type 16</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 21 (lot 307610)</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Compton Type 34</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Holden Type 35</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Wan Type 50</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="4">ADV C</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">267</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">533</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">Type 5</td><td colspan="1" rowspan="1">533</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">Type 6</td><td colspan="1" rowspan="1">533</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">ADVE</td><td colspan="1" rowspan="1">Type 4</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">3x</td></tr></table>

\* In silico analysis revealed little homology between the strain sequence and the H1 primer sequences. The degree of mis-match to the H1 subtype primer sequences resulted in decreased reactivity to these non-contemporary influenza strains for the H1 subtyping result.

\*\*In silico analysis revealed little homology between the strain sequence and the 2009 H1N1 primer sequences. The degree of mis-match to the 2009 H1N1 subtype primer sequences resulted in decreased reactivity to these influenza strains for the 2009 H1N1 subtyping result.

HRV strain 3, used for LoD determination, had a $T C l D _ { 5 0 } / m |$ of 0.0016. HRV strains 1A FO2-2547, 277G were detected at a higher LoD multiple to the reference strain, respectively with their corresponding $\tau \mathrm { c l D } _ { 5 0 } / m |$ values of 0.9, 0.2, and 0.2. In silico analysis revealed mis-matches between the capture probe sequence and the HRV strains (2, 3 and 1 mis-match(es) respectively).

oi ll  . y n  on  p y

# Supplemental Analytical Inclusivity Testing of Other Influenza Subtypes

Additional analytical inclusivity testing was carrid out wih either live isolates or purified genomc RNA of Influenza A strains.

Table 5: Additional Analytical Reactivity (Inclusivity) Results   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Testing Conc.</td><td rowspan=1 colspan=1>RVP Result</td></tr><tr><td rowspan=1 colspan=1>Flu A H1N2</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>A/NWS/34 (HA) x A/Rockefeller Institute/5/57 (NA),(H1N2), Reassortant NWS-F, RNA</td><td rowspan=1 colspan=1>0.74 ng</td><td rowspan=1 colspan=1>Flu A H1</td></tr><tr><td rowspan=2 colspan=1>Flu A H2N2</td><td rowspan=4 colspan=1>Avian</td><td rowspan=1 colspan=1>A/Japan/305/57, RNA</td><td rowspan=1 colspan=1>1.625 ng</td><td rowspan=1 colspan=1>FluA</td></tr><tr><td rowspan=1 colspan=1>AKorea/426/68 (HA, NA) X A/Puerto Rico/8/34, RNA</td><td rowspan=1 colspan=1>3.12 ng</td><td rowspan=1 colspan=1>Flu A</td></tr><tr><td rowspan=1 colspan=1>Flu A H5N3</td><td rowspan=1 colspan=1>A/duck/Singapore/645/97, Wild Type</td><td rowspan=1 colspan=1>1.26 TCIDso/ml</td><td rowspan=1 colspan=1>Flu A</td></tr><tr><td rowspan=1 colspan=1>Flu A H10N7</td><td rowspan=1 colspan=1>A/chicken/Germany/N/49</td><td rowspan=1 colspan=1>1.26 TCIDso/ml</td><td rowspan=1 colspan=1>Flu A</td></tr></table>

Simulated Reactivity Information for Influenza Strains of Human. Swine, and Avian Oriqin For any strains that were not available for testing with the eSensor RVP, such as Flu A H5 and Flu A H7 strains in silico analysis was performed. The eSensor RVP primers for Flu A, Flu A H1, Flu A 2009 H1N1 and Flu A H3 were aligned to the GenBank® sequences of the reactivity strains. Exclusivity was predicted based on the number and location of mismatches between assay primers and available strain sequences.

Simulated reactivity of the eSensor RVP with influenza strains was generated using a bioinformatics approach. Assay primer sequences, signal probes and capture probes were aligned with GenBank sequences corresponding to the appropriate gene targets and reactivity were predicated based on the number and location of mismatches in the targeted region shown in Table 6.

Table 6: Supplemental Reactivity of eSensor RVP Influenza A with Human, Swine, and Avian Influenza Strains   

<table><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBankID</td><td colspan="1" rowspan="1">SimulatedRVPReactivityResult</td></tr><tr><td colspan="1" rowspan="4">H2N2</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">A/Albany/20/1957(H2N2)</td><td colspan="1" rowspan="1">CY022014</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="3">Avian</td><td colspan="1" rowspan="1">A/chicken/New York/13828-3/1995(H2N2)</td><td colspan="1" rowspan="1">CY014822</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/Japan/305/1957(H2N2)</td><td colspan="1" rowspan="1">CY014977</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">A/Korea/426/1968(H2N2)</td><td colspan="1" rowspan="1">CY031596</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">H4N6</td><td colspan="1" rowspan="2">Avian</td><td colspan="1" rowspan="1">A/blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td><td colspan="1" rowspan="1">CY063978</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/peregrine falcon/Aomori/7/2011</td><td colspan="1" rowspan="1">AB629716</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBankID</td><td colspan="1" rowspan="1">SimulatedVPReactivityResult</td></tr><tr><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">A/chicken/West Bengal/239022/2010</td><td colspan="1" rowspan="1">CY061305</td><td colspan="1" rowspan="1">FIUA</td></tr><tr><td colspan="1" rowspan="1">A/chicken/West Bengal/193936/2009</td><td colspan="1" rowspan="1">GU272009</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Hunan/1/2009</td><td colspan="1" rowspan="1">HM172150</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Hunan/8/2008</td><td colspan="1" rowspan="1">GU182162</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/West Bengal/106181/2008</td><td colspan="1" rowspan="1">GU083632</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Primorsky/85/2008</td><td colspan="1" rowspan="1">FJ654298</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="8">Avian</td><td colspan="1" rowspan="1">A/chicken/West Bengal/82613/2008</td><td colspan="1" rowspan="1">GU083648</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/duck/France/080036/2008</td><td colspan="1" rowspan="1">CY046185</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/duck/Vietnam/G12/2008</td><td colspan="1" rowspan="1">AB593450</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="10">H5N1</td><td colspan="1" rowspan="1">A/chicken/Thailand/PC-340/2008</td><td colspan="1" rowspan="1">EU620664</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/great egret/Hong Kong/807/2008</td><td colspan="1" rowspan="1">CY036240</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/rook/Rostov-on-Don/26/2007(H5N1)</td><td colspan="1" rowspan="1">EU814504</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/turkey/VA/505477-18/2007(H5N1)</td><td colspan="1" rowspan="1">GU186510</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Bangladesh/1151-10/2010(H5N1)</td><td colspan="1" rowspan="1">HQ156766</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="4">Human</td><td colspan="1" rowspan="1">A/Bangladesh/3233/2011</td><td colspan="1" rowspan="1">CY088772</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">ACambodia/R0405050/2007(H5N1)</td><td colspan="1" rowspan="1">HQ200572</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/S1211394/2008</td><td colspan="1" rowspan="1">HQ200597</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/486/97(H5N1)</td><td colspan="1" rowspan="1">AF255368</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Swine</td><td colspan="1" rowspan="1">A/swine/East Java/UT6010/2007(H5N1)</td><td colspan="1" rowspan="1">HM440124</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="11">H5N2</td><td colspan="1" rowspan="11">Avian</td><td colspan="1" rowspan="1">Aduck/Pennsylvania/10218/1984(H5N2)</td><td colspan="1" rowspan="1">AB286120</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/American black duck/llinois/08OS2688/2008</td><td colspan="1" rowspan="1">CY079453</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">A/American green-winged teal/California/HKWF609/2007</td><td colspan="1" rowspan="1">CY033447</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/Canada goose/New York/475813-2/2007</td><td colspan="1" rowspan="1">GQ923358</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Ablue-winged teal/Saskatchewan/22542/2007</td><td colspan="1" rowspan="1">CY047705</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Taiwan/A703-1/2008</td><td colspan="1" rowspan="1">AB507267</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">A/duck/France/080032/2008</td><td colspan="1" rowspan="1">CY046177</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/duck/New York/481172/2007</td><td colspan="1" rowspan="1">GQ117202</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/gadwall/Altai/1202/2007</td><td colspan="1" rowspan="1">CY049759</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">A/mallard/Louisiana/476670-4/2007</td><td colspan="1" rowspan="1">GQ923390</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/waterfowl/Colorado/476466-2/2007</td><td colspan="1" rowspan="1">GQ923374</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">H5N3</td><td colspan="1" rowspan="3">Avian</td><td colspan="1" rowspan="1">A/duck/Singapore/F119/3/1997(H5N3)</td><td colspan="1" rowspan="1">GU052803</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">H6N1</td><td colspan="1" rowspan="1">Aduck/PA/486/1969(H6N1)</td><td colspan="1" rowspan="1">EU743287</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">H6N2</td><td colspan="1" rowspan="1">A/mallard/Czech Republic/15902-17K/2009(H6N2)</td><td colspan="1" rowspan="1">HQ244433</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="5">H7N2</td><td colspan="1" rowspan="5">Avian</td><td colspan="1" rowspan="1">A/chicken/Hebei/1/2002</td><td colspan="1" rowspan="1">AY724263</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/PA/149092-1/02</td><td colspan="1" rowspan="1">AY241609</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/NJ/294508-12/2004</td><td colspan="1" rowspan="1">EU743254</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/New York/23165-6/2005</td><td colspan="1" rowspan="1">CY031077</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/muscovy duck/New York/23165-13/2005</td><td colspan="1" rowspan="1">CY033226</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBankID</td><td colspan="1" rowspan="1">SimulatedRVPReactivityResult</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">−</td><td colspan="1" rowspan="1">A/muscovy duck/New York/87493-3/2005</td><td colspan="1" rowspan="1">CY034791</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/mallard/Netherlands/29/2006</td><td colspan="1" rowspan="1">CY043833</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/northern shoveler/California/JN1447/2007</td><td colspan="1" rowspan="1">CY076873</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">H7N2</td><td colspan="1" rowspan="2">Human</td><td colspan="1" rowspan="1">A/New York/107/2003(H7N2)</td><td colspan="1" rowspan="1">EU587373</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">H7N3</td><td colspan="1" rowspan="1">A/Canada/rv504/2004(H7N3)</td><td colspan="1" rowspan="1">CY015007</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="8">H7N7.</td><td colspan="1" rowspan="7">Avian.</td><td colspan="1" rowspan="1">A/American green-winged teal/Mississippi/09OS046/2009</td><td colspan="1" rowspan="1">CY079309</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Achicken/Germany/R28/03</td><td colspan="1" rowspan="1">AJ619676</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Netherlands/1/03</td><td colspan="1" rowspan="1">AY340091</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/mallard/California/HKWF1971/2007</td><td colspan="1" rowspan="1">CY033383</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/mallard/Korea/GH171/2007</td><td colspan="1" rowspan="1">FJ959087</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">A/mute swan/Hungary/5973/2007</td><td colspan="1" rowspan="1">GQ240816</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">A/northern shoveler/Mississippi/ 09OS643/2009</td><td colspan="1" rowspan="1">CY079413</td><td colspan="1" rowspan="1">FiuA</td></tr><tr><td colspan="1" rowspan="2">Human</td><td colspan="1" rowspan="1">A/Netherlands/219/03(H7N7)</td><td colspan="1" rowspan="1">AY340089</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="2">H9N2</td><td colspan="1" rowspan="1">A/Hong Kong/1073/99(H9N2)</td><td colspan="1" rowspan="1">AJ278647</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="2">Avian</td><td colspan="1" rowspan="1">A/turkeyWisconsin/1/1966(H9N2)</td><td colspan="1" rowspan="1">CY014664</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">H11N9</td><td colspan="1" rowspan="1">A/duck/Memphis/546/1974(H11N9)</td><td colspan="1" rowspan="1">GQ257441</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="4">H1N1</td><td colspan="1" rowspan="2">Swine</td><td colspan="1" rowspan="2">Aswine/Wisconsin/1/1971 (H1N1)</td><td colspan="1" rowspan="1">CY022414</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY022417</td><td colspan="1" rowspan="1">Flu A H1</td></tr><tr><td colspan="1" rowspan="6">Human</td><td colspan="1" rowspan="2">AVCalifornia/UR06-0393/2007(H1N1)</td><td colspan="1" rowspan="1">CY026540</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY026539</td><td colspan="1" rowspan="1">Flu A H1</td></tr><tr><td colspan="1" rowspan="2">H1N2</td><td colspan="1" rowspan="2">A/New York/297/2003(H1N2)</td><td colspan="1" rowspan="1">CY002665</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY002664</td><td colspan="1" rowspan="1">Flu A H1</td></tr><tr><td colspan="1" rowspan="2">H1N1(209)</td><td colspan="1" rowspan="2">A/Aalborg/INS133/2009(H1N1)</td><td colspan="1" rowspan="1">CY063607</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY063606</td><td colspan="1" rowspan="1">2009 H1N1</td></tr><tr><td colspan="1" rowspan="4">H1N2</td><td colspan="1" rowspan="4">Swine</td><td colspan="1" rowspan="2">A/swine/Hong Kong/NS857/2001 (H1N2)</td><td colspan="1" rowspan="1">GQ229350</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">GQ229348</td><td colspan="1" rowspan="1">Flu A H1</td></tr><tr><td colspan="1" rowspan="2">A/swine/Sweden/1021/2009(H1N2)</td><td colspan="1" rowspan="1">GQ495135</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">GQ495132</td><td colspan="1" rowspan="1">Flu A H1</td></tr><tr><td colspan="1" rowspan="2">H3N1</td><td colspan="1" rowspan="2">Avian</td><td colspan="1" rowspan="2">Ablue-winged teal/ALB/452/1983(H3N1)</td><td colspan="1" rowspan="1">CY004635</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY005940</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="8">H3N2</td><td colspan="1" rowspan="6">Swine</td><td colspan="1" rowspan="2">A/swine/NY/A01104005/2011(H3N2)*</td><td colspan="1" rowspan="1">N/A*</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">JN940422</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">A/Maine/06/2011(H3N2)</td><td colspan="1" rowspan="1">JN866181</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">JN866186</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">A/Indiana/08/2011(H3N2)</td><td colspan="1" rowspan="1">JN655558</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">JN638733</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">Human</td><td colspan="1" rowspan="2">A/Boston/38/2008(H3N2)</td><td colspan="1" rowspan="1">CY044581</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY044580</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="10">Avian</td><td colspan="1" rowspan="2">AAmerican black duck/North Carolina/675-075/2004(H3N2)</td><td colspan="1" rowspan="1">GU051136</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">GU051135</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">H3N5</td><td colspan="1" rowspan="2">Amallard/Netherlands/2/1999(H3N5)</td><td colspan="1" rowspan="1">CY060264</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY060261</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">H3N6</td><td colspan="1" rowspan="2">A/American black duck/New Brunswick/25182/2007(H3N6)</td><td colspan="1" rowspan="1">CY047697</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY047696</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">H3N7</td><td colspan="1" rowspan="2">Anorthern shoveler/California/HKWF1367/2007(H3N7)</td><td colspan="1" rowspan="1">CY033375</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">CY033372</td><td colspan="1" rowspan="1">Flu A H3</td></tr><tr><td colspan="1" rowspan="2">H3N8</td><td colspan="1" rowspan="2">A/American black duck/Washington/699/1978(H3N8)</td><td colspan="1" rowspan="1">GU052300</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">GU052299</td><td colspan="1" rowspan="1">Flu A H3</td></tr></table>

N   DC o conerved Matrix Protein) sequence from 2009 H1Therefore Fu A should also be able to be detecte

# Reproducibility

# Multisite Reproducibility

Multisite reproducibility of the eSensor RVP was performed to evaluate the major sources of variability such as lot-to-lot, extraction-to-extraction, site/instrument-to-site/instrument,day-to-day and operator/run-to-operator/run. Reproducibility testing occurred at three sites, utilizing a panel of samples containing viral material from culture derived isolates in media, simulating NPS specimen. Each of the selected panel targets was prepared at concentrations representing the following three viral load levels: Moderate Positive ( $100 \%$ positive, 3x LoD), Low Positive $( 9 5 \%$ positive, 1x LoD), and Negative ( $100 \%$ negative). Each simulated sample was divided into aliquots, blinded, and stored frozen $( - 7 0 ^ { \circ } C )$ prior to testing. Each site received three lots of RVP Cartridges/Reagents, a set of samples for two operators and one XT-8 instrument. All samples were independently extracted using the bioMérieux easyMAG extraction system. Every analyte at each concentration was tested a total of 108 times (two operators at three sites, each testing three replicates on six testing days). Each lot of RVP Cartridges/Reagents was used in 36 tests per analyte/concentration. Summary results for each tested analyte are summarized below.

Table 7: Summary of Influenza A Calls (H3N2)   

<table><tr><td colspan="1" rowspan="1">Flu AConcentration</td><td colspan="1" rowspan="1">Slte</td><td colspan="1" rowspan="1"># Positive</td><td colspan="1" rowspan="1"># Negative,</td><td colspan="1" rowspan="1">% AgreommentResults</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Mean(nA)</td><td colspan="1" rowspan="1">Std Dev</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="4">MOD POS(3x LD)1.3 TCIDso/ml</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">243.0</td><td colspan="1" rowspan="1">23.6</td><td colspan="1" rowspan="1">9.7</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">246.4</td><td colspan="1" rowspan="1">29.7</td><td colspan="1" rowspan="1">12.0</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">235.0</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">13.9</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">0/108</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.6%-100%</td><td colspan="1" rowspan="1">241.5</td><td colspan="1" rowspan="1">29.0</td><td colspan="1" rowspan="1">12.0</td></tr><tr><td colspan="1" rowspan="4">LOW POS(1x LoD)0.4 TCIDso/ml</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">248.3</td><td colspan="1" rowspan="1">28.6</td><td colspan="1" rowspan="1">11.5</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">244.7</td><td colspan="1" rowspan="1">26.4</td><td colspan="1" rowspan="1">10.8</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">232.8</td><td colspan="1" rowspan="1">23.2</td><td colspan="1" rowspan="1">10.0</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1"> 0/108</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.6%-100%</td><td colspan="1" rowspan="1">242.0</td><td colspan="1" rowspan="1">26.7</td><td colspan="1" rowspan="1">11.1</td></tr><tr><td colspan="1" rowspan="1">Flu AConcentratlon</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1"># Positive</td><td colspan="1" rowspan="1"># Negative</td><td colspan="1" rowspan="1">% Agreementwith ExpectedResults</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Mean(nA)</td><td colspan="1" rowspan="1">Std Dev</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">3/288</td><td colspan="1" rowspan="1">285/288</td><td colspan="1" rowspan="1">99.0%</td><td colspan="1" rowspan="1">97.0%-99.8%</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">2.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">1/288</td><td colspan="1" rowspan="1">287/288</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.1%-100%</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">2/288</td><td colspan="1" rowspan="1">286/288</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">97.5%-99.9%</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">6/864</td><td colspan="1" rowspan="1">858/864</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">98.5%-99.7%</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">N/A</td></tr></table>

Table 8: Summary of Influenza A H3 Calls   

<table><tr><td rowspan=1 colspan=1>Flu A H3Concentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>% A mendResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>% cv</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD4.7 X 103TCIDso/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>86.5</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>27.7</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>77.4</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>34.5</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>86.0</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>83.3</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>32.7</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD) 1.6 x 103TCIDso/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>81.5</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>32.0</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>68.2</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>43.4</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>84.9</td><td rowspan=1 colspan=1>21.3</td><td rowspan=1 colspan=1>25.1</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>78.2</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>34.1</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%100%</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>1/288</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1%-100%</td><td rowspan=1 colspan=1>.0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/864</td><td rowspan=1 colspan=1>863/864</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.4%-100%</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Table 9: Summary of Adenovirus B/E Calls   

<table><tr><td rowspan=1 colspan=1>Con aon</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1># Negative</td><td rowspan=1 colspan=1>%A ened dResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD)47.4 TCIDso/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>109.1</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>10.0</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>102.6</td><td rowspan=1 colspan=1>11,5</td><td rowspan=1 colspan=1>11.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>102.2</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>104.7</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD15.8 TCIDso/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>92.7</td><td rowspan=1 colspan=1>11.4</td><td rowspan=1 colspan=1>12.3</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>89.9</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>16.9</td><td rowspan=1 colspan=1>20.0</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>89.1</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>15.3</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site.1</td><td rowspan=1 colspan=1>1/288</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1%-100%</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>.288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>·1.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%·100%</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>1/864</td><td rowspan=1 colspan=1>863/864</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.4%-100%</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Table 10: Summary of hMPV Calls   

<table><tr><td rowspan=1 colspan=1>hMPVConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1># Positive</td><td rowspan=1 colspan=1># Negative</td><td rowspan=1 colspan=1>% Agreementith pectedResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD)13 TCID5/</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>91.2</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>28.6</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>40.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>100.4</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>22.6</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>30.9</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD)4 TCID5o/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>56.4</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>53.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>1/36</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>85.5%-99.9%</td><td rowspan=1 colspan=1>51.0</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>61.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>63.8</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>44.0</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>107/108</td><td rowspan=1 colspan=1>1/108</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>94.9%-100%</td><td rowspan=1 colspan=1>57.1</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>52.5</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>8/288</td><td rowspan=1 colspan=1>280/288</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>94.6%-98.8%</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>8/864</td><td rowspan=1 colspan=1>856/864</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>98.2%-99.6%</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>NIA</td></tr></table>

Table 11: Summary of PIV 3 Calls   

<table><tr><td rowspan=1 colspan=1>PIV3Concentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1># Positive</td><td rowspan=1 colspan=1># Negative</td><td rowspan=1 colspan=1>me dResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD84 TCIDso/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>178.1</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>15.4</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>193.8</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>15.4</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>160.8</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>17.0</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>177.6</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>17.5</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoDD)28 TCIDso/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>139.0</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>25.1</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>162.4</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>17.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>126.9</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>30.7</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>142.8</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/864</td><td rowspan=1 colspan=1>864/864</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Table 12: Summary of RSV A Calls   

<table><tr><td colspan="1" rowspan="1">RSV AConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1"># Positive</td><td colspan="1" rowspan="1"># Negative</td><td colspan="1" rowspan="1">% Agreementwithh ExpectedResults</td><td colspan="1" rowspan="1">95% Cl</td><td colspan="1" rowspan="1">Mean(nA)</td><td colspan="1" rowspan="1">Std Dev</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="4">MOD POS(3x LoD)8.4 TCIDso/ml</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">166.3</td><td colspan="1" rowspan="1">19.2</td><td colspan="1" rowspan="1">11.5</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">156.7</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">20.2</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">156.4</td><td colspan="1" rowspan="1">22.9</td><td colspan="1" rowspan="1">14.7</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">0/108</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.6%-100%</td><td colspan="1" rowspan="1">159.8</td><td colspan="1" rowspan="1">25.3</td><td colspan="1" rowspan="1">15.8</td></tr><tr><td colspan="1" rowspan="4">LOW POS(1xxxLoD)2.8 TCIDso/ml</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">146.6</td><td colspan="1" rowspan="1">22.7</td><td colspan="1" rowspan="1">15.5</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">124.6</td><td colspan="1" rowspan="1">41.0</td><td colspan="1" rowspan="1">32.9</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">1/36</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">85.5%-99.9%</td><td colspan="1" rowspan="1">128.2</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">26.1</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">1/108</td><td colspan="1" rowspan="1">99.1%</td><td colspan="1" rowspan="1">94.9%-100%</td><td colspan="1" rowspan="1">133.1</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">25.8</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">4/288</td><td colspan="1" rowspan="1">284/288</td><td colspan="1" rowspan="1">98.6%</td><td colspan="1" rowspan="1">96.5%-99.6%</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">0/288</td><td colspan="1" rowspan="1">288/288</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.7%-100%</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">RSV AConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1"># Positive</td><td colspan="1" rowspan="1"># Negative</td><td colspan="1" rowspan="1">% Agreementwith ExpectedResuits</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Mean(A)</td><td colspan="1" rowspan="1">Std Dev</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">0/288</td><td colspan="1" rowspan="1">288/288</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.7%-100%</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">4/864</td><td colspan="1" rowspan="1">860/864</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">98.8%-99.9%.</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">N/A</td></tr></table>

# Testing of Dual Infection Samples

# Clinically Relevant Co-Infections

An internal co-infection study was performed to determine the capability of the eSensor RVP to detect clinically relevant dual co-infections in patient samples. Nine clinically relevant coinfections were evaluated in this study. Dual co-infections were prepared by using the representative viral cultures at two different concentrations - Virus A at 1x LoD and Virus B at 10,000x LoD, as well as Virus A at $1 0 , 0 0 0 \times$ LoD and Virus B at $\boldsymbol { \mathsf { 1 x } }$ LoD. Relevant medical literature was sourced for selection of viral mix composition of common or expected co-infections.

The table below summarizes the $\mathtt { T C l D } _ { 5 0 } / \ m \mathrm { m l }$ and LoD multiple detected in each viral co-infection.

Table 13: Dual Infection Reproducibility Summary Results   

<table><tr><td rowspan=2 colspan=1>Viral Co-Infection(Virus 1/Virus 2)</td><td rowspan=1 colspan=2>Virus 1 Detected</td><td rowspan=1 colspan=2>Virus 2 Detected</td></tr><tr><td rowspan=1 colspan=1>LoDMultiple</td><td rowspan=1 colspan=1>TCIDso/ml</td><td rowspan=1 colspan=1>LoDMultiple</td><td rowspan=1 colspan=1>TCIDso/ml</td></tr><tr><td rowspan=1 colspan=1>H3-RSV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>1.58 x 10^3$</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>2.81 x 10^4</td></tr><tr><td rowspan=1 colspan=1>RSV-H3</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>8.43 x10^}$</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.58 x 10&#x27;</td></tr><tr><td rowspan=1 colspan=1>H3-FLUB</td><td rowspan=1 colspan=1>1× LoD</td><td rowspan=1 colspan=1>1.58 x 10</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>3.16 x 10^$</td></tr><tr><td rowspan=1 colspan=1>FLUB-H3</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>3.16 x 10</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.58 x 10&#x27;</td></tr><tr><td rowspan=1 colspan=1>H1N1-HRV</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>3.15 x 10</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.58 x 10</td></tr><tr><td rowspan=1 colspan=1>HRV-H1N1</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>1.58 x 10-3$</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.05 x 10^$</td></tr><tr><td rowspan=1 colspan=1>H1N1-PIV3</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>1.05 x 10</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>2.81 x 10r}$</td></tr><tr><td rowspan=1 colspan=1>PIV3-H1N1</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>2.81 x 10</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.05 x 10^{}$</td></tr><tr><td rowspan=1 colspan=1>H1N1-RSV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>1.05 x 101</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>2.81 x 104</td></tr><tr><td rowspan=1 colspan=1>RSV-H1N1</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>8.43 x 10 $</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.05 x 10$</td></tr><tr><td rowspan=1 colspan=1>RSV-ADV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>2.81 x 10 $</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>8.89 x 10^{2</td></tr><tr><td rowspan=1 colspan=1>ADV-RSV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>8.89 x 102</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>2:81x 104</td></tr><tr><td rowspan=1 colspan=1>HMPV-RSV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>4.17 x 10</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>2.81 x 104</td></tr><tr><td rowspan=1 colspan=1>RSV-HMPV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>2.81 x 10}$</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>4.17 x 10^4</td></tr><tr><td rowspan=1 colspan=1>HMPV-ADV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>4.17 x 100$</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>8.89 x 102</td></tr><tr><td rowspan=1 colspan=1>ADV-HMPV</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>8.89 x 10^2</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>4.17 x 104</td></tr><tr><td rowspan=1 colspan=1>HRV-RSV</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>4.74× 10^3</td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>2.81 x 104</td></tr><tr><td rowspan=1 colspan=1>RSV-HRV</td><td rowspan=1 colspan=1>1x LD</td><td rowspan=1 colspan=1>2.81 x 10 </td><td rowspan=1 colspan=1>10,000x LoD</td><td rowspan=1 colspan=1>1.58 x 10</td></tr></table>

# Interference

# Interfering Substances

Potentially interfering substances were selected based on the fact that they could pre-exist in the specimen (e.g. blood, nasal secretions or mucus, and nasal and throat medications used to relieve congestion, nasal dryness, irritation, or asthma and allergy symptoms) as well as those that could be introduced during specimen collection and preparation.Each potentially interfering substance was tested individually with the exception of Luffa opperculata, Galphimia glauca, Histaminum hydrochloricum, and Sulfur, which were tested together as Zicam® Allergy Relief Nasal spray and Oxymetazoline and Menthol, which were tested together as Afrin® No Drip Severe Congestion nasal spray, thereby bringing the total to 21 potentially interfering test combinations. Viral samples representative of the 14 viral targets on the eSensor RVP were obtained from commercially available cultured cell lines as indicated in Table 14. Seven viral mixes were made, each containing unique viral targets. Viral mixes were added to each potentially interfering substance resulting in a final testing concentration of 3X LoD for each analyte. Each was extracted in triplicate with each extract tested once with the eSensor RVP. Twenty-four (24) potentially interfering substances were tested in this study with 21 combinations. Additionally, nine potentially interfering microorganisms (viral and bacterial) were also tested in the same manner as described above. The microorganisms and their testing concentrations are listed in Table 14. All substances and microorganisms tested for interference were shown to be compatible with the eSensor RVP. No potentially interfering substance or microorganism was shown to inhibit the eSensor RVP at all tested concentrations®.

Table 14: Potentially Interfering Substances   

<table><tr><td colspan="1" rowspan="1">Potentially InterferingSubstance</td><td colspan="1" rowspan="1">ActiveIngredient</td><td colspan="1" rowspan="1">SubstanceForm</td><td colspan="1" rowspan="1">TestedConcentration</td></tr><tr><td colspan="1" rowspan="1">Sample Matrix</td><td colspan="1" rowspan="1">Control for no interfering substance</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Viral transport medium</td><td colspan="1" rowspan="1">Becton Dickinson VTM</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="2">Blood (human)</td><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">2% v/V</td></tr><tr><td colspan="1" rowspan="1">Human gDNA</td><td colspan="1" rowspan="1">50 ng/rxn</td><td colspan="1" rowspan="1">50 ng/rxn</td></tr><tr><td colspan="1" rowspan="2">Throat lozenges, oral anestheticand analgesic</td><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">30% w/v</td></tr><tr><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">Nasal Spray</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Mucin: bovine submaxillary gland,type l-S</td><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1% w/V</td></tr><tr><td colspan="1" rowspan="3">Nasal spraysor drops</td><td colspan="1" rowspan="1">Phenylephrine (Neo-Synephrine)</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1.5% v/V</td></tr><tr><td colspan="1" rowspan="1">Oxymetazoline* (also contains BenzalkoniumChloride, Menthol, Eucalyptol, Camphor, benzylalcohol and phosphate buffers)</td><td colspan="1" rowspan="1">Nasal Spray</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Sodium chloride</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">0.8% w/v</td></tr><tr><td colspan="1" rowspan="1">Antibacterial, systemic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">5% W/N</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, nasal ointment</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">2% w/V</td></tr><tr><td colspan="1" rowspan="6">Nasal corticosteroids</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1.5% w/v</td></tr><tr><td colspan="1" rowspan="1">Budesonide (Pulmicort)</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">1.5% w/V</td></tr><tr><td colspan="1" rowspan="1">Fluticasone (Flonase)</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">3% w/v</td></tr><tr><td colspan="1" rowspan="2">Nasal gel</td><td colspan="1" rowspan="1">Luffa opperculata**</td><td colspan="1" rowspan="1">Nasal Gel</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Sulfur**</td><td colspan="1" rowspan="1">Nasal Gel</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="1" rowspan="2">Homeopathic allergy relief medicine</td><td colspan="1" rowspan="1">Galphimia glauca**</td><td colspan="1" rowspan="1">Nasal Gel</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Histaminum hydrochloricum**</td><td colspan="1" rowspan="1">Nasal Gel</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">FluMist</td><td colspan="1" rowspan="1">Live intranasal influenza virus vaccine</td><td colspan="1" rowspan="1">Liquid</td><td colspan="1" rowspan="1">0.5%-1% v/V</td></tr><tr><td colspan="1" rowspan="2">Anti-viral drugs</td><td colspan="1" rowspan="1">Zanamivir (Relenza)</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">550 ng/ml</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir (Tamiflu)</td><td colspan="1" rowspan="1">Dry</td><td colspan="1" rowspan="1">142 ng/ml</td></tr><tr><td colspan="1" rowspan="2">Virus</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Culture</td><td colspan="1" rowspan="2">1 x 103 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="7">Bacteria</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Culture</td><td colspan="1" rowspan="7">1 x 10 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">Culture</td></tr></table>

\*Tested together (Afrin No Drip Severe Congestion nasal spray) Tested together Zicam Allergy Rel) l resulted in positive calls for Flu A, Flu A H3, Flu A 2009 H1N1 and Flu B. This was due to the live attenuated influenza virus present in the vaccine. Testing of FluMist at $1 \%$ /) resulting in an inhibition in the detection of hMPV.FluMist did not inhibit the detection of hMPV when tested at $0 . 5 \%$ (v/v).

# Cross-Reactivity

# Cross-Reactivity Evaluation for Viruses Detected by the eSensor RVP

Cross-reactivity of each viral target (14 viral targets) was evaluated at high concentrations with the eSensor RVP by making three serial dilutions of viral reference strains with viral transport media (Remel M5) at 10,000x, 1000x and $1 0 0 \times$ the LoD. The titer of each virus dilution and corresponding LoD values were determined and provided in the table below. Cross-reactivity was not observed with any of the RVP viral targets at the concentrations tested. Table 15 summarizes the cross-reactivity results.

Table 15: Within Panel Cross-Reactivity Final Results   

<table><tr><td colspan="1" rowspan="1">. Viral Taget</td><td colspan="1" rowspan="1">Straln</td><td colspan="1" rowspan="1">LoDConcentration(TCIDs/mL)</td><td colspan="1" rowspan="1">Highest TestConcentration(TCIDso/mL)</td><td colspan="1" rowspan="1">Highest Multipleof Lo Tested</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">FlUA</td><td colspan="1" rowspan="1">H1N1 Brisbane/59/07</td><td colspan="1" rowspan="1">4.17× 101</td><td colspan="1" rowspan="1">4.17 × 103</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">FluAH1</td><td colspan="1" rowspan="1">H1N1 Brisbane/59/07</td><td colspan="1" rowspan="1">4.17×10-</td><td colspan="1" rowspan="1">4.17× 10^}$</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">FIUA</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">1.58 × 10}$</td><td colspan="1" rowspan="1">1.58 × 10</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Flu A H3</td><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">1.58 X 10}}$</td><td colspan="1" rowspan="1">1.58× 10{</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Flu A 2009 H1N1</td><td colspan="1" rowspan="1">NY/2009</td><td colspan="1" rowspan="1">1.05 x 10'</td><td colspan="1" rowspan="1">1.05× 10}$</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">Florida/02/06</td><td colspan="1" rowspan="1">3.16 × 10^</td><td colspan="1" rowspan="1">3.16× 103}}</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">4.17×100}$</td><td colspan="1" rowspan="1">4.17×104</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.58 x×10}$</td><td colspan="1" rowspan="1">1.58× 10'</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">2.81x 10^2</td><td colspan="1" rowspan="1">2.81X 10^2}$</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Viral Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LODConcentration(TCIDso/mL)</td><td colspan="1" rowspan="1">Highest TestConcentration(TSso/mL)</td><td colspan="1" rowspan="1">Highest Multipleof LoD Tested</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">$2.81x10}$</td><td colspan="1" rowspan="1">2.81x 104</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">C 243</td><td colspan="1" rowspan="1">2.81x 101</td><td colspan="1" rowspan="1">2.81× 10}$</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">2.81x 10}$</td><td colspan="1" rowspan="1">2.81× 104</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">9320</td><td colspan="1" rowspan="1">1.58  10}$</td><td colspan="1" rowspan="1">1.58 × 104</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">Type 7Type 4</td><td colspan="1" rowspan="1">8.89 x 10^21.58 × 101</td><td colspan="1" rowspan="1">1.58 x 10}$</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">ADVC</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">8.89 x 10'</td><td colspan="1" rowspan="1">8.89 x 10}$</td><td colspan="1" rowspan="1">10,000x</td><td colspan="1" rowspan="1">Not Observed</td></tr></table>

Cross-Reactivity with Other Respiratory Viruses Not Targeted by the eSensor RVP Cross-reactivity with 5 respiratory viruses known to circulate with low frequency in the general population was assessed.All viral strains were diluted in M5 transport media to a titer of $\overline { { 1 0 ^ { 5 } } }$ PFU/mL and extracted using the bioMérieux easyMAG extraction method in triplicate. Following extraction, each replicate was tested once in the RVP assay.

Table 16: Cross-Reactivity Results of Rare Respiratory Virus   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test Concentrations</td><td rowspan=1 colspan=1>Cross-Reactivity Results</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>2.92 x 10^3 PFU/mL</td><td rowspan=1 colspan=1>Not Observed</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43*</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>5.96 x 10* PFU/mL</td><td rowspan=1 colspan=1>Not Observed</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.36 x 10 PFU/mL</td><td rowspan=1 colspan=1>Not Observed</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63**</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>9.89 x 104 PFU/mL</td><td rowspan=1 colspan=1>Not Observed</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical Isolate</td><td rowspan=1 colspan=1>N/A\}$</td><td rowspan=1 colspan=1>Not Observed</td></tr></table>

\*OC43 had one replicate fail the IC control at high $( 1 0 ^ { 5 } )$ concentration. \*NL63 was tested at the highest concentration available $. 9 . 8 9 x$ 10 PFU/mL. The Coronavirus HKU1 sample was a clinical isolate identified during the method comparison study. The method used was qualitative so no copy information was available.

Cross-Reactivity with 17 additional viruses that are not targets of the eSensor RVP were also assessed (Table 17.) All viral strains were diluted in M5 transport media to a titer of $\overline { { 1 0 ^ { 5 } } }$ PFU/mL and extracted using the bioMérieux easyMAG extraction method in triplicate reactions.

Table 17: Cross- Reactivity Results of with other Viruses   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Test Concentrations</td><td colspan="1" rowspan="1">Cross-Reactivityesults</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 18 (A)</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">2.37 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 9 (D)</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">4.63 x 10° PFU/mL</td><td colspan="1" rowspan="1">ADV C False Positive*</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 41 (F)</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">8.05 x 10° PFU/mL</td><td colspan="1" rowspan="1">ADV C False Positive*</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">Zeptometrix 0810047CF</td><td colspan="1" rowspan="1">2.92 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A10</td><td colspan="1" rowspan="1">Zeptometrix 0810106CF</td><td colspan="1" rowspan="1">1.72 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">Zeptometrix 0810017CF</td><td colspan="1" rowspan="1">2.21 X 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Echovirus E6</td><td colspan="1" rowspan="1">Zeptometrix 0810076CF</td><td colspan="1" rowspan="1">7.16 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B2</td><td colspan="1" rowspan="1">ATCC VR-29</td><td colspan="1" rowspan="1">6.22 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">Zeptometrix 0810074CF</td><td colspan="1" rowspan="1">1.06 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">Zeptometrix 0810075CF</td><td colspan="1" rowspan="1">8.04 x 10 PFU/mL</td><td colspan="1" rowspan="1">2/3 Not Observed1 HRV Positive**</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Test Concentrations</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5</td><td colspan="1" rowspan="1">Zeptometrix 081019CF</td><td colspan="1" rowspan="1">7.16 x 10' PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">Zeptometrix 081007CF</td><td colspan="1" rowspan="1">1.41 x 10° PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Echovirus 25</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">1.93 x 10b PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Echovirus 30</td><td colspan="1" rowspan="1">Zeptometrix 0810078CF</td><td colspan="1" rowspan="1">9.89 x 10* PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A21</td><td colspan="1" rowspan="1">Zeptometrix 0810018CF</td><td colspan="1" rowspan="1">2.92 x 10° PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24</td><td colspan="1" rowspan="1">ATCC VR-583</td><td colspan="1" rowspan="1">7.00 x 10° PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">ATCC VR-561</td><td colspan="1" rowspan="1">1.40 X 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Poliovirus</td><td colspan="1" rowspan="1">ATCC VR-193</td><td colspan="1" rowspan="1">1.11 x 10° FU/mL,</td><td colspan="1" rowspan="1">HRV False Positive</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">Clinical Isolate</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus  Herpes Simplex</td><td colspan="1" rowspan="1">Zeptometrix 0810005CF</td><td colspan="1" rowspan="1">1.01 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus  Varicella Zoster</td><td colspan="1" rowspan="1">Zeptometrix 081 0026CF</td><td colspan="1" rowspan="1">2.35 x 10 copies/mL."</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus 4: Epstein Barr</td><td colspan="1" rowspan="1">Zeptometrix 0810008CF</td><td colspan="1" rowspan="1">1.06 x 103 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus :Cytomegalovirus</td><td colspan="1" rowspan="1">Zeptometrix 0810003CF</td><td colspan="1" rowspan="1">6.68 x 10b PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">1.37 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Zeptometrix 0810079CF</td><td colspan="1" rowspan="1">1.93 x 10 PFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr></table>

\*ADV C cross-reactive signal was also obtained from Adenovirus 9 (D) and Adenovirus 41 (F) when it was dilute 00 fold from heinitial testng concentratinDue he gene milarity between Adenovi D and F, the eSensor RVP cannot reliably differentiate them. A positive eSensorRVP Adenovius species Cresult olslaivtqnaletiv is needed.

One replicate of Coxsackievirus B4 at high concentration $( 8 . 0 4 \times 1 0 ^ { 6 }$ PFU/mL ) generated a HRV positive result which was generated a positive call for HRV.

Oue to the genetic similarity between human rhinovirus and poliovirus, the eSensor RVP cannot reliably differentiate them a polio infection is suspected, a positive eSensor RVP human rhinovirus HRV) reult should be confirmed using an alternate method (e.g., cell culture).

time RT-PCR and provided in copies/mL

# Cross-Reactivity with Bacteria and Fungus

Bacterial and fungal strains were tested for cross-reactivity with the eSensor RVP and were diluted in M5 transport media to a titer of $1 0 ^ { 6 } C F U / m L$ . These organisms were extracted in triplicate with the bioMérieux easyMAG system.

Following extraction, each replicate was tested once using the eSensor RVP as shown in Table 18.

Table 18: Cross-Reactivity Results of with Bacteria and Fungus   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">s   Source</td><td colspan="1" rowspan="1">Test Concentrations</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">Zeptometrix 0801597</td><td colspan="1" rowspan="1">5.2 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Zeptometrix 0801461</td><td colspan="1" rowspan="1">9.8 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Zeptometrix 0801459</td><td colspan="1" rowspan="1">5.8 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">2.3 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Zeptometrix 0801504</td><td colspan="1" rowspan="1">1.0 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Zeptometrix 0801535</td><td colspan="1" rowspan="1">9.73 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae DNA</td><td colspan="1" rowspan="1">ABI 08-942-250</td><td colspan="1" rowspan="1">1.4 x 10' copies/mL .</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">3.58 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Test Concentrations</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Zeptometrix 0801624</td><td colspan="1" rowspan="1">1.5 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Zeptometrix 0801680</td><td colspan="1" rowspan="1">2.6  10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix 0801506</td><td colspan="1" rowspan="1">1.07 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Zeptometrix 0801540</td><td colspan="1" rowspan="1">2.12 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus planarum</td><td colspan="1" rowspan="1">Zeptometrix 0801507</td><td colspan="1" rowspan="1">1.75 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Zeptometrix 0801645</td><td colspan="1" rowspan="1">2.6 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Zeptometrix 0801509</td><td colspan="1" rowspan="1">3.9 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">Zeptometrix 0801660</td><td colspan="1" rowspan="1">2.2 x 10CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix 0801579</td><td colspan="1" rowspan="1">2.47 × 10 CCU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningiditis</td><td colspan="1" rowspan="1">Zeptometrix 0801511</td><td colspan="1" rowspan="1">3.37 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">Zeptometrix 0801754</td><td colspan="1" rowspan="1">3.37 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">3.55x10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">1.0 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Zeptometrix 0801519</td><td colspan="1" rowspan="1">1.05 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Zeptometrix 0801723</td><td colspan="1" rowspan="1">6.1 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (COL)</td><td colspan="1" rowspan="1">Zeptometrix 0801638</td><td colspan="1" rowspan="1">8.4 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (MSSA)</td><td colspan="1" rowspan="1">Zeptometrix 0801675</td><td colspan="1" rowspan="1">1.2 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (MSSE)</td><td colspan="1" rowspan="1">Zeptometrix 0801689</td><td colspan="1" rowspan="1">2.2 × 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (MRSE)</td><td colspan="1" rowspan="1">Zeptometrix 0801651</td><td colspan="1" rowspan="1">6.2 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Zeptometrix 0801591</td><td colspan="1" rowspan="1">2.16 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Zeptometrix 0801545</td><td colspan="1" rowspan="1">2.2 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">Zeptometrix 0801516</td><td colspan="1" rowspan="1">6.46 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Zeptometrix 0801695</td><td colspan="1" rowspan="1">2.43 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix 0801439</td><td colspan="1" rowspan="1">2.8 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyrogenes</td><td colspan="1" rowspan="1">Zeptometrix 0801512</td><td colspan="1" rowspan="1">1.55 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">6.53 x 10 CFU/mL</td><td colspan="1" rowspan="1">Not Observed</td></tr></table>

# Carryover/Cross-Contamination

The carryover/cross-contamination study challenged the extraction, RT-PCR, and detection portions of the assay within and between runs and operators tested over the course of five testing days. A representative strain of Parainfluenza Virus 3 was obtained as a commercially available cultured cell line. Positive Parainfluenza Virus 3 samples were prepared at a concentration of $1 . 0 0 \times 1 0 ^ { 5 } \tau \mathsf { C l } \mathsf { D } _ { 5 0 } t \mathsf { m L }$ (3559x LoD) while negative samples were un-inoculated Remel M5 transport media. All samples were extracted using the bioMérieux easyMAG System. Five sets of alternating high concentration positive and negative samples were extracted and tested in a checkerboard pattern. Each set of samples contained 24 tests (12 positive and 12 negative). Total number of tests for the duration of the study was 120 samples (60 positive and 60 negative).

No carryover/cross-contamination was observed in the eSensor RVP, as $100 \%$ of the PIV 3 negative samples were reported as 'Target Not Detected'.

# CLINICAL PERFORMANCE DATA

# Clinical Performance

# Expected Values

A prospective clinical study testing nasopharyngeal (NP) swab specimens was conducted during the $2 0 1 0 / 1 1$ influenza season at three North American clinical laboratories. The expected values of individual analytes based on eSensor RVP results in prospective samples are summarized in Tables 19 and 20. The expected values of mixed co-infections based on eSensor RVP results in prospective samples are summarized in Tables 21 and 22.

Table 19: Expected Value (As Determined by eSensor RVP) Summary by Age Group in the Prospective Clinical Evaluation   

<table><tr><td rowspan=2 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=1>Age 0-1(N = 270)</td><td rowspan=1 colspan=1>Age &gt;1-5(N = 136)</td><td rowspan=1 colspan=1>Age &gt;5-21(N =127)</td><td rowspan=1 colspan=1>Age &gt;21-655N = 333 </td><td rowspan=1 colspan=1>Age &gt;65(N =171)</td><td rowspan=1 colspan=1>All Ages(N = 1037)</td></tr><tr><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N {(%</td><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td></tr><tr><td rowspan=1 colspan=1>Infuenza A (Un-Subtypable</td><td rowspan=1 colspan=1>2(0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.6)</td><td rowspan=1 colspan=1>5(1.5)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>10 (1.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Total)</td><td rowspan=1 colspan=1>25 (9.3)</td><td rowspan=1 colspan=1>22 (16.2)</td><td rowspan=1 colspan=1>17 (13.4)</td><td rowspan=1 colspan=1>84 (25.2)</td><td rowspan=1 colspan=1>31 (18.1)</td><td rowspan=1 colspan=1>179 (17.3)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>12 (4.8)</td><td rowspan=1 colspan=1>15 (11.0)</td><td rowspan=1 colspan=1>7(5.5)</td><td rowspan=1 colspan=1>43 (12.9)</td><td rowspan=1 colspan=1>22 (12.9)</td><td rowspan=1 colspan=1>99 (9.5)</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>10 (3.7)</td><td rowspan=1 colspan=1>8 (5.9)</td><td rowspan=1 colspan=1>6(4.7)</td><td rowspan=1 colspan=1>33 (9.9)</td><td rowspan=1 colspan=1>7(4.1)</td><td rowspan=1 colspan=1>64 (6.2)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>10 (3.7)</td><td rowspan=1 colspan=1>17(12.5)</td><td rowspan=1 colspan=1>33 (26.0)</td><td rowspan=1 colspan=1>15(4.5)</td><td rowspan=1 colspan=1>7(4.1)</td><td rowspan=1 colspan=1>82 (7.9)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>18 (6.7)</td><td rowspan=1 colspan=1>11 (8.1)</td><td rowspan=1 colspan=1>3(2.4)</td><td rowspan=1 colspan=1>15 (4.5)</td><td rowspan=1 colspan=1>10 (5.9)</td><td rowspan=1 colspan=1>57 (5.5)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>82 (30.4)</td><td rowspan=1 colspan=1>27 (19.9)</td><td rowspan=1 colspan=1>21 (16.6)</td><td rowspan=1 colspan=1>26 (7.8)</td><td rowspan=1 colspan=1>11 (6.4)</td><td rowspan=1 colspan=1>167 (16.1)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>3 (1.1)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>5(0.5)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>1 (0.7).</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>7(0.7)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>43 (15.9)</td><td rowspan=1 colspan=1>15 (11.0)</td><td rowspan=1 colspan=1>5 (3.9)</td><td rowspan=1 colspan=1>18 (5.4)</td><td rowspan=1 colspan=1>5 (2.9)</td><td rowspan=1 colspan=1>86 (8.3)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>69 (25.6)</td><td rowspan=1 colspan=1>21 (15.4)</td><td rowspan=1 colspan=1>8 (6.3)</td><td rowspan=1 colspan=1>11(3.3)</td><td rowspan=1 colspan=1>10 (5.8)</td><td rowspan=1 colspan=1>119 (11.4)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>28 (10.4)</td><td rowspan=1 colspan=1>17 (12.5)</td><td rowspan=1 colspan=1>4 (3.2)</td><td rowspan=1 colspan=1>14 (4.2)</td><td rowspan=1 colspan=1>6 (3.5)</td><td rowspan=1 colspan=1>69 (6.7)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B/E</td><td rowspan=1 colspan=1>6 (2.2)</td><td rowspan=1 colspan=1>8(5.9)</td><td rowspan=1 colspan=1>3 (1.6)</td><td rowspan=1 colspan=1>5(1.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>22 (2.1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>21 (7.7)</td><td rowspan=1 colspan=1>4 (2.9)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>9 (2.7)</td><td rowspan=1 colspan=1>6 (3.5)</td><td rowspan=1 colspan=1>41 (3.9)</td></tr></table>

Table 20: Expected Value (As Determined by eSensor RVP) Summary by Site in the Prospective Clinical Evaluation   

<table><tr><td colspan="1" rowspan="2">Virus (Analyte)</td><td colspan="1" rowspan="1">Site 1(N = 245)</td><td colspan="1" rowspan="1">Site 2(N = 533)</td><td colspan="1" rowspan="1">Site 3(N = 259)</td><td colspan="1" rowspan="1">All Sites.(N = 1037)</td></tr><tr><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (Un-Subtypable</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">8(1.5)</td><td colspan="1" rowspan="1">2 (0.8)</td><td colspan="1" rowspan="1">10 (1.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (Total)</td><td colspan="1" rowspan="1">58 (23.7)</td><td colspan="1" rowspan="1">89 (16.7)</td><td colspan="1" rowspan="1">32 (12.4)</td><td colspan="1" rowspan="1">179 (17.3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">32 (13.1)</td><td colspan="1" rowspan="1">54 (10.1)</td><td colspan="1" rowspan="1">13 (5.0)</td><td colspan="1" rowspan="1">99 (9.5)</td></tr><tr><td colspan="1" rowspan="2">Virus (Analyte)</td><td colspan="1" rowspan="1">Site 1(N = 245)</td><td colspan="1" rowspan="1">Site 2(N = 533)</td><td colspan="1" rowspan="1">Site 3(N = 259)</td><td colspan="1" rowspan="1">All Sites(N = 1037)</td></tr><tr><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009 H1N1</td><td colspan="1" rowspan="1">19 (7.8)</td><td colspan="1" rowspan="1">28 (5.3)</td><td colspan="1" rowspan="1">17 (6.6)</td><td colspan="1" rowspan="1">64 (6.2)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">4(1.6)</td><td colspan="1" rowspan="1">59 (11.1)</td><td colspan="1" rowspan="1">19 (7.3)</td><td colspan="1" rowspan="1">82 (7.9)</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">23 (9.4)</td><td colspan="1" rowspan="1">25 (4.7)</td><td colspan="1" rowspan="1">9 (3.5)</td><td colspan="1" rowspan="1">57 (5.5)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">44 (18.0)</td><td colspan="1" rowspan="1">99 (18.6)</td><td colspan="1" rowspan="1">24 (9.3)</td><td colspan="1" rowspan="1">167 (16.1)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (0.8)</td><td colspan="1" rowspan="1">1 (0.4)</td><td colspan="1" rowspan="1">5(0.5)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">1 (0.4)</td><td colspan="1" rowspan="1">6(1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">7(0.7)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">3 (1.2)</td><td colspan="1" rowspan="1">68 (12.8)</td><td colspan="1" rowspan="1">15 (5.8)</td><td colspan="1" rowspan="1">86 (8.3)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">17 (6.9)</td><td colspan="1" rowspan="1">85 (15.9)</td><td colspan="1" rowspan="1">17(6.6)</td><td colspan="1" rowspan="1">119 (11.4)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">15 (6.1)</td><td colspan="1" rowspan="1">41 (7.7)</td><td colspan="1" rowspan="1">13 (5.0)</td><td colspan="1" rowspan="1">69 (6.7)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B/E</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">14 (2.6)</td><td colspan="1" rowspan="1">8 (3.1)</td><td colspan="1" rowspan="1">22 (2.1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C</td><td colspan="1" rowspan="1">16 (6.5)</td><td colspan="1" rowspan="1">19 (3.6)</td><td colspan="1" rowspan="1">6 (2.3)</td><td colspan="1" rowspan="1">41 (3.9)</td></tr></table>

Table 21: Expected Value (Co-infections as Determined by eSensor RVP) Summary by Age Group in the Prospective Clinical Evaluation   

<table><tr><td rowspan=2 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Age O-1(N =270</td><td rowspan=1 colspan=1>Age &gt;1-5(N = 136)</td><td rowspan=1 colspan=1>Age &gt;5-21N =127</td><td rowspan=1 colspan=1>Age &gt;21-65(N = 333)</td><td rowspan=1 colspan=1>Age &gt;65(N =171</td><td rowspan=1 colspan=1>All AgesN = 1037)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N (%)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + Flu B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2(0.2)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + HRV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + PIV3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2(0.2)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + HMPV + HRV + RSV A + RSV B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + Flu B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + HMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>ADV C + HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6 (0.6</td></tr><tr><td rowspan=1 colspan=1>ADV C + PIV3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + RSV A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4(0.4)</td></tr><tr><td rowspan=1 colspan=1>ADV C + RSV B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>ADV C + HRV + PIV3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + HRV + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV B/E</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1(0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6 (0.6)</td></tr><tr><td rowspan=1 colspan=1>Flu A + Flu B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + HMPV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4 (0.4)</td></tr><tr><td rowspan=1 colspan=1>Flu A + PIV2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2(0.2)</td></tr></table>

Table 22: Expected Value (Co-infections as Determined by eSensor RVP) Summary by Site in the Prospective Clinical Evaluation   

<table><tr><td rowspan=2 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Age 0-1(N=270</td><td rowspan=1 colspan=1>Age &gt;1-5N = 136)</td><td rowspan=1 colspan=1>Age &gt;5-21N = 127</td><td rowspan=1 colspan=1>Age &gt;21-65(N =333)</td><td rowspan=1 colspan=1>Age &gt;65N = 171</td><td rowspan=1 colspan=1>All Ages(N =1037</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N (%)</td></tr><tr><td rowspan=1 colspan=1>FluA + RSVA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>FluA + RSVB</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + HRV + PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + RSV A + RSV B</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV C + HRV + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV C + HRV + PIV3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu B + HRV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4(0.4)</td></tr><tr><td rowspan=1 colspan=1>Flu B + PIV3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>Flu B + RSVA</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5(0.5)</td></tr><tr><td rowspan=1 colspan=1>Flu B + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu B + HRV + PIV2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu B + HRV + RSV A</td><td rowspan=1 colspan=1>2-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>HMPV+ HRV</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5(0.5)</td></tr><tr><td rowspan=1 colspan=1>HMPV + PIV3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HMPV + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2(0.2)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11 (1.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + RSV A</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16 (1.6)</td></tr><tr><td rowspan=1 colspan=1>HRV + RSVB</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8(0.8)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV3 + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV3 + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>PIV3 + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>.0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6(0.6)</td></tr><tr><td rowspan=1 colspan=1>PIV3 + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr></table>

<table><tr><td colspan="1" rowspan="2">Co-Infection</td><td colspan="1" rowspan="1">Site 1(N = 245)</td><td colspan="1" rowspan="1">Site 2N = 533)</td><td colspan="1" rowspan="1">Site 3N = 259)</td><td colspan="1" rowspan="1">All Sites(N = 1037</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + Flu B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + HRV</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + HMPV + HRV + RSV A +RSVB</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">. •</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1 (0.1)-</td></tr><tr><td colspan="1" rowspan="1">ADV C + Flu B</td><td colspan="1" rowspan="1">0.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HMPV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="2">Co-Infection</td><td colspan="1" rowspan="1">Site 1N = 245)</td><td colspan="1" rowspan="1">Site 2N =533</td><td colspan="1" rowspan="1">Site 3(N = 259)</td><td colspan="1" rowspan="1">All SitesN =1037</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6(0.6</td></tr><tr><td colspan="1" rowspan="1">ADV C + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + RSV A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4 (0.4)</td></tr><tr><td colspan="1" rowspan="1">ADV C + RSV B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV +PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV B/E</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A +ADV C</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6(0.6)</td></tr><tr><td colspan="1" rowspan="1">FluA + Flu B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A+ HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4 (0.4)</td></tr><tr><td colspan="1" rowspan="1">Flu A + PIV2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSVA</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSVA + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADVC + HRV + RSVA</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV C + HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu B + HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4 (0.4)</td></tr><tr><td colspan="1" rowspan="1">Flu B + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">Flu B + RSVA</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5(0.5)</td></tr><tr><td colspan="1" rowspan="1">Flu B + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu B + HRV + PIV2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu B + HRV + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">HMPV + HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5 (0.5)</td></tr><tr><td colspan="1" rowspan="1">HMPV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HMPV + RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">11 (1.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + RSV A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16 (1.6)</td></tr><tr><td colspan="1" rowspan="1">HRV + RSVB</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">8 (0.8)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV3 + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV3 + RSV 8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">PIV3 + RSVA</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6 (0.6)</td></tr><tr><td colspan="1" rowspan="1">PIV3 + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1 (0.1)</td></tr></table>

# Prospective Clinical Study

All clinical specimens in the prospective clinical study were nasopharyngeal (NP) swab specimens, prospectively collected and tested during the $2 0 1 0 / 1 1$ influenza season at three North American clinical laboratories. Clinical laboratories were located in Cleveland, Ohio; Providence, RI; and Albuquerque, NM. Demographic details for patient population are summarized in Table 23.Study sites enrolled subjects from diverse demographic groups; about $40 \%$ of the specimens were obtained from patients enrolled at a hospital. The remaining specimens were collected from outpatients and patients in an emergency department. A total of 1182 patient samples were collected prospectively across the three clinical sites from January 2011 until May 2011. Out of these patient samples, 1037 were evaluable. A total of 145 samples were excluded for the following reasons: samples not tested within 5 days of specimen collection (72/145), operator and/or easyMAG mechanical errors (62/145), samples not retested (11/145). Out of the 1037 samples collected, an even split of patients were male and female. Approximately one quarter of the samples came from children under the age of 1, and patients aged 21-65 contributed the largest share of the samples.

Table 23: General Demographic Data for Prospectively Collected Specimens $( N = 1 0 3 7 )$   

<table><tr><td rowspan=2 colspan=1>Demographic</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=1 colspan=1>N = 245 (%)</td><td rowspan=1 colspan=1>N = 533 (%)</td><td rowspan=1 colspan=1>N = 259 (%)</td><td rowspan=1 colspan=1>N = 1037 (%)</td></tr><tr><td rowspan=1 colspan=5>SEX</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>105 (42.9)</td><td rowspan=1 colspan=1>296 (55.5)</td><td rowspan=1 colspan=1>117 (45.2)</td><td rowspan=1 colspan=1>518 (50.0)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>140 (57.1)</td><td rowspan=1 colspan=1>237 (44.5)</td><td rowspan=1 colspan=1>142 (54.8)</td><td rowspan=1 colspan=1>519 (50.0)</td></tr><tr><td rowspan=1 colspan=5>:                            AGE (yrs)</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>46 (18.8)</td><td rowspan=1 colspan=1>197 (37.0)</td><td rowspan=1 colspan=1>27 (10.4)</td><td rowspan=1 colspan=1>270 (26.0)</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>20 (8.2)</td><td rowspan=1 colspan=1>94 (17.6)</td><td rowspan=1 colspan=1>22 (8.4)</td><td rowspan=1 colspan=1>136 (13.1)</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>19 (7.8)</td><td rowspan=1 colspan=1>82 (15.4)</td><td rowspan=1 colspan=1>26 (10.0)</td><td rowspan=1 colspan=1>127 (12.2)</td></tr><tr><td rowspan=1 colspan=1>&gt;21-65</td><td rowspan=1 colspan=1>97 (39.6)</td><td rowspan=1 colspan=1>106 (19.9)</td><td rowspan=1 colspan=1>130 (50.2)</td><td rowspan=1 colspan=1>333 (32.1)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65</td><td rowspan=1 colspan=1>63 (25.7)</td><td rowspan=1 colspan=1>54 (10.1)</td><td rowspan=1 colspan=1>54 (20.8)</td><td rowspan=1 colspan=1>171 (16.5).</td></tr><tr><td rowspan=1 colspan=5>SUBJECT STATUS</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>7 (2.9)</td><td rowspan=1 colspan=1>219 (41.1)</td><td rowspan=1 colspan=1>90 (34.7)</td><td rowspan=1 colspan=1>316 (30.5)</td></tr><tr><td rowspan=1 colspan=1>.Hospitalized</td><td rowspan=1 colspan=1>131 (53.5)</td><td rowspan=1 colspan=1>162 (30.4)</td><td rowspan=1 colspan=1>114 (44.0)</td><td rowspan=1 colspan=1>407 (39.2)</td></tr><tr><td rowspan=1 colspan=1>Emergency Department</td><td rowspan=1 colspan=1>107 (43.7)</td><td rowspan=1 colspan=1>152 (28.5)</td><td rowspan=1 colspan=1>55 (21.2)</td><td rowspan=1 colspan=1>314 (30.3)</td></tr></table>

A total of 1037 specimens were evaluated for all 14 RVP panel viruses with the prospectively collected samples, the performance for each respiratory virus was described by the clinical sensitivity and specificity. Sensitivity for a respiratory virus is the ability of the test to obtain positive results for this respiratory virus in the samples with positive results obtained by the comparator method for the particular virus. Specificity for a respiratory virus is the ability of the test to obtain negative results for this respiratory virus in the samples with negative results obtained by the comparator method for this respiratory virus. Depending on the comparator method used for a particular virus, performance is described as sensitivity/specificity or Positive Percent Agreement (PPA)/Negative Percent Agreement (NPA).

The performance of the RVP assay was compared to the established gold standard reference method of viral culture for most viral targets. For respiratory viruses in which culture was not available, a composite (multi-test) reference method (a predetermined algorithm that combined the results of a few tests) was used as the comparator method.As seen in Table 24, viral culture followed by DFA identification testing was used as the comparator method for Influenza A, Influenza B, RSV, Parainfluenza Viruses (PIV1, PIV2, PIV3), and adenovirus. Since viral culture cannot determine the subtype for influenza A, RSVs, and adenoviruses, these viruses were subtyped by an independently developed qRT-PCR assay or qPCR assay followed by bidirectional sequencing to determine the subtypes (Influenza A H1, Influenza A H3, Influenza A 2009 H1N1, RSVA, RSVB, ADVB/E and ADVC). HRV and HMPV were evaluated using two independently developed and validated qRT-PCR assays followed by bidirectional sequencing.

Table 24: Comparator Methods used to assess RVP performance   

<table><tr><td rowspan=1 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=1>Comparator Method</td><td rowspan=1 colspan=1>Subtyping</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=8 colspan=1>Viral culture followed by DFA identification</td><td rowspan=8 colspan=1>qRT-PCR + BidirectionalSequencing</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td></tr><tr><td rowspan=1 colspan=1>RSVA</td></tr><tr><td rowspan=1 colspan=1>RSVB.</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B/E</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=4 colspan=1>Viral culture followed by DFA identification²2</td><td rowspan=4 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=2 colspan=1>2 qRT-PCR (2 methods) with BidirectionalSequencing3</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td></tr></table>

Validated Performance of the eSensor RVP assay detecting Influenza A, RSV or ADV respectively was compared to viral culture followed by fluorecent antibody ientiication. "True" InfluenzaA, RVADV positives respectively, were considered as any sample that tested positive for Influenza A, RSV or ADV respectively, by viral culture followed by DFA testing. True positive samples were subtyped usig one analytically validated qRT-PCR assay with bi-directional sequence confiation. The comparator assays were designed to amplify a different sequence from that amplified by the eSensor RVP assay(s). None of the comparator PCR assays overlapped any RVP amplicon sequence even if the same gene was targeted. True Influenza A H1, H3, or 2009 H1N1 positives, respectively, were considered as any sample that tested positive Izylcultu a que ee ualy criteria that matched Influenza A/H1, AH3, or A/2009 H1 sequences deposited in the National Center for Botenology Information(NCB) GenBank database (ww.cbi.nlm.ih.gov), espectively, with acceptableEval. Tr"RV A or RV B positives, respeively, wee consdered as ny sample that test positiv Inza  byvial cultureand hadbidirectinal sequencgdataeeting re-define qualityacceptane criteria that matched RSV A or RSV B sequences deposited in the National Center for Biotechnology Inatin NCB) GenBank database w.cbi.ih.gov resectively with acceptable -valus. ADV C ADV B/ postives,espectively, were consiered s any sample hat teste positive orInfluza l uluhl g ateeg eef ualypanci matched ADV Cor ADV B/E sequences deposited in the National Center for Biotechnology Information (NCBl) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values.

Perfoance of the eSensor RVP assay detecting Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2 and Parainfuenza Virus 3 respectively was compared to viral culture followed by fluorescent antibody identification. "True" Influenza B, Parainfluenza Virus 1, Parainfuenza Virus 2 or Parainfuenza Virus 3 piv ivey eoer plat st ostiv Inr ParaiuenzaVi  r Paraifenza Vi especively by vial cultu ollowe by DA te 3Performance of the eSensor RVP assay detecting Human Rhinovirus or Human Metapneumovirus, respectively, was compared to a predetermined algorithm that used composite comparator methods. The mets consit ftanalyticaly valiatd Rassays olwe byirectinal sequenciTrue Hu

Rhinovirus or Human Metapneumovirus positives, respectively, were considered as any sample that had idiecional sequencing datameeting re-defined qualityaccetancecriterithat matched HumanRhinvi Human Metapneumovirus sequences deposited in the National Center for Biotechnology Information (NCBl) GenBank database (www.ncbi.nlm.nih. gov), respectively, with acceptable E-values.

Sensitivity or positive percent agreement (PPA) was calculated by dividing the number of true positive TP) results by the sum of true positive and false negative (FN) results, while speciicity or negative percent agreement (NPV) was calculated by dividing the number of true negative (TN) results by the sum of true negative and false positive (FP) results. A TP result was one where the positive RVP result matched the positive reference/comparator result, while a TN result was one whereby a negative RVP result matched a negative reference/comparator result. The two-sided $9 5 \%$ confidence interval was also calculated. The results are summarized below.

Table 25: Performance in Prospective Clinical Specimens $( N = 1 0 3 7 )$   

<table><tr><td rowspan=2 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A&#x27;</td><td rowspan=1 colspan=1>132/137a</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>91.7% - 98.8%</td><td rowspan=1 colspan=1>850/897</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>93.1% - 96.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1*</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1027/1027</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>74174</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.1% - 100.0%</td><td rowspan=1 colspan=1>927/952</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>96.2% - 98.3%</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.7% - 100.0%</td><td rowspan=1 colspan=1>956/971</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>97.5% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>64/69°</td><td rowspan=1 colspan=1>92.8%</td><td rowspan=1 colspan=1>83.9% - 97.6%</td><td rowspan=1 colspan=1>947/965</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>97.1% - 98.9%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1*</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>39.8% - 100.0%</td><td rowspan=1 colspan=1>1029/1030&#x27;</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2*</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>35.9% - 99.6%</td><td rowspan=1 colspan=1>1026/1028*</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>64/68</td><td rowspan=1 colspan=1>94.1%</td><td rowspan=1 colspan=1>85.6% - 98.4%</td><td rowspan=1 colspan=1>944/966™</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>96.6% - 98.6%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7% - 100.0%</td><td rowspan=1 colspan=1>905/956</td><td rowspan=1 colspan=1>94.7%</td><td rowspan=1 colspan=1>93.1% - 96.0%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.7% - 100.0%</td><td rowspan=1 colspan=1>955/996</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>94.5% - 97.0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B/E*</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>75.3% - 100.0%</td><td rowspan=1 colspan=1>1012/1021</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>98.3% - 99.5%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C*</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>54.1% - 100.0%</td><td rowspan=1 colspan=1>993/10289</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>95.3% - 97.5%</td></tr><tr><td rowspan=2 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.5% - 100.0%</td><td rowspan=1 colspan=1>979/981</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>·132/148</td><td rowspan=1 colspan=1>89.2%</td><td rowspan=1 colspan=1>83.0% - 93.7%</td><td rowspan=1 colspan=1>853/888h</td><td rowspan=1 colspan=1>96.1%</td><td rowspan=1 colspan=1>94.6% - 97.3%</td></tr></table>

These viral targets were supplemented with retrospective samples as shown below.   
nnzaA esults conta Flu A smples without a positive subtype n smples with either nza A H3 or 2009 H1N1 positive results.   
"Flu A was not detected in all5 RVP False Negative samples using independently developed and validated qPCR assays.   
viruses wee confe positiv /47 RVP Fale ositive sampessig bidetional seuenc Flu  3vises we onfe postiv  /5 RVP Fal siti smpless bietional eu dFlu A 2009 H1N1 viruses were confimed positive in 14/15 RVP False Positive samples using bidirectional sequencing.   
Flu B was not detected in 4/5 RVP False Negative samples using bidirectional sequencing.   
' Flu B was confirmed positive in 11/18 RVP False Positive samples using bidirectional sequencing.   
hMPV was confirmed positive in 1/2 RVP False Positive samples using bidirectional sequencing.   
HRV was confirmed positive in 7/35 RVP False Positive samples using bidirectional sequencing.   
PIV 1 was not detected in this RVP False Positive sample by bidirectional sequencing.   
PV 2 was not detected in this RVP Fale Negative sample using independently developed and validated qPCR assays.

The eSensor RVP system detected a total of 128 mixed infections in the prospective clinical evaluation (1037 tested and analyzed specimens). This represents $1 8 . 4 \%$ of the total positive specimens (128/696). One hundred fourteen ( $1 1 4 / 1 2 8 ; 8 9 . 1 \% )$ were double infections, eleven $( 1 1 / 1 2 8 ; 8 . 6 \% )$ were triple infections, and three $( 3 / 1 2 8 ; 2 . 3 \% )$ samples with four or more RVP analytes were identified. Ninety five of the 128 samples contained one or more analytes that the reference/comparator method failed to detect.

Table 26: Distinct Co-infection Combinations Detected by the eSensor RVP Assay in the Prospective Clinical Trial   

<table><tr><td colspan="5" rowspan="1">Distinct Co-infection Combinations Detectedby eSensor RVP</td><td colspan="1" rowspan="2">Total NumberofCo-infections</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-infections</td><td colspan="1" rowspan="2">DiscrepantAnalyte(s)</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ADV B (2), Flu B (1)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ADV B (3)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ADV B (1), RSVA (2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSVA (1), RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">ADVC</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">· ADVC(1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ADV C (3)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">ADV C (4), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C(1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">ADV C (3), RSV A (2)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ADV C (3), RSV B (2)</td></tr><tr><td colspan="1" rowspan="1">ADVC</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ADV C (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADV B/E</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">ADV C (6)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Flu A (2), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H1N1 (1), H3 (1), HMPV (1) .</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H1N1 (1), HRV (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Flu A (1), PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1)</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-infection Combinations Detectedby eSensor RVP                i </td><td colspan="1" rowspan="2">Total NumberofCo-infections</td><td colspan="1" rowspan="2">Number ofDiscrepaantCo-infections</td><td colspan="1" rowspan="2">Discrepantnalyte(s)</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td></tr><tr><td colspan="1" rowspan="1">FluA</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSV B (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H1N1 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSVA (2), RSV B (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADVC</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">·Flu A</td><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1), Flu A (1), PIV3 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Flu B (1), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Flu B (2), PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Flu B (2), RSVA (5)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV (1), PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSVA (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HMPV (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PIV1 (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">HRV (4) PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">HRV (5), RSV A (6)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HRV (1). RSV B (5)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSVA (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">PIV3 (4), RSVA (5)</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV3 (1), RSV B (1)</td></tr><tr><td colspan="5" rowspan="1">Total Number of Co-infections</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">117/278'</td></tr><tr><td colspan="5" rowspan="1">Total Number of Double Infections</td><td colspan="1" rowspan="1">114</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">99/232</td></tr><tr><td colspan="5" rowspan="1">Total Number f Triple Infections</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">11/33</td></tr><tr><td colspan="5" rowspan="1">Total Number of Quadruple Infections</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5/8</td></tr><tr><td colspan="5" rowspan="1">Total Number of Quintuple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2/5</td></tr></table>

Adicrepant co-nfection or discrepant analyte was defined as one that was detected by RVP but not the   
reference/comparator methods.   
aly a analyte in question in 58/117 cases.   
$a _ { G / 6 }$ nynal the analyte in question in 5/6 cases   
$\textcircled { 2 4 } 1 2 4$ nnawngas na the analyte in question in 11/24 cases   
$c _ { 6 } / 6$ a analyte in question in 3/6 cases   
${ \mathsf { d } } _ { \pmb { 4 } / \pmb { 4 } }$ eant u   nalys wenvgatsteaethnal uen identified the analyte in question in 4/4 cases   
EPY $^ { e } 1 / 1$ n aygaal al   
the analyte in question in 1/1 cases   
EY $1 2 / 2$ aa   
the analyte in question in 1/2 cases   
EY ${ \mathfrak { s } } _ { 1 9 / 1 9 }$ dnalva ay   
the analyte in question in 3/19 cases   
the analyte in question in $\mathfrak { 3 } / 1 2$ cases   
aa   
the analyte in question in 17/27 cases   
  
the analyte in question in 11/17 cases

Table 27: Additional Co-Infection Combinations Detected by Reference/Comparator Methods, But Not by the eSensor RVP Assay in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=2>Distinct Co-InfectionCombinations*</td><td rowspan=2 colspan=1>TotalNumber ofCo-Infections</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=2 colspan=1>DiscrepantAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Flu B (2), HRV (3)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Flu B (1),RSV/B(1)</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>HRV (3), PIV3 (3)</td></tr></table>

\*This table includes only co-infections that were detected by the reference/comparator method but not by RVP; the remaining co-infections detected by the reference/comparator method are already represented in Table above.

Table 28: Mixed Infections Detected by eSensor RVP in Prospective Samples   

<table><tr><td colspan="1" rowspan="1">OrganismCombinations</td><td colspan="1" rowspan="1">NumberofSamples</td><td colspan="1" rowspan="1">% ofSamplesAnalyzed(N=1037</td><td colspan="1" rowspan="1">OrganismCombinations</td><td colspan="1" rowspan="1">NumberofSamples</td><td colspan="1" rowspan="1">% ofSamplesAnalyzed(N=1037)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + Flu B</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">Flu A + HRV + PIV3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">Flu A + RSV A + RSV B</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + PIV3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">Flu A + ADV C + HRV + PIV3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + RSV A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">Flu A + ADV C + HRV + RSVA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">Flu B + HRV</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + HMPV + HRV + RSV A +RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">Flu B + PIV3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">ADV C + Flu B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">Flu B + RSV A</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1">ADV C + HMPV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">Flu B + RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">Flu B + HRV + PIV2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">ADV C + PIV3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">Flu B + HRV + RSV A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">ADV C + RSV A</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">HMPV + HRV</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1">ADV C + RSVB</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">HMPV + PIV3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV + PIV3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">HMPV + RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV + RSV A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">HRV + PIV1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">OrganismCombinations</td><td colspan="1" rowspan="1">NumberofSamples</td><td colspan="1" rowspan="1">% ofSamplesAnalyz dN1037)</td><td colspan="1" rowspan="1">OrganismCombinations</td><td colspan="1" rowspan="1">NumberofSamples</td><td colspan="1" rowspan="1">% ofS mplesAnalyzedN=1037</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV B/E</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">HRV + PIV2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Flu A +ADV C</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">HRV + PIV3</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.1</td></tr><tr><td colspan="1" rowspan="1">Flu A + Fiu B</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">HRV + RSV A</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1.6</td></tr><tr><td colspan="1" rowspan="1">Flu A + HMPV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">HRV + RSV B</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.8</td></tr><tr><td colspan="1" rowspan="1">Flu A + HRV</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">HRV + PIV3 + RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Flu A + PIV2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">HRV + PIV3 + RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Flu A + PIV3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">PIV3 + RSV A</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSV A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">PIV3 + RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSVB</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">Total Mixed Infections</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">12.3</td></tr></table>

$93 \%$ (963/1037) of the evaluable prospective clinical specimens yielded valid results on the first attempt. Invalid results or no results were obtained for the remaining 74 specimens (45 of which generated results on the first run, but required retesting due to a negative control failure caused by operator error). Data generated from the retests was used in the final analysis. All 74 specimens yielded valid results after a single retest when tested according the retest recommendations.

# Testing of Preselected Archived Samples

Banked samples previously characterized as positive for Influenza A H1, Parainfluenza Virus 1, Parainfluenza Virus 2, Adenovirus B/E, and Adenovirus C were used to supplement the performance studies for these analytes. These frozen banked samples were collected from various sites across the United States or from the Centers for Disease Control and Prevention (CDC). Upon arrival at GenMark, banked samples were blinded and intermixed with negative samples before being sent for testing, which was conducted by multiple sites involved in the prospective analysis of the patient samples. Testing of the banked samples was performed identically to prospectively-collected patient specimens. Results from the banked samples are presented separately from the prospectively collected specimens.

A total of 343 retrospective banked samples were collected for analysis. Out of this sample set, 11 samples were sent which didn't contain a banked viral target so these eleven samples were not tested further. Eight additional samples were excluded as they didn't contain a banked viral target as originally reported by the collection site and confirmed by comparator testing. Two samples reported errors on targets but were not retested as indicated. One sample was not sequenced. One sample had an internal control failure but was not retested as indicated. After these data were excluded, a total of 320 banked samples (including negative samples) for 5 viral targets were collected and analyzed.

With the exception of Flu A H1 samples, these banked samples were also sent to Beckman Coulter for comparator testing, and the results from the Beckman Coulter testing were compared to the results obtained by the eSensor RVP. Since the Flu A H1 samples came from the Centers for Disease Control and Prevention and were verified to be Flu A H1, these samples were not sent to Beckman Coulter for further testing. The results are summarized in Table 29.

Table 29: Performance in Retrospective Clinical Specimens $( N = 3 2 0 )$   

<table><tr><td rowspan=2 colspan=1>Virus</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>82.8% - 99.9%</td><td rowspan=1 colspan=1>290/290</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.7% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.3% - 100.0%</td><td rowspan=1 colspan=1>289/295</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>95.6% - 99.3%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.8% - 100.0%</td><td rowspan=1 colspan=1>284/294</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>93.8% - 98.4%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B/E</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.3% - 100.0%</td><td rowspan=1 colspan=1>290/295</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>96.1% - 99.4%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>80.6% - 100.0%</td><td rowspan=1 colspan=1>270/304</td><td rowspan=1 colspan=1>88.8%</td><td rowspan=1 colspan=1>84.8% - 91.9%</td></tr></table>

# eSensor RVP Performance in Fresh vs. Frozen Clinical Specimen

Simulated viral specimens were prepared by spiking viral transport media (Remel M5) with two different concentrations of ADV C viral culture (3x LoD and 1x LoD). To evaluate the performance of frozen specimens, 128 aliquots of ADV C (64 replicates each at $_ { 3 \times }$ and $1 \times$ LoD) were prepared. Sixty four aliquots (32 at each testing concentration) were tested immediately after preparation (fresh) while 64 aliquots were tested after undergoing two freeze/thaw cycles (frozen). Positive percent agreement between RVP results from fresh versus frozen aliquots for all concentrations tested was calculated. The positive percent agreement between RVP results from fresh versus frozen aliquots was $100 \%$ $9 5 \%$ confidence interval $8 9 . 3 \% - 1 0 0 \%$

# Prospective 2X2 Performance Tables:

Table 30: Prospective Influenza A Results   

<table><tr><td rowspan=1 colspan=4>Influenza A</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>47^</td><td rowspan=1 colspan=1>179</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>$5b$</td><td rowspan=1 colspan=1>850</td><td rowspan=1 colspan=1>855</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>897</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 96.4% (95% CI: 91.7% - 98.8%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 94.8% (95% CI: 93.1% - 96.1%)</td></tr></table>

a Influenza A virus was confirmed positive in 35/47 RVP False Positive samples using bidirectional sequencing. b Influenza A virus was not detected in all 5 RVP False Negative samples using independently developed and validated $\mathsf { q P C R }$ assays.

Table 31: Prospective Influenza A H1 Results   

<table><tr><td rowspan=1 colspan=4>Influenza A H1</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1027</td><td rowspan=1 colspan=1>1027</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1027</td><td rowspan=1 colspan=1>1027</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: N/A</td></tr><tr><td rowspan=1 colspan=4>Specificity: 100.0% (95% CI: 99.6% - 100.0%)</td></tr></table>

Table 32: Prospective Influenza A H3 Results   

<table><tr><td rowspan=1 colspan=4>Influenza A H3</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>25a</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>927</td><td rowspan=1 colspan=1>927</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>952</td><td rowspan=1 colspan=1>1026</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 95.1% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 97.4% (95% Cl: 96.2% - 98.3%)</td></tr></table>

a Influenza A H3 virus was confirmed positive in 22/25 RVP False Positive samples using bidirectional sequencing.

Table 33: Prospective Influenza A 2009 H1N1 Results   

<table><tr><td rowspan=1 colspan=4>Influenza A 2009 H1N1</td></tr><tr><td rowspan=2 colspan=1>eSensor RVp</td><td rowspan=1 colspan=3>          Reference          v:i .</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>64</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>956</td><td rowspan=1 colspan=1>956</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>971</td><td rowspan=1 colspan=1>1020</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 92.7% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 98.5% (95% CI: 97.5% - 99.1%)</td></tr></table>

a Influenza A $2 0 0 9 \mathsf { H l N } \mathsf { l }$ virus was confirmed positive in 14/15 RVP False Positive samples using bidirectional sequencing.

Table 34: Prospective Influenza B Results   

<table><tr><td rowspan=1 colspan=4>Influenza B</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1> -Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>18a</td><td rowspan=1 colspan=1>82</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>$5bb}$</td><td rowspan=1 colspan=1>947</td><td rowspan=1 colspan=1>952</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>965</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 92.8% (95% CI: 83.9% - 97.6%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 98.1% (95% CI: 97.1% - 98.9%)</td></tr></table>

a Influenza B virus was confirmed positive in 11/18 RVP False Positive samples using bidirectional sequencing.

b Influenza B virus was not detected in $4 / 5$ RVP False Negative samples using independently developed and validated qPCR assays.

Table 35: Prospective RSVA Results   

<table><tr><td rowspan=1 colspan=4>Respiratory Syncytial Virus A</td></tr><tr><td rowspan=2 colspan=1>*4eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>51a</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>905</td><td rowspan=1 colspan=1>905</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>956</td><td rowspan=1 colspan=1>1024</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 94.7% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 94.7% (95% CI: 93.1% - 96.0%)</td></tr></table>

a Respiratory Syncytial Virus type A was confirmed positive in 43/51 RVP False Positive samples using bidirectional sequencing.

Table 36: Prospective RSVB Results   

<table><tr><td rowspan=1 colspan=4>Respiratory Syncytial Virus B</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>28.</td><td rowspan=1 colspan=1>41a</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>955</td><td rowspan=1 colspan=1>955</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>996</td><td rowspan=1 colspan=1>1024</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 87.7% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 95.9% (95% Cl: 94.5% - 97.0%)</td></tr></table>

a Respiratory Syncytial Virus type B was confirmed positive in 35/41 RVP False Positive samples using bidirectional sequencing.

Table 37: Prospective PIV1 Results   

<table><tr><td rowspan=1 colspan=4>Parainfluenza Virus 1</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>:    Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1029</td><td rowspan=1 colspan=1>1029</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1030</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 39.8% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.9% (95% Cl: 99.5% - 100.0%)</td></tr></table>

a PIV 1 was not detected in this RVP False Positive sample by bidirectional sequencing.

Table 38: Prospective PIV2 Results   

<table><tr><td rowspan=1 colspan=4>Parainfluenza Virus 2</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>1026</td><td rowspan=1 colspan=1>1027</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1028</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 83.3% (95% CI: 35.9% - 99.6%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.8% (95% CI: 99.3% - 100.0%)</td></tr></table>

a Parainifluenza type 2 virus was confirmed positive in 0/2 RVP False Positive samples by bidirectional sequencing. b Parainfluenza type 2 virus was not detected in this RVP False Negative sample using independently developed and validated $\mathsf { q P C R }$ assays.

Table 39: Prospective PIV3 Results   

<table><tr><td rowspan=1 colspan=4>Parainfluenza Virus 3</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>SPositive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>944</td><td rowspan=1 colspan=1>948</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 94.1% (95% CI: 85.6% - 98.4%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 97.7% (95% CI: 96.6% - 98.6%)</td></tr></table>

a Parainfluenza type 3 virus was confirmed positive in $1 0 / 2 2$ RVP False Positive samples using bidirectional sequencing. b Parainfluenza type 3 virus was not detected in $4 / 4$ RVP False Negative samples using independcntly developed and validated $\mathsf { q P C R }$ assays.

Table 40: Prospective HMPV Results   

<table><tr><td rowspan=1 colspan=4>Human Metapneumovirus</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>.2a</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>979</td><td rowspan=1 colspan=1>979</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>981</td><td rowspan=1 colspan=1>1036</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 93.5% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.8% (95% CI: 99.3% - 100.0%)</td></tr></table>

a Human metapneumovirus was confirmed positive in 1/2 RVP False Positive samples using bidirectional sequencing.

Table 41: Prospective HRV Results   

<table><tr><td rowspan=1 colspan=4>Human Rhinovirus</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>35a</td><td rowspan=1 colspan=1>167</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>869</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>888</td><td rowspan=1 colspan=1>1036</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 89.2% (95% CI: 83.0% - 93.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 96.1% (95% CI: 94.6% - 97.3%)</td></tr></table>

a Human rhinovirus was confirmed positive in 7/35 RVP False Positive samples using bidirectional sequencing.

Table 42: Prospective ADV B/E Results   

<table><tr><td rowspan=1 colspan=4>Adenovirus B/E</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>?&quot;&quot;Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>9a</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1012</td><td rowspan=1 colspan=1>1012</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1021</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI:75.3% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.1% (95% CI: 98.3% - 99.5%)</td></tr></table>

a Adenovirus type B/E was confirmed positive in 8/9 RVP False Positive samples using bidirectional sequencing.

Table 43: Prospective ADV C Results   

<table><tr><td rowspan=1 colspan=4>Adenovirus C</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8  Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>35a</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>993</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1028</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (95% CI: 54.1% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 96.6% (95% CI: 95.3% - 97.5%)</td></tr></table>

a Adenovirus type C was confirmed positive in 16/35 RVP False Positive samples using bidirectional sequencing.

# Retrospective 2X2 Performance Tables:

Table 44: Retrospective Influenza A H1 Results   

<table><tr><td rowspan=1 colspan=4>Influenza A (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>291</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 96.7% (95% Cl: 82.8% - 99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 100.0% (95% CI: 98.7% - 100.0%)</td></tr></table>

Table 45: Retrospective PIV1 Results   

<table><tr><td rowspan=1 colspan=4>PIV1 (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative 3 x</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>.289</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (95% CI: 86.3% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 98.0% (95% CI: 95.6% - 99.3%)</td></tr></table>

Table 46: Retrospective PIV2 Samples   

<table><tr><td rowspan=1 colspan=4>PIV2 (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>284</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>294</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (95% CI: 86.8% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 96.6% (95% CI: 93.8% - 98.4%)</td></tr></table>

Table 47: Retrospective ADV B/E Results   

<table><tr><td rowspan=1 colspan=4>ADV B/E (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>290</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (95% CI: 86.3% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 98.3% (95% CI: 96.1% - 99.4%)</td></tr></table>

Table 48: Retrospective ADV C Results   

<table><tr><td rowspan=1 colspan=4>ADV C (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0 </td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (95% Cl: 80.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 88.8% (95% CI: 84.8% - 91.9%)</td></tr></table>

GenMark Diagnostics, Inc.   
c/o Joel Centeno   
VP Regulatory, Quality, Clinical 5964 La Place Court   
Carlsbad, CA 92008

Re: k113731 Trade/Device Name: eSensor® Respitory Viral Panel (RVP) Regulation Number: 21 CFR 866.3980 Regulation Name: Respitory viral panel multiplex nucleic acid assay Regulatory Class: Class II Product Code: OCC, OEM, OOU, OEP, OQW, NSU, OUL, JJH Dated: September 4, 2012 Received: September 6, 2012

Dear Mr. Centeno:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling. and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class I (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register. .

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice

requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

# Indications for Use Form

510(k) Number (f known):K113731

Device Name:Sensor® Respiratory Viral Panel (RVP)

Indications for Use:

Vi innasopharyngea swabs (NP)obtainedfrom individualsexhibiting sins nd symptoms f respiratory infecion.

Tolloye bseSVn SubtyfS uby,ubty,ReySn Rsy n by V Human Metapneumovirus, Human Rhinovirus, Adenovirus species B/E, and Adenovlrus species C.

Thedetection and ientification  speciiiral nucle acis frondividualsexhibiting ns and sypt epidemiological information.

Naiv  vn uo e ul t aral nd viral culure,immunofuorescenc nd radiography and clinicl presentation must e taken to consideration in the final diagnosis of respiratory viral infection.

Peoanc haris nfenzwestabliheduringhe0/20nfz n wheInfn 2009 H11 and H3N2 were he predominant Infuenza  viruses in circulation.When othernfuenzaviruses merge performance characteristics may vary.

remneypuleahuorit seens houlecol popriaten contoet attempted in these cases unless a BSL $\mathfrak { s } \mathrm { + }$ facility is available to receive and culture specimens.

Prescription Use X Over-The-Counter Use CFR 80 Sub DNDOR (21 CFR 801 Subpart C)

(PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE OF NEEDED)

Concurrence of CDRH, Office of In Vitro Diagnostic Devices (OIVD)

Vamar Keldb Division Sign-Off Office of In Vitro Diagnostic Device Evaluation and Safety 510(k) K113731