# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k113731

To obtain a Substantial Equivalence Determination for a new device

# C. Measurand:

Viral RNA/DNA sequences of the following respiratory viruses   
Adenovirus species B/E (combined result) and species C   
Human Rhinovirus   
Human Metapneumovirus   
Influenza A and Influenza A subtypes H1, H3 and 2009 H1N1   
Influenza B   
Parainfluenza Virus 1, 2, and 3   
Respiratory Syncytial Virus subtypes A and B

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the eSensor instrument for the qualitative in vitro detection and identification of multiple respiratory viral pathogen nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections

E. Applicant: Clinical MicroSensors, Inc d.b.a GenMark Diagnostics

F. Proprietary and Established Names: eSensor® Respiratory Viral Panel (RVP) eSensor® XT-8 System

G. Regulatory Information:

<table><tr><td>Product Code</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>OCC</td><td>Class II</td><td>21 CFR 866.3980 Respitory Viral Panel Multiplex Nucleic Acid Assay</td><td>Microbiology (83)</td></tr><tr><td>OEM</td><td>Class II</td><td>21 CFR 866.3980 Respitory Viral Panel Multiplex Nucleic Acid Assay (human metapneumovirus)</td><td>Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respitory Viral PanelMultiplex Nucleic Acid Assay(parainfluenza)</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEP</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respitory Viral PanelMultiplex Nucleic Acid Assay(influenza a virus subtypedifferentiation)</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OQW</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3332Reagents for detection of specificnovel influenza A viruses</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">NSU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570Instrumentation for ClinicalMultiplex Test Systems</td><td colspan="1" rowspan="1">Clinical Chemistry (75)</td></tr><tr><td colspan="1" rowspan="1">OUL</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 862.2310Thermocycler Generic</td><td colspan="1" rowspan="1">Clinical Chemistry (75)</td></tr><tr><td colspan="1" rowspan="1">JJH</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">21 CFR 862.2310Clinical Sample Concentrator</td><td colspan="1" rowspan="1">Clinical Chemistry (75)</td></tr></table>

# H. Intended Use:

# 1. Intended use(s):

The eSensor Respiratory Viral Panel (RVP) is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on the eSensor XT-8 system for the simultaneous detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals exhibiting signs and symptoms of respiratory infection.

The following virus types and subtypes are identified using the eSensor RVP: Influenza A, Influenza A H1 Seasonal Subtype, Influenza A H3 Seasonal Subtype, Influenza A 2009 H1N1 subtype, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Human Metapneumovirus, Human Rhinovirus, Adenovirus species B/E, and Adenovirus species C.

The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and epidemiological information.

Negative results do not preclude respiratory viral infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Positive results do not rule out bacterial infection, or co-infection with other viruses. The agent detected may not be the definite cause of disease. The use of additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence and radiography) and clinical presentation must be taken into consideration in the final diagnosis of respiratory viral infection.

Performance characteristics for Influenza A were established during the 2010/2011 influenza season when Influenza A 2009 H1N1 and H3N2 were the predominant Influenza A viruses in circulation. When other Influenza A viruses emerge, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: eSensor XT-8 System

# I. Device Description:

The eSensor Respiratory Viral Panel (RVP) reagent kit including primers and signal probes for the following respitory viral pathogens: Adenovirus (with species specific oligonucleotides for type C, B and E), Influenza A (with subtyping for hemagglutinin genes H1, 2009 H1 and H3), Influenza B, human Metapneumovirus, Parainfluenza Virus (serotypes 1, 2, and 3), Respiratory Syncytial virus (sub-types A and B), and Rhinovirus. The eSensor XT-8 consumable has a plurality of electrode locations that are coated with analyte specific capture probe oligonucleotide for multiplex amplicon detection. The eSensor XT-8 System accepts the consumable and completes the hybridization and detection of each electrode using an assay specific protocol. A summary of the eSensor Respiratory Viral Panel assays and targets is presented in the following table:

<table><tr><td colspan="1" rowspan="1">Organism (Abbreviation)</td><td colspan="3" rowspan="1">Assay Type</td><td colspan="1" rowspan="1">Gene Target</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C(Adv C)</td><td colspan="3" rowspan="2">Adenovirus(DNA)</td><td colspan="1" rowspan="2">Hexon</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B/E (Adv B/E)</td></tr><tr><td colspan="1" rowspan="1">Influenza A(Flu A)</td><td colspan="3" rowspan="3">Orthomyxovirus</td><td colspan="1" rowspan="1">Matrix Protein</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1 (Flu A H1)</td><td colspan="1" rowspan="1">Hemagglutinin (HA)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3 (Flu A H3)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Hemagglutinin (HA)</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009 H1N1(2009 H1N1)</td><td colspan="3" rowspan="2">(RNA)</td><td colspan="1" rowspan="1">Hemagglutinin (HA)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (Flu B)</td><td colspan="1" rowspan="1">RNA polymerase subunitPB1</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1 (PIV 1)</td><td colspan="3" rowspan="3">Paramyxovirus(RNA)</td><td colspan="1" rowspan="3">Hemagglutinin-neuraminidase(HN)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 2 (PIV 2)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 3 (PIV 3)</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="3" rowspan="1">Paramyxovirus</td><td colspan="1" rowspan="1">Nucleocapsid (N)</td></tr><tr><td colspan="1" rowspan="1">Organism (Abbreviation)</td><td colspan="1" rowspan="1">Assay Type</td><td colspan="1" rowspan="1">Gene Target</td></tr><tr><td colspan="1" rowspan="1">(hMPV)</td><td colspan="1" rowspan="3">(RNA)</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Respitory Syncytial Virus A(RSV A)</td></tr><tr><td colspan="1" rowspan="1">Respitory Syncytial Virus B(RSV B)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (HRV)</td><td colspan="1" rowspan="1">Picornavirus(RNA)</td><td colspan="1" rowspan="1">5'Untranslated Region(UTR)</td></tr></table>

# eSensor Respiratory Viral Panel (RVP) Reagent Kit

The RVP Reagent Kit contains all materials required to complete approximately 60 tests. The contents of a test kit are described below:

<table><tr><td rowspan=1 colspan=1>Box</td><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Packaging andQuantity</td><td rowspan=1 colspan=1>StorageConditions</td></tr><tr><td rowspan=2 colspan=1>eSensor RespitoryViral PanelCartridge Packs</td><td rowspan=1 colspan=1>eSensor RVPCartridges</td><td rowspan=1 colspan=1>6 foil bags with 8cartridges ineach cartridge pack</td><td rowspan=1 colspan=1>10-25 </td></tr><tr><td rowspan=1 colspan=1>eSensor RespiratoryViral PanelProduct Insert</td><td rowspan=1 colspan=1>1 Copy</td><td rowspan=1 colspan=1>Dry place</td></tr><tr><td rowspan=3 colspan=1>eSensor RespitoryViral PanelAmplificationReagents</td><td rowspan=1 colspan=1>RVP Enzyme Mix(reverse transcription)</td><td rowspan=1 colspan=1>2 vials for a total ofapprox.60 reactions</td><td rowspan=3 colspan=1>-15 to -30 °(in a designated pre-PCR location)</td></tr><tr><td rowspan=1 colspan=1>RVP PCR Mix(amplification reagent)</td><td rowspan=1 colspan=1>2 vials for a total ofapprox.70 reactions</td></tr><tr><td rowspan=1 colspan=1>MS2 Internal Control</td><td rowspan=1 colspan=1>2 vials with 300 μL each</td></tr><tr><td rowspan=2 colspan=1>eSensor RespitoryViral PanelDetection Reagents</td><td rowspan=1 colspan=1>Exonuclease(amplicon digestion)</td><td rowspan=1 colspan=1>2 vials for a total ofapprox.60 reactions</td><td rowspan=2 colspan=1>-15 to -30 °C(in a designated pre-PCR location)</td></tr><tr><td rowspan=1 colspan=1>RVP Signal BufferXT Buffer-1 and -2(hybridization anddetection)</td><td rowspan=1 colspan=1>2 vials per reagent for atotal of approx. 60-70reactions.</td></tr></table>

# The Assay Cartridge (eSensor XT-8 Cartridge)

The eSensor XT-8 cartridge device consists of a printed circuit board (PCB) with a multilayer laminate and a plastic cover that forms a hybridization chamber. The cartridge is fitted with a pump and check valves that circulate the hybridization solution when inserted into the eSensor XT-8 instrument. The PCB chip consists of an array of 72 gold-plated working electrodes, a silver reference electrode, and two gold-plated auxiliary electrodes. Each working electrode has a connector contact pad on the opposite side of the chip for electrical connection to the eSensor XT-8 instrument. Each electrode is modified with a multicomponent; self-assembled monolayer that includes oligonucleotide capture probes specific for each polymorphic site on the test panel and insulator molecules. The cartridge also contains an electrically erasable and programmable read only memory component

(EEPROM) that stores information related to the cartridge (e.g., assay identifier, cartridge lot number, and expiration date).

# The eSensor XT-8 Instrument

The eSensor XT-8 is a clinical multiplex instrument that has a modular design consisting of a base module and one, two, or three cartridge-processing towers containing eight, 16, or 24 cartridge slots, respectively. The cartridge slots operate independently of each other. Any number of cartridges can be loaded at one time, and the remaining slots are available for use while the instrument is running. The base module controls each processing tower, provides power, and stores and analyzes data. The instrument is designed to be operated solely with the touch screen interface. Entering patient accession numbers and reagent lot numbers can be performed by the bar code scanner or the touch screen.

Each processing tower consists of eight cartridge modules, each containing a cartridge connector, a precision-controlled heater, an air pump, and electronics. The air pumps drive the pump and valve system in the cartridge, eliminating fluid contact between the instrument and the cartridge. The pneumatic pumping enables recirculation of the hybridization solution allowing the target DNA and the signal probes to hybridize with the complementary capture probes on the electrodes. The pump in the cartridge is connected to a pneumatic source from the eSensor XT- 8 instrument and provides unidirectional pumping of the hybridization mixture through the channel during hybridization. Using this process to circulate the hybridization solution minimizes the unstirred boundary layer at the electrode surface and continuously replenishes the volume above the electrode that has been depleted of complementary targets and signal probes.

The XT-8 instrument provides electrochemical detection of bound signal probes by ACV and subsequent data analysis and test report generating functions. All hybridization, ACV scanning and analysis parameters are defined by a scanning protocol loaded into the XT-8 software, and then specified for use by the EEPROM on each cartridge.

# Materials Required But Not Provided Equipment

The XT-8 instrument is only for use with XT-8 test cartridges and is not integrated or connected with other laboratory systems.

§GenMark eSensor XT-8 instrument   
§Vortex Mixer (with platform head for 96-well PCR plate mixing)   
§Dry Heat Block   
§Cold Block or Ice   
§Adjustable Pipettes   
§PCR Thermal Cycler compatible with $0 . 2 \mathrm { m l }$ reaction tubes and 96-well reaction plates   
§Microcentrifuge (with adaptor if using PCR tubes or strips)   
§Plate Centrifuge (with adaptor if using 96-well PCR plates)   
§Dead Air Hood   
§bioMérieux NucliSENS® easyMAG® extraction system

# Consumables

§RNase/DNase-free PCR tubes and caps $\mathrm { 0 . 2 m L }$ , thin-walled), strips of 8 tubes with   
individual caps, or 96-well PCR plates and seals   
§Water, Molecular Biology Grade, RNAase/DNase-free   
§Disposable Gloves   
§Microfuge Tubes, RNase/DNase-free   
§Microfuge Tube Racks   
§Pipette Tips, Aerosol Resistant, RNase/DNase-free   
§Nucleic acid decontaminating solutions or $10 \%$ bleach for appropriate surfaces   
§bioMérieux NucliSENS® easyMAG® Consumables (buffers and disposables)

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Luminex® xTAG™ Respiratory Viral Panel (RVP) eSensor XT-8 System

2. Predicate 510(k) number(s): k081483

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>eSensor Respiratory Viral Panel (RVP)</td><td>Luminex xTAG RVP</td></tr><tr><td>Intended Use</td><td>The eSensor® Respiratory Viral Panel (RVP) is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on the eSensor XT-8T system for the simultaneous detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals exhibiting signs and symptoms of respiratory infection. The following virus types and subtypes are identified using the eSensor RVP: Influenza A, Influenza A H1 Seasonal Subtype, Influenza A H3 Seasonal Subtype, Influenza A 2009 H1N1 subtype, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B,</td><td>The xTAG RVP (Respiratory Viral Panel) is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of multiple respiratory virus nucleic acids in nasopharyngeal swabs from individuals suspected of respiratory tract infections. The following virus types and subtypes are identified using RVP: Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B, Para influenza 1, Para influenza 2, and Para influenza 3 virus, Human Metapneumovirus, Rhinovirus, and Adenovirus. The detection and identification of specific viral nucleic acids from individuals</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">eSensor Respiratory ViralPanel (RVP)</td><td colspan="1" rowspan="1">Luminex xTAG RVP</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1,Parainfluenza Virus 2,Parainfluenza Virus 3, HumanMetapneumovirus, HumanRhinovirus, Adenovirus speciesB/E, and Adenovirus species C.The detection and identification ofspecific viral nucleic acids fromindividuals exhibiting signs andsymptoms of respiratory infectionaids in the diagnosis of respiratoryviral infection if used inconjunction with other clinical andepidemiological information.Negative results do not precluderespiratory viral infection andshould not be used as the sole basisfor diagnosis, treatment or otherpatient management decisions.Positive results do not rule outbacterial infection, or co-infectionwith other viruses. The agentdetected may not be the definitecause of disease. The use ofadditional laboratory testing (e.g.bacterial and viral culture,immunofluorescence andradiography) and clinicalpresentation must be taken intoconsideration in the final diagnosisof respiratory viral infection.Performance characteristics forInfluenza A were establishedduring the 2010/2011 influenzaseason when Influenza A 2009H1N1 and H3N2 were thepredominant Influenza A viruses incirculation. When other InfluenzaA viruses emerge, performancecharacteristics may vary.If infection with a novel InfluenzaA virus is suspected based oncurrent clinical and</td><td colspan="1" rowspan="1">exhibiting signs and symptoms ofrespiratory infection aids in thediagnosis of respiratory viralinfection if used in conjunctionwith other clinical and laboratoryfindings. It is recommended thatspecimens found to be negativeafter examination using RVP beconfirmed by cell culture.Negative results do not precluderespiratory viral infection andshould not be used as the sole basisfor diagnosis, treatment or othermanagement decisions. Positiveresults do not rule out bacterialinfection or co-infection with otherorganisms. The agent detectedmay not be the definite cause ofdisease. The use of additionallaboratory testing (e.g., bacterialand viral culture,immunofluorescence, andradiography) and clinicalpresentation must be taken intoconsideration in order to obtain thefinal diagnosis of respiratoryinfection. Due to seasonalprevalence, performancecharacteristics for Influenza A/H1were established primarily withretrospective specimens. The RVPassay cannot adequately detectAdenovirus species C, or serotypes7a and 41. The RVP primers fordetection of rhinovirus cross-reactwith enterovirus. A rhinovirusreactive result should be confirmedby an alternate method (e.g. cellculture). Performancecharacteristics for Influenza AVirus were established whenInfluenza A/H3 and A/H1 were thepredominant Influenza A viruses incirculation. When other InfluenzaA viruses are emerging,performance characteristics mayvary. If infections with a novel</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">epidemiological screening criteriarecommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novelvirulent influenza viruses and sentto state or local health departmentsfor testing. Viral culture shouldnot be attempted in these casesunless a BSL 3+ facility isavailable to receive and culturespecimens.</td><td colspan="1" rowspan="1">Influenza A virus is suspectedbased on current clinical andepidemiological screening criteriarecommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novelvirulent Influenza viruses and sentto a state or local health departmentfor testing. Viral culture shouldnot be attempted in these casesunless a BSL 3+ facility isavailable to receive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Influenza A, Influenza A H1Seasonal Subtype, Influenza A H3Seasonal Subtype, Influenza A2009 H1N1 strain, HumanMetapneumovirus, HumanRhinovirus</td><td colspan="1" rowspan="1">Same except for the differenceslisted in the "Differences" tablerow 1.</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Nasopharyngeal swabs (NPS)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal control added to eachsample. External control processedwith each batch of samples.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay type</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Extraction</td><td colspan="1" rowspan="1">EasyMag® extraction system</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">eSensor Respiratory ViralPanel (RVP)</td><td colspan="1" rowspan="1">Luminex xTAG RVP</td></tr><tr><td colspan="1" rowspan="1">Detectiontechnology</td><td colspan="1" rowspan="1">eSensor XT-8Micro-array hybridization and solid Iphase electrochemical detection</td><td colspan="1" rowspan="1">Luminex® 100/200TM instrumentFluorescence-activated sorting oflabeled beads coupled tostreptavidin-conjugatedbiotinylated products</td></tr><tr><td colspan="1" rowspan="1">Software</td><td colspan="1" rowspan="1">Application software andembedded firmware (controlshardware functions) on XT-8 inaddition to Assay Analysis Module(AAM) for RVP-IVD</td><td colspan="1" rowspan="1">IS or xPONENT software; xTAGData Analysis Software RVP (US)</td></tr><tr><td colspan="1" rowspan="1">Time toresult</td><td colspan="1" rowspan="1">Approximately 6 hours</td><td colspan="1" rowspan="1">Approximately 8 hours</td></tr><tr><td colspan="1" rowspan="1">Assay callmethod</td><td colspan="1" rowspan="1">Automated test interpretation andreport generation. User can access</td><td colspan="1" rowspan="1">Semi-automated test interpretation.User must review all "no call"</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">raw data.</td><td colspan="1" rowspan="1">results to determine cause andretesting strategy.</td></tr></table>

# K. Standard/Guidance Documents Referenced:

User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute Approved Guideline, EP12-A2   
· Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005) Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses (July 15, 2011) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays (October 9, 2009) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 9, 2009) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays (October 9, 2009)   
· Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems (March 10, 2005) Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999)

# L. Test Principle:

The eSensor RVP is a multiplex microarray-based genotyping test system that accepts a nasopharyngeal specimen. It is based on the principles of competitive DNA hybridization using a sandwich assay format, wherein a single stranded target binds concurrently to sequence-specific solution-phase signal probe and solid-phase electrode-bound capture probe. The test employs reverse transcription polymerase chain reaction amplification, exonuclease digestion and hybridization of target DNA/RNA. In the process, the doublestranded PCR amplicons are digested with exonuclease to generate single stranded DNA suitable for hybridization. Hybridization occurs in the eSensor XT-8 Cartridge (described below) where the single-stranded target DNA is mixed with a hybridization solution containing labeled signal probes.

eSensor technology uses a solid-phase electrochemical method for determining the presence of one or more of a defined panel of virus target sequences. Purified DNA/RNA is isolated from the patient specimen and the extracted nucleic acid is reverse transcribed and/or amplified using virus specific primers with an RT-PCR enzyme mix. The amplified DNA is converted to single-stranded DNA via exonuclease digestion and is then combined with a signal buffer containing ferrocene labeled signal probes that are specific for the different viral targets. The mixture of amplified sample and signal buffer is loaded onto a cartridge containing single-stranded oligonucleotide capture probes bound to gold-plated electrodes. The cartridge is inserted into the XT-8 instrument where the single-stranded targets hybridize to the complementary sequences of the capture probes and signal probes. The presence of each target is determined by voltammetry, which generates specific electrical signals from the ferrocene-labeled signal probe. The eSensor RVP provides a qualitative result, the presence (Positive) or absence (Target Not Detected) of the viruses contained in the panel, along with the internal MS2 control, based upon whether the underlying electrical signals are above or below a pre-defined cut-off signal intensity.

Results are interpreted based on the following tables:

<table><tr><td rowspan=1 colspan=1>Virus Result</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>The run was successfully completedANDThe analyte indicated as POSITIVE was detected.</td><td rowspan=1 colspan=1>Report results</td></tr><tr><td rowspan=1 colspan=1>Target Not Detected</td><td rowspan=1 colspan=1>The run was successfully completedANDThe internal control or at least one other analytewas detectedANDThe result for the analyte indicated wasNEGATIVE</td><td rowspan=1 colspan=1>Report results</td></tr><tr><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Electrode or instrument failure</td><td rowspan=4 colspan=1>Retest sample. SeeTable Below: Summaryof Sample RetestRecommendations byReport Type</td></tr><tr><td rowspan=1 colspan=1>Fail(Internal ControlFailure</td><td rowspan=1 colspan=1>The internal control was unsuccessfulORThe run was unsuccessful</td></tr><tr><td rowspan=1 colspan=1>Indeterminate resultsbetween Influenza Asubtypes(Influenza A only, nosubtype)</td><td rowspan=1 colspan=1>The run was successfully completedANDThe result for Influenza A was POSITIVEANDThe results for Influenza A H1, H3 and 2009H1N1 were NEGATIVE</td></tr><tr><td rowspan=1 colspan=1>Target Not Detected(for external positivecontrol)</td><td rowspan=1 colspan=1>The run was successfully completedANDThe internal control or at least one other analytewas detectedANDThe results for the external positive controls wereNEGATIVE</td></tr></table>

The table below describes the interpretation of possible eSensor RVP results for Influenza A.

<table><tr><td colspan="1" rowspan="1">AssayFinal Result</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">2009 H1N1</td><td colspan="1" rowspan="1">RequiredFollow-up</td></tr><tr><td colspan="1" rowspan="1">Target Not Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="4">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="2">Influenza A 2009 H1N1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Retest to</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">confirmresult,see Table 8</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1 andInfluenza A H3</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="3">Multiple FluA subtypeinfections arepossible butrare. Retestto confirmresult</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1 andInfluenza A 2009 H1N1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3 andInfluenza A 2009 H1N1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Influenza A (no subtypedetected)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">See below</td></tr></table>

# For Influenza A (no subtype detected)

If the Influenza A assay is positive, but none of the hemagglutinin subtyping assays are positive, then the interpretation is ‘Indeterminate result between Influenza A subtypes (no subtype detected)’. This result could occur when the titer of the virus in the specimen is low and not detected by the subtyping assays. This result could also indicate the presence of a novel Influenza A strain. In both cases, the sample in question should be retested. If the retest provides a different result, test the sample a third time to ensure the accuracy of the result. If the retest provides the same result, then the function of the device should be verified by testing with appropriate external control materials (known positive samples for Influenza A H1, Influenza A H3 and Influenza A 2009 H1N1), and a negative control reaction should also be run to test for PCR-product contamination. If the eSensor RVP accurately identifies the external and negative controls, contact local or state public health authorities for confirmatory testing.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Repeatability was evaluated using representative cultures for each viral panel member (see table below) at a concentration of 1x LoD. The study was executed at a single site by a single operator on a single day. One representative culture was spiked into viral transport media then extracted using the NucliSENS easyMAG system. A total of 20 extraction replicates we run per analyte. Viral strains used in the repeatability study are provided below:

<table><tr><td rowspan=1 colspan=1>TARGET</td><td rowspan=1 colspan=1>STRAIN</td><td rowspan=1 colspan=1>SOURCE</td><td rowspan=1 colspan=1>LOT</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>H1N1 Brisbane/59/07</td><td rowspan=1 colspan=1>Zeptometrix 0810036CF</td><td rowspan=1 colspan=1>305254</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>ATCC VR-547</td><td rowspan=1 colspan=1>7683464</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>NY/2009</td><td rowspan=1 colspan=1>Zeptometrix 080109CFN</td><td rowspan=1 colspan=1>305984</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>Zeptometrix 0810037CF</td><td rowspan=1 colspan=1>305445</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>58224956</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>58189459</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>58023566</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>5006417</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>58102987</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>Zeptometrix 0810061CFN</td><td rowspan=1 colspan=1>305462</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC VR-483</td><td rowspan=1 colspan=1>2W</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 7</td><td rowspan=1 colspan=1>ATCC VR-7</td><td rowspan=1 colspan=1>7578661</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>ATCC VR-1</td><td rowspan=1 colspan=1>4703343</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E</td><td rowspan=1 colspan=1>Type 4</td><td rowspan=1 colspan=1>ATCC VR-1572</td><td rowspan=1 colspan=1>7588003</td></tr></table>

The repeatability study results are presented below:

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Viral TiterTested(TCID50/mL)</td><td rowspan=1 colspan=1>%Positives</td><td rowspan=1 colspan=1>MeanSignal(nA)</td><td rowspan=1 colspan=1>StandardDeviation(nA)</td><td rowspan=1 colspan=1>%CV(SD/Mean)</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H1N1Brisbane/59/07</td><td rowspan=1 colspan=1>4.17E-01</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>253.6</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>9.2</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>H1N1Brisbane/59/07</td><td rowspan=1 colspan=1>4.17E-01</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>212.1</td><td rowspan=1 colspan=1>14.0</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58E+03</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>264.1</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>8.1</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58E+03</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>50.9</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>20.9</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009H1N1</td><td rowspan=1 colspan=1>NY/2009</td><td rowspan=1 colspan=1>1.05E-01</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>43.2</td><td rowspan=1 colspan=1>21.1</td><td rowspan=1 colspan=1>48.0</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>3.16E-02</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.5</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>36.7</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.1</td><td rowspan=1 colspan=1>17.4</td><td rowspan=1 colspan=1>40.4</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.58E-03</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>43.8</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>54.1</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>2.81E-03</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>200.5</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>8.3</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>2.81E-01</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>259.0</td><td rowspan=1 colspan=1>68.6</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.81E+01</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.6</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>48.1</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>2.81E+00</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>79.3</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>1.58E+00</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>158.8</td><td rowspan=1 colspan=1>22.3</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>ADV B/E</td><td rowspan=1 colspan=1>Type 4</td><td rowspan=1 colspan=1>1.58E+00</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>147.6</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>16.3</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>8.89E-01</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>196.3</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>7.5</td></tr></table>

# Reproducibility

A multisite reproducibility study was carried out to evaluate the reproducibility of the eSensor RVP using a representative subset of targeted analytes. Sources of variation due to lot, run, day, site and operator were included in the study design.

<table><tr><td rowspan=1 colspan=1>Variance Component</td><td rowspan=1 colspan=1>Instances</td></tr><tr><td rowspan=1 colspan=1>Instruments</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Sites</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Users Per Site</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Reagent Lots</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Non-consecutive Days ofTesting</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Concentrations TestedPer Analyte</td><td rowspan=1 colspan=1>3x LoD, 1x LoD,negative</td></tr></table>

The following five viral strains were selected for the multisite reproducibility study:

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>StockConcentration(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>Aichi/2/68/H3N2</td><td rowspan=1 colspan=1>ATCC, VR-547</td><td rowspan=1 colspan=1>58167241</td><td rowspan=1 colspan=1>1.60E+08</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC, VR-1540</td><td rowspan=1 colspan=1>58224956</td><td rowspan=1 colspan=1>2.80E+06</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>ATCC, VR-93</td><td rowspan=1 colspan=1>58102987</td><td rowspan=1 colspan=1>2.80E+07</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>Zeptometrix,810159CF</td><td rowspan=1 colspan=1>307012</td><td rowspan=1 colspan=1>5.01E+05</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>Type 7</td><td rowspan=1 colspan=1>ATCC, VR-7</td><td rowspan=1 colspan=1>7578661</td><td rowspan=1 colspan=1>8.90E+05</td></tr></table>

A culture stock of each selected virus was diluted in VTM to construct the positive samples of the reproducibility study panel. The positive samples of the reproducibility study panel were prepared so that each viral strain would be tested at two concentrations: a “low positive” $\mathrm { C } _ { 9 5 }$ concentration, expected to be positive approximately $9 5 \%$ of the time), and a “moderate positive” $( 3 \mathrm { ~ x ~ L o D }$ , expected to be positive approximately $100 \%$ of the time). The positive samples for a particular virus strain also served as “negative” samples for the other four virus strains in the reproducibility study panel. All reproducibility study panel samples were randomized and blinded prior to shipment to the study sites.

Summary reproducibility study results are presented in the tables below:

ADV B/E Results   

<table><tr><td rowspan=1 colspan=1>ADV BConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD)47.4TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>109.1</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>10.0</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>102.6</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>11.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>102.2</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>104.7</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD)15.8TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>92.7</td><td rowspan=1 colspan=1>11.4</td><td rowspan=1 colspan=1>12.3</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>89.9</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>16.9</td><td rowspan=1 colspan=1>20.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>89.1</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>15.3</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>1/288</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1%-100%</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>1/864</td><td rowspan=1 colspan=1>863/864</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.4%-100%</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Influenza A Results

<table><tr><td rowspan=1 colspan=1>Flu AConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD)1.3 TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>243.0</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>9.7</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>246.4</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>12.0</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>235.0</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>241.5</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>12.0</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD)0.4 TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>248.3</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>11.5</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>244.7</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>10.8</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>232.8</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>10.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>242.0</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>11.1</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>3/288</td><td rowspan=1 colspan=1>285/288</td><td rowspan=1 colspan=1>99.0%</td><td rowspan=1 colspan=1>97.0%-99.8%</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>1/288</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1%-100%</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>2/288</td><td rowspan=1 colspan=1>286/288</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>97.5%-99.9%</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>6/864</td><td rowspan=1 colspan=1>858/864</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>98.5%-99.7%</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# Influenza A H3 Results

<table><tr><td colspan="1" rowspan="1">Flu A H3Concentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResults</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Mean(nA)</td><td colspan="1" rowspan="1">StdDev</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="4">MOD POS(3x LoD)4.7 x 103TCID50/ml</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">86.5</td><td colspan="1" rowspan="1">23.9</td><td colspan="1" rowspan="1">27.7</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">77.4</td><td colspan="1" rowspan="1">26.7</td><td colspan="1" rowspan="1">34.5</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">86.0</td><td colspan="1" rowspan="1">30.5</td><td colspan="1" rowspan="1">35.4</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">0/108</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.6%-100%</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">27.3</td><td colspan="1" rowspan="1">32.7</td></tr><tr><td colspan="1" rowspan="4">LOW POS(1x LoD)1.6 x 103TCID50/ml</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">81.5</td><td colspan="1" rowspan="1">26.1</td><td colspan="1" rowspan="1">32.0</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">68.2</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">43.4</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">84.9</td><td colspan="1" rowspan="1">21.3</td><td colspan="1" rowspan="1">25.1</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">0/108</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.6%-100%</td><td colspan="1" rowspan="1">78.2</td><td colspan="1" rowspan="1">26.6</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">0/288</td><td colspan="1" rowspan="1">288/288</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.7%-100%</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">0/288</td><td colspan="1" rowspan="1">288/288</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.7%-100%</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">1/288</td><td colspan="1" rowspan="1">287/288</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98.1%-100%</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">1/864</td><td colspan="1" rowspan="1">863/864</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99.4%-100%</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">RSV AConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResults</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">Mean(nA)</td><td colspan="1" rowspan="1">StdDev</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="4">MOD POS(3x LoD)8.4 TCID50/ml</td><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">166.3</td><td colspan="1" rowspan="1">19.2</td><td colspan="1" rowspan="1">11.5</td></tr><tr><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">156.7</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">20.2</td></tr><tr><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">156.4</td><td colspan="1" rowspan="1">22.9</td><td colspan="1" rowspan="1">14.7</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">0/108</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">96.6%-100%</td><td colspan="1" rowspan="1">159.8</td><td colspan="1" rowspan="1">25.3</td><td colspan="1" rowspan="1">15.8</td></tr><tr><td colspan="1" rowspan="4">LOW POS(1x LoD)2.8 TCID50/ml</td><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">146.6</td><td colspan="1" rowspan="1">22.7</td><td colspan="1" rowspan="1">15.5</td></tr><tr><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">0/36</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">90.3%-100%</td><td colspan="1" rowspan="1">124.6</td><td colspan="1" rowspan="1">41.0</td><td colspan="1" rowspan="1">32.9</td></tr><tr><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">1/36</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">85.5%-99.9%</td><td colspan="1" rowspan="1">128.2</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">26.1</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">1/108</td><td colspan="1" rowspan="1">99.1%</td><td colspan="1" rowspan="1">94.9%-100%</td><td colspan="1" rowspan="1">133.1</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">25.8</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">4/288</td><td colspan="1" rowspan="1">284/288</td><td colspan="1" rowspan="1">98.6%</td><td colspan="1" rowspan="1">96.5%-99.6%</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">0/288</td><td colspan="1" rowspan="1">288/288</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.7%-100%</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">0/288</td><td colspan="1" rowspan="1">288/288</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.7%-100%</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">4/864</td><td colspan="1" rowspan="1">860/864</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">98.8%-99.9%</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">N/A</td></tr></table>

Human Metapneumovirus Results   

<table><tr><td rowspan=1 colspan=1>hMPVConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD)13 TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>91.2</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>28.6</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>40.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>100.4</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>22.6</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>30.9</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD)4 TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>56.4</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>53.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>1/36</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>85.5%-99.9%</td><td rowspan=1 colspan=1>51.0</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>61.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>63.8</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>44.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>107/108</td><td rowspan=1 colspan=1>1/108</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>94.9%-100%</td><td rowspan=1 colspan=1>57.1</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>52.5</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>8/288</td><td rowspan=1 colspan=1>280/288</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>94.6%-98.8%</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>8/864</td><td rowspan=1 colspan=1>856/864</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>98.2%-99.6%</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Parainfluenza Virus 3 Results   

<table><tr><td rowspan=1 colspan=1>PIV3Concentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResults</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean(nA)</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=4 colspan=1>MOD POS(3x LoD)84 TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>178.1</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>15.4</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>193.8</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>15.4</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>160.8</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>17.0</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>177.6</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>17.5</td></tr><tr><td rowspan=4 colspan=1>LOW POS(1x LoD)28 TCID50/ml</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>139.0</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>25.1</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>162.4</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>17.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>0/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3%-100%</td><td rowspan=1 colspan=1>126.9</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>30.7</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>0/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6%-100%</td><td rowspan=1 colspan=1>142.8</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>0/288</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/864</td><td rowspan=1 colspan=1>864/864</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6%-100%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td></tr></table>

An additional reproducibility study was performed to supplement the original study. The new study was requested to provide at least 90 replicates per analyte per concentration and to include data for lot to lot variance evaluation. The additional study was performed at three sites to evaluate the following sources of variation:

Site instrument to site instrument · Operator run to operator run · Extraction to extraction · Day to day · Lot to Lot

A schematic of the study design is shown below:

![](images/40d01fdf70b92edf5dd7fe36cd339b85e243323896a2c0f1a2188af0bae6750a.jpg)

Although the “Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay” indicates that every claimed analyte should be represented by the reproducibility study samples, the Agencies thinking regarding the reproducibility study design for multiplexed microbiology device is evolving based on the feedback we obtained from the FDA public meeting “Advancing Regulatory Science for Highly Multiplexed Microbiology/MCM Devices held on October 13, 2011.” Instead of including every claimed analyte in the additional reproducibility study again, testing a representative number of claimed analytes was assessed to be sufficient in this case. A list of representative analytes was reviewed prior to initiation of the additional study. The five representative analytes used in the study are listed below:

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>Aichi/2/68/H3N2</td><td rowspan=1 colspan=1>ATCC, VR-547</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC, VR-1540</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>ATCC,VR-93</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>Zeptometrix,810159CF</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>Type 7</td><td rowspan=1 colspan=1>ATCC, VR-7</td></tr></table>

The detailed numerical results from each analyte concentration level were also reported and the variance calculations, based on the raw data, are presented below for each of the variance components included in the study.

<table><tr><td rowspan=1 colspan=4>Adenovirus B</td><td rowspan=1 colspan=5>Influenza A (H3N2)</td></tr><tr><td rowspan=1 colspan=4>Low Positive</td><td rowspan=1 colspan=5>Low Positive</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>89.06</td><td rowspan=1 colspan=1>13.66</td><td rowspan=1 colspan=1>15.34</td><td rowspan=1 colspan=2>108</td><td rowspan=1 colspan=1>241.95</td><td rowspan=1 colspan=1>26.74</td><td rowspan=1 colspan=1>11.05</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=3>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>2.31</td><td></td><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>0.58</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>6.04</td><td rowspan=1 colspan=1>6.78</td><td></td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>13.86</td><td rowspan=1 colspan=1>5.73</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>3.07</td><td rowspan=1 colspan=1>3.45</td><td></td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>5.23</td><td rowspan=1 colspan=1>2.16</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>3.96</td><td rowspan=1 colspan=1>4.44</td><td></td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>9.60</td><td rowspan=1 colspan=1>3.97</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>11.71</td><td rowspan=1 colspan=1>13.15</td><td></td><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>21.92</td><td rowspan=1 colspan=1>9.06</td></tr><tr><td rowspan=1 colspan=4>Moderate Positive</td><td rowspan=1 colspan=5>Moderate Positive</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>104.65</td><td rowspan=1 colspan=1>12.67</td><td rowspan=1 colspan=1>12.10</td><td></td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>241.47</td><td rowspan=1 colspan=1>28.99</td><td rowspan=1 colspan=1>12.00</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=3>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>3.30</td><td rowspan=1 colspan=1>3.16</td><td rowspan=1 colspan=3>Site</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>4.57</td><td rowspan=1 colspan=1>4.36</td><td rowspan=1 colspan=3>Lot</td><td rowspan=1 colspan=1>6.35</td><td rowspan=1 colspan=1>2.63</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=3>Day</td><td rowspan=1 colspan=1>9.94</td><td rowspan=1 colspan=1>4.12</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=3>Operator</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>11.76</td><td rowspan=1 colspan=1>11.24</td><td rowspan=1 colspan=3>ResidualVariance</td><td rowspan=1 colspan=1>26.81</td><td rowspan=1 colspan=1>11.10</td></tr><tr><td rowspan=1 colspan=4>Negative</td><td rowspan=1 colspan=5>Negative</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=8 colspan=1></td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>864   1.09</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=3>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>0.00</td><td></td><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>0.00</td><td></td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>0.11</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>0.00</td><td></td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>0.17</td><td></td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>3.46</td><td></td><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>1.94</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Parainfluenza Virus 3</td></tr><tr><td rowspan=1 colspan=4>Low Positive</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>142.77</td><td rowspan=1 colspan=1>36.98</td><td rowspan=1 colspan=1>25.90</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>14.86</td><td rowspan=1 colspan=1>10.41</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>3.97</td><td rowspan=1 colspan=1>2.78</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>19.65</td><td rowspan=1 colspan=1>13.76</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>6.20</td><td rowspan=1 colspan=1>4.34</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>28.43</td><td rowspan=1 colspan=1>19.91</td></tr><tr><td rowspan=1 colspan=4>Moderate Positive</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>177.57</td><td rowspan=1 colspan=1>31.07</td><td rowspan=1 colspan=1>17.50</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>13.83</td><td rowspan=1 colspan=1>7.79</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>15.41</td><td rowspan=1 colspan=1>6.68</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>7.31</td><td rowspan=1 colspan=1>4.12</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>23.78</td><td rowspan=1 colspan=1>13.39</td></tr><tr><td rowspan=1 colspan=4>Negative</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>0.05</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>0.12</td></tr></table>

<table><tr><td colspan="4" rowspan="1">Respiratory Syncytial Virus A</td></tr><tr><td colspan="4" rowspan="1">Low Positive</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">GrandMean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">133.14</td><td colspan="1" rowspan="1">34.29</td><td colspan="1" rowspan="1">25.76</td></tr><tr><td colspan="2" rowspan="1">VarianceComponent</td><td colspan="1" rowspan="1">SD estimate</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="2" rowspan="1">Site</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="2" rowspan="1">Lot</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">1.66</td></tr><tr><td colspan="2" rowspan="1">Day</td><td colspan="1" rowspan="1">3.89</td><td colspan="1" rowspan="1">6.02</td></tr><tr><td colspan="2" rowspan="1">Operator</td><td colspan="1" rowspan="1">16.15</td><td colspan="1" rowspan="1">9.03</td></tr><tr><td colspan="2" rowspan="1">ResidualVariance</td><td colspan="1" rowspan="1">30.68</td><td colspan="1" rowspan="1">12.28</td></tr><tr><td colspan="4" rowspan="1">Moderate Positive</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">GrandMean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">159.82</td><td colspan="1" rowspan="1">25.32</td><td colspan="1" rowspan="1">15.84</td></tr><tr><td colspan="2" rowspan="1">VarianceComponent</td><td colspan="1" rowspan="1">SD estimate</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="2" rowspan="1">Site</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="2" rowspan="1">Lot</td><td colspan="1" rowspan="1">2.66</td><td colspan="1" rowspan="1">1.66</td></tr><tr><td colspan="2" rowspan="1">Day</td><td colspan="1" rowspan="1">9.63</td><td colspan="1" rowspan="1">6.02</td></tr><tr><td colspan="2" rowspan="1">Operator</td><td colspan="1" rowspan="1">12.43</td><td colspan="1" rowspan="1">9.03</td></tr><tr><td colspan="2" rowspan="1">ResidualVariance</td><td colspan="1" rowspan="1">19.63</td><td colspan="1" rowspan="1">12.28</td></tr><tr><td colspan="4" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">GrandMean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="8"></td></tr><tr><td colspan="1" rowspan="1">864</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="2" rowspan="1">VarianceComponent</td><td colspan="1" rowspan="1">SD estimate</td></tr><tr><td colspan="2" rowspan="1">Site</td><td colspan="1" rowspan="1">0.18</td></tr><tr><td colspan="2" rowspan="1">Lot</td><td colspan="1" rowspan="1">0.07</td></tr><tr><td colspan="2" rowspan="1">Day</td><td colspan="1" rowspan="1">0.27</td></tr><tr><td colspan="2" rowspan="1">Operator</td><td colspan="1" rowspan="1">0.06</td></tr><tr><td colspan="2" rowspan="1">ResidualVariance</td><td colspan="1" rowspan="1">2.31</td></tr><tr><td colspan="4" rowspan="1">Human Metapneumovirus</td></tr><tr><td colspan="4" rowspan="1">Low Positive</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">GrandMean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">57.10</td><td colspan="1" rowspan="1">29.99</td><td colspan="1" rowspan="1">52.52</td></tr><tr><td colspan="2" rowspan="1">VarianceComponent</td><td colspan="1" rowspan="1">SD estimate</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="2" rowspan="1">Site</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="2" rowspan="1">Lot</td><td colspan="1" rowspan="1">22.17</td><td colspan="1" rowspan="1">38.82</td></tr><tr><td colspan="2" rowspan="1">Day</td><td colspan="1" rowspan="1">7.11</td><td colspan="1" rowspan="1">12.46</td></tr><tr><td colspan="2" rowspan="1">Operator</td><td colspan="1" rowspan="1">10.07</td><td colspan="1" rowspan="1">17.64</td></tr><tr><td colspan="2" rowspan="1">ResidualVariance</td><td colspan="1" rowspan="1">20.94</td><td colspan="1" rowspan="1">36.67</td></tr><tr><td colspan="4" rowspan="1">Moderate Positive</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">GrandMean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">94.72</td><td colspan="1" rowspan="1">29.26</td><td colspan="1" rowspan="1">30.89</td></tr><tr><td colspan="2" rowspan="1">VarianceComponent</td><td colspan="1" rowspan="1">SD estimate</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="2" rowspan="1">Site</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Lot</td><td colspan="1" rowspan="1">17.02</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Day</td><td colspan="1" rowspan="1">10.81</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Operator</td><td colspan="1" rowspan="1">17.23</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">ResidualVariance</td><td colspan="1" rowspan="1">17.87</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">GrandMean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="8"></td></tr><tr><td colspan="1" rowspan="1">864</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="2" rowspan="1">VarianceComponent</td><td colspan="1" rowspan="1">SD estimate</td></tr><tr><td colspan="2" rowspan="1">Site</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="2" rowspan="1">Lot</td><td colspan="1" rowspan="1">0.07</td></tr><tr><td colspan="2" rowspan="1">Day</td><td colspan="1" rowspan="1">1.21</td></tr><tr><td colspan="2" rowspan="1">Operator</td><td colspan="1" rowspan="1">0.46</td></tr><tr><td colspan="2" rowspan="1">ResidualVariance</td><td colspan="1" rowspan="1">2.01</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Influenza A H3</td></tr><tr><td rowspan=1 colspan=4>Low Positive</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>78.16</td><td rowspan=1 colspan=1>26.64</td><td rowspan=1 colspan=1>34.08</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>7.36</td><td rowspan=1 colspan=1>9.41</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>13.78</td><td rowspan=1 colspan=1>17.63</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>7.69</td><td rowspan=1 colspan=1>9.84</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>1.31</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>22.09</td><td rowspan=1 colspan=1>28.26</td></tr><tr><td rowspan=1 colspan=4>Moderate Positive</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>83.32</td><td rowspan=1 colspan=1>27.25</td><td rowspan=1 colspan=1>32.71</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>1.61</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>14.67</td><td rowspan=1 colspan=1>17.60</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>7.42</td><td rowspan=1 colspan=1>8.91</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>23.33</td><td rowspan=1 colspan=1>28.00</td></tr><tr><td rowspan=1 colspan=4>Negative</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>GrandMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>VarianceComponent</td><td rowspan=1 colspan=1>SD estimate</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=1>0.18</td></tr><tr><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=1>0.02</td></tr><tr><td rowspan=1 colspan=2>ResidualVariance</td><td rowspan=1 colspan=1>0.32</td></tr></table>

# b. Linearity/assay reportable range:

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Assay Controls

Negative Control

A negative control is not included in the eSensor RVP kit but should be included with each viral nucleic acid isolation run, and with each eSensor RVP run. RNase/DNase-free water should be used as the control sample (blank transport media may also be used). These negative controls are to be run through the assay process from viral nucleic acid isolation to RT-PCR and XT-8 detection. The results are identified as the negative controls on the XT-8 reports and the software will determine whether the negative controls are valid. If a negative control fails, all results of the samples in the same run are invalid and the samples should be re-extracted and re-run.

# External Controls (Positive Controls for Viral Targets)

External positive controls are not provided in the eSensor RVP kit. However, known strains of the targeted viruses should be included in routine quality control procedures (“external controls”). These external controls are viral target positive controls and should be included with each batch of patient specimens. These controls may be previously characterized positive samples or obtained commercially. External controls should be prepared and tested in the same manner as patient samples. Results from external controls should be examined before the results from the patient samples. If an external positive control dose not perform as expected, all results of the batch of samples are invalid and the samples should be re-run.

# Internal Control (IC)

An internal control (Bacteriophage MS2; IC) is supplied. This allows the user to control for the viral nucleic acid isolation procedure and to check for possible RT-PCR inhibition. IC must be added to each sample during viral nucleic acid isolation; this includes patient samples, negative controls, and external viral controls. The XT-8 will analyze samples for IC signal. Failure to detect IC signal in a sample or a control, in the absence of signal from any of the RVP analytes, will be identified and the user will be required to retest the sample. Specific instructions on adding the IC to a sample for extraction are detailed in the “Viral Nucleic Acid Isolation” section of the product package insert.

# Reagent Kit Stability

An analytical study was performed to establish stability of the eSensor RVP assay components. Functional performance of RVP reagents were evaluated with three stability test panels (IVT-1, IVT-2, IVT-3) composed of in vitro transcripts. Each stability test panel consisted of a unique combination of claimed viruses, so that the three stability test panels covered all the claimed viruses. All viruses in a particular stability test panel were tested at least once during each stability study. Virus targets were present at a concentration of $2 . 0 \mathrm { x }$ $1 0 ^ { 4 }$ copies/ $\mu \mathrm { l }$ . Since each viral target has a unique LoD concentration, a concentration of 2.0 $\mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies/ul represents a range of $1 0 \mathrm { x }$ to $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { { x } }$ LoD.

<table><tr><td rowspan=1 colspan=1>Viral Target</td><td rowspan=1 colspan=1>IVT-1(RVPSet 1)</td><td rowspan=1 colspan=1>IVT-2(RVPSet 2)</td><td rowspan=1 colspan=1>IVT-3(RVPSet 3)</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009H1N1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total positivetargets per RVPSet</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td></tr></table>

Kits were stored according to the package insert $( 1 0 - 2 5 ^ { \circ } \mathrm { C } )$ for up to 6 months. Testing was performed after time 0, 3 and 6 months of storage. One test is defined as the extraction of one sample using the NucliSENS EasyMag kit, amplification and analysis on the eSensor. A total of 16 tests were performed per time point and all three sets IVT panels were tested along with a negative control at each time point. A successful time point was determined as a statistically equivalent number of correct calls between the time point tested and the data from time 0. The Fisher’s exact test for independence was used to determine statistical equivalence.

Shelf life was determined by acceptable performance at the last time point tested. Acceptable performance was defined as no statistical difference in correct call rates between the particular time point data and the data at time zero. Final results for the reagent stability studies are shown below.

<table><tr><td rowspan=3 colspan=1>Timepoint</td><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>T1</td><td rowspan=1 colspan=3>T2</td></tr><tr><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=3>3 Months</td><td rowspan=1 colspan=3>6 Months</td></tr><tr><td rowspan=1 colspan=1>%CorrectCalls(95%CI)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>%CorrectCalls(95%CI)**</td><td rowspan=1 colspan=1># ofNoCallS</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>%CorrectCalls(95%CI)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>MasterLot 1</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106/108=98.1(93.5,99.5)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MasterLot 2</td><td rowspan=1 colspan=1>105/108=97.2(92.1,99.0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>104/108=96.3(90.9,98.6)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>105/108=97.2(92.1,99.0)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MasterLot 3</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>319/324=98.5(96.4,99.3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>317/324=97.8(95.6,98.9)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>319/324=98.5(96.4,99.3)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=3 colspan=1>Timepoint</td><td rowspan=1 colspan=3>T3</td><td rowspan=1 colspan=3>T4</td><td rowspan=1 colspan=3>T5</td></tr><tr><td rowspan=1 colspan=2>9 Months</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>12 Months</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>15 Months</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>%CorrectCalls(95%CI)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>%CorrectCalls(95%CI)**</td><td rowspan=1 colspan=1># ofNoCallS</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>%CorrectCalls(95%CI)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>MasterLot 1</td><td rowspan=1 colspan=1>99/108=91.7(84.9,95.5)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>MasterLot 2</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105/108=97.2(92.1,99.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>MasterLot 3</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>314/324=96.9(94.4,98.3</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>323/324=99.7(98.3,99.9)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>319/324=98.5(96.4,99.3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr></table>

The no call and miscall results were attributed to an electrical problem with the RVP cartridge/XT-8 interface. The electrical failures are not reagent or amplicon related.

A statistical analysis of the data was performed using the Fisher’s exact test method. There were no statistically significant $( \mathbf { p } > 0 . 0 5 )$ difference in the call rates up to time point T2.

These data are acceptable to support reagent kit stability claims as published in the product labeling. The product labeling states:

<table><tr><td rowspan=1 colspan=1>Box</td><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Packaging &amp; Quantity</td><td rowspan=1 colspan=1>Storage</td></tr><tr><td rowspan=2 colspan=1>eSensor RespiratoryViral PanelCartridges</td><td rowspan=1 colspan=1>eSensor RVPCartridges</td><td rowspan=1 colspan=1>6 foil bags with 8cartridges in each foil bag</td><td rowspan=1 colspan=1>10-25 </td></tr><tr><td rowspan=1 colspan=1>eSensor RespiratoryViral PanelPackage Insert</td><td rowspan=1 colspan=1>1 copy</td><td rowspan=1 colspan=1>Dry place(retain forreference)</td></tr><tr><td rowspan=3 colspan=1>eSensor RespiratoryViral PanelAmplificationReagents</td><td rowspan=1 colspan=1>RVP Enzyme Mix</td><td rowspan=1 colspan=1>2 vials with 40 µL each</td><td rowspan=3 colspan=1>-15 to -30 (in a designatedpre-PCR location)</td></tr><tr><td rowspan=1 colspan=1>RVP PCR Mix</td><td rowspan=1 colspan=1>2 vials with 1000 µL each</td></tr><tr><td rowspan=1 colspan=1>MS2 Internal Control</td><td rowspan=1 colspan=1>2 vials with 300 μL each</td></tr><tr><td rowspan=4 colspan=1>eSensor RespiratoryViral Panel DetectionReagents</td><td rowspan=1 colspan=1>RVP Signal Buffer</td><td rowspan=1 colspan=1>2 vials with 2200μL each</td><td rowspan=4 colspan=1>-15 to -30 °(in a designatedpost-PCR location)</td></tr><tr><td rowspan=1 colspan=1>Exonuclease</td><td rowspan=1 colspan=1>2 vials with 145 μL each</td></tr><tr><td rowspan=1 colspan=1>Buffer-1</td><td rowspan=1 colspan=1>2 vials with 350 μL each</td></tr><tr><td rowspan=1 colspan=1>Buffer-2</td><td rowspan=1 colspan=1>2 vials with 700 µL each</td></tr></table>

# Double Stranded Amplicon Stability

An analytical study was performed to establish performance for assay intermediates after storage. Functional performance was evaluated with three stability test panels (IVT-1, IVT2, IVT-3) of in vitro transcripts. Each stability test panel consisted of a unique combination of claimed viruses, so that the three stability test panels covered all the claimed viruses. All viruses in a particular stability test panel were tested at least once during each stability study. Virus targets were present at a concentration of $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies/ul. Since each viral target has a unique LoD concentration, a concentration of $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { \dot { 4 } }$ copies/ul represents a range of $1 0 \mathrm { x }$ to $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { x }$ LoD of an amplicon. Three sets of IVT viral panels were used which covered all analytes in the RVP (same sets as presented above). After amplification, all replicates of the same sample set were pooled and labeled. Aliquots from each RVP amplicon set were removed and stored according to the test plan in the figure below. The remaining volume was used for testing the $\mathrm { T 0 }$ time point. A schematic of the test plan is shown below.

<table><tr><td rowspan=1 colspan=1>Time Point</td><td rowspan=1 colspan=1>TestingCondition</td><td rowspan=1 colspan=1>Number ofCartridges</td><td rowspan=1 colspan=1>Amplicon Tested</td></tr><tr><td rowspan=1 colspan=1>TO(0 days)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T1_2-8°C(7 days)</td><td rowspan=2 colspan=1>2-8°C</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T2_2-8°C(9 days)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T1_&lt;-15°C(4 weeks)</td><td rowspan=2 colspan=1>&lt;-15°C</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T2_&lt;-15°C(5 weeks)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr></table>

After the specified storage time, aliquots were removed from storage, exonuclease digested and mixed with hybridization buffer prior to RVP cartridge detection. A summary of the performance at three time points using a $- 1 5$ to ${ } _ { - 3 0 } \mathrm { \Omega } ^ { \circ } \mathrm { C }$ storage condition are shown below:

<table><tr><td rowspan=2 colspan=1>SampleDetection</td><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>T1 (4 weeks)</td><td rowspan=1 colspan=3>T2 (5 weeks)</td></tr><tr><td rowspan=1 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls (95%C1)***</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0))</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=96.4(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39/40=100(91.2,100)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100 (51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>106/108=98.1(93.5,99.5)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td></tr></table>

The no call results at the 5 week time point were attributed to an electrical problem with the RVP cartridge/XT-8 interface. The electrical failures are not reagent or amplicon related.

A summary of the performance at three time points using a $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ storage condition are shown below:

<table><tr><td rowspan=2 colspan=1>SampleDetection</td><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>T1 (7 days)</td><td rowspan=1 colspan=3>T2 (9 days)</td></tr><tr><td rowspan=1 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls (95%C1)***</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0))</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=96.4(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40/40=100(91.2,100)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108/108=100(96.6, 100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108/108=100(96.6, 100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108/108=100(96.6, 100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

The time points tested exhibited no statistically significant difference in the positive detection rate. These data are acceptable to support amplicon stability claims as published in the product labeling. The product labeling states:

…store the amplified samples refrigerated at $2 \substack { - 8 \mathrm { \Omega } ^ { \circ } C }$ for up to 1 week or frozen at -15 to $- 3 0 ~ ^ { \circ } C$ for up to one month before processing.

# Single Stranded amplicon Stability

An analytical study was performed to establish performance for assay intermediates after storage. Functional performance was evaluated with three stability test panels (IVT-1, IVT2, IVT-3) of in vitro transcripts. Each stability test panel consisted of a unique combination of claimed viruses, so that the three stability test panels covered all the claimed viruses. All viruses in a particular stability test panel were tested at least once during each stability study. Virus targets were present at a concentration of $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies $\mathbf { \mu } ^ { \prime } \mathbf { \mu } \mathbf { l }$ . Since each viral target has a unique LoD concentration, a concentration of $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { \overline { { 4 } } }$ copies/ul represents a range of $1 0 \mathrm { x }$ to $1 \mathrm { ~ \ i ~ { ~ I ~ } ~ } 0 ^ { 7 } \mathrm { { x } }$ LoD of an amplicon. Three sets of IVT viral panels were used which covered all analytes in the RVP (same sets as presented above). Once amplification and exonuclease digestion were complete, all replicates from the same sample set were pooled and labeled. Aliquots from each RVP amplicon set were removed and stored according to the test plan in the figure below. The remaining volume was used for testing the T0 time point. Final time point performance will be considered acceptable is the number of correctly called samples is statistically equivalent to the T0 time point.

After amplification and exonuclease cleavage, all replicates of the same sample sets were pooled and labeled. Aliquots from each RVP amplicon set were removed and stored according to the test plan in the figure below. The remaining volume was used for testing the T0 time point. A schematic of the test plan is shown below.

<table><tr><td rowspan=1 colspan=1>Time Point</td><td rowspan=1 colspan=1>Condition</td><td rowspan=1 colspan=1>Number of Cartridges</td><td rowspan=1 colspan=1>Amplicon Tested</td></tr><tr><td rowspan=1 colspan=1>TO(0 days)</td><td rowspan=1 colspan=1>+2-8°CAnd&lt;15°C</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1RVP Set 2RVP Set 3RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T1_2-8°C(7 days)</td><td rowspan=2 colspan=1>+2-8°C</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1RVP Set 2RVP Set 3RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T2_2-8 RC(9 days)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1RVP Set 2RVP Set 3RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T1_&lt;-15 °C(4 weeks)</td><td rowspan=3 colspan=1>&lt;-15°C</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1RVP Set 2RVP Set 3RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T2_ &lt;-15 °C(5 weeks)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1RVP Set 2RVP Set 3RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T3_&lt;-15 °C(10 weeks) FIO</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1RVP Set 2RVP Set 3RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>Total Number of Cartridges Required</td><td rowspan=1 colspan=3>96</td></tr></table>

After the specified storage time, aliquots were removed from storage mixed with hybridization buffer prior to RVP cartridge detection. Performance was tested at three time points using two different storage conditions ( -15 to ${ } _ { - 3 0 } \mathrm { { } ^ { \circ } C }$ and 2 to $8 ^ { \circ } \mathrm { C }$ ) are shown below.

The overall detection rate for exonuclease cleaved amplicons stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ is shown below:

<table><tr><td rowspan=2 colspan=1>SampleDetection</td><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>T1 (7 days)</td><td rowspan=1 colspan=3>T2 (9 days)</td></tr><tr><td rowspan=1 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls (95%C1)***</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0))</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27/28=96.4(82.3,99.4)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39/40=97.5(87.1,99.6)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40/40=100(91.2,100)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td></tr></table>

No calls were a result of a software error and an electrical connection problem between the RVP cartridge and the XT-8 instrument. These failures were non-reagent or amplicon related.

The overall detection rate for exonuclease cleaved amplicons stored at -15 to $- 3 0 ^ { \circ } \mathrm { C }$ is shown

below:   

<table><tr><td rowspan=3 colspan=1>SampleDetection</td><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>T1 (4 weeks)</td><td rowspan=1 colspan=3>T2 (5 weeks)</td></tr><tr><td rowspan=2 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=1 colspan=1># of</td><td rowspan=1 colspan=1># of</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=1>#of</td><td rowspan=1 colspan=1>#of</td><td rowspan=2 colspan=1>% CorrectCalls (95%C1)**</td><td rowspan=2 colspan=1>#ofNoCalls</td><td rowspan=2 colspan=1>#ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Mis-Calls</td><td rowspan=1 colspan=1>Calls (95%C1)**</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Mis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27/28=96.4(82.3,99.4)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39/40=97.5(87.1,99.6)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>106/108=98.1(93.5,99.5)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108/108=100(96.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>X0</td></tr></table>

No calls were a result of an electrical connection problem between the RVP cartridge and the XT-8 instrument. These failures were non-reagent or amplicon related.

The time points tested exhibited no statistically significant difference in the positive detection rate. These data are acceptable to support exonuclease cleaved amplicon stability claims as published in the product labeling. The product labeling states:

Proceed immediately with the next step, or store the exonuclease-digested samples refrigerated at $2 \substack { - 8 \mathrm { \Omega } ^ { \circ } C }$ for up to 1 week or frozen at -15 to ${ } _ { - 3 0 } \ { } ^ { \circ } C$ for up to 1 month before processing.

# Open pouch stability

An analytical study was performed to establish performance of cartridges in previously opened storage pouches. Functional performance was evaluated with three stability test panels (IVT-1, IVT-2, IVT-3) of in vitro transcripts. Each stability test panel consisted of a unique combination of claimed viruses, so that the three stability test panels covered all the claimed viruses. All viruses in a particular stability test panel were tested at least once during each stability study. Virus targets were present at a concentration of $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies $\mu \mathrm { l }$ . Since each viral target has a unique LoD concentration, a concentration of $2 . 0 \mathrm { x }$ $1 0 ^ { \dot { 4 } }$ copies/ul represents a range of $1 0 \mathrm { x }$ to $\mathrm { 1 ~ \dot { x } ~ 1 0 ^ { 7 } x ~ L o D }$ of an amplicon. Three sets of IVT viral panels were used which covered all analytes in the RVP (same sets as presented above). At time $\mathrm { T 0 }$ all RVP cartridges were opened. All cartridges except those designated for T0 testing were stored in a re-sealable plastic bag and placed in controlled incubators for the scheduled storage time. After storage, cartridges were used to analyze test panels according to product labeling instructions. Final time point performance will be considered acceptable is the number of correctly called samples is statistically equivalent to the T0 time point. A schematic of the storage conditions and test time points is shown below:

<table><tr><td rowspan=1 colspan=1>Time Point and Condition</td><td rowspan=1 colspan=1>Number of Cartridges</td><td rowspan=1 colspan=1>Amplicon Tested</td></tr><tr><td rowspan=1 colspan=1>TO0 hours</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T12 weeks30 °C, Ambient Humidity</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T24 weeks30 °C, Ambient Humidity</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T38 weeks30 °C, Ambient Humidity</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>T49 weeks30 °C, Ambient Humidity</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; RVP Set 4</td></tr><tr><td rowspan=1 colspan=1>Total Number of Cartridges Required</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1></td></tr></table>

A total of 100 tests were run for each time point and a summary of the results is shown below:

<table><tr><td rowspan=2 colspan=1>SampleDetection</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>T1 (2 weeks)</td><td rowspan=1 colspan=3>T2 (4 weeks)</td></tr><tr><td rowspan=1 colspan=1>% Correct Calls(95% C1)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Cails</td><td rowspan=1 colspan=1>% Correct Calls(95% C1)**</td><td rowspan=1 colspan=1># of NoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>% Correct Calls(95% CI1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>35/35=100(90.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35/35=100(90.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35/35=100(90.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>49/50=98.0(89.5,99.7)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50/50=100(92.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50/50=100(92.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>44/45=97.8(88.4,99.6)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45/45=100(92.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45/45=100(92.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>5/5=100(56.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5/5=100(56.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5/5=100(56.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>133/135=98.5(94.5,99.6)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135/135=100(97.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135/135=100(97.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

<table><tr><td rowspan=2 colspan=1>SampleDetection</td><td rowspan=1 colspan=2>TO</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>T3 (8 weeks)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>T4 9</td><td rowspan=1 colspan=1>eeks)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% Correct Calls(95% C1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>% Correct Calls(95% CI)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>% Correct Calls(95% CI)**</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># of Mis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>35/35=100(90.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35/35=100(90.1,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33/35=94.3(81.4,98.4)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>49/50=98.0(89.5,99.7)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50/50=100(92.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48/50=96.0(86.5,98.9)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>44/45=97.8(88.4,99.6)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43/45=96.6(85.2,98.8)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43/45=96.6(85.2,98.8)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>5/5=100(56.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5/5=100(56.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5/5=100(56.6,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>133/135=98.5(94.5,99.6)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>133/135=98.5(94.5,99.6)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>129/135=95.6(90.6,97.9)</td><td rowspan=1 colspan=1>6*</td><td rowspan=1 colspan=1>0</td></tr></table>

No call results were due to hardware/software problems include electrical contacts issues between the reader and the cartridge and software curve fitting errors.

The “no call” results during week 9 of testing were followed up and further test results pointed to hardware/software problems but the results were not conclusive.

The time points tested exhibited no statistically significant difference in the positive detection rate. These data are acceptable to support open package stability claims (at ambient storage conditions) as published in the product labeling. The product labeling states:

Cartridges must be used within 4 weeks of opening the pouch.

Extracted and Non-extracted Sample Storage and Freeze Thaw Stability

An analytical study was conducted to demonstrate that freezing samples does not affect the accuracy of test results compared to freshly collected specimens or freshly prepared samples. Fourteen viral targets were selected for the sample stability portion of the study, each positive for one viral target. All viral reference strains for this study were acquired from commercial sources and grown and titered prior to being used in the study. The viral strains used in this study are shown below:

<table><tr><td rowspan=1 colspan=1>RVP Target</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>H1N1 Brisbane/59/07</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>H3N2 ATCC VR-547</td></tr><tr><td rowspan=1 colspan=1>Influenza A2009 H1N1</td><td rowspan=1 colspan=1>NY/2009</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Florida/02/06</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9320</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>C35</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>Greer</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>C 243</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus</td><td rowspan=1 colspan=1>B2</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 7</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>Type 1</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E</td><td rowspan=1 colspan=1>Type 4</td></tr></table>

Fourteen samples each containing a unique viral strain at 3x LoD were prepared in negative patient nasopharyngeal swab (NPS) samples. Internal control was added to all samples per the product labeling prior to extraction. Prior to the initiation of the study an aliquot of each sample was extracted and used in the RNA/DNA stability portion of the study.

An outline of the study design is shown below:

![](images/ef86f06280c23f380bc9a01845c7442331a4f5528f8427f0b95eb26e45135418.jpg)

# Non-extracted Sample Stability

For each positive sample a total of 5 aliquots were prepared and labeled with the appropriate storage conditions. At each time point for each storage condition, a total of 15 extractions were performed. The 9 day and 5 week storage time points were tested at 2 temperatures according to the schematic above.

Overall detection results for unextracted samples after 9 days of storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ .   

<table><tr><td rowspan=4 colspan=1>Target</td><td rowspan=1 colspan=9>Sample Stability</td></tr><tr><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>9 days</td><td rowspan=1 colspan=3>9 days + 3 Freeze Thaws</td></tr><tr><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=3>Storage at 2-8°C</td><td rowspan=1 colspan=3>Storage at -70°C</td></tr><tr><td rowspan=1 colspan=1>Correct Calls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>Correct Calls(%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>Correct Calls(%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>FLU B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FinalCall</td><td rowspan=1 colspan=1>44/45 (97.8%)(88.4,99.6)**</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>44/45(97.8%)(88.4, 99.6)**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>44/45(97.8%)(88.4,99.6)**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

One no call and two missed calls noted in the table above were due to hardware errors unrelated to the biochemical portion of the assay.

Mean signals $\scriptstyle ( \mathtt { n } = 3 )$ ) for each positive viral target call after 9 days in cold storage and the control sample data are shown below:

<table><tr><td rowspan=5 colspan=1>Target</td><td rowspan=2 colspan=3></td><td rowspan=2 colspan=1>%</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=7>Sample Stability</td></tr><tr><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>9 days</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>9 days + 3 Freeze Thaws</td></tr><tr><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Storage at 2-8°C</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Storage at -70°C</td></tr><tr><td rowspan=1 colspan=1>Signal(nA)</td><td rowspan=1 colspan=1>StdDev(nA)</td><td rowspan=1 colspan=1>%CV*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Signal(nA)</td><td rowspan=1 colspan=1>StdDev(nA)</td><td rowspan=1 colspan=1>%CV*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Signal(nA)</td><td rowspan=1 colspan=1>StdDev(nA)</td><td rowspan=1 colspan=1>%CV*</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>111.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>110.9</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>73.2</td><td rowspan=1 colspan=1>20.3</td><td rowspan=1 colspan=1>27.7</td></tr><tr><td rowspan=1 colspan=1>ADVC</td><td rowspan=1 colspan=1>194.4</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>172.8</td><td rowspan=1 colspan=1>56.1</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>117.2</td><td rowspan=1 colspan=1>45.9</td><td rowspan=1 colspan=1>39.2</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>125.9</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>167.7</td><td rowspan=1 colspan=1>75.1</td><td rowspan=1 colspan=1>44.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>60.9</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>65.1</td></tr><tr><td rowspan=1 colspan=1>FLUB</td><td rowspan=1 colspan=1>117.2</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>128.3</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>63.0</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>48.9</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>254.3</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>261.5</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>149.7</td><td rowspan=1 colspan=1>25.6</td><td rowspan=1 colspan=1>17.1</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>209.1</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>212.1</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>171.0</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>107.7</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>47.7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>42.4</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>70.7</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>56.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40.9</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>27.0</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>82.6</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>17.7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>61.2</td><td rowspan=1 colspan=1>16.1</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>44.7</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>38.4</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>110.2</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>55.5</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>51.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.8</td><td rowspan=1 colspan=1>11.6</td><td rowspan=1 colspan=1>53.3</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>218.9</td><td rowspan=1 colspan=1>43.4</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>178.5</td><td rowspan=1 colspan=1>85.5</td><td rowspan=1 colspan=1>47.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>157.6</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>17.9</td></tr><tr><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>276.2</td><td rowspan=1 colspan=1>41.0</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>190.8</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>275.5</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>6.2</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>17.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>55.9</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>46.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>59.0</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>10.4</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>117.4</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>19.7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>77.1</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>48.9</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>182.4</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>164.7</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>105.4</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>32.2</td></tr></table>

Overall detection results after 5 weeks of storage at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ and $- 7 0 \mathrm { { } ^ { \circ } C }$   

<table><tr><td rowspan=4 colspan=1>Target</td><td rowspan=1 colspan=9>Sample Stability</td></tr><tr><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=6>5 week incubation</td></tr><tr><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=3>Storage at -70°C</td><td rowspan=1 colspan=3>Storage at -20°C</td></tr><tr><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>ADV 8</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVC</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%}</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FLU B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2/3 (66.79%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FinalCall</td><td rowspan=1 colspan=1>44/45=97.8(88.4, 99.6)**</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>44/45-97.8{88.4,99.6)**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>44/45=97.8(88.4,99.6)**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

Missed calls were attributed to electrical connection problems between the eSensor cartridge and the XT-8 instrument.

Mean signals $\scriptstyle ( \mathrm { n } = 3 )$ ) for each positive viral target call after 5 weeks in cold storage and the control sample data are shown below:

<table><tr><td rowspan=3 colspan=2>Target</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>Sample Stability</td></tr><tr><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>5 week incubation</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Storage at -70°C</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Storage at -20°C</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Signal(nA)</td><td rowspan=1 colspan=1>Std Dev(nA)</td><td rowspan=1 colspan=1>%CV*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Signal(nA)</td><td rowspan=1 colspan=1>Std Dev(nA)</td><td rowspan=1 colspan=1>%CV*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Signal(nA)</td><td rowspan=1 colspan=1>Std Dev(nA)</td><td rowspan=1 colspan=1>%CV*</td></tr><tr><td rowspan=1 colspan=2>ADV B</td><td rowspan=1 colspan=1>111.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>117.9</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>68.7</td><td rowspan=1 colspan=1>21.9</td><td rowspan=1 colspan=1>31.9</td></tr><tr><td rowspan=1 colspan=2>ADV C</td><td rowspan=1 colspan=1>194.4</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>215.0</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>17.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>205.0</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>7.2</td></tr><tr><td rowspan=1 colspan=2>ADVE</td><td rowspan=1 colspan=1>125.9</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>179.5</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>166.5</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>12.9</td></tr><tr><td rowspan=1 colspan=2>FLU B</td><td rowspan=1 colspan=1>117.2</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>129.9</td><td rowspan=1 colspan=1>10.2</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>143.3</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>9.0</td></tr><tr><td rowspan=1 colspan=2>Flu A</td><td rowspan=1 colspan=1>254.3</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>261.0</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>265.4</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>12.0</td></tr><tr><td rowspan=1 colspan=2>H1</td><td rowspan=1 colspan=1>209.1</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>162.0</td><td rowspan=1 colspan=1>51.8</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>201.4</td><td rowspan=1 colspan=1>54.7</td><td rowspan=1 colspan=1>27.1</td></tr><tr><td rowspan=1 colspan=2>H1N1</td><td rowspan=1 colspan=1>107.7</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>73.5</td></tr><tr><td rowspan=1 colspan=2>H3</td><td rowspan=1 colspan=1>70.7</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>56.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>113.2</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>130.1</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>23.2</td></tr><tr><td rowspan=1 colspan=2>HMPV</td><td rowspan=1 colspan=1>82.6</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>17.7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>79.7</td><td rowspan=1 colspan=1>11.7</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>76.4</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>16.7</td></tr><tr><td rowspan=1 colspan=2>HRV</td><td rowspan=1 colspan=1>110.2</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>64.2</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>82.8</td><td rowspan=1 colspan=1>15.6</td><td rowspan=1 colspan=1>18.8</td></tr><tr><td rowspan=1 colspan=2>PIV 1</td><td rowspan=1 colspan=1>218.9</td><td rowspan=1 colspan=1>43.4</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>242.1</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>203.1</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=2>PIV 2</td><td rowspan=1 colspan=1>276.2</td><td rowspan=1 colspan=1>41.0</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>240.7</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>252.1</td><td rowspan=1 colspan=1>43.1</td><td rowspan=1 colspan=1>17.1</td></tr><tr><td rowspan=1 colspan=2>PIV 3</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>17.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>71.2</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>84.8</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>13.0</td></tr><tr><td rowspan=1 colspan=2>RSV A</td><td rowspan=1 colspan=1>117.4</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>19.7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>85.6</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>14.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>108.1</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>22.7</td></tr><tr><td rowspan=1 colspan=2>RSV B</td><td rowspan=1 colspan=1>182.4</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>165.0</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>15.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>172.3</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>19.3</td></tr></table>

# Extracted Sample Stability

A test of the stability of extracted nucleic acid was run in parallel with the sample stability testing employing the same storage and testing conditions. Each target dilution was extracted 6 times at time 0 (T0) to generate sets of RNA/DNA used to establish extracted stability performance. At the start of the study (T0 time point) these sample extracts were then stored at the specified conditions and tested according to the study schematic at the beginning of this section.

Overall detection results for extracted samples after 9 days of storage at $2 { - } 8 ^ { \circ } \mathrm { C }$   

<table><tr><td rowspan=4 colspan=1>Target</td><td rowspan=1 colspan=9>Nucleic Acid</td></tr><tr><td rowspan=1 colspan=3>TO</td><td rowspan=1 colspan=3>9 days</td><td rowspan=1 colspan=3>9 days + 3 Freeze Thaws</td></tr><tr><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=3>Storage at 2-8°C</td><td rowspan=1 colspan=3>Storage at -70°</td></tr><tr><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>CorrectCalls (9%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVC</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FLU B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1/3 (33.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Final</td><td rowspan=1 colspan=1>44/45=97.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=2 colspan=1>44/45=97.888.4.99.61*8</td><td rowspan=2 colspan=1>0</td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>43/45=95.6(85.2,98.8)**</td><td rowspan=2 colspan=1>0</td><td rowspan=2 colspan=1>2*</td></tr><tr><td rowspan=1 colspan=1>Call</td><td rowspan=1 colspan=1>8R</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

All no calls and miscalls were due to signals below the threshold of <3 nA (false negative) no

hardware failure was reported.

Overall detection results for extracted samples after 9 days of storage at $2 { - } 8 ^ { \circ } \mathrm { C }$   

<table><tr><td rowspan=4 colspan=1>Target</td><td rowspan=2 colspan=9>Nucleic Acid StabilityTO                                       5 week</td></tr><tr><td rowspan=1 colspan=3>TO</td></tr><tr><td rowspan=1 colspan=3>Control</td><td rowspan=1 colspan=3>Storage at-70°C</td><td rowspan=1 colspan=3>Storage at -20°</td></tr><tr><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1># ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVC</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%}</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>2/3 (66.7%}</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FLU B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%}</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FinalCall</td><td rowspan=1 colspan=1>44/45=97.8(88.4,99.6)**</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45/45=100(92.1, 100.0)**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45/45=100(92.1, 100.0)**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

The single no call result was due to signal below the threshold of $< 3$ nA (false negative) no hardware failure was reported.

The time points tested exhibited no statistically significant difference in the positive detection rate. These data are acceptable to support extracted and unextracted sample stability claims as published in the product labeling. Product labeling states:

Transport and Storage- Clinical specimens can be stored between $2 ~ ^ { \circ } C$ and $8 ~ ^ { \circ } C$ for up to 7 days after collection in viral transport media. Specimens can also be stored at $< -$ $I 5 ^ { \circ } C$ for up to 1 month prior to extraction and undergo up to two freeze/thaw cycles.

Note: Purified nucleic acids have been validated for storage between $2 ~ ^ { \circ } C$ and $8 ~ ^ { \circ } C$ for up to 7 days, at ${ < } { - } I 5 ~ ^ { \circ } C$ for up to 1 month, and can undergo up to two freeze/thaw cycles.

Reagent Kit Freeze Thaw (FT) Stability

An analytical study was performed to determine the stability of the RT-PCR and detection reagents used in the eSensor RVP when subjected to storage and freeze/thaw (FT) cycles. Reagents were evaluated for functional performance after 4 and 6 freeze thaw cycles at 4 and 6 weeks. The same set of IVT mixes (IVT1-3) were used in this study and have been previously described above. At each time point and freeze/thaw cycle, samples were spiked with internal control, extracted and amplified then analyzed using the eSensor RVP. An outline of the time points and freeze thaw cycles is shown in the schematic below:

<table><tr><td rowspan=1 colspan=1>Time Point andFreeze/Thaw (FT) Cycle</td><td rowspan=1 colspan=1>Number of Cartridges</td><td rowspan=1 colspan=1>IVT Tested</td></tr><tr><td rowspan=1 colspan=1>FTO; Control(0 day)</td><td rowspan=1 colspan=1>4 replicates for each IVT Set1 DCM17 total</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; MS2</td></tr><tr><td rowspan=1 colspan=1>FT1</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FT2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FT3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FT4(4 weeks)</td><td rowspan=1 colspan=1>4 replicates for each IVT Set16 total</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; MS2</td></tr><tr><td rowspan=1 colspan=1>FT5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FT6(6 weeks)</td><td rowspan=1 colspan=1>4 replicates for each IVT Set16 total</td><td rowspan=1 colspan=1>4 Replicates of:RVP Set 1; RVP Set 2RVP Set 3; MS2</td></tr></table>

Two reagent packs designated for FT4 and FT6 were opened at the beginning of the study and underwent three and five FT cycles respectively prior to the final thaw at the time of testing. For each FT cycle all reagents (except the RVP enzyme mix) were thawed at $3 7 ^ { \circ } \mathrm { C }$ for 15-20 minutes before use or before the next FT cycle. The RVP enzyme mix was thawed on wet ice between FT cycles. A total of 16 reactions per time point plus one negative control were prepared. Final time point performance was considered acceptable if the numbers of correctly called samples were statistically equivalent for each tested condition.

Overall detection results for IVT mixes tested with reagent kits upon opening (FT0) and stored at 2 test conditions (FT4: 4 weeks of storage at $- 1 5 ^ { \circ } \mathrm { C }$ to $- 3 0 ^ { \circ } \mathrm { C }$ and 3 FT cycles and FT6: 6 weeks of storage at $- 1 5 ^ { \circ } \mathrm { C }$ to $- 3 0 ^ { \circ } \mathrm { C }$ and 5 FT cycles ) as shown below:

<table><tr><td rowspan=2 colspan=1>SampleDetection</td><td rowspan=1 colspan=3>FTO</td><td rowspan=1 colspan=3>FT4 (4 week)</td><td rowspan=1 colspan=3>FT6 (6 week)</td></tr><tr><td rowspan=1 colspan=1>% CorrectCalls(95% Cl)**</td><td rowspan=1 colspan=1># of NoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls(95% C1)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1># ofMis-Calls</td><td rowspan=1 colspan=1>% CorrectCalls(95% CI)**</td><td rowspan=1 colspan=1>#ofNoCalls</td><td rowspan=1 colspan=1>#ofMis-Calls</td></tr><tr><td rowspan=1 colspan=1>IVT-1</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27/28=96.4(82.3,99.4)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28/28=100(87.9,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-2</td><td rowspan=1 colspan=1>39/40=97.5(87.1, 99.6)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39/40=97.5(87.1,99.6)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40/40=100(91.2,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>IVT-3</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36/36=100(90.4,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>2/4=50(15,85.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>4/4=100(51.0,100.0)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/4=75(30.1,95.4)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1*</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>105/108=97.2(92.1,99)</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>106/108=98.1(93.5,99.5)</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107/108=99.1(94.9,99.8)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1*</td></tr></table>

Miscalls were attributed to electrical connection problems between the eSensor cartridge and the XT-8 instrument. Miscalls for the MS2 samples (negative controls) were attributed to carry-over contamination.

The time points tested exhibited no statistically significant difference in the positive detection rate. These data are acceptable to support reagent kit freeze thaw stability claims as published in the product labeling. The product labeling states:

§Freeze remaining reagents immediately after use.   
§Store frozen reagents in a non-frost-free freezer (-15 to $- 3 0 ~ ^ { \circ } C )$ .   
§Reagents may be frozen and thawed up to 5 times.

# d. Detection limit:

A range finding study was executed initially to estimate the LoD of each claimed virus using pooled negative NPS specimens in viral transport media as the sample matrix. Re-grown and quantified reference strains of one representative strain for each claimed virus was tested in the study. Each virus strain was diluted in log intervals into transport media (a swab was not used to transfer the virus strain into transport media). Three replicates of each virus strain were tested per concentration starting from nucleic acid extraction. The LoD was chosen as the lowest concentration of each virus that generated a positive result by the eSensor RVP assay for all three replicates. The initial estimates of LoD are shown below:

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Spiked Conc.(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>1:106}$</td><td rowspan=1 colspan=1>4.17E-01</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>1.58E+03</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009 H1N1</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>1.05E-01</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1:108</td><td rowspan=1 colspan=1>3.16E-02</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>2.81E+00</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1:105}$</td><td rowspan=1 colspan=1>1.58E+00</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>2.81E-03</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>2.81E-01</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>1:106}$</td><td rowspan=1 colspan=1>2.81E+01</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>4.17E+00</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>1.58E-03</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>8.89E-02</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>1:108</td><td rowspan=1 colspan=1>8.89E-01</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>1.58E+00</td></tr></table>

The above estimated LoD concentrations were further tested to verify the LoD estimates from the previous range finding study in a LoD confirmation study. The LoD confirmation study tested 20 replicates of each virus strain at the estimated LoD concentrations. The LoD is confirmed if greater than or equal to $9 5 \%$ detection rate is achieved (i.e., at least 19/20 replicates test positive). Results from the first LoD confirmation study are shown below:

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=1># of POSsamples</td><td rowspan=1 colspan=1># of NEGsamples</td><td rowspan=1 colspan=1>% Positives</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>1:10f}$</td><td rowspan=1 colspan=1>4.17E-01</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>1:10f}$</td><td rowspan=1 colspan=1>1.58E+03</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009 H1N1</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>1.05E-01</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1:108}$</td><td rowspan=1 colspan=1>3.16E-02</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1:106$</td><td rowspan=1 colspan=1>2.81E+00</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1:10$</td><td rowspan=1 colspan=1>1.58E+00</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>PIV 1*</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>2.81E-03</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>2.81E-01</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>1:10f}$</td><td rowspan=1 colspan=1>2.81E+01</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>1.58E-03</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>8.89E-02</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>1:108</td><td rowspan=1 colspan=1>8.89E-01</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>1.58E+00</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>75</td></tr></table>

\*19 of 20 data points available due to cartridge failure.

During the first confirmation study it was observed that five virus strains (PIV 1, PIV 2, ADV C, ADV E and Flu A 2009 H1N1) had detection rates below the target of $9 5 \%$ . A second LoD confirmation study was conducted testing 20 additional extraction replicates at the confirmed LoD concentrations for all the virus strains from the first study except for the five virus strains that had below $9 5 \%$ detection rates. The concentration of these five virus strains were increased (a lower dilution of the same stock) and tested in the second LoD confirmation study. The results of the second LoD confirmation study are shown below:

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=1># of POSsamples</td><td rowspan=1 colspan=1># of NEGsamples</td><td rowspan=1 colspan=1>% Positives</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>1:10f}$</td><td rowspan=1 colspan=1>4.17E-01</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>$1:10^r}$</td><td rowspan=1 colspan=1>1.58E+03</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009 H1N1</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>1.05E-01</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1:108}$</td><td rowspan=1 colspan=1>3.16E-02</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1:10}$</td><td rowspan=1 colspan=1>2.81E+00</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1:10fr}$</td><td rowspan=1 colspan=1>1.58E+00</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>1:106</td><td rowspan=1 colspan=1>2.81E-02</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>1:106}$</td><td rowspan=1 colspan=1>2.81E+00</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>1:10fr}$</td><td rowspan=1 colspan=1>2.81E+01</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>1:10fr}$</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>1.58E-03</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>1:107</td><td rowspan=1 colspan=1>8.89E-02</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>1:106}$</td><td rowspan=1 colspan=1>8.89E+01</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>1:10r}$</td><td rowspan=1 colspan=1>1.58E+01</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

Based on the combined results from the two confirmation studies, Flu B was retested at an increased concentration of $3 . 1 6 \mathrm { x } 1 0 ^ { - 1 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ and the Flu B LoD was confirmed in a third separate study.

A summary of the final LoD claims are listed below:

<table><tr><td rowspan=1 colspan=1>Viral Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Spiked Conc.(TCID5o/mL)</td><td rowspan=1 colspan=1># of POS / TotalSamples</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H1N1 Brisbane/59/07</td><td rowspan=1 colspan=1>4.17 x 10-1</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>H1N1 Brisbane/59/07</td><td rowspan=1 colspan=1>4.17 x 10-1</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58 x 103</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58 x 103</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009 H1N1</td><td rowspan=1 colspan=1>NY/2009</td><td rowspan=1 colspan=1>1.05 x 10-1</td><td rowspan=1 colspan=1>76/80</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>3.16 x 10-1</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>4.17 x 100</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.58 x 10-3</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>2.81 x 10-2</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>2.81 x 100</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.81 x 101</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>2.81 x 100</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>1.58 x 100</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ADV B/E</td><td rowspan=1 colspan=1>Type 4</td><td rowspan=1 colspan=1>1.58 x 101</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>8.89 x 101</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td></tr></table>

# e. Analytical reactivity:

An analytical reactivity (inclusivity) study was performed to determine whether the eSensor RVP is able to detect a variety of strains (inclusivity panel) that represent the temporal and geographic diversity of each of the RVP targeted viruses. This study expanded upon the limit of detection studies by determining whether different strains of the same virus can be detected at or close to the established LoD concentrations.

This study involved inclusivity test panels of diverse virus strains spiked into simulated NPS samples. All virus identities were confirmed prior to the study. All virus titers were confirmed except for three HRV strains where extracted RNA samples provided by Biota, Inc. were tested directly in the eSensor RVP. A negative assay control was included with each eSensor RVP extraction run.

Inclusivity virus strains were initially tested near the limit of detection (3x LoD) for each strain as determined in the LoD study. If a specimen containing a particular strain was positive (detected) at the $3 \mathrm { ~ x ~ }$ LoD level, no further testing of that strain was required. If a strain was not detected at the $3 \mathrm { ~ x ~ }$ LoD concentration, the strain was retested at a higher concentration. If necessary, the same approach was followed until a positive result was obtained or the maximum concentration possible for that strain stock had been tested.

The inclusivity panel represented the evolutionary, temporal, and geographical diversity of the RVP analytes. For example, the inclusivity panel included 14 different Adenovirus serotypes, 32 Influenza A strains (subtypes H3, H1 and $2 0 0 9 \mathrm { H } 1$ ) isolated from around the world as early as 1930 and as recent as 2009, and 17 strains of the group that comprised of Human Rhinovirus A, B and C. However, for some analytes, the inclusivity panel is restricted to only two strains either because of limited strain diversity, or due to limited availability.

Each viral strain was diluted in M5 transport medium to a titer of 3x LoD for the corresponding viral target and extracted in triplicate using the bioMerieux NucliSENS easyMAG System. Following extraction, each replicate was tested once using the eSensor RVP. In the case that a viral strain is not detected at 3X LoD, 1000 fold serial dilutions were made from the viral stock and then each dilution was extracted in triplicate and tested using the eSensor RVP.

The summary of all isolates tested appears below:

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">ConcentrationDetected(TCID50/ml)</td><td colspan="1" rowspan="1">LoD MultipleDetected</td></tr><tr><td colspan="1" rowspan="5">Flu A H1</td><td colspan="1" rowspan="1">A/NewCaledonia/20/1999</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">FM/1/47H1</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Denver/1/57</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/SolomonIslands/3/2006</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="5">−</td><td colspan="1" rowspan="1">A/Taiwan/42/06</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">1260</td><td colspan="1" rowspan="1">Flu A- 3x;H1- 3000x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">Flu A- 3x;H1- not detected*</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">6372</td><td colspan="1" rowspan="1">Flu A- 3xH1- 15172x*</td></tr><tr><td colspan="1" rowspan="1">A/Fort Monmouth/1/1947(H1N1)</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">Flu A- 3xH1-13x</td></tr><tr><td colspan="1" rowspan="10">Flu A H3</td><td colspan="1" rowspan="1">A/Aichi/2/68 H3N2</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Perth/16/2009</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Alice (vaccine)A/England/42/72</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">MRC-2 RecombinantStrain</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Nanchang/933/95</td><td colspan="1" rowspan="1">4743</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="10">Flu A 2009H1N1</td><td colspan="1" rowspan="1">A/NY/02/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/New Jersey/8/76</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/California/7/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Swine NY/01/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Swine NY/03/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/09</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Virginia/ATCC1/2009</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Virginia/ATCC2/2009</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">Flu A- 3x2009 H1N1- 6x</td></tr><tr><td colspan="1" rowspan="1">A/Virginia/ATCC3/2009</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">Flu A- 3x2009 H1N1- 27x</td></tr><tr><td colspan="1" rowspan="1">A/Iowa/15/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">Flu A- 3x;2009 H1N1-1000x</td></tr><tr><td colspan="1" rowspan="2">Flu A H2N2</td><td colspan="1" rowspan="1">A/Japan/305/57, RNA</td><td colspan="1" rowspan="1">1.625 ng</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">A/Korea/426/68 (HA,NA) x A/PuertoRico/8/34, RNA</td><td colspan="1" rowspan="1">3.12 ng</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">Flu A H1N2</td><td colspan="1" rowspan="1">A/NWS/34 (HA) xA/RockefellerInstitute/5/57 (NA), RNA</td><td colspan="1" rowspan="1">0.74 ng</td><td colspan="1" rowspan="1">FluA H1</td></tr><tr><td colspan="1" rowspan="1">Flu A H5N3</td><td colspan="1" rowspan="1">A/duck/Singapore/645/97,Wild Type</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="1">Flu AH10N7</td><td colspan="1" rowspan="1">A/chicken/Germany/N/49</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">FluA</td></tr><tr><td colspan="1" rowspan="5">Flu B</td><td colspan="1" rowspan="1">B/Florida/02/06</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/5/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">RSV A</td><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Long</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="3">RSV B</td><td colspan="1" rowspan="1">9320</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">WV/14617/85</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Wash/18537/62</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">PIV1</td><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">0.084</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">0.084</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">PIV2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">PIV3</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="4">HMPV</td><td colspan="1" rowspan="1">IA3-2002 G, A1</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">IA14-2003 G, A2</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Peru2-2002 G, B1</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Peru6-2003 G, B2</td><td colspan="1" rowspan="1">12.6</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="9">HRV A</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">450x#</td></tr><tr><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">569x</td></tr><tr><td colspan="1" rowspan="1">A7</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A16</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Detected**</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">A34</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A57</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A77</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">277G</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">100x#</td></tr><tr><td colspan="1" rowspan="7">HRV B</td><td colspan="1" rowspan="1">B3</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">80x</td></tr><tr><td colspan="1" rowspan="1">B14</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">14x</td></tr><tr><td colspan="1" rowspan="1">B17</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">253x</td></tr><tr><td colspan="1" rowspan="1">B42</td><td colspan="1" rowspan="1">0.005</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">FO2-2547</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">89x#</td></tr><tr><td colspan="1" rowspan="1">B83</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">127x</td></tr><tr><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">Detected**</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">HRVC</td><td colspan="1" rowspan="1">C$\$</td><td colspan="1" rowspan="1">Detected**</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="9">ADV B</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 7A</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 11 (lot 306523)</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">De Wit Type 14</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Ch.79 Type 16</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 21 (lot 307610)</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Compton Type 34</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Holden Type 35</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Wan Type 50</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="4">ADV C</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">267</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">533</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">Type 5</td><td colspan="1" rowspan="1">533</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">Type 6</td><td colspan="1" rowspan="1">533</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">ADV E</td><td colspan="1" rowspan="1">Type 4</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">3x</td></tr></table>

\* In silico analysis revealed little homology between the strain sequence and the H1 primers.

# HRV strain 3, used for LoD determination, had a $\mathrm { T C I D 5 0 / m l }$ of 0.0016. HRV strains 1A, FO2-2547, 277G were detected at a higher LoD multiple to the reference strain, respectively with their corresponding TCID50/ml values of 0.9, 0.2, and 0.2.

$\ast \ast _ { \mathrm { N o } }$ concentration available since it was an extracted RNA sample. \$ Only one test done for HRV C due to limited sample availability

Two Flu A isolates from non-contemporary strains (Flu A/NWS/33 and Flu A/PR/8/34) were successfully detected at 3x the LoD. The Flu A/NWS/33 strain subtype was differentiated as Flu A Hl subtype at $3 0 0 0 \mathrm { { x } }$ the LoD. In silico analysis revealed several mismatches of this strain with the Hl primers which would result in reduced sensitivity for the Hl target. The subtype for isolate Flu A/PR/8/34 strain was not differentiated at any concentration tested. In silico analysis revealed low homology between the strain sequence and the H1 primers.

Flu A/lowa/15/30 strain was detected as FluA only at 3x the LoD; Flu A 2009 H1N1 subtype was detected at $1 0 0 0 \mathrm { x }$ LoD. In silico analysis revealed several mismatches of this strain with the 2009 H1N1 primers which would result in reduced sensitivity for the 2009 H1N1 target.

Rhinovirus strains 1A, F02-2547, 277G were detected at 450X, 89X, and 100X the LoD, respectively with their corresponding $\mathrm { T C I D 5 0 / m l }$ values of 0.9, 0.2, and 0.2. The HRV strain 3, which was used for the LoD determination, had a TCIDSO/ml of 0.0016. The detection levels for HRV reactivity strains were determined relative to this value as described below.

Adenovirus C types 2, 5, and 6 were detected at 6x the LoD. All three strains have a single mismatch with ADVC type 1 capture probe and ADVC type 6 has an additional mismatch with the signal probe which would result in reduced sensitivity for these strains. One mismatch was observed between the three target strains and the ADV C capture probe.

In addition to laboratory testing, bioinformatics resources were also used to predict reactivity of additional species, strains and serotypes with the eSensor RVP system. Simulated reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s).

f. Analytical specificity and cross-reactivity:

An analytical specificity study was carried out to assess the potential for false positive results due to cross-reactivity between eSensor RVP assay targets and other RVP or non-RVP organisms.

Cross-Reactivity Evaluation for Viruses Detected by the eSensor RVP

Cross-reactivity of each viral target (14 viral targets) was evaluated at high concentrations with the eSensor RVP by making three serial dilutions of viral reference strains with viral transport media (Remel M5) at 10,000x, 1000x and $1 0 0 \mathrm { x }$ the LoD. The titer of each virus dilution and corresponding LoD values were determined and provided in the table below. Cross-reactivity was not observed with any of the RVP viral targets at the concentrations tested. The table below summarizes the cross-reactivity results:

<table><tr><td rowspan=1 colspan=1>Viral Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoDConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>Highest TestConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>HighestMultipleof LoDTested</td><td rowspan=1 colspan=1>Cross-ReactivityResults</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H1N1Brisbane/59/07</td><td rowspan=1 colspan=1>4.17 x 10-1</td><td rowspan=1 colspan=1>4.17 x 103</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>H1N1Brisbane/59/07</td><td rowspan=1 colspan=1>4.17 x 10-1</td><td rowspan=1 colspan=1>4.17 x 103</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58 x 103</td><td rowspan=1 colspan=1>1.58 x 107</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>H3N2</td><td rowspan=1 colspan=1>1.58 x 103</td><td rowspan=1 colspan=1>1.58 x 107</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>Flu A 2009H1N1</td><td rowspan=1 colspan=1>NY/2009</td><td rowspan=1 colspan=1>1.05 x 10-1</td><td rowspan=1 colspan=1>1.05 x 103</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Florida/02/06</td><td rowspan=1 colspan=1>3.16 x 10-1</td><td rowspan=1 colspan=1>3.16 x 103</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>4.17 x 100</td><td rowspan=1 colspan=1>4.17 x 104</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.58 x 10-3</td><td rowspan=1 colspan=1>1.58 x 101</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>2.81 x 10-2</td><td rowspan=1 colspan=1>2.81 x 102</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>2.81 x 100</td><td rowspan=1 colspan=1>2.81 x 104</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.81 x 101</td><td rowspan=1 colspan=1>2.81 x 105</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>2.81 x 100</td><td rowspan=1 colspan=1>2.81 x 104</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>1.58 x 100</td><td rowspan=1 colspan=1>1.58 x 104</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>ADV B/E</td><td rowspan=1 colspan=1>Type 7Type 4</td><td rowspan=1 colspan=1>8.89 x 10-21.58 x 101</td><td rowspan=1 colspan=1>1.58 x 105</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>8.89 x 101</td><td rowspan=1 colspan=1>8.89 x 105</td><td rowspan=1 colspan=1>10,000x</td><td rowspan=1 colspan=1>NotObserved</td></tr></table>

As shown in the table below, the signal intensities observed for each tested target and panel analyte indicating that cross-reactivity only exits between the ADV E and ADV B targets. These analytes are reported as a single result due to the experimentally confirmed genetic similarity between these species. All other panel members showed no cross-reactivity at levels below the cut-off of $3 \mathrm { n A }$ for all panel members tested.

<table><tr><td rowspan=2 colspan=1>RVPTarget</td><td rowspan=1 colspan=15>Reference Reagent, Signal Resuits (nA)</td></tr><tr><td rowspan=1 colspan=1>ADVB</td><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>ADVc</td><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>FluAH1</td><td rowspan=1 colspan=1>Flu AHIN1</td><td rowspan=1 colspan=1>Flu AH3</td><td rowspan=1 colspan=1>Flu8</td><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>RSVB</td></tr><tr><td rowspan=1 colspan=1>ADV B/E</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>125.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>ADVC</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>127.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>238.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>250.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Flu A HIN1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>253.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>94.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>129.6</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>105.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>132.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>228.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>266.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>260.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>191.7</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>160.4</td></tr></table>

# Cross-Reactivity with Other Respiratory Viruses Not Targeted by the eSensor RVP

Cross-reactivity with 5 respiratory viruses known to circulate with low frequency in the general population was assessed. All viral strains were diluted in M5 transport media to a titer of $\mathrm { i 0 ^ { 5 } p f u / m L }$ (except Echovirus 30 which was tested at $9 . 8 9 \mathrm { { x } 1 0 ^ { 4 } \ p f i / \dot { \mathrm { / m l } } ) }$ and extracted using the bioMérieux NucliSENS easyMAG extraction method in triplicate. Following extraction, each replicate was tested once in the RVP assay.

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test Concentrations</td><td rowspan=1 colspan=1>Cross-ReactivityResults</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>2.92x105 pfu/mL</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43*</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>5.96x104 pfu/mL</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1.36x105 pfu/mL</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63**</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>9.89x104 pfu/mL</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>ClinicalIsolate</td><td rowspan=1 colspan=1>N/AS</td><td rowspan=1 colspan=1>Not Detected</td></tr></table>

$^ { * } \mathrm { O C } 4 3$ had one replicate fail the IC control at high $\overline { { ( 1 0 ^ { 5 } ) } }$ concentration. \*\*NL63 was tested at the highest concentration available - $9 . 8 9 \mathrm { { x } 1 0 ^ { 4 } \ p f u / m L }$ . \$The Coronavirus HKU1 sample was a clinical isolate identified during the method comparison study. The method used was qualitative so no genome copy number information was available.

Cross-Reactivity with 17 additional viruses that are not targets of the eSensor RVP were also assessed. All viral strains were diluted in M5 transport media to a titer of $1 0 ^ { 5 } \mathrm { p f u / m L }$ and extracted using the bioMérieux NucliSENS easyMAG extraction method in triplicate reactions prior to detection using the eSensor RVP cartridge.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">TestConcentrations</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 18 (A)</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">2.37x10³ pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 9 (D)</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">4.63x105 pfu/mL</td><td colspan="1" rowspan="1">ADVC FalsePositive*</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 41 (F)</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">8.05x105 pfu/mL</td><td colspan="1" rowspan="1">ADVC FalsePositive*</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">Zeptometrix0810047CF</td><td colspan="1" rowspan="1">2.92x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A10</td><td colspan="1" rowspan="1">Zeptometrix0810106CF</td><td colspan="1" rowspan="1">1.72E x10pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">Zeptometrix0810017CF</td><td colspan="1" rowspan="1">2.21x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Echovirus E6</td><td colspan="1" rowspan="1">Zeptometrix0810076CF</td><td colspan="1" rowspan="1">7.16x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B2</td><td colspan="1" rowspan="1">ATCC VR-29</td><td colspan="1" rowspan="1">6.22E+08 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">Zeptometrix0810074CF</td><td colspan="1" rowspan="1">1.06x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">Zeptometrix0810075CF</td><td colspan="1" rowspan="1">8.04E+06 pfu/mL</td><td colspan="1" rowspan="1">2/3 Not Detected1 HRV Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5</td><td colspan="1" rowspan="1">Zeptometrix081019CF</td><td colspan="1" rowspan="1">7.16E+07 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">Zeptometrix081007CF</td><td colspan="1" rowspan="1">1.41E+05 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Echovirus 25</td><td colspan="1" rowspan="1">Zeptometrix VPL-030</td><td colspan="1" rowspan="1">1.93x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Echovirus 30</td><td colspan="1" rowspan="1">Zeptometrix0810078CF</td><td colspan="1" rowspan="1">9.89E+04 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A21</td><td colspan="1" rowspan="1">Zeptometrix0810018CF</td><td colspan="1" rowspan="1">2.92x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24</td><td colspan="1" rowspan="1">ATCC VR-583</td><td colspan="1" rowspan="1">7.00E+05 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">ATCC VR-561</td><td colspan="1" rowspan="1">1.40E+06 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Poliovirus</td><td colspan="1" rowspan="1">ATCC VR-193</td><td colspan="1" rowspan="1">1.11x105 pfu/mL,</td><td colspan="1" rowspan="1">HRV FalsePositive*</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">Clinical Isolate</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus 1: HerpesSimplex</td><td colspan="1" rowspan="1">Zeptometrix0810005CF</td><td colspan="1" rowspan="1">1.01x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus 3: VaricellaZoster</td><td colspan="1" rowspan="1">Zeptometrix 0810026CF</td><td colspan="1" rowspan="1">2.35x106copies/mL#</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus 4: Epstein Barr</td><td colspan="1" rowspan="1">Zeptometrix0810008CF</td><td colspan="1" rowspan="1">1.06x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Herpesvirus 5:Cytomegalovirus</td><td colspan="1" rowspan="1">Zeptometrix0810003CF</td><td colspan="1" rowspan="1">6.68x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">1.37x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Zeptometrix0810079CF</td><td colspan="1" rowspan="1">1.93x105 pfu/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr></table>

$^ { * } \mathrm { A D V C }$ cross-reactive signal was also obtained from Adenovirus 9 (D) and Adenovirus 41 (F) when it was diluted 1000 folds from the initial testing concentration. Due to the genetic similarity between Adenovirus C, D, and F, the eSensor RVP cannot reliably differentiate them. A positive eSensor RVP Adenovirus specie C result should be followed-up using an alternative method (e.g., sequence analysis) if definitive Adenovirus speciation is needed.

\*\*Due to the genetic similarity between human rhinovirus and poliovirus, the eSensor RVP cannot reliably differentiate them. If a polio infection is suspected, a positive eSensor RVP human rhinovirus (HRV) result should be confirmed using an alternate method (e.g., cell culture).

#Quantification of the viral RNA contained in the Herpesvirus-3 (Varicella Zoster Virus) sample was performed using real-time RT-PCR and provided in copies/mL

Systematic false positives observed during the cross-reactivity study will be communicated to the user by the following three statements in the product labeling:

Due to the genetic similarity between Adenovirus species C, D and $F _ { s }$ , and between Adenovirus species B and $E$ , the eSensor RVP cannot reliably differentiate between Adenovirus species C, D and F, or between Adenovirus species B and $E$ . A positive eSensor RVP Adenovirus specie $C$ result or a positive Adenovirus species B/E result should be followed-up using an alternative method (e.g., sequence analysis) if definitive Adenovirus speciation is needed.

Due to the genetic similarity between human rhinovirus and poliovirus, the eSensor RVP cannot reliably differentiate them. If a polio infection is suspected, a positive eSensor RVP human rhinovirus (HRV) result should be confirmed using an alternate method (e.g., cell culture).

Due to the genetic similarity between Adenovirus species C, D (serotype 9) and $F$ (serotype 41), the eSensor RVP cannot reliably differentiate between Adenovirus species C, D (serotype 9) and $F$ (serotype 41). A positive eSensor RVP Adenovirus species $C$ result should be followed-up using an alternative method (e.g., sequence analysis) if definitive Adenovirus speciation between species C, $D$ (serotype 9) and $F$ (serotype 41) is needed.

Cross-Reactivity with Bacteria and Fungus

The non-RVP target exclusivity panel consisted of 32 bacteria, 25 viruses, and 2 fungi (Candida albicans and Candida glabrata). The organisms included in the non-RVP exclusivity panel were selected either because they are related to RVP target organisms, are clinically relevant (colonize the upper respiratory tract or cause respiratory symptoms), are common skin flora or laboratory contaminants, or are microorganisms with a high prevalence of infection (Herpes Simplex virus, etc.).

Bacterial and fungal strains tested for cross-reactivity with the eSensor RVP were diluted in viral transport media to a titer of at least $1 0 ^ { 6 } \mathrm { C F U / m L }$ . Nucleic acids of these organisms, listed below, were extracted in triplicate with the bioMérieux NucliSENS easyMAG system. Following extraction, each replicate was tested once using the eSensor RVP. A “not detected” result indicates all three replicates yielded the same result.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">TestConcentrations</td><td colspan="1" rowspan="1">Cross-ReactivityResults</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">Zeptometrix001597</td><td colspan="1" rowspan="1">5.2x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Zeptometrix8801461</td><td colspan="1" rowspan="1">9.8x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Zeptometrix801459</td><td colspan="1" rowspan="1">5.8x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">2.3x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Zeptometrix001504</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Zeptometrix001535</td><td colspan="1" rowspan="1">9.73x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniaeDNA</td><td colspan="1" rowspan="1">ABI 08-942-250</td><td colspan="1" rowspan="1">1.4x107 copies/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">3.58x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Zeptometrix0801624</td><td colspan="1" rowspan="1">1.5x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Zeptometrix0801680</td><td colspan="1" rowspan="1">2.6x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix801506</td><td colspan="1" rowspan="1">1.07x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Zeptometrix080540</td><td colspan="1" rowspan="1">2.12x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus planarum</td><td colspan="1" rowspan="1">Zeptometrix801507</td><td colspan="1" rowspan="1">1.75x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Zeptometrix001645</td><td colspan="1" rowspan="1">2.6x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Zeptometrix001509</td><td colspan="1" rowspan="1">3.9x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">Zeptometrix801660</td><td colspan="1" rowspan="1">2.2x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix801579</td><td colspan="1" rowspan="1">2.47x106 cCU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Zeptometrix8801511</td><td colspan="1" rowspan="1">3.37x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">Zeptometrix001754</td><td colspan="1" rowspan="1">3.37x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">Zeptometrix BacT-05</td><td colspan="1" rowspan="1">3.55x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Zeptometrix BacT-0</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Zeptometrix801519</td><td colspan="1" rowspan="1">1.05x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Zeptometrix0801723</td><td colspan="1" rowspan="1">6.1x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (COL)</td><td colspan="1" rowspan="1">Zeptometrix0801638</td><td colspan="1" rowspan="1">8.4x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (MSSA)</td><td colspan="1" rowspan="1">Zeptometrix801675</td><td colspan="1" rowspan="1">1.2x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis(MSSE)</td><td colspan="1" rowspan="1">Zeptometrix801689</td><td colspan="1" rowspan="1">2.2x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis(MRSE)</td><td colspan="1" rowspan="1">Zeptometrix0801651</td><td colspan="1" rowspan="1">6.2x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Zeptometrix0801591</td><td colspan="1" rowspan="1">2.16x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Zeptometrix0801545</td><td colspan="1" rowspan="1">2.2x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">Zeptometrix0801516</td><td colspan="1" rowspan="1">6.46x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Zeptometrix0801695</td><td colspan="1" rowspan="1">2.43x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix0801439</td><td colspan="1" rowspan="1">2.8x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Zeptometrix0801512</td><td colspan="1" rowspan="1">1.55x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">Zeptometrix BacT-050</td><td colspan="1" rowspan="1">6.53x106 CFU/mL</td><td colspan="1" rowspan="1">Not Detected</td></tr></table>

No positive test results were observed during testing of the bacterial or fungal species listed above at the tested concentrations.

# g. Assay cut-off:

The eSensor RVP provides a qualitative result detecting the presence (Positive) or absence (Target Not Detected) of the viruses contained in the panel, along with the internal MS2 control, based upon underlying target signals and appropriate signal boundaries. Signals are generated from an electrode and are analyzed by electrochemical voltammetry with the units of nanoamperes (nA). Boundary settings were based upon the negative data as a reference point for viral target cutoffs and the internal control (MS2) data as a reference point for the control cutoffs. Each viral target can have signal from either of two measuring electrodes. If the target signal on at least one of the two measuring electrodes is greater than the assay cutoff level, then that target is considered positive.

To establish the cut-off values for the eSensor RVP targets, blank negative samples consisting of three different viral transport media (M4, M5, and UTM) were spiked with MS2 internal control and then extracted 24 times for each transport media, resulting in a total of 72 tests. The signals obtained from these blank negatives were then combined with results from negative electrodes of samples used in a comparator study during assay development. Negative electrode data from the comparator study were obtained from the negative electrodes of samples which were positive for another viral target. The number of data points obtained from this analysis varied between targets due to the differing number of samples which were positive for each viral target. The total data set included between 419 and 585 observations for each target. The mean and standard deviation were calculated for each target, and a cut-off value of $3 \mathrm { n A }$ was established for all viral targets. This level was chosen to be the largest value after applying the formula [Signal Cut-off $=$ Mean (Target Signal from Negative sample) $+ 5 ^ { * }$ Observed Std Dev (Target Signal)] was $3 . 0 2 \mathrm { n A }$ (RSV A analyte).

For the MS2 control, the cut-off value was determined by analyzing the signals on MS2 electrodes from 68 samples which were tested with no MS2 and the 72 samples from the three transport media (MS2 positive) listed above. After applying the formula [Signal Cutoff $=$ Mean (MS2 Signal from Negative sample) $+ 5 ^ { * }$ Observed Std Dev (MS2 Signal)] a 1 nA boundary was then established.

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Mean + 5 SD</td><td rowspan=1 colspan=1>Number ofSamples Tested</td></tr><tr><td rowspan=1 colspan=1>ADV E</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>564</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>581</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>2.48</td><td rowspan=1 colspan=1>419</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>559</td></tr><tr><td rowspan=1 colspan=1>Flu A H1N1</td><td rowspan=1 colspan=1>1.98</td><td rowspan=1 colspan=1>466</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>2.32</td><td rowspan=1 colspan=1>550</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>541</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>574</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>576</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>538</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>3.02</td><td rowspan=1 colspan=1>498</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>542</td></tr></table>

To verify the established cut-off values, signals from 200 archived patient samples positive by either DFA or RT-PCR were compared against the $3 \mathrm { n A }$ cut-off for each viral target. Out of the 200 confirmed positive samples, 191 generated signals above $5 \mathrm { n A }$ ; the remaining 9 positives generated signals less than $1 . 5 \mathrm { n A }$ . Therefore, an assay cut-off at $3 \mathrm { n A }$ for the viral targets was chosen to maximize assay sensitivity and specificity. For the MS2 control, the 1 nA boundary was compared against MS2 signals from 200 archived patient samples, a 1 nA cut-off was chosen based on this data set.

# h. Interfering substances

Potentially interfering substances were selected based on the fact that they could pre-exist in the specimen (e.g. blood, nasal secretions or mucus, and nasal and throat medications used to relieve congestion, nasal dryness, irritation, or asthma and allergy symptoms) as well as those that could be introduced during specimen collection and preparation. Each potentially interfering substance was tested individually with the exception of Luffa opperculata, Galphimia glauca, Histaminum hydrochloricum, and Sulfur, which were tested together as Zicam $^ \mathrm { \textregistered }$ Allergy Relief Nasal spray and Oxymetazoline and Menthol, which were tested together as Afrin $^ \mathrm { \textregistered }$ No Drip Severe Congestion nasal spray, thereby bringing the total to 21 potentially interfering test combinations. Viral samples representative of the 14 viral targets on the eSensor RVP were obtained from commercially available cultured cell lines as indicated in Table 31. Seven viral mixes were made, each containing unique viral targets. Viral mixes were added to each potentially interfering substance resulting in a final testing concentration of 3X LoD for each analyte. Each was extracted in triplicate with each extract tested once with the eSensor RVP. Twenty-one (21) combinations of 24 potentially interfering substances were tested in this study. Additionally, nine potentially interfering microorganisms (viral and bacterial) were also tested in the same manner as described above. The microorganisms and their testing concentrations are listed in Table 31. All substances and microorganisms tested for interference were shown to be compatible with the eSensor RVP. No potentially interfering substance or microorganism was shown to inhibit the eSensor RVP at all tested concentrationsa.

<table><tr><td rowspan=1 colspan=1>Potentially InterferingSubstance</td><td rowspan=1 colspan=1>ActiveIngredient</td><td rowspan=1 colspan=1>SubstanceForm</td><td rowspan=1 colspan=1>TestedConcentration</td></tr><tr><td rowspan=1 colspan=1>Sample Matrix</td><td rowspan=1 colspan=1>Control for no interfering substance</td><td rowspan=1 colspan=1>Liquid</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Viral transport medium</td><td rowspan=1 colspan=1>Becton Dickinson VTM</td><td rowspan=1 colspan=1>Liquid</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>Blood (human)</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Liquid</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Human gDNA</td><td rowspan=1 colspan=1>50 ng/rxn</td><td rowspan=1 colspan=1>50 ng/rxn</td></tr><tr><td rowspan=2 colspan=1>Throat lozenges, oral anestheticand analgesic</td><td rowspan=1 colspan=1>Benzocaine</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>30% w/v</td></tr><tr><td rowspan=1 colspan=1>Menthol*</td><td rowspan=1 colspan=1>NasalSpray</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucin: bovine submaxillarygland,type I-S</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=3 colspan=1>Nasal spraysor drops</td><td rowspan=1 colspan=1>Phenylephrine (Neo-Synephrine)</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Oxymetazoline* (also containsBenzalkonium Chloride, Menthol,Eucalyptol, Camphor, benzyl alcohol andphosphate buffers)</td><td rowspan=1 colspan=1>NasalSpray</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Sodium chloride</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>0.8% w/v</td></tr><tr><td rowspan=1 colspan=1>Antibacterial, systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>5% w/v</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>2% w/v</td></tr><tr><td rowspan=6 colspan=1>Nasal corticosteroids</td><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Budesonide (Pulmicort)</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Fluticasone (Flonase)</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>3% w/v</td></tr><tr><td rowspan=2 colspan=1>Nasal gel</td><td rowspan=1 colspan=1>Luffa opperculata**</td><td rowspan=1 colspan=1>Nasal Gel</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Sulfur**</td><td rowspan=1 colspan=1>Nasal Gel</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=2 colspan=1>Homeopathic allergy reliefmedicine</td><td rowspan=1 colspan=1>Galphimia glauca**</td><td rowspan=1 colspan=1>Nasal Gel</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>Histaminum hydrochloricum**</td><td rowspan=1 colspan=1>Nasal Gel</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>FluMist#</td><td rowspan=1 colspan=1>Live intranasal influenza virus vaccine#</td><td rowspan=1 colspan=1>Liquid</td><td rowspan=1 colspan=1>0.5%-1% v/v</td></tr><tr><td rowspan=2 colspan=1>Anti-viral drugs</td><td rowspan=1 colspan=1>Zanamivir (Relenza)</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>550 ng/ml</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir (Tamiflu)</td><td rowspan=1 colspan=1>Dry</td><td rowspan=1 colspan=1>142 ng/ml</td></tr><tr><td rowspan=2 colspan=1>Virus</td><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Culture</td><td rowspan=2 colspan=1>1 x 105PFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=7 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Culture</td><td rowspan=7 colspan=1>1 x 106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenza</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Culture</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diptheriae</td><td rowspan=1 colspan=1>Culture</td></tr></table>

\*Tested together (Afrin No Drip Severe Congestion nasal spray) \*\* Tested together (Zicam Allergy Relief) # FluMist vaccine: Addition of FluMist Live Intranasal Influenza Vaccine to the transport media control resulted in positive calls for Flu A, Flu A H3, Flu A 2009

H1N1 and Flu B. This was due to the live attenuated influenza virus present in the vaccine.

a Testing of FluMist at $1 \% ( \mathrm { v / v } )$ resulting in an inhibition in the detection of hMPV.

FluMist did not inhibit the detection of hMPV when tested at $0 . 5 \%$ (v/v).

Raw data from the interfering substances study was compared with data from the control runs (VTM only) to generate the table below. The signal ratio is calculated by dividing the mean nA signal of samples with interfering substance by mean nA signal of control samples without interfering substance. The results of this calculation are summarized below:

<table><tr><td colspan="1" rowspan="2">Potentially InterferingSubstance</td><td colspan="8" rowspan="1">Percent Interference</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu AH1</td><td colspan="1" rowspan="1">Flu AH3</td><td colspan="1" rowspan="1">Flu A2009H1N1</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">HRV</td></tr><tr><td colspan="1" rowspan="1">Remel M5 (Control)</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td></tr><tr><td colspan="1" rowspan="1">BD VTM</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">1.38</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">1.17</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="1">Blood gDNA</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">1.24</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">1.76</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">1.21</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">1.28</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">2.16</td><td colspan="1" rowspan="1">1.91</td><td colspan="1" rowspan="1">0.91</td></tr><tr><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">1.19</td><td colspan="1" rowspan="1">1.13</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">1.98</td><td colspan="1" rowspan="1">1.62</td><td colspan="1" rowspan="1">0.81</td></tr><tr><td colspan="1" rowspan="1">Sodium Chloride</td><td colspan="1" rowspan="1">1.17</td><td colspan="1" rowspan="1">1.01</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">1.42</td><td colspan="1" rowspan="1">1.40</td><td colspan="1" rowspan="1">1.27</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">1.07</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">1.39</td><td colspan="1" rowspan="1">1.27</td></tr><tr><td colspan="1" rowspan="1">Oxymetazoline-HCl(Afrin)</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">1.42</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">1.20</td></tr><tr><td colspan="1" rowspan="1">Tobramycin Sulfate salt</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">1.02</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">1.75</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">1.38</td><td colspan="1" rowspan="1">1.33</td><td colspan="1" rowspan="1">1.12</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">2.06</td><td colspan="1" rowspan="1">1.03</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">1.44</td><td colspan="1" rowspan="1">1.22</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">1.35</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">1.81</td><td colspan="1" rowspan="1">1.18</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">2.23</td><td colspan="1" rowspan="1">0.84</td></tr><tr><td colspan="1" rowspan="1">TriamcinoloneAcetomide</td><td colspan="1" rowspan="1">1.25</td><td colspan="1" rowspan="1">1.40</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">1.32</td><td colspan="1" rowspan="1">2.13</td><td colspan="1" rowspan="1">1.08</td></tr><tr><td colspan="1" rowspan="1">(R )-(-)-Phenylephrine-HCI</td><td colspan="1" rowspan="1">1.33</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">1.52</td><td colspan="1" rowspan="1">2.36</td><td colspan="1" rowspan="1">1.23</td></tr><tr><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">1.16</td><td colspan="1" rowspan="1">1.23</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">0.84</td><td colspan="1" rowspan="1">2.37</td><td colspan="1" rowspan="1">1.11</td></tr><tr><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">1.01</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">2.30</td><td colspan="1" rowspan="1">0.74</td></tr><tr><td colspan="1" rowspan="1">Fluticasone proprionate</td><td colspan="1" rowspan="1">1.28</td><td colspan="1" rowspan="1">1.19</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">0.84</td><td colspan="1" rowspan="1">2.27</td><td colspan="1" rowspan="1">2.11</td><td colspan="1" rowspan="1">1.15</td></tr><tr><td colspan="1" rowspan="1">Zicam</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">1.21</td><td colspan="1" rowspan="1">1.39</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">2.00</td><td colspan="1" rowspan="1">0.82</td></tr><tr><td colspan="1" rowspan="1">FluMist (1%)</td><td colspan="1" rowspan="1">1.25</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">1.99</td><td colspan="1" rowspan="1">3.98</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.57</td></tr><tr><td colspan="1" rowspan="1">Zanamivir (Relenza)</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">1.54</td><td colspan="1" rowspan="1">1.22</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir (Tamiflu)</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">1.92</td><td colspan="1" rowspan="1">1.10</td></tr><tr><td colspan="1" rowspan="1">EY</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">PotentiallyInterferingSubstance</td><td colspan="8" rowspan="1">Percent Interference</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">ADVC</td><td colspan="1" rowspan="1">ADVB</td><td colspan="1" rowspan="1">ADVE</td><td colspan="1" rowspan="1">ADVB/E*</td></tr><tr><td colspan="1" rowspan="1">Remel M5 (Control)</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.00</td></tr><tr><td colspan="1" rowspan="1">BD VTM</td><td colspan="1" rowspan="1">1.07</td><td colspan="1" rowspan="1">1.51</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">1.01</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">1.03</td></tr><tr><td colspan="1" rowspan="1">Blood gDNA</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">1.72</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">0.90</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">1.21</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">0.88</td></tr><tr><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">1.07</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.97</td></tr><tr><td colspan="1" rowspan="1">Sodium Chloride</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">1.66</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">0.84</td><td colspan="1" rowspan="1">0.93</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">1.69</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">1.23</td><td colspan="1" rowspan="1">1.33</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">1.12</td><td colspan="1" rowspan="1">0.92</td></tr><tr><td colspan="1" rowspan="1">Oxymetazoline-HCI(Afrin)</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">1.79</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">1.43</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">0.81</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">Tobramycin Sulfatesalt</td><td colspan="1" rowspan="1">1.02</td><td colspan="1" rowspan="1">1.19</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">0.96</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">1.47</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">1.77</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">1.19</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">1.28</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.76</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">0.84</td></tr><tr><td colspan="1" rowspan="1">TriamcinoloneAcetomide</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">1.32</td><td colspan="1" rowspan="1">1.79</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">(R )-(-)-Phenylephrine-HCI</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">1.64</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.89</td></tr><tr><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">1.28</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">1.48</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">0.88</td></tr><tr><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">1.72</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">1.73</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">0.87</td></tr><tr><td colspan="1" rowspan="1">Fluticasoneproprionate</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">1.22</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">1.45</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">0.86</td></tr><tr><td colspan="1" rowspan="1">Zicam</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">1.52</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">1.49</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">1.00</td></tr><tr><td colspan="1" rowspan="1">FluMist (1%)</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">1.02</td></tr><tr><td colspan="1" rowspan="1">Zanamivir(Relenza)</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">1.44</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">1.48</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">1.17</td><td colspan="1" rowspan="1">1.05</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir(Tamiflu)</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">1.58</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">1.47</td><td colspan="1" rowspan="1">1.01</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">0.91</td></tr></table>

An analytical interference study was carried out by testing RVP viral target mixes against a panel of potentially interfering microorganisms. Seven different RVP viral target mixes were used during the study. Each of the mixes contained two RVP panel members at a concentration of 3X LoD. All sample/substance mixtures were extracted in triplicate according to the package insert. Potentially interfering microorganisms and the concentrations tested are shown below:

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Potential InterferingSubstance</td><td rowspan=1 colspan=1>Lot Number</td><td rowspan=1 colspan=2>StockConcentration</td><td rowspan=1 colspan=1>TestingConcentration</td></tr><tr><td rowspan=2 colspan=1>Viral</td><td rowspan=1 colspan=1>Cytomegalovirus (CMV)</td><td rowspan=1 colspan=1>305811</td><td rowspan=1 colspan=2>4.17E+05</td><td rowspan=2 colspan=1>1 x 105 pfu/ml</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>308126</td><td rowspan=1 colspan=2>7.24E+05</td></tr><tr><td rowspan=8 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>305705</td><td rowspan=1 colspan=2>2.80E+10</td><td rowspan=8 colspan=1>1 x 106 cfu/ml</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>305539</td><td rowspan=1 colspan=2>5.80E+09</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenza</td><td rowspan=1 colspan=1>305130</td><td rowspan=1 colspan=2>2.60E+06</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>306813</td><td rowspan=1 colspan=2>2.70E+07</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Staphylococcus aureus</td><td rowspan=2 colspan=1>305344</td><td rowspan=2 colspan=2>1.20E+08</td></tr><tr></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>306072</td><td rowspan=1 colspan=2>3.37E+08</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diptheriae</td><td rowspan=1 colspan=1>305956</td><td rowspan=1 colspan=2>3.58E+09</td></tr></table>

Each potential interfering substance was analyzed for detection interference between the substance and the added RVP viral target. No interfering effects were observed with regard to the call rate of the RVP viral targets, this data is shown below.

<table><tr><td colspan="11" rowspan="1">Microorganism Tested</td></tr><tr><td colspan="1" rowspan="1">RVPViralTarget</td><td colspan="1" rowspan="1">() [u)</td><td colspan="1" rowspan="1">Chtoc</td><td colspan="1" rowspan="1">LT</td><td colspan="1" rowspan="1">euounedss</td><td colspan="1" rowspan="1">PessB</td><td colspan="1" rowspan="1">Huu! H</td><td colspan="1" rowspan="1">etounedM</td><td colspan="1" rowspan="1">anssi</td><td colspan="1" rowspan="1">wiiiN</td><td colspan="1" rowspan="1">cehee</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">Flu AH1</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">Flu AH3</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">2009H1N1</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">PIV 2</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">ADVB</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">RVPViralTarget</td><td colspan="1" rowspan="1">) 0.)</td><td colspan="1" rowspan="1">Chtc</td><td colspan="1" rowspan="1">TT </td><td colspan="1" rowspan="1">entuounads</td><td colspan="1" rowspan="1">PeessB</td><td colspan="1" rowspan="1">!znnur=</td><td colspan="1" rowspan="1">eutounedM</td><td colspan="1" rowspan="1">annusi</td><td colspan="1" rowspan="1">wiitN</td><td colspan="1" rowspan="1">rhdaeU</td></tr><tr><td colspan="1" rowspan="1">ADVC</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr><tr><td colspan="1" rowspan="1">ADVE</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,10%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td><td colspan="1" rowspan="1">3/3,100%</td></tr></table>

Raw data from the interfering microorganisms study was compared with data from the control runs (VTM only) to generate a table of percent interference for each microorganism and analyte combination. Percent interference is calculated by dividing the mean nA signal of samples with interfering substance by mean nA signal of control samples without interfering substance $\textbf { x } 1 0 0$ . The results of the percent interference calculations are summarized below:

<table><tr><td rowspan=1 colspan=11>Microorganism Tested</td></tr><tr><td rowspan=1 colspan=1>RVPViralTarget</td><td rowspan=1 colspan=1>() C)</td><td rowspan=1 colspan=1>Chtc</td><td rowspan=1 colspan=1>Lh</td><td rowspan=1 colspan=1>Sounue s</td><td rowspan=1 colspan=1>SeeS</td><td rowspan=1 colspan=1>Huu! &#x27;H</td><td rowspan=1 colspan=1>Mounud E</td><td rowspan=1 colspan=1>snnss</td><td rowspan=1 colspan=1>mwiitN</td><td rowspan=1 colspan=1>hdeU</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>0.95</td></tr><tr><td rowspan=1 colspan=1>Flu AH1</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.01</td></tr><tr><td rowspan=1 colspan=1>Flu AH3</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>1.11</td></tr><tr><td rowspan=1 colspan=1>2009H1N1</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>1.00</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>1.09</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>0.94</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.98</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>0.91</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>1.04</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>1.02</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=1 colspan=1>ADV B</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>1.06</td></tr><tr><td rowspan=1 colspan=1>ADV C</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>0.91</td></tr><tr><td rowspan=1 colspan=1>ADVE</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.99</td></tr></table>

Based on the analytical study data of potentially interfering substances and microorganism the following limitations have been added to the product labeling:

The effect of interfering substances has only been evaluated for those listed in the labeling. Interference by substances other than those described in the “Interference” section below can lead to erroneous results.

Recent administration of a live intranasal influenza virus vaccine may cause false positive results for Influenza A, H1, H3, 2009 H1N1, and/or Influenza B.

# i. Carryover contamination

The carryover/cross-contamination study challenged the extraction, RT-PCR, and detection portions of the assay within and between runs and operators tested over the course of five testing days. A representative strain of Parainfluenza Virus 3 was obtained as a commercially available cultured cell line. Positive Parainfluenza Virus 3 samples were prepared at a concentration of $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $3 5 5 9 \mathrm { X }$ LoD) while negative samples were un-inoculated Remel M5 transport media. All samples were extracted using the bioMérieux easyMAG System. Five sets of alternating high concentration positive and negative samples were extracted and tested in a checkerboard pattern. Each set of samples contained 24 tests (12 positive and 12 negative). Total number of tests for the duration of the study was 120 samples (60 positive and 60 negative).

No carryover/cross-contamination was observed in the eSensor RVP, as $100 \%$ of the PIV 3 negative samples were reported as ‘Target Not Detected’.

2. Comparison studies:

a. Method comparison with predicate device:

N/A

# b.Matrix comparison:

All analytical studies were performed using viral targets diluted in transport media. A bridging study was conducted to determine the effects of natural clinical matrix (NCM) on assay performance. Excess sample from the clinical study which tested negative for all panel members by the eSensor RVP constituted the NCM in this study. The bridging study compared assay performance of representative viral targets diluted in negative natural clinical matrix (NCM) compared to targets diluted in simulated matrix (SM). Panel members were tested by the eSensor RVP at $3 \mathbf { x }$ LoD in NCM and SM.

Positive rate for each target in the natural matrix is the same as the positive rate for the corresponding target in the simulated matrix control,

<table><tr><td rowspan=2 colspan=1>Sample Matrix</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu AH3</td><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>ADVB/E</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>IC</td></tr><tr><td rowspan=1 colspan=1>POSCalls</td><td rowspan=1 colspan=1>POSCalls</td><td rowspan=1 colspan=1>POSCalls</td><td rowspan=1 colspan=1>POSCalls</td><td rowspan=1 colspan=1>POSCalls</td><td rowspan=1 colspan=1>POSCalls</td><td rowspan=1 colspan=1>POS Calls</td></tr><tr><td rowspan=1 colspan=1>Natural Clinical Matirx(NCM)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>25/25</td></tr><tr><td rowspan=1 colspan=1>Sinluated Matrix(Control)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>25/25</td></tr></table>

<table><tr><td rowspan=2 colspan=1>SampleMatrix</td><td rowspan=1 colspan=4>Flu A (n=5)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Flu A H3 (n=5)</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td></tr><tr><td rowspan=1 colspan=1>Natural ClinicalMatrix (NCM)</td><td rowspan=1 colspan=1>230.8</td><td rowspan=1 colspan=1>18.9</td><td rowspan=1 colspan=1>8.2</td><td rowspan=2 colspan=1>96.9</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>11.3</td><td rowspan=1 colspan=1>12.1</td><td rowspan=2 colspan=1>108.9</td></tr><tr><td rowspan=1 colspan=1>SimulatedMatrix (Control)</td><td rowspan=1 colspan=1>238.3</td><td rowspan=1 colspan=1>17.9</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>31.3</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Sample Matrix</td><td rowspan=1 colspan=4>HMPV (n=5)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>ADV B/E (n=5)</td></tr><tr><td rowspan=1 colspan=1>MeanSignalnA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Mean SignalnA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td></tr><tr><td rowspan=1 colspan=1>Natural ClinicalMatrix (NCM)</td><td rowspan=1 colspan=1>76.0</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>15.7</td><td rowspan=2 colspan=1>81.6</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>9.9</td><td rowspan=2 colspan=1>100.5</td></tr><tr><td rowspan=1 colspan=1>SimulatedMatrix (Control)</td><td rowspan=1 colspan=1>93.1</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>16.6</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>8.4</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Sample Matrix</td><td rowspan=1 colspan=4>RSV A (n=5)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>PIV3 (n=5)</td></tr><tr><td rowspan=1 colspan=1>Mean ignalnA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td></tr><tr><td rowspan=1 colspan=1>Natural ClinicalMatrix (NCM)</td><td rowspan=1 colspan=1>167.3</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>5.8</td><td rowspan=2 colspan=1>94.0</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>136.0</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>24.6</td><td rowspan=2 colspan=1>93.9</td></tr><tr><td rowspan=1 colspan=1>SimulatedMatrix (Control)</td><td rowspan=1 colspan=1>178.0</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>144.8</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>24.1</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Sample Matrix</td><td rowspan=1 colspan=4>MS2 IC (n=25)</td></tr><tr><td rowspan=1 colspan=1>MeanSignal(nA)</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>%Signal</td></tr><tr><td rowspan=1 colspan=1>Natural ClinicalMatrix (NCM)</td><td rowspan=1 colspan=1>200.0</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>19.0</td><td rowspan=2 colspan=1>96.5</td></tr><tr><td rowspan=1 colspan=1>SimulatedMatrix (Control)</td><td rowspan=1 colspan=1>207.3</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>4.8</td></tr></table>

An analysis of the raw data reveals that the p-values for the PIV 3, Flu A, ADV B and Flu A H3 datasets are not statistically different $( \mathtt { p } > 0 . 0 5 )$ . The hMPV and RSV A datasets are statistically different $( \mathbf { p } < 0 . 0 5 )$ however there is no clinically significance difference ( $100 \%$ agreement of positive calls).

# 3. Clinical Studies:

Prospective Clinical Study

The study was designed to collect and test prospective nasopharyngeal samples from individuals representing the intended use population. The study was performed in two arms at a total of 3 external clinical study sites representing a geographic diversity within the United States. All clinical specimens in the prospective clinical study were nasopharyngeal (NP) swab specimens, prospectively collected and tested during the 2010/11 influenza season at three North American clinical laboratories. Clinical laboratories were located in Cleveland, Ohio; Providence, RI; and Albuquerque, NM. Demographic details for patient population are summarized in the table below. Study sites enrolled subjects from diverse demographic groups; about $40 \%$ of the specimens were obtained from patients enrolled at a hospital. The remaining specimens were collected from outpatients and patients in an emergency department.

A total of 1182 patient samples were collected prospectively across the three clinical sites from January 2011 until May 2011. Out of these patient samples, 1037 were evaluable. A total of 145 samples were excluded for the following reasons: samples not tested within 5 days of specimen collection (72/145), operator and/or easyMAG mechanical errors (62/145), samples not retested (11/145). Of the evaluable prospective clinical specimens, $9 3 \%$ (963/1037) yielded valid results on the first attempt. Invalid results or no results were obtained for the remaining 74 specimens (45 of which generated results on the first run, but required retesting due to a negative control failure caused by operator error). Data generated from the retests was used in the final analysis. All 74 specimens yielded valid results after a single retest when tested according the retest recommendations.

Out of the 1037 samples collected, an even split of patients were male and female. Approximately one quarter of the samples came from children under the age of 1; patients aged 21-65 contributed the largest share of the samples. A summary of the patient demographics is shown below:

General Demographic Data for Prospectively Collected Specimens $\scriptstyle ( \mathbf { N } = \mathbf { 1 0 3 7 } )$   

<table><tr><td colspan="1" rowspan="2">Demographic</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites</td></tr><tr><td colspan="1" rowspan="1">N = 245(%)</td><td colspan="1" rowspan="1">N = 533(%)</td><td colspan="1" rowspan="1">N = 259(%)</td><td colspan="1" rowspan="1">N = 1037(%)</td></tr><tr><td colspan="5" rowspan="1">SEX</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">105 (42.9)</td><td colspan="1" rowspan="1">296 (55.5)</td><td colspan="1" rowspan="1">117 (45.2)</td><td colspan="1" rowspan="1">518 (50.0)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">140 (57.1)</td><td colspan="1" rowspan="1">237 (44.5)</td><td colspan="1" rowspan="1">142 (54.8)</td><td colspan="1" rowspan="1">519 (50.0)</td></tr><tr><td colspan="5" rowspan="1">AGE (yrs)</td></tr><tr><td colspan="1" rowspan="1">0-1</td><td colspan="1" rowspan="1">46 (18.8)</td><td colspan="1" rowspan="1">197 (37.0)</td><td colspan="1" rowspan="1">27 (10.4)</td><td colspan="1" rowspan="1">270 (26.0)</td></tr><tr><td colspan="1" rowspan="1">&gt;1-5</td><td colspan="1" rowspan="1">20 (8.2)</td><td colspan="1" rowspan="1">94 (17.6)</td><td colspan="1" rowspan="1">22 (8.4)</td><td colspan="1" rowspan="1">136 (13.1)</td></tr><tr><td colspan="1" rowspan="1">&gt;5-21</td><td colspan="1" rowspan="1">19 (7.8)</td><td colspan="1" rowspan="1">82 (15.4)</td><td colspan="1" rowspan="1">26 (10.0)</td><td colspan="1" rowspan="1">127 (12.2)</td></tr><tr><td colspan="1" rowspan="1">&gt; 21-65</td><td colspan="1" rowspan="1">97 (39.6)</td><td colspan="1" rowspan="1">106 (19.9)</td><td colspan="1" rowspan="1">130 (50.2)</td><td colspan="1" rowspan="1">333 (32.1)</td></tr><tr><td colspan="1" rowspan="1">&gt;65</td><td colspan="1" rowspan="1">63 (25.7)</td><td colspan="1" rowspan="1">54 (10.1)</td><td colspan="1" rowspan="1">54 (20.8)</td><td colspan="1" rowspan="1">171 (16.5)</td></tr><tr><td colspan="5" rowspan="1">SUBJECT STATUS</td></tr><tr><td colspan="1" rowspan="1">Outpatient</td><td colspan="1" rowspan="1">7 (2.9)</td><td colspan="1" rowspan="1">219 (41.1)</td><td colspan="1" rowspan="1">90 (34.7)</td><td colspan="1" rowspan="1">316 (30.5)</td></tr><tr><td colspan="1" rowspan="1">Hospitalized</td><td colspan="1" rowspan="1">131 (53.5)</td><td colspan="1" rowspan="1">162 (30.4)</td><td colspan="1" rowspan="1">114 (44.0)</td><td colspan="1" rowspan="1">407 (39.2)</td></tr><tr><td colspan="1" rowspan="1">EmergencyDepartment</td><td colspan="1" rowspan="1">107 (43.7)</td><td colspan="1" rowspan="1">152 (28.5)</td><td colspan="1" rowspan="1">55 (21.2)</td><td colspan="1" rowspan="1">314 (30.3)</td></tr></table>

The performance of the RVP assay was compared to the established gold standard reference method of viral culture for most viral targets. For respiratory viruses in which culture was not available, a composite (multi-test) reference method (a predetermined algorithm that combined the results of a few tests) was used as the comparator method. Composite methods were designed to target a different genomic region for each viral target than the one used by the RVP assay. The comparator methods were analytically validated prior to use in this study. As seen in the table below, viral culture followed by DFA identification testing was used as the comparator method for Influenza A, Influenza B, RSV, Parainfluenza Viruses (PIV1, PIV2, PIV3), and adenovirus. Since viral culture cannot determine the subtype for influenza A, RSVs, and adenoviruses, these viruses were subtyped by an independently developed qRT-PCR assay or qPCR assay followed by bidirectional sequencing to determine the subtypes (Influenza A H1, Influenza A H3, Influenza A 2009 H1N1, RSVA, RSVB, ADVB/E and ADVC). All molecular comparator and subtyping testing was performed on extracted nucleic acids. All clinical samples, regardless of the RVP or culture/DFA result, were sent for HMPV and HRV testing with two validated qRT-PCR assays for each viral target. If at least one of the assays was positive for the specific target, then the positive sample was sequenced using target specific primers.

Samples whose RVP result didn’t match the culture/DFA result were sent for discordant testing. A third party performed DNA sequencing on the extracted nucleic acid samples supplied by GenMark using primer pairs for the specific viral target being tested. The results of this discordant testing were used to footnote the sensitivity and specificity tables.

Assays were developed to perform discordant testing for Flu B, PIV1, PIV2, and PIV3. Samples that required discordant testing for Flu B were tested in the comparator qRT-PCR assay for Flu B, and any positive samples were then sequenced using a target specific primer for Flu B. Similarly, any sample that required testing for PIV1, PIV2, or PIV3 were tested in the corresponding comparator qRT-PCR assay for that specific viral target and any positive samples were sequenced using a target specific primer for the specific Parainfluenza virus. A viral target was determined to be present in a specimen only if indicated by sequencing.

<table><tr><td rowspan=1 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=1>Comparator Method</td><td rowspan=1 colspan=1>Subtyping</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=8 colspan=1>Viral culture followed by DFAidentification</td><td rowspan=8 colspan=1>qRT-PCR +BidirectionalSequencing</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009H1N1</td></tr><tr><td rowspan=1 colspan=1>RSV A</td></tr><tr><td rowspan=1 colspan=1>RSV B</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B/E</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=4 colspan=1>Viral culture followed by DFAidentification2</td><td rowspan=4 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=2 colspan=1>2 qRT-PCR (2 methods) withBidirectional Sequencing</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td></tr></table>

1Validated performance of the eSensor RVP assay detecting Influenza A, RSV or ADV respectively was compared to viral culture followed by fluorescent antibody identification. “True” Influenza A, RSV or ADV positives respectively, were considered as any sample that tested positive for Influenza A, RSV or ADV respectively, by viral culture followed by DFA testing. True positive samples were subtyped using one analytically validated qRT-PCR assay with bi-directional sequence confirmation. The comparator assays were designed to amplify a different sequence from that amplified by the eSensor RVP assay(s). None of the comparator PCR assays overlapped any RVP amplicon sequence even if the same gene was targeted. “True” Influenza A H1, H3, or 2009 H1N1 positives, respectively, were considered as any sample that tested positive for Influenza A by viral culture, and had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched Influenza A/H1, A/H3, or A/2009 H1 sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” RSV A or RSV B positives, respectively, were considered as any sample that tested positive for Influenza A by viral culture, and had bi-directional sequencing data meeting predefined quality acceptance criteria that matched RSV A or RSV B sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” ADV C or ADV B/E positives, respectively, were considered as any sample that tested positive for Influenza A by viral culture, and had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched ADV C or ADV B/E sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values.

2Performance of the eSensor RVP assay detecting Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2 and Parainfluenza Virus 3 respectively was compared to viral culture followed by fluorescent antibody identification. “True” Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2 or Parainfluenza Virus 3 positives, respectively, were considered as any sample that tested positive for Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, or Parainfluenza Virus 3, respectively, by viral culture followed by DFA testing.

3Performance of the eSensor RVP assay detecting Human Rhinovirus or Human Metapneumovirus, respectively, was compared to a predetermined algorithm that used composite comparator methods. The methods consist of two analytically validated PCR assays followed by bi-directional sequencing. “True” Human Rhinovirus or Human Metapneumovirus positives, respectively, were considered as any sample that had bidirectional sequencing data meeting pre-defined quality acceptance criteria that matched Human Rhinovirus or Human Metapneumovirus sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih. gov), respectively, with acceptable E-values.

Depending on the comparator method used for a particular virus, performance is described as sensitivity/specificity or Positive Percent Agreement (PPA)/Negative Percent Agreement (NPA). Sensitivity or positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of true positive and false negative (FN) results, while specificity or negative percent agreement (NPV) was calculated by dividing the number of true negative (TN) results by the sum of true negative and false positive (FP) results. A TP result was one where the positive RVP result matched the positive reference/comparator result, while a TN result was one whereby a negative RVP result matched a negative reference/comparator result. The two-sided $9 5 \%$ confidence interval was also calculated. The detailed performance results from each viral target are summarized below.

eSensor RVP prospective clinical performance   

<table><tr><td rowspan=2 colspan=1>Virus(Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A1</td><td rowspan=1 colspan=1>132/137a</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>91.7% -98.8%</td><td rowspan=1 colspan=1>850/897b</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>93.1% -96.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza AH1*</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1027/1027</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.1% -100.0%</td><td rowspan=1 colspan=1>927/952</td><td rowspan=1 colspan=1>97.4%</td><td rowspan=1 colspan=1>96.2% -98.3%</td></tr><tr><td rowspan=1 colspan=1>Influenza A2009 H1N1</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.7% -100.0%</td><td rowspan=1 colspan=1>956/971d</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>97.5% -99.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>64/69e</td><td rowspan=1 colspan=1>92.8%</td><td rowspan=1 colspan=1>83.9% -97.6%</td><td rowspan=1 colspan=1>947/965f</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>97.1% -98.9%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 1*</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>39.8% -100.0%</td><td rowspan=1 colspan=1>1029/1030</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% -100.0%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 2*</td><td rowspan=1 colspan=1>5/6i</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>35.9% -99.6%</td><td rowspan=1 colspan=1>1026/1028k</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>64/681</td><td rowspan=1 colspan=1>94.1%</td><td rowspan=1 colspan=1>85.6% -98.4%</td><td rowspan=1 colspan=1>944/966m</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>96.6% -98.6%</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial VirusA</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7% -100.0%</td><td rowspan=1 colspan=1>905/956n</td><td rowspan=1 colspan=1>94.7%</td><td rowspan=1 colspan=1>93.1% -96.0%</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial VirusB</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.7% -100.0%</td><td rowspan=1 colspan=1>955/996°</td><td rowspan=1 colspan=1>95.9%</td><td rowspan=1 colspan=1>94.5% -97.0%</td></tr><tr><td rowspan=1 colspan=1>AdenovirusB/E*</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>75.3% -100.0%</td><td rowspan=1 colspan=1>1012/1021P</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>98.3% -99.5%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C*</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>54.1% -100.0%</td><td rowspan=1 colspan=1>993/10289</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>95.3% -97.5%</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.5% -100.0%</td><td rowspan=1 colspan=1>979/9818</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>132/148</td><td rowspan=1 colspan=1>89.2%</td><td rowspan=1 colspan=1>83.0% -93.7%</td><td rowspan=1 colspan=1>853/888h</td><td rowspan=1 colspan=1>96.1%</td><td rowspan=1 colspan=1>94.6% -97.3%</td></tr></table>

\*These viral targets were supplemented with retrospective samples as shown below.

1Influenza A results contain 14 Flu A samples without a positive subtype and 123 samples with either Influenza A H3 or 2009 H1N1 positive results.

a Flu A was not detected in all 5 RVP False Negative samples using independently developed and validated qPCR assays.

b Flu A viruses were confirmed positive in 35/47 RVP False Positive samples using bidirectional sequencing.

c Flu A H3 viruses were confirmed positive in 22/25 RVP False Positive samples using bidirectional sequencing.

d Flu A 2009 H1N1 viruses were confirmed positive in 14/15 RVP False Positive samples using bidirectional sequencing.

e Flu B was not detected in 4/5 RVP False Negative samples using bidirectional sequencing.

f Flu B was confirmed positive in 11/18 RVP False Positive samples using bidirectional sequencing.

$\mathrm { ^ g h M P V }$ was confirmed positive in 1/2 RVP False Positive samples using bidirectional sequencing.

h HRV was confirmed positive in 7/35 RVP False Positive samples using bidirectional sequencing.

i PIV 1 was not detected in this RVP False Positive sample by bidirectional sequencing.

j PIV 2 was not detected in this RVP False Negative sample using independently developed and validated qPCR assays.

k PIV 2 virus was confirmed positive in 0/2 RVP False Positive samples by bidirectional sequencing.

PIV 3 was not detected in 4/4 RVP False Negative samples using independently developed and validated qPCR assays.

m PIV 3 virus was confirmed positive in 10/22 RVP False Positive samples using bidirectional sequencing.

n RSV A were confirmed positive in 43/51 RVP False Positive samples using bidirectional sequencing.

o RSV B was confirmed positive in 35/41 RVP False Positive samples using bidirectional sequencing.

p ADV B/E was confirmed positive in 8/9 RVP False Positive samples using bidirectional sequencing.

q ADV C was confirmed positive in 16/35 False Positive samples using bidirectional sequencing.

Mixed Infections

The eSensor RVP system detected a total of 128 mixed infections in the prospective clinical evaluation (1037 tested and analyzed specimens). This represents $1 8 . 4 \%$ of the total positive specimens (128/696). One hundred fourteen (114/128; $8 9 . 1 \%$ ) were double infections, eleven $( 1 1 / 1 2 8 ; 8 . 6 \% )$ were triple infections, and three (3/128; $2 . 3 \%$ ) samples with four or more RVP analytes were identified. Ninety five of the 128 samples contained one or more analytes that the reference/comparator method failed to detect.

<table><tr><td colspan="5" rowspan="1">Distinct Co-infection Combinations Detectedby eSensor RVP</td><td colspan="1" rowspan="2">TotalNumberofCo-infections</td><td colspan="1" rowspan="2">Number ofDiscrepantCo-infections</td><td colspan="1" rowspan="2">DiscrepantAnalyte(s) *</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td></tr><tr><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ADV B (2), Flu B (1)</td></tr><tr><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ADV B (3)</td></tr><tr><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ADV B (1), RSV A (2)</td></tr><tr><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1), RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ADV C (3)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">ADV C (4), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">ADV C (3), RSV A (2))</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ADV C (3), RSV B (2)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1)</td></tr><tr><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADVB/E</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">ADV C (6)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Flu A (2), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H1N1 (1), H3 (1), HMPV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H1N1 (1), HRV (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Flu A (1), PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSV B (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H1N1 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSV A (2), RSV B (2)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">ADV C</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ADV C (1), Flu A (1), PIV3(1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Flu B (1), HRV (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Flu B (2), PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Flu B (2), RSV A (5)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV (1), PIV2 (1)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HMPV (1)</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">HMPV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV1 (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">HRV (4), PIV3 (2)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">HRV (5), RSV A (6)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HRV (1), RSV B (5)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV A (1)</td></tr><tr><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV B (1)</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">PIV3 (4), RSV A (5)</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV3 (1), RSV B (1)</td></tr><tr><td colspan="5" rowspan="1">Total Number of Co-infections</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">117/2781</td></tr><tr><td colspan="5" rowspan="1">Total Number of Double Infections</td><td colspan="1" rowspan="1">114</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">99/232</td></tr><tr><td colspan="5" rowspan="1">Total Number of Triple Infections</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">11/33</td></tr><tr><td colspan="5" rowspan="1">Total Number of Quadruple Infections</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5/8</td></tr><tr><td colspan="5" rowspan="1">Total Number of Quintuple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2/5</td></tr></table>

$^ * \mathrm { A }$ discrepant co-infection or discrepant analyte was defined as one that was detected by RVP but not the reference/comparator methods.

$^ { 1 } 1 1 7 / 1 1 7$ discrepant analytes were investigated using an alternate method; bi-directional sequence analysis identified the analyte in question in 58/117 cases.

$^ { \mathrm { a } } 6 / 6$ discrepant ADV B/E analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 5/6 cases   
$\mathsf { b } _ { 2 4 / 2 4 }$ discrepant ADV C analytes were investigated using an alternate method; bi  
directional sequence analysis identified the analyte in question in 11/24 cases   
$^ \mathrm { c } 6 / 6$ discrepant Flu B analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in $3 / 6$ cases   
${ ^ \mathrm { d } } _ { 4 / 4 }$ discrepant Flu A 2009 H1N1 analytes were investigated using an alternate method; bi  
directional sequence analysis identified the analyte in question in 4/4 cases   
$^ { \mathrm { e } } 1 / 1$ discrepant Flu A H3 analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 1/1 cases   
$\mathrm { f } _ { 2 / \bar { 2 } }$ discrepant HMPV analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 1/2 cases   
$^ { \mathrm { g } } 1 9 / 1 9$ discrepant HRV analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 3/19 cases   
$^ { \mathrm { h } } 1 \dot { 2 } / 1 2$ discrepant PIV3 analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 3/12 cases   
$\dot { 2 } \dot { 7 } / 2 7$ discrepant RSV A analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 17/27 cases   
$^ { \mathrm { j } } 1 \bar { 7 } / 1 7$ discrepant RSV B analytes were investigated using an alternate method; bi-directional   
sequence analysis identified the analyte in question in 11/17 cases

During the prospective clinical trial, there were a number of co-infection combinations which were detected by the reference/comparator methods but not detected by the eSensor RVP assay. These co-infections are summarized below:

<table><tr><td rowspan=1 colspan=2>Distinct Co-InfectionCombinations*</td><td rowspan=2 colspan=1>TotalNumber ofCo-Infections</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=2 colspan=1>DiscrepantAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Flu B (2), HRV (3)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Flu B (1), RSV B(1)</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>HRV (3), PIV3 (3)</td></tr></table>

\*This table includes only co-infections that were detected by the reference/comparator method but not by RVP; the remaining coinfections detected by the reference/comparator method are already represented in Table above.

# Retrospective Clinical Study

Banked samples previously characterized as positive for Influenza A H1, Parainfluenza Virus 1, Parainfluenza Virus 2, Adenovirus B/E, and Adenovirus C were used to supplement the performance studies for these analytes. These frozen banked samples were collected from various sites across the United States or from the Centers for Disease Control and Prevention (CDC). Upon arrival at GenMark, banked samples were blinded and intermixed with negative samples before being sent for testing, which was conducted by multiple sites involved in the prospective analysis of the patient samples. Testing of the banked samples was performed identically to prospectively-collected patient specimens. Results from the banked samples are presented separately from the prospectively collected specimens.

A total of 343 retrospective banked samples were collected for analysis. Out of this sample set, 11 samples were sent which didn't contain a banked viral target so these eleven samples were not tested further. Eight additional samples were excluded as they didn't contain a banked viral target as originally reported by the collection site and confirmed by comparator testing. Two samples reported errors on targets but were not retested as indicated. One sample was not sequenced. One sample had an internal control failure but was not retested as indicated. After these data were excluded, a total of 320 banked samples (including negative samples) for 5 viral targets were tested and analyzed.

With the exception of Flu A H1 samples, these banked samples were also sent to a third party lab for comparator testing, and the results from the third party lab testing were compared to the results obtained by the eSensor RVP. Since the Flu A H1 samples came from the Centers for Disease Control and Prevention and were verified to be Flu A H1, these samples were not tested further. The individual analyte results are presented below.

<table><tr><td rowspan=1 colspan=4>Influenza A H1 (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>291</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 96.7% (95% CI: 82.8% - 99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 100.0% (95% CI: 98.7% - 100.0%)</td></tr></table>

<table><tr><td rowspan=1 colspan=4>PIV1 (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>289</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (95% CI: 86.3% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 98.0% (95% CI: 95.6% - 99.3%)</td></tr></table>

<table><tr><td rowspan=1 colspan=4>PIV2 (Banked Samples)</td></tr><tr><td rowspan=2 colspan=1>eSensor RVP</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>284</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>294</td><td rowspan=1 colspan=1>320</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (95% CI: 86.8% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 96.6% (95% CI: 93.8% - 98.4%)</td></tr></table>

<table><tr><td colspan="4" rowspan="1">ADV B/E (Banked Samples)</td></tr><tr><td colspan="1" rowspan="2">eSensor RVP</td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">290</td><td colspan="1" rowspan="1">290</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">295</td><td colspan="1" rowspan="1">320</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement: 100.0% (95% CI: 86.3% - 100.0%)</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement: 98.3% (95% CI: 96.1% - 99.4%)</td></tr><tr><td colspan="4" rowspan="1">ADV C (Banked Samples)</td></tr><tr><td colspan="1" rowspan="2">eSensor RVP</td><td colspan="3" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">270</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">304</td><td colspan="1" rowspan="1">320</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement: 100.0% (95% CI: 80.6% - 100.0%)</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement: 88.8% (95% CI: 84.8% - 91.9%)</td></tr></table>

A summary of the individual performance tables is shown below:

<table><tr><td rowspan=2 colspan=1>Virus</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza AH1</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>82.8% -99.9%</td><td rowspan=1 colspan=1>290/290</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.7% -100.0%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 1</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.3% -100.0%</td><td rowspan=1 colspan=1>289/295</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>95.6% -99.3%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 2</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.8% -100.0%</td><td rowspan=1 colspan=1>284/294</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>93.8% -98.4%</td></tr><tr><td rowspan=1 colspan=1>AdenovirusB/E</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.3% -100.0%</td><td rowspan=1 colspan=1>290/295</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>96.1% -99.4%</td></tr><tr><td rowspan=1 colspan=1>AdenovirusC</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>80.6% -100.0%</td><td rowspan=1 colspan=1>270/304</td><td rowspan=1 colspan=1>88.8%</td><td rowspan=1 colspan=1>84.8% -91.9%</td></tr></table>

4. Clinical cut-off: N/A

5. Expected values/Reference range:

A prospective clinical study testing nasopharyngeal (NP) swab specimens was conducted during the 2010/11 influenza season at three North American clinical laboratories. The expected values of individual analytes and mixed co-infections based on eSensor RVP prospective sample testing results are summarized below:

<table><tr><td colspan="1" rowspan="2">Virus (Analyte)</td><td colspan="1" rowspan="1">Age 0-1(N =270)</td><td colspan="1" rowspan="1">Age &gt;1-5(N =136)</td><td colspan="1" rowspan="1">Age &gt;5-21(N = 127)</td><td colspan="1" rowspan="1">Age &gt;21-65(N = 333)</td><td colspan="1" rowspan="1">Age &gt;65(N =171)</td><td colspan="1" rowspan="1">All Ages(N =1037</td></tr><tr><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (Un-Subtypable)</td><td colspan="1" rowspan="1">2 (0.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.6)</td><td colspan="1" rowspan="1">5 (1.5)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">10 (1.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (Total)</td><td colspan="1" rowspan="1">25 (9.3)</td><td colspan="1" rowspan="1">22 (16.2)</td><td colspan="1" rowspan="1">17 (13.4)</td><td colspan="1" rowspan="1">84 (25.2)</td><td colspan="1" rowspan="1">31 (18.1)</td><td colspan="1" rowspan="1">179 (17.3)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">12 (4.8)</td><td colspan="1" rowspan="1">15 (11.0)</td><td colspan="1" rowspan="1">7 (5.5)</td><td colspan="1" rowspan="1">43 (12.9)</td><td colspan="1" rowspan="1">22(12.9)</td><td colspan="1" rowspan="1">99(9.5)</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009 H1N1</td><td colspan="1" rowspan="1">10 (3.7)</td><td colspan="1" rowspan="1">8 (5.9)</td><td colspan="1" rowspan="1">6 (4.7)</td><td colspan="1" rowspan="1">33 (9.9)</td><td colspan="1" rowspan="1">7 (4.1)</td><td colspan="1" rowspan="1">64 (6.2)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">10 (3.7)</td><td colspan="1" rowspan="1">17 (12.5)</td><td colspan="1" rowspan="1">33 (26.0)</td><td colspan="1" rowspan="1">15 (4.5)</td><td colspan="1" rowspan="1">7 (4.1)</td><td colspan="1" rowspan="1">82 (7.9)</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">18 (6.7)</td><td colspan="1" rowspan="1">11 (8.1)</td><td colspan="1" rowspan="1">3 (2.4)</td><td colspan="1" rowspan="1">15 (4.5)</td><td colspan="1" rowspan="1">10 (5.9)</td><td colspan="1" rowspan="1">57 (5.5)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">82 (30.4)</td><td colspan="1" rowspan="1">27 (19.9)</td><td colspan="1" rowspan="1">21 (16.6)</td><td colspan="1" rowspan="1">26 (7.8)</td><td colspan="1" rowspan="1">11 (6.4)</td><td colspan="1" rowspan="1">167 (16.1)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">3 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">5 (0.5)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">1 (0.4)</td><td colspan="1" rowspan="1">1 (0.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">7 (0.7)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">43 (15.9)</td><td colspan="1" rowspan="1">15 (11.0)</td><td colspan="1" rowspan="1">5 (3.9)</td><td colspan="1" rowspan="1">18 (5.4)</td><td colspan="1" rowspan="1">5 (2.9)</td><td colspan="1" rowspan="1">86 (8.3)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial VirusA</td><td colspan="1" rowspan="1">69 (25.6)</td><td colspan="1" rowspan="1">21 (15.4)</td><td colspan="1" rowspan="1">8 (6.3)</td><td colspan="1" rowspan="1">11(3.3)</td><td colspan="1" rowspan="1">10 (5.8)</td><td colspan="1" rowspan="1">119 (11.4)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial VirusB</td><td colspan="1" rowspan="1">28 (10.4)</td><td colspan="1" rowspan="1">17 (12.5)</td><td colspan="1" rowspan="1">4 (3.2)</td><td colspan="1" rowspan="1">14 (4.2)</td><td colspan="1" rowspan="1">6 (3.5)</td><td colspan="1" rowspan="1">69 (6.7)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B/E</td><td colspan="1" rowspan="1">6 (2.2)</td><td colspan="1" rowspan="1">8 (5.9)</td><td colspan="1" rowspan="1">3 (1.6)</td><td colspan="1" rowspan="1">5 (1.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">22 (2.1)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C</td><td colspan="1" rowspan="1">21 (7.7)</td><td colspan="1" rowspan="1">4 (2.9)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">9 (2.7)</td><td colspan="1" rowspan="1">6 (3.5)</td><td colspan="1" rowspan="1">41 (3.9)</td></tr></table>

Below is the summary of expected values (as determined by eSensor RVP) by site during the prospective clinical sample testing.

<table><tr><td rowspan=2 colspan=1>Virus (Analyte)</td><td rowspan=1 colspan=1>Site 1(N =245)</td><td rowspan=1 colspan=1>Site 3(N =533)</td><td rowspan=1 colspan=1>Site 4(N =259)</td><td rowspan=1 colspan=1>All Sites(N = 1037)</td></tr><tr><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td><td rowspan=1 colspan=1>N (%)</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Un-Subtypable)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (1.5)</td><td rowspan=1 colspan=1>2 (0.8)</td><td rowspan=1 colspan=1>10 (1.0)</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Total)</td><td rowspan=1 colspan=1>58 (23.7)</td><td rowspan=1 colspan=1>89 (16.7)</td><td rowspan=1 colspan=1>32 (12.4)</td><td rowspan=1 colspan=1>179 (17.3)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>32 (13.1)</td><td rowspan=1 colspan=1>54 (10.1)</td><td rowspan=1 colspan=1>13 (5.0)</td><td rowspan=1 colspan=1>99 (9.5)</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>19 (7.8)</td><td rowspan=1 colspan=1>28 (5.4)</td><td rowspan=1 colspan=1>17 (6.6)</td><td rowspan=1 colspan=1>64 (6.2)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>4 (1.6)</td><td rowspan=1 colspan=1>59 (11.1)</td><td rowspan=1 colspan=1>19 (7.3)</td><td rowspan=1 colspan=1>82 (7.9)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>23 (9.4)</td><td rowspan=1 colspan=1>25 (4.7)</td><td rowspan=1 colspan=1>9 (3.5)</td><td rowspan=1 colspan=1>57 (5.5)</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>44 (18.0)</td><td rowspan=1 colspan=1>99 (18.6)</td><td rowspan=1 colspan=1>24 (9.3)</td><td rowspan=1 colspan=1>167 (16.1)</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (0.8)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>5 (0.5)</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 2</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>6 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>7 (0.7)</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>68 (12.8)</td><td rowspan=1 colspan=1>15 (5.8)</td><td rowspan=1 colspan=1>86 (8.3)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial VirusA</td><td rowspan=1 colspan=1>17 (6.9)</td><td rowspan=1 colspan=1>85 (15.9)</td><td rowspan=1 colspan=1>17(6.6)</td><td rowspan=1 colspan=1>119 (11.4)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial VirusB</td><td rowspan=1 colspan=1>15 (6.1)</td><td rowspan=1 colspan=1>41 (7.7)</td><td rowspan=1 colspan=1>13 (5.0)</td><td rowspan=1 colspan=1>69 (6.7)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B/E</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>14 (2.6)</td><td rowspan=1 colspan=1>8 (3.1)</td><td rowspan=1 colspan=1>22 (2.1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>16 (6.5)</td><td rowspan=1 colspan=1>19 (3.6)</td><td rowspan=1 colspan=1>6 (2.3)</td><td rowspan=1 colspan=1>41 (3.9)</td></tr></table>

Below is the summary of expected values (as determined by eSensor RVP) by age group during the prospective clinical sample testing.

<table><tr><td rowspan=2 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Age 0-1(N =270)</td><td rowspan=1 colspan=1>Age &gt;1-5(N =136)</td><td rowspan=1 colspan=1>Age &gt;5-21(N =127)</td><td rowspan=1 colspan=1>Age &gt;21-65(N = 333)</td><td rowspan=1 colspan=1>Age &gt;65(N =171)</td><td rowspan=1 colspan=1>All Ages(N =1037)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>N (%)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + Flu B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + HRV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + PIV3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV B/E + HMPV + HRV + RSV A+ RSV B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + Flu B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + HMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>ADV C+HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6 (0.6</td></tr><tr><td rowspan=1 colspan=1>ADV C + PIV3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADV C + RSV A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4 (0.4)</td></tr><tr><td rowspan=1 colspan=1>ADV C +RSV B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>ADV C + HRV + PIV3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>ADVC+HRV+RSVA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV B/E</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6 (0.6)</td></tr><tr><td rowspan=1 colspan=1>Flu A + Flu B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + HMPV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + HRV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4 (0.4)</td></tr><tr><td rowspan=1 colspan=1>Flu A + PIV2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + HRV + PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A+RSV A+RSV B</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV C + HRV + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu A + ADV C + HRV + PIV3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu B + HRV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4 (0.4)</td></tr><tr><td rowspan=1 colspan=1>Flu B + PIV3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3 (0.3)</td></tr><tr><td rowspan=1 colspan=1>Flu B + RSV A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5 (0.5)</td></tr><tr><td rowspan=1 colspan=1>Flu B + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu B + HRV + PIV2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>Flu B + HRV + RSV A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>HMPV + HRV</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5 (0.5)</td></tr><tr><td rowspan=1 colspan=1>HMPV + PIV3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HMPV + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (0.2)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11 (1.1)</td></tr><tr><td rowspan=1 colspan=1>HRV +RSVA</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16 (1.6)</td></tr><tr><td rowspan=1 colspan=1>HRV +RSV B</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8 (0.8)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV3 + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>HRV + PIV3 + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr><tr><td rowspan=1 colspan=1>PIV3 + RSV A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6 (0.6)</td></tr><tr><td rowspan=1 colspan=1>PIV3 + RSV B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.1)</td></tr></table>

Below is the summary of expected values for co-infection (as determined by eSensor RVP) by site during the prospective clinical sample testing.

<table><tr><td colspan="1" rowspan="2">Co-Infection</td><td colspan="1" rowspan="1">Site 1(N =245)</td><td colspan="1" rowspan="1">Site 2(N =533)</td><td colspan="1" rowspan="1">Site 3(N =259)</td><td colspan="1" rowspan="1">All Sites(N =1037)</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N (%)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + Flu B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + HRV</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV B/E + HMPV + HRV + RSVA+RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + Flu B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HMPV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6 (0.6</td></tr><tr><td colspan="1" rowspan="1">ADV C + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + RSV A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4 (0.4)</td></tr><tr><td colspan="1" rowspan="1">ADV C + RSV B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">ADV C + HRV + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV B/E</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV C</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6 (0.6)</td></tr><tr><td colspan="1" rowspan="1">Flu A + Flu B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + HMPV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4 (0.4)</td></tr><tr><td colspan="1" rowspan="1">Flu A + PIV2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + RSV A + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV C + HRV + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu A + ADV C + HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu B + HRV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4 (0.4)</td></tr><tr><td colspan="1" rowspan="1">Flu B + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3 (0.3)</td></tr><tr><td colspan="1" rowspan="1">Flu B + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5 (0.5)</td></tr><tr><td colspan="1" rowspan="1">Flu B + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu B + HRV + PIV2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">Flu B + HRV +RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">HMPV + HRV</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5 (0.5)</td></tr><tr><td colspan="1" rowspan="1">HMPV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HMPV + RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2 (0.2)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">11 (1.1)</td></tr><tr><td colspan="1" rowspan="1">HRV +RSV A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">16 (1.6)</td></tr><tr><td colspan="1" rowspan="1">HRV +RSV B</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">8 (0.8)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV3 + RSV A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">HRV + PIV3 + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1 (0.1)</td></tr><tr><td colspan="1" rowspan="1">PIV3 + RSV A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6 (0.6)</td></tr><tr><td colspan="1" rowspan="1">PIV3 + RSV B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1 (0.1)</td></tr></table>

# N. Instrument Name:

eSensor Respiratory Viral Panel (RVP)

# O. System Descriptions:

1. Modes of Operation:

The eSensor XT-8 is a clinical multiplex instrument that has a modular design consisting of a base module and one, two, or three cartridge-processing towers containing 8, 16, or 24 cartridge slots, respectively. The cartridge slots operate independently of each other. Any number of cartridges can be loaded at one time, and the remaining slots are available for use while the instrument is running.

The base module controls each processing tower, provides power, and stores and analyzes data. The base module includes the user interface, and a 15 inch portrait-orientation display and touch panel. The instrument is designed to be operated solely with the touch screen interface. Entering patient accession numbers and reagent lot codes can be performed by the bar code scanner, the touch screen, or uploading a text file from a USB memory stick.

Each processing tower consists of eight cartridge modules, each containing a cartridge connector, a precision-controlled heater, an air pump, and electronics. The air pumps drive the diaphragm pump and valve system in the cartridge, eliminating fluid contact between the instrument and the cartridge. The pneumatic pumping enables recirculation of the hybridization solution allowing the target DNA and the signal probes to hybridize with the complementary capture probes on the electrodes. The diaphragm pump in the cartridge is connected to a pneumatic source from the eSensor XT-8 instrument and provides unidirectional pumping of the hybridization mixture through the microfluidic channel during hybridization. Using microfluidic technology to circulate the hybridization solution minimizes the unstirred boundary layer at the electrode surface and continuously replenishes the volume above the electrode that has been depleted of complementary targets and signal probes.

The XT-8 instrument provides electrochemical detection of bound signal probes by ACV and subsequent data analysis and test report generating functions. All hybridization, ACV scanning and analysis parameters are defined by a scanning protocol loaded into the XT-8 Software, and then specified for use by the EEPROM on each cartridge

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this

line of product types: Yes ___X_ __ or No

Level of Concern: Moderate

# Software Description:

The eSensor XT-8 system software consists of instrument software and assay specific software called “Assay Analysis Module” or “AAM”. The instrument software consists of: 1) Application Software, which is the user interface; 2) Instrument Firmware; 3) Windows XP Operating System; 4) Printer Software; and 5) On-line User Manual for the instrument. The Application software was developed using $\mathrm { C } \#$ in the Microsoft Visual Studio .NET 2.0 development framework. All of the hardware functions of the instrument are controlled by embedded firmware under the command of the Application Software. The embedded firmware was developed using the C language with the Greenhills compiler for the CPU firmware and Texas Instrument Code Composer for the DSP firmware.

The AAM is a software plug-in specifically designed to generate detection, contamination, and configuration reports for specific Assays. AAM consists of four sub-components: 1) AAM Instrument Application Interface, 2) AAM Data Analysis Module, 3) AAM Assay Specific Report Module, and 4) AAM Assay Specific Configuration Files. The AAM Instrument Application Interface is used to interface the AAM with XT-8 Application Software. The AAM Data Analysis Module performs signal classification, signal error detection, and statistics. The AAM Assay Specific Report Module performs genotyping, while the Configuration Files are used to specify configurable parameters for the assay specific polymorphisms. Among all the software components, only the AAM Assay Specific Report Module and the Configuration Files are assay specific to Respiratory Viral Panel Test. By keeping these assay-specific AAMs separate from the other modules, they are individually validated and installed on an instrument without having to revalidate the rest of the software such as Instrument software and two AAM sub-components (i.e., AAM Instrument Application Interface and AAM Data Analysis Module), which have been cleared previously by the FDA in K073720 and K090901.

FDA has reviewed the hazard analysis and software documentation for the cleared XT-8 system under 510(k) K073720. The XT-8 Instrument Software (including Application Software, Instrument Firmware, Window XP OS, Printer Software, and On-line User Manual for the instrument) and two AAM sub-components (i.e., AAM Instrument Application Interface and AAM Data Analysis Module) have not been changed in this 510(k). The only changes are to the two “assay specific” modules, AAM Assay Specific Report Module and AAM Assay Specific Configuration Files. The validation study results are provided in Section 7.0. The data provided in Section 7.0 also include a description of software design, evaluation of the hazard analysis and AAM validation testing.

The XT-8 instrument is only for use with XT-8 test cartridges and is not integrated or connected with other laboratory systems.

Device Hazard Analysis:   
Acceptable as reviewed in document RVP-21   
Architecture Design Chart:   
Acceptable as reviewed in document REC-1004   
Software Design Specification:   
Acceptable as reviewed in document REC-1004   
Traceability Analysis:   
Acceptable as reviewed in document REC-1527 section 6   
Software Development Environment Description:   
Acceptable as reviewed in document REC-1004   
Verification and Validation Testing:   
Acceptable as reviewed in document REC-1527   
Revision Level History:   
Acceptable as reviewed in document REC-1527 (RVP IVD Version 1.0) Unresolved Anomalies:   
Acceptable as reviewed in document REC-1527 section 3.2   
Off the Shelf Software or Software of Unknown Pedigree:   
None reported.

3. Specimen Identification:

Specimens are manually identified and sample ID’s are entered into the system software.

4. Specimen Sampling and Handling:

The specimen type is a Nasopharyngeal Swab (NPS) Collection. The NPS specimen collection should be performed according to standard technique and placed in viral transport media.

Minimum Sample Volume‐ $2 0 0 \mu \mathrm { L }$ NPS specimen is required for testing.

Transport and Storage‐ Clinical specimens can be stored between $2 \ { } ^ { \circ } \mathrm { C }$ and $8 ~ ^ { \circ } \mathrm { C }$ for up to 7 days after collection in viral transport media. Specimens can also be stored at ${ < } { - } 1 5 ~ ^ { \circ } \mathrm { C }$ for up to 1 month prior to extraction and undergo 2 freeze thaw cycles.

Note: Storage of purified nucleic acids have been validated for storage between $2 \ { } ^ { \circ } \mathrm { C }$ and 8 $^ \circ C$ for up to 7 days and at ${ < } { - } 1 5 ~ ^ { \circ } \mathrm { C }$ for up to 1 month with up to 2 freeze thaws.

# 5. Calibration:

No routine calibration or user maintenance is required.

# 6. Quality Control:

See section M.c for a discussion of the quality control materials

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

None

Q. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.