# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: k051824   
B. Purpose for Submission: Clearance of new device

F. Proprietary and Established Names: Invader® UGT1A1 Molecular Assay

# G. Regulatory Information:

2. Classification: Class II

3. Product code: NTI, drug metabolizing enzyme genotyping system

4. Panel: Toxicology (91)

# H. Intended Use:

1. Intended use(s): See indications for use below

2. Indication(s) for use:

The Invader® UGT1A1 Molecular Assay is an in vitro diagnostic test for the detection and genotyping of the $^ { * 1 }$ (TA6) and $^ { * } 2 8$ (TA7) alleles of the UDP glucuronosyltransferase 1A1 (UGT1A1) gene in genomic DNA from whole peripheral blood as an aid in the identification of patients with greater risk for decreased UDP-glucuronosyltransferase activity.

3. Special conditions for use statement(s): For professional use.

The information provided from this test may supplement therapeutic decision-making and should only be used in conjunction with routine monitoring by a physician. Because of the variability in the knowledge of clinical utility with certain drugs that are metabolized by UGT1A1, clinicians should use professional judgment in the interpretation of results from this type of test. The Invader® UGT1A1 Molecular Assay should not be used: 1) as the only test to determine specific drug dose. Other clinical information and patient history should primarily be considered, 2) to aid in predicting a patient’s drug response for drugs that are not metabolized by the enzyme encoded by UGT1A1, and 3) to aid in predicting a patient’s response to drugs for which the mutant UGT1A1 phenotype has not been clearly established.

4. Special instrument requirements:

Tecan GENios, Tecan GENios FL, or Bio-Tek FLX800 fluorometers Call Reporting Software (CRS), UGT1A1_CRS_v060705.xlt

# I. Device Description:

The Invader® UGT1A1 Molecular Assay is an in vitro diagnostic test that utilizes sequence-specific DNA probes, a structure-specific cleavage enzyme and a universal fluorescent resonance energy transfer (FRET) system combined with interpretative software and third party microtiter plate reader instrumentation. The assay is designed to identify specific nucleic acid sequences and query for the presence of known sequence polymorphisms through analysis of structure-specific cleavage of a series of probes that are specifically complementary either to wild type or mutant sequences of the “TATA Box” promoter region of the UGT1A1 $^ { * 1 }$ (TA6) and $^ { * } 2 8$ alleles (TA7) of the UDP glucuronosyltransferase 1A1 (UGT1A1) gene.

J. Substantial Equivalence Information:

1. Predicate device name(s): AmpliChip CYP450 Test for CYP2C19

2. Predicate 510(k) number(s): k043576

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Invader® UGT1A1Molecular Assay</td><td rowspan=1 colspan=1>AmpliChip CYP450 Test forCYP2C19 (K043576)</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>genomic DNA obtainedfrom a human wholeperipheral blood sample</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Reference method</td><td rowspan=1 colspan=1>bi-directional DNAsequencing</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Detection</td><td rowspan=1 colspan=1>Single nucleotidepolymorphism</td><td rowspan=1 colspan=1>Same</td></tr></table>

Page - 3/9   

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Outcome data</td><td rowspan=1 colspan=1>Assay signal results areinterpreted by a softwareprogram and are assigned agenotype that is presented tothe end user in a reportformat</td><td rowspan=1 colspan=1>same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Invader® UGT1A1 MolecularAssay</td><td rowspan=1 colspan=1>AmpliChip CYP450 Test forCYP2C19 (K043576)</td></tr><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>UGT1A1 *1 and *28</td><td rowspan=1 colspan=1>2C19 *1, *2, and *3</td></tr><tr><td rowspan=1 colspan=1>Reaction conditions</td><td rowspan=1 colspan=1>1) No thermal cycling;isothermal reaction at 63° C2) Utilizes signal amplification3) Reactions occur in multipleplastic microtiter wells</td><td rowspan=1 colspan=1>1) Utilizes thermal cycling) Utilizes target DNAamplification3) Reactions occur on a singleglass slide</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

<table><tr><td rowspan=2 colspan=1>User Fees and Refunds for PremarketNotification Submissions (510(k)s) -Guidance for Industry and FDA Staff</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/mdufma/guidance/1511.html</td></tr><tr><td></td></tr><tr><td rowspan=1 colspan=1>Drug Metabolizing Enzyme GenotypingSystem - Class II Special Controls GuidanceDocument - Guidance for Industry and FDAStaff</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/oivd/guidance/1551.html</td></tr><tr><td rowspan=1 colspan=1>Shelf Life of Medical Devices</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/ode/415.pdf</td></tr><tr><td rowspan=1 colspan=1>Guidance for Off-the-Shelf Software Use inMedical Devices; Final</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/ode/guidance/585.html</td></tr><tr><td rowspan=1 colspan=1>A New 510(k) Paradigm - AlternateApproaches to Demonstrating SubstantialEquivalence in Premarket Notifications</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/ode/parad510.html</td></tr><tr><td rowspan=1 colspan=1>Guidance for the Content of PremarketSubmissions for Software Contained inMedical Devices; Final</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/ode/377.html</td></tr><tr><td rowspan=1 colspan=1>Statistical Guidance on Reporting Resultsfrom Studies Evaluating Diagnostic Tests;Draft Guidance for Industry and FDAReviewers</td><td rowspan=1 colspan=1>http://www.fda.gov/cdrh/osb/guidance/1428.html</td></tr></table>

# L. Test Principle:

The Invader $\textsuperscript { \textregistered }$ UGT1A1 Molecular Assay permits the analysis of specific nucleic acid sequences and a query for the presence of known sequence polymorphisms. The basis of the Invader assay is the ability of an enzyme, provided with the test, to recognize and cleave specific structures formed by the addition of two oligonucleotides. The two oligonucleotides, a discriminatory primary probe and an invader probe, hybridize in tandem to the target DNA to an overlapping structure. The $_ 5 \cdot$ -end of the discriminatory probe includes a $5 '$ -flap that does not hybridize to the target DNA. The $_ { 3 } ,$ - nucleotide of the bound invader probe overlaps the primary probe, but need not hybridize to the target DNA. A specific enzyme (the cleavase enzyme) recognizes this overlapping structure and cleaves off the unpaired $5 '$ -flap of the discriminatory probe, releasing it into the buffer. The discriminatory probe is designed to have a melting temperature close to the reaction temperature. Thus, under the assay conditions, the discriminatory probe cycles on the target DNA isothermally. This allows for multiple round of discriminatory probe cleavage for each target DNA, and therefore to an amplification of the number of released $_ 5 \cdot$ -flaps.

In a second reaction within the same reaction tube, each released $5 '$ -flap can serve as an invader oligonucleotide on a fluorescence resonance energy transfer (FRET) cassette to create another overlapping structure that is recognized and cleaved by the same enzyme mentioned above (cleavase enzyme). When the FRET cassette is cleaved, a fluorophore is separated from a quencher and detectable fluorescence signal is generated. Similar to the initial reaction, the released $5 '$ -flap and FRET cassette cycle, results in amplified fluorescence signal. The initial and secondary reactions run concurrently in the same well.

The biplex format of the assay enables simultaneous detection of two DNA sequences in a single well. It allows the detection of the UGT1A1 genotype specified in the indication for use and an alpha actin internal control in each well. The biplex format uses two different discriminatory probes, each with a unique $5 '$ -flap, and two different fluorescence resonance energy transfer cassettes, each with a spectrally distinct fluorophore.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the Invader $\textsuperscript { \textregistered }$ UGT1A1 Molecular Assay was evaluated with a panel of 20 blinded whole blood samples that represented UGT1A1 $^ { * 1 }$ $( \mathrm { T A } _ { 6 / 6 } )$ $( \mathtt { n } = \mathtt { 6 } )$ ), UGT1A1 $^ { * } 2 8$ heterozygous $( \mathrm { T A } _ { 6 / 7 } )$ $( \mathtt { n } = 4 )$ ), UGT1A1 $^ { * } 2 8$ homozygous $( \mathrm { T A } _ { 7 / 7 } )$ $( \mathtt { n } = 4 )$ , and other genotypes $( \mathtt { n } = 5 )$ , at a concentration of 100 to $7 0 0 \mathrm { n g }$ per reaction. The testing was conducted at three different sites, including two external sites and a laboratory at Third Wave Technologies Inc. using one lot or reagents. The samples underwent DNA extraction (in triplicate) using the Qiagen QIAamp DNA Extraction methodology. DNA from the samples underwent subsequent Invader analysis at each site on each of five days. Each site utilized a different fluorometer to determine the performance of the assay. The genotype of the different blood samples was determined by bi-directional DNA sequence analysis. The results from this study are summarized below:

<table><tr><td rowspan=1 colspan=12>Table 1. Inter-laboratory reproducibility of the Invader® UGT1A1 Molecular Assay</td></tr><tr><td rowspan=2 colspan=1>UGT1A1Genotype</td><td rowspan=2 colspan=1>#tested</td><td rowspan=2 colspan=1>1struntestspersite</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=5>1st Run Results</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1>CorrectCallRatefor allruns</td></tr><tr><td rowspan=1 colspan=1>GenotypeCalls</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>In-correctCalls</td><td rowspan=1 colspan=1>#Invalid 2</td><td rowspan=1 colspan=1>CorrectCallRate</td><td rowspan=1 colspan=1>#Invalid after2drun2</td><td rowspan=1 colspan=1>In-correctCallsfor allruns</td></tr><tr><td rowspan=3 colspan=1>*1(TA6/6)</td><td rowspan=3 colspan=1>6</td><td rowspan=3 colspan=1>90</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>793</td><td rowspan=1 colspan=1>3$</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>87.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>$3^3}$</td><td rowspan=1 colspan=1>94.4%3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.9%</td></tr><tr><td rowspan=3 colspan=1>*28hetero-zygous(TA6/7)</td><td rowspan=3 colspan=1>5</td><td rowspan=3 colspan=1>75</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>92%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.7%</td></tr><tr><td rowspan=3 colspan=1>*28 homozygous(TA7/7)</td><td rowspan=3 colspan=1>4</td><td rowspan=3 colspan=1>60</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>503</td><td rowspan=1 colspan=1>3^3}$</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>$3^3}$</td><td rowspan=1 colspan=1>93.3%3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=3 colspan=1>Other</td><td rowspan=3 colspan=1>5</td><td rowspan=3 colspan=1>75</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>63^3}$</td><td rowspan=1 colspan=1>53}$</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>84.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>$5^r3}$</td><td rowspan=1 colspan=1>92.0%3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=2>900</td><td rowspan=1 colspan=1>851</td><td rowspan=1 colspan=1>8403</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>98.1%4</td></tr></table>

1 Genotype determined using bi-directional sequencing 2 Invalid $=$ reported to user as “Low Signal”. Insufficient signal generated to make a genotype call 3 Site 1 experienced user error that resulted in mis-order of 9 genotyping reactions (3 samples for all 3 replicates on day 5). 4 Correct call rate for all runs is based on 883/900 genotype calls.

Correct genotype calls for UGT1A1 were obtained for 840/900 $( 9 3 . 3 \% )$ samples after the $1 ^ { \mathrm { s t } }$ run. After retesting 49 samples, 43/49 were correctly identified after the $2 ^ { \mathrm { n d } }$ run. The total correct genotype calls for the combined first and second runs was 883/900 $( 9 8 . 1 \% )$ . Nine samples were mis-ordered in the reaction plate at site 1 on day 5. As a result, the 3 replicates for each sample produced genotype results that were inconsistent with the bi-directional sequencing result for those samples. Discrepancy resolution by sequencing of the template in the original Invader® assay plates confirmed that the genotypes present in the assay wells were consistent with the reported genotypes of the assay.

From the 49 invalid calls after the 1st run, 20 were due to an invalid “Positive Control” result, 20 were due to an invalid “Negative Control” result, and the remaining nine were due to “Low Signal” as a result. After retesting, 6 genotype reactions were still reported as “Low Signal.”

# Lot-to-Lot reproducibility:

Whole blood samples $( \mathtt { n } = 4 0 )$ underwent DNA extraction and subsequent bi-directional DNA sequence analysis. The same DNA samples were then analyzed using the Invader® UGT1A1 Molecular Assay using three different lots of the reagents. The observed agreement between all three lots of the Invader® UGT1A1 Molecular Assay and bi

directional DNA sequencing was $100 \%$ (120/120). The $9 5 \%$ one-sided lower confidence limit was $9 2 . 8 \%$ .

b. Linearity/assay reportable range: Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Assay Shelf Life Stability: Archived genomic DNA samples representing UGT1A1 $^ { * 1 }$ , UGT1A1 $^ { * } 2 8$ (heterozygous) and UGT1A1 $^ { * } 2 8$ (homozygous) genotypes were used for testing. These samples were derived from whole blood samples anticoagulated with tripotassium EDTA, subjected to genomic DNA extraction using the Qiagen QIAamp 96 Blood Kit method, and quantitated using the PicoGreen method. The DNA had been stored at $+ 4 { - } 8 ^ { \circ } \mathrm { C }$ in AE buffer for ${ \sim } 3$ months in a monitored refrigerator, according to Qiagen’s recommended storage conditions. The stability of the Invader® UGT1A1 Molecular Assay was evaluated using three different lots of assay components. Each lot was tested for Standard Storage conditions (unopened kit stored at ${ } _ { - 2 0 ^ { \circ } \mathrm { C } }$ until time of testing). The assay demonstrated $100 \%$ agreement with bi-directional DNA sequencing genotypes up to the expiration date of 30 days, when stored at $- 2 0 \mathrm { { ^ \circ C } }$ .

# d. Detection limit:

Lower Limit of Detection: The lower limit of detection of the Invader® UGT1A1 Molecular Assay was determined by analysis of dilutions of genomic DNA samples to 150, 100 and 50 ng DNA/reaction. Forty (40) replicates of genomic DNA samples representing the $^ { * 1 }$ , $^ { * } 2 8$ heterozygous, and $^ { * } 2 8$ homozygous UGT1A1 alleles were tested and compared to bidirectional DNA sequencing results. There was $100 \%$ agreement between the Invader® UGT1A1 Molecular Assay and bi-directional DNA sequencing for all samples at each of tested concentrations. The ability of the assay to detect UGT1A1 $^ { * 1 }$ , UGT1A1 $^ { * } 2 8$ (heterozygous) and UGT1A1 $^ { * } 2 8$ (homozygous) genotypes is not adversely affected by lower than recommended patient sample DNA concentrations. The lower limit of patient sample DNA is defined as $1 0 0 ~ \mathrm { n g }$ per reaction, or $1 0 \mathrm { n g / \mu L }$ .

Upper Limit of Detection: The upper limit of detection of the Invader® UGT1A1 Molecular Assay was determined by analysis of genomic DNA samples at $8 0 0 ~ \mathrm { n g }$ DNA/reaction. Forty (40) replicates of genomic DNA samples representing the $^ { * 1 }$ , $^ { * } 2 8$ heterozygous, and $^ { * } 2 8$ homozygous UGT1A1 alleles were tested at a concentration of $8 0 0 \mathrm { n g / }$ reaction and compared to bi-directional DNA sequencing results. There was $100 \%$ agreement between the Invader® UGT1A1 Molecular Assay and bi-directional DNA sequencing demonstrating the capability to accurately detect UGT1A1 TATA box sequences at DNA concentration levels at the recommended upper DNA concentration limit of $7 0 0 \mathrm { n g }$ per reaction, or 70 $\mathrm { n g / \mu L }$ .

# e. Analytical specificity:

Interfering Substances The potentially interfering substances Bilirubin, Lipids, K2 EDTA, $5 \%$ Qiagen Buffer AW2, $1 \%$ Qiagen Buffer AW2, and Hemoglobin were either added to the blood sample or directly to the purified genomic DNA, depending on the nature of the proposed interferent.

Compounds added to the blood samples include endogenous substances and those associated with blood collection. Specifically, the substances included bilirubin, lipids, and dipotassium EDTA. Bilirubin and lipids were added to blood samples to evaluate assay performance with samples acquired from individuals with "high-test" levels of either substance. Dipotassium EDTA was added to blood samples to represent an incomplete blood draw, resulting in an increased amount of anticoagulant in the whole blood sample. Compounds that were added directly to purified genomic DNA include Qiagen Buffer AW2 and hemoglobin. Buffer AW2 was added directly to the purified DNA sample, to mimic the impact of incomplete removal of residual wash buffer. While hemoglobin is endogenous to the original blood sample, it was added to the purified DNA to determine its direct impact on assay performance. Sixteen samples of known UGT1A1 genotype (as determined through bidirectional DNA sequence analysis) underwent genomic DNA isolation using the Qiagen QIAamp methodology, and were tested along with four subtractive blanks, two controls, and a No Target Control (NTC). Interference was measured by calculation of percent agreement of genotype calls between spiked and non-spiked samples. The sponsor reports that the percent agreement of the genotype between the spiked and non-spiked samples varied between $8 7 . 5 \%$ and $100 \%$ . The Invader® UGT1A1 Molecular Assay performance was not adversely affected by higher than normal blood levels of bilirubin (5-fold, or $8 \mathrm { m g / d L }$ ) or lipids (mono-, di-, and triglycerides, 2-fold, or $1 5 0 \mathrm { m g / d L }$ ). Hemoglobin (2-fold, or $0 . 0 2 5 \%$ ) or $1 \%$ residual Qiagen AW2 DNA extraction buffer also did not affect the assay. Decreased performance was seen with DNA samples containing $5 \%$ residual Qiagen DNA extraction AW2 buffer. $5 \%$ AW2 buffer resulted in 1 incorrect genotype call and 1 ˝low signal˝.

$f .$ Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device: Whole blood samples $\mathrm { \Delta } \mathrm { n } = 2 8 5 \mathrm { \Omega }$ underwent DNA extraction and subsequent bi-directional DNA sequence analysis. The same DNA samples were then analyzed using the Invader® UGT1A1 Molecular Assay. The observed agreement between the Invader® UGT1A1 Molecular Assay and bi-directional DNA sequencing was $100 \%$ (285/285). The one-sided $9 5 \%$ confidence lower limit ranged from $9 0 . 5 \%$ to $9 7 . 3 \%$ . The one-sided $9 5 \%$ confidence lower limit overall agreement with bi-directional DNA sequencing was $9 8 . 9 5 \%$ (see Table):

<table><tr><td rowspan=1 colspan=1>UGT1A1Genotype (bi-directionalsequencing)</td><td rowspan=1 colspan=1># ofgeno-typestested</td><td rowspan=1 colspan=1># ofreplicatespersample</td><td rowspan=1 colspan=1># of correctgenotype callsin 1st run</td><td rowspan=1 colspan=1># of invalidcalls in 1strun</td><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95%One-sidedCIlowerlimit</td></tr><tr><td rowspan=1 colspan=1>*1(TA6/6)</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.0%</td></tr><tr><td rowspan=1 colspan=1>*28 heterozygous(TA6/7)</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.3%</td></tr><tr><td rowspan=1 colspan=1>*28homozygous(TA7/7)</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.5%</td></tr><tr><td rowspan=1 colspan=1>Others</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>285</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>285</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.95%</td></tr></table>

# b. Matrix comparison:

Component Tolerance Study   
Whole blood samples $( \mathrm { n } { = } 2 0 )$ ) anticoagulated with dipotassium EDTA were subjected to the Qiagen QIAamp DNA extraction procedure. An aliquot of the DNA samples were tested using the Invader $\textsuperscript { \textregistered }$ UGT1A1 Molecular Assay with non-standard reaction conditions by manipulating the parameters of both sample and reagent volume addition to the reaction. The reagents, Oligo Mix and the Enzyme-Buffer Mix, were manipulated separately. A second aliquot of the DNA samples were PCR amplified and analyzed by bi-directional sequencing to assign a reference genotype. Twenty genomic DNA samples, 2 sample controls, and a NTC sample were tested in singlicate with each assay Oligo Mix. Four Subtractive Water Blank (Blank) samples were also included on each plate. Standard reagent and sample volumes and $+ / - 1 0 \%$ or $+ / - 2 0 \%$ of the standard sample or reagent volume were added to the appropriate reaction mixes. One reaction mix condition was tested per plate. The percent agreement of final calls between each non-standard and standard reaction condition was determined by comparison of the Invader® UGT1A1 Molecular Assay result to the genotype as confirmed using bi-directional DNA sequencing. The sponsor determined $100 \%$ agreement $8 6 . 1 \%$ at one sided $9 5 \%$ confidence lower limit) across all reagent and sample manipulations between the reference genotypes and the Invader genotypes for the 20 samples tested. The Invader® UGT1A1 Molecular Assay performance was not adversely affected by $\pm 2 0 \%$ variation in reagent or sample volumes. Freeze –Thaw Stability   
The Freeze-Thaw Stability of the Invader $\textsuperscript { \textregistered }$ UGT1A1 DNA Assay was evaluated using identical sets of assay components from a single lot. Whole blood samples $( \mathrm { n } { = } 2 0 )$ )   
anticoagulated with potassium EDTA were subjected to the Qiagen QIAamp DNA extraction procedure and tested in singlicate using Invader $\textsuperscript { \textregistered }$ UGT1A1 DNA Assay kits that were subjected to differing numbers of freeze thaw cycles (1, 3, 5, 10). Two controls and an NTC sample were also included in the test sample set. Four Subtractive Water Blank (Blank) samples were also included on each plate. For each freeze thaw cycle, all components were stored at $- 2 0 ^ { \circ } \mathrm { C }$ until frozen solid (excluding the Enzyme/Buffer mix which does not freeze due to the presence of glycerol as an ingredient) and then thawed at room temperature until liquid. The percent agreement of final calls between reagents subjected to each freeze thaw cycle (3, 5, 10) and reagents subjected to 1 cycle (baseline) was determined. The sponsor observed that the Invader $\textsuperscript { \textregistered }$ UGT1A1 Molecular Assay accurately detected each of the UGT1A1 genotypes present in the test samples. Performance was not adversely affected after being subjected to 10 freeze-thaw cycles, supporting the recommendation in the package insert of no more than five (5) cycles. Agreement between the Invader $\textsuperscript { \textregistered }$ UGT1A1 Molecular Assay and the bi-directional DNA sequence comparison genotype was $100 \%$ . The-sided $9 5 \%$ Confidence Lower Limit for the agreement of the 20 samples at each cycle point (1, 3, 5, & 10 freeze-thaw cycles) was $8 6 . 1 \%$ .

3. Clinical studies:

a. Clinical Sensitivity: Not applicable

b. Clinical specificity: Not applicable

c. Other clinical supportive data (when a. and b. are not applicable):

Sample preparation Equivalency Study: The performance of the Invader® UGT1A1 Molecular Assay was evaluated with samples extracted with both the Qiagen QIAamp® 96 DNA Blood Kit and the Qiagen QIAamp® DNA Blood Mini Kit (Qiagen Inc, Valencia, CA). Sixty (60) human whole blood samples anticoagulated with dipotassium EDTA, representing UGT1A1 $^ { * 1 }$ $( \mathrm { T A } _ { ( 6 / 6 ) } )$ $( \mathtt { n } = 1 3 )$ ), UGT1A1 $^ { * } 2 8$ heterozygous $\left( \mathrm { T A } _ { ( 6 / 7 ) } \right)$ $( \mathtt { n } = 2 4 )$ ), homozygous $( \mathrm { T A } _ { ( 7 / 7 ) } ) ( \mathrm { n } = 1 0 )$ , and other genotypes $( \mathtt { n } = 1 3 $ ). The listed genotypes were again established through bi-directional sequence analysis, and were processed using both extraction methods. The resultant genomic DNA samples from both extraction methods were tested in singlicate using the Invader® UGT1A1 Molecular Assay. In addition to the processed blood samples, two positive control samples, a No Target Control, and four Subtractive Blanks (Blank) were also included on each plate. Genomic DNA from each sample was previously quantified by the PicoGreen methodology (Molecular Probes-Invitrogen, Carlsbad, CA). The ranges of genomic DNA concentrations obtained from the two Qiagen QIAamp® extraction methods were recorded and assessed for suitability for use in the Invader® UGT1A1 Molecular Assay. Results: There was $100 \%$ agreement, with a 1-sided $9 5 \%$ confidence lower limit of $9 5 . 1 \%$ , between the Invader® UGT1A1 Molecular Assay results obtained for each sample preparation method on all 60 samples. The results demonstrate that either DNA extraction/purification methodology provides genomic DNA samples suitable for use with the Invader® UGT1A1 Molecular Assay. DNA concentrations ranged from $1 3 ~ \mathrm { n g / \mu L }$ to 78 $\mathrm { n g / u L }$ , covering most of the range of $1 0 - 7 0 ~ \mathrm { { n g / \mu L } }$ recommended in the package insert.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range: Not applicable

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this Premarket notification is complete and supports a substantial equivalence decision.