# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: k070804   
B. Purpose for Submission: New device   
C. Measurand: Genotype of Cytochrome P450 2C9 (CYP450 2C9) and Vitamin K epoxide reductase complex subunit 1 (VKORC1)   
D. Type of Test: Qualitative genetic test for single nucleotide polymorphism detection   
E. Applicant: Nanosphere, Inc.   
F. Proprietary and Established Names: $^ \mathrm { \textregistered }$ Verigene Warfarin Metabolism Nucleic Acid Test Verigene System

# G. Regulatory Information:

1. Regulation section: 21 CFR §862.3360 Drug Metabolizing Enzyme Genotyping Systems 21 CFR $\ S 8 6 4 . 7 7 5 0 $ Prothrombin time test 21 CFR $\ S 8 6 2 . 2 5 7 0$ Instrumentation for Clinical Multiplex Test Systems

2. Classification: Class II

3. Product code: ODW Cytochrome P450 2C9 (CYP450 2C9) Drug Metabolizing Enzyme Genotyping System ODV Vitamin K epoxide reductase complex subunit 1 (VKORC1) Genotyping System NSU Instrumentation for Clinical Multiplex Test Systems

4. Panel: Toxicology (91), Hematology (81), Chemistry (75)

# H. Intended Use:

1. Intended use(s): See Indications for use below.

2. Indication(s) for use: Instrument: The Verigene System is an in vitro diagnostic device intended for processing and genotyping multiple genes in a DNA sample utilizing gold nanoparticle probe technology. The Verigene System consists of the Verigene Processor and the Verigene Reader, each with its own onboard proprietary software.

Assay: The Verigene Warfarin Metabolism Nucleic Acid Test is an in vitro diagnostic for the detection and genotyping of the $^ { * 2 }$ and $^ { * 3 }$ alleles of the

CYP2C9 gene locus and a single-point mutation (C to T at position 1173) of the VKORC1 gene locus, from EDTA-anticoagulated whole blood samples, as an aid in the identification of patients at risk for increased warfarin sensitivity.

3. Special conditions for use statement(s): For prescription use only.

The information provided from this test may supplement therapeutic decisionmaking and should only be used in conjunction with routine monitoring by a physician. Clinicians should use professional judgment in the interpretation of results from this type of test.

4. Special instrument requirements: The Verigene® System

# I. Device Description:

Verigene System: an in vitro diagnostic device for processing and genotyping multiple genes in a DNA sample. The Verigene System consists of two instruments, the Verigene Processor and the Verigene Reader, and utilizes single-use, disposable Test Cartridges to process and genotype multiple genes in a DNA sample in approximately $1 \%$ hours. Up to eight Verigene Processors may be connected to a single Verigene Reader. There are four hybridization modules in each Verigene Processor, and the modules within a Verigene Processor can simultaneously run different tests.

The Verigene Reader controls the hybridization process in the modules of the Verigene Processors. It utilizes a graphical user interface to guide the user through the process of ordering tests and reporting results. Interaction with the touch screen is minimized through barcode use. This instrument also serves as the reader of the hybridization substrate using optical detection.

Verigene Processor: When a Test Cartridge is inserted, a barcode reader internal to the Verigene Processor modules reads the cartridge ID and sends it to the Verigene Reader. From this information the Verigene Reader will establish the hybridization parameters and automatically start the hybridization process.

The Test Cartridge consists of two parts: a Reagent Pack which is preloaded with the reagents necessary for running the desired test in the Verigene Processor, and a slide where the analysis takes place. All of the steps in the genotyping process take place within the Test Cartridge.

J. Substantial Equivalence Information:

Predicate device name(s): Third Wave Technologies, Inc. Invader $\textsuperscript { \textregistered }$ UGT1A1 Molecular Assay

2. Predicate 510(k) number(s): k051824

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Feature</td><td rowspan=1 colspan=1>Predicate device</td><td rowspan=1 colspan=1>Proposeddevice</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Genomic DNA obtained from ahuman whole blood sample</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Detection</td><td rowspan=1 colspan=1>Single nucleotide polymorphism</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DNAextraction</td><td rowspan=1 colspan=1>Performed off-line</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Reaction</td><td rowspan=1 colspan=1>Utilizes signal amplification</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Feature</td><td rowspan=1 colspan=1>Predicate device</td><td rowspan=1 colspan=1>Proposed device</td></tr><tr><td rowspan=1 colspan=1>Genes</td><td rowspan=1 colspan=1>UGT1A1</td><td rowspan=1 colspan=1>Cytochrome P450 2C9 andVKORC1</td></tr><tr><td rowspan=1 colspan=1>No. ofLocigenotyped</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Reactionlocation</td><td rowspan=1 colspan=1>Plastic microtiterwells</td><td rowspan=1 colspan=1>Test cartridge</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable): None referenced.

# L. Test Principle:

The Verigene Warfarin Metabolism Nucleic Acid Test genotyping process occurs with a hybridization of the target analyte to a synthetic gene-specific oligonucleotide capture strand on the Test Cartridge’s substrate. A synthetic mediator target-specific oligonucleotide is included with the test-specific sample buffer to form a hybridization “sandwich” with the gene sequence of interest, followed by washing steps to remove the unbound DNA from the hybridization chamber. A probe, composed of a gold nanoparticle with covalently bound oligonucleotides complementary to a sequence on the intermediate oligonucleotide, is then introduced, followed by a series of washing steps to remove any unbound probe. A two-part signal enhancement reagent is added to the chamber and reacts with the gold nanoparticle to amplify the signal for the Verigene Reader scanning and analysis. Upon completion of this step, the user removes the Test Cartridge. At this point, the substrate is inserted in the Verigene Reader. The Verigene Reader illuminates the signal-enhanced nanoparticles specifically bound to either the wild type of mutant captures for the gene. A photosensor reads the relative brightness of each spot and the Verigene Reader outputs a result based on the relative levels of brightness of the wild type to mutant signals.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Three sites participated in the reproducibility study. At each site, five genomic DNA samples, which covered all possible genotypes for all three alleles, were each tested in triplicate on a daily basis by the same operator for three days. Two to three different lots of disposable Test Cartridges were used at each site.

One site (Site 1) performed the reproducibility testing twice, using two different operators. Each day, each of the two operators performed the testing using the same aliquots of solutions on the same instruments.

There were 498 calls made out of 540 possible calls $( 9 2 . 2 \%$ call rate), including 165 out of 180 wild types (wt) $( 9 1 . 7 \% )$ , 234 out of 252 heterozygotes $( 9 2 . 9 \% )$ , and 99 out of 108 homozygous mutants $( 9 1 . 7 \% )$

There were no incorrect genotype calls.

Two cartridges, one at Site 1 and one at Site 2 were defective and failed their run.

Samples:   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Genotype</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>*2</td><td rowspan=1 colspan=1>*3</td><td rowspan=1 colspan=1>VKORC1</td></tr><tr><td rowspan=1 colspan=1>R1</td><td rowspan=1 colspan=1>Wt/wt</td><td rowspan=1 colspan=1>wt/*3</td><td rowspan=1 colspan=1>1173/1173</td></tr><tr><td rowspan=1 colspan=1>R2</td><td rowspan=1 colspan=1>Wt/wt</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>wt/wt</td></tr><tr><td rowspan=1 colspan=1>R3</td><td rowspan=1 colspan=1>Wt/*2</td><td rowspan=1 colspan=1>wt/*3</td><td rowspan=1 colspan=1>wt/1173</td></tr><tr><td rowspan=1 colspan=1>R4</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>wt/1173</td></tr><tr><td rowspan=1 colspan=1>R5</td><td rowspan=1 colspan=1>Wt/wt</td><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>wt/1173</td></tr></table>

Correct call rate per sample (e.g. a correct call occurs only when all three loci are genotyped accurately):

<table><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>#samplestested</td><td rowspan=1 colspan=1>#genotypingcalls made</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>#incorrectcalls</td><td rowspan=1 colspan=1>PercentAgreementor Call Rate</td></tr><tr><td rowspan=1 colspan=1>R1</td><td rowspan=1 colspan=1>2C9*2 wt/wt2C9*3 wt/*3VKORC11173/1173</td><td rowspan=1 colspan=1>36 (35*)</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>83.3%(85.7%*)</td></tr><tr><td rowspan=1 colspan=1>R2</td><td rowspan=1 colspan=1>2C9*2 wt/wt2C9*3 wt/wtVKORC1 wt/wt</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.7%</td></tr><tr><td rowspan=1 colspan=1>R3</td><td rowspan=1 colspan=1>2C9*2 wt/*22C9*3 wt/*3VKORC1 wt/1173</td><td rowspan=1 colspan=1>36 (35*)</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>94.4%(97.1%*)</td></tr><tr><td rowspan=1 colspan=1>R4</td><td rowspan=1 colspan=1>2C9*2 *2/*22C9*3 wt/wtVKORC1 wt/1173</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>94.4%</td></tr><tr><td rowspan=1 colspan=1>R5</td><td rowspan=1 colspan=1>2C9*2 wt/wt2C9*3 *3/*3VKORC1 wt/1173</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.2%</td></tr></table>

\* excludes one pre-insertion error (i.e., tests were not run) for a total of two on two cartridges in this study.

The reproducibility results per operator:

<table><tr><td rowspan=1 colspan=1>Operator ID</td><td rowspan=1 colspan=1>Locus</td><td rowspan=1 colspan=1>#Samples</td><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1>Incorrectcalls</td><td rowspan=1 colspan=1>Nocalls</td><td rowspan=1 colspan=1>%Correctcall rate</td></tr><tr><td rowspan=3 colspan=1>Site 1/0per 1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>95.5</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>95.5</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>95.5</td></tr><tr><td rowspan=3 colspan=1>Site 1/0per 2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>93.3</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>93.3</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>93.3</td></tr><tr><td rowspan=3 colspan=1>Site 2/0per 1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>88.8</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>88.8</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>88.8</td></tr><tr><td rowspan=3 colspan=1>Site 3/Oper 1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>91.1</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>91.1</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>91.1</td></tr></table>

\*includes 1 pre-insertion error

b. Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Stability testing protocols, acceptance criteria for stability testing and release criteria for cartridge lots has been reviewed and found to be acceptable.

Controls: Positive or negative DNA assay controls are not included as part of the assay. The manufacturer recommends that the user obtain quality control materials (previously characterized patient samples) to be used with the assay during installation, system validation, to verify the performance of a new lot/batch of cartridges, or when the integrity of storage conditions is in question in addition to following local, state and federal guidelines regarding quality control testing. See internal controls described below in Section O.: System Descriptions.

d. Detection limit:

A Limit of Detection (LoD) study was performed to assess the genotyping performance of the Verigene Warfarin Metabolism panel across a range of genomic DNA input concentrations. Five genomic DNA input concentrations were evaluated each across 12 Verigene Warfarin Metabolism Test Cartridges. Input concentrations were $3 0 \mathrm { n g / \mu L }$ , $4 0 \mathrm { n g / \mu L }$ , $2 0 0 ~ \mathrm { { n g / \mu L } }$ , 400 $\mathrm { n g / \mu L }$ , and $5 0 0 ~ \mathrm { { n g / \mu L } }$ .

Assay performance was evaluated with respect to the following criteria: genotyping signal intensity, genotyping ratio (genotype number), call rate, and mis-call rate.

<table><tr><td rowspan=1 colspan=1>Amt. of DNA(ng/ml)</td><td rowspan=1 colspan=1>No. ofTests</td><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1>Mis-calls</td><td rowspan=1 colspan=1>% Correctcall rate</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>92</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>92</td></tr></table>

Across the recommended DNA concentration range of $4 0 ~ \mathrm { n g / \mu L } - 4 0 0 ~ \mathrm { n g / \mu L } ,$ , the call rate ranges from $9 2 - 1 0 0 \%$ , with no incorrect genotype calls.

e. Analytical specificity:

An evaluation of potential assay interferences was performed to assess the genotyping performance of the Verigene Warfarin Metabolism Test Cartridges, when potential contaminants are present in a purified DNA sample.

Potential interfering substances that were tested included:

• 1.5 units/mL heparin; • $3 ~ \mathrm { m g / m L }$ hemoglobin • 6mM magnesium chloride • $4 0 0 \mathrm { p p m }$ lithium chloride • ${ 5 } \mathrm { x }$ concentration magnetic beads

Assay performance was evaluated with respect to the following criteria:

genotyping signal intensity, genotyping ratio (genotype number), call rate, and mis-call rate.

Samples with heparin, hemoglobin, magnesium chloride, and lithium chloride augmentation accurately detected $100 \%$ of made calls. There was no significant difference in the signal intensities for $2 C 9 ^ { \ast } 2$ and $2 C 9 ^ { \ast } 3$ wild type or mutant captures.

The samples with added hemoglobin did have significantly lower signal intensity at both VKORC1 wild type and mutant captures, however the genotyping was still accurate. The hemoglobin samples also showed lower imaging control intensities, higher background and higher virtual exposure indicating an overall loss of signal.

The samples with added heparin also showed a decrease in the nonspecific signal at the VKORC1 wild type mismatch capture, but the genotyping result was still accurate. Additionally, the heparin contaminated samples had lower negative control signal and background.

Samples augmented with additional magnesium chloride did not differ from the control. Lithium chloride supplemented samples did have a significantly higher exposure time which indicates a loss of signal.

The magnetic bead samples in which a 5X bead concentration was compared to its beadless DNA elution, showed a decrease in specific signal intensity; however, this did not impact either the call rate ( $100 \%$ for both test group and control), or genotyping capabilities. Additionally, there was no significant difference in the intensities of the imaging control, negative control, background, or exposure time. This data shows that, although high levels of magnetic beads in the DNA elution does decrease signal, it does not impact the ability of the system to correctly genotype the patient.

f. Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device: Method comparison studies were conducted at three sites, one internal and two external sites. Testing was performed using multiple lots of Test Cartridges at each site. Samples were collected and the DNA extracted by a $3 ^ { \mathrm { r d } }$ -party laboratory or by Nanosphere. Whole blood samples were extracted using common commercial DNA extraction methods. Seventy-five to eightythree samples were tested at each site. Testing was performed by laboratory personnel. There were no incorrect genotyping calls. Three Verigene cartridges were defective and failed to run due to pre-insertion errors.

Bi-directional sequencing was performed on each sample by an independent reference laboratory.

All $C Y P 2 C 9 ^ { \ast } 2$ genotype results combined   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>CYP2C9*2genotype</td></tr><tr><td rowspan=4 colspan=1>Sequenceanalysis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Hetero-zygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>NoCalls</td></tr><tr><td rowspan=1 colspan=1>Wt</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr></table>

Total call rate $= 2 { \cal I } 3 / 2 3 4 = 9 { \cal I } . \theta \%$ $( 9 5 \% C I { = } 8 6 . 6 \%$ to $94 . 4 \% ,$ ): Wild-type call rate $= 1 7 6 / 1 9 1 = 9 2 . 1 \%$ $9 5 \% C \mathrm { { C I = 8 7 . 4 \% } }$ to $9 5 . 5 \%$ ) Heterozygous call rate $= 3 5 / 4 0 = 8 7 . 5 \%$ $( 9 5 \% C \mathrm { I } { = } 7 3 . 2 \%$ to $9 5 . 8 \%$ ) Mutant call rate $= 2 / 3 = 6 6 . 7 \%$ $9 5 \% C \mathrm { I } { = } 9 . 4 \%$ to $9 9 . 2 \%$ ) Pre-insertion error $\mathrm { ( P I E ) } = 3$ No sequencing result $( \mathrm { N R } ) = 1$

# All $C Y P 2 C 9 ^ { * } 3$ genotype results combined

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>CYP2C9*3genotype</td></tr><tr><td rowspan=4 colspan=1>Sequenceanalysis</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Hetero-zygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>NoCalls</td></tr><tr><td rowspan=1 colspan=1>Wt</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr></table>

Total call rate $= 2 { \cal I } 3 / 2 3 4 = 9 { \cal I } . \theta \%$ $( 9 5 \% C I { = } 8 6 . 6 \%$ to $94 . 4 \% ,$ ): Wild-type call rate $= 1 8 2 / 1 9 9 = 9 1 . 5 \%$ $9 5 \% C \mathrm { I } { = } 8 6 . 7 \%$ to $9 4 . 9 \%$ ) Heterozygous call rate $= 3 0 / 3 4 = 8 8 . 2$ $( 9 5 \% C ] = 7 2 . 6 \%$ to $9 6 . 7 \%$ ) Mutant call rate $= 1 / 1 = 1 0 0 \%$ $9 5 \% C \mathrm { I } { = 2 . 5 \% }$ to $1 0 0 . 0 \%$ ) Pre-insertion error $\mathrm { ( P I E ) } = 3$ No sequencing result $( \mathrm { N R } ) = 1$

# All VKORC1 genotype results combined

VKORC1 1173C>T has been shown to be in strong linkage disequilibrium with another frequently assayed polymorphism, $- 1 6 3 9 \mathrm { G } \mathrm { > A }$ . Both polymorphisms were sequenced in all samples used in this study to confirm the strong linkage disequilibrium between these two polymorphisms and the results are included in the table below:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>VKORC1 1173C&gt;T [VKORC1 -1639G&gt;A] genotype</td><td rowspan=1 colspan=1>genotype</td></tr><tr><td rowspan=4 colspan=1>Sequenceanalysis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Hetero-zygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>NoCalls</td></tr><tr><td rowspan=1 colspan=1>wt</td><td rowspan=1 colspan=1>79 [79]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97 [98]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34 [34]</td><td rowspan=1 colspan=1>1</td></tr></table>

Total call rate $= 2 I 0 / 2 3 I = 9 0 . 9 \ \%$ $( 9 5 \% C I = 8 6 . 4 \% \ t o \ 9 4 . 3 \% ) \ [ = 2 I I / 2 3 2 = 9 0 . 9 \%$ $( 9 5 \% C I { = } 8 6 . 5 \%$ to $9 4 . 3 \%$ ]: Wild-type call rate $= 7 9 / 8 7 = 9 0 . 8 \%$ ( $9 5 \% C \mathrm { I } { = } 8 2 . 7 \%$ to $9 5 . 9 \%$ ) Heterozygous call rate $= 9 7 / 1 0 9 = 8 9 . 0 \%$ $9 5 \% C \mathrm { I } { = } 8 1 . 6 \%$ to $9 4 . 2 \%$ ) Mutant call rate $= 3 4 / 3 5 = 9 7 . 1 \%$ $( 9 5 \% C \mathrm { I } { = } 8 5 . 0 \%$ to $9 9 . 9 \%$ ) Pre-insertion error $\mathrm { ( P I E ) } = 3$ No sequencing result $( \mathrm { N R } ) = 4$ [4]

# Verigene System VKORC1 1173C>T correlation with VKORC1 -1639G>A $=$ 21 calls out of 211 calls

No sequencing result (NR) for 1 VKORC1 -1639G>A sequences   
Pre-insertion error $\mathrm { ( P I E ) } = 3$   
No calls $= 2 1$

# Overall results; all genotypes combined

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>All genotypescombined</td></tr><tr><td rowspan=4 colspan=1>Sequenceanalysis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Hetero-zygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>NoCalls</td></tr><tr><td rowspan=1 colspan=1>wt</td><td rowspan=1 colspan=1>437</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>2</td></tr></table>

Overall call rate $= 6 3 6 / 6 9 9 = 9 1 . 0 \ \%$ $( 9 5 \% C I { = } 8 8 . 6 \%$ to $93 . 0 \%$ ): Wild-type call rate $= 4 3 7 / 4 7 7 = 9 1 . 6 \%$ $9 5 \% C \mathrm { { C I } = 8 8 . 8 \% }$ to $9 3 . 9 \%$ ) Heterozygous call rate $= 1 6 2 / 1 8 3 = 8 8 . 5 \%$ $( 9 5 \% C \mathrm { I } { = } 8 3 . 0 \%$ to $9 2 . 8 \%$ ) Mutant call rate $= 3 7 / 3 9 = 9 4 . 9 \%$ $( 9 5 \% C \mathrm { I } { = } 8 2 . 7 \%$ to $9 9 . 4 \%$ ) Pre-insertion error $\mathrm { ( P I E ) } = 9$ No sequencing result $( \mathrm { N R } ) = 6$

From an overall panel view, the total panel read rate was $9 1 . 1 \%$ $( = 2 1 4 / 2 3 5 )$ . For $C Y P 2 C 9 ^ { * } 2$ , the overall call rate was $9 1 . 0 \%$ . For $C Y P 2 C 9 ^ { * } 3$ , the overall call rate was $9 1 . 0 \%$ . For VKORC1, the overall call rate was $9 0 . 9 \%$ . The agreement between VKORC1 $1 1 7 3 \mathrm { C } \mathrm { > } \mathrm { T }$ and VKORC1 -1639G>A was $100 \%$ , revealing complete linkage disequilibrium in this small sample set.

# Extraction study:

23 whole blood samples were sent to three sites: 1 internal and 2 external.

Samples were extracted using 3 commonly used DNA extraction methods and tested using the Verigene kit. The results are below:

2C9\*2   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Samplestested</td><td rowspan=1 colspan=1>Run</td><td rowspan=1 colspan=1>#Genotypingcalls made</td><td rowspan=1 colspan=1>#Correctcalls</td><td rowspan=1 colspan=1>#Incorrectcalls</td><td rowspan=1 colspan=1>Correctcall rate(%)</td></tr><tr><td rowspan=2 colspan=1>Site 1</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.3%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>Site 2</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.3%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>Site 3</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.6%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

2C9\*3   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Samplestested</td><td rowspan=1 colspan=1>Run</td><td rowspan=1 colspan=1>#Genotypingcalls made</td><td rowspan=1 colspan=1>#Correctcalls</td><td rowspan=1 colspan=1>#Incorrectcalls</td><td rowspan=1 colspan=1>Correctcall rate(%)</td></tr><tr><td rowspan=2 colspan=1>Site 1</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.3%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>Site 2</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.3%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>Site 3</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.6%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

# VKORC1 1173

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Samplestested</td><td rowspan=1 colspan=1>Run</td><td rowspan=1 colspan=1>#Genotypingcalls made</td><td rowspan=1 colspan=1>#Correctcalls</td><td rowspan=1 colspan=1>#Incorrectcalls</td><td rowspan=1 colspan=1>Correctcall rate(%)</td></tr><tr><td rowspan=2 colspan=1>Site 1</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.3%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>Site 2</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91.3%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=2 colspan=1>Site 3</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.6%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.   
b. Clinical specificity: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

Table 1. Allele frequency across various ethnic groups   

<table><tr><td rowspan=1 colspan=1>Ethnic</td><td rowspan=1 colspan=1>CYP2C9*2</td><td rowspan=1 colspan=1>CYP2C9*3</td><td rowspan=1 colspan=2>VKORC1 1173</td></tr><tr><td rowspan=1 colspan=1>Caucasian</td><td rowspan=1 colspan=1>0.9 - 20%{2</td><td rowspan=1 colspan=1>0 - 14.5%2</td><td rowspan=1 colspan=1>37%1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>African</td><td rowspan=1 colspan=1>0 - 8.7%2</td><td rowspan=1 colspan=1>0 - 4.3%2</td><td rowspan=1 colspan=2>14%</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>0%2</td><td rowspan=1 colspan=1>0 - 8.2%2</td><td rowspan=1 colspan=2>89%</td></tr></table>

1 Rieder MJ, Reiner AP, Gage BF, Nickerson DA, Eby CS, McLeod HL, Blough DK, Thummel KE, Veenstra DL, Rettie AE. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. NEJM 2005; 352: 2285-2293.   
2 Lee CR, Goldstein JA, Pieper JA. Cytochrome P450 2C9 polymorphisms: a comprehensive review of the in-vitro and human data. Pharmacogenetics 2002; 12: 251-263.

# N. Instrument Name: Verigene System

# O. System Descriptions:

1. Modes of Operation:

The instrument has a single mode of operation with four hybridization modules that can run 4 different reactions at a time.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development   
processes for this line of product types:   
Yes ___X_ _ or No

The sponsor’s submitted software documentation demonstrated that the software design meets the stated requirements for this device and were verified and validated in part by testing the system with known wild type, mutant, and negative control samples determining that signal detection configurations results in correct calls/results.

3. Specimen Identification:

Sample identification is performed through entering in the sample ID and the barcode of the test cartridge to be used with the sample. When the sample is to be run, the barcode of the test cartridge is scanned.

4. Specimen Sampling and Handling:

Whole blood samples collected from a patient undergo an extraction step. Following the extraction of DNA the sample is then used for analysis by the device.

5. Calibration: The Verigene System does not require user calibration and the user does not have the capability to re-calibrate the instrument. The temperature control system on the Verigene Processor device is the only component that requires calibration. This is performed at the time of manufacture or by a Nanosphere service technician. All control parameters are verified by the system’s 32-bit cyclical redundancy check which is established at the time of software release. If the protected configuration is modified, the application will be prevented from running.

6. Quality Control: For single nucleotide polymorphism (SNP) genotyping, it can be expected that either the mutant or the wild type sequence is present at the loci tested. Therefore, a system and panel positive control check are built into each hybridization chamber.

A negative control sequence is included in each well to detect if the effective hybridization $\mathrm { T } _ { \mathrm { m } }$ (melt temperature) of the hybridization chamber is too low. This negative control is intended to detect if the hybridization temperature is incorrect due to an instrument malfunction or if the hybridization buffer was incorrectly added due to a user or instrument failure. The Verigene Reader automatically checks the wild type or mutant target oligonucleotide signal relative to the negative control oligonucleotide signal. If the target signal is not adequately higher than the negative control signal, then a “No Call” results.

The threshold limits for the negative control and background checks were determined empirically with nominal and failure condition testing. The operator is informed if the “No Call” result is due to the background or the negative control signal being too high.

A set of oligonucleotide spots on the substrate are called imaging controls. The imaging controls act as a quality check for proper fluid control and movement between the Test Cartridge and the instrument. If the imaging controls are absent, then a “No Call” is output and the user is informed that the reason is due to an absence of imaging controls.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.