# Nanosphere

510(k) Summary

<table><tr><td colspan="2">This summary of the 510(k) safety and effectiveness information is being submitted in accordance with the requirements of SMDA 1900 and CFR 807.92.</td></tr><tr><td>510(k) number</td><td>k070804</td></tr><tr><td>Summary preparation date</td><td>September 11, 2007</td></tr><tr><td>Submitted by</td><td>Nanosphere, Inc. 4088 Commercial Avenue Northbrook, IL 60062 Phone: 847-400-9000 Fax: 847-400-9199</td></tr><tr><td>Contact</td><td>Sue Kent — Manager, Clinical &amp; Regulatory Affairs</td></tr><tr><td>Proprietary names and classifications</td><td>For the assay: Verigene® Warfarin Metabolism Nucleic Acid Test Regulations: 21 CFR §862.3360 - Drug Metabolizing Enzyme Genotyping System</td></tr><tr><td></td><td>21 CFR §864.7750 - Prothrombin Time Test Panels: 91 Toxicology &amp; 81 Hematology Classification:I</td></tr><tr><td></td><td>Product codes: ODW - Cytochrome P450 2C9 (CYP450 2C9) Drug Metabolizing Enzyme Genotyping System</td></tr><tr><td>For the instrument:</td><td>ODV - Vitamin K epoxide reductase complex subunit 1 (VKORC1) Genotyping System</td></tr><tr><td></td><td>Verigene® System</td></tr><tr><td></td><td>Regulation: 21 CFR §862.2570 - Instrumentation for Clinical Multiplex Test Systems Panel: 75 Clinical Chemistry Classification: II</td></tr><tr><td></td><td>Product code: NSU - Instrumentation for Clinical Multiplex Test Systems</td></tr><tr><td>Common names</td><td>For the assays:</td></tr><tr><td></td><td>warfarin metabolism CYP2C9*2 warfarin panel CYP2C9*3</td></tr><tr><td></td><td>warfarin VKORC1 For the instrument:</td></tr><tr><td></td><td>Bench-top molecular diagnostics workstation</td></tr><tr><td></td><td></td></tr><tr><td>Intended uses</td><td>The Verigene® Warfarin Metabolism Nucleic Acid Test is an in vitro diagnostic for the detection and genotyping of the *2 and *3 alleles of the CYP2C9 gene and a single-point</td></tr><tr><td></td><td></td></tr><tr><td></td><td></td></tr><tr><td></td><td>polymorphism (C to T at position 1173) of the VKORC1 gene, from EDTA-anticoagulated</td></tr><tr><td></td><td></td></tr><tr><td></td><td>whole blood samples, as an aid in the identification of patients at risk for increased warfarin sensitivity. The test is intended to be used on the Verigene System.</td></tr><tr><td>genotyping multiple genes in a DNA sample utilizing gold nanoparticle probe</td><td>The Verigene® System is an in vitro diagnostic device intended for processing and</td></tr></table>

# Device descriptions

The Verigene System is an in vitro diagnostic device for processing and genotyping multiple genes in a DNA sample. The Verigene System consists of two instruments, the Verigene Processor and the Verigene Reader, and utilizes single-use, disposable Test Cartridges to process and genotype multiple genes in a DNA sample in approximately $1 \%$ hours.

The analysis sequence is the same for each of the three tests (i.e., $C Y P 2 C 9 ^ { \ast } 2$ and $^ { \star } 3$ and VKORC1). After extracted and purified DNA, mixed with hybridization buffer, is loaded into the sample well of the Test Cartridge, it is ready for processing and is inserted into the Verigene Processor. An internal barcode reader reads the cartridge ID and sends the information to the Verigene Reader. From this information, the Verigene Reader establishes the hybridization parameters and starts the hybridization process.

The genotyping process occurs with a hybridization of the target analyte to a synthetic genespecific oligonucleotide capture strand on the Test Cartridge's substrate. A synthetic mediator target-specific oligonucleotide is included with the test-specific sample buffer to form a hybridization "sandwich" with the gene sequence of interest. Washing steps following the target hybridization remove the unbound DNA from the hybridization chamber. A probe, composed of a gold nanoparticle with covalently bound oligonucleotides complementary to a sequence on the intermediate oligonucleotide, is introduced after the target wash. After the probe hybridization is completed, a series of washing steps remove the unbound probe from the hybridization chamber. A two-part signal enhancement reagent is added to the hybridization chamber and reacts with the gold nanoparticle to amplify the signal for the Verigene Reader scanning and analysis.

Upon completion of the genotyping process, the user removes the Test Cartridge from the Verigene Processor which is now ready for the next test.

Once the reagent portion of the Test Cartridge is removed by the user, the substrate is inserted into the Verigene Reader. The Verigene Reader illuminates the signal-enhanced nanoparticles specifically bound to either the wild type or mutant captures for the gene. A photosensor reads the relative brightness of each spot and the Verigene Reader outputs a result based on relative levels of brightness of the wild type to mutant signals.

Comparison to technological features of the predicate device   

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>Third Wave Technologies, Inc.,Invader® UGT1A1 MolecularAssay</td><td rowspan=1 colspan=1>Nanosphere, Inc., VerigeneWarfarin Metabolism NucleicAcid Test</td></tr><tr><td rowspan=1 colspan=1>DNA sequencedetection</td><td rowspan=1 colspan=1>Detects specific DNA sequencesthrough direct recognition of DNAtargets</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Reactionconditions</td><td rowspan=1 colspan=1>) No thermal cycling -isothermic reaction2) Utilizes signal amplification3) Reactions occur in multipleplastic microtiter wells</td><td rowspan=1 colspan=1>) Same as predicate2) Same as predicate3) Reactions occur on a singleglass slide using microfluidics,with up to 32 cartridges (slides)hybridized and enhancedsimultaneously</td></tr><tr><td rowspan=1 colspan=1>Assay results</td><td rowspan=1 colspan=1>Assay signal results areinterpreted by a software programand are assigned a genotype thatis presented to the end-user in areport format.</td><td rowspan=1 colspan=1>Same as predicate</td></tr></table>

The following is a comparison of the Nanosphere, Inc., Verigene System to the Third Wave Technologies, Inc., Invader® UGT1A1 Molecular Assay:

# Performance characteristics

# Reproducibility

In an initial reproducibility study at each of three sites, five genomic DNA samples, covering all possible genotypes for all three alleles, were each tested in triplicate on a daily basis by the same operator for three days. One site performed the same reproducibility testing twice each day, using two different operators. The table below shows the number of samples tested, correct calls, incorrect calls, no calls, and percent agreement (or call rate) by locus for each of the three sites.

Initial reproducibility results   
\*includes 1 pre-insertion error   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Locus</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls(Mis-Calls)</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Call Rate(% agreement)</td></tr><tr><td rowspan=3 colspan=1>Site 1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>94%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>94%</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>94%</td></tr><tr><td rowspan=3 colspan=1>Site 2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>89%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>89%</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5*</td><td rowspan=1 colspan=1>89%</td></tr><tr><td rowspan=3 colspan=1>Site 3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>91%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>91%</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>91%</td></tr></table>

In a second reproducibility study (see table below) that evaluated three common DNA extraction methods, aliquots of a panel of 23 blood specimens were utilized. At each of the 3 sites, 1 operator extracted the DNA from each of the 23 aliquots of blood and ran the assay. Each site used a different DNA extraction procedure/kit. 3 lots of cartridges were tested. 1 re-test run was performed if there was a "no call" or "mis-cal" on the first run but no mis-calls were observed. The genotypes of the DNA samples were confirmed by bi

ectional sequencing. The genotypes of the 23 samples - ${ } ^ { 2 0 9 } { } ^ { 2 }$ 18 wild type, 4 heterozygous, 1 mutant - $^ { 2 0 9 } 3$ 20 wild type, 3 heterozygous, 0 mutant - VKORC1: 9 wild type, 11 heterozygous, 3 mutant

Extraction method reproducibility results   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>Run</td><td rowspan=1 colspan=1>GenotypingCalls Made</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls(Mis-Calls)</td><td rowspan=1 colspan=1>Call Rate(% agreement)</td></tr><tr><td rowspan=2 colspan=1>Site 1</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Site 2</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>91%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Site 3</td><td rowspan=2 colspan=1>23</td><td rowspan=1 colspan=1>Afterrun 1</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96%</td></tr><tr><td rowspan=1 colspan=1>Afterrun 2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100 %</td></tr></table>

# Accuracy (percent agreement compared to bi-directional DNA sequencing) -

A total of 238 samples were analyzed using the Verigene Warfarin Metabolism Nucleic Acid Test at three sites and by bi-directional sequencing analysis at an independent reference laboratory (see results in the tables below). All purified DNA samples were from whole blood collected using EDTA as the anticoagulant. These data are based on the original run only (i.e., no re-testing was performed). Three Verigene cartridges were defective and failed to run due to pre-insertion errors.

CYP2C9\*2 method comparison results   

<table><tr><td rowspan=4 colspan=1>Seenceeansjs</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Heterozygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>IncorrectCall Rate(Mis-Calls)</td><td rowspan=1 colspan=1>Correct Call Rate[% agreement](No Calls)</td></tr><tr><td rowspan=1 colspan=1>wt</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%(0)</td><td rowspan=1 colspan=1>92%(15)</td></tr><tr><td rowspan=1 colspan=1>het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0% (0)</td><td rowspan=1 colspan=1>87% 5)</td></tr><tr><td rowspan=1 colspan=1>mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0% (0)</td><td rowspan=1 colspan=1>67%1</td></tr></table>

$C Y P 2 C 9 ^ { \ast } 3$ method comparison results   

<table><tr><td rowspan=4 colspan=1>Seenceaan</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wild-type(wt)</td><td rowspan=1 colspan=1>Heterozygous(het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>IncorrectCatl Rate(Mis-Calis)</td><td rowspan=1 colspan=1>Correct Call Rate[% agreement](No Calls)</td></tr><tr><td rowspan=1 colspan=1>wt</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0% (0)</td><td rowspan=1 colspan=1>91%(17)</td></tr><tr><td rowspan=1 colspan=1>het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%(0)</td><td rowspan=1 colspan=1>88%4)</td></tr><tr><td rowspan=1 colspan=1>mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0%(0)</td><td rowspan=1 colspan=1>100% (0)</td></tr></table>

VKORC1 1173 method comparison results   

<table><tr><td rowspan=4 colspan=1>Seunbe aakes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Wid-type(wt)</td><td rowspan=1 colspan=1>Heterozygous{het)</td><td rowspan=1 colspan=1>Mutant(mut)</td><td rowspan=1 colspan=1>IncorrectCall Rate(Mis-Calls)</td><td rowspan=1 colspan=1>Correct Call Rate[% agreement](No Calls)</td></tr><tr><td rowspan=1 colspan=1>wt</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%0)</td><td rowspan=1 colspan=1>91%(8)</td></tr><tr><td rowspan=1 colspan=1>het</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%(0</td><td rowspan=1 colspan=1>89% (12)</td></tr><tr><td rowspan=1 colspan=1>mut</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0% (0)</td><td rowspan=1 colspan=1>97%(1)</td></tr></table>

The Verigene Warfarin Metabolism Nucleic Acid Test results will be reported only if calls are made for all three targets (i.e., the panel must be complete, based on calls made for $C \forall P 2 C 9 ^ { \star } 2 ,$ $C Y P 2 C 9 ^ { \star } 3$ and VKORC1 calls). If only one or two allele calls are made, the entire test's results are not reported on the screen or printout.

Therefore, from an overall panel view, the total panel read rate was $9 1 . 1 \%$ (=214/235).

# Limit of Detection (analytical sensitivity)-

The table below shows the results of a DNA concentration study. When DNA concentrations outside of the range of $4 0 \ \cap { \mathfrak { g } } / \mu \mathsf { L } - 4 0 0 \ \mathsf { n g } / \mu \mathsf { L }$ are studied, no mis-calls are made but the call rate decreases. At $4 0 \approxg / \mu L$ , the call rate is $9 2 \%$ .

Limit of detection study results   

<table><tr><td rowspan=1 colspan=1>DNAConcentration</td><td rowspan=1 colspan=1>Number ofCartridges</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls(Mis-Calls)</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Call Rate(% agreement)</td></tr><tr><td rowspan=1 colspan=1>30 ng/μL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>40 ng/μL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92%</td></tr><tr><td rowspan=1 colspan=1>200 ng/μL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>400 ng/μL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>500 ng/μL</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92%</td></tr></table>

Other characteristics of the Verigene Warfarin Metabolism Nucleic Acid Test   

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>CYP2C9*2</td><td rowspan=1 colspan=1>CYP2C9*3</td><td rowspan=1 colspan=1>VKORC1</td></tr><tr><td rowspan=1 colspan=1>Interferences</td><td rowspan=1 colspan=3>Performance not affected bymagnetic beadsheparinhemoglobinmagnesium chloridelithium chlorideother possible interferences are not known.</td></tr><tr><td rowspan=1 colspan=1>Reagent stability</td><td rowspan=1 colspan=3>The Test Cartridges are to be stored from 2°C to 8°C until theexpiration date printed on the label.The Sample Buffer is to be stored from 2°C to 8°C until theexpiration date printed on the label.Neither the Test Cartridges nor the Sample Buffer should be frozen.</td></tr><tr><td rowspan=1 colspan=1>Precautions andwarnings</td><td rowspan=1 colspan=2>In the CYP2C9 gene, additional raremutations other than R149C (CYP2C9*2)and I359L (CYP2C9*3) have beenobserved. These rare CYP2C9 alleles arenot detected by this test. NOTE: Theprevalence of these additional allelesis low and there is insufficient information inthe scientific literature to predict the impactthis polymorphism will have on anindividual&#x27;s sensitivity to warfarin.</td><td rowspan=1 colspan=1>In the VKORC1gene, additional rarepolymorphisms otherthan 1173C&gt;T havebeen observed. Theserare VKORC1 allelesare not detected by thistest.</td></tr></table>

# Conclusion

The above pre-clinical and clinical test results support the safety and effectiveness of the devices - the Verigene System and Verigene Warfarin Metabolism Nucleic Acid Test.

# SEP 1 7 2007

Re: k070804 Trade/Device Name: Verigene Warfarin Metabolism Nucleic Acid Test, Verigene System Regulation Number: 21 CFR 862.3360 Regulation Name: Drug metabolizing enzyme genotyping system Regulatory Class: Class II Product Code: ODW, ODV, NSU Dated: August 7, 2007 Received: August 8, 2007

Dear Ms. Kent:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into either class II (Special Controls) or class ⅢI (PMA), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820).

This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific information about the application of labeling requirements to your device, or questions on the promotion and advertising of your device, please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (240) 276-0490. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (240) 276-3150 or at its Internet address at   
http://www.fda.gov/cdrh/industry/support/index.html. Sincerely yours,   
Jean M.Coopes, M.S. D.V.M. Jean M. Cooper, M.S., D.V.M.   
Director   
Division of Chemistry and Toxicology   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and   
Radiological Health

# Indications for use

510(k) Number (if known): K070804

Device Name: Verigene® System

Indications for Use: The Verigene System is an in vitro diagnostic device intended for processing and genotyping multiple genes in a DNA sample utilizing gold nanoparticle probe technology. The Verigene System consists of the Verigene Processor and the Verigene Reader, each with its own onboard proprietary software.

Device Name: Verigene Warfarin Metabolism Nucleic Acid Test

Indications for Use: The Verigene Warfarin Metabolism Nucleic Acid Test is an in vitro diagnostic for the detection and genotyping of the $^ { \star 2 }$ and $^ { \star } 3$ alleles of the CYP2C9 gene and a single-point polymorphism (C to T at position 1173) of the VKORC1 gene, from EDTAanticoagulated whole blood samples, as an aid in the identification of patients at risk for increased warfarin sensitivity. The test is intended to be used on the Verigene System.

Concurrence of CDRH, Office of In Vitro Diagnostic Devices (OIVD)

Cooll Benar ffice of In Vitro Diagnostic Device Ivaluation and Safety

![](images/9a8bbdde429a8a64c8aa96125e87e8a7cda4b89eb3d3cc60258fdc191ccb8952.jpg)