# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: k073720   
B. Purpose for Submission: New device   
C. Measurand: Genotype of Cytochrome P450 2C9 (CYP450 2C9) and Vitamin K epoxide reductase complex subunit I (VKORC1)   
D. Type of Test: Qualitative genetic test for single nucleotide polymorphism detection   
E. Applicant: Osmetech Molecular Diagnostics   
F. Proprietary and Established Names: eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test eSensor $\textsuperscript { \textregistered }$ XT-8 System

# G. Regulatory Information:

1. Regulation section: 21CFR §862.3360 – Drug Metabolism Enzyme Genotyping Test 21CFR §864.7750 – Prothrombin Time Test 21CFR §862.2570 – Instrument for Clinical Multiplex Test Systems

2. Classification: Class II

3. Product code: ODW Cytochrome P450 2C9 (CYP450 2C9) Drug Metabolizing Enzyme Genotyping System ODV Vitamin K epoxide reductase complex subunit 1 (VKORC1) Genotyping System NSU Instrumentation for Clinical Multiplex Test Systems

4. Panel: Toxicology (91), Hematology (81), Chemistry (75)

# H. Intended Use:

1. Intended use(s): See Indications for use below.

2. Indication(s) for use:

Assay: The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test is an in vitro diagnostic for the detection and genotyping of the $^ { * 2 }$ and $^ * 3$ alleles of the cytochrome P450 (CYP450) 2C9 gene locus and the Vitamin K epoxide reductase C1 (VKORC1) gene promoter polymorphism $\left( - 1 6 3 9 \mathrm { G } > \mathrm { A } \right)$ from genomic DNA extracted from whole blood samples preserved with EDTA, as an aid in the identification of patients at risk for increased warfarin sensitivity.

Instrument: The eSensor $^ \mathrm { \textregistered }$ XT-8 Instrument is an in vitro diagnostic device intended for genotyping multiple mutations or polymorphisms in an amplified DNA sample utilizing electrochemical detection technology.

3. Special conditions for use statement(s): For Prescription use only.

4. Special instrument requirements:

The eSensor $\textsuperscript { \textregistered }$ XT-8 Instrument

# I. Device Description:

The eSensor $\textsuperscript { \textregistered }$ XT-8 System is an in vitro diagnostic device for performing hybridization and genotyping of multiple mutations and/or polymorphisms in an amplified DNA sample. The XT-8 Instrument is configured with one to three processing towers which perform up to 8 simultaneous tests per tower. The XT-8 System uses a single-use, disposable test cartridge to perform hybridization and genotyping in approximately 40 minutes per sample. The cartridge contains an EEPROM chip which transmits the cartridge lot number, expiration date and protocol identity to the instrument.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Verigene $\textsuperscript { \textregistered }$ Warfarin Metabolism Nucleic Acid Test

2. Predicate 510(k) number(s): k070804

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Predicate (k07084)</td><td colspan="1" rowspan="1">Device (k073720)</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Verigene® WarfarinMetabolism Nucleic AcidTest and Verigene®</td><td colspan="1" rowspan="1">eSensor® WarfarinSensitivity Test and XT-8System</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">System</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Test type</td><td colspan="1" rowspan="1">Qualitative genetic test forsingle nucleotidepolymorphism detection</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Genomic DNA obtainedfrom a human whole bloodsample</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Gene/Mutations genotyped</td><td colspan="1" rowspan="1">CYP2C9*2 (430C&gt;T),CYP2C9*3 (1075A&gt;C)</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">Genotyping principle</td><td colspan="1" rowspan="1">Sandwich hybridization test</td><td colspan="1" rowspan="1">Same as predicate</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Predicate</td><td rowspan=1 colspan=1>Device</td></tr><tr><td rowspan=1 colspan=1>Gene/Mutations genotyped</td><td rowspan=1 colspan=1>VKORC1(1173C&gt;T)</td><td rowspan=1 colspan=1>VKORC1(-1639G&gt;A)</td></tr></table>

# K. Standard/Guidance Document referenced (if applicable):

# CLSI EP7A-2

# L. Test Principle:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test uses a solid-phase electrochemical method for determining the genotype of patient blood specimens for the polymorphisms.

Three process blocks are required to generate a genotyping result from a patient sample. The first is the generation of single-stranded amplified targets from a genomic DNA sample, the second is the genotyping reaction that determines the genotype of the sample, and the third is the acquisition and analysis of data from the genotyping reaction, and the report generation for that sample. The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test provides all reagents needed for PCR, exonuclease treatment, and genotyping.

The analysis process for each sample consists of three steps: 1) Genomic DNA isolated from whole blood obtained using EDTA as anti-coagulant is combined with PCR Mix and Taq polymerase enzyme and is subjected to amplification of target sequences by PCR using a thermal cycler. 2) Amplified DNA is treated with exonuclease enzyme to generate singlestranded target DNA. 3) Single-stranded, amplified target DNA is mixed with hybridization and genotyping reagents, transferred to an eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test cartridge, and the cartridge is inserted in the eSensor $\textsuperscript { \textregistered }$ XT-8 Instrument. The instrument controls the circulation of the sample through the cartridge containing to allow hybridization at a controlled temperature, and then detects and genotypes the sample by voltammetry.

Genotyping of the test panel polymorphisms is achieved by a sandwich assay principle: 1) each pair of electrodes contains a different synthetic oligonucleotide capture probe which is complementary to one of the target DNA fragments. 2) The hybridization reagents contain pairs of ferrocene-labeled synthetic oligonucleotide signal probes; one member of each pair is complementary to the major allele sequence of the target polymorphism, while the second member of the pair is complementary to the minor allele sequence. Each member of the probe pair has a ferrocene labels with a different oxidation potential for each allele. 3) Single-stranded, amplified target DNA hybridizes to its specific capture probe, and in turn hybridizes to the allele-specific, ferrocene-labeled signal probe. 4) Each electrode of the array is analyzed by voltammetry; the target polymorphism is determined by the location of the electrode containing the capture probe, and the genotype is identified by the ratio of signals from the allele-specific ferrocene labels. The array also includes positive and negative controls to confirm the hybridization reaction and detect non-specific signals.

Upon completion of the test, the EEPROM chip on the cartridge contains information that prevents its re-use with a new sample. The instrument analyzes the results and provides a report of the test results. The operator removes the used cartridge from the XT-8 Instrument, and that slot is ready to accept a new test.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A reproducibility study was performed at three sites, two external and one internal. Five genomic DNA samples covering all possible genotypes for all three alleles in the Warfarin Sensitivity Test (see Table below) were tested in duplicate runs on a daily basis by the same operator per site for 5 days at 3 different sites (2 external sites and 1 internal site). The internal site performed the same reproducibility testing twice each day, using two different operators and the same testing materials. Three kit lots were randomized throughout the study. Overall, a combined 263 tests were performed using samples that represented all genotypes of each polymorphism. There were 9 first pass no-calls, for an overall first pass no-call rate of $4 . 5 \%$ . Following an additional round of testing of these no-call samples, the results obtained were in $100 \%$ agreement with DNA sequencing. There were no incorrect genotype calls in this study.

The genotypes of the samples tested in the reproducibility studies are as follows:

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=2>Genotype</td></tr><tr><td rowspan=1 colspan=1>2C9</td><td rowspan=1 colspan=1>VKORC1</td></tr><tr><td rowspan=1 colspan=1>gDNAsample 1</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>gDNAsample 2</td><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>G/A</td></tr><tr><td rowspan=1 colspan=1>gDNAsample 3</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>gDNAsample 4</td><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>gDNAsample 5</td><td rowspan=1 colspan=1>wt/*3</td><td rowspan=1 colspan=1>A/A</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>Allele</td><td rowspan=1 colspan=1>Totaltests</td><td rowspan=1 colspan=1>First-passcorrectcalls</td><td rowspan=1 colspan=1>First-pass no-calls</td><td rowspan=1 colspan=1>Finalcorrectcalls</td><td rowspan=1 colspan=1>Finalincorrectcalls</td><td rowspan=1 colspan=1>% Correct CallRate(95% LCB)</td></tr><tr><td rowspan=6 colspan=1>1</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>All</td><td rowspan=3 colspan=1>All</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (98.5%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (98.5%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (98.5%)</td></tr></table>

Summary of Reproducibility Results sorted by sample and genotype.   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>TotalTests</td><td rowspan=1 colspan=1>First-passcorrectcalls</td><td rowspan=1 colspan=1>First-pass no-calls</td><td rowspan=1 colspan=1>Finalcorrectcalls</td><td rowspan=1 colspan=1>Finalincorrectcalls</td><td rowspan=1 colspan=1>% Correct CallRate(95% LCB)</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>2C9 wt/wt VKORC1G/G</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>2C9 *2/*3VKORC1G/A</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>2C9 *2/*2VKORC1G/G</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>2C9 *3/*3VKORC1G/G</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>2C9 wt/*3VKORC1A/A</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr></table>

Overall reproducibility for all mutations was $100 \%$ after additional testing of first-pass no-calls. The first pass no-call rate was $4 . 5 \%$ , and the miscall rate was $0 \%$ .

An extraction method study was carried out in order to demonstrate that personnel at different laboratories can isolate genomic DNA starting from whole blood samples using standard DNA purification kits and utilize that DNA in the eSensor $^ \mathrm { \textregistered }$ Warfarin Sensitivity Test to generate correct genotype calls. Three generic DNA extraction methods were evaluated at three different sites using aliquots of a panel of 7 whole blood samples of different genotypes. Each site used a different extraction method, representing examples of magnetic bead, silica membrane, and precipitation methodologies. Each site performed three independent extractions of each blood sample and assayed them using a single eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test kit from the same kit lot.

Genotypes of samples tested in the Extraction Reproducibility Study:

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=2>Genotype</td></tr><tr><td rowspan=1 colspan=1>2C9</td><td rowspan=1 colspan=1>VKORC1</td></tr><tr><td rowspan=1 colspan=1>Bloodsample 1</td><td rowspan=1 colspan=1>wt/*2</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>Bloodsample 2</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>Bloodsample 3</td><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>Bloodsample 4</td><td rowspan=1 colspan=1>wt/*3</td><td rowspan=1 colspan=1>G/G</td></tr><tr><td rowspan=1 colspan=1>Bloodsample 5</td><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>A/A</td></tr><tr><td rowspan=1 colspan=1>BloodSample 6</td><td rowspan=1 colspan=1>wt/*3</td><td rowspan=1 colspan=1>G/A</td></tr><tr><td rowspan=1 colspan=1>BloodSample 7</td><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>A/A</td></tr></table>

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test genotyping data were evaluated after first-pass results. Table 5 summarizes the percent agreement between results obtained at each of the sites and DNA sequencing. All first-pass results agreed with DNA Sequencing.

Summary of Inter-laboratory Extraction Reproducibility Results   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Allele</td><td rowspan=1 colspan=1>#TotalTests</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls</td><td rowspan=1 colspan=1>No Calls</td><td rowspan=1 colspan=1>% Correct Call rate(95% LCB)</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%6 (86.7%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%6 (86.7%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1># TotalTests</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls</td><td rowspan=1 colspan=1>No Calls</td><td rowspan=1 colspan=1>%CorrectCall Rate</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>2C9 wt/wt VKORC1 G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>2C9 wt/*2 VKORC1G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>2C9wt/*3VKORC1G/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>2C9wt/*3VKORC1G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>2C9 *3/*3VKORC1G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>06</td><td rowspan=1 colspan=1>2C9 *2/*3 VKORC1 A/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>07</td><td rowspan=1 colspan=1>2C9 *2/*3 VKORC1 A/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Overall reproducibility for all mutations was $100 \%$ , regardless of extraction method. The first pass correct call rate was $100 \%$ and the no call rate and miscall rate were $0 \%$ .

$b$ . Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Stability: Test kit cartridges are stable at $1 0 { - } 2 5 ^ { \circ } \mathrm { C }$ for up to 3 months. Reagents are stable at $- 2 0 ^ { \circ } \mathrm { C }$ for up to 3 months. As results are obtained from ongoing real-time stability testing, the product expiry dating will be extended based on these additional data. Cartridges can be stored for up to 14 days after opening the foil pouches. Once open, reagents can be stored at $- 2 0 ^ { \circ } \mathrm { C }$ for up to 30 days. Reagents can be thawed up to 3 times. Whole blood stored in EDTA can be stored at $4 \mathrm { { } ^ { \circ } C }$ for up to 4 weeks after collection prior to extraction of gDNA for use in the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test. PCR products can be stored at $4 \mathrm { { } ^ { \circ } C }$ or $- 2 0 ^ { \circ } \mathrm { C }$ for up to 7 days. Exonucleasedigested PCR product can be stored store at $4 \mathrm { { } ^ { \circ } C }$ or ${ } _ { - 2 0 ^ { \circ } \mathrm { C } }$ for up to 7 days. After combining the exonuclease-digested PCR with hybridization reagents, the hybridization reaction can be loaded on the cartridge and held at $1 0 { - } 2 5 ^ { \circ } \mathrm { C }$ for up to 8 hours before initiating hybridization of the cartridge on the XT-8 instrument.

Controls: Positive or negative sample controls are not included with this assay. It is required that a non-template control, called a DNA Contamination Monitor, be included with each PCR run of the eSensor Warfarin Sensitivity Test.

Each test contains internal positive and negative controls to assure proper functioning of the system. Each cartridge contains two electrodes coated with a capture probe that is complementary to a synthetic target DNA present in the hybridization mixture. The target is in turn complementary to a control signal probe in the hybridization mixture, and thus generates an appropriate signal in the assay. The positive control is designed to detect a systematic failure of the hybridization and/or detection processes. A lack of signal for the positive control indicates a genotyping assay failure. If a correct signal is observed for the positive control, but one or more genotyping assays are invalid due to low signals, then a failure of DNA isolation or PCR amplification is indicated. A negative control is present on each cartridge, consisting of a capture probe that does not hybridize to any sequence within the target DNA or signal probes. Signals on the negative control indicate an assay system failure.

# d. Detection limit:

An upper and lower limit of detection study was performed to assess the genotyping performance of the eSensor XT-8 system across a range of genomic DNA input concentrations.

In order to determine the lowest concentration of DNA at which this assay is accurate, serial dilutions $( 1 0 0 0 , 1 0 0 , 1 0 , 1 , 0 . 1 ~ \mathrm { n g } )$ of two genomic DNA samples of different genotypes were assayed 20 times using the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test. An additional run was performed for tests that gave a first-pass no-call result. All replicates were correctly genotyped at $0 . 1 { \mathrm { n g } }$ of purified DNA per reaction, and $90 \%$ of samples gave a genotype at $\boldsymbol { . 0 0 0 \mathrm { n g } }$ of purified DNA per reaction. Results are as summarized in the table below. The recommended range of DNA input amounts for the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test is from $1 0 ~ \mathrm { n g }$ .

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>DNAinputamount(ng)</td><td rowspan=1 colspan=1>Replicatestested</td><td rowspan=1 colspan=1>First-passcorrectcalls</td><td rowspan=1 colspan=1>First-pass no-calls</td><td rowspan=1 colspan=1>Finalcorrectcalls</td><td rowspan=1 colspan=1>Finalincorrectcalls</td></tr><tr><td rowspan=5 colspan=1>1</td><td rowspan=5 colspan=1>2C9 wt/wt VKORC1 G/G</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>2</td><td rowspan=5 colspan=1>2C9 wt/*3 VKORC1 G/A</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr></table>

# e. Analytical specificity:

Potentially interfering substances were selected based on the criteria described in the Clinical and Laboratory Standards Institute approved guideline EP7-A2, Interference Testing in Clinical Chemistry and an internally-performed risk assessment.

Alternate description:

For the interference studies, 16 individual samples were treated with high concentrations of 8 different interfering substances. Studies contained four replicates of a pooled sample (containing 3 whole blood samples from wild-type donors) that were extracted by three different extraction methods (and therefore 12 samples), and four replicates of a sample (with genotype $^ { * } 2 / { ^ * } 3$ for 2C9) extracted by one extraction method.

Each extracted sample was tested with the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test.   
Genotypes of all samples were confirmed by sequencing and included in the final report.

<table><tr><td colspan="1" rowspan="1">Interfering Substance</td><td colspan="1" rowspan="1">Highconcentration</td><td colspan="1" rowspan="1">Correct call</td><td colspan="1" rowspan="1">Miscal1</td><td colspan="1" rowspan="1">No-call</td></tr><tr><td colspan="1" rowspan="1">HumanSerum Albumin</td><td colspan="1" rowspan="1">3 g/dL</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Bilirubin (conjugated)</td><td colspan="1" rowspan="1">30 mg/dL</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Human Immunoglobulin G</td><td colspan="1" rowspan="1">3 g/dL</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1*</td></tr><tr><td colspan="1" rowspan="1">Triglycerides</td><td colspan="1" rowspan="1">500 mg/dL</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">~20 g/dL addedas purified redblood cells</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">EDTA treated plasma tubes</td><td colspan="1" rowspan="1">5xrecommended</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Warfarin</td><td colspan="1" rowspan="1">32.5 μmol/L</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Heparin Sodium (fromPorcine Intestinal Mucosa)</td><td colspan="1" rowspan="1">3,000 U/L</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">*NEG control failure due toplatform error; not related tothe interferent</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td></tr></table>

f. Assay cut-off: Not Applicable

2. Comparison studies:

# a. Method comparison with predicate device:

In a method comparison study, a total of 157 samples were genotyped using the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test and bi-directional DNA Sequencing. All first-pass sample results (157/157) obtained with the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test agreed with the results obtained by DNA sequencing. The $9 5 \%$ lower confidence bound on a per-sample basis was $9 8 . 1 \%$ , and $9 9 . 4 \%$ on a per-SNP basis (471/471). The following table summarizes the results of the method comparison study:

<table><tr><td rowspan=2 colspan=1>DNASequencingResult</td><td rowspan=1 colspan=3>eSensor® Warfarin Sensitivity TestResult</td></tr><tr><td rowspan=1 colspan=1>2C9 wt/wt</td><td rowspan=1 colspan=1>2C9 wt/*2</td><td rowspan=1 colspan=1>2C9*2/*2</td></tr><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>36.8%</td></tr><tr><td rowspan=2 colspan=1>DNASequencingResult</td><td rowspan=1 colspan=3>eSensor® Warfarin Sensitivity TestResult</td></tr><tr><td rowspan=1 colspan=1>2C9 wt/wt</td><td rowspan=1 colspan=1>2C9 wt/*3</td><td rowspan=1 colspan=1>2C9 *3/*3</td></tr><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>87.3%</td><td rowspan=1 colspan=1>22.4%</td></tr><tr><td rowspan=2 colspan=1>DNASequencingResult</td><td rowspan=1 colspan=3>eSensor® Warfarin Sensitivity TestResult</td></tr><tr><td rowspan=1 colspan=1>VKORC1G/G</td><td rowspan=1 colspan=1>VKORC1GG/A</td><td rowspan=1 colspan=1>VKORC1AA</td></tr><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>95.6%</td><td rowspan=1 colspan=1>95.4%</td><td rowspan=1 colspan=1>89.5%</td></tr></table>

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.   
b. Clinical specificity: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

Table 7: Genotype frequencies for all alleles genotyped by the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test

<table><tr><td rowspan=1 colspan=1>Ethnicity</td><td rowspan=1 colspan=1>CYP2C9*2</td><td rowspan=1 colspan=1>CYP2C9*3</td><td rowspan=1 colspan=1>VKOR</td></tr><tr><td rowspan=1 colspan=1>Caucasian</td><td rowspan=1 colspan=1>0.9-20%1</td><td rowspan=1 colspan=1>0-14.5%1</td><td rowspan=1 colspan=1>37%2</td></tr><tr><td rowspan=1 colspan=1>African</td><td rowspan=1 colspan=1>0.8-7%1</td><td rowspan=1 colspan=1>0.4-3%1</td><td rowspan=1 colspan=1>14%2</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>0%1</td><td rowspan=1 colspan=1>0-8.2%</td><td rowspan=1 colspan=1>89%2</td></tr></table>

# N. Instrument Name:

# eSensor XT- 8 system

The basic model XT-8 instrument consists of the user interface display and a processing tower with eight independent cartridge slots where hybridization and scanning of the cartridge occur. The XT-8 is also available in two and three tower versions with 16 and 24 independent slots respectively. The XT-8 instrument includes several software modules which together allow users to process genetic tests and create reports of the testing results. The primary software module is the application software, which is what a user interacts with while using the instrument. All of the hardware functions of the instrument are controlled by embedded firmware under the command of the application software. A separate Assay Analysis Module or AAM is used by the application software to create assay-specific results and reports. By keeping these assay-specific AAMs separate from the application, they may be individually validated and installed on an instrument without having to revalidate the application software.

The application software and AAM were developed using $\mathbf { C \# }$ in the Microsoft Visual Studio .net 2.0 development framework. The embedded firmware was developed using the C language using the Greenhills compiler for the CPU firmware and Texas Instrument Code Composer for the DSP firmware.

# O. System Descriptions:

1. Modes of Operation:

The instrument has a single mode of operation with eight hybridization slots that can run 8 different reactions at a time.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_ ___ or No

3. Specimen Identification:

The sponsor’s submitted software documentation demonstrated that the software design meets the stated requirements for this device and were verified and validated in part by testing the system with known wild type, mutant, and negative control samples determining that signal detection configurations result in correct call/results.

4. Specimen Sampling and Handling:

Sample identification is performed through entering in the sample ID and the barcode of the test cartridge to be used with the sample. The barcode of the cartridge is scanned as the sample is about to be run.

# 5. Calibration:

The instrument does not require calibration. Quality control testing during instrument manufacture is performed to confirm that measurement and control of temperature and electrical current are within specification:

The operator software allows the user to confirm the thermal and electrical performance of the system at will:

• The thermal tests cycle all slots through their useful temperature range, and confirm that they reach thermal equilibrium within defined time limits.

• The electrical test characterizes the electronics for its linearity at many different frequencies and voltages spanning its operational capabilities. This test is first run during manufacturing and results of this characterization are stored in the instrument’s permanent memory. The user may then run the same test. If the results of the user test agree with the permanently stored data, this indicates that the electronics have not drifted from their original state since manufacturing.

In addition to the QC and user tests, the instrument performs functional self tests each time it is turned on. These tests confirm that the thermal sensors are functional and communicating, that the on-board memory is working, that the switching electronics is functional and the embedded firmware is not corrupted. If any of the tests fail, the corresponding elements are locked out from use by the operator software.

# 6. Quality Control:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test on the XT-8 Instrument uses a combination of electronic and molecular controls and validity criteria to verify the proper operation of the system and assure the delivery of accurate results.

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test provides a system of controls for proper instrument setup and function, preparation of reagents, handling of samples, and function of the cartridge and reagents. The criteria for control and test results comprise consistent and stringent test for reporting of results. Furthermore, results from control tests provide useful information to troubleshoot test failures and to identify the potential root cause(s) and corrective action(s). A defect in product or process occurring at any step of the process is expected to cause a failed control or a signal which does not meet the criteria for a valid test result. In any of these cases, no result is reported, and further troubleshooting and/or a repeat of the test is performed.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.