# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: k073071   
B. Purpose for Submission: New device   
C. Measurand: Genotype of Cytochrome P450 2C9 (CYP450 2C9) and Vitamin K epoxide reductase complex subunit 1 (VKORC1)   
D. Type of Test: Genotyping Realtime PCR   
E. Applicant: TrimGen Corporation   
F. Proprietary and Established Names: eQ-PCR™ LC Warfarin Genotyping kit

G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>Cytochrome P450 2C9(CYP450 2C9) DrugMetabolizing EnzymeGenotyping System(ODW)</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.3360Drug metabolizingenzyme genotypingsystem</td><td rowspan=1 colspan=1>Toxicology (91)</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>INSTRUMENTATIONFOR CLINICALMULTIPLEXTEST SYSTEMS(NSU)</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 -INSTRUMENTATIONFOR CLINICALMULTIPLEX TESTSYSTEMS</td><td rowspan=1 colspan=1>CLINICALCHEMISTRY</td></tr><tr><td rowspan=1 colspan=1>VITAMIN KEPOXIDEREDUCTASECOMPLEXSUBUNIT ONE(VKORC1)GENOTYPINGSYSTEM (ODV)</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 864.7750 -PROTHROMBINTIME TEST</td><td rowspan=1 colspan=1>HEMATOLOGY</td></tr></table>

# H. Intended Use:

1. Intended use(s): Refer to Indications for use below.

2. Indication(s) for use:

The eQ-PCR™ LC Warfarin Genotyping kit is an in vitro diagnostic test for the detection and genotyping of two single nucleotide polymorphisms (SNP) in the cytochrome P450 enzyme gene CYP2C9 known as $\mathrm { C Y P 2 C 9 ^ { * } } 2$ (C430T) and $\mathrm { C Y P 2 C 9 ^ { * } } 3$ (A1075C), and a SNP in the vitamin K epoxide reductase complex 1 gene VKORC1, known as VKORC1 $\left( - 1 6 3 9 \mathrm { G } \mathrm { > } \mathrm { A } \right)$ obtained from EDTA – anticoagulated whole blood samples. The eQ-PCR LC Warfarin Genotyping kit is used as an aid in the identification of patients at risk for increased warfarin sensitivity. It is a qualitative assay for use in clinical laboratories upon prescription by the attending physician.

3. Special conditions for use statement(s): For Prescription use only

4. Special instrument requirements: For use with LightCycler $^ \mathrm { \textregistered }$ Real-Time PCR System instrument model 1.2.

I. Device Description: eQ-PCRTM LC Warfarin Genotyping Kit reagents

The reagents in the eQ-PCR™ LC Warfarin Genotyping Kit are designed to be used specifically with the Roche LightCycler $\textsuperscript { \textregistered }$ . They are provided for 32 tests per kit and include:

• The PCR Mix is a pre-mixed reagent for PCR amplification including a DNA polymerase, PCR Buffer and dNTPs.   
• The LC1 is the Probe Mix 1 containing primer and probes designed to detect the $2 \mathrm { C } 9 ^ { \ast } 3$ (A1075C) SNP in the CYP2C9 gene and the $\left( - 1 6 3 9 \mathrm { G } \mathrm { > A } \right)$ ) SNP in the VKORC1 gene.   
• The LC2 is the Probe Mix 2 containing primers and probes designed to detect the $2 \mathrm { C } 9 ^ { \ast } 2$ (C430T) SNP in the CYP2C9 gene.   
• The NF Water is nuclease-free water.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Verigene $\textsuperscript { \textregistered }$ Warfarin Metabolism Nucleic Acid Test

2. Predicate 510(k) number(s): k070804

3. Comparison with predicate: The following table summarizes the similarities and differences between the new and predicate devices.

# Similarities:

<table><tr><td rowspan=2 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Nanosphere Warfarin Assay</td><td rowspan=1 colspan=1>eQ-PCRTM LC WarfarinGenotyping Kit</td></tr><tr><td rowspan=1 colspan=1>Predicate device (K070804)</td><td rowspan=1 colspan=1>Proposed device</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Verigene systemwith Verigene cartridge</td><td rowspan=1 colspan=1>Roche LightCyclerwith glass capillary tubes</td></tr><tr><td rowspan=1 colspan=1>Method todeterminespecificity</td><td rowspan=1 colspan=1>Hybridization</td><td rowspan=1 colspan=1>PCRHybridizationMelting Curve Analysis</td></tr><tr><td rowspan=1 colspan=1>Assay conditions</td><td rowspan=1 colspan=1>A closed system</td><td rowspan=1 colspan=1>A closed tube assay</td></tr><tr><td rowspan=1 colspan=1>Procedures</td><td rowspan=1 colspan=1>Reaction and SNP analysisrequire separate operations(transfer cartridge to reader)</td><td rowspan=1 colspan=1>PCR and SNP analysis in sametube (no transfer steps)</td></tr></table>

# Differences:

<table><tr><td rowspan=1 colspan=1>Gene</td><td rowspan=1 colspan=1>CYP2C9 and VKORC1</td><td rowspan=1 colspan=1>CYP2C9 and VKORC1</td></tr><tr><td rowspan=1 colspan=1>Specimen type</td><td rowspan=1 colspan=1>Purified DNA from humanwhole blood samples</td><td rowspan=1 colspan=1>Purified DNA from humanwhole blood samples</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Drug Metabolizing Enzyme Genotyping System – Class II Special Controls Guidance Document

# L. Test Principle:

The eQ-PCR™ LC Warfarin Genotyping Assay uses real time PCR technology to amplify and detect the target DNA. The extracted DNA sample is mixed with a PCR Mix, and an eQ-PCR™ specific Probe Mix reagent containing specific primers and fluorescent labeled probes for the CYP2C9 and/or VKORC1 gene polymorphisms. Amplification and detection are then performed in the Roche Diagnostics LightCycler $\textsuperscript { \textregistered }$ Real-Time PCR System instrument model 1.2 using conditions defined in the specific eQ- PCR™ LC Warfarin Genotyping Kit labeling. After the PCR reaction is completed, the LightCycler System automatically proceeds to the melting curve-based detection method. The signal detection is based on fluorescence resonance energy transfer (FRET) technology. Two probes are designed for each SNP and labeled with different fluorophores. The anchor probe selectively hybridizes to the sequence flanking the SNP site, and the SNP detection probe recognizes the SNP and hybridizes to the sequence containing the SNP site. During FRET, the fluorophore on the anchor probe is excited by the light source of the LightCycler and the excitation energy is transferred to the fluorophore on the SNP detection probe. The emitted fluorescence is then measured at the respective wavelength.

The SNP detection is performed by melting curve analysis, a method that discriminates different allelic forms of DNA by melting temperature. The SNP detection probe is designed to have a unique melting temperature to the wild type and mutant SNP. With change in temperature, the probe dissociates from the target sequence at a specific melting temperature depending on the allelic composition of SNP. The SNP type is then detected based on the melting curve specific to each SNP.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

To evaluate the within-laboratory, between operator, and between laboratory reproducibility of the eQ-PCR™ LC Warfarin Genotyping Kit on the Roche LightCycler $\textsuperscript { \textregistered }$ , three sites (two external and the sponsor’s laboratory) and five operators participated. A panel of 31samples was tested at each of the three sites on each of five days. At the two external sites, two operators per site performed the testing on alternate days. The panel consisted of whole blood (WB) samples collected in EDTA from 20 donors, and 9 genomic DNA samples (cctl below). These samples represented all the genotypes tested by this device. Each site used a different DNA extraction method for isolating DNA from whole blood samples. Each site then tested the resulting extracted nucleic acids using the eQ-PCR™ LC Warfarin Genotyping Kit on the Roche LightCycler $\textsuperscript { \textregistered }$ . Genotypes for each sample were determined by bi-directional sequencing and compared with the results from the device.

The composition of the reproducibility panel with the call rate for all three sites combined for each sample is described in Table below. There were 6 no calls, but no incorrect calls reported. The entire study produced a correct call rate of $9 8 . 7 \%$ .

Genotypic composition and percent call rate per sample:   

<table><tr><td colspan="1" rowspan="2">SampleID</td><td colspan="3" rowspan="1">Genotype</td><td colspan="1" rowspan="1">#samples</td><td colspan="1" rowspan="1">#genotype</td><td colspan="1" rowspan="1">#correct</td><td colspan="1" rowspan="2">#nocalls</td><td colspan="1" rowspan="2">#incorrectcalls</td><td colspan="1" rowspan="2">Correctcall rate%</td></tr><tr><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">tested</td><td colspan="1" rowspan="1">calls made</td><td colspan="1" rowspan="1">calls</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/*3</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93.3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93.3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">86.7</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">wt/*2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93.3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93.3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">wt/*2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">wt/*2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">wt/*2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93.3</td></tr><tr><td colspan="1" rowspan="1">cctl-1</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-3</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-4</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-5</td><td colspan="1" rowspan="1">wt/*2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-6</td><td colspan="1" rowspan="1">*2/*2</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-7</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-8</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">wt/*3</td><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">cctl-9</td><td colspan="1" rowspan="1">wt/wt</td><td colspan="1" rowspan="1">*3/*3</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="4" rowspan="1">Total</td><td colspan="1" rowspan="1">465</td><td colspan="1" rowspan="1">459</td><td colspan="1" rowspan="1">459</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">98.7</td></tr></table>

Reproducibility per genotype:   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>#samplestested</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>#incorrect calls</td><td rowspan=1 colspan=1>#nocalls</td><td rowspan=1 colspan=1>Correct callrate%</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>461</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>99.1</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>464</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>464</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.7</td></tr></table>

£ Total tests for each genotype: 31 samples x 3 sites x 5 days repeats $= 4 6 5$ tests

Reproducibility per site per Operator:

<table><tr><td rowspan=1 colspan=1>Site/Operator</td><td rowspan=1 colspan=1>Locus</td><td rowspan=1 colspan=1>#Samples</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>#incorrectcalls</td><td rowspan=1 colspan=1>#nocalls</td><td rowspan=1 colspan=1>Correctcall rate%</td></tr><tr><td rowspan=3 colspan=1>Site 1/0perator 1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>97.85</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>Site 1/0perator 2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>Site 2/0perator 3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=3 colspan=1>Site 2/0perator 4</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>96.8</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>Site 3/Operator 5</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

# Lot to lot reproducibility results

A study was conducted at the sponsor’s facility to determine the reproducibility between kit reagent lots of the eQ-PCR™ LC Warfarin Genotyping assay. A panel of 24 whole blood samples was tested using three reagent lots for five days. The results per genotype for each lot are illustrated in the table below.

<table><tr><td rowspan=1 colspan=1>Loci</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>#samplestested</td><td rowspan=1 colspan=1>#tests$</td><td rowspan=1 colspan=1>Genotypingcalls made</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>#incorrectcalls</td><td rowspan=1 colspan=1>#nocalls</td><td rowspan=1 colspan=1>Correctcall rate%</td></tr><tr><td rowspan=3 colspan=1>2C9 *2</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>2C9 *3</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>VKORC1</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

§ Total tests for each Lot: 24 samples x 5 repeats $= 1 2 0$ tests

b. Linearity/assay reportable range: Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The sponsor conducted stability studies for DNA stored at $- 2 0 ^ { \circ } \mathrm { C }$ and $2 { - } 8 ^ { \circ } \mathrm { C }$ using 9 commercially available genomic DNA samples with 3 lots of the reagent kit. Stability studies for DNA stored at $- 2 0 ^ { \circ } \mathrm { C }$ were conducted up to one year at 3-month intervals and for DNA stored at ${ } - 2 { - } 8 ^ { \circ } \mathrm { C }$ for 30 days. Results demonstrated consistent accurate results. The sponsor is continuing the stability studies for DNA stored at $- 2 0 ^ { \circ } \mathrm { C }$ studies for 2 yrs. However, the sponsor recommends using DNA from fresh whole blood samples for genotyping assay.

Reagents for the Warfarin Genotyping Kit are shipped to the user laboratory on dry ice. The sponsor recommends storing all reagents at $- 2 0 ^ { \circ } \mathrm { C }$ in dark. The kit is stable at - $2 0 \mathrm { { ^ \circ C } }$ for one year. After first use, all reagents should be stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ in dark. The sponsor states that under these conditions, reagents are suitable for use up to three months.

Controls: The sponsor does not provide positive or negative controls to be used with this assay. However, the sponsor has recommended that a positive (heterozygous and/or homozygous for the three genotypes) sample for each variant, a negative control (a sample that does not contain the mutation of interest, i.e., a wild type sample); and a “NonTemplate Control” be included with each test run. All quality control requirements and testing should be performed in conformance with local, state and/or federal regulations.

The sponsor conducted preliminary studies to establish the Melting Temperature (Tm) cut-off values to identify the presence of individual SNP positions by the eQ-PCR™ LC Warfarin Genotyping Kit. A total of 64 individual samples were tested to evaluate the melting temperature Cut-Off value for each SNP type. The average melting temperature for each SNP was calculated and the Cut–Off value was determined based on Average value $\pm 3$ SD. The SNP result for each sample was confirmed by bidirectional sequencing. Results are listed in the table below.

Tm values obtained for each genotype:   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>VKORC1</td><td rowspan=1 colspan=2>2C9*2</td><td rowspan=1 colspan=2>2C9*3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Tm Low(variant)</td><td rowspan=1 colspan=1>Tm High(Wt)</td><td rowspan=1 colspan=1>Tm Low(variant)</td><td rowspan=1 colspan=1>Tm High(Wt)</td><td rowspan=1 colspan=1>Tm Low(Wt)</td><td rowspan=1 colspan=1>Tm High(variant)</td></tr><tr><td rowspan=1 colspan=1>Min:</td><td rowspan=1 colspan=1>53.31</td><td rowspan=1 colspan=1>62.36</td><td rowspan=1 colspan=1>54.52</td><td rowspan=1 colspan=1>63.47</td><td rowspan=1 colspan=1>53.26</td><td rowspan=1 colspan=1>60.32</td></tr><tr><td rowspan=1 colspan=1>Max:</td><td rowspan=1 colspan=1>55.92</td><td rowspan=1 colspan=1>64.52</td><td rowspan=1 colspan=1>58.44</td><td rowspan=1 colspan=1>66.74</td><td rowspan=1 colspan=1>56.49</td><td rowspan=1 colspan=1>63.16</td></tr><tr><td rowspan=1 colspan=1>Avg.</td><td rowspan=1 colspan=1>54.94</td><td rowspan=1 colspan=1>63.57</td><td rowspan=1 colspan=1>57.34</td><td rowspan=1 colspan=1>65.58</td><td rowspan=1 colspan=1>55.23</td><td rowspan=1 colspan=1>61.89</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.86</td></tr></table>

The established Cut-Off value for each SNP:   

<table><tr><td rowspan=1 colspan=1>SNP ID</td><td rowspan=1 colspan=1>Melting temperature forWild type</td><td rowspan=1 colspan=1>Melting temperature forvariant</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>63.57 ± 1.950C</td><td rowspan=1 colspan=1>54.94 ± 2.400C</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>65.58 ± 1.770C</td><td rowspan=1 colspan=1>57.34 ± 2.700C</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>55.23 ± 2.700C</td><td rowspan=1 colspan=1>61.89 ± 2.580C</td></tr></table>

# d. Limit of Detection:

A study was conducted at the sponsor’s facility to validate the recommended amounts of DNA required for genotyping test using eQ-PCR™ LC Warfarin Genotyping test. Six human genomic DNA samples at 7 different concentrations to obtain 0.1, 1, 10, 50, 100, 200 and $6 0 0 \mathrm { n g }$ in the final reaction tube were used and 26 repeats conducted for each test level (total of 3312 tests). These six genomic DNA samples covered all 9 possible $2 \mathrm { C } 9 ^ { \ast } 2$ , $2 \mathrm { C } 9 ^ { \ast } 3$ , and VKORC-1 genotypes. Two of the six human genomic DNA samples were tested in two additional concentrations $( 3 0 0 \mathrm { n g } \ : \& \ : 4 0 0 \mathrm { n g } )$ and repeated three times. Based on the results, there were two “no calls” at $0 . 1 \ \mathrm { n g } \ \mathrm { D N A }$ . The results of DNA limitation study are given in the tables below. For eQ-PCR™ LC Warfarin Genotyping assay, the sponsor recommends using a total of $5 0 - 2 0 0 \mathrm { n g }$ genomic DNA in the final reaction tube.

Call rate based on DNA amount:   

<table><tr><td colspan="1" rowspan="1">InputDNAAmount(ng)</td><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">#samples</td><td colspan="1" rowspan="1">#repeats</td><td colspan="1" rowspan="1">#tests</td><td colspan="1" rowspan="1">#genotypecalls</td><td colspan="1" rowspan="1">#correctcalls</td><td colspan="1" rowspan="1">#incorrectcalls</td><td colspan="1" rowspan="1">#nocalls</td><td colspan="1" rowspan="1">Correctcall rate(%)</td></tr><tr><td colspan="1" rowspan="3">0.1</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">466</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">2</td><td colspan="1" rowspan="3">99.6</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="3">10</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="3">50€</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="3">100€</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="3">200</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="3">300</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="3">18</td><td colspan="1" rowspan="3">18</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="3">400</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="3">18</td><td colspan="1" rowspan="3">18</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="3">600</td><td colspan="1" rowspan="1">2C9*2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">468</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">100</td></tr><tr><td colspan="1" rowspan="1">2C9*3</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1">VKORC1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">156</td></tr></table>

£ The sponsor recommended DNA amount for eQ-PCR Warfarin Genotyping assay

Call rate based on genotype:

<table><tr><td rowspan=1 colspan=2>Allele</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>#samples</td><td rowspan=1 colspan=1>#tests</td><td rowspan=1 colspan=1>#genotypecalls</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>#incorrectcalls</td><td rowspan=1 colspan=1>#no calls</td><td rowspan=1 colspan=1>Correctcall rate(%)</td></tr><tr><td rowspan=3 colspan=2>2C9*2</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>740</td><td rowspan=1 colspan=1>740</td><td rowspan=1 colspan=1>739</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>wt/*2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=2>2C9*3</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>728</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>wt/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=2>VKORC1</td><td rowspan=1 colspan=1>G/G</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>552</td><td rowspan=1 colspan=1>552</td><td rowspan=1 colspan=1>552</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>G/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>370</td><td rowspan=1 colspan=1>370</td><td rowspan=1 colspan=1>370</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>A/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3312</td><td rowspan=1 colspan=1>3312</td><td rowspan=1 colspan=1>3310</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>99.9</td></tr></table>

# e. Analytical specificity:

The sponsor conducted studies to demonstrate the interference of albumin $( 5 \mathrm { m g / m l } )$ , bilirubin $( 1 0 \mathrm { u M } )$ , triglycerides $( 4 \mathrm { m g / m l } )$ , heparin $\left( 1 4 \mathrm { U / m l } \right)$ , hemoglobin $\left( 8 0 \mathrm { m g / m l } \right)$ , magnesium chloride $( 6 \mathrm { m M } )$ , and lithium chloride $( 9 \mathrm { m M } )$ and excess of extraction buffers on eQ-PCR assay. Nine blood samples covering all of the genotypes under evaluation were included. An aliquot of each sample was spiked with the above interfering substances and the results compared against the sample aliquot with no substance. The tests were repeated 6 times for each sample. The test results indicated no interference from above substances at the concentration levels tested and call rates remained unaffected.

f. Assay cut-off: Not Applicable.

2. Comparison studies:

a. Method comparison with predicate device:

The sponsor conducted initial method comparison studies at 3 sites to demonstrate the accuracy of the device in comparison to bi-directional sequencing. This study included whole blood samples collected in EDTA from 150 donors. DNA was extracted at the sponsor’s facility and aliquots of 50 unique DNA samples distributed to 3 sites (two external and the sponsor’s facility) that participated in the study. The studies on the 150 samples resulted in 3 no-calls and 2 incorrect calls The sponsor’s study using 150 samples did not cover $^ { * } 3 / { ^ * } 3$ variant genotypes tested by the device. Therefore, the sponsor conducted a second study that included 2 additional $\ast _ { 3 } / \ast _ { 3 }$ samples.

Genotyping results confirmed $\ast _ { 3 } / \ast _ { 3 }$ variant genotype for one of the samples, while the other new sample gave an incorrect genotype call of $w t / { } ^ { * } 2 , { } ^ { * } 3 / { } ^ { * } 3$ for the 2C9 gene. The sponsor performed a third study that included 7 new samples $( 4 ^ { * } 3 / { } ^ { * } 3$ and $3 ~ { ^ { * } } 2 / { ^ { * } } 2$ samples). There were no incorrect calls or no-calls reported in this third study.

Genotype call-rates for three studies combined are given in table below.

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1># ofsamples</td><td rowspan=1 colspan=1># ofcorrectcalls</td><td rowspan=1 colspan=1># ofincorrectcalls</td><td rowspan=1 colspan=1># ofnocalls</td><td rowspan=1 colspan=1>Correctcall rate</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3$</td><td rowspan=1 colspan=1>98.11%</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>1$fra }$</td><td rowspan=1 colspan=1>3}$</td><td rowspan=1 colspan=1>97.48%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>1$}$</td><td rowspan=1 colspan=1>3$</td><td rowspan=1 colspan=1>97.48%</td></tr></table>

‡ The incorrect genotype call reported a genotype of $w t / { ^ { * } } 2$ , $\ast _ { 3 } / \ast _ { 3 }$ † No-calls resulted from a system failure.

The genotype call-rate for each specific genotype based on all three studies combined is given in the table below.

<table><tr><td rowspan=1 colspan=1>Allele</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>#samples</td><td rowspan=1 colspan=1>#genotypecalls</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>#incorrectcalls</td><td rowspan=1 colspan=1>#no calls</td><td rowspan=1 colspan=1>Correctcall rate(%)</td><td rowspan=1 colspan=1>95%0ne-sidedConfidenceLowerLimit*</td></tr><tr><td rowspan=4 colspan=1>2C9*2</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>94.40%</td></tr><tr><td rowspan=1 colspan=1>wt /*2</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1‡</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>87.68%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.98%</td></tr><tr><td rowspan=1 colspan=1>Sub-total</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>94.59%</td></tr><tr><td rowspan=4 colspan=1>2C9*3</td><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.09%</td></tr><tr><td rowspan=1 colspan=1>wt /*3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>75.32%</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1‡$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>83.3</td><td rowspan=1 colspan=1>54.28%</td></tr><tr><td rowspan=1 colspan=1>Sub-total</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>96.40%</td></tr><tr><td rowspan=4 colspan=1>VKORC1</td><td rowspan=1 colspan=1>G/G</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>93.16%</td></tr><tr><td rowspan=1 colspan=1>G/A</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>94.32%</td></tr><tr><td rowspan=1 colspan=1>A/A</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.52%</td></tr><tr><td rowspan=1 colspan=1>Sub-total</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>95.54%</td></tr><tr><td rowspan=1 colspan=3>Total for Assay</td><td rowspan=1 colspan=1>477</td><td rowspan=1 colspan=1>466</td><td rowspan=1 colspan=1>2‡</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>96.18%</td></tr></table>

‡ The incorrect genotype call reported a genotype of $w t / ^ { * } 2$ , $\ast _ { 3 } / \ast _ { 3 }$ . † No-calls resulted from a system failure.

b. Matrix comparison:

The sponsor recommends only fresh whole blood samples only for use with this device.   
Therefore, Matrix comparison studies were not applicable.

3. Clinical studies:

a. Clinical Sensitivity: Not Applicable.

b. Clinical specificity: Not Applicable.

c. Other clinical supportive data (when a. and b. are not applicable):

4. Clinical cut-off: Not Applicable.

5. Expected values/Reference range:

The following table provides the allele frequency across ethnic groups for the CYP2C9 and VKORC1 variants.

Expected Genotypic Frequency of Different Ethnic Groups:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Caucasian*1</td><td rowspan=1 colspan=1>African 2,3</td><td rowspan=1 colspan=1>Asian1</td></tr><tr><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>78.7%</td><td rowspan=1 colspan=1>87.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>wt /*2</td><td rowspan=1 colspan=1>20.4%</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>wt/wt</td><td rowspan=1 colspan=1>88.0%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>92.7%</td></tr><tr><td rowspan=1 colspan=1>wt /*3</td><td rowspan=1 colspan=1>11.6%</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>7.3%</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>39.1%</td><td rowspan=1 colspan=1>79.2%</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>46.7%</td><td rowspan=1 colspan=1>20.8%</td><td rowspan=1 colspan=1>18.3%</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>14.2%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>79.8%</td></tr></table>

† The data presented are based on the genotypic frequencies in the following 3 publications:

1. Yuan et al., A novel functional VKORC1 promoter polymorphism is associated with inter-individual and inter-ethnic differences in warfarin sensitivity. Human Molecular Genetics 14(13): 1745-1751, 2005.

2. Kirchheiner and Brockmoller. Clinical Consequences of Cytochrome P450 2C9 Polymorphisms. Clinical Pharmacology & Therapeutics 77(1):1-16, 2005.

3. Schelleman et al., Warfarin response and Vitamin K Epoxide Reductase Complex in African Americans and Caucasians. Nature 81(5): 742-747, 2007

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.