# 510(k) Summary

JUL 1 7 2008

This summary of the 510(k) safety and effectiveness information is being submitted in accordance with the requirements of SMDA 1900 and CFR 807.92

<table><tr><td rowspan=1 colspan=1>510(k) number</td><td></td></tr><tr><td rowspan=1 colspan=1>SummaryPreparationDate</td><td rowspan=1 colspan=1>July 15, 2008</td></tr><tr><td rowspan=1 colspan=1>Submitted by</td><td rowspan=1 colspan=1>Osmetech Molecular Diagnostics757 South Raymond Ave.Pasadena, CA 91105Phone: 626 463-2000      Fax: 626 463-2012</td></tr><tr><td rowspan=1 colspan=1>Contact</td><td rowspan=1 colspan=1>Robert Dicheck, Vice President of Quality &amp; Regulatory Affairs (Official Correspondent)Michael Reed, Ph.D., Director of Product Development &amp; Project Manager</td></tr><tr><td rowspan=1 colspan=1>Proprietarynames andclassifications</td><td rowspan=1 colspan=1>For the assay:eSensor® Warfarin Sensitivity TestRegulations: 21CFR §862.3360 - Drug Metabolism Enzyme Genotyping Test21CFR §864.7750 - Prothrombin Time TestPanels: 91 Toxicology &amp; 81 HematologyClassification: IIProduct Codes:  ODWCytochrome P450 2C9 (CYP450 2C9) Drug Metabolizing Enzyme GenotypingSystemODV Vitamin K epoxide reductase complex subunit 1 (VKORC1) GenotypingSystemFor the instrument:eSensor® XT-8 SystemRegulation: 21CFR §862.2570  Instrument for Clinical Multiplex Test SystemsPanel: 75 Clinical ChemistryClassification: IIProduct Code:   NSU Instrumentation for Clinical Multiplex Test Systems</td></tr><tr><td rowspan=1 colspan=1>Commonnames</td><td rowspan=1 colspan=1>For the Assay:Warfarin Sensitivity Test (CYP2C9*2, CYP2C9*3, VKORCI)For the Instrument.Bench-top molecular diagnostics workstation</td></tr><tr><td rowspan=1 colspan=1>Intended uses</td><td rowspan=1 colspan=1>The eSensor® Warfarin Sensitivity Test is an in vitro diagnostic or the detection and genotyping of the *2and * 3 alleles of the cytochrome P450 (CYP450) 2C9 gene locus and the Vitamin K epoxide reductaseC1 (VKORC1) gene promoter polymorphism (-1639G&gt;A) from genomic DNA extracted from fresh wholeblood samples preserved with EDTA, as an aid in the identification of patients at risk for increasedwarfarin sensitivity. The eSensor® Warfarin Sensitivity Test is for Rx only professional use within theconfines of a licensed laboratory, as defined by the Clinical Laboratory Improvement Amendments (CLIA)of 1988.The eSensor® T-8 Instrument is an invitro diagnostic device intended or genotyping multiple mutationsor polymorphisms in an amplified DNA sample utilizing electrochemical detection technology.</td></tr><tr><td rowspan=1 colspan=1>Specialconditions forusestatement(s)</td><td rowspan=1 colspan=1>The eSensor® Warfarin Sensitivity Test is for Rx only professional use within the confines of a licensedlaboratory, as defined by the Clinical Laboratory Improvement Amendments (CLIA) of 1988.</td></tr><tr><td rowspan=1 colspan=1>Predicatedevices</td><td rowspan=1 colspan=1>Nanosphere Verigene® Warfarin Metabolism Nucleic Acid Test and Verigene@ System</td></tr></table>

# Device descriptions

Comparison to technological features of the predicate device   

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>Verigene® Warfarin MetabolismNucleic Acid Test and Verigene®System</td><td rowspan=1 colspan=1>eSensor® Warfarin SensitivityTest and XT-8 System</td></tr><tr><td rowspan=1 colspan=1>Test type</td><td rowspan=1 colspan=1>Qualitative genetic test for singlenucleotide polymorphism detection</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Genomic DNA obtained from a humanwhole blood sample</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Target of detection</td><td rowspan=1 colspan=1>Single-nucleotide polymorphism</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>DNA extraction</td><td rowspan=1 colspan=1>Performed off-line</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Genes</td><td rowspan=1 colspan=1>Cytochrome P450 2C9 and VKORC1</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Number of Locigenotyped</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Genotyping reactionlocation</td><td rowspan=1 colspan=1>Test cartridge</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Genotyping principle</td><td rowspan=1 colspan=1>Sandwich hybridization test</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>User interface</td><td rowspan=1 colspan=1>Graphical user interface with touchscreen</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Instrument operatingsystem</td><td rowspan=1 colspan=1>Random access compatible withmultiple simultaneous test types</td><td rowspan=1 colspan=1>Same as predicate</td></tr><tr><td rowspan=1 colspan=1>Assay results</td><td rowspan=1 colspan=1>Assay signal results are interpreted by asoftware program and are assigned agenotype that is presented to the end-user in a report format</td><td rowspan=1 colspan=1>Same as predicate</td></tr></table>

# Performance characteristics

Summary of Inter-laboratory and Inter-Operator Reproducibility Results   

<table><tr><td>Reproducibility: Site to Site, Operator to Operator, Lot to Lot, Day to Day and Run to Run reproducibility</td></tr><tr><td>A reproducibility study was performed at three sites, two external and one internal. Five genomic DNA samples covering all possible genotypes for all three alleles in the Warfarin Sensitivity Test were tested in duplicate runs on a daily basis by the same operator for 5 days at 3 different sites. One site performed the same reproducibility testing twice each day, using two different operators and the same testing materials. Three kit lots were randomized throughout the study. An</td></tr><tr><td>additional run using the same kit lot and sample as for the first-pass test were performed for tests that gave a no-call result. The data were evaluated after first-pass results and following the additional run for no-calls. All samples gave 100% agreement with DNA sequencing. There were 9 first-pass no-calls: one was</td></tr><tr><td>due to a cartridge manufacturing assembly error, and the remaining eight were due to operator error in set-up of the exonuclease reaction The following tables summarize the percent agreement between results obtained at each of the sites and DNA sequencing, before (first- pass) and after additional testing of no-calls (final):</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>Allele</td><td rowspan=1 colspan=1>Totalteests</td><td rowspan=1 colspan=1>First-passcorrectcalls</td><td rowspan=1 colspan=1>rst-passno-calls</td><td rowspan=1 colspan=1>Finalcorrectcalls</td><td rowspan=1 colspan=1>Fnalincorrect&quot;caloA</td><td rowspan=1 colspan=1> Agreement(95%. LCB)</td></tr><tr><td rowspan=6 colspan=1>1</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (94.2%)</td></tr><tr><td rowspan=3 colspan=1>All</td><td rowspan=3 colspan=1>All</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (98.5%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (98.5%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (98.5%)</td></tr></table>

# Summary of Reproducibility Results sorted by sample and genotype.

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>TotalTests</td><td rowspan=1 colspan=1>First-pascorrectcalls</td><td rowspan=1 colspan=1>IrstDa1ss no.callan</td><td rowspan=1 colspan=1>Finalcorrectcalls</td><td rowspan=1 colspan=1>FinalIncorrectcalls</td><td rowspan=1 colspan=1>% Agreement(95% LCB)</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>2C9 *1/*1 VKORC1 G/G</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>2C9 *2/*3 VKORC1 G/A</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>2C9 *2/*2 VKORC1G/G</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>2C9 *3/*3 VKORC1| G/G</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>2C9 *1/*3 VKORC1 A/A</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr></table>

Genomic DNA extraction reproducibility

Three different sites extracted 7 whole blood samples of different genotypes in triplicate and tested them using the eSensor® Warfarin Sensitivity Test. Each site used a different commercially available extraction method, which yielded DNA with A260/280 ratios of 1.7 to 3.3. First pass results were in $100 \%$ agreement with DNA sequencing, as shown in the following tables:

Summary of Inter-laboratory Extraction Reproducibility Results   

<table><tr><td rowspan=1 colspan=1>Siteate</td><td rowspan=1 colspan=1>Allela</td><td rowspan=1 colspan=1>TotalTestsaa</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>Incorrectcalls</td><td rowspan=1 colspan=1>No callss</td><td rowspan=1 colspan=1>% Agreement(93% LCB)E</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr><tr><td rowspan=1 colspan=1>VKORC1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (86.7%)</td></tr></table>

Summary of Extraction Reproducibility Results (sorted by sample and genotype).   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=2>Total   CorrectTests.d   Calls</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>Incorrectcalls.</td><td rowspan=1 colspan=1>No Calls</td></tr><tr><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>2C9 *1/*1 VKORC1 G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>02</td><td rowspan=1 colspan=1>2C9*1/*2 VKORC1G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>03</td><td rowspan=1 colspan=1>2C9*1/*3VKORC1G/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>04</td><td rowspan=1 colspan=1>2C9*1/*3VKORC1G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1>2C9*3/*3VKORC1G/G</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>06</td><td rowspan=1 colspan=1>2C9*2/*3 VKORC1A/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>07</td><td rowspan=1 colspan=1>2C9 *2/*3 VKORC1A/A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Method comparison to bi-directional DNA sequencing:

In a method comparison study, a total of 157 samples with A260/280 ratios of 1.2 to 2.3 were genotyped using the eSensor $\textcircled{8}$ Warfarin Sensitivity Test and DNA Sequencing. All first-pass sample results (157/157) obtained with the eSensor $\bigoplus$ Warfarin Sensitivity Test agreed with the results obtained by DNA sequencing. The $9 5 \%$ lower confidence bound on a per-sample basis was $9 8 . 1 \%$ , and $9 9 . 4 \%$ on a per-SNP basis (471/471). The following table summarizes the results of the method comparison study:

<table><tr><td rowspan=2 colspan=1>DNASequencingResult</td><td rowspan=1 colspan=1>@Sensor© Warfarin Sensitivity Test Result</td></tr><tr><td rowspan=1 colspan=1>2C91/1    2C9M7/M2        2C0*2/2.</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>36.8%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>87.3%</td><td rowspan=1 colspan=1>22.4%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>No-Calls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Miscalls</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>95.6%</td><td rowspan=1 colspan=1>95.4%</td><td rowspan=1 colspan=1>89.5%</td></tr></table>

<table><tr><td colspan="3"></td></tr><tr><td></td><td colspan="2"></td></tr><tr><td rowspan="2">Other characteristics of the eSensor®</td><td colspan="2"></td></tr><tr><td>Characteristic Limit of detection</td><td>Result Two genomic DNA samples of different genotypes were serially diluted to 1000, 100, 10, 1, and 0.1, nanograms and assayed 20 times each using the eSensor® Warfarin Sensitivity Test. An</td></tr><tr><td rowspan="4">Warfarin Sensitivity Test</td><td></td><td>additional test was performed for tests that gave a first pass no call result. All input amounts for both samples gave equivalent first-pass and final performance. The lower detection limit was determined to be O.1 ng of purified DNA per reaction and the upper detection limit was determined to be 1000 ng of purified DNA per reaction. The recommended range of DNA input amounts for the eSensor® Warfarin Sensitivity Test is from 10 to 1000 ng.. Test performance was not affected by addition of the following</td></tr><tr><td>Interfering substances to extraction:</td><td>substances to two whole blood samples of different genotypes prior Human serum albumin (3 g added/dL whole blood). Bilurubin (50 μg added/mL whole blood). Human immunoglobulin G (3 g added/dL whole blood). Triglycerides (3 g added/dL whole blood). Hemoglobin (20 g added as purified red blood cells/dL whole blood). Warfarin (32.5 µM added to whole blood).</td></tr><tr><td>Interfering mutations and polymorphisms</td><td>Heparin sodium (3,000 U/L added to whole blood). EDTA (at a concentration equivalent to 5-fold higher than that provided by a standard EDTA blood collection tube). Samples containing the following CYP450 2C9 polymorphisms have been tested and found to give accurate results in the eSensor® Warfarin Sensitivity Test: 1076T&gt;C(*4) 1080C&gt;G(*5)</td></tr><tr><td>374G&gt;A(*14) 485C&gt;A(*15) 895A&gt;G(*16).</td><td>818delA (*6) 1003C&gt;T(*11) In the VKORC1 gene, additional polymorphisms other than -1639G&gt;A, as well as rare mutations have been observed. These additional polymorphisms and mutations are not detected by the eSensor® Warfarin Sensitivity Test.</td></tr><tr><td rowspan="2">Kit stability</td><td colspan="2">eSensor® Warfarin Sensitivity Test kit components should be stored under the appropriate conditio the expiration date printed on the label: PCR Box containing Warfarin Sensitivity Test PCR Mix and Taq Polymerase: Store at -20°C in a designated pre-PCR area.</td></tr><tr><td colspan="2">Cartridges: Store at 10° to 25° •Genotyping Box containing Exonuclease, Warfarin Sensitivity Test Signal Buffer, XT-Buffer 1 an Buffer 2: Store at -20°C in a designated post-PCR area. In-process stability has been established for the following components, working reagents and samp Cartridges can be stored for up to 14 days after opening the foil pouches. If stored, cartridges sh kept in their original foil pouch at room temperature in a dry place with the zip-loc closure sealed Once open, reagents can be stored at -20°C for up to 30 days. Reagents can be thawed up to 3 times. Whole blood stored in EDTA can be stored for up to 4 weeks after collection prior to extraction o for use in the eSensor® Warfarin Sensitivity Test. PCR product can be stored at 4°C or -20°C for up to 7 days.</td></tr><tr><td></td><td>Exonuclease-digested PCR product can be stored store at 4°C or -20°C for up to 7 days. After combining the exonuclease-digested PCR with hybridization reagents, the hybridization reaction can be loaded on the cartridge and held at ambient temperature for up to 8 hours before initiating hybridization of the cartridge on the XT-8 instrument.</td></tr><tr><td>Conclusion</td><td>Theabove internal and clinical test results support the safety and effectiveness of theeSensor® Warfar Sensitivity Test and the eSensor® XT-8 System, and demonstrate substantial equivalence to the predicate device.</td></tr></table>

eSenso $r \textsuperscript { \textregistered }$ is a registered trademark of Osmetech and its subsidiaries.   
Verigene $\otimes$ is a registered trademark of Nanosphere, Inc.   
Osmctech Molecular Diagnostics   
c/o Mr. Robert Dicheck   
Vice President of Quality & Regularoty Affairs   
757 South Raymond Avenue   
Pasadena, CA 91105

# JUL 1 7 2008

Re: k073720 Trade Name: eSensor $^ \mathrm { \textregistered }$ Warfarin Sensitivity Test, eSensor $\textsuperscript { \textregistered }$ XT-8 System Regulation Number: 21 CFR 862.3360 Regulation Name: Drug Metabolism Enzyme Genotyping Test Regulatory Class: Class II Product Codes: ODW, ODV NSU Dated: May 22, 2008 Received: May 27, 2008

Dear Mr. Dicheck:

We have reviewed your Section $5 1 0 ( \mathbf { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into either class ⅡI (Special Controls) or class II (PMA), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820).

This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA fnding f substantial equivalence of your device to a legally imarketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific information about the application of labeling requirements to your device, or questions on the promotion and advertising of your device, please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (240) 276-0490. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800).638-2041 er (240) 276-3150-er at its Internct address at   
http://www.fda.gov/cdrh/industry/support/index.htrml.

# Indications for Use

510(k) Number (if known): K073720

Device Name: eSensor $\mathfrak { Q }$ Warfarin Sensitivity Test and XT-8 System

Indications For Use:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Test is an in vitro diagnostic for the detection and genotyping of the \*2 and \* 3 alleles of the cytochrome P450 (CYP450) 2C9 gene locus and the Vitamin K epoxide reductase C1 (VKORC1) gene promoter polymorphism (-1639G>A) from genomic DNA extracted from whole blood samples preserved with EDTA, as an aid in the identification of patients at risk for increased warfarin sensitivity.

The eSensor $\mathfrak { B }$ XT-8 Instrument is an in vitro diagnostic device intended for genotyping multiple mutations or polymorphisms in an amplified DNA sample utilizing electrochemical detection technology.

Concurrence of CDRH, Office of In Vitro Diagnostic Device Evaluation and Safety (OIVD)

![](images/8ee0130f5a06acf577fcc764cba65511ab0a29daf87dde04ac0a664e3fd81909.jpg)

Qision Sign-Off   
fice of In Vitro Diagnostic Device   
Evaluation and Safety