# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k092967   
B. Purpose for Submission: Modified device   
C. Measurand: Gene expression profile for 15 common tumor types   
D. Type of Test: Gene expression microarray   
E. Applicant: Pathwork Diagnostics Inc.   
F. Proprietary and Established Names: Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test Kit - FFPE

# G. Regulatory Information:

1. Regulation section: 21 CFR $\$ 862.3100$ Amphetamine Test System   
2. Classification: Class II   
3. Product code: OIW, Software, similarity score algorithm, tissue of origin for malignant tumor types   
4. Panel: Toxicology (91)

# H. Intended Use:

1. Intended use(s): The Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test is an in vitro diagnostic intended to measure the degree of similarity between the RNA expression patterns in a patient’s formalin-fixed, paraffin-embedded (FFPE) tumor and the RNA expression patterns in a database of fifteen tumor types (poorly differentiated, undifferentiated and metastatic cases) that were diagnosed according to then current clinical and pathological practice. This test should be evaluated by a qualified physician in the context of the patient’s clinical history and other diagnostic test results.

Limitations: The Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test is not intended to establish the origin of tumors (e.g. cancer of unknown primary) that cannot be diagnosed according to current clinical and pathological practice. It is not intended to subclassify or modify the classification of tumors that can be diagnosed by current clinical and pathological practice, nor to predict disease course or survival or treatment efficacy, nor to distinguish primary from metastatic tumor. Tumor types not in the Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test database may have RNA expression patterns that are similar to patterns in the database. Therefore, results cannot be used to distinguish tumor types in the database from tumor types not in the database.

2. Indication(s) for use: Same as Intended use

3. Special conditions for use statement(s): For prescription use only   
4. Special instrument requirements: Affymetrix GeneChip $^ \mathrm { \textregistered }$ Microarray Instrumentation System (k080995)

# I. Device Description:

The Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test Kit-FFPE (Formalin-Fixed Paraffin-Embedded) is a modification of the Tissue of Origin Test kit (k080896) which was FDA cleared on 7/30/2008. Differences between the modified product and the original test system are in the specimen assessed, the method of preparation of the specimen for hybridization to the chip, the algorithm employed to convert the expression data to a diagnostics report, and the guide to report interpretation.

The Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test is a test kit consists of the Pathchip microarray, reagents, software, Pathwork Specimen Processing Guide and Guide to Report Interpretation (GRI). The Pathchip microarray employed in Origin Test Kit-FFPE test is the same one as employed in the predicate Pathwork Tissue of Origin Test (k080896). As with the original test kit cleared by $\mathrm { k 0 8 0 8 9 6 }$ , the specimen processing up to chip hybridization will be done at the clinical laboratory.

Pathwork Diagnostics, in its labeling for the Origin Test Kit-FFPE test, recommends Pathwork reagents for RNA extraction, in vitro transcription (IVT), biotinylated cDNA synthesis, and array preparation. These reagents are manufactured for Pathwork and are controlled under Pathwork’s Quality System.

The Pathwork FFPE Specimen Processing Guide (SPG) provides instructions to enable the user to process FFPE tissue specimens.

Software for the Origin Test Kit-FFPE product is composed of transmission software and analytical software. Transmission software is provided to the laboratory processing the specimen on the Pathchip. Analytic software is not provided to the laboratory, but remains at Pathwork. The transmission software allows the user to upload the CEL file to Pathwork, where it is analyzed, and allows reverse transmission of a report from Pathwork to the user laboratory. This software is unchanged from k080896. The Pathwork analytical software converts the CEL file to gene expression measurements, performs data verification,

standardizes the data to correct for lab-to-lab variation, converts the gene expression values to Similarity Scores, stores the results and produces a report (pdf file) summarizing these results. This software employs an algorithm different from the one employed for frozen tissue and reported in k080896.

The algorithm was built upon the earlier algorithm for the predicate device, and was adapted for use with FFPE specimens. The test development used a machine learning approach based on marker selection and a support vector machine (SVM) algorithm to build a predictive model. The model consists of a list of markers, a set of reference (support) samples, and a set of coefficients. These components are combined to produce 15 probabilities (Similarity Scores), each corresponding to likelihood that the input specimen has a molecular signature of the corresponding tissue of origin.

As with the predicate, report presents, in a graphical format, 15 computed Similarity Scores (SS), one for each tissue on the test panel. The 15 tumor types evaluated are bladder (BL), breast (BR), colorectal (CO), gastric (GA), testicular germ cell (GC), hepatocellular (LI), kidney (KI), non-small cell lung (LU), non-Hodgkin’s lymphoma (LY), melanoma (ME), ovarian (OV), pancreatic (PA), prostate (PR), sarcoma (SC), and thyroid (TH).

# J. Substantial Equivalence Information:

1. Predicate device name(s): Pathwork $^ \mathrm { \textregistered }$ Tissue of Origin Test   
2. Predicate 510(k) number(s): k080896   
3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>New Device Tissue of Origin Test kit  FFPE k092967</td><td>Predicate Device Tissue of Origin Test k080896</td></tr><tr><td>Intended use</td><td>The Pathwork® Tissue of Origin Test is an in vitro diagnostic intended to measure the degree of similarity between the RNA expression patterns in a patient's tumor and the RNA expression patterns in a database of fifteen tumor types (poorly differentiated, undifferentiated and metastatic cases) that were diagnosed according to then current clinical and pathological practice. This test should be evaluated by a qualified physician in the context of the patient's clinical history and other diagnostic test results The Pathwork® Tissue of Origin Test is</td><td>Same</td></tr><tr><td>Limitations in the Intended use</td><td>not intended to establish the origin of tumors (e.g. cancer of unknown primary) that cannot be diagnosed according to current clinical and pathological practice. It is not intended to subclassify or modify the classification of tumors that can be diagnosed by current clinical and pathological practice, nor to predict disease course or survival or treatment efficacy, nor to distinguish primary from metastatic tumor. Tumor types not in the Pathwork® Tissue of Origin Test database may have RNA expression patterns that are similar to patterns in the database. Therefore, results cannot be used to distinguish tumor types in the database from tumor types not in the</td><td>Same</td></tr><tr><td>Device Description</td><td>database. Pathchip microarray, reagents, software,</td><td>Same</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New DeviceTissue of Origin Test kit  FFPEk092967</td><td colspan="1" rowspan="1">Predicate DeviceTissue of Origin Testk080896</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pathwork Specimen Processing Guideand Guide to Report Interpretation(GRI)</td><td colspan="1" rowspan="1"></td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>New DeviceTissue of Origin Test kit - FFPE</td><td rowspan=1 colspan=1>Predicate DeviceTissue of Origin Test</td></tr><tr><td rowspan=1 colspan=1>Test Sample</td><td rowspan=1 colspan=1>FFPE tumor samples</td><td rowspan=1 colspan=1>Frozen biopsy tissues</td></tr><tr><td rowspan=1 colspan=1>Pathwork SoftwareSystem (PSS; AnalysisSoftware)</td><td rowspan=1 colspan=1>Pathwork Systems S/W 4.0Algorithm Version 42.4</td><td rowspan=1 colspan=1>Pathwork Systems S/W2.1Algorithm Version 21.2</td></tr><tr><td rowspan=1 colspan=1>Report</td><td rowspan=1 colspan=1>Equivalent graphic presentation withmodified Guide to Report Interpretation</td><td rowspan=1 colspan=1>Graphic presentation ofrelative probabilities for15 tissues of origin withGuide to ReportInterpretation</td></tr><tr><td rowspan=1 colspan=1>Extraction/ amplificationreagents/ amplificationprocedure</td><td rowspan=1 colspan=1>Procedure for FFPE tumor samples</td><td rowspan=1 colspan=1>Procedure for frozenbiopsy tumor samples</td></tr><tr><td rowspan=1 colspan=1>Analyte detected on chip</td><td rowspan=1 colspan=1>Labeled cDNA</td><td rowspan=1 colspan=1>Labeled cRNA</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable.

# L. Test Principle:

The specimen used for this assay is a formalin-fixed paraffin-embedded (FFPE) tissue block. The test uses curls from a formaldehyde-fixed, paraffin-embedded (FFPE) block. The Origin Test Kit-FFPE test can be performed on a single $1 0 \mathrm { - } \mu \mathrm { m }$ -thick curl cut from the block, or on up to five curls. It is not recommended to assess more than five curls per microarray.

Routine histological sectioning and H&E staining should be performed to determine percent tumor (includes tumor, hyper- and hypocellular stroma). The specimen must contain at least $60 \%$ viable tumor. The tissue must be de-paraffinized and total RNA must be isolated per the Pathwork Specimen Processing Guide (SPG). Thirty (30) nanograms of total RNA are required to perform the test.

The procedural steps after isolation of total RNA from the specimen begin with a two-cycle RNA amplification: 1) round 1 reverse transcription (RT) of total RNA with dT and random RT primers; 2) purification of round 1 cDNA; 3) tailing of first-strand cDNA with terminal deoxynucleotidyl transferase; 4) synthesis of T7/T3 promoter sequence; 5) T7 in vitro transcription 6) round 2 RT of sense RNA with a dT and random primers; 7) T3 in vitro transcription (linear amplification); 8) purification of round 2 sense RNA; 9) RT of round 2 sense RNA with random primers; and 10) purification and fragmentation of biotinylated DNA. This is followed by hybridization of the biotin-labeled cDNA target to the Pathchip microarray; washing and scanning of the hybridized Pathchip microarray; data acquisition (signal intensity per feature; CEL file), data verification, standardization of the signal

intensities; determination of similarity to 15 tissues of origin and generation of the Report. The CEL file from each laboratory is transported from the Affymetrix Workstation with GeneChip $^ \mathrm { \textregistered }$ Operating System (GCOSDx) using the Pathwork File Transfer Software (PFTS) through a secure FTP transfer protocol to Pathwork Diagnostics for analysis. The Pathwork System Software (PSS) then converts the CEL file to gene expression measurements, performs data verification, normalizes (standardizes) the data to correct for technical sources of variation, performs a series of multiplex statistical tests, and produces a report summarizing these results. Each specimen analyzed will produce 15 Similarity Scores, one for each tissue on the panel. Each Similarity Score is a measure of the similarity of the gene expression profile of the specimen to the profile of the indicated tissue, ranging from 0 (very low similarity) to 100 (very high similarity). Similarity Scores for all 15 tissues sum to 100. For each Origin Test Kit–FFPE test performed, a test report is generated that quantifies the similarity of the RNA expression pattern found in a tumor specimen (poorly or un-differentiated primary tumors, as well as metastatic tumors) to expression patterns found in tumor specimens from 15 known tissues of origin and provided back to the laboratory over a secure internet connection in pdf format.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: Inter-laboratory reproducibility study: Replicate samples from 60 individual tumors were distributed among the three laboratories for a total of 177 tissue samples (Site 1 did not receive two specimens and Site 3 did not receive one specimen due to insufficient stocks at the time specimen processing began). Of the 60 specimens, 27 $( 4 5 \% )$ were metastases and 33 $( 5 5 \% )$ were primaries classified as poorly differentiated to undifferentiated. This study required each laboratory to perform the full protocol from FFPE curls. Each of the 15 tissues in the Origin Test Kit-FFPE test panel was represented by four specimens. Each curl was processed at three different laboratories. The methods for RNA extraction amplification and purification, target preparation, cDNA labeling and hybridization, array washing, staining and scanning were according to the procedure in the Pathwork Specimen Processing Guide (SPG) for Formalin-Fixed, ParaffinEmbedded Tissues. Array CEL files were transferred to Pathwork for analysis with Pathwork System Software. Pathwork’s predetermined acceptance criteria were that (i) at least $8 5 \%$ concordance must be observed between the tissue of origin predictions when an overall pairwise comparison is made and (ii) that there is no evidence for differences across sites.

Table 1:   

<table><tr><td rowspan=1 colspan=6>Origin-FFPE Reproducibility Study  Regression AnalysisStandardized Expression (SE)</td></tr><tr><td rowspan=2 colspan=1>Comparison(Y vs. X)</td><td rowspan=2 colspan=1>Specimens(n)</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Intercept</td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>r</td></tr><tr><td rowspan=1 colspan=1>Points</td><td rowspan=1 colspan=1>[95% CI]</td><td rowspan=1 colspan=1>[95% CI]</td><td rowspan=1 colspan=1>[95% CI]</td></tr><tr><td rowspan=1 colspan=1>Sites 2 vs. 1</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>99421</td><td rowspan=1 colspan=1>0.02[0.02, 0.03]</td><td rowspan=1 colspan=1>0.63[0.63, 0.64]</td><td rowspan=1 colspan=1>0.59[0.59, 0.60]</td></tr><tr><td rowspan=1 colspan=1>Sites 3 vs.1</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>99421</td><td rowspan=1 colspan=1>0.03[0.03, 0.03]</td><td rowspan=1 colspan=1>0.67[0.67, 0.68]</td><td rowspan=1 colspan=1>0.68[0.68, 0.68]</td></tr><tr><td rowspan=1 colspan=1>Sites 3 vs. 2</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>103479</td><td rowspan=1 colspan=1>0.01[0.01, 0.02]</td><td rowspan=1 colspan=1>0.69,[0.69, 0.70]</td><td rowspan=1 colspan=1>0.63[0.63, 0.63]</td></tr></table>

Pathwork has determined that Origin-FFPE SE regressions demonstrate more scatter than Origin-Frozen SE regressions. This is ostensibly due to the lower signal that is produced on Origin-FFPE microarrays. This lower signal is, in turn, due to the more challenging nature of gene expression profiling of FFPE total RNA. However, Similarity Score (SS) is considered as the primary analytical output of the test.

Similarity Score

Matched pairs of specimens which were successfully processed at both laboratories were evaluated using linear regression and correlation analysis to evaluate the reproducibility of Similarity Score values. Across all three laboratories, three pairwise comparisons showed slopes of 0.93 to 0.97 and $\tilde { \mathbf { r } } ^ { \prime \prime }$ of 0.92 to 0.93.

Table 2:   

<table><tr><td rowspan=1 colspan=6>Origin-FFPE Reproducibility Study  Regression AnalysisSimilarity Score (SS)</td></tr><tr><td rowspan=2 colspan=1>Comparison(Y vs. X)</td><td rowspan=2 colspan=1>Specimens(n)</td><td rowspan=2 colspan=1>TotalPoints</td><td rowspan=1 colspan=1>Intercept</td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>r</td></tr><tr><td rowspan=1 colspan=1>[95% CI]</td><td rowspan=1 colspan=1>[95% CI]</td><td rowspan=1 colspan=1>[95% CI]</td></tr><tr><td rowspan=1 colspan=1>Sites 2 vs. 1</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>0.26[0.22, 0.73]</td><td rowspan=1 colspan=1>0.96[0.93, 0.99]</td><td rowspan=1 colspan=1>0.93[0.92, 0.94]</td></tr><tr><td rowspan=1 colspan=1>Sites 3 vs.1</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>0.19[0.28, 0.67]</td><td rowspan=1 colspan=1>0.97[0.94, 1.00]</td><td rowspan=1 colspan=1>0.93[0.92, 0.94]</td></tr><tr><td rowspan=1 colspan=1>Sites 3 vs. 2</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>765</td><td rowspan=1 colspan=1>0.49[0.00, 0.99]</td><td rowspan=1 colspan=1>0.93,[0.90, 0.95]</td><td rowspan=1 colspan=1>0.92[0.91, 0.93]</td></tr></table>

In the reproducibility study, each specimen was tested at the three sites and for each specimen, an average of the similarity scores associated with the actual tissue of origin (Available Diagnosis) was calculated, along with standard deviation (SD) and percent coefficient of variation $( \% \mathrm { C V } )$ .

Table 3:   

<table><tr><td rowspan=1 colspan=5>Reproducibility of Similarity Scores</td></tr><tr><td rowspan=1 colspan=1>AverageSimilarity Score(Available Diagnosis)</td><td rowspan=1 colspan=1>Specimens(n)</td><td rowspan=1 colspan=1>Replicates*</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>0 to 20</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>54.5</td></tr><tr><td rowspan=1 colspan=1>20 to 40</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>25.8</td></tr><tr><td rowspan=1 colspan=1>40 to 60</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>16.9</td><td rowspan=1 colspan=1>36.2</td></tr><tr><td rowspan=1 colspan=1>60 to 80</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>18.5</td></tr><tr><td rowspan=1 colspan=1>80 to 100</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>7.8</td></tr></table>

\*Each specimen was tested as $\mathfrak { n } = 3$ (Sites 1-3). Missing replicates are due to depleted aliquots, insufficient yields, or data verification errors (low overall signal).

To assess the accuracy of the results reported by each laboratory, the Origin Test Kit– FFPE results were compared to available diagnosis established for each specimen. Each laboratory was expected to achieve the overall acceptance criteria of at least $80 \%$ agreement with the available diagnosis, with no more than $20 \%$ nonagreement.

Table 4:   

<table><tr><td rowspan=1 colspan=6>Tissue of Origin Test Results</td></tr><tr><td rowspan=3 colspan=1>Stratificationby Lab</td><td rowspan=3 colspan=1>#Specimens</td><td rowspan=1 colspan=2>% Agreement</td><td rowspan=1 colspan=2>% Non- Agreement</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>ratio</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>ratio</td></tr><tr><td rowspan=1 colspan=2>[95% CI]</td><td rowspan=1 colspan=2>[95% CI]</td></tr><tr><td rowspan=2 colspan=1>Site 1</td><td rowspan=2 colspan=1>52</td><td rowspan=1 colspan=1>82.7%</td><td rowspan=1 colspan=1>(43/52)</td><td rowspan=1 colspan=1>17.3%</td><td rowspan=1 colspan=1>(9/52)</td></tr><tr><td rowspan=1 colspan=2>[69.7%, 91..8%</td><td rowspan=1 colspan=2>[8.2%, 30.3%]</td></tr><tr><td rowspan=2 colspan=1>Site 2</td><td rowspan=2 colspan=1>55</td><td rowspan=1 colspan=1>81.8%</td><td rowspan=1 colspan=1>(45/55)</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>(10/55)</td></tr><tr><td rowspan=1 colspan=2>[69.1%, 90.9%]</td><td rowspan=1 colspan=2>[9.1%, 30.9%]</td></tr><tr><td rowspan=2 colspan=1>Site 3</td><td rowspan=2 colspan=1>55</td><td rowspan=1 colspan=1>81.8%</td><td rowspan=1 colspan=1>(45/55)</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>(10/55)</td></tr><tr><td rowspan=1 colspan=2>[69.1%, 90.9%]</td><td rowspan=1 colspan=2>[9.1%, 30.9%]</td></tr><tr><td rowspan=2 colspan=1>All Sites(cumulative)</td><td rowspan=2 colspan=1>162</td><td rowspan=1 colspan=1>82.1%</td><td rowspan=1 colspan=1>(133/162)</td><td rowspan=1 colspan=1>17.9%</td><td rowspan=1 colspan=1>(29/162)</td></tr><tr><td rowspan=1 colspan=2>[74.0%, 86.6%]</td><td rowspan=1 colspan=2>[13.4%, 26.0%]</td></tr></table>

b. Linearity/assay reportable range: Linearity is not applicable for this range of assay.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Quality Control A suitable total RNA sample should be run with every batch of specimens to serve a a positive/negative control as described in the Pathwork Specimen Processing Guid Specimen Processing Controls:   
The following quality control checks are required during specimen processing to assure reliable results:   
1) Amount of total RNA extracted from tissue specimens: minimum of $3 0 \mathrm { n g }$ .

2) Amount of fragmented, labeled cDNA used for hybridization to Pathchip microarrays: $3 . 0 \left( \pm 0 . 5 \right) \mu \mathrm { g }$ .

To ensure that a sufficient quantity of high quality labeled cDNA is obtained for hybridization to Pathchip microarrays, it is recommended that:

1) All procedures in the Pathwork Specimen Processing Guide are followed efficiently to reduce RNase degradation.   
2) FFPE block size is sufficient to take a $5 \mu \mathrm { m }$ section for H&E staining and a $1 0 \mu \mathrm { m }$ $\pm 1 \mu \mathrm { m }$ section (curl) with a minimum tissue area of at least $2 5 \mathrm { m m } ^ { 2 }$ or equivalent for the test.   
3) Specimen contain at least $60 \%$ viable tumor.   
4) Absorbance measurements of total RNA give an A260/A280 ratio of greater than 1.0 (typically 1.0 to 2.2).   
5) The minimum amount of labeled cDNA used for fragmentation is $2 . 5 \mu \mathrm { g }$ .

Endogenous Pathchip Controls:

In addition to these external controls, the Origin Test Kit–FFPE test utilizes endogenous mRNA markers that are found in human tissue specimens and are captured on the Pathchip microarray to perform a series of data verifications that detect laboratory processing anomalies found to unfavorably influence the accuracy of the Origin Test Kit – FFPE test. These data verifications include the detection of regional discontinuities and low overall signal. When a submitted file fails one or more of the data verifications, the system software will return an Origin Test Kit – FFPE Report flagged as “Unacceptable”, with caution messages appropriate for the specific type of failure that occurred.

Device stability:

Storage conditions for the Origin Test Kit - FFPE reagents range from ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ to ambient temperature.

# d. Detection limit:

Dilution study – Labeled cDNA:

A dilution study was performed to demonstrate that the Origin Test Kit – FFPE test is robust against reasonable variations in the amount of labeled cDNA used in hybridization. The study used three different FFPE specimens. RNA was extracted and processed to the point of labeled cDNA. For each specimen, Pathchip arrays were hybridized in triplicate with each of the five dilutions (1, 2, 4, 6 and $7 . 5 \mu \mathrm { g } )$ of labeled cDNA. Subsequent processing of all 3 specimens x 5 dilutions x triplicates $( =$ 45 arrays) was performed per the recommended protocol. One Pathchip array failed the percent present requirement, thus 44 arrays were used for further analysis. The data was analyzed using linear regression and correlation analysis to assess equivalence. Based on the data, the Pathwork® Tissue of Origin Test Kit–FFPE User Guide specifies $3 . 0 \pm 0 . 5 ~ \mu \mathrm { g }$ labeled cDNA.

# e. Analytical specificity:

Several potential interfering substances in tumor biopsies were evaluated for potential

adverse effects.

RNases: Performance of the test using specimens derived from biopsies of the pancreas, a tissue known to contain high levels of endogenous ribonuclease (RNase), was equivalent to performance of the test for specimens derived from other tissue types. Nonetheless, Pathwork recommends that care be taken to limit specimen exposure to RNases during storage, extraction and processing of specimens.

Necrotic tissue: To examine whether necrotic tissue produce adverse effects in use of the Origin Test Kit-FFPE, the results from the FFPE specimens used in the clinical validation study were stratified by percent necrosis. Based on the $\%$ agreement, the Origin Test Kit-FFPE demonstrates adequate performance with up to $40 \%$ necrotic tissue in the sample.

Adipose Tissue: To examine whether the adipose tissue in breast tumors is a potential interferrent and produce adverse effects in the Tissue of Origin test, the results from 58 breast-related specimens included in the clinical validation were compared to its available clinical diagnosis. Available clinical diagnoses from Breast-related specimens were in $9 6 . 6 \%$ agreement with PGC, compared to all specimens which had $8 8 . 5 \%$ agreement.

Fibrous material: To examine whether fibrous material in skin-related specimens, produce adverse effects in the Tissue of Origin Test, the results from 27 skin-related specimens included in the clinical validation were compared to its available clinical diagnosis. Available clinical diagnoses from skin-related specimens were in $8 5 . 2 \%$ agreement with PGC, compared to all specimens which had $8 8 . 5 \%$ agreement. Tissue of Origin Test does not demonstrate apparent adverse effects with this interfering substance.

$f .$ Assay cut-off:

The Similarity Score (SS) is a measure of the similarity of the RNA expression pattern of the specimen to the RNA expression pattern of the indicated tissue. Similarity Scores range from 0 (very low similarity) to 100 (very high similarity) and sum to 100 across all 15 tissues on the panel.

The highest Similarity Score indicates the likely tissue of origin. A Similarity Score less than or equal to 5 rules out that tissue type as the likely tissue of origin.   
• Performance has not been established for results where the highest Similarity Score is less than 20.

The TOO Test Kit - FFPE does not produce an indeterminate result. The highest Similarity Score (SS) indicates the likely tissue of origin, unless it is $< 2 0$ . When the highest SS is $< 2 0$ , the accuracy or reproducibility of the top score is not assured, hence the statement in the Guide to Report Interpretation that “Performance has not been established for results where the highest Similarity Score is less than 20.”

In the predicate device (TOO Test with frozen specimens), the entire assay was indeterminate if there were no $\mathrm { S } \mathrm { S } > 3 0$ . This is not the case with the TOO Test Kit – FFPE, which uses different reagents and a different algorithm than the predicate device.

# 2. Comparison studies:

a. Method comparison with available diagnosis:

The clinical validation studies were performed to assess the predictive capability of the Pathwork Origin-FFPE test in determining the tissue of origin of poorly differentiated, undifferentiated and metastatic FFPE tumor specimens with tissue of origin within the 15-tissue panel. The study was performed at three different processing sites. For each specimen, an H&E slide was received at Pathwork and a Pathwork pathologist examined it for $\%$ tumor before accepting the specimen into final analysis. Only specimens with ${ > } 6 0 \%$ tumor were analyzed. The sites received the curls, processed the specimens per the Specimen processing Guide, and sent the scanned CEL files to Pathwork for analysis. Primary end-point was overall $\%$ agreement of Test result with available diagnosis. Each laboratory was expected to achieve the overall acceptance criteria of at least $80 \%$ agreement with the available diagnosis, with no more than $20 \%$ non-agreement.

The acceptance criterion for the primary endpoint specified that the lower bound of the $9 5 \%$ confidence interval of the overall percent agreement with available diagnosis shall be $2 7 5 \%$ ).

The study included 25 to 57 specimens per tissue on the panel. Of the total 598 tumor specimens processed in this study, 462 specimens met the labeling limitations for tumor grade and available diagnosis and passed the data verification quality tests. To assess the accuracy of the results reported by each laboratory, the Origin Test Kit– FFPE results were compared to available diagnosis established for each specimen. For individual tumors, average $\%$ agreement ranged from $72 \%$ (gastric) to $9 6 . 5 \%$ (breast). Overall agreement with Available Diagnosis was $8 8 . 5 \%$ , with the lower and upper bounds of the $9 5 \%$ CI as 85.3 and 91.3 respectively. Any tissue type with a Similarity Score less than or equal to 5 had a $9 9 . 8 \%$ , $9 5 \%$ CI [99.7, 99.9] probability of not being the tissue of origin.

The detailed results are presented in the following Tables.

Positive Percent Agreement (PPA) $- 1 0 0 ^ { * } \mathrm { T P } / \mathrm { P O S } .$ , where TP is the number of test results that match the available diagnoses for the given tissue of origin and POS is the total number of positive specimens as per available diagnosis for the given tissue of origin.

Negative Percent Agreement $\mathbf { ( N P A ) } - 1 0 0 ^ { * } ( 1 \mathbf { - } \mathbf { ( F P / N E G ) } )$ , where FP is the number of test results that are false positive (as per the available diagnoses) for the given tissue of origin and NEG is the number of negative specimens as per the available diagnosis for the given tissue of origin.

Non-Agreement $( \% ) -$ the percent of POS specimens in which the Pathwork test result does not

agree with the available diagnosis.

Table 5: Tissue of Origin –FFPE Test Accuracy Performance - Agreement with   

<table><tr><td rowspan=1 colspan=7>Available Diagnosis</td></tr><tr><td rowspan=3 colspan=1>AvailableDiagnosis</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Non-Agreement</td><td rowspan=1 colspan=1>NegativePercentAgreement</td><td rowspan=3 colspan=1>Area UnderROCcurve</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>ratio</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>ratio</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[95% CI]</td><td rowspan=1 colspan=1>[95% CI]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[95% CI]</td></tr><tr><td rowspan=2 colspan=1>Bladder</td><td rowspan=1 colspan=1>79.3%</td><td rowspan=1 colspan=1>23/29</td><td rowspan=1 colspan=1>20.7</td><td rowspan=1 colspan=1>6/29</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>0.992</td></tr><tr><td rowspan=1 colspan=2>[60.3,92.0]</td><td rowspan=1 colspan=2>[8.0, 39.7]</td><td rowspan=1 colspan=1>[ 99.2, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Breast</td><td rowspan=1 colspan=2>96.5%    55/57</td><td rowspan=1 colspan=2>3.5     2/57</td><td rowspan=1 colspan=1>99</td><td rowspan=2 colspan=1>0.998</td></tr><tr><td rowspan=1 colspan=2>[87.9,99.6]</td><td rowspan=1 colspan=2>[0.4, 12.1]</td><td rowspan=1 colspan=1>[97.5, 99.7]</td></tr><tr><td rowspan=2 colspan=1>Colorectal</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>33/36</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>3/36</td><td rowspan=1 colspan=1>96.9</td><td rowspan=2 colspan=1>0.989</td></tr><tr><td rowspan=1 colspan=1>[77.5,</td><td rowspan=1 colspan=1>98.2]</td><td rowspan=1 colspan=1>[1.8, 22.5]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[94.8, 98.4]</td></tr><tr><td rowspan=2 colspan=1>Gastric</td><td rowspan=1 colspan=1>72.0</td><td rowspan=1 colspan=1>18/25</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>7/25</td><td rowspan=1 colspan=1>98.6</td><td rowspan=2 colspan=1>0.978</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[50.6, 87.9]</td><td rowspan=1 colspan=1>[12.1,</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[97.0, 99.5]</td></tr><tr><td rowspan=2 colspan=1>Hepatocellular</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>1/25</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>[79.6,</td><td rowspan=1 colspan=1>99.9]</td><td rowspan=1 colspan=1>[0.1, 20.4]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[ 99.2, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Kidney</td><td rowspan=1 colspan=1>89.3</td><td rowspan=1 colspan=1>25/28</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>3/28</td><td rowspan=1 colspan=1>99.5</td><td rowspan=2 colspan=1>0.996</td></tr><tr><td rowspan=1 colspan=1>[71.8,</td><td rowspan=1 colspan=1>97.7]</td><td rowspan=1 colspan=1>[0.3, 28.2]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[98.3, 99.9]</td></tr><tr><td rowspan=2 colspan=1>Melanoma</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>21/25</td><td rowspan=1 colspan=1>16.0</td><td rowspan=1 colspan=1>4/25</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>[63.9,</td><td rowspan=1 colspan=1>95.5]</td><td rowspan=1 colspan=1>[0.5, 36.1]</td><td rowspan=1 colspan=1>[0.5, 36.1]</td><td rowspan=1 colspan=1>[ 99.2, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Non-Hodgkin&#x27;sLymphoma</td><td rowspan=1 colspan=1>89.7</td><td rowspan=1 colspan=1>26/29</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1>3/29</td><td rowspan=1 colspan=1>99.5</td><td rowspan=2 colspan=1>0.997</td></tr><tr><td rowspan=1 colspan=2>[72.6, 97.8]</td><td rowspan=1 colspan=2>[2.2, 27.4]</td><td rowspan=1 colspan=1>[ 98.3, 99.9]</td></tr><tr><td rowspan=2 colspan=1>Non-small CellLung</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>23/27</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>4/27</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>0.998</td></tr><tr><td rowspan=1 colspan=2>[66.3,95.8]</td><td rowspan=1 colspan=2>[4.2, 33.7]</td><td rowspan=1 colspan=1>[ 99.2, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Ovarian</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>40/45</td><td rowspan=1 colspan=1>11.1</td><td rowspan=1 colspan=1>5/45</td><td rowspan=1 colspan=1>98.8</td><td rowspan=2 colspan=1>0.97</td></tr><tr><td rowspan=1 colspan=1>[75.9,</td><td rowspan=1 colspan=1>96.3]</td><td rowspan=1 colspan=1>[3.7, 24.1]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[97.2, 99.6]</td></tr><tr><td rowspan=2 colspan=1>Pancreas</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>24/28</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>4/28</td><td rowspan=1 colspan=1>98.8</td><td rowspan=2 colspan=1>0.993</td></tr><tr><td rowspan=1 colspan=1>[67.3,</td><td rowspan=1 colspan=1>96.0]</td><td rowspan=1 colspan=1>[4.0, 32.7]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[97.3, 99.6]</td></tr><tr><td rowspan=2 colspan=1>Prostate</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>1/25</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>0.997</td></tr><tr><td rowspan=1 colspan=1>[79.6,</td><td rowspan=1 colspan=1>99.9]</td><td rowspan=1 colspan=1>[0.1, 20.4]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[99.2, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Sarcoma</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>11.1</td><td rowspan=1 colspan=1>3/27</td><td rowspan=1 colspan=1>96.3</td><td rowspan=2 colspan=1>0.967</td></tr><tr><td rowspan=1 colspan=1>[70.8,</td><td rowspan=1 colspan=1>97.6]</td><td rowspan=1 colspan=1>[2.4, 29.2]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[94.1, 97.9]</td></tr><tr><td rowspan=2 colspan=1>Testicular GermCell</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>21/25</td><td rowspan=1 colspan=1>16.0</td><td rowspan=1 colspan=1>4/25</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>0.998</td></tr><tr><td rowspan=1 colspan=2>[63.9,95.5]</td><td rowspan=1 colspan=2>[4.5, 36.1]</td><td rowspan=1 colspan=1>[ 99.2, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Thyroid</td><td rowspan=1 colspan=1>90.3</td><td rowspan=1 colspan=1>28/31</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>3/31</td><td rowspan=1 colspan=1>100</td><td rowspan=2 colspan=1>0.98</td></tr><tr><td rowspan=1 colspan=1>[74.2,</td><td rowspan=1 colspan=1>98.0]</td><td rowspan=1 colspan=1>[2.0, 25.8</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[99.1, 100.0]</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>409/462</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>53/462</td><td rowspan=1 colspan=1>99.1</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=2>[85.3, 91.3]</td><td rowspan=1 colspan=2>[8.7, 14.7]</td><td rowspan=1 colspan=1>[97.6, 99.7]</td></tr></table>

Table 6: Overall Tissue of Origin Test Performance: Stratification by Metastatic and Primary Tumor Specimens   

<table><tr><td rowspan=2 colspan=1>Metastasis orPrimary</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Non-Agreement</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>ratio</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>ratio</td></tr><tr><td rowspan=1 colspan=1>Metastatic Tumors</td><td rowspan=1 colspan=1>91.1%</td><td rowspan=1 colspan=1>163/179</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>16/179</td></tr><tr><td rowspan=1 colspan=1>Poorly &amp;UndifferentiatedPrimary Tumors</td><td rowspan=1 colspan=1>86.9%</td><td rowspan=1 colspan=1>246/283</td><td rowspan=1 colspan=1>13.1%</td><td rowspan=1 colspan=1>37/283</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>88.5%</td><td rowspan=1 colspan=1>409/462</td><td rowspan=1 colspan=1>11.5%</td><td rowspan=1 colspan=1>53/462</td></tr><tr><td rowspan=1 colspan=2>95% CI [85.3, 91.3]</td><td rowspan=1 colspan=2>95% CI [8.7, 14.7]</td></tr></table>

$b$ . Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity and Clinical Specificity: Clinical sensitivity was not examined because a reference standard defining diagnostic truth was not employed for the clinical studies. Instead, positive percent agreement with the available diagnosis was considered (See Tables 5 and 6)

Clinical specificity was not examined because a reference standard defining diagnostic truth was not employed for the clinical studies. Instead, negative percent agreement with the available diagnosis was considered (See table 5 and 6)

Below is a table with “On-panel Tumor types with RNA expression patterns that are similar to patterns in the database”.

Clinical validation study specimens $\scriptstyle ( \mathtt { n } = 4 6 2$ ) are shown below:

Table 7: On-panel tumor types with RNA expression patterns that are similar to patterns in the database.   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>TotalSpecimens(n=462)</td><td rowspan=1 colspan=15>Distribution of origin Test Kit - FFPE across the 15 tissues on the TISSUE OF ORIGIN Test Kit  FFPE panel</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>BL</td><td rowspan=1 colspan=1>BR</td><td rowspan=1 colspan=1>CO</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>GC</td><td rowspan=1 colspan=1>KI</td><td rowspan=1 colspan=1>LI</td><td rowspan=1 colspan=1>LU</td><td rowspan=1 colspan=1>LY</td><td rowspan=1 colspan=1>ME</td><td rowspan=1 colspan=1>OV</td><td rowspan=1 colspan=1>PA</td><td rowspan=1 colspan=1>PR</td><td rowspan=1 colspan=1>SC</td><td rowspan=1 colspan=1>TH</td></tr><tr><td rowspan=1 colspan=1>Bladder(BL)</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Breast(BR)</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Colorectal(CO)</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Gastric(GA)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>TesticularGerm(GC)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Kidney(KI)</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hepatocellular (LI)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Non-smallcel lung(LU)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>→</td></tr><tr><td rowspan=1 colspan=1>Non-Hodgkin&#x27;sLymphoma(LY)</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Melanoma(ME)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Ovary(OV)</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pancreas(PA)</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Prostate(PR)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Soft tissuesarcoma(SC)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Thyroid(TH)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>GrandTotal</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>28</td></tr></table>

The majority of specimens are on the diagonal, as is expected from the $8 8 . 5 \%$ overall agreement with available diagnosis. Diagnostic Odds Ratio was calculated for each individual tissue in the Origin test. DOR was significantly greater than 1 for all 15 tissues, indicating that all 15 tests are informative.

The Off-Panel Specimen Study was to assess off-panel specimen “cross-reactivity” in the Origin test Kit-FFPE test for tumor specimens that do not originate from one of the 15 tissues on the Origin Test Kit – FFPE panel.

Tissue specimens that are off-panel (i.e. not one of the fifteen tissues on the Tissue of Origin Test panel) were assessed for similarity in RNA expression pattern with one of the 15 tissues on the panel. A review of published sources and interviews with practitioners were conducted to identify the cancers not included in the 15 tissues on the Origin Test Kit–FFPE test panel that should be evaluated in this study. Criteria for selection included:

• Commonly known to metastasize

Challenging or difficult to diagnose Likely to present as an uncertain primary cancer

This review selected the following: cancer of the uterine cervix, endometrium, esophagus, small cell lung, and squamous cell carcinoma of the head & neck. The study involved 101 off-panel tissue specimens from tissues from tumors of CNS origin, adult (CN, $\mathtt { n } = 1 3$ ), endometrium (EN, ${ \tt n } = 1 2$ ), esophagus (ES, $\mathtt { n } = 1 1 \mathtt { \Omega } _ { . }$ ), head and neck, squamous, (HN, $\mathtt { n } = 1 0$ ), mesothelioma (MT, $\mathtt { n } = 1 0$ ), neuroendocrine (NE, ${ \tt n } = 1 2$ ), ovarian germ cell (OG, $\mathrm { n } { = } 1 1$ ), small cell lung $\mathrm { S L }$ , ${ \mathfrak { n } } = 1 1$ ), and uterine cervix (UC, $\mathrm { n } = 1 1$ ), of which 82/101 $( 8 1 . 2 \% )$ yielded Origin Test Kit – FFPE results.

Table 8 below shows off-panel specimen results and how highest Similarity Scores are distributed across the 15 tissues on the Origin Test Kit–FFPE panel. For off-panel tissue specimens, this indicates a false positive association with that cancer. For each off panel tissue type, a high false positive percentage was observed, especially for several cancers (CN, EN, MT, OG) with ovarian and HN and SL with non-small cell lung. The table indicates that the Pathwork Origin-FFPE Test, when applied to tumors having origins not included in the panel, will give a result indicating one of the tissues on the panel. Therefore, no conclusion can be drawn about the presence of absence of off-panel tissues from the results of this test. This limitation will be noted on the labeling of the product.

# Table 8: Off-panel tumor types with RNA expression patterns that are similar to patterns in the database.

Off-panel study specimens $( \boldsymbol { \mathsf { n } } = 8 2 )$ ) are shown below:

<table><tr><td colspan="1" rowspan="2">SpecimenOrigin</td><td colspan="1" rowspan="2">TotalSpecimens(n)</td><td colspan="15" rowspan="1">Distribution of Results Across the 15 Tissues on the Origin-FFPE Panel</td></tr><tr><td colspan="1" rowspan="1">BL</td><td colspan="1" rowspan="1">BR</td><td colspan="1" rowspan="1">CO</td><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">GC</td><td colspan="1" rowspan="1">LI</td><td colspan="1" rowspan="1">KI</td><td colspan="1" rowspan="1">LY</td><td colspan="1" rowspan="1">LU</td><td colspan="1" rowspan="1">ME</td><td colspan="1" rowspan="1">OV</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">PR</td><td colspan="1" rowspan="1">SC</td><td colspan="1" rowspan="1">TH</td></tr><tr><td colspan="1" rowspan="1">CNS Origin(CN)</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Endometrium(EN)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Esophagus(ES)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Head &amp; Neck,sq. (HN)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mesothelioma(MT)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neuroendocrine (NE)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Ovarian GermCell (OG)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Small CellLung (SL)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Uterine cervix(CX)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">1</td></tr></table>

$b$ . Other clinical supportive data (when a. is not applicable): None

4. Clinical cut-off: Same as Assay cut-off

5. Expected values/Reference range: Not applicable

N. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

O. Conclusion: The submitted information in this premarket notification is complete and supports a substantial equivalence decision.