# 510(k) SUMMARY

# Submitted By

AutoGenomics, Inc. 2980 Scott Street Vista, CA 92081, USA Telephone: (760) 477-2248 FAX: (760) 477-2251 .

Contact: Evelyn Lopez Vice President, Regulatory Affairs

Date Prepared: October 22, 2010

K1016Y3   
OCT 2 5 2010

# Device Name

Trade or Proprietary Name: INFINITI CYP2C19 Assay

Common or Usual Name: Cytochrome P450 CYP2C19 Test

# Regulations and Product Codes

Regulations: 21CFR§862.3360 Drug Metabolizing Enzyme Genotyping Systems 21CFR862.2570 Instrumentation for Clinical Multiplex Test Systems

Classification: Class II .

Product Codes: NTI Drug Metabolizing Enzyme Genotyping System NSU Instrumentation for Clinical Multiplex Test Systems

Predicate Device

(aINFINITI Warfarin Assay (k073014) (AmpliChip CYP450 Microarray (k043576)

# Device Description

The INFINITI CYP2C19 Assay is an in vitro diagnostic device which utilizes proprietary film-based microarray technology combined with process automation, reagent management, and software technology for the detection and genotyping of the $2 \mathrm { C } 1 9 ~ { ^ { \ast } } 2$ , $^ { * 3 }$ , and $* 1 7$ mutations in genomic deoxyribonucleic acid (DNA) obtained from EDTAanticoagulated whole blood samples.

The INFINITI CYP2C19 Assay is comprised of the BioFilmChip™" Microarray, the Intellipac Reagent Module and the PCR Amplification Mix. The INFINITI CYP2C19 Assay should be run using the AutoGenomics INFINITI Analyzer.

The BioFilmChip Microarray consists of a polyester film coated with proprietary multilayer components designed for DNA analysis. The layers have been designed to provide a versatile surface to enhance test performance. There can be up to 240 spots per microarray with each spot representing a different allele. The microarrays are designed to be assay specific.

The Intellipac Reagent Module contains up four reservoirs that house the test reagents and has an integrated memory chip. Information on the reagent such as lot number, expiration date and remaining tests, are archived in the memory.

The PCR Amplification Mix consists of the reagents needed for the PCR amplification step of the assay.

The INFINITI CYP2C19 Assay is based on the following processes:

() DNA extraction PCR amplification of purified DNA from human genomic DNA   
(c) Labeling of the amplified product (allele specific primer extension)   
(d) Hybridization of the labeled amplified product to a microarray by signature Tag/Capture probe hybridization under isothermal conditions.   
(e) Scanning of the microarray   
()Signal detection and analysis   
Steps (c) through (f) are automated by the INFINITI Analyzer.

The INFINITI Analyzer automates the 2C19 assay and integrates all the discrete processes of sample (PCR amplicon) handling, reagent management, hybridization, detection, and results analysis. The assays are processed automatically and read by the built-in confocal microscope. Results are analyzed and presented as genotype calls.

# Intended Use

The INFINITI CYP2C19 Assay is an in vitro diagnostic test for the identification of a patient's CYP450 2C19 genotype in genomic deoxyribonucleic acid (DNA) obtained from EDTA-anticoagulated whole blood samples. The INFINITI CYP2C19 Assay is a qualitative assay for use in clinical laboratories upon prescription by the attending physician.

# Indication for Use

The INFINITI CYP2C19 Assay is indicated for use as an aid to clinicians in determining therapeutic strategy for therapeutics that are metabolized by the CYP450 2C19 gene product, specifically $^ { * } 2 , ^ { * } 3 , ^ { * } 1 7$ .

The INFINIT1 CYP2C19 Assay is not indicated to be used to predict drug response or nonresponse.

Substantial Equivalence

Table la and Table 1b provide a comparison of the INFINITI CYP2C19 Assay to the predicate devices. The comparison demonstrates that the INFINITI CYP2C19 Assay is substantially equivalent to the predicate devices.

Table la   

<table><tr><td rowspan=2 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Predicate</td><td rowspan=1 colspan=1>Subject Device</td></tr><tr><td rowspan=1 colspan=1>INFINITI Warfarin Assay(k073014)</td><td rowspan=1 colspan=1>INFINITI CYP2C19 Assay</td></tr><tr><td rowspan=1 colspan=3>x Similarities                    :             *.:</td></tr><tr><td rowspan=1 colspan=1>DNA sequence</td><td rowspan=1 colspan=1>Detects specific DNA sequences through recognition ofDNA targets</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Microarray-based genotyping test for simultaneousdetection (multiplex system) of DNA sequences</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Purified DNA from EDTA-anticoagulated whole bloodsample</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Reaction Conditions</td><td rowspan=1 colspan=1>Utilizes thermal cyclingUtilizes target DNA amplificationReactions occur on a single biofilm microarray chip</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=1>Assay signal results are interpreted by a softwareprogramAssay results are provided as genotype callsreported to the end user in a report format</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>INFINITI Analyzer</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=2>Differences   :te.   &quot;a i35:in</td><td rowspan=1 colspan=1>1i4:&quot;     ST     .;$</td></tr><tr><td rowspan=1 colspan=1>Target Genes</td><td rowspan=1 colspan=1>CYP4502C9 and VKORC1</td><td rowspan=1 colspan=1>CYP4502C19</td></tr></table>

Table 1b   

<table><tr><td rowspan=2 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Predicate</td><td rowspan=1 colspan=1>Subject Device</td></tr><tr><td rowspan=1 colspan=1>AmpliChip CYP450 Microarray (k043576)</td><td rowspan=1 colspan=1>INFINITI CYP2C19 Assay</td></tr><tr><td rowspan=1 colspan=3> - 1            45      Similarities</td></tr><tr><td rowspan=1 colspan=1>Indication for use</td><td rowspan=1 colspan=1>As an aid in determining therapeutic strategyand treatment dose for therapeutics that aremetabolized by the CYP450 2C19 geneproduct.</td><td rowspan=1 colspan=1>As an aid in determining therapeutic strategyfor therapeutics that are metabolized by theCYP450 2C19 gene product. Not intended topredict drug response or non-response</td></tr><tr><td rowspan=1 colspan=1>DNA sequence</td><td rowspan=1 colspan=1>Detects specific DNA sequences throughrecognition of DNA targets</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Purified DNA from EDTA-anticoagulatedwhole blood sample</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Microarray-based genotyping test forsimultaneous detection (multiplex system) ofDNA sequences</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>ReactionConditions</td><td rowspan=1 colspan=1>. Utilizes thermal cyclingUtilizes target DNA amplificationReactions occur on a single biofilmmicroarray chip</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=1>Assay signal results are interpreted by asoftware programAssay results are provided as genotype callsreported to the end user in a report format</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target Gene</td><td rowspan=1 colspan=1>CYP450 2C19</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=3>Differences                t                                      v.*:</td></tr><tr><td rowspan=1 colspan=1>Target GeneMutations</td><td rowspan=1 colspan=1>CYP450 2C19 *2 and *3</td><td rowspan=1 colspan=1>CYP450 2C19 *2, *3, and *17</td></tr></table>

# Performance

The following are performance characteristics of the INFINITI CYP2C19 Assay:

# Analytical Specificity

S relatd  seciciy wee condud urg assy deveoent.CR prie pecficiy wa ete y lio z  nq he l  eccy  yhec cal t sys   pt pro ey   yyizi and demonstrating that correct oligo hybridizes to the known spot.

# Limit of Detection (analytical sensitivity)

The analyical sensitiviy Limi  Detecion of the FTAssy wasassesse by analysi w blo samples t serial dilutions representing 500ng, 400ng, 200ng, 100ng, 50ng, 20ng, 10ng andn DNA inut (per test) to determine the lowest level of genomic DNA (ng input per test) that would give $a \geq 9 0 \%$ correct call rate of the allele with no incorrect calls.

The following whole blood samples were used in the LOD studies: $^ { * } 1 / { } ^ { * } 1 , ^ { * } 1 / { } ^ { * } 2 , ^ { * } 1 / { } ^ { * } 3 , ^ { * } 1 / { } ^ { * } 1 7 , ^ { * } 2 / { } ^ { * }$ and $\ast 2 / \ast 1 7$ Sample genotype was determined by bi-directional sequencing.

Two extraction methods were used in the studies: Qiagen QlAamp DNA Blood Kit and the PSS Magration System.   
The concentration and $\mathsf { A } _ { 2 6 0 } / \mathsf { A } _ { 2 8 0 }$ were determined by spectrophotometry for the extracted DNA.

A 56  e  Os 0%    i  l was obtained at DNA input levels from $4 0 0 \mathrm { n g } /$ test to 5ng/test. There was one incorrect call at the Sng DNA input level. The incorct al was probably due to the ow DNA concentration, therefore, we are not recomending DNA input level at this low concentration.

Te ev   hi  al We belive that establishing the lowst dtectable evel t ng DA/test is conevative n should prece incorrect call.

Table 2 provies asary f the LOD studi r the FINIT CYP19Assy.

Table 2 Limit of Detection   

<table><tr><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">gDNAinput pertest2</td><td colspan="1" rowspan="1">Replicatesxtested  i$$\fa}$</td><td colspan="1" rowspan="1">.:Correctcalls  1$</td><td colspan="1" rowspan="1">- } '.Incorrectcalls</td><td colspan="1" rowspan="1">s :No calls</td><td colspan="1" rowspan="1">+..     ;% correct4calls 1time.runr2    t*i</td><td colspan="1" rowspan="1">95% One-sidedConfidenceLower Limit4</td></tr><tr><td colspan="1" rowspan="9">*1/*1</td><td colspan="1" rowspan="8">AG44AG105AG208AG209AG211AG219</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">72.7%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">305</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">e15</td><td colspan="1" rowspan="1">95.3%</td><td colspan="1" rowspan="1">∴92.8%</td></tr><tr><td colspan="1" rowspan="9">*1/*2</td><td colspan="1" rowspan="8">AG81AG197AG210AG223</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">82.9%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">79.5%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">Total      </td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">229</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">111 3.</td><td colspan="1" rowspan="1">95.4% 53$</td><td colspan="1" rowspan="1">92.6%</td></tr><tr><td colspan="1" rowspan="9">*2/*2</td><td colspan="1" rowspan="8">AG82AG199AG214AG233</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">'  39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1"> …. To</td><td colspan="1" rowspan="1">Total     :</td><td colspan="1" rowspan="1">320  ÷'</td><td colspan="1" rowspan="1">304</td><td colspan="1" rowspan="1">0,</td><td colspan="1" rowspan="1">16.</td><td colspan="1" rowspan="1">95.0% 2a}$</td><td colspan="1" rowspan="1">  92.5%</td></tr><tr><td colspan="1" rowspan="9">*1/*3</td><td colspan="1" rowspan="8">AG152AG162</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">.40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3   320</td><td colspan="1" rowspan="1">317</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">99:1%</td><td colspan="1" rowspan="1">97.9%  </td></tr><tr><td colspan="1" rowspan="1">jenc</td><td colspan="1" rowspan="1">SampleID</td><td colspan="1" rowspan="1">ngDNAinputpertest</td><td colspan="1" rowspan="1">...Replicatestesed</td><td colspan="1" rowspan="1">SA*fa}$Correctc alls</td><td colspan="1" rowspan="1">.      ; &lt;IncorrectXccallsi$$          ,'1.   tir.</td><td colspan="1" rowspan="1">"4}$, 'Fi:+No calls$\fra}$</td><td colspan="1" rowspan="1">calls 1*         2time run,</td><td colspan="1" rowspan="1">95% One-sidedConfidenceLower Limit= 4" s$24*5..</td></tr><tr><td colspan="1" rowspan="9">*1/*17</td><td colspan="1" rowspan="8">AG74AG94AG180AG235</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">79.5%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">.:311 '</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">97:2%</td><td colspan="1" rowspan="1">95.2%</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="3">*2/*17</td><td colspan="1" rowspan="2">AG191AG195</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="2" rowspan="1">Total    : </td><td colspan="1" rowspan="1">40;</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">:0  2i</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">'-,  100%</td><td colspan="1" rowspan="1">98.8%</td></tr></table>

a determined by bi-directional sequencing

# Percent Agreement vs. Bi-directional Sequencing

The F PAssy s cpare i-ireinal equencn s he cparar methdTree u  he crn sda   n pat s wi he y samples were de-identified to protect patient's identity.

A total of 317 samples were tested.   
There were no incorrect calls   
Six samples $( 1 . 9 \% )$ had to be repeated because of "no call" due to NTCE Error. NTCE is reported when the quality or quantiy f the A in the sample/PCR product is poor.All si no calls gave correct cal un the repeat for a $100 \%$ agreement of the INFINITI CYP2C19 Assay with bi-directional sequencing. The repeat test was done on the same extracted DNA. Only one repeat was done for each sample.

The results of the comparison studies comparing the INFINIT CYP2 Assay to bi-directional sequencin are provided in Table 3.

Table 3 Agreement between INFINITI CYP2C19 Assay with Bi-directional Sequencing   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Number4;Tested∴</td><td rowspan=1 colspan=2>12Replicatesper Sampleie.</td><td rowspan=1 colspan=1> Correct:&#x27;rGenotype4.2 Calls..3</td><td rowspan=1 colspan=1>-3Numbr.of:IncorrectCalls}$  ao</td><td rowspan=1 colspan=1>,3NoCallsi$fra</td><td rowspan=1 colspan=1>Agreement,i;.</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit5      . .</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>95.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.2%</td><td rowspan=1 colspan=1>91.5%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8%</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>50.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>95.3%</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.5%</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*3/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>50.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>96.4%</td></tr></table>

a Genotype determined by bi-directional sequencing; $\yen 123,456$ samples are wild-type for $^ { * } 2 , * 3$ and $\$ 12$

# Assay Inter-Laboratory Reproducibility

A-si uy w cn srate the erucibily  eFINTYPy.The of reproducibility studies, each with a different operator and instrument).

T   enical smpl cris   whole bloo sleThe  we blinded  sp eny.A 0 s       o lTe h e etrad Ahe  r iht hav bee caus by peatue radint duri R, l  he  tbs e etg .The noye alm he ept asy  100%.

Results of the inter-laboratory reproducibility study are summarized in Table 4a and Table 4b.

Table 4a Inter-Laboratory Reproducibility of the INFINITI CYP2C19 Assay by Genotype Call   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>,Siteb</td><td rowspan=1 colspan=1>2       .Replicatesper Site.</td><td rowspan=1 colspan=1>Replicateswith GenotypewECalls made byINFINITI</td><td rowspan=1 colspan=1>?Calls</td><td rowspan=1 colspan=1>$\frac{}$Calls</td><td rowspan=1 colspan=1>NoCalls5</td><td rowspan=1 colspan=1>.&#x27; CorrectCalls&#x27;</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=4 colspan=1>*1/*1</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>3b</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=4 colspan=1>U</td><td rowspan=4 colspan=1>1,</td><td rowspan=1 colspan=1>$1$f}$</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0&quot;</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>$$ &#x27;598.8  a.</td></tr><tr><td rowspan=1 colspan=1>2.</td><td rowspan=1 colspan=1>,; r&quot;&quot;30T.+</td><td rowspan=1 colspan=1>.30&#x27;</td><td rowspan=1 colspan=1>.30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>::   98.3</td></tr><tr><td rowspan=1 colspan=1>.3A</td><td rowspan=1 colspan=1>430F</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0,</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2100</td><td rowspan=1 colspan=1>.   98.3  *</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=4 colspan=1>*2/*2</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>87.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>70.0</td><td rowspan=1 colspan=1>47.4</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>82.8</td></tr><tr><td rowspan=4 colspan=1>*1/*3</td><td rowspan=4 colspan=1>,2</td><td rowspan=1 colspan=1>.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20    </td><td rowspan=1 colspan=1>i20</td><td rowspan=1 colspan=1>  :0</td><td rowspan=1 colspan=1>.*.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10214&#x27;</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5&quot; 10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>.3</td><td rowspan=1 colspan=1>..10.  : :</td><td rowspan=1 colspan=1>A810</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>:0:</td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>: t ∴95.0</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1 40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>   98.8:</td></tr><tr><td rowspan=4 colspan=1>*1/*17</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>85.0</td><td rowspan=1 colspan=1>66.9</td></tr><tr><td rowspan=1 colspan=1>$^{}$</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>66.4</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>86.1</td></tr><tr><td rowspan=4 colspan=1>*17/*17</td><td rowspan=4 colspan=1>-&#x27;·2&#x27;:</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1> 0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>2.</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1:</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>82.9</td></tr><tr><td rowspan=1 colspan=1>:;$3^a}$</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>$1x^}$ 10</td><td rowspan=1 colspan=1>10.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>693$</td><td rowspan=1 colspan=1>&quot;69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>430</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>$1frac{ }$</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>94.7</td></tr></table>

a determined by bi-directional sequencing; ${ } ^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { \ast 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$ EMPY b Internal site (Site 1) had two sets, one operator each Excludes samples with No Calls Initial result was incorrect $( \ast 1 / \ast 2$ instead of $\yen 23,452$ ).The root cause of the incorrect call was not definitively determined. gave the correct call gave the correct calls Samples with correct calls/samples tested Site 3: one sample had A260/A280 of 1.16 (1.6 is required), therefore only one sample was tested h Site 1: one sample had A260/A280 of 1.46 (1.6 is required), therefore only two samples were tested Site 3: one sample had A260/A280 of 1.46 (1.6 is required), therefore only one sample was tested Site 3: one sample had A260/A280 of 1.45 (1.6 is required), therefore only one sample was tested

Table 4b Inter-Laboratory Reproducibility of the INFINIT1 CYP2C19 Assay by Sample   

<table><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>i:Genotype.</td><td rowspan=1 colspan=1>Replicates.ipersample</td><td rowspan=1 colspan=1>Replicates with Genotype CallsmadebyINFINITE</td><td rowspan=1 colspan=1>Samples3withCorrect:S  alls ÷*</td><td rowspan=1 colspan=1>Samples33.</td><td rowspan=1 colspan=1>SampleswithincorrectCals</td><td rowspan=1 colspan=1>    7Correct</td><td rowspan=1 colspan=1>rx95% One-sided$</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>83.1</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>76.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>83.1</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.6</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.6</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.6</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>430</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>94.7</td></tr></table>

Determined by bi-directional sequencing. ${ } ^ { * } 1 / { } ^ { * } 1$ samples ae wild-type or ,   1 Excludes samples with No Calls A sample with correct call indicates a correct call at all loci. is poor. Repeat tests gave the correct calls $( { } ^ { * } \mathbb { I } / { } ^ { * } \mathbb { 2 }$ instead of $\yen 2$ T  cu l was  eivy e Samples with crct calls/smples ested

Aby   b INFINITI CYP2C19 Assay for additional samples including $\ast _ { 2 } / \ast _ { 1 7 }$ and $\ast _ { 2 } / \ast _ { 3 }$ . The study involved three lots of the IFTYPAssay.The e  enticlsamp copri   enic whole blood sapThe s w y  a consecutive days.

A total of 255 tests were completed. Overall correct call rate for the first time run was $9 7 . 6 \%$ .There were no incorrect calls. There were six no calls:

O o l a   t    p whe ot py aligned.   
Five were due to NTCE error.NTCE is reported if the quality or quantity of the DNA in the sample/PCR product is poor.These NTCE errors might have been caused by temperature gradient during PCR, improper sealing of the PCR tubes, or operator pipetting error.

Results of the inter-laboratory reproducibility studies are summarized in Table 5a and Table 5b.

Table 5a Inter-Laboratory Reproducibility of the INFINITI CYP2C19 Assay by Genotype calls   

<table><tr><td rowspan=1 colspan=1>5.1Genotype:</td><td rowspan=1 colspan=1>.:3SamplesTested</td><td rowspan=1 colspan=1>: &#x27; .•Site2. .&#x27;:;</td><td rowspan=1 colspan=1>&quot;per Site*</td><td rowspan=1 colspan=1>Replicateswith GenotypeCalls made byINFINITI</td><td rowspan=1 colspan=1>2  ErCorrect Calls*4:</td><td rowspan=1 colspan=1>*2IncorrectCalls,i.J:=</td><td rowspan=1 colspan=1>4$NoCallsi&#x27;</td><td rowspan=1 colspan=1>$\ra }$%CorrectCalls</td><td rowspan=1 colspan=1>! •.95% One-sidedConfidence•Lower Limit&#x27;      .</td></tr><tr><td rowspan=4 colspan=1>*1/*1</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=4 colspan=1>.*1/*2</td><td rowspan=4 colspan=1>:1</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>.15</td><td rowspan=1 colspan=1>.15</td><td rowspan=1 colspan=1>.0  :</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>196.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>v..15&#x27;</td><td rowspan=1 colspan=1>&quot;  15E.</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100$</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>:3</td><td rowspan=1 colspan=1>&quot;15</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>$0$</td><td rowspan=1 colspan=1>$2.</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>66:1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2.</td><td rowspan=1 colspan=1>95:6</td><td rowspan=1 colspan=1>88.4</td></tr><tr><td rowspan=4 colspan=1>*1/*3</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>$\}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=4 colspan=1>*1/*17-</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15i.</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>:0X</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>.2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>.   15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>∴100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15I</td><td rowspan=1 colspan=1>.15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>.96.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45:</td><td rowspan=1 colspan=1>:   45   ;</td><td rowspan=1 colspan=1>45.</td><td rowspan=1 colspan=1>*.0 .:</td><td rowspan=1 colspan=1>∴0:</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>.98.9</td></tr><tr><td rowspan=4 colspan=1>*2/*3</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>56.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>84.9</td></tr><tr><td rowspan=4 colspan=1>*2/*17</td><td rowspan=4 colspan=1>-e1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>, .15.</td><td rowspan=1 colspan=1>y : &quot;.. ,15</td><td rowspan=1 colspan=1>&#x27;15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100=</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>.2</td><td rowspan=1 colspan=1>.15.)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>i0  i</td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3.</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>i  14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>-0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>77.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>44.</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>95.6</td></tr></table>

Di y ale; $\mp 1 / \ast 1$ samples are wild-type for $^ { * 2 }$ , \*3 and $^ { * 1 7 }$ EMPY Excludes samples with No Calls T      A Si 1: DNA concentration was ${ \mathfrak { s n g } } / { \mu \updownarrow }$ , below the LOD of $1 0 \Im \mathrm { / \mu l } ,$ therefore no sample tested

Table 5b Inter-Laboratory Reproducibility of the INFINITI CYP2C19 Assay by Sample   

<table><tr><td rowspan=1 colspan=1>SampleID•</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>.Replicatesper sample.</td><td rowspan=1 colspan=1>Replicateswith GenotypeCalls made byINFINITI</td><td rowspan=1 colspan=1>:CorrectCallsec.</td><td rowspan=1 colspan=1>No Callsd</td><td rowspan=1 colspan=1>IncorrectCallsi</td><td rowspan=1 colspan=1>.Correct CallRate (%.</td><td rowspan=1 colspan=1>95% One-sidedConfidence LowerL Limit</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>${b}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.4</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>84.9</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>88.4</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>95.6</td></tr></table>

$\neq 1 / \ast 1$ se  y r Excludes samples with No Calls NTCE Error - NTCE is reported if the quality or quantity of the DNA in the sample/PCR product is poor. eSamples with correct calls/samples tested

Table 6 provides a summary of the combined reproducibility studies.

Tab :Summary of the Inte-Laboratory Reproducibility of the INFINIT CYP9 Assay by Genotype calls   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>SamplesTested:</td><td rowspan=1 colspan=1>, &#x27;&quot;.Replicatesper Samplei</td><td rowspan=1 colspan=1>Replicaateswith Genotype.Calls made by.INFINITI&quot;s</td><td rowspan=1 colspan=1>.CorrectCalls1</td><td rowspan=1 colspan=1>.. :1$$</td><td rowspan=1 colspan=1>IncorrectCallls17</td><td rowspan=1 colspan=1>$Ratel (%):5</td><td rowspan=1 colspan=1>95% One-sidedConfidence Loweri$Limit5..</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.0</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.4</td></tr><tr><td rowspan=1 colspan=1>*21*2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>82.8</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>915</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>84.9</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>685</td><td rowspan=1 colspan=1>665</td><td rowspan=1 colspan=1>664</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>95.6</td></tr></table>

"Genotype determined by bi-directional sequencing; $\mp 1 / \mp 1$ samples are wild-type for \*2, \*3 and $^ { * } 1 7$ Excludes samples with No Calls Ae wi callie  rt altll i $\dot { }$ 18TC T   l  A/R Inal u  c $[ { ^ { * } } 1 / { ^ { * } } 2$ instead of $\blacktriangleleft / \ast 2$ T  u     v . Samples with correct calls/samples tested

# Interference

In  a o s wepely he o blos exin t  t INFINIT CYP2C19 Assay.Genotype results were compared to those obtained from non-spiked samples.Smple notype was veried by bi-directional DA sequenciTheintererence studi monstrated that eFNIT CYP2C19 Assay performance was not affected by the addition of the following substances.

<table><tr><td rowspan=1 colspan=1>.          Substance Added     $$*</td><td rowspan=1 colspan=1>$}$    Concentration</td></tr><tr><td rowspan=1 colspan=1>Bilirubin (conjugated)</td><td rowspan=1 colspan=1>60mg/dl</td></tr><tr><td rowspan=1 colspan=1>Bilirubin (unconjugated)</td><td rowspan=1 colspan=1>60mg/dl</td></tr><tr><td rowspan=1 colspan=1>Triglycerides (Intralipid)</td><td rowspan=1 colspan=1>3000mg/dl</td></tr><tr><td rowspan=1 colspan=1>Human albumin</td><td rowspan=1 colspan=1>6g/dl</td></tr></table>

No studies were conducted with oral anti-coagulants, and no claims are being made.

# Sample Carry-Over

No ve n and when 300ng of a positive sample was followed by a $\sin _ { 0 }$ Template Control" or water. All genotype calls were $100 \%$ EMP correct.

# Reagent Stability

BioFilmChip Microarray: Intellipac Reagent: Amplification Mix:

12 months at RT (15 to $3 0 ^ { \circ } \mathrm { C }$ )   
12 months Refrigerated (2 to $8 ^ { \circ } \mathrm { C }$   
18 months Frozen (-30 to $\cdot 1 5 ^ { \circ } \mathrm { C }$ t

Conclusion

The above pre-clinical and clinical data support the safety and effectiveness of the INFINITI CYP2C19 Assay.

AutoGenomics, Inc.   
c/o Evelyn Lopez   
Vice President, Regulatory Affairs 2980 Scott Street   
Vista, CA 92081

Re: k101683 Trade/Device Name: INFINITI CYP2C19 Assay Regulation Number: 21 CFR $\ S 8 6 2 . 3 3 6 0$ E Regulation Name: Drug Metabolizing Enzyme Genotyping System Regulatory Class: Class II Product Codes: NTI, NSU Dated: October 6, 2010 Received: October 8, 2010

Dear Evelyn Lopez:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration:

If your device is classified (see above) into either class II (Special Controls) or class III (MA), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to.895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820).

Page 2 -

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket   
notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

GC.

Coulthey Harper, Ph.D.   
Director   
Division of Chemistry and Toxicology   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

# Premarket Notification 510(k) INFINIT1® CYP2C19 Assay

# Indications for Use

K101683   
OCT 25 2010

510(k) Number (if known):

# K101683

Device Name:

INFINITI®CYP2C19 Assay

Indications For Use:

The INFINITI CYP2C19 Assay is an in vitro diagnostic test for the identification of a patient's CYP450 2C19 genotype in genomic deoxyribonucleic acid (DNA) obtained from EDTA-anticoagulated whole blood samples. The INFINITI CYP2C19 Assay is a qualitative assay for use in clinical laboratories upon prescription by the attending physician.

The INFINITI CYP2C19 Assay is indicated for use as an aid to clinicians in determining therapeutic strategy for therapeutics that are metabolized by the CYP450 2C19 gene product, specifically $^ { * } 2 , ^ { * } 3 , ^ { * } 1 7$ .

The INFINITI CYP2C19 Assay is not indicated to be used to predict drug response or non-response.

Prescription Use X (Part 21 CFR 801 Subpart D)

AND/OR

Over-The-Counter Use (21 CFR 801 Subpart C)

(PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE IF NEEDED)