# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: k101683   
B. Purpose for Submission: New device   
C. Measurand: Genotype of cytochrome P450 2C19 (CYP2C19)   
D. Type of Test: Genotyping microarray   
E. Applicant: AutoGenomics, Inc.   
F. Proprietary and Established Names: INFINITI CYP2C19 Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR $\ S 8 6 2 . 3 3 6 0$ , Drug Metabolizing Enzyme Genotyping Systems 21 CFR $\ S 8 6 2 . 2 5 7 0$ , Instrumentation for Clinical Multiplex Test Systems   
2. Classification: Class II   
3. Product code: NTI, Drug Metabolizing Enzyme Genotyping Systems NSU, Instrumentation for Clinical Multiplex Test Systems   
4. Panel: Toxicology (91) Chemistry (75)

# H. Intended Use:

1. Intended use(s): See Indications for use below

2. Indication(s) for use: The INFINITI CYP2C19 Assay is an in vitro diagnostic test for the identification of a patient’s CYP450 2C19 genotype in genomic deoxyribonucleic acid (DNA) obtained from EDTA-anticoagulated whole blood samples. The INFINITI CYP2C19 Assay is a qualitative assay for use in clinical laboratories upon prescription by the attending physician.

The INFINITI CYP2C19 Assay is indicated for use as an aid to clinicians in determining therapeutic strategy for therapeutics that are metabolized by the CYP450 2C19 gene product, specifically $^ { * } 2 , ^ { * } 3 , ^ { * } 1 7$ . The INFINITI CYP2C19 Assay is not intended to be used to predict drug response or non-response.

3. Special conditions for use statement(s): For prescription use only

The information provided from this test may supplement decision making and should only be used in conjunction with routine monitoring by a physician. Because of the variability in the knowledge of clinical utility with specific drugs that are metabolized by CYP2C19, clinicians should use professional judgment in the interpretation of results from this test. Results from this type of assay should not be used in predicting a patient’s response to drugs for which the drug metabolizing enzyme activity of that allele, or the drug metabolic pathway, has not been clearly established.

# 4. Special instrument requirements: AutoGenomics INFINITI Analyzer

# I. Device Description:

The INFINITI CYP2C19 Assay is comprised of the R-Chip BioFilmChip Microarray, the Intellipac Reagent Module, the PCR Amplification Mix and the GAP/Header CD. The INFINITY CYP2C19 Assay is run on the AutoGenomics INFINITI Analyzer.

The BioFilmChip Microarray consists of a polyester film coated with proprietary multi-layer components designed for DNA analysis. There can be up to 240 spots per microarray with each spot representing a different allele. The microarrays are assay specific.

The Intellipac Reagent Module which acts as a communication link, contains up to four reservoirs that house the test reagents and has an integrated memory chip. Information such as lot number, expiration date of reagents and number of tests are archived in the memory chip.

The PCR Amplification Mix consists of the reagents needed for the PCR amplification step of the assay.

The GAP/Header CD contains the assay protocol which specifies the assay steps, parameters and conditions, and the assay Header which specifies the algorithm, assay multipliers and ratios/cut-offs. The GAP/Header CD is loaded into the INFINITI Analyzer.

The INFINITI Analyzer is an instrument used for clinical multiplex systems intended to measure and sort multiple signals from a clinical sample. The INFINITI Analyzer is designed to measure fluorescence signals of labeled DNA target hybridized to

BioFilmChip microarrays. The INFINITI Analyzer automates the INFINITI CYP2C19 Assay and integrates all the discrete processes of sample (PCR amplicon) handling, reagent management, hybridization, detection, and results analysis. The assays are processed automatically and read by the built-in confocal microscope.

Results are analyzed and presented as genotype calls.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Roche AmpliChip CYP450 microarray   
2. Predicate 510(k) number(s): k043576   
3. Comparison with predicates:

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Proposed device</td><td rowspan=1 colspan=1>AmpliChip CYP450Microarray (k043576)</td></tr><tr><td rowspan=1 colspan=2>Similarities</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intendeduse/indicationsfor use</td><td rowspan=1 colspan=1>In vitro diagnostic test for theidentification of a patient&#x27;s CYP4502C19 genotype in genomic DNAobtained from whole blood samples.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Limitation</td><td rowspan=1 colspan=1>The INFINITI CYP2C19 Assay is notintended to be used to predict drugresponse or non-response.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Test principle</td><td rowspan=1 colspan=1>Microarray-based genotyping test forsimultaneous detection (multiplexsystem) of DNA sequences</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Reactionconditions</td><td rowspan=1 colspan=1>Utilizes thermal cycling and target DNAamplification</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen type</td><td rowspan=1 colspan=1>Genomic DNA from EDTA-anticoagulated whole blood samples</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target gene</td><td rowspan=1 colspan=1>CYP2C19</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Targetmutations</td><td rowspan=1 colspan=1>*2, *3, *17</td><td rowspan=1 colspan=1>*2 and *3</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Drug Metabolizing Enzyme Genotyping System – Class II Special Controls Guidance Document

# L. Test Principle:

The INFINITI CYP2C19 Assay utilizes film-based microarray technology combined with process automation, reagent management, and software technology for the detection and genotyping of the $2 { \mathrm { C l } } 9 ^ { * } 2$ , $^ { * 3 }$ and $^ { * } 1 7$ mutations from DNA obtained from EDTAanticoagulated whole blood human samples. The assay is based on the following major processes:

# Performed off-line

(a) DNA extraction (b) PCR amplification of purified DNA

# Performed by the INFINITI Analyzer

(c) Labeling of the amplified product (allele specific primer extension)   
(d) Hybridization of the labeled amplified product to a microarray   
(e) Scanning of the microarray   
(f) Signal detection and analysis

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: a. Precision/Reproducibility: Site-to-site Study 1: In the first study performed to assess site-to-site reproducibility, three clinical sites performed testing using four analyzers and four operators. Three lots of reagents were used. All three sites tested the same 12 whole blood samples. Each site used a different DNA extraction method for the clinical samples. During the initial run, the study gave a $9 6 . 5 ~ \%$ call rate based on 415 correct calls out of a total of 430 genotype calls. There were 14 No Calls in the study, and one incorrect call. The data is summarized below:

Inter-Laboratory Reproducibility study results (data sorted by genotype call)   

<table><tr><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>Siteb</td><td rowspan=1 colspan=1>Replicatesper Site</td><td rowspan=1 colspan=1>Replicateswith GenotypeCalls made byINFINITI</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCallsd</td><td rowspan=1 colspan=1>NoCallse</td><td rowspan=1 colspan=1>CorrectCalls (%)</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=4 colspan=1>*1/*1</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=4 colspan=1>*1/*2</td><td rowspan=4 colspan=1>3</td><td rowspan=1 colspan=1>1h</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=4 colspan=1>*2/*2</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>87.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>70.0</td><td rowspan=1 colspan=1>47.4</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>82.8</td></tr><tr><td rowspan=4 colspan=1>*1/*3</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=4 colspan=1>*1/*17</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>85.0</td><td rowspan=1 colspan=1>66.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>66.4</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>86.1</td></tr><tr><td rowspan=4 colspan=1>*17/*17</td><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>82.9</td></tr><tr><td rowspan=1 colspan=1>3j</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.0</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>430</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>1d</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>94.7</td></tr></table>

a determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$   
b Internal site (Site 1) had two sets, one operator each   
c Excludes samples with No Calls   
d Initial result was incorrect $^ { * } 1 / { } ^ { * } 2$ instead of $^ { * } 2 / { ^ { * } 2 }$ ). The root cause was not definitively determined. One No Call was due to Registration Spot Error - this error is reported when the microarray chip is not properly aligned. Repeat test gave the correct call 13 reported NTCE Error - NTCE is reported if the quality or quantity of the DNA in the sample/PCR product is poor. Repeat tests gave the correct calls Samples with correct calls/samples tested   
g Site 3: one sample had A260/A280 of 1.16 (1.6 is required), therefore only one sample was tested   
h Site 1: one sample had A260/A280 of 1.46 (1.6 is required), therefore only two samples were tested Site 3: one sample had A260/A280 of 1.46 (1.6 is required), therefore only one sample was tested   
j Site 3: one sample had A260/A280 of 1.45 (1.6 is required), therefore only one sample was tested

Inter-Laboratory Reproducibility study results (data sorted by sample)   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>Replicatespersample</td><td rowspan=1 colspan=1>Replicates withGenotype Callsmade byINFINITIb</td><td rowspan=1 colspan=1>CorrectCallsc</td><td rowspan=1 colspan=1>NoCallsd</td><td rowspan=1 colspan=1>IncorrectCallse</td><td rowspan=1 colspan=1>CorrectCall Ratef(%)</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>83.1</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>76.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>83.1</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.6</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.6</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>88.6</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>430</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>94.7</td></tr></table>

a Determined by bi-directional sequencing. $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * } 2 , { } ^ { * } 3$ and $^ { * } 1 7$ b Excludes samples with No Calls c A sample with correct call indicates a correct call at all loci. d One No Call was due to Registration Spot Error - this error is reported when the microarray chip is not properly aligned. Repeat test gave the correct call; 13 reported NTCE Error - NTCE is reported if the quality or quantity of the DNA in the sample/PCR product is poor. Repeat tests gave the correct calls e Initial result was incorrect $^ { * } 1 / { ^ * } 2$ instead of $^ { * } 2 / { ^ { * } 2 }$ ). The root cause was not definitively determined. Samples with correct calls/samples tested

Site-to-site Study 2: A second study was conducted at the same three sites to demonstrate the reproducibility of the assay for six additional samples. A total of 255 tests were completed. The overall correct call rate for the first run was $9 7 . 6 \%$ . There were six No Calls and no incorrect calls. The data is summarized below:

Inter-Laboratory Reproducibility study results (data sorted by genotype call)   

<table><tr><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Replicatesper Site</td><td rowspan=1 colspan=1>Replicateswith GenotypeCalls made byINFINITIb</td><td rowspan=1 colspan=1>CorrectCalls</td><td rowspan=1 colspan=1>IncorrectCalls</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>%CorrectCallsd</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=4 colspan=1>*1/*1</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=4 colspan=1>*1/*2</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>66.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>88.4</td></tr><tr><td rowspan=4 colspan=1>*1/*3</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1e</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=4 colspan=1>*1/*17</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=4 colspan=1>*2/*3</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>56.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>84.9</td></tr><tr><td rowspan=4 colspan=1>*2/*17</td><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>77.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>95.6</td></tr></table>

a Determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * } 2 , { } ^ { * } 3$ and $^ { * 1 7 }$ b Excludes samples with No Calls c One no call was due to Registration Spot Error - this error is reported when the microarray chip is not properly aligned; 5 reported NTCE Error - NTCE is reported if the quality or quantity of the DNA in the sample/PCR product is poor. d Samples with correct calls/samples tested e Site 1: DNA concentration was $8 \mathrm { n g / \mu l }$ , below the LOD of $1 0 \mathrm { n g / \mu l }$ , therefore no sample tested

Inter-Laboratory Reproducibility study results (data sorted by sample)   

<table><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>Replicatespersample</td><td rowspan=1 colspan=1>Replicateswith GenotypeCalls made byINFINITIb</td><td rowspan=1 colspan=1>CorrectCallsc</td><td rowspan=1 colspan=1>NoCallsd</td><td rowspan=1 colspan=1>IncorrectCallse</td><td rowspan=1 colspan=1>CorrectCallRatee(%)</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.4</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>84.9</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>88.4</td></tr><tr><td rowspan=1 colspan=2>All</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>95.6</td></tr></table>

a Genotype determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$ b Excludes samples with No Calls c A sample with correct call indicates a correct call at all loci. d One no call was due to Registration Spot Error - this error is reported when the microarray chip is not properly aligned;

Summary of the combined reproducibility studies (data sorted by genotype call):

<table><tr><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>ReplicatesperSample</td><td rowspan=1 colspan=1>Replicateswith GenotypeCalls made byINFINITIb</td><td rowspan=1 colspan=1>CorrectCallsc</td><td rowspan=1 colspan=1>NoCallsd</td><td rowspan=1 colspan=1>IncorrectCallse</td><td rowspan=1 colspan=1>CorrectCallRatef(%)</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.0</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.4</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.0</td><td rowspan=1 colspan=1>82.8</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>91.5</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>84.9</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>685</td><td rowspan=1 colspan=1>665</td><td rowspan=1 colspan=1>664</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>95.6</td></tr></table>

a Genotype determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$ b Excludes samples with No Calls c A sample with correct call indicates a correct call at all loci. d Two No Calls were due to Registration Spot Error - this error is reported when the microarray chip is not properly aligned; 18 reported NTCE Error - NTCE is reported if the quality or quantity of the DNA in the sample/PCR product is poor. e Initial result was incorrect $^ { * } 1 / { ^ * } 2$ instead of $^ { \ast } 2 / ^ { \ast } 2$ ) the root cause was not definitively determined. Samples with correct calls/samples tested

$b$ . Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Controls: Positive or negative controls are not included with this assay. The manufacturer recommends that positive samples for each mutation (heterozygous and/or homozygous), a negative control (a sample that does not contain the mutations, i.e., a wild type sample) and a non-template-control be included with each run. All quality control requirements and testing should be performed in conformance with local, state and/or federal regulations.

Pre-immobilized negative and positive controls: These controls are related to the hybridization process, and indicate that the hybridization has taken place. There are five (5) negative control spots and three (3) positive control spots. The negative control is a 3’biotinylated 24mer to which nothing should bind. The positive control is a 3’biotinylated 24mer that hybridizes the complementary Cy5 labeled oligonucleotide.

Reagent Stability: The sponsor states that the BioFilmChip Microarray is stable for 12 months at 15 to $3 0 ~ ^ { \mathrm { { o } } } \mathrm { { C } }$ , the Intellipac Reagent is stable for 12 months when stored at 2 to $8 ^ { \mathrm { o } } \mathrm { C }$ , and the Amplification Mix is stable for 18 months when stored at -30 to $- 1 5 ~ ^ { \circ } \mathrm { C }$ .

Sample Stability: The sponsor recommends that specimens (EDTA anticoagulated whole blood samples) be kept refrigerated $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ) and extracted within 9 days from the day the specimen was collected. The samples should not be frozen or thawed. Extracted DNA samples should be kept refrigerated ( $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ) and assayed within 2 days from the day the specimen was extracted.

# d. Detection limit:

The upper and lower limits of detection of the assay were assessed by analysis of whole blood samples $( ^ { * } 1 / { ^ * } 1 , { ^ * } 1 / { ^ * } 2 , { ^ * } 1 / { ^ * } 3 ,$ . $^ { * } 1 / { } ^ { * } 1 7$ , $^ { * } 2 / { ^ * } 2$ and ${ } ^ { * } 2 / { } ^ { * } 1 7 $ ) at serial dilutions of DNA concentrations in replicates of 20 or 40 to determine the highest and lowest level of genomic DNA that would give $a \geq 9 0 \%$ correct call rate with no incorrect calls. A total of 1,560 tests were completed. $\mathbf { A } \geq$ $90 \%$ correct call rate with no incorrect calls was obtained at DNA input levels from $4 0 0 \mathrm { n g / }$ test to 20ng/test. There was one incorrect call at the 5ng DNA input level. A $^ { * } 1 / { ^ * } 3$ sample was incorrectly called $^ { * } 3 / { } ^ { * } 1 7$ . The results are summarized below:

Limit of detection study results (data sorted by genotype call)   

<table><tr><td colspan="1" rowspan="1">Genotypea</td><td colspan="1" rowspan="1">.ng DNAinput pertest</td><td colspan="1" rowspan="1">Replicatestested</td><td colspan="1" rowspan="1">Correctcalls</td><td colspan="1" rowspan="1">Incorrectcalls</td><td colspan="1" rowspan="1">Nocalls</td><td colspan="1" rowspan="1">%correctcalls 1sttime run</td><td colspan="1" rowspan="1">95% One-sidedConfidenceLower Limit</td></tr><tr><td colspan="1" rowspan="9">*1/*1</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">72.7%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">305</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">95.3%</td><td colspan="1" rowspan="1">92.8%</td></tr><tr><td colspan="1" rowspan="9">*1/*2</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">82.9%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">79.5%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">229</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">95.4%</td><td colspan="1" rowspan="1">92.6%</td></tr><tr><td colspan="1" rowspan="9">*2/*2</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">92.5%</td><td colspan="1" rowspan="1">83.1%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">304</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">92.5%</td></tr><tr><td colspan="1" rowspan="1">*1/*3</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">317</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">99.1%</td><td colspan="1" rowspan="1">97.9%</td></tr><tr><td colspan="1" rowspan="9">*1/*17</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">79.5%</td></tr><tr><td colspan="1" rowspan="1">400</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">87.0%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">91.4%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">311</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">95.2%</td></tr><tr><td colspan="1" rowspan="3">*2/*17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.5%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8%</td></tr></table>

a Genotype determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$

The sponsor states that the lower limit of detection for the assay is 20ng DNA per test. The recommended DNA input is $5 0 \mathrm { n g }$ per test and the maximum recommended is $4 0 0 \mathrm { n g }$ per test.

# e. Analytical specificity:

Interference from potential interfering substances was evaluated using eight whole blood samples (two samples each with genotypes of $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 1 / { ^ { * } } 2$ , $^ { * } 1 / { ^ * } 3$ , and $^ { * } 1 / { } ^ { * } 1 7$ ). The potential interfering substances were added separately to the whole blood sample prior to DNA extraction and testing. Genotype results were compared to those obtained from non-spiked samples. The following compounds at the following concentrations were tested: albumin at $6 \mathrm { g } / \mathrm { d L }$ , bilirubin (conjugated and unconjugated) at $6 0 \mathrm { m g / d L }$ , and triglycerides (Intralipid) at $3 0 0 0 \mathrm { m g / d L }$

Sample genotypes were verified by bi-directional DNA sequencing. All samples gave the correct call. The sponsor concluded that the compounds do not interfere with the performance of the assay.

# Sample carry-over:

Four genomic DNA samples with different genotypes $( ^ { * } 2 / { ^ * } 3 , ^ { * } 1 / { ^ * } 2 , ^ { * } 1 / { ^ * } 1 7$ and $^ { * } 1 / { } ^ { * } 1$ ) and water were used in this study. Four samples were tested in this order: a high DNA concentration sample $( 3 0 0 \mathrm { n g } )$ , a low DNA concentration sample $( 1 0 \mathrm { n g } )$ of a different genotype, followed by the high DNA concentration sample again, then a water blank. This protocol was run multiple times, and in all cases, the correct call (or an indeterminate call for the water ‘sample’) was obtained, illustrating no sample carry-over between samples.

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device: The method comparison study was completed at two external sites and internally. All three sites compared the results of samples run on the proposed device to bi-directional sequencing results (sequencing was completed at one site). Each site tested its own clinical samples. Of the 317 samples tested, six samples gave a No Call result for the initial testing. The results are summarized below:

<table><tr><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>NumberTested</td><td rowspan=1 colspan=2>ReplicatesperSample</td><td rowspan=1 colspan=1>Number ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>NumberofIncorrectCalls</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% One-sidedConfidenceLower Limit</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>95.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.2%</td><td rowspan=1 colspan=1>91.5%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8%</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>50.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>95.3%</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.5%</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*3/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>50.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>96.4%</td></tr></table>

a Genotype determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$

Agreement between the device and bi-directional DNA Sequencing (final results after one repeat of the No Calls):

<table><tr><td rowspan=1 colspan=1>Genotypea</td><td rowspan=1 colspan=1>Number ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>Number ofIncorrectCalls</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% One-sidedConfidence LowerLimit</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.8%</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>50.0%</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3%</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.5%</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*3/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>50.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.8%</td></tr></table>

Genotype determined by bi-directional sequencing; $^ { * } 1 / { } ^ { * } 1$ samples are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity: Not applicable

b. Clinical specificity: Not applicable

c. Other clinical supportive data (when a. and b. are not applicable):

The following table lists the alleles recognized by the device, the single nucleotide polymorphisms (SNPs) recognized by the device for each allele, enzyme activity and references:

<table><tr><td rowspan=1 colspan=1>CYP2C19Allele</td><td rowspan=1 colspan=1>SNPsRecognizedby INFINITICYP2C19 Assay</td><td rowspan=1 colspan=1>SNPs Associated withthe Allele b</td><td rowspan=1 colspan=1>EnzymeActivity</td><td rowspan=1 colspan=1>References</td></tr><tr><td rowspan=1 colspan=1>*1a</td><td rowspan=1 colspan=1>None a</td><td rowspan=1 colspan=1>None a</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>Romkes et al, 1991 [1]Richardson et al, 1995 [2]Blaisdell et al, 2002 [3]</td></tr><tr><td rowspan=1 colspan=1>*2</td><td rowspan=1 colspan=1>19154G&gt;A</td><td rowspan=1 colspan=1>-98T&gt;C; 99C&gt;T;12122G&gt;A; 12460G&gt;C12662A&gt;G; 12834G&gt;C;19154G&gt;A; 19520A&gt;G;57740C&gt;G; 79936T&gt;A;80160C&gt;T; 80161A&gt;G;87275G&gt;A</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>de Morais et al, 1994 [4]Ibeanu et al, 1998 [5]Fukushima-Uesaka et al,2005 16]Lee et al, 2009 [7]</td></tr><tr><td rowspan=1 colspan=1>*3</td><td rowspan=1 colspan=1>17948G&gt;A</td><td rowspan=1 colspan=1>-889T&gt;G; 12013T&gt;G;12122G&gt;A; 12306G&gt;A;13166T&gt;C; 17948G&gt;A;18911A&gt;G; 80161A&gt;G;80248G&gt;A; 87313A&gt;C</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>de Morais et al, 1994 [4]Fukushima-Uesaka et al,20056]</td></tr><tr><td rowspan=1 colspan=1>*17</td><td rowspan=1 colspan=1>-806C&gt;T</td><td rowspan=1 colspan=1>-3402C&gt;T; -806C&gt;T;99C&gt;T;80161A&gt;G</td><td rowspan=1 colspan=1>Increased</td><td rowspan=1 colspan=1>Sim et al, 2006 [8]Rudberg et al, 2008 [9]</td></tr></table>

a $^ { * 1 }$ genotype for the INFINITI CYP2C19 Assay indicates only the absence of $^ { * 2 }$ , $^ { * 3 }$ and $^ { * 1 7 }$ alleles b SNPs in bold are the major SNPs/alterations responsible for the phenotype of the corresponding allele and are unique to the mutation.

1 Romkes M et al. Cloning and expression complementary DNAs for multiple members of the human cytochrome P450IIC subfamily. Biochemistry (1991) Apr 2;30(13):3247-55.   
2 Richardson TH et al. A universal approach to the expression of human and rabbit cytochrome P450s of the 2C subfamily in Escherichia coli. Arch Biochem Biophys.1995 Oct 20;323(1):87-96.   
3 Blaisdell J et al. Identification and functional characterization of new potentially defective alleles of human CYP2C19. Pharmacogenetics. 2002   
Dec;12(9):703-11.   
de Morais SM et al. The major genetic defect responsible for the polymorphism of S-mephenytoin metabolism in humans. J Biol Chem. 1994 Jun 3;269(22):15419-22.   
5 Ibeanu GC et al. Identification of new human CYP2C19 alleles

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The package insert states:

$^ { \mathrm { < } } \mathrm { C Y P 2 C 1 9 ^ { \ast } } 2$ and $\mathrm { C Y P 2 C 1 } 9 ^ { * } 3$ , which encode for non-functional proteins, are responsible for the vast majority of poor metabolizer (PM) phenotypes. The allele frequency of PM phenotype varies significantly between populations, ranging from 2 to $5 \%$ in White and Black populations to 13- $23 \%$ in Asian populations.1

The frequency of the $\mathrm { C Y P 2 C l 9 ^ { * } l 7 }$ allele is equally high $( 1 8 \% )$ in Ethiopians and Swedes. A lower frequency has been reported in a Japanese population $( 1 . 3 \% )$ and in Chinese subjects $( 4 \% )$ . A study reported a $\mathrm { C Y P 2 C l 9 ^ { * } l 7 }$ allele frequency of $2 7 \%$ in a Polish population.” 2

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.