# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: k120466   
B. Purpose for Submission: New device   
C. Measurand: Genotype of Cytochrome P450 2C19 (CYP2C19) variants   
D. Type of Test: Genotyping microarray   
E. Applicant: Nanosphere, Inc.   
F. Proprietary and Established Names: Verigene® CYP2C19 Nucleic Acid Test (CYP2C19)

# G. Regulatory Information:

1. Regulation section: 21 CFR $\$ 862.3360$ ; Drug Metabolizing Enzyme Genotyping Systems 21 CFR $\$ 862.2570$ ; Instrumentation for Clinical Multiplex Test Systems   
2. Classification: Class II   
3. Product code: NTI; Drug metabolizing enzyme genotyping systems NSU; Instrumentation for Clinical Multiplex Test Systems   
4. Panel: Toxicology (91) Clinical Chemistry (75)

# H. Intended Use:

1. Intended use(s): See Indications for use below

2. Indication(s) for use: The Verigene® CYP2C19 Nucleic Acid Test (2C19), performed using the sample-to-result Verigene System, is a qualitative multiplexed in vitro diagnostic test for the simultaneous detection and identification of an individual’s CYP45O 2C19 genotype in genomic deoxyribonucleic acid (DNA) obtained from EDTAanticoagulated whole blood samples. The Verigene CYP2C19 Nucleic Acid Test (2C19) is indicated for use in clinical laboratories upon prescription by the attending physician as an aid to clinicians in determining therapeutic strategy for therapeutics that are metabolized by the CYP450 2C19 gene product, specifically $^ { * } 2 , { ^ * } 3$ , and $^ { * } 1 7$ . The Verigene CYP2C19 Nucleic Acid Test (2C19) is not indicated to be used to predict drug response or non-response.

3. Special conditions for use statement(s): For prescription use only

The information provided from this test may supplement decision-making and should only be used in conjunction with routine monitoring by a physician. Because of the variability in the knowledge of clinical utility with specific drugs that are metabolized by CYP2C19, clinicians should use professional judgment in the interpretation of results from this test. Results from this type of assay should not be used in predicting a patient's response to drugs for which the drug metabolizing enzyme activity of the allele, or the drug metabolic pathway, has not been clearly established.

4. Special instrument requirements: Verigene® System

# I. Device Description:

The Verigene CYP2C19 Nucleic Acid Test (2C19) is a molecular assay which detects specific nucleic acid targets in a microarray format. For each of the nucleic acid sequences detected by the 2C19 test, Capture and Mediator oligonucleotides are utilized for gold nanoparticle probe-based endpoint detection. The Capture oligonucleotides bind to a specific portion of the nucleic acid target and are themselves bound onto a substrate in the microarray. The Mediator oligonucleotides bind to a different portion of the same nucleic acid target and also allow binding of a gold nanoparticle probe to a portion of the Mediator oligonucleotide that is complementary to the oligonucleotides attached to the gold nanoparticle probe. Specific silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency. The 2C19 test is performed on the Verigene System, a “sample-to-result”, fully automated, bench-top molecular diagnostics workstation. The Verigene System consists of 2 components: the Verigene Reader and the Verigene Processor $S P$ . The 2C19 test utilizes single-use disposable test consumables and a self-contained Verigene Test Cartridge for each sample tested. For the 2C19 test, the Verigene System allows automated nucleic acid extraction from blood specimens and target detection of specific DNA.

The Reader is the Verigene System’s user interface, which serves as the central control unit for all aspects of test processing and results generation. The Reader’s graphical user interface guides the user through test processing and test results using a barcode scanner. The user inserts the Test Cartridge into the Verigene Processor

$S P$ , which executes the test procedure, automating the steps of (1) sample preparation; i.e.; cell lysis and magnetic bead-based DNA isolation from EDTAanticoagulated whole blood samples and (2) hybridization; i.e.; detection and identification of DNA in a microarray format using gold nanoparticle probe-based technology.

After test processing is complete, the user removes the Test Cartridge from the Processor $S P$ , removes the reagent pack from the substrate holder, and inserts the substrate holder into the Reader for analysis. Light scatter from the capture spots is imaged by the Reader. Presence of the gold nanoparticle probes at a particular location on the substrate is assessed optically. The relative density of probes adhered at each set of capture locations on the substrate is then translated into a signal intensity which is used to assess the relative level of captured DNA target to determine a genotype.

# J. Substantial Equivalence Information:

1. Predicate device name(s): INFINITI CYP2C19 Assay

2. Predicate K number(s): k101683

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Verigene CYP2C19 Test</td><td rowspan=1 colspan=1>INFINITICYP2C19 Assay(k101683)</td></tr><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>IntendedUse/Indications forUse</td><td rowspan=1 colspan=1>In vitro diagnostic test for the identification of apatient&#x27;s CYP450 2C19 genotype in genomic DNAobtained from whole blood samples</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Warnings andPrecautions</td><td rowspan=1 colspan=1>For use in clinical laboratories upon prescription by theattending physician</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Contraindication (s)</td><td rowspan=1 colspan=1>Assay is not intended to be used to predict drugresponse or non-response</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Test Cartridge</td><td rowspan=1 colspan=1>Disposable single-use, multi-chambered fluidiccartridge.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Quality control</td><td rowspan=1 colspan=1>Internal procedural/instrument quality controls; InternalNegative Control, Sample processing control, externalpositive and negative assay controls</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target Mutations</td><td rowspan=1 colspan=1>*2, *3, and *17 genotype of CYP 2C19</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Type of Test</td><td rowspan=1 colspan=1>Genotyping microarray</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Interference</td><td rowspan=1 colspan=1>None observed for bilirubin, triglyceride, cholesterol,and albumin</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Detection Method</td><td rowspan=1 colspan=1>Gold/Ag nanoparticle probe detection of DNA oncomplementary oligo-microarray</td><td rowspan=1 colspan=1>Measures fluorescentsignals of labeledDNA targethybridized to themicroarray</td></tr><tr><td rowspan=1 colspan=1>Sample Type used bythe test system</td><td rowspan=1 colspan=1>EDTA-anticoagulated whole blood</td><td rowspan=1 colspan=1>Genomic DNAextracted fromEDTA-anticoagulatedwhole blood</td></tr><tr><td rowspan=1 colspan=1>DNA ExtractionMethod</td><td rowspan=1 colspan=1>Automated DNA extraction with &quot;blood sample toresult system&quot;</td><td rowspan=1 colspan=1>DNA extractionusing methodvalidated by thelaboratory</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): None

# L. Test Principle:

The Verigene® System is comprised of test consumables and shared instrumentation. All Verigene tests are formatted in self-contained test specific Verigene Test Cartridges which serve to analyze a nucleic acid sample that is presented to them. Nucleic acids are prepared directly from a whole blood specimen using magnetic glass particles in an extraction tray and input automatically into a Test Cartridge inside the Verigene Processor $S P _ { \circ }$ . Test progress is tracked and directed by the

Verigene Reader instrument, which serves as a central control unit for each Verigene System.

# Test Procedure

Genomic DNA is extracted from the white blood cells in a whole blood specimen, fragmented and denatured. This fragmented, single-stranded genomic DNA hybridizes to complementary sequence-specific DNA oligonucleotides, known as capture oligonucleotides, arrayed on the surface of a substrate (glass slide). A second DNA oligonucleotide is then hybridized to the captured genomic DNA that was captured initially. This oligonucleotide is known as a mediator oligonucleotide containing 2 sequence domains: 1 domain is complementary to the genomic DNA target and a second domain is complementary to a common oligonucleotide attached to a signal generating gold nanoparticle probe. After washing away any DNA not affixed to the captures, the probe is exposed to the captured mediator/target compound where it hybridizes to any captured mediators. Presence of the gold nanoparticle probes at a particular location on the substrate is assessed optically. The relative density of probes at each set of captures on the substrate is then translated into a signal intensity which is used to assess relative levels of captured DNA target to determine a genotype.

# Result Interpretation

A genotype result is calculated using a normalized ratio of the signal of wild type capture locations on the microarray to the mutant capture locations on the microarray.

Ratio $=$ (wild type signal – mutant signal) / (wild type signal $^ +$ mutant signal) When the ratio is near 1.0, the wild type signal is higher and the genotype result is wild type. When the ratio is near 0, the wild type signal is equivalent to the mutant signal and the genotype result is heterozygous. When the ratio is near -1.0, the mutant signal is higher and the genotype result is mutant.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The precision study was conducted at 1 testing site (Nanosphere) and the reproducibility study was conducted at 3 sites (2 external and 1 internal sites). All 3 sites tested the same 8 member sample panel consisting of 8 whole blood specimens containing genotypes $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 1 / { ^ { * } 2 }$ , $^ { * } 2 / { ^ { * } 2 }$ , $^ { * } 2 / { ^ { * } 1 7 }$ , $^ { * } 1 / { } ^ { * } 3$ (2 tested), $^ { * } 1 / { } ^ { * } 1 7$ , and $^ { * } 1 7 / { } ^ { * } 1 7$ . Precision and reproducibility results generated by the 2C19 test were compared to results generated by bi-directional sequencing (BDS), confirming the genotype of the panel member before prior to testing.

Intra-laboratory precision was evaluated by testing the 8 member panel in duplicate twice daily by 2 operators for 12 non-consecutive days at 1 testing site, generating a total of 48 replicates per specimen and an overall total of

384 data points. Three lots of test cartridges and 3 lots of extraction trays were used for the study. Of the 384 samples tested, there was $100 \%$ agreement for all panel members as compared to BDS. There were 9 “No Calls” in the study for an initial call rate of $9 7 . 7 \%$ (375/384). A final call rate of $100 \%$ (384/384) was achieved upon repeat testing of all 9 samples. There were 2 Pre-Analysis Errors (Pre-AE) (1 tip test error and 1 tip seal error) for a $0 . 6 \%$ error rate. The precision study results are summarized below:

<table><tr><td rowspan=3 colspan=1>edeoue</td><td rowspan=3 colspan=1>Nn  eeun</td><td rowspan=3 colspan=1>Reem/rm</td><td rowspan=1 colspan=5>Initial Results</td><td rowspan=1 colspan=6>Final Results</td></tr><tr><td rowspan=1 colspan=2>No. ofGenotypeCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>Agreement</td><td rowspan=2 colspan=1>95%One-sidedCI LL</td><td rowspan=1 colspan=3>No. of GenotypeCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>Agree-ment</td><td rowspan=2 colspan=1>95%One-sided CILL</td></tr><tr><td rowspan=1 colspan=1>Corr-ect</td><td rowspan=1 colspan=1>In-correct</td><td rowspan=1 colspan=2>Correct</td><td rowspan=1 colspan=1>Incorrect</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>ALL</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>384</td><td rowspan=1 colspan=1>375</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>384</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.2</td></tr></table>

Reproducibility was evaluated with the 8 member panel tested in duplicate twice daily by 2 operators for 5 non-consecutive days. The testing regimen generated 60 replicates (20 replicates per site) per specimen for a total of 480 data points. Four lots of test cartridges and 4 lots of extraction trays were used for the study. There were 15 initial “No Calls” in the study for an initial call rate of $9 6 . 9 \%$ (465/480). A final call rate of $9 9 . 6 \%$ (478/480) was achieved upon successful repeat testing of 13 samples, where 2 samples did not resolve after re-testing. Of the 480 samples tested, with 2 un-resolved No Call samples, the percent agreement for all panel members compared to BDS was $9 9 . 6 \%$ (478/480). There were 6 Pre-AEs in the study for a $1 . 4 \%$ error rate. The data is summarized below:

<table><tr><td rowspan=3 colspan=1>Ceeoe</td><td rowspan=3 colspan=1>NqurrRese</td><td rowspan=3 colspan=1>Redes/rd</td><td rowspan=1 colspan=5>Initial Results</td><td rowspan=1 colspan=5>Final Results</td></tr><tr><td rowspan=1 colspan=2>No. ofGenotypeCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>Agree-ment</td><td rowspan=1 colspan=1>95%One-sided</td><td rowspan=1 colspan=2>No. ofGenotypeCalls</td><td rowspan=2 colspan=1>NoCalls</td><td rowspan=2 colspan=1>Agree-ment</td><td rowspan=2 colspan=1>95% One-sided CI LL</td></tr><tr><td rowspan=1 colspan=1>Corr-ect</td><td rowspan=1 colspan=1>In-correct</td><td rowspan=1 colspan=1>CI LL</td><td rowspan=1 colspan=1>Corr-ect</td><td rowspan=1 colspan=1>Incorrect</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>85.4</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.1</td></tr><tr><td rowspan=1 colspan=1>ALL</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>480</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>478</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.7</td></tr></table>

$b$ . Linearity/assay reportable range: Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Controls: Positive or negative DNA assay quality controls are not included as part of the assay. When external quality control is needed, the sponsor recommends that previously-characterized clinical specimens (positive control samples to test each genotype and at least 1 negative control wild type) be used with the assay during instrument installation, test validation, verification of the performance of a new lot of consumables, or when the integrity of storage conditions is in question. The package insert outlines QC options for evaluating (1) DNA extraction and hybridization steps together and (2) DNA extraction and hybridization steps separately. For (1), combined QC steps (DNA extraction and hybridization together), fresh samples of known genotype stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ be used within 10 days of collection. In the event that insufficient volume of whole blood for certain genotypes cannot be obtained, QC option (2) can be used where the extraction and hybridization control steps are divided. For the extraction control step, whole blood samples (stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 10 days after collection) should be used where the following results are obtained: DNA concentration range within $1 6 0 ~ \mathrm { { n g / \mu L } }$ and $4 0 0 ~ \mathrm { { n g / \mu L } }$ and absorbance $\mathrm { A } _ { 2 6 0 } / \mathrm { A } _ { 2 8 0 }$ ratio range within 1.7-2.2. To QC the hybridization control step, previously characterized genomic DNA (stored at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ for up to 9 months) with a concentration range of $1 6 0 ~ \mathrm { { n g / \mu L } }$ to 400 $\mathrm { n g / \mu L }$ and absorbance $\mathrm { A } _ { 2 6 0 } / \mathrm { A } _ { 2 8 0 }$ ratio range from 1.7-2.2 should be used and the test results should match the expected genotype result for the sample.

All quality control requirements and testing should be performed in conformance with local, state and/or federal regulations or accreditation requirements.

Imaging Controls: A set of oligonucleotide spots on the substrate is used as Imaging Controls. The imaging controls act as a quality check for proper fluid control and movement between the Test Cartridge and the instrument. If the imaging controls are absent, then a “No Call” is an output and the user is informed that the reason is due to an absence of imaging controls. The imaging control signal is due to signal probe hybridization to an oligonucleotide on the substrate. Signal will only be present if the probe reagent and the signal enhancement reagents are pumped into and held in the hybridization chamber for the specified time period. A Negative Control sequence is included as a control for non-specific binding within the test owing to factors including inadequate hybridization stringency, fluidic failures resulting in high non-specific signals, and other user- or instrument-related failures. A set of background spots on the substrate is used as a background control. The user is informed if the “No Call” result is due to the background signal being too high. Each target oligonucleotide and background location has 6 or more separate spots in the array. The array spots are grouped in at least 3 different locations on the substrate surface. Mathematical algorithms, called “spot filters”, compare the “like” array spots as a set. If individual spots in the set differ, then they are discarded. If the total number of spots in a set becomes less than 4 due to the spot filters, then a “No Call” results. The spot comparison algorithms protect against some hybridization failures; they also protect against fluidics failures such as the hybridization chamber being partially filled.

Reagent Stability: Stability testing of the CYP2C19 test involved taking individual test kit units arranged in the final packaging configuration and placing them in qualified stability chambers in a temperature controlled environment and testing at defined time intervals. For shipping stability, a subset of 2C19 test kit lots were shipped in their intended outer packaging to several sites and returned to Nanosphere and tested for performance. Successful product testing for a minimum of twelve weeks nominal exposure is required prior to marketing of the product.

Sample Stability: The sponsor has demonstrated that specimens (EDTA anticoagulated whole blood samples) can be kept refrigerated (2 to $8 ^ { \circ } \mathrm { C }$ ) 10 days after specimen collection and used with the CYP2C19 test. Previously characterized genomic DNA $1 6 0 ~ \mathrm { { n g / \mu L } }$ to $4 0 0 ~ \mathrm { { n g / \mu L } }$ and $\mathrm { A } _ { 2 6 0 } / \mathrm { A } _ { 2 8 0 }$ from 1.7- 2.2) can be stored at ${ - 2 0 ^ { \circ } C }$ for up to 9 months and used with the CYP2C19 test.

# d. Detection limit:

The upper and lower limits of detection of the assay were assessed by analysis of 7 whole blood samples $( ^ { * } 1 / { ^ * } 1 , ^ { * } 1 / { ^ * } 2 , ^ { * } 1 / { ^ * } 3 , ^ { * } 1 / { ^ * } 1 7 , ^ { * } 2 / { ^ * } 2 , ^ { * } 2 / { ^ * } 1 7$ and

$^ { * } 1 7 / ^ { * } 1 7$ genotypes) with 5 input volumes (between $8 0 0 \mu \mathrm { L }$ and $1 2 0 0 ~ \mu \mathrm { L }$ in $1 0 0 \mu \mathrm { L }$ increments in replicates of 40) to determine the lowest and highest input volumes that would consistently give an initial call rate of $2 9 0 \%$ and $100 \%$ accuracy (vs. BDS). The blood samples were then tested with the assay over a period of 4 days by 2 different users. No repeat testing was performed for samples yielding a “no call” result. A total of 1,400 tests were completed. No incorrect calls were observed for the 1,348 valid test results generated (i.e., $54 \%$ no-calls; $\mathrm { n } { = } 5 2$ ). Overall, the CYP2C19 test detected each genotype consistently $( \geq 9 0 . 0 \% )$ and accurately ( $100 \%$ vs. BDS) for all genotypes tested between the specimen volume ranges of $1 0 0 0 \pm 2 0 0 \mu \mathrm { L }$ . There were no incorrect calls at any of the input levels (data summarized below). The Sponsor states that a significant degradation in test performance was not observed when testing the blood input volumes at either end of the $20 \%$ range around the recommended input value of $1 \mathrm { m l }$ (i.e., $1 0 0 0 \pm 2 0 0 ~ \mu \mathrm { L } )$ . Therefore, $1 . 0 \pm 0 . 0 5 \mathrm { m L }$ of whole blood is the recommended sample volume tolerance for the assay. Furthermore, an additional study evaluating the impact of test performance at blood input volumes of 0, 500 and $7 5 0 ~ \mu \mathrm { L }$ (in replicates of 40 for the $^ { * } 1 / { ^ { * } 2 }$ specimen) demonstrated initial call rates of 0, 67.5 (27/40) and $9 7 . 5 \%$ (39/40) for all 3 additional input volumes tested, respectively. Initial call rate for blood input volumes greater than $8 0 0 ~ \mu \mathrm { L }$ are summarized in the table below:

<table><tr><td colspan="1" rowspan="2">BloodInput(μL)</td><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Initial Call Rate and 95% Confidence Interval("1) for Each Genotypea</td></tr><tr><td colspan="1" rowspan="1">*1/*1</td><td colspan="1" rowspan="1">*1/*2</td><td colspan="1" rowspan="1">*1/*3</td><td colspan="1" rowspan="1">*1/*17</td><td colspan="1" rowspan="1">*2/*2</td><td colspan="1" rowspan="1">*2/*17</td><td colspan="1" rowspan="1">*17/*17</td><td colspan="1" rowspan="1">All</td></tr><tr><td colspan="1" rowspan="1">800</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">92.5%(37/40)79.6-98.4</td><td colspan="1" rowspan="1">97.1%(272/280)94.5-98.8</td></tr><tr><td colspan="1" rowspan="1">900</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">96.8%(271/280)94.0-98.5</td></tr><tr><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">100%(40/40)91.2-100</td><td colspan="1" rowspan="1">98.2%(275/280)95.9-99.4</td></tr><tr><td colspan="1" rowspan="1">1100</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-</td><td colspan="1" rowspan="1">92.5%(37/40)79.6-</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-</td><td colspan="1" rowspan="1">92.5%(37/40)79.6-</td><td colspan="1" rowspan="1">92.5%(37/40)79.6-</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-</td><td colspan="1" rowspan="1">94.6%(265/280)91.3-97.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1200</td><td colspan="1" rowspan="1">95.0%(38/40)83.1-99.4</td><td colspan="1" rowspan="1">92.5%(37/40)79.6-98.4</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">90.0%(36/40)75.1-99.8</td><td colspan="1" rowspan="1">92.5%(37/40)79.6-98.4</td><td colspan="1" rowspan="1">97.5%(39/40)86.8-99.9</td><td colspan="1" rowspan="1">94.6%(265/280)91.3-97.0</td></tr><tr><td colspan="1" rowspan="1">Total ByGenotype</td><td colspan="1" rowspan="1">97.0%(194/200)93.6-98.9</td><td colspan="1" rowspan="1">96.0%(192/200)92.3-98.3</td><td colspan="1" rowspan="1">96.5%(193/200)92.9-98.6</td><td colspan="1" rowspan="1">96.5%(193/200)92.9-98.6</td><td colspan="1" rowspan="1">96.5%(193/200)92.9-98.6</td><td colspan="1" rowspan="1">95.0%(190/200)91.0-97.6</td><td colspan="1" rowspan="1">96.5%(193/200)92.9-98.6</td><td colspan="1" rowspan="1">96.3%(1348/1400)95.2-97.2</td></tr></table>

1) $\% \mathrm { C I }$ values were computed using the SAS 9.1.3, exact binomial method. a) Genotype determined by bi-directional sequencing

The Sponsor has noted in their labeling that the ability of the CYP2C19 test to consistently detect the genotype of a specimen will be negatively impacted at whole blood sample input volumes less than $7 5 0 \mu \mathrm { L }$ , based on the call rates in the study results cited above.

# e. Analytical specificity:

Interference from potential interfering substances was evaluated to assess the genotyping assay performance of the CYP2C19 test in the presence of excess amounts of endogenous substances. Seven EDTA anticoagulated whole blood control samples (each containing individually containing genotypes $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 1 / { ^ { * } 2 }$ , $^ { * } 1 / { } ^ { * } 3$ , $^ { * } 1 / { } ^ { * } 1 7$ , $^ { * } 2 / { ^ { * } 2 }$ , $^ { * } 2 / { ^ { * } 1 } 7$ , an $\mathrm { 1 ^ { * } 1 7 / * 1 7 }$ ) were processed in replicates of 30 using $1 . 0 \mathrm { m L }$ of blood for each replicate. These samples were used as the base matrix into which the 5 interfering substances were spiked and compared to unspiked control samples. The following compounds at the following concentrations were tested: albumin (6000 $\mathrm { m g / d L ) }$ , bilirubin (conjugated and unconjugated, $2 0 \mathrm { m g / d L } )$ , triglycerides $\mathrm { 3 0 0 0 m g / d L } )$ and cholesterol $( 5 0 0 \mathrm { m g / d L } )$ . The concentrations of these endogenous substances were at higher levels for each of the substances than is normally observed in human blood. Sample genotypes were verified by BDS.

In the presence of these substances, each genotype was detected consistently with a final call rate of $100 \%$ (except for $^ { * } 1 / { ^ { * } 3 }$ in the presence of albumin, $^ { * } 2 / { ^ { * } 1 } 7$ in the presence of conjugated bilirubin and $^ { * } 1 7 / ^ { * } 1 7$ in the presence of cholesterol that yielded final call rates of 29/30) and $100 \%$ accuracy vs. BDS. The sponsor concluded that the compounds do not interfere with the performance of the assay.

# Sample carry-over:

To evaluate possible cross-contamination and carry-over between test runs, samples of alternating genotypes were tested consecutively on the same device. Demonstration of a $2 9 0 \%$ no call rate and $2 9 9 \%$ concordance with BDS was the criteria by which to establish that there is no sample carry-over and cross-over contamination between procedures on the Verigene Processor $S P$ using different sample cartridges. Whole blood with $^ { * } 1 / { } ^ { * } 2$ genotype was tested on 10 Verigene instruments, followed immediately by whole blood with $^ { * } 1 / { } ^ { * } 1$ genotype. Next, whole blood with $^ { * } 1 / { } ^ { * } 1 7$ genotype was tested on the same 10 instruments, followed by whole blood with $^ { * } 1 / { } ^ { * } 1$ genotype. This was repeated 3 times, such that 30 replicates were generated for $^ { * } 1 / { ^ { * } 2 }$ , 30 replicates for $^ { * } 1 / { } ^ { * } 1 7$ , and 60 replicates for $^ { * } 1 / { } ^ { * } 1$ on a total of 10 instruments. A total of 120 test runs were generated. All samples were concordance with BDS and there were a total of 4 “no calls” during the study as summarized below:

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Total #Replicates</td><td rowspan=1 colspan=1>Calls</td><td rowspan=1 colspan=1>No Calls</td><td rowspan=1 colspan=1>Total Call Rate (%)</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>96.7</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>93.3</td></tr></table>

The sponsor concluded that the no calls observed in the study reflected the typical performance of the CYP2C19 assay and that no sample carry-over was illustrated between the samples.

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device:

Method comparison studies were conducted at 3 external clinical study sites using according to CLSI EP9-A2 guidelines. Each site utilized 1 Verigene Reader and up to 12 Verigene Processor SP instruments, depending on each site’s specimen enrollment and testing volume. Clinical samples, consisting of EDTA anti-coagulated whole blood samples from the general population of known Asian donors, were collected by 2 third party facilities and shipped to each study site. Specimen tests resulting in a “No Call” or a pre-analysis error (pre-AE) were repeat tested once. Residual DNA from each clinical specimen after testing was sequenced using BDS. A total of 670 samples (125-281 samples per site) were tested. The table below shows the number of samples tested, the initial and final call rates at each of the sites and the combined statistics for each. There were 35 initial no calls of which 34 were successfully repeated. The initial call rate was $9 4 . 8 \%$ (635/670) and the final call rate was $9 9 . 9 \%$ (669/670), as summarized below:

<table><tr><td rowspan=1 colspan=1>Genotype(a)</td><td rowspan=1 colspan=1>No.Tested</td><td rowspan=1 colspan=2>Reps perSample</td><td rowspan=1 colspan=1>No.CorrectGenotypeCalls</td><td rowspan=1 colspan=1>No.Incorrect No CallsCalls</td><td rowspan=1 colspan=1>Incorrect No Calls</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>95% Two-sidedConfidenceInterval LowerLimit</td></tr><tr><td rowspan=1 colspan=1>*1/*1(b)</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.6%</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1c)</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.9%</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9%</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>95.2%</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>89.1%</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.3%</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.4%</td></tr><tr><td rowspan=1 colspan=1>*3/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>81.5%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>670</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>667</td><td rowspan=1 colspan=1>2c</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.7%</td></tr></table>

a) Genotype determined by bi-directional sequencing b) $^ { * } 1 / { } ^ { * } 1$ samples are inferred if they are wild-type for $^ { * 2 }$ , $^ { * 3 }$ and $^ { * } 1 7$ c) CYP2C19 result was no-call for both initial and final test runs; BDS result $^ { * } 1 / { } ^ { * } 2$ .

$b$ . Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity: Not applicable

b. Clinical specificity: Not applicable

c. Other clinical supportive data (when a. and b. are not applicable): The following table lists the alleles recognized by the device, the single nucleotide polymorphisms (SNPs) recognized by the device for each allele, enzyme activity, and references:

<table><tr><td>Allele</td><td>Recognized Polymorphism</td><td>Polymorphisms Associated with Allele</td><td>Enzyme Activity</td><td>References</td></tr><tr><td>*2(b</td><td>19154G&gt;A</td><td>-98T&gt;C; 99C&gt;T; 12122G&gt;A; 12460G&gt;C; 12662A&gt;G; 12834G&gt;C; 19154G&gt;A; 19520A&gt;G; 57740C&gt;G;</td><td>None</td><td>de Morais et al, 19942 Ibeanu et al, 19983</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">80161A&gt;G; 87275G&gt;A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Useaka et al,20054</td></tr><tr><td colspan="1" rowspan="1">*3</td><td colspan="1" rowspan="1">17948G&gt;A</td><td colspan="1" rowspan="1">-889T&gt;G; 12031T&gt;G;12122G&gt;A; 12306G&gt;A;13166T&gt;C; 17948G&gt;A;18911A&gt;G; 80161A&gt;G;80248G&gt;A; 87313A&gt;C</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">de Morais et al,19942Ibeanu et al,19983Fukushima-Useaka et al,20054</td></tr><tr><td colspan="1" rowspan="1">*17</td><td colspan="1" rowspan="1">-806C&gt;T</td><td colspan="1" rowspan="1">-3402C&gt;T; -806C&gt;T;99C&gt;T; 80161A&gt;G</td><td colspan="1" rowspan="1">Increased</td><td colspan="1" rowspan="1">Sim et al, 20065Rudberg et al,20081</td></tr></table>

a) SNPs in bold are the major SNPs/alterations responsible for the phenotype of the corresponding allele and are unique to the mutation b) The $^ { * } 1 0$ mutation is adjacent to the $^ { * 2 }$ mutation, therefore the CYP2C19 Test contains two capture probes for $^ { * } 1 0$ . These probes are used to ensure the accuracy of $^ { * 2 }$ allele detection; however, the $^ { * } 1 0$ mutations are not reported by the assay.

1 Rudberg, I; et al. Impact of the ultrarapid $\mathrm { C Y P 2 C l 9 ^ { * } l 7 }$ allele on serum concentration of escitalopram in psychiatric patients. Clin Pharmacol Ther. 2008 Feb;83(2):322-7   
2 de Morais SM et al. The major genetic defect responsible for the polymorphism of S-mephenytoin metabolism in humans. J Biol Chem. 1994 Jun 3; 269(22):15419-22   
3 Ibeanu GC et al. Identification of new human CYP2C19 alleles(CYP2C19\*6 and $\mathrm { C Y P 2 C l 9 ^ { * } 2 B }$ ) in a Caucasian poor metabolizer of mephenytoin. J Pharmacol Exp Ther. 1998 Sep;286(3):1490-5.   
4 Fukushima-Uesaka H et al. Genetic variations and haplotypes of CYP2C19 in a Japanese population. Drug Metab Pharmacokinet. 2005 Aug;20(4):300-7   
5 Sim SC et al. A common novel CYP2C19 gene variant causes ultra-rapid drug metabolism relevant for the drug response to proton pump inhibitors and anti depressants. Clin Pharmacol Ther. 2006; 79:103-113

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The package insert states:

$\mathrm { C Y P 2 C 1 } 9 ^ { * } 2$ arises from a $\mathrm { G } {  } \mathrm { A }$ transition in exon 5 at position 681 of CYP2C19, creating an aberrant splice site. The $\mathrm { C Y P 2 C } 1 9 ^ { * } 2$ polymorphism is seen in approximately $1 5 \%$ of Caucasians, $1 5 \%$ of African-Americans, and $30 \%$ of Asians. $\mathrm { C Y P 2 C 1 } 9 ^ { * } 3$ arises from a $\mathrm { G } {  } \mathrm { A }$ transition at position 636 in exon 4 of CYP2C19, which produces a truncated protein. The $\mathrm { C Y P 2 C } 1 9 ^ { * } 3$ polymorphism is seen in approximately $< 1 \%$ of Caucasians, $< 1 \%$ of African-Americans, and $8 \%$ of Asians1. A poor metabolizer phenotype (caused by 2 non-functional CYP2C19 alleles is present in $4 \%$ of Caucasians, $5 \%$ of African-Americans, and up to $2 5 \%$ of Asians1.

The $\mathrm { C Y P 2 C l 9 ^ { * } l 7 }$ polymorphism is seen in approximately $1 8 \%$ of Swedes, $1 8 \%$ of Ethiopians, and more than $4 \%$ of Chinese2.

1ARUP Laboratories. Test Sheet: Cytochrome P450 2C19 (CYP2C19) 10 Mutations – for Detection of CYP2C19 Mutations Affecting Drug Metabolism. August 2010.

2Rudberg, I; et al. Impact of the ultrarapid $C Y P 2 C I 9 ^ { * } I 7$ allele on serum concentration of escitalopram in psychiatric patients. Clin Pharmacol Ther. 2008 Feb;83(2):322-7.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.