# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

# E. Applicant:

# G. Regulatory Information:

1. Regulation section: 21 CFR §862.3360, Drug Metabolizing Enzyme Genotyping Systems

2. Classification: Class II

3. Product code: NTI, Drug Metabolizing Enzyme Genotyping Systems

4. Panel: Toxicology (91)

# H. Intended Use:

1. Intended use(s): See Indications for use below

2. Indication(s) for use:

The xTAG $^ \mathrm { \textregistered }$ CYP2D6 Kit v3 is a device used to simultaneously detect and identify a panel of nucleotide variants found within the highly polymorphic CYP2D6 gene located on chromosome 22 from genomic DNA extracted from an EDTA or citrate anticoagulated whole blood sample. This kit can also identify gene rearrangements associated with the deletion $( ^ { * } 5 )$ and duplication genotypes. The xTAG $\textsuperscript { \textregistered }$ CYP2D6 Kit v3 is a qualitative genotyping assay which can be used as an aid to clinicians in determining therapeutic strategy for therapeutics that are metabolized by the CYP2D6 gene product. This kit is not indicated for stand- alone diagnostic purposes. This test is not intended to be used to predict drug response or non-response.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Luminex $\overset { \vartriangle } { \boldsymbol { \mathbb { B } } } 1 0 0 / 2 0 0 ^ { \mathrm { T M } }$ instrument

# I. Device Description:

The xTAG $^ \mathrm { \textregistered }$ CYP2D6 Kit v3 includes the following components:

• xTAG 2D6 v3 PCR Mix A (includes dNTPs)   
• xTAG 2D6 v3 PCR Mix B (includes dNTPs) xTAG 2D6 v3 ASPE Mix (includes dNTPs) xTAG 2D6 v3 Bead Mix xTAG Reporter Buffer xTAG Shrimp Alkaline Phosphatase xTAG Exonuclease I xTAG Streptavidin, R-Phycoerythrin conjugate   
• Platinum TFI Exo(-) DNA Polymerase and associated $\mathrm { M g C l } _ { 2 }$ and PCR buffer   
• xTAG $^ \mathrm { \textregistered }$ Data Analysis Software (TDAS) CYP2D6   
• xTAG Hot Start Taq DNA Polymerase and 10x Taq Buffer

# J. Substantial Equivalence Information:

1. Predicate device name(s): Luminex xTAG CYP2D6 Kit v3

2. Predicate 510(k) number(s): k093420

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Candidate Device:xTAG® CYP2D6 Kit v3(k130189)</td><td rowspan=1 colspan=1>Predicate Device:xTAG® CYP2D6 Kit v3(k093420)</td></tr><tr><td rowspan=1 colspan=1>Intended Use andIndications for use</td><td rowspan=1 colspan=1>Intended to identify apatient&#x27;s CYP2D6 genotypefrom genomic DNAextracted from a wholeblood sample.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>EDTA or citrateanticoagulated whole blood</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Limitation</td><td rowspan=1 colspan=1>This test is not intended tobe used to predict drugresponse or non-response</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Luminex® 100/200TM</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Target Gene</td><td rowspan=1 colspan=1>CYP2D6</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>CYP2D6 star genotypesdetected by the test</td><td rowspan=1 colspan=1>*1, *2, *3, *4, *5, *6, *7,*8, *9, *10, *11, *15, *17,*29, *35, *41</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Amplification Method</td><td rowspan=1 colspan=1>Multiplex PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Detection Method</td><td rowspan=1 colspan=1>Fluorescence based</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Candidate Device:xTAG® CYP2D6 Kit v3(k130189)</td><td colspan="1" rowspan="1">Predicate Device:xTAG® CYP2D6 Kit v3(k093420)</td></tr><tr><td colspan="1" rowspan="1">Buffer Reagent</td><td colspan="1" rowspan="1">xTAG® Reporter Bufferalready at reactionconcentration (pre-diluted);dilution no longer required</td><td colspan="1" rowspan="1">xTAG® 10X Bufferrequired dilution by users</td></tr><tr><td colspan="1" rowspan="1">ASPE Primer MixComponent</td><td colspan="1" rowspan="1">xTAG® 2D6 v3 ASPEPrimer Mix has been</td><td colspan="1" rowspan="1">xTAG® 2D6 v3 ASPEPrimer Mix</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">modified. The concentrationof one primer set wasreduced and a substituteprimer set for 3183G&gt;Awas included.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TDAS CYP2D6 Software</td><td colspan="1" rowspan="1">Minor adjustments to allelethresholds for: 138insT,1661G&gt;A, 1707T&gt;del,18466G&gt;A, 2988G&gt;A,3183G&gt;A, 4180G&gt;C, andDUP variant; version 1.01Build 13D221</td><td colspan="1" rowspan="1">First Release</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guidance Document: Drug Metabolizing Enzyme   
Genotyping System- Guidance for Industry and FDA Staff   
Class II Special Controls Guidance Document: Instrumentation for Clinical Mulitplex   
Test Systems- Guidance for Industry and FDA Staff   
Guidance for the Content of Premarket Submissions for Software Contained in   
Medical Devices   
CLSI Guideline MM13-PE: Collection, Transport, Preparation, and Storage of   
Specimens for Molecular Methods   
CLSI Guideline EP05-A2: Evaluation of Precision Performance of Clinical   
Chemistry Devices   
CLSI Guideline EP07-A2E: Interference Testing in Clinical Chemistry   
CLSI Guideline EP12-A: User Protocol for Evaluation of Qualitative Test   
Performance   
CLSI Guideline EP17-A: Protocols for Determining Limits of Detection and Limits of   
Quantitation

# L. Test Principle:

For each genomic sample being tested, two separate PCR reactions are performed. PCR-A produces an alpha fragment used to detect the mutations in the table below, as well as a duplication amplimer which indicates the presence of the duplication genotype. PCR-B produces a beta fragment used to detect the mutations in the table below, as well as a deletion amplimer indicative of the deletion genotype.

Following PCR amplification, the two reactions (PCR-A and PCR-B) are pooled. To enable efficient incorporation of biotin-dCTP during the ASPE reaction, the pooled PCR product is treated with Shrimp Alkaline Phosphatase to dephosphorylate any remaining nucleotides (particularly dCTP), and with Exonuclease I to degrade any primers left over from the PCR reaction.

ASPE is then carried out using universally-tagged primers supplied in the ASPE primer mix. An aliquot of the ASPE reaction is hybridized with the universal array (beat Mix) in the presence of the hybridization buffer and incubated with Streptavidin, R-Phycoerythrin conjugate (reporter solution).

Samples are read on the Luminex 100 or 200 Instrument and a signal is generated for each of the loci as well as for the duplication and deletion amplimers, if present. These fluorescence values are then analyzed to determine whether the wild-type/mutant allele for each of the loci has been detected, or whether the samples carry an allele(s) with the deletion or duplication.

<table><tr><td rowspan=1 colspan=1>Star (*) Genotype</td><td rowspan=1 colspan=2>Mutations and Polymorphisms detected by xTAG® CYP2D6</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PCR A</td><td rowspan=1 colspan=1>PCR B</td></tr><tr><td rowspan=1 colspan=1>*1</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>*2</td><td rowspan=1 colspan=1>-1584 C&gt;G, 1661G&gt;C</td><td rowspan=1 colspan=1>2850C&gt;T, 4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2549A&gt;del</td></tr><tr><td rowspan=1 colspan=1>*4</td><td rowspan=1 colspan=1>100C&gt;T, 1661G&gt;C, 1846G&gt;A</td><td rowspan=1 colspan=1>2850C&gt;T, 4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Deletion</td></tr><tr><td rowspan=1 colspan=1>*6</td><td rowspan=1 colspan=1>1707T&gt;del</td><td rowspan=1 colspan=1>4180C&gt;C</td></tr><tr><td rowspan=1 colspan=1>*7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2935A&gt;C</td></tr><tr><td rowspan=1 colspan=1>*8</td><td rowspan=1 colspan=1>1661G&gt;C, 1758G&gt;T</td><td rowspan=1 colspan=1>2850C&gt;T, 4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2613delAGA</td></tr><tr><td rowspan=1 colspan=1>*10</td><td rowspan=1 colspan=1>100C&gt;T, 1661G&gt;C</td><td rowspan=1 colspan=1>4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*11</td><td rowspan=1 colspan=1>883G&gt;C, 1661 G&gt;C</td><td rowspan=1 colspan=1>2850C&gt;T, 4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*15</td><td rowspan=1 colspan=1>138InsT</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>*17</td><td rowspan=1 colspan=1>1023C&gt;T, 1661G&gt;C</td><td rowspan=1 colspan=1>2850C&gt;T, 4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*29</td><td rowspan=1 colspan=1>1659G&gt;A, 1661G&gt;C</td><td rowspan=1 colspan=1>2850C&gt;T, 3183G&gt;A,4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*35</td><td rowspan=1 colspan=1>-1584C&gt;G, 31G&gt;A, 1661G&gt;C</td><td rowspan=1 colspan=1>2850C&gt;T, 4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>*41</td><td rowspan=1 colspan=1>1661G&gt;C</td><td rowspan=1 colspan=1>2850C&gt;T, 2988G&gt;A,4180G&gt;C</td></tr><tr><td rowspan=1 colspan=1>DUP</td><td rowspan=1 colspan=1>Duplication</td><td rowspan=1 colspan=1></td></tr></table>

Nucleotide changes that define the star genotype $( ^ { * } )$ are shown in bold font.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Please see k093420 for the evaluation of lot to lot and operator to operator repeatability. For the modified device, a condensed repreatability study was performed blinded using three lots of kits. Three operators each performed one run for three non-consecutive days, with each run containing two replicates per run, resulting in 18 replicates for each sample set in this study. Each repeatability sample set contained 11 unique samples consisting of whole blood samples collected in tubes containing either EDTA or citrate, and pre-extracted genomic DNA samples. All samples were sequenced using bi-directional DNA sequencing to establish the genotype.

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Total # ofReplicates</td><td rowspan=1 colspan=1>TotalCorrectCalls</td><td rowspan=1 colspan=1># NoCalls</td><td rowspan=1 colspan=1>#IncorrectCalls</td><td rowspan=1 colspan=1>95% LCB</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>*1/*4</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*1/*5</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*1/*41</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*2/*4 DUP</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*4/*35</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>76.23</td><td rowspan=1 colspan=1>94.4</td></tr><tr><td rowspan=1 colspan=1>*5/*17</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*17/*29</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*29/*29</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>*35/*41</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Total # ofReplicates</td><td rowspan=1 colspan=1>198</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.63</td><td rowspan=1 colspan=1>99.5</td></tr></table>

One sample resulted in a no call, resulting in $9 9 . 5 \%$ overall repeatability. A root cause investigation for the no-call was carried out. After re-running this sample, the overall reproducibility was $100 \%$ .

b. Linearity/assay reportable range: Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Quality control materials are not provided with the kit. The package insert states that previously characterized clinical samples or commercially available 2D6 controls are recommended for quality control testing. All quality control requirements and testing should be performed in conformance with local, state, and/or federal regulations or requirement.

See k093240 for stability information. The current claimed shelf-life for the xTAG CYP2D6 Kit v3 is 12 months. Repeated freeze-thaw cycles (up to 6) are acceptable.

# d. Detection limit:

The lower limit of detection and range of the $\mathbf { x T A G } ^ { \mathbb { ( B ) } }$ CYP2D6 Kit v3 was assessed by analyzing serial dilutions of three genomic DNA samples and three whole blood samples. The six samples used in the study had the following genotypes: $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 2 / { ^ { * } } 4$ DUP, $^ { * } 5 / { ^ { * } 1 7 }$ , $^ { * } 1 7 / { ^ { * } 1 7 }$ , $^ { * } 1 7 / { ^ { * } 2 9 }$ , and $^ { * } 2 9 / { ^ { * } 2 9 }$ . The data were acquired by two different Luminex 100/200 instruments.

Forty replicates of the six samples were run at six concentrations within the assay range, which generated 1400 data points across all samples and concentrations. The positive concordance was determined from the number of correct genotype calls. The results were as follows:

<table><tr><td rowspan=1 colspan=1>[InputDNAJ(ng/μL)</td><td rowspan=1 colspan=1>TotalInputDNA(ng)</td><td rowspan=1 colspan=1>NumberofSamples</td><td rowspan=1 colspan=1>NumberofReplicates</td><td rowspan=1 colspan=1>Number ofSampleswithCorrectGenotyping</td><td rowspan=1 colspan=1>PositiveConcordance</td><td rowspan=1 colspan=1>LowerBoundof 95%CI</td><td rowspan=1 colspan=1>UpperBoundof 95%CI</td></tr><tr><td rowspan=1 colspan=1>~100.0</td><td rowspan=1 colspan=1>~600.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>240/240 =100%</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>300.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>240/240=100%</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>60.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>240/240 =100%</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>240/240 =100%</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>238/240 =99.2%</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>99.97%</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>209/240 =87.1%</td><td rowspan=1 colspan=1>82.2%</td><td rowspan=1 colspan=1>90.8%</td></tr></table>

The limit of detection for the system was established at $2 \mathrm { n g / \mu L }$ ( $1 2 { \mathrm { n g } }$ input DNA).

However, it is recommended that in order to mitigate the potential risk of a user using a low concentration sample, the recommended lower bound of the assay be $4 . 0 \mathrm { n g / \mu L }$ (24.0 ng total input DNA). The claimed assay range for the xTAG CYP2D6 Kit v3 with refined ASPE formulation is from $4 . 0 \mathrm { n g / \mu L }$ to $3 0 0 ~ \mathrm { { n g / \mu L } }$ .

e. Analytical specificity: See k093420 for specificity information.

f. Assay cut-off: Not Applicable

2. Comparison studies:

a. Method comparison with predicate device:

Method comparison studies were performed using bi-directional dideoxy-DNA sequencing as the comparator for the $\mathbf { x T A G } ^ { \mathbb { ( B ) } }$ CYP2D6 Kit v3 with the reformulated ASPE primer mix. DNA sequence analysis for genotype confirmation was performed for 279 clinical samples that were tested and collected in either EDTA or Citrate analyzed by xTAG® CYP2D6 Kit v3 with the reformulated ASPE primer mix. The agreement between the $\mathrm { x T A G } ^ { \mathbb { B } } \mathrm { C Y P 2 D 6 } \mathrm { K i t } \mathrm { v } 3$ with the reformulated ASPE primer mix and bi-directional dideoxy sequencing was $100 \%$ for the CYP2DG alleles. In addition, the agreement between results from the $\mathbf { x T A G } ^ { \mathbb { ( B ) } }$ CYP2D6 Kit v3 with the reformulated ASPE primer mix and the previous version of the xTAG® CYP2D6 Kit v3 was $1 0 0 . 0 \%$ . The results on a “per allele” basis are summarized below.

<table><tr><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1"># ofAllelesSequenced</td><td colspan="1" rowspan="1"># ofHomozygousSamples</td><td colspan="1" rowspan="1"># ofHeterozygousSamples</td><td colspan="1" rowspan="1"># ofCorrectCalls</td><td colspan="1" rowspan="1"># ofIncorrectCalls</td><td colspan="1" rowspan="1"># ofNocalls</td><td colspan="1" rowspan="1">PercentAgreement</td></tr><tr><td colspan="1" rowspan="1">*1</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*2</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*3</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*4</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*5</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*6</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*7</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*9</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*10</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*11</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*15</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*17</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*29</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*35</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">*41</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">504</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">450</td><td colspan="1" rowspan="1">504</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr></table>

The percent agreement for genotype detection of the $\mathrm { x T A G } ^ { \mathbb { B } } \mathrm { C Y P 2 D 6 } \mathrm { K i t } \mathrm { v } 3$ was calculated by determining the percentage of tested samples with the correct genotype assigned, compared to the total number of samples of that genotype. The results on a “per genotype” basis are summarized below.

<table><tr><td colspan="1" rowspan="1">CYP2D6Genotype</td><td colspan="1" rowspan="1">TotalUniqueSamples</td><td colspan="1" rowspan="1"># CorrectCalls</td><td colspan="1" rowspan="1"># ofMiscalls</td><td colspan="1" rowspan="1"># ofNoCalls</td><td colspan="1" rowspan="1">PercentAgreement</td></tr><tr><td colspan="1" rowspan="1">*1/*1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*1DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*2DUP</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*3DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*4DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*5DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*6</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*10</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*10DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*15</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*17</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*29</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*29DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*35</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*35DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*1/*41</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*2DUP</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*4DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*9</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*10</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*10DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*11</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*17</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*17DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*29</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*35</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*41</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*2/*41DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*3/*3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*3/*4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*3/*5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*3/*9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*3/*35</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*4DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*5DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*9DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*10</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*10DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*17</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*17DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*29</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*29DUP</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*35</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*35DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*4/*41</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*10</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*17</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*29</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*35</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*5/*41</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*6/*6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*6/*29</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*6/*35</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*9/*9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*9/*35</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*9/*41</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*10/*10</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*10/*17</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*10/*29</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*10/*35</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*10/*41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*15/*41</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*17/*17</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*17/*29</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*17/*29DUP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*17/*35</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*17/*41</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*29/*29</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*29/*35</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*29/*41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*35/*35</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*35/*41</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">*41/*41</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">279</td><td colspan="1" rowspan="1">279</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.00%</td></tr></table>

b. Matrix comparison: Please see k093420 for the matrix comparison information for the xTAG CYP2D6 Kit v3.

3. Clinical studies:

a. Clinical Sensitivity:

Not Applicable

b. Clinical specificity: Not Applicable   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

<table><tr><td rowspan=1 colspan=1>Final TDASOutput (*genotype or *allele)</td><td rowspan=1 colspan=1>SNPs thatxTAGCYP2D6 v3Kit detects</td><td rowspan=1 colspan=1>Frequency inthe U.S.CaucasianPopulation</td><td rowspan=1 colspan=1>Frequency inthe AfricanAmericanPopulation</td><td rowspan=1 colspan=1>PredictedEnzymeActivity</td><td rowspan=1 colspan=1>Referencewhere the effectof the genotypeon drugmetabolism isdescribed</td></tr><tr><td rowspan=1 colspan=1>*1</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>37 - 40%</td><td rowspan=1 colspan=1>29- 35%</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>(Kimura, Umenoet al. 1989;Marez, Legrandet al. 1997;Sachse,Brockmoller etal. 1997)</td></tr><tr><td rowspan=4 colspan=1>*2</td><td rowspan=4 colspan=1>-1584C&gt;G,1661G&gt;C,2850C&gt;T,4180G&gt;C</td><td rowspan=4 colspan=1>26 - 33%</td><td rowspan=1 colspan=1>18 - 27%</td><td rowspan=3 colspan=1>Normal</td><td rowspan=4 colspan=1>(Johansson,Lundqvist et al.1993; Panserat,Mura et al.1994; Marez,Legrand et al.1997;Raimundo,Fischer et al.2000;Sakuyama,Sasaki et al.2008)</td></tr><tr><td rowspan=2 colspan=1></td></tr><tr><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="1" rowspan="1">*3</td><td colspan="1" rowspan="1">2549A&gt;del</td><td colspan="1" rowspan="1">1%</td><td colspan="1" rowspan="1">0.2%-0.6%</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Kagimoto,Heim et al.1990; Marez,Legrand etal. 1997</td></tr><tr><td colspan="1" rowspan="1">*4</td><td colspan="1" rowspan="1">100C&gt;T,1661G&gt;C,1846G&gt;A,4180G&gt;C,2850C&gt;T</td><td colspan="1" rowspan="1">18 - 20%</td><td colspan="1" rowspan="1">6 - 9%</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Gough, Miles etal. 1990;Hanioka,Kimura et al.1990; Kagimoto,Heim et al.1990;Sachse etal, 1997; Marezet al, 1997)</td></tr><tr><td colspan="1" rowspan="1">*5</td><td colspan="1" rowspan="1">deletion</td><td colspan="1" rowspan="1">2 - 4%</td><td colspan="1" rowspan="1">6 - 7%</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Gaedigk, Blumet al. 1991;Steen, Molven etal. 1995)</td></tr><tr><td colspan="1" rowspan="1">*6</td><td colspan="1" rowspan="1">1707T&gt;del,4180G&gt;C</td><td colspan="1" rowspan="1">1%</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Evert, Griese etal. 1994;Saxena, Shaw etal. 1994; Daly,Leathart et al.1995; Marez,Legrand et al.1997</td></tr><tr><td colspan="1" rowspan="1">*7</td><td colspan="1" rowspan="1">2935A&gt;C1661G&gt;C,1758G&gt;T,2850C&gt;T,4180G&gt;C</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Evert, Griese etal. 1994)</td></tr><tr><td colspan="1" rowspan="1">*8</td><td colspan="1" rowspan="1">1661G&gt;C,1758G&gt;T,2850C&gt;T,4180G&gt;C</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Broly, Marez etal. 1995)</td></tr><tr><td colspan="1" rowspan="1">*9</td><td colspan="1" rowspan="1">2613delAGA</td><td colspan="1" rowspan="1">2 - 3%</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">Reduced</td><td colspan="1" rowspan="1">(Tyndale,Aoyama et al.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1991; Broly andMeyer 1993)</td></tr><tr><td colspan="1" rowspan="1">*10</td><td colspan="1" rowspan="1">100C&gt;T,1661G&gt;C,4180G&gt;C</td><td colspan="1" rowspan="1">2 - 8%</td><td colspan="1" rowspan="1">0.3%-0.4%</td><td colspan="1" rowspan="1">Reduced</td><td colspan="1" rowspan="1">(Yokota,Tamura et al.1993;Johansson,Oscarson et al.1994; Ishiguro,Kubota et al.2004;Sakuyama,Sasaki et al.2008)</td></tr><tr><td colspan="1" rowspan="1">*11</td><td colspan="1" rowspan="1">883G&gt;C,1661G&gt;C,2850C&gt;T,4180G&gt;C</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">Not Known</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Marez, Sabbaghet al. 1995)</td></tr><tr><td colspan="1" rowspan="1">*15</td><td colspan="1" rowspan="1">138insT</td><td colspan="1" rowspan="1">Not known</td><td colspan="1" rowspan="1">Not Known</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">(Sachse,Brockmoller etal. 1996)</td></tr><tr><td colspan="1" rowspan="1">*17</td><td colspan="1" rowspan="1">1023C&gt;T,1661G&gt;C,2850C&gt;T,4180G&gt;C</td><td colspan="1" rowspan="1">0.2-0.3%</td><td colspan="1" rowspan="1">15 - 26%</td><td colspan="1" rowspan="1">Reduced</td><td colspan="1" rowspan="1">(Masimirembwa,Persson et al.1996; Oscarson,Hidestrand et al.1997)</td></tr><tr><td colspan="1" rowspan="1">*29</td><td colspan="1" rowspan="1">1659G&gt;A,1661G&gt;C,2850C&gt;T,3183G&gt;A,4180G&gt;C</td><td colspan="1" rowspan="1">Not known3</td><td colspan="1" rowspan="1">Not known3</td><td colspan="1" rowspan="1">Reduced</td><td colspan="1" rowspan="1">(Marez, Legrandet al. 1997;Wennerholm,Johansson et al.2001;Wennerholm,Dandara et al.2002)</td></tr><tr><td colspan="1" rowspan="1">*35</td><td colspan="1" rowspan="1">-1584C&gt;G,31G&gt;A,1661G&gt;C,2850C&gt;T,4180G&gt;C</td><td colspan="1" rowspan="1">7.4%4</td><td colspan="1" rowspan="1">1%4</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">(Marez, Legrandet al. 1997;Gaedigk, Ryderet al. 2003)</td></tr><tr><td colspan="1" rowspan="1">*41</td><td colspan="1" rowspan="1">1661G&gt;C,2850C&gt;T,2988G&gt;A,4180G&gt;C</td><td colspan="1" rowspan="1">9%4</td><td colspan="1" rowspan="1">11%4</td><td colspan="1" rowspan="1">Reduced</td><td colspan="1" rowspan="1">(Raimundo,Fischer et al.2000;Raimundo,Toscano et al.2004)</td></tr></table>

1,2 Bradford, L. D. (2002). "CYP2D6 allele frequency in European Caucasians, Asians, Africans and their descendants." Pharmacogenomics 3(2): 229-43.   
3 The frequency of the $^ { \ast } 2 9$ allele in the U.S. population is not known; however, it is very common in Tanzanian Africans, with an allele frequency of 20 percent (Wennerholm, A., I. Johansson, et al. (2001). "Characterization of the $\mathrm { C Y P 2 D 6 ^ { * } 2 9 }$ allele commonly present in a black Tanzanian population causing reduced catalytic activity." Pharmacogenetics 11 (5): 417-27).   
4 Gaedigk, A., D. L. Ryder, et al. (2003). "CYP2D6 poor metabolizer status can be ruled out by a single genotyping assay for the -1584G promoter polymorphism." Clinical Chemistry 49(6 Pt 1): 1008-11.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.