# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

Modification of a previously cleared device; addition of new collection devices, ORAcollect•DX (models OCD-100 and OCD-100A - k152464) and Oragene•Dx (models OGD-510, OGD-600, OGD-610, and OGD-675 - k152556)

# C. Measurand:

Genotype of Cytochrome P450 2C9 (CYP450 2C9) and Vitamin K epoxide reductase complex subunit I (VKORC1)

. Type of Test: Qualitative genetic test for single nucleotide polymorphism detection

E. Applicant: Genmark Diagnostics, Incorporated

F. Proprietary and Established Names: eSensor Warfarin Sensitivity Saliva Test

# G. Regulatory Information:

1. Regulation section:

21 CFR §862.3360 – Drug Metabolism Enzyme Genotyping Test   
21 CFR §864.7750 – Prothrombin Time Test   
21 CFR $\ S 8 6 2 . 2 5 7 0$ - Instrumentation for Clinical Multiplex Test Systems

2. Classification: Class II

3. Product code: ODW - Cytochrome P450 2C9 (CYP450 2C9) Drug Metabolizing Enzyme Genotyping System ODV - Vitamin K epoxide reductase complex subunit 1 (VKORC1) Genotyping System NSU - Instrumentation for Clinical Multiplex Test Systems

4. Panel: Toxicology (91), Hematology (81), Chemistry (75)

# H. Intended Use:

1. Intended use(s):

See indications for use below.

2. Indication(s) for use:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test is an in vitro diagnostic for the detection and genotyping of the $^ { * 2 }$ and $^ { * 3 }$ alleles of the cytochrome P450 (CYP450) 2C9 gene locus and the Vitamin K epoxide reductase C1 (VKORC1) gene promoter polymorphism $\left( - 1 6 3 9 \mathrm { G } > \mathrm { A } \right)$ from genomic DNA extracted from human saliva samples collected using the Oragene $\ @ \bullet \mathrm { \mathbf { D } } \mathbf { x }$ and ORAcollect $\textsuperscript { \textregistered }$ ·Dx devices, as an aid in the identification of patients at risk for increased warfarin sensitivity.

3. Special conditions for use statement(s):

For prescription use only

Samples for this test should only be collected with the Oragene•Dx collection devices cleared under k110701, k152556, and k152464: models OCD-100, OCD100A, OGD-500, OGD-510, OGD-575, OGD-600, OGD-610, OGD-675, OXD525, and OYD-500

4. Special instrument requirements:

eSensor $\textsuperscript { \textregistered }$ XT-8 Instrument (cleared with the test system under k073720)

# I. Device Description:

The kit consists of the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test cartridge, the eSensor $^ \mathrm { \textregistered }$ Warfarin Sensitivity Saliva Test amplification reagents (including PCR mix and DNA polymerase), the eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test detection reagents (including exonuclease, probes and hybridization buffer ingredients) and the eSensor $\textsuperscript { \textregistered }$ XT-8 System. One eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test Kit has sufficient materials for 24 tests.

The eSensor $\textsuperscript { \textregistered }$ XT-8 System uses a solid-phase electrochemical method for determining the genotype of a defined panel of polymorphisms. The genotype of each polymorphism is determined by voltammetry, which generates specific electrical signals from the allelespecific signal probes.

# J. Substantial Equivalence Information:

1. Predicate device name(s): eSensor Warfarin Sensitivity Saliva Test

2. Predicate K number(s): k110786

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Proposed Device</td><td rowspan=1 colspan=1>Predicate Device(k110786)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>For the detection and genotyping ofthe *2 and *3 alleles of thecytochrome P450 (CYP450) 2C9 genelocus and the Vitamin K epoxidereductase C1 (VKORC1) genepromoter polymorphism (-1639G&gt;A)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Indication forUse</td><td rowspan=1 colspan=1>as an aid in the identification ofpatients at risk for increased warfarinsensitivity</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DeviceComponents</td><td rowspan=1 colspan=1>Test cartridge, amplification reagents(including PCR mix and DNApolymerase), detection reagents(including exonuclease, probes andhybridization buffer ingredients) andthe eSensor® XT-8 System.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Proposed Device</td><td rowspan=1 colspan=1>Predicate Device(k110786)</td></tr><tr><td rowspan=1 colspan=1>SpecimenCollection Kit</td><td rowspan=1 colspan=1>ORAcollect ® Dx Device: ODC-100and OCD-100A</td><td rowspan=1 colspan=1>Oragene ® Dx Device</td></tr><tr><td rowspan=1 colspan=1>DNAExtractionMethod</td><td rowspan=1 colspan=1>Addition of QIAAMP DNA Mini Kitextraction method; specified inAttachment B of the package insert</td><td rowspan=1 colspan=1>Only Manual ethanolextractionmethod; specified inAttachment A of thepackage insert</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

None cited.

# L. Test Principle:

The eSensor $\textsuperscript { \textregistered }$ Warfarin Sensitivity Saliva Test uses an electrochemical detection based microarray method for determining the genotype of a defined panel of polymorphisms from purified genomic DNA isolated from human saliva. This

method was cleared under k073720 using genomic DNA from blood as the sample type. In the process, regions of the genome containing the polymorphisms of interest are amplified by PCR, and the resulting double stranded PCR amplicons are digested with exonuclease lambda to generate single stranded target DNA which is then mixed with a hybridization solution containing a pair of allele-specific oligonucleotide signal probes for each polymorphism. Each signal probe within the pair is labeled with a genotype-specific ferrocene derivative.

The mixture of amplified target DNA sample and signal buffer is loaded onto a test cartridge containing single-stranded oligonucleotide capture probes that are covalently bound to gold-plated electrodes. The cartridge is then inserted into the XT8 Instrument. During the hybridization in the XT-8 Instrument, each target DNA binds to a capture probe. Each pair of working electrodes on the array contains a different capture probe.

The signal and capture probes are designed with sequences complementary to immediately adjacent regions on the corresponding target DNA sequence and so both signal and capture probes bind to complementary sequences on the target DNA. In this manner, a three-member complex is formed among capture probe, target, and signal probe based on sequence-specific hybridization. This process brings the end of the signal probe containing electrochemically active ferrocene labels into close proximity to the electrode surface.

Hybridization of the three-member complex at the electrode surface and subsequent application of an excitation voltage causes the ferrous ion in each ferrocene group to undergo cyclic oxidation and reduction at its characteristic redox potential, leading to loss or gain of an electron, and the generation of an alternating current at the electrode surface that is measured using voltammetry. Higher-order harmonic signal analysis also facilitates discrimination of ferrocene-dependent faradaic current from background capacitive current. Signals from the ferrocene labels are detected and measured by instrument software, and the ratio of signals from the different labels allows identification of genotype. Genotyping boundaries and signal threshold for each polymorphism are pre-programmed into instrument software, and genotypes are called by comparison of the signal ratio observed for an unknown sample to the SNPspecific genotyping boundaries and signal threshold. Sequential analysis of each electrode allows genotyping of multiple mutations or polymorphisms.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Three reproducibility studies were performed as part of the clearance of k152464 and referenced to support this clearance.

An additional reproducibility study was conducted by Genmark to establish the DNA collection with the new tube type in k152556. Three samples (collected using three lots of Oragene•Dx format OGD-500) from each of ten donors, covering all possible genotypes of $\mathrm { C Y P 2 C 9 ^ { * } } 2$ , $\mathrm { C Y P 2 C 9 ^ { * } } 3$ , and VKORC1, were tested in triplicate by four different operators at three different sites. Each operator extracted DNA from each sample using the QIAamp mini kit alcohol precipitation method, followed by determination of DNA concentration and A260/A280 ratio for all samples by an independent operator at one of the sites. Four operators at three sites tested the extracted DNA samples on the eSensor Warfarin Sensitivity Test. Genotyping data was evaluated after one run, and also after re-testing. First run failures were a result of five no-call results, one incorrect call, and two invalidated runs (46 samples) due to failure of the negative control. There was $100 \%$ agreement between the eSensor results and DNA sequencing in the second run.

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>SNP</td><td rowspan=1 colspan=1>SamplesTested</td><td rowspan=1 colspan=1>First-passCorrectCalls</td><td rowspan=1 colspan=2>First-passincorrectcalls</td><td rowspan=1 colspan=1>First-passno-calls</td><td rowspan=1 colspan=1>FinalCorrectCalls</td><td rowspan=1 colspan=1>Final %agreement</td><td rowspan=1 colspan=1>95%LCB</td></tr><tr><td rowspan=6 colspan=1>Site 1</td><td rowspan=3 colspan=1>Operator1</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.6%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.6%</td></tr><tr><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.6%</td></tr><tr><td rowspan=3 colspan=1>Operator2</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.6%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.6%</td></tr><tr><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.6%</td></tr><tr><td rowspan=3 colspan=1>Site 2</td><td rowspan=3 colspan=1>Operator3</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=2>1a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=2>1a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=3 colspan=1>Site 3</td><td rowspan=3 colspan=1>Operator4</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=3 colspan=1>Combined</td><td rowspan=3 colspan=1>Combined</td><td rowspan=1 colspan=1>2C9*2</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>302</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>51b</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=1>2C9*3</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>51b</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=1>VKOR</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>302</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>51b</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.2%</td></tr></table>

a Same sample.

b 46 first-run no-calls were due to two runs (23 samples per run) invalidated due to failure of the negative control. The other five first-pass no-calls were low signal for the $2 \mathrm { C } 9 ^ { \ast } 2$ allele (three), positive control failure (one) and contradictory score at the $2 \mathrm { C } 9 ^ { \ast } 3$ allele (one).

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): See k073720.

d. Detection limit:

The recommended input for the eSensor Warfarin Sensitivity Saliva Test is 10 ng of genomic DNA ( $5 \mu \mathrm { L }$ of $2 \mathrm { n g / \mu L }$ genomic DNA sample). To support the use of the new collection device, detection limit studies were provided in k152464 and referenced to support clearance of this submission.

e. Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device: A method comparison study was provided in k152464 and referenced to support this clearance.   
b. Matrix comparison: A matrix comparison study was provided in k152464 and referenced to support this clearance.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.   
b. Clinical specificity: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable):

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range: Not applicable for this device type.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.