# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K220026   
B Applicant Genomadix Inc.   
C Proprietary and Established Names Genomadix Cube CYP2C19 System   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>NTI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.3360 -Drug MetabolizingEnzyme GenotypingSystem</td><td rowspan=1 colspan=1>Toxicology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New Device   
B Measurand: Genotype of Cytochrome P450 2C19 (CYP2C19)   
C Type of Test: Genotyping microarray

# III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

The Genomadix Cube CYP2C19 System is a qualitative in vitro diagnostic test for the identification of a patient’s $\mathrm { C Y P 2 C } 1 9 ^ { * } 2 , ^ { * } 3$ , and $^ { * } 1 7$ genotypes determined from genomic DNA obtained from a buccal swab sample.

The Genomadix Cube CYP2C19 System can be used to aid clinicians in determining therapeutic strategy for therapeutics that are metabolized by the cytochrome P450 2C19 gene product, specifically $^ { * } 2 , { ^ * } 3$ , and $^ { * } 1 7$ alleles. This test is not intended to be used to predict drug response or non-response.

The Genomadix Cube CYP2C19 Test Kit is indicated for use with the Genomadix Cube CYP2C19 Platform.

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

The Genomadix Cube CYP2C19 System is not indicated for stand-alone diagnostic purposes. The information provided from this test may supplement decision making and should only be used in conjunction with routine monitoring by a physician. Clinicians should use professional judgment in the interpretation of results from this test. Results from this type of assay should not be used in predicting a patient's response to drugs for which the drug metabolizing enzyme activity of that allele, or the drug metabolic pathway, has not been clearly established.

# D Special Instrument Requirements:

Genomadix Cube CYP2C19 Platform

# IV Device/System Characteristics:

# A Device Description:

The Genomadix Cube CYP2C19 System is a sample-to-result DNA testing system with integrated DNA extraction and amplification. Genotypes are determined using Polymerase Chain Reaction (PCR) and fluorescent probe detection. The Genomadix Cube CYP2C19 System is comprised of the Genomadix Cube CYP2C19 Platform (Genomadix Cube, computer, and barcode scanner) and the Genomadix Cube CYP2C19 Test Kit (swabs and cartridges). The test is run on the Genomadix Cube CYP2C19 Platform.

The Genomadix Cube CYP2C19 Platform has the following components:

The Genomadix Cube is a thermal cycling instrument that automatically integrates extraction of DNA from the buccal sample, PCR amplification, fluorescence-based detection of CYP2C19 alleles, and genotype calling.   
The computer is a consumer-grade netbook computer that serves as the user interface for logging into the platform, inputting patient information, and starting the test.   
The barcode scanner is used to automatically scan barcode information on the pouch of the cartridge, which includes sample collection kit lot information and fields to manually enter in patient information.

The Genomadix Cube CYP2C19 Test Kit has the following components:

• Cartridges (20, one per pouch) contain the reagents needed for DNA extraction, PCR amplification, and detection of the CYP2C19 alleles, in three separate tubes specific for the $^ { * } 2 , { ^ * } 3$ , or $^ { * } 1 7$ alleles.   
• Swab pouches (20 pouches with 3 swabs per pouch) are used to collect buccal samples from the inside of a patient’s cheek and transfer it to the cartridge tubes. The label on the pouch contains fields to manually track information including patient ID, swab collector, date, and time.

The External Control cartridges determines if the Genomadix Cube is performing correctly. The internal controls are internal software-based controls that monitor performance and automatically inform the user of insufficient buccal material and if testing is running within acceptable performance limits. The External Control cartridges and the Genomadix Cube CYP2C19 System are sold separately from the Genomadix Cube CYP2C19 Test Kits.

# B Principle of Operation:

The Genomadix Cube CYP2C19 System utilized PCR amplification of target DNA and detection of the presence or absence of specific CYP2C19 alleles $( ^ { * } 2 , ^ { * } 3 , ^ { * } 1 7 )$ using fluorescent oligonucleotide probes. The Genomadix Cube CYP2C19 System makes use of a thermal cycling PCR program which combines the annealing and extension steps.

The System uses a sample obtained with a buccal swab. The operator collects the buccal swab samples from a patient, places the cartridge in the Genomadix Cube, inserts 3 swab tips into each of the 3 tubes in the cartridge, and starts the test. The test system then integrates and automates DNA extraction, PCR-based amplification of the target gene, detection of the $^ { * } 2 , { ^ * } 3$ , and $^ { * } 1 7$ alleles using fluorescent oligonucleotide probes by fluorescent signal detection and analysis, and genotype determination. Results are displayed on the screen as genotype calls. The system detects the CYP2C19 $^ { * 2 }$ , $^ { * 3 }$ , and $^ { * } 1 7$ genotypes in separate reagent tubes.

# C Instrument Description Information:

1. Instrument Name: Genomadix Cube CYP2C19 Platform

2. Specimen Identification:

Specimen information is manually entered into the platform software using the fields generated by scanning the barcode on the pouch containing the cartridge.

3. Specimen Sampling and Handling:

The system is automated. After each buccal swab is taken, it is directly placed into one of three separate tubes in the cartridge and the closed cartridge is loaded into the platform.

4. Calibration: The Genomadix Cube does not require any end user maintenance or calibration

5. Quality Control:

There are two internal controls (internal positive control (IPC) and positive system control (PSC)) that are run automatically during each test. The IPC ensures that sufficient buccal material was added and that the sample was correctly processed. The PSC assesses system performance to ensure proper function of the system. The sponsor also offers an External Control Cartridge (ECC) to ensure that the instrument is working properly.

Users are also instructed to follow local, state and federal guidelines by testing quality control samples.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Spartan Rx Cyp2c19 Test System B Predicate 510(k) Number(s): K123891 C Comparison with Predicate(s):

Due to an administrative error, the reference to “Test kits cannot be shipped” was erroneously included and it is now updated to “Samples cannot be shipped.”

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K220026</td><td colspan="1" rowspan="1">K123891</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Genomadix CubeCYP2C19 System</td><td colspan="1" rowspan="1">Spartan RX CYP2C19Test System</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">For use as an aid toclinicians indetermining therapeuticstrategies for</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">therapeuticsthat are metabolized bythe Cytochrome P4502C19 gene product, andthat are specificallyaffected by the*2, *3, and *17 alleles</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Target Gene</td><td colspan="1" rowspan="1">CYP4502C19 *2, *3,and *17</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Limitation</td><td colspan="1" rowspan="1">Not intended to be usedto predict drug responseor non-response</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Utilizes thermal cyclingand target DNAamplification;fluorescent probedetection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Assay signal results areinterpreted by asoftware program.Assay results areprovided as genotypecalls reported to the enduser in a report format.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">DNA from a buccalswab sample</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Genomadix Cube</td><td colspan="1" rowspan="1">Spartan Rx Analyzer</td></tr><tr><td colspan="1" rowspan="1">Sample Shipping</td><td colspan="1" rowspan="1">Samples cannot beshipped.</td><td colspan="1" rowspan="1">Utilizes a SampleTransport System toship samples attemperatures between -80°C and -20°C.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI EP25-A: Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline.   
CLSI EP07: Interference Testing in Clinical Chemistry-Third Edition.   
CLSI EP12-A2: User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline-Second Edition.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

Site-to-Site Reproducibility

The reproducibility study was conducted at 3 sites. Samples used for the study were buccal samples from 8 individuals. The same eight individuals were sampled at all 3 sites. The following genotypes were assessed and confirmed by bi-directional sequencing: $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 1 / { } ^ { * } 2$ , $^ { * } 1 / { } ^ { * } 3$ , $^ { * } 1 / { } ^ { * } 1 7$ , $^ { * } 1 7 / { } ^ { * } 1 7$ , $^ { * } 2 / { ^ { * } 1 } 7$ , $^ { * } 2 / { ^ { * } 2 }$ , and $^ { * } 2 / { ^ * } 3$ . Testing was conducted by 2 operators, twice per day, in duplicate, over 5 non-consecutive days, per site, for a total of 320 tests per site (8 subjects $_ { \textrm { X 2 } }$ operators $_ { \textrm { X 2 } }$ sessions x 2 replicates x 5 days). Reproducibility was assessed using 2 lots per site, for a total of 3 lots (site 1: Lot $\mathrm { A } / \mathrm { B }$ , site 2: Lot $\mathbf { B } / \mathbf { C }$ , site 3: Lot C/A).

A total of 960 tests were performed across the 3 sites. First pass data is the result of testing the first set of samples (comprised of 1 test per allele: $^ { * } 2 , { ^ * } 3$ , and $^ { * } 1 7$ , respectively) from an individual. If an inconclusive result was obtained on the first set of samples, the second set of samples from the same patient were used for a second pass test. The correct call rate in the first pass was $9 9 . 1 \%$ with a $9 5 \%$ lower confidence limit of $9 8 . 2 \%$ . There were 9 inconclusive calls in the first pass across three lots. The following are the results from the first pass:

<table><tr><td rowspan=1 colspan=8>Inter-Laboratory Reproducibility</td></tr><tr><td rowspan=2 colspan=1>Sampletype</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=6>First Pass</td></tr><tr><td rowspan=1 colspan=1>#Samplestested</td><td rowspan=1 colspan=1>#Inconclusivecalls</td><td rowspan=1 colspan=1>#Incorrectcalls</td><td rowspan=1 colspan=1>#Correctcalls</td><td rowspan=1 colspan=1>%Correctcall rate</td><td rowspan=1 colspan=1>95% 2-sided(lower)</td></tr><tr><td rowspan=4 colspan=1>*1/*1</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9</td></tr><tr><td rowspan=4 colspan=1>*1/*2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9</td></tr><tr><td rowspan=4 colspan=1>*1/*3</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>87.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>95.4</td></tr><tr><td rowspan=4 colspan=1>*1/*17</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.9</td></tr><tr><td rowspan=4 colspan=1>*17/*17</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>87.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>95.4</td></tr><tr><td rowspan=4 colspan=1>*2/*17</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>80.1</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>87.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>87.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>90.6</td></tr><tr><td rowspan=4 colspan=1>*2/*2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>87.1</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>95.4</td></tr><tr><td rowspan=4 colspan=1>*2/*3</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>87.1</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>95.4</td></tr><tr><td rowspan=1 colspan=2>Total samples(all sites, alllots)</td><td rowspan=1 colspan=1>960</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>951</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>98.2%</td></tr></table>

After the second pass testing there were 3 inconclusive results (2 with a $^ { * } 2 / { ^ { * } 1 7 }$ genotype and 1 with a $^ { * } 2 / { ^ * } 3$ genotype) and no incorrect calls. The correct call rate on the second pass was $9 9 . 7 \%$ with a $9 5 \%$ one-sided lower confidence limit of $9 9 . 1 \%$ .

Lot-to-lot Reproducibility

Lot-to-lot reproducibility was assessed as part of the site-to-site reproducibility study using 3 independent lots. After second pass testing of inconclusive genotype results, the correct call rates were $9 9 . 7 \%$ , $9 9 . 4 \%$ , and $100 \%$ for Lot A, Lot B and Lot C, respectively.

2. Linearity:

Not applicable

# 3. Analytical Specificity/Interference:

Interference from potential exogenous and endogenous substances was evaluated using buccal swab samples collected from individuals with the following genotypes: $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 2 / { ^ { * } 1 7 }$ , $^ { * } 1 7 / { } ^ { * } 1 7$ and $^ { * 3 }$ -containing ( $^ { * } 1 / { ^ * } 3$ or $^ { * } 2 / { ^ { * } 3 }$ ). Genotypes of all individuals were confirmed by bi-directional sequencing prior to initiation of the study. To test the impact of endogenous and exogenous substances, buccal swab samples were collected from individuals after they were exposed to the substance (no rinsing prior to sample collection is recommended per the instructions for use). Endogenous substances were tested through adding substances directly to the reaction or dipping swab in a blood/saliva mixture prior to testing.

The following exogenous and endogenous substances were tested under the following conditions:

Exogenous:

$\bigcirc$ Antiseptic mouthwash: Individuals rinsed $2 0 ~ \mathrm { m L }$ in mouth for 10 seconds   
$\bigcirc$ Baking Soda: Individuals rinsed $3 0 ~ \mathrm { m L }$ in mouth for 10 seconds   
o Chewing gum: Individual chewed 1 piece for 1 minute   
o Cough Syrup: Individual rinsed $3 0 ~ \mathrm { m L }$ in mouth for 10 seconds   
o Cranberry juice: Individual rinsed $3 0 ~ \mathrm { m L }$ in mouth for 10 seconds   
o Denture paste: Individual applied 3 strips on roof of mouth for 5 minutes before removing   
o Hard candy: Individual dissolved 1 piece fully in mouth   
$\bigcirc$ Horse meat: Individual chewed $1 5 \mathrm { g }$ for 10 seconds   
$\bigcirc$ Salt solution: Individual rinsed $3 0 ~ \mathrm { m L }$ in mouth for 10 seconds   
$\bigcirc$ Sugar solution: Individuals rinsed $3 0 ~ \mathrm { m L }$ in mouth for seconds   
$\bigcirc$ Tobacco smoking: Individual inhaled and exhaled 5 times   
$\bigcirc$ Toothpaste: Individual brushed with $1 . 9 \mathrm { c m }$ of toothpaste for 2 minutes

Endogenous:

$\bigcirc$ Bacteria: $\sim 9 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ cells were added to reaction $\bigcirc$ Whole blood: Swab was dipped in a $1 . 7 \%$ blood/saliva mixture

A total of 16 samples were tested for each of the 14 potential interfering substances (4 replicates of each genotype tested for a total of 224 samples tested). The first pass testing results showed that there were 4 inconclusive calls with the following substances: 2 inconclusive calls for toothpaste, 1 inconclusive call for sugar solution, and 1 inconclusive call from tobacco smoking. These 4 inconclusive calls were retested using the second sample set. The final result after the second pass retesting showed $100 \%$ correct calls.

4. Assay Reportable Range:

Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Sample stability studies were conducted to determine the stability of the buccal sample on the system swabs. It was determined that a sample, collected on the swab, can be stored for up to 20 hours at room temperature ( $1 8 { - } 2 5 ^ { \circ } \mathrm { C }$ , $30 \mathrm { - } 7 0 \%$ relative humidity) after collection. Swab stability studies were conducted by testing at extreme cases, outside the claimed range, for temperature, humidity, and duration, by exceeding those values by $2 1 0 \%$ .

# 6. Detection Limit:

The upper limit of detection (ULoD) and lower LoD (LLoD) were assessed by analyzing buccal swab samples collection from people with different genotypes $( ^ { * } 1 / { ^ * } 1 , * 1 / { ^ * } 2 , * 2 / { ^ * } 1 7$ , a $^ { * 3 }$ containing ( $^ { * } 1 / { ^ * } 3$ or $^ { * } 2 / { ^ { * } 3 }$ ), $^ { * } 1 / { } ^ { * } 1 7$ and $^ { * } 1 7 / ^ { * } 1 7$ ), as confirmed by bidirectional sequencing. The LLoD was assessed by using a single stroke (1 up-and-down stroke) or a single cheek touch, where the swab is pressed against the inside of the cheek without a stroke. The recommendations in the instructions for use (IFU) were also assessed (i.e., 3 upand-down strokes). The ULoD was assessed by swabbing 5 times (15 up-and-down strokes).

In the first pass, there were a total of 208 tests and 9 inconclusive calls (6 inconclusive calls from the cheek touch studies, 2 inconclusive calls from the IFU condition, and 1 inconclusive call from the 5x IFU condition). After the second pass testing there was 1 inconclusive call (for $^ { * 3 }$ containing genotype) in one of the cheek touch studies. The final call rate was $9 9 . 5 \%$ .

The amount of DNA expected was determined for the LLoD (cheek touch), IFU, and ULoD (5x IFU) conditions using 18 samples each, as shown in the table below.

<table><tr><td rowspan=1 colspan=1>cYanmaGellob</td><td rowspan=1 colspan=1>Swabbing condition</td><td rowspan=1 colspan=1>Total DNA Range (ng)</td></tr><tr><td rowspan=1 colspan=1>Lower LoD (LLoD)</td><td rowspan=1 colspan=1>Cheek Touch</td><td rowspan=1 colspan=1>2 - 30</td></tr><tr><td rowspan=1 colspan=1>IFUcondition</td><td rowspan=1 colspan=1>IFU</td><td rowspan=1 colspan=1>2 - 218</td></tr><tr><td rowspan=1 colspan=1>Upper LoD (ULoD)</td><td rowspan=1 colspan=1>5X IFU</td><td rowspan=1 colspan=1>10 - 248</td></tr></table>

Device performance at the lowest and highest expected DNA concentrations was assessed using gDNA collected from individuals with the following genotypes: $^ { * } 1 / { } ^ { * } 1$ , $^ { * } 1 / { } ^ { * } 2$ , $^ { * } 1 / { ^ { * } 3 }$ , $^ { * } 1 / { } ^ { * } 1 7$ , $^ { * } 2 / { ^ { * } 2 }$ , $^ { * } 3 / { ^ * } 3$ , and $^ { * } 1 7 / { } ^ { * } 1 7$ . Samples were either diluted to $1 . 0 \mathrm { n g }$ (to represent the lowest concentration expected from a sample) or not diluted and tested between a range of 316.7 to $3 6 1 . 9 \mathrm { n g }$ per reaction (to represent the highest concentration expected from a sample). For the low concentration, 20 replicates were used from the 7 individuals for a total of 140 samples. For the high concentration, 3 replicates were used from the 7 individuals for a total of 21 samples tested. One (1) inconclusive result was generated at the low concentration in the first pass which was resolved in the second pass. The final call rate for the LLoD and ULoD based on expected DNA was $100 \%$ for all genotypes tested.

7. Assay Cut-Off:

Not applicable

8. Accuracy (Instrument): Accuracy was established in the method comparison study (see below).

9. Carry-Over:

A total of 122 samples were tested. Sixty-one $^ { * } 1 / { } ^ { * } 1$ genotypes were tested alternately with non- $^ { * } 1 / { } ^ { * } 1$ genotypes (twenty-one $^ { * } 2 / { ^ { * } 1 } 7$ , twenty $^ { * } 2 / { ^ { * } 2 }$ and twenty $^ { * } 1 7 / ^ { * } 1 7$ ) on the same set of platforms. The results generated were $100 \%$ concordant results (122/122) after a second pass.

# B Comparison Studies:

# 1. Method Comparison with Predicate Device:

Method comparison testing was evaluated at 3 sites by 10 operators. Samples from 444 unique patients were collected, including saliva samples for bi-directional sequencing and swabs for testing on the Genomadix Cube CYP2C19 System. Of the 444 unique patient samples, 11 samples were excluded for low quality (Phred score ${ < } 3 0$ , as determined by bidirectional sequencing, which is used as the comparator method). The percent agreement $( \%$ correct call rate) for genotype detection of the Genomadix Cube CYP2C19 System was calculated by determining the percentage of tested samples with the correct genotype assigned compared to the total number of samples of that genotype. Two (2) incorrect calls and 17 inconclusive calls were identified, resulting in $9 6 \%$ correct call rate in the first pass. After second pass testing, 2 inconclusive calls remained, for a final correct call rate of $9 9 \%$ (429/433) and a $9 5 \%$ lower confidence interval of $9 8 \%$ . The 2 incorrect calls ( $^ { * } 1 / { ^ * } 2$ and $^ { * } 2 / { ^ { * } 2 } )$ ) were suspected to be a result of swapped samples.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>First Pass</td></tr><tr><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>#Samplestested</td><td rowspan=1 colspan=1>#Inconclusivecalls</td><td rowspan=1 colspan=1>#Incorrectcalls</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>%correct callrate</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>83</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>85</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>*3/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>433</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>414</td><td rowspan=1 colspan=1>96%</td></tr></table>

<table><tr><td rowspan=2 colspan=1>SampleType</td><td rowspan=1 colspan=6>Combined Call (after second pass retesting)</td></tr><tr><td rowspan=1 colspan=1>#Samplestested</td><td rowspan=1 colspan=1>#Inconclusivecalls</td><td rowspan=1 colspan=1>#Incorrectcalls</td><td rowspan=1 colspan=1>#correctcalls</td><td rowspan=1 colspan=1>% correctcall rate</td><td rowspan=1 colspan=1>95% lowerconfidenceinterval</td></tr><tr><td rowspan=1 colspan=1>*1/*1</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>*1/*2</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>*1/*3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>*1/*17</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>*2/*2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>*2/*3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>*2/*17</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87</td></tr><tr><td rowspan=1 colspan=1>*3/*3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>*3/*17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>*17/*17</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>433</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>429</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>98%</td></tr></table>

2. Matrix Comparison: Not applicable

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable

D Clinical Cut-Off: Not applicable   
E Expected Values/Reference Range: Not applicable   
F Other Supportive Instrument Performance Characteristics Data: Not applicable

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.